{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Comparison of ETL document analyzer libraries for Python\n",
    "\n",
    "The tools that take place in this analysis are the ones listed below.\n",
    "\n",
    "* **Fully commercial tools**\n",
    "    - [Reducto.ai](https://reducto.ai/)\n",
    "    - [Extend](https://www.extend.app/)\n",
    "* **Comercial tools with free tier**\n",
    "    - [Unstructured](https://unstructured.io/)\n",
    "    - [Datalab](https://www.datalab.to/) \n",
    "    - [Pymupdf](https://pymupdf.readthedocs.io/en/latest/) (The handling of office files requires the commercial extension of `PyMuPDFPro`. Does not seem to give anything new. RAG functionality available bbut very standard and not special. No good conversion from DOCX to MD)\n",
    "\n",
    "* **Full open-source tools**\n",
    "    - [MarkItDown](https://github.com/microsoft/markitdown) (Very simple to use and easy conversion to markdown format. Uses `python-docx` as engine. Possible integration with LLMs, but they only mention figure analysis.)\n",
    "    - [Docling](https://github.com/DS4SD/docling) (Very simple to use and easy conversion to markdown format and `html`. Uses `python-docx` as engine. Integration with LangChain framework.)\n",
    "    - [Everything2Markdown](https://github.com/wisupai/e2m)\n",
    "\n",
    "\n",
    "The analysis will be working on the following criteria:\n",
    "* Price (free, paid, free tier)\n",
    "* Latency time on conversion\n",
    "* Integration with other tools (LLMs, Langchain)\n",
    "* Supported file formats\n",
    "* Accuracy of data extraction using OpenAI (`Figure legends` and `Data availability` sections)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Notebook preparation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PyMuPDFPro: Trial Mode, 59 days 9 hours left.\n"
     ]
    }
   ],
   "source": [
    "from glob import glob\n",
    "import os\n",
    "from dotenv import load_dotenv\n",
    "\n",
    "load_dotenv()  # Load .env file\n",
    "import pymupdf.pro\n",
    "pymupdf.pro.unlock(os.getenv('PYMUPDF_KEY'))\n",
    "\n",
    "\n",
    "PDF_FILES = glob('./data/pdf/*')\n",
    "DOCX_FILES = glob('./data/docx/*')\n",
    "GROUND_TRUTH = glob('./data/ground_truth/*')\n",
    "\n",
    "\n",
    "# Organize the list of files making sure that the same file names are in the same index\n",
    "PDF_FILES.sort()\n",
    "DOCX_FILES.sort()\n",
    "GROUND_TRUTH.sort()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 0. Using `pypandoc` as the baseline\n",
    "\n",
    "This is the library that we are currently using in `soda-curation` to convert the documents to `html`.\n",
    "It gives us a good idea of the baseline time. This one also does not include any other functionalities.\n",
    "\n",
    "* `pypandoc` does not support `.pdf` files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'<p><em><strong>LINE-1 RNA triggers matrix formation in bone cells via a\\nPKR-mediated inflammatory response</strong></em></p>\\n<p>Arianna Mangiavacchi,<sup>1@</sup> Gabriele Morelli,<sup>1</sup> Sjur\\nReppe,<sup>2,3,4</sup> Alfonso Saera-Vila,<sup>5</sup> Peng\\nLiu,<sup>1</sup> Benjamin Eggerschwiler,<sup>6,7</sup> Huoming\\nZhang,<sup>8</sup> Dalila Bensaddek,<sup>8</sup> Elisa A.\\nCasanova,<sup>5</sup> Carolina Medina Gomez,<sup>9</sup> Vid\\nPrijatelj,<sup>9</sup> Francesco Della Valle,<sup>1</sup> Nazerke\\nAtinbayeva,<sup>1</sup> Juan Carlos Izpisua Belmonte,<sup>10</sup>\\nFernando Rivadeneira,<sup>9</sup> Paolo Cinelli,<sup>5,11</sup> Kaare\\nMorten Gautvik,<sup>3*</sup> Valerio Orlando.<sup>1*@</sup></p>\\n<p><sup>1</sup> King Abdullah University of Science and Technology\\n(KAUST), Biological Environmental Science and Engineering Division,\\nThuwal 23500-6900, Kingdom of Saudi Arabia\\u200b</p>\\n<p><sup>2</sup> Oslo University Hospital, Department of Medical\\nBiochemistry, Oslo, Norway.</p>\\n<p><sup>3</sup>Lovisenberg Diaconal Hospital, Unger-Vetlesen Institute,\\nOslo, Norway.</p>\\n<p><sup>4</sup>Oslo University Hospital, Department of Plastic and\\nReconstructive Surgery, Oslo, Norway.</p>\\n<p><sup>5</sup> Sequentia Biotech, Carrer Comte D\\'Urgell 240, Barcelona,\\n08036, Spain.</p>\\n<p><sup>6</sup> Department of Trauma, University Hospital Zurich,\\nSternwartstrasse 14, 8091, Zurich, Switzerland</p>\\n<p><sup>7</sup> Life Science Zurich Graduate School, University of\\nZurich, Winterthurerstrasse 190, 8057, Zurich, Switzerland.</p>\\n<p><sup>8</sup> Core Labs, King Abdullah University of Science and\\nTechnology (KAUST), Thuwal 23500-6900, Kingdom of Saudi Arabia\\u200b.</p>\\n<p><sup>9</sup> Department of Internal Medicine, Erasmus Medical Centre,\\nRotterdam, the Netherlands.</p>\\n<p><sup>10</sup> Altos Labs, San Diego, CA, USA</p>\\n<p><sup>11</sup> Center for Applied Biotechnology and Molecular\\nMedicine, University of Zurich, Winterthurerstrasse 190, 8057, Zurich,\\nSwitzerland.</p>\\n<p>*senior author</p>\\n<p><sup>@</sup>corresponding author</p>\\n<p><strong>Abstract</strong></p>\\n<p>Transposable elements (TEs) are mobile genetic modules of viral\\nderivation that have been co-opted to become modulators of mammalian\\ngene expression. TEs are a major source of endogenous dsRNAs, signaling\\nmolecules able to coordinate inflammatory responses in various\\nphysiological processes. Here, we provide evidence for a positive\\ninvolvement of TEs in inflammation-driven bone repair and\\nmineralization. In newly fractured mice bone, we observed an early\\ntransient upregulation of repeats occurring concurrently with the\\ninitiation of the inflammatory stage. In human bone biopsies, analysis\\nrevealed a significant correlation between repeats expression,\\nmechanical stress and bone mineral density. We investigated a potential\\nlink between LINE-1 (L1) expression and bone mineralization by\\ndelivering a synthetic L1 RNA to osteoporotic patient-derived\\nmesenchymal stem cells and observed a dsRNA-triggered protein kinase\\n(PKR)-mediated stress response that led to strongly increased\\nmineralization. This response was associated with a strong and transient\\ninflammation, accompanied by a global translation attenuation induced by\\neIF2α phosphorylation. We demonstrated that L1 transfection reshaped the\\nsecretory profile of osteoblasts, triggering a paracrine activity that\\nstimulated the mineralization of recipient cells.</p>\\n<p><strong>Keywords:</strong> dsRNA/inflammation/osteoblast\\n/PKR/transposable elements</p>\\n<p><strong>Introduction</strong></p>\\n<p>Repetitive elements are, in origin, parasitic transposable elements\\n(TEs) of viral derivation and during evolution have become regulatory\\nmodules of gene expression, eventually integrated into developmental,\\nresilience, and cell identity programs.(Cosby <em>et al</em>, 2019)\\n(Mangiavacchi <em>et al</em>, 2021) (Jachowicz <em>et al</em>, 2017)\\nAlthough cells have evolved several defense mechanisms to prevent\\ndeleterious uncontrolled retrotransposon reactivation,(Slotkin &amp;\\nMartienssen, 2007) increasing evidence demonstrates that\\nretrotransposons are also involved in non-pathological contexts,\\nparticularly as non-coding RNAs.(Mangiavacchi <em>et al</em>, 2021) As\\npart of resilience mechanisms, all retrotransposons are a major source\\nof endogenous double-stranded RNAs (dsRNAs), that serve as a cellular\\nsignaling molecule to coordinate inflammatory and immune responses in\\nvarious physiological processes.(Fukuda <em>et al</em>, 2021) (De Cecco\\n<em>et al</em>, 2019a) (Simon <em>et al</em>, 2019) (Sadeq <em>et\\nal</em>, 2021) (Chen &amp; Hur, 2022) (Zhang <em>et al</em>, 2020) The\\ntransient and controlled breaching of the activation threshold of dsRNA\\nsensors allows the integration of innate immune functions within the\\nframe of physiological processes.(Chen &amp; Hur, 2022) For example,\\nrepetitive elements transcribed during development drive RIG-I-like\\nreceptors (RLR)-mediated inflammatory signals that regulate\\nhematopoietic stem and progenitor cells (HSPC) formation.(Lefkopoulos\\n<em>et al</em>, 2020) During cell replication, dsRNA-mediated activation\\nof PKR ensures proper progression of mitosis.(Kim <em>et al</em>, 2014)\\nMoreover, epigenetically mediated derepression of repetitive elements\\nleads to dsRNA production and to the activation of an inflammatory\\nresponse stimulating anti-tumor T cell immunity.(Sheng <em>et al</em>,\\n2018) Furthermore, after skin injury, dsRNAs induce inflammatory\\npathways contributing to wound healing and hair regeneration.(Nelson\\n<em>et al</em>, 2015) Several studies have demonstrated a positive\\neffect of proinflammatory mediators on bone anabolism <em>in vitro</em>\\nand <em>in vivo</em>,(Croes <em>et al</em>, 2015) (Croes <em>et al</em>,\\n2016) (Li <em>et al</em>, 2016) (Croes <em>et al</em>, 2017) (Laroche\\n<em>et al</em>, 2011) suggesting that an inflammatory reaction may be\\nharnessed for bone regenerative purposes. Indeed, sterile inflammation\\nis a fundamental component of the osteogenic microenvironment: the\\ninflammatory reaction is the earliest response to injury and is crucial\\nto initiate and orchestrate fracture repair and activate bone anabolic\\nprocesses. (Mountziaris <em>et al</em>, 2011) (Marsell &amp; Einhorn,\\n2011) (Bahney <em>et al</em>, 2019) Accordingly, a dysregulated\\ninflammation negatively impacts optimal bone regeneration, (Wheatley\\n<em>et al</em>, 2019) (Gerstenfeld <em>et al</em>, 2001) (Recknagel\\n<em>et al</em>, 2013) (Hurtgen <em>et al</em>, 2016) (Jiao <em>et\\nal</em>, 2015) and often precedes the excessive bone formation\\nassociated with heterotopic ossification and several vertebral column\\npathological conditions. (Lories &amp; Schett, 2012) (Balboni <em>et\\nal</em>, 2006) Therefore, identifying the key molecular factors\\ntriggering the sterile inflammation that precedes regeneration in bone\\nwould allow us to modulate the inflammatory reaction in a spatiotemporal\\nand contextual manner as well as to develop new anabolic strategies for\\nthe treatment of bone loss conditions, such as osteoporosis, or impaired\\nbone repair.(Roberts &amp; Ke, 2018)</p>\\n<p>Here, we show that 1) In mice, immediately after bone fracture, a\\ntransient upregulation of TEs concurs with the initiation of the\\ninflammatory stage; 2) in humans, an increased expression of repeats is\\nobserved in bones with a higher mechanical stress-induced anabolic\\ndemand and expression of TEs correlates with local mineral density; 3)\\nThe delivery of L1 RNA, but not control RNA, to human bone\\nmarrow-derived mesenchymal stem cells committed to osteoblasts\\nstimulates a unique mineralizing phenotype in a dose-dependent manner;\\n4) L1-treated osteoblasts show upregulation of inflammatory genes and\\ntranscriptional changes characteristic for the earliest stage of bone\\nrepair; 5) cytoplasmic L1 RNA is sensed by dsRNA-activated protein\\nkinase (PKR), which mediates eIF2α phosphorylation and a global\\nattenuation of protein synthesis; 6) the inhibition of PKR prevents\\ninflammation, inhibits translation and induces mineralization caused by\\nelevated levels of cytoplasmic L1 RNA; 7) cells transfected with L1 RNA\\nundergo significant changes in their secretome composition and initiate\\na paracrine effect stimulating the mineralization of osteogenic\\ncompetent recipient cells.</p>\\n<p>Our results indicate that TEs are induced by bone damage and\\nstimulate bone mineralization through an inflammatory response triggered\\nby dsRNA sensing and mediated by a paracrine activity.</p>\\n<p><strong>Results</strong></p>\\n<p><strong>Retrotransposon expression is triggered by fracture <em>in\\nvivo</em> and correlates with bone mineral density in human\\nweight-bearing bones</strong></p>\\n<p>Inflammation is the earliest response in fracture healing. To\\ninvestigate a possible involvement of TEs in the sterile inflammatory\\nresponse triggered by bone injury, we analyzed TEs expression in\\ntime-course RNA-Seq repository data obtained from a full-fracture bone\\nhealing mice model.(Coates <em>et al</em>, 2019) We focused on TEs\\nexpression dynamics immediately after fracture (acute inflammatory\\nstage) and observed four clusters of differentially expressed TEs (logFC\\n&gt; 0.5) between intact and post-fracture bone after 4h, a time point\\ncorresponding to the initiation of acute inflammation (Figure 1A).\\nCluster 1 includes TEs subfamilies whose expression slightly increases\\nafter fracture and during the whole healing process. Clusters 2 and 4\\ninclude TEs subfamilies that are downregulated after fracture but show\\nincreased expression at later time points. The major group, cluster 3,\\nis represented by TEs subfamilies whose expression is highly and\\ntransiently upregulated after fracture (Figure 1A and 1B).\\nInterestingly, the induction of these TEs, mostly Long Interspersed\\nNuclear Elements (LINEs) and Long Terminal Repeats (LTRs) (Figure 1B),\\nis transient and limited to the earliest phase of the inflammatory stage\\nnecessary and sufficient to initiate and orchestrate fracture repair and\\nactivate bone anabolic processes.(Mountziaris <em>et al</em>,\\n2011)<sup>,</sup>(Marsell &amp; Einhorn, 2011) These results suggest\\nthat, in bone, retrotransposon reactivation is an early event in\\nresponse to fracture, possibly linked to inflammation initiation which\\nis fundamental for a healthy healing. Indeed, retrotransposon\\ninvolvement in posttraumatic tissue regeneration was previously found in\\nother organisms.(Mashanov <em>et al</em>, 2012)<sup>,</sup>(Zhu <em>et\\nal</em>, 2012)</p>\\n<p>To assess gene expression dynamics immediately after fracture in\\nhumans represents a technical and ethical hurdle. A comparison between\\nbones experiencing different degrees of mechanical loading may represent\\nan alternative model to study stress-induced bone anabolism in humans.\\nMechanically loaded bones like the femur, which are subject to recurring\\nmicrofractures that need to be repaired, are indicated to be more\\nmetabolically active than less loaded bones, like the iliac crest, as\\npointed out by previous transcriptome studies.(Aerssens <em>et al</em>,\\n1997) (Varanasi <em>et al</em>, 2010)</p>\\n<p>We compared the expression of TEs in trabecular bone from ilium and\\nfemoral head, two different skeletal sites, the latter experiencing a\\nhigher degree of mechanical loading, anabolic demand and bone\\nturnover.(Aerssens <em>et al</em>, 1997) Femur (n=48) and ilium biopsies\\n(n=71) were isolated from a cohort of clinically well-characterized\\ndonors (refer to cohort description and Appendix Table S1 and S2). Each\\ncohort was divided into normal (BMD T-score &gt;-1), osteopenic (−2.5\\n&lt; BMD T-score ≤ −1), and osteoporotic (BMD T-score ≤ −2.5, with at\\nleast one fragility fracture). TEs show generally higher upregulation in\\nhealthy femurs (n=27) compared to healthy ilia (n=34) (Figure 1C and\\n1E). Each TEs order showed a high percentage of upregulated subfamilies,\\nfrom 60% of DNA transposon to 75% of LINEs, and a low percentage of\\ndownregulated subfamilies, from 9% of LTRs to 3% of Short interspersed\\nnuclear elements (SINEs) and LINEs, in femur compared to ilium (Figure\\n1D). These results suggested a positive correlation between TEs global\\nexpression and bone anabolic activity.</p>\\n<p>To further test our hypothesis, we compared TEs expression in femurs\\nfrom healthy donors (n=27), osteopenic (n=12) and osteoporotic patients\\n(n=9) whose bone anabolism is increasingly compromised. As shown in\\nFigure 2A, the global expression of TEs is reduced in patients. To\\nfurther assess whether TEs expression is involved in local anabolism and\\nassociated with mineral density, we divided the femur cohort according\\nto the DXA T-score measured in the femoral neck (FN). We divided the\\ncohort into “high BMD” (FN T-score &gt;-1) and “low BMD” (FN T-score\\n&lt;-1). Globally, TEs are upregulated in femurs with high bone mineral\\ndensity (Figure 2B). In particular, more than 90% of differentially\\nexpressed TEs subfamilies are upregulated (Figure 2C). Interestingly,\\nalmost all SINE elements are unchanged between the two groups (Figure\\n2C). Finally, we assessed whether a positive correlation exists between\\nTEs expression levels and local mineral density (FN T-score). We found\\nthat approximately 30% of L1 and LTR subfamilies expressed in the\\nfemoral bone are positively correlated with local BMD (Figure 2D).\\nAltogether, these results strongly suggest a positive involvement of TEs\\nin response to stress-induced anabolic demand and bone\\nmineralization.</p>\\n<p><strong>Increased cytoplasmic L1 repeat RNA stimulates osteoblast\\nmineralization activity</strong></p>\\n<p>The <em>in vivo</em> evidence suggested a positive involvement of\\nretrotransposons in post-traumatic bone repair in mice, and a strong\\ncorrelation between retrotransposon expression and bone mineral density\\nin humans, and higher expression in loaded compared to unloaded bone. We\\nthen corroborated the results by studying the effects of increasing\\nrepeat RNA levels on differentiating osteoblasts derived from\\nmesenchymal stem cells (MSCs) isolated from femurs of healthy donors. At\\nday 5 of differentiation, a Cy5-conjugated full-length L1 RNA consensus\\nsequence was transfected. Capping, 2’-O-Methylation of 5’ end,\\npolyadenylation (200 adenosines), full substitution with\\n5-methylcytidine (m5C) and 75% substitution with pseudouridine were used\\nto stabilize the RNA and to bypass the intracellular innate immune\\nsystem.(Koski <em>et al</em>, 2004) (Pardi <em>et al</em>, 2013) (Ludwig\\n<em>et al</em>, 2010) (Karikó <em>et al</em>, 2005) (Nallagatla &amp;\\nBevilacqua, 2008) (Karikó <em>et al</em>, 2008) (Anderson <em>et\\nal</em>, 2010) (Kormann <em>et al</em>, 2011) Red fluorescent protein\\n(RFP) mRNA, with the same modifications, was included as negative\\ncontrol. The exogenous L1 RNA accumulated in the cytoplasm (Figure 3A)\\nand was gradually cleared by secretion along with matrix components\\n(Figure 3B and 3C). The effect of L1 RNA delivery was tested in\\ndifferentiating MSCs from two different and unrelated healthy donors. As\\nshown, L1 RNA transfection markedly stimulated the production of\\nmineralized matrix (Figure 3D). Osteoblast mineralization increased\\ndose-dependently with the concentration of the delivered L1 RNA, in\\nagreement with saturation kinetics (Figure 3E). Moreover, the strongly\\nenhanced mineralization was highly specific for L1 RNA, as demonstrated\\nby the lack of significant changes in mineral deposition in cells\\ntransfected with negative control RNA, even at concentrations\\n100-200-fold higher (figure 3E). We also isolated MSCs from the femur of\\nosteoporotic patients and selected those showing a markedly delayed and\\nreduced <em>in vitro</em> production of bone matrix in contrast to those\\nderived from healthy donors (Figure 3F). Strikingly, also in these cells\\nwith low osteogenic capacity, the delivery of L1 RNA triggered a strong\\nmineralization response despite their clearly compromised anabolic\\nactivity (Figure 3G). These results demonstrate convincingly that the\\nectopic delivery of L1 RNA stimulates the mineralization of bone-forming\\ncells regardless of their prior intrinsic differentiation/anabolic\\npotential.</p>\\n<p>Osteoporotic patient-derived cells transfected with L1 RNA show a\\nhigher expression of early differentiation markers, with a peak 48h\\nafter L1 transfection (day 7) (Figure 3H). Bone sialoprotein\\n(<em>IBSP</em>), a late osteogenic gene, shows a similar profile, being\\nalmost 6 times more expressed in L1- than in RFP-transfected cells on\\nday 7 (Appendix Figure S1A). Strikingly, although alkaline phosphatase\\n(<em>ALPL</em>) expression slightly increases at later time points\\n(Appendix Figure S1A), its activity is not enhanced in L1-treated cells\\n(Appendix Figure S2A). However, the activity of ectonucleotide\\npyrophosphatase/phosphodiesterase 1 (ENPP1) is gradually and\\nsignificantly reduced by L1 RNA transfection (Appendix Figure S2B). As\\nENPP1 is a major source of inorganic pyrophosphate (PPi), one of the\\nmain physiological inhibitors of mineralization,(FLEISCH <em>et al</em>,\\n1966) (Johnson <em>et al</em>, 2000) (Hessle <em>et al</em>, 2002) the\\nreduction of its activity may contribute to the enhanced mineralization\\nobserved in L1-RNA transfected cells.</p>\\n<p>Altogether, these data suggest that L1 RNA induces a unique\\nmineralizing phenotype and apparently distinct from the well-known\\ncanonical differentiation mechanisms.</p>\\n<p><strong>L1 RNA delivery induces inflammatory pathways significantly\\noverlapping those involved in bone fracture repair</strong></p>\\n<p>To investigate the molecular response of osteoblasts to L1 RNA\\ntransfection, we profiled their transcriptome by Illumina RNA-Seq 24h\\npost-transfection. We found 482 differentially expressed genes (DEG)\\n(FDR&lt;0.05) between L1 and RFP RNA-treated OB (268 upregulated and 214\\ndownregulated) (Appendix Figure S3). Gene ontology (GO) enrichment\\nanalysis revealed that the early transcriptional signature of L1-treated\\nosteoblasts is typical of that of an inflammatory response (Figure 4A\\nand 4B). This is in accordance with the recent evidence of a link\\nbetween L1 and other retrotransposons accumulation in the cytoplasm and\\ninflammatory response <em>in vitro</em> and <em>in vivo</em>.(Fukuda\\n<em>et al</em>, 2021) (De Cecco <em>et al</em>, 2019b) (Simon <em>et\\nal</em>, 2019) GO term enrichment analysis comparison between the early\\nstage of fracture-induced bone healing <em>in vivo</em>(Coates <em>et\\nal</em>, 2019) and differentiating osteoblasts <em>in vitro</em> 24h\\npost-L1 delivery (Figure 4A) revealed a matching inflammatory response\\nand shared upregulated pathways (Figure 4C) essential for effective bone\\nrepair.(Maruyama <em>et al</em>, 2020) (Gerstenfeld <em>et al</em>,\\n2003) (Kon <em>et al</em>, 2001) Recruitment of neutrophils at the site\\nof injury also indicates a healthy repair process, particularly in\\nbone.(Kovtun <em>et al</em>, 2016) Extracellular signal-regulated kinase\\n1 (ERK1) and 2 (ERK2), play crucial roles in bone formation(Xiao <em>et\\nal</em>, 2000) (Xiao <em>et al</em>, 2002b) (Xiao <em>et al</em>, 2002a)\\n(Matsushita <em>et al</em>, 2009) (Ge <em>et al</em>, 2007) and seem to\\nbe the most upstream initiators of tissue regeneration in\\nplanaria.(Owlarn <em>et al</em>, 2017) Upon bone injury, an initial\\ntransient stage of acute inflammation is a crucial factor in ensuring an\\neffective regeneration(Maruyama <em>et al</em>, 2020) whereas an\\nexcessive/prolonged (chronic) inflammation is deleterious for the\\nhealing environment.(Chan <em>et al</em>, 2012) (Hurtgen <em>et al</em>,\\n2016) (Clark <em>et al</em>, 2017) (Brem &amp; Tomic-Canic, 2007)\\n(Timmen <em>et al</em>, 2014) (Schmidt-Bleek <em>et al</em>, 2012)\\n(Weckbach <em>et al</em>, 2013) We followed the time-course expression\\nof “inflammatory response” (GO:0006954) and “immune response”\\n(GO:0006955) genes upregulated by L1 RNA immediately after transfection.\\nAs shown in Figure 4D, their expression is already strongly reduced at\\nday 10 and almost entirely silenced by day 14, suggesting the transient\\ndynamics of the inflammatory response triggered <em>in vitro</em> by L1\\nRNA. This is coherent with our previous observation that exogenous L1\\nRNA is gradually secreted by the cell together with matrix components\\n(Figure 3B and C), eventually clearing out the initial inflammatory\\nstimulus.</p>\\n<p><strong>Cellular response to increased cytoplasmic level of L1 RNA is\\nmediated by PKR</strong></p>\\n<p>It has been previously demonstrated that the cytoplasmic accumulation\\nof L1-derived cDNA triggers an inflammatory response mediated by the\\ncyclic GMP-AMP synthase (cGAS)-stimulator of interferon genes (STING)\\ncytosolic DNA sensing pathway.(De Cecco <em>et al</em>, 2019a)\\nImmunofluorescence (IF) analysis reveals that the exogenous L1 RNA forms\\nDNA:RNA hybrids (Appendix Figure S4), suggesting that L1 RNA is reversed\\ntranscribed into cDNA after its transfection. Notably, ORF1p was\\nundetectable by WB in differentiating osteoblasts and in L1 transfected\\nosteoblasts (Appendix Figure S5), indicating that the exogenous L1 RNA\\nis not translated after transfection. To assess whether L1-induced\\ninflammation was triggered by the sensing of its cDNA, we used\\nLamivudine 3TC and G140 (Figure 5A) to inhibit the ORF2-mediated reverse\\ntranscription of L1 RNA and cGAS activity, respectively. Moreover, we\\nalso performed a siRNA-mediated knockdown (KD) of cGAS (Appendix Figure\\nS6). (Jones <em>et al</em>, 2008) (Lama <em>et al</em>, 2019)\\nSurprisingly, none of the treatments impaired the mineralization induced\\nby L1 RNA delivery (Figure 5A, Appendix Figure S6C), suggesting that the\\nobserved cellular response depends entirely on RNA sensing. As\\npreviously mentioned, repeats are a major source of endogenous\\ndsRNA,(Sadeq <em>et al</em>, 2021) (Chen &amp; Hur, 2022) and L1 RNA, in\\nparticular, is able to form intramolecular double-stranded\\nstructures.(Hur, 2019) As expected, we found a tight colocalization\\nbetween L1 RNA and dsRNA IF signal (Figure 5B). One of the main cellular\\nsensors of dsRNA, including those derived from retrotransposons, is\\nPKR.(Kim <em>et al</em>, 2018) PKR activation results in the\\nphosphorylation of eukaryotic translation initiation factor 2α (eIF2α)\\nand in the subsequent inhibition of global protein synthesis and cell\\ngrowth.(Donnelly <em>et al</em>, 2013) Interestingly, PKR has been shown\\nto interact with L1 RNA <em>in vitro</em>.(Kim <em>et al</em>, 2018)\\nMoreover, eIF2α phosphorylation has been demonstrated to promote\\nautophagy in osteoblasts and to counteract BMD reduction in osteoporotic\\novariectomized mice.(Li <em>et al</em>, 2019)</p>\\n<p>To demonstrate that L1 RNA is able to trigger a PKR-mediated\\nintracellular response, we transfected L1 RNA in the presence or absence\\nof the PKR inhibitor C16 and evaluated the levels of eIF2α\\nphosphorylation by western blot (Figure 5C). The delivery of L1, but not\\nRFP, strongly induces eIF2α phosphorylation and this effect is prevented\\nby PKR inhibition (Figure 5C). Notably, eIF2α phosphorylation occurs\\nalready 6 hours post-transfection. This observation suggests that PKR\\nactivation is one of the first events induced by L1 RNA, thus\\ncorroborating the hypothesis of PKR as an endogenous L1 RNA sensor.</p>\\n<p>We then characterized the proteome of cells 24h post-L1 RNA\\ntransfection by diaPASEF mass spectrometry (MS).(Meier <em>et al</em>,\\n2020a) We detected almost 8000 proteins in each sample, and observed\\nthat more than 88% of significantly differentially expressed proteins\\n(DEP) were downregulated in L1-treated cells (Figure 5D, left panel).\\nMoreover, among the top 15 downregulated biological processes and\\ncellular components were translation and ribosomal subunits (Figure 5D,\\nright panels, yellow circles), indicating that L1 RNA induces a global\\nattenuation of protein synthesis. Indeed, a cap-dependent translation\\nshutdown is the main consequence of eIF2α phosphorylation.(Donnelly\\n<em>et al</em>, 2013) Finally, we show that neither PKR KD (Appendix\\nFigure S6A, B) nor its pharmacological inhibition by C16 treatment\\n(Figure 5E and 5F) prevents L1 RNA-triggered induction of inflammatory\\ngenes (Figure 5E) and mineral matrix deposition (Figure 5F, Appendix\\nFigure S6B). As previously mentioned, a global reduction of protein\\nsynthesis occurs 24h after L1 transfection and only a little percentage\\nof DEP is upregulated in L1 compared to RFP control (Figure 5D).\\nInterestingly, some of the most upregulated proteins are involved in\\nautophagy and vesicle trafficking (Figure 5G). Autophagy is a crucial\\nprocess for mineralization and bone homeostasis both <em>in vitro</em>\\nand <em>in vivo</em>,(Nollet <em>et al</em>, 2014) as autophagic\\nvacuoles are exploited to secrete apatite crystals.</p>\\n<p>Altogether these data demonstrate that the PKR-mediated sensing of\\nL1-derived dsRNA globally attenuates the translation via eIF2α\\nphosphorylation and indicate autophagy as a possible mechanism involved\\nin L1-induced osteoblasts mineralization.</p>\\n<p><strong>L1 RNA delivery reshapes the secretory profile of\\ndifferentiating osteoblasts</strong></p>\\n<p>An inflammatory response translates into a strong secretory activity\\nand production of paracrine signals, and we therefore tested the effect\\nof “L1-primed” osteoblast-derived conditioned medium in recipient\\ndifferentiating osteoblasts (Figure 6A). We found that the “secretome”\\nof cells transfected with L1 RNA has a significant paracrine effect on\\nrecipient osteoblasts since they showed an earlier formation of\\nmineralized nodules, already 24h post the addition of conditioned media\\n(Figure 6A, right panel), and higher deposition of mineral matrix at day\\n10 (Figure 6A, left panel). We subsequently isolated bulk and\\nexosome-derived proteomes from conditioned media of untreated, RFP- and\\nL1-treated differentiating osteoblasts and characterized them by MS.\\nDifferential expression analysis of MS data revealed a unique secretome\\nprofile (for both bulk and vesicular proteomes) of L1-primed osteoblasts\\ncompared to RFP-primed and untreated osteoblasts (Figure 6B). Gene\\nOntology (GO) analysis of differentially expressed proteins showed an\\nenrichment of pro-inflammatory factors (i.e. interleukins and\\nchemokines) involved in immune response and chemotactic migration of\\nimmune cells (Figure 6C and D), crucial processes for tissue repair\\nmechanisms <em>in vivo</em>. Pro-inflammatory molecules are also a major\\nconstituent of the senescence-associated secretory phenotype\\n(SASP),(Coppé <em>et al</em>, 2010) whose transient delivery supports\\ncellular plasticity and tissue regeneration.(Ritschka <em>et al</em>,\\n2017) Notably, the most enriched protein in L1-specific bulk secretome\\nis interleukin 8 (IL-8), an inflammatory chemokine involved in several\\nregenerative processes, such as skin wound healing,(JIANG <em>et\\nal</em>, 2012) therapeutic angiogenesis after stroke and ischemia,(Choi\\n<em>et al</em>, 2021) (Hou <em>et al</em>, 2014) and osteochondral bone\\nrepair.(Lin <em>et al</em>, 2019) (Yang <em>et al</em>, 2018a) Tumor\\nnecrosis factor-inducible gene 6 (TSG6) is among the top 10 proteins\\nenriched in the bulk secretome of L1-primed osteoblasts compared to RFP.\\nTSG6 is an inflammatory factor with suggestive therapeutic effects in\\ncorneal wounds, myocardial infarction, injured central nervous system,\\nchronic liver damage, and intervertebral disc degeneration.(Zhang <em>et\\nal</em>, 2013) (Lee <em>et al</em>, 2015) (Lee <em>et al</em>, 2009)\\n(Wang <em>et al</em>, 2017) (Yang <em>et al</em>, 2018b) Moreover, TSG6\\ninduces autophagy influx both <em>in vivo</em> and <em>in\\nvitro.</em>(Wang <em>et al</em>, 2017) Another protein positively\\ninvolved in autophagy and found specifically in the secretome of L1\\nprimed osteoblasts was ATG7 (autophagy-related 7). Osteoblast-specific\\nATG7 conditional knockout in mice reduced bone mass during developmental\\nand in adult age.(Li <em>et al</em>, 2018) Type 1 lysophosphatidic acid\\nreceptor (LPAR1) is also specifically secreted by L1-primed osteoblasts\\nand has proven to be positively involved in bone mineralization <em>in\\nvitro</em> and <em>in vivo</em>.(Alioli <em>et al</em>, 2020) (Gennero\\n<em>et al</em>, 2011)</p>\\n<p>Altogether, our findings demonstrate that cytoplasmic L1 RNA is\\nsensed by PKR, whose activation leads to eIF2α phosphorylation with\\nconsequent reprogramming of transcription and translation. The ultimate\\nconsequence of this L1-induced, PKR-mediated stress response in\\nosteoblasts is a major reprogramming of their secretory activity,\\nleading to a paracrine effect on recipient osteogenic competent cells to\\ntrigger their mineralization activity.</p>\\n<p><strong>Discussion</strong></p>\\n<p>The concerted co-option of TEs markedly changed whole regulatory\\nnetworks and integrated new functions into the eukaryote genome.(Cosby\\n<em>et al</em>, 2019) (Mangiavacchi <em>et al</em>, 2021) (Carelli\\n<em>et al</em>, 2022) However, while the role of TEs as evolutionary\\ndrivers is today well-established, their contribution to cell\\nphysiology, particularly in somatic cells, remains to be elucidated.\\nIndeed, aside from their role in the nucleus, (Della Valle <em>et\\nal</em>, 2022) (Jachowicz <em>et al</em>, 2017) (Lu <em>et al</em>,\\n2021) (Percharde <em>et al</em>, 2018) TEs represent a major source of\\nendogenous dsRNA whose role in triggering inflammatory and innate immune\\nresponses in various physiological processes has increasingly gained\\nattention.(Sadeq <em>et al</em>, 2021) (Chen &amp; Hur, 2022) For\\ninstance, an early acute inflammatory response is of vital importance in\\ninitiating and orchestrating bone repair and induce anabolism leading to\\nfracture healing. (Mountziaris <em>et al</em>, 2011) (Marsell &amp;\\nEinhorn, 2011) (Bahney <em>et al</em>, 2019) Moreover, several studies\\ndemonstrated a positive effect of proinflammatory mediators on bone\\nmineralization <em>in vitro</em> and <em>in vivo</em>,(Croes <em>et\\nal</em>, 2015) (Croes <em>et al</em>, 2016) (Li <em>et al</em>, 2016)\\n(Croes <em>et al</em>, 2017) indicating that the modulation of\\ninflammatory reaction is a tool to pursue regenerative strategies in\\nbone.(Roberts &amp; Ke, 2018) We found that, in the bone of newly\\nfractured mice, a group of TEs, mostly LINE and LTR, was early, markedly\\nand transiently upregulated. Interestingly, the expression of these TEs\\nwas limited to the earliest phase of the inflammatory stage. In line\\nwith the hypothesis of a positive correlation between stress-induced\\nbone mineralization and TEs activation, the expression dynamics in human\\nbone biopsies revealed a significant upregulation of TEs in the femur, a\\nbone subjected to high mechanical stresses, high turnover and anabolism,\\ncompared to ilium, not weight bearing. Furthermore, we detected a lower\\nTEs expression in donors with compromised bone mineralization and a\\nsignificant correlation between TEs expression and local BMD. We\\nhypothesized that the expression dynamics of TEs in fracture/mechanical\\nstress observed <em>in vivo</em> could mimic the immunological threshold\\nmodel of the sterile activation of dsRNA sensors as a response to\\nstressful conditions. Indeed, the transient and controlled breaching of\\nthe activation threshold of dsRNA sensors may lead to sterile\\ninflammation being integrated into physiological processes.(Chen &amp;\\nHur, 2022) To test further this hypothesis, we delivered a full-length\\nL1 consensus sequence as source of dsRNA(Hur, 2019) to differentiating\\nosteoblasts <em>in vitro</em> and showed that L1 RNA initially\\naccumulated in the cytoplasm and is gradually cleared by secretion. The\\nincreased cytoplasmic levels of L1 RNA are immediately sensed by PKR, a\\ncytoplasmic dsRNA sensor whose activation induces the phosphorylation of\\neIF2α(Κιμ\\xa0<em>ετ\\xa0αλ</em>,\\xa02018)\\xa0(∆οννελλψ\\xa0<em>ετ\\xa0αλ</em>,\\xa02013), leading\\nto the attenuation of global protein synthesis (Donnelly <em>et al</em>,\\n2013), induction of inflammatory response, and stimulation of mineral\\nmatrix deposition. In support of the evidence that PKR is the early\\nsensor of increased L1 RNA cytoplasmic levels, we show that inhibition\\nof PKR activation by C16 prevents L1-induced eIF2α\\xa0phosphorylation,\\ninduction of inflammatory genes, and mineral matrix deposition.\\nMoreover, the inhibition of L1 cDNA formation and sensing by Lamivudine\\n3TC, G140, or cGAS KD does not affect the L1-induced phenotype. Thus,\\nthe evidence does not support that the L1-triggered inflammation is\\ncausally connected to a cGAS-STING-dependent sensing of cytosolic L1\\ncDNA, a mechanism demonstrated operative in senescence.(De Cecco <em>et\\nal</em>, 2019a)</p>\\n<p>L1 RNA response showed high specificity, as demonstrated by the\\ndose-dependent increase of mineralization and the lack of significant\\nchanges in mineral deposition when negative control RNA was transfected,\\neven at 100-200-fold higher concentration. Although the molecular\\nmechanism behind L1-induced stress response which stimulates the\\nmineralization activity of osteoblasts remains elusive, our results\\nindicate that L1 stimulates the mineral matrix deposition by\\nbone-forming cells regardless of their prior intrinsic anabolic\\npotential. Moreover, MS results suggested that increased autophagy, a\\nprocess crucial for mineralization and bone homeostasis <em>in\\nvitro</em> and <em>in vivo</em>(Nollet <em>et al</em>, 2014) may\\ncontribute to L1-induced hydroxyapatite deposition. Intriguingly, the\\nosteoinductive properties of L1-primed osteoblast-derived secretome and\\nthe observation that L1 RNA was gradually secreted by the cells may\\nsuggest an involvement of L1 RNA in cell-to-cell communication, in line\\nwith the recent finding of other retrotransposons (i.e. ERV) acting as\\nparacrine molecules.(Liu <em>et al</em>, 2021) This, together with the\\nobserved L1-dependent induction of CCL and CXCL chemokines and\\ncytokines, being chemoattractants of immune cells and endothelial\\ncells,(Bischoff <em>et al</em>, 2011) (Eash <em>et al</em>, 2010)\\n(Strieter <em>et al</em>, 2005) and the composition of L1-primed\\nosteoblast secretome, makes the <em>in vivo</em> delivery of L1 RNA a\\nnecessary step to evaluate its actual contribution to the paracrine\\nprocesses that orchestrate bone repair mechanisms. (Chaparro &amp;\\nLinero, 2016) (Nimiritsky <em>et al</em>, 2019)</p>\\n<p>The identification of repeats RNA, particularly L1, as potential\\nresilient molecular factors involved in stress-induced,\\ninflammation-mediated bone production points out new avenues to develop\\nanabolic strategies for the treatment of bone loss conditions, such as\\nosteoporosis or impaired bone repair.(Roberts &amp; Ke, 2018) On the\\nother hand, repeats may be novel targets in the treatment of those\\nconditions characterized by excessive or heterotopic ossification\\npreceded by deregulated/chronic inflammation.(Lories &amp; Schett, 2012)\\n(Balboni <em>et al</em>, 2006)</p>\\n<p><strong>Materials and Methods</strong></p>\\n<p><strong>Participants and ethics</strong></p>\\n<p>Femoral bone biopsies were obtained from the caput region of\\npostmenopausal women or men with a wide BMD range, i.e., from healthy to\\nosteoporotic, who were undergoing hip replacement surgery due to\\nosteoarthritis or fracture at Lovisenberg Diaconal Hospital (Oslo,\\nNorway) or Diakonhiemmet Hospital (Oslo, Norway), respectively. The\\ndonors of femoral bones are listed in Appendix Table S1. The\\npostmenopausal iliac bone donors were recruited from the outpatient\\nclinic of Lovisenberg Diaconal Hospital (Oslo, Norway). Candidates\\nfilled out a questionnaire that included medication and lifestyle\\nfactors, and selected donors were deemed representative of the\\nOslo-based Norwegian ethnic female population aged 50 to 86\\u2009years. The\\niliac bone donors are listed in Appendix Table S2 and have been\\ndescribed previously in detail.(Reppe <em>et al</em>, 2010) All donors\\ntaking medication or having diseases, other than primary osteoporosis,\\nknown to affect bone metabolism were excluded. The presence of\\nbone-impairing diseases/conditions was excluded by extensive biochemical\\nserum and urine analyses supported by X-ray examinations. The\\nsite-specific BMD of all donors was evaluated with Lunar Prodigy DEXA\\n(GE Lunar, Madison, WI, USA) following the manufacturer\\'s instructions.\\nThe precision of the instrument for measuring the lumbar spine\\n(L<sub>2</sub>–L<sub>4</sub>) and hip BMD was 1.7% and 1.1%,\\nrespectively. The study was approved by the Norwegian Regional Ethical\\nCommittee (REK no 2010/2539, Norway), all volunteers gave their written\\ninformed consent, and sampling and procedures were according to the Act\\nof Biobanking in Norway.</p>\\n<p><strong>Cell culture</strong></p>\\n<p>For L1 RNA transfection experiments, human MSCs were isolated from\\nosteoporotic femoral heads with the consent of the patient according to\\nSwiss (BASEC-Nr. 2016-02159) ethical guidelines. Femoral heads were\\ncrushed and incubated in DMEM (PAN Biotech, Germany, Cat. No. P04-03550)\\nsupplemented with 1% (v/v) 100× Penicillin-Streptomycin Solution\\n(Biowest, France, Cat. No. L0022), 10% (v/v) FCS (Biowest, France, Cat.\\nNo. S181S), 1% (v/v) 200mM L-Glutamine solution (Sigma, USA, Cat. No.\\nG7513), 5ng/ml FGF-2 (Sigma, USA, Cat. No. F0291) and 10ng/ml FGF-4\\n(Sigma, USA, Cat. No. F8424) in a humidified atmosphere containing 5%\\nCO2. To induce osteogenic differentiation, MSCs were seeded on Nunc™\\n24-well plates (Thermo Fisher, USA, Cat. No. 142475) or 48-well plates\\n(Thermo Fisher, USA, Cat. No. 150687) at a density of\\n1.5\\u2009×\\u200910<sup>4</sup> cells/cm2. After 24\\u2009h, differentiation was induced\\nusing StemPro® Osteogenesis Kit (Gibco/Life Technologies, USA, Cat. No.\\nA10072-01). The medium was exchanged every 3 days. Cells were treated\\nwith Lamivudine 3TC (Sigma), 1μM final concentration; C16 (Merck), 500nM\\nfinal concentration; G140 (InvivoGen), 150μM final concentration.</p>\\n<p><strong>L1 RNA transfection</strong></p>\\n<p>The vector human-L1_pBluescript II sk (+) carrying the full-length L1\\nsequence was custom-prepared by GenScript, USA. Large-scale human L1\\nmRNA was in vitro transcribed, modified, and purified by TriLink\\nBiotechnologies, USA, (ARCA capped and 2’Omethymalted (CapI), fully\\nsubstituted with 5-methyl-C, 25% substitution of Cyanine-5-U and 75%\\nsubstitution of Pseudo-U, enzymatically polyadenylated, DNase and\\nphosphatase treated, silica membrane purified). Synthetic L1 RNA was\\ntransfected in MSCs differentiating to osteoblasts at day 5 using\\nLipofectamine<sup>TM</sup> MessengerMAX<sup>TM</sup> (Invitrogen, USA,\\nCat. No. LMRNA003) with a modified protocol for low RNA amount. RFP mRNA\\n(System Bioscience, USA, Cat. No. MR800A-1) was used as negative\\ncontrol. Bone matrix was quantified with OsteoImage Mineralization Assay\\n(Lonza, Basel, Switzerland, Cat. No. LOPA503) and Alizarin Red staining.\\nIn conditioned media experiments, the medium was exchanged 6h after\\ntransfection. Conditioned medium was collected after three days and\\ndelivered to the recipient, differentiating osteoblasts at day 5. In\\nknockdown experiments, siRNAs were transfected with Lipofectamine\\nRNAiMAX (Thermo Scientific) in differentiating osteoblasts at day 3,\\nfollowed by L1 RNA transfection at day 7. siRNAs: EIF2AK2- Targeting\\nSMARTpool (Dharmacon, Horizon, E-003527-00-0010), MB21D1-Targeting\\nSMARTpool (Dharmacon, Horizon, L-015607-02-0020), Non-targeting Pool\\n(D-001810-10).</p>\\n<p><strong>Quantitative mineralization assay</strong></p>\\n<p>Cells were washed in PBS and fixed in 4% (v/v) formaldehyde (Sigma,\\nUSA, Cat. No. F8775) in 1× PBS for 15 minutes. Mineralization was\\nassessed by using the OsteoImage Mineralization Assay (Lonza, Basel,\\nSwitzerland, Cat. No. LOPA503) according to the manufacturer’s\\nindication. Mineralization was quantified with GloMax® discover\\nmicroplate reader (Promega, USA), selecting appropriate excitation (492)\\n/ emission (520) wavelengths.</p>\\n<p><strong>Alizarin Red staining</strong></p>\\n<p>Osteoblasts were washed with 1× PBS (Kantonsapotheke Zürich,\\nSwitzerland, Cat. No. A171012) and fixed with 4% (v/v) formaldehyde\\n(Sigma, USA, Cat. No. F8775) in 1× PBS for 30\\u2009min. After washing twice\\nwith ddH2O, Alizarin Red staining solution (0.7\\u2009g Alizarin Red S (Sigma,\\nUSA, Cat. No. A5533) diluted in 50\\u2009ml ddH2O at pH\\u2009=\\u20094.2) was added for\\n20\\u2009min. Afterwards, cells were washed four times with ddH2O, dried, and\\nstored in the dark until image acquisition. For absorbance measurement,\\nAlizarin Red S was eluted from stained osteoblasts with 300\\u2009μl 10% (w/v)\\ncetylpyridinium chloride in an aqueous 0.01\\u2009M Na2HPO4/NaH2PO4 solution\\nat pH\\u2009=\\u20097 for 1\\u2009h. One hundred fifty microliters were transferred on a\\n96-well plate, and absorbance was measured at 560\\u2009nm. Ten percent (w/v)\\ncetylpyridinium chloride in an aqueous 0.01\\u2009M Na2HPO4/NaH2PO4 solution\\nwas used as blank. Images were acquired, processed and analysed as\\npreviously described(Eggerschwiler <em>et al</em>, 2019)</p>\\n<p><strong>ALPL activity assay:</strong></p>\\n<p>ALPL activity was measured as in Liu et al., 2016.(Liu <em>et\\nal</em>, 2016) Briefly, cells were fixed in 3.7% formaldehyde at RT for\\n10’ and then stained with a solution of 25% naphthol AS-BI phosphate\\n(Thermo) and 0.75% Fast Blue BB (Sigma) dissolved in 0.1\\u2009M Tris buffer\\n(pH 9.3) at RT for 15’. After staining, cells were quickly washed with\\nPBS five times.</p>\\n<p><strong>ENPP1 activity assay</strong></p>\\n<p>ENPP1 activity was calculated colorimetrically using the chromogenic\\nsubstrate p-nitrophenyl-thymidine-5′-monophosphate, as described in\\n(Ferreira et al., 2013). After PBS washing, cells were lysed in a\\nsolution of 0.1% Triton X-100, 0.2 M Tris-base, 1.6 mM MgCl2, pH 8.1. 50\\nμl of 1mM thymidine monophosphate p-nitrophenyl ester (Sigma) were added\\nto 50 μl of cell lysate and incubated for 1 hour at 37°C. Then, four\\nvolumes of 0.1M NaOH were added to stop the reaction and absorbance was\\nread at 410 nm using a spectrophotometer (Tecan Nanoquant Infinite 200\\nPro Multimode). Absorbance was normalized to DNA content.</p>\\n<p><strong>Immunohistochemistry:</strong></p>\\n<p>Cells were fixed in 4% paraformaldehyde in PBS at room temperature\\n(RT) for 10’. After fixation, cells were permeabilized in 3% Triton\\nX-100 in PBS for 3’ at RT. Blocking was performed with 4% BSA in PBS for\\n30’ at RT. After blocking, cells were incubated with primary antibodies\\novernight at 4°C, then washed with PBS and incubated with secondary\\nantibodies for 1 hour at RT. After washing with PBS, nuclei were stained\\nwith Hoechst (Thermo Scientific) and mounted using Fluoro Gel with\\nDABCO™ Mounting Medium (Electron Microscopy Sciences). Primary\\nantibodies: DNA-RNA Hybrid (mouse, Merck Millipore, MABE1095), dsRNA\\n(mouse, Merck Millipore, MABE1134).</p>\\n<p><strong>RNA extraction and cDNA preparation</strong></p>\\n<p>Cells were harvested and resuspended in 1ml of QIAzol Lysis reagent\\n(Qiagen, Cat. No.79306). Total RNA was then purified with RNeasy Plus\\nMini kit (Qiagen, cat. No. 74134) with minimal modifications to\\nmanufacturer’s instructions. DNase treatment (RNase-free DNase set,\\nQiagen, Cat. No. 79254) was performed to remove any residual DNA. RNA\\nquality and concentration were checked using a Nanodrop<sup>TM</sup>\\n2000 spectrophotometer (ThermoFisher). cDNA was synthesized from 200ng\\nof each RNA sample using a Superscript III first-strand cDNA synthesis\\nsystem (ThermoFisher, cat. No.18080051) according to the manufacturer’s\\nprotocol. Processing of human bone biopsies and RNA isolation has been\\ndescribed previously.(Reppe <em>et al</em>, 2010)</p>\\n<p><strong>Gene expression analysis</strong></p>\\n<p>Real-time quantitative polymerase chain reaction (qPCR) was performed\\nwith a 7900HT Fast Real-Time PCR system (Applied Biosystems). Each\\nsample was analyzed in triplicate and normalized with the endogenous\\ncontrol Ribosomal Protein L13A (<em>RPL13A)</em> for cDNA input\\nconcentration. No template and no RT were included as negative controls.\\nFor each 15μl reaction, 10ng (1ng for L1) of cDNA was mixed with 1μM\\nspecific primers mix and 7.5μl of Sybr<sup>TM</sup> Select Master mix\\n(Applied Biosystems, USA, Cat. No. 4472908). The reaction was incubated\\nat 95°C for 10 minutes, followed by 40 cycles of denaturation at 95°C\\nfor 15 seconds, annealing at 60°C for 30 seconds and elongation at 72°C\\nfor 30 seconds. Ct values were calculated by 7900HT Fast Real-Time PCR\\nRQ manager software (Applied Biosystems, USA) and then normalized as ∆Ct\\nbetween the gene of interest and the endogenous calibrator. Primers used\\nin this study for gene expression analysis were designed using Primer3\\n(<a\\nhref=\"http://www.ncbi.nlm.nih.gov/tools/primer-blast/\"><u>http://www.ncbi.nlm.nih.gov/tools/primer-blast/</u></a>).\\nIn all primer pairs, each primer matches a different exon. Amplicons\\nlength was 80-130 nucleotides. Primer sequences are reported in Appendix\\nTable S3.</p>\\n<p><strong>Cell cycle analysis</strong></p>\\n<p>2x10<sup>5</sup> MSCs were trypsinized for 5 min at 37°C, washed with\\nPBS and 2% BSA, passed through a 70μM strainer (Corning, USA, Cat. No.\\n352350) and then fixed at -20°C for 30 min in 70% ethanol. After washing\\nwith PBS and 4% BSA, cells were resuspended in PBS and incubated for 1\\nhour at 37°C with RNase. Cells were then washed and resuspended in 100μl\\nof Flow Cytometry Staining Buffer (R&amp;D System, USA, Cat. No. FC001).\\n10μl of 1mg/ml Propidium iodide (PI) staining solution (Invitrogen, USA,\\nCat. No. P3566) was added to the single-cell solution, gently mixed, and\\nincubated for 5 min in the dark. Cell cycle analysis was performed on BD\\nFACSCanto II Flow Cytometry System (BD-Biosciences), using BD FACSDiva\\nSoftware (BD-Biosciences).</p>\\n<p><strong>RNA-Seq and data analysis</strong></p>\\n<p>RNA from human bone biopsies was sequenced at the Human Genotyping\\nfacility (HuGe-F) of Erasmus MC.</p>\\n<p>Total RNA-Seq library, from L1 RNA delivery experiments, was prepared\\nwith CORALL Total RNA library prep with RiboCop rRNA for Human/Mouse/Rat\\ndepletion kit (Lexogen GmbH, Vienna, Austria) following manufacturer’s\\ninstructions (library type: fr-secondstrand) by IGA Technology service\\n(Italy). The final libraries were checked using both Qubit 2.0\\nFluorometer (Invitrogen, Carlsbad, CA) and Agilent Bioanalyzer DNA assay\\nor Caliper (PerkinElmer, Waltham, MA). Libraries were then prepared for\\nsequencing and sequenced on paired-end 150 bp mode on NovaSeq6000\\n(Illumina, San Diego, CA). RNA-Seq read quality control (QC) analyses\\nand filtering of high-quality reads were executed using FastQC v0.11.9\\n(http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) and BBDuk\\nv35.85 (<a\\nhref=\"https://urldefense.com/v3/__https:/jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbduk-guide/__;!!Nmw4Hv0!0EikfGZVN-B-Xp5VkKLqowTItYk7Yav6lOsagjvuQwPqWY_Y3yszwKePkxm4puTKivdPL5LvCMjrSQwcKpyz9m7YVqqz09ekvfk$\">https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbduk-guide/</a>)\\xa0by\\nsetting a minimum read length of 35 bp and a minimum Phred-quality score\\nof 25. After trimming quality control, high-quality reads were aligned\\nto the human genome reference (GRCh38) with STAR 2.7.3a,(Dobin <em>et\\nal</em>, 2013) while FeatureCounts 1.6.3 package(Liao <em>et al</em>,\\n2014) was used to assign reads to genes. Next, lowly expressed genes\\nacross one or more experimental conditions were filtered to eliminate\\nthe “uninformative” genes using HTSFilter v1.30.1.(Rau <em>et al</em>,\\n2013) Filtered gene data were further processed with the EdgeR package\\nv3.32.1(Robinson <em>et al</em>, 2010) to normalize (Trimmed Mean of M-7\\nvalues, TMM, method) the raw counts and perform differential gene\\nexpression analysis. Multiple testing correction was performed with the\\nFDR method(Benjamini &amp; Hochberg, 1995) and significance level was\\nset at FDR &lt; 0.05. Gene Ontology (GO) term enrichment was analyzed by\\nperforming hypergeometric tests11 for each individual term, and FDR\\ncorrection was applied (FDR &lt; 0.05). Expression of Interspersed\\nRepeat elements was quantified using SQuiRE 0.9.9.92\\n(https://github.com/wyang17/SquIRE). SQuiRE provides locus-specific\\nexpression quantification along with subfamily-level expression\\nestimates counting unambiguously mapped reads, as well as ambiguously\\nmapped reads using an expectation–maximization (EM) algorithm.(Yang\\n<em>et al</em>, 2019) Briefly, reference genome and Repeatmasker\\nannotation were downloaded from UCSC and prepared for the analysis with\\nsquire Fetch and squire Clean, respectively. High-quality reads were\\nmapped against the reference genome with STAR using squire Map and\\nexpression was quantified using squire Call. Differential expression\\nanalysis was performed with squire Call.</p>\\n<p><strong>DIA-MS analysis using TimsTOF MS</strong></p>\\n<p>Total protein extracts were prepared with RIPA buffer (50mMTris-cl pH\\n8.0, 5mM EDTA, 150mM Nacl, 15mM MgCl<sub>2</sub>, 1% NP-40, 1mM PMSF and\\n1X Protease Inhibitor Cocktail (PIC)). Further sonication step was\\nincluded: (30S ON, 30S OFF, 10 cycles with Bioruptor). Equal 50 μg\\nprotein extracts were concentrated to 30 μl volume, then diluted in 8M\\nurea in 0.1M Tris-HCl, followed by protein digestion with trypsin,\\naccording to the FASP protocol.(Wiśniewski <em>et al</em>, 2009) After\\novernight digestion, the peptides were eluted from the filters with 25\\nmM ammonium bicarbonate buffer. The eluted peptides were processed in\\nthe desalting step using Sep-Pag C18 Column (waters) based on the\\nmanufacturer’s instructions. Approximately 200 ng of peptide mixture per\\nsample was analyzed using a timsTOF Pro 2 QTOF mass spectrometer coupled\\nwith a nanoElute liquid chromatography system (Bruker Daltonik GmbH,\\nGermany). The sample was injected directly into an RP-C18 Aurora emitter\\ncolumn (75 µm\\u202fi.d.×\\u202f250 mm, 1.6 μm, 120 Å pore size) (Ion Opticks,\\nAustralia) using a one-column separation method. An 80-min gradient was\\nestablished using mobile phase A (0.1% FA in H2O) and mobile phase B\\n(0.1% FA in Acetonitrile): 2–25% B for 60\\u2009min, 25–37% for 10\\u2009min,\\nramping 37% to 95% in 5\\u2009min, and maintaining 95% B for 5\\u2009min. The column\\ntemperature was set at 50<sup>o</sup>C and the flow rate at 250 nl/min.\\nThe sample eluting from the separation column was introduced into the\\nmass spectrometer via a CaptiveSpray nano-electrospray ion source\\n(Bruker Daltonik GmbH) with an electrospray voltage of 1.6\\u2009kV. The ion\\nsource temperature was set to 180\\u2009°C and a dry gas of 3 l/min. The\\nsamples were analyzed using diaPASEF scheme(Meier <em>et al</em>, 2020b)\\nconsisting of 24 cycles including a total of 48 mass width windows (13\\nDa (m/z) from m/z 400 to 1,000 and TIMS scan range from\\u20090.63\\u2009to\\n1.35\\u2009Vs\\u2009cm−2 (1/K0). The collisional energy increased linearly from\\n20.01 eV at 0.6 (1/K0) to 52.00 eV at 1.35\\u2009Vs\\u2009cm−2 (1/K0). The scan\\nrange for MS and MS/MS spectra was set to 100-1700 m/z. TIMS ramping\\ntime and accumulation time were set to 100 milliseconds. The diaPASEF\\ndata were analyzed by directDIA approach using Spectronaut software\\n(version 14) following manufacture instructions. Up or down regulated\\nproteins were determined using DEP (Differential Enrichment analysis of\\nProteomics data) R package. Significant results (adjusted p-value &lt;\\n0.05) were subjected to Gene Ontology enrichment analysis with\\nclusterProfiler R package.</p>\\n<p><strong>Western Blot</strong></p>\\n<p>Total protein extracts were prepared by lysing cells in extraction\\nbuffer (HEPES KOH [pH 8.5], NaCl 400 mM, EDTA 0.1 mM, EGTA 0.1 mM, DTT 1\\nmM, 1× protease inhibitor, SDS 1%). Proteins were separated by\\nelectrophoresis on BOLT 4%–12% bis-tris polyacrylamide precast gels in\\nMES buffer (Life Technologies) and transferred to a 0,2 μm\\nnitrocellulose membrane. Non-specific signals were blocked with 5%\\nMilk-PBS-Tween0.5% and the membrane hybridized overnight at 4°C with\\nprimary and secondary antibodies diluted in blocking buffer. Horseradish\\nperoxidase-conjugated secondary antibodies were revealed with the ECL\\nchemiluminescence kit (Amersham), and signals were detected using\\nChemiDoc (Bio-Rad). Primary antibodies: PKR (rabbit, Abcam, ab32052),\\ncGAS (rabbit, Abcam, ab224144), LINE-1 ORF1 (mouse, Merck Millipore,\\nMABC1152), EIF2a (mouse, Abcam, ab5369),\\xa0phospho-EIF2α\\xa0(rabbit, Abcam,\\nab32157), Histone H3 (rabbit, Merk, 06-755).</p>\\n<p><strong>Exosome isolation</strong></p>\\n<p>Exosomes were isolated using Total Exosome Isolation Reagent (from\\ncell culture media) (Thermo), following manufacturer’s instructions. The\\nexosome pellet was resuspended in 50 μl of RIPA buffer and kept on ice\\nfor 30’, mixing every 5’. Samples were then sonicated (30’’ ON/30” OFF,\\n10 cycles), centrifuged at 13,000 x g for 20’ at 4°C and collected as\\nsupernatants.</p>\\n<p><strong>Statistical analysis</strong></p>\\n<p>Statistic tests used for data analysis are indicated in the figure\\nlegends.</p>\\n<p><strong>Data availability</strong></p>\\n<p>The RNA-seq transcriptomics data on human bone biopsies have been\\ndeposited to BioProject with the dataset identifier PRJNA764663\\n(https://www.ncbi.nlm.nih.gov/bioproject/PRJNA764663). The RNA-seq\\ntranscriptomics data on transfected osteoblasts have been deposited to\\nGEO with the dataset identifier GSE201774\\n(https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201774) (enter\\ntoken: ififogaehzoxlgh).</p>\\n<p>The mass spectrometry proteomics data have been deposited to the\\nProteomeXchange Consortium via the PRIDE partner repository with the\\ndataset identifier PXD051195. Reviewer account details are username: <a\\nhref=\"mailto:reviewer_pxd051195@ebi.ac.uk\">reviewer_pxd051195@ebi.ac.uk</a>\\n; password: R6v0mIYe</p>\\n<p><strong>Acknowledgment</strong></p>\\n<p>We acknowledge kaust Smarth Health Initiative (KSHI); the\\nSouth-Eastern Norway Regional Health Authority; the Norwegian\\nOsteoporosis Society; Anders Jahre Foundation for the Promotion of\\nScience; the Lovisenberg Diaconal Hospital research fund; and the\\nResearch Council of Norway.</p>\\n<p><strong>Funding:</strong> We are grateful for support from: KAUST\\nBAS/1/1037-01-01, the European Union project OSTEOGENE (no. FP6-502491);\\nZonMw\\xa0VIDI\\xa0016.136.367\\xa0grant, for funding the creation of the RNA-Seq\\ndataset of hip primary bone and Oslo University Hospital, Ullevål,\\nproject #29750104.</p>\\n<p><strong>Disclosure and competing interests statement:</strong> All\\nthe authors declare no competing interests.</p>\\n<p><strong>References</strong></p>\\n<p>Aerssens J, Boonen S, Joly J &amp; Dequeker J (1997) Variations in\\ntrabecular bone composition with anatomical site and age: potential\\nimplications for bone quality assessment. <em>J Endocrinol</em> 155:\\n411–421</p>\\n<p>Alioli CA, Demesmay L, Laurencin-Dalacieux S, Beton N, Farlay D,\\nFollet H, Saber A, Duboeuf F, Chun J, Rivera R, <em>et al</em> (2020)\\nExpression of the type 1 lysophosphatidic acid receptor in osteoblastic\\ncell lineage controls both bone mineralization and osteocyte\\nspecification. <em>Biochim Biophys Acta - Mol Cell Biol Lipids</em>\\n1865: 158715</p>\\n<p>Anderson BR, Muramatsu H, Nallagatla SR, Bevilacqua PC, Sansing LH,\\nWeissman D &amp; Karikó K (2010) Incorporation of pseudouridine into\\nmRNA enhances translation by diminishing PKR activation. <em>Nucleic\\nAcids Res</em></p>\\n<p>Bahney CS, Zondervan RL, Allison P, Theologis A, Ashley JW, Ahn J,\\nMiclau T, Marcucio RS &amp; Hankenson KD (2019) Cellular biology of\\nfracture healing. <em>J Orthop Res</em> 37: 35–50</p>\\n<p>Balboni TA, Gobezie R &amp; Mamon HJ (2006) Heterotopic ossification:\\nPathophysiology, clinical features, and the role of radiotherapy for\\nprophylaxis. <em>Int J Radiat Oncol</em> 65: 1289–1299</p>\\n<p>Benjamini Y &amp; Hochberg Y (1995) Controlling the False Discovery\\nRate: A Practical and Powerful Approach to Multiple Testing. <em>J R\\nStat Soc Ser B</em> 57: 289–300</p>\\n<p>Bischoff DS, Sakamoto T, Ishida K, Makhijani NS, Gruber HE &amp;\\nYamaguchi DT (2011) CXC receptor knockout mice: characterization of\\nskeletal features and membranous bone healing in the adult mouse.\\n<em>Bone</em> 48: 267–274</p>\\n<p>Brem H &amp; Tomic-Canic M (2007) Cellular and molecular basis of\\nwound healing in diabetes. <em>J Clin Invest</em> 117: 1219–1222</p>\\n<p>Carelli FN, Cerrato C, Dong Y, Appert A, Dernburg A &amp; Ahringer J\\n(2022) Widespread transposon co-option in the Caenorhabditis germline\\nregulatory network. <em>Sci Adv</em> 8</p>\\n<p>De Cecco M, Ito T, Petrashen AP, Elias AE, Skvir NJ, Criscione SW,\\nCaligiana A, Brocculi G, Adney EM, Boeke JD, <em>et al</em> (2019a) L1\\ndrives IFN in senescent cells and promotes age-associated inflammation.\\n<em>Nature</em> 566: 73–78</p>\\n<p>De Cecco M, Ito T, Petrashen AP, Elias AE, Skvir NJ, Criscione SW,\\nCaligiana A, Brocculi G, Adney EM, Boeke JD, <em>et al</em> (2019b) L1\\ndrives IFN in senescent cells and promotes age-associated inflammation.\\n<em>Nature</em> 566: 73–78</p>\\n<p>Chan JK, Roth J, Oppenheim JJ, Tracey KJ, Vogl T, Feldmann M, Horwood\\nN &amp; Nanchahal J (2012) Alarmins: awaiting a clinical response. <em>J\\nClin Invest</em> 122: 2711</p>\\n<p>Chaparro O &amp; Linero I (2016) Regenerative Medicine: A New\\nParadigm in Bone Regeneration. In <em>Advanced Techniques in Bone\\nRegeneration</em> InTech</p>\\n<p>Chen YG &amp; Hur S (2022) Cellular origins of dsRNA, their\\nrecognition and consequences. <em>Nat Rev Mol Cell Biol</em> 23:\\n286–301</p>\\n<p>Choi J, Choi W, Joo Y, Chung H, Kim D, Oh SJ &amp; Kim S-H (2021)\\nFGF2-primed 3D spheroids producing IL-8 promote therapeutic angiogenesis\\nin murine hindlimb ischemia. <em>NPJ Regen Med</em> 6: 48</p>\\n<p>Clark D, Nakamura M, Miclau T &amp; Marcucio R (2017) Effects of\\nAging on Fracture Healing. <em>Curr Osteoporos Rep</em> 15: 601</p>\\n<p>Coates BA, McKenzie JA, Buettmann EG, Liu X, Gontarz PM, Zhang B\\n&amp; Silva MJ (2019) Transcriptional Profiling of Intramembranous and\\nEndochondral Ossification after Fracture in Mice. <em>Bone</em> 127:\\n577</p>\\n<p>Coppé J-P, Desprez P-Y, Krtolica A &amp; Campisi J (2010) The\\nSenescence-Associated Secretory Phenotype: The Dark Side of Tumor\\nSuppression. <em>Annu Rev Pathol Mech Dis</em> 5: 99–118</p>\\n<p>Cosby RL, Chang NC &amp; Feschotte C (2019) Host-transposon\\ninteractions: conflict, cooperation, and cooption. <em>Genes Dev</em>\\n33: 1098–1116</p>\\n<p>Croes M, Boot W, Kruyt MC, Weinans H, Pouran B, van der Helm YJM,\\nGawlitta D, Vogely HC, Alblas J, Dhert WJA, <em>et al</em> (2017)\\nInflammation-Induced Osteogenesis in a Rabbit Tibia Model. <em>Tissue\\nEng Part C Methods</em> 23: 673–685</p>\\n<p>Croes M, Oner FC, Kruyt MC, Blokhuis TJ, Bastian O, Dhert WJA &amp;\\nAlblas J (2015) Proinflammatory Mediators Enhance the Osteogenesis of\\nHuman Mesenchymal Stem Cells after Lineage Commitment. <em>PLoS One</em>\\n10: e0132781</p>\\n<p>Croes M, Öner FC, van Neerven D, Sabir E, Kruyt MC, Blokhuis TJ,\\nDhert WJA &amp; Alblas J (2016) Proinflammatory T cells and IL-17\\nstimulate osteoblast differentiation. <em>Bone</em> 84: 262–270</p>\\n<p>Dobin A, Davis CA, Schlesinger F, Drenkow J, Zaleski C, Jha S, Batut\\nP, Chaisson M &amp; Gingeras TR (2013) STAR: ultrafast universal RNA-seq\\naligner. <em>Bioinformatics</em> 29: 15–21</p>\\n<p>Donnelly N, Gorman AM, Gupta S &amp; Samali A (2013) The eIF2α\\nkinases: their structures and functions. <em>Cell Mol Life Sci</em> 70:\\n3493–3511</p>\\n<p>Eash KJ, Greenbaum AM, Gopalan PK &amp; Link DC (2010) CXCR2 and\\nCXCR4 antagonistically regulate neutrophil trafficking from murine bone\\nmarrow. <em>J Clin Invest</em> 120: 2423–2431</p>\\n<p>Eggerschwiler B, Canepa DD, Pape HC, Casanova EA &amp; Cinelli P\\n(2019) Automated digital image quantification of histological staining\\nfor the analysis of the trilineage differentiation potential of\\nmesenchymal stem cells. <em>Stem Cell Res Ther</em></p>\\n<p>FLEISCH H, RUSSELL RGG &amp; STRAUMANN F (1966) Effect of\\nPyrophosphate on Hydroxyapatite and Its Implications in Calcium\\nHomeostasis. <em>Nature</em> 212: 901–903</p>\\n<p>Fukuda S, Varshney A, Fowler BJ, Wang S Bin, Narendran S, Ambati K,\\nYasuma T, Magagnoli J, Leung H, Hirahara S, <em>et al</em> (2021)\\nCytoplasmic synthesis of endogenous Alu complementary DNA via reverse\\ntranscription and implications in age-related macular degeneration.\\n<em>Proc Natl Acad Sci U S A</em> 118</p>\\n<p>Ge C, Xiao G, Jiang D &amp; Franceschi RT (2007) Critical role of the\\nextracellular signal–regulated kinase–MAPK pathway in osteoblast\\ndifferentiation and skeletal development. <em>J Cell Biol</em> 176:\\n709</p>\\n<p>Gennero I, Laurencin-Dalicieux S, Conte-Auriol F, Briand-Mésange F,\\nLaurencin D, Rue J, Beton N, Malet N, Mus M, Tokumura A, <em>et al</em>\\n(2011) Absence of the lysophosphatidic acid receptor LPA1 results in\\nabnormal bone development and decreased bone mass. <em>Bone</em> 49:\\n395–403</p>\\n<p>Gerstenfeld LC, Cho T-J, Kon T, Aizawa T, Cruceta J, Graves BD &amp;\\nEinhorn TA (2001) Impaired Intramembranous Bone Formation during Bone\\nRepair in the Absence of Tumor Necrosis Factor-Alpha Signaling.\\n<em>Cells Tissues Organs</em> 169: 285–294</p>\\n<p>Gerstenfeld LC, Cullinane DM, Barnes GL, Graves DT &amp; Einhorn TA\\n(2003) Fracture healing as a post-natal developmental process:\\nMolecular, spatial, and temporal aspects of its regulation. <em>J Cell\\nBiochem</em> 88: 873–884</p>\\n<p>Hessle L, Johnson KA, Anderson HC, Narisawa S, Sali A, Goding JW,\\nTerkeltaub R &amp; Millan JL (2002) Tissue-nonspecific alkaline\\nphosphatase and plasma cell membrane glycoprotein-1 are central\\nantagonistic regulators of bone mineralization. <em>Proc Natl Acad Sci U\\nS A</em> 99: 9445–9</p>\\n<p>Hou Y, Ryu CH, Jun JA, Kim SM, Jeong CH &amp; Jeun S (2014) IL-8\\nenhances the angiogenic potential of human bone marrow mesenchymal stem\\ncells by increasing vascular endothelial growth factor. <em>Cell Biol\\nInt</em> 38: 1050–9</p>\\n<p>Hur S (2019) Double-Stranded RNA Sensors and Modulators in Innate\\nImmunity. <em>Annu Rev Immunol</em> 37: 349–375</p>\\n<p>Hurtgen BJ, Ward CL, Garg K, Pollot BE, Goldma SM, McKinley TO, Wenke\\nJC &amp; Corona BT (2016) Severe muscle trauma triggers heightened and\\nprolonged local musculoskeletal inflammation and impairs adjacent tibia\\nfracture healing. <em>J Musculoskelet Neuronal Interact</em> 16: 122</p>\\n<p>Jachowicz JW, Bing X, Pontabry J, Bošković A, Rando OJ &amp;\\nTorres-Padilla ME (2017) LINE-1 activation after fertilization regulates\\nglobal chromatin accessibility in the early mouse embryo. <em>Nat\\nGenet</em></p>\\n<p>JIANG WG, SANDERS AJ, RUGE F &amp; HARDING KG (2012) Influence of\\ninterleukin-8 (IL-8) and IL-8 receptors on the migration of human\\nkeratinocytes, the role of PLC-γ and potential clinical implications.\\n<em>Exp Ther Med</em> 3: 231–236</p>\\n<p>Jiao H, Xiao E &amp; Graves DT (2015) Diabetes and Its Effect on Bone\\nand Fracture Healing. <em>Curr Osteoporos Rep</em> 13: 327–335</p>\\n<p>Johnson KA, Hessle L, Vaingankar S, Wennberg C, Mauro S, Narisawa S,\\nGoding JW, Sano K, Millan JL &amp; Terkeltaub R (2000) Osteoblast\\ntissue-nonspecific alkaline phosphatase antagonizes and regulates PC-1.\\n<em>Am J Physiol Regul Integr Comp Physiol</em> 279: R1365-77</p>\\n<p>Jones RB, Garrison KE, Wong JC, Duan EH, Nixon DF &amp; Ostrowski MA\\n(2008) Nucleoside analogue reverse transcriptase inhibitors\\ndifferentially inhibit human LINE-1 retrotransposition. <em>PLoS\\nOne</em></p>\\n<p>Karikó K, Buckstein M, Ni H &amp; Weissman D (2005) Suppression of\\nRNA recognition by Toll-like receptors: The impact of nucleoside\\nmodification and the evolutionary origin of RNA. <em>Immunity</em></p>\\n<p>Karikó K, Muramatsu H, Welsh FA, Ludwig J, Kato H, Akira S &amp;\\nWeissman D (2008) Incorporation of pseudouridine into mRNA yields\\nsuperior nonimmunogenic vector with increased translational capacity and\\nbiological stability. <em>Mol Ther</em></p>\\n<p>Kim Y, Lee JH, Park J-E, Cho J, Yi H &amp; Kim VN (2014) PKR is\\nactivated by cellular dsRNAs during mitosis and acts as a mitotic\\nregulator. <em>Genes Dev</em> 28: 1310–1322</p>\\n<p>Kim Y, Park J, Kim S, Kim MA, Kang MG, Kwak C, Kang M, Kim B, Rhee HW\\n&amp; Kim VN (2018) PKR Senses Nuclear and Mitochondrial Signals by\\nInteracting with Endogenous Double-Stranded RNAs. <em>Mol Cell</em> 71:\\n1051-1063.e6</p>\\n<p>Kon T, Cho TJ, Aizawa T, Yamazaki M, Nooh N, Graves D, Gerstenfeld LC\\n&amp; Einhorn TA (2001) Expression of osteoprotegerin, receptor\\nactivator of NF-kappaB ligand (osteoprotegerin ligand) and related\\nproinflammatory cytokines during fracture healing. <em>J Bone Miner\\nRes</em> 16: 1004–1014</p>\\n<p>Kormann MSD, Hasenpusch G, Aneja MK, Nica G, Flemmer AW, Herber-Jonat\\nS, Huppmann M, Mays LE, Illenyi M, Schams A, <em>et al</em> (2011)\\nExpression of therapeutic proteins after delivery of chemically modified\\nmRNA in mice. <em>Nat Biotechnol</em></p>\\n<p>Koski GK, Karikó K, Xu S, Weissman D, Cohen PA &amp; Czerniecki BJ\\n(2004) Cutting Edge: Innate Immune System Discriminates between RNA\\nContaining Bacterial versus Eukaryotic Structural Features That Prime\\nfor High-Level IL-12 Secretion by Dendritic Cells. <em>J\\nImmunol</em></p>\\n<p>Kovtun A, Bergdolt S, Wiegner R, Radermacher P, Huber-Lang M &amp;\\nIgnatius A (2016) The crucial role of neutrophil granulocytes in bone\\nfracture healing. <em>Eur Cells Mater</em> 32: 152–162</p>\\n<p>Lama L, Adura C, Xie W, Tomita D, Kamei T, Kuryavyi V, Gogakos T,\\nSteinberg JI, Miller M, Ramos-Espiritu L, <em>et al</em> (2019)\\nDevelopment of human cGAS-specific small-molecule inhibitors for\\nrepression of dsDNA-triggered interferon expression. <em>Nat Commun</em>\\n10: 2261</p>\\n<p>Laroche M, Livideanu C, Paul C &amp; Cantagrel A (2011) Interferon\\nAlpha and Pamidronate in Osteoporosis with Fracture Secondary to\\nMastocytosis. <em>Am J Med</em> 124: 776–778</p>\\n<p>Lee MJ, Kim DH, Ryu JS, Ko AY, Ko JH, Kim MK, Wee WR, Khwarg SI &amp;\\nOh JY (2015) Topical TSG-6 Administration Protects the Ocular Surface in\\nTwo Mouse Models of Inflammation-Related Dry Eye. <em>Invest Ophthalmol\\nVis Sci</em> 56: 5175–81</p>\\n<p>Lee RH, Pulin AA, Seo MJ, Kota DJ, Ylostalo J, Larson BL,\\nSemprun-Prieto L, Delafontaine P &amp; Prockop DJ (2009) Intravenous\\nhMSCs improve myocardial infarction in mice because cells embolized in\\nlung are activated to secrete the anti-inflammatory protein TSG-6.\\n<em>Cell Stem Cell</em> 5: 54–63</p>\\n<p>Lefkopoulos S, Polyzou A, Derecka M, Bergo V, Clapes T, Cauchy P,\\nJerez-Longres C, Onishi-Seebacher M, Yin N, Martagon-Calderón N-A,\\n<em>et al</em> (2020) Repetitive Elements Trigger RIG-I-like Receptor\\nSignaling that Regulates the Emergence of Hematopoietic Stem and\\nProgenitor Cells. <em>Immunity</em> 53: 934-951.e9</p>\\n<p>Li C, Li G, Liu M, Zhou T &amp; Zhou H (2016) Paracrine effect of\\ninflammatory cytokine-activated bone marrow mesenchymal stem cells and\\nits role in osteoblast function. <em>J Biosci Bioeng</em> 121:\\n213–219</p>\\n<p>Li H, Li D, Ma Z, Qian Z, Kang X, Jin X, Li F, Wang X, Chen Q, Sun H,\\n<em>et al</em> (2018) Defective autophagy in osteoblasts induces\\nendoplasmic reticulum stress and causes remarkable bone loss.\\n<em>Autophagy</em> 14: 1726–1741</p>\\n<p>Li J, Li X, Liu D, Hamamura K, Wan Q, Na S, Yokota H &amp; Zhang P\\n(2019) eIF2α signaling regulates autophagy of osteoblasts and the\\ndevelopment of osteoclasts in OVX mice. <em>Cell Death Dis</em> 10:\\n921</p>\\n<p>Liao Y, Smyth GK &amp; Shi W (2014) featureCounts: an efficient\\ngeneral purpose program for assigning sequence reads to genomic\\nfeatures. <em>Bioinformatics</em> 30: 923–930</p>\\n<p>Lin D, Chai Y, Ma Y, Duan B, Yuan Y &amp; Liu C (2019) Rapid\\ninitiation of guided bone regeneration driven by spatiotemporal delivery\\nof IL-8 and BMP-2 from hierarchical MBG-based scaffold.\\n<em>Biomaterials</em> 196: 122–137</p>\\n<p>Liu W, Zhou L, Zhou C, Zhang S, Jing J, Xie L, Sun N, Duan X, Jing W,\\nLiang X, <em>et al</em> (2016) GDF11 decreases bone mass by stimulating\\nosteoclastogenesis and inhibiting osteoblast differentiation. <em>Nat\\nCommun</em> 7: 12794</p>\\n<p>Liu X, Liu Z, Sun L, Ren J, Wu Z, Jiang X, Ji Q, Wang Q, Fan Y, Cai\\nY, <em>et al</em> (2021) Resurrection of human endogenous retroviruses\\nduring aging reinforces senescence. <em>bioRxiv</em></p>\\n<p>Lories RJU &amp; Schett G (2012) Pathophysiology of New Bone\\nFormation and Ankylosis in Spondyloarthritis. <em>Rheum Dis Clin North\\nAm</em> 38: 555–567</p>\\n<p>Lu JY, Chang L, Li T, Wang T, Yin Y, Zhan G, Han X, Zhang K, Tao Y,\\nPercharde M, <em>et al</em> (2021) Homotypic clustering of L1 and B1/Alu\\nrepeats compartmentalizes the 3D genome. <em>Cell Res</em> 31:\\n613–630</p>\\n<p>Ludwig J, Schuberth C, Goldeck M, Schlee M, Li H, Juranek S, Sheng G,\\nMicura R, Tuschl T, Hartmann G, <em>et al</em> (2010) Structural and\\nfunctional insights into 5′-ppp RNA pattern recognition by the innate\\nimmune receptor RIG-I. <em>Nat Struct Mol Biol</em></p>\\n<p>Mangiavacchi A, Liu P, Della Valle F &amp; Orlando V (2021) New\\ninsights into the functional role of retrotransposon dynamics in\\nmammalian somatic cells. <em>Cell Mol Life Sci</em> 78: 5245–5256</p>\\n<p>Marsell R &amp; Einhorn TA (2011) The biology of fracture healing.\\n<em>Injury</em> 42: 551–555</p>\\n<p>Maruyama M, Rhee C, Utsunomiya T, Zhang N, Ueno M, Yao Z &amp;\\nGoodman SB (2020) Modulation of the Inflammatory Response and Bone\\nHealing. <em>Front Endocrinol (Lausanne)</em> 11: 386</p>\\n<p>Mashanov VS, Zueva OR &amp; García-Arrarás JE (2012) Posttraumatic\\nregeneration involves differential expression of long terminal repeat\\n(LTR) retrotransposons. <em>Dev Dyn</em> 241: 1625–1636</p>\\n<p>Matsushita T, Chan YY, Kawanami A, Balmes G, Landreth GE &amp;\\nMurakami S (2009) Extracellular signal-regulated kinase 1 (ERK1) and\\nERK2 play essential roles in osteoblast differentiation and in\\nsupporting osteoclastogenesis. <em>Mol Cell Biol</em> 29: 5843–5857</p>\\n<p>Meier F, Brunner AD, Frank M, Ha A, Bludau I, Voytik E,\\nKaspar-Schoenefeld S, Lubeck M, Raether O, Bache N, <em>et al</em>\\n(2020a) diaPASEF: parallel accumulation-serial fragmentation combined\\nwith data-independent acquisition. <em>Nat Methods</em> 17:\\n1229–1236</p>\\n<p>Meier F, Brunner AD, Frank M, Ha A, Bludau I, Voytik E,\\nKaspar-Schoenefeld S, Lubeck M, Raether O, Bache N, <em>et al</em>\\n(2020b) diaPASEF: parallel accumulation-serial fragmentation combined\\nwith data-independent acquisition. <em>Nat Methods</em> 17:\\n1229–1236</p>\\n<p>Mountziaris PM, Spicer PP, Kasper FK &amp; Mikos AG (2011) Harnessing\\nand Modulating Inflammation in Strategies for Bone Regeneration.\\n<em>Tissue Eng Part B Rev</em> 17: 393</p>\\n<p>Nallagatla SR &amp; Bevilacqua PC (2008) Nucleoside modifications\\nmodulate activation of the protein kinase PKR in an RNA\\nstructure-specific manner. <em>RNA</em></p>\\n<p>Nelson AM, Reddy SK, Ratliff TS, Hossain MZ, Katseff AS, Zhu AS,\\nChang E, Resnik SR, Page C, Kim D, <em>et al</em> (2015) dsRNA Released\\nby Tissue Damage Activates TLR3 to Drive Skin Regeneration. <em>Cell\\nStem Cell</em> 17: 139–151</p>\\n<p>Nimiritsky P, Eremichev R, Alexandrushkina N, Efimenko A, Tkachuk V\\n&amp; Makarevich P (2019) Unveiling Mesenchymal Stromal Cells’\\nOrganizing Function in Regeneration. <em>Int J Mol Sci</em> 20: 823</p>\\n<p>Nollet M, Santucci-Darmanin S, Breuil V, Al-Sahlanee R, Cros C, Topi\\nM, Momier D, Samson M, Pagnotta S, Cailleteau L, <em>et al</em> (2014)\\nAutophagy in osteoblasts is involved in mineralization and bone\\nhomeostasis. <em>Autophagy</em> 10: 1965–1977</p>\\n<p>Owlarn S, Klenner F, Schmidt D, Rabert F, Tomasso A, Reuter H, Mulaw\\nMA, Moritz S, Gentile L, Weidinger G, <em>et al</em> (2017) Generic\\nwound signals initiate regeneration in missing-tissue contexts. <em>Nat\\nCommun 2017 81</em> 8: 1–13</p>\\n<p>Pardi N, Muramatsu H, Weissman D &amp; Karikó K (2013) In vitro\\ntranscription of long RNA containing modified nucleosides. <em>Methods\\nMol Biol</em></p>\\n<p>Percharde M, Lin CJ, Yin Y, Guan J, Peixoto GA, Bulut-Karslioglu A,\\nBiechele S, Huang B, Shen X &amp; Ramalho-Santos M (2018) A\\nLINE1-Nucleolin Partnership Regulates Early Development and ESC\\nIdentity. <em>Cell</em></p>\\n<p>Rau A, Gallopin M, Celeux G &amp; Jaffrézic F (2013) Data-based\\nfiltering for replicated high-throughput transcriptome sequencing\\nexperiments. <em>Bioinformatics</em> 29: 2146–2152</p>\\n<p>Recknagel S, Bindl R, Brochhausen C, Göckelmann M, Wehner T,\\nSchoengraf P, Huber-Lang M, Claes L &amp; Ignatius A (2013) Systemic\\ninflammation induced by a thoracic trauma alters the cellular\\ncomposition of the early fracture callus. <em>J Trauma Acute Care\\nSurg</em> 74: 531–537</p>\\n<p>Reppe S, Refvem H, Gautvik VT, Olstad OK, Høvring PI, Reinholt FP,\\nHolden M, Frigessi A, Jemtland R &amp; Gautvik KM (2010) Eight genes are\\nhighly associated with BMD variation in postmenopausal Caucasian women.\\n<em>Bone</em></p>\\n<p>Ritschka B, Storer M, Mas A, Heinzmann F, Ortells MC, Morton JP,\\nSansom OJ, Zender L &amp; Keyes WM (2017) The senescence-associated\\nsecretory phenotype induces cellular plasticity and tissue regeneration.\\n<em>Genes Dev</em> 31: 172–183</p>\\n<p>Roberts SJ &amp; Ke HZ (2018) Anabolic Strategies to Augment Bone\\nFracture Healing. <em>Curr Osteoporos Rep</em> 16: 289–298</p>\\n<p>Robinson MD, McCarthy DJ &amp; Smyth GK (2010) edgeR: a Bioconductor\\npackage for differential expression analysis of digital gene expression\\ndata. <em>Bioinformatics</em> 26: 139–140</p>\\n<p>Sadeq S, Al-Hashimi S, Cusack CM &amp; Werner A (2021) Endogenous\\nDouble-Stranded RNA. <em>Non-Coding RNA</em> 7: 15</p>\\n<p>Schmidt-Bleek K, Schell H, Schulz N, Hoff P, Perka C, Buttgereit F,\\nVolk HD, Lienau J &amp; Duda GN (2012) Inflammatory phase of bone\\nhealing initiates the regenerative healing cascade. <em>Cell Tissue\\nRes</em> 347: 567–573</p>\\n<p>Sheng W, LaFleur MW, Nguyen TH, Chen S, Chakravarthy A, Conway JR, Li\\nY, Chen H, Yang H, Hsu P-H, <em>et al</em> (2018) LSD1 Ablation\\nStimulates Anti-tumor Immunity and Enables Checkpoint Blockade.\\n<em>Cell</em> 174: 549-563.e19</p>\\n<p>Simon M, Van Meter M, Ablaeva J, Ke Z, Gonzalez RS, Taguchi T, De\\nCecco M, Leonova KI, Kogan V, Helfand SL, <em>et al</em> (2019) LINE1\\nDerepression in Aged Wild-Type and SIRT6-Deficient Mice Drives\\nInflammation. <em>Cell Metab</em> 29: 871-885.e5</p>\\n<p>Slotkin RK &amp; Martienssen R (2007) Transposable elements and the\\nepigenetic regulation of the genome. <em>Nat Rev Genet</em> 8:\\n272–285</p>\\n<p>Strieter RM, Burdick MD, Gomperts BN, Belperio JA &amp; Keane MP\\n(2005) CXC chemokines in angiogenesis. <em>Cytokine Growth Factor\\nRev</em> 16: 593–609</p>\\n<p>Timmen M, Hidding H, Wieskötter B, Baum W, Pap T, Raschke MJ, Schett\\nG, Zwerina J &amp; Stange R (2014) Influence of antiTNF-alpha antibody\\ntreatment on fracture healing under chronic inflammation. <em>BMC\\nMusculoskelet Disord</em> 15: 184</p>\\n<p>Della Valle F, Reddy P, Yamamoto M, Liu P, Saera-Vila A, Bensaddek D,\\nZhang H, Prieto Martinez J, Abassi L, Celii M, <em>et al</em> (2022)\\nLINE-1 RNA causes heterochromatin erosion and is a target for\\namelioration of senescent phenotypes in progeroid syndromes. <em>Sci\\nTransl Med</em> 14: eabl6057</p>\\n<p>Varanasi SS, Olstad OK, Swan DC, Sanderson P, Gautvik VT, Reppe S,\\nFrancis RM, Gautvik KM &amp; Datta HK (2010) Skeletal Site-Related\\nVariation in Human Trabecular Bone Transcriptome and Signaling. <em>PLoS\\nOne</em> 5: e10692</p>\\n<p>Wang S, Lee C, Kim J, Hyun J, Lim M, Cha H-J, Oh S-H, Choi YH &amp;\\nJung Y (2017) Tumor necrosis factor-inducible gene 6 protein ameliorates\\nchronic liver damage by promoting autophagy formation in mice. <em>Exp\\nMol Med</em> 49: e380</p>\\n<p>Weckbach S, Hohmann C, Braumueller S, Denk S, Klohs B, Stahel PF,\\nGebhard F, Huber-Lang MS &amp; Perl M (2013) Inflammatory and apoptotic\\nalterations in serum and injured tissue after experimental polytrauma in\\nmice: distinct early response compared with single trauma or\\n‘double-hit’ injury. <em>J Trauma Acute Care Surg</em> 74: 489–498</p>\\n<p>Wheatley BM, Nappo KE, Christensen DL, Holman AM, Brooks DI &amp;\\nPotter BK (2019) Effect of NSAIDs on Bone Healing Rates. <em>J Am Acad\\nOrthop Surg</em> 27: e330–e336</p>\\n<p>Wiśniewski JR, Zougman A, Nagaraj N &amp; Mann M (2009) Universal\\nsample preparation method for proteome analysis. <em>Nat Methods</em> 6:\\n359–362</p>\\n<p>Xiao G, Gopalakrishnan R, Jiang D, Reith E, Benson MD &amp;\\nFranceschi RT (2002a) Bone morphogenetic proteins, extracellular matrix,\\nand mitogen-activated protein kinase signaling pathways are required for\\nosteoblast-specific gene expression and differentiation in MC3T3-E1\\ncells. <em>J Bone Miner Res</em> 17: 101–110</p>\\n<p>Xiao G, Jiang D, Gopalakrishnan R &amp; Franceschi RT (2002b)\\nFibroblast growth factor 2 induction of the osteocalcin gene requires\\nMAPK activity and phosphorylation of the osteoblast transcription\\nfactor, Cbfa1/Runx2. <em>J Biol Chem</em> 277: 36181–36187</p>\\n<p>Xiao G, Jiang D, Thomas P, Benson MD, Guan K, Karsenty G &amp;\\nFranceschi RT (2000) MAPK Pathways Activate and Phosphorylate the\\nOsteoblast-specific Transcription Factor, Cbfa1 *. <em>J Biol Chem</em>\\n275: 4453–4459</p>\\n<p>Yang A, Lu Y, Xing J, Li Z, Yin X, Dou C, Dong S, Luo F, Xie Z, Hou\\nT, <em>et al</em> (2018a) IL-8 Enhances Therapeutic Effects of BMSCs on\\nBone Regeneration via CXCR2-Mediated PI3k/Akt Signaling Pathway.\\n<em>Cell Physiol Biochem</em> 48: 361–370</p>\\n<p>Yang H, Tian W, Wang S, Liu X, Wang Z, Hou L, Ge J, Zhang X, He Z\\n&amp; Wang X (2018b) TSG-6 secreted by bone marrow mesenchymal stem\\ncells attenuates intervertebral disc degeneration by inhibiting the\\nTLR2/NF-κB signaling pathway. <em>Lab Investig</em> 98: 755–772</p>\\n<p>Yang WR, Ardeljan D, Pacyna CN, Payer LM &amp; Burns KH (2019) SQuIRE\\nreveals locus-specific regulation of interspersed repeat expression.\\n<em>Nucleic Acids Res</em> 47: e27–e27</p>\\n<p>Zhang R, Liu Y, Yan K, Chen L, Chen X-R, Li P, Chen F-F &amp; Jiang\\nX-D (2013) Anti-inflammatory and immunomodulatory mechanisms of\\nmesenchymal stem cell transplantation in experimental traumatic brain\\ninjury. <em>J Neuroinflammation</em> 10: 106</p>\\n<p>Zhang X, Zhang R &amp; Yu J (2020) New Understanding of the Relevant\\nRole of LINE-1 Retrotransposition in Human Disease and Immune\\nModulation. <em>Front cell Dev Biol</em> 8: 657</p>\\n<p>Zhu W, Kuo D, Nathanson J, Satoh A, Pao GM, Yeo GW, Bryant S V., Voss\\nSR, Gardiner DM &amp; Hunter T (2012) Retrotransposon long interspersed\\nnucleotide element-1 (LINE-1) is activated during salamander limb\\nregeneration. <em>Dev Growth Differ</em> 54: 673–685</p>\\n<p><strong>Figure Legends</strong></p>\\n<p><strong>Figure 1 - TEs expression is induced after fracture and in\\nmechanically loaded bone.</strong></p>\\n<p><strong>A)</strong> Heatmap representation of RNAseq differentially\\nexpressed TEs (LogFC&gt;0.5) between 4h post-injury and intact femurs at\\ndifferent time points of the bone healing process. N=5 biological\\nreplicates for each time point. N=10 biological replicates for intact\\nfemur. <strong>B)</strong> Heatmap representation of RNAseq upregulated\\nTEs (LogFC&gt;0.5) between 4h and intact at different time points of the\\nbone healing process. N=5 biological replicates for each time point.\\nN=10 biological replicates for intact femur. <strong>C)</strong> Heatmap\\nrepresentation of RNAseq differentially expressed TEs analysis\\n[Fragments Per Kilobase of transcript per Million (FPKM) fold change] in\\nfemoral (n=27) and iliac (n=34) bone biopsies from healthy donors.\\n<strong>D)</strong> Pie charts showing the number and percentage of\\ndifferentially expressed TEs subfamilies between femoral and iliac\\nhealthy bone. <strong>E)</strong> Heatmap representation of RNAseq\\ndifferentially expressed TEs analysis [Fragments Per Kilobase of\\ntranscript per Million (FPKM) fold change] in femoral (n=27) and iliac\\n(n=34) bone biopsies from healthy donors. One heatmap for each TEs order\\nis shown.</p>\\n<p><strong>Figure 2 - TEs expression correlates with bone mineral\\ndensity in human weight-bearing bone.</strong></p>\\n<p><strong>A)</strong> Heatmap representation of RNAseq differentially\\nexpressed TEs analysis [Fragments Per Kilobase of transcript per Million\\n(FPKM) fold change] in femoral bone from healthy (n=27), osteopenic\\n(n=12), and osteoporotic (n=9) donors. <strong>B)</strong> Heatmap\\nrepresentation of RNAseq differentially expressed TEs analysis\\n[Fragments Per Kilobase of transcript per Million (FPKM) fold change] in\\nfemoral bone with high BMD (FN T-score &gt;-1) (n=27) and low BMD (FN\\nT-score &lt;-1) (n=21). <strong>C)</strong> Upper panels: pie charts\\nshowing the number and percentage of differentially expressed TEs\\nsubfamilies between high BMD and low BMD femoral bone. Lower panels:\\nheatmap representation of RNAseq differentially expressed TEs analysis\\n[Fragments Per Kilobase of transcript per Million (FPKM) fold change] in\\nfemoral bone with high BMD (n=27) and low BMD (n=21). One heatmap for\\neach TEs order is shown. <strong>D)</strong> Upper panels: Heatmap\\nrepresentation of correlation analysis (<em>P</em> value&lt;0.05)\\nbetween TEs expression (FPKM) and local BMD (FN T-score) in femoral bone\\nbiopsies (n=48). Correlation between TE expression (FPKM values) and FN\\nt-score was evaluated using Pearson method. Significant correlations\\n(p-value &lt; 0.05) were selected for the heatmap representation and\\narranged according to the Pearson correlation coefficient (r). r &gt; 0,\\npositive correlations; r &lt; 0, negative correlations. Lower panels:\\nPie charts showing the percentage of TEs subfamilies positively\\ncorrelated to local BMD.</p>\\n<p><strong>Figure 3 - L1 RNA delivery stimulates the mineralization of\\ndifferentiating osteoblasts</strong></p>\\n<p><strong>A)</strong> Experimental workflow and flow cytometer analysis\\nshowing the percentage of positive cells 6 hours after L1 RNA delivery\\nat day 5 of <em>ex vivo</em> osteogenesis. Intracellular localization of\\ncy5 conjugated synthetic L1 RNA (red spots) three days after\\ntransfection is also shown from a typical experiment (right).\\n<strong>B)</strong> qPCR analysis of intracellular L1 RNA level 24 h\\n(d6), 5 days (d10), and 9 days (d14) post-transfection. The graph is\\nshown as mean\\u2009±\\u2009s.d. of n\\u2009=3 independent experiments.\\n<strong>C)</strong> cy5 conjugated synthetic L1 RNA (red) and osteoimage\\nstained mineral matrix (green) detection 9 days post L1 RNA\\ntransfection. The white arrowheads indicate the colocalization between\\nL1 RNA and hydroxyapatite. <strong>D)</strong> Osteoimage stained\\nmineral matrix quantification (upper panels), images (central panels),\\nand Alizarin Red (lower panels) 9 days after L1 RNA or RFP RNA delivery\\nin 2 healthy donors-derived MSCs (D188 and D170). RFU= Relative\\nFluorescence Units. The graph is shown as mean\\u2009±\\u2009s.d. of n\\u2009=\\u200911\\ntechnical replicates. ****P\\u2009&lt;\\u20090.00005 in Wilcoxon test.\\n<strong>E)</strong> Alizarin Red staining of osteoblasts transfected\\nwith increasing doses of RFP RNA (upper panels) and L1 RNA (lower\\npanels). <strong>F)</strong> Alizarin Red images (upper panel) and\\nquantification (lower panel) of MSCs mineralization after 14, 17 and 21\\ndays of <em>ex vivo</em> differentiation. MSCs were obtained from the\\nfemur of 4 healthy (D188, D239, D247, D170) and 4 OP patients (HUK7,\\nHUK9, HUK12, HUK16). N=9 technical replicates for each donor and time\\npoint. Boxes in the boxplot indicate the interquartile range (50% of\\ndata), while the lower end and the upper end represent the\\n1<sup>st</sup> and the 3<sup>rd</sup> quartile, respectively. The solid\\nline inside the box represents the median. Whiskers represent the max\\nand min values. Values that are not within 1.5 times the interquartile\\nrange are considered as outliers and lie outside the whiskers.\\n<strong>G)</strong> Osteoimage stained mineral matrix quantification\\n(upper panels), images (central panels), and Alizarin Red (lower panels)\\n9 days after L1 RNA or RFP RNA delivery in 3 OP patients derived MSCs\\n(HUK9, HUK12, HUK16). RFU= Relative Fluorescence Units. The graph is\\nshown as mean\\u2009±\\u2009s.d. of n\\u2009=\\u200910-12 technical replicates.\\n****P\\u2009&lt;\\u20090.00005 in Wilcoxon test. <strong>H)</strong> qRT-PCR of\\nearly osteogenic genes in RFP and L1 transfected osteoblasts at\\ndifferent time points of osteogenic differentiation. Expression level is\\nnormalized on day 5 (not transfected osteoblasts). N=3 biological\\nreplicates. The graph is shown as mean\\u2009±\\u2009s.d. of n\\u2009=\\u20093 independent\\nexperiments. *P\\u2009&lt;\\u20090.05; **P\\u2009&lt;\\u20090.005, ***P\\u2009&lt;\\u20090.0005,\\n****P\\u2009&lt;\\u20090.00005 in Student’s t test.</p>\\n<p><strong>Figure 4 - L1 RNA delivery induces an inflammatory response\\ncharacteristic of bone repair process</strong></p>\\n<p><strong>A)</strong> Left panel: heatmap representation of RNAseq\\ndifferentially expressed gene analysis [Fragments Per Kilobase of\\ntranscript per Million (FPKM) fold change] in osteoblasts transfected\\nwith negative control RNA (RFP) or L1 RNA. Right panel: bubble plot\\nshowing gene ontology (GO) enrichment analysis of L1 upregulated\\nbiological processes. <strong>B)</strong> Tree plot showing gene\\nontology (GO) enrichment analysis of L1 upregulated biological processes\\n(left) and cellular components (right). <strong>C)</strong> Top 25\\nupregulated biological processes 24h post-L1 RNA delivery <em>in\\nvitro</em> (yellow) and 4h post fracture <em>in vivo</em> (blue). Shared\\nGO terms are shown in green. <strong>D)</strong> qPCR analysis of\\n“inflammatory response” (GO:0006954) and “immune response” (GO:0006955)\\ngenes at different time points post L1 RNA transfection at day 5. *\\n**P\\u2009&lt;\\u20090.005, ****P\\u2009&lt;\\u20090.00005 in two way ANOVA.</p>\\n<p><strong>Figure 5 - PKR mediates L1 RNA-induced stress response and\\nmineralization</strong></p>\\n<p><strong>A)</strong> Upper panel: schematic representation of the\\npathway inhibited by Lamivudine 3TC and G140. Lower panel: Alizarin Red\\nstaining of differentiating osteoblasts transfected with L1 RNA and\\ntreated with Lamivudine 3TC and G140. <strong>B)</strong> IF on RFP and\\nL1 transfected osteoblasts, 6h post-transfection, shows the\\ncolocalization between cy5 signal (L1 RNA, red) and 488-Anti-dsRNA\\nsignal (dsRNA, green). Nuclei are stained with Hoechst (blue).</p>\\n<p><strong>C)</strong>Western blot analysis showing the ratio between\\ntotal eIF2α and phosphorylated eIF2α\\xa0(P- eIF2α). H3: endogenous standard\\nHistone 3. <strong>D)</strong> Volcano plot (left) showing the number of\\nsignificantly upregulated and downregulated protein 24h post-L1 RNA\\ntransfection. Right: tree plots showing gene ontology (GO) enrichment\\nanalysis of biological processes (upper panel) and cellular components\\n(lower panel) downregulated by L1 RNA delivery (MS data).\\n<strong>E)</strong> qPCR analysis of “inflammatory response”\\n(GO:0006954) and “immune response” (GO:0006955) genes 24h post L1 RNA\\ntransfection with and without PKR inhibitor C16. <strong>F)</strong>\\nAlizarin Red staining of differentiating osteoblasts transfected with L1\\nRNA with and without PKR inhibitor C16. <strong>G)</strong> Tree plots\\nshowing gene ontology (GO) enrichment analysis of biological processes\\nupregulated by L1 RNA 24h after transfection (MS data).</p>\\n<p><strong>Figure 6 - L1-RNA-induced changes in osteoblast\\nsecretome</strong></p>\\n<p><strong>A)</strong> Left: Alizarin red staining on recipient OB 9\\ndays after the delivery of conditioned media. Right: Microscope images\\nof recipient OB 24h from the delivery of conditioned media. Mineralized\\nnodules in OB receiving conditioned media from L1 treated OB are shown\\n(orange arrows). <strong>B)</strong> Heat map of differentially\\nexpressed proteins in bulk secretome and exosomes derived from\\nuntransfected (NT), RFP- and L1-transfected osteoblasts. N=3 biological\\nreplicates. <strong>C)</strong> GO enrichment analysis of differentially\\nexpressed protein (adjusted p-value &lt;0.05) in the bulk secretome of\\nL1 compared to RFP transfected osteoblasts. <strong>D)</strong> GO\\nenrichment analysis of differentially expressed protein (adjusted\\np-value &lt;0.05) in the exosomes of L1 compared to RFP transfected\\nosteoblasts.</p>\\n'"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pypandoc\n",
    "\n",
    "pypandoc_html = pypandoc.convert_file(DOCX_FILES[0], 'html')\n",
    "pypandoc_html"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'***LINE-1 RNA triggers matrix formation in bone cells via a PKR-mediated\\ninflammatory response***\\n\\nArianna Mangiavacchi,^1@^ Gabriele Morelli,^1^ Sjur Reppe,^2,3,4^\\nAlfonso Saera-Vila,^5^ Peng Liu,^1^ Benjamin Eggerschwiler,^6,7^ Huoming\\nZhang,^8^ Dalila Bensaddek,^8^ Elisa A. Casanova,^5^ Carolina Medina\\nGomez,^9^ Vid Prijatelj,^9^ Francesco Della Valle,^1^ Nazerke\\nAtinbayeva,^1^ Juan Carlos Izpisua Belmonte,^10^ Fernando\\nRivadeneira,^9^ Paolo Cinelli,^5,11^ Kaare Morten Gautvik,^3\\\\*^ Valerio\\nOrlando.^1\\\\*@^\\n\\n^1^ King Abdullah University of Science and Technology (KAUST),\\nBiological Environmental Science and Engineering Division, Thuwal\\n23500-6900, Kingdom of Saudi Arabia\\u200b\\n\\n^2^ Oslo University Hospital, Department of Medical Biochemistry, Oslo,\\nNorway.\\n\\n^3^Lovisenberg Diaconal Hospital, Unger-Vetlesen Institute, Oslo,\\nNorway.\\n\\n^4^Oslo University Hospital, Department of Plastic and Reconstructive\\nSurgery, Oslo, Norway.\\n\\n^5^ Sequentia Biotech, Carrer Comte D\\\\\\'Urgell 240, Barcelona, 08036,\\nSpain.\\n\\n^6^ Department of Trauma, University Hospital Zurich, Sternwartstrasse\\n14, 8091, Zurich, Switzerland\\n\\n^7^ Life Science Zurich Graduate School, University of Zurich,\\nWinterthurerstrasse 190, 8057, Zurich, Switzerland.\\n\\n^8^ Core Labs, King Abdullah University of Science and Technology\\n(KAUST), Thuwal 23500-6900, Kingdom of Saudi Arabia\\u200b.\\n\\n^9^ Department of Internal Medicine, Erasmus Medical Centre, Rotterdam,\\nthe Netherlands.\\n\\n^10^ Altos Labs, San Diego, CA, USA\\n\\n^11^ Center for Applied Biotechnology and Molecular Medicine, University\\nof Zurich, Winterthurerstrasse 190, 8057, Zurich, Switzerland.\\n\\n\\\\*senior author\\n\\n^@^corresponding author\\n\\n**Abstract**\\n\\nTransposable elements (TEs) are mobile genetic modules of viral\\nderivation that have been co-opted to become modulators of mammalian\\ngene expression. TEs are a major source of endogenous dsRNAs, signaling\\nmolecules able to coordinate inflammatory responses in various\\nphysiological processes. Here, we provide evidence for a positive\\ninvolvement of TEs in inflammation-driven bone repair and\\nmineralization. In newly fractured mice bone, we observed an early\\ntransient upregulation of repeats occurring concurrently with the\\ninitiation of the inflammatory stage. In human bone biopsies, analysis\\nrevealed a significant correlation between repeats expression,\\nmechanical stress and bone mineral density. We investigated a potential\\nlink between LINE-1 (L1) expression and bone mineralization by\\ndelivering a synthetic L1 RNA to osteoporotic patient-derived\\nmesenchymal stem cells and observed a dsRNA-triggered protein kinase\\n(PKR)-mediated stress response that led to strongly increased\\nmineralization. This response was associated with a strong and transient\\ninflammation, accompanied by a global translation attenuation induced by\\neIF2α phosphorylation. We demonstrated that L1 transfection reshaped the\\nsecretory profile of osteoblasts, triggering a paracrine activity that\\nstimulated the mineralization of recipient cells.\\n\\n**Keywords:** dsRNA/inflammation/osteoblast /PKR/transposable elements\\n\\n**Introduction**\\n\\nRepetitive elements are, in origin, parasitic transposable elements\\n(TEs) of viral derivation and during evolution have become regulatory\\nmodules of gene expression, eventually integrated into developmental,\\nresilience, and cell identity programs.(Cosby *et al*, 2019)\\n(Mangiavacchi *et al*, 2021) (Jachowicz *et al*, 2017) Although cells\\nhave evolved several defense mechanisms to prevent deleterious\\nuncontrolled retrotransposon reactivation,(Slotkin & Martienssen, 2007)\\nincreasing evidence demonstrates that retrotransposons are also involved\\nin non-pathological contexts, particularly as non-coding\\nRNAs.(Mangiavacchi *et al*, 2021) As part of resilience mechanisms, all\\nretrotransposons are a major source of endogenous double-stranded RNAs\\n(dsRNAs), that serve as a cellular signaling molecule to coordinate\\ninflammatory and immune responses in various physiological\\nprocesses.(Fukuda *et al*, 2021) (De Cecco *et al*, 2019a) (Simon *et\\nal*, 2019) (Sadeq *et al*, 2021) (Chen & Hur, 2022) (Zhang *et al*,\\n2020) The transient and controlled breaching of the activation threshold\\nof dsRNA sensors allows the integration of innate immune functions\\nwithin the frame of physiological processes.(Chen & Hur, 2022) For\\nexample, repetitive elements transcribed during development drive\\nRIG-I-like receptors (RLR)-mediated inflammatory signals that regulate\\nhematopoietic stem and progenitor cells (HSPC) formation.(Lefkopoulos\\n*et al*, 2020) During cell replication, dsRNA-mediated activation of PKR\\nensures proper progression of mitosis.(Kim *et al*, 2014) Moreover,\\nepigenetically mediated derepression of repetitive elements leads to\\ndsRNA production and to the activation of an inflammatory response\\nstimulating anti-tumor T cell immunity.(Sheng *et al*, 2018)\\nFurthermore, after skin injury, dsRNAs induce inflammatory pathways\\ncontributing to wound healing and hair regeneration.(Nelson *et al*,\\n2015) Several studies have demonstrated a positive effect of\\nproinflammatory mediators on bone anabolism *in vitro* and *in\\nvivo*,(Croes *et al*, 2015) (Croes *et al*, 2016) (Li *et al*, 2016)\\n(Croes *et al*, 2017) (Laroche *et al*, 2011) suggesting that an\\ninflammatory reaction may be harnessed for bone regenerative purposes.\\nIndeed, sterile inflammation is a fundamental component of the\\nosteogenic microenvironment: the inflammatory reaction is the earliest\\nresponse to injury and is crucial to initiate and orchestrate fracture\\nrepair and activate bone anabolic processes. (Mountziaris *et al*, 2011)\\n(Marsell & Einhorn, 2011) (Bahney *et al*, 2019) Accordingly, a\\ndysregulated inflammation negatively impacts optimal bone regeneration,\\n(Wheatley *et al*, 2019) (Gerstenfeld *et al*, 2001) (Recknagel *et al*,\\n2013) (Hurtgen *et al*, 2016) (Jiao *et al*, 2015) and often precedes\\nthe excessive bone formation associated with heterotopic ossification\\nand several vertebral column pathological conditions. (Lories & Schett,\\n2012) (Balboni *et al*, 2006) Therefore, identifying the key molecular\\nfactors triggering the sterile inflammation that precedes regeneration\\nin bone would allow us to modulate the inflammatory reaction in a\\nspatiotemporal and contextual manner as well as to develop new anabolic\\nstrategies for the treatment of bone loss conditions, such as\\nosteoporosis, or impaired bone repair.(Roberts & Ke, 2018)\\n\\nHere, we show that 1) In mice, immediately after bone fracture, a\\ntransient upregulation of TEs concurs with the initiation of the\\ninflammatory stage; 2) in humans, an increased expression of repeats is\\nobserved in bones with a higher mechanical stress-induced anabolic\\ndemand and expression of TEs correlates with local mineral density; 3)\\nThe delivery of L1 RNA, but not control RNA, to human bone\\nmarrow-derived mesenchymal stem cells committed to osteoblasts\\nstimulates a unique mineralizing phenotype in a dose-dependent manner;\\n4) L1-treated osteoblasts show upregulation of inflammatory genes and\\ntranscriptional changes characteristic for the earliest stage of bone\\nrepair; 5) cytoplasmic L1 RNA is sensed by dsRNA-activated protein\\nkinase (PKR), which mediates eIF2α phosphorylation and a global\\nattenuation of protein synthesis; 6) the inhibition of PKR prevents\\ninflammation, inhibits translation and induces mineralization caused by\\nelevated levels of cytoplasmic L1 RNA; 7) cells transfected with L1 RNA\\nundergo significant changes in their secretome composition and initiate\\na paracrine effect stimulating the mineralization of osteogenic\\ncompetent recipient cells.\\n\\nOur results indicate that TEs are induced by bone damage and stimulate\\nbone mineralization through an inflammatory response triggered by dsRNA\\nsensing and mediated by a paracrine activity.\\n\\n**Results**\\n\\n**Retrotransposon expression is triggered by fracture *in vivo* and\\ncorrelates with bone mineral density in human weight-bearing bones**\\n\\nInflammation is the earliest response in fracture healing. To\\ninvestigate a possible involvement of TEs in the sterile inflammatory\\nresponse triggered by bone injury, we analyzed TEs expression in\\ntime-course RNA-Seq repository data obtained from a full-fracture bone\\nhealing mice model.(Coates *et al*, 2019) We focused on TEs expression\\ndynamics immediately after fracture (acute inflammatory stage) and\\nobserved four clusters of differentially expressed TEs (logFC \\\\> 0.5)\\nbetween intact and post-fracture bone after 4h, a time point\\ncorresponding to the initiation of acute inflammation (Figure 1A).\\nCluster 1 includes TEs subfamilies whose expression slightly increases\\nafter fracture and during the whole healing process. Clusters 2 and 4\\ninclude TEs subfamilies that are downregulated after fracture but show\\nincreased expression at later time points. The major group, cluster 3,\\nis represented by TEs subfamilies whose expression is highly and\\ntransiently upregulated after fracture (Figure 1A and 1B).\\nInterestingly, the induction of these TEs, mostly Long Interspersed\\nNuclear Elements (LINEs) and Long Terminal Repeats (LTRs) (Figure 1B),\\nis transient and limited to the earliest phase of the inflammatory stage\\nnecessary and sufficient to initiate and orchestrate fracture repair and\\nactivate bone anabolic processes.(Mountziaris *et al*, 2011)^,^(Marsell\\n& Einhorn, 2011) These results suggest that, in bone, retrotransposon\\nreactivation is an early event in response to fracture, possibly linked\\nto inflammation initiation which is fundamental for a healthy healing.\\nIndeed, retrotransposon involvement in posttraumatic tissue regeneration\\nwas previously found in other organisms.(Mashanov *et al*, 2012)^,^(Zhu\\n*et al*, 2012)\\n\\nTo assess gene expression dynamics immediately after fracture in humans\\nrepresents a technical and ethical hurdle. A comparison between bones\\nexperiencing different degrees of mechanical loading may represent an\\nalternative model to study stress-induced bone anabolism in humans.\\nMechanically loaded bones like the femur, which are subject to recurring\\nmicrofractures that need to be repaired, are indicated to be more\\nmetabolically active than less loaded bones, like the iliac crest, as\\npointed out by previous transcriptome studies.(Aerssens *et al*, 1997)\\n(Varanasi *et al*, 2010)\\n\\nWe compared the expression of TEs in trabecular bone from ilium and\\nfemoral head, two different skeletal sites, the latter experiencing a\\nhigher degree of mechanical loading, anabolic demand and bone\\nturnover.(Aerssens *et al*, 1997) Femur (n=48) and ilium biopsies (n=71)\\nwere isolated from a cohort of clinically well-characterized donors\\n(refer to cohort description and Appendix Table S1 and S2). Each cohort\\nwas divided into normal (BMD T-score \\\\>-1), osteopenic (−2.5 \\\\< BMD\\nT-score ≤ −1), and osteoporotic (BMD T-score ≤ −2.5, with at least one\\nfragility fracture). TEs show generally higher upregulation in healthy\\nfemurs (n=27) compared to healthy ilia (n=34) (Figure 1C and 1E). Each\\nTEs order showed a high percentage of upregulated subfamilies, from 60%\\nof DNA transposon to 75% of LINEs, and a low percentage of downregulated\\nsubfamilies, from 9% of LTRs to 3% of Short interspersed nuclear\\nelements (SINEs) and LINEs, in femur compared to ilium (Figure 1D).\\nThese results suggested a positive correlation between TEs global\\nexpression and bone anabolic activity.\\n\\nTo further test our hypothesis, we compared TEs expression in femurs\\nfrom healthy donors (n=27), osteopenic (n=12) and osteoporotic patients\\n(n=9) whose bone anabolism is increasingly compromised. As shown in\\nFigure 2A, the global expression of TEs is reduced in patients. To\\nfurther assess whether TEs expression is involved in local anabolism and\\nassociated with mineral density, we divided the femur cohort according\\nto the DXA T-score measured in the femoral neck (FN). We divided the\\ncohort into \"high BMD\" (FN T-score \\\\>-1) and \"low BMD\" (FN T-score\\n\\\\<-1). Globally, TEs are upregulated in femurs with high bone mineral\\ndensity (Figure 2B). In particular, more than 90% of differentially\\nexpressed TEs subfamilies are upregulated (Figure 2C). Interestingly,\\nalmost all SINE elements are unchanged between the two groups (Figure\\n2C). Finally, we assessed whether a positive correlation exists between\\nTEs expression levels and local mineral density (FN T-score). We found\\nthat approximately 30% of L1 and LTR subfamilies expressed in the\\nfemoral bone are positively correlated with local BMD (Figure 2D).\\nAltogether, these results strongly suggest a positive involvement of TEs\\nin response to stress-induced anabolic demand and bone mineralization.\\n\\n**Increased cytoplasmic L1 repeat RNA stimulates osteoblast\\nmineralization activity**\\n\\nThe *in vivo* evidence suggested a positive involvement of\\nretrotransposons in post-traumatic bone repair in mice, and a strong\\ncorrelation between retrotransposon expression and bone mineral density\\nin humans, and higher expression in loaded compared to unloaded bone. We\\nthen corroborated the results by studying the effects of increasing\\nrepeat RNA levels on differentiating osteoblasts derived from\\nmesenchymal stem cells (MSCs) isolated from femurs of healthy donors. At\\nday 5 of differentiation, a Cy5-conjugated full-length L1 RNA consensus\\nsequence was transfected. Capping, 2\\'-O-Methylation of 5\\' end,\\npolyadenylation (200 adenosines), full substitution with\\n5-methylcytidine (m5C) and 75% substitution with pseudouridine were used\\nto stabilize the RNA and to bypass the intracellular innate immune\\nsystem.(Koski *et al*, 2004) (Pardi *et al*, 2013) (Ludwig *et al*,\\n2010) (Karikó *et al*, 2005) (Nallagatla & Bevilacqua, 2008) (Karikó *et\\nal*, 2008) (Anderson *et al*, 2010) (Kormann *et al*, 2011) Red\\nfluorescent protein (RFP) mRNA, with the same modifications, was\\nincluded as negative control. The exogenous L1 RNA accumulated in the\\ncytoplasm (Figure 3A) and was gradually cleared by secretion along with\\nmatrix components (Figure 3B and 3C). The effect of L1 RNA delivery was\\ntested in differentiating MSCs from two different and unrelated healthy\\ndonors. As shown, L1 RNA transfection markedly stimulated the production\\nof mineralized matrix (Figure 3D). Osteoblast mineralization increased\\ndose-dependently with the concentration of the delivered L1 RNA, in\\nagreement with saturation kinetics (Figure 3E). Moreover, the strongly\\nenhanced mineralization was highly specific for L1 RNA, as demonstrated\\nby the lack of significant changes in mineral deposition in cells\\ntransfected with negative control RNA, even at concentrations\\n100-200-fold higher (figure 3E). We also isolated MSCs from the femur of\\nosteoporotic patients and selected those showing a markedly delayed and\\nreduced *in vitro* production of bone matrix in contrast to those\\nderived from healthy donors (Figure 3F). Strikingly, also in these cells\\nwith low osteogenic capacity, the delivery of L1 RNA triggered a strong\\nmineralization response despite their clearly compromised anabolic\\nactivity (Figure 3G). These results demonstrate convincingly that the\\nectopic delivery of L1 RNA stimulates the mineralization of bone-forming\\ncells regardless of their prior intrinsic differentiation/anabolic\\npotential.\\n\\nOsteoporotic patient-derived cells transfected with L1 RNA show a higher\\nexpression of early differentiation markers, with a peak 48h after L1\\ntransfection (day 7) (Figure 3H). Bone sialoprotein (*IBSP*), a late\\nosteogenic gene, shows a similar profile, being almost 6 times more\\nexpressed in L1- than in RFP-transfected cells on day 7 (Appendix Figure\\nS1A). Strikingly, although alkaline phosphatase (*ALPL*) expression\\nslightly increases at later time points (Appendix Figure S1A), its\\nactivity is not enhanced in L1-treated cells (Appendix Figure S2A).\\nHowever, the activity of ectonucleotide\\npyrophosphatase/phosphodiesterase 1 (ENPP1) is gradually and\\nsignificantly reduced by L1 RNA transfection (Appendix Figure S2B). As\\nENPP1 is a major source of inorganic pyrophosphate (PPi), one of the\\nmain physiological inhibitors of mineralization,(FLEISCH *et al*, 1966)\\n(Johnson *et al*, 2000) (Hessle *et al*, 2002) the reduction of its\\nactivity may contribute to the enhanced mineralization observed in\\nL1-RNA transfected cells.\\n\\nAltogether, these data suggest that L1 RNA induces a unique mineralizing\\nphenotype and apparently distinct from the well-known canonical\\ndifferentiation mechanisms.\\n\\n**L1 RNA delivery induces inflammatory pathways significantly\\noverlapping those involved in bone fracture repair**\\n\\nTo investigate the molecular response of osteoblasts to L1 RNA\\ntransfection, we profiled their transcriptome by Illumina RNA-Seq 24h\\npost-transfection. We found 482 differentially expressed genes (DEG)\\n(FDR\\\\<0.05) between L1 and RFP RNA-treated OB (268 upregulated and 214\\ndownregulated) (Appendix Figure S3). Gene ontology (GO) enrichment\\nanalysis revealed that the early transcriptional signature of L1-treated\\nosteoblasts is typical of that of an inflammatory response (Figure 4A\\nand 4B). This is in accordance with the recent evidence of a link\\nbetween L1 and other retrotransposons accumulation in the cytoplasm and\\ninflammatory response *in vitro* and *in vivo*.(Fukuda *et al*, 2021)\\n(De Cecco *et al*, 2019b) (Simon *et al*, 2019) GO term enrichment\\nanalysis comparison between the early stage of fracture-induced bone\\nhealing *in vivo*(Coates *et al*, 2019) and differentiating osteoblasts\\n*in vitro* 24h post-L1 delivery (Figure 4A) revealed a matching\\ninflammatory response and shared upregulated pathways (Figure 4C)\\nessential for effective bone repair.(Maruyama *et al*, 2020)\\n(Gerstenfeld *et al*, 2003) (Kon *et al*, 2001) Recruitment of\\nneutrophils at the site of injury also indicates a healthy repair\\nprocess, particularly in bone.(Kovtun *et al*, 2016) Extracellular\\nsignal-regulated kinase 1 (ERK1) and 2 (ERK2), play crucial roles in\\nbone formation(Xiao *et al*, 2000) (Xiao *et al*, 2002b) (Xiao *et al*,\\n2002a) (Matsushita *et al*, 2009) (Ge *et al*, 2007) and seem to be the\\nmost upstream initiators of tissue regeneration in planaria.(Owlarn *et\\nal*, 2017) Upon bone injury, an initial transient stage of acute\\ninflammation is a crucial factor in ensuring an effective\\nregeneration(Maruyama *et al*, 2020) whereas an excessive/prolonged\\n(chronic) inflammation is deleterious for the healing environment.(Chan\\n*et al*, 2012) (Hurtgen *et al*, 2016) (Clark *et al*, 2017) (Brem &\\nTomic-Canic, 2007) (Timmen *et al*, 2014) (Schmidt-Bleek *et al*, 2012)\\n(Weckbach *et al*, 2013) We followed the time-course expression of\\n\"inflammatory response\" (GO:0006954) and \"immune response\" (GO:0006955)\\ngenes upregulated by L1 RNA immediately after transfection. As shown in\\nFigure 4D, their expression is already strongly reduced at day 10 and\\nalmost entirely silenced by day 14, suggesting the transient dynamics of\\nthe inflammatory response triggered *in vitro* by L1 RNA. This is\\ncoherent with our previous observation that exogenous L1 RNA is\\ngradually secreted by the cell together with matrix components (Figure\\n3B and C), eventually clearing out the initial inflammatory stimulus.\\n\\n**Cellular response to increased cytoplasmic level of L1 RNA is mediated\\nby PKR**\\n\\nIt has been previously demonstrated that the cytoplasmic accumulation of\\nL1-derived cDNA triggers an inflammatory response mediated by the cyclic\\nGMP-AMP synthase (cGAS)-stimulator of interferon genes (STING) cytosolic\\nDNA sensing pathway.(De Cecco *et al*, 2019a) Immunofluorescence (IF)\\nanalysis reveals that the exogenous L1 RNA forms DNA:RNA hybrids\\n(Appendix Figure S4), suggesting that L1 RNA is reversed transcribed\\ninto cDNA after its transfection. Notably, ORF1p was undetectable by WB\\nin differentiating osteoblasts and in L1 transfected osteoblasts\\n(Appendix Figure S5), indicating that the exogenous L1 RNA is not\\ntranslated after transfection. To assess whether L1-induced inflammation\\nwas triggered by the sensing of its cDNA, we used Lamivudine 3TC and\\nG140 (Figure 5A) to inhibit the ORF2-mediated reverse transcription of\\nL1 RNA and cGAS activity, respectively. Moreover, we also performed a\\nsiRNA-mediated knockdown (KD) of cGAS (Appendix Figure S6). (Jones *et\\nal*, 2008) (Lama *et al*, 2019) Surprisingly, none of the treatments\\nimpaired the mineralization induced by L1 RNA delivery (Figure 5A,\\nAppendix Figure S6C), suggesting that the observed cellular response\\ndepends entirely on RNA sensing. As previously mentioned, repeats are a\\nmajor source of endogenous dsRNA,(Sadeq *et al*, 2021) (Chen & Hur,\\n2022) and L1 RNA, in particular, is able to form intramolecular\\ndouble-stranded structures.(Hur, 2019) As expected, we found a tight\\ncolocalization between L1 RNA and dsRNA IF signal (Figure 5B). One of\\nthe main cellular sensors of dsRNA, including those derived from\\nretrotransposons, is PKR.(Kim *et al*, 2018) PKR activation results in\\nthe phosphorylation of eukaryotic translation initiation factor 2α\\n(eIF2α) and in the subsequent inhibition of global protein synthesis and\\ncell growth.(Donnelly *et al*, 2013) Interestingly, PKR has been shown\\nto interact with L1 RNA *in vitro*.(Kim *et al*, 2018) Moreover, eIF2α\\nphosphorylation has been demonstrated to promote autophagy in\\nosteoblasts and to counteract BMD reduction in osteoporotic\\novariectomized mice.(Li *et al*, 2019)\\n\\nTo demonstrate that L1 RNA is able to trigger a PKR-mediated\\nintracellular response, we transfected L1 RNA in the presence or absence\\nof the PKR inhibitor C16 and evaluated the levels of eIF2α\\nphosphorylation by western blot (Figure 5C). The delivery of L1, but not\\nRFP, strongly induces eIF2α phosphorylation and this effect is prevented\\nby PKR inhibition (Figure 5C). Notably, eIF2α phosphorylation occurs\\nalready 6 hours post-transfection. This observation suggests that PKR\\nactivation is one of the first events induced by L1 RNA, thus\\ncorroborating the hypothesis of PKR as an endogenous L1 RNA sensor.\\n\\nWe then characterized the proteome of cells 24h post-L1 RNA transfection\\nby diaPASEF mass spectrometry (MS).(Meier *et al*, 2020a) We detected\\nalmost 8000 proteins in each sample, and observed that more than 88% of\\nsignificantly differentially expressed proteins (DEP) were downregulated\\nin L1-treated cells (Figure 5D, left panel). Moreover, among the top 15\\ndownregulated biological processes and cellular components were\\ntranslation and ribosomal subunits (Figure 5D, right panels, yellow\\ncircles), indicating that L1 RNA induces a global attenuation of protein\\nsynthesis. Indeed, a cap-dependent translation shutdown is the main\\nconsequence of eIF2α phosphorylation.(Donnelly *et al*, 2013) Finally,\\nwe show that neither PKR KD (Appendix Figure S6A, B) nor its\\npharmacological inhibition by C16 treatment (Figure 5E and 5F) prevents\\nL1 RNA-triggered induction of inflammatory genes (Figure 5E) and mineral\\nmatrix deposition (Figure 5F, Appendix Figure S6B). As previously\\nmentioned, a global reduction of protein synthesis occurs 24h after L1\\ntransfection and only a little percentage of DEP is upregulated in L1\\ncompared to RFP control (Figure 5D). Interestingly, some of the most\\nupregulated proteins are involved in autophagy and vesicle trafficking\\n(Figure 5G). Autophagy is a crucial process for mineralization and bone\\nhomeostasis both *in vitro* and *in vivo*,(Nollet *et al*, 2014) as\\nautophagic vacuoles are exploited to secrete apatite crystals.\\n\\nAltogether these data demonstrate that the PKR-mediated sensing of\\nL1-derived dsRNA globally attenuates the translation via eIF2α\\nphosphorylation and indicate autophagy as a possible mechanism involved\\nin L1-induced osteoblasts mineralization.\\n\\n**L1 RNA delivery reshapes the secretory profile of differentiating\\nosteoblasts**\\n\\nAn inflammatory response translates into a strong secretory activity and\\nproduction of paracrine signals, and we therefore tested the effect of\\n\"L1-primed\" osteoblast-derived conditioned medium in recipient\\ndifferentiating osteoblasts (Figure 6A). We found that the \"secretome\"\\nof cells transfected with L1 RNA has a significant paracrine effect on\\nrecipient osteoblasts since they showed an earlier formation of\\nmineralized nodules, already 24h post the addition of conditioned media\\n(Figure 6A, right panel), and higher deposition of mineral matrix at day\\n10 (Figure 6A, left panel). We subsequently isolated bulk and\\nexosome-derived proteomes from conditioned media of untreated, RFP- and\\nL1-treated differentiating osteoblasts and characterized them by MS.\\nDifferential expression analysis of MS data revealed a unique secretome\\nprofile (for both bulk and vesicular proteomes) of L1-primed osteoblasts\\ncompared to RFP-primed and untreated osteoblasts (Figure 6B). Gene\\nOntology (GO) analysis of differentially expressed proteins showed an\\nenrichment of pro-inflammatory factors (i.e. interleukins and\\nchemokines) involved in immune response and chemotactic migration of\\nimmune cells (Figure 6C and D), crucial processes for tissue repair\\nmechanisms *in vivo*. Pro-inflammatory molecules are also a major\\nconstituent of the senescence-associated secretory phenotype\\n(SASP),(Coppé *et al*, 2010) whose transient delivery supports cellular\\nplasticity and tissue regeneration.(Ritschka *et al*, 2017) Notably, the\\nmost enriched protein in L1-specific bulk secretome is interleukin 8\\n(IL-8), an inflammatory chemokine involved in several regenerative\\nprocesses, such as skin wound healing,(JIANG *et al*, 2012) therapeutic\\nangiogenesis after stroke and ischemia,(Choi *et al*, 2021) (Hou *et\\nal*, 2014) and osteochondral bone repair.(Lin *et al*, 2019) (Yang *et\\nal*, 2018a) Tumor necrosis factor-inducible gene 6 (TSG6) is among the\\ntop 10 proteins enriched in the bulk secretome of L1-primed osteoblasts\\ncompared to RFP. TSG6 is an inflammatory factor with suggestive\\ntherapeutic effects in corneal wounds, myocardial infarction, injured\\ncentral nervous system, chronic liver damage, and intervertebral disc\\ndegeneration.(Zhang *et al*, 2013) (Lee *et al*, 2015) (Lee *et al*,\\n2009) (Wang *et al*, 2017) (Yang *et al*, 2018b) Moreover, TSG6 induces\\nautophagy influx both *in vivo* and *in vitro.*(Wang *et al*, 2017)\\nAnother protein positively involved in autophagy and found specifically\\nin the secretome of L1 primed osteoblasts was ATG7 (autophagy-related\\n7). Osteoblast-specific ATG7 conditional knockout in mice reduced bone\\nmass during developmental and in adult age.(Li *et al*, 2018) Type 1\\nlysophosphatidic acid receptor (LPAR1) is also specifically secreted by\\nL1-primed osteoblasts and has proven to be positively involved in bone\\nmineralization *in vitro* and *in vivo*.(Alioli *et al*, 2020) (Gennero\\n*et al*, 2011)\\n\\nAltogether, our findings demonstrate that cytoplasmic L1 RNA is sensed\\nby PKR, whose activation leads to eIF2α phosphorylation with consequent\\nreprogramming of transcription and translation. The ultimate consequence\\nof this L1-induced, PKR-mediated stress response in osteoblasts is a\\nmajor reprogramming of their secretory activity, leading to a paracrine\\neffect on recipient osteogenic competent cells to trigger their\\nmineralization activity.\\n\\n**Discussion**\\n\\nThe concerted co-option of TEs markedly changed whole regulatory\\nnetworks and integrated new functions into the eukaryote genome.(Cosby\\n*et al*, 2019) (Mangiavacchi *et al*, 2021) (Carelli *et al*, 2022)\\nHowever, while the role of TEs as evolutionary drivers is today\\nwell-established, their contribution to cell physiology, particularly in\\nsomatic cells, remains to be elucidated. Indeed, aside from their role\\nin the nucleus, (Della Valle *et al*, 2022) (Jachowicz *et al*, 2017)\\n(Lu *et al*, 2021) (Percharde *et al*, 2018) TEs represent a major\\nsource of endogenous dsRNA whose role in triggering inflammatory and\\ninnate immune responses in various physiological processes has\\nincreasingly gained attention.(Sadeq *et al*, 2021) (Chen & Hur, 2022)\\nFor instance, an early acute inflammatory response is of vital\\nimportance in initiating and orchestrating bone repair and induce\\nanabolism leading to fracture healing. (Mountziaris *et al*, 2011)\\n(Marsell & Einhorn, 2011) (Bahney *et al*, 2019) Moreover, several\\nstudies demonstrated a positive effect of proinflammatory mediators on\\nbone mineralization *in vitro* and *in vivo*,(Croes *et al*, 2015)\\n(Croes *et al*, 2016) (Li *et al*, 2016) (Croes *et al*, 2017)\\nindicating that the modulation of inflammatory reaction is a tool to\\npursue regenerative strategies in bone.(Roberts & Ke, 2018) We found\\nthat, in the bone of newly fractured mice, a group of TEs, mostly LINE\\nand LTR, was early, markedly and transiently upregulated. Interestingly,\\nthe expression of these TEs was limited to the earliest phase of the\\ninflammatory stage. In line with the hypothesis of a positive\\ncorrelation between stress-induced bone mineralization and TEs\\nactivation, the expression dynamics in human bone biopsies revealed a\\nsignificant upregulation of TEs in the femur, a bone subjected to high\\nmechanical stresses, high turnover and anabolism, compared to ilium, not\\nweight bearing. Furthermore, we detected a lower TEs expression in\\ndonors with compromised bone mineralization and a significant\\ncorrelation between TEs expression and local BMD. We hypothesized that\\nthe expression dynamics of TEs in fracture/mechanical stress observed\\n*in vivo* could mimic the immunological threshold model of the sterile\\nactivation of dsRNA sensors as a response to stressful conditions.\\nIndeed, the transient and controlled breaching of the activation\\nthreshold of dsRNA sensors may lead to sterile inflammation being\\nintegrated into physiological processes.(Chen & Hur, 2022) To test\\nfurther this hypothesis, we delivered a full-length L1 consensus\\nsequence as source of dsRNA(Hur, 2019) to differentiating osteoblasts\\n*in vitro* and showed that L1 RNA initially accumulated in the cytoplasm\\nand is gradually cleared by secretion. The increased cytoplasmic levels\\nof L1 RNA are immediately sensed by PKR, a cytoplasmic dsRNA sensor\\nwhose activation induces the phosphorylation of\\neIF2α(Κιμ\\xa0*ετ\\xa0αλ*,\\xa02018)\\xa0(∆οννελλψ\\xa0*ετ\\xa0αλ*,\\xa02013), leading to the\\nattenuation of global protein synthesis (Donnelly *et al*, 2013),\\ninduction of inflammatory response, and stimulation of mineral matrix\\ndeposition. In support of the evidence that PKR is the early sensor of\\nincreased L1 RNA cytoplasmic levels, we show that inhibition of PKR\\nactivation by C16 prevents L1-induced eIF2α\\xa0phosphorylation, induction\\nof inflammatory genes, and mineral matrix deposition. Moreover, the\\ninhibition of L1 cDNA formation and sensing by Lamivudine 3TC, G140, or\\ncGAS KD does not affect the L1-induced phenotype. Thus, the evidence\\ndoes not support that the L1-triggered inflammation is causally\\nconnected to a cGAS-STING-dependent sensing of cytosolic L1 cDNA, a\\nmechanism demonstrated operative in senescence.(De Cecco *et al*, 2019a)\\n\\nL1 RNA response showed high specificity, as demonstrated by the\\ndose-dependent increase of mineralization and the lack of significant\\nchanges in mineral deposition when negative control RNA was transfected,\\neven at 100-200-fold higher concentration. Although the molecular\\nmechanism behind L1-induced stress response which stimulates the\\nmineralization activity of osteoblasts remains elusive, our results\\nindicate that L1 stimulates the mineral matrix deposition by\\nbone-forming cells regardless of their prior intrinsic anabolic\\npotential. Moreover, MS results suggested that increased autophagy, a\\nprocess crucial for mineralization and bone homeostasis *in vitro* and\\n*in vivo*(Nollet *et al*, 2014) may contribute to L1-induced\\nhydroxyapatite deposition. Intriguingly, the osteoinductive properties\\nof L1-primed osteoblast-derived secretome and the observation that L1\\nRNA was gradually secreted by the cells may suggest an involvement of L1\\nRNA in cell-to-cell communication, in line with the recent finding of\\nother retrotransposons (i.e. ERV) acting as paracrine molecules.(Liu *et\\nal*, 2021) This, together with the observed L1-dependent induction of\\nCCL and CXCL chemokines and cytokines, being chemoattractants of immune\\ncells and endothelial cells,(Bischoff *et al*, 2011) (Eash *et al*,\\n2010) (Strieter *et al*, 2005) and the composition of L1-primed\\nosteoblast secretome, makes the *in vivo* delivery of L1 RNA a necessary\\nstep to evaluate its actual contribution to the paracrine processes that\\norchestrate bone repair mechanisms. (Chaparro & Linero, 2016)\\n(Nimiritsky *et al*, 2019)\\n\\nThe identification of repeats RNA, particularly L1, as potential\\nresilient molecular factors involved in stress-induced,\\ninflammation-mediated bone production points out new avenues to develop\\nanabolic strategies for the treatment of bone loss conditions, such as\\nosteoporosis or impaired bone repair.(Roberts & Ke, 2018) On the other\\nhand, repeats may be novel targets in the treatment of those conditions\\ncharacterized by excessive or heterotopic ossification preceded by\\nderegulated/chronic inflammation.(Lories & Schett, 2012) (Balboni *et\\nal*, 2006)\\n\\n**Materials and Methods**\\n\\n**Participants and ethics**\\n\\nFemoral bone biopsies were obtained from the caput region of\\npostmenopausal women or men with a wide BMD range, i.e., from healthy to\\nosteoporotic, who were undergoing hip replacement surgery due to\\nosteoarthritis or fracture at Lovisenberg Diaconal Hospital (Oslo,\\nNorway) or Diakonhiemmet Hospital (Oslo, Norway), respectively. The\\ndonors of femoral bones are listed in Appendix Table S1. The\\npostmenopausal iliac bone donors were recruited from the outpatient\\nclinic of Lovisenberg Diaconal Hospital (Oslo, Norway). Candidates\\nfilled out a questionnaire that included medication and lifestyle\\nfactors, and selected donors were deemed representative of the\\nOslo-based Norwegian ethnic female population aged 50 to 86\\u2009years. The\\niliac bone donors are listed in Appendix Table S2 and have been\\ndescribed previously in detail.(Reppe *et al*, 2010) All donors taking\\nmedication or having diseases, other than primary osteoporosis, known to\\naffect bone metabolism were excluded. The presence of bone-impairing\\ndiseases/conditions was excluded by extensive biochemical serum and\\nurine analyses supported by X-ray examinations. The site-specific BMD of\\nall donors was evaluated with Lunar Prodigy DEXA (GE Lunar, Madison, WI,\\nUSA) following the manufacturer\\\\\\'s instructions. The precision of the\\ninstrument for measuring the lumbar spine (L~2~--L~4~) and hip BMD was\\n1.7% and 1.1%, respectively. The study was approved by the Norwegian\\nRegional Ethical Committee (REK no 2010/2539, Norway), all volunteers\\ngave their written informed consent, and sampling and procedures were\\naccording to the Act of Biobanking in Norway.\\n\\n**Cell culture**\\n\\nFor L1 RNA transfection experiments, human MSCs were isolated from\\nosteoporotic femoral heads with the consent of the patient according to\\nSwiss (BASEC-Nr. 2016-02159) ethical guidelines. Femoral heads were\\ncrushed and incubated in DMEM (PAN Biotech, Germany, Cat. No. P04-03550)\\nsupplemented with 1% (v/v) 100× Penicillin-Streptomycin Solution\\n(Biowest, France, Cat. No. L0022), 10% (v/v) FCS (Biowest, France, Cat.\\nNo. S181S), 1% (v/v) 200mM L-Glutamine solution (Sigma, USA, Cat. No.\\nG7513), 5ng/ml FGF-2 (Sigma, USA, Cat. No. F0291) and 10ng/ml FGF-4\\n(Sigma, USA, Cat. No. F8424) in a humidified atmosphere containing 5%\\nCO2. To induce osteogenic differentiation, MSCs were seeded on Nunc™\\n24-well plates (Thermo Fisher, USA, Cat. No. 142475) or 48-well plates\\n(Thermo Fisher, USA, Cat. No. 150687) at a density of 1.5\\u2009×\\u200910^4^\\ncells/cm2. After 24\\u2009h, differentiation was induced using StemPro®\\nOsteogenesis Kit (Gibco/Life Technologies, USA, Cat. No. A10072-01). The\\nmedium was exchanged every 3 days. Cells were treated with Lamivudine\\n3TC (Sigma), 1μM final concentration; C16 (Merck), 500nM final\\nconcentration; G140 (InvivoGen), 150μM final concentration.\\n\\n**L1 RNA transfection**\\n\\nThe vector human-L1_pBluescript II sk (+) carrying the full-length L1\\nsequence was custom-prepared by GenScript, USA. Large-scale human L1\\nmRNA was in vitro transcribed, modified, and purified by TriLink\\nBiotechnologies, USA, (ARCA capped and 2\\'Omethymalted (CapI), fully\\nsubstituted with 5-methyl-C, 25% substitution of Cyanine-5-U and 75%\\nsubstitution of Pseudo-U, enzymatically polyadenylated, DNase and\\nphosphatase treated, silica membrane purified). Synthetic L1 RNA was\\ntransfected in MSCs differentiating to osteoblasts at day 5 using\\nLipofectamine^TM^ MessengerMAX^TM^ (Invitrogen, USA, Cat. No. LMRNA003)\\nwith a modified protocol for low RNA amount. RFP mRNA (System\\nBioscience, USA, Cat. No. MR800A-1) was used as negative control. Bone\\nmatrix was quantified with OsteoImage Mineralization Assay (Lonza,\\nBasel, Switzerland, Cat. No. LOPA503) and Alizarin Red staining. In\\nconditioned media experiments, the medium was exchanged 6h after\\ntransfection. Conditioned medium was collected after three days and\\ndelivered to the recipient, differentiating osteoblasts at day 5. In\\nknockdown experiments, siRNAs were transfected with Lipofectamine\\nRNAiMAX (Thermo Scientific) in differentiating osteoblasts at day 3,\\nfollowed by L1 RNA transfection at day 7. siRNAs: EIF2AK2- Targeting\\nSMARTpool (Dharmacon, Horizon, E-003527-00-0010), MB21D1-Targeting\\nSMARTpool (Dharmacon, Horizon, L-015607-02-0020), Non-targeting Pool\\n(D-001810-10).\\n\\n**Quantitative mineralization assay**\\n\\nCells were washed in PBS and fixed in 4% (v/v) formaldehyde (Sigma, USA,\\nCat. No. F8775) in 1× PBS for 15 minutes. Mineralization was assessed by\\nusing the OsteoImage Mineralization Assay (Lonza, Basel, Switzerland,\\nCat. No. LOPA503) according to the manufacturer\\'s indication.\\nMineralization was quantified with GloMax® discover microplate reader\\n(Promega, USA), selecting appropriate excitation (492) / emission (520)\\nwavelengths.\\n\\n**Alizarin Red staining**\\n\\nOsteoblasts were washed with 1× PBS (Kantonsapotheke Zürich,\\nSwitzerland, Cat. No. A171012) and fixed with 4% (v/v) formaldehyde\\n(Sigma, USA, Cat. No. F8775) in 1× PBS for 30\\u2009min. After washing twice\\nwith ddH2O, Alizarin Red staining solution (0.7\\u2009g Alizarin Red S (Sigma,\\nUSA, Cat. No. A5533) diluted in 50\\u2009ml ddH2O at pH\\u2009=\\u20094.2) was added for\\n20\\u2009min. Afterwards, cells were washed four times with ddH2O, dried, and\\nstored in the dark until image acquisition. For absorbance measurement,\\nAlizarin Red S was eluted from stained osteoblasts with 300\\u2009μl 10% (w/v)\\ncetylpyridinium chloride in an aqueous 0.01\\u2009M Na2HPO4/NaH2PO4 solution\\nat pH\\u2009=\\u20097 for 1\\u2009h. One hundred fifty microliters were transferred on a\\n96-well plate, and absorbance was measured at 560\\u2009nm. Ten percent (w/v)\\ncetylpyridinium chloride in an aqueous 0.01\\u2009M Na2HPO4/NaH2PO4 solution\\nwas used as blank. Images were acquired, processed and analysed as\\npreviously described(Eggerschwiler *et al*, 2019)\\n\\n**ALPL activity assay:**\\n\\nALPL activity was measured as in Liu et al., 2016.(Liu *et al*, 2016)\\nBriefly, cells were fixed in 3.7% formaldehyde at RT for 10\\' and then\\nstained with a solution of 25% naphthol AS-BI phosphate (Thermo) and\\n0.75% Fast Blue BB (Sigma) dissolved in 0.1\\u2009M Tris buffer (pH 9.3) at RT\\nfor 15\\'. After staining, cells were quickly washed with PBS five times.\\n\\n**ENPP1 activity assay**\\n\\nENPP1 activity was calculated colorimetrically using the chromogenic\\nsubstrate p-nitrophenyl-thymidine-5′-monophosphate, as described in\\n(Ferreira et al., 2013). After PBS washing, cells were lysed in a\\nsolution of 0.1% Triton X-100, 0.2 M Tris-base, 1.6 mM MgCl2, pH 8.1. 50\\nμl of 1mM thymidine monophosphate p-nitrophenyl ester (Sigma) were added\\nto 50 μl of cell lysate and incubated for 1 hour at 37°C. Then, four\\nvolumes of 0.1M NaOH were added to stop the reaction and absorbance was\\nread at 410 nm using a spectrophotometer (Tecan Nanoquant Infinite 200\\nPro Multimode). Absorbance was normalized to DNA content.\\n\\n**Immunohistochemistry:**\\n\\nCells were fixed in 4% paraformaldehyde in PBS at room temperature (RT)\\nfor 10\\'. After fixation, cells were permeabilized in 3% Triton X-100 in\\nPBS for 3\\' at RT. Blocking was performed with 4% BSA in PBS for 30\\' at\\nRT. After blocking, cells were incubated with primary antibodies\\novernight at 4°C, then washed with PBS and incubated with secondary\\nantibodies for 1 hour at RT. After washing with PBS, nuclei were stained\\nwith Hoechst (Thermo Scientific) and mounted using Fluoro Gel with\\nDABCO™ Mounting Medium (Electron Microscopy Sciences). Primary\\nantibodies: DNA-RNA Hybrid (mouse, Merck Millipore, MABE1095), dsRNA\\n(mouse, Merck Millipore, MABE1134).\\n\\n**RNA extraction and cDNA preparation**\\n\\nCells were harvested and resuspended in 1ml of QIAzol Lysis reagent\\n(Qiagen, Cat. No.79306). Total RNA was then purified with RNeasy Plus\\nMini kit (Qiagen, cat. No. 74134) with minimal modifications to\\nmanufacturer\\'s instructions. DNase treatment (RNase-free DNase set,\\nQiagen, Cat. No. 79254) was performed to remove any residual DNA. RNA\\nquality and concentration were checked using a Nanodrop^TM^ 2000\\nspectrophotometer (ThermoFisher). cDNA was synthesized from 200ng of\\neach RNA sample using a Superscript III first-strand cDNA synthesis\\nsystem (ThermoFisher, cat. No.18080051) according to the manufacturer\\'s\\nprotocol. Processing of human bone biopsies and RNA isolation has been\\ndescribed previously.(Reppe *et al*, 2010)\\n\\n**Gene expression analysis**\\n\\nReal-time quantitative polymerase chain reaction (qPCR) was performed\\nwith a 7900HT Fast Real-Time PCR system (Applied Biosystems). Each\\nsample was analyzed in triplicate and normalized with the endogenous\\ncontrol Ribosomal Protein L13A (*RPL13A)* for cDNA input concentration.\\nNo template and no RT were included as negative controls. For each 15μl\\nreaction, 10ng (1ng for L1) of cDNA was mixed with 1μM specific primers\\nmix and 7.5μl of Sybr^TM^ Select Master mix (Applied Biosystems, USA,\\nCat. No. 4472908). The reaction was incubated at 95°C for 10 minutes,\\nfollowed by 40 cycles of denaturation at 95°C for 15 seconds, annealing\\nat 60°C for 30 seconds and elongation at 72°C for 30 seconds. Ct values\\nwere calculated by 7900HT Fast Real-Time PCR RQ manager software\\n(Applied Biosystems, USA) and then normalized as ∆Ct between the gene of\\ninterest and the endogenous calibrator. Primers used in this study for\\ngene expression analysis were designed using Primer3\\n([[http://www.ncbi.nlm.nih.gov/tools/primer-blast/]{.underline}](http://www.ncbi.nlm.nih.gov/tools/primer-blast/)).\\nIn all primer pairs, each primer matches a different exon. Amplicons\\nlength was 80-130 nucleotides. Primer sequences are reported in Appendix\\nTable S3.\\n\\n**Cell cycle analysis**\\n\\n2x10^5^ MSCs were trypsinized for 5 min at 37°C, washed with PBS and 2%\\nBSA, passed through a 70μM strainer (Corning, USA, Cat. No. 352350) and\\nthen fixed at -20°C for 30 min in 70% ethanol. After washing with PBS\\nand 4% BSA, cells were resuspended in PBS and incubated for 1 hour at\\n37°C with RNase. Cells were then washed and resuspended in 100μl of Flow\\nCytometry Staining Buffer (R&D System, USA, Cat. No. FC001). 10μl of\\n1mg/ml Propidium iodide (PI) staining solution (Invitrogen, USA, Cat.\\nNo. P3566) was added to the single-cell solution, gently mixed, and\\nincubated for 5 min in the dark. Cell cycle analysis was performed on BD\\nFACSCanto II Flow Cytometry System (BD-Biosciences), using BD FACSDiva\\nSoftware (BD-Biosciences).\\n\\n**RNA-Seq and data analysis**\\n\\nRNA from human bone biopsies was sequenced at the Human Genotyping\\nfacility (HuGe-F) of Erasmus MC.\\n\\nTotal RNA-Seq library, from L1 RNA delivery experiments, was prepared\\nwith CORALL Total RNA library prep with RiboCop rRNA for Human/Mouse/Rat\\ndepletion kit (Lexogen GmbH, Vienna, Austria) following manufacturer\\'s\\ninstructions (library type: fr-secondstrand) by IGA Technology service\\n(Italy). The final libraries were checked using both Qubit 2.0\\nFluorometer (Invitrogen, Carlsbad, CA) and Agilent Bioanalyzer DNA assay\\nor Caliper (PerkinElmer, Waltham, MA). Libraries were then prepared for\\nsequencing and sequenced on paired-end 150 bp mode on NovaSeq6000\\n(Illumina, San Diego, CA). RNA-Seq read quality control (QC) analyses\\nand filtering of high-quality reads were executed using FastQC v0.11.9\\n(http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) and BBDuk\\nv35.85\\n([https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbduk-guide/](https://urldefense.com/v3/__https:/jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbduk-guide/__;!!Nmw4Hv0!0EikfGZVN-B-Xp5VkKLqowTItYk7Yav6lOsagjvuQwPqWY_Y3yszwKePkxm4puTKivdPL5LvCMjrSQwcKpyz9m7YVqqz09ekvfk$))\\xa0by\\nsetting a minimum read length of 35 bp and a minimum Phred-quality score\\nof 25. After trimming quality control, high-quality reads were aligned\\nto the human genome reference (GRCh38) with STAR 2.7.3a,(Dobin *et al*,\\n2013) while FeatureCounts 1.6.3 package(Liao *et al*, 2014) was used to\\nassign reads to genes. Next, lowly expressed genes across one or more\\nexperimental conditions were filtered to eliminate the \"uninformative\"\\ngenes using HTSFilter v1.30.1.(Rau *et al*, 2013) Filtered gene data\\nwere further processed with the EdgeR package v3.32.1(Robinson *et al*,\\n2010) to normalize (Trimmed Mean of M-7 values, TMM, method) the raw\\ncounts and perform differential gene expression analysis. Multiple\\ntesting correction was performed with the FDR method(Benjamini &\\nHochberg, 1995) and significance level was set at FDR \\\\< 0.05. Gene\\nOntology (GO) term enrichment was analyzed by performing hypergeometric\\ntests11 for each individual term, and FDR correction was applied (FDR \\\\<\\n0.05). Expression of Interspersed Repeat elements was quantified using\\nSQuiRE 0.9.9.92 (https://github.com/wyang17/SquIRE). SQuiRE provides\\nlocus-specific expression quantification along with subfamily-level\\nexpression estimates counting unambiguously mapped reads, as well as\\nambiguously mapped reads using an expectation--maximization (EM)\\nalgorithm.(Yang *et al*, 2019) Briefly, reference genome and\\nRepeatmasker annotation were downloaded from UCSC and prepared for the\\nanalysis with squire Fetch and squire Clean, respectively. High-quality\\nreads were mapped against the reference genome with STAR using squire\\nMap and expression was quantified using squire Call. Differential\\nexpression analysis was performed with squire Call.\\n\\n**DIA-MS analysis using TimsTOF MS**\\n\\nTotal protein extracts were prepared with RIPA buffer (50mMTris-cl pH\\n8.0, 5mM EDTA, 150mM Nacl, 15mM MgCl~2~, 1% NP-40, 1mM PMSF and 1X\\nProtease Inhibitor Cocktail (PIC)). Further sonication step was\\nincluded: (30S ON, 30S OFF, 10 cycles with Bioruptor). Equal 50 μg\\nprotein extracts were concentrated to 30 μl volume, then diluted in 8M\\nurea in 0.1M Tris-HCl, followed by protein digestion with trypsin,\\naccording to the FASP protocol.(Wiśniewski *et al*, 2009) After\\novernight digestion, the peptides were eluted from the filters with 25\\nmM ammonium bicarbonate buffer. The eluted peptides were processed in\\nthe desalting step using Sep-Pag C18 Column (waters) based on the\\nmanufacturer\\'s instructions. Approximately 200 ng of peptide mixture per\\nsample was analyzed using a timsTOF Pro 2 QTOF mass spectrometer coupled\\nwith a nanoElute liquid chromatography system (Bruker Daltonik GmbH,\\nGermany). The sample was injected directly into an RP-C18 Aurora emitter\\ncolumn (75 µm\\u202fi.d.×\\u202f250 mm, 1.6 μm, 120 Å pore size) (Ion Opticks,\\nAustralia) using a one-column separation method. An 80-min gradient was\\nestablished using mobile phase A (0.1% FA in H2O) and mobile phase B\\n(0.1% FA in Acetonitrile): 2--25% B for 60\\u2009min, 25--37% for 10\\u2009min,\\nramping 37% to 95% in 5\\u2009min, and maintaining 95% B for 5\\u2009min. The column\\ntemperature was set at 50^o^C and the flow rate at 250 nl/min. The\\nsample eluting from the separation column was introduced into the mass\\nspectrometer via a CaptiveSpray nano-electrospray ion source (Bruker\\nDaltonik GmbH) with an electrospray voltage of 1.6\\u2009kV. The ion source\\ntemperature was set to 180\\u2009°C and a dry gas of 3 l/min. The samples were\\nanalyzed using diaPASEF scheme(Meier *et al*, 2020b) consisting of 24\\ncycles including a total of 48 mass width windows (13 Da (m/z) from m/z\\n400 to 1,000 and TIMS scan range from\\u20090.63\\u2009to 1.35\\u2009Vs\\u2009cm−2 (1/K0). The\\ncollisional energy increased linearly from 20.01 eV at 0.6 (1/K0) to\\n52.00 eV at 1.35\\u2009Vs\\u2009cm−2 (1/K0). The scan range for MS and MS/MS spectra\\nwas set to 100-1700 m/z. TIMS ramping time and accumulation time were\\nset to 100 milliseconds. The diaPASEF data were analyzed by directDIA\\napproach using Spectronaut software (version 14) following manufacture\\ninstructions. Up or down regulated proteins were determined using DEP\\n(Differential Enrichment analysis of Proteomics data) R package.\\nSignificant results (adjusted p-value \\\\< 0.05) were subjected to Gene\\nOntology enrichment analysis with clusterProfiler R package.\\n\\n**Western Blot**\\n\\nTotal protein extracts were prepared by lysing cells in extraction\\nbuffer (HEPES KOH \\\\[pH 8.5\\\\], NaCl 400 mM, EDTA 0.1 mM, EGTA 0.1 mM, DTT\\n1 mM, 1× protease inhibitor, SDS 1%). Proteins were separated by\\nelectrophoresis on BOLT 4%--12% bis-tris polyacrylamide precast gels in\\nMES buffer (Life Technologies) and transferred to a 0,2 μm\\nnitrocellulose membrane. Non-specific signals were blocked with 5%\\nMilk-PBS-Tween0.5% and the membrane hybridized overnight at 4°C with\\nprimary and secondary antibodies diluted in blocking buffer. Horseradish\\nperoxidase-conjugated secondary antibodies were revealed with the ECL\\nchemiluminescence kit (Amersham), and signals were detected using\\nChemiDoc (Bio-Rad). Primary antibodies: PKR (rabbit, Abcam, ab32052),\\ncGAS (rabbit, Abcam, ab224144), LINE-1 ORF1 (mouse, Merck Millipore,\\nMABC1152), EIF2a (mouse, Abcam, ab5369),\\xa0phospho-EIF2α\\xa0(rabbit, Abcam,\\nab32157), Histone H3 (rabbit, Merk, 06-755).\\n\\n**Exosome isolation**\\n\\nExosomes were isolated using Total Exosome Isolation Reagent (from cell\\nculture media) (Thermo), following manufacturer\\'s instructions. The\\nexosome pellet was resuspended in 50 μl of RIPA buffer and kept on ice\\nfor 30\\', mixing every 5\\'. Samples were then sonicated (30\\'\\' ON/30\" OFF,\\n10 cycles), centrifuged at 13,000 x g for 20\\' at 4°C and collected as\\nsupernatants.\\n\\n**Statistical analysis**\\n\\nStatistic tests used for data analysis are indicated in the figure\\nlegends.\\n\\n**Data availability**\\n\\nThe RNA-seq transcriptomics data on human bone biopsies have been\\ndeposited to BioProject with the dataset identifier PRJNA764663\\n(https://www.ncbi.nlm.nih.gov/bioproject/PRJNA764663). The RNA-seq\\ntranscriptomics data on transfected osteoblasts have been deposited to\\nGEO with the dataset identifier GSE201774\\n(https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201774) (enter\\ntoken: ififogaehzoxlgh).\\n\\nThe mass spectrometry proteomics data have been deposited to the\\nProteomeXchange Consortium via the PRIDE partner repository with the\\ndataset identifier PXD051195. Reviewer account details are username:\\n<reviewer_pxd051195@ebi.ac.uk> ; password: R6v0mIYe\\n\\n**Acknowledgment**\\n\\nWe acknowledge kaust Smarth Health Initiative (KSHI); the South-Eastern\\nNorway Regional Health Authority; the Norwegian Osteoporosis Society;\\nAnders Jahre Foundation for the Promotion of Science; the Lovisenberg\\nDiaconal Hospital research fund; and the Research Council of Norway.\\n\\n**Funding:** We are grateful for support from: KAUST BAS/1/1037-01-01,\\nthe European Union project OSTEOGENE (no. FP6-502491);\\nZonMw\\xa0VIDI\\xa0016.136.367\\xa0grant, for funding the creation of the RNA-Seq\\ndataset of hip primary bone and Oslo University Hospital, Ullevål,\\nproject #29750104.\\n\\n**Disclosure and competing interests statement:** All the authors\\ndeclare no competing interests.\\n\\n**References**\\n\\nAerssens J, Boonen S, Joly J & Dequeker J (1997) Variations in\\ntrabecular bone composition with anatomical site and age: potential\\nimplications for bone quality assessment. *J Endocrinol* 155: 411--421\\n\\nAlioli CA, Demesmay L, Laurencin-Dalacieux S, Beton N, Farlay D, Follet\\nH, Saber A, Duboeuf F, Chun J, Rivera R, *et al* (2020) Expression of\\nthe type 1 lysophosphatidic acid receptor in osteoblastic cell lineage\\ncontrols both bone mineralization and osteocyte specification. *Biochim\\nBiophys Acta - Mol Cell Biol Lipids* 1865: 158715\\n\\nAnderson BR, Muramatsu H, Nallagatla SR, Bevilacqua PC, Sansing LH,\\nWeissman D & Karikó K (2010) Incorporation of pseudouridine into mRNA\\nenhances translation by diminishing PKR activation. *Nucleic Acids Res*\\n\\nBahney CS, Zondervan RL, Allison P, Theologis A, Ashley JW, Ahn J,\\nMiclau T, Marcucio RS & Hankenson KD (2019) Cellular biology of fracture\\nhealing. *J Orthop Res* 37: 35--50\\n\\nBalboni TA, Gobezie R & Mamon HJ (2006) Heterotopic ossification:\\nPathophysiology, clinical features, and the role of radiotherapy for\\nprophylaxis. *Int J Radiat Oncol* 65: 1289--1299\\n\\nBenjamini Y & Hochberg Y (1995) Controlling the False Discovery Rate: A\\nPractical and Powerful Approach to Multiple Testing. *J R Stat Soc Ser\\nB* 57: 289--300\\n\\nBischoff DS, Sakamoto T, Ishida K, Makhijani NS, Gruber HE & Yamaguchi\\nDT (2011) CXC receptor knockout mice: characterization of skeletal\\nfeatures and membranous bone healing in the adult mouse. *Bone* 48:\\n267--274\\n\\nBrem H & Tomic-Canic M (2007) Cellular and molecular basis of wound\\nhealing in diabetes. *J Clin Invest* 117: 1219--1222\\n\\nCarelli FN, Cerrato C, Dong Y, Appert A, Dernburg A & Ahringer J (2022)\\nWidespread transposon co-option in the Caenorhabditis germline\\nregulatory network. *Sci Adv* 8\\n\\nDe Cecco M, Ito T, Petrashen AP, Elias AE, Skvir NJ, Criscione SW,\\nCaligiana A, Brocculi G, Adney EM, Boeke JD, *et al* (2019a) L1 drives\\nIFN in senescent cells and promotes age-associated inflammation.\\n*Nature* 566: 73--78\\n\\nDe Cecco M, Ito T, Petrashen AP, Elias AE, Skvir NJ, Criscione SW,\\nCaligiana A, Brocculi G, Adney EM, Boeke JD, *et al* (2019b) L1 drives\\nIFN in senescent cells and promotes age-associated inflammation.\\n*Nature* 566: 73--78\\n\\nChan JK, Roth J, Oppenheim JJ, Tracey KJ, Vogl T, Feldmann M, Horwood N\\n& Nanchahal J (2012) Alarmins: awaiting a clinical response. *J Clin\\nInvest* 122: 2711\\n\\nChaparro O & Linero I (2016) Regenerative Medicine: A New Paradigm in\\nBone Regeneration. In *Advanced Techniques in Bone Regeneration* InTech\\n\\nChen YG & Hur S (2022) Cellular origins of dsRNA, their recognition and\\nconsequences. *Nat Rev Mol Cell Biol* 23: 286--301\\n\\nChoi J, Choi W, Joo Y, Chung H, Kim D, Oh SJ & Kim S-H (2021)\\nFGF2-primed 3D spheroids producing IL-8 promote therapeutic angiogenesis\\nin murine hindlimb ischemia. *NPJ Regen Med* 6: 48\\n\\nClark D, Nakamura M, Miclau T & Marcucio R (2017) Effects of Aging on\\nFracture Healing. *Curr Osteoporos Rep* 15: 601\\n\\nCoates BA, McKenzie JA, Buettmann EG, Liu X, Gontarz PM, Zhang B & Silva\\nMJ (2019) Transcriptional Profiling of Intramembranous and Endochondral\\nOssification after Fracture in Mice. *Bone* 127: 577\\n\\nCoppé J-P, Desprez P-Y, Krtolica A & Campisi J (2010) The\\nSenescence-Associated Secretory Phenotype: The Dark Side of Tumor\\nSuppression. *Annu Rev Pathol Mech Dis* 5: 99--118\\n\\nCosby RL, Chang NC & Feschotte C (2019) Host-transposon interactions:\\nconflict, cooperation, and cooption. *Genes Dev* 33: 1098--1116\\n\\nCroes M, Boot W, Kruyt MC, Weinans H, Pouran B, van der Helm YJM,\\nGawlitta D, Vogely HC, Alblas J, Dhert WJA, *et al* (2017)\\nInflammation-Induced Osteogenesis in a Rabbit Tibia Model. *Tissue Eng\\nPart C Methods* 23: 673--685\\n\\nCroes M, Oner FC, Kruyt MC, Blokhuis TJ, Bastian O, Dhert WJA & Alblas J\\n(2015) Proinflammatory Mediators Enhance the Osteogenesis of Human\\nMesenchymal Stem Cells after Lineage Commitment. *PLoS One* 10: e0132781\\n\\nCroes M, Öner FC, van Neerven D, Sabir E, Kruyt MC, Blokhuis TJ, Dhert\\nWJA & Alblas J (2016) Proinflammatory T cells and IL-17 stimulate\\nosteoblast differentiation. *Bone* 84: 262--270\\n\\nDobin A, Davis CA, Schlesinger F, Drenkow J, Zaleski C, Jha S, Batut P,\\nChaisson M & Gingeras TR (2013) STAR: ultrafast universal RNA-seq\\naligner. *Bioinformatics* 29: 15--21\\n\\nDonnelly N, Gorman AM, Gupta S & Samali A (2013) The eIF2α kinases:\\ntheir structures and functions. *Cell Mol Life Sci* 70: 3493--3511\\n\\nEash KJ, Greenbaum AM, Gopalan PK & Link DC (2010) CXCR2 and CXCR4\\nantagonistically regulate neutrophil trafficking from murine bone\\nmarrow. *J Clin Invest* 120: 2423--2431\\n\\nEggerschwiler B, Canepa DD, Pape HC, Casanova EA & Cinelli P (2019)\\nAutomated digital image quantification of histological staining for the\\nanalysis of the trilineage differentiation potential of mesenchymal stem\\ncells. *Stem Cell Res Ther*\\n\\nFLEISCH H, RUSSELL RGG & STRAUMANN F (1966) Effect of Pyrophosphate on\\nHydroxyapatite and Its Implications in Calcium Homeostasis. *Nature*\\n212: 901--903\\n\\nFukuda S, Varshney A, Fowler BJ, Wang S Bin, Narendran S, Ambati K,\\nYasuma T, Magagnoli J, Leung H, Hirahara S, *et al* (2021) Cytoplasmic\\nsynthesis of endogenous Alu complementary DNA via reverse transcription\\nand implications in age-related macular degeneration. *Proc Natl Acad\\nSci U S A* 118\\n\\nGe C, Xiao G, Jiang D & Franceschi RT (2007) Critical role of the\\nextracellular signal--regulated kinase--MAPK pathway in osteoblast\\ndifferentiation and skeletal development. *J Cell Biol* 176: 709\\n\\nGennero I, Laurencin-Dalicieux S, Conte-Auriol F, Briand-Mésange F,\\nLaurencin D, Rue J, Beton N, Malet N, Mus M, Tokumura A, *et al* (2011)\\nAbsence of the lysophosphatidic acid receptor LPA1 results in abnormal\\nbone development and decreased bone mass. *Bone* 49: 395--403\\n\\nGerstenfeld LC, Cho T-J, Kon T, Aizawa T, Cruceta J, Graves BD & Einhorn\\nTA (2001) Impaired Intramembranous Bone Formation during Bone Repair in\\nthe Absence of Tumor Necrosis Factor-Alpha Signaling. *Cells Tissues\\nOrgans* 169: 285--294\\n\\nGerstenfeld LC, Cullinane DM, Barnes GL, Graves DT & Einhorn TA (2003)\\nFracture healing as a post-natal developmental process: Molecular,\\nspatial, and temporal aspects of its regulation. *J Cell Biochem* 88:\\n873--884\\n\\nHessle L, Johnson KA, Anderson HC, Narisawa S, Sali A, Goding JW,\\nTerkeltaub R & Millan JL (2002) Tissue-nonspecific alkaline phosphatase\\nand plasma cell membrane glycoprotein-1 are central antagonistic\\nregulators of bone mineralization. *Proc Natl Acad Sci U S A* 99:\\n9445--9\\n\\nHou Y, Ryu CH, Jun JA, Kim SM, Jeong CH & Jeun S (2014) IL-8 enhances\\nthe angiogenic potential of human bone marrow mesenchymal stem cells by\\nincreasing vascular endothelial growth factor. *Cell Biol Int* 38:\\n1050--9\\n\\nHur S (2019) Double-Stranded RNA Sensors and Modulators in Innate\\nImmunity. *Annu Rev Immunol* 37: 349--375\\n\\nHurtgen BJ, Ward CL, Garg K, Pollot BE, Goldma SM, McKinley TO, Wenke JC\\n& Corona BT (2016) Severe muscle trauma triggers heightened and\\nprolonged local musculoskeletal inflammation and impairs adjacent tibia\\nfracture healing. *J Musculoskelet Neuronal Interact* 16: 122\\n\\nJachowicz JW, Bing X, Pontabry J, Bošković A, Rando OJ & Torres-Padilla\\nME (2017) LINE-1 activation after fertilization regulates global\\nchromatin accessibility in the early mouse embryo. *Nat Genet*\\n\\nJIANG WG, SANDERS AJ, RUGE F & HARDING KG (2012) Influence of\\ninterleukin-8 (IL-8) and IL-8 receptors on the migration of human\\nkeratinocytes, the role of PLC-γ and potential clinical implications.\\n*Exp Ther Med* 3: 231--236\\n\\nJiao H, Xiao E & Graves DT (2015) Diabetes and Its Effect on Bone and\\nFracture Healing. *Curr Osteoporos Rep* 13: 327--335\\n\\nJohnson KA, Hessle L, Vaingankar S, Wennberg C, Mauro S, Narisawa S,\\nGoding JW, Sano K, Millan JL & Terkeltaub R (2000) Osteoblast\\ntissue-nonspecific alkaline phosphatase antagonizes and regulates PC-1.\\n*Am J Physiol Regul Integr Comp Physiol* 279: R1365-77\\n\\nJones RB, Garrison KE, Wong JC, Duan EH, Nixon DF & Ostrowski MA (2008)\\nNucleoside analogue reverse transcriptase inhibitors differentially\\ninhibit human LINE-1 retrotransposition. *PLoS One*\\n\\nKarikó K, Buckstein M, Ni H & Weissman D (2005) Suppression of RNA\\nrecognition by Toll-like receptors: The impact of nucleoside\\nmodification and the evolutionary origin of RNA. *Immunity*\\n\\nKarikó K, Muramatsu H, Welsh FA, Ludwig J, Kato H, Akira S & Weissman D\\n(2008) Incorporation of pseudouridine into mRNA yields superior\\nnonimmunogenic vector with increased translational capacity and\\nbiological stability. *Mol Ther*\\n\\nKim Y, Lee JH, Park J-E, Cho J, Yi H & Kim VN (2014) PKR is activated by\\ncellular dsRNAs during mitosis and acts as a mitotic regulator. *Genes\\nDev* 28: 1310--1322\\n\\nKim Y, Park J, Kim S, Kim MA, Kang MG, Kwak C, Kang M, Kim B, Rhee HW &\\nKim VN (2018) PKR Senses Nuclear and Mitochondrial Signals by\\nInteracting with Endogenous Double-Stranded RNAs. *Mol Cell* 71:\\n1051-1063.e6\\n\\nKon T, Cho TJ, Aizawa T, Yamazaki M, Nooh N, Graves D, Gerstenfeld LC &\\nEinhorn TA (2001) Expression of osteoprotegerin, receptor activator of\\nNF-kappaB ligand (osteoprotegerin ligand) and related proinflammatory\\ncytokines during fracture healing. *J Bone Miner Res* 16: 1004--1014\\n\\nKormann MSD, Hasenpusch G, Aneja MK, Nica G, Flemmer AW, Herber-Jonat S,\\nHuppmann M, Mays LE, Illenyi M, Schams A, *et al* (2011) Expression of\\ntherapeutic proteins after delivery of chemically modified mRNA in mice.\\n*Nat Biotechnol*\\n\\nKoski GK, Karikó K, Xu S, Weissman D, Cohen PA & Czerniecki BJ (2004)\\nCutting Edge: Innate Immune System Discriminates between RNA Containing\\nBacterial versus Eukaryotic Structural Features That Prime for\\nHigh-Level IL-12 Secretion by Dendritic Cells. *J Immunol*\\n\\nKovtun A, Bergdolt S, Wiegner R, Radermacher P, Huber-Lang M & Ignatius\\nA (2016) The crucial role of neutrophil granulocytes in bone fracture\\nhealing. *Eur Cells Mater* 32: 152--162\\n\\nLama L, Adura C, Xie W, Tomita D, Kamei T, Kuryavyi V, Gogakos T,\\nSteinberg JI, Miller M, Ramos-Espiritu L, *et al* (2019) Development of\\nhuman cGAS-specific small-molecule inhibitors for repression of\\ndsDNA-triggered interferon expression. *Nat Commun* 10: 2261\\n\\nLaroche M, Livideanu C, Paul C & Cantagrel A (2011) Interferon Alpha and\\nPamidronate in Osteoporosis with Fracture Secondary to Mastocytosis. *Am\\nJ Med* 124: 776--778\\n\\nLee MJ, Kim DH, Ryu JS, Ko AY, Ko JH, Kim MK, Wee WR, Khwarg SI & Oh JY\\n(2015) Topical TSG-6 Administration Protects the Ocular Surface in Two\\nMouse Models of Inflammation-Related Dry Eye. *Invest Ophthalmol Vis\\nSci* 56: 5175--81\\n\\nLee RH, Pulin AA, Seo MJ, Kota DJ, Ylostalo J, Larson BL, Semprun-Prieto\\nL, Delafontaine P & Prockop DJ (2009) Intravenous hMSCs improve\\nmyocardial infarction in mice because cells embolized in lung are\\nactivated to secrete the anti-inflammatory protein TSG-6. *Cell Stem\\nCell* 5: 54--63\\n\\nLefkopoulos S, Polyzou A, Derecka M, Bergo V, Clapes T, Cauchy P,\\nJerez-Longres C, Onishi-Seebacher M, Yin N, Martagon-Calderón N-A, *et\\nal* (2020) Repetitive Elements Trigger RIG-I-like Receptor Signaling\\nthat Regulates the Emergence of Hematopoietic Stem and Progenitor Cells.\\n*Immunity* 53: 934-951.e9\\n\\nLi C, Li G, Liu M, Zhou T & Zhou H (2016) Paracrine effect of\\ninflammatory cytokine-activated bone marrow mesenchymal stem cells and\\nits role in osteoblast function. *J Biosci Bioeng* 121: 213--219\\n\\nLi H, Li D, Ma Z, Qian Z, Kang X, Jin X, Li F, Wang X, Chen Q, Sun H,\\n*et al* (2018) Defective autophagy in osteoblasts induces endoplasmic\\nreticulum stress and causes remarkable bone loss. *Autophagy* 14:\\n1726--1741\\n\\nLi J, Li X, Liu D, Hamamura K, Wan Q, Na S, Yokota H & Zhang P (2019)\\neIF2α signaling regulates autophagy of osteoblasts and the development\\nof osteoclasts in OVX mice. *Cell Death Dis* 10: 921\\n\\nLiao Y, Smyth GK & Shi W (2014) featureCounts: an efficient general\\npurpose program for assigning sequence reads to genomic features.\\n*Bioinformatics* 30: 923--930\\n\\nLin D, Chai Y, Ma Y, Duan B, Yuan Y & Liu C (2019) Rapid initiation of\\nguided bone regeneration driven by spatiotemporal delivery of IL-8 and\\nBMP-2 from hierarchical MBG-based scaffold. *Biomaterials* 196: 122--137\\n\\nLiu W, Zhou L, Zhou C, Zhang S, Jing J, Xie L, Sun N, Duan X, Jing W,\\nLiang X, *et al* (2016) GDF11 decreases bone mass by stimulating\\nosteoclastogenesis and inhibiting osteoblast differentiation. *Nat\\nCommun* 7: 12794\\n\\nLiu X, Liu Z, Sun L, Ren J, Wu Z, Jiang X, Ji Q, Wang Q, Fan Y, Cai Y,\\n*et al* (2021) Resurrection of human endogenous retroviruses during\\naging reinforces senescence. *bioRxiv*\\n\\nLories RJU & Schett G (2012) Pathophysiology of New Bone Formation and\\nAnkylosis in Spondyloarthritis. *Rheum Dis Clin North Am* 38: 555--567\\n\\nLu JY, Chang L, Li T, Wang T, Yin Y, Zhan G, Han X, Zhang K, Tao Y,\\nPercharde M, *et al* (2021) Homotypic clustering of L1 and B1/Alu\\nrepeats compartmentalizes the 3D genome. *Cell Res* 31: 613--630\\n\\nLudwig J, Schuberth C, Goldeck M, Schlee M, Li H, Juranek S, Sheng G,\\nMicura R, Tuschl T, Hartmann G, *et al* (2010) Structural and functional\\ninsights into 5′-ppp RNA pattern recognition by the innate immune\\nreceptor RIG-I. *Nat Struct Mol Biol*\\n\\nMangiavacchi A, Liu P, Della Valle F & Orlando V (2021) New insights\\ninto the functional role of retrotransposon dynamics in mammalian\\nsomatic cells. *Cell Mol Life Sci* 78: 5245--5256\\n\\nMarsell R & Einhorn TA (2011) The biology of fracture healing. *Injury*\\n42: 551--555\\n\\nMaruyama M, Rhee C, Utsunomiya T, Zhang N, Ueno M, Yao Z & Goodman SB\\n(2020) Modulation of the Inflammatory Response and Bone Healing. *Front\\nEndocrinol (Lausanne)* 11: 386\\n\\nMashanov VS, Zueva OR & García-Arrarás JE (2012) Posttraumatic\\nregeneration involves differential expression of long terminal repeat\\n(LTR) retrotransposons. *Dev Dyn* 241: 1625--1636\\n\\nMatsushita T, Chan YY, Kawanami A, Balmes G, Landreth GE & Murakami S\\n(2009) Extracellular signal-regulated kinase 1 (ERK1) and ERK2 play\\nessential roles in osteoblast differentiation and in supporting\\nosteoclastogenesis. *Mol Cell Biol* 29: 5843--5857\\n\\nMeier F, Brunner AD, Frank M, Ha A, Bludau I, Voytik E,\\nKaspar-Schoenefeld S, Lubeck M, Raether O, Bache N, *et al* (2020a)\\ndiaPASEF: parallel accumulation-serial fragmentation combined with\\ndata-independent acquisition. *Nat Methods* 17: 1229--1236\\n\\nMeier F, Brunner AD, Frank M, Ha A, Bludau I, Voytik E,\\nKaspar-Schoenefeld S, Lubeck M, Raether O, Bache N, *et al* (2020b)\\ndiaPASEF: parallel accumulation-serial fragmentation combined with\\ndata-independent acquisition. *Nat Methods* 17: 1229--1236\\n\\nMountziaris PM, Spicer PP, Kasper FK & Mikos AG (2011) Harnessing and\\nModulating Inflammation in Strategies for Bone Regeneration. *Tissue Eng\\nPart B Rev* 17: 393\\n\\nNallagatla SR & Bevilacqua PC (2008) Nucleoside modifications modulate\\nactivation of the protein kinase PKR in an RNA structure-specific\\nmanner. *RNA*\\n\\nNelson AM, Reddy SK, Ratliff TS, Hossain MZ, Katseff AS, Zhu AS, Chang\\nE, Resnik SR, Page C, Kim D, *et al* (2015) dsRNA Released by Tissue\\nDamage Activates TLR3 to Drive Skin Regeneration. *Cell Stem Cell* 17:\\n139--151\\n\\nNimiritsky P, Eremichev R, Alexandrushkina N, Efimenko A, Tkachuk V &\\nMakarevich P (2019) Unveiling Mesenchymal Stromal Cells\\' Organizing\\nFunction in Regeneration. *Int J Mol Sci* 20: 823\\n\\nNollet M, Santucci-Darmanin S, Breuil V, Al-Sahlanee R, Cros C, Topi M,\\nMomier D, Samson M, Pagnotta S, Cailleteau L, *et al* (2014) Autophagy\\nin osteoblasts is involved in mineralization and bone homeostasis.\\n*Autophagy* 10: 1965--1977\\n\\nOwlarn S, Klenner F, Schmidt D, Rabert F, Tomasso A, Reuter H, Mulaw MA,\\nMoritz S, Gentile L, Weidinger G, *et al* (2017) Generic wound signals\\ninitiate regeneration in missing-tissue contexts. *Nat Commun 2017 81*\\n8: 1--13\\n\\nPardi N, Muramatsu H, Weissman D & Karikó K (2013) In vitro\\ntranscription of long RNA containing modified nucleosides. *Methods Mol\\nBiol*\\n\\nPercharde M, Lin CJ, Yin Y, Guan J, Peixoto GA, Bulut-Karslioglu A,\\nBiechele S, Huang B, Shen X & Ramalho-Santos M (2018) A LINE1-Nucleolin\\nPartnership Regulates Early Development and ESC Identity. *Cell*\\n\\nRau A, Gallopin M, Celeux G & Jaffrézic F (2013) Data-based filtering\\nfor replicated high-throughput transcriptome sequencing experiments.\\n*Bioinformatics* 29: 2146--2152\\n\\nRecknagel S, Bindl R, Brochhausen C, Göckelmann M, Wehner T, Schoengraf\\nP, Huber-Lang M, Claes L & Ignatius A (2013) Systemic inflammation\\ninduced by a thoracic trauma alters the cellular composition of the\\nearly fracture callus. *J Trauma Acute Care Surg* 74: 531--537\\n\\nReppe S, Refvem H, Gautvik VT, Olstad OK, Høvring PI, Reinholt FP,\\nHolden M, Frigessi A, Jemtland R & Gautvik KM (2010) Eight genes are\\nhighly associated with BMD variation in postmenopausal Caucasian women.\\n*Bone*\\n\\nRitschka B, Storer M, Mas A, Heinzmann F, Ortells MC, Morton JP, Sansom\\nOJ, Zender L & Keyes WM (2017) The senescence-associated secretory\\nphenotype induces cellular plasticity and tissue regeneration. *Genes\\nDev* 31: 172--183\\n\\nRoberts SJ & Ke HZ (2018) Anabolic Strategies to Augment Bone Fracture\\nHealing. *Curr Osteoporos Rep* 16: 289--298\\n\\nRobinson MD, McCarthy DJ & Smyth GK (2010) edgeR: a Bioconductor package\\nfor differential expression analysis of digital gene expression data.\\n*Bioinformatics* 26: 139--140\\n\\nSadeq S, Al-Hashimi S, Cusack CM & Werner A (2021) Endogenous\\nDouble-Stranded RNA. *Non-Coding RNA* 7: 15\\n\\nSchmidt-Bleek K, Schell H, Schulz N, Hoff P, Perka C, Buttgereit F, Volk\\nHD, Lienau J & Duda GN (2012) Inflammatory phase of bone healing\\ninitiates the regenerative healing cascade. *Cell Tissue Res* 347:\\n567--573\\n\\nSheng W, LaFleur MW, Nguyen TH, Chen S, Chakravarthy A, Conway JR, Li Y,\\nChen H, Yang H, Hsu P-H, *et al* (2018) LSD1 Ablation Stimulates\\nAnti-tumor Immunity and Enables Checkpoint Blockade. *Cell* 174:\\n549-563.e19\\n\\nSimon M, Van Meter M, Ablaeva J, Ke Z, Gonzalez RS, Taguchi T, De Cecco\\nM, Leonova KI, Kogan V, Helfand SL, *et al* (2019) LINE1 Derepression in\\nAged Wild-Type and SIRT6-Deficient Mice Drives Inflammation. *Cell\\nMetab* 29: 871-885.e5\\n\\nSlotkin RK & Martienssen R (2007) Transposable elements and the\\nepigenetic regulation of the genome. *Nat Rev Genet* 8: 272--285\\n\\nStrieter RM, Burdick MD, Gomperts BN, Belperio JA & Keane MP (2005) CXC\\nchemokines in angiogenesis. *Cytokine Growth Factor Rev* 16: 593--609\\n\\nTimmen M, Hidding H, Wieskötter B, Baum W, Pap T, Raschke MJ, Schett G,\\nZwerina J & Stange R (2014) Influence of antiTNF-alpha antibody\\ntreatment on fracture healing under chronic inflammation. *BMC\\nMusculoskelet Disord* 15: 184\\n\\nDella Valle F, Reddy P, Yamamoto M, Liu P, Saera-Vila A, Bensaddek D,\\nZhang H, Prieto Martinez J, Abassi L, Celii M, *et al* (2022) LINE-1 RNA\\ncauses heterochromatin erosion and is a target for amelioration of\\nsenescent phenotypes in progeroid syndromes. *Sci Transl Med* 14:\\neabl6057\\n\\nVaranasi SS, Olstad OK, Swan DC, Sanderson P, Gautvik VT, Reppe S,\\nFrancis RM, Gautvik KM & Datta HK (2010) Skeletal Site-Related Variation\\nin Human Trabecular Bone Transcriptome and Signaling. *PLoS One* 5:\\ne10692\\n\\nWang S, Lee C, Kim J, Hyun J, Lim M, Cha H-J, Oh S-H, Choi YH & Jung Y\\n(2017) Tumor necrosis factor-inducible gene 6 protein ameliorates\\nchronic liver damage by promoting autophagy formation in mice. *Exp Mol\\nMed* 49: e380\\n\\nWeckbach S, Hohmann C, Braumueller S, Denk S, Klohs B, Stahel PF,\\nGebhard F, Huber-Lang MS & Perl M (2013) Inflammatory and apoptotic\\nalterations in serum and injured tissue after experimental polytrauma in\\nmice: distinct early response compared with single trauma or\\n\\'double-hit\\' injury. *J Trauma Acute Care Surg* 74: 489--498\\n\\nWheatley BM, Nappo KE, Christensen DL, Holman AM, Brooks DI & Potter BK\\n(2019) Effect of NSAIDs on Bone Healing Rates. *J Am Acad Orthop Surg*\\n27: e330--e336\\n\\nWiśniewski JR, Zougman A, Nagaraj N & Mann M (2009) Universal sample\\npreparation method for proteome analysis. *Nat Methods* 6: 359--362\\n\\nXiao G, Gopalakrishnan R, Jiang D, Reith E, Benson MD & Franceschi RT\\n(2002a) Bone morphogenetic proteins, extracellular matrix, and\\nmitogen-activated protein kinase signaling pathways are required for\\nosteoblast-specific gene expression and differentiation in MC3T3-E1\\ncells. *J Bone Miner Res* 17: 101--110\\n\\nXiao G, Jiang D, Gopalakrishnan R & Franceschi RT (2002b) Fibroblast\\ngrowth factor 2 induction of the osteocalcin gene requires MAPK activity\\nand phosphorylation of the osteoblast transcription factor, Cbfa1/Runx2.\\n*J Biol Chem* 277: 36181--36187\\n\\nXiao G, Jiang D, Thomas P, Benson MD, Guan K, Karsenty G & Franceschi RT\\n(2000) MAPK Pathways Activate and Phosphorylate the Osteoblast-specific\\nTranscription Factor, Cbfa1 \\\\*. *J Biol Chem* 275: 4453--4459\\n\\nYang A, Lu Y, Xing J, Li Z, Yin X, Dou C, Dong S, Luo F, Xie Z, Hou T,\\n*et al* (2018a) IL-8 Enhances Therapeutic Effects of BMSCs on Bone\\nRegeneration via CXCR2-Mediated PI3k/Akt Signaling Pathway. *Cell\\nPhysiol Biochem* 48: 361--370\\n\\nYang H, Tian W, Wang S, Liu X, Wang Z, Hou L, Ge J, Zhang X, He Z & Wang\\nX (2018b) TSG-6 secreted by bone marrow mesenchymal stem cells\\nattenuates intervertebral disc degeneration by inhibiting the TLR2/NF-κB\\nsignaling pathway. *Lab Investig* 98: 755--772\\n\\nYang WR, Ardeljan D, Pacyna CN, Payer LM & Burns KH (2019) SQuIRE\\nreveals locus-specific regulation of interspersed repeat expression.\\n*Nucleic Acids Res* 47: e27--e27\\n\\nZhang R, Liu Y, Yan K, Chen L, Chen X-R, Li P, Chen F-F & Jiang X-D\\n(2013) Anti-inflammatory and immunomodulatory mechanisms of mesenchymal\\nstem cell transplantation in experimental traumatic brain injury. *J\\nNeuroinflammation* 10: 106\\n\\nZhang X, Zhang R & Yu J (2020) New Understanding of the Relevant Role of\\nLINE-1 Retrotransposition in Human Disease and Immune Modulation. *Front\\ncell Dev Biol* 8: 657\\n\\nZhu W, Kuo D, Nathanson J, Satoh A, Pao GM, Yeo GW, Bryant S V., Voss\\nSR, Gardiner DM & Hunter T (2012) Retrotransposon long interspersed\\nnucleotide element-1 (LINE-1) is activated during salamander limb\\nregeneration. *Dev Growth Differ* 54: 673--685\\n\\n**Figure Legends**\\n\\n**Figure 1 - TEs expression is induced after fracture and in\\nmechanically loaded bone.**\\n\\n**A)** Heatmap representation of RNAseq differentially expressed TEs\\n(LogFC\\\\>0.5) between 4h post-injury and intact femurs at different time\\npoints of the bone healing process. N=5 biological replicates for each\\ntime point. N=10 biological replicates for intact femur. **B)** Heatmap\\nrepresentation of RNAseq upregulated TEs (LogFC\\\\>0.5) between 4h and\\nintact at different time points of the bone healing process. N=5\\nbiological replicates for each time point. N=10 biological replicates\\nfor intact femur. **C)** Heatmap representation of RNAseq differentially\\nexpressed TEs analysis \\\\[Fragments Per Kilobase of transcript per\\nMillion (FPKM) fold change\\\\] in femoral (n=27) and iliac (n=34) bone\\nbiopsies from healthy donors. **D)** Pie charts showing the number and\\npercentage of differentially expressed TEs subfamilies between femoral\\nand iliac healthy bone. **E)** Heatmap representation of RNAseq\\ndifferentially expressed TEs analysis \\\\[Fragments Per Kilobase of\\ntranscript per Million (FPKM) fold change\\\\] in femoral (n=27) and iliac\\n(n=34) bone biopsies from healthy donors. One heatmap for each TEs order\\nis shown.\\n\\n**Figure 2 - TEs expression correlates with bone mineral density in\\nhuman weight-bearing bone.**\\n\\n**A)** Heatmap representation of RNAseq differentially expressed TEs\\nanalysis \\\\[Fragments Per Kilobase of transcript per Million (FPKM) fold\\nchange\\\\] in femoral bone from healthy (n=27), osteopenic (n=12), and\\nosteoporotic (n=9) donors. **B)** Heatmap representation of RNAseq\\ndifferentially expressed TEs analysis \\\\[Fragments Per Kilobase of\\ntranscript per Million (FPKM) fold change\\\\] in femoral bone with high\\nBMD (FN T-score \\\\>-1) (n=27) and low BMD (FN T-score \\\\<-1) (n=21).\\n**C)** Upper panels: pie charts showing the number and percentage of\\ndifferentially expressed TEs subfamilies between high BMD and low BMD\\nfemoral bone. Lower panels: heatmap representation of RNAseq\\ndifferentially expressed TEs analysis \\\\[Fragments Per Kilobase of\\ntranscript per Million (FPKM) fold change\\\\] in femoral bone with high\\nBMD (n=27) and low BMD (n=21). One heatmap for each TEs order is shown.\\n**D)** Upper panels: Heatmap representation of correlation analysis (*P*\\nvalue\\\\<0.05) between TEs expression (FPKM) and local BMD (FN T-score) in\\nfemoral bone biopsies (n=48). Correlation between TE expression (FPKM\\nvalues) and FN t-score was evaluated using Pearson method. Significant\\ncorrelations (p-value \\\\< 0.05) were selected for the heatmap\\nrepresentation and arranged according to the Pearson correlation\\ncoefficient (r). r \\\\> 0, positive correlations; r \\\\< 0, negative\\ncorrelations. Lower panels: Pie charts showing the percentage of TEs\\nsubfamilies positively correlated to local BMD.\\n\\n**Figure 3 - L1 RNA delivery stimulates the mineralization of\\ndifferentiating osteoblasts**\\n\\n**A)** Experimental workflow and flow cytometer analysis showing the\\npercentage of positive cells 6 hours after L1 RNA delivery at day 5 of\\n*ex vivo* osteogenesis. Intracellular localization of cy5 conjugated\\nsynthetic L1 RNA (red spots) three days after transfection is also shown\\nfrom a typical experiment (right). **B)** qPCR analysis of intracellular\\nL1 RNA level 24 h (d6), 5 days (d10), and 9 days (d14)\\npost-transfection. The graph is shown as mean\\u2009±\\u2009s.d. of n\\u2009=3 independent\\nexperiments. **C)** cy5 conjugated synthetic L1 RNA (red) and osteoimage\\nstained mineral matrix (green) detection 9 days post L1 RNA\\ntransfection. The white arrowheads indicate the colocalization between\\nL1 RNA and hydroxyapatite. **D)** Osteoimage stained mineral matrix\\nquantification (upper panels), images (central panels), and Alizarin Red\\n(lower panels) 9 days after L1 RNA or RFP RNA delivery in 2 healthy\\ndonors-derived MSCs (D188 and D170). RFU= Relative Fluorescence Units.\\nThe graph is shown as mean\\u2009±\\u2009s.d. of n\\u2009=\\u200911 technical replicates.\\n\\\\*\\\\*\\\\*\\\\*P\\u2009\\\\<\\u20090.00005 in Wilcoxon test. **E)** Alizarin Red staining of\\nosteoblasts transfected with increasing doses of RFP RNA (upper panels)\\nand L1 RNA (lower panels). **F)** Alizarin Red images (upper panel) and\\nquantification (lower panel) of MSCs mineralization after 14, 17 and 21\\ndays of *ex vivo* differentiation. MSCs were obtained from the femur of\\n4 healthy (D188, D239, D247, D170) and 4 OP patients (HUK7, HUK9, HUK12,\\nHUK16). N=9 technical replicates for each donor and time point. Boxes in\\nthe boxplot indicate the interquartile range (50% of data), while the\\nlower end and the upper end represent the 1^st^ and the 3^rd^ quartile,\\nrespectively. The solid line inside the box represents the median.\\nWhiskers represent the max and min values. Values that are not within\\n1.5 times the interquartile range are considered as outliers and lie\\noutside the whiskers. **G)** Osteoimage stained mineral matrix\\nquantification (upper panels), images (central panels), and Alizarin Red\\n(lower panels) 9 days after L1 RNA or RFP RNA delivery in 3 OP patients\\nderived MSCs (HUK9, HUK12, HUK16). RFU= Relative Fluorescence Units. The\\ngraph is shown as mean\\u2009±\\u2009s.d. of n\\u2009=\\u200910-12 technical replicates.\\n\\\\*\\\\*\\\\*\\\\*P\\u2009\\\\<\\u20090.00005 in Wilcoxon test. **H)** qRT-PCR of early\\nosteogenic genes in RFP and L1 transfected osteoblasts at different time\\npoints of osteogenic differentiation. Expression level is normalized on\\nday 5 (not transfected osteoblasts). N=3 biological replicates. The\\ngraph is shown as mean\\u2009±\\u2009s.d. of n\\u2009=\\u20093 independent experiments.\\n\\\\*P\\u2009\\\\<\\u20090.05; \\\\*\\\\*P\\u2009\\\\<\\u20090.005, \\\\*\\\\*\\\\*P\\u2009\\\\<\\u20090.0005, \\\\*\\\\*\\\\*\\\\*P\\u2009\\\\<\\u20090.00005 in\\nStudent\\'s t test.\\n\\n**Figure 4 - L1 RNA delivery induces an inflammatory response\\ncharacteristic of bone repair process**\\n\\n**A)** Left panel: heatmap representation of RNAseq differentially\\nexpressed gene analysis \\\\[Fragments Per Kilobase of transcript per\\nMillion (FPKM) fold change\\\\] in osteoblasts transfected with negative\\ncontrol RNA (RFP) or L1 RNA. Right panel: bubble plot showing gene\\nontology (GO) enrichment analysis of L1 upregulated biological\\nprocesses. **B)** Tree plot showing gene ontology (GO) enrichment\\nanalysis of L1 upregulated biological processes (left) and cellular\\ncomponents (right). **C)** Top 25 upregulated biological processes 24h\\npost-L1 RNA delivery *in vitro* (yellow) and 4h post fracture *in vivo*\\n(blue). Shared GO terms are shown in green. **D)** qPCR analysis of\\n\"inflammatory response\" (GO:0006954) and \"immune response\" (GO:0006955)\\ngenes at different time points post L1 RNA transfection at day 5. \\\\*\\n\\\\*\\\\*P\\u2009\\\\<\\u20090.005, \\\\*\\\\*\\\\*\\\\*P\\u2009\\\\<\\u20090.00005 in two way ANOVA.\\n\\n**Figure 5 - PKR mediates L1 RNA-induced stress response and\\nmineralization**\\n\\n**A)** Upper panel: schematic representation of the pathway inhibited by\\nLamivudine 3TC and G140. Lower panel: Alizarin Red staining of\\ndifferentiating osteoblasts transfected with L1 RNA and treated with\\nLamivudine 3TC and G140. **B)** IF on RFP and L1 transfected\\nosteoblasts, 6h post-transfection, shows the colocalization between cy5\\nsignal (L1 RNA, red) and 488-Anti-dsRNA signal (dsRNA, green). Nuclei\\nare stained with Hoechst (blue).\\n\\n**C)**Western blot analysis showing the ratio between total eIF2α and\\nphosphorylated eIF2α\\xa0(P- eIF2α). H3: endogenous standard Histone 3.\\n**D)** Volcano plot (left) showing the number of significantly\\nupregulated and downregulated protein 24h post-L1 RNA transfection.\\nRight: tree plots showing gene ontology (GO) enrichment analysis of\\nbiological processes (upper panel) and cellular components (lower panel)\\ndownregulated by L1 RNA delivery (MS data). **E)** qPCR analysis of\\n\"inflammatory response\" (GO:0006954) and \"immune response\" (GO:0006955)\\ngenes 24h post L1 RNA transfection with and without PKR inhibitor C16.\\n**F)** Alizarin Red staining of differentiating osteoblasts transfected\\nwith L1 RNA with and without PKR inhibitor C16. **G)** Tree plots\\nshowing gene ontology (GO) enrichment analysis of biological processes\\nupregulated by L1 RNA 24h after transfection (MS data).\\n\\n**Figure 6 - L1-RNA-induced changes in osteoblast secretome**\\n\\n**A)** Left: Alizarin red staining on recipient OB 9 days after the\\ndelivery of conditioned media. Right: Microscope images of recipient OB\\n24h from the delivery of conditioned media. Mineralized nodules in OB\\nreceiving conditioned media from L1 treated OB are shown (orange\\narrows). **B)** Heat map of differentially expressed proteins in bulk\\nsecretome and exosomes derived from untransfected (NT), RFP- and\\nL1-transfected osteoblasts. N=3 biological replicates. **C)** GO\\nenrichment analysis of differentially expressed protein (adjusted\\np-value \\\\<0.05) in the bulk secretome of L1 compared to RFP transfected\\nosteoblasts. **D)** GO enrichment analysis of differentially expressed\\nprotein (adjusted p-value \\\\<0.05) in the exosomes of L1 compared to RFP\\ntransfected osteoblasts.\\n'"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pypandoc_md = pypandoc.convert_file(DOCX_FILES[0], 'markdown')\n",
    "pypandoc_md"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1. PymuPDF"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pymupdf\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Is possible to use `TOC` to find the sections in the documents?\n",
    "\n",
    "It does not seem to be the case, as shown below:\n",
    "\n",
    "The engine behind `MuPDF` has its own functionality to convert `docx` files into something readable."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 69,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "docx_doc = pymupdf.open(DOCX_FILES[0])\n",
    "docx_doc.get_toc()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[[1, 'Manuscript Text', 1],\n",
       " [1, 'Figure 1', 65],\n",
       " [1, 'Figure 2', 66],\n",
       " [1, 'Figure 3', 67],\n",
       " [1, 'Figure 4', 68],\n",
       " [1, 'Figure 5', 69],\n",
       " [1, 'Figure 6', 70],\n",
       " [1, 'Figure 7', 71],\n",
       " [1, 'Figure 8', 72],\n",
       " [1, 'Figure EV1', 73],\n",
       " [1, 'Figure EV2', 74],\n",
       " [1, 'Figure EV3', 75],\n",
       " [1, 'Figure EV4', 76],\n",
       " [1, 'Figure EV5', 77],\n",
       " [1, 'Table EV1', 78],\n",
       " [1, 'Table EV2', 79],\n",
       " [1, 'Table EV3', 81]]"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pdf_doc = pymupdf.open(PDF_FILES[0])\n",
    "pdf_doc.get_toc()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Converting the files to markdown format"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pymupdf4llm"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing ./data/docx/EMBOJ-2023-115257.docx...\n",
      "[                                        ] (0/3[=                                       ] ( 1/3[==                                      ] ( 2/32[===                                     ] ( 3/3=[=====                                   ] ( 4/3[======                                  ] ( 5/32[=======                                 ] ( 6/3[========                                ] ( 7/32=[==========                              ] ( 8/32[===========                             ] ( 9/3[============                            ] (10/32[=============                           ] (11/3=[===============                         ] (12/3[================                        ] (13/32[=================                       ] (14/3[==================                      ] (15/32=[====================                    ] (16/32[=====================                   ] (17/3[======================                  ] (18/32[=======================                 ] (19/3=[=========================               ] (20/3[==========================              ] (21/32[===========================             ] (22/3[============================            ] (23/32=[==============================          ] (24/32[===============================         ] (25/3[================================        ] (26/32[=================================       ] (27/3=[===================================     ] (28/3[====================================    ] (29/32[=====================================   ] (30/3[======================================  ] (31/32=[========================================] (32/32]\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "\"LINE-1 RNA triggers matrix formation in bone cells via a PKR-mediated inflammatory response\\n\\nArianna Mangiavacchi,[1@] Gabriele Morelli,[1] Sjur Reppe,[2,3,4 ]Alfonso Saera-Vila,[5] Peng Liu,[1] Benjamin Eggerschwiler,[6,7] Huoming\\n\\nZhang,[8] Dalila Bensaddek,[8] Elisa A. Casanova,[5] Carolina Medina Gomez,[9] Vid Prijatelj,[9] Francesco Della Valle,[1 ] Nazerke\\n\\nAtinbayeva,[1] Juan Carlos Izpisua Belmonte,[10] Fernando Rivadeneira,[9] Paolo Cinelli,[5,11] Kaare Morten Gautvik,[3*] Valerio Orlando.\\n\\n1*@\\n\\n1 King Abdullah University of Science and Technology (KAUST), Biological Environmental Science and\\n\\nEngineering Division, Thuwal 23500-6900, Kingdom of Saudi Arabia\\u200b\\n\\n2 Oslo University Hospital, Department of Medical Biochemistry, Oslo, Norway.\\n\\n3Lovisenberg Diaconal Hospital, Unger-Vetlesen Institute, Oslo, Norway.\\n\\n4Oslo University Hospital, Department of Plastic and Reconstructive Surgery, Oslo, Norway.\\n\\n5 Sequentia Biotech, Carrer Comte D'Urgell 240, Barcelona, 08036, Spain.\\n\\n6 Department of Trauma, University Hospital Zurich, Sternwartstrasse 14, 8091, Zurich, Switzerland\\n\\n7 Life Science Zurich Graduate School, University of Zurich, Winterthurerstrasse 190, 8057, Zurich, Switzerland.\\n\\n8 Core Labs, King Abdullah University of Science and Technology (KAUST), Thuwal 23500-6900, Kingdom of\\n\\nSaudi Arabia\\u200b.\\n\\n9 Department of Internal Medicine, Erasmus Medical Centre, Rotterdam, the Netherlands.\\n\\n10 Altos Labs, San Diego, CA, USA\\n\\n11 Center for Applied Biotechnology and Molecular Medicine, University of Zurich, Winterthurerstrasse 190, 8057,\\n\\nZurich, Switzerland.\\n\\n*senior author\\n\\n@corresponding author\\n\\n# Abstract\\n\\nTransposable elements (TEs) are mobile genetic modules of viral derivation that have been co-opted to become\\n\\nmodulators of mammalian gene expression. TEs are a major source of endogenous dsRNAs, signaling molecules\\n\\nable to coordinate inflammatory responses in various physiological processes. Here, we provide evidence for a\\n\\npositive involvement of TEs in inflammation-driven bone repair and mineralization. In newly fractured mice bone,\\n\\n\\n-----\\n\\nwe observed an early transient upregulation of repeats occurring concurrently with the initiation of the inflammatory\\n\\nstage. In human bone biopsies, analysis revealed a significant correlation between repeats expression, mechanical\\n\\nstress and bone mineral density. We investigated a potential link between LINE-1 (L1) expression and bone\\n\\nmineralization by delivering a synthetic L1 RNA to osteoporotic patient-derived mesenchymal stem cells and\\n\\nobserved a dsRNA-triggered protein kinase (PKR)-mediated stress response that led to strongly increased\\n\\nmineralization. This response was associated with a strong and transient inflammation, accompanied by a global\\n\\ntranslation attenuation induced by eIF2\\uf061 phosphorylation. We demonstrated that L1 transfection reshaped the\\n\\nsecretory profile of osteoblasts, triggering a paracrine activity that stimulated the mineralization of recipient cells.\\n\\nKeywords: dsRNA/inflammation/osteoblast /PKR/transposable elements\\n\\n# Introduction\\n\\nRepetitive elements are, in origin, parasitic transposable elements (TEs) of viral derivation and during evolution\\n\\nhave become regulatory modules of gene expression, eventually integrated into developmental, resilience, and cell\\n\\nidentity programs.(Cosby et al, 2019) (Mangiavacchi et al, 2021) (Jachowicz et al, 2017) Although cells have\\n\\nevolved several defense mechanisms to prevent deleterious uncontrolled retrotransposon reactivation,(Slotkin &\\n\\nMartienssen, 2007) increasing evidence demonstrates that retrotransposons are also involved in non-pathological\\n\\ncontexts, particularly as non-coding RNAs.(Mangiavacchi et al, 2021) As part of resilience mechanisms, all\\n\\nretrotransposons are a major source of endogenous double-stranded RNAs (dsRNAs), that serve as a cellular\\n\\nsignaling molecule to coordinate inflammatory and immune responses in various physiological processes.(Fukuda et\\n\\nal, 2021) (De Cecco et al, 2019a) (Simon et al, 2019) (Sadeq et al, 2021) (Chen & Hur, 2022) (Zhang et al, 2020)\\n\\nThe transient and controlled breaching of the activation threshold of dsRNA sensors allows the integration of innate\\n\\nimmune functions within the frame of physiological processes.(Chen & Hur, 2022) For example, repetitive elements\\n\\ntranscribed during development drive RIG-I-like receptors (RLR)-mediated inflammatory signals that regulate\\n\\nhematopoietic stem and progenitor cells (HSPC) formation.(Lefkopoulos et al, 2020) During cell replication,\\n\\ndsRNA-mediated activation of PKR ensures proper progression of mitosis.(Kim et al, 2014) Moreover,\\n\\nepigenetically mediated derepression of repetitive elements leads to dsRNA production and to the activation of an\\n\\ninflammatory response stimulating anti-tumor T cell immunity.(Sheng et al, 2018) Furthermore, after skin injury,\\n\\n\\n-----\\n\\ndsRNAs induce inflammatory pathways contributing to wound healing and hair regeneration.(Nelson et al, 2015)\\n\\nSeveral studies have demonstrated a positive effect of proinflammatory mediators on bone anabolism in vitro and in\\n\\nvivo,(Croes et al, 2015) (Croes et al, 2016) (Li et al, 2016) (Croes et al, 2017) (Laroche et al, 2011) suggesting that\\n\\nan inflammatory reaction may be harnessed for bone regenerative purposes. Indeed, sterile inflammation is a\\n\\nfundamental component of the osteogenic microenvironment: the inflammatory reaction is the earliest response to\\n\\ninjury and is crucial to initiate and orchestrate fracture repair and activate bone anabolic processes. (Mountziaris et\\n\\nal, 2011) (Marsell & Einhorn, 2011) (Bahney et al, 2019) Accordingly, a dysregulated inflammation negatively\\n\\nimpacts optimal bone regeneration, (Wheatley et al, 2019) (Gerstenfeld et al, 2001) (Recknagel et al, 2013)\\n\\n(Hurtgen et al, 2016) (Jiao et al, 2015) and often precedes the excessive bone formation associated with heterotopic\\n\\nossification and several vertebral column pathological conditions. (Lories & Schett, 2012) (Balboni et al, 2006)\\n\\nTherefore, identifying the key molecular factors triggering the sterile inflammation that precedes regeneration in\\n\\nbone would allow us to modulate the inflammatory reaction in a spatiotemporal and contextual manner as well as to\\n\\ndevelop new anabolic strategies for the treatment of bone loss conditions, such as osteoporosis, or impaired bone\\n\\nrepair.(Roberts & Ke, 2018)\\n\\nHere, we show that 1) In mice, immediately after bone fracture, a transient upregulation of TEs concurs with the\\n\\ninitiation of the inflammatory stage; 2) in humans, an increased expression of repeats is observed in bones with a\\n\\nhigher mechanical stress-induced anabolic demand and expression of TEs correlates with local mineral density; 3)\\n\\nThe delivery of L1 RNA, but not control RNA, to human bone marrow-derived mesenchymal stem cells committed\\n\\nto osteoblasts stimulates a unique mineralizing phenotype in a dose-dependent manner; 4) L1-treated osteoblasts\\n\\nshow upregulation of inflammatory genes and transcriptional changes characteristic for the earliest stage of bone\\n\\nrepair; 5) cytoplasmic L1 RNA is sensed by dsRNA-activated protein kinase (PKR), which mediates eIF2α\\n\\nphosphorylation and a global attenuation of protein synthesis; 6) the inhibition of PKR prevents inflammation,\\n\\ninhibits translation and induces mineralization caused by elevated levels of cytoplasmic L1 RNA; 7) cells\\n\\ntransfected with L1 RNA undergo significant changes in their secretome composition and initiate a paracrine effect\\n\\nstimulating the mineralization of osteogenic competent recipient cells.\\n\\nOur results indicate that TEs are induced by bone damage and stimulate bone mineralization through an\\n\\ninflammatory response triggered by dsRNA sensing and mediated by a paracrine activity.\\n\\n\\n-----\\n\\n# Results\\n\\nRetrotransposon expression is triggered by fracture in vivo and correlates with bone mineral density in\\n\\nhuman weight-bearing bones\\n\\nInflammation is the earliest response in fracture healing. To investigate a possible involvement of TEs in the sterile\\n\\ninflammatory response triggered by bone injury, we analyzed TEs expression in time-course RNA-Seq repository\\n\\ndata obtained from a full-fracture bone healing mice model.(Coates et al, 2019) We focused on TEs expression\\n\\ndynamics immediately after fracture (acute inflammatory stage) and observed four clusters of differentially\\n\\nexpressed TEs (logFC > 0.5) between intact and post-fracture bone after 4h, a time point corresponding to the\\n\\ninitiation of acute inflammation (Figure 1A). Cluster 1 includes TEs subfamilies whose expression slightly increases\\n\\nafter fracture and during the whole healing process. Clusters 2 and 4 include TEs subfamilies that are downregulated\\n\\nafter fracture but show increased expression at later time points. The major group, cluster 3, is represented by TEs\\n\\nsubfamilies whose expression is highly and transiently upregulated after fracture (Figure 1A and 1B). Interestingly,\\n\\nthe induction of these TEs, mostly Long Interspersed Nuclear Elements (LINEs) and Long Terminal Repeats (LTRs)\\n\\n(Figure 1B), is transient and limited to the earliest phase of the inflammatory stage necessary and sufficient to\\n\\ninitiate and orchestrate fracture repair and activate bone anabolic processes.(Mountziaris et al, 2011)[,](Marsell &\\n\\nEinhorn, 2011) These results suggest that, in bone, retrotransposon reactivation is an early event in response to\\n\\nfracture, possibly linked to inflammation initiation which is fundamental for a healthy healing. Indeed,\\n\\nretrotransposon involvement in posttraumatic tissue regeneration was previously found in other organisms.\\n\\n(Mashanov et al, 2012)[,](Zhu et al, 2012)\\n\\nTo assess gene expression dynamics immediately after fracture in humans represents a technical and ethical hurdle.\\n\\nA comparison between bones experiencing different degrees of mechanical loading may represent an alternative\\n\\nmodel to study stress-induced bone anabolism in humans. Mechanically loaded bones like the femur, which are\\n\\nsubject to recurring microfractures that need to be repaired, are indicated to be more metabolically active than less\\n\\nloaded bones, like the iliac crest, as pointed out by previous transcriptome studies.(Aerssens et al, 1997) (Varanasi et\\n\\nal, 2010)\\n\\nWe compared the expression of TEs in trabecular bone from ilium and femoral head, two different skeletal sites, the\\n\\nlatter experiencing a higher degree of mechanical loading, anabolic demand and bone turnover.(Aerssens et al,\\n\\n1997) Femur (n=48) and ilium biopsies (n=71) were isolated from a cohort of clinically well-characterized donors\\n\\n\\n-----\\n\\n(refer to cohort description and Appendix Table S1 and S2). Each cohort was divided into normal (BMD T-score >\\n1), osteopenic (−2.5 < BMD T-score ≤ −1), and osteoporotic (BMD T-score ≤ −2.5, with at least one fragility\\n\\nfracture). TEs show generally higher upregulation in healthy femurs (n=27) compared to healthy ilia (n=34) (Figure\\n\\n1C and 1E). Each TEs order showed a high percentage of upregulated subfamilies, from 60% of DNA transposon to\\n\\n75% of LINEs, and a low percentage of downregulated subfamilies, from 9% of LTRs to 3% of Short interspersed\\n\\nnuclear elements (SINEs) and LINEs, in femur compared to ilium (Figure 1D). These results suggested a positive\\n\\ncorrelation between TEs global expression and bone anabolic activity.\\n\\nTo further test our hypothesis, we compared TEs expression in femurs from healthy donors (n=27), osteopenic\\n\\n(n=12) and osteoporotic patients (n=9) whose bone anabolism is increasingly compromised. As shown in Figure 2A,\\n\\nthe global expression of TEs is reduced in patients. To further assess whether TEs expression is involved in local\\n\\nanabolism and associated with mineral density, we divided the femur cohort according to the DXA T-score measured\\n\\nin the femoral neck (FN). We divided the cohort into “high BMD” (FN T-score >-1) and “low BMD” (FN T-score <\\n1). Globally, TEs are upregulated in femurs with high bone mineral density (Figure 2B). In particular, more than\\n\\n90% of differentially expressed TEs subfamilies are upregulated (Figure 2C). Interestingly, almost all SINE\\n\\nelements are unchanged between the two groups (Figure 2C). Finally, we assessed whether a positive correlation\\n\\nexists between TEs expression levels and local mineral density (FN T-score). We found that approximately 30% of\\n\\nL1 and LTR subfamilies expressed in the femoral bone are positively correlated with local BMD (Figure 2D).\\n\\nAltogether, these results strongly suggest a positive involvement of TEs in response to stress-induced anabolic\\n\\ndemand and bone mineralization.\\n\\nIncreased cytoplasmic L1 repeat RNA stimulates osteoblast mineralization activity\\n\\nThe in vivo evidence suggested a positive involvement of retrotransposons in post-traumatic bone repair in mice, and\\n\\na strong correlation between retrotransposon expression and bone mineral density in humans, and higher expression\\n\\nin loaded compared to unloaded bone. We then corroborated the results by studying the effects of increasing repeat\\n\\nRNA levels on differentiating osteoblasts derived from mesenchymal stem cells (MSCs) isolated from femurs of\\n\\nhealthy donors. At day 5 of differentiation, a Cy5-conjugated full-length L1 RNA consensus sequence was\\n\\ntransfected. Capping, 2’-O-Methylation of 5’ end, polyadenylation (200 adenosines), full substitution with 5\\nmethylcytidine (m5C) and 75% substitution with pseudouridine were used to stabilize the RNA and to bypass the\\n\\n\\n-----\\n\\nintracellular innate immune system.(Koski et al, 2004) (Pardi et al, 2013) (Ludwig et al, 2010) (Karikó et al, 2005)\\n\\n(Nallagatla & Bevilacqua, 2008) (Karikó et al, 2008) (Anderson et al, 2010) (Kormann et al, 2011) Red fluorescent\\n\\nprotein (RFP) mRNA, with the same modifications, was included as negative control. The exogenous L1 RNA\\n\\naccumulated in the cytoplasm (Figure 3A) and was gradually cleared by secretion along with matrix components\\n\\n(Figure 3B and 3C). The effect of L1 RNA delivery was tested in differentiating MSCs from two different and\\n\\nunrelated healthy donors. As shown, L1 RNA transfection markedly stimulated the production of mineralized matrix\\n\\n(Figure 3D). Osteoblast mineralization increased dose-dependently with the concentration of the delivered L1 RNA,\\n\\nin agreement with saturation kinetics (Figure 3E). Moreover, the strongly enhanced mineralization was highly\\n\\nspecific for L1 RNA, as demonstrated by the lack of significant changes in mineral deposition in cells transfected\\n\\nwith negative control RNA, even at concentrations 100-200-fold higher (figure 3E). We also isolated MSCs from the\\n\\nfemur of osteoporotic patients and selected those showing a markedly delayed and reduced in vitro production of\\n\\nbone matrix in contrast to those derived from healthy donors (Figure 3F). Strikingly, also in these cells with low\\n\\nosteogenic capacity, the delivery of L1 RNA triggered a strong mineralization response despite their clearly\\n\\ncompromised anabolic activity (Figure 3G). These results demonstrate convincingly that the ectopic delivery of L1\\n\\nRNA stimulates the mineralization of bone-forming cells regardless of their prior intrinsic differentiation/anabolic\\n\\npotential.\\n\\nOsteoporotic patient-derived cells transfected with L1 RNA show a higher expression of early differentiation\\n\\nmarkers, with a peak 48h after L1 transfection (day 7) (Figure 3H). Bone sialoprotein (IBSP), a late osteogenic gene,\\n\\nshows a similar profile, being almost 6 times more expressed in L1- than in RFP-transfected cells on day 7\\n\\n(Appendix Figure S1A). Strikingly, although alkaline phosphatase (ALPL) expression slightly increases at later time\\n\\npoints (Appendix Figure S1A), its activity is not enhanced in L1-treated cells (Appendix Figure S2A). However, the\\n\\nactivity of ectonucleotide pyrophosphatase/phosphodiesterase 1 (ENPP1) is gradually and significantly reduced by\\n\\nL1 RNA transfection (Appendix Figure S2B). As ENPP1 is a major source of inorganic pyrophosphate (PPi), one of\\n\\nthe main physiological inhibitors of mineralization,(FLEISCH et al, 1966) (Johnson et al, 2000) (Hessle et al, 2002)\\n\\nthe reduction of its activity may contribute to the enhanced mineralization observed in L1-RNA transfected cells.\\n\\nAltogether, these data suggest that L1 RNA induces a unique mineralizing phenotype and apparently distinct from\\n\\nthe well-known canonical differentiation mechanisms.\\n\\n\\n-----\\n\\nL1 RNA delivery induces inflammatory pathways significantly overlapping those involved in bone fracture\\n\\nrepair\\n\\nTo investigate the molecular response of osteoblasts to L1 RNA transfection, we profiled their transcriptome by\\n\\nIllumina RNA-Seq 24h post-transfection. We found 482 differentially expressed genes (DEG) (FDR<0.05) between\\n\\nL1 and RFP RNA-treated OB (268 upregulated and 214 downregulated) (Appendix Figure S3). Gene ontology (GO)\\n\\nenrichment analysis revealed that the early transcriptional signature of L1-treated osteoblasts is typical of that of an\\n\\ninflammatory response (Figure 4A and 4B). This is in accordance with the recent evidence of a link between L1 and\\n\\nother retrotransposons accumulation in the cytoplasm and inflammatory response in vitro and in vivo.(Fukuda et al,\\n\\n2021) (De Cecco et al, 2019b) (Simon et al, 2019) GO term enrichment analysis comparison between the early stage\\n\\nof fracture-induced bone healing in vivo(Coates et al, 2019) and differentiating osteoblasts in vitro 24h post-L1\\n\\ndelivery (Figure 4A) revealed a matching inflammatory response and shared upregulated pathways (Figure 4C)\\n\\nessential for effective bone repair.(Maruyama et al, 2020) (Gerstenfeld et al, 2003) (Kon et al, 2001) Recruitment of\\n\\nneutrophils at the site of injury also indicates a healthy repair process, particularly in bone.(Kovtun et al, 2016)\\n\\nExtracellular signal-regulated kinase 1 (ERK1) and 2 (ERK2), play crucial roles in bone formation(Xiao et al, 2000)\\n\\n(Xiao et al, 2002b) (Xiao et al, 2002a) (Matsushita et al, 2009) (Ge et al, 2007) and seem to be the most upstream\\n\\ninitiators of tissue regeneration in planaria.(Owlarn et al, 2017) Upon bone injury, an initial transient stage of acute\\n\\ninflammation is a crucial factor in ensuring an effective regeneration(Maruyama et al, 2020) whereas an\\n\\nexcessive/prolonged (chronic) inflammation is deleterious for the healing environment.(Chan et al, 2012) (Hurtgen\\n\\net al, 2016) (Clark et al, 2017) (Brem & Tomic-Canic, 2007) (Timmen et al, 2014) (Schmidt-Bleek et al, 2012)\\n\\n(Weckbach et al, 2013) We followed the time-course expression of “inflammatory response” (GO:0006954) and\\n\\n“immune response” (GO:0006955) genes upregulated by L1 RNA immediately after transfection. As shown in\\n\\nFigure 4D, their expression is already strongly reduced at day 10 and almost entirely silenced by day 14, suggesting\\n\\nthe transient dynamics of the inflammatory response triggered in vitro by L1 RNA. This is coherent with our\\n\\nprevious observation that exogenous L1 RNA is gradually secreted by the cell together with matrix components\\n\\n(Figure 3B and C), eventually clearing out the initial inflammatory stimulus.\\n\\nCellular response to increased cytoplasmic level of L1 RNA is mediated by PKR\\n\\nIt has been previously demonstrated that the cytoplasmic accumulation of L1-derived cDNA triggers an\\n\\n\\n-----\\n\\ninflammatory response mediated by the cyclic GMP-AMP synthase (cGAS)-stimulator of interferon genes (STING)\\n\\ncytosolic DNA sensing pathway.(De Cecco et al, 2019a) Immunofluorescence (IF) analysis reveals that the\\n\\nexogenous L1 RNA forms DNA:RNA hybrids (Appendix Figure S4), suggesting that L1 RNA is reversed\\n\\ntranscribed into cDNA after its transfection. Notably, ORF1p was undetectable by WB in differentiating osteoblasts\\n\\nand in L1 transfected osteoblasts (Appendix Figure S5), indicating that the exogenous L1 RNA is not translated after\\n\\ntransfection. To assess whether L1-induced inflammation was triggered by the sensing of its cDNA, we used\\n\\nLamivudine 3TC and G140 (Figure 5A) to inhibit the ORF2-mediated reverse transcription of L1 RNA and cGAS\\n\\nactivity, respectively. Moreover, we also performed a siRNA-mediated knockdown (KD) of cGAS (Appendix Figure\\n\\nS6). (Jones et al, 2008) (Lama et al, 2019) Surprisingly, none of the treatments impaired the mineralization induced\\n\\nby L1 RNA delivery (Figure 5A, Appendix Figure S6C), suggesting that the observed cellular response depends\\n\\nentirely on RNA sensing. As previously mentioned, repeats are a major source of endogenous dsRNA,(Sadeq et al,\\n\\n2021) (Chen & Hur, 2022) and L1 RNA, in particular, is able to form intramolecular double-stranded structures.\\n\\n(Hur, 2019) As expected, we found a tight colocalization between L1 RNA and dsRNA IF signal (Figure 5B). One of\\n\\nthe main cellular sensors of dsRNA, including those derived from retrotransposons, is PKR.(Kim et al, 2018) PKR\\n\\nactivation results in the phosphorylation of eukaryotic translation initiation factor 2\\uf061 (eIF2α) and in the subsequent\\n\\ninhibition of global protein synthesis and cell growth.(Donnelly et al, 2013) Interestingly, PKR has been shown to\\n\\ninteract with L1 RNA in vitro.(Kim et al, 2018) Moreover, eIF2α phosphorylation has been demonstrated to promote\\n\\nautophagy in osteoblasts and to counteract BMD reduction in osteoporotic ovariectomized mice.(Li et al, 2019)\\n\\nTo demonstrate that L1 RNA is able to trigger a PKR-mediated intracellular response, we transfected L1 RNA in the\\n\\npresence or absence of the PKR inhibitor C16 and evaluated the levels of eIF2α phosphorylation by western blot\\n\\n(Figure 5C). The delivery of L1, but not RFP, strongly induces eIF2α phosphorylation and this effect is prevented by\\n\\nPKR inhibition (Figure 5C). Notably, eIF2α phosphorylation occurs already 6 hours post-transfection. This\\n\\nobservation suggests that PKR activation is one of the first events induced by L1 RNA, thus corroborating the\\n\\nhypothesis of PKR as an endogenous L1 RNA sensor.\\n\\nWe then characterized the proteome of cells 24h post-L1 RNA transfection by diaPASEF mass spectrometry (MS).\\n\\n(Meier et al, 2020a) We detected almost 8000 proteins in each sample, and observed that more than 88% of\\n\\nsignificantly differentially expressed proteins (DEP) were downregulated in L1-treated cells (Figure 5D, left panel).\\n\\nMoreover, among the top 15 downregulated biological processes and cellular components were translation and\\n\\n\\n-----\\n\\nribosomal subunits (Figure 5D, right panels, yellow circles), indicating that L1 RNA induces a global attenuation of\\n\\nprotein synthesis. Indeed, a cap-dependent translation shutdown is the main consequence of eIF2α phosphorylation.\\n\\n(Donnelly et al, 2013) Finally, we show that neither PKR KD (Appendix Figure S6A, B) nor its pharmacological\\n\\ninhibition by C16 treatment (Figure 5E and 5F) prevents L1 RNA-triggered induction of inflammatory genes (Figure\\n\\n5E) and mineral matrix deposition (Figure 5F, Appendix Figure S6B). As previously mentioned, a global reduction\\n\\nof protein synthesis occurs 24h after L1 transfection and only a little percentage of DEP is upregulated in L1\\n\\ncompared to RFP control (Figure 5D). Interestingly, some of the most upregulated proteins are involved in\\n\\nautophagy and vesicle trafficking (Figure 5G). Autophagy is a crucial process for mineralization and bone\\n\\nhomeostasis both in vitro and in vivo,(Nollet et al, 2014) as autophagic vacuoles are exploited to secrete apatite\\n\\ncrystals.\\n\\nAltogether these data demonstrate that the PKR-mediated sensing of L1-derived dsRNA globally attenuates the\\n\\ntranslation via eIF2α phosphorylation and indicate autophagy as a possible mechanism involved in L1-induced\\n\\nosteoblasts mineralization.\\n\\nL1 RNA delivery reshapes the secretory profile of differentiating osteoblasts\\n\\nAn inflammatory response translates into a strong secretory activity and production of paracrine signals, and we\\n\\ntherefore tested the effect of “L1-primed” osteoblast-derived conditioned medium in recipient differentiating\\n\\nosteoblasts (Figure 6A). We found that the “secretome” of cells transfected with L1 RNA has a significant paracrine\\n\\neffect on recipient osteoblasts since they showed an earlier formation of mineralized nodules, already 24h post the\\n\\naddition of conditioned media (Figure 6A, right panel), and higher deposition of mineral matrix at day 10 (Figure\\n\\n6A, left panel). We subsequently isolated bulk and exosome-derived proteomes from conditioned media of\\n\\nuntreated, RFP- and L1-treated differentiating osteoblasts and characterized them by MS. Differential expression\\n\\nanalysis of MS data revealed a unique secretome profile (for both bulk and vesicular proteomes) of L1-primed\\n\\nosteoblasts compared to RFP-primed and untreated osteoblasts (Figure 6B). Gene Ontology (GO) analysis of\\n\\ndifferentially expressed proteins showed an enrichment of pro-inflammatory factors (i.e. interleukins and\\n\\nchemokines) involved in immune response and chemotactic migration of immune cells (Figure 6C and D), crucial\\n\\nprocesses for tissue repair mechanisms in vivo. Pro-inflammatory molecules are also a major constituent of the\\n\\nsenescence-associated secretory phenotype (SASP),(Coppé et al, 2010) whose transient delivery supports cellular\\n\\n\\n-----\\n\\nplasticity and tissue regeneration.(Ritschka et al, 2017) Notably, the most enriched protein in L1-specific bulk\\n\\nsecretome is interleukin 8 (IL-8), an inflammatory chemokine involved in several regenerative processes, such as\\n\\nskin wound healing,(JIANG et al, 2012) therapeutic angiogenesis after stroke and ischemia,(Choi et al, 2021) (Hou\\n\\net al, 2014) and osteochondral bone repair.(Lin et al, 2019) (Yang et al, 2018a) Tumor necrosis factor-inducible gene\\n\\n6 (TSG6) is among the top 10 proteins enriched in the bulk secretome of L1-primed osteoblasts compared to RFP.\\n\\nTSG6 is an inflammatory factor with suggestive therapeutic effects in corneal wounds, myocardial infarction,\\n\\ninjured central nervous system, chronic liver damage, and intervertebral disc degeneration.(Zhang et al, 2013) (Lee\\n\\net al, 2015) (Lee et al, 2009) (Wang et al, 2017) (Yang et al, 2018b) Moreover, TSG6 induces autophagy influx both\\n\\nin vivo and in vitro.(Wang et al, 2017) Another protein positively involved in autophagy and found specifically in\\n\\nthe secretome of L1 primed osteoblasts was ATG7 (autophagy-related 7). Osteoblast-specific ATG7 conditional\\n\\nknockout in mice reduced bone mass during developmental and in adult age.(Li et al, 2018) Type 1 lysophosphatidic\\n\\nacid receptor (LPAR1) is also specifically secreted by L1-primed osteoblasts and has proven to be positively\\n\\ninvolved in bone mineralization in vitro and in vivo.(Alioli et al, 2020) (Gennero et al, 2011)\\n\\nAltogether, our findings demonstrate that cytoplasmic L1 RNA is sensed by PKR, whose activation leads to eIF2α\\n\\nphosphorylation with consequent reprogramming of transcription and translation. The ultimate consequence of this\\n\\nL1-induced, PKR-mediated stress response in osteoblasts is a major reprogramming of their secretory activity,\\n\\nleading to a paracrine effect on recipient osteogenic competent cells to trigger their mineralization activity.\\n\\n# Discussion\\n\\nThe concerted co-option of TEs markedly changed whole regulatory networks and integrated new functions into the\\n\\neukaryote genome.(Cosby et al, 2019) (Mangiavacchi et al, 2021) (Carelli et al, 2022) However, while the role of\\n\\nTEs as evolutionary drivers is today well-established, their contribution to cell physiology, particularly in somatic\\n\\ncells, remains to be elucidated. Indeed, aside from their role in the nucleus, (Della Valle et al, 2022) (Jachowicz et al\\n\\n, 2017) (Lu et al, 2021) (Percharde et al, 2018) TEs represent a major source of endogenous dsRNA whose role in\\n\\ntriggering inflammatory and innate immune responses in various physiological processes has increasingly gained\\n\\nattention.(Sadeq et al, 2021) (Chen & Hur, 2022) For instance, an early acute inflammatory response is of vital\\n\\nimportance in initiating and orchestrating bone repair and induce anabolism leading to fracture healing. (Mountziaris\\n\\net al, 2011) (Marsell & Einhorn, 2011) (Bahney et al, 2019) Moreover, several studies demonstrated a positive effect\\n\\n\\n-----\\n\\nof proinflammatory mediators on bone mineralization in vitro and in vivo,(Croes et al, 2015) (Croes et al, 2016) (Li\\n\\net al, 2016) (Croes et al, 2017) indicating that the modulation of inflammatory reaction is a tool to pursue\\n\\nregenerative strategies in bone.(Roberts & Ke, 2018) We found that, in the bone of newly fractured mice, a group of\\n\\nTEs, mostly LINE and LTR, was early, markedly and transiently upregulated. Interestingly, the expression of these\\n\\nTEs was limited to the earliest phase of the inflammatory stage. In line with the hypothesis of a positive correlation\\n\\nbetween stress-induced bone mineralization and TEs activation, the expression dynamics in human bone biopsies\\n\\nrevealed a significant upregulation of TEs in the femur, a bone subjected to high mechanical stresses, high turnover\\n\\nand anabolism, compared to ilium, not weight bearing. Furthermore, we detected a lower TEs expression in donors\\n\\nwith compromised bone mineralization and a significant correlation between TEs expression and local BMD. We\\n\\nhypothesized that the expression dynamics of TEs in fracture/mechanical stress observed in vivo could mimic the\\n\\nimmunological threshold model of the sterile activation of dsRNA sensors as a response to stressful conditions.\\n\\nIndeed, the transient and controlled breaching of the activation threshold of dsRNA sensors may lead to sterile\\n\\ninflammation being integrated into physiological processes.(Chen & Hur, 2022) To test further this hypothesis, we\\n\\ndelivered a full-length L1 consensus sequence as source of dsRNA(Hur, 2019) to differentiating osteoblasts in vitro\\n\\nand showed that L1 RNA initially accumulated in the cytoplasm and is gradually cleared by secretion. The increased\\n\\ncytoplasmic levels of L1 RNA are immediately sensed by PKR, a cytoplasmic dsRNA sensor whose activation\\n\\ninduces the phosphorylation of eIF2\\uf061\\uf028\\uf04b\\uf069\\uf06d \\uf065\\uf074 \\uf061\\uf06c\\uf02c \\uf032\\uf030\\uf031\\uf038\\uf029 \\uf028\\uf044\\uf06f\\uf06e\\uf06e\\uf065\\uf06c\\uf06c\\uf079 \\uf065\\uf074 \\uf061\\uf06c\\uf02c \\uf032\\uf030\\uf031\\uf033\\uf029, leading to the attenuation of\\n\\nglobal protein synthesis (Donnelly et al, 2013), induction of inflammatory response, and stimulation of mineral\\n\\nmatrix deposition. In support of the evidence that PKR is the early sensor of increased L1 RNA cytoplasmic levels,\\n\\nwe show that inhibition of PKR activation by C16 prevents L1-induced eIF2\\uf061 phosphorylation, induction of\\n\\ninflammatory genes, and mineral matrix deposition. Moreover, the inhibition of L1 cDNA formation and sensing by\\n\\nLamivudine 3TC, G140, or cGAS KD does not affect the L1-induced phenotype. Thus, the evidence does not\\n\\nsupport that the L1-triggered inflammation is causally connected to a cGAS-STING-dependent sensing of cytosolic\\n\\nL1 cDNA, a mechanism demonstrated operative in senescence.(De Cecco et al, 2019a)\\n\\nL1 RNA response showed high specificity, as demonstrated by the dose-dependent increase of mineralization and\\n\\nthe lack of significant changes in mineral deposition when negative control RNA was transfected, even at 100-200\\nfold higher concentration. Although the molecular mechanism behind L1-induced stress response which stimulates\\n\\n\\n-----\\n\\nthe mineralization activity of osteoblasts remains elusive, our results indicate that L1 stimulates the mineral matrix\\n\\ndeposition by bone-forming cells regardless of their prior intrinsic anabolic potential. Moreover, MS results\\n\\nsuggested that increased autophagy, a process crucial for mineralization and bone homeostasis in vitro and in vivo\\n\\n(Nollet et al, 2014) may contribute to L1-induced hydroxyapatite deposition. Intriguingly, the osteoinductive\\n\\nproperties of L1-primed osteoblast-derived secretome and the observation that L1 RNA was gradually secreted by\\n\\nthe cells may suggest an involvement of L1 RNA in cell-to-cell communication, in line with the recent finding of\\n\\nother retrotransposons (i.e. ERV) acting as paracrine molecules.(Liu et al, 2021) This, together with the observed\\n\\nL1-dependent induction of CCL and CXCL chemokines and cytokines, being chemoattractants of immune cells and\\n\\nendothelial cells,(Bischoff et al, 2011) (Eash et al, 2010) (Strieter et al, 2005) and the composition of L1-primed\\n\\nosteoblast secretome, makes the in vivo delivery of L1 RNA a necessary step to evaluate its actual contribution to\\n\\nthe paracrine processes that orchestrate bone repair mechanisms. (Chaparro & Linero, 2016) (Nimiritsky et al, 2019)\\n\\nThe identification of repeats RNA, particularly L1, as potential resilient molecular factors involved in stress\\ninduced, inflammation-mediated bone production points out new avenues to develop anabolic strategies for the\\n\\ntreatment of bone loss conditions, such as osteoporosis or impaired bone repair.(Roberts & Ke, 2018) On the other\\n\\nhand, repeats may be novel targets in the treatment of those conditions characterized by excessive or heterotopic\\n\\nossification preceded by deregulated/chronic inflammation.(Lories & Schett, 2012) (Balboni et al, 2006)\\n\\n# Materials and Methods\\n\\nParticipants and ethics\\n\\nFemoral bone biopsies were obtained from the caput region of postmenopausal women or men with a wide BMD\\n\\nrange, i.e., from healthy to osteoporotic, who were undergoing hip replacement surgery due to osteoarthritis or\\n\\nfracture at Lovisenberg Diaconal Hospital (Oslo, Norway) or Diakonhiemmet Hospital (Oslo, Norway), respectively.\\n\\nThe donors of femoral bones are listed in Appendix Table S1. The postmenopausal iliac bone donors were recruited\\n\\nfrom the outpatient clinic of Lovisenberg Diaconal Hospital (Oslo, Norway). Candidates filled out a questionnaire\\n\\nthat included medication and lifestyle factors, and selected donors were deemed representative of the Oslo-based\\n\\nNorwegian ethnic female population aged 50 to 86 years. The iliac bone donors are listed in Appendix Table S2 and\\n\\nhave been described previously in detail.(Reppe et al, 2010) All donors taking medication or having diseases, other\\n\\nthan primary osteoporosis, known to affect bone metabolism were excluded. The presence of bone-impairing\\n\\n\\n-----\\n\\ndiseases/conditions was excluded by extensive biochemical serum and urine analyses supported by X-ray\\n\\nexaminations. The site-specific BMD of all donors was evaluated with Lunar Prodigy DEXA (GE Lunar, Madison,\\n\\nWI, USA) following the manufacturer's instructions. The precision of the instrument for measuring the lumbar spine\\n\\n(L2–L4) and hip BMD was 1.7% and 1.1%, respectively. The study was approved by the Norwegian Regional Ethical\\n\\nCommittee (REK no 2010/2539, Norway), all volunteers gave their written informed consent, and sampling and\\n\\nprocedures were according to the Act of Biobanking in Norway.\\n\\nCell culture\\n\\nFor L1 RNA transfection experiments, human MSCs were isolated from osteoporotic femoral heads with the consent\\n\\nof the patient according to Swiss (BASEC-Nr. 2016-02159) ethical guidelines. Femoral heads were crushed and\\n\\nincubated in DMEM (PAN Biotech, Germany, Cat. No. P04-03550) supplemented with 1% (v/v) 100× Penicillin\\nStreptomycin Solution (Biowest, France, Cat. No. L0022), 10% (v/v) FCS (Biowest, France, Cat. No. S181S), 1%\\n\\n(v/v) 200mM L-Glutamine solution (Sigma, USA, Cat. No. G7513), 5ng/ml FGF-2 (Sigma, USA, Cat. No. F0291)\\n\\nand 10ng/ml FGF-4 (Sigma, USA, Cat. No. F8424) in a humidified atmosphere containing 5% CO2. To induce\\n\\nosteogenic differentiation, MSCs were seeded on Nunc™ 24-well plates (Thermo Fisher, USA, Cat. No. 142475) or\\n\\n48-well plates (Thermo Fisher, USA, Cat. No. 150687) at a density of 1.5 × 10[4] cells/cm2. After 24 h, differentiation\\n\\nwas induced using StemPro® Osteogenesis Kit (Gibco/Life Technologies, USA, Cat. No. A10072-01). The medium\\n\\nwas exchanged every 3 days. Cells were treated with Lamivudine 3TC (Sigma), 1\\uf06dM final concentration; C16\\n\\n(Merck), 500nM final concentration; G140 (InvivoGen), 150\\uf06dM final concentration.\\n\\nL1 RNA transfection\\n\\nThe vector human-L1_pBluescript II sk (+) carrying the full-length L1 sequence was custom-prepared by GenScript,\\n\\nUSA. Large-scale human L1 mRNA was in vitro transcribed, modified, and purified by TriLink Biotechnologies,\\n\\nUSA, (ARCA capped and 2’Omethymalted (CapI), fully substituted with 5-methyl-C, 25% substitution of Cyanine\\n5-U and 75% substitution of Pseudo-U, enzymatically polyadenylated, DNase and phosphatase treated, silica\\n\\nmembrane purified). Synthetic L1 RNA was transfected in MSCs differentiating to osteoblasts at day 5 using\\n\\nLipofectamine[TM] MessengerMAX[TM ] (Invitrogen, USA, Cat. No. LMRNA003) with a modified protocol for low\\n\\nRNA amount. RFP mRNA (System Bioscience, USA, Cat. No. MR800A-1) was used as negative control. Bone\\n\\nmatrix was quantified with OsteoImage Mineralization Assay (Lonza, Basel, Switzerland, Cat. No. LOPA503) and\\n\\nAlizarin Red staining. In conditioned media experiments, the medium was exchanged 6h after transfection.\\n\\n\\n-----\\n\\nConditioned medium was collected after three days and delivered to the recipient, differentiating osteoblasts at day\\n\\n5. In knockdown experiments, siRNAs were transfected with Lipofectamine RNAiMAX (Thermo Scientific) in\\n\\ndifferentiating osteoblasts at day 3, followed by L1 RNA transfection at day 7. siRNAs: EIF2AK2- Targeting\\n\\nSMARTpool (Dharmacon, Horizon, E-003527-00-0010), MB21D1-Targeting SMARTpool (Dharmacon, Horizon,\\n\\nL-015607-02-0020), Non-targeting Pool (D-001810-10).\\n\\nQuantitative mineralization assay\\n\\nCells were washed in PBS and fixed in 4% (v/v) formaldehyde (Sigma, USA, Cat. No. F8775) in 1× PBS for 15\\n\\nminutes. Mineralization was assessed by using the OsteoImage Mineralization Assay (Lonza, Basel, Switzerland,\\n\\nCat. No. LOPA503) according to the manufacturer’s indication. Mineralization was quantified with GloMax®\\n\\ndiscover microplate reader (Promega, USA), selecting appropriate excitation (492) / emission (520) wavelengths.\\n\\nAlizarin Red staining\\n\\nOsteoblasts were washed with 1× PBS (Kantonsapotheke Zürich, Switzerland, Cat. No. A171012) and fixed with\\n\\n4% (v/v) formaldehyde (Sigma, USA, Cat. No. F8775) in 1× PBS for 30 min. After washing twice with ddH2O,\\n\\nAlizarin Red staining solution (0.7 g Alizarin Red S (Sigma, USA, Cat. No. A5533) diluted in 50 ml ddH2O at\\n\\npH = 4.2) was added for 20 min. Afterwards, cells were washed four times with ddH2O, dried, and stored in the dark\\n\\nuntil image acquisition. For absorbance measurement, Alizarin Red S was eluted from stained osteoblasts with 300\\n\\nμl 10% (w/v) cetylpyridinium chloride in an aqueous 0.01 M Na2HPO4/NaH2PO4 solution at pH = 7 for 1 h. One\\n\\nhundred fifty microliters were transferred on a 96-well plate, and absorbance was measured at 560 nm. Ten percent\\n\\n(w/v) cetylpyridinium chloride in an aqueous 0.01 M Na2HPO4/NaH2PO4 solution was used as blank. Images were\\n\\nacquired, processed and analysed as previously described(Eggerschwiler et al, 2019)\\n\\nALPL activity assay:\\n\\nALPL activity was measured as in Liu et al., 2016.(Liu et al, 2016) Briefly, cells were fixed in 3.7% formaldehyde\\n\\nat RT for 10’ and then stained with a solution of 25% naphthol AS-BI phosphate (Thermo) and 0.75% Fast Blue BB\\n\\n(Sigma) dissolved in 0.1 M Tris buffer (pH 9.3) at RT for 15’. After staining, cells were quickly washed with PBS\\n\\nfive times.\\n\\nENPP1 activity assay\\n\\nENPP1 activity was calculated colorimetrically using the chromogenic substrate p-nitrophenyl-thymidine-5′\\nmonophosphate, as described in (Ferreira et al., 2013). After PBS washing, cells were lysed in a solution of 0.1%\\n\\n\\n-----\\n\\nTriton X-100, 0.2 M Tris-base, 1.6 mM MgCl2, pH 8.1. 50 \\uf06dl of 1mM thymidine monophosphate p-nitrophenyl\\n\\nester (Sigma) were added to 50 \\uf06dl of cell lysate and incubated for 1 hour at 37°C. Then, four volumes of 0.1M\\n\\nNaOH were added to stop the reaction and absorbance was read at 410 nm using a spectrophotometer (Tecan\\n\\nNanoquant Infinite 200 Pro Multimode). Absorbance was normalized to DNA content.\\n\\nImmunohistochemistry:\\n\\nCells were fixed in 4% paraformaldehyde in PBS at room temperature (RT) for 10’. After fixation, cells were\\n\\npermeabilized in 3% Triton X-100 in PBS for 3’ at RT. Blocking was performed with 4% BSA in PBS for 30’ at RT.\\n\\nAfter blocking, cells were incubated with primary antibodies overnight at 4°C, then washed with PBS and incubated\\n\\nwith secondary antibodies for 1 hour at RT. After washing with PBS, nuclei were stained with Hoechst (Thermo\\n\\nScientific) and mounted using Fluoro Gel with DABCO™ Mounting Medium (Electron Microscopy Sciences).\\n\\nPrimary antibodies: DNA-RNA Hybrid (mouse, Merck Millipore, MABE1095), dsRNA (mouse, Merck Millipore,\\n\\nMABE1134).\\n\\nRNA extraction and cDNA preparation\\n\\nCells were harvested and resuspended in 1ml of QIAzol Lysis reagent (Qiagen, Cat. No.79306). Total RNA was then\\n\\npurified with RNeasy Plus Mini kit (Qiagen, cat. No. 74134) with minimal modifications to manufacturer’s\\n\\ninstructions. DNase treatment (RNase-free DNase set, Qiagen, Cat. No. 79254) was performed to remove any\\n\\nresidual DNA. RNA quality and concentration were checked using a Nanodrop[TM] 2000 spectrophotometer\\n\\n(ThermoFisher). cDNA was synthesized from 200ng of each RNA sample using a Superscript III first-strand cDNA\\n\\nsynthesis system (ThermoFisher, cat. No.18080051) according to the manufacturer’s protocol. Processing of human\\n\\nbone biopsies and RNA isolation has been described previously.(Reppe et al, 2010)\\n\\nGene expression analysis\\n\\nReal-time quantitative polymerase chain reaction (qPCR) was performed with a 7900HT Fast Real-Time PCR\\n\\nsystem (Applied Biosystems). Each sample was analyzed in triplicate and normalized with the endogenous control\\n\\nRibosomal Protein L13A (RPL13A) for cDNA input concentration. No template and no RT were included as\\n\\nnegative controls. For each 15\\uf06dl reaction, 10ng (1ng for L1) of cDNA was mixed with 1\\uf06dM specific primers mix\\n\\nand 7.5\\uf06dl of Sybr[TM ]Select Master mix (Applied Biosystems, USA, Cat. No. 4472908). The reaction was incubated\\n\\nat 95°C for 10 minutes, followed by 40 cycles of denaturation at 95°C for 15 seconds, annealing at 60°C for 30\\n\\nseconds and elongation at 72°C for 30 seconds. Ct values were calculated by 7900HT Fast Real-Time PCR RQ\\n\\n\\n-----\\n\\nmanager software (Applied Biosystems, USA) and then normalized as \\uf044Ct between the gene of interest and the\\n\\nendogenous calibrator. Primers used in this study for gene expression analysis were designed using Primer3 (\\n\\n[http://www.ncbi.nlm.nih.gov/tools/primer-blast/). In all primer pairs, each primer matches a different exon.](http://www.ncbi.nlm.nih.gov/tools/primer-blast/)\\n\\nAmplicons length was 80-130 nucleotides. Primer sequences are reported in Appendix Table S3.\\n\\nCell cycle analysis\\n\\n2x10[5] MSCs were trypsinized for 5 min at 37°C, washed with PBS and 2% BSA, passed through a 70\\uf06dM strainer\\n\\n(Corning, USA, Cat. No. 352350) and then fixed at -20°C for 30 min in 70% ethanol. After washing with PBS and\\n\\n4% BSA, cells were resuspended in PBS and incubated for 1 hour at 37°C with RNase. Cells were then washed and\\n\\nresuspended in 100\\uf06dl of Flow Cytometry Staining Buffer (R&D System, USA, Cat. No. FC001). 10\\uf06dl of 1mg/ml\\n\\nPropidium iodide (PI) staining solution (Invitrogen, USA, Cat. No. P3566) was added to the single-cell solution,\\n\\ngently mixed, and incubated for 5 min in the dark. Cell cycle analysis was performed on BD FACSCanto II Flow\\n\\nCytometry System (BD-Biosciences), using BD FACSDiva Software (BD-Biosciences).\\n\\nRNA-Seq and data analysis\\n\\nRNA from human bone biopsies was sequenced at the Human Genotyping facility (HuGe-F) of Erasmus MC.\\n\\nTotal RNA-Seq library, from L1 RNA delivery experiments, was prepared with CORALL Total RNA library prep\\n\\nwith RiboCop rRNA for Human/Mouse/Rat depletion kit (Lexogen GmbH, Vienna, Austria) following\\n\\nmanufacturer’s instructions (library type: fr-secondstrand) by IGA Technology service (Italy). The final libraries\\n\\nwere checked using both Qubit 2.0 Fluorometer (Invitrogen, Carlsbad, CA) and Agilent Bioanalyzer DNA assay or\\n\\nCaliper (PerkinElmer, Waltham, MA). Libraries were then prepared for sequencing and sequenced on paired-end\\n\\n150 bp mode on NovaSeq6000 (Illumina, San Diego, CA). RNA-Seq read quality control (QC) analyses and\\n\\nfiltering of high-quality reads were executed using FastQC v0.11.9\\n\\n(http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) and BBDuk v35.85 [(https://jgi.doe.gov/data-and-](https://urldefense.com/v3/__https:/jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbduk-guide/__;!!Nmw4Hv0!0EikfGZVN-B-Xp5VkKLqowTItYk7Yav6lOsagjvuQwPqWY_Y3yszwKePkxm4puTKivdPL5LvCMjrSQwcKpyz9m7YVqqz09ekvfk$)\\n\\n[tools/software-tools/bbtools/bb-tools-user-guide/bbduk-guide/) by setting a minimum read length of 35 bp and a](https://urldefense.com/v3/__https:/jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbduk-guide/__;!!Nmw4Hv0!0EikfGZVN-B-Xp5VkKLqowTItYk7Yav6lOsagjvuQwPqWY_Y3yszwKePkxm4puTKivdPL5LvCMjrSQwcKpyz9m7YVqqz09ekvfk$)\\n\\nminimum Phred-quality score of 25. After trimming quality control, high-quality reads were aligned to the human\\n\\ngenome reference (GRCh38) with STAR 2.7.3a,(Dobin et al, 2013) while FeatureCounts 1.6.3 package(Liao et al,\\n\\n2014) was used to assign reads to genes. Next, lowly expressed genes across one or more experimental conditions\\n\\nwere filtered to eliminate the “uninformative” genes using HTSFilter v1.30.1.(Rau et al, 2013) Filtered gene data\\n\\nwere further processed with the EdgeR package v3.32.1(Robinson et al, 2010) to normalize (Trimmed Mean of M-7\\n\\n\\n-----\\n\\nvalues, TMM, method) the raw counts and perform differential gene expression analysis. Multiple testing correction\\n\\nwas performed with the FDR method(Benjamini & Hochberg, 1995) and significance level was set at FDR < 0.05.\\n\\nGene Ontology (GO) term enrichment was analyzed by performing hypergeometric tests11 for each individual term,\\n\\nand FDR correction was applied (FDR < 0.05). Expression of Interspersed Repeat elements was quantified using\\n\\nSQuiRE 0.9.9.92 (https://github.com/wyang17/SquIRE). SQuiRE provides locus-specific expression quantification\\n\\nalong with subfamily-level expression estimates counting unambiguously mapped reads, as well as ambiguously\\n\\nmapped reads using an expectation–maximization (EM) algorithm.(Yang et al, 2019) Briefly, reference genome and\\n\\nRepeatmasker annotation were downloaded from UCSC and prepared for the analysis with squire Fetch and squire\\n\\nClean, respectively. High-quality reads were mapped against the reference genome with STAR using squire Map and\\n\\nexpression was quantified using squire Call. Differential expression analysis was performed with squire Call.\\n\\nDIA-MS analysis using TimsTOF MS\\n\\nTotal protein extracts were prepared with RIPA buffer (50mMTris-cl pH 8.0, 5mM EDTA, 150mM Nacl, 15mM\\n\\nMgCl2, 1% NP-40, 1mM PMSF and 1X Protease Inhibitor Cocktail (PIC)). Further sonication step was included:\\n\\n(30S ON, 30S OFF, 10 cycles with Bioruptor). Equal 50 \\uf06dg protein extracts were concentrated to 30 \\uf06dl volume, then\\n\\ndiluted in 8M urea in 0.1M Tris-HCl, followed by protein digestion with trypsin, according to the FASP protocol.\\n\\n(Wiśniewski et al, 2009) After overnight digestion, the peptides were eluted from the filters with 25 mM ammonium\\n\\nbicarbonate buffer. The eluted peptides were processed in the desalting step using Sep-Pag C18 Column (waters)\\n\\nbased on the manufacturer’s instructions. Approximately 200 ng of peptide mixture per sample was analyzed using a\\n\\ntimsTOF Pro 2 QTOF mass spectrometer coupled with a nanoElute liquid chromatography system (Bruker Daltonik\\n\\nGmbH, Germany). The sample was injected directly into an RP-C18 Aurora emitter column (75 µm i.d.× 250 mm,\\n\\n1.6 μm, 120 Å pore size) (Ion Opticks, Australia) using a one-column separation method. An 80-min gradient was\\n\\nestablished using mobile phase A (0.1% FA in H2O) and mobile phase B (0.1% FA in Acetonitrile): 2–25% B for\\n\\n60 min, 25–37% for 10 min, ramping 37% to 95% in 5 min, and maintaining 95% B for 5 min. The column\\n\\ntemperature was set at 50[o]C and the flow rate at 250 nl/min. The sample eluting from the separation column was\\n\\nintroduced into the mass spectrometer via a CaptiveSpray nano-electrospray ion source (Bruker Daltonik GmbH)\\n\\nwith an electrospray voltage of 1.6 kV. The ion source temperature was set to 180 °C and a dry gas of 3 l/min. The\\n\\nsamples were analyzed using diaPASEF scheme(Meier et al, 2020b) consisting of 24 cycles including a total of 48\\n\\nmass width windows (13 Da (m/z) from m/z 400 to 1,000 and TIMS scan range from 0.63 to 1.35 Vs cm−2 (1/K0).\\n\\n\\n-----\\n\\nThe collisional energy increased linearly from 20.01 eV at 0.6 (1/K0) to 52.00 eV at 1.35 Vs cm−2 (1/K0). The scan\\n\\nrange for MS and MS/MS spectra was set to 100-1700 m/z. TIMS ramping time and accumulation time were set to\\n\\n100 milliseconds. The diaPASEF data were analyzed by directDIA approach using Spectronaut software (version 14)\\n\\nfollowing manufacture instructions. Up or down regulated proteins were determined using DEP (Differential\\n\\nEnrichment analysis of Proteomics data) R package. Significant results (adjusted p-value < 0.05) were subjected to\\n\\nGene Ontology enrichment analysis with clusterProfiler R package.\\n\\nWestern Blot\\n\\nTotal protein extracts were prepared by lysing cells in extraction buffer (HEPES KOH [pH 8.5], NaCl 400 mM,\\n\\nEDTA 0.1 mM, EGTA 0.1 mM, DTT 1 mM, 1× protease inhibitor, SDS 1%). Proteins were separated by\\n\\nelectrophoresis on BOLT 4%–12% bis-tris polyacrylamide precast gels in MES buffer (Life Technologies) and\\n\\ntransferred to a 0,2 \\uf06dm nitrocellulose membrane. Non-specific signals were blocked with 5% Milk-PBS-Tween0.5%\\n\\nand the membrane hybridized overnight at 4°C with primary and secondary antibodies diluted in blocking buffer.\\n\\nHorseradish peroxidase-conjugated secondary antibodies were revealed with the ECL chemiluminescence kit\\n\\n(Amersham), and signals were detected using ChemiDoc (Bio-Rad). Primary antibodies: PKR (rabbit, Abcam,\\n\\nab32052), cGAS (rabbit, Abcam, ab224144), LINE-1 ORF1 (mouse, Merck Millipore, MABC1152), EIF2a (mouse,\\n\\nAbcam, ab5369\\uf029\\uf02c phospho-EIF2\\uf061 (rabbit, Abcam, ab32157), Histone H3 (rabbit, Merk, 06-755).\\n\\nExosome isolation\\n\\nExosomes were isolated using Total Exosome Isolation Reagent (from cell culture media) (Thermo), following\\n\\nmanufacturer’s instructions. The exosome pellet was resuspended in 50 \\uf06dl of RIPA buffer and kept on ice for 30’,\\n\\nmixing every 5’. Samples were then sonicated (30’’ ON/30” OFF, 10 cycles), centrifuged at 13,000 x g for 20’ at\\n\\n4°C and collected as supernatants.\\n\\nStatistical analysis\\n\\nStatistic tests used for data analysis are indicated in the figure legends.\\n\\nData availability\\n\\nThe RNA-seq transcriptomics data on human bone biopsies have been deposited to BioProject with the dataset\\n\\nidentifier PRJNA764663 (https://www.ncbi.nlm.nih.gov/bioproject/PRJNA764663). The RNA-seq transcriptomics\\n\\ndata on transfected osteoblasts have been deposited to GEO with the dataset identifier GSE201774\\n\\n\\n-----\\n\\n(https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201774) (enter token: ififogaehzoxlgh).\\n\\nThe mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE\\n\\npartner repository with the dataset identifier PXD051195. Reviewer account details are username:\\n\\n[reviewer_pxd051195@ebi.ac.uk ; password: R6v0mIYe](mailto:reviewer_pxd051195@ebi.ac.uk)\\n\\n# Acknowledgment\\n\\nWe acknowledge kaust Smarth Health Initiative (KSHI); the South-Eastern Norway Regional Health Authority; the\\n\\nNorwegian Osteoporosis Society; Anders Jahre Foundation for the Promotion of Science; the Lovisenberg Diaconal\\n\\nHospital research fund; and the Research Council of Norway.\\n\\n# Funding: We are grateful for support from: KAUST BAS/1/1037-01-01, the European Union project\\n\\nOSTEOGENE (no. FP6-502491); ZonMw VIDI 016.136.367 grant, for funding the creation of the RNA-Seq dataset\\n\\nof hip primary bone and Oslo University Hospital, Ullevål, project #29750104.\\n\\n# Disclosure and competing interests statement: All the authors declare no competing interests.\\n\\n References\\n\\nAerssens J, Boonen S, Joly J & Dequeker J (1997) Variations in trabecular bone composition with anatomical site\\n\\nand age: potential implications for bone quality assessment. J Endocrinol 155: 411–421\\n\\nAlioli CA, Demesmay L, Laurencin-Dalacieux S, Beton N, Farlay D, Follet H, Saber A, Duboeuf F, Chun J, Rivera\\n\\nR, et al (2020) Expression of the type 1 lysophosphatidic acid receptor in osteoblastic cell lineage controls\\n\\nboth bone mineralization and osteocyte specification. Biochim Biophys Acta - Mol Cell Biol Lipids 1865:\\n\\n158715\\n\\nAnderson BR, Muramatsu H, Nallagatla SR, Bevilacqua PC, Sansing LH, Weissman D & Karikó K (2010)\\n\\nIncorporation of pseudouridine into mRNA enhances translation by diminishing PKR activation. Nucleic Acids\\n\\nRes\\n\\nBahney CS, Zondervan RL, Allison P, Theologis A, Ashley JW, Ahn J, Miclau T, Marcucio RS & Hankenson KD\\n\\n\\n-----\\n\\n(2019) Cellular biology of fracture healing. J Orthop Res 37: 35–50\\n\\nBalboni TA, Gobezie R & Mamon HJ (2006) Heterotopic ossification: Pathophysiology, clinical features, and the\\n\\nrole of radiotherapy for prophylaxis. Int J Radiat Oncol 65: 1289–1299\\n\\nBenjamini Y & Hochberg Y (1995) Controlling the False Discovery Rate: A Practical and Powerful Approach to\\n\\nMultiple Testing. J R Stat Soc Ser B 57: 289–300\\n\\nBischoff DS, Sakamoto T, Ishida K, Makhijani NS, Gruber HE & Yamaguchi DT (2011) CXC receptor knockout\\n\\nmice: characterization of skeletal features and membranous bone healing in the adult mouse. Bone 48: 267–\\n\\n274\\n\\nBrem H & Tomic-Canic M (2007) Cellular and molecular basis of wound healing in diabetes. J Clin Invest 117:\\n\\n1219–1222\\n\\nCarelli FN, Cerrato C, Dong Y, Appert A, Dernburg A & Ahringer J (2022) Widespread transposon co-option in the\\n\\nCaenorhabditis germline regulatory network. Sci Adv 8\\n\\nDe Cecco M, Ito T, Petrashen AP, Elias AE, Skvir NJ, Criscione SW, Caligiana A, Brocculi G, Adney EM, Boeke\\n\\nJD, et al (2019a) L1 drives IFN in senescent cells and promotes age-associated inflammation. Nature 566: 73–\\n\\n78\\n\\nDe Cecco M, Ito T, Petrashen AP, Elias AE, Skvir NJ, Criscione SW, Caligiana A, Brocculi G, Adney EM, Boeke\\n\\nJD, et al (2019b) L1 drives IFN in senescent cells and promotes age-associated inflammation. Nature 566: 73–\\n\\n78\\n\\nChan JK, Roth J, Oppenheim JJ, Tracey KJ, Vogl T, Feldmann M, Horwood N & Nanchahal J (2012) Alarmins:\\n\\nawaiting a clinical response. J Clin Invest 122: 2711\\n\\nChaparro O & Linero I (2016) Regenerative Medicine: A New Paradigm in Bone Regeneration. In Advanced\\n\\nTechniques in Bone Regeneration InTech\\n\\nChen YG & Hur S (2022) Cellular origins of dsRNA, their recognition and consequences. Nat Rev Mol Cell Biol 23:\\n\\n286–301\\n\\nChoi J, Choi W, Joo Y, Chung H, Kim D, Oh SJ & Kim S-H (2021) FGF2-primed 3D spheroids producing IL-8\\n\\npromote therapeutic angiogenesis in murine hindlimb ischemia. NPJ Regen Med 6: 48\\n\\nClark D, Nakamura M, Miclau T & Marcucio R (2017) Effects of Aging on Fracture Healing. Curr Osteoporos Rep\\n\\n15: 601\\n\\n\\n-----\\n\\nCoates BA, McKenzie JA, Buettmann EG, Liu X, Gontarz PM, Zhang B & Silva MJ (2019) Transcriptional\\n\\nProfiling of Intramembranous and Endochondral Ossification after Fracture in Mice. Bone 127: 577\\n\\nCoppé J-P, Desprez P-Y, Krtolica A & Campisi J (2010) The Senescence-Associated Secretory Phenotype: The Dark\\n\\nSide of Tumor Suppression. Annu Rev Pathol Mech Dis 5: 99–118\\n\\nCosby RL, Chang NC & Feschotte C (2019) Host-transposon interactions: conflict, cooperation, and cooption.\\n\\nGenes Dev 33: 1098–1116\\n\\nCroes M, Boot W, Kruyt MC, Weinans H, Pouran B, van der Helm YJM, Gawlitta D, Vogely HC, Alblas J, Dhert\\n\\nWJA, et al (2017) Inflammation-Induced Osteogenesis in a Rabbit Tibia Model. Tissue Eng Part C Methods\\n\\n23: 673–685\\n\\nCroes M, Oner FC, Kruyt MC, Blokhuis TJ, Bastian O, Dhert WJA & Alblas J (2015) Proinflammatory Mediators\\n\\nEnhance the Osteogenesis of Human Mesenchymal Stem Cells after Lineage Commitment. PLoS One 10:\\n\\ne0132781\\n\\nCroes M, Öner FC, van Neerven D, Sabir E, Kruyt MC, Blokhuis TJ, Dhert WJA & Alblas J (2016)\\n\\nProinflammatory T cells and IL-17 stimulate osteoblast differentiation. Bone 84: 262–270\\n\\nDobin A, Davis CA, Schlesinger F, Drenkow J, Zaleski C, Jha S, Batut P, Chaisson M & Gingeras TR (2013) STAR:\\n\\nultrafast universal RNA-seq aligner. Bioinformatics 29: 15–21\\n\\nDonnelly N, Gorman AM, Gupta S & Samali A (2013) The eIF2α kinases: their structures and functions. Cell Mol\\n\\nLife Sci 70: 3493–3511\\n\\nEash KJ, Greenbaum AM, Gopalan PK & Link DC (2010) CXCR2 and CXCR4 antagonistically regulate neutrophil\\n\\ntrafficking from murine bone marrow. J Clin Invest 120: 2423–2431\\n\\nEggerschwiler B, Canepa DD, Pape HC, Casanova EA & Cinelli P (2019) Automated digital image quantification of\\n\\nhistological staining for the analysis of the trilineage differentiation potential of mesenchymal stem cells. Stem\\n\\nCell Res Ther\\n\\nFLEISCH H, RUSSELL RGG & STRAUMANN F (1966) Effect of Pyrophosphate on Hydroxyapatite and Its\\n\\nImplications in Calcium Homeostasis. Nature 212: 901–903\\n\\nFukuda S, Varshney A, Fowler BJ, Wang S Bin, Narendran S, Ambati K, Yasuma T, Magagnoli J, Leung H, Hirahara\\n\\nS, et al (2021) Cytoplasmic synthesis of endogenous Alu complementary DNA via reverse transcription and\\n\\nimplications in age-related macular degeneration. Proc Natl Acad Sci U S A 118\\n\\n\\n-----\\n\\nGe C, Xiao G, Jiang D & Franceschi RT (2007) Critical role of the extracellular signal–regulated kinase–MAPK\\n\\npathway in osteoblast differentiation and skeletal development. J Cell Biol 176: 709\\n\\nGennero I, Laurencin-Dalicieux S, Conte-Auriol F, Briand-Mésange F, Laurencin D, Rue J, Beton N, Malet N, Mus\\n\\nM, Tokumura A, et al (2011) Absence of the lysophosphatidic acid receptor LPA1 results in abnormal bone\\n\\ndevelopment and decreased bone mass. Bone 49: 395–403\\n\\nGerstenfeld LC, Cho T-J, Kon T, Aizawa T, Cruceta J, Graves BD & Einhorn TA (2001) Impaired Intramembranous\\n\\nBone Formation during Bone Repair in the Absence of Tumor Necrosis Factor-Alpha Signaling. Cells Tissues\\n\\nOrgans 169: 285–294\\n\\nGerstenfeld LC, Cullinane DM, Barnes GL, Graves DT & Einhorn TA (2003) Fracture healing as a post-natal\\n\\ndevelopmental process: Molecular, spatial, and temporal aspects of its regulation. J Cell Biochem 88: 873–884\\n\\nHessle L, Johnson KA, Anderson HC, Narisawa S, Sali A, Goding JW, Terkeltaub R & Millan JL (2002) Tissue\\nnonspecific alkaline phosphatase and plasma cell membrane glycoprotein-1 are central antagonistic regulators\\n\\nof bone mineralization. Proc Natl Acad Sci U S A 99: 9445–9\\n\\nHou Y, Ryu CH, Jun JA, Kim SM, Jeong CH & Jeun S (2014) IL-8 enhances the angiogenic potential of human\\n\\nbone marrow mesenchymal stem cells by increasing vascular endothelial growth factor. Cell Biol Int 38:\\n\\n1050–9\\n\\nHur S (2019) Double-Stranded RNA Sensors and Modulators in Innate Immunity. Annu Rev Immunol 37: 349–375\\n\\nHurtgen BJ, Ward CL, Garg K, Pollot BE, Goldma SM, McKinley TO, Wenke JC & Corona BT (2016) Severe\\n\\nmuscle trauma triggers heightened and prolonged local musculoskeletal inflammation and impairs adjacent\\n\\ntibia fracture healing. J Musculoskelet Neuronal Interact 16: 122\\n\\nJachowicz JW, Bing X, Pontabry J, Bošković A, Rando OJ & Torres-Padilla ME (2017) LINE-1 activation after\\n\\nfertilization regulates global chromatin accessibility in the early mouse embryo. Nat Genet\\n\\nJIANG WG, SANDERS AJ, RUGE F & HARDING KG (2012) Influence of interleukin-8 (IL-8) and IL-8 receptors\\n\\non the migration of human keratinocytes, the role of PLC-γ and potential clinical implications. Exp Ther Med\\n\\n3: 231–236\\n\\nJiao H, Xiao E & Graves DT (2015) Diabetes and Its Effect on Bone and Fracture Healing. Curr Osteoporos Rep 13:\\n\\n327–335\\n\\nJohnson KA, Hessle L, Vaingankar S, Wennberg C, Mauro S, Narisawa S, Goding JW, Sano K, Millan JL &\\n\\n\\n-----\\n\\nTerkeltaub R (2000) Osteoblast tissue-nonspecific alkaline phosphatase antagonizes and regulates PC-1. Am J\\n\\nPhysiol Regul Integr Comp Physiol 279: R1365-77\\n\\nJones RB, Garrison KE, Wong JC, Duan EH, Nixon DF & Ostrowski MA (2008) Nucleoside analogue reverse\\n\\ntranscriptase inhibitors differentially inhibit human LINE-1 retrotransposition. PLoS One\\n\\nKarikó K, Buckstein M, Ni H & Weissman D (2005) Suppression of RNA recognition by Toll-like receptors: The\\n\\nimpact of nucleoside modification and the evolutionary origin of RNA. Immunity\\n\\nKarikó K, Muramatsu H, Welsh FA, Ludwig J, Kato H, Akira S & Weissman D (2008) Incorporation of\\n\\npseudouridine into mRNA yields superior nonimmunogenic vector with increased translational capacity and\\n\\nbiological stability. Mol Ther\\n\\nKim Y, Lee JH, Park J-E, Cho J, Yi H & Kim VN (2014) PKR is activated by cellular dsRNAs during mitosis and\\n\\nacts as a mitotic regulator. Genes Dev 28: 1310–1322\\n\\nKim Y, Park J, Kim S, Kim MA, Kang MG, Kwak C, Kang M, Kim B, Rhee HW & Kim VN (2018) PKR Senses\\n\\nNuclear and Mitochondrial Signals by Interacting with Endogenous Double-Stranded RNAs. Mol Cell 71:\\n\\n1051-1063.e6\\n\\nKon T, Cho TJ, Aizawa T, Yamazaki M, Nooh N, Graves D, Gerstenfeld LC & Einhorn TA (2001) Expression of\\n\\nosteoprotegerin, receptor activator of NF-kappaB ligand (osteoprotegerin ligand) and related proinflammatory\\n\\ncytokines during fracture healing. J Bone Miner Res 16: 1004–1014\\n\\nKormann MSD, Hasenpusch G, Aneja MK, Nica G, Flemmer AW, Herber-Jonat S, Huppmann M, Mays LE, Illenyi\\n\\nM, Schams A, et al (2011) Expression of therapeutic proteins after delivery of chemically modified mRNA in\\n\\nmice. Nat Biotechnol\\n\\nKoski GK, Karikó K, Xu S, Weissman D, Cohen PA & Czerniecki BJ (2004) Cutting Edge: Innate Immune System\\n\\nDiscriminates between RNA Containing Bacterial versus Eukaryotic Structural Features That Prime for High\\nLevel IL-12 Secretion by Dendritic Cells. J Immunol\\n\\nKovtun A, Bergdolt S, Wiegner R, Radermacher P, Huber-Lang M & Ignatius A (2016) The crucial role of\\n\\nneutrophil granulocytes in bone fracture healing. Eur Cells Mater 32: 152–162\\n\\nLama L, Adura C, Xie W, Tomita D, Kamei T, Kuryavyi V, Gogakos T, Steinberg JI, Miller M, Ramos-Espiritu L, et\\n\\nal (2019) Development of human cGAS-specific small-molecule inhibitors for repression of dsDNA-triggered\\n\\ninterferon expression. Nat Commun 10: 2261\\n\\n\\n-----\\n\\nLaroche M, Livideanu C, Paul C & Cantagrel A (2011) Interferon Alpha and Pamidronate in Osteoporosis with\\n\\nFracture Secondary to Mastocytosis. Am J Med 124: 776–778\\n\\nLee MJ, Kim DH, Ryu JS, Ko AY, Ko JH, Kim MK, Wee WR, Khwarg SI & Oh JY (2015) Topical TSG-6\\n\\nAdministration Protects the Ocular Surface in Two Mouse Models of Inflammation-Related Dry Eye. Invest\\n\\nOphthalmol Vis Sci 56: 5175–81\\n\\nLee RH, Pulin AA, Seo MJ, Kota DJ, Ylostalo J, Larson BL, Semprun-Prieto L, Delafontaine P & Prockop DJ\\n\\n(2009) Intravenous hMSCs improve myocardial infarction in mice because cells embolized in lung are\\n\\nactivated to secrete the anti-inflammatory protein TSG-6. Cell Stem Cell 5: 54–63\\n\\nLefkopoulos S, Polyzou A, Derecka M, Bergo V, Clapes T, Cauchy P, Jerez-Longres C, Onishi-Seebacher M, Yin N,\\n\\nMartagon-Calderón N-A, et al (2020) Repetitive Elements Trigger RIG-I-like Receptor Signaling that\\n\\nRegulates the Emergence of Hematopoietic Stem and Progenitor Cells. Immunity 53: 934-951.e9\\n\\nLi C, Li G, Liu M, Zhou T & Zhou H (2016) Paracrine effect of inflammatory cytokine-activated bone marrow\\n\\nmesenchymal stem cells and its role in osteoblast function. J Biosci Bioeng 121: 213–219\\n\\nLi H, Li D, Ma Z, Qian Z, Kang X, Jin X, Li F, Wang X, Chen Q, Sun H, et al (2018) Defective autophagy in\\n\\nosteoblasts induces endoplasmic reticulum stress and causes remarkable bone loss. Autophagy 14: 1726–1741\\n\\nLi J, Li X, Liu D, Hamamura K, Wan Q, Na S, Yokota H & Zhang P (2019) eIF2α signaling regulates autophagy of\\n\\nosteoblasts and the development of osteoclasts in OVX mice. Cell Death Dis 10: 921\\n\\nLiao Y, Smyth GK & Shi W (2014) featureCounts: an efficient general purpose program for assigning sequence\\n\\nreads to genomic features. Bioinformatics 30: 923–930\\n\\nLin D, Chai Y, Ma Y, Duan B, Yuan Y & Liu C (2019) Rapid initiation of guided bone regeneration driven by\\n\\nspatiotemporal delivery of IL-8 and BMP-2 from hierarchical MBG-based scaffold. Biomaterials 196: 122–\\n\\n137\\n\\nLiu W, Zhou L, Zhou C, Zhang S, Jing J, Xie L, Sun N, Duan X, Jing W, Liang X, et al (2016) GDF11 decreases\\n\\nbone mass by stimulating osteoclastogenesis and inhibiting osteoblast differentiation. Nat Commun 7: 12794\\n\\nLiu X, Liu Z, Sun L, Ren J, Wu Z, Jiang X, Ji Q, Wang Q, Fan Y, Cai Y, et al (2021) Resurrection of human\\n\\nendogenous retroviruses during aging reinforces senescence. bioRxiv\\n\\nLories RJU & Schett G (2012) Pathophysiology of New Bone Formation and Ankylosis in Spondyloarthritis. Rheum\\n\\nDis Clin North Am 38: 555–567\\n\\n\\n-----\\n\\nLu JY, Chang L, Li T, Wang T, Yin Y, Zhan G, Han X, Zhang K, Tao Y, Percharde M, et al (2021) Homotypic\\n\\nclustering of L1 and B1/Alu repeats compartmentalizes the 3D genome. Cell Res 31: 613–630\\n\\nLudwig J, Schuberth C, Goldeck M, Schlee M, Li H, Juranek S, Sheng G, Micura R, Tuschl T, Hartmann G, et al\\n\\n(2010) Structural and functional insights into 5′-ppp RNA pattern recognition by the innate immune receptor\\n\\nRIG-I. Nat Struct Mol Biol\\n\\nMangiavacchi A, Liu P, Della Valle F & Orlando V (2021) New insights into the functional role of retrotransposon\\n\\ndynamics in mammalian somatic cells. Cell Mol Life Sci 78: 5245–5256\\n\\nMarsell R & Einhorn TA (2011) The biology of fracture healing. Injury 42: 551–555\\n\\nMaruyama M, Rhee C, Utsunomiya T, Zhang N, Ueno M, Yao Z & Goodman SB (2020) Modulation of the\\n\\nInflammatory Response and Bone Healing. Front Endocrinol (Lausanne) 11: 386\\n\\nMashanov VS, Zueva OR & García-Arrarás JE (2012) Posttraumatic regeneration involves differential expression of\\n\\nlong terminal repeat (LTR) retrotransposons. Dev Dyn 241: 1625–1636\\n\\nMatsushita T, Chan YY, Kawanami A, Balmes G, Landreth GE & Murakami S (2009) Extracellular signal-regulated\\n\\nkinase 1 (ERK1) and ERK2 play essential roles in osteoblast differentiation and in supporting\\n\\nosteoclastogenesis. Mol Cell Biol 29: 5843–5857\\n\\nMeier F, Brunner AD, Frank M, Ha A, Bludau I, Voytik E, Kaspar-Schoenefeld S, Lubeck M, Raether O, Bache N,\\n\\net al (2020a) diaPASEF: parallel accumulation-serial fragmentation combined with data-independent\\n\\nacquisition. Nat Methods 17: 1229–1236\\n\\nMeier F, Brunner AD, Frank M, Ha A, Bludau I, Voytik E, Kaspar-Schoenefeld S, Lubeck M, Raether O, Bache N,\\n\\net al (2020b) diaPASEF: parallel accumulation-serial fragmentation combined with data-independent\\n\\nacquisition. Nat Methods 17: 1229–1236\\n\\nMountziaris PM, Spicer PP, Kasper FK & Mikos AG (2011) Harnessing and Modulating Inflammation in Strategies\\n\\nfor Bone Regeneration. Tissue Eng Part B Rev 17: 393\\n\\nNallagatla SR & Bevilacqua PC (2008) Nucleoside modifications modulate activation of the protein kinase PKR in\\n\\nan RNA structure-specific manner. RNA\\n\\nNelson AM, Reddy SK, Ratliff TS, Hossain MZ, Katseff AS, Zhu AS, Chang E, Resnik SR, Page C, Kim D, et al\\n\\n(2015) dsRNA Released by Tissue Damage Activates TLR3 to Drive Skin Regeneration. Cell Stem Cell 17:\\n\\n139–151\\n\\n\\n-----\\n\\nNimiritsky P, Eremichev R, Alexandrushkina N, Efimenko A, Tkachuk V & Makarevich P (2019) Unveiling\\n\\nMesenchymal Stromal Cells’ Organizing Function in Regeneration. Int J Mol Sci 20: 823\\n\\nNollet M, Santucci-Darmanin S, Breuil V, Al-Sahlanee R, Cros C, Topi M, Momier D, Samson M, Pagnotta S,\\n\\nCailleteau L, et al (2014) Autophagy in osteoblasts is involved in mineralization and bone homeostasis.\\n\\nAutophagy 10: 1965–1977\\n\\nOwlarn S, Klenner F, Schmidt D, Rabert F, Tomasso A, Reuter H, Mulaw MA, Moritz S, Gentile L, Weidinger G, et\\n\\nal (2017) Generic wound signals initiate regeneration in missing-tissue contexts. Nat Commun 2017 81 8: 1–\\n\\n13\\n\\nPardi N, Muramatsu H, Weissman D & Karikó K (2013) In vitro transcription of long RNA containing modified\\n\\nnucleosides. Methods Mol Biol\\n\\nPercharde M, Lin CJ, Yin Y, Guan J, Peixoto GA, Bulut-Karslioglu A, Biechele S, Huang B, Shen X & Ramalho\\nSantos M (2018) A LINE1-Nucleolin Partnership Regulates Early Development and ESC Identity. Cell\\n\\nRau A, Gallopin M, Celeux G & Jaffrézic F (2013) Data-based filtering for replicated high-throughput transcriptome\\n\\nsequencing experiments. Bioinformatics 29: 2146–2152\\n\\nRecknagel S, Bindl R, Brochhausen C, Göckelmann M, Wehner T, Schoengraf P, Huber-Lang M, Claes L & Ignatius\\n\\nA (2013) Systemic inflammation induced by a thoracic trauma alters the cellular composition of the early\\n\\nfracture callus. J Trauma Acute Care Surg 74: 531–537\\n\\nReppe S, Refvem H, Gautvik VT, Olstad OK, Høvring PI, Reinholt FP, Holden M, Frigessi A, Jemtland R &\\n\\nGautvik KM (2010) Eight genes are highly associated with BMD variation in postmenopausal Caucasian\\n\\nwomen. Bone\\n\\nRitschka B, Storer M, Mas A, Heinzmann F, Ortells MC, Morton JP, Sansom OJ, Zender L & Keyes WM (2017)\\n\\nThe senescence-associated secretory phenotype induces cellular plasticity and tissue regeneration. Genes Dev\\n\\n31: 172–183\\n\\nRoberts SJ & Ke HZ (2018) Anabolic Strategies to Augment Bone Fracture Healing. Curr Osteoporos Rep 16: 289–\\n\\n298\\n\\nRobinson MD, McCarthy DJ & Smyth GK (2010) edgeR: a Bioconductor package for differential expression\\n\\nanalysis of digital gene expression data. Bioinformatics 26: 139–140\\n\\nSadeq S, Al-Hashimi S, Cusack CM & Werner A (2021) Endogenous Double-Stranded RNA. Non-Coding RNA 7:\\n\\n\\n-----\\n\\n15\\n\\nSchmidt-Bleek K, Schell H, Schulz N, Hoff P, Perka C, Buttgereit F, Volk HD, Lienau J & Duda GN (2012)\\n\\nInflammatory phase of bone healing initiates the regenerative healing cascade. Cell Tissue Res 347: 567–573\\n\\nSheng W, LaFleur MW, Nguyen TH, Chen S, Chakravarthy A, Conway JR, Li Y, Chen H, Yang H, Hsu P-H, et al\\n\\n(2018) LSD1 Ablation Stimulates Anti-tumor Immunity and Enables Checkpoint Blockade. Cell 174: 549\\n563.e19\\n\\nSimon M, Van Meter M, Ablaeva J, Ke Z, Gonzalez RS, Taguchi T, De Cecco M, Leonova KI, Kogan V, Helfand\\n\\nSL, et al (2019) LINE1 Derepression in Aged Wild-Type and SIRT6-Deficient Mice Drives Inflammation.\\n\\nCell Metab 29: 871-885.e5\\n\\nSlotkin RK & Martienssen R (2007) Transposable elements and the epigenetic regulation of the genome. Nat Rev\\n\\nGenet 8: 272–285\\n\\nStrieter RM, Burdick MD, Gomperts BN, Belperio JA & Keane MP (2005) CXC chemokines in angiogenesis.\\n\\nCytokine Growth Factor Rev 16: 593–609\\n\\nTimmen M, Hidding H, Wieskötter B, Baum W, Pap T, Raschke MJ, Schett G, Zwerina J & Stange R (2014)\\n\\nInfluence of antiTNF-alpha antibody treatment on fracture healing under chronic inflammation. BMC\\n\\nMusculoskelet Disord 15: 184\\n\\nDella Valle F, Reddy P, Yamamoto M, Liu P, Saera-Vila A, Bensaddek D, Zhang H, Prieto Martinez J, Abassi L,\\n\\nCelii M, et al (2022) LINE-1 RNA causes heterochromatin erosion and is a target for amelioration of\\n\\nsenescent phenotypes in progeroid syndromes. Sci Transl Med 14: eabl6057\\n\\nVaranasi SS, Olstad OK, Swan DC, Sanderson P, Gautvik VT, Reppe S, Francis RM, Gautvik KM & Datta HK\\n\\n(2010) Skeletal Site-Related Variation in Human Trabecular Bone Transcriptome and Signaling. PLoS One 5:\\n\\ne10692\\n\\nWang S, Lee C, Kim J, Hyun J, Lim M, Cha H-J, Oh S-H, Choi YH & Jung Y (2017) Tumor necrosis factor\\ninducible gene 6 protein ameliorates chronic liver damage by promoting autophagy formation in mice. Exp\\n\\nMol Med 49: e380\\n\\nWeckbach S, Hohmann C, Braumueller S, Denk S, Klohs B, Stahel PF, Gebhard F, Huber-Lang MS & Perl M\\n\\n(2013) Inflammatory and apoptotic alterations in serum and injured tissue after experimental polytrauma in\\n\\nmice: distinct early response compared with single trauma or ‘double-hit’ injury. J Trauma Acute Care Surg\\n\\n\\n-----\\n\\n74: 489–498\\n\\nWheatley BM, Nappo KE, Christensen DL, Holman AM, Brooks DI & Potter BK (2019) Effect of NSAIDs on Bone\\n\\nHealing Rates. J Am Acad Orthop Surg 27: e330–e336\\n\\nWiśniewski JR, Zougman A, Nagaraj N & Mann M (2009) Universal sample preparation method for proteome\\n\\nanalysis. Nat Methods 6: 359–362\\n\\nXiao G, Gopalakrishnan R, Jiang D, Reith E, Benson MD & Franceschi RT (2002a) Bone morphogenetic proteins,\\n\\nextracellular matrix, and mitogen-activated protein kinase signaling pathways are required for osteoblast\\nspecific gene expression and differentiation in MC3T3-E1 cells. J Bone Miner Res 17: 101–110\\n\\nXiao G, Jiang D, Gopalakrishnan R & Franceschi RT (2002b) Fibroblast growth factor 2 induction of the osteocalcin\\n\\ngene requires MAPK activity and phosphorylation of the osteoblast transcription factor, Cbfa1/Runx2. J Biol\\n\\nChem 277: 36181–36187\\n\\nXiao G, Jiang D, Thomas P, Benson MD, Guan K, Karsenty G & Franceschi RT (2000) MAPK Pathways Activate\\n\\nand Phosphorylate the Osteoblast-specific Transcription Factor, Cbfa1 *. J Biol Chem 275: 4453–4459\\n\\nYang A, Lu Y, Xing J, Li Z, Yin X, Dou C, Dong S, Luo F, Xie Z, Hou T, et al (2018a) IL-8 Enhances Therapeutic\\n\\nEffects of BMSCs on Bone Regeneration via CXCR2-Mediated PI3k/Akt Signaling Pathway. Cell Physiol\\n\\nBiochem 48: 361–370\\n\\nYang H, Tian W, Wang S, Liu X, Wang Z, Hou L, Ge J, Zhang X, He Z & Wang X (2018b) TSG-6 secreted by bone\\n\\nmarrow mesenchymal stem cells attenuates intervertebral disc degeneration by inhibiting the TLR2/NF-κB\\n\\nsignaling pathway. Lab Investig 98: 755–772\\n\\nYang WR, Ardeljan D, Pacyna CN, Payer LM & Burns KH (2019) SQuIRE reveals locus-specific regulation of\\n\\ninterspersed repeat expression. Nucleic Acids Res 47: e27–e27\\n\\nZhang R, Liu Y, Yan K, Chen L, Chen X-R, Li P, Chen F-F & Jiang X-D (2013) Anti-inflammatory and\\n\\nimmunomodulatory mechanisms of mesenchymal stem cell transplantation in experimental traumatic brain\\n\\ninjury. J Neuroinflammation 10: 106\\n\\nZhang X, Zhang R & Yu J (2020) New Understanding of the Relevant Role of LINE-1 Retrotransposition in Human\\n\\nDisease and Immune Modulation. Front cell Dev Biol 8: 657\\n\\nZhu W, Kuo D, Nathanson J, Satoh A, Pao GM, Yeo GW, Bryant S V., Voss SR, Gardiner DM & Hunter T (2012)\\n\\nRetrotransposon long interspersed nucleotide element-1 (LINE-1) is activated during salamander limb\\n\\n\\n-----\\n\\nregeneration. Dev Growth Differ 54: 673–685\\n\\n# Figure Legends\\n\\nFigure 1 - TEs expression is induced after fracture and in mechanically loaded bone.\\n\\nA) Heatmap representation of RNAseq differentially expressed TEs (LogFC>0.5) between 4h post-injury and intact\\n\\nfemurs at different time points of the bone healing process. N=5 biological replicates for each time point. N=10\\n\\nbiological replicates for intact femur. B) Heatmap representation of RNAseq upregulated TEs (LogFC>0.5) between\\n\\n4h and intact at different time points of the bone healing process. N=5 biological replicates for each time point.\\n\\nN=10 biological replicates for intact femur. C) Heatmap representation of RNAseq differentially expressed TEs\\n\\nanalysis [Fragments Per Kilobase of transcript per Million (FPKM) fold change] in femoral (n=27) and iliac (n=34)\\n\\nbone biopsies from healthy donors. D) Pie charts showing the number and percentage of differentially expressed\\n\\nTEs subfamilies between femoral and iliac healthy bone. E) Heatmap representation of RNAseq differentially\\n\\nexpressed TEs analysis [Fragments Per Kilobase of transcript per Million (FPKM) fold change] in femoral (n=27)\\n\\nand iliac (n=34) bone biopsies from healthy donors. One heatmap for each TEs order is shown.\\n\\nFigure 2 - TEs expression correlates with bone mineral density in human weight-bearing bone.\\n\\nA) Heatmap representation of RNAseq differentially expressed TEs analysis [Fragments Per Kilobase of transcript\\n\\nper Million (FPKM) fold change] in femoral bone from healthy (n=27), osteopenic (n=12), and osteoporotic (n=9)\\n\\ndonors. B) Heatmap representation of RNAseq differentially expressed TEs analysis [Fragments Per Kilobase of\\n\\ntranscript per Million (FPKM) fold change] in femoral bone with high BMD (FN T-score >-1) (n=27) and low BMD\\n\\n(FN T-score <-1) (n=21). C) Upper panels: pie charts showing the number and percentage of differentially expressed\\n\\nTEs subfamilies between high BMD and low BMD femoral bone. Lower panels: heatmap representation of RNAseq\\n\\ndifferentially expressed TEs analysis [Fragments Per Kilobase of transcript per Million (FPKM) fold change] in\\n\\nfemoral bone with high BMD (n=27) and low BMD (n=21). One heatmap for each TEs order is shown. D) Upper\\n\\npanels: Heatmap representation of correlation analysis (P value<0.05) between TEs expression (FPKM) and local\\n\\nBMD (FN T-score) in femoral bone biopsies (n=48). Correlation between TE expression (FPKM values) and FN t\\nscore was evaluated using Pearson method. Significant correlations (p-value < 0.05) were selected for the heatmap\\n\\nrepresentation and arranged according to the Pearson correlation coefficient (r). r > 0, positive correlations; r < 0,\\n\\nnegative correlations. Lower panels: Pie charts showing the percentage of TEs subfamilies positively correlated to\\n\\n\\n-----\\n\\nlocal BMD.\\n\\nFigure 3 - L1 RNA delivery stimulates the mineralization of differentiating osteoblasts\\n\\nA) Experimental workflow and flow cytometer analysis showing the percentage of positive cells 6 hours after L1\\n\\nRNA delivery at day 5 of ex vivo osteogenesis. Intracellular localization of cy5 conjugated synthetic L1 RNA (red\\n\\nspots) three days after transfection is also shown from a typical experiment (right). B) qPCR analysis of intracellular\\n\\nL1 RNA level 24 h (d6), 5 days (d10), and 9 days (d14) post-transfection. The graph is shown as mean ± s.d. of n =3\\n\\nindependent experiments. C) cy5 conjugated synthetic L1 RNA (red) and osteoimage stained mineral matrix (green)\\n\\ndetection 9 days post L1 RNA transfection. The white arrowheads indicate the colocalization between L1 RNA and\\n\\nhydroxyapatite. D) Osteoimage stained mineral matrix quantification (upper panels), images (central panels), and\\n\\nAlizarin Red (lower panels) 9 days after L1 RNA or RFP RNA delivery in 2 healthy donors-derived MSCs (D188\\n\\nand D170). RFU= Relative Fluorescence Units. The graph is shown as mean ± s.d. of n = 11 technical replicates.\\n\\n****P < 0.00005 in Wilcoxon test. E) Alizarin Red staining of osteoblasts transfected with increasing doses of RFP\\n\\nRNA (upper panels) and L1 RNA (lower panels). F) Alizarin Red images (upper panel) and quantification (lower\\n\\npanel) of MSCs mineralization after 14, 17 and 21 days of ex vivo differentiation. MSCs were obtained from the\\n\\nfemur of 4 healthy (D188, D239, D247, D170) and 4 OP patients (HUK7, HUK9, HUK12, HUK16). N=9 technical\\n\\nreplicates for each donor and time point. Boxes in the boxplot indicate the interquartile range (50% of data), while\\n\\nthe lower end and the upper end represent the 1[st] and the 3[rd] quartile, respectively. The solid line inside the box\\n\\nrepresents the median. Whiskers represent the max and min values. Values that are not within 1.5 times the\\n\\ninterquartile range are considered as outliers and lie outside the whiskers. G) Osteoimage stained mineral matrix\\n\\nquantification (upper panels), images (central panels), and Alizarin Red (lower panels) 9 days after L1 RNA or RFP\\n\\nRNA delivery in 3 OP patients derived MSCs (HUK9, HUK12, HUK16). RFU= Relative Fluorescence Units. The\\n\\ngraph is shown as mean ± s.d. of n = 10-12 technical replicates. ****P < 0.00005 in Wilcoxon test. H) qRT-PCR of\\n\\nearly osteogenic genes in RFP and L1 transfected osteoblasts at different time points of osteogenic differentiation.\\n\\nExpression level is normalized on day 5 (not transfected osteoblasts). N=3 biological replicates. The graph is shown\\n\\nas mean ± s.d. of n = 3 independent experiments. *P < 0.05; **P < 0.005, ***P < 0.0005, ****P < 0.00005 in\\n\\nStudent’s t test.\\n\\nFigure 4 - L1 RNA delivery induces an inflammatory response characteristic of bone repair process\\n\\nA) Left panel: heatmap representation of RNAseq differentially expressed gene analysis [Fragments Per Kilobase of\\n\\n\\n-----\\n\\ntranscript per Million (FPKM) fold change] in osteoblasts transfected with negative control RNA (RFP) or L1 RNA.\\n\\nRight panel: bubble plot showing gene ontology (GO) enrichment analysis of L1 upregulated biological processes.\\n\\nB) Tree plot showing gene ontology (GO) enrichment analysis of L1 upregulated biological processes (left) and\\n\\ncellular components (right). C) Top 25 upregulated biological processes 24h post-L1 RNA delivery in vitro (yellow)\\n\\nand 4h post fracture in vivo (blue). Shared GO terms are shown in green. D) qPCR analysis of “inflammatory\\n\\nresponse” (GO:0006954) and “immune response” (GO:0006955) genes at different time points post L1 RNA\\n\\ntransfection at day 5. * **P < 0.005, ****P < 0.00005 in two way ANOVA.\\n\\nFigure 5 - PKR mediates L1 RNA-induced stress response and mineralization\\n\\nA) Upper panel: schematic representation of the pathway inhibited by Lamivudine 3TC and G140. Lower panel:\\n\\nAlizarin Red staining of differentiating osteoblasts transfected with L1 RNA and treated with Lamivudine 3TC and\\n\\nG140. B) IF on RFP and L1 transfected osteoblasts, 6h post-transfection, shows the colocalization between cy5\\n\\nsignal (L1 RNA, red) and 488-Anti-dsRNA signal (dsRNA, green). Nuclei are stained with Hoechst (blue).\\n\\nC)Western blot analysis showing the ratio between total eIF2\\uf061 and phosphorylated eIF2\\uf061 (P- eIF2\\uf061\\uf029. H3:\\n\\nendogenous standard Histone 3. D) Volcano plot (left) showing the number of significantly upregulated and\\n\\ndownregulated protein 24h post-L1 RNA transfection. Right: tree plots showing gene ontology (GO) enrichment\\n\\nanalysis of biological processes (upper panel) and cellular components (lower panel) downregulated by L1 RNA\\n\\ndelivery (MS data). E) qPCR analysis of “inflammatory response” (GO:0006954) and “immune response”\\n\\n(GO:0006955) genes 24h post L1 RNA transfection with and without PKR inhibitor C16. F) Alizarin Red staining\\n\\nof differentiating osteoblasts transfected with L1 RNA with and without PKR inhibitor C16. G) Tree plots showing\\n\\ngene ontology (GO) enrichment analysis of biological processes upregulated by L1 RNA 24h after transfection (MS\\n\\ndata).\\n\\nFigure 6 - L1-RNA-induced changes in osteoblast secretome\\n\\nA) Left: Alizarin red staining on recipient OB 9 days after the delivery of conditioned media. Right: Microscope\\n\\nimages of recipient OB 24h from the delivery of conditioned media. Mineralized nodules in OB receiving\\n\\nconditioned media from L1 treated OB are shown (orange arrows). B) Heat map of differentially expressed proteins\\n\\nin bulk secretome and exosomes derived from untransfected (NT), RFP- and L1-transfected osteoblasts. N=3\\n\\nbiological replicates. C) GO enrichment analysis of differentially expressed protein (adjusted p-value <0.05) in the\\n\\nbulk secretome of L1 compared to RFP transfected osteoblasts. D) GO enrichment analysis of differentially\\n\\n\\n-----\\n\\nexpressed protein (adjusted p-value <0.05) in the exosomes of L1 compared to RFP transfected osteoblasts.\\n\\n\\n-----\\n\\n\""
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pymupdf4llm.to_markdown(DOCX_FILES[0])\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing ./data/pdf/EMM-2023-18636.pdf...\n",
      "[                                        ] (0/82[                                        ] ( 1/8[                                        ] ( 2/82[=                                       ] ( 3/82[=                                       ] ( 4/8[==                                      ] ( 5/8[==                                      ] ( 6/82[===                                     ] ( 7/82[===                                     ] ( 8/8[====                                    ] ( 9/8[====                                    ] (10/82[=====                                   ] (11/82[=====                                   ] (12/8[======                                  ] (13/8[======                                  ] (14/82[=======                                 ] (15/82[=======                                 ] (16/8[========                                ] (17/8[========                                ] (18/82[=========                               ] (19/82[=========                               ] (20/8[==========                              ] (21/8[==========                              ] (22/82[===========                             ] (23/82[===========                             ] (24/8[============                            ] (25/8[============                            ] (26/82[=============                           ] (27/[=============                           ] (28/8[==============                          ] (29/8[==============                          ] (30/82[===============                         ] (31/82[===============                         ] (32/8[================                        ] (33/8[================                        ] (34/82[=================                       ] (35/82[=================                       ] (36/8[==================                      ] (37/8[==================                      ] (38/82[===================                     ] (39/82[===================                     ] (40/[====================                    ] (41/82[====================                    ] (42/8[====================                    ] (43/82[=====================                   ] (44/82[=====================                   ] (45/8[======================                  ] (46/8[======================                  ] (47/82[=======================                 ] (48/82[=======================                 ] (49/8[========================                ] (50/8[========================                ] (51/82[=========================               ] (52/82[=========================               ] (53/8[==========================              ] (54/8[==========================              ] (55/82[===========================             ] (56/82[===========================             ] (57/8[============================            ] (58/8[============================            ] (59/82[=============================           ] (60/82[=============================           ] (61/8[==============================          ] (62/8[==============================          ] (63/82[===============================         ] (64/82[===============================         ] (65/8[================================        ] (66/8[================================        ] (67/82[=================================       ] (68/[=================================       ] (69/8[==================================      ] (70/8[==================================      ] (71/82[===================================     ] (72/[===================================     ] (73/8[====================================    ] (74/8[====================================    ] (75/82[=====================================   ] (76/82[=====================================   ] (77/8[======================================  ] (78/8[======================================  ] (79/82[======================================= ] (80/[======================================= ] (81/82[========================================] (82/82]\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "\"1\\n\\n2\\n\\n3 **DUSP6 inhibition overcomes Neuregulin/HER3-driven therapy**\\n\\n4 **tolerance in HER2+ breast cancer**\\n\\n5\\n\\n6 Majid Momeny[1,2§], Mari Tienhaara[3,4], Mukund Sharma[1,4], Deepankar Chakroborty[3,4],\\n\\n7 Roosa Varjus[1], Iina Takala[3,4], Joni Merisaari[1], Artur Padzik[1], Andreas Vogt[5], Ilkka\\n\\n8 Paatero[1], Klaus Elenius[3,4], Teemu D. Laajala[6], Kari J. Kurppa[3,4], Jukka\\n\\n9 Westermarck[1,4,§]\\n\\n10\\n\\n11 1Turku Bioscience Centre, University of Turku and Åbo Akademi University, Turku,\\n\\n12 Finland\\n\\n13 2The Brown Foundation Institute of Molecular Medicine, McGovern Medical School,\\n\\n14 The University of Texas Health Science Center at Houston, TX, USA.\\n\\n15 3Medicity Research Laboratories, Faculty of Medicine, University of Turku, Turku,\\n\\n16 Finland\\n\\n17 4Institute of Biomedicine, University of Turku, Turku, Finland\\n\\n18 5University of Pittsburgh Drug Discovery Institute, Department of Computational and\\n\\n19 Systems Biology, Pittsburgh Technology Center, Pittsburgh, PA, USA\\n\\n20 6Department of Mathematics and Statistics, University of Turku, Turku, Finland\\n\\n21\\n\\n22\\n\\n23 § Co-corresponding authors: jukwes@utu.fi, majid.momeny@uth.tmc.edu\\n\\n24\\n\\n25\\n\\n26\\n\\n\\n-----\\n\\n27 **Abstract**\\n\\n28\\n\\n29 Despite clinical benefits of tyrosine kinase inhibitors (TKIs) in cancer, most tumors\\n\\n30 can reactivate proliferation under TKI therapy. Here we present transcriptional\\n\\n31 profiling of HER2+ breast cancer cells transitioning from dormant drug tolerant cells\\n\\n32 to re-proliferating cells under continuous HER2 inhibitor (HER2i) therapy. Focusing\\n\\n33 on phosphatases, expression of dual-specificity phosphatase DUSP6 was found\\n\\n34 inhibited in dormant cells, but strongly induced upon re-growth. DUSP6 expression\\n\\n35 also selectively associated with poor patient survival in HER2+ breast cancers.\\n\\n36 DUSP6 overexpression conferred apoptosis resistance, whereas its pharmacological\\n\\n37 blockade prevented therapy tolerance development under HER2i therapy. DUSP6\\n\\n38 targeting also synergized with clinically used HER2i combination therapies.\\n\\n39 Mechanistically DUSP6 is a positive regulator of HER3 expression, and its impact\\n\\n40 on HER2i tolerance was mediated by neuregulin-HER3 axis. _In vivo,_ genetic\\n\\n41 targeting of _DUSP6_ reduced tumor growth in brain metastasis model, whereas its\\n\\n42 pharmacological targeting induced synthetic lethal therapeutic effect in combination\\n\\n43 with HER2i. Collectively this work demonstrates that _DUSP6 drives escape from_\\n\\n44 HER2i-induced dormancy, and that DUSP6 is a druggable target to overcome\\n\\n45 HER3-driven TKI resistance.\\n\\n46 Keywords: Non-genetic drug tolerance, Lapatinib, Neratinib, BCI, FOXM1\\n\\n47\\n\\n\\n-----\\n\\n48\\n\\n49 **The paper explained:**\\n\\n50\\n\\n51 **Problem**\\n\\n52 The molecular mechanisms for the acquisition of resistance to HER2-targeted\\n\\n53 therapies in breast cancer are still elusive, especially when the drug-tolerant\\n\\n54 persister cells start to regrow under the treatment.\\n\\n55 **Results**\\n\\n56 The oncogenic phosphatase DUSP6 is increased in the drug-tolerant emerging\\n\\n57 persister cells and plays central roles in the acquisition of HER2i resistance.\\n\\n58 Moreover, DUSP6 blockade potentiates therapeutic sensitivity in the primary\\n\\n59 resistant models via inhibition of the HER3 signaling pathway. Inhibition of DUSP6\\n\\n60 offers advantages over AKT blockade, including induction of apoptotic cell death,\\n\\n61 preempting ligand-induced rescue, and inhibition of HER3. Ultimately, DUSP6\\n\\n62 inhibition reverses HER2i resistance _in vivo_ and reduces the outgrowth of HER2+\\n\\n63 cell in the brain metastasis models.\\n\\n64 **Impact**\\n\\n65 These findings have implications in the clinical management of HER2+ breast cancer\\n\\n66 and warrant development and clinical investigation of DUSP6 inhibitors in\\n\\n67 combination with HER2-directed therapies in the patients.\\n\\n68\\n\\n69\\n\\n70\\n\\n\\n-----\\n\\n71 **Introduction**\\n\\n72\\n\\n73 To develop therapeutic resistance, tumor cells undergo distinct evolutionary stages,\\n\\n74 starting with induction of dormancy and non-genetic drug tolerance, followed by\\n\\n75 epigenetic changes and finally resistance-conferring genetic mutations (De Conti et\\n\\n76 _al, 2021; Hata et al, 2016; Marine et al, 2020). These different phases were originally_\\n\\n77 demonstrated for EGFR-targeted therapies in non-small cell lung cancer (NSCLC)\\n\\n78 cells (Sharma et al, 2010), but the concept has been expanded more recently to\\n\\n79 other malignancies including HER2+ breast cancer (Chang et al, 2022; De Conti et\\n\\n80 _al., 2021; Dhimolea et al, 2021; Hata et al., 2016; Kurppa et al, 2020; Sharma et al.,_\\n\\n81 2010). Based on these studies, there is ample of omics data from dormant cells (also\\n\\n82 called as drug tolerant persisters; DTPs) from different cancer types treated with\\n\\n83 variety of therapies. However, our understanding of the molecular mechanisms\\n\\n84 behind the regrowth of DTPs under continuous therapy is still rudimentary.\\n\\n85 Especially, to our knowledge there are no published studies describing\\n\\n86 transcriptional landscapes of transition from DTP to drug tolerant expanding cells\\n\\n87 (DTEP), or transition of DTEP cells to long term resistant (LR) cells upon TKI\\n\\n88 therapies.\\n\\n89\\n\\n90 The human epidermal growth factor receptor 2 (HER2; encoded by _ERBB2) is_\\n\\n91 overexpressed in ∼15-25% of human breast cancers and associates with a poor\\n\\n92 patient survival (Arteaga & Engelman, 2014; Haikala & Janne, 2021). HER2 belongs\\n\\n93 to the ERBB family of receptor tyrosine kinase (RTK) with four members: HER1\\n\\n94 (EGFR), HER2, HER3 and HER4. Upon ligand binding, the ERBB receptors homo\\n95 and heterodimerize and activate downstream signaling pathways including PI3K/AKT\\n\\n\\n-----\\n\\n96 and RAS/MAPK/ERK, which regulate cell proliferation, survival and the metastatic\\n\\n97 dissemination (Arteaga & Engelman, 2014; Haikala & Janne, 2021). Multiple HER2\\n98 targeted therapies, including the monoclonal antibody trastuzumab and small\\n\\n99 molecule tyrosine kinase inhibitors (TKIs) have been approved for the treatment of\\n\\n100 HER2-overexpressing (HER2+) breast cancer (Goutsouliak et al, 2020). Application\\n\\n101 of anti-HER2 agents in combination with chemotherapy has significantly improved\\n\\n102 the patients’ outcome. However, patients initially responsive to the HER2 inhibitors\\n\\n103 (HER2is) almost inevitably succumb to disease relapse (Goutsouliak et al., 2020).\\n\\n104 Moreover, HER2+ breast tumors have an inherent tendency to develop brain\\n\\n105 metastasis, a significant clinical challenge for the treatment of these patients (Fecci\\n\\n106 _et al, 2019). Therefore, there is a pressing need for novel and more efficacious_\\n\\n107 therapeutic strategies to overcome resistance to HER2is.\\n\\n108\\n\\n109 HER3 is an obligate heterodimerization partner for HER2 and plays essential roles in\\n\\n110 HER2-driven tumorigenesis, and resistance to HER2is (Haikala & Janne, 2021;\\n\\n111 Wilson et al, 2012). Consistent with the effects of HER3 overexpression, the HER3\\n\\n112 ligand neuregulin (NRG, a.k.a Heregulin; HRG) promotes trastuzumab resistance in\\n\\n113 HER2+ breast cancer cells (Haikala & Janne, 2021). Importantly, the NRG-HER3\\n\\n114 axis also promotes resistance to a wide range of TKIs and chemotherapies (Erjala et\\n\\n115 _al, 2006; Haikala & Janne, 2021; Knuefermann et al, 2003; Recondo et al, 2020;_\\n\\n116 Wilson et al., 2012; Yonesaka et al, 2011). Despite the importance of HER3 in\\n\\n117 cancer progression and therapy resistance, development of HER3 small molecule\\n\\n118 inhibitors has been challenging due to its impaired kinase activity (Haikala & Janne,\\n\\n119 2021; Xie et al, 2014). Moreover, the clinical activity of HER3 monoclonal antibodies\\n\\n120 either as monotherapies or in combination with chemo- and targeted therapies have\\n\\n\\n-----\\n\\n121 been marginal (Cleary et al, 2017; Haikala & Janne, 2021; Schneeweiss et al, 2018).\\n\\n122 To this end, there is a pressing need to identify novel strategies to inhibit HER3\\n\\n123 activity and/or expression for the treatment of HER3-dependent human malignancies\\n\\n124 (Gaborit et al, 2015; Haikala & Janne, 2021; Xie et al., 2014).\\n\\n125\\n\\n126 There is emerging evidence that phosphatases are novel and “druggable” targets in\\n\\n127 oncology (Lazo et al, 2018; Vainonen et al, 2021). Inhibition of oncogenic\\n\\n128 phosphatases, or re-activation of tumor suppressor phosphatases, by small molecule\\n\\n129 therapies halt tumor growth, retard malignant progression, and enhance therapeutic\\n\\n130 sensitivity in various neoplasms (Lazo et al., 2018; Vainonen et al., 2021). Despite\\n\\n131 this, the contribution of phosphatases to resistance to HER2is is still poorly\\n\\n132 understood. Dual-specificity phosphatases (DUSPs) belong to the superfamily of\\n\\n133 protein tyrosine phosphatases and dephoshorylate both tyrosines and serines or\\n\\n134 threonines. A subgroup of the DUSPs are mitogen-activated protein kinase (MAPK)\\n\\n135 phosphatases that selectively interact with and dephosphorylate the MAPKs\\n\\n136 (Patterson et al, 2009; Zandi et al, 2022). For instance, DUSP6 displays a high\\n\\n137 degree of substrate selectivity for the extracellular signal-regulated kinase (ERK), but\\n\\n138 not P38 or c-Jun N-terminal kinase (JNK) (Zandi et al., 2022). However, recent\\n\\n139 studies indicate that at least some cancer relevant DUSP6 functions may by ERK\\n140 independent (Kong et al, 2023). DUSP6 is indicated in clinical cancer progression,\\n\\n141 and its genetic inhibition prevents tumor cell growth (Shojaee et al, 2015; Wu et al,\\n\\n142 2018). Notably, DUSP6 is a druggable phosphatase (Korotchenko et al, 2014;\\n\\n143 Molina et al, 2009; Vainonen et al., 2021; Zandi et al., 2022). The best characterized\\n\\n144 DUSP6 inhibitor molecule BCI ((E)-2-Benzylidene-3-(cyclohexylamino)-2,3-dihydro\\n145 1H-inden-1-one), is a semi-allosteric inhibitor of both DUSP1 and 6 that phenocopies\\n\\n\\n-----\\n\\n146 genetic DUSP6 inhibition in several cancer models (Kong et al., 2023; Korotchenko\\n\\n147 _et al., 2014; Molina et al., 2009; Vainonen et al., 2021). However, the role and_\\n\\n148 potential of therapeutic targeting of DUSP6 in overcoming resistance to HER2is is\\n\\n149 currently unknown.\\n\\n150\\n\\n151 Here we present transcriptional analysis of HER2i treated HER2+ cancer cells upon\\n\\n152 9 months of continuous HER2i treatment. In addition to revealing first global gene\\n\\n153 expression programs associated with the DTP-DTEP and DTEP-LR therapy\\n\\n154 tolerance transitions, we use complementary genetic and pharmacological\\n\\n155 approaches to demonstrate that _DUSP6 has critical role in regrowth of DTEP cells_\\n\\n156 under HER2i therapies. Mechanistically, DUSP6 drives non-genetic HER2i tolerance\\n\\n157 via regulation of HER3 expression, and by abrogating neuregulin-elicited apoptosis\\n\\n158 resistance. Collectively our findings provide a strong pre-clinical rationale to further\\n\\n159 advance in DUSP6 blockade for HER3 targeting in general, and especially for the\\n\\n160 clinical management of HER2+ breast cancer patients with resistance to HER2i.\\n\\n161\\n\\n162\\n\\n\\n-----\\n\\n163 **Results**\\n\\n164\\n\\n165 **Development of acquired HER2i resistance by long-term treatment of drug**\\n\\n166 **sensitive HER2+ breast cancer cells with lapatinib**\\n\\n167\\n\\n168 To model the full range of development of HER2i resistance starting from primary\\n\\n169 sensitive phase, via drug-tolerant persister (DTP) development, and re-emergence of\\n\\n170 proliferative drug-tolerant expanded persister (DTEP) cells, to long-term resistant\\n\\n171 cells (LR), the HER2i sensitive cell line BT474, and its brain seeking variant BT474Br\\n\\n172 (Zhang et al, 2013), were exposed to therapeutically relevant 1 \\uf06dM of lapatinib every\\n\\n173 3 days for up to 9 months (Fig. 1A). Both cell lines followed a similar pattern of\\n\\n174 lapatinib tolerance development, where at the 9-day timepoint only a few DTP cells\\n\\n175 could be microscopically observed, whereas the emergence DTEP population took\\n\\n176 about 6 months (Fig. 1A). Following this, the plates were fully populated by the LR\\n\\n177 cells after 9 months of continuous lapatinib treatment (Fig. 1A). Importantly, in\\n\\n178 addition to lapatinib, the LR clones of both BT474 and BT474Br displayed strong\\n\\n179 cross resistance to tucatinib (a HER2i), afatinib (a HER2/EGFR inhibitor), and\\n\\n180 neratinib (a HER2/HER4/EGFR inhibitor) (Figure EV1). This indicates that the\\n\\n181 acquired resistance is not specific to lapatinib but is driven by a mechanism that is\\n\\n182 generally relevant to the ERBB family of RTKs.\\n\\n183\\n\\n184 **Transcriptomic landscape of acquired lapatinib resistance in BT474 cells**\\n\\n185\\n\\n186 Recent studies have focused on the molecular characterization of DTP cells in\\n\\n187 response to kinase inhibitor therapies (Chang et al., 2022; Kurppa et al., 2020;\\n\\n\\n-----\\n\\n188 Marsolier et al, 2022) but there is no published information about transcriptional\\n\\n189 profiles of TKI treated cells at the DTP-DTEP or DTEP-LR transitions. To this end,\\n\\n190 the transcriptional profiles from each functional state of lapatinib drug tolerance and\\n\\n191 resistance development in BT474 cells were surveyed by bulk RNA-sequencing.\\n\\n192 From three technical replicates per condition, and by using statistical criteria of\\n\\n193 |logFC|>2 and Benjamini-Hochberg adjusted p <0.05, upregulation of 144, 1169 and\\n\\n194 16 genes was found upon the control-DTP, DTP-DTEP, and DTEP-LR transitions of\\n\\n195 BT474 cells, respectively (Fig. 1B, Dataset EV1). On the other hand, the number of\\n\\n196 downregulated genes were 517, 930 and 28, respectively (Fig. 1B, Dataset EV1).\\n\\n197 The highest number of differentially regulated genes upon the DTP-DTEP transition\\n\\n198 indicates that this is the transition phase where the BT474 cell fate is most robustly\\n\\n199 impacted during the resistance development. When assessing the patterns of gene\\n\\n200 expression changes by unsupervised soft clustering analysis (Futschik & Carlisle,\\n\\n201 2005), we identified six approximately similar size gene clusters with distinct\\n\\n202 regulation patterns (Appendix Figure S1A). The genes included in these clusters are\\n\\n203 listed in the Dataset EV2. Such regulation patterns indicate that neither gene\\n\\n204 activation nor gene repression characterize lapatinib resistance development in\\n\\n205 BT474 cells, but unique gene expression programs are involved in each of these\\n\\n206 steps.\\n\\n207\\n\\n208 **Gene regulatory mechanisms and hallmarks associated with HER2i resistance**\\n\\n209 **transitions**\\n\\n210\\n\\n211 To understand gene regulatory mechanisms controlling transitions between the\\n\\n212 different phases on lapatinib tolerance and resistance development, we predicted the\\n\\n\\n-----\\n\\n213 transcription factor binding sites enriched on promoter regions of the differentially\\n\\n214 regulated genes (Fig. 1B, Dataset EV3). Using FDR < 0.05 as a cut-off, the most\\n\\n215 highly enriched transcription factor elements in genes downregulated upon the\\n\\n216 control-DTP transition were E2F4, FOXM1, FLI1, E2F1 and NFYA (Fig. 1B, Dataset\\n\\n217 EV3). Strikingly, most of these transcription factor binding sites were enriched also in\\n\\n218 the genes significantly upregulated in DTEP cells (Fig. 1B in bold, Dataset EV3).\\n\\n219 This indicates that these transcription factors are inactivated co-ordinately upon the\\n\\n220 development of the DTP state and re-activated upon re-growth of the DTEPs. E2F4,\\n\\n221 FOXM1 and E2F1 are all indicated in development and progression of breast cancer.\\n\\n222 We validated inhibition of selected BT474 cell DTP FOXM1 and E2F target genes\\n\\n223 across different HER2+ cells treated with lapatinib to reach the DTP state (Figure\\n\\n224 EV2). On the other hand, binding sites for transcription factors CBX2, ESR1, TCF12\\n\\n225 and ZNF217 were found significantly enriched in the genes downregulated upon\\n\\n226 DTP to DTEP transition, whereas STAT1 and STAT2 were enriched among the\\n\\n227 genes upregulated in LR cells as compared to DTEPs (Fig. 1B, Dataset EV3).\\n\\n228\\n\\n229 To identify cancer hallmark processes involved in each transition phase, the entire\\n\\n230 transcriptomics data was re-analysed by Gene Set Variation Analysis (GSVA).\\n\\n231 Consistent with recent evidence from other DTP models (Chang et al., 2022;\\n\\n232 Dhimolea et al., 2021; Kurppa et al., 2020), MYC signaling was inhibited in the\\n\\n233 lapatinib-treated BT474 DTPs, but reactivated in DTEPs (Fig. 1C, gene sets 3 and\\n\\n234 6). Similar gene regulation pattern was observed for E2F1 targets, G2/M checkpoint,\\n\\n235 mTOR signaling, and androgen response (Fig. 1C, gene set 3), ROS signaling,\\n\\n236 oxidative phosphorylation, and DNA repair (Fig. 1C, gene set 6). Interestingly, the\\n\\n237 DTP-DTEP transition was associated also with downregulation of several cancer\\n\\n\\n-----\\n\\n238 hallmark gene sets, most apparently seen in set 4 where KRAS signaling, EMT,\\n\\n239 WNT/ß-catenin, and inflammatory response genes were all suppressed upon\\n\\n240 proliferation reactivation (Fig. 1C). Additional hallmark gene sets regulated between\\n\\n241 the lapatinib resistance development transitions are displayed in appendix figure\\n\\n242 S1B.\\n\\n243\\n\\n244 Collectively these results demonstrate that unique gene clusters and biological\\n\\n245 processes are involved in each step of lapatinib resistance development. This bulk\\n\\n246 RNA-sequencing data provides a rich resource for future studies of gene regulatory\\n\\n247 mechanisms in HER2i tolerance and resistance development. However, it is clear\\n\\n248 that future single cell RNA sequencing studies are needed to understand clonality of\\n\\n249 gene expression changes induced during these transitions.\\n\\n250\\n\\n251\\n\\n252 **Phosphatase gene expression landscape in DTPs and DTEPs**\\n\\n253\\n\\n254 Recent data indicates that development of cancer therapy resistance is initiated by\\n\\n255 non-genetic signaling rewiring mediated by post-translational regulation of\\n\\n256 intracellular signaling pathways (De Conti et al., 2021; Hata et al., 2016; Marine et\\n\\n257 _al., 2020). Protein phosphorylation is the most prevalent post-translational_\\n\\n258 modification in cancer cells, and cancer cell phosphoproteomes are regulated by\\n\\n259 kinases and phosphatases. Whereas the role of kinases in non-genetic therapy\\n\\n260 tolerance development has been extensively studied (Marine et al., 2020), the\\n\\n261 importance of phosphatases to development of non-genetic kinase inhibitor therapy\\n\\n262 tolerance have been thus far poorly characterized. Therefore, we focused on the\\n\\n263 dynamics of phosphatase gene regulation during lapatinib tolerance development.\\n\\n\\n-----\\n\\n264 Importantly, there were only four phosphatase genes (CDC25A, CDC25C, DUSP6\\n\\n265 and SYNJ1) that were synchronously downregulated in DTP cells but upregulated in\\n\\n266 DTEPs versus DTPs (Fig. 2A-C, Dataset EV1). We rationalized that these four\\n\\n267 phosphatases might be particularly relevant for allowing the regrowth of the DTEP\\n\\n268 population under continuous lapatinib treatment. Even though a large group of other\\n\\n269 genes were found differentially regulated between the DTEP and LR populations\\n\\n270 (Fig. 1B, Dataset EV1 and EV2), none of the phosphatase genes were significantly\\n\\n271 regulated in the DTEP-LR transition (Dataset EV1). This indicates that regulation of\\n\\n272 phosphatase gene expression could be primarily relevant to the early non-genetic\\n\\n273 phases of acquired lapatinib resistance.\\n\\n274\\n\\n275\\n\\n276 **Clinical association of DUSP6 with poor prognosis HER2+ breast cancer**\\n\\n277\\n\\n278 Based on the above analysis, CDC25A, CDC25C, DUSP6 and SYNJ1 were the only\\n\\n279 phosphatases that were significantly regulated during both therapy tolerance\\n\\n280 transitions (Fig. 2A-C). To evaluate potential clinical relevance of the selected four\\n\\n281 phosphatases in human breast cancer, we first examined their expression levels\\n\\n282 across breast cancer subtypes in the METABRIC dataset (Dataref:(Pereira et al,\\n\\n283 2016). Interestingly, DUSP6 was the only one of these four phosphatases that was\\n\\n284 selectively overexpressed in the target HER2+ subtype of breast cancers (Fig. 2D).\\n\\n285 The closest functional orthologue for DUSP6, DUSP1 did not show HER2+ selective\\n\\n286 overexpression (Appendix Figure S2A). _DUSP6 overexpression in HER2+ breast_\\n\\n287 cancer was confirmed in the TCGA breast invasive carcinoma dataset (Dataref:\\n\\n288 (Cerami et al, 2012; de Bruijn et al, 2023; Gao et al, 2013)(Appendix Figure S2B).\\n\\n289 Further, when the 1082 breast invasive carcinoma samples from the TCGA BRCA\\n\\n-----\\n\\n290 dataset (Dataref: (Cerami et al., 2012; de Bruijn et al., 2023; Gao et al., 2013) were\\n\\n291 divided to DUSP6high and DUSP6low groups based on their _DUSP6_ mRNA\\n\\n292 expression levels, HER2+ and luminal A subtypes were clearly enriched among the\\n\\n293 DUSP6high samples (Fig. 2E). HER2 positivity was also found enriched among\\n\\n294 DUSP6high tumors based on HER2 immunohistochemistry of samples available for\\n\\n295 staining in the Breast Invasive Carcinoma (TCGA, Firehose legacy) dataset (Dataref:\\n\\n296 (Cerami et al., 2012; de Bruijn et al., 2023; Gao et al., 2013) (Appendix Figure S2C).\\n\\n297\\n\\n298 To study the prognostic value of DUSP6 expression in HER2+ breast cancers from\\n\\n299 the TCGA BRCA-dataset, the patients with high _ERBB2_ (gene coding for HER2)\\n\\n300 expression were further divided into DUSP6high and low groups. Comparison of the\\n\\n301 survival outcomes between the two groups indicate that high _DUSP6 expression_\\n\\n302 predicts poor overall and disease-free survival among HER2+ patients (Fig. 2F,G). In\\n\\n303 multivariable analysis of high _ERBB2_ expressing tumors, increase in _DUSP6_\\n\\n304 expression and large tumor size (T4) remained significant independent prognostic\\n\\n305 factors (Appendix Figure S2D). However, _DUSP6 mRNA expression was neither a_\\n\\n306 prognostic factor in Luminal B or Basal subtypes, nor across unselected breast\\n\\n307 cancer patient population (Appendix Figure S2E), further highlighting the selective\\n\\n308 connection between DUSP6 and HER2 in breast cancer progression.\\n\\n309\\n\\n310 Interestingly, luminal B cancers were clearly enriched in _DUSP6low samples as_\\n\\n311 compared to _DUSP6high (Fig. 2E). Luminal B are estrogen receptor (ER) positive_\\n\\n312 cancers that often also express HER2. Clinically luminal B cancers are less\\n\\n313 aggressive than HER2+ cancers. This data may suggest that DUSP6 has a role in\\n\\n314 suppressing ER positivity, and thereby increasing the relative numbers of HER2+\\n\\n\\n-----\\n\\n315 cancers over luminal B cancers. Although the details of this regulation remain\\n\\n316 unclear, this hypothesis is supported by strongly decreased expression of ER protein\\n\\n317 and _ESR1 mRNA, coding for ER, in HER2+_ _DUSP6high samples as compared to_\\n\\n318 HER2+ DUSP6low samples (Appendix Figure S2F,G).\\n\\n319\\n\\n320 These data demonstrate that DUSP6 is clinically associated with the aggressive\\n\\n321 HER2+ breast cancer but may also have a broader role in defining breast cancer\\n\\n322 subtype development.\\n\\n323\\n\\n324 **DUSP6 promotes the HER2i tolerance and DTP-DTEP transition under**\\n\\n325 **continuous lapatinib treatment**\\n\\n326\\n\\n327 Clinical relevance of DUSP6 in HER2+ breast cancers among the four candidate\\n\\n328 phosphatases differentially regulated at DTP-DTEP transition, together with the\\n\\n329 feasibility of DUSP6 targeting by small molecules (Kong et al., 2023; Korotchenko et\\n\\n330 _al., 2014; Molina et al., 2009; Vainonen et al., 2021; Zandi et al., 2022), motivated us_\\n\\n331 to select DUSP6 as the phosphatase to be focused in this study. We confirmed\\n\\n332 differential expression of DUSP6 across different resistance acquisition transitions in\\n\\n333 BT474 cells. Consistent with the RNA sequencing results (Fig. 2C), DUSP6 protein\\n\\n334 expression was strongly induced upon the DTP-DTEP transition (Fig. 3A). While the\\n\\n335 _DUSP6 mRNA levels were diminished in the fully resistant LR clones compared to_\\n\\n336 DTEPs (Fig. 2C), its protein levels remained robustly elevated presumably via post\\n337 translational stabilization mechanisms (Fig. 3A). Validating that these effects were\\n\\n338 not cell line specific, DUSP6 protein was also increased in BT474BrLR cells as\\n\\n339 compared to the parental cells (Appendix Figure 3A). We further validated inhibition\\n\\n\\n-----\\n\\n340 of DUSP6 upon DTP phase across independent set of lapatinib treated HER2+ cells\\n\\n341 including BT474Br, and two cell lines (EFM192A, HCC1419) from (Data ref:(Chang\\n\\n342 _et al., 2022)( GSE155342), or NSCLC, melanoma, and colorectal cancer cells_ _de_\\n\\n343 _novo treated with various TKIs (Fig. 3B). Therefore, inhibition of DUSP6 expression_\\n\\n344 appears as a general mechanism associated with establishment of TKI-induced\\n\\n345 therapy tolerance. Selected E2F1 and FOXM1 target genes were also confirmed to\\n\\n346 be downregulated in TKI-treated NSCLC, MM, and CRC DTP cells (Appendix Figure\\n\\n347 S3B). Notably, among the transcription factors differentially implicated upon HER2i\\n\\n348 tolerance development (Fig. 1B), forkhead box transcription factor M1 (FOXM1) and\\n\\n349 NFYA bind to DUSP6 promoter (Dataset EV4), and coinciding with _DUSP6_\\n\\n350 expression, FOXM1 gene expression is downregulated in DTPs and upregulated in\\n\\n351 DTEPs (Appendix Figure S3C). Functionally, a small molecule inhibitor of FOXM1\\n\\n352 (FDI-6) (Wang et al, 2021) inhibited DUSP6 expression in a time-dependent fashion\\n\\n353 in BT474 cells (Appendix Figure S3D,E). Together with its role as breast cancer\\n\\n354 oncogene involved in therapy resistance (Zhang et al, 2021), these data imply\\n\\n355 FOXM1 as a viable candidate inducing _DUSP6 expression during the DTP-DTEP_\\n\\n356 transition.\\n\\n357\\n\\n358 To evaluate whether the hallmark gene sets co-regulated with DUSP6 during therapy\\n\\n359 tolerance transitions (Fig. 1C) could be functionally downstream of DUSP6, we\\n\\n360 performed a RNAseq analysis from _DUSP6 depleted MDA-MB-453 cells and_\\n\\n361 compared these two gene sets. Notably, there was a marked overlap between\\n\\n362 hallmark gene sets from these two conditions in which DUSP6 expression was\\n\\n363 suppressed (Fig. 1C versus Fig. 3C; overlapping gene sets highlighted in red).\\n\\n364 Especially interesting finding was that the DUSP6 knockdown cells displayed a gene\\n\\n\\n-----\\n\\n365 expression pattern linked to dormant cancer cells such as inhibition of MYC, E2F1\\n\\n366 targets, and the PI3K/AKT/mTOR signaling, as well as activation of the interferon\\n\\n367 response (Chang et al., 2022; Dhimolea et al., 2021; Kurppa et al., 2020; Marine et\\n\\n368 _al., 2020). The finding that siRNA-mediated DUSP6 depletion (Fig. 3C) recapitulates_\\n\\n369 the gene expression profile in the lapatinib-induced DTP cells (Fig. 1C), clearly\\n\\n370 indicates that inhibition of DUSP6-driven gene expression programs functionally\\n\\n371 contribute to the HER2i-elicited growth inhibition.\\n\\n372\\n\\n373\\n\\n374 To functionally validate that increased DUSP6 expression in DTEP cells contributes\\n\\n375 to their survival, we ectopically overexpressed DUSP6 in BT474 cells, and subjected\\n\\n376 the cells to treatment with lapatinib, neratinib, afatinib, or tucatinib. Importantly,\\n\\n377 DUSP6 overexpression was able to dampen both cell viability inhibition, and\\n\\n378 apoptosis induction, by all four tested HER2is (Fig. 3D,E and Figure EV3A-D).\\n\\n379 Mechanistically, the anti-apoptotic activity of DUSP6 in lapatinib treated BT474 cells\\n\\n380 appeared to be independent of ERK MAPK regulation, as overexpression of the KIM\\n\\n381 mutant of DUSP6 (R64A,R65A), incapable in ERK binding (Nichols et al, 2000), had\\n\\n382 equally strong impact as the wild-type DUSP6 (Fig. 3F and Appendix Figure S3F).\\n\\n383 ERK-independent antiapoptotic activity for DUSP6 in cancer cells was suggested\\n\\n384 also recently (Kong et al., 2023). As a complementary pharmacological approach,\\n\\n385 BT474 cells were treated with lapatinib or neratinib alone, or in combination with\\n\\n386 small molecule DUSP6 inhibitor BCI for 6 months. BCI is a semi-allosteric inhibitor of\\n\\n387 both DUSP1 and DUSP6 and several studies have demonstrated that BCI\\n\\n388 phenocopies genetic DUSP6 inhibition in cancer (Kong et al., 2023; Ramkissoon et\\n\\n389 _al, 2019; Shojaee et al., 2015). Notably, as compared to the monotherapies,_\\n\\n390 combination with BCI preempted the DTEP development in both lapatinib and\\n\\n\\n-----\\n\\n391 neratinib treated cells (Fig. 3G). Strongly indicative of selective drug interaction\\n\\n392 rather than overall toxicity by BCI, the BCI used at the given concentration for 6\\n\\n393 months did not kill all BT474 cells but potently synergized with the HER2is (Fig. 3G).\\n\\n394 The fact that DUSP6 overexpression inhibited HER2i-elicited cell killing (Fig. 3E),\\n\\n395 whereas BCI abrogated development of HER2i tolerant cells (Fig. 3G), strongly\\n\\n396 support selective effects for BCI on DUSP6. Indeed, the role of DUSP1 as a primary\\n\\n397 BCI target was overruled by the results that DUSP1 was neither expressed at the\\n\\n398 protein level in BT474 cells (Fig. 3A), nor its mRNA was found differentially regulated\\n\\n399 between any of the acquired resistance phases (Fig. 2A, B, Dataset EV1). In\\n\\n400 harmony, DUSP1 overexpression had clearly weaker activity than DUSP6\\n\\n401 overexpression in the rescue experiments (Figure EV3D). To further validate the\\n\\n402 selectivity of BCI as a DUSP6 inhibitor, we generated DUSP6 knockout (DUSP6KO)\\n\\n403 MDA-MB-453 cells by CRISPR/CAS9 (Appendix Figure S3G). Indeed, three\\n\\n404 independent single cell clones of DUSP6KO cells were significantly less sensitive to\\n\\n405 BCI-elicited inhibition of cell viability as compared to the control cells (Appendix\\n\\n406 Figure S3H).\\n\\n407\\n\\n408 Collectively, these results identify DUSP6 downregulation to be functionally relevant\\n\\n409 for establishment of the DTP phase, whereas its transcriptional induction contributes\\n\\n410 to DTP-DTEP transition in lapatinib treated HER2+ cells.\\n\\n411\\n\\n412 **DUSP6 targeting kills HER2i resistant breast cancer cells and synergizes with**\\n\\n413 **HER2i combination therapies**\\n\\n414\\n\\n\\n-----\\n\\n415 After discovering the role for DUSP6 in the development of HER2i tolerance, we\\n\\n416 wanted to address its role in HER2+ breast cancer cells with stable HER2i\\n\\n417 resistance. To this end, we compared pharmacological DUSP6 targeting against a\\n\\n418 library of available small molecule modulators of other phosphatases (Lazo et al.,\\n\\n419 2018; Vainonen et al., 2021). Across either HER2i resistant cells lines, inhibition of\\n\\n420 cell viability by phosphatase targeting was observed only with DUSP6 inhibitor BCI,\\n\\n421 its derivative BCI-215, and with FTY-720 that reactivates protein phosphatase 2A\\n\\n422 (PP2A) by SET inhibition (Saddoughi et al, 2013)(Figure EV4). The phenocopying\\n\\n423 results between BCI and BCI-215 is an important additional evidence for the\\n\\n424 selectivity of BCI type of drugs towards DUSPs, as BCI-215 was demonstrated to\\n\\n425 activate only the DUSP1 and 6 target MAPKs among the 43 tested kinases (Chan et\\n\\n426 _al, 2020; Kaltenmeier et al, 2017). Based on the response to kinase inhibitors and_\\n\\n427 anti-apoptotic antagonists, the cells that are HER2i resistant, but sensitive to DUSP6\\n\\n428 inhibition, are co-dependent on PI3K/AKT, PLK1 and AXL kinase activities, and on\\n\\n429 the anti-apoptotic proteins IkB, survivin, cIAP and/or XIAP (Figure EV4). Further\\n\\n430 corroborating the role for DUSP6 as the anti-apoptotic target in stably HER2i\\n\\n431 resistance cells, siRNA-mediated depletion of _DUSP6 induced apoptosis in MDA-_\\n\\n432 MB-453 cells (Fig. 4A). This was not seen with DUSP1 inhibition (Fig. 4A). DUSP6\\n\\n433 depletion also induced apoptosis in another HER2i resistant HER2+ cell line MDA\\n434 MB-361 (Appendix Figure S4A). Furthermore, independent clones of CRISPR/CAS9\\n\\n435 targeted MDA-MB-453 _DUSP6 KO cells showed impaired long-term colony growth_\\n\\n436 potential (Fig. 4B). This was due to loss of DUSP6 expression, as lentiviral re\\n437 expression of DUSP6 rescued the phenotype (Appendix Figure S4B-E).\\n\\n438\\n\\n\\n-----\\n\\n439 Notably, the genetic DUSP6 targeting also sensitized MDA-MB-453 cells to several\\n\\n440 HER2 targeting approaches. Indeed, whereas the parental MDA-MB-453 cells were\\n\\n441 resistant to the clinically relevant concentrations of lapatinib, neratinib and\\n\\n442 trastuzumab, this resistance was abrogated in the DUSP6-siRNA targeted cells (Fig.\\n\\n443 4C, Appendix Figure S4F). These results also validate that DUSP6 inhibition\\n\\n444 sensitizes to HER2 inhibition regardless of whether small molecule inhibitors or\\n\\n445 therapeutic antibody (trastuzumab) is used. The impact of genetic DUSP6 targeting\\n\\n446 in HER2i sensitization was recapitulated by BCI treatment (Fig. 4D). Additionally,\\n\\n447 _DUSP6 depletion increased sensitivity to the combination of neratinib and_\\n\\n448 capecitabine (Fig. 4E), which is a combination therapy in clinical use (Saura et al,\\n\\n449 2020). Furthermore, _DUSP6 depletion enhanced sensitivity to the combination of_\\n\\n450 tucatinib+trastuzumab+capecitabine (Appendix Figure S4G), which improves\\n\\n451 progression free survival and overall survival in patients with HER2+ metastatic\\n\\n452 breast cancer (Murthy et al, 2020).\\n\\n453\\n\\n454 Together with DUSP6 overexpression experiments in HER2i sensitive cells (Fig. 3G\\n455 I), these data provide strong evidence that DUSP6 contributes to the HER2i\\n\\n456 resistance both in monotherapy and combination therapy settings.\\n\\n457\\n\\n458 **DUSP6 inhibition overcomes HER2 inhibitor resistance in vivo**\\n\\n459\\n\\n460 To validate the _in vivo_ relevance of the results, we used both genetic and\\n\\n461 pharmacological targeting of DUSP6 in HER2i resistant xenograft models. To start\\n\\n462 with, we evaluated the impact of CRISPR/CAS9-mediated _DUSP6 knockout on the_\\n\\n463 xenograft growth of MDA-MB-453 cells in immunocompromised BALB/cOlaHsd\\n\\n-----\\n\\n464 Foxn1nu mice. Importantly, the two _DUSP6 KO clones showed significant and_\\n\\n465 indistinguishable antitumor effects as compared to the control cells (Fig. 5A). Next,\\n\\n466 we asked whether pharmacological DUSP6 blockade overcomes HER2i resistance\\n\\n467 _in vivo. For this purpose, we used two different HER2+ cell models, MDA-MB-453 or_\\n\\n468 HCC1954. When xenografts with these two cell lines had reached the approximate\\n\\n469 size of 100 mm[3], the mice were randomized into four treatment groups; vehicle,\\n\\n470 lapatinib/neratinib (50 mg/kg), BCI (50 mg/kg), and lapatinib/neratinib+BCI.\\n\\n471 Importantly, validating the _in vivo HER2i resistance of both chosen HER2+ cell_\\n\\n472 models, tumors from both cell lines were fully resistant to clinically relevant doses of\\n\\n473 either lapatinib or neratinib (Fig. 5B-E). Notably, both MDA-MB-453 and HCC1954\\n\\n474 tumors also displayed strong resistance to BCI monotherapy (Fig. 5B-E), indicating\\n\\n475 for tumor microenvironment-mediated impact as compared to the _in vitro cultures._\\n\\n476 However, combination of BCI very efficiently preempted the lapatinib or neratinib\\n\\n477 resistance phenotype (Fig. 5B-E). Indicating for potential clinical utility, DUSP6 and\\n\\n478 HER2-targeted therapies displayed a clear synthetic lethal drug interaction when\\n\\n479 assessed by a waterfall blot in both cell models (Fig. 5C, E). The dramatic\\n\\n480 combinatorial activity of DUSP6 and HER2 targeting on cellular viability was also\\n\\n481 evidenced by lack of cytoplasmic eosin staining of the MDA-MB-453 xenograft tumor\\n\\n482 after 24 d of treatment (Fig. 5F). Consistent with previous studies with BCI\\n\\n483 (Kesarwani et al, 2017; Ramkissoon et al., 2019; Shojaee et al., 2015; Wu et al.,\\n\\n484 2018), we did not observe any apparent signs of toxicity or weight loss (Appendix\\n\\n485 Figure S4D) in any treatment groups. In addition, normal fibroblasts displayed\\n\\n486 several folds decreased sensitivity to BCI in an in vitro cell viability assay (Appendix\\n\\n487 Figure S4E), further indicating for a favorable dose-window between impact of\\n\\n488 DUSP6 inhibition in HER2i resistance, and its effects to normal cells.\\n\\n\\n-----\\n\\n489\\n\\n490 **DUSP6 targeting does not lead to the compensatory HER3 induction**\\n\\n491 **characteristic to AKT inhibition**\\n\\n492\\n\\n493 Constitutive activity of PI3K/AKT signaling pathway is strongly associated with\\n\\n494 HER2i therapy resistance (Berns et al, 2007; Majewski et al, 2015; Nagata et al,\\n\\n495 2004). Further, clinical studies of HER2i plus PI3K/AKT inhibitors demonstrated\\n\\n496 some clinical activity, but did not lead to approval of these combinations (Hudis et al,\\n\\n497 2013; Saura et al, 2014). On the other hand, PI3K/AKT activation was one of the\\n\\n498 DUSP6-driven hallmark gene sets associated with the DTP-DTEP transition (Fig. 1C\\n\\n499 and 3F). Therefore, it was relevant to compare the quantitative and qualitative\\n\\n500 differences between DUSP6 blockade versus AKT inhibition in combination with the\\n\\n501 HER2i in the resistant models. Having demonstrated phenocopying growth effects\\n\\n502 between genetic DUSP6 inhibition and BCI, as well as resistance of DUSP KO cells\\n\\n503 to BCI, these experiments were mostly performed by comparing the compounds\\n\\n504 MK2206 (AKTi) and BCI (DUSP6i) as the alternative pharmacological HER2i\\n\\n505 combination approaches.\\n\\n506\\n\\n507 In cell viability assay, both BCI and MK2206 synergized with already low micromolar\\n\\n508 concentrations of lapatinib and neratinib in MDA-MB-453 and HCC1954 cells,\\n\\n509 respectively (Fig. 6A,B, and Appendix Figures S4C and S5A). However, as\\n\\n510 compared to AKT inhibition, DUSP6 targeting had qualitatively superior pro-apoptotic\\n\\n511 activity. Regardless of efficient inhibition of AKT phosphorylation, MK2206 did not\\n\\n512 induce apoptosis alone, or in combination with lapatinib (Fig. 6C,D). In contrast,\\n\\n\\n-----\\n\\n513 treatment with BCI plus lapatinib triggered apoptosis across all the tested cell models\\n\\n514 (Fig. 6C,D, and Appendix Figure S5B).\\n\\n515\\n\\n516 HER3 is a key pro-survival receptor in HER2+ breast cancer cells (Haikala & Janne,\\n\\n517 2021). Further, compensatory induction of HER3 is thought to be one of the primary\\n\\n518 mechanisms behind the resistance to the combination of HER2 inhibition and AKT\\n\\n519 targeting (Chandarlapaty et al, 2011; Wilson et al., 2012). Accordingly, AKT inhibition\\n\\n520 resulted in HER3 induction in MDA-MBA-453 cells treated with either MK2206 alone,\\n\\n521 and with MK2206-lapatinib combination (Fig. 6C,E). Pharmacological inhibition of the\\n\\n522 AKT upstream kinase PI3K also induced HER3 expression (Appendix Figure S5C).\\n\\n523 In contrast, BCI decreased HER2 and HER3 expressions as a monotherapy, and in\\n\\n524 combination with lapatinib (Fig. 6C, F). Notably, both HER2 and HER3 were\\n\\n525 downregulated 3 h already after BCI treatment, and this coincided with the induction\\n\\n526 of ERK phosphorylation, and destabilization of DUSP6, both serving as signs for BCI\\n\\n527 target engagement (Fig. 6G). Accordingly, siRNA-mediated knockdown of _DUSP6_\\n\\n528 reduced HER2 and HER3 protein levels in MDA-MB-453, HCC1954, and the HER3+\\n\\n529 triple negative breast cancer cell line MDA-MB-468 (Fig. 6H and Appendix Figures\\n\\n530 S5D,E). Indicative of transcriptional regulation, _DUSP6 inhibition decreased_ _HER2_\\n\\n531 and _HER3 mRNA expression, but did not impact HER2 and HER3 protein stability_\\n\\n532 (Appendix Figure S5F,G). Further, indicative of selective effects on HER2 and HER3\\n\\n533 among the ERBB receptors, EGFR expression was unaffected by DUSP6 targeting\\n\\n534 in MDA-MB-468 (Appendix Figure S5E). The tumor material from the BCI-treated\\n\\n535 MDA-MB-453 xenograft model was further used for _in vivo_ validation of HER3 as\\n\\n536 DUSP6 downstream target. Indeed, BCI-treated tumors displayed vastly decreased\\n\\n537 tumor cell immunopositivity for HER3, and to lesser extent HER2 (Fig. 6I). Finally,\\n\\n\\n-----\\n\\n538 clinically high _DUSP6_ tumors had significantly higher expression of tyrosine 1298\\n\\n539 phosphorylated HER3 (Fig. 6J).\\n\\n540\\n\\n541 These findings indicate that when HER2i is combined AKTi, the HER2+ cells survive\\n\\n542 due to compensatory induction of HER3, whereas DUSP6 blockade rather inhibits\\n\\n543 HER3, and therefore BCI+lapatinib treated cells succumb to apoptosis.\\n\\n544\\n\\n545 **DUSP6 targeting overcomes NRG-mediated HER2i therapy tolerance**\\n\\n546\\n\\n547 HER3 is activated in tumors by tumor microenvironment derived ligand NRG, and\\n\\n548 NRG-activated HER3 drives HER2i tolerance (Kodack et al, 2017; Leung et al,\\n\\n549 2015). On the other hand, our results indicate that DUSP6 may drive HER2i\\n\\n550 resistance by promoting the expression of HER3. With this background, we\\n\\n551 compared the capacity of DUSP6 and AKT inhibition to reverse HER2i tolerance by\\n\\n552 the NRG/HER3 axis. As expected, (Kodack et al., 2017; Leung et al., 2015), NRG\\n\\n553 treatment induced tolerance to both neratinib and lapatinib across all four tested\\n\\n554 primarily sensitive cell lines (Fig. 7A and Appendix Figure S6A). The NRG-mediated\\n\\n555 rescue from both HER2is was abrogated in a concentration dependent manner after\\n\\n556 treatment with BCI, but not with MK2206 (Fig. 7A and Appendix Figure S6A). The\\n\\n557 only exception was HCC2218 cells, in which also MK2206 reversed NRG-elicited\\n\\n558 HER2i tolerance (Fig. 7A and Appendix Figure S6A). These results were validated at\\n\\n559 the level of differential apoptosis induction between DUSP6 and AKT targeting.\\n\\n560 Whereas NRG completely prevented HER2i-elicited induction of apoptosis in BT474\\n\\n561 cells, and AKT inhibition could not reverse this, DUSP6 inhibition restored the\\n\\n562 apoptotic activity (Fig. 7B). On the other hand, RNAi-mediated _HER3 knockdown_\\n\\n\\n-----\\n\\n563 triggered apoptosis in MDA-MB-453 cells (Appendix Figure S6B). To functionally\\n\\n564 prove that inhibition of _HER3 expression mediates BCI effects, we performed a_\\n\\n565 rescue experiment in MDA-MB-453 cells with ectopic stable overexpression of HER3\\n\\n566 under heterologous CMV promoter (Appendix Figure S6C). As compared to control\\n\\n567 cells, the HER3 overexpressing cells displayed significant resistance to BCI (Fig.\\n\\n568 7C).\\n\\n569\\n\\n570 To link these results back to the development of HER2i tolerance, we found that\\n\\n571 targeting of DUSP6, but not AKT, reversed the NRG-elicited effects on lapatinib\\n\\n572 tolerance in the DTP assay (Fig. 7D). This BCI effect was likely due to DUSP6\\n\\n573 inhibition, as siRNA-mediated _DUSP6 depletion also abrogated the NRG-induced_\\n\\n574 cell survival in BT474 cells (Appendix Figure S6D). Importantly, deregulation of\\n\\n575 NRG-ERBB axis was evidenced also during the DTP-DTEP transition of lapatinib\\n\\n576 treated cells as, concomitantly with DUSP6 regulation (Fig. 2C), NRG3 and ERBB4\\n\\n577 were found upregulated in DTEP versus DTP cells (Appendix Figure S6E). Further,\\n\\n578 providing clinical validation for the link between NRG and DUSP6, NRG mRNA was\\n\\n579 significantly overexpressed in breast tumors with high _DUSP6_ mRNA expression\\n\\n580 (Fig. 7E).\\n\\n581\\n\\n582 Due to brain microenvironment derived NRG, HER3 has a key role in brain\\n\\n583 metastatic growth of HER2+ breast cancers (Da Silva et al, 2010; Haikala & Janne,\\n\\n584 2021). Based on the newly discovered role for DUSP6 in promoting HER3\\n\\n585 expression, we investigated contribution of DUSP6 in the brain metastatic outgrowth\\n\\n586 by using GFP-positive MDA-MB-361 cells as yet additional HER2+ in _vivo model. To_\\n\\n587 this end, we quantitated the zebrafish intracranial tumor area derived from GFP+\\n\\n\\n-----\\n\\n588 MDA-MB-361 cells transfected prior tumor implantation either with control or DUSP6\\n\\n589 targeting siRNA. As shown in Fig. 7F, the zebrafish larva with DUSP6 targeted cells\\n\\n590 had significantly smaller tumors 3d after the intracranial injection. Further, mice\\n\\n591 bearing the intracranial _DUSP6 KO MDA-MB-453 tumors had a significantly longer_\\n\\n592 survival time as compared with the CAS9 control group (Fig. 7G).\\n\\n593\\n\\n594 Collectively these results demonstrate that DUSP6 inhibition is superior to AKT\\n\\n595 blockade as a HER2i combination therapy strategy due to its newly discovered\\n\\n596 capacity to inhibit the NRG-HER3 axis and has potential in inhibiting brain metastatic\\n\\n597 growth of HER2+ cells.\\n\\n598\\n\\n599 **A DUSP6-HER3 feed forward loop drives the HER2i tolerance**\\n\\n600\\n\\n601 After identifying previously unrecognized regulation of HER3 by DUSP6 and\\n\\n602 demonstrating its functional relevance, we asked whether the HER3 reciprocally\\n\\n603 regulates DUSP6 expression. Indeed, _DUSP6 mRNA expression was potently_\\n\\n604 inhibited already with the smallest tested concentration of neratinib or lapatinib in the\\n\\n605 HER2i sensitive cells (BT474, BT474Br) (Fig. 8A and Appendix Figure S7A; in\\n\\n606 green). However, among the HER2i resistant cell lines, DUSP6 was not inhibited by\\n\\n607 HER2is in MDA-MB-361, MDA-MB-453, BT474Br-LR, and BT474-LR cells, whereas\\n\\n608 HCC1954 showed an intermediate phenotype (Fig. 8A and Appendix Figure S7A; in\\n\\n609 red). Additionally, this regulation was specific to _DUSP6, as_ _DUSP1_ mRNA\\n\\n610 expression was not inhibited by HER2 targeting in any of the tested cell modes (Fig.\\n\\n611 8A and Appendix Figure S7A). Clinically DUSP6 and HER3 mRNA expression also\\n\\n612 correlated in HER2+ cancer samples in the TCGA-BRCA dataset (Cerami et al.,\\n\\n613 2012)(Appendix Figure S7B). Differential regulation of DUSP6 expression by HER2i\\n\\n\\n-----\\n\\n614 in sensitive (BT474Br, BT474) versus resistant (MDA-MB-361, MDA-MB-453,\\n\\n615 BT474BrLR) cells was validated at the protein level by western blotting (Fig. 8B,C\\n\\n616 and Figure EV5). Importantly, RNAi-mediated knockdown of _HER3_ decreased\\n\\n617 DUSP6 expression (Fig. 8D). HER3 dependent DUSP6 regulation was further\\n\\n618 supported by induction of _DUSP6 mRNA and protein expression by treatment of_\\n\\n619 serum-starved BT474 or BT474Br cells with the HER3 ligand NRG (Fig. 8E, F and\\n\\n620 Appendix Figures S7C,D). Induction of DUSP6 was selective for NRG, since\\n\\n621 hepatocyte growth factor (HGF), which activates MET RTK, failed to increase\\n\\n622 DUSP6 in BT474 cells (Appendix Figure S7C,D). To gain insights into the\\n\\n623 mechanism by which HER3 regulates DUSP6, the NRG treated cells were co-treated\\n\\n624 either with AKTi MK2206, or the MEK1/2 inhibitor trametinib. Consistent with a\\n\\n625 previous report that DUSP6 is a transcriptional ERK1/2 target (Zandi et al., 2022),\\n\\n626 treatment with trametinib inhibited both ERK phosphorylation and DUSP6 expression\\n\\n627 in all tested cell lines (Fig. 8E-G and Appendix Figure S7E), whereas AKT inhibition\\n\\n628 by MK2206 further increased DUSP6 expression (Fig. 8E-G). Notably, HER2i\\n629 elicited inhibition of DUSP6 expression correlated with the capability of HER2i to\\n\\n630 impact ERK phosphorylation (Fig. 8F, G and Figure EV5A,B).\\n\\n631\\n\\n632 Collectively these results reveal that inhibition of the newly discovered DUSP6-HER3\\n\\n633 feed-forward loop has a major contribution to HER2i therapy response, and that lack\\n\\n634 of DUSP6 inhibition is a novel HER2i resistance mechanism in acquired resistant\\n\\n635 cells in which HER2i fails to target MEK activity (Fig. 8H).\\n\\n636\\n\\n637\\n\\n\\n-----\\n\\n638 **Discussion**\\n\\n639\\n\\n640 Cancer therapy resistance is initiated by non-genetic signaling rewiring resulting in\\n\\n641 major changes in the epigenetic and transcriptional landscapes of DTPs (Chang et\\n\\n642 _al., 2022; De Conti et al., 2021; Marine et al., 2020; Sharma et al., 2010). It is_\\n\\n643 commonly believed that transition from dormant DTPs to proliferating DTEPs is a\\n\\n644 pivotal step towards development of the disseminated disease, and eventually late\\n645 stage genetic changes which determine the ultimate therapy resistance (Aguirre\\n646 Ghiso, 2021; Marine et al., 2020). Here, we report the first transcriptomic analysis of\\n\\n647 the DTP-DTEP transition in TKI treated cancer cells. In addition to importance of the\\n\\n648 discovery of the role of DUSP6 in DTP-DTEP transition, the transcriptome data will\\n\\n649 provide a rich resource for future studies to understand different stages of the HER2i\\n\\n650 resistance development. Based on the fuzzy clustering data (Appendix figure S1 and\\n\\n651 Dataset EV2), the gene clusters 1, 2 and 6, contain genes that are strongly regulated\\n\\n652 upon the DTP-DTEP transition and are therefore contain many additional genes\\n\\n653 involved in the evasion of the DTP cells from lapatinib-induced growth inhibition. The\\n\\n654 longitudinal transcriptomics profiles also allowed us to identify potential master\\n\\n655 transcription factors involved in the lapatinib resistance development. The functional\\n\\n656 relevance of the differentially expressed genes associated with each therapy\\n\\n657 tolerance transition obviously need to be validated in future studies stemming from\\n\\n658 this novel resource. Further, to understand therapy tolerance development\\n\\n659 projectories at the single cell level, similar studies are needed by employing single\\n\\n660 cell RNA sequencing technologies.\\n\\n661\\n\\n\\n-----\\n\\n662 Phosphatases have recently emerged as novel druggable targets for cancer therapy\\n\\n663 (Lazo et al., 2018; Vainonen et al., 2021; Zandi et al., 2022). However, their\\n\\n664 contribution to development of non-genetic TKI therapy tolerance is still poorly\\n\\n665 understood. In the present study, we asked which phosphatases might contribute to\\n\\n666 the development of HER2i tolerance and resistance and explored their potential role\\n\\n667 as therapy targets in combination with the HER2i therapies in HER2+ breast cancer.\\n\\n668 Via transcriptomics analysis of lapatinib tolerance development over six months, we\\n\\n669 discovered an important role for DUSP6 in the re-growth of HER2i tolerant cells and\\n\\n670 showed that its genetic and pharmacological blockade potentiates HER2i sensitivity\\n\\n671 _in vitro and_ _in vivo in multiple different HER2+ cell models. Mechanistically, we_\\n\\n672 discovered that transcriptional _DUSP6_ inhibition is an important contributor to\\n\\n673 effective HER2i response whereas the lack of HER2i-elicited DUSP6 inhibition in the\\n\\n674 resistant cells provides a novel explanation for drug resistance in these cells (Fig.\\n\\n675 8H). We further discovered DUSP6 as a novel activator of _HER3 expression and_\\n\\n676 demonstrated that _HER3_ inhibition is a prerequisite for the therapeutic impact of\\n\\n677 DUSP6 inhibition. Inhibition of HER3 by DUSP6 targeting appears also an important\\n\\n678 mechanistic explanation for the superior apoptotic activity as compared to PI3K/AKT\\n\\n679 targeting in combination with HER2i. As we demonstrate, the therapeutic impact of\\n\\n680 DUSP6 targeting is not compromised by the NRG-mediated feedback to HER3 as is\\n\\n681 the case with AKT targeting and thereby our data identify DUSP6 targeting as a\\n\\n682 novel approach to target NRG-HER3 axis to overcome HER2i resistance. These are\\n\\n683 important advantages favoring DUSP6 blockade versus AKT inhibition as a future\\n\\n684 strategy for treatment of HER2+ breast cancer. A particularly important translational\\n\\n685 finding in our study is that DUSP6 defines therapy sensitivity of the lapatinib resistant\\n\\n686 cells in response to several ERBB TKIs with different specificities, and to their\\n\\n\\n-----\\n\\n687 chemotherapy combinations. This is consistent with the proposed HER3-DUSP6\\n\\n688 positive feedback mechanism, as HER3 or NRG overexpression confer resistance to\\n\\n689 several cancer drugs (Erjala et al., 2006; Haikala & Janne, 2021; Knuefermann et\\n\\n690 _al., 2003; Recondo et al., 2020; Wilson et al., 2012; Yonesaka et al., 2011). Thereby_\\n\\n691 identification of DUSP6 targeting as a novel approach for HER3 inhibition may have\\n\\n692 broad ramifications in different combination therapy settings across different cancer\\n\\n693 types. HER2 and 3 play also essential roles in the development of brain metastasis\\n\\n694 which causes death of more than 30% of stage IV HER2+ breast cancer patients\\n\\n695 (Berghoff et al, 2014; Da Silva et al., 2010; Fecci et al., 2019). To this end, DUSP6\\n696 mediated regulation of HER2 and 3 suggested that its inhibition may retard\\n\\n697 development of brain metastasis. Indeed, our findings directly support these\\n\\n698 conclusions as _DUSP6 knockout in tumor cells improves survival of mice with_\\n\\n699 intracranial HER2+ tumors and the outgrowth of HER2+ breast tumor cells in a\\n\\n700 zebrafish intracranial model. Linking our results further to future therapy of HER2+\\n\\n701 brain metastasis, we showed that _DUSP6 knockdown increases sensitivity to_\\n\\n702 tucatinib+trastuzumab+capecitabine combination regimen, which show significant\\n\\n703 clinical activity in HER2+ breast cancer patients with brain metastasis (Murthy et al.,\\n\\n704 2020).\\n\\n705\\n\\n706 While demonstrated here for the first time in the context of HER2i therapy resistance,\\n\\n707 genetic DUSP6 inhibition has recently been shown to inhibit malignant phenotypes in\\n\\n708 other cancer types (Shojaee et al., 2015; Wu et al., 2018; Zandi et al., 2022).\\n\\n709 Consistent with these reports, genetic inhibition of _DUSP6 in our study resulted in_\\n\\n710 significant inhibition of HER2+ breast cancer cell viability, HER2i resistance, colony\\n\\n711 forming potential, and _in vivo tumor growth. Mechanistically, DUSP6 inhibition_\\n\\n\\n-----\\n\\n712 converted cytostatic response to HER2i to an apoptotic response, which is\\n\\n713 considered as a paramount for cancer therapy strategies aiming for cancer cure. On\\n\\n714 the other hand, overexpression of DUSP6 converted HER2i sensitive cells to\\n\\n715 resistant in both cell viability and apoptosis assays. Importantly, all our main\\n\\n716 conclusions remained valid regardless of whether DUSP6 was inhibited either by\\n\\n717 siRNA, CRISPR/CAS9, or by pharmacological inhibitors BCI and BCI-215. We\\n\\n718 further demonstrated nearly immediate target engagement, as well as resistance of\\n\\n719 three independent _DUSP6 knock-out clones to BCI-elicited cell viability inhibition._\\n\\n720 Therefore, together with recent results from the others (Kong et al., 2023; Shojaee et\\n\\n721 _al., 2015), we are confident that the results achieved by BCI and BCI-215 are_\\n\\n722 dependent on their inhibitory effects on DUSP6 despite a recent report questioned\\n\\n723 their selectivity using other cellular system (Thompson et al, 2022).\\n\\n724\\n\\n725 The results identified DUSP6 as a novel combination therapy target with existing\\n\\n726 clinical HER2 targeting strategies including both small molecule and antibody\\n\\n727 mediated HER2 inhibition and chemotherapy combinations. In that regard, our\\n\\n728 xenograft results demonstrate significant potential for pharmacological DUSP6\\n\\n729 inhibition in overcoming HER2i resistance _in vivo by using two different HER2i_\\n\\n730 resistant cell lines, and two different HER2is. Whereas neither HER2 nor DUSP6\\n\\n731 inhibition alone did not have therapeutic effect on tumors, the combination showed\\n\\n732 very potent synthetic lethal phenotype further validating our in vitro results. The lack\\n\\n733 of monotherapy effect of BCI, combined with clear evidence for downstream target\\n\\n734 mechanism engagement _in vivo also alleviates concerns about overall cellular_\\n\\n735 toxicity behind the BCI-elicited HER2i synergy. This is consistent with previous\\n\\n736 findings demonstrating antitumor effects with BCI and BCI-215 without obvious\\n\\n\\n-----\\n\\n737 systemic toxicity in _in vivo models of gastric cancer, leukemia, and malignant_\\n\\n738 peripheral nerve sheath tumor (Kesarwani et al., 2017; Ramkissoon et al., 2019;\\n\\n739 Shojaee et al., 2015; Wu et al., 2018).\\n\\n740\\n\\n741 In summary, we provide first transcriptional map of DTP-DTEP transition under TKI\\n\\n742 tolerance development in cancer. The results specifically identify DUSP6 targeting\\n\\n743 as a novel approach to target HER3-mediated ERBB TKI resistance. Ultimately, the\\n\\n744 work provides proof-of-principle evidence to encourage development of next\\n\\n745 generation DUSP6 inhibitors (with brain penetrance) to test the clinical relevance of\\n\\n746 the presented therapy scenarios.\\n\\n747\\n\\n\\n-----\\n\\n748 **Materials and Methods**\\n\\n749\\n\\n750 **Reagents**\\n\\n751 Tissue culture reagents including regular RPMI, DMEM, RPMI and FBS were\\n\\n752 purchased from Sigma. The recombinant NRG, HGF and β-estradiol (E2) were from\\n\\n753 Peprotech. BCI was purchased from Axon Medchem. BCI-215 was provided by Dr.\\n\\n754 Andreas Vogt, University of Pittsburgh Drug Discovery Institute, Pittsburgh, PA,\\n\\n755 USA. The HER2 inhibitors and the other compounds used in the drug screening\\n\\n756 were purchased from Adooq Bioscience. All the agents were dissolved in DMSO and\\n\\n757 the final concentration of DMSO did not exceed 0.1% [v/v] in all the treatments.\\n\\n758 **Cloning and plasmids**\\n\\n759 pBABE-puro-gateway-ERBB2 was a gift from Matthew Meyerson (Addgene plasmid\\n\\n760 No. 40978; http://n2t.net/addgene:40978; RRID:Addgene_40978). ERBB3 wild-type\\n\\n761 was cloned from pBABE-puro-gateway-ERBB3 (Koivu et al, 2021) into pLenti CMV\\n\\n762 Puro DEST (w118-1), a gift from Eric Campeau and Paul Kaufman (Addgene\\n\\n763 plasmid # 17452;http://n2t.net/addgene:17452 ; RRID:Addgene_17452) through\\n\\n764 Gateway cloning (Chakroborty et al, 2019) to create pLenti-CMV-Puro-ERBB3. The\\n\\n765 DUSP6 plasmid was purchased from Addgene (#27975) and the Kinase interaction\\n\\n766 motif (KIM) mutants were prepared by site directed mutagenesis (GenScript Inc)\\n\\n767 generating R64A, R65A double mutant defective in ERK binding as described\\n\\n768 (Nichols et al., 2000).\\n\\n769 **Generation of stable lines**\\n\\n770 To overexpress wild-type ERBB2, pBABE-puro-gateway-ERBB2 was transfected\\n\\n771 (using Fugene6 transfection reagent; Promega Catalog # E2692) into amphotropic\\n\\n772 Phoenix HEK293T cells (a gift from Dr. Garry Nolan) to generate retroviruses, which\\n\\n773 were used to transduce MDA-MB-453, as described previously (Chakroborty et al.,\\n\\n\\n-----\\n\\n774 2019). pLenti-CMV-Puro-ERBB3 was co-transfected with virus-packaging plasmids\\n\\n775 pMLDg/pRRE (addgene #12251), pMD2.G (addgene #12259), and pRSV-Rev\\n\\n776 (addgene #12253) into HEK293T cells using Fugene6 transfection reagent to\\n\\n777 produce lentiviruses. The lentivirus-containing supernatant was used to transduce\\n\\n778 MDA-MB-453 to over-express wild-type ERBB3. After viral transduction cells were\\n\\n779 treated with 1 µg/mL puromycin (Gibco) for 48 h to select the cells with stable\\n\\n780 expression of the respective introduced transgenes.\\n\\n781 For DUSP1 and 6 overexpression, lentiviral particles containing full length of either\\n\\n782 DUSP1 (Genecopoeia), DUSP6 (Addgene #27975), or control empty (Genecopoeia)\\n\\n783 vector were generated in HEK293FT packaging cell line (complete medium: high\\n\\n784 glucose DMEM, 10% FBS, 0,1mM NEAA, 1mM MEM Sodium Pyruvate, 6mM L\\n785 Glutamine, 1% Pen/Strep and 0,5mg/ml Geneticin) by transient transfection of\\n\\n786 transfer vector 2[nd] generation packaging plasmid-psPAX2 (Addgene #12259) and\\n\\n787 envelope vector-pMD2 (Addgene #12260) with the ratio (7:2:1) using calcium\\n788 phosphate precipitation method. Seventy-two hours post-transfection medium\\n\\n789 containing viral vectors was collected, concentrated for 2 h by ultracentrifugation in\\n\\n790 swing-out rotor SW-32Ti (Beckman Coulter), 26,000g, resuspended in residual\\n\\n791 medium and flash-frozen in liquid nitrogen. Functional titer  ̴1x10[8] was measured in\\n\\n792 HEK293FT cells and FACS (BD LSRFortessa, Becton Dickinson). To obtain stable\\n\\n793 overexpression of DUSP1, DUSP6 or double DUSP1+6 population on day zero,\\n\\n794 8x10[4] cells were seeded in a 24-well plate. 24 h later, the cells were transduced with\\n\\n795 MOI 60 of lentiviral stocks in a low volume of full media. Medium containing viral\\n\\n796 particles was removed 16 h later. Cells expressing DUSP1 and GFP indicative of\\n\\n797 lentiviral integration were collected by fluorescence assisted cell sorting (BD\\n\\n798 FACSaria II cell sorter, Becton Dickinson). FACS gating was set at 10% top high\\n\\n\\n-----\\n\\n799 fluorescence signal. DUSP6 transduced BT474 cells were selected with 3 µg/mL of\\n\\n800 puromycin. DUSP1 and 6 expressing cells were obtained by sequential transduction\\n\\n801 (MOI 2, 6, 10), puromycin selection and later GFP fluorescence assisted cell sorting\\n\\n802 (BD FACSaria II cell sorter, Becton Dickinson). The levels of protein were confirmed\\n\\n803 by Western blot analysis.\\n\\n804 For stable overexpression of DUSP6 and DUSP6 KIM mutant, DUSP6\\n\\n805 (Addgene #27975) and DUSP6[R64-65A ] lentiviral vectors were co-transfected with\\n\\n806 virus-packaging plasmids pMLDg/pRRE, (Addgene #12251), pMD2.G (Addgene\\n\\n807 #12259), and pRSV-Rev (Addgene #12253) into HEK293T cells using Fugene6\\n\\n808 transfection reagent to produce lentiviruses. 48 h after transfection viral supernatants\\n\\n809 were collected, filtered, and added to pre-plated BT474 cells along with 10 μg/mL\\n\\n810 polybrene. Transduced cells were selected with 2 μg/mL puromycin, and the protein\\n\\n811 levels were confirmed by Western blot analysis.\\n\\n812 To rescue the DUSP6 KO MDA-MB-453 clones, the pLEX307-hygro-DUSP6\\n\\n813 lentiviral vector was co-transfected with virus-packaging plasmids pMLDg/pRRE,\\n\\n814 (Addgene #12251), pMD2.G (Addgene #12259), and pRSV-Rev (Addgene #12253)\\n\\n815 into HEK293T cells using FuGENE6 transfection reagent. 48 hours after transfection\\n\\n816 viral supernatants were collected, filtered, and added to MDA-MB-453 DUSP6 KO\\n\\n817 cells (g1, clone 22) along with 10 μg/mL polybrene. Transduced cells were selected\\n\\n818 with hygromycin (250µg/ml).The MDA-MB-453 CAS9 control, DUSP6 KO, and\\n\\n819 DUSP6 transduced KO cells were seeded at low density and maintained for 10 d.\\n\\n820 The colonies were stained with 0.5% crystal violet in methanol and imaged using an\\n\\n821 Epson scanner. Colony quantification was done by using the ImageJ ColonyArea\\n\\n822 plugin.\\n\\n823\\n\\n\\n-----\\n\\n824 **FACS**\\n\\n825 MDA-MB-453 cells transduced with lentiviruses encoding wild-type ERBB3 were\\n\\n826 washed with azide-free PBS, trypsinized and suspended in ice-cold sorting buffer\\n\\n827 (PBS +1% Goat serum, Life technologies catalog # PCN5000). The cells were\\n\\n828 incubated with anti-ERBB3 (MAB3481, R&D systems) for 1 h on ice, and with anti\\n829 mouse Alexa Fluor 405 (A-31553, Invitrogen) for 30 min on ice in a dark\\n\\n830 environment. Single cell suspension was analyzed and sorted on Sony SH800 Cell\\n\\n831 Sorter to select cell-pools with high surface ERBB3 expression.\\n\\n832 **CRISPR/CAS9 knockout system**\\n\\n833 A two-component CRISPR system was used to generate _DUSP6 KO cells (Adli,_\\n\\n834 2018). DUSP6 sgRNAs (seq#1- CATCGAGTCGGCCATCAACG, seq#2\\n835 GACTGGAACGAGAATACGGG, seq#3- CCATGATAGATACGCTCAGA) were\\n\\n836 selected using DeskGEN platform and cloned according to F. Zhang lab protocol.\\n\\n837 Separate lentivectors containing spCas9 (lentiCas9-Blast a gift from Feng Zhang\\n\\n838 (Addgene plasmid # 52962) and sgRNA (lentiGuide-Puro a gift from Feng Zhang\\n\\n839 (Addgene plasmid # 52963) were produced in HEK293FT packaging cell line by\\n\\n840 transient cotransfection. Shortly, 40-70% confluent HEK293FT cells were used for\\n\\n841 transfections with 14 μg of transfer vector, 4 μg of packaging vector psPAX2 (gift\\n\\n842 from Didier Trono (Addgene plasmid # 12260), 2 μg envelope vector pMD2.G (gift\\n\\n843 from Didier Trono (Addgene plasmid # 12259) mixed in 0.45 mL water, 2.5M CaCl2,\\n\\n844 and 2x HeBS (274 mM NaCl, 10 mM KCl, 1.4 mM Na2HPO4, 15 mM D-glucose, 42\\n\\n845 mM Hepes, pH 7.06) per 10 cm dish. Before adding to the cells, the DNA-HeBS mix\\n\\n846 was incubated for 30 min at room temperature. After overnight incubation medium\\n\\n847 with DNA precipitate was gently removed from the cells and replaced with a full fresh\\n\\n848 medium. Media containing viral particles was collected after 72 h, spun at 300 rpm\\n\\n\\n-----\\n\\n849 for 5 min at room temperature to remove cell debris, filtered through 0.45 µm PES\\n\\n850 filter, and concentrated by ultracentrifugation for 2 hours at 25,000rpm, 4°C\\n\\n851 (Beckman Coulter). The pellet containing lentiviral particles was suspended in the\\n\\n852 residual medium, incubated for ~2 hours in +4˚C with occasional mild vortex,\\n\\n853 aliquoted, snap-frozen and stored in -70˚C. P24 ELISA measured physical lentiviral\\n\\n854 titer with a serial dilution of virus stock according to manufacturer protocol.\\n\\n855 To generate DUSP6KO clones in MDA-MB-453, 1e+05 cells were seeded on a 24\\n856 well plate. The next day the cells were transduced with Lenti-Cas9 (MOI 1, 5, 10),\\n\\n857 and 72 h later, 8 μg/mL of Blasticidin was applied to select only Cas9 expressing\\n\\n858 cells. Cells transduced with the smallest number of Lenti-Cas9 particles that survived\\n\\n859 after the parallel control well was cleared proceeded to the next step. In the second\\n\\n860 stage, the mixed pool of stably expressing Cas9 cells was transduced with Lenti\\n\\n861 sgRNA vectors (MOI 5,10,15,20), and 72 h later, 1 μg/uL Puromycin was applied on\\n\\n862 the cells to select double-positive Cas9+/Lenti sgRNA+ cells. Based on Western blot\\n\\n863 results, cell populations showing the highest reduction in DUSP6 protein levels were\\n\\n864 single sorted (Sony SH800 cell sorter, Sony Biotechnology Inc) and re-grown into a\\n\\n865 clonal cell population. On average, about 20 clones per sgRNA population were\\n\\n866 screened using Western blotting and qPCR. Sanger sequencing was used to confirm\\n\\n867 full knockout status.\\n\\n868 **Cell Culture and Transfections**\\n\\n869 All cell lines were purchased from the American Type Culture Collection (ATCC) and\\n\\n870 were maintained at 37 °C and 5% CO2 in a humidified incubator and cultured\\n\\n871 according to the ATCC recommendations. Cell line authentication was performed by\\n\\n872 STR profiling and using Cell ID[TM] system (Promega). The cultures were routinely\\n\\n873 tested for mycoplasma contamination. The information about siRNAs for _DUSP1,_\\n\\n\\n-----\\n\\n874 _DUSP6,_ _HER2,_ _HER3,_ _AKT1 and negative control siRNAs are in Table EV1._\\n\\n875 Transient transfections were performed with lipofectamine RNAiMAX reagent\\n\\n876 (ThermoFisher) according to the manufacturer's instructions.\\n\\n877 **Cell growth assays**\\n\\n878 The cells were seeded at a density of 2 x 10[3] into 96-well plates and treated with\\n\\n879 increasing concentrations of the drugs for 48 h. Cell viability was determined using\\n\\n880 WST-1 assay (Sigma). Vehicle-treated cells were used as the control group. For the\\n\\n881 clonogenic survival assay, cells were seeded in 6-well plates at a density of 1000\\n\\n882 cells/well. After 24 h, the media was changed, and the cells were maintained for\\n\\n883 another 10 d. The resulting colonies were stained/fixed with 0.5% crystal violet\\n\\n884 imaged using an inverted microscope.\\n\\n885 To yield DTPs in NSCLC, melanoma and colorectal cancer cell lines, the following\\n\\n886 treatments were carried out: NSCLC cell line HCC827, melanoma cell line A375, and\\n\\n887 colorectal cancer HT-29 cell line were cultured under standard conditions. The\\n\\n888 following treatments were applied for 10 d: HCC827, 1 µM osimertinib; A375, 1 µM\\n\\n889 dabrafenib+100 nM trametinib; HT-29, 1 µM dabrafenib+10 µg/ml cetuximab.\\n\\n890 **Drug combination analysis**\\n\\n891 To explore the efficacy of drug combinations, growth inhibition was determined by\\n\\n892 WST-1 assay and the results were evaluated by Bliss SynergyFinder (Ianevski et al,\\n\\n893 2017). Visualization of synergy scores is depicted as a synergy map. An average\\n\\n894 synergy score of 0 is considered additive, less than 0 as antagonistic and over 0 as a\\n\\n895 synergistic outcome.\\n\\n896 **Caspase 3/7 activity assay**\\n\\n897 The Caspase-Glo® 3/7 Assay System (Promega), a luminescence-based assay for\\n\\n898 detection of active caspase-3 and 7, was used to quantitatively determine apoptotic\\n\\n\\n-----\\n\\n899 cell death. Following caspase cleavage of the proluciferin DEVD substrate, a\\n\\n900 substrate for luciferase is released and results in luciferase reaction and production\\n\\n901 of light.\\n\\n902 **Western blot analysis**\\n\\n903 The cells were lysed for 30 min in ice-cold RIPA buffer (50 mM Tris-HCl, pH 8.0, 150\\n\\n904 mM NaCl, 1.0% NP-40, 0.5% sodium deoxycholate and 0.1% SDS) containing\\n\\n905 protease and phosphatase inhibitors (ThermoFisher). The assay was performed with\\n\\n906 primary and Licor secondary antibodies (Licor Biosciences). β-actin was used as the\\n\\n907 loading control. The list of antibodies is in Table EV2.\\n\\n908 **Analysis of gene expression by quantitative reverse transcription-PCR**\\n\\n909 Quantitative reverse transcription-PCR (qRT-PCR) analysis was done on a\\n\\n910 QuantStudio Real-Time PCR instrument (ThermoFisher) using PowerUp™ SYBR®\\n\\n911 Green Master Mix (ThermoFisher). The primer sequences are listed in Table EV3.\\n\\n912 The target gene expression levels were normalized to beta-2-microglobulin (B2M)\\n\\n913 levels. For calculations, 2 [–ΔΔCT] formula was used, with ΔΔCT = (CTTarget – CTB2M)\\n\\n914 experimental sample – (CTTarget – CTB2M) control samples, where CT is the cycle\\n\\n915 threshold.\\n\\n916 **Animal studies**\\n\\n917 The animal experiments were performed according to the Animal Experiment Board\\n\\n918 in Finland (ELLA) for the care and use of animals under the licenses\\n\\n919 4161/04.10.07/2015 and 9241/2018. The animals were kept under pathogen-free\\n\\n920 conditions in individually ventilated cages in an animal care facility. Mice were kept\\n\\n921 on a 12 h light/dark cycle with access to autoclaved water and irradiated chow _ad_\\n\\n922 _libitum and were allowed to adapt to the facility for 1 week before starting the_\\n\\n923 experiments. For the subcutaneous experiments, MDA-MB-453 (5 x 10[6]) and\\n\\n\\n-----\\n\\n924 HCC1954 (3 x 10[6]) cells were injected into the right flank of six- to eight-week-old\\n\\n925 BALB/cOlaHsd-Foxn1nu mice (Envigo, France). Mice with tumor size ∼100 mm[3]\\n\\n926 were randomized into experimental and control groups. Tumor dimensions were\\n\\n927 measured with Vernier calipers and tumor volume were calculated as 1/2 larger\\n\\n928 diameter x (smaller diameter)[2]. For the intracranial model, 1 x 10[5] cells in 5 µL of\\n\\n929 PBS were inoculated into the brain of anaesthetized mice. The mice were then\\n\\n930 imaged with bioluminescence and based on the bioluminescence signal were\\n\\n931 randomized to experimental and control groups, as descried earlier (Merisaari et al,\\n\\n932 2020). Mice were euthanized when they became moribund when they reached\\n\\n933 defined study end points.\\n\\n934 **Zebrafish studies**\\n\\n935 Zebrafish embryo xenograft studies were performed under the license\\n\\n936 ESAVI/9339/04.10.07/2016 (National Animal Experimentation Board, Regional State\\n\\n937 Administrative Agency for Southern Finland). Briefly, the _DUSP6 knockdown GFP-_\\n\\n938 MDA-MB-361 cells were injected into the brain of zebrafish embryos from the dorsal\\n\\n939 side. One day after injection (1 dpi), successfully transplanted embryos were placed\\n\\n940 in CellView glass bottom 96-well plate (1 embryo/well) and embryos were incubated\\n\\n941 in E3 + PTU at 33 °C. The xenografted embryos were imaged using a Nikon Eclipse\\n\\n942 Ti2 fluorescence microscope and a 2x Nikon Plan-Apochromat (NA 0.06) objective.\\n\\n943 Each embryo was imaged at 1dpi and 4dpi using brightfield illumination and a GFP\\n\\n944 fluorescence filter set (excitation with 470nm LED). Each image was inspected\\n\\n945 manually to filter out severely malformed, dead or out of focus embryos. The tumor\\n\\n946 area was measured using ImageJ (NIH). The fold change in tumor size was\\n\\n947 calculated as follows: GFP intensity (4dpi)/GFP intensity (1dpi)\\n\\n948 **RNA-sequencing**\\n\\n\\n-----\\n\\n949 RNA-sequencing was conducted at the Finnish Functional Genomic Center, The\\n\\n950 University of Turku, Finland. RNA was harvested using the NucleoSpin RNA\\n\\n951 purification kit (Macherey‐Nagel), followed by treatment with DNase to remove\\n\\n952 genomic DNA. RNA (300 ng) was reverse transcribed using the Illumina TruSeq\\n\\n953 Stranded Total mRNA kit. The quality of the samples was ensured using Agilent\\n\\n954 Bioanalyzer 2100 or Advanced Analytical Fragment Analyzer. Sample concentration\\n\\n955 was measured with Qubit® Fluorometric Quantitation (Life Technologies) and/or\\n\\n956 KAPA Library Quantification kit for Illumina platform, KAPA Biosystems. Sequencing\\n\\n957 run was performed using the Illumina NovaSeq 6000 instrument. Genomic alignment\\n\\n958 was performed using Rsubread v. 2.0.0 and the reads were mapped to the human\\n\\n959 reference genome hg38. Aligned reads were assigned to RefSeq gene models using\\n\\n960 the same R package with its default settings.\\n\\n961 Differentially expressed genes and pathways were identified using the R package\\n\\n962 limma. Latest hallmark gene sets were downloaded from the Molecular Signatures\\n\\n963 Database version 7.4 (http://www.gsea-msigdb.org/gsea/msigdb/index.jsp) and used\\n\\n964 within Gene Set Variation Analysis (GSVA) allowing pathway enrichment estimates\\n\\n965 for each sample (Hanzelmann et al, 2013). Data was transformed using log(x+1)\\n\\n966 after normalization and the RNA-sequencing pipeline run. Heatmaps were plotted\\n\\n967 using the ComplexHeatmap package (Gu et al, 2016). R statistical software version\\n\\n968 4.0.3 was used for the statistical analyses and visualizations.\\n\\n969 **cBioPortal database analyses**\\n\\n970 The correlation analysis between gene expression and breast cancer subtypes was\\n\\n971 examined using the METABRIC breast cancer cohort with PAM50 classification\\n\\n972 (Pereira et al., 2016), TCGA breast invasive carcinoma dataset and the TCGA\\n\\n973 Firehose legacy dataset (Gao et al., 2013). The RNA-seq data from breast invasive\\n\\n\\n-----\\n\\n974 carcinoma samples and the relevant clinical information are available at the TCGA\\n\\n975 data portal, cBioPortal for Cancer Genomics (http://www.cbioportal.org/). According\\n\\n976 to the PAM50 classification, METABRIC breast cancer patients were divided into 5\\n\\n977 subtypes, including the basal (n=199), HER2+(n=220), Lum A (n=679), Lum B\\n\\n978 (n=461) and Normal-like (n=140). ER and HER2 status were assessed using the\\n\\n979 patient’s IHC information. The survival outcomes were extracted from the TCGA\\n\\n980 breast invasive carcinoma dataset.\\n\\n981 For multivariable Cox regression analyses, N=121 HER2+ patients from the TCGA\\n982 BRCA cohort were extracted, with reported pathological T-stages and lymph node\\n\\n983 statuses included as clinical factors and relapse free survival modelled as the end\\n984 point. Further, transcriptomics was included as log2-transformed values for RSEM\\n\\n985 normalized expression of DUSP6.\\n\\n986 **Transcription factor binding analyses**\\n\\n987 To predict transcription factors enriched in the significantly differentially regulated\\n\\n988 genes during the therapy tolerance transitions we used ChEA3: transcription factor\\n\\n989 enrichment analysis by orthogonal omics integration tool (Keenan et al, 2019).\\n\\n990 Differentially regulated genes were provided as input to the tool and the significantly\\n\\n991 associated transcription factors were obtained using ENCODE (Encyclopedia of\\n\\n992 DNA Elements) ChIP-seq library. The output file consists of a list of transcription\\n\\n993 factors arranged based on a scaled rank where a lower value means a higher\\n\\n994 significance in the gene list. An FDR value < 0.05 was considered significant.\\n\\n995 To predict the transcription factors binding the DUSP6 promotor region, Cistrome DB\\n\\n996 Toolkit was used (Zheng et al, 2019). As an input 4000 base pairs upstream of the\\n\\n997 DUSP6 gene were entered in the Cistrome DB Toolkit. This provided a list of\\n\\n\\n-----\\n\\n998 transcription factors based on the regulatory potential (RP score) derived by\\n\\n999 comparing the CHIP-seq data sets.\\n\\n1000 **Statistical analysis**\\n\\n1001 Differentially expressed genes and pathways were identified using the R package\\n\\n1002 limma (Ritchie et al, 2015). Latest hallmark gene sets were downloaded from the\\n\\n1003 Molecular Signatures Database version 7.4 (http://www.gsea\\n1004 msigdb.org/gsea/msigdb/index.jsp) and used within Gene Set Variation Analysis\\n\\n1005 (GSVA) allowing pathway enrichment estimates for each sample (Hanzelmann et al.,\\n\\n1006 2013), with FDR-cutoff <0.25 used for the pathway enrichment analyses. Data was\\n\\n1007 transformed using log(x+1) after normalization and the RNA-sequencing pipeline\\n\\n1008 run. Heatmaps were plotted using the ComplexHeatmap package (Gu et al., 2016).\\n\\n1009 R statistical software (R Core Team (2023). R: A language and environment for\\n\\n1010 statistical computing. R Foundation for Statistical Computing, Vienna, Austria. URL:\\n\\n1011 https://www.R-project.org) version 4.0.3 was used for the statistical analyses and\\n\\n1012 visualizations.\\n\\n1013 All data were evaluated in triplicate against the vehicle-treated control cells and\\n\\n1014 collected from three independent experiments. In addition to R analyses, data were\\n\\n1015 visualized and analyzed using GraphPad Prism 8.3.0 using one-way ANOVA and the\\n\\n1016 unpaired two-tailed student’s t test. All such data are presented as mean ± standard\\n\\n1017 deviation (SD).\\n\\n1018 **Data Availability**\\n\\n1019 The gene expression data from this publication have been deposited to the GEO\\n\\n1020 database (https://www.ncbi.nlm.nih.gov/geo/) and assigned the identifier\\n\\n1021 GSE231526 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE231526).\\n\\n1022\\n\\n\\n-----\\n\\n1023 **Disclosure and competing interests statement**\\n\\n1024\\n\\n1025 The authors declare no competing interests.\\n\\n1026\\n\\n1027\\n\\n1028 **Author contributions**\\n\\n1029 **Majid Momeny: Conceptualization; Validation; Supervision; Visualization; Writing;**\\n\\n1030 Investigation; Formal analysis; Methodology. Mari Tienhaara: Validation;\\n\\n1031 Investigation; Methodology. Mukund Sharma: Validation; Investigation;\\n\\n1032 Methodology. Deepankar Chakroborty: Methodology. Roosa Varjus: Formal\\n\\n1033 analysis. Iina Takala: Methodology. Joni Merisaari: Methodology. Artur Padzik:\\n\\n1034 Methodology; Validation. Andreas Vogt: Methodology. Ilkka Paatero: Methodology;\\n\\n1035 Validation. Klaus Elenius: Supervision; Methodology. Teemu D. Laajala:\\n\\n1036 Methodology; Formal analysis. Kari J. Kurppa: Supervision; Investigation;\\n\\n1037 Methodology. Jukka Westermarck: Conceptualization; Supervision; Validation;\\n\\n1038 Project administration; Writing-review and editing.\\n\\n1039 **Acknowledgement**\\n\\n1040 Taina Kalevo-Mattila is acknowledged for superior technical support and the entire\\n\\n1041 Turku Bioscience Centre personnel is thanked for excellent working environment.\\n\\n1042 We acknowledge Zhong-Yin Zhang for PRL1i compound, Norma O'Donovan,\\n\\n1043 National Institute for Cellular Biotechnology, Dublin City University, Ireland, for some\\n\\n1044 HER2+ breast cancer cell lines used in this study, and Dihua Yu, The University of\\n\\n1045 Texas MD Anderson Cancer Center, USA, for the BT474Br cells. We further\\n\\n1046 acknowledge important contributions of the following core facilities of Turku\\n\\n1047 Bioscience Centre (University of Turku and Åbo Akademi University) supported by\\n\\n1048 Biocentre Finland: Finnish Functional Genomics Center, Screening unit, and\\n\\n1049 Zebrafish unit. This study was supported by funding from Finnish Cancer\\n\\n\\n-----\\n\\n1050 Associations (JW), Foundation of Finnish Cancer Institute (MM), Maud Kuistila\\n\\n1051 Foundation (MM), Turku University Foundation (MM), Finnish Cancer Institute (TDL),\\n\\n1052 Finnish Cultural Foundation (TDL), and Finnish Cultural Foundation (KJK).\\n\\n1053 **References:**\\n\\n1054 Adli M (2018) The CRISPR tool kit for genome editing and beyond. Nat Commun 9:\\n\\n1055 1911\\n\\n1056 Aguirre-Ghiso JA (2021) Translating the Science of Cancer Dormancy to the Clinic.\\n\\n1057 _Cancer Res 81: 4673-4675_\\n\\n1058 Arteaga CL, Engelman JA (2014) ERBB receptors: from oncogene discovery to\\n\\n1059 basic science to mechanism-based cancer therapeutics. Cancer Cell 25: 282-303\\n\\n1060 Berghoff AS, Bartsch R, Preusser M, Ricken G, Steger GG, Bago-Horvath Z, Rudas\\n\\n1061 M, Streubel B, Dubsky P, Gnant M et al (2014) Co-overexpression of HER2/HER3 is\\n\\n1062 a predictor of impaired survival in breast cancer patients. Breast 23: 637-643\\n\\n1063 Berns K, Horlings HM, Hennessy BT, Madiredjo M, Hijmans EM, Beelen K, Linn SC,\\n\\n1064 Gonzalez-Angulo AM, Stemke-Hale K, Hauptmann M et al (2007) A functional\\n\\n1065 genetic approach identifies the PI3K pathway as a major determinant of trastuzumab\\n\\n1066 resistance in breast cancer. Cancer Cell 12: 395-402\\n\\n1067 [DATASET] Cerami E, Gao J, Dogrusoz U, Gross BE, Sumer SO, Aksoy BA,\\n\\n1068 Jacobsen A, Byrne CJ, Heuer ML, Larsson E et al (2012) The cBio cancer genomics\\n\\n1069 portal: an open platform for exploring multidimensional cancer genomics data.\\n\\n1070 _Cancer Discov 2: 401-404_\\n\\n1071 Chakroborty D, Kurppa KJ, Paatero I, Ojala VK, Koivu M, Tamirat MZ, Koivunen JP,\\n\\n1072 Janne PA, Johnson MS, Elo LL et al (2019) An unbiased in vitro screen for activating\\n\\n1073 epidermal growth factor receptor mutations. J Biol Chem 294: 9377-9389\\n\\n\\n-----\\n\\n1074 Chan LN, Murakami MA, Robinson ME, Caeser R, Sadras T, Lee J, Cosgun KN,\\n\\n1075 Kume K, Khairnar V, Xiao G et al (2020) Signalling input from divergent pathways\\n\\n1076 subverts B cell transformation. Nature 583: 845-851\\n\\n1077 Chandarlapaty S, Sawai A, Scaltriti M, Rodrik-Outmezguine V, Grbovic-Huezo O,\\n\\n1078 Serra V, Majumder PK, Baselga J, Rosen N (2011) AKT inhibition relieves feedback\\n\\n1079 suppression of receptor tyrosine kinase expression and activity. Cancer Cell 19: 58\\n1080 71\\n\\n1081 [DATASET] Chang CA, Jen J, Jiang S, Sayad A, Mer AS, Brown KR, Nixon AML,\\n\\n1082 Dhabaria A, Tang KH, Venet D et al (2022) Ontogeny and Vulnerabilities of Drug\\n1083 Tolerant Persisters in HER2+ Breast Cancer. _Cancer discovery 12: 1022-1045._\\n\\n1084 https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE155342\\n\\n1085 Cleary JM, McRee AJ, Shapiro GI, Tolaney SM, O'Neil BH, Kearns JD, Mathews S,\\n\\n1086 Nering R, MacBeath G, Czibere A et al (2017) A phase 1 study combining the HER3\\n\\n1087 antibody seribantumab (MM-121) and cetuximab with and without irinotecan. _Invest_\\n\\n1088 _New Drugs 35: 68-78_\\n\\n1089 Da Silva L, Simpson PT, Smart CE, Cocciardi S, Waddell N, Lane A, Morrison BJ,\\n\\n1090 Vargas AC, Healey S, Beesley J et al (2010) HER3 and downstream pathways are\\n\\n1091 involved in colonization of brain metastases from breast cancer. Breast Cancer Res\\n\\n1092 12: R46\\n\\n1093 [DATASET] de Bruijn I, Kundra R, Mastrogiacomo B, Tran TN, Sikina L, Mazor T, Li\\n\\n1094 X, Ochoa A, Zhao G, Lai B et al (2023) Analysis and Visualization of Longitudinal\\n\\n1095 Genomic and Clinical Data from the AACR Project GENIE Biopharma Collaborative\\n\\n1096 in cBioPortal. Cancer Res 83: 3861-3867\\n\\n1097 De Conti G, Dias MH, Bernards R (2021) Fighting Drug Resistance through the\\n\\n1098 Targeting of Drug-Tolerant Persister Cells. Cancers (Basel) 13\\n\\n\\n-----\\n\\n1099 Dhimolea E, de Matos Simoes R, Kansara D, Al'Khafaji A, Bouyssou J, Weng X,\\n\\n1100 Sharma S, Raja J, Awate P, Shirasaki R et al (2021) An Embryonic Diapause-like\\n\\n1101 Adaptation with Suppressed Myc Activity Enables Tumor Treatment Persistence.\\n\\n1102 _Cancer Cell 39: 240-256 e211_\\n\\n1103 Erjala K, Sundvall M, Junttila TT, Zhang N, Savisalo M, Mali P, Kulmala J, Pulkkinen\\n\\n1104 J, Grenman R, Elenius K (2006) Signaling via ErbB2 and ErbB3 associates with\\n\\n1105 resistance and epidermal growth factor receptor (EGFR) amplification with sensitivity\\n\\n1106 to EGFR inhibitor gefitinib in head and neck squamous cell carcinoma cells. _Clin_\\n\\n1107 _Cancer Res 12: 4103-4111_\\n\\n1108 Fecci PE, Champion CD, Hoj J, McKernan CM, Goodwin CR, Kirkpatrick JP, Anders\\n\\n1109 CK, Pendergast AM, Sampson JH (2019) The Evolving Modern Management of\\n\\n1110 Brain Metastasis. Clin Cancer Res 25: 6570-6580\\n\\n1111 Futschik ME, Carlisle B (2005) Noise-robust soft clustering of gene expression time\\n1112 course data. J Bioinform Comput Biol 3: 965-988\\n\\n1113 Gaborit N, Abdul-Hai A, Mancini M, Lindzen M, Lavi S, Leitner O, Mounier L,\\n\\n1114 Chentouf M, Dunoyer S, Ghosh M et al (2015) Examination of HER3 targeting in\\n\\n1115 cancer using monoclonal antibodies. Proc Natl Acad Sci U S A 112: 839-844\\n\\n1116 [DATASET] Gao J, Aksoy BA, Dogrusoz U, Dresdner G, Gross B, Sumer SO, Sun Y,\\n\\n1117 Jacobsen A, Sinha R, Larsson E et al (2013) Integrative analysis of complex cancer\\n\\n1118 genomics and clinical profiles using the cBioPortal. Sci Signal 6: pl1\\n\\n1119 Goutsouliak K, Veeraraghavan J, Sethunath V, De Angelis C, Osborne CK, Rimawi\\n\\n1120 MF, Schiff R (2020) Towards personalized treatment for early stage HER2-positive\\n\\n1121 breast cancer. Nat Rev Clin Oncol 17: 233-250\\n\\n1122 Gu Z, Eils R, Schlesner M (2016) Complex heatmaps reveal patterns and\\n\\n1123 correlations in multidimensional genomic data. Bioinformatics 32: 2847-2849\\n\\n\\n-----\\n\\n1124 Haikala HM, Janne PA (2021) Thirty Years of HER3: From Basic Biology to\\n\\n1125 Therapeutic Interventions. Clin Cancer Res\\n\\n1126 Hanzelmann S, Castelo R, Guinney J (2013) GSVA: gene set variation analysis for\\n\\n1127 microarray and RNA-seq data. BMC Bioinformatics 14: 7\\n\\n1128 Hata AN, Niederst MJ, Archibald HL, Gomez-Caraballo M, Siddiqui FM, Mulvey HE,\\n\\n1129 Maruvka YE, Ji F, Bhang HE, Krishnamurthy Radhakrishna V et al (2016) Tumor\\n\\n1130 cells can follow distinct evolutionary paths to become resistant to epidermal growth\\n\\n1131 factor receptor inhibition. Nat Med 22: 262-269\\n\\n1132 Hudis C, Swanton C, Janjigian YY, Lee R, Sutherland S, Lehman R, Chandarlapaty\\n\\n1133 S, Hamilton N, Gajria D, Knowles J et al (2013) A phase 1 study evaluating the\\n\\n1134 combination of an allosteric AKT inhibitor (MK-2206) and trastuzumab in patients\\n\\n1135 with HER2-positive solid tumors. Breast Cancer Res 15: R110\\n\\n1136 Ianevski A, He L, Aittokallio T, Tang J (2017) SynergyFinder: a web application for\\n\\n1137 analyzing drug combination dose–response matrix data. _Bioinformatics 33: 2413-_\\n\\n1138 2415\\n\\n1139 Kaltenmeier CT, Vollmer LL, Vernetti LA, Caprio L, Davis K, Korotchenko VN, Day\\n\\n1140 BW, Tsang M, Hulkower KI, Lotze MT et al (2017) A Tumor Cell-Selective Inhibitor of\\n\\n1141 Mitogen-Activated Protein Kinase Phosphatases Sensitizes Breast Cancer Cells to\\n\\n1142 Lymphokine-Activated Killer Cell Activity. J Pharmacol Exp Ther 361: 39-50\\n\\n1143 Keenan AB, Torre D, Lachmann A, Leong AK, Wojciechowicz ML, Utti V, Jagodnik\\n\\n1144 KM, Kropiwnicki E, Wang Z, Ma'ayan A (2019) ChEA3: transcription factor\\n\\n1145 enrichment analysis by orthogonal omics integration. _Nucleic Acids Res 47: W212-_\\n\\n1146 W224\\n\\n1147 Kesarwani M, Kincaid Z, Gomaa A, Huber E, Rohrabaugh S, Siddiqui Z, Bouso MF,\\n\\n1148 Latif T, Xu M, Komurov K et al (2017) Targeting c-FOS and DUSP1 abrogates\\n\\n\\n-----\\n\\n1149 intrinsic resistance to tyrosine-kinase inhibitor therapy in BCR-ABL-induced\\n\\n1150 leukemia. Nat Med 23: 472-482\\n\\n1151 Knuefermann C, Lu Y, Liu B, Jin W, Liang K, Wu L, Schmidt M, Mills GB,\\n\\n1152 Mendelsohn J, Fan Z (2003) HER2/PI-3K/Akt activation leads to a multidrug\\n\\n1153 resistance in human breast adenocarcinoma cells. Oncogene 22: 3205-3212\\n\\n1154 Kodack DP, Askoxylakis V, Ferraro GB, Sheng Q, Badeaux M, Goel S, Qi X,\\n\\n1155 Shankaraiah R, Cao ZA, Ramjiawan RR et al (2017) The brain microenvironment\\n\\n1156 mediates resistance in luminal breast cancer to PI3K inhibition through HER3\\n\\n1157 activation. Sci Transl Med 9\\n\\n1158 Koivu MKA, Chakroborty D, Tamirat MZ, Johnson MS, Kurppa KJ, Elenius K (2021)\\n\\n1159 Identification of Predictive ERBB Mutations by Leveraging Publicly Available Cell\\n\\n1160 Line Databases. Mol Cancer Ther 20: 564-576\\n\\n1161 Kong T, Laranjeira ABA, Yang K, Fisher DAC, Yu L, Poittevin De La Fregonniere L,\\n\\n1162 Wang AZ, Ruzinova MB, Fowles JS, Fulbright MC et al (2023) DUSP6 mediates\\n\\n1163 resistance to JAK2 inhibition and drives leukemic progression. _Nat Cancer 4: 108-_\\n\\n1164 127\\n\\n1165 Korotchenko VN, Saydmohammed M, Vollmer LL, Bakan A, Sheetz K, Debiec KT,\\n\\n1166 Greene KA, Agliori CS, Bahar I, Day BW et al (2014) In vivo structure-activity\\n\\n1167 relationship studies support allosteric targeting of a dual specificity phosphatase.\\n\\n1168 _Chembiochem 15: 1436-1445_\\n\\n1169 Kurppa KJ, Liu Y, To C, Zhang T, Fan M, Vajdi A, Knelson EH, Xie Y, Lim K, Cejas P\\n\\n1170 _et al (2020) Treatment-Induced Tumor Dormancy through YAP-Mediated_\\n\\n1171 Transcriptional Reprogramming of the Apoptotic Pathway. Cancer Cell 37: 104-122\\n\\n1172 e112\\n\\n\\n-----\\n\\n1173 Lazo JS, McQueeney KE, Burnett JC, Wipf P, Sharlow ER (2018) Small molecule\\n\\n1174 targeting of PTPs in cancer. Int J Biochem Cell Biol 96: 171-181\\n\\n1175 Leung WY, Roxanis I, Sheldon H, Buffa FM, Li JL, Harris AL, Kong A (2015)\\n\\n1176 Combining lapatinib and pertuzumab to overcome lapatinib resistance due to NRG1\\n1177 mediated signalling in HER2-amplified breast cancer. Oncotarget 6: 5678-5694\\n\\n1178 Majewski IJ, Nuciforo P, Mittempergher L, Bosma AJ, Eidtmann H, Holmes E,\\n\\n1179 Sotiriou C, Fumagalli D, Jimenez J, Aura C et al (2015) PIK3CA mutations are\\n\\n1180 associated with decreased benefit to neoadjuvant human epidermal growth factor\\n\\n1181 receptor 2-targeted therapies in breast cancer. J Clin Oncol 33: 1334-1339\\n\\n1182 Marine JC, Dawson SJ, Dawson MA (2020) Non-genetic mechanisms of therapeutic\\n\\n1183 resistance in cancer. Nat Rev Cancer 20: 743-756\\n\\n1184 Marsolier J, Prompsy P, Durand A, Lyne AM, Landragin C, Trouchet A, Bento ST,\\n\\n1185 Eisele A, Foulon S, Baudre L et al (2022) H3K27me3 conditions chemotolerance in\\n\\n1186 triple-negative breast cancer. Nat Genet 54: 459-468\\n\\n1187 Merisaari J, Denisova OV, Doroszko M, Le Joncour V, Johansson P, Leenders WPJ,\\n\\n1188 Kastrinsky DB, Zaware N, Narla G, Laakkonen P et al (2020) Monotherapy efficacy\\n\\n1189 of blood-brain barrier permeable small molecule reactivators of protein phosphatase\\n\\n1190 2A in glioblastoma. Brain Commun 2: fcaa002\\n\\n1191 Molina G, Vogt A, Bakan A, Dai W, Queiroz de Oliveira P, Znosko W, Smithgall TE,\\n\\n1192 Bahar I, Lazo JS, Day BW et al (2009) Zebrafish chemical screening reveals an\\n\\n1193 inhibitor of Dusp6 that expands cardiac cell lineages. Nat Chem Biol 5: 680-687\\n\\n1194 Murthy RK, Loi S, Okines A, Paplomata E, Hamilton E, Hurvitz SA, Lin NU, Borges\\n\\n1195 V, Abramson V, Anders C et al (2020) Tucatinib, Trastuzumab, and Capecitabine for\\n\\n1196 HER2-Positive Metastatic Breast Cancer. N Engl J Med 382: 597-609\\n\\n\\n-----\\n\\n1197 Nagata Y, Lan KH, Zhou X, Tan M, Esteva FJ, Sahin AA, Klos KS, Li P, Monia BP,\\n\\n1198 Nguyen NT et al (2004) PTEN activation contributes to tumor inhibition by\\n\\n1199 trastuzumab, and loss of PTEN predicts trastuzumab resistance in patients. Cancer\\n\\n1200 _Cell 6: 117-127_\\n\\n1201 Nichols A, Camps M, Gillieron C, Chabert C, Brunet A, Wilsbacher J, Cobb M,\\n\\n1202 Pouyssegur J, Shaw JP, Arkinstall S (2000) Substrate recognition domains within\\n\\n1203 extracellular signal-regulated kinase mediate binding and catalytic activation of\\n\\n1204 mitogen-activated protein kinase phosphatase-3. J Biol Chem 275: 24613-24621\\n\\n1205 Patterson KI, Brummer T, O'Brien PM, Daly RJ (2009) Dual-specificity\\n\\n1206 phosphatases: critical regulators with diverse cellular targets. _Biochem J 418: 475-_\\n\\n1207 489\\n\\n1208 Pereira B, Chin SF, Rueda OM, Vollan HK, Provenzano E, Bardwell HA, Pugh M,\\n\\n1209 Jones L, Russell R, Sammut SJ et al (2016) The somatic mutation profiles of 2,433\\n\\n1210 breast cancers refines their genomic and transcriptomic landscapes. _Nature_\\n\\n1211 _communications 7: 11479_\\n\\n1212 Ramkissoon A, Chaney KE, Milewski D, Williams KB, Williams RL, Choi K, Miller A,\\n\\n1213 Kalin TV, Pressey JG, Szabo S et al (2019) Targeted Inhibition of the Dual\\n\\n1214 Specificity Phosphatases DUSP1 and DUSP6 Suppress MPNST Growth via JNK.\\n\\n1215 _Clin Cancer Res 25: 4117-4127_\\n\\n1216 Recondo G, Bahcall M, Spurr LF, Che J, Ricciuti B, Leonardi GC, Lo YC, Li YY,\\n\\n1217 Lamberti G, Nguyen T et al (2020) Molecular Mechanisms of Acquired Resistance to\\n\\n1218 MET Tyrosine Kinase Inhibitors in Patients with MET Exon 14-Mutant NSCLC. Clin\\n\\n1219 _Cancer Res 26: 2615-2625_\\n\\n\\n-----\\n\\n1220 Ritchie ME, Phipson B, Wu D, Hu Y, Law CW, Shi W, Smyth GK (2015) limma\\n\\n1221 powers differential expression analyses for RNA-sequencing and microarray studies.\\n\\n1222 _Nucleic Acids Res 43: e47_\\n\\n1223 Saddoughi SA, Gencer S, Peterson YK, Ward KE, Mukhopadhyay A, Oaks J,\\n\\n1224 Bielawski J, Szulc ZM, Thomas RJ, Selvam SP et al (2013) Sphingosine analogue\\n\\n1225 drug FTY720 targets I2PP2A/SET and mediates lung tumour suppression via\\n\\n1226 activation of PP2A-RIPK1-dependent necroptosis. EMBO Mol Med 5: 105-121\\n\\n1227 Saura C, Bendell J, Jerusalem G, Su S, Ru Q, De Buck S, Mills D, Ruquet S, Bosch\\n\\n1228 A, Urruticoechea A et al (2014) Phase Ib study of Buparlisib plus Trastuzumab in\\n\\n1229 patients with HER2-positive advanced or metastatic breast cancer that has\\n\\n1230 progressed on Trastuzumab-based therapy. Clin Cancer Res 20: 1935-1945\\n\\n1231 Saura C, Oliveira M, Feng YH, Dai MS, Chen SW, Hurvitz SA, Kim SB, Moy B,\\n\\n1232 Delaloge S, Gradishar W et al (2020) Neratinib Plus Capecitabine Versus Lapatinib\\n\\n1233 Plus Capecitabine in HER2-Positive Metastatic Breast Cancer Previously Treated\\n\\n1234 With 2 HER2-Directed Regimens: Phase III NALA Trial. J Clin Oncol 38: 3138-3149\\n\\n1235 Schneeweiss A, Park-Simon TW, Albanell J, Lassen U, Cortes J, Dieras V, May M,\\n\\n1236 Schindler C, Marme F, Cejalvo JM et al (2018) Phase Ib study evaluating safety and\\n\\n1237 clinical activity of the anti-HER3 antibody lumretuzumab combined with the anti\\n1238 HER2 antibody pertuzumab and paclitaxel in HER3-positive, HER2-low metastatic\\n\\n1239 breast cancer. Invest New Drugs 36: 848-859\\n\\n1240 Sharma SV, Lee DY, Li B, Quinlan MP, Takahashi F, Maheswaran S, McDermott U,\\n\\n1241 Azizian N, Zou L, Fischbach MA et al (2010) A chromatin-mediated reversible drug\\n1242 tolerant state in cancer cell subpopulations. Cell 141: 69-80\\n\\n1243 Shojaee S, Caeser R, Buchner M, Park E, Swaminathan S, Hurtz C, Geng H, Chan\\n\\n1244 LN, Klemm L, Hofmann WK et al (2015) Erk Negative Feedback Control Enables\\n\\n\\n-----\\n\\n1245 Pre-B Cell Transformation and Represents a Therapeutic Target in Acute\\n\\n1246 Lymphoblastic Leukemia. Cancer Cell 28: 114-128\\n\\n1247 Thompson EM, Patel V, Rajeeve V, Cutillas PR, Stoker AW (2022) The cytotoxic\\n\\n1248 action of BCI is not dependent on its stated DUSP1 or DUSP6 targets in\\n\\n1249 neuroblastoma cells. FEBS Open Bio 12: 1388-1405\\n\\n1250 Vainonen JP, Momeny M, Westermarck J (2021) Druggable cancer phosphatases.\\n\\n1251 _Sci Transl Med 13_\\n\\n1252 Wang SP, Wu SQ, Huang SH, Tang YX, Meng LQ, Liu F, Zhu QH, Xu YG (2021)\\n\\n1253 FDI-6 inhibits the expression and function of FOXM1 to sensitize BRCA-proficient\\n\\n1254 triple-negative breast cancer cells to Olaparib by regulating cell cycle progression\\n\\n1255 and DNA damage repair. Cell death & disease 12: 1138\\n\\n1256 Wilson TR, Fridlyand J, Yan Y, Penuel E, Burton L, Chan E, Peng J, Lin E, Wang Y,\\n\\n1257 Sosman J et al (2012) Widespread potential for growth-factor-driven resistance to\\n\\n1258 anticancer kinase inhibitors. Nature 487: 505-509\\n\\n1259 Wu QN, Liao YF, Lu YX, Wang Y, Lu JH, Zeng ZL, Huang QT, Sheng H, Yun JP, Xie\\n\\n1260 D et al (2018) Pharmacological inhibition of DUSP6 suppresses gastric cancer\\n\\n1261 growth and metastasis and overcomes cisplatin resistance. _Cancer Lett 412: 243-_\\n\\n1262 255\\n\\n1263 Xie T, Lim SM, Westover KD, Dodge ME, Ercan D, Ficarro SB, Udayakumar D,\\n\\n1264 Gurbani D, Tae HS, Riddle SM et al (2014) Pharmacological targeting of the\\n\\n1265 pseudokinase Her3. Nat Chem Biol 10: 1006-1012\\n\\n1266 Yonesaka K, Zejnullahu K, Okamoto I, Satoh T, Cappuzzo F, Souglakos J, Ercan D,\\n\\n1267 Rogers A, Roncalli M, Takeda M et al (2011) Activation of ERBB2 signaling causes\\n\\n1268 resistance to the EGFR-directed therapeutic antibody cetuximab. Sci Transl Med 3:\\n\\n1269 99ra86\\n\\n\\n-----\\n\\n1270 Zandi Z, Kashani B, Alishahi Z, Pourbagheri-Sigaroodi A, Esmaeili F, Ghaffari SH,\\n\\n1271 Bashash D, Momeny M (2022) Dual-specificity phosphatases: therapeutic targets in\\n\\n1272 cancer therapy resistance. J Cancer Res Clin Oncol 148: 57-70\\n\\n1273 Zhang S, Huang WC, Zhang L, Zhang C, Lowery FJ, Ding Z, Guo H, Wang H,\\n\\n1274 Huang S, Sahin AA et al (2013) SRC family kinases as novel therapeutic targets to\\n\\n1275 treat breast cancer brain metastases. Cancer Res 73: 5764-5774\\n\\n1276 Zhang YL, Ma Y, Zeng YQ, Liu Y, He EP, Liu YT, Qiao FL, Yu R, Wang YS, Wu XY\\n\\n1277 _et al (2021) A narrative review of research progress on FoxM1 in breast cancer_\\n\\n1278 carcinogenesis and therapeutics. Ann Transl Med 9: 1704\\n\\n1279 Zheng R, Wan C, Mei S, Qin Q, Wu Q, Sun H, Chen CH, Brown M, Zhang X, Meyer\\n\\n1280 CA et al (2019) Cistrome Data Browser: expanded datasets and new tools for gene\\n\\n1281 regulatory analysis. Nucleic Acids Res 47: D729-D735\\n\\n1282\\n\\n1283\\n\\n1284\\n\\n1285\\n\\n\\n-----\\n\\n1286\\n\\n1287 **Figure legends**\\n\\n1288\\n\\n1289 **Figure 1: Transcriptional landscape of lapatinib tolerance and resistance**\\n\\n1290 **development in HER2+ cells. (A) Development of lapatinib resistance in HER2+**\\n\\n1291 breast cancer cells. BT474 and BT474Br cells were treated with 1 \\uf06dM of lapatinib for\\n\\n1292 9 days (d), 6 months (mos) and 9 mos to yield Drug-tolerant persister (DTP), Drug\\n1293 tolerant expanding perister (DTEP) and Long term resistant (LR) clones,\\n\\n1294 respectively. The cells were stained with crystal violet (0.5% w/v) and the images\\n\\n1295 were acquired with an inverted microscope. The transcriptional profiles from each\\n\\n1296 functional state of lapatinib drug tolerance and resistance development from three\\n\\n1297 parallel BT474 cell plates were surveyed by RNA-sequencing. (B) The number of the\\n\\n1298 genes with a significant change in their expression during the resistance acquisition\\n\\n1299 (see Dataset EV1 for individual gene names). Transcription factor (TF) binding motifs\\n\\n1300 significantly enriched in significantly regulated genes in each transition are indicated.\\n\\n1301 Bolding indicates shared TF binding sites between DTP downregulated and DTEP\\n\\n1302 upregulated genes. (C) Differentially expressed pathways were identified using the R\\n\\n1303 package limma and hallmark gene sets were used for GSVA analysis to reveal\\n\\n1304 hallmarks and signal transduction pathways involved in each step of the resistance\\n\\n1305 acquisition. Red indicates those hallmarks and pathways that are overlapping with\\n\\n1306 processes regulated by DUSP6 depletion in figure 3F.\\n\\n1307\\n\\n1308 **Figure 2: Clinical association of** **_DUSP6 overexpression with poor prognosis_**\\n\\n1309 **HER2+ breast cancers. (A, B)** Volcano plots visualizing differentially expressed\\n\\n1310 genes in (A) Control-DTP and (B) DTP-DTEP transitions. The volcano blots indicate\\n\\n\\n-----\\n\\n1311 all genes that were significantly regulated during these transitions (|logFC |<2 and\\n\\n1312 FDR <0.05), whereas only the phosphatase genes among these are indicated by\\n\\n1313 names. The four phosphatase genes significantly regulated in both transitions\\n\\n1314 (DUSP6, CDC25A, _CDC25C, and_ _SYNJ1) are indicated in bold. Differentially_\\n\\n1315 expressed genes were identified using the R package limma (n=3). (C) Changes in\\n\\n1316 the DUSP6, CDC25A, CDC25C, and SYNJ1 mRNA levels during the acquisition of\\n\\n1317 lapatinib resistance in BT474 cells. Data is based on RNA sequencing analysis\\n\\n1318 (Dataset EV1) and was analyzed by one-way ANOVA followed by Tukey's multiple\\n\\n1319 comparisons test. Statistically significant values of *p < 0.05, **p < 0.01 and ***p <\\n\\n1320 0.001 were determined (n=3). **(D) Differential expression of** _DUSP6,_ _CDC25A,_\\n\\n1321 _CDC25C, and SYNJ1 in different breast cancer subtypes. Data were extracted from_\\n\\n1322 the METABRIC dataset and categorized into five molecular subtypes according to\\n\\n1323 the PAM50 gene expression subtype classification (basal, claudin-low, HER2+,\\n\\n1324 Luminal A, and Luminal B). Data were analyzed by one-way ANOVA followed by\\n\\n1325 Tukey's multiple comparisons test and shown as mean \\uf0b1 standard deviation (SD).\\n\\n1326 Statistically significant values of *p < 0.05, **p < 0.01 and ****p < 0.0001 were\\n\\n1327 determined (basal=209, claudin-low=218, HER2+=224, LumA=700 and LumB=475).\\n\\n1328 **(E) Breast cancer patients from the TCGA-BRCA dataset were divided into DUSP6**\\n\\n1329 high (LogFC>1, FDR<0.05) and low expression (LogFC<-1, FDR<0.05) groups and\\n\\n1330 the clinical breast cancer subtypes were compared between the two groups. NA; not\\n\\n1331 available. **(F,G) Subgroup of 113 patient cases with high tumor** _ERBB2 mRNA_\\n\\n1332 expression (LogFC>1, FDR<0.05) were divided into _DUSP6high and_ _DUSP6low_\\n\\n1333 groups and their overall survival (OS) (G) (Log-rank Test _p value=0.0220) and_\\n\\n1334 disease-specific progression free survival (PFS)(H)(Log-rank Test _p value=0.0259)_\\n\\n1335 was tested according to DUSP6 status.\\n\\n\\n-----\\n\\n1336\\n\\n1337 **Figure 3: Functional involvement of DUSP6 in HER2i tolerance development.**\\n\\n1338 **(A) Expression of DUSP6 protein in different stages of lapatinib resistance**\\n\\n1339 development by Western blot analysis. (B) Relative expression of DUSP6 mRNA in\\n\\n1340 indicated cell lines either at the untreated control situation, or in the DTP state after\\n\\n1341 following treatments: BT474Br, Lapatinib 1\\uf06dM for 9 days; EFM192A and HCC1419,\\n\\n1342 Lapatinib 2.5 \\uf06dM for 14 days; HCC827(EGFRmut NSCLC), 1 µM Osimertinib for 10\\n\\n1343 days; A375(BRAFV600E mutant malignant melanoma (MM)), 1 µM dabrafenib+100\\n\\n1344 nM trametinib for 10 days; and HT-29(BRAFV600E mutant colorectal cancer (CRC)),\\n\\n1345 1 µM dabrafenib+10 µg/ml cetuximab for 10 days. Data for EFM192 and HCC1419\\n\\n1346 cells was obtained from (DataRef: (Chang et al., 2022)(GSE155342), and for other\\n\\n1347 cells by qRT-PCR analysis of the de novo treated samples. Shown is data from two\\n1348 three repeat samples. (C) Transcriptional profile of MDA-MB-453 cells after DUSP6\\n\\n1349 knockdown by 3 different siRNA and compared with 3 different scramble controls,\\n\\n1350 followed by the GSVA analysis of the Hallmark gene sets. The Hallmark gene sets\\n\\n1351 overlapping with the gene sets regulated to same direction in DUSP6 low expressing\\n\\n1352 DTEP cells (Fig. 1C) are indicated with red. **(D, E) Ectopic overexpression of**\\n\\n1353 DUSP6 in BT474 cells inhibits lapatinib effects on cell viability (G) and apoptosis (H),\\n\\n1354 as measured by WST1 cell viability assay and caspase 3/7 activity, respectively.\\n\\n1355 Data was analyzed by two-way ANOVA followed by Tukey' post hoc test.\\n\\n1356 Statistically significant values of **p < 0.01 and ****p < 0.0001 were determined. n=3.\\n\\n1357 **(F) Ectopic overexpression of either wild-type DUSP6 (DUSPWT) or ERK binding**\\n\\n1358 deficient KIM mutant of DUSP6 (DUSP6MUT) in BT474 cells inhibits lapatinib effects\\n\\n1359 on apoptosis. Shown is a result from a representative experiment from three repeats\\n\\n1360 with similar results. Data were analyzed by two-way ANOVA followed by Tukey' post\\n\\n\\n-----\\n\\n1361 hoc test. Statistically significant values of **p < 0.01 and ****p < 0.0001 were\\n\\n1362 determined. **(G) DUSP6 inhibitor BCI preempts DTEP development in BT474 cells**\\n\\n1363 treated with either lapatinib or neratinib for 6 months. The cells were stained and\\n\\n1364 fixed with crystal violet in methanol (0.5% w/v) and the images were acquired with an\\n\\n1365 inverted microscope.\\n\\n1366\\n\\n1367 **Figure 4: Both genetic and pharmacological DUSP6 targeting overcomes**\\n\\n1368 **HER2i resistance. (A)** RNAi-mediated _DUSP6 knockdown, but not of_ _DUSP1,_\\n\\n1369 induces apoptotic cell death in MDA-MB-453 cells, as shown by Western blotting for\\n\\n1370 PARP-1 cleavage. **(B) CRISPR/CAS9-mediated** _DUSP6 knockout hinders the_\\n\\n1371 clonogenic growth of MDA-MB-453 cells, as compared to the CAS9 expressing\\n\\n1372 controls. Shown are three independent single cell clones created with two\\n\\n1373 independent gRNAs (g1 and g2). The cells were seeded at low density and\\n\\n1374 maintained for 10 d. The colonies were stained/fixed with 0.5% crystal violet in\\n\\n1375 methanol and imaged using an inverted microscope. (C) DUSP6 siRNA knockdown\\n\\n1376 increases sensitivity of HER2i resistant MDA-MB-453 cells to HER2-targeted\\n\\n1377 therapies. Cell viability was measured by WST-1 assay after 48 h of drug treatment.\\n\\n1378 Data were collected from three independent experiments each performed in triplicate\\n\\n1379 and analyzed by one-way ANOVA followed by Tukey's multiple comparisons test.\\n\\n1380 Statistically significant values of **p < 0.01, ***p < 0.001 and ****p < 0.0001 were\\n\\n1381 determined. **(D) A 2D synergy map of neratinib-BCI combination in MDA-MB-453**\\n\\n1382 cells calculated by Bliss SynergyFinder (Ianevski et al., 2017). Higher score (in red)\\n\\n1383 indicates for higher degree of drug synergy. The cultures were treated with\\n\\n1384 increasing concentrations of the compounds for 48 h and cell viability was measured\\n\\n1385 by WST-1 assay. **(E)** _DUSP6 siRNA knockdown increases sensitivity of HER2i_\\n\\n\\n-----\\n\\n1386 resistant MDA-MB-453 cells to combination with capecitabine and neratinib. Cell\\n\\n1387 viability was measured by WST-1 assay after 48 h of drug treatment. Data were\\n\\n1388 collected from three independent experiments each performed in triplicate and\\n\\n1389 analyzed by one-way ANOVA followed by Tukey's multiple comparisons test.\\n\\n1390 Statistically significant values of ****p < 0.0001 were determined.\\n\\n1391\\n\\n1392 **Figure 5: DUSP6 inhibition overcomes HER2 inhibitor resistance** **_in vivo. (A)_**\\n\\n1393 Subcutaneous xenograft growth of two independent _DUSP6 single cell knockout_\\n\\n1394 clones of MDA-MB-453 cells targeted with two different gRNA guides. One-way\\n\\n1395 ANOVA followed by Tukey's multiple comparisons test **p < 0.01 comparing each\\n\\n1396 clone to CAS9 expressing control cells . Data are shown as mean \\uf0b1 SD (n=5) (B-E)\\n\\n1397 The effect of BCI in combination with lapatinib or neratinib in two HER2 inhibitor\\n\\n1398 resistant xenograft models: HCC1954 (B, C) or MDA-MB-453 (D, E). Data are shown\\n\\n1399 as mean \\uf0b1 SD (n=10 in each treatment group). Mice with tumor size ∼100 mm[3] were\\n\\n1400 randomized into the experimental and the control groups and tumor volumes were\\n\\n1401 measured every 3 d. Data were analyzed by one-way ANOVA followed by Tukey's\\n\\n1402 multiple comparisons test. Statistically significant values of ****p < 0.0001 were\\n\\n1403 determined. C and E panels display percentual change in the tumor volume from the\\n\\n1404 start of the therapy as water-fall blots in HCC1954 and MDA-MB453 models,\\n\\n1405 respectively. **(F) H&E staining of the representative MDA-MB-453 xenograft tumors**\\n\\n1406 from the control, lapatinib, BCI and lapatinib+BCI groups at day 24. Scale bar 200\\n\\n1407 µm.\\n\\n1408\\n\\n1409 **Figure 6: DUSP6 targeting inhibits HER3 expression** **(A) A 2D synergy map of**\\n\\n1410 lapatinib+MK2206(AKTi) or (B) lapatinib+BCI(DUSP6i) combination in MDA-MB-453\\n\\n\\n-----\\n\\n1411 cells calculated by the Bliss SynergyFinder. Higher score (in red) indicates for higher\\n\\n1412 degree of drug synergy. The cultures were treated with increasing concentrations of\\n\\n1413 the compounds for 48 h and cell viability was measured by WST-1 assay. **(C)**\\n\\n1414 Comparison of the effects of lapatinib+BCI and lapatinib+MK2206 on apoptosis\\n\\n1415 induction (PARP-1 cleavage), and HER2 or HER3 protein levels by Western blot\\n\\n1416 analysis. The cells were treated with lapatinib (1 \\uf06dM), MK2206 (2.5 µM) and BCI (2.5\\n\\n1417 µM) and their combinations for 48 h. **(D)** Quantification of PARP1 cleavage from\\n\\n1418 three repeats of (C). Data were analyzed by one-way ANOVA followed by Tukey's\\n\\n1419 multiple comparisons test. Statistically significant values of **p < 0.01 and ****p <\\n\\n1420 0.0001 were determined. (E,F) The dose-dependent effects of MK2206 (E) and BCI\\n\\n1421 (F) on the expression of HER2 and HER3 protein levels in MDA-MB-453 cells by\\n\\n1422 Western blot analysis after 48h of treatment. (G) The time-dependent effects of BCI\\n\\n1423 (2.5 µM) on the expression of HER2 and HER3 protein levels and apoptosis\\n\\n1424 induction (PARP cleavage) in MDA-MB-453 cells by Western blot analysis. Increase\\n\\n1425 in phosphorylated ERK (p-ERK1/2) and inhibition of DUSP6 both indicate for early\\n\\n1426 target engagement by BCI. **(H)** _DUSP6 knockdown by siRNA inhibits HER2 and_\\n\\n1427 HER3 protein expression in MDA-MB-453 cells. (I) Effects of BCI (50 mg/kg) therapy\\n\\n1428 on HER2 and HER3 protein levels in the MDA-MB-453 xenograft tissue on day 24.\\n\\n1429 Shown is immunohistochemical analysis of HER2 and HER3 from the adjacent\\n\\n1430 paraffin embedded tissue slices from Fig. 5F. Scale bar 200 µm. (J) Breast cancer\\n\\n1431 patients from the TCGA-BRCA dataset were divided into _DUSP6 high (LogFC>1,_\\n\\n1432 FDR<0.05) and low expression (LogFC<-1, FDR<0.05) profiles and expression of\\n\\n1433 phosphorylated HER3 (p-HER3[Y1298]) was compared between the two groups. Data\\n\\n1434 were analyzed by two-tailed t test; **p < 0.01. (DUSP6high=142, DUSP6low=149).\\n\\n1435\\n\\n\\n-----\\n\\n1436 **Fig. 7.** **DUSP6 targeting overcomes Neuregulin/HER3-mediated HER2i therapy**\\n\\n1437 **tolerance**\\n\\n1438 **(A) Comparison of the potential of BCI or MK2206 treatment to overcome Neuregulin**\\n\\n1439 (NRG)-mediated rescue from the anti-proliferative activity of neratinib. The cells were\\n\\n1440 treated with NRG (10 ng/mL), lapatinib (1 µM), MK2206 (1, 2.5, and 5 µM), and BCI\\n\\n1441 (1, 2.5, and 5 µM) for 48 h and cell viability was measured by WST-1 assay. Data\\n\\n1442 were collected from three independent experiments each performed with three\\n\\n1443 technical repeat samples. **(B) Comparison of the effects of BCI and MK2206 on**\\n\\n1444 NRG-mediated evasion from neratinib-induced apoptotic cell death, as measured by\\n\\n1445 Western blot analysis for cleaved PARP-1. The cells were treated with NRG (10\\n\\n1446 ng/mL), lapatinib (1 \\uf06dM), MK2206 (2.5 µM) and BCI (2.5 µM) for 48 h. **(C) HER3**\\n\\n1447 overexpression rescues MDA-MB-453 cells from BCI-elicited inhibition of cell\\n\\n1448 viability. The control or HER3 overexpressing MDA-MB-453 were treated with BCI (3\\n\\n1449 µM) for 48 h and cell viability was measured by WST-1 assay. Shown is data from\\n\\n1450 four technical replicate samples from a representative of three experiments with\\n\\n1451 similar results. The data was analysed by Two-way ANOVA + Tukey’s post hoc test,\\n\\n1452 ****p < 0.0001 (D) Comparison of the effects of lapatinib+BCI and lapatinib+MK2206\\n\\n1453 on NRG-mediated rescue from the anti-growth activity of lapatinib, as shown by\\n\\n1454 crystal violet staining. The cells were treated with NRG (10 ng/mL), lapatinib (1 \\uf06dM),\\n\\n1455 MK2206 (2.5 µM) and BCI (2.5 µM) for 48 h, stained/fixed with 0.5% crystal violet in\\n\\n1456 methanol and imaged by an inverted microscope (images acquired at 10x\\n\\n1457 magnification). **(E) Breast cancer patients from the TCGA-BRCA dataset were**\\n\\n1458 divided into _DUSP6 high (LogFC>1, FDR<0.05) and low expression (LogFC<-1,_\\n\\n1459 FDR<0.05) profiles and the neuregulin (NRG1) mRNA levels were compared\\n\\n1460 between the two groups. Data were analyzed by two-tailed t test; ****p < 0.0001.\\n\\n\\n-----\\n\\n1461 (DUSP6high=167, DUSP6low=181). **(F)** The effect of _DUSP6 knockdown on the_\\n\\n1462 brain metastatic outgrowth of MDA-MB-361 cells in a zebrafish model. GFP positive\\n\\n1463 MDA-MB-361 cells transfected either with control scrambled siRNA or DUSP6 siRNA\\n\\n1464 were injected into zebrafish embryo brain and the GFP intensity was measured 3\\n\\n1465 days after by microscopy. Data were analyzed by two-tailed _t test; ****p < 0.0001._\\n\\n1466 Scale bar 100 µm (siSCR=37, siDUSP6=43). **(G) Improved overall survival of mice**\\n\\n1467 with intracranially injected DUSP6 KO MDA-MB-453 cells as compared to the CAS9\\n\\n1468 positive control cell injected mice. Survival data were analyzed by log-rank Mantel–\\n\\n1469 Cox test, **p < 0.01.\\n\\n1470\\n\\n1471 **Figure 8. Feed-forward loop between NRG/HER3 and DUSP6 determines HER2i**\\n\\n1472 **sensitivity**\\n\\n1473\\n\\n1474 **(A)** _DUSP1 and_ _DUSP6 mRNA levels were determined by qRT-PCR analysis after_\\n\\n1475 treatment with increasing concentrations of neratinib for 48 h in indicated cell lines.\\n\\n1476 Red denotes for HER2i resistant cell lines and green HER2i sensitive cells. Data\\n\\n1477 were collected from three independent experiments each performed in triplicate. (B,\\n\\n1478 **C) Comparison of the effect of neratinib treatment (48 h) on DUSP6, p-AKT and p-**\\n\\n1479 ERK1/2 between HER2i sensitive BT474Br (B) and HER2i resistant MDA-MB-361\\n\\n1480 (C) cells, respectively. **(D) The effect of siRNA-mediated** _HER3 knockdown on_\\n\\n1481 DUSP6 expression in MDA-MB-453 cells by Western blot analysis. **(E) NRG-**\\n\\n1482 mediated induction of DUSP6 mRNA via MEK activation as measured by qRT-PCR\\n\\n1483 analysis after treatment with NRG (10 ng/mL), MK2206(AKTi) (2.5 µM), and\\n\\n1484 trametinib(MEKi) (100 nM) for 48 h. Data were analyzed by one-way ANOVA\\n\\n1485 followed by Tukey's multiple comparisons test. Statistically significant values of *p <\\n\\n\\n-----\\n\\n1486 0.05 and ****p < 0.0001 were determined (n=3). **(F)** The effect of NRG on DUSP6\\n\\n1487 protein expression BT474 cells via MEK activation. The cells were serum-starved for\\n\\n1488 24 h, followed by treatment with NRG (10 ng/mL), MK2206 (2.5 µM) and trametinib\\n\\n1489 (100 nM) for 48 h. (G) Inhibition of DUSP6 expression in HER2i resistant MDA-MB\\n1490 453 cells by MEKi Trametinib. The cells were treated with MK2206 (2.5 µM) or\\n\\n1491 trametinib (100 nM) for 48 h. **(H) A schematic illustration of the discovered**\\n\\n1492 HER3/DUSP6 feed forward loop in HER2+ breast cancer cells. NRG binding to\\n\\n1493 HER3 induces MEK/ERK-mediated _DUSP6 expression which feeds back to_\\n\\n1494 increased HER2 and HER3 expression (left panel). In HER2i sensitive cells (middle\\n\\n1495 panel) inhibition of HER3 results in _DUSP6 inhibition and loss of DUSP6 driven_\\n\\n1496 cancer hallmarks. In HER2i resistant cells (right panel), MEK is not inhibited by\\n\\n1497 HER2i but its constitutive activity (MEKca.) drives DUSP6-HER2/3 positive feed\\n1498 back loop resulting in HER3-mediated multitherapy resistance and cancer\\n\\n1499 progression.\\n\\n1500\\n\\n1501 **Expanded view figure legends**\\n\\n1502\\n\\n1503 **Figure EV1:** **Profiling of a panel of HER2+ breast cancer cell lines for their**\\n\\n1504 **sensitivity to HER2 targeting small molecule tyrosine kinase inhibitors**\\n\\n1505 The primary HER2i sensitive cells are marked on green, and the acquired HER2i\\n\\n1506 resistant cells in red. The long-term resistant (LR) BT-474 and BT-474Br generated\\n\\n1507 by 9-month treatment with lapatinib in this study are denoted in bold. The cells were\\n\\n1508 treated with the increasing concentrations of the indicated HER2i compounds (in µM)\\n\\n1509 for 48 h and cell viability was measured using WST-1 assay.\\n\\n1510\\n\\n\\n-----\\n\\n1511 **Figure EV2: E2F1 and FOXM1 target genes are inhibited across HER2+ DTP**\\n\\n1512 **cells**\\n\\n1513 **A) mRNA levels of selected predicted FOXM1 and E2F1 target genes regulated**\\n\\n1514 during different steps of acquisition of lapatinib resistance in BT474 cells (Fig. 1).\\n\\n1515 Data is blotted based on RNA sequencing analysis (Dataset EV1). (B, C) Expression\\n\\n1516 of the predicted FOXM1 and E2F1 target genes in HER2+ HCC1419 and EFM192A\\n\\n1517 cells between the DTP and the control cells. The RNA-seq data was obtained from\\n\\n1518 (Dataref: (Chang et al., 2022)(GSE155342). The cells were treated with lapatinib (2.5\\n\\n1519 µM) for 14 days to reach the DTP state. Data are shown as mean \\uf0b1 SD (n=2) **(D)**\\n\\n1520 Changes in the expression of FOXM1, PLK1, CCNB1, E2F1, CCNE2 and CDC6 in\\n\\n1521 BT474Br cells between the control and the DTP cells treated for 9 days with lapatinib\\n\\n1522 (1 µM). Data is based on Q-PCR analysis from three technical replicates. The\\n\\n1523 analysis was limited to only these genes due to lack of sufficient mRNA material from\\n\\n1524 the strongly growth suppressed DTP cells. Data are shown as mean \\uf0b1 SD (n=3).\\n\\n1525\\n\\n1526 **Figure EV3: DUSP6 overexpression protects HER2+ cells from HER2i-induced**\\n\\n1527 **cell death**\\n\\n1528 **(A, B) Ectopic overexpression of DUSP6 in BT474 cells inhibits neratinib-elicited**\\n\\n1529 effects on cell viability and apoptosis, as measured by WST1 cell viability assay and\\n\\n1530 caspase 3/7 activity, respectively. Data were collected from three independent\\n\\n1531 experiments each performed in triplicate and analyzed by two-way ANOVA followed\\n\\n1532 by Tukey' post hoc test. Statistically significant values of ****p < 0.0001 were\\n\\n1533 determined. **(C) Ectopic overexpression of DUSP6 in BT474 cells inhibits the**\\n\\n1534 afatinib-elicited effects on cell viability, as measured by WST1 assay. Data were\\n\\n1535 collected from three independent experiments each performed in triplicate and\\n\\n\\n-----\\n\\n1536 analyzed by two-way ANOVA followed by Tukey' post hoc test. Statistically\\n\\n1537 significant values of ****p < 0.0001 were determined. (D) Ectopic overexpression of\\n\\n1538 DUSP6, but not of DUSP1, inhibits the Tucatinib-elicited effects on cell viability, as\\n\\n1539 measured by WST1 assay. Data were collected from three independent experiments\\n\\n1540 each performed in triplicate and analyzed by two-way ANOVA followed by Tukey'\\n\\n1541 post hoc test. Statistically significant values of ****p < 0.0001 were determined.\\n\\n1542\\n\\n1543 **Figure EV4: The anti-proliferative activities of a library of small molecule**\\n\\n1544 **modulators of phosphatases, kinases, and anti-apoptotic proteins in HER2i**\\n\\n1545 **resistant cells. The indicated cells were treated with the increasing concentrations**\\n\\n1546 (in \\uf06dM) of the compounds for 48 h and cell viability was measured using WST-1\\n\\n1547 assay. The long-term resistant (LR) BT474 and BT474Br generated _de novo by 9-_\\n\\n1548 month treatment with lapatinib in this study had comparable drug sensitivity profile to\\n\\n1549 acquired resistant cell line MDA-MB-453. The primary target of the used compound\\n\\n1550 is indicated in parenthesis.\\n\\n1551\\n\\n1552 **Figure EV5: Lack of DUSP6 inhibition is associated with HER2i resistance**\\n\\n1553 **(A,B)** The effects of tucatinib on DUSP6 expression and the signaling pathway\\n\\n1554 activities in (A) HER2i sensitive (green) BT474 cells or (B) HER2i resistant (red)\\n\\n1555 MDA-MB-453 and BT474BrLR cells. The cells were treated with increasing\\n\\n1556 concentrations of tucatinib for 48 h, followed by Western blot analysis. **(C) DUSP6**\\n\\n1557 expression in MDA-MB-453 cells is resistant to multiple HER2is including antibody\\n\\n1558 therapy with trastuzumab. The cells were treated with indicated drugs for 48 h,\\n\\n1559 followed by Western blot analysis.\\n\\n\\n-----\\n\\nBT474\\n\\nBT474Br\\n\\n\\n9 d 6 mos 9 mos\\n\\n\\n### B\\n\\n\\n**_E2F4, FOXM1,_**\\n**_NFYA, E2F1_**\\nSP1\\n\\n\\n_STAT2,_\\n_STAT1_\\n\\n\\n**Re-growth** **Long-term**\\n**by DTEPs** **resistant**\\n\\n\\n**_E2F4, FOXM1,_**\\n_FLI1, E2F1_\\n**_NFYA_**\\n\\n\\n**Re-growth** **Long-term**\\n**by DTEPs** **resistant**\\n\\n930 28\\n\\n_CBX2, ESR1,_\\n_TCF12, ZNF217_\\n\\n\\n### C\\n\\n\\nGSVA (Gene Set Variation Analysis) Hallmark gene sets\\n\\nGene set 1 Gene set 2 Gene set 3 Gene set 4\\n\\n\\nControl\\n\\nTolerance\\n(DTP)\\n\\n\\ncon_1\\n\\ncon_2\\n\\ncon_3\\n\\n\\nDTP_1\\n\\nDTP_2\\n\\nDTP_3\\n\\nDTEP_1\\n\\nDTEP_2\\n\\nDTEP_3\\n\\n\\nExpansion\\n(DTEP)\\n\\nResistance\\n(LR)\\n\\n\\nresistant_1\\n\\nresistant_2\\n\\nresistant_3\\n\\n1\\n\\n\\n-----\\n\\n4\\n\\n2\\n\\n\\n4\\n\\n2\\n\\n\\n0\\n\\n\\n0\\n\\n\\nDUSP4\\n\\nDUSP23\\n**CDC25C**\\n\\nPPM1KFBP1 INPP4B DUSP5P1CDC25ADUSP6PPP1R3D\\nPPP3CC DUSP5 PPP1R12B\\n\\nCDC14B DUSP14 PFKFB1SYNJ1\\n\\n-10 -5 0 5 10\\n\\nlog (fold change)\\n\\n|Col1|Col2|Col3|Col4|Col5|\\n|---|---|---|---|---|\\n||||||\\n||||||\\n\\n|4|Col2|Col3|Col4|Col5|\\n|---|---|---|---|---|\\n||||||\\n||||||\\n||||||\\n\\n\\nControl\\n\\nDTP\\n\\n\\n**C**\\n\\n**D**\\n\\n\\n-10 -5 0 5 10\\n\\nlog (fold change)\\n\\n1000\\n\\n\\n_DUSP6_\\n\\n\\n100\\n\\n10\\n\\n\\nDTEP\\n\\nLR\\n\\n\\n1\\n\\n0.1\\n_DUSP6_ _CDC25A_ _CDC25C_ _SYNJ1_\\n\\n\\n_CDC25C_\\n\\n\\n_CDC25A_\\n\\n\\n_SYNJ1_\\n\\n\\nns\\n\\n\\n9\\n\\n8\\n\\n7\\n\\n6\\n\\n5\\n\\n4\\n\\n\\n12\\n\\n10\\n\\n8\\n\\n6\\n\\n4\\n\\n\\n8\\n\\n7\\n\\n6\\n\\n5\\n\\n4\\n\\n\\n**E** **F** **G**\\n\\n100 OS in ERBB2high breast cancer\\n\\n100%\\n\\nBasal 90%\\n\\n80 80%\\n\\nHER2\\n\\n70%\\n\\n60 LumA 60%\\n\\nLumB 50%\\n\\n40 Normal 40%\\n\\nNA 30% _ERBB2high/DUSP6low_\\n\\n20 20% _ERBB2high/DUSP6high_\\n\\n\\n100%\\n90%\\n80%\\n70%\\n60%\\n50%\\n40%\\n30%\\n20%\\n\\n\\n10 ns ns\\n\\n9\\n\\n8\\n\\n7\\n\\n6\\n\\n5\\n\\nPFS in ERBB2high breast cancer\\n\\n|Col1|Col2|\\n|---|---|\\n\\n|Col1|Col2|\\n|---|---|\\n\\n|Col1|Col2|\\n|---|---|\\n\\n\\n-----\\n\\n#### B\\n\\n D\\n\\n\\nDUSP1\\n\\nß-actin\\n\\n_DUSP6_\\n\\n1.0\\n\\nControl\\nDTP\\n\\n0.5\\n\\n\\n0.0\\n\\n150\\n\\n\\nDUSP1\\n\\nß-actin\\n\\n_DUSP6_\\n\\n1.0\\n\\n\\nControl\\nDTP\\n\\n\\n_DUSP6_1_\\n\\n_DUSP6_2_\\n\\n_DUSP6_3_\\n\\nSCR_1\\n\\nSCR_2\\n\\nSCR_3\\n\\n|0|Col2|Col3|Col4|Col5|Col6|Col7|Col8|\\n|---|---|---|---|---|---|---|---|\\n|||||||||\\n|||||||||\\n|||||||||\\n\\n\\n800\\n\\n600\\n\\n400\\n\\n200\\n\\n0\\n\\n\\nGSVA enrichment score\\n\\n#### E F\\n\\n\\nHER2+ NSCLC\\n\\n\\nMM CRC\\n\\n\\nBT474 BT474DUSP6WT BT474DUSP6MUT\\n\\n1500\\n\\n\\nBT474 BT474DUSP6\\n\\n\\nBT474 BT474DUSP6\\n\\n\\n100\\n\\n50\\n\\n\\n1000\\n\\n500\\n\\n\\n0\\n\\n\\n0\\n\\n\\n#### G\\n\\n\\nBT474 DTEP 6 months\\n\\n\\n-----\\n\\nDUSP1\\n\\nDUSP6\\n\\nß-actin\\n\\nMDA-MB-453\\n\\n\\nCl-PARP, 89\\n\\nCl-PARP, 25\\n\\nDUSP1\\n\\n\\n#### C\\n\\n\\nCl-PARP, 89\\n\\nCl-PARP, 25\\n\\nDUSP1\\n\\nDUSP6\\n\\n\\nMDA-MB-453 MDA-MB-453\\n\\nMDA-MB-453\\n\\n\\nCl-PARP, 25\\n\\nDUSP1\\n\\nDUSP6\\n\\nß-actin\\n\\nMDA-MB-453\\n\\n150 **\\n\\n\\n_DUSP6 KD+trastuzumab_\\n\\nMDA-MB-453\\n\\n\\n150\\n\\n100\\n\\n\\nSCR\\n\\nLapatinib (1 µM)\\n\\n\\nNeratinib (100 nM)\\n\\nTrastuzumab (10 µg/mL)\\n\\n\\n50\\n\\n\\n_DUSP6 KD_\\n\\n_DUSP6 KD+lapatinib_\\n\\n_DUSP6 KD+neratinib_\\n\\n\\n0\\n\\n### D\\nMDA-MB-453\\n\\n\\n****\\n****\\n****\\n\\n***\\n**\\n\\n### E\\n\\n|Col1|**|Col3|Col4|Col5|Col6|Col7|Col8|Col9|Col10|\\n|---|---|---|---|---|---|---|---|---|---|\\n||** ** *** **|||||||||\\n|||||||||||\\n|||||||||||\\n|||||||||||\\n\\n\\nBliss synergy score: 16.194\\n\\n\\n****\\n\\n****\\n\\n\\n10\\n\\n5\\n\\n\\n150\\n\\n100\\n\\n\\nSCSCR\\n\\nCaCapecitabine+neratinib\\n\\n\\n2.5\\n\\n1\\n\\n\\nSynergy\\nscore\\n\\n40\\n20\\n0\\n−20\\n−40\\n\\n\\n****\\n\\n\\n0.5\\n\\n0.25\\n\\n\\n50\\n\\n\\n_DUDUSP6 KD_\\n\\n_DUDUSP6 KD+_\\n\\n\\n-----\\n\\n1500\\n\\n1000\\n\\n\\n1000\\n\\n500\\n\\n\\n500\\n\\n0\\n0 3 6 9 12 15 18\\n\\nTime (days)\\n\\n\\n**\\n\\n|DUSP6 KO, g1 CAS9|Col2|\\n|---|---|\\n|DUSP6 KO, g1 DUSP6 KO, g2 **||\\n|||\\n\\n\\n**C** **D**\\n\\n**HCC 1954**\\n\\n\\n0\\n\\n800\\n\\n600\\n\\n400\\n\\n200\\n\\n\\n0 3 6 9 12 15 18\\n\\nTime (days)\\n\\n**MDA-MB-453**\\n\\n\\n1500\\n\\n1000\\n\\n500\\n\\n\\nControl BCI (50mg/kg)\\n\\nLapatinib (50mg/kg) BCI+Lapatinib\\n\\n\\n50\\n\\n\\nControl BCI\\nNeratinib BCI+Neratinib\\n\\nControl BCI Neratinib BCI+Neratinib\\n\\n\\n0\\n\\n-50\\n\\n\\n0\\n\\n\\n**E** **F**\\n\\n**MDA-MB-453**\\n\\n\\n0 3 6 9 12 15 18 21 24\\nTime (days)\\n\\n**MDA-MB-453 xenograft**\\n\\n\\nBCI\\nControl\\n\\nBCI\\nLapatinib +\\nlapatinib\\n\\n\\n1000\\n\\n750\\n\\n500\\n\\n250\\n\\n\\nControl BCI\\nLapatinib BCI+lapatinib\\n\\n\\n-----\\n\\n5\\n\\n\\n5\\n\\n\\n2.5\\n\\n1\\n\\n\\nCl-PARP1, 25\\n\\n\\n0.5\\n\\n\\np-AKT\\n\\nHER2\\n\\nHER3\\n\\n\\n0.25\\n\\n\\n0.1\\n\\n0\\n\\n\\np-AKT\\n\\nHER2\\n\\nHER3\\n\\nß-actin\\n\\n_ _ _\\n\\n\\n**D** **E**\\n\\n**** MDA-MB-453\\n\\n1.0 **** MK2206 (μM): 0 0.1 0.5 1 2.5\\n\\n****\\n\\nHER2\\n\\n0.8\\n\\nControl HER3\\n\\nLapatinib\\n\\np-AKT\\n\\n0.6 BCI\\n\\nAKT\\n\\n** MK2206\\n\\n0.4 Lapatinib+BCI ß-actin\\n\\nLapatinib+MK2206 MK2206, 48h\\n\\n0.2\\n\\n|Col1|Col2|Col3|Col4|\\n|---|---|---|---|\\n||**** **** **|||\\n|||||\\n|||||\\n|||||\\n|||||\\n\\n\\nMK2206\\n\\nBCI (µM)\\n\\n**F**\\n\\nMDA-MB-453\\nμM): 0 0.1 0.5 1 2.5\\n\\nHER2 BCI (μM):\\n\\nHER2\\n\\nHER3\\n\\nHER3\\n\\nß-actin\\n\\nAKT\\n\\n\\nHER2\\n\\nHER3\\n\\nß-actin\\n\\nBCI, 48h\\n\\n### J\\n\\nTCGA, breast cancer\\n\\n\\np-AKT\\n\\nAKT\\n\\nß-actin\\n\\nMK2206, 48h\\n\\n### H J\\n\\n\\n**F**\\n\\nMDA-MB-453\\nMK2206 (μM): 0 0.1 0.5 1 2.5\\n\\nHER2\\n\\nHER3\\n\\np-AKT\\n\\n\\nMDA-MB-453\\n\\nμM): 0 0.1 0.5 1 2.5\\n\\nHER2\\n\\nHER3\\n\\n\\n### G\\n\\n\\nBCI (µM)\\n\\n**F**\\n\\nMDA-MB-453\\nMK2206 (μM): 0 0.1 0.5 1 2.5\\n\\nHER2 BCI (\\n\\nHER3\\n\\n\\nBCI (μM): 0 0.1 0.5 1 2.5\\n\\nHER2\\n\\nHER3\\n\\n\\n0.0\\nMDA-MB-453\\n\\nMDA-MB-453\\n\\nBCI (h) 0 3 6 12 24 48\\n\\n\\np-AKT\\n\\nAKT\\n\\nß-actin\\n\\nMK2206, 48h\\n\\n### H J\\n\\n\\n_ _ _\\n\\n_ +_ _ + +_\\n\\n_ _ +_ +_\\n\\n+ +\\n\\n\\nHER2\\n\\nß-actin\\nHER3\\n\\n\\nß-actin\\n\\nMK2206, 48h\\n\\n### J\\n\\nHER3\\n\\nHER2\\n\\nDUSP6\\n\\nß-actin\\n\\n\\nPARP1, 110\\nCl-PARP, 89\\n\\nDUSP6\\n\\n\\np-ERK1/2\\n\\nß-actin\\n\\n\\n### J\\n\\nHER3\\n\\nHER2\\n\\nDUSP6\\n\\nß-actin\\n\\nMDA-MB-453\\n\\nMDA-MB-453 xenograft\\n\\nHER2\\n\\n\\n-----\\n\\nNRG +Neratinib\\n\\nNRG +Neratinib\\n\\n\\n50\\n\\n0\\n\\n\\nCl-PARP, 25\\n\\nβ-actin\\n\\nNeratinib\\n\\nNRG\\nBCI\\n\\nMK2206\\n\\n### D\\n\\n\\n100\\n\\n50\\n\\n\\n100\\n\\n-actin\\n\\n_\\n+ + + + 50\\n\\n_ _\\nNRG + + +\\n_ _ _ _\\nBCI +\\n\\n_ _ _ _\\nMK2206 + 0\\n\\nBT474 48 hours\\n\\n\\nNRG +Neratinib\\n\\n### E\\n\\n\\n-actin\\n\\n_\\n+ + + +\\n\\n_ _\\nNRG + + +\\n_ _ _ _\\nBCI +\\n\\n_ _ _ _\\nMK2206 +\\n\\n\\n+\\n+_\\n_\\n\\n\\n+\\n+\\n+_\\n\\n\\nDMSO BCI\\n\\n+\\n+_\\n+\\n\\n\\nLapatinib\\nNRG\\nMK2206\\nBCI\\n\\n\\n_\\n_\\n\\n_\\n_\\n\\n\\n### F\\n\\n\\n**G**\\n\\n\\n+_\\n_\\n_\\n\\n100\\n\\n\\nTCGA, breast cancer\\n\\n\\nMDA-MB-361\\n\\n\\n15\\n\\n10\\n\\n\\n****\\n\\n\\nMDA-MB-453 mouse intracranial xenograft\\n\\nCAS9 _DUSP6 KO, g1_\\n\\nP=0.0019\\n\\n\\nBrain Tumor Area\\n\\n2.0 p=5.767e-007\\n\\n1.5\\n\\n\\n5\\n\\n\\n75\\n\\n50\\n\\n\\n25\\n\\n\\n0.5\\n\\n\\n-----\\n\\nHCC1954\\nMDA-MB-361\\nMDA-MB-453\\nBT474Br-LR\\nBT474-LR\\n\\n\\n### C\\n\\n F\\n\\n G\\n\\n\\nNeratinib (nM) 0 10 25 50 100\\n\\nDUSP6\\n\\np-AKT\\n\\nAKT\\n\\np-ERK1/2\\n\\nERK1/2\\n\\nß-actin\\n\\n### E\\nns\\n\\n15\\n\\nSFM\\n\\n10 NRG\\n\\nNRG+MK2206\\n\\n5 NRG+trametinib\\n\\n\\n### D\\n\\nMDA-MB-361\\n\\nNeratinib (nM) 0 10 25 50 100\\n\\nDUSP6\\n\\np-AKT HER3\\n\\nAKT\\n\\nDUSP6\\n\\np-ERK1/2\\n\\nERK1/2 ß-actin\\n\\nß-actin\\n\\nBT474\\n\\nDUSP6\\n\\np-AKT\\n\\nAKT\\n\\np-ERK1/2\\n\\nERK1/2\\n\\nß-actin\\n\\n\\nDUSP6\\n\\np-AKT HER3\\n\\nAKT\\n\\nDUSP6\\n\\np-ERK1/2\\n\\nERK1/2 ß-actin\\n\\nß-actin\\n\\nBT474\\n\\nDUSP6\\n\\np-AKT\\n\\nAKT\\n\\np-ERK1/2\\n\\nERK1/2\\n\\nß-actin\\n\\n**_**\\n\\nNRG (10 ng/mL) **+**\\n\\n\\nDUSP6\\n\\np-AKT HER3\\n\\nAKT\\n\\nDUSP6\\n\\np-ERK1/2\\n\\nERK1/2 ß-actin\\n\\nß-actin\\n\\nBT474\\n\\nDUSP6\\n\\np-AKT\\n\\nAKT\\n\\np-ERK1/2\\n\\n\\nHCC1954\\nMDA-MB-361\\nMDA-MB-453\\nBT474Br-LR\\nBT474-LR\\n\\n0 10 25 50 100\\n\\nNeratinib (nM)\\n\\n### C D\\n\\nMDA-MB-361\\n\\nNeratinib (nM) 0 10 25 50 100\\n\\nDUSP6\\n\\np-AKT\\n\\nAKT\\n\\n\\n**+_** **+** **+_**\\n**_** **_[+]**\\n\\n**+**\\n\\nMDA-MB-453\\n\\n\\n**_**\\n**_** **+_** **+** **+_**\\n**_** **_** **_[+]**\\n\\n**+**\\n\\nMDA-MB-453\\n\\nDUSP6\\n\\np-AKT\\n\\nAKT\\n\\n\\np-ERK1/2\\n\\nERK1/2\\n\\nß-actin\\n\\n**_**\\n**_** **+_** **+** **+_**\\n**_** **_** **_[+]**\\n\\n**+**\\n\\n\\nAKT\\n\\np-ERK1/2\\n\\nERK1/2\\n\\n\\n-----\\n\\nBT474Br\\n\\n\\n80\\n\\n60\\n\\n\\n80\\n\\n60\\n\\n\\nBT474Br\\n\\n\\nHCC2218\\n\\nSKBR3\\n\\n\\nHCC2218\\n\\nSKBR3\\n\\n\\nHCC1954\\n\\nHCC1569\\n\\nMDA-MB-361\\n\\nMDA-MB-453\\n\\n\\nHCC1954\\n\\nHCC1569\\n\\nMDA-MB-361\\n\\nMDA-MB-453\\n\\n\\n**BT474Br-LR**\\n\\n**BT474-LR**\\n\\n\\n40\\n\\n20\\n\\n\\n**BT474Br-LR**\\n\\n**BT474-LR**\\n\\n\\n40\\n\\n20\\n\\n\\nHCC2218\\n\\nSKBR3\\n\\n\\nBT474\\n\\nBT474Br\\n\\n\\nNeratinib\\n100\\n\\n80\\n\\n\\nBT474\\n\\nBT474Br\\n\\n\\nTucatinib\\n100\\n\\n80\\n\\n\\nHCC2218\\n\\nSKBR3\\n\\n\\n60\\n\\n40\\n\\n\\nHCC1954\\n\\nHCC1569\\n\\nMDA-MB-361\\n\\nMDA-MB-453\\n\\n\\n60\\n\\n40\\n\\n\\nHCC1954\\n\\nHCC1569\\n\\nMDA-MB-361\\n\\nMDA-MB-453\\n\\n\\n**BT474Br-LR**\\n\\n**BT474-LR**\\n\\n\\n**BT474Br-LR**\\n\\n**BT474-LR**\\n\\n\\n20\\n\\n\\n20\\n\\n\\n-----\\n\\n## A B\\n\\n#### BT474, RNAseq.\\n\\n\\n#### HCC1419, RNAseq, GSE155341\\n20000\\nControl\\n\\n\\nDTP\\n\\n\\nFOXM1\\n\\nPLK1\\n\\n\\n15000\\n\\n10000\\n\\n\\nCCNB1\\n\\n\\nBIRC5\\n\\nCDC25A\\n\\nCDC25C\\n\\n\\n5000\\n\\n|Col1|HCC1419, RNAseq, GSE155341|Col3|Col4|Col5|Col6|Col7|Col8|Col9|Col10|Col11|Col12|Col13|Col14|Col15|Col16|Col17|Col18|\\n|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|\\n||Con DTP|||||||||||||||||\\n|||||||||||||||||||\\n\\n\\nSYNJ1\\n\\n\\n400\\n\\n300\\n\\n\\n0\\n\\n8000 **EFM192A, RNAseq, GSE155341**\\nControl\\n\\n|Col1|EFM192A, RNAseq, GSE155341|Col3|Col4|Col5|Col6|Col7|\\n|---|---|---|---|---|---|---|\\n||Control DTP||||||\\n||||||||\\n||||||||\\n||||||||\\n||||||||\\n\\n\\nFOXM1 target genes E2F target genes\\n\\n#### BT474BR, Q-PCR\\n\\n1.5 Control\\n\\n\\nAURKB\\n\\nAURKA\\n\\nE2F1\\n\\n\\nCCNE2\\n\\nCDC6\\n\\n\\n## C\\n\\n D\\n\\n\\n6000\\n\\n4000\\n\\n\\nCENPF\\n\\nNCAPG\\n\\nCDK1\\n\\nTYMS\\n\\n\\n200\\n\\n100\\n\\n\\n2000\\n\\n0\\n\\n\\nRAD51\\n\\nPCNA\\n\\nRRM1\\n\\nRFC3\\n\\n\\n1.0\\n\\n0.5\\n\\n\\nDTP\\n\\n\\n0.0\\nFOXM1 PLK1 CCNB1 E2F1 CCNE2 CDC6\\n\\n\\n-----\\n\\n# A B\\n\\n**BT474** **BT474DUSP6-OE**\\n\\n**150**\\n\\n\\n**800**\\n\\n**600**\\n\\n\\n**BT474** **BT474DUSP6-OE**\\n\\n\\n**100**\\n\\n**50**\\n\\n\\n**400**\\n\\n**200**\\n\\n\\n**0**\\n\\n\\n**0**\\n\\n**BT474** **BT474DUSP1-OE** **BT474DUSP6-OE**\\n\\n\\n# C\\n\\n\\n# D\\n\\n\\n**BT474** **BT474DUSP6-OE**\\n\\n\\n**150**\\n\\n**100**\\n\\n\\n**150**\\n\\n**100**\\n\\n\\n**50**\\n\\n**0**\\n\\n\\n**50**\\n\\n**0**\\n\\n\\n-----\\n\\nBCI (DUSP6i)\\n\\nBCI-215(DUSP6i)\\n\\nPTP1Bi\\n\\nFTY720(PP2A reactivator)\\n\\nPRL1i\\n\\nDT061(PP2A reactivator)\\n\\nMK2206(AKTi)\\n\\nTemsirolimus(mTORi)\\n\\nDactolisib(PI3K/mTORi)\\n\\nTrametinib(MEKi)\\n\\nBI2536(PLKi)\\n\\nBAY117082(NF-kBi)\\n\\nTamoxifen(ERi)\\n\\nR428(AXLi)\\n\\nForetinib(METi)\\n\\nDasatinib(SRCi)\\n\\nLinsitinib(IGF-1Ri)\\n\\nSP600125(JNKi)\\n\\nSB203580(P38i)\\n\\nAZD5582(IAPi)\\n\\nSabutoclax(BCL2i)\\n\\nYM155(survivin antagonist)\\n\\nAlisertib(AURKAi)\\n\\nPalbociclib(CDK4/6i)\\n\\nCediranib(VEGFRi)\\n\\nRegorafenib(VEGFRi)\\n\\nVeliparib(PARPi)\\n\\n5FU\\n\\nDrug concentration (μM):\\n\\n\\n-----\\n\\n### A B\\nBT474 MDA-MB-453 BT474BrLR\\n\\nTucatinib (nM):  0    100  250   500  1000 Tucatinib (nM):  0    100  250   500  1000 Tucatinib (nM):  0    100  250   500  1000\\n\\n\\n### C MDA-MB-453\\n\\nDUSP6\\nActin\\n\\n\\nDUSP6\\n\\n\\nDUSP6\\n\\n\\nDUSP6\\n\\n\\np-ERK1/2\\n\\nERK1/2\\n\\n\\np-ERK1/2\\n\\nERK1/2\\n\\n\\np-ERK1/2\\n\\nERK1/2\\n\\n\\np-S6\\n\\nS6\\n\\n\\np-S6\\n\\nS6\\n\\n\\np-S6\\n\\nS6\\n\\n\\nActin\\n\\n\\nActin\\n\\n### Expanded View 5\\n\\n\\nActin\\n\\n\\nDUSP6\\nActin\\n\\n\\n-----\\n\\n**Expanded view table 1:**\\n\\n**Used siRNA sequences**\\n\\n**Target Gene** **Cat number** **Company** **Concentration**\\n_HER2_ s613 ThermoFisher 20 nM\\n_HER3_ SI02660245 Qiagen 100 nM\\n_AKT1_ s659 ThermoFisher 20 nM\\n_DUSP1_ SI03100048 Qiagen 150 nM\\n\\n\\nQiagen\\nThermoFisher\\nThermoFisher\\n\\nQiagen\\nThermoFisher\\nThermoFisher\\n\\n\\n_DUSP6_\\n\\nNegative controls\\n\\n\\nSI03106404\\n\\ns4379\\ns4378\\n\\nSI03650318\\n\\n4390844\\n4390847\\n\\n\\n150 nM\\n\\n   \\n\\n-----\\n\\n**Expanded view Table 2:**\\n\\n**Antibodies**\\n\\n**Target protein** **Cat number** **Company** **Clone** **Dilution**\\n\\nHER2 sc-33684 Santa Cruz 3B5 1:500-1:5000\\n\\np-HER2 2243 CST 6B12 1:1000\\n\\n\\nCST\\n\\nMillipore\\n\\n\\nD22C5\\n\\n2F12\\n\\n\\n1:1000\\n\\n1:500\\n\\n\\nHER3\\n\\n\\n12708\\n\\n05-390\\n\\n\\np-HER3 4791 CST 21D3 1:1000\\n\\nDUSP1 07-535 Millipore - 1:500\\n\\nDUSP6 ab76310 Abcam EPR129Y 1:1000\\n\\nAKT sc-5298 Santa Cruz B-1 1:500\\n\\np-AKT 4060 CST D9E 1:1000\\n\\nERK1/2 sc-514302 Santa Cruz C-9 1;1000\\n\\np-ERK1/2 9101 CST - 1:1000\\n\\nβ-actin sc-47778 Santa Cruz C4 1:5000\\n\\nP38 8690 CST D13E1 1:1000\\n\\n\\n-----\\n\\np-P38 4511 CST D3F9 1:1000\\n\\nJNK1/2 3708 CST 2C6 1:1000\\n\\np-JNK1/2 4668 CST 81E11 1:1000\\n\\nPARP-1 9532 CST 46D11 1:1000\\n\\n\\nCST\\n\\nAbcam\\n\\n\\nD64E10\\n\\nE51\\n\\n\\n1:1000\\n\\n1:2500\\n\\n\\nCl-PARP-1\\n\\n\\n5625\\n\\nab32064\\n\\n\\nGAPDH 5G4-6C5 HyTest   - 1:5000\\n\\nCST: Cell signaling technology.\\n\\n\\n-----\\n\\n**Expanded view Table 3**\\n\\n**qRT-PCR primers**\\n\\nGene Accession number Forward primer (5’→3’) Reverse primer (5’→3’) Size\\n\\n(bp)\\n\\n_DUSP1_ NM_004417 AGGCCATTGACTTCATAGACTCC TGGGAGAGATGATGCTTCGC 177\\n\\n_DUSP6_ NM_001946 CCTGGAAGGTGGCTTCAGTAA GCACTATTGGGGTCTCGGTC 180\\n\\n_B2M_ NM_004048 GATGAGTATGCCTGCCGTGT CTGCTTACATGTCTCGATCCCA 79\\n\\n_HER3_ NM_001982 GGTGATGGGGAACCTTGAGAT CTGTCACTTCTCGAATCCACTG 80\\n\\n_HER2_ NM_004448 TGTGACTGCCTGTCCCTACAA CCAGACCATAGCACACTCGG 152\\n\\n_BIRC5_ NM_001168 CCAGATGACGACCCCATAGAG TTGTTGGTTTCCTTTGCAATTTT 152\\n\\n_E2F1_ NM_005225 TGGACTCTTCGGAGAACTTTCA TGATCCCACCTACGGTCTCC 93\\n\\n_CCNE2_ NM_057749 TAGCTGGTCTGGCGAGGTTT GGCCTGGATTATCTGGGCTT 130\\n\\n_CDC6_ NM_001254 GCGAGGCCTGAGCTGTG AGGCAGGGCTTTTACACGAG 181\\n\\n_CENPF_ NM_016343 CGTCCCCGAGAGCAAGTTTA GTAGGCAGCCCTTCTTTCCA 98\\n\\n_NCAPG_ NM_022346 AGTCCACATAGAGAAGAATGATGC TCCACAGCATCCCAAGCATA 199\\n\\n_CDK1_ NM_001786 AAACTACAGGTCAAGTGGTAGCC TCCTGCATAAGCACATCCTGA 148\\n\\n_TYMS_ NM_001071 CTGCTGACAACCAAACGTGTG GCATCCCAGATTTTCACTCCCTT 116\\n\\n\\n-----\\n\\n_RAD51_ NM_133487 CAACCCATTTCACGGTTAGAGC TTCTTTGGCGCATAGGCAACA 107\\n\\n_PCNA_ NM_002592 GCGTGAACCTCACCAGTATGT TCTTCGGCCCTTAGTGTAATGAT 76\\n\\n_RRM1_ NM_001033 GCCAGGATCGCTGTCTCTAAC GAGAGTGTTTGCCATTATGTGGA 106\\n\\n_RFC3_ NM_002915 GTGGACAAGTATCGGCCCTG TGATGGTCCGTACACTAACAGAT 120\\n\\n_ESCO2_ NM_001017420 TGTGTGCAAGTCTTGTGGTATG CCCATCCCAAAACTCTGCTACT 145\\n\\n_FOXM1_ NM_202002 ATAGCAAGCGAGTCCGCATT AGCAGCACTGATAAACAAAGAAAGA 151\\n\\n_CDC25A_ NM_001789 CTACCTCAGAAGCTGTTGGGA AAAGGCTTCATTTTCCTTGTTCTCA 117\\n\\n_AURKB_ NM_001313950 ACCTGCACCATCCCAACATC TGCCAACTCCTCCATGATCG 164\\n\\n_PLK1_ NM_005030 TCTTCCAGGATCACACCAAGC AGGAGACTCAGGCGGTATGT 100\\n\\n_CCNB1_ NM_031966 GATACTGCCTCTCCAAGCCC TGACTGCTTGCTCTTCCTCAA 196\\n\\n_CDC25C_ NM_001287582 CTACCCAGTCGGAAGGCAGA AAACAAAACCTAGCTCAAGCCT 159\\n\\n_AURKA_ NM_001323303 CTCAGTGGCGGACGAGGA GGAGTGAGACCCTCTAGCTGT 200\\n\\n_SYNJ1_ NM_003895 AGCTCTTGAAGGGAAAGCGAA TGTCTGCTCAGAAACACGCAA 193\\n\\n\\n-----\\n\\n\""
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pymupdf4llm.to_markdown(PDF_FILES[0])\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Document loading using langchain"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain_community.document_loaders import PyMuPDFLoader\n",
    "loader = PyMuPDFLoader(PDF_FILES[0])\n",
    "data = loader.load()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain_community.document_loaders import PyMuPDFLoader\n",
    "loader = PyMuPDFLoader(DOCX_FILES[0])\n",
    "data = loader.load()\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Searching for text in the document\n",
    "\n",
    "PyMuPDF supports searching for text at the page level. This might be of use for us to limit the context sent to the LLM."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Text found on page\n",
      "Text found on page\n"
     ]
    }
   ],
   "source": [
    "# Loop through the pages int he document and return true if text is found\n",
    "docx_doc = pymupdf.open(DOCX_FILES[0])\n",
    "for page in docx_doc:\n",
    "    if page.search_for('Figure legends'):\n",
    "        print('Text found on page')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Text found on page\n",
      "The collisional energy increased linearly from 20.01 eV at 0.6 (1/K0) to 52.00 eV at 1.35 Vs cm−2 (1/K0). The scan\n",
      "range for MS and MS/MS spectra was set to 100-1700 m/z. TIMS ramping time and accumulation time were set to\n",
      "100 milliseconds. The diaPASEF data were analyzed by directDIA approach using Spectronaut software (version 14)\n",
      "following manufacture instructions. Up or down regulated proteins were determined using DEP (Differential\n",
      "Enrichment analysis of Proteomics data) R package. Significant results (adjusted p-value < 0.05) were subjected to\n",
      "Gene Ontology enrichment analysis with clusterProfiler R package.\n",
      "Western Blot\n",
      "Total protein extracts were prepared by lysing cells in extraction buffer (HEPES KOH [pH 8.5], NaCl 400 mM,\n",
      "EDTA 0.1 mM, EGTA 0.1 mM, DTT 1 mM, 1× protease inhibitor, SDS 1%). Proteins were separated by\n",
      "electrophoresis on BOLT 4%–12% bis-tris polyacrylamide precast gels in MES buffer (Life Technologies) and\n",
      "transferred to a 0,2 m nitrocellulose membrane. Non-specific signals were blocked with 5% Milk-PBS-Tween0.5%\n",
      "and the membrane hybridized overnight at 4°C with primary and secondary antibodies diluted in blocking buffer.\n",
      "Horseradish peroxidase-conjugated secondary antibodies were revealed with the ECL chemiluminescence kit\n",
      "(Amersham), and signals were detected using ChemiDoc (Bio-Rad). Primary antibodies: PKR (rabbit, Abcam,\n",
      "ab32052), cGAS (rabbit, Abcam, ab224144), LINE-1 ORF1 (mouse, Merck Millipore, MABC1152), EIF2a (mouse,\n",
      "Abcam, ab5369 phospho-EIF2 (rabbit, Abcam, ab32157), Histone H3 (rabbit, Merk, 06-755).\n",
      "Exosome isolation\n",
      "Exosomes were isolated using Total Exosome Isolation Reagent (from cell culture media) (Thermo), following\n",
      "manufacturer’s instructions. The exosome pellet was resuspended in 50 l of RIPA buffer and kept on ice for 30’,\n",
      "mixing every 5’. Samples were then sonicated (30’’ ON/30” OFF, 10 cycles), centrifuged at 13,000 x g for 20’ at\n",
      "4°C and collected as supernatants.\n",
      "Statistical analysis\n",
      "Statistic tests used for data analysis are indicated in the figure legends. \n",
      "Data availability\n",
      "The RNA-seq transcriptomics data on human bone biopsies have been deposited to BioProject with the dataset\n",
      "identifier PRJNA764663 (https://www.ncbi.nlm.nih.gov/bioproject/PRJNA764663). The RNA-seq transcriptomics\n",
      "data on transfected osteoblasts have been deposited to GEO with the dataset identifier GSE201774\n",
      "\n"
     ]
    }
   ],
   "source": [
    "# Loop through the pages int he document and return true if text is found\n",
    "docx_doc = pymupdf.open(DOCX_FILES[0])\n",
    "for page in docx_doc:\n",
    "    if page.search_for('Data availability'):\n",
    "        print('Text found on page')\n",
    "        print(page.get_text())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Text found on page\n",
      "The collisional energy increased linearly from 20.01 eV at 0.6 (1/K0) to 52.00 eV at 1.35 Vs cm−2 (1/K0). The scan\n",
      "range for MS and MS/MS spectra was set to 100-1700 m/z. TIMS ramping time and accumulation time were set to\n",
      "100 milliseconds. The diaPASEF data were analyzed by directDIA approach using Spectronaut software (version 14)\n",
      "following manufacture instructions. Up or down regulated proteins were determined using DEP (Differential\n",
      "Enrichment analysis of Proteomics data) R package. Significant results (adjusted p-value < 0.05) were subjected to\n",
      "Gene Ontology enrichment analysis with clusterProfiler R package.\n",
      "Western Blot\n",
      "Total protein extracts were prepared by lysing cells in extraction buffer (HEPES KOH [pH 8.5], NaCl 400 mM,\n",
      "EDTA 0.1 mM, EGTA 0.1 mM, DTT 1 mM, 1× protease inhibitor, SDS 1%). Proteins were separated by\n",
      "electrophoresis on BOLT 4%–12% bis-tris polyacrylamide precast gels in MES buffer (Life Technologies) and\n",
      "transferred to a 0,2 m nitrocellulose membrane. Non-specific signals were blocked with 5% Milk-PBS-Tween0.5%\n",
      "and the membrane hybridized overnight at 4°C with primary and secondary antibodies diluted in blocking buffer.\n",
      "Horseradish peroxidase-conjugated secondary antibodies were revealed with the ECL chemiluminescence kit\n",
      "(Amersham), and signals were detected using ChemiDoc (Bio-Rad). Primary antibodies: PKR (rabbit, Abcam,\n",
      "ab32052), cGAS (rabbit, Abcam, ab224144), LINE-1 ORF1 (mouse, Merck Millipore, MABC1152), EIF2a (mouse,\n",
      "Abcam, ab5369 phospho-EIF2 (rabbit, Abcam, ab32157), Histone H3 (rabbit, Merk, 06-755).\n",
      "Exosome isolation\n",
      "Exosomes were isolated using Total Exosome Isolation Reagent (from cell culture media) (Thermo), following\n",
      "manufacturer’s instructions. The exosome pellet was resuspended in 50 l of RIPA buffer and kept on ice for 30’,\n",
      "mixing every 5’. Samples were then sonicated (30’’ ON/30” OFF, 10 cycles), centrifuged at 13,000 x g for 20’ at\n",
      "4°C and collected as supernatants.\n",
      "Statistical analysis\n",
      "Statistic tests used for data analysis are indicated in the figure legends. \n",
      "Data availability\n",
      "The RNA-seq transcriptomics data on human bone biopsies have been deposited to BioProject with the dataset\n",
      "identifier PRJNA764663 (https://www.ncbi.nlm.nih.gov/bioproject/PRJNA764663). The RNA-seq transcriptomics\n",
      "data on transfected osteoblasts have been deposited to GEO with the dataset identifier GSE201774\n",
      "\n",
      "Text found on page\n",
      "regeneration. Dev Growth Differ 54: 673–685\n",
      "Figure Legends\n",
      "Figure 1 - TEs expression is induced after fracture and in mechanically loaded bone.\n",
      "A) Heatmap representation of RNAseq differentially expressed TEs (LogFC>0.5) between 4h post-injury and intact\n",
      "femurs at different time points of the bone healing process. N=5 biological replicates for each time point. N=10\n",
      "biological replicates for intact femur. B) Heatmap representation of RNAseq upregulated TEs (LogFC>0.5) between\n",
      "4h and intact at different time points of the bone healing process. N=5 biological replicates for each time point.\n",
      "N=10 biological replicates for intact femur. C) Heatmap representation of RNAseq differentially expressed TEs\n",
      "analysis [Fragments Per Kilobase of transcript per Million (FPKM) fold change] in femoral (n=27) and iliac (n=34)\n",
      "bone biopsies from healthy donors. D) Pie charts showing the number and percentage of differentially expressed\n",
      "TEs subfamilies between femoral and iliac healthy bone. E) Heatmap representation of RNAseq differentially\n",
      "expressed TEs analysis [Fragments Per Kilobase of transcript per Million (FPKM) fold change] in femoral (n=27)\n",
      "and iliac (n=34) bone biopsies from healthy donors. One heatmap for each TEs order is shown. \n",
      "Figure 2 - TEs expression correlates with bone mineral density in human weight-bearing bone.\n",
      "A) Heatmap representation of RNAseq differentially expressed TEs analysis [Fragments Per Kilobase of transcript\n",
      "per Million (FPKM) fold change] in femoral bone from healthy (n=27), osteopenic (n=12), and osteoporotic (n=9)\n",
      "donors. B) Heatmap representation of RNAseq differentially expressed TEs analysis [Fragments Per Kilobase of\n",
      "transcript per Million (FPKM) fold change] in femoral bone with high BMD (FN T-score >-1) (n=27) and low BMD\n",
      "(FN T-score <-1) (n=21). C) Upper panels: pie charts showing the number and percentage of differentially expressed\n",
      "TEs subfamilies between high BMD and low BMD femoral bone. Lower panels: heatmap representation of RNAseq\n",
      "differentially expressed TEs analysis [Fragments Per Kilobase of transcript per Million (FPKM) fold change] in\n",
      "femoral bone with high BMD (n=27) and low BMD (n=21). One heatmap for each TEs order is shown. D) Upper\n",
      "panels: Heatmap representation of correlation analysis (P value<0.05) between TEs expression (FPKM) and local\n",
      "BMD (FN T-score) in femoral bone biopsies (n=48). Correlation between TE expression (FPKM values) and FN t-\n",
      "score was evaluated using Pearson method. Significant correlations (p-value < 0.05) were selected for the heatmap\n",
      "representation and arranged according to the Pearson correlation coefficient (r).  r > 0, positive correlations; r < 0,\n",
      "negative correlations. Lower panels: Pie charts showing the percentage of TEs subfamilies positively correlated to\n",
      "\n"
     ]
    }
   ],
   "source": [
    "# Loop through the pages int he document and return true if text is found\n",
    "docx_doc = pymupdf.open(DOCX_FILES[0])\n",
    "for page in docx_doc:\n",
    "    if page.search_for('Figure Legends'):\n",
    "        print('Text found on page')\n",
    "        print(page.get_text())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'expressed protein (adjusted p-value <0.05) in the exosomes of L1 compared to RFP transfected osteoblasts.\\n \\n'"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "page.get_text()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Splitting the text to find sections in the document\n",
    "\n",
    "To efficiently extract specific sections from your document using LangChain's PyMuPDFLoader, it's advisable to utilize a text splitter that aligns with the document's inherent structure. Given that you're working with a DOCX file, employing the MarkdownHeaderTextSplitter can be particularly effective, as it splits the text based on Markdown headers, preserving the logical organization of your document.\n",
    "\n",
    "Here's how you can implement this approach:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing ./data/docx/EMM-2023-19044.docx...\n",
      "[                                        ] (0/4[=                                       ] ( 1/4[==                                      ] ( 2/40[===                                     ] ( 3/4[====                                    ] ( 4/40[=====                                   ] ( 5/4[======                                  ] ( 6/40[=======                                 ] ( 7/4[========                                ] ( 8/40[=========                               ] ( 9/4[==========                              ] (10/40[===========                             ] (11/4[============                            ] (12/40[=============                           ] (13/4[==============                          ] (14/40[===============                         ] (15/4[================                        ] (16/40[=================                       ] (17/4[==================                      ] (18/40[===================                     ] (19/4[====================                    ] (20/40[=====================                   ] (21/4[======================                  ] (22/40[=======================                 ] (23/4[========================                ] (24/40[=========================               ] (25/4[==========================              ] (26/40[===========================             ] (27/4[============================            ] (28/40[=============================           ] (29/4[==============================          ] (30/40[===============================         ] (31/4[================================        ] (32/40[=================================       ] (33/4[==================================      ] (34/40[===================================     ] (35/4[====================================    ] (36/40[=====================================   ] (37/4[======================================  ] (38/40[======================================= ] (39/4[========================================] (40/40]\n",
      "****page_content='Research Article\n",
      "\n",
      "Title: Ependymal cell lineage reprogramming as a potential therapeutic intervention for\n",
      "\n",
      "hydrocephalus\n",
      "\n",
      "Authors: Konstantina Kaplani[1], Maria - Eleni Lalioti[1], Stella Vassalou[1], Georgia Lokka[1], Evangelia\n",
      "\n",
      "Parlapani[1], Georgios Kritikos[1], Zoi Lygerou[2], Stavros Taraviras[1,]\n",
      "\n",
      "Affiliations:\n",
      "\n",
      "1Department of Physiology, School of Medicine, University of Patras, Patras, Greece\n",
      "\n",
      "2Department of General Biology, School of Medicine, University of Patras, Patras, Greece\n",
      "\n",
      "Corresponding author: Stavros Taraviras\n",
      "\n",
      "Corresponding author’s address: Department of Physiology, School of Medicine, University of Patras,\n",
      "\n",
      "Asklepiou Street 1, Rio 26504, Patras, Greece\n",
      "\n",
      "Corresponding author’s phone and fax: +30 2610 997676\n",
      "\n",
      "Corresponding author’s e-mail address: [taraviras@med.upatras.gr](mailto:taraviras@med.upatras.gr)\n",
      "\n",
      "Running title: GemC1 and McIdas promote ependymal reprogramming.\n",
      "\n",
      "Keywords: ependymal cells, reprogramming, hydrocephalus, McIdas, GemC1\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "ABSTRACT\n",
      "\n",
      "Hydrocephalus is a common neurological condition, characterized by the excessive accumulation of\n",
      "\n",
      "cerebrospinal fluid in the cerebral ventricles. Primary treatments for hydrocephalus mainly involve\n",
      "\n",
      "neurosurgical cerebrospinal fluid diversion, which hold high morbidity and failure rates, highlighting the\n",
      "\n",
      "necessity for the discovery of novel therapeutic approaches. Although the pathophysiology of\n",
      "\n",
      "hydrocephalus is highly multifactorial, impaired function of the brain ependymal cells plays a\n",
      "\n",
      "fundamental role in hydrocephalus. Here we show that GemC1 and McIdas, key regulators of\n",
      "\n",
      "multiciliated ependymal cell fate determination, induce direct cellular reprogramming towards\n",
      "\n",
      "ependyma. Our study reveals that ectopic expression of GemC1 and McIdas reprograms cortical\n",
      "\n",
      "astrocytes and programs mouse embryonic stem cells into ependyma. McIdas is sufficient to establish\n",
      "\n",
      "functional activity in the reprogrammed astrocytes. Furthermore, we show that McIdas’ expression\n",
      "\n",
      "promotes ependymal cells regeneration in two different postnatal hydrocephalus mouse models: an\n",
      "\n",
      "intracranial hemorrhage and a genetic form of hydrocephalus and ameliorates the cytoarchitecture of the\n",
      "\n",
      "neurogenic niche. Our study provides evidence on the restoration of ependyma in animal models\n",
      "\n",
      "mimicking hydrocephalus that could be exploited towards future therapeutic interventions.\n",
      "\n",
      "2\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "INTRODUCTION\n",
      "\n",
      "Hydrocephalus is a neurological disorder defined by the abnormal accumulation of cerebrospinal fluid\n",
      "\n",
      "(CSF) within the brain ventricles, leading to increased head circumference (Schrander-Stumpel & Fryns,\n",
      "\n",
      "1998). The condition can arise as a primary clinical feature triggered by a genetic cause (congenital\n",
      "\n",
      "hydrocephalus) or secondary (acquired hydrocephalus) to other insults such as central nervous system\n",
      "\n",
      "(CNS) infections, tumors, trauma, or hemorrhage (Rekate, 2009; Kahle et al, 2016; Kousi & Katsanis,\n",
      "\n",
      "2016). Malfunction of the ependymal cells is a leading cause of the pathophysiological mechanism of\n",
      "\n",
      "hydrocephalus as these cells have a key role in CSF circulation and composition (Ji et al, 2022). In\n",
      "\n",
      "addition, hydrocephalus is linked with disorganization of the subventricular neurogenic niche’s\n",
      "\n",
      "cytoarchitecture and neurogenesis defects, as ependymal cells were shown to provide structural and\n",
      "\n",
      "functional support to the adult neural stem cells (Paez-Gonzalez et al, 2011; McAllister, James P. et al,\n",
      "\n",
      "2017; Rodríguez & Guerra, 2017).\n",
      "\n",
      "Under homeostatic conditions, ependymal cells form one of the largest epithelia of the brain, lying\n",
      "\n",
      "adjacent to the ventricular lumen and bear unique structural characteristics. The cell surface exposed to\n",
      "\n",
      "the ventricular lumen contains a patch of approximately 50 modified centrioles, called basal bodies,\n",
      "\n",
      "which are tiny microtubule-based organelles responsible for nucleating an equal number of motile cilia.\n",
      "\n",
      "Cilia are anchored οn their apical surface, beating in the same direction and in a coordinated fashion,\n",
      "\n",
      "thus contributing to efficient CSF circulation throughout the brain (Mirzadeh et al, 2008, 2010).\n",
      "\n",
      "Constant ciliary beating facilitates secreted molecules dispersion to other regions of the CNS. Such\n",
      "\n",
      "circulating molecules in the CSF influence neurogenesis and neuroblasts’ migration to the olfactory\n",
      "\n",
      "bulbs (Lim et al, 2000; Sawamoto et al, 2006).\n",
      "\n",
      "Ependymal cell differentiation is a multi-step process orchestrated by a tightly regulated transcriptional\n",
      "\n",
      "program which is responsible for activating numerous genes that promote amplification and docking of\n",
      "\n",
      "centrioles and the formation of cilia (Kyrousi et al, 2017; Arbi et al, 2018). We have previously provided\n",
      "\n",
      "evidence that the Geminin family proteins, GEMC1 and MCIDAS, are the earliest regulators for the cell\n",
      "\n",
      "fate commitment to the ependymal lineage (Kyrousi et al, 2015). GemC1 lies upstream of McIdas\n",
      "\n",
      "inducing its transcriptional activation, while both upregulate genes that are essential for centriole\n",
      "\n",
      "amplification and for promoting the downstream transcriptional machinery of multiciliogenesis (Kyrousi\n",
      "\n",
      "et al, 2015; Arbi et al, 2016; Lalioti et al, 2019a). GEMC1 and MCIDAS cooperate with E2F4/5\n",
      "\n",
      "3\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "transcription factors, to upregulate cMyb and Ccno, which are implicated in centriole amplification (Tan\n",
      "\n",
      "et al, 2013; Wallmeier et al, 2014; Funk et al, 2015; Arbi et al, 2018). In addition, we and others have\n",
      "\n",
      "provided evidence that GemC1 and McIdas drive the transcriptional activation of P73 and FOXJ1\n",
      "\n",
      "transcription factors, which are implicated in basal bodies docking and cilia motility (Kyrousi et al,\n",
      "\n",
      "2015; Arbi et al, 2016; Nemajerova et al, 2016; Marshall et al, 2016; Lalioti et al, 2019a).\n",
      "\n",
      "Given the multifaceted roles that ependymal cells have in hydrocephalus pathophysiology, restoration of\n",
      "\n",
      "ependymal cells could significantly contribute to hydrocephalus therapy. With the advancements in\n",
      "\n",
      "direct reprogramming-based strategies for conditions affecting the central nervous system (Bocchi et al,\n",
      "\n",
      "2022), it is plausible to anticipate that cellular reprogramming directed towards ependymal cells could\n",
      "\n",
      "represent a significant breakthrough in the treatment of hydrocephalus. In the present study we showed\n",
      "\n",
      "that ectopic expression of GemC1 or McIdas promotes programming of mouse pluripotent embryonic\n",
      "\n",
      "stem cells and reprogramming of cortical astrocytes into ependymal cells, with McIdas being more\n",
      "\n",
      "efficient on establishing functional motile cilia in reprogrammed astrocytes. In addition, forced\n",
      "\n",
      "expression of McIdas in both a genetic and an acquired hydrocephalus mouse model promoted in vivo\n",
      "\n",
      "reprogramming of cells residing in the ventricular walls of the diseased brain into functional ependymal\n",
      "\n",
      "cells. Importantly, reprogrammed ependymal cells were able to beat their cilia in a coordinated fashion\n",
      "\n",
      "and formed pinwheel structures together with neural stem cells in hydrocephalic mice, revealing their\n",
      "\n",
      "potential to regenerate the architecture of the neurogenic niche, which is disrupted in hydrocephalus.\n",
      "\n",
      "Collectively, our data suggest that GEMC1 and MCIDAS can orchestrate the transcriptional program of\n",
      "\n",
      "multiciliogenesis establishing ependymal cell fate and differentiation, providing proof of principle\n",
      "\n",
      "evidence that restoration of ependymal cells could potentially contribute to hydrocephalus management.\n",
      "\n",
      "4\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "RESULTS\n",
      "\n",
      "GemC1 and McIdas promote programming of mouse embryonic stem cells and reprogramming of\n",
      "\n",
      "cortical astrocytes into ependymal cells\n",
      "\n",
      "GemC1 and McIdas are the most upstream regulators driving multiciliogenesis in vertebrate organisms\n",
      "\n",
      "(Stubbs et al, 2012; Ma et al, 2014; Boon et al, 2014; Kyrousi et al, 2015; Terré et al, 2016; Arbi et al,\n",
      "\n",
      "2016). We therefore postulated that GemC1 and McIdas would be capable of inducing stem cell\n",
      "\n",
      "programming towards multiciliated ependymal cells. To examine this hypothesis, mouse embryonic\n",
      "\n",
      "stem cells (mESCs) were transfected with plasmids encoding McIdas or GemC1 in conjunction with\n",
      "\n",
      "GFP (referred to as McIdas and GemC1 respectively), or GFP alone as a control. Transfected mESCs\n",
      "\n",
      "were cultured for four days in the absence of Leukemia Inhibitory Factor (LIF) and subjected to\n",
      "\n",
      "immunostaining for FoxJ1, a key transcription factor for the differentiation of ependymal cells, and\n",
      "\n",
      "Pericentrin which marks nascent basal bodies (Fig EV1A). Our analysis showed that 87% of McIdas\n",
      "and 39% of GemC1-overexpressing mESCs were expressing FoxJ1, while only 1% FoxJ1 positive cells\n",
      "\n",
      "were detected in the control (Fig EV1B). In addition, we identified that 90% of McIdas- and 59% of\n",
      "\n",
      "GemC1-overexpressing mESCs showed accumulation of Pericentrin staining (Fig EV1C), thus\n",
      "\n",
      "presented multiple nascent basal bodies, corresponding to differentiating ependymal cells. Furthermore,\n",
      "\n",
      "we examined whether McIdas and GemC1-overexpressing mESCs acquire multiple cilia, a unique\n",
      "\n",
      "structural characteristic of the ependymal cells. Double immunofluorescence experiments were\n",
      "\n",
      "performed six days following mESCs transfection using anti-Pericentrin and anti-acetylated-a-tubulin\n",
      "\n",
      "antibodies to mark the basal bodies and cilia, respectively (Fig EV1D). Our analysis revealed that 66%\n",
      "\n",
      "of McIdas- and 21% of GemC1- overexpressing mESCs develop multiple cilia arising from numerous\n",
      "\n",
      "basal bodies. On the contrary, control mESCs develop neither multiple basal bodies nor cilia (Fig\n",
      "\n",
      "EV1D, E). Our data show that ectopic expression of McIdas and GemC1 programs mESCs into\n",
      "\n",
      "5\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "multiciliated ependymal cells.\n",
      "\n",
      "Next, we sought out to explore the reprogramming potential of McIdas and GemC1 in astrocytes that\n",
      "\n",
      "under physiological conditions do not differentiate into ependymal cells. Astrocytes were isolated from\n",
      "\n",
      "the cortex of neonatal mice and were subsequently infected with lentiviruses expressing GFP-McIdas or\n",
      "\n",
      "GFP-GemC1 (referred to as McIdas and GemC1, respectively), while lentiviruses expressing GFP alone\n",
      "\n",
      "were used as control. Cells were subsequently cultured under differentiating conditions and\n",
      "\n",
      "immunofluorescence experiments were conducted to assess the expression of known ependymal and\n",
      "\n",
      "astrocytic markers (Fig. 1A). Ectopic expression of both McIdas and GemC1 resulted in P73 expression,\n",
      "\n",
      "a marker of the early steps of ependymogenesis, in 37% and 36% of McIdas- and GemC1- infected\n",
      "\n",
      "cells, respectively, while in the control condition only 4% of the transduced cells was expressing P73\n",
      "\n",
      "(Fig. 1B, C, and EV 2A, B). Notably, concurrent changes in the expression pattern of the astrocytic\n",
      "\n",
      "marker S100β were observed in approximately 20% of McIdas- and 18% of GemC1- infected\n",
      "\n",
      "astrocytes, as S100β expression was detected around their cell body, which is characteristic for\n",
      "\n",
      "ependymal cells (Mirzadeh et al, 2008) (Fig. 1B-D and EV. 2A-C). On the contrary, control astrocytes\n",
      "\n",
      "retained a cytoplasmic expression of the S100β protein, a typical feature of astrocytes. Overall, our data\n",
      "\n",
      "suggest that ectopic expression of McIdas and GemC1 can lead to the acquisition of the ependymal cell\n",
      "\n",
      "fate.\n",
      "\n",
      "Subsequently, we examined whether McIdas- and GemC1-transduced astrocytes successfully\n",
      "\n",
      "differentiate into ependymal cells. We performed immunofluorescence experiments using specific\n",
      "\n",
      "antibodies recognizing FoxJ1 and Pericentrin, to identify astrocytes that acquired the ependymal fate and\n",
      "\n",
      "successfully managed to multiply their basal bodies, respectively. Our analysis showed that fourteen\n",
      "\n",
      "days post infection 49% of McIdas- and 19% of GemC1- infected astrocytes were expressing FoxJ1\n",
      "\n",
      "(Fig. 1E, F and EV2D, E). In addition, 28% of the McIdas- and 12% of GemC1- infected astrocytes\n",
      "\n",
      "6\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "possessed multiple basal bodies, indicated by the accumulation of Pericentrin signal (Fig. 1G and\n",
      "\n",
      "EV2F). Moreover, expression of a GFP lentivirus did not result into FoxJ1 expression nor accumulation\n",
      "\n",
      "of Pericentrin.\n",
      "\n",
      "In addition, we examined the ability of McIdas and GemC1 to repress the astrocytic identity of the\n",
      "\n",
      "transduced astrocytes. Towards this direction, the combined expression of the astrocytic markers GFAP\n",
      "\n",
      "and S100β was assessed through immunofluorescence experiments at fourteen days post infection. Our\n",
      "\n",
      "analysis revealed that McIdas ectopic expression resulted to 23% of GFAP+/S100β+ double positive\n",
      "\n",
      "cells compared to 31% in the control condition (Appendix Fig. S1A, B). We did not observe statistically\n",
      "\n",
      "significant change in the percentage of GFAP+/S100β+ double positive cells following GemC1 ectopic\n",
      "\n",
      "expression (Appendix Fig. S1C). In addition, we examined the percentage of infected cells that were\n",
      "\n",
      "GFAP- and reveal S100β expression around the cell body, which correspond to ependymal cells. We\n",
      "\n",
      "showed that 28% McIdas- and 22% GemC1- infected astrocytes present this expression pattern\n",
      "\n",
      "(Appendix Fig. S1A, B), which is in accordance with our analysis performed in earlier stages of\n",
      "\n",
      "reprogramming (Fig. 1D). Our results indicate that a significant percentage of the infected astrocytes are\n",
      "\n",
      "reprogrammed to the ependymal lineage and possess unique ependymal morphological characteristics.\n",
      "\n",
      "Moreover, we showed that McIdas is more potent in the downregulation of the astrocytic identity of the\n",
      "\n",
      "transduced cells.\n",
      "\n",
      "GemC1 is positioned upstream of McIdas in the molecular pathway governing ependymogenesis, as it\n",
      "\n",
      "regulates the transcriptional activation of McIdas and controls the initial fate commitment towards the\n",
      "\n",
      "ependymal lineage (Kyrousi et al, 2015, 2017). Our results suggest that GemC1 was less effective than\n",
      "\n",
      "McIdas in reprogramming of astrocytes. We have therefore examined whether ectopic expression of\n",
      "\n",
      "GemC1 in astrocytes would not induce McIdas expression at sufficient levels. To test this hypothesis, we\n",
      "\n",
      "conducted immunofluorescence experiments using an McIdas specific antibody and quantified McIdas\n",
      "\n",
      "7\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "fluorescence in McIdas- and GemC1-infected astrocytes seven days post infection (Appendix Fig. S2A).\n",
      "\n",
      "Our results revealed that McIdas’ ectopic expression led to higher expression levels compared to GemC1\n",
      "\n",
      "(Appendix Fig. S2B), providing a potential mechanism for their differential ability to induce\n",
      "\n",
      "reprogramming.\n",
      "\n",
      "Given the greater capacity of McIdas to induce the ependymal differentiation in astrocytes compared to\n",
      "\n",
      "GemC1, we hypothesized that McIdas would reprogram cortical astrocytes into fully mature ependymal\n",
      "\n",
      "cells carrying multiple basal bodies nucleating motile cilia. To address this point, we initially performed\n",
      "\n",
      "immunofluorescence experiments in transduced astrocytes twenty-two days post infection using\n",
      "\n",
      "antibodies against Centriolin (also known as Cep110), a known centrosomal marker, and Meig1 which\n",
      "\n",
      "was recently characterized as ependymal cell marker (MacDonald et al, 2021). We detected\n",
      "\n",
      "accumulation of both Centriolin (Appendix Fig.S3A) and Meig1 (Appendix Fig.S3B) signal in McIdas\n",
      "infected astrocytes, verifying the generation of multiple basal bodies upon McIdas’ ectopic expression.\n",
      "\n",
      "Next, we performed immunofluorescence experiments making use of antibodies against acetylated a\n",
      "tubulin, which stains ciliary axonemes, and the astrocytic marker glial fibrillary acidic protein (GFAP).\n",
      "\n",
      "The ectopic expression of McIdas resulted in the downregulation of GFAP expression and the generation\n",
      "\n",
      "of multiple cilia on the surface of the reprogrammed astrocytes, revealing the loss of the astrocytic\n",
      "\n",
      "identity and the establishment of the ependymal differentiation (Fig. 2A). Our analysis showed that 18%\n",
      "\n",
      "of McIdas- infected astrocytes were reprogrammed into multiciliated ependymal cells based on\n",
      "\n",
      "acetylated-a-tubulin staining (Fig. 2C), which was consistent with the percentage of McIdas- infected\n",
      "\n",
      "cells that downregulate the expression of the astrocytic marker GFAP (Fig. 2B). Additionally, live\n",
      "\n",
      "imaging microscopy experiments were performed in transduced astrocytes twenty-two days post\n",
      "\n",
      "infection to address the ability of the reprogrammed astrocytes multicilia to beat (Movies EV1-4). Using\n",
      "\n",
      "high-speed video microscopy, we acquired fast video recordings from GFP (Movies EV1-2) and\n",
      "\n",
      "8\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "McIdas- infected astrocytes (Movies EV3-4), which revealed the presence of ciliary motility in McIdas\n",
      "infected cells as opposed to the absence of motility in GFP- infected cells. Importantly, our analysis\n",
      "\n",
      "showed that McIdas expression established cilia movement that was sufficient to propel fluorescent\n",
      "\n",
      "particles added to the imaging medium, highlighting the functionality of the ependymal cells and their\n",
      "\n",
      "capability to create a fluid flow. On the contrary, astrocytes infected with a control virus were unable to\n",
      "\n",
      "propel the fluorescent particles, as were only observed to have a rather stable forward and backward\n",
      "\n",
      "motion (Fig. 2D).\n",
      "\n",
      "Our findings show that GemC1 and McIdas promote the early steps of the multiciliogenesis program in\n",
      "\n",
      "both embryonic stem cells and astrocytes. Importantly, McIdas has a greater capacity for eliciting direct\n",
      "\n",
      "reprogramming into functional ependymal cells.\n",
      "\n",
      "McIdas induces ependymal cells regeneration in a mouse model of intracranial hemorrhage\n",
      "\n",
      "hydrocephalus\n",
      "\n",
      "In both human and murine hydrocephalus, the disrupted ependymal cell population is replaced by\n",
      "\n",
      "astrocytes. However, this astrocytic scarring along the denuded ventricular wall fails to restore the\n",
      "\n",
      "ependymal functions (Sival et al, 2011; Roales-Buján et al, 2012; Guerra et al, 2015; McAllister, James\n",
      "\n",
      "P. et al, 2017). Based on our finding that McIdas can successfully reprogram cortical astrocytes into\n",
      "\n",
      "functional ependymal cells, we examined whether McIdas could induce direct reprogramming of\n",
      "\n",
      "periventricular cells in hydrocephalic models. We used an established mouse model of intracranial\n",
      "\n",
      "hemorrhage hydrocephalus induced by the elevated concentration of Lysophosphatidic Acid (LPA) in\n",
      "\n",
      "the brain (Yung et al, 2015; Lummis et al, 2019), which closely mirrors the most common type of\n",
      "\n",
      "human hydrocephalus.\n",
      "\n",
      "9\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "Based on previously established protocols (Yung et al, 2011; Lummis et al, 2019), postnatal day five\n",
      "\n",
      "(P5) mice received intracranial injections of LPA and mouse brains were examined two days later to\n",
      "\n",
      "evaluate hydrocephalus occurrence. Although no obvious differences were observed in the size of LPA\n",
      "treated brains macroscopically (Fig. EV3A), coronal brain sections revealed ventricular dilation in LPA\n",
      "injected animals compared to non-injected ones (Fig. EV3B), confirming the development of\n",
      "\n",
      "hydrocephalus. In addition, acetylated-a-tubulin immunofluorescence revealed the ciliary disruption in\n",
      "\n",
      "ependymal cells alongside the ventricular walls upon LPA administration (Fig. EV3C), in line with\n",
      "\n",
      "previous studies (Lummis et al, 2019). On the contrary, ependymal cells with multiple cilia covered the\n",
      "\n",
      "entire ventricular walls in control animals (Fig. EV3C). Given that the ventricular walls of the LPA\n",
      "injected hydrocephalic mice were denuded from ependymal cells we wished to determine their cellular\n",
      "\n",
      "composition. We performed immunofluorescence with an antibody against the astrocytic marker glial\n",
      "\n",
      "fibrillary acidic protein (GFAP). Our analysis showed that GFAP positive cells with long processes,\n",
      "\n",
      "which likely corresponded to reactive astrocytes, covered the walls of the lateral ventricles in regions\n",
      "\n",
      "where ependymal cells localize under physiological conditions (Fig. EV3C), in line with previous\n",
      "\n",
      "findings (Roales-Buján et al, 2012).\n",
      "\n",
      "To assess whether McIdas can induce direct reprogramming into ependyma in LPA-induced\n",
      "\n",
      "hydrocephalus, GFP-McIdas (McIdas) or GFP expressing plasmids were electroporated onto P7 brains\n",
      "\n",
      "isolated from LPA-treated mice by targeting the dorsolateral wall of one lateral ventricle to introduce\n",
      "\n",
      "plasmids in cells located at periventricular regions. Subsequently, thick coronal brain sections at the\n",
      "\n",
      "level of the lateral ventricles were cultured and immunofluorescence was performed using established\n",
      "\n",
      "ependymal markers (Fig. 3A). Five days post electroporation, 77% of the McIdas-electroporated cells\n",
      "\n",
      "expressed P73, as opposed to 13% of the GFP-electroporated cells, suggesting that astrocytes that\n",
      "\n",
      "replaced damaged ependymal cells in LPA treated hydrocephalic mouse models acquired the ependymal\n",
      "\n",
      "10\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "fate (Fig. 3B).\n",
      "\n",
      "As ependymal cells normally require approximately two weeks until they acquire their mature\n",
      "\n",
      "characteristics in vivo, we followed differentiation of electroporated cells at later time points and\n",
      "\n",
      "examined whether they differentiate into fully mature multiciliated cells. Towards this direction, we\n",
      "\n",
      "performed immunolabeling experiments making use of antibodies recognizing Pericentrin to mark basal\n",
      "\n",
      "bodies and acetylated-a-tubulin to detect cilia (Fig. 3C). Nine days post electroporation 67% of McIdas\n",
      "\n",
      "electroporated cells carried multiple basal bodies (Fig. 3D), while 16% of McIdas-electroporated cells\n",
      "\n",
      "carried multiple cilia as revealed by acetylated-a-tubulin immunostaining (Fig. 3E). Basal bodies\n",
      "\n",
      "amplification or multiple cilia formation was not detected in any of the control cells, electroporated with\n",
      "\n",
      "a GFP expressing plasmid.\n",
      "\n",
      "Collectively, our data show that McIdas forced expression in brain periventricular cells has the potential\n",
      "\n",
      "to regenerate the ependymal cells in a mouse model of intracranial hemorrhage hydrocephalus.\n",
      "\n",
      "McIdas promotes in vivo reprogramming towards functional ependymal cells in a model of\n",
      "\n",
      "congenital hydrocephalus\n",
      "\n",
      "We have recently described that genetic mutations in the human and mouse GemC1 gene lead to\n",
      "\n",
      "hydrocephalus (Lalioti et al, 2019b). Moreover, we have shown that radial glial cells lacking functional\n",
      "\n",
      "GemC1 fail to commit to the ependymal lineage and therefore do not develop ependymal cells during\n",
      "\n",
      "embryogenesis. We took advantage of the GemC1-knockout genetic mouse model of hydrocephalus we\n",
      "\n",
      "have generated to investigate whether reprogramming towards the ependymal cells could be achieved in\n",
      "\n",
      "a model of congenital hydrocephalus.\n",
      "\n",
      "We initially examined whether McIdas could program radial glial cells derived from GemC1-knockout\n",
      "\n",
      "11\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "hydrocephalic mice into ependymal cells ex vivo. Postnatal radial glial cells isolated from GemC1\n",
      "knockout mice were infected with lentiviruses expressing either a GFP-McIdas (McIdas) fusion protein\n",
      "\n",
      "or GFP as a control. Infected cells were analyzed at different time points for the expression of known\n",
      "\n",
      "ependymal markers by immunofluorescence experiments, using specific antibodies recognizing FOXJ1,\n",
      "\n",
      "Pericentrin and acetylated-a-tubulin proteins (Fig. EV4A). At five days post infection, 86% of the\n",
      "\n",
      "McIdas-infected cells expressed FoxJ1 and showed accumulation of multiple basal bodies, based on\n",
      "\n",
      "Pericentrin immunostaining (Fig. EV4B, C). Moreover, our analysis revealed that 97% of the McIdas\n",
      "\n",
      "transduced cells were multiciliated 15 days following infection, based on acetylated-a-tubulin and\n",
      "\n",
      "Pericentrin staining (Fig. EV3D, E). By contrast, in GFP transduced cells neither FoxJ1 expression nor\n",
      "\n",
      "multiple basal bodies and cilia were observed. In addition, similar experiments were performed by\n",
      "\n",
      "ectopically expressing P73 in postnatal radial glial cells isolated from GemC1-knockout mice. Ectopic\n",
      "\n",
      "expression of P73 was not sufficient to induce FoxJ1 expression nor multiple basal bodies or multiple\n",
      "\n",
      "cilia formation in GemC1-knockout radial glial cells (Fig. EV4A-E). These results show that cells\n",
      "\n",
      "derived from a genetic model of hydrocephalus can be programmed ex vivo into multiciliated ependymal\n",
      "\n",
      "cells.\n",
      "\n",
      "In order to examine whether McIdas promotes cellular reprogramming towards the ependymal cell\n",
      "\n",
      "lineage in vivo, we have used our genetic mouse model of hydrocephalus which was previously\n",
      "\n",
      "established (Lalioti et al, 2019b). Plasmid vectors encoding McIdas in conjunction with GFP or GFP\n",
      "\n",
      "alone were electroporated into one lateral ventricle of newborn GemC1-knockout mouse brains targeting\n",
      "\n",
      "periventricular cells which could be either neural progenitor cells or reactive astrocytes that have\n",
      "\n",
      "replaced the lost ependymal cell population (Roales-Buján et al, 2012; Gonzalez-Cano et al, 2016; Abdi\n",
      "\n",
      "et al, 2018; Lalioti et al, 2019b). Reprogramming towards the ependymal cells was evaluated through\n",
      "\n",
      "immunofluorescence, using molecular markers that determine different stages of ependymogenesis.\n",
      "\n",
      "12\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "Interestingly, four days post electroporation 61% and 58% of McIdas-electroporated cells were\n",
      "\n",
      "immunopositive for the ependymal markers P73 (Fig. 4A, B) and FoxJ1, respectively (Fig. EV5A, B).\n",
      "\n",
      "The expression of P73 and FoxJ1 was not detected in control animals, in agreement to what was\n",
      "\n",
      "previously described (Lalioti et al, 2019b), as GemC1-knockout mice lack ependymal cells. We also\n",
      "\n",
      "showed that McIdas further promoted the differentiation of the electroporated ependymal cells as basal\n",
      "\n",
      "bodies amplification was observed, based on Pericentrin immunofluorescence (Fig. 4C). More\n",
      "\n",
      "specifically, upon McIdas overexpression 44% of the electroporated cells acquired multiple basal bodies\n",
      "\n",
      "(Fig. 4D). In addition, immunofluorescence against acetylated-a-tubulin revealed that McIdas\n",
      "\n",
      "overexpression resulted in the formation of cilia in a significant number of cells, corresponding to 7% of\n",
      "\n",
      "McIdas electroporated cells (Fig. 4E). Noticeably, ciliated cells were located close to the ventricles,\n",
      "\n",
      "resembling normal mature multiciliated ependymal cells, which form a single-layered ciliated\n",
      "\n",
      "epithelium adjacent to the brain ventricle.\n",
      "\n",
      "In order to examine whether the cilia that were generated upon McIdas ectopic expression were motile,\n",
      "\n",
      "thick coronal brain sections from GemC1-knockout mice covering the lateral brain ventricles were\n",
      "\n",
      "obtained and used in live imaging microscopy experiments (Fig. 5A). Electroporated cells were\n",
      "\n",
      "identified under a FITC filter and their cilia beating capacity was examined in fast video records. A\n",
      "\n",
      "proportion of cells with multiple beating cilia that were GFP-negative in sections derived from McIdas\n",
      "electroporated brains were observed. We have previously shown that GemC1-knockout mice are unable\n",
      "\n",
      "to generate ependymal cells (Lalioti et al, 2019b), therefore cells carrying multiple beating cilia that do\n",
      "\n",
      "not appear GFP-positive were possibly observed due to the loss of the transgene because of its transient\n",
      "\n",
      "expression and were thus undetectable under the FITC filter. In addition to this, we cannot exclude the\n",
      "\n",
      "possibility of dilution of the GFP signal due to continued proliferation. In agreement with what was\n",
      "\n",
      "previously described cilia were not detected in brain sections following GFP overexpression. However,\n",
      "\n",
      "13\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "McIdas electroporated cells exhibited functional beating cilia (Fig. 5B and Movies EV5, 6).\n",
      "\n",
      "Kymographs were generated (Fig. 5C) following previously described methods by Francis et al. (Francis\n",
      "\n",
      "& Lo, 2013) and ciliary beating frequency of McIdas induced ependymal cells was estimated to have a\n",
      "\n",
      "mean value of 10 Hz (Fig. 5D).\n",
      "\n",
      "Our findings show that McIdas induces ependymal cells restoration in the diseased hydrocephalic brain.\n",
      "\n",
      "McIdas expression contributes to the restoration of the subventricular zone niche cytoarchitecture\n",
      "\n",
      "in hydrocephalic mice.\n",
      "\n",
      "The adult subventricular zone niche has a well-defined architecture formed by pinwheel-like structures.\n",
      "\n",
      "These structures are assembled by the large apical domain of ependymal cells which surround the\n",
      "\n",
      "smaller apical domain of adult neural stem cells (aNSCs) and constitute a unique micro-environment for\n",
      "\n",
      "coordinating adult neurogenesis (Mirzadeh et al, 2008; Paez-Gonzalez et al, 2011; Kokovay et al, 2012).\n",
      "\n",
      "To assess whether McIdas is able to restore the subventricular zone niche’s cytoarchitecture in\n",
      "\n",
      "hydrocephalic mice, whole-mount immunofluorescence was performed on ventricular walls of the\n",
      "\n",
      "lateral ventricles of GemC1-knockout hydrocephalic mice at P7-P9, previously electroporated with an\n",
      "\n",
      "McIdas-expressing plasmid or GFP alone. Specific antibodies against VCAM1 and GFAP, both markers\n",
      "\n",
      "of aNSCs, and β-catenin to delineate cell boundaries were used. Our analysis showed that McIdas\n",
      "electroporated cells had a larger apical surface compared to control GFP-electroporated cells, which is\n",
      "\n",
      "characteristic for multiciliated ependymal cells. In addition, McIdas-electroporated cells were\n",
      "\n",
      "surrounding VCAM1- (Fig. 6A) and GFAP-immunopositive cells (Fig. 6B) reminiscent of pinwheel\n",
      "\n",
      "structures. On the contrary, control GFP-electroporated cells had a small apical surface and pinwheel\n",
      "\n",
      "structures were not detectable as it was previously described (Lalioti et al, 2019b). We also quantified\n",
      "\n",
      "the number of electroporated cells that were detected in pinwheel structures, revealing that 18% of\n",
      "\n",
      "14\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "McIdas-expressing cells were able to form pinwheel structures as opposed to GFP-electroporated cells\n",
      "\n",
      "that were not detected in pinwheels (Fig. 6C). Our data suggest that McIdas expression is sufficient to\n",
      "\n",
      "instruct the regeneration of ependymal cells in hydrocephalic mice which have the potential to re-build\n",
      "\n",
      "the lost architecture of the subventricular zone niche.\n",
      "\n",
      "DISCUSSION\n",
      "\n",
      "During mammalian brain development neural progenitor cells transition from an uncommitted to a\n",
      "\n",
      "restricted state, through a process that is influenced by lineage-specific gene expression programs,\n",
      "\n",
      "resulting to terminally differentiated cells. Reprogramming allows the conversion of differentiated cells\n",
      "\n",
      "from one lineage to another type of differentiated cell. Efficient reprogramming is highly dependent on\n",
      "\n",
      "the selection of pioneer factors related to the desired cell type. Here, we demonstrate that GEMC1 and\n",
      "\n",
      "MCIDAS, members of the geminin family, constitute potent reprogramming factors towards the\n",
      "\n",
      "ependymal fate both ex vivo and in vivo.\n",
      "\n",
      "GemC1 and McIdas drive the cell fate commitment of a subpopulation of brain radial glial cells towards\n",
      "\n",
      "multiciliated ependymal cells by coordinating gene expression of essential factors for ependymal cell\n",
      "\n",
      "differentiation (Kyrousi et al, 2015). In addition, we have recently provided evidence that GemC1\n",
      "\n",
      "expression is necessary and sufficient to establish the appropriate chromatin organization at multiple loci\n",
      "\n",
      "for ependymal cells differentiation (Lalioti et al, 2019b). In light of this evidence, we hypothesized that\n",
      "\n",
      "GemC1 and McIdas could promote direct reprogramming towards the ependymal lineage in different\n",
      "\n",
      "cellular populations under physiology and disease. We initially demonstrated that both GemC1 and\n",
      "\n",
      "McIdas overexpression were able to trigger the differentiation of mESCs into ependymal cells, as\n",
      "\n",
      "evidenced by the upregulation of key factors of ependymogenesis and the generation of multiple cilia in\n",
      "\n",
      "15\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "the GemC1- and McIdas overexpressing cells. We also showed that GemC1 and McIdas overexpression\n",
      "\n",
      "induce the expression of P73 and FoxJ1, two well-established transcription factors of the molecular\n",
      "\n",
      "pathway of multiciliogenesis, in cultured cortical astrocytes. Importantly, we demonstrated that GemC1\n",
      "\n",
      "and McIdas activate the ependymal fate and induce the formation of multiple basal bodies and cilia in\n",
      "\n",
      "the reprogrammed cells. The ectopic expression of GemC1 resulted in a lower percentage of cells with\n",
      "\n",
      "multiple basal bodies, a unique structural feature of ependymal cells, revealing a differential\n",
      "\n",
      "reprogramming potential of the two factors. We showed that GemC1 does not induce sufficient levels of\n",
      "\n",
      "McIdas expression, that could contribute to the reduced ability of GemC1 to execute as efficiently the\n",
      "\n",
      "ependymal differentiation program. Moreover, using video microscopy experiments we showed that\n",
      "\n",
      "McIdas ectopic expression was sufficient to reprogram cortical astrocytes into fully functional\n",
      "\n",
      "multiciliated ependymal cells, as reprogrammed cells possessed multiple beating cilia that were able to\n",
      "\n",
      "beat in a co-ordinated way and generate fluid flow. In addition, McIdas overexpression was sufficient to\n",
      "\n",
      "induce the ependymal fate in radial glial cells derived from the brain of hydrocephalic mice in great\n",
      "\n",
      "contrast to P73, further highlighting McIdas’ ability to induce the ependymal cell fate. The stem cell\n",
      "\n",
      "properties of the mESCs and radial glial cells used in our study are likely to facilitate the adoption of the\n",
      "\n",
      "ependymal fate. Since this aligns with their differentiation spectrum their conversion to ependyma could\n",
      "\n",
      "be considered as differentiation into the ependymal cell lineage.\n",
      "\n",
      "It has been shown that in human and experimental hydrocephalus, disrupted ependyma is replaced by\n",
      "\n",
      "astroglial cells (Sival et al, 2011; Roales-Buján et al, 2012; Guerra et al, 2015; McAllister, James P. et al\n",
      "\n",
      ", 2017). This astrocytic scarring along the denuded ventricular wall contributes to the pathophysiology\n",
      "\n",
      "of the disease as astrocytes do not restore ependymal functions; as they do not display cilia, produce\n",
      "\n",
      "appropriate trophic signals, or facilitate the production of neurons (Sival et al, 2011; Roales-Buján et al,\n",
      "\n",
      "2012). Therefore, our strategy revealing the successful conversion of isolated cortical astrocytes into\n",
      "\n",
      "16\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "ependymal cells is of particular importance for treatment of hydrocephalus.\n",
      "\n",
      "To investigate whether McIdas would also reprogram cells in vivo, we used a mouse model of congenital\n",
      "\n",
      "hydrocephalus in which GemC1 has been inactivated, and a model of acquired hydrocephalus, triggered\n",
      "\n",
      "by intracranial hemorrhage. It was previously shown that GemC1 deficiency in mice results in the loss\n",
      "\n",
      "of the ependymal cells due to failure on activating the molecular program for the cell fate commitment\n",
      "\n",
      "towards the ependymal lineage (Terré et al, 2016; Lalioti et al, 2019b). Moreover, GemC1 deficiency\n",
      "\n",
      "has been recently associated with congenital hydrocephalus in both humans and mice (Lalioti et al,\n",
      "\n",
      "2019b). In this study, we showed that McIdas forced expression induces the upregulation of P73 and\n",
      "\n",
      "FoxJ1 in brain periventricular cells, which are likely to be neural progenitor cells or/and reactive\n",
      "\n",
      "astrocytes that have replaced the ependymal cell population in the hydrocephalic brain. The activation of\n",
      "\n",
      "the P73 and FoxJ1 genes uncovered the ability of McIdas to promote the initial steps of the ependymal\n",
      "\n",
      "cell’s differentiation program in the hydrocephalic brain. We also showed that McIdas-overexpressing\n",
      "\n",
      "cells generated multiple basal bodies and cilia, both of which are unique features of mature multiciliated\n",
      "\n",
      "ependymal cells, thus revealing the successful reprogramming towards ependyma. Importantly, the\n",
      "\n",
      "multiple cilia of the reprogrammed cells presented functional activity, as we were able to detect beating\n",
      "\n",
      "cilia upon in vivo overexpression of McIdas in the hydrocephalic mice. Using the LPA-induced\n",
      "\n",
      "intracranial hydrocephalus mouse model, we demonstrated McIdas ability to convert periventricular\n",
      "\n",
      "cells into ependymal cells. Previous study has shown that LPA causes substantial depletion of the\n",
      "\n",
      "ependymal monolayer, loss of ependymal cilia and subsequent ependymal apoptotic cell death (Lummis\n",
      "\n",
      "et al, 2019). In line with these findings, we did not observe multiciliated ependymal cells in\n",
      "\n",
      "periventricular regions after LPA treatment. Instead, we observed GFAP-positive cells, likely reactive\n",
      "\n",
      "astrocytes, covering the lateral ventricle walls where ependymal cells typically reside under normal\n",
      "\n",
      "conditions, as it has been previously shown in hydrocephalic mice (Páez et al, 2007; Roales-Buján et al,\n",
      "\n",
      "17\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "2012). Therefore, it is highly likely that McIdas reprograms astrocytes into ependymal cells in LPA\n",
      "treated mice. However, we cannot rule out the possibility that McIdas might also enhance the survival of\n",
      "\n",
      "a subset of ependymal cells resistant to LPA-induced apoptosis.​\n",
      "\n",
      "Collectively, in our experiments we employed diverse cellular systems and technical approaches,\n",
      "\n",
      "including plasmid and lentiviral vectors, to induce the overexpression of McIdas and GemC1 suggesting\n",
      "\n",
      "that both GemC1 and McIdas can induce the programming or reprogramming of different cellular\n",
      "\n",
      "populations irrespectively of the cell system and vector used. While our approach in the genetic mouse\n",
      "\n",
      "model of hydrocephalus and the LPA-induced intracranial hydrocephalus likely targets periventricular\n",
      "\n",
      "cells, which are neural progenitor cells or/and reactive astrocytes, we cannot rule out the possibility that\n",
      "\n",
      "other cellular populations might also be targeted. Recent evidence from reprogramming experiments in\n",
      "\n",
      "the mouse retina and the brain indicates that the insert sequences of overexpression constructs can alter\n",
      "\n",
      "the specificity of the construct's promoter, introducing a bias toward the initial cell population being\n",
      "\n",
      "targeted (Wang et al, 2021; Le et al, 2022). Additionally, future genetic lineage analysis using transgenic\n",
      "\n",
      "mice to label neural progenitors and/or astrocytes would be needed to clarify which cell population is\n",
      "\n",
      "differentiated into ependymal cells in our approach. One of the major clinical features in human patients\n",
      "\n",
      "with hydrocephalus is the ventricular zone disruption phenomenon, characterized by the loss of neural\n",
      "\n",
      "stem cells and ependyma (McAllister, James P. et al, 2017). Of note, sustained neuron generation\n",
      "\n",
      "depends on the maintenance of the proper cytoarchitecture of the Ventricular-Subventricular Zone (V\n",
      "SVZ) niche of the brain, which requires preserving the structure of the ependymal cell monolayer intact\n",
      "\n",
      "(Paez-Gonzalez et al, 2011). In the present study, we targeted V-SVZ-resident cells in the hydrocephalic\n",
      "\n",
      "brain in which McIdas overexpression was induced and observed that McIdas-overexpressing cells\n",
      "\n",
      "populated areas of the ventricular zone niche next to the lateral ventricles, where ependymal cells\n",
      "\n",
      "normally reside under homeostasis. We have used SVZ electroporation to target the periventricular cells\n",
      "\n",
      "18\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "of one lateral ventricle of the hydrocephalic brain of GemC1-knockout mice, which results to a limited\n",
      "\n",
      "number of electroporated cells. Therefore, following this experimental approach, the small number of\n",
      "\n",
      "ependymal cells which is generated is not possible to contribute to a significant reduction in the volume\n",
      "\n",
      "of the lateral ventricles as a readout of resolved or reduced hydrocephalus. It has been described that\n",
      "\n",
      "disruption of the ventricular/subventricular zone (V/SVZ) is a critical and consistent factor in the\n",
      "\n",
      "pathophysiology of hydrocephalus (McAllister, James P. et al, 2017; Jiménez et al, 2014). In our study\n",
      "\n",
      "we showed that McIdas ectopic expression contributes to the restoration of the lost cytoarchitecture of\n",
      "\n",
      "the niche in hydrocephalic mice, as we identified reprogrammed ependymal cells forming pinwheel-like\n",
      "\n",
      "structures with resident neural stem cells in the targeted ventricular wall. Our findings suggest that\n",
      "\n",
      "McIdas forced expression in human patients with hydrocephalus would contribute to the re\n",
      "establishment of the SVZ niche leading to brain functions’ improvement by enhancing neuronal\n",
      "\n",
      "production in human patients with hydrocephalus and ameliorating brain function. The in vivo\n",
      "\n",
      "reprogramming approach indicates that the in-situ conversion of cells into ependyma in its native\n",
      "\n",
      "microenvironment is feasible. Our in vivo approach allowed the exploration of the influence of the\n",
      "\n",
      "diseased environment of the hydrocephalic brain on the reprogramming process. Our data support the\n",
      "\n",
      "idea that functional maturation and integration of the reprogrammed cells in the native tissue can be\n",
      "\n",
      "achieved through our in vivo reprogramming method. Since the molecular program of multiciliogenesis\n",
      "\n",
      "is conserved across different organisms and tissues (Arbi et al, 2018), it is tempting to speculate that\n",
      "\n",
      "such a reprogramming approach could be applied to other diseases that impact ciliated tissues, such as\n",
      "\n",
      "the lung and the reproductive system. For instance, a rare genetic disorder affecting the mucociliary\n",
      "\n",
      "clearance mechanism, known as Reduced Generation of Multiple Motile Cilia (RGMC), has been linked\n",
      "\n",
      "to mutations in MCIDAS. Therefore, it would be highly interesting to investigate whether these\n",
      "\n",
      "conditions could be addressed in the future through a genetic treatment implementing McIdas’\n",
      "\n",
      "19\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "overexpression in the affected tissue.\n",
      "\n",
      "Together, our analysis proposes MCIDAS and GEMC1 as potent reprogramming factors towards the\n",
      "\n",
      "ependymal cell lineage and demonstrate a proof of concept that restoring ependymal cells could\n",
      "\n",
      "contribute in managing hydrocephalus. Although the present study yielded intriguing findings, our\n",
      "\n",
      "implementation holds certain limitations. In particular, our in vivo studies have been conducted to mouse\n",
      "\n",
      "models of hydrocephalus. Despite sharing a similar pathophysiology with the human disease, the\n",
      "\n",
      "anatomical and physiological characteristics of the mouse and human brain possess certain differences.\n",
      "\n",
      "Therefore, to validate the effectiveness of our approach, additional experiments in disease models closer\n",
      "\n",
      "to human pathophysiology will be required. The pig model demonstrates anatomical and physiological\n",
      "\n",
      "similarities to humans and replicates the pathophysiology of human hydrocephalus, exhibiting features\n",
      "\n",
      "such as ventriculomegaly, V/SVZ disruption, and neuroinflammation and could thus be a valuable model\n",
      "\n",
      "for examining the translational success of our approach to the human conditions (Garcia-Bonilla et al,\n",
      "\n",
      "2023). Additionally, since hydrocephalus in our study appeared during early developmental stages, our\n",
      "\n",
      "findings are particularly relevant to postnatal hydrocephalus. Therefore, their applicability to adult\n",
      "\n",
      "hydrocephalus will require further investigation using appropriate adult animal models. Although we\n",
      "\n",
      "achieved McIdas overexpression in the periventricular cells of the diseased brain through\n",
      "\n",
      "electroporation, such an approach cannot be used in humans. Therefore, alternative gene delivery\n",
      "\n",
      "methods compatible with therapeutic protocols, such as AAV gene delivery strategies in the human brain\n",
      "\n",
      "(Chamakioti et al, 2022; Au et al, 2022) should be established. Our study shows that regeneration of\n",
      "\n",
      "multiciliated ependymal cells can be succeeded in hydrocephalic mouse models while further\n",
      "\n",
      "experiments are needed to show the applicability of our findings in treating hydrocephalus in humans.\n",
      "\n",
      "20\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "METHODS\n",
      "\n",
      "Mouse knockout strain\n",
      "\n",
      "C57/Bl6 mice carrying a knockout (KO) allele of the GemC1 gene were initially generated by the trans\n",
      "NIH Knock-Out Mouse Project (KOMP) and obtained from the KOMP repository. The GemC1-KO\n",
      "\n",
      "allele was maintained on a mixed strain background. Inactivation of the GemC1 gene was carried out\n",
      "\n",
      "using the “knockout-first” strategy, as previously described (Arbi et al, 2016; Lalioti et al, 2019b). This\n",
      "\n",
      "approach involves disrupting gene function by adding RNA processing signals without removing any of\n",
      "\n",
      "the targeted genes (Testa et al, 2004). The GemC1-KO allele was created by placing a lacZ-neo cassette\n",
      "\n",
      "between the second and third exon of the gene.\n",
      "\n",
      "Mice were housed in the animal house of the University of Patras, where a standard light cycle of 12\n",
      "\n",
      "hours of light and 12 hours of darkness was maintained. Continuous access to food and water was\n",
      "\n",
      "ensured at all times. To ensure animal welfare handling and environmental stressors like noise,\n",
      "\n",
      "vibrations, and strong odors were minimized to necessary levels. All animal-related procedures were\n",
      "\n",
      "conducted in strict accordance with EU directives and approved by the Veterinary Administrations of the\n",
      "\n",
      "Prefecture of Achaia, Greece, and the Research Ethics Committee of the University of Patras, Greece.\n",
      "\n",
      "In all experiments, mice were used without regard to their gender. Experiments involving animal usage\n",
      "\n",
      "were conducted during the neonatal stage when the gender of the animals was not yet distinguishable.\n",
      "\n",
      "LPA preparation and injections\n",
      "\n",
      "To generate an inducible mouse model of intracranial-hemorrhage hydrocephalus, we performed\n",
      "\n",
      "intraventricular delivery of lysophosphatidic acid (LPA) in wild-type (WT) mice at postnatal day five\n",
      "\n",
      "(P5), following previously published works (Yung et al, 2011; Lummis et al, 2019).\n",
      "\n",
      "Powdered 18:1 LPA (857130P, Avanti Polar Lipids) was initially dissolved in methanol and subsequently\n",
      "\n",
      "vacuum-dried. LPA was then reconstituted in Hanks’ balanced salt solution (HBSS 14065-049, Gibco)\n",
      "\n",
      "by sonicating in a water bath for 5 min to generate a 10 mM stock, as previously described (Yung et al,\n",
      "\n",
      "2011; Lummis et al, 2019). Stock solution was stored at −20°C until use. Using a glass microcapillary, 3\n",
      "\n",
      "μl of the 10 mM LPA stock solution together with Fast Green (0.1%, Sigma) were injected into the\n",
      "\n",
      "lateral ventricle of P5 mice upon hypothermia-anesthetization. Mice were sacrificed two days post LPA\n",
      "\n",
      "injections at P7 and were examined or further processed as described in the respective experiments.\n",
      "\n",
      "21\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "Plasmid constructs\n",
      "\n",
      "For neonatal electroporation and mESCs transfection experiments mouse cDNAs for McIdas and\n",
      "\n",
      "GemC1 were cloned in pCAGGS-IRES-GFP vectors between SacI/EcoRV and NheI/SmaI restriction\n",
      "\n",
      "sites respectively, as described previously (Kyrousi et al, 2015).\n",
      "\n",
      "Neonatal electroporation\n",
      "\n",
      "Neonatal electroporation was performed at P0-P1. Anesthesia was induced via hypothermia. Using a\n",
      "\n",
      "pulled glass microcapillary, 2 μg plasmid DNA together with Fast Green (0.1%, Sigma) were injected in\n",
      "\n",
      "the lateral ventricle of the mouse brains. Mice were subsequently electroporated with 5 pulses of 100V\n",
      "\n",
      "for 50ms each followed by intervals of 950ms. An ECM830 Electroporator (Harvard Apparatus) and 7\n",
      "\n",
      "mm platinum electrodes (BTX) were used. Following electroporation mice were placed back to cages\n",
      "\n",
      "and left to grow for 4 or 6-14 days before sacrificed as described in the respective experiments.\n",
      "\n",
      "Ex vivo electroporation and organotypic brain slice cultures\n",
      "\n",
      "For ex vivo electroporation, mice were sacrificed 2 days post LPA-injections at P7. Mouse brains were\n",
      "\n",
      "dissected and placed in cold DMEM/F-12 medium (Gibco). Subsequently, 2 μg plasmid DNA of a\n",
      "\n",
      "pCAGGS-IRES-GFP (GFP) or pCAGGS-McIdas-IRES-GFP (McIdas) vector together with Fast Green\n",
      "\n",
      "(0.1%, Sigma) were injected in the lateral ventricle using a pulled glass microcapillary. Brains were then\n",
      "\n",
      "electroporated with 5 pulses applied at 100V for 50ms each at intervals of 950ms. An ECM830\n",
      "\n",
      "Electroporator (Harvard Apparatus) and 7 mm platinum electrodes (BTX) were used.\n",
      "\n",
      "For organotypic brain slice cultures 250 μm thick coronal slices were obtained from the electroporated\n",
      "\n",
      "brains using a vibrating microtome (Leica VT1000 S). Consecutive slices containing the lateral\n",
      "\n",
      "ventricles were collected and placed in culturing medium [2% FBS (Gibco), 1% GlutaMAX Supplement\n",
      "\n",
      "(Gibco) and 1% Penicillin-Streptomycin (Gibco) in DMEM/F-12]. Subsequently, brain slices were\n",
      "\n",
      "placed onto 0.4 μm membrane inserts (Millipore) in 6-well plates and cultured for 5 or 9 days in\n",
      "\n",
      "culturing medium, performing medium change every three days.\n",
      "\n",
      "Lentiviral production\n",
      "\n",
      "A second generation packaging system was used to generate lentiviral expression particles as previously\n",
      "\n",
      "described (Arbi et al, 2016). The pLV-Dest-CAG lentiviral expression vector was used and was kindly\n",
      "\n",
      "22\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "provided by Dr. M. Gotz, Helmholtz Center, Munich. The cloning strategy for generating the lentiviral\n",
      "\n",
      "expression vectors for GFP and GemC1 with an N-terminal GFP tag (GFP-GemC1) was described\n",
      "\n",
      "elsewhere (Arbi et al, 2016).\n",
      "\n",
      "McIdas with an N-terminal GFP tag (GFP-McIdas) was initially cloned into the KpnI/XhoI restriction\n",
      "\n",
      "sites of the pENTR1AminusCmR vector. p73 with a C-terminal GFP tag (p73-GFP) was initially cloned\n",
      "\n",
      "into the SalI/NotI restriction sites of the pENTR1AminusCmR vector. An LR recombination reaction,\n",
      "\n",
      "using the Gateway LR Clonase II enzyme mix (Invitrogen), was performed between the attL-containing\n",
      "\n",
      "entry clone and the attR-containing destination pLVDest-CAG vector.\n",
      "\n",
      "Primary cultures of postnatal cortical astroglia and lentiviral infection\n",
      "\n",
      "To establish primary cultures of postnatal cortical astroglia, the cerebral cortex of postnatal P5-P7 wild\n",
      "type mice was dissected from the brain. Cells were subsequently mechanically dissociated and cultured\n",
      "\n",
      "under proliferating conditions for 7 days in astro medium containing DMEM/F-12 (Gibco), 10% FBS\n",
      "\n",
      "(Gibco), 2% B27 Supplement (Gibco), 1% GlutaMAX Supplement (Gibco), 1% Penicillin-Streptomycin\n",
      "\n",
      "(Gibco), epidermal growth factor (10ng/ml) and basic fibroblast growth factor (10ng/ml), as previously\n",
      "\n",
      "described (Heinrich et al, 2011).\n",
      "\n",
      "For lentiviral infection, astroglial cells were plated onto poly-D-lysine pre-coated coverslips 7 days upon\n",
      "\n",
      "proliferation initiation and were left to attach and grow for 2-4 hours. Cells were then transduced with\n",
      "\n",
      "concentrated lentiviral vectors. The infection mix was removed 24 hours later and was replaced by astro\n",
      "\n",
      "medium which did not contain FBS, epidermal growth factor and basic fibroblast growth factor to favor\n",
      "\n",
      "differentiation. Cells were analyzed 7 and 14 days later.\n",
      "\n",
      "Primary cultures of postnatal radial glial cells and lentiviral infection\n",
      "\n",
      "To establish primary cultures of postnatal radial glial cells (pRGCs), the lateral walls of the lateral\n",
      "\n",
      "ventricles of P0-P1 mice were dissected, mechanically dissociated, and then plated onto poly-D-lysine\n",
      "\n",
      "pre-coated coverslips. pRGCs were cultured in proliferation medium [DMEM-high glucose (Gibco),\n",
      "\n",
      "10% FBS (Gibco), and 1% penicillin/streptomycin (Gibco)] for three days (Paez-Gonzalez et al, 2011;\n",
      "\n",
      "Kyrousi et al, 2015; Lalioti et al, 2019b). Subsequently, pRGCs were infected with unconcentrated viral\n",
      "\n",
      "particles performing plate-spin infection. Proliferation medium was changed 24 hours post-infection and\n",
      "\n",
      "later replaced with differentiation medium [DMEM-high glucose (Gibco), 2% FBS (Gibco), and 1%\n",
      "\n",
      "23\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "penicillin/streptomycin (Gibco)] 48 hours post-infection. Cells were processed for analysis 5 or 15 days\n",
      "\n",
      "upon differentiation initiation.\n",
      "\n",
      "Cultures of mouse embryonic stem cells (mESCs) and transfection\n",
      "\n",
      "Mouse embryonic stem cells were grown onto gelatin-coated (0.1% gelatin, Sigma) flasks in GMEM\n",
      "\n",
      "BHK-21 medium (Gibco), supplemented with 10% FBS (Biosera, Embryonic Stem Cells tested), 0.1\n",
      "\n",
      "mM β-mercaptoethanol (Gibco), 2 mM L-glutamine (Sigma), 1 mM sodium pyruvate (Pan Biotech), 1%\n",
      "\n",
      "penicillin/streptomycin (Gibco), 1% MEM Non-Essential Amino Acids (Biosera) and 2.000 U/ml LIF\n",
      "\n",
      "(Millipore). Cells were passaged every other day and were mycoplasma tested.\n",
      "\n",
      "Reverse mESCs transfection was performed using the jet-OPTIMUS reagent according to\n",
      "\n",
      "manufacturer’s instructions. Briefly, the transfection mixes containing DNA, jet-OPTIMUS buffer and\n",
      "\n",
      "transfection reagent were prepared. Afterwards, transfection mixes were placed on gelatin pre-coated\n",
      "\n",
      "glass coverslips and then mESCs were immediately seeded. Cells were cultured in the presence of LIF\n",
      "\n",
      "for 24 hours. The following day the medium was changed (LIF withdrawal), and the cells were cultured\n",
      "\n",
      "in differentiating conditions for up to 6 days and were then processed for analysis.\n",
      "\n",
      "Video microscopy of ependymal cilia\n",
      "\n",
      "For the study of ependymal cilia in primary cultures of reprogrammed postnatal cortical astroglia, cells\n",
      "\n",
      "were grown under proliferating conditions in astro medium for 7 days. Cells were then plated onto poly\n",
      "D-lysine coated glass-bottom dishes (ibidi) and transduced with unconcentrated lentiviral vectors. 24\n",
      "\n",
      "hours post infection, medium was replaced by astro medium which did not contain FBS, epidermal\n",
      "\n",
      "growth factor and basic fibroblast growth factor to favor differentiation. Cells were used in video\n",
      "\n",
      "microscopy experiments 22 days later.\n",
      "\n",
      "To perform video microscopy experiments in reprogrammed postnatal cortical astroglia, the culturing\n",
      "\n",
      "medium was replaced by live imaging medium containing MEM (Sigma), 20% FBS (Gibco), 4 mM L\n",
      "glutamine (Sigma) and 1 mM sodium pyruvate (Pan Biotech). In addition, to track cilia generated fluid\n",
      "\n",
      "flow 0.5 μm red fluorescent microbeads (1μl/ml medium; Sigma) were added to the imaging medium.\n",
      "\n",
      "Video microscopy experiments were performed using an Olympus IX83 microscope equipped with a\n",
      "\n",
      "Hamamatsu Orca-Flash 4.0 sCMOS camera. During live imaging microscopy experiments cells were\n",
      "\n",
      "maintained at 37°C and 5% CO2. Videos were obtained using a 60x water immersion objective at 80\n",
      "\n",
      "24\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "frames/sec, while 500 frames were acquired for each field.\n",
      "\n",
      "For the study of brain ependymal cilia, mice were sacrificed at P7-P15, brains were dissected in HBSS\n",
      "\n",
      "at room temperature and 200 μm thick coronal slices were obtained using a vibrating microtome. Slices\n",
      "\n",
      "containing the lateral ventricles were placed in DMEM-GlutaMAX (Gibco) media pre-warmed at 37°C.\n",
      "\n",
      "To perform video microscopy experiments, brain slices were placed onto glass-bottom dishes containing\n",
      "\n",
      "live imaging medium with 0.5 μm red fluorescent microbeads (1μl/ml medium). Video recordings were\n",
      "\n",
      "obtained as described in the preceding paragraph.\n",
      "\n",
      "For cilia beat frequency analysis, the ImageJ image processing program was used to firstly generate a\n",
      "\n",
      "graphical representation of cilia beating, in the form of a kymograph, as previously described by\n",
      "\n",
      "Drummond I. (Drummond, 2009). Cilia beat frequency for each multiciliated cell was then calculated\n",
      "\n",
      "with the correlation of the time scale of the video recording with the kymograph, following the method\n",
      "\n",
      "described by Francis R. and Lo C. (Francis & Lo, 2013).\n",
      "\n",
      "To track the trajectory of the moving microbeads the MTrackJ Plugin of the ImageJ program was used.\n",
      "\n",
      "In short, the point selection tool was used to manually mark the position of each red fluorescent\n",
      "\n",
      "microbead in each frame. Consecutive marks on the corresponding video recordings created the\n",
      "\n",
      "trajectory of the microbeads.\n",
      "\n",
      "Whole-mount dissection\n",
      "\n",
      "SVZ whole mounts were obtained as previously described (Lalioti et al, 2019b). Briefly, the brain was\n",
      "\n",
      "isolated and the electroporated lateral ventricle was carefully separated from the posterior region of the\n",
      "\n",
      "telencephalon, followed by the extraction of the hippocampus and septum. Fixation of the dissected\n",
      "\n",
      "lateral wall was performed in 4% PFA/0.1% Triton X-100 at 4[o]C overnight. Staining was then performed\n",
      "\n",
      "(see below, immunofluorescence) and the lateral wall was further dissected from the surrounding brain\n",
      "\n",
      "parenchyma, resulting in tissue slivers with a thickness of 200-300 micrometers. These slivers were then\n",
      "\n",
      "affixed to a glass slide using Mowiol 4-88 (Calbiochem) mounting media and covered with a coverslip.\n",
      "\n",
      "Immunofluorescence\n",
      "\n",
      "For immunofluorescence, cultured cells were fixed for 10 min with 4% PFA (or 10 min PFA 4%\n",
      "\n",
      "25\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "followed by 10 min fixation in Methanol where stated) and incubated in blocking solution, for 1hr at\n",
      "\n",
      "room temperature. Postnatal cortical astroglial cells were incubated in blocking solution containing 5%\n",
      "\n",
      "FBS, 3% BSA, 0.1% Triton X-100 in 1× PBS. pRGCs and mESCs were incubated in blocking solution\n",
      "\n",
      "containing 5% Normal Goat Serum (Jackson Immunoresearch) and 0.3% Triton-X100 in 1x PBS. Cells\n",
      "\n",
      "were incubated with primary antibodies in blocking solution at 4°C, overnight (Kalogeropoulou et al,\n",
      "\n",
      "2022).\n",
      "\n",
      "Brains dissected from newborn mice (up to P7) were fixed overnight with 4% PFA. Subsequently,\n",
      "\n",
      "brains were rinsed with 1x PBS and cryopreserved with incubation in 30% sucrose in 1x PBS for ~48\n",
      "\n",
      "hours. Brains were then frozen in 7.5% gelatin supplemented with 15% sucrose and sectioned at 12 μm\n",
      "\n",
      "using a CM1850 Leica cryostat. Brain coronal cryosections were postfixed with 4% PFA for 10 min,\n",
      "\n",
      "treated with 0.3% Triton X-100 in 1× PBS for 5 min, and incubated in blocking solution containing 5%\n",
      "\n",
      "FBS, 3% BSA, 0.1% Triton X-100 in 1× PBS, for 1-3 hrs. Samples were incubated with primary\n",
      "\n",
      "antibodies in blocking solution at 4°C, overnight.\n",
      "\n",
      "For immunofluorescence of cultured brain slices, membrane inserts were removed, rinsed in 1x PBS and\n",
      "\n",
      "fixed with 4% PFA at 4°C, overnight. Membrane inserts were then subjected to consecutive washes in\n",
      "\n",
      "1x PBS (5 min), 0.1% Triton X-100 in 1× PBS (5 min) and 0.5% Triton X-100 in 1× PBS (20 min once)\n",
      "\n",
      "and were incubated in blocking solution containing 1% FBS, 0.1% Triton X-100 in 1× PBS for 2-3 hrs\n",
      "\n",
      "at room temperature. Samples were incubated with primary antibodies in blocking solution at 4°C,\n",
      "\n",
      "overnight.\n",
      "\n",
      "Immunofluorescence of SVZ whole-mounts was performed as previously described (Mirzadeh et al,\n",
      "\n",
      "2008, 2010). Briefly, whole-mounts were incubated for 1-2 hrs at room temperature in blocking solution,\n",
      "\n",
      "containing 10% normal goat serum in 0.1 M PBS with 0.5-2% Triton-X100. Samples were incubated\n",
      "\n",
      "with primary antibodies in blocking solution at 4[o]C for 24-48 hrs.\n",
      "\n",
      "The following primary antibodies were used: chicken anti-GFP (1020, Aves Laboratories, 1:500 for\n",
      "\n",
      "brain sections; 1:1000 for cells, 1:250 for whole mounts), mouse anti-S100β (S2532, Sigma, 1:250),\n",
      "\n",
      "rabbit anti-P73 (ab40658, Abcam, 1:300), mouse anti-FoxJ1 (14-9965, eBioscience, 1:500), rabbit anti\n",
      "pericentrin (PRB432C, Covance, 1:750 for brain sections; 1:1500 for cells), rabbit anti-GFAP (Z0334,\n",
      "\n",
      "DakoCytomation, 1:1000 for cells, 1:500 for whole-mounts), mouse anti-acetylated α-tubulin (T6793,\n",
      "\n",
      "Sigma, 1:1000 for brain sections; 1:1500 for cells), rabbit anti-McIdas ((Pefani et al, 2011), 1:250),\n",
      "\n",
      "26\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "mouse anti-centriolin (sc-365521, Santa Cruz, 1:100, cells’ fixation in PFA-Methanol), rabbit anti\n",
      "Meig1 (orb185566, Biorbyt, 1:100), rat anti-VCAM1 (550547, BD Biosciences, 1:100), mouse anti β\n",
      "catenin (610153, BD Biosciences, 1:500), guinea pig anti-double-cortin (AB2253, Millipore, 1:500).\n",
      "\n",
      "The following Alexa Fluor-labeled secondary antibodies (Invitrogen) were used in the respective\n",
      "\n",
      "blocking solution (1:1000) for 1 hr (cryosections and cells) at room temperature or overnight (membrane\n",
      "\n",
      "inserts) at 4°C: Alexa Fluor 488 goat anti-chicken, Alexa Fluor 568 goat anti-mouse IgG1, Alexa Fluor\n",
      "\n",
      "647 donkey anti-rabbit, Alexa Fluor 568 goat anti-rabbit, Alexa Fluor 647 goat anti-mouse IgG1, Alexa\n",
      "\n",
      "Fluor 568 goat anti-mouse, Alexa Fluor 568 goat anti-rabbit, Alexa Fluor 647 donkey anti-mouse, Alexa\n",
      "\n",
      "Fluor 568 goat anti-guinea pig and Alexa Fluor 568 goat anti-rat. DNA was stained either with Draq-5\n",
      "\n",
      "(1:1000, Biostatus) or Dapi (1:1500, Sigma). Samples were mounted in Mowiol 4-88 (Calbiochem).\n",
      "\n",
      "Microscopy and data analysis\n",
      "\n",
      "Images of fluorescent samples were acquired with a confocal Leica TCS SP5 and a Leica TCS SP8\n",
      "\n",
      "system equipped with a fluorescent Leica DMI600B or Leica DMi8 microscope, respectively. 20x, 40x\n",
      "\n",
      "and 63x lenses were used. Digital images were processed with Adobe Photoshop and the ImageJ image\n",
      "\n",
      "processing program. For still images related to video microscopy experiments, individual frames were\n",
      "\n",
      "cropped and adjusted for brightness and contrast in ImageJ.\n",
      "\n",
      "Quantification of cells on overexpressed brains from in vivo experiments was performed on\n",
      "\n",
      "cryosections, vibratome sections or whole-mounts. In all instances, each brain region containing\n",
      "\n",
      "electroporated cells was imaged and quantified per frame. All quantifications are expressed as\n",
      "\n",
      "percentages of the total cell count. Quantification of radial glial cells, cortical astrocytes, or mESCs from\n",
      "\n",
      "ex-vivo experiments was performed in at least two independent experiments, with a minimum of 15\n",
      "\n",
      "different images analyzed for each condition in each experiment. Fluorescent intensity per nucleus was\n",
      "\n",
      "measured using the open-source platform ImageJ.\n",
      "\n",
      "The number of independent experiments performed for each analysis is mentioned in the corresponding\n",
      "\n",
      "figure legend. Normality was tested for each experiment using the Shapiro-Wilk test to determine if the\n",
      "\n",
      "sample data came from a normally distributed population. Since our data did not follow a normal\n",
      "\n",
      "distribution, statistical analysis was performed with the non-parametric two-tailed Mann–Whitney test.\n",
      "\n",
      "Significant differences in medians were determined by p value: P<0.05 (*P<0.05, **P<0.01,\n",
      "\n",
      "***P<0.001). Each graph displayed the median ± interquartile range (IQR). All statistical analyses and\n",
      "\n",
      "27\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "graph preparation were performed in GraphPad Prism 6.\n",
      "\n",
      "The study did not incorporate blinding or randomization, and all samples were included without\n",
      "\n",
      "applying specific inclusion or exclusion criteria. These choices were guided by the exploratory nature of\n",
      "\n",
      "the research.DATA AVAILABILITY\n",
      "\n",
      "This study includes no data deposited in external repositories.\n",
      "\n",
      "ACKNOWLEDGMENTS\n",
      "\n",
      "We thank the Advanced Light Microscopy Facility of the Medical School at the University of Patras and\n",
      "\n",
      "the Experimental Animal Facility of the University of Patras for support with experiments. We are also\n",
      "\n",
      "grateful to Andriana Charalampopoulou and members from Professor’s Benedikt Berninger laboratory\n",
      "\n",
      "for their assistance in astrocytic reprogramming experiments. We also thank Dr. Christina Kyrousi and\n",
      "\n",
      "members of our laboratories for helpful discussions and Nikolaos Giakoumakis for his assistance with\n",
      "\n",
      "whole-mount images processing. This study was supported by the Hellenic Foundation for Research and\n",
      "\n",
      "Innovation (H.F.R.I.) under the “2nd Call for H.F.R.I. Research Projects to support Faculty Members &\n",
      "\n",
      "Researchers” (Project Number: 2735), the Hydrocephalus Association USA and Fondation Santé to S.T.,\n",
      "\n",
      "and the State Scholarships Foundation (IKY) to K.K.\n",
      "\n",
      "DISCLOSURE AND COMPETING INTERESTS STATEMENT\n",
      "\n",
      "Zoi Lygerou is an EMBO Council member. This has no bearing on the editorial consideration of this\n",
      "\n",
      "article for publication.\n",
      "\n",
      "THE PAPER EXPLAINED:\n",
      "\n",
      "28\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "Medical issue\n",
      "\n",
      "Hydrocephalus is a condition where fluid builds up in the brain cavities, leading to serious health issues.\n",
      "Standard treatment involves surgery, which is risky and often fails over time. Ependymal cells have\n",
      "multiple tiny hair-like structures (cilia) on their surface to circulate the fluid, and their impairment may\n",
      "lead to hydrocephalus.\n",
      "\n",
      "Results\n",
      "\n",
      "We showed that GemC1 and McIdas, which are known regulators of multiciliated ependymal cell fate\n",
      "determination, can instruct different cell types to differentiate into ependymal cells. We further managed\n",
      "to convert brain cells of hydrocephalic animal models into ependymal cells.\n",
      "\n",
      "Clinical impact\n",
      "\n",
      "Our findings suggest new therapeutic intervention aiming at the regeneration of damaged ependymal\n",
      "cells in human hydrocephalus. .\n",
      "\n",
      "REFERENCES\n",
      "\n",
      "Abdi K, Lai C-H, Paez-Gonzalez P, Lay M, Pyun J & Kuo CT (2018) Uncovering inherent cellular\n",
      "\n",
      "plasticity of multiciliated ependyma leading to ventricular wall transformation and\n",
      "hydrocephalus. Nat Commun 9: 1655\n",
      "\n",
      "Arbi M, Pefani D, Kyrousi C, Lalioti M, Kalogeropoulou A, Papanastasiou AD, Taraviras S & Lygerou\n",
      "\n",
      "Z (2016) GemC1 controls multiciliogenesis in the airway epithelium. EMBO Rep 17: 400–413\n",
      "\n",
      "Arbi M, Pefani D-E, Taraviras S & Lygerou Z (2018) Controlling centriole numbers: Geminin family\n",
      "\n",
      "members as master regulators of centriole amplification and multiciliogenesis. Chromosoma 127:\n",
      "151–174\n",
      "\n",
      "Au HKE, Isalan M & Mielcarek M (2022) Gene Therapy Advances: A Meta-Analysis of AAV Usage in\n",
      "\n",
      "Clinical Settings. Frontiers in Medicine 8\n",
      "\n",
      "Bocchi R, Masserdotti G & Götz M (2022) Direct neuronal reprogramming: Fast forward from new\n",
      "\n",
      "concepts toward therapeutic approaches. Neuron 110: 366–393\n",
      "\n",
      "Boon M, Wallmeier J, Ma L, Loges NT, Jaspers M, Olbrich H, Dougherty GW, Raidt J, Werner C,\n",
      "\n",
      "Amirav I, et al (2014) MCIDAS mutations result in a mucociliary clearance disorder with\n",
      "reduced generation of multiple motile cilia. Nat Commun 5: 4418\n",
      "\n",
      "Chamakioti M, Karantzelis N & Taraviras S (2022) Advanced Gene-Targeting Therapies for Motor\n",
      "\n",
      "Neuron Diseases and Muscular Dystrophies. International Journal of Molecular Sciences 23:\n",
      "4824\n",
      "\n",
      "29\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "Drummond I (2009) Studying cilia in zebrafish. Methods Cell Biol 93: 197–217\n",
      "\n",
      "Francis R & Lo C (2013) Ex vivo method for high resolution imaging of cilia motility in rodent airway\n",
      "\n",
      "epithelia. J Vis Exp\n",
      "\n",
      "Funk MC, Bera AN, Menchen T, Kuales G, Thriene K, Lienkamp SS, Dengjel J, Omran H, Frank M &\n",
      "\n",
      "Arnold SJ (2015) Cyclin O (Ccno) functions during deuterosome-mediated centriole\n",
      "amplification of multiciliated cells. EMBO J 34: 1078–1089\n",
      "\n",
      "Garcia-Bonilla M, Nair A, Moore J, Castaneyra-Ruiz L, Zwick SH, Dilger RN, Fleming SA, Golden\n",
      "\n",
      "RK, Talcott MR, Isaacs AM, et al (2023) Impaired neurogenesis with reactive astrocytosis in the\n",
      "hippocampus in a porcine model of acquired hydrocephalus. Experimental Neurology 363:\n",
      "114354\n",
      "\n",
      "Gonzalez-Cano L, Fuertes-Alvarez S, Robledinos-Anton N, Bizy A, Villena-Cortes A, Fariñas I,\n",
      "\n",
      "Marques MM & Marin MC (2016) p73 is required for ependymal cell maturation and neurogenic\n",
      "SVZ cytoarchitecture: p73 Regulates SVZ Cytoarchitecture. Developmental Neurobiology 76:\n",
      "730–747\n",
      "\n",
      "Guerra MM, Henzi R, Ortloff A, Lichtin N, Vío K, Jiménez AJ, Dominguez-Pinos MD, González C,\n",
      "\n",
      "Jara MC, Hinostroza F, et al (2015) Cell Junction Pathology of Neural Stem Cells Is Associated\n",
      "With Ventricular Zone Disruption, Hydrocephalus, and Abnormal Neurogenesis. J Neuropathol\n",
      "Exp Neurol 74: 653–671\n",
      "\n",
      "Heinrich C, Gascón S, Masserdotti G, Lepier A, Sanchez R, Simon-Ebert T, Schroeder T, Götz M &\n",
      "\n",
      "Berninger B (2011) Generation of subtype-specific neurons from postnatal astroglia of the mouse\n",
      "cerebral cortex. Nat Protoc 6: 214–228\n",
      "\n",
      "Ji W, Tang Z, Chen Y, Wang C, Tan C, Liao J, Tong L & Xiao G (2022) Ependymal Cilia: Physiology\n",
      "\n",
      "and Role in Hydrocephalus. Frontiers in Molecular Neuroscience 15\n",
      "\n",
      "Jiménez AJ, Domínguez-Pinos M-D, Guerra MM, Fernández-Llebrez P & Pérez-Fígares J-M (2014)\n",
      "\n",
      "Structure and function of the ependymal barrier and diseases associated with ependyma\n",
      "disruption. Tissue Barriers 2: e28426\n",
      "\n",
      "Kahle KT, Kulkarni AV, Limbrick DD & Warf BC (2016) Hydrocephalus in children. The Lancet 387:\n",
      "\n",
      "788–799\n",
      "\n",
      "Kalogeropoulou A, Mougkogianni M, Iliadou M, Nikolopoulou E, Flordelis S, Kanellou A, Arbi M,\n",
      "\n",
      "Nikou S, Nieminuszczy J, Niedzwiedz W, et al (2022) Intrinsic neural stem cell properties define\n",
      "brain hypersensitivity to genotoxic stress. Stem Cell Reports 17: 1395–1410\n",
      "\n",
      "Kokovay E, Wang Y, Kusek G, Wurster R, Lederman P, Lowry N, Shen Q & Temple S (2012) VCAM1\n",
      "\n",
      "is essential to maintain the structure of the SVZ niche and acts as an environmental sensor to\n",
      "regulate SVZ lineage progression. Cell Stem Cell 11: 220–230\n",
      "\n",
      "Kousi M & Katsanis N (2016) The Genetic Basis of Hydrocephalus. Annu Rev Neurosci 39: 409–435\n",
      "\n",
      "30\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "Kyrousi C, Arbi M, Pilz G-A, Pefani D-E, Lalioti M-E, Ninkovic J, Götz M, Lygerou Z & Taraviras S\n",
      "\n",
      "(2015) Mcidas and GemC1 are key regulators for the generation of multiciliated ependymal cells\n",
      "in the adult neurogenic niche. Development 142: 3661–3674\n",
      "\n",
      "Kyrousi C, Lygerou Z & Taraviras S (2017) How a radial glial cell decides to become a multiciliated\n",
      "\n",
      "ependymal cell. Glia 65: 1032–1042\n",
      "\n",
      "Lalioti M-E, Arbi M, Loukas I, Kaplani K, Kalogeropoulou A, Lokka G, Kyrousi C, Mizi A,\n",
      "\n",
      "Georgomanolis T, Josipovic N, et al (2019a) GemC1 governs multiciliogenesis through direct\n",
      "interaction with and transcriptional regulation of p73. J Cell Sci 132\n",
      "\n",
      "Lalioti M-E, Kaplani K, Lokka G, Georgomanolis T, Kyrousi C, Dong W, Dunbar A, Parlapani E,\n",
      "\n",
      "Damianidou E, Spassky N, et al (2019b) GemC1 is a critical switch for neural stem cell\n",
      "generation in the postnatal brain. Glia\n",
      "\n",
      "Le N, Appel H, Pannullo N, Hoang T & Blackshaw S (2022) Ectopic insert-dependent neuronal\n",
      "\n",
      "expression of GFAP promoter-driven AAV constructs in adult mouse retina. Front Cell Dev Biol\n",
      "10\n",
      "\n",
      "Lim DA, Tramontin AD, Trevejo JM, Herrera DG, García-Verdugo JM & Alvarez-Buylla A (2000)\n",
      "\n",
      "Noggin Antagonizes BMP Signaling to Create a Niche for Adult Neurogenesis. Neuron 28: 713–\n",
      "726\n",
      "\n",
      "Lummis NC, Sánchez-Pavón P, Kennedy G, Frantz AJ, Kihara Y, Blaho VA & Chun J (2019) LPA1/3\n",
      "\n",
      "overactivation induces neonatal posthemorrhagic hydrocephalus through ependymal loss and\n",
      "ciliary dysfunction. Sci Adv 5: eaax2011\n",
      "\n",
      "Ma L, Quigley I, Omran H & Kintner C (2014) Multicilin drives centriole biogenesis via E2f proteins.\n",
      "\n",
      "Genes Dev 28: 1461–1471\n",
      "\n",
      "MacDonald A, Lu B, Caron M, Caporicci-Dinucci N, Hatrock D, Petrecca K, Bourque G & Stratton JA\n",
      "\n",
      "(2021) Single Cell Transcriptomics of Ependymal Cells Across Age, Region and Species Reveals\n",
      "Cilia-Related and Metal Ion Regulatory Roles as Major Conserved Ependymal Cell Functions.\n",
      "Front Cell Neurosci 15\n",
      "\n",
      "Marshall CB, Mays DJ, Beeler JS, Rosenbluth JM, Boyd KL, Santos Guasch GL, Shaver TM, Tang LJ,\n",
      "\n",
      "Liu Q, Shyr Y, et al (2016) p73 Is Required for Multiciliogenesis and Regulates the Foxj1Associated Gene Network. Cell Reports 14: 2289–2300\n",
      "\n",
      "McAllister, James P., Guerra, Maria Montserrat, Ruiz, Leandro Castaneyra, Jimenez, Antonio J.,\n",
      "\n",
      "Dominguez-Pinos, Dolores, Sival, Deborah, den Dunnen, Wilfred, Morales, Diego M., Schmidt,\n",
      "Robert E., Rodriguez, Esteban M., et al (2017) Ventricular Zone Disruption in Human Neonates\n",
      "With Intraventricular Hemorrhage. Journal of neuropathology and experimental neurology 76:\n",
      "358–375\n",
      "\n",
      "Mirzadeh Z, Doetsch F, Sawamoto K, Wichterle H & Alvarez-Buylla A (2010) The subventricular zone\n",
      "\n",
      "en-face: wholemount staining and ependymal flow. J Vis Exp: 1938\n",
      "\n",
      "31\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "Mirzadeh Z, Merkle FT, Soriano-Navarro M, Garcia-Verdugo JM & Alvarez-Buylla A (2008) Neural\n",
      "\n",
      "Stem Cells Confer Unique Pinwheel Architecture to the Ventricular Surface in Neurogenic\n",
      "Regions of the Adult Brain. Cell Stem Cell 3: 265–278\n",
      "\n",
      "Nemajerova A, Kramer D, Siller SS, Herr C, Shomroni O, Pena T, Suazo CG, Glaser K, Wildung M,\n",
      "\n",
      "Steffen H, et al (2016) TAp73 is a central transcriptional regulator of airway multiciliogenesis.\n",
      "Genes Dev\n",
      "\n",
      "Páez P, Bátiz L-F, Roales-Buján R, Rodríguez-Pérez L-M, Rodríguez S, Jiménez AJ, Rodríguez EM &\n",
      "\n",
      "Pérez-Fígares JM (2007) Patterned neuropathologic events occurring in hyh congenital\n",
      "hydrocephalic mutant mice. J Neuropathol Exp Neurol 66: 1082–1092\n",
      "\n",
      "Paez-Gonzalez P, Abdi K, Luciano D, Liu Y, Soriano-Navarro M, Rawlins E, Bennett V, Garcia-Verdugo\n",
      "\n",
      "JM & Kuo CT (2011) Ank3-Dependent SVZ Niche Assembly Is Required for the Continued\n",
      "Production of New Neurons. Neuron 71: 61–75\n",
      "\n",
      "Rekate HL (2009) A Contemporary Definition and Classification of Hydrocephalus. Seminars in\n",
      "\n",
      "Pediatric Neurology 16: 9–15\n",
      "\n",
      "Roales-Buján R, Páez P, Guerra M, Rodríguez S, Vío K, Ho-Plagaro A, García-Bonilla M, Rodríguez\n",
      "Pérez L-M, Domínguez-Pinos M-D, Rodríguez E-M, et al (2012) Astrocytes acquire\n",
      "morphological and functional characteristics of ependymal cells following disruption of\n",
      "ependyma in hydrocephalus. Acta Neuropathol 124: 531–546\n",
      "\n",
      "Rodríguez EM & Guerra MM (2017) Neural Stem Cells and Fetal-Onset Hydrocephalus. Pediatr\n",
      "\n",
      "Neurosurg 52: 446–461\n",
      "\n",
      "Sawamoto K, Wichterle H, Gonzalez-Perez O, Cholfin JA, Yamada M, Spassky N, Murcia NS, Garcia\n",
      "Verdugo JM, Marin O, Rubenstein JLR, et al (2006) New neurons follow the flow of\n",
      "cerebrospinal fluid in the adult brain. Science 311: 629–632\n",
      "\n",
      "Schrander-Stumpel C & Fryns JP (1998) Congenital hydrocephalus: nosology and guidelines for clinical\n",
      "\n",
      "approach and genetic counselling. Eur J Pediatr 157: 355–362\n",
      "\n",
      "Sival DA, Guerra M, den Dunnen WFA, Bátiz LF, Alvial G, Castañeyra-Perdomo A & Rodríguez EM\n",
      "\n",
      "(2011) Neuroependymal denudation is in progress in full-term human foetal spina bifida aperta.\n",
      "Brain Pathol 21: 163–179\n",
      "\n",
      "Stubbs JL, Vladar EK, Axelrod JD & Kintner C (2012) Multicilin promotes centriole assembly and\n",
      "\n",
      "ciliogenesis during multiciliate cell differentiation. Nat Cell Biol 14: 140–147\n",
      "\n",
      "Tan FE, Vladar EK, Ma L, Fuentealba LC, Hoh R, Espinoza FH, Axelrod JD, Alvarez-Buylla A, Stearns\n",
      "\n",
      "T, Kintner C, et al (2013) Myb promotes centriole amplification and later steps of the\n",
      "multiciliogenesis program. Development 140: 4277–4286\n",
      "\n",
      "Terré B, Piergiovanni G, Segura-Bayona S, Gil-Gómez G, Youssef SA, Attolini CS-O, Wilsch\n",
      "Bräuninger M, Jung C, Rojas AM, Marjanović M, et al (2016) GEMC1 is a critical regulator of\n",
      "multiciliated cell differentiation. EMBO J 35: 942–960\n",
      "\n",
      "32\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "Testa G, Schaft J, van der Hoeven F, Glaser S, Anastassiadis K, Zhang Y, Hermann T, Stremmel W &\n",
      "\n",
      "Stewart AF (2004) A reliable lacZ expression reporter cassette for multipurpose, knockout-first\n",
      "alleles. Genesis 38: 151–158\n",
      "\n",
      "Wallmeier J, Al-Mutairi DA, Chen C-T, Loges NT, Pennekamp P, Menchen T, Ma L, Shamseldin HE,\n",
      "\n",
      "Olbrich H, Dougherty GW, et al (2014) Mutations in CCNO result in congenital mucociliary\n",
      "clearance disorder with reduced generation of multiple motile cilia. Nat Genet 46: 646–651\n",
      "\n",
      "Wang L-L, Serrano C, Zhong X, Ma S, Zou Y & Zhang C-L (2021) Revisiting astrocyte to neuron\n",
      "\n",
      "conversion with lineage tracing in vivo. Cell 184: 5465-5481.e16\n",
      "\n",
      "Yung YC, Mutoh T, Lin M, Noguchi K, Rivera RR, Choi JW, Kingsbury MA & Chun J (2011)\n",
      "\n",
      "Lysophosphatidic Acid Signaling May Initiate Fetal Hydrocephalus. Sci Transl Med 3: 99ra87\n",
      "\n",
      "Yung YC, Stoddard NC, Mirendil H & Chun J (2015) Lysophosphatidic acid (LPA) signaling in the\n",
      "\n",
      "nervous system. Neuron 85: 669–682\n",
      "\n",
      "FIGURE LEGENDS\n",
      "\n",
      "Figure 1.\n",
      "\n",
      "McIdas forced expression promotes ex vivo reprogramming of murine cortical astrocytes into\n",
      "\n",
      "ependymal cells.\n",
      "\n",
      "(A) Schematic representation of experimental procedures for (B-G). Astrocytes were isolated from the\n",
      "\n",
      "cortex of neonatal mice and were cultured under proliferating conditions. Subsequently, cells were\n",
      "\n",
      "transduced with lentiviral vectors encoding GFP or GFP-McIdas (McIdas) and cultured under\n",
      "\n",
      "differentiating conditions for indicated time points.\n",
      "\n",
      "(B) Cortical astrocytes were immunostained against GFP (green) to mark infected cells, P73 (grey) and\n",
      "\n",
      "S100β (red) at day 7 of differentiation. Arrows point to McIdas-infected cells which are P73+ and\n",
      "\n",
      "represent cells committed to the ependymal lineage.\n",
      "\n",
      "(C-D) The graph depicts the percentage of the infected cells that co-express the ependymal marker P73\n",
      "\n",
      "over the total number of the infected cells (C) The percentage of the infected cells which display S100β\n",
      "\n",
      "staining around their cell body, which corresponds to ependymal cells, is depicted in the graph (D) Data\n",
      "\n",
      "are presented as the median ±interquartile range (IQR) of three independent experiments. Statistical\n",
      "\n",
      "significance was determined using the non-parametric two-tailed Mann–Whitney test (***p < 0.0001).\n",
      "\n",
      "(E) Transduced astrocytes were immunostained at differentiation day 14 against GFP (green), FoxJ1\n",
      "\n",
      "33\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "(grey) and pericentrin (PCNT, red). Note the presence of multiple basal bodies, based on PCNT signal\n",
      "\n",
      "accumulation, in McIdas-infected cells which co-express the ependymal marker FoxJ1 (arrows).\n",
      "\n",
      "(F-G) The graphs present the percentage of the infected cells that express FoxJ1 over the total number of\n",
      "\n",
      "the infected cells (F) and the percentage of FoxJ1-positive infected cells which also display\n",
      "\n",
      "accumulation of PCNT signal (G). Data are presented as the median ±IQR of three independent\n",
      "\n",
      "experiments. Statistical significance was determined using the non-parametric two-tailed Mann–\n",
      "\n",
      "Whitney test (***p < 0.0001).\n",
      "\n",
      "DNA was stained with Dapi (blue). Scale bars, 10 μm.\n",
      "\n",
      "Abbreviations, dd: differentiation day\n",
      "\n",
      "Figure 2.\n",
      "\n",
      "McIdas reprograms cortical astrocytes into functional ependymal cells.\n",
      "\n",
      "(A) Immunofluorescence of transduced cortical astrocytes at differentiation day 22 with antibodies\n",
      "\n",
      "against GFP (green), Glial Fibrillary Acidic Protein (GFAP, grey) which labels astrocytes and acetylated\n",
      "\n",
      "a-tubulin (ACT, red), a marker of ciliary axonemes. McIdas expression triggers the formation of\n",
      "\n",
      "multiple cilia, tiny hair-like structures which appear on the surface of the reprogrammed astrocytes\n",
      "\n",
      "(arrow). Note the downregulation of the GFAP signal in the reprogrammed multiciliated cells. DNA was\n",
      "\n",
      "stained with Dapi (blue). Scale bars, 10 μm\n",
      "\n",
      "(B-C) The graph shows the percentage of infected cells that express GFAP and correspond to an\n",
      "\n",
      "astrocytic population (B) The percentage of the infected cells which possess multiple cilia based on ACT\n",
      "\n",
      "staining is depicted in the graph (C) Data are presented as the median ±IQR of two independent\n",
      "\n",
      "experiments. Statistical significance was determined using the non-parametric two-tailed Mann–\n",
      "\n",
      "Whitney test (***p < 0.0001).\n",
      "\n",
      "(D) Live imaging microscopy experiments were performed on cortical astrocytes transduced either with\n",
      "\n",
      "GFP or McIdas lentiviruses. The positions of fluorescent beads included in the live imaging medium,\n",
      "\n",
      "were marked with colored dots on selected video frames. Τrack of one representative bead is shown in\n",
      "\n",
      "the right panel. Two independent experiments were performed. Scale bar, 10 μm\n",
      "\n",
      "Abbreviations, dd: differentiation day\n",
      "\n",
      "Figure 3.\n",
      "\n",
      "McIdas induces ependymal cells regeneration in a mouse model of intracranial hemorrhage\n",
      "\n",
      "34\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "hydrocephalus.\n",
      "\n",
      "(A) Mice received unilateral intracranial injection of Lysophosphatidic acid (LPA) to induce\n",
      "\n",
      "hydrocephalus. Coronal brain slices derived from brains electroporated either with a GFP or GFP\n",
      "McIdas (McIdas) plasmid were cultured for 5 days. Immunofluorescence experiments were performed\n",
      "\n",
      "with antibodies against GFP (green) and the ependymal marker P73 (red). Inserts show higher\n",
      "\n",
      "magnification images. Arrows indicate GFP+P73+ cells.\n",
      "\n",
      "(B) The graph depicts the percentage of the electroporated cells that co-express the ependymal marker\n",
      "\n",
      "P73, over the total number of the electroporated cells, at day 5 of culture. Data are presented as the\n",
      "\n",
      "median ± IQR of three independent experiments. Statistical significance was determined using the non\n",
      "parametric two-tailed Mann–Whitney test (***p = 0.0002).\n",
      "\n",
      "(C) Coronal brain slices were cultured for 9 days and immunostained against GFP (green), pericentrin\n",
      "\n",
      "(PCNT, red) and acetylated a-tubulin (ACT, grey) to detect mature multiciliated cells. Note the presence\n",
      "\n",
      "of multiple basal bodies and cilia in McIdas electroporated cells.\n",
      "\n",
      "(D-E) The percentage of the electroporated cells that carry multiple basal bodies, based on PCNT\n",
      "\n",
      "accumulation signal, upon 9 days of culture, is presented in the graph (D), (***p < 0.0001). The graph\n",
      "\n",
      "shows the percentage of multiciliated cells based on PCNT and ACT staining (E) upon 9 days of culture,\n",
      "\n",
      "(p=0.0013). Data are presented as the median ±IQR of three independent experiments. Statistical\n",
      "\n",
      "significance was determined using the non-parametric two-tailed Mann–Whitney test.\n",
      "\n",
      "DNA was stained with Dapi (blue). Scale bars, 10 μm. .\n",
      "\n",
      "Figure 4.\n",
      "\n",
      "McIdas ectopic expression promotes in vivo reprogramming into ependymal cells in a genetic\n",
      "\n",
      "mouse model of hydrocephalus.\n",
      "\n",
      "(A) Neonatal subventricular zone electroporation was conducted at GemC1-knockout mice at postnatal\n",
      "\n",
      "day 1 (P1) with plasmids encoding GFP, or GFP-McIdas (McIdas). Mice were sacrificed 4 days post\n",
      "\n",
      "electroporation and coronal brain sections were immunostained with antibodies against GFP (green) and\n",
      "\n",
      "P73 (red). Arrows point to McIdas-electroporated cells co-expressing P73, which represent cells\n",
      "\n",
      "committed to the ependymal lineage. Higher magnification of the boxed region is shown in the right\n",
      "\n",
      "panel.\n",
      "\n",
      "(B) The graph shows the percentage of the electroporated cells expressing P73. Data are presented as the\n",
      "\n",
      "median ±IQR of three independent experiments. Statistical significance was determined using the non\n",
      "35\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "parametric two-tailed Mann–Whitney test (***p < 0.0001).\n",
      "\n",
      "(C) Coronal brain sections from GemC1-knockout electroporated mice were stained with specific\n",
      "\n",
      "antibodies against GFP (green), Pericentrin (PCNT, red) and acetylated a-tubulin (ACT grey). Multiple\n",
      "\n",
      "basal bodies and cilia on the surface of McIdas electroporated cells are indicated by an arrow. Higher\n",
      "\n",
      "magnification of the boxed region is shown in the right panel.\n",
      "\n",
      "(D-E) Quantification of the percentage of the electroporated cells with multiple basal bodies, based on\n",
      "\n",
      "PCNT accumulation (D), (***p < 0.0001), and the percentage of multiciliated cells based on PCNT and\n",
      "\n",
      "ACT staining over the total number of the electroporated cells (E), (P=0.0041). Data are presented as the\n",
      "\n",
      "median ±IQR of two independent experiments. Statistical significance was determined using the non\n",
      "parametric two-tailed Mann–Whitney test.\n",
      "\n",
      "DNA was stained with Dapi (blue). Scale bars, 10 μm. .\n",
      "\n",
      "Abbreviations, LV: lateral ventricle\n",
      "\n",
      "Figure 5.\n",
      "\n",
      "McIdas establishes cilia motility in reprogrammed cells in a genetic mouse model of\n",
      "\n",
      "hydrocephalus.\n",
      "\n",
      "(A) Schematic representation of experimental procedures for (B-D). GFP or GFP-McIdas (McIdas)\n",
      "\n",
      "expressing plasmids were electroporated at the lateral ventricle of postnatal day 1 (P1) GemC1-knockout\n",
      "\n",
      "mouse brains. Coronal brain sections, that contained the lateral ventricles, were obtained from the\n",
      "\n",
      "electroporated mice at ages P7-P15 and were used in live imaging experiments.\n",
      "\n",
      "(B) Cilia still images from GFP or McIdas electroporated brains at indicated time points in milliseconds.\n",
      "\n",
      "FITC fluorescent cells are presented together with brightfield images. Arrows indicate the positions of\n",
      "\n",
      "cilia at indicated time points. Cilia motility was observed upon McIdas ectopic expression as evidenced\n",
      "\n",
      "by cilia changing positions at indicated time points. Higher magnification of the boxed regions is shown\n",
      "\n",
      "in the right panels.\n",
      "\n",
      "(C) Representative kymograms depicting cilia motility upon GFP or McIdas ectopic expression. Left\n",
      "\n",
      "panel images show the lines used to obtain the kymograms. These images are extracted from the image\n",
      "\n",
      "shown in B, which was utilized to generate the kymograph.(D) The graph shows cilia beat frequency of\n",
      "\n",
      "electroporated cells quantified from high-speed brightfield cilia motility movies. Data are presented as\n",
      "\n",
      "the median ±IQR (n=2 for GFP (control) and n=4 for McIdas). Statistical significance was determined\n",
      "\n",
      "using the non-parametric two-tailed Mann–Whitney test (***p < 0.0001).\n",
      "\n",
      "36\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "Figure 6.\n",
      "\n",
      "Ectopic expression of McIdas restores the subventricular zone niche cytoarchitecture.\n",
      "\n",
      "(A-B) GemC1-knockout mouse brains were electroporated with plasmids encoding GFP or McIdas-GFP\n",
      "\n",
      "(McIdas) at postnatal day 1 (P1). Mice were sacrificed at ages P7-P9 and lateral ventricular wall whole\n",
      "mount immunofluorescence was conducted with antibodies against GFP (green), β-catenin (red), and\n",
      "\n",
      "Vascular Cell adhesion Molecule 1 (VCAM1, grey) (A) or Glial Fibrillary Acidic Protein (GFAP, grey)\n",
      "\n",
      "(B).\n",
      "\n",
      "Formation of pinwheel structures by McIdas-reprogrammed ependymal cells surrounding VCAM1+\n",
      "\n",
      "neural stem cells (arrow, A) or GFAP-expressing neural stem cells (B) was observed. Note the enlarged\n",
      "\n",
      "apical domain of McIdas-reprogrammed ependymal cells.\n",
      "\n",
      "Higher magnification of the boxed region is shown in the right panel.\n",
      "\n",
      "(C) The graph shows the percentage of electroporated cells, shown in A, identified in pinwheel\n",
      "\n",
      "structures. Data are presented as the median ±IQR (n=3 for GFP (control) and n=4 for McIdas).\n",
      "\n",
      "Statistical significance was determined using the non-parametric two-tailed Mann–Whitney test (***p <\n",
      "\n",
      "0.0001.).\n",
      "\n",
      "Scale bars, 10 μm.\n",
      "\n",
      "EV Figure Legends\n",
      "\n",
      "Figure EV1.\n",
      "\n",
      "Mouse embryonic stem cells programming into ependymal cells.\n",
      "\n",
      "(A) Mouse embryonic stem cells (mESCs) were transfected with GFP, GFP-McIdas (McIdas) or GFP\n",
      "GemC1 (GemC1) expressing vectors and subsequently cultured under differentiating conditions.\n",
      "\n",
      "Immunofluorescence was performed with antibodies against GFP (green), FoxJ1 (red) and Pericentrin\n",
      "\n",
      "(PCNT, magenta) at differentiation day 4. The arrow points to multiple basal bodies in McIdas and\n",
      "\n",
      "GemC1 transfected cells which co-express the ependymal marker FoxJ1. Higher magnification of the\n",
      "\n",
      "boxed regions is shown in the right panels.\n",
      "\n",
      "(B-C) The graphs present the percentage of transfected mESCs that express FoxJ1 over the total number\n",
      "\n",
      "of the transfected cells (B) and the percentage of transfected cells which display multiple basal bodies\n",
      "\n",
      "37\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "based on the accumulation of PCNT signal (C) Data are presented as the median ±IQR of two\n",
      "\n",
      "independent experiments. Statistical significance was determined using the non-parametric two-tailed\n",
      "\n",
      "Mann–Whitney test (***p < 0.0001).\n",
      "\n",
      "(D) GFP (green), PCNT (magenta) and acetylated-a-tubulin (ACT red) immunostaining in transfected\n",
      "\n",
      "mESCs at differentiation day 6. McIdas and GemC1 transfected mESCs present multiple basal bodies\n",
      "\n",
      "and cilia.\n",
      "\n",
      "(E) The graph shows the percentage of transfected cells with multiple basal bodies (PCNT\n",
      "\n",
      "accumulation) and multiple cilia (ACT labeled cilia) over the total number of the transfected cells. Data\n",
      "\n",
      "are presented as the median ±IQR of two independent experiments. Statistical significance was\n",
      "\n",
      "determined using the non-parametric two-tailed Mann–Whitney test (***p < 0.0001).\n",
      "\n",
      "DNA was stained with Dapi (blue). Scale bars, 10 μm,\n",
      "\n",
      "Abbreviations, dd: differentiation day\n",
      "\n",
      "Figure EV2.\n",
      "\n",
      "GemC1 drives the commitment of murine cortical astrocytes towards the ependymal cell lineage.\n",
      "\n",
      "(A) Cortical astrocytes were transduced with lentiviruses encoding GFP or GFP-GemC1 (GemC1).\n",
      "\n",
      "Immunofluorescence experiments were performed at day 7 of differentiation with specific antibodies\n",
      "\n",
      "against GFP (green) to mark the infected cells, P73 (grey) and S100β (red). Arrow point to a P73+\n",
      "\n",
      "infected cell, committed to the ependymal lineage.\n",
      "\n",
      "(B-C) Graph depicting the percentage of infected cells which express P73 over the total number of\n",
      "\n",
      "infected cells (B). The graph shows the percentage of the infected cells which display S100β staining\n",
      "\n",
      "around their cell body (C). Three independent experiments were analyzed. Data are presented as the\n",
      "\n",
      "median ±IQR. Statistical significance was determined using the non-parametric two-tailed Mann–\n",
      "\n",
      "Whitney test (***p < 0.0001).\n",
      "\n",
      "(D) Astrocytes were immunostained against GFP (green), FoxJ1 (grey) and pericentrin (PCNT red) at\n",
      "\n",
      "differentiation day 14. The arrow points to a GemC1-infected cell which expresses FoxJ1 and possesses\n",
      "\n",
      "multiple basal bodies.\n",
      "\n",
      "(E-F) Graph presenting the percentage of the infected cells that express FoxJ1 over the total number of\n",
      "\n",
      "the infected cells (E) The graph shows the percentage of infected cells which express the ependymal\n",
      "\n",
      "marker FoxJ1 and display accumulation of PCNT signal (F). Data are presented as the median ±IQR\n",
      "\n",
      "from three independent experiments. Statistical significance was determined using the non-parametric\n",
      "\n",
      "38\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "two-tailed Mann–Whitney test (***p < 0.0001).\n",
      "\n",
      "DNA was stained with Dapi (blue). Scale bars, 10 μm\n",
      "\n",
      "Abbreviations, dd: differentiation day\n",
      "\n",
      "Figure EV3.\n",
      "\n",
      "Lysophosphatidic Acid (LPA) administration causes hydrocephalus in early postnatal stages in\n",
      "\n",
      "mice.\n",
      "\n",
      "(A) Top views of Postnatal day 7 (P7) wild type mouse brains that either received or not injection of\n",
      "\n",
      "LPA at the lateral ventricle at P5. No differences on the size of the brain were observed macroscopically.\n",
      "\n",
      "(B) Dapi staining on coronal brain sections from P7 mouse brains reveals the dilation of the lateral\n",
      "\n",
      "ventricles after LPA injections. Scale bar, 100 μm.\n",
      "\n",
      "(C) Immunofluorescence for Glial Fibrillary Acidic Protein (GFAP, green) which labels astrocytes and\n",
      "\n",
      "acetylated a-tubulin (ACT, red), a marker of ciliary axonemes on coronal sections from P7 mouse brains\n",
      "\n",
      "at the dorsal-lateral and lateral regions of the lateral walls. Arrowheads point to multiple cilia in control\n",
      "\n",
      "ependymal cells (ep). Arrows show GFAP positive astrocytes (as) in the lateral wall of LPA-injected\n",
      "\n",
      "brains, where ciliary disruption is observed. Higher magnification of the boxed regions is shown in the\n",
      "\n",
      "right panels. Scale bars, 10 μm\n",
      "\n",
      "DNA was stained with Dapi or Draq-5 (blue).\n",
      "\n",
      "Figure EV4.\n",
      "\n",
      "Ex vivo reprogramming of radial glial cells to ependyma can be achieved through McIdas ectopic\n",
      "\n",
      "expression.\n",
      "\n",
      "(A) Schematic representation of experimental procedures for (B-E). Postnatal radial glial cells (pRGCs)\n",
      "\n",
      "were isolated from GemC1-knockout newborn mice, cultured and infected with lentiviruses expressing a\n",
      "\n",
      "GFP-McIdas (McIdas) or a GFP-P73 (P73) fusion protein, while GFP alone was used as a control. Cells\n",
      "\n",
      "were then cultured under differentiating conditions and analyzed at indicated timepoints.\n",
      "\n",
      "(B) Transduced radial glial cells were co-stained with antibodies against GFP (green) to mark infected\n",
      "\n",
      "cells, FoxJ1 (red) and pericentrin (PCNT, grey) 5 days after the initiation of differentiation. The arrow\n",
      "\n",
      "indicates the accumulation of PCNT signal in FoxJ1 expressing cells upon McIdas ectopic expression.\n",
      "\n",
      "(C) The graph presents the percentage of infected cells which express FoxJ1 and display multiple basal\n",
      "\n",
      "bodies (accumulation of PCNT signal). Three independent experiments were analyzed. Data are\n",
      "\n",
      "39\n",
      "\n",
      "\n",
      "'****\n",
      "\n",
      "****page_content='\n",
      "presented as the median ±IQR. Statistical significance was determined using the non-parametric two\n",
      "tailed Mann–Whitney test (***p < 0.0001).\n",
      "\n",
      "(D) Radial glial cells infected with GFP, McIdas, or P73 lentiviruses were labeled with antibodies\n",
      "\n",
      "against GFP (green), pericentrin (PCNT, grey) and acetylated a-tubulin (ACT, red) to detect mature\n",
      "\n",
      "multiciliated cells at differentiation day 15 (arrow).\n",
      "\n",
      "(E) The percentage of the infected cells which displayed multiple basal bodies, based on PCNT staining\n",
      "\n",
      "and simultaneously multiple cilia, based on ACT staining, was analyzed. Data are presented as the\n",
      "\n",
      "median ±IQR of three independent experiments. Statistical significance was determined using the non\n",
      "parametric two-tailed Mann–Whitney test (***p < 0.0001).\n",
      "\n",
      "DNA was stained with Hoechst (blue). Scale bars, 10 μm,.\n",
      "\n",
      "Figure EV5.\n",
      "\n",
      "McIdas drives the commitment to the ependymal cell lineage in a genetic mouse model for\n",
      "\n",
      "hydrocephalus.\n",
      "\n",
      "(A) Subventricular zone electroporation was conducted at postnatal day 1 (P1) GemC1-knockout mice\n",
      "\n",
      "with plasmids encoding GFP, or GFP-McIdas (McIdas). Coronal brain sections were stained with\n",
      "\n",
      "antibodies against GFP (green) to mark the electroporated cells and FoxJ1 (red), a marker of committed\n",
      "\n",
      "ependymal cells, 4 days post electroporation. Arrows point to McIdas-electroporated cells expressing\n",
      "\n",
      "FoxJ1. Higher magnification of the boxed region is shown in the right panel.\n",
      "\n",
      "(B) Graph depicting the percentage of electroporated cells that express FoxJ1 over the total number of\n",
      "\n",
      "the electroporated cells. Two independent experiments were analyzed. Data are presented as the median\n",
      "\n",
      "±IQR. Statistical significance was determined using the non-parametric two-tailed Mann–Whitney test\n",
      "\n",
      "(***p < 0.0001).\n",
      "\n",
      "DNA was stained with Dapi (blue). Scale bars, 10 μm\n",
      "\n",
      "40\n",
      "\n",
      "\n",
      "'****\n",
      "\n"
     ]
    }
   ],
   "source": [
    "import pymupdf4llm\n",
    "from langchain_text_splitters.markdown import ExperimentalMarkdownSyntaxTextSplitter\n",
    "\n",
    "# Get the MD text\n",
    "md_text = pymupdf4llm.to_markdown(DOCX_FILES[1])  # get markdown for all pages\n",
    "\n",
    "# Initialize the MarkdownHeaderTextSplitter\n",
    "splitter = ExperimentalMarkdownSyntaxTextSplitter()\n",
    "\n",
    "chunks = splitter.split_text(md_text)\n",
    "\n",
    "for chunk in chunks:\n",
    "    print(f\"****{chunk}****\\n\")\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Note that the markdown text is indeed `plain text`. The reason is that there is a comercial use of this library that is the one that actually supports passign DOCX to md."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\"Research Article\\n\\nTitle: Ependymal cell lineage reprogramming as a potential therapeutic intervention for\\n\\nhydrocephalus\\n\\nAuthors: Konstantina Kaplani[1], Maria - Eleni Lalioti[1], Stella Vassalou[1], Georgia Lokka[1], Evangelia\\n\\nParlapani[1], Georgios Kritikos[1], Zoi Lygerou[2], Stavros Taraviras[1,]\\n\\nAffiliations:\\n\\n1Department of Physiology, School of Medicine, University of Patras, Patras, Greece\\n\\n2Department of General Biology, School of Medicine, University of Patras, Patras, Greece\\n\\nCorresponding author: Stavros Taraviras\\n\\nCorresponding author’s address: Department of Physiology, School of Medicine, University of Patras,\\n\\nAsklepiou Street 1, Rio 26504, Patras, Greece\\n\\nCorresponding author’s phone and fax: +30 2610 997676\\n\\nCorresponding author’s e-mail address: [taraviras@med.upatras.gr](mailto:taraviras@med.upatras.gr)\\n\\nRunning title: GemC1 and McIdas promote ependymal reprogramming.\\n\\nKeywords: ependymal cells, reprogramming, hydrocephalus, McIdas, GemC1\\n\\n\\n-----\\n\\nABSTRACT\\n\\nHydrocephalus is a common neurological condition, characterized by the excessive accumulation of\\n\\ncerebrospinal fluid in the cerebral ventricles. Primary treatments for hydrocephalus mainly involve\\n\\nneurosurgical cerebrospinal fluid diversion, which hold high morbidity and failure rates, highlighting the\\n\\nnecessity for the discovery of novel therapeutic approaches. Although the pathophysiology of\\n\\nhydrocephalus is highly multifactorial, impaired function of the brain ependymal cells plays a\\n\\nfundamental role in hydrocephalus. Here we show that GemC1 and McIdas, key regulators of\\n\\nmulticiliated ependymal cell fate determination, induce direct cellular reprogramming towards\\n\\nependyma. Our study reveals that ectopic expression of GemC1 and McIdas reprograms cortical\\n\\nastrocytes and programs mouse embryonic stem cells into ependyma. McIdas is sufficient to establish\\n\\nfunctional activity in the reprogrammed astrocytes. Furthermore, we show that McIdas’ expression\\n\\npromotes ependymal cells regeneration in two different postnatal hydrocephalus mouse models: an\\n\\nintracranial hemorrhage and a genetic form of hydrocephalus and ameliorates the cytoarchitecture of the\\n\\nneurogenic niche. Our study provides evidence on the restoration of ependyma in animal models\\n\\nmimicking hydrocephalus that could be exploited towards future therapeutic interventions.\\n\\n2\\n\\n\\n-----\\n\\nINTRODUCTION\\n\\nHydrocephalus is a neurological disorder defined by the abnormal accumulation of cerebrospinal fluid\\n\\n(CSF) within the brain ventricles, leading to increased head circumference (Schrander-Stumpel & Fryns,\\n\\n1998). The condition can arise as a primary clinical feature triggered by a genetic cause (congenital\\n\\nhydrocephalus) or secondary (acquired hydrocephalus) to other insults such as central nervous system\\n\\n(CNS) infections, tumors, trauma, or hemorrhage (Rekate, 2009; Kahle et al, 2016; Kousi & Katsanis,\\n\\n2016). Malfunction of the ependymal cells is a leading cause of the pathophysiological mechanism of\\n\\nhydrocephalus as these cells have a key role in CSF circulation and composition (Ji et al, 2022). In\\n\\naddition, hydrocephalus is linked with disorganization of the subventricular neurogenic niche’s\\n\\ncytoarchitecture and neurogenesis defects, as ependymal cells were shown to provide structural and\\n\\nfunctional support to the adult neural stem cells (Paez-Gonzalez et al, 2011; McAllister, James P. et al,\\n\\n2017; Rodríguez & Guerra, 2017).\\n\\nUnder homeostatic conditions, ependymal cells form one of the largest epithelia of the brain, lying\\n\\nadjacent to the ventricular lumen and bear unique structural characteristics. The cell surface exposed to\\n\\nthe ventricular lumen contains a patch of approximately 50 modified centrioles, called basal bodies,\\n\\nwhich are tiny microtubule-based organelles responsible for nucleating an equal number of motile cilia.\\n\\nCilia are anchored οn their apical surface, beating in the same direction and in a coordinated fashion,\\n\\nthus contributing to efficient CSF circulation throughout the brain (Mirzadeh et al, 2008, 2010).\\n\\nConstant ciliary beating facilitates secreted molecules dispersion to other regions of the CNS. Such\\n\\ncirculating molecules in the CSF influence neurogenesis and neuroblasts’ migration to the olfactory\\n\\nbulbs (Lim et al, 2000; Sawamoto et al, 2006).\\n\\nEpendymal cell differentiation is a multi-step process orchestrated by a tightly regulated transcriptional\\n\\nprogram which is responsible for activating numerous genes that promote amplification and docking of\\n\\ncentrioles and the formation of cilia (Kyrousi et al, 2017; Arbi et al, 2018). We have previously provided\\n\\nevidence that the Geminin family proteins, GEMC1 and MCIDAS, are the earliest regulators for the cell\\n\\nfate commitment to the ependymal lineage (Kyrousi et al, 2015). GemC1 lies upstream of McIdas\\n\\ninducing its transcriptional activation, while both upregulate genes that are essential for centriole\\n\\namplification and for promoting the downstream transcriptional machinery of multiciliogenesis (Kyrousi\\n\\net al, 2015; Arbi et al, 2016; Lalioti et al, 2019a). GEMC1 and MCIDAS cooperate with E2F4/5\\n\\n3\\n\\n\\n-----\\n\\ntranscription factors, to upregulate cMyb and Ccno, which are implicated in centriole amplification (Tan\\n\\net al, 2013; Wallmeier et al, 2014; Funk et al, 2015; Arbi et al, 2018). In addition, we and others have\\n\\nprovided evidence that GemC1 and McIdas drive the transcriptional activation of P73 and FOXJ1\\n\\ntranscription factors, which are implicated in basal bodies docking and cilia motility (Kyrousi et al,\\n\\n2015; Arbi et al, 2016; Nemajerova et al, 2016; Marshall et al, 2016; Lalioti et al, 2019a).\\n\\nGiven the multifaceted roles that ependymal cells have in hydrocephalus pathophysiology, restoration of\\n\\nependymal cells could significantly contribute to hydrocephalus therapy. With the advancements in\\n\\ndirect reprogramming-based strategies for conditions affecting the central nervous system (Bocchi et al,\\n\\n2022), it is plausible to anticipate that cellular reprogramming directed towards ependymal cells could\\n\\nrepresent a significant breakthrough in the treatment of hydrocephalus. In the present study we showed\\n\\nthat ectopic expression of GemC1 or McIdas promotes programming of mouse pluripotent embryonic\\n\\nstem cells and reprogramming of cortical astrocytes into ependymal cells, with McIdas being more\\n\\nefficient on establishing functional motile cilia in reprogrammed astrocytes. In addition, forced\\n\\nexpression of McIdas in both a genetic and an acquired hydrocephalus mouse model promoted in vivo\\n\\nreprogramming of cells residing in the ventricular walls of the diseased brain into functional ependymal\\n\\ncells. Importantly, reprogrammed ependymal cells were able to beat their cilia in a coordinated fashion\\n\\nand formed pinwheel structures together with neural stem cells in hydrocephalic mice, revealing their\\n\\npotential to regenerate the architecture of the neurogenic niche, which is disrupted in hydrocephalus.\\n\\nCollectively, our data suggest that GEMC1 and MCIDAS can orchestrate the transcriptional program of\\n\\nmulticiliogenesis establishing ependymal cell fate and differentiation, providing proof of principle\\n\\nevidence that restoration of ependymal cells could potentially contribute to hydrocephalus management.\\n\\n4\\n\\n\\n-----\\n\\nRESULTS\\n\\nGemC1 and McIdas promote programming of mouse embryonic stem cells and reprogramming of\\n\\ncortical astrocytes into ependymal cells\\n\\nGemC1 and McIdas are the most upstream regulators driving multiciliogenesis in vertebrate organisms\\n\\n(Stubbs et al, 2012; Ma et al, 2014; Boon et al, 2014; Kyrousi et al, 2015; Terré et al, 2016; Arbi et al,\\n\\n2016). We therefore postulated that GemC1 and McIdas would be capable of inducing stem cell\\n\\nprogramming towards multiciliated ependymal cells. To examine this hypothesis, mouse embryonic\\n\\nstem cells (mESCs) were transfected with plasmids encoding McIdas or GemC1 in conjunction with\\n\\nGFP (referred to as McIdas and GemC1 respectively), or GFP alone as a control. Transfected mESCs\\n\\nwere cultured for four days in the absence of Leukemia Inhibitory Factor (LIF) and subjected to\\n\\nimmunostaining for FoxJ1, a key transcription factor for the differentiation of ependymal cells, and\\n\\nPericentrin which marks nascent basal bodies (Fig EV1A). Our analysis showed that 87% of McIdas\\nand 39% of GemC1-overexpressing mESCs were expressing FoxJ1, while only 1% FoxJ1 positive cells\\n\\nwere detected in the control (Fig EV1B). In addition, we identified that 90% of McIdas- and 59% of\\n\\nGemC1-overexpressing mESCs showed accumulation of Pericentrin staining (Fig EV1C), thus\\n\\npresented multiple nascent basal bodies, corresponding to differentiating ependymal cells. Furthermore,\\n\\nwe examined whether McIdas and GemC1-overexpressing mESCs acquire multiple cilia, a unique\\n\\nstructural characteristic of the ependymal cells. Double immunofluorescence experiments were\\n\\nperformed six days following mESCs transfection using anti-Pericentrin and anti-acetylated-a-tubulin\\n\\nantibodies to mark the basal bodies and cilia, respectively (Fig EV1D). Our analysis revealed that 66%\\n\\nof McIdas- and 21% of GemC1- overexpressing mESCs develop multiple cilia arising from numerous\\n\\nbasal bodies. On the contrary, control mESCs develop neither multiple basal bodies nor cilia (Fig\\n\\nEV1D, E). Our data show that ectopic expression of McIdas and GemC1 programs mESCs into\\n\\n5\\n\\n\\n-----\\n\\nmulticiliated ependymal cells.\\n\\nNext, we sought out to explore the reprogramming potential of McIdas and GemC1 in astrocytes that\\n\\nunder physiological conditions do not differentiate into ependymal cells. Astrocytes were isolated from\\n\\nthe cortex of neonatal mice and were subsequently infected with lentiviruses expressing GFP-McIdas or\\n\\nGFP-GemC1 (referred to as McIdas and GemC1, respectively), while lentiviruses expressing GFP alone\\n\\nwere used as control. Cells were subsequently cultured under differentiating conditions and\\n\\nimmunofluorescence experiments were conducted to assess the expression of known ependymal and\\n\\nastrocytic markers (Fig. 1A). Ectopic expression of both McIdas and GemC1 resulted in P73 expression,\\n\\na marker of the early steps of ependymogenesis, in 37% and 36% of McIdas- and GemC1- infected\\n\\ncells, respectively, while in the control condition only 4% of the transduced cells was expressing P73\\n\\n(Fig. 1B, C, and EV 2A, B). Notably, concurrent changes in the expression pattern of the astrocytic\\n\\nmarker S100β were observed in approximately 20% of McIdas- and 18% of GemC1- infected\\n\\nastrocytes, as S100β expression was detected around their cell body, which is characteristic for\\n\\nependymal cells (Mirzadeh et al, 2008) (Fig. 1B-D and EV. 2A-C). On the contrary, control astrocytes\\n\\nretained a cytoplasmic expression of the S100β protein, a typical feature of astrocytes. Overall, our data\\n\\nsuggest that ectopic expression of McIdas and GemC1 can lead to the acquisition of the ependymal cell\\n\\nfate.\\n\\nSubsequently, we examined whether McIdas- and GemC1-transduced astrocytes successfully\\n\\ndifferentiate into ependymal cells. We performed immunofluorescence experiments using specific\\n\\nantibodies recognizing FoxJ1 and Pericentrin, to identify astrocytes that acquired the ependymal fate and\\n\\nsuccessfully managed to multiply their basal bodies, respectively. Our analysis showed that fourteen\\n\\ndays post infection 49% of McIdas- and 19% of GemC1- infected astrocytes were expressing FoxJ1\\n\\n(Fig. 1E, F and EV2D, E). In addition, 28% of the McIdas- and 12% of GemC1- infected astrocytes\\n\\n6\\n\\n\\n-----\\n\\npossessed multiple basal bodies, indicated by the accumulation of Pericentrin signal (Fig. 1G and\\n\\nEV2F). Moreover, expression of a GFP lentivirus did not result into FoxJ1 expression nor accumulation\\n\\nof Pericentrin.\\n\\nIn addition, we examined the ability of McIdas and GemC1 to repress the astrocytic identity of the\\n\\ntransduced astrocytes. Towards this direction, the combined expression of the astrocytic markers GFAP\\n\\nand S100β was assessed through immunofluorescence experiments at fourteen days post infection. Our\\n\\nanalysis revealed that McIdas ectopic expression resulted to 23% of GFAP+/S100β+ double positive\\n\\ncells compared to 31% in the control condition (Appendix Fig. S1A, B). We did not observe statistically\\n\\nsignificant change in the percentage of GFAP+/S100β+ double positive cells following GemC1 ectopic\\n\\nexpression (Appendix Fig. S1C). In addition, we examined the percentage of infected cells that were\\n\\nGFAP- and reveal S100β expression around the cell body, which correspond to ependymal cells. We\\n\\nshowed that 28% McIdas- and 22% GemC1- infected astrocytes present this expression pattern\\n\\n(Appendix Fig. S1A, B), which is in accordance with our analysis performed in earlier stages of\\n\\nreprogramming (Fig. 1D). Our results indicate that a significant percentage of the infected astrocytes are\\n\\nreprogrammed to the ependymal lineage and possess unique ependymal morphological characteristics.\\n\\nMoreover, we showed that McIdas is more potent in the downregulation of the astrocytic identity of the\\n\\ntransduced cells.\\n\\nGemC1 is positioned upstream of McIdas in the molecular pathway governing ependymogenesis, as it\\n\\nregulates the transcriptional activation of McIdas and controls the initial fate commitment towards the\\n\\nependymal lineage (Kyrousi et al, 2015, 2017). Our results suggest that GemC1 was less effective than\\n\\nMcIdas in reprogramming of astrocytes. We have therefore examined whether ectopic expression of\\n\\nGemC1 in astrocytes would not induce McIdas expression at sufficient levels. To test this hypothesis, we\\n\\nconducted immunofluorescence experiments using an McIdas specific antibody and quantified McIdas\\n\\n7\\n\\n\\n-----\\n\\nfluorescence in McIdas- and GemC1-infected astrocytes seven days post infection (Appendix Fig. S2A).\\n\\nOur results revealed that McIdas’ ectopic expression led to higher expression levels compared to GemC1\\n\\n(Appendix Fig. S2B), providing a potential mechanism for their differential ability to induce\\n\\nreprogramming.\\n\\nGiven the greater capacity of McIdas to induce the ependymal differentiation in astrocytes compared to\\n\\nGemC1, we hypothesized that McIdas would reprogram cortical astrocytes into fully mature ependymal\\n\\ncells carrying multiple basal bodies nucleating motile cilia. To address this point, we initially performed\\n\\nimmunofluorescence experiments in transduced astrocytes twenty-two days post infection using\\n\\nantibodies against Centriolin (also known as Cep110), a known centrosomal marker, and Meig1 which\\n\\nwas recently characterized as ependymal cell marker (MacDonald et al, 2021). We detected\\n\\naccumulation of both Centriolin (Appendix Fig.S3A) and Meig1 (Appendix Fig.S3B) signal in McIdas\\ninfected astrocytes, verifying the generation of multiple basal bodies upon McIdas’ ectopic expression.\\n\\nNext, we performed immunofluorescence experiments making use of antibodies against acetylated a\\ntubulin, which stains ciliary axonemes, and the astrocytic marker glial fibrillary acidic protein (GFAP).\\n\\nThe ectopic expression of McIdas resulted in the downregulation of GFAP expression and the generation\\n\\nof multiple cilia on the surface of the reprogrammed astrocytes, revealing the loss of the astrocytic\\n\\nidentity and the establishment of the ependymal differentiation (Fig. 2A). Our analysis showed that 18%\\n\\nof McIdas- infected astrocytes were reprogrammed into multiciliated ependymal cells based on\\n\\nacetylated-a-tubulin staining (Fig. 2C), which was consistent with the percentage of McIdas- infected\\n\\ncells that downregulate the expression of the astrocytic marker GFAP (Fig. 2B). Additionally, live\\n\\nimaging microscopy experiments were performed in transduced astrocytes twenty-two days post\\n\\ninfection to address the ability of the reprogrammed astrocytes multicilia to beat (Movies EV1-4). Using\\n\\nhigh-speed video microscopy, we acquired fast video recordings from GFP (Movies EV1-2) and\\n\\n8\\n\\n\\n-----\\n\\nMcIdas- infected astrocytes (Movies EV3-4), which revealed the presence of ciliary motility in McIdas\\ninfected cells as opposed to the absence of motility in GFP- infected cells. Importantly, our analysis\\n\\nshowed that McIdas expression established cilia movement that was sufficient to propel fluorescent\\n\\nparticles added to the imaging medium, highlighting the functionality of the ependymal cells and their\\n\\ncapability to create a fluid flow. On the contrary, astrocytes infected with a control virus were unable to\\n\\npropel the fluorescent particles, as were only observed to have a rather stable forward and backward\\n\\nmotion (Fig. 2D).\\n\\nOur findings show that GemC1 and McIdas promote the early steps of the multiciliogenesis program in\\n\\nboth embryonic stem cells and astrocytes. Importantly, McIdas has a greater capacity for eliciting direct\\n\\nreprogramming into functional ependymal cells.\\n\\nMcIdas induces ependymal cells regeneration in a mouse model of intracranial hemorrhage\\n\\nhydrocephalus\\n\\nIn both human and murine hydrocephalus, the disrupted ependymal cell population is replaced by\\n\\nastrocytes. However, this astrocytic scarring along the denuded ventricular wall fails to restore the\\n\\nependymal functions (Sival et al, 2011; Roales-Buján et al, 2012; Guerra et al, 2015; McAllister, James\\n\\nP. et al, 2017). Based on our finding that McIdas can successfully reprogram cortical astrocytes into\\n\\nfunctional ependymal cells, we examined whether McIdas could induce direct reprogramming of\\n\\nperiventricular cells in hydrocephalic models. We used an established mouse model of intracranial\\n\\nhemorrhage hydrocephalus induced by the elevated concentration of Lysophosphatidic Acid (LPA) in\\n\\nthe brain (Yung et al, 2015; Lummis et al, 2019), which closely mirrors the most common type of\\n\\nhuman hydrocephalus.\\n\\n9\\n\\n\\n-----\\n\\nBased on previously established protocols (Yung et al, 2011; Lummis et al, 2019), postnatal day five\\n\\n(P5) mice received intracranial injections of LPA and mouse brains were examined two days later to\\n\\nevaluate hydrocephalus occurrence. Although no obvious differences were observed in the size of LPA\\ntreated brains macroscopically (Fig. EV3A), coronal brain sections revealed ventricular dilation in LPA\\ninjected animals compared to non-injected ones (Fig. EV3B), confirming the development of\\n\\nhydrocephalus. In addition, acetylated-a-tubulin immunofluorescence revealed the ciliary disruption in\\n\\nependymal cells alongside the ventricular walls upon LPA administration (Fig. EV3C), in line with\\n\\nprevious studies (Lummis et al, 2019). On the contrary, ependymal cells with multiple cilia covered the\\n\\nentire ventricular walls in control animals (Fig. EV3C). Given that the ventricular walls of the LPA\\ninjected hydrocephalic mice were denuded from ependymal cells we wished to determine their cellular\\n\\ncomposition. We performed immunofluorescence with an antibody against the astrocytic marker glial\\n\\nfibrillary acidic protein (GFAP). Our analysis showed that GFAP positive cells with long processes,\\n\\nwhich likely corresponded to reactive astrocytes, covered the walls of the lateral ventricles in regions\\n\\nwhere ependymal cells localize under physiological conditions (Fig. EV3C), in line with previous\\n\\nfindings (Roales-Buján et al, 2012).\\n\\nTo assess whether McIdas can induce direct reprogramming into ependyma in LPA-induced\\n\\nhydrocephalus, GFP-McIdas (McIdas) or GFP expressing plasmids were electroporated onto P7 brains\\n\\nisolated from LPA-treated mice by targeting the dorsolateral wall of one lateral ventricle to introduce\\n\\nplasmids in cells located at periventricular regions. Subsequently, thick coronal brain sections at the\\n\\nlevel of the lateral ventricles were cultured and immunofluorescence was performed using established\\n\\nependymal markers (Fig. 3A). Five days post electroporation, 77% of the McIdas-electroporated cells\\n\\nexpressed P73, as opposed to 13% of the GFP-electroporated cells, suggesting that astrocytes that\\n\\nreplaced damaged ependymal cells in LPA treated hydrocephalic mouse models acquired the ependymal\\n\\n10\\n\\n\\n-----\\n\\nfate (Fig. 3B).\\n\\nAs ependymal cells normally require approximately two weeks until they acquire their mature\\n\\ncharacteristics in vivo, we followed differentiation of electroporated cells at later time points and\\n\\nexamined whether they differentiate into fully mature multiciliated cells. Towards this direction, we\\n\\nperformed immunolabeling experiments making use of antibodies recognizing Pericentrin to mark basal\\n\\nbodies and acetylated-a-tubulin to detect cilia (Fig. 3C). Nine days post electroporation 67% of McIdas\\n\\nelectroporated cells carried multiple basal bodies (Fig. 3D), while 16% of McIdas-electroporated cells\\n\\ncarried multiple cilia as revealed by acetylated-a-tubulin immunostaining (Fig. 3E). Basal bodies\\n\\namplification or multiple cilia formation was not detected in any of the control cells, electroporated with\\n\\na GFP expressing plasmid.\\n\\nCollectively, our data show that McIdas forced expression in brain periventricular cells has the potential\\n\\nto regenerate the ependymal cells in a mouse model of intracranial hemorrhage hydrocephalus.\\n\\nMcIdas promotes in vivo reprogramming towards functional ependymal cells in a model of\\n\\ncongenital hydrocephalus\\n\\nWe have recently described that genetic mutations in the human and mouse GemC1 gene lead to\\n\\nhydrocephalus (Lalioti et al, 2019b). Moreover, we have shown that radial glial cells lacking functional\\n\\nGemC1 fail to commit to the ependymal lineage and therefore do not develop ependymal cells during\\n\\nembryogenesis. We took advantage of the GemC1-knockout genetic mouse model of hydrocephalus we\\n\\nhave generated to investigate whether reprogramming towards the ependymal cells could be achieved in\\n\\na model of congenital hydrocephalus.\\n\\nWe initially examined whether McIdas could program radial glial cells derived from GemC1-knockout\\n\\n11\\n\\n\\n-----\\n\\nhydrocephalic mice into ependymal cells ex vivo. Postnatal radial glial cells isolated from GemC1\\nknockout mice were infected with lentiviruses expressing either a GFP-McIdas (McIdas) fusion protein\\n\\nor GFP as a control. Infected cells were analyzed at different time points for the expression of known\\n\\nependymal markers by immunofluorescence experiments, using specific antibodies recognizing FOXJ1,\\n\\nPericentrin and acetylated-a-tubulin proteins (Fig. EV4A). At five days post infection, 86% of the\\n\\nMcIdas-infected cells expressed FoxJ1 and showed accumulation of multiple basal bodies, based on\\n\\nPericentrin immunostaining (Fig. EV4B, C). Moreover, our analysis revealed that 97% of the McIdas\\n\\ntransduced cells were multiciliated 15 days following infection, based on acetylated-a-tubulin and\\n\\nPericentrin staining (Fig. EV3D, E). By contrast, in GFP transduced cells neither FoxJ1 expression nor\\n\\nmultiple basal bodies and cilia were observed. In addition, similar experiments were performed by\\n\\nectopically expressing P73 in postnatal radial glial cells isolated from GemC1-knockout mice. Ectopic\\n\\nexpression of P73 was not sufficient to induce FoxJ1 expression nor multiple basal bodies or multiple\\n\\ncilia formation in GemC1-knockout radial glial cells (Fig. EV4A-E). These results show that cells\\n\\nderived from a genetic model of hydrocephalus can be programmed ex vivo into multiciliated ependymal\\n\\ncells.\\n\\nIn order to examine whether McIdas promotes cellular reprogramming towards the ependymal cell\\n\\nlineage in vivo, we have used our genetic mouse model of hydrocephalus which was previously\\n\\nestablished (Lalioti et al, 2019b). Plasmid vectors encoding McIdas in conjunction with GFP or GFP\\n\\nalone were electroporated into one lateral ventricle of newborn GemC1-knockout mouse brains targeting\\n\\nperiventricular cells which could be either neural progenitor cells or reactive astrocytes that have\\n\\nreplaced the lost ependymal cell population (Roales-Buján et al, 2012; Gonzalez-Cano et al, 2016; Abdi\\n\\net al, 2018; Lalioti et al, 2019b). Reprogramming towards the ependymal cells was evaluated through\\n\\nimmunofluorescence, using molecular markers that determine different stages of ependymogenesis.\\n\\n12\\n\\n\\n-----\\n\\nInterestingly, four days post electroporation 61% and 58% of McIdas-electroporated cells were\\n\\nimmunopositive for the ependymal markers P73 (Fig. 4A, B) and FoxJ1, respectively (Fig. EV5A, B).\\n\\nThe expression of P73 and FoxJ1 was not detected in control animals, in agreement to what was\\n\\npreviously described (Lalioti et al, 2019b), as GemC1-knockout mice lack ependymal cells. We also\\n\\nshowed that McIdas further promoted the differentiation of the electroporated ependymal cells as basal\\n\\nbodies amplification was observed, based on Pericentrin immunofluorescence (Fig. 4C). More\\n\\nspecifically, upon McIdas overexpression 44% of the electroporated cells acquired multiple basal bodies\\n\\n(Fig. 4D). In addition, immunofluorescence against acetylated-a-tubulin revealed that McIdas\\n\\noverexpression resulted in the formation of cilia in a significant number of cells, corresponding to 7% of\\n\\nMcIdas electroporated cells (Fig. 4E). Noticeably, ciliated cells were located close to the ventricles,\\n\\nresembling normal mature multiciliated ependymal cells, which form a single-layered ciliated\\n\\nepithelium adjacent to the brain ventricle.\\n\\nIn order to examine whether the cilia that were generated upon McIdas ectopic expression were motile,\\n\\nthick coronal brain sections from GemC1-knockout mice covering the lateral brain ventricles were\\n\\nobtained and used in live imaging microscopy experiments (Fig. 5A). Electroporated cells were\\n\\nidentified under a FITC filter and their cilia beating capacity was examined in fast video records. A\\n\\nproportion of cells with multiple beating cilia that were GFP-negative in sections derived from McIdas\\nelectroporated brains were observed. We have previously shown that GemC1-knockout mice are unable\\n\\nto generate ependymal cells (Lalioti et al, 2019b), therefore cells carrying multiple beating cilia that do\\n\\nnot appear GFP-positive were possibly observed due to the loss of the transgene because of its transient\\n\\nexpression and were thus undetectable under the FITC filter. In addition to this, we cannot exclude the\\n\\npossibility of dilution of the GFP signal due to continued proliferation. In agreement with what was\\n\\npreviously described cilia were not detected in brain sections following GFP overexpression. However,\\n\\n13\\n\\n\\n-----\\n\\nMcIdas electroporated cells exhibited functional beating cilia (Fig. 5B and Movies EV5, 6).\\n\\nKymographs were generated (Fig. 5C) following previously described methods by Francis et al. (Francis\\n\\n& Lo, 2013) and ciliary beating frequency of McIdas induced ependymal cells was estimated to have a\\n\\nmean value of 10 Hz (Fig. 5D).\\n\\nOur findings show that McIdas induces ependymal cells restoration in the diseased hydrocephalic brain.\\n\\nMcIdas expression contributes to the restoration of the subventricular zone niche cytoarchitecture\\n\\nin hydrocephalic mice.\\n\\nThe adult subventricular zone niche has a well-defined architecture formed by pinwheel-like structures.\\n\\nThese structures are assembled by the large apical domain of ependymal cells which surround the\\n\\nsmaller apical domain of adult neural stem cells (aNSCs) and constitute a unique micro-environment for\\n\\ncoordinating adult neurogenesis (Mirzadeh et al, 2008; Paez-Gonzalez et al, 2011; Kokovay et al, 2012).\\n\\nTo assess whether McIdas is able to restore the subventricular zone niche’s cytoarchitecture in\\n\\nhydrocephalic mice, whole-mount immunofluorescence was performed on ventricular walls of the\\n\\nlateral ventricles of GemC1-knockout hydrocephalic mice at P7-P9, previously electroporated with an\\n\\nMcIdas-expressing plasmid or GFP alone. Specific antibodies against VCAM1 and GFAP, both markers\\n\\nof aNSCs, and β-catenin to delineate cell boundaries were used. Our analysis showed that McIdas\\nelectroporated cells had a larger apical surface compared to control GFP-electroporated cells, which is\\n\\ncharacteristic for multiciliated ependymal cells. In addition, McIdas-electroporated cells were\\n\\nsurrounding VCAM1- (Fig. 6A) and GFAP-immunopositive cells (Fig. 6B) reminiscent of pinwheel\\n\\nstructures. On the contrary, control GFP-electroporated cells had a small apical surface and pinwheel\\n\\nstructures were not detectable as it was previously described (Lalioti et al, 2019b). We also quantified\\n\\nthe number of electroporated cells that were detected in pinwheel structures, revealing that 18% of\\n\\n14\\n\\n\\n-----\\n\\nMcIdas-expressing cells were able to form pinwheel structures as opposed to GFP-electroporated cells\\n\\nthat were not detected in pinwheels (Fig. 6C). Our data suggest that McIdas expression is sufficient to\\n\\ninstruct the regeneration of ependymal cells in hydrocephalic mice which have the potential to re-build\\n\\nthe lost architecture of the subventricular zone niche.\\n\\nDISCUSSION\\n\\nDuring mammalian brain development neural progenitor cells transition from an uncommitted to a\\n\\nrestricted state, through a process that is influenced by lineage-specific gene expression programs,\\n\\nresulting to terminally differentiated cells. Reprogramming allows the conversion of differentiated cells\\n\\nfrom one lineage to another type of differentiated cell. Efficient reprogramming is highly dependent on\\n\\nthe selection of pioneer factors related to the desired cell type. Here, we demonstrate that GEMC1 and\\n\\nMCIDAS, members of the geminin family, constitute potent reprogramming factors towards the\\n\\nependymal fate both ex vivo and in vivo.\\n\\nGemC1 and McIdas drive the cell fate commitment of a subpopulation of brain radial glial cells towards\\n\\nmulticiliated ependymal cells by coordinating gene expression of essential factors for ependymal cell\\n\\ndifferentiation (Kyrousi et al, 2015). In addition, we have recently provided evidence that GemC1\\n\\nexpression is necessary and sufficient to establish the appropriate chromatin organization at multiple loci\\n\\nfor ependymal cells differentiation (Lalioti et al, 2019b). In light of this evidence, we hypothesized that\\n\\nGemC1 and McIdas could promote direct reprogramming towards the ependymal lineage in different\\n\\ncellular populations under physiology and disease. We initially demonstrated that both GemC1 and\\n\\nMcIdas overexpression were able to trigger the differentiation of mESCs into ependymal cells, as\\n\\nevidenced by the upregulation of key factors of ependymogenesis and the generation of multiple cilia in\\n\\n15\\n\\n\\n-----\\n\\nthe GemC1- and McIdas overexpressing cells. We also showed that GemC1 and McIdas overexpression\\n\\ninduce the expression of P73 and FoxJ1, two well-established transcription factors of the molecular\\n\\npathway of multiciliogenesis, in cultured cortical astrocytes. Importantly, we demonstrated that GemC1\\n\\nand McIdas activate the ependymal fate and induce the formation of multiple basal bodies and cilia in\\n\\nthe reprogrammed cells. The ectopic expression of GemC1 resulted in a lower percentage of cells with\\n\\nmultiple basal bodies, a unique structural feature of ependymal cells, revealing a differential\\n\\nreprogramming potential of the two factors. We showed that GemC1 does not induce sufficient levels of\\n\\nMcIdas expression, that could contribute to the reduced ability of GemC1 to execute as efficiently the\\n\\nependymal differentiation program. Moreover, using video microscopy experiments we showed that\\n\\nMcIdas ectopic expression was sufficient to reprogram cortical astrocytes into fully functional\\n\\nmulticiliated ependymal cells, as reprogrammed cells possessed multiple beating cilia that were able to\\n\\nbeat in a co-ordinated way and generate fluid flow. In addition, McIdas overexpression was sufficient to\\n\\ninduce the ependymal fate in radial glial cells derived from the brain of hydrocephalic mice in great\\n\\ncontrast to P73, further highlighting McIdas’ ability to induce the ependymal cell fate. The stem cell\\n\\nproperties of the mESCs and radial glial cells used in our study are likely to facilitate the adoption of the\\n\\nependymal fate. Since this aligns with their differentiation spectrum their conversion to ependyma could\\n\\nbe considered as differentiation into the ependymal cell lineage.\\n\\nIt has been shown that in human and experimental hydrocephalus, disrupted ependyma is replaced by\\n\\nastroglial cells (Sival et al, 2011; Roales-Buján et al, 2012; Guerra et al, 2015; McAllister, James P. et al\\n\\n, 2017). This astrocytic scarring along the denuded ventricular wall contributes to the pathophysiology\\n\\nof the disease as astrocytes do not restore ependymal functions; as they do not display cilia, produce\\n\\nappropriate trophic signals, or facilitate the production of neurons (Sival et al, 2011; Roales-Buján et al,\\n\\n2012). Therefore, our strategy revealing the successful conversion of isolated cortical astrocytes into\\n\\n16\\n\\n\\n-----\\n\\nependymal cells is of particular importance for treatment of hydrocephalus.\\n\\nTo investigate whether McIdas would also reprogram cells in vivo, we used a mouse model of congenital\\n\\nhydrocephalus in which GemC1 has been inactivated, and a model of acquired hydrocephalus, triggered\\n\\nby intracranial hemorrhage. It was previously shown that GemC1 deficiency in mice results in the loss\\n\\nof the ependymal cells due to failure on activating the molecular program for the cell fate commitment\\n\\ntowards the ependymal lineage (Terré et al, 2016; Lalioti et al, 2019b). Moreover, GemC1 deficiency\\n\\nhas been recently associated with congenital hydrocephalus in both humans and mice (Lalioti et al,\\n\\n2019b). In this study, we showed that McIdas forced expression induces the upregulation of P73 and\\n\\nFoxJ1 in brain periventricular cells, which are likely to be neural progenitor cells or/and reactive\\n\\nastrocytes that have replaced the ependymal cell population in the hydrocephalic brain. The activation of\\n\\nthe P73 and FoxJ1 genes uncovered the ability of McIdas to promote the initial steps of the ependymal\\n\\ncell’s differentiation program in the hydrocephalic brain. We also showed that McIdas-overexpressing\\n\\ncells generated multiple basal bodies and cilia, both of which are unique features of mature multiciliated\\n\\nependymal cells, thus revealing the successful reprogramming towards ependyma. Importantly, the\\n\\nmultiple cilia of the reprogrammed cells presented functional activity, as we were able to detect beating\\n\\ncilia upon in vivo overexpression of McIdas in the hydrocephalic mice. Using the LPA-induced\\n\\nintracranial hydrocephalus mouse model, we demonstrated McIdas ability to convert periventricular\\n\\ncells into ependymal cells. Previous study has shown that LPA causes substantial depletion of the\\n\\nependymal monolayer, loss of ependymal cilia and subsequent ependymal apoptotic cell death (Lummis\\n\\net al, 2019). In line with these findings, we did not observe multiciliated ependymal cells in\\n\\nperiventricular regions after LPA treatment. Instead, we observed GFAP-positive cells, likely reactive\\n\\nastrocytes, covering the lateral ventricle walls where ependymal cells typically reside under normal\\n\\nconditions, as it has been previously shown in hydrocephalic mice (Páez et al, 2007; Roales-Buján et al,\\n\\n17\\n\\n\\n-----\\n\\n2012). Therefore, it is highly likely that McIdas reprograms astrocytes into ependymal cells in LPA\\ntreated mice. However, we cannot rule out the possibility that McIdas might also enhance the survival of\\n\\na subset of ependymal cells resistant to LPA-induced apoptosis.\\u200b\\n\\nCollectively, in our experiments we employed diverse cellular systems and technical approaches,\\n\\nincluding plasmid and lentiviral vectors, to induce the overexpression of McIdas and GemC1 suggesting\\n\\nthat both GemC1 and McIdas can induce the programming or reprogramming of different cellular\\n\\npopulations irrespectively of the cell system and vector used. While our approach in the genetic mouse\\n\\nmodel of hydrocephalus and the LPA-induced intracranial hydrocephalus likely targets periventricular\\n\\ncells, which are neural progenitor cells or/and reactive astrocytes, we cannot rule out the possibility that\\n\\nother cellular populations might also be targeted. Recent evidence from reprogramming experiments in\\n\\nthe mouse retina and the brain indicates that the insert sequences of overexpression constructs can alter\\n\\nthe specificity of the construct's promoter, introducing a bias toward the initial cell population being\\n\\ntargeted (Wang et al, 2021; Le et al, 2022). Additionally, future genetic lineage analysis using transgenic\\n\\nmice to label neural progenitors and/or astrocytes would be needed to clarify which cell population is\\n\\ndifferentiated into ependymal cells in our approach. One of the major clinical features in human patients\\n\\nwith hydrocephalus is the ventricular zone disruption phenomenon, characterized by the loss of neural\\n\\nstem cells and ependyma (McAllister, James P. et al, 2017). Of note, sustained neuron generation\\n\\ndepends on the maintenance of the proper cytoarchitecture of the Ventricular-Subventricular Zone (V\\nSVZ) niche of the brain, which requires preserving the structure of the ependymal cell monolayer intact\\n\\n(Paez-Gonzalez et al, 2011). In the present study, we targeted V-SVZ-resident cells in the hydrocephalic\\n\\nbrain in which McIdas overexpression was induced and observed that McIdas-overexpressing cells\\n\\npopulated areas of the ventricular zone niche next to the lateral ventricles, where ependymal cells\\n\\nnormally reside under homeostasis. We have used SVZ electroporation to target the periventricular cells\\n\\n18\\n\\n\\n-----\\n\\nof one lateral ventricle of the hydrocephalic brain of GemC1-knockout mice, which results to a limited\\n\\nnumber of electroporated cells. Therefore, following this experimental approach, the small number of\\n\\nependymal cells which is generated is not possible to contribute to a significant reduction in the volume\\n\\nof the lateral ventricles as a readout of resolved or reduced hydrocephalus. It has been described that\\n\\ndisruption of the ventricular/subventricular zone (V/SVZ) is a critical and consistent factor in the\\n\\npathophysiology of hydrocephalus (McAllister, James P. et al, 2017; Jiménez et al, 2014). In our study\\n\\nwe showed that McIdas ectopic expression contributes to the restoration of the lost cytoarchitecture of\\n\\nthe niche in hydrocephalic mice, as we identified reprogrammed ependymal cells forming pinwheel-like\\n\\nstructures with resident neural stem cells in the targeted ventricular wall. Our findings suggest that\\n\\nMcIdas forced expression in human patients with hydrocephalus would contribute to the re\\nestablishment of the SVZ niche leading to brain functions’ improvement by enhancing neuronal\\n\\nproduction in human patients with hydrocephalus and ameliorating brain function. The in vivo\\n\\nreprogramming approach indicates that the in-situ conversion of cells into ependyma in its native\\n\\nmicroenvironment is feasible. Our in vivo approach allowed the exploration of the influence of the\\n\\ndiseased environment of the hydrocephalic brain on the reprogramming process. Our data support the\\n\\nidea that functional maturation and integration of the reprogrammed cells in the native tissue can be\\n\\nachieved through our in vivo reprogramming method. Since the molecular program of multiciliogenesis\\n\\nis conserved across different organisms and tissues (Arbi et al, 2018), it is tempting to speculate that\\n\\nsuch a reprogramming approach could be applied to other diseases that impact ciliated tissues, such as\\n\\nthe lung and the reproductive system. For instance, a rare genetic disorder affecting the mucociliary\\n\\nclearance mechanism, known as Reduced Generation of Multiple Motile Cilia (RGMC), has been linked\\n\\nto mutations in MCIDAS. Therefore, it would be highly interesting to investigate whether these\\n\\nconditions could be addressed in the future through a genetic treatment implementing McIdas’\\n\\n19\\n\\n\\n-----\\n\\noverexpression in the affected tissue.\\n\\nTogether, our analysis proposes MCIDAS and GEMC1 as potent reprogramming factors towards the\\n\\nependymal cell lineage and demonstrate a proof of concept that restoring ependymal cells could\\n\\ncontribute in managing hydrocephalus. Although the present study yielded intriguing findings, our\\n\\nimplementation holds certain limitations. In particular, our in vivo studies have been conducted to mouse\\n\\nmodels of hydrocephalus. Despite sharing a similar pathophysiology with the human disease, the\\n\\nanatomical and physiological characteristics of the mouse and human brain possess certain differences.\\n\\nTherefore, to validate the effectiveness of our approach, additional experiments in disease models closer\\n\\nto human pathophysiology will be required. The pig model demonstrates anatomical and physiological\\n\\nsimilarities to humans and replicates the pathophysiology of human hydrocephalus, exhibiting features\\n\\nsuch as ventriculomegaly, V/SVZ disruption, and neuroinflammation and could thus be a valuable model\\n\\nfor examining the translational success of our approach to the human conditions (Garcia-Bonilla et al,\\n\\n2023). Additionally, since hydrocephalus in our study appeared during early developmental stages, our\\n\\nfindings are particularly relevant to postnatal hydrocephalus. Therefore, their applicability to adult\\n\\nhydrocephalus will require further investigation using appropriate adult animal models. Although we\\n\\nachieved McIdas overexpression in the periventricular cells of the diseased brain through\\n\\nelectroporation, such an approach cannot be used in humans. Therefore, alternative gene delivery\\n\\nmethods compatible with therapeutic protocols, such as AAV gene delivery strategies in the human brain\\n\\n(Chamakioti et al, 2022; Au et al, 2022) should be established. Our study shows that regeneration of\\n\\nmulticiliated ependymal cells can be succeeded in hydrocephalic mouse models while further\\n\\nexperiments are needed to show the applicability of our findings in treating hydrocephalus in humans.\\n\\n20\\n\\n\\n-----\\n\\nMETHODS\\n\\nMouse knockout strain\\n\\nC57/Bl6 mice carrying a knockout (KO) allele of the GemC1 gene were initially generated by the trans\\nNIH Knock-Out Mouse Project (KOMP) and obtained from the KOMP repository. The GemC1-KO\\n\\nallele was maintained on a mixed strain background. Inactivation of the GemC1 gene was carried out\\n\\nusing the “knockout-first” strategy, as previously described (Arbi et al, 2016; Lalioti et al, 2019b). This\\n\\napproach involves disrupting gene function by adding RNA processing signals without removing any of\\n\\nthe targeted genes (Testa et al, 2004). The GemC1-KO allele was created by placing a lacZ-neo cassette\\n\\nbetween the second and third exon of the gene.\\n\\nMice were housed in the animal house of the University of Patras, where a standard light cycle of 12\\n\\nhours of light and 12 hours of darkness was maintained. Continuous access to food and water was\\n\\nensured at all times. To ensure animal welfare handling and environmental stressors like noise,\\n\\nvibrations, and strong odors were minimized to necessary levels. All animal-related procedures were\\n\\nconducted in strict accordance with EU directives and approved by the Veterinary Administrations of the\\n\\nPrefecture of Achaia, Greece, and the Research Ethics Committee of the University of Patras, Greece.\\n\\nIn all experiments, mice were used without regard to their gender. Experiments involving animal usage\\n\\nwere conducted during the neonatal stage when the gender of the animals was not yet distinguishable.\\n\\nLPA preparation and injections\\n\\nTo generate an inducible mouse model of intracranial-hemorrhage hydrocephalus, we performed\\n\\nintraventricular delivery of lysophosphatidic acid (LPA) in wild-type (WT) mice at postnatal day five\\n\\n(P5), following previously published works (Yung et al, 2011; Lummis et al, 2019).\\n\\nPowdered 18:1 LPA (857130P, Avanti Polar Lipids) was initially dissolved in methanol and subsequently\\n\\nvacuum-dried. LPA was then reconstituted in Hanks’ balanced salt solution (HBSS 14065-049, Gibco)\\n\\nby sonicating in a water bath for 5 min to generate a 10 mM stock, as previously described (Yung et al,\\n\\n2011; Lummis et al, 2019). Stock solution was stored at −20°C until use. Using a glass microcapillary, 3\\n\\nμl of the 10 mM LPA stock solution together with Fast Green (0.1%, Sigma) were injected into the\\n\\nlateral ventricle of P5 mice upon hypothermia-anesthetization. Mice were sacrificed two days post LPA\\n\\ninjections at P7 and were examined or further processed as described in the respective experiments.\\n\\n21\\n\\n\\n-----\\n\\nPlasmid constructs\\n\\nFor neonatal electroporation and mESCs transfection experiments mouse cDNAs for McIdas and\\n\\nGemC1 were cloned in pCAGGS-IRES-GFP vectors between SacI/EcoRV and NheI/SmaI restriction\\n\\nsites respectively, as described previously (Kyrousi et al, 2015).\\n\\nNeonatal electroporation\\n\\nNeonatal electroporation was performed at P0-P1. Anesthesia was induced via hypothermia. Using a\\n\\npulled glass microcapillary, 2 μg plasmid DNA together with Fast Green (0.1%, Sigma) were injected in\\n\\nthe lateral ventricle of the mouse brains. Mice were subsequently electroporated with 5 pulses of 100V\\n\\nfor 50ms each followed by intervals of 950ms. An ECM830 Electroporator (Harvard Apparatus) and 7\\n\\nmm platinum electrodes (BTX) were used. Following electroporation mice were placed back to cages\\n\\nand left to grow for 4 or 6-14 days before sacrificed as described in the respective experiments.\\n\\nEx vivo electroporation and organotypic brain slice cultures\\n\\nFor ex vivo electroporation, mice were sacrificed 2 days post LPA-injections at P7. Mouse brains were\\n\\ndissected and placed in cold DMEM/F-12 medium (Gibco). Subsequently, 2 μg plasmid DNA of a\\n\\npCAGGS-IRES-GFP (GFP) or pCAGGS-McIdas-IRES-GFP (McIdas) vector together with Fast Green\\n\\n(0.1%, Sigma) were injected in the lateral ventricle using a pulled glass microcapillary. Brains were then\\n\\nelectroporated with 5 pulses applied at 100V for 50ms each at intervals of 950ms. An ECM830\\n\\nElectroporator (Harvard Apparatus) and 7 mm platinum electrodes (BTX) were used.\\n\\nFor organotypic brain slice cultures 250 μm thick coronal slices were obtained from the electroporated\\n\\nbrains using a vibrating microtome (Leica VT1000 S). Consecutive slices containing the lateral\\n\\nventricles were collected and placed in culturing medium [2% FBS (Gibco), 1% GlutaMAX Supplement\\n\\n(Gibco) and 1% Penicillin-Streptomycin (Gibco) in DMEM/F-12]. Subsequently, brain slices were\\n\\nplaced onto 0.4 μm membrane inserts (Millipore) in 6-well plates and cultured for 5 or 9 days in\\n\\nculturing medium, performing medium change every three days.\\n\\nLentiviral production\\n\\nA second generation packaging system was used to generate lentiviral expression particles as previously\\n\\ndescribed (Arbi et al, 2016). The pLV-Dest-CAG lentiviral expression vector was used and was kindly\\n\\n22\\n\\n\\n-----\\n\\nprovided by Dr. M. Gotz, Helmholtz Center, Munich. The cloning strategy for generating the lentiviral\\n\\nexpression vectors for GFP and GemC1 with an N-terminal GFP tag (GFP-GemC1) was described\\n\\nelsewhere (Arbi et al, 2016).\\n\\nMcIdas with an N-terminal GFP tag (GFP-McIdas) was initially cloned into the KpnI/XhoI restriction\\n\\nsites of the pENTR1AminusCmR vector. p73 with a C-terminal GFP tag (p73-GFP) was initially cloned\\n\\ninto the SalI/NotI restriction sites of the pENTR1AminusCmR vector. An LR recombination reaction,\\n\\nusing the Gateway LR Clonase II enzyme mix (Invitrogen), was performed between the attL-containing\\n\\nentry clone and the attR-containing destination pLVDest-CAG vector.\\n\\nPrimary cultures of postnatal cortical astroglia and lentiviral infection\\n\\nTo establish primary cultures of postnatal cortical astroglia, the cerebral cortex of postnatal P5-P7 wild\\ntype mice was dissected from the brain. Cells were subsequently mechanically dissociated and cultured\\n\\nunder proliferating conditions for 7 days in astro medium containing DMEM/F-12 (Gibco), 10% FBS\\n\\n(Gibco), 2% B27 Supplement (Gibco), 1% GlutaMAX Supplement (Gibco), 1% Penicillin-Streptomycin\\n\\n(Gibco), epidermal growth factor (10ng/ml) and basic fibroblast growth factor (10ng/ml), as previously\\n\\ndescribed (Heinrich et al, 2011).\\n\\nFor lentiviral infection, astroglial cells were plated onto poly-D-lysine pre-coated coverslips 7 days upon\\n\\nproliferation initiation and were left to attach and grow for 2-4 hours. Cells were then transduced with\\n\\nconcentrated lentiviral vectors. The infection mix was removed 24 hours later and was replaced by astro\\n\\nmedium which did not contain FBS, epidermal growth factor and basic fibroblast growth factor to favor\\n\\ndifferentiation. Cells were analyzed 7 and 14 days later.\\n\\nPrimary cultures of postnatal radial glial cells and lentiviral infection\\n\\nTo establish primary cultures of postnatal radial glial cells (pRGCs), the lateral walls of the lateral\\n\\nventricles of P0-P1 mice were dissected, mechanically dissociated, and then plated onto poly-D-lysine\\n\\npre-coated coverslips. pRGCs were cultured in proliferation medium [DMEM-high glucose (Gibco),\\n\\n10% FBS (Gibco), and 1% penicillin/streptomycin (Gibco)] for three days (Paez-Gonzalez et al, 2011;\\n\\nKyrousi et al, 2015; Lalioti et al, 2019b). Subsequently, pRGCs were infected with unconcentrated viral\\n\\nparticles performing plate-spin infection. Proliferation medium was changed 24 hours post-infection and\\n\\nlater replaced with differentiation medium [DMEM-high glucose (Gibco), 2% FBS (Gibco), and 1%\\n\\n23\\n\\n\\n-----\\n\\npenicillin/streptomycin (Gibco)] 48 hours post-infection. Cells were processed for analysis 5 or 15 days\\n\\nupon differentiation initiation.\\n\\nCultures of mouse embryonic stem cells (mESCs) and transfection\\n\\nMouse embryonic stem cells were grown onto gelatin-coated (0.1% gelatin, Sigma) flasks in GMEM\\n\\nBHK-21 medium (Gibco), supplemented with 10% FBS (Biosera, Embryonic Stem Cells tested), 0.1\\n\\nmM β-mercaptoethanol (Gibco), 2 mM L-glutamine (Sigma), 1 mM sodium pyruvate (Pan Biotech), 1%\\n\\npenicillin/streptomycin (Gibco), 1% MEM Non-Essential Amino Acids (Biosera) and 2.000 U/ml LIF\\n\\n(Millipore). Cells were passaged every other day and were mycoplasma tested.\\n\\nReverse mESCs transfection was performed using the jet-OPTIMUS reagent according to\\n\\nmanufacturer’s instructions. Briefly, the transfection mixes containing DNA, jet-OPTIMUS buffer and\\n\\ntransfection reagent were prepared. Afterwards, transfection mixes were placed on gelatin pre-coated\\n\\nglass coverslips and then mESCs were immediately seeded. Cells were cultured in the presence of LIF\\n\\nfor 24 hours. The following day the medium was changed (LIF withdrawal), and the cells were cultured\\n\\nin differentiating conditions for up to 6 days and were then processed for analysis.\\n\\nVideo microscopy of ependymal cilia\\n\\nFor the study of ependymal cilia in primary cultures of reprogrammed postnatal cortical astroglia, cells\\n\\nwere grown under proliferating conditions in astro medium for 7 days. Cells were then plated onto poly\\nD-lysine coated glass-bottom dishes (ibidi) and transduced with unconcentrated lentiviral vectors. 24\\n\\nhours post infection, medium was replaced by astro medium which did not contain FBS, epidermal\\n\\ngrowth factor and basic fibroblast growth factor to favor differentiation. Cells were used in video\\n\\nmicroscopy experiments 22 days later.\\n\\nTo perform video microscopy experiments in reprogrammed postnatal cortical astroglia, the culturing\\n\\nmedium was replaced by live imaging medium containing MEM (Sigma), 20% FBS (Gibco), 4 mM L\\nglutamine (Sigma) and 1 mM sodium pyruvate (Pan Biotech). In addition, to track cilia generated fluid\\n\\nflow 0.5 μm red fluorescent microbeads (1μl/ml medium; Sigma) were added to the imaging medium.\\n\\nVideo microscopy experiments were performed using an Olympus IX83 microscope equipped with a\\n\\nHamamatsu Orca-Flash 4.0 sCMOS camera. During live imaging microscopy experiments cells were\\n\\nmaintained at 37°C and 5% CO2. Videos were obtained using a 60x water immersion objective at 80\\n\\n24\\n\\n\\n-----\\n\\nframes/sec, while 500 frames were acquired for each field.\\n\\nFor the study of brain ependymal cilia, mice were sacrificed at P7-P15, brains were dissected in HBSS\\n\\nat room temperature and 200 μm thick coronal slices were obtained using a vibrating microtome. Slices\\n\\ncontaining the lateral ventricles were placed in DMEM-GlutaMAX (Gibco) media pre-warmed at 37°C.\\n\\nTo perform video microscopy experiments, brain slices were placed onto glass-bottom dishes containing\\n\\nlive imaging medium with 0.5 μm red fluorescent microbeads (1μl/ml medium). Video recordings were\\n\\nobtained as described in the preceding paragraph.\\n\\nFor cilia beat frequency analysis, the ImageJ image processing program was used to firstly generate a\\n\\ngraphical representation of cilia beating, in the form of a kymograph, as previously described by\\n\\nDrummond I. (Drummond, 2009). Cilia beat frequency for each multiciliated cell was then calculated\\n\\nwith the correlation of the time scale of the video recording with the kymograph, following the method\\n\\ndescribed by Francis R. and Lo C. (Francis & Lo, 2013).\\n\\nTo track the trajectory of the moving microbeads the MTrackJ Plugin of the ImageJ program was used.\\n\\nIn short, the point selection tool was used to manually mark the position of each red fluorescent\\n\\nmicrobead in each frame. Consecutive marks on the corresponding video recordings created the\\n\\ntrajectory of the microbeads.\\n\\nWhole-mount dissection\\n\\nSVZ whole mounts were obtained as previously described (Lalioti et al, 2019b). Briefly, the brain was\\n\\nisolated and the electroporated lateral ventricle was carefully separated from the posterior region of the\\n\\ntelencephalon, followed by the extraction of the hippocampus and septum. Fixation of the dissected\\n\\nlateral wall was performed in 4% PFA/0.1% Triton X-100 at 4[o]C overnight. Staining was then performed\\n\\n(see below, immunofluorescence) and the lateral wall was further dissected from the surrounding brain\\n\\nparenchyma, resulting in tissue slivers with a thickness of 200-300 micrometers. These slivers were then\\n\\naffixed to a glass slide using Mowiol 4-88 (Calbiochem) mounting media and covered with a coverslip.\\n\\nImmunofluorescence\\n\\nFor immunofluorescence, cultured cells were fixed for 10 min with 4% PFA (or 10 min PFA 4%\\n\\n25\\n\\n\\n-----\\n\\nfollowed by 10 min fixation in Methanol where stated) and incubated in blocking solution, for 1hr at\\n\\nroom temperature. Postnatal cortical astroglial cells were incubated in blocking solution containing 5%\\n\\nFBS, 3% BSA, 0.1% Triton X-100 in 1× PBS. pRGCs and mESCs were incubated in blocking solution\\n\\ncontaining 5% Normal Goat Serum (Jackson Immunoresearch) and 0.3% Triton-X100 in 1x PBS. Cells\\n\\nwere incubated with primary antibodies in blocking solution at 4°C, overnight (Kalogeropoulou et al,\\n\\n2022).\\n\\nBrains dissected from newborn mice (up to P7) were fixed overnight with 4% PFA. Subsequently,\\n\\nbrains were rinsed with 1x PBS and cryopreserved with incubation in 30% sucrose in 1x PBS for ~48\\n\\nhours. Brains were then frozen in 7.5% gelatin supplemented with 15% sucrose and sectioned at 12 μm\\n\\nusing a CM1850 Leica cryostat. Brain coronal cryosections were postfixed with 4% PFA for 10 min,\\n\\ntreated with 0.3% Triton X-100 in 1× PBS for 5 min, and incubated in blocking solution containing 5%\\n\\nFBS, 3% BSA, 0.1% Triton X-100 in 1× PBS, for 1-3 hrs. Samples were incubated with primary\\n\\nantibodies in blocking solution at 4°C, overnight.\\n\\nFor immunofluorescence of cultured brain slices, membrane inserts were removed, rinsed in 1x PBS and\\n\\nfixed with 4% PFA at 4°C, overnight. Membrane inserts were then subjected to consecutive washes in\\n\\n1x PBS (5 min), 0.1% Triton X-100 in 1× PBS (5 min) and 0.5% Triton X-100 in 1× PBS (20 min once)\\n\\nand were incubated in blocking solution containing 1% FBS, 0.1% Triton X-100 in 1× PBS for 2-3 hrs\\n\\nat room temperature. Samples were incubated with primary antibodies in blocking solution at 4°C,\\n\\novernight.\\n\\nImmunofluorescence of SVZ whole-mounts was performed as previously described (Mirzadeh et al,\\n\\n2008, 2010). Briefly, whole-mounts were incubated for 1-2 hrs at room temperature in blocking solution,\\n\\ncontaining 10% normal goat serum in 0.1 M PBS with 0.5-2% Triton-X100. Samples were incubated\\n\\nwith primary antibodies in blocking solution at 4[o]C for 24-48 hrs.\\n\\nThe following primary antibodies were used: chicken anti-GFP (1020, Aves Laboratories, 1:500 for\\n\\nbrain sections; 1:1000 for cells, 1:250 for whole mounts), mouse anti-S100β (S2532, Sigma, 1:250),\\n\\nrabbit anti-P73 (ab40658, Abcam, 1:300), mouse anti-FoxJ1 (14-9965, eBioscience, 1:500), rabbit anti\\npericentrin (PRB432C, Covance, 1:750 for brain sections; 1:1500 for cells), rabbit anti-GFAP (Z0334,\\n\\nDakoCytomation, 1:1000 for cells, 1:500 for whole-mounts), mouse anti-acetylated α-tubulin (T6793,\\n\\nSigma, 1:1000 for brain sections; 1:1500 for cells), rabbit anti-McIdas ((Pefani et al, 2011), 1:250),\\n\\n26\\n\\n\\n-----\\n\\nmouse anti-centriolin (sc-365521, Santa Cruz, 1:100, cells’ fixation in PFA-Methanol), rabbit anti\\nMeig1 (orb185566, Biorbyt, 1:100), rat anti-VCAM1 (550547, BD Biosciences, 1:100), mouse anti β\\ncatenin (610153, BD Biosciences, 1:500), guinea pig anti-double-cortin (AB2253, Millipore, 1:500).\\n\\nThe following Alexa Fluor-labeled secondary antibodies (Invitrogen) were used in the respective\\n\\nblocking solution (1:1000) for 1 hr (cryosections and cells) at room temperature or overnight (membrane\\n\\ninserts) at 4°C: Alexa Fluor 488 goat anti-chicken, Alexa Fluor 568 goat anti-mouse IgG1, Alexa Fluor\\n\\n647 donkey anti-rabbit, Alexa Fluor 568 goat anti-rabbit, Alexa Fluor 647 goat anti-mouse IgG1, Alexa\\n\\nFluor 568 goat anti-mouse, Alexa Fluor 568 goat anti-rabbit, Alexa Fluor 647 donkey anti-mouse, Alexa\\n\\nFluor 568 goat anti-guinea pig and Alexa Fluor 568 goat anti-rat. DNA was stained either with Draq-5\\n\\n(1:1000, Biostatus) or Dapi (1:1500, Sigma). Samples were mounted in Mowiol 4-88 (Calbiochem).\\n\\nMicroscopy and data analysis\\n\\nImages of fluorescent samples were acquired with a confocal Leica TCS SP5 and a Leica TCS SP8\\n\\nsystem equipped with a fluorescent Leica DMI600B or Leica DMi8 microscope, respectively. 20x, 40x\\n\\nand 63x lenses were used. Digital images were processed with Adobe Photoshop and the ImageJ image\\n\\nprocessing program. For still images related to video microscopy experiments, individual frames were\\n\\ncropped and adjusted for brightness and contrast in ImageJ.\\n\\nQuantification of cells on overexpressed brains from in vivo experiments was performed on\\n\\ncryosections, vibratome sections or whole-mounts. In all instances, each brain region containing\\n\\nelectroporated cells was imaged and quantified per frame. All quantifications are expressed as\\n\\npercentages of the total cell count. Quantification of radial glial cells, cortical astrocytes, or mESCs from\\n\\nex-vivo experiments was performed in at least two independent experiments, with a minimum of 15\\n\\ndifferent images analyzed for each condition in each experiment. Fluorescent intensity per nucleus was\\n\\nmeasured using the open-source platform ImageJ.\\n\\nThe number of independent experiments performed for each analysis is mentioned in the corresponding\\n\\nfigure legend. Normality was tested for each experiment using the Shapiro-Wilk test to determine if the\\n\\nsample data came from a normally distributed population. Since our data did not follow a normal\\n\\ndistribution, statistical analysis was performed with the non-parametric two-tailed Mann–Whitney test.\\n\\nSignificant differences in medians were determined by p value: P<0.05 (*P<0.05, **P<0.01,\\n\\n***P<0.001). Each graph displayed the median ± interquartile range (IQR). All statistical analyses and\\n\\n27\\n\\n\\n-----\\n\\ngraph preparation were performed in GraphPad Prism 6.\\n\\nThe study did not incorporate blinding or randomization, and all samples were included without\\n\\napplying specific inclusion or exclusion criteria. These choices were guided by the exploratory nature of\\n\\nthe research.DATA AVAILABILITY\\n\\nThis study includes no data deposited in external repositories.\\n\\nACKNOWLEDGMENTS\\n\\nWe thank the Advanced Light Microscopy Facility of the Medical School at the University of Patras and\\n\\nthe Experimental Animal Facility of the University of Patras for support with experiments. We are also\\n\\ngrateful to Andriana Charalampopoulou and members from Professor’s Benedikt Berninger laboratory\\n\\nfor their assistance in astrocytic reprogramming experiments. We also thank Dr. Christina Kyrousi and\\n\\nmembers of our laboratories for helpful discussions and Nikolaos Giakoumakis for his assistance with\\n\\nwhole-mount images processing. This study was supported by the Hellenic Foundation for Research and\\n\\nInnovation (H.F.R.I.) under the “2nd Call for H.F.R.I. Research Projects to support Faculty Members &\\n\\nResearchers” (Project Number: 2735), the Hydrocephalus Association USA and Fondation Santé to S.T.,\\n\\nand the State Scholarships Foundation (IKY) to K.K.\\n\\nDISCLOSURE AND COMPETING INTERESTS STATEMENT\\n\\nZoi Lygerou is an EMBO Council member. This has no bearing on the editorial consideration of this\\n\\narticle for publication.\\n\\nTHE PAPER EXPLAINED:\\n\\n28\\n\\n\\n-----\\n\\nMedical issue\\n\\nHydrocephalus is a condition where fluid builds up in the brain cavities, leading to serious health issues.\\nStandard treatment involves surgery, which is risky and often fails over time. Ependymal cells have\\nmultiple tiny hair-like structures (cilia) on their surface to circulate the fluid, and their impairment may\\nlead to hydrocephalus.\\n\\nResults\\n\\nWe showed that GemC1 and McIdas, which are known regulators of multiciliated ependymal cell fate\\ndetermination, can instruct different cell types to differentiate into ependymal cells. We further managed\\nto convert brain cells of hydrocephalic animal models into ependymal cells.\\n\\nClinical impact\\n\\nOur findings suggest new therapeutic intervention aiming at the regeneration of damaged ependymal\\ncells in human hydrocephalus. .\\n\\nREFERENCES\\n\\nAbdi K, Lai C-H, Paez-Gonzalez P, Lay M, Pyun J & Kuo CT (2018) Uncovering inherent cellular\\n\\nplasticity of multiciliated ependyma leading to ventricular wall transformation and\\nhydrocephalus. Nat Commun 9: 1655\\n\\nArbi M, Pefani D, Kyrousi C, Lalioti M, Kalogeropoulou A, Papanastasiou AD, Taraviras S & Lygerou\\n\\nZ (2016) GemC1 controls multiciliogenesis in the airway epithelium. EMBO Rep 17: 400–413\\n\\nArbi M, Pefani D-E, Taraviras S & Lygerou Z (2018) Controlling centriole numbers: Geminin family\\n\\nmembers as master regulators of centriole amplification and multiciliogenesis. Chromosoma 127:\\n151–174\\n\\nAu HKE, Isalan M & Mielcarek M (2022) Gene Therapy Advances: A Meta-Analysis of AAV Usage in\\n\\nClinical Settings. Frontiers in Medicine 8\\n\\nBocchi R, Masserdotti G & Götz M (2022) Direct neuronal reprogramming: Fast forward from new\\n\\nconcepts toward therapeutic approaches. Neuron 110: 366–393\\n\\nBoon M, Wallmeier J, Ma L, Loges NT, Jaspers M, Olbrich H, Dougherty GW, Raidt J, Werner C,\\n\\nAmirav I, et al (2014) MCIDAS mutations result in a mucociliary clearance disorder with\\nreduced generation of multiple motile cilia. Nat Commun 5: 4418\\n\\nChamakioti M, Karantzelis N & Taraviras S (2022) Advanced Gene-Targeting Therapies for Motor\\n\\nNeuron Diseases and Muscular Dystrophies. International Journal of Molecular Sciences 23:\\n4824\\n\\n29\\n\\n\\n-----\\n\\nDrummond I (2009) Studying cilia in zebrafish. Methods Cell Biol 93: 197–217\\n\\nFrancis R & Lo C (2013) Ex vivo method for high resolution imaging of cilia motility in rodent airway\\n\\nepithelia. J Vis Exp\\n\\nFunk MC, Bera AN, Menchen T, Kuales G, Thriene K, Lienkamp SS, Dengjel J, Omran H, Frank M &\\n\\nArnold SJ (2015) Cyclin O (Ccno) functions during deuterosome-mediated centriole\\namplification of multiciliated cells. EMBO J 34: 1078–1089\\n\\nGarcia-Bonilla M, Nair A, Moore J, Castaneyra-Ruiz L, Zwick SH, Dilger RN, Fleming SA, Golden\\n\\nRK, Talcott MR, Isaacs AM, et al (2023) Impaired neurogenesis with reactive astrocytosis in the\\nhippocampus in a porcine model of acquired hydrocephalus. Experimental Neurology 363:\\n114354\\n\\nGonzalez-Cano L, Fuertes-Alvarez S, Robledinos-Anton N, Bizy A, Villena-Cortes A, Fariñas I,\\n\\nMarques MM & Marin MC (2016) p73 is required for ependymal cell maturation and neurogenic\\nSVZ cytoarchitecture: p73 Regulates SVZ Cytoarchitecture. Developmental Neurobiology 76:\\n730–747\\n\\nGuerra MM, Henzi R, Ortloff A, Lichtin N, Vío K, Jiménez AJ, Dominguez-Pinos MD, González C,\\n\\nJara MC, Hinostroza F, et al (2015) Cell Junction Pathology of Neural Stem Cells Is Associated\\nWith Ventricular Zone Disruption, Hydrocephalus, and Abnormal Neurogenesis. J Neuropathol\\nExp Neurol 74: 653–671\\n\\nHeinrich C, Gascón S, Masserdotti G, Lepier A, Sanchez R, Simon-Ebert T, Schroeder T, Götz M &\\n\\nBerninger B (2011) Generation of subtype-specific neurons from postnatal astroglia of the mouse\\ncerebral cortex. Nat Protoc 6: 214–228\\n\\nJi W, Tang Z, Chen Y, Wang C, Tan C, Liao J, Tong L & Xiao G (2022) Ependymal Cilia: Physiology\\n\\nand Role in Hydrocephalus. Frontiers in Molecular Neuroscience 15\\n\\nJiménez AJ, Domínguez-Pinos M-D, Guerra MM, Fernández-Llebrez P & Pérez-Fígares J-M (2014)\\n\\nStructure and function of the ependymal barrier and diseases associated with ependyma\\ndisruption. Tissue Barriers 2: e28426\\n\\nKahle KT, Kulkarni AV, Limbrick DD & Warf BC (2016) Hydrocephalus in children. The Lancet 387:\\n\\n788–799\\n\\nKalogeropoulou A, Mougkogianni M, Iliadou M, Nikolopoulou E, Flordelis S, Kanellou A, Arbi M,\\n\\nNikou S, Nieminuszczy J, Niedzwiedz W, et al (2022) Intrinsic neural stem cell properties define\\nbrain hypersensitivity to genotoxic stress. Stem Cell Reports 17: 1395–1410\\n\\nKokovay E, Wang Y, Kusek G, Wurster R, Lederman P, Lowry N, Shen Q & Temple S (2012) VCAM1\\n\\nis essential to maintain the structure of the SVZ niche and acts as an environmental sensor to\\nregulate SVZ lineage progression. Cell Stem Cell 11: 220–230\\n\\nKousi M & Katsanis N (2016) The Genetic Basis of Hydrocephalus. Annu Rev Neurosci 39: 409–435\\n\\n30\\n\\n\\n-----\\n\\nKyrousi C, Arbi M, Pilz G-A, Pefani D-E, Lalioti M-E, Ninkovic J, Götz M, Lygerou Z & Taraviras S\\n\\n(2015) Mcidas and GemC1 are key regulators for the generation of multiciliated ependymal cells\\nin the adult neurogenic niche. Development 142: 3661–3674\\n\\nKyrousi C, Lygerou Z & Taraviras S (2017) How a radial glial cell decides to become a multiciliated\\n\\nependymal cell. Glia 65: 1032–1042\\n\\nLalioti M-E, Arbi M, Loukas I, Kaplani K, Kalogeropoulou A, Lokka G, Kyrousi C, Mizi A,\\n\\nGeorgomanolis T, Josipovic N, et al (2019a) GemC1 governs multiciliogenesis through direct\\ninteraction with and transcriptional regulation of p73. J Cell Sci 132\\n\\nLalioti M-E, Kaplani K, Lokka G, Georgomanolis T, Kyrousi C, Dong W, Dunbar A, Parlapani E,\\n\\nDamianidou E, Spassky N, et al (2019b) GemC1 is a critical switch for neural stem cell\\ngeneration in the postnatal brain. Glia\\n\\nLe N, Appel H, Pannullo N, Hoang T & Blackshaw S (2022) Ectopic insert-dependent neuronal\\n\\nexpression of GFAP promoter-driven AAV constructs in adult mouse retina. Front Cell Dev Biol\\n10\\n\\nLim DA, Tramontin AD, Trevejo JM, Herrera DG, García-Verdugo JM & Alvarez-Buylla A (2000)\\n\\nNoggin Antagonizes BMP Signaling to Create a Niche for Adult Neurogenesis. Neuron 28: 713–\\n726\\n\\nLummis NC, Sánchez-Pavón P, Kennedy G, Frantz AJ, Kihara Y, Blaho VA & Chun J (2019) LPA1/3\\n\\noveractivation induces neonatal posthemorrhagic hydrocephalus through ependymal loss and\\nciliary dysfunction. Sci Adv 5: eaax2011\\n\\nMa L, Quigley I, Omran H & Kintner C (2014) Multicilin drives centriole biogenesis via E2f proteins.\\n\\nGenes Dev 28: 1461–1471\\n\\nMacDonald A, Lu B, Caron M, Caporicci-Dinucci N, Hatrock D, Petrecca K, Bourque G & Stratton JA\\n\\n(2021) Single Cell Transcriptomics of Ependymal Cells Across Age, Region and Species Reveals\\nCilia-Related and Metal Ion Regulatory Roles as Major Conserved Ependymal Cell Functions.\\nFront Cell Neurosci 15\\n\\nMarshall CB, Mays DJ, Beeler JS, Rosenbluth JM, Boyd KL, Santos Guasch GL, Shaver TM, Tang LJ,\\n\\nLiu Q, Shyr Y, et al (2016) p73 Is Required for Multiciliogenesis and Regulates the Foxj1Associated Gene Network. Cell Reports 14: 2289–2300\\n\\nMcAllister, James P., Guerra, Maria Montserrat, Ruiz, Leandro Castaneyra, Jimenez, Antonio J.,\\n\\nDominguez-Pinos, Dolores, Sival, Deborah, den Dunnen, Wilfred, Morales, Diego M., Schmidt,\\nRobert E., Rodriguez, Esteban M., et al (2017) Ventricular Zone Disruption in Human Neonates\\nWith Intraventricular Hemorrhage. Journal of neuropathology and experimental neurology 76:\\n358–375\\n\\nMirzadeh Z, Doetsch F, Sawamoto K, Wichterle H & Alvarez-Buylla A (2010) The subventricular zone\\n\\nen-face: wholemount staining and ependymal flow. J Vis Exp: 1938\\n\\n31\\n\\n\\n-----\\n\\nMirzadeh Z, Merkle FT, Soriano-Navarro M, Garcia-Verdugo JM & Alvarez-Buylla A (2008) Neural\\n\\nStem Cells Confer Unique Pinwheel Architecture to the Ventricular Surface in Neurogenic\\nRegions of the Adult Brain. Cell Stem Cell 3: 265–278\\n\\nNemajerova A, Kramer D, Siller SS, Herr C, Shomroni O, Pena T, Suazo CG, Glaser K, Wildung M,\\n\\nSteffen H, et al (2016) TAp73 is a central transcriptional regulator of airway multiciliogenesis.\\nGenes Dev\\n\\nPáez P, Bátiz L-F, Roales-Buján R, Rodríguez-Pérez L-M, Rodríguez S, Jiménez AJ, Rodríguez EM &\\n\\nPérez-Fígares JM (2007) Patterned neuropathologic events occurring in hyh congenital\\nhydrocephalic mutant mice. J Neuropathol Exp Neurol 66: 1082–1092\\n\\nPaez-Gonzalez P, Abdi K, Luciano D, Liu Y, Soriano-Navarro M, Rawlins E, Bennett V, Garcia-Verdugo\\n\\nJM & Kuo CT (2011) Ank3-Dependent SVZ Niche Assembly Is Required for the Continued\\nProduction of New Neurons. Neuron 71: 61–75\\n\\nRekate HL (2009) A Contemporary Definition and Classification of Hydrocephalus. Seminars in\\n\\nPediatric Neurology 16: 9–15\\n\\nRoales-Buján R, Páez P, Guerra M, Rodríguez S, Vío K, Ho-Plagaro A, García-Bonilla M, Rodríguez\\nPérez L-M, Domínguez-Pinos M-D, Rodríguez E-M, et al (2012) Astrocytes acquire\\nmorphological and functional characteristics of ependymal cells following disruption of\\nependyma in hydrocephalus. Acta Neuropathol 124: 531–546\\n\\nRodríguez EM & Guerra MM (2017) Neural Stem Cells and Fetal-Onset Hydrocephalus. Pediatr\\n\\nNeurosurg 52: 446–461\\n\\nSawamoto K, Wichterle H, Gonzalez-Perez O, Cholfin JA, Yamada M, Spassky N, Murcia NS, Garcia\\nVerdugo JM, Marin O, Rubenstein JLR, et al (2006) New neurons follow the flow of\\ncerebrospinal fluid in the adult brain. Science 311: 629–632\\n\\nSchrander-Stumpel C & Fryns JP (1998) Congenital hydrocephalus: nosology and guidelines for clinical\\n\\napproach and genetic counselling. Eur J Pediatr 157: 355–362\\n\\nSival DA, Guerra M, den Dunnen WFA, Bátiz LF, Alvial G, Castañeyra-Perdomo A & Rodríguez EM\\n\\n(2011) Neuroependymal denudation is in progress in full-term human foetal spina bifida aperta.\\nBrain Pathol 21: 163–179\\n\\nStubbs JL, Vladar EK, Axelrod JD & Kintner C (2012) Multicilin promotes centriole assembly and\\n\\nciliogenesis during multiciliate cell differentiation. Nat Cell Biol 14: 140–147\\n\\nTan FE, Vladar EK, Ma L, Fuentealba LC, Hoh R, Espinoza FH, Axelrod JD, Alvarez-Buylla A, Stearns\\n\\nT, Kintner C, et al (2013) Myb promotes centriole amplification and later steps of the\\nmulticiliogenesis program. Development 140: 4277–4286\\n\\nTerré B, Piergiovanni G, Segura-Bayona S, Gil-Gómez G, Youssef SA, Attolini CS-O, Wilsch\\nBräuninger M, Jung C, Rojas AM, Marjanović M, et al (2016) GEMC1 is a critical regulator of\\nmulticiliated cell differentiation. EMBO J 35: 942–960\\n\\n32\\n\\n\\n-----\\n\\nTesta G, Schaft J, van der Hoeven F, Glaser S, Anastassiadis K, Zhang Y, Hermann T, Stremmel W &\\n\\nStewart AF (2004) A reliable lacZ expression reporter cassette for multipurpose, knockout-first\\nalleles. Genesis 38: 151–158\\n\\nWallmeier J, Al-Mutairi DA, Chen C-T, Loges NT, Pennekamp P, Menchen T, Ma L, Shamseldin HE,\\n\\nOlbrich H, Dougherty GW, et al (2014) Mutations in CCNO result in congenital mucociliary\\nclearance disorder with reduced generation of multiple motile cilia. Nat Genet 46: 646–651\\n\\nWang L-L, Serrano C, Zhong X, Ma S, Zou Y & Zhang C-L (2021) Revisiting astrocyte to neuron\\n\\nconversion with lineage tracing in vivo. Cell 184: 5465-5481.e16\\n\\nYung YC, Mutoh T, Lin M, Noguchi K, Rivera RR, Choi JW, Kingsbury MA & Chun J (2011)\\n\\nLysophosphatidic Acid Signaling May Initiate Fetal Hydrocephalus. Sci Transl Med 3: 99ra87\\n\\nYung YC, Stoddard NC, Mirendil H & Chun J (2015) Lysophosphatidic acid (LPA) signaling in the\\n\\nnervous system. Neuron 85: 669–682\\n\\nFIGURE LEGENDS\\n\\nFigure 1.\\n\\nMcIdas forced expression promotes ex vivo reprogramming of murine cortical astrocytes into\\n\\nependymal cells.\\n\\n(A) Schematic representation of experimental procedures for (B-G). Astrocytes were isolated from the\\n\\ncortex of neonatal mice and were cultured under proliferating conditions. Subsequently, cells were\\n\\ntransduced with lentiviral vectors encoding GFP or GFP-McIdas (McIdas) and cultured under\\n\\ndifferentiating conditions for indicated time points.\\n\\n(B) Cortical astrocytes were immunostained against GFP (green) to mark infected cells, P73 (grey) and\\n\\nS100β (red) at day 7 of differentiation. Arrows point to McIdas-infected cells which are P73+ and\\n\\nrepresent cells committed to the ependymal lineage.\\n\\n(C-D) The graph depicts the percentage of the infected cells that co-express the ependymal marker P73\\n\\nover the total number of the infected cells (C) The percentage of the infected cells which display S100β\\n\\nstaining around their cell body, which corresponds to ependymal cells, is depicted in the graph (D) Data\\n\\nare presented as the median ±interquartile range (IQR) of three independent experiments. Statistical\\n\\nsignificance was determined using the non-parametric two-tailed Mann–Whitney test (***p < 0.0001).\\n\\n(E) Transduced astrocytes were immunostained at differentiation day 14 against GFP (green), FoxJ1\\n\\n33\\n\\n\\n-----\\n\\n(grey) and pericentrin (PCNT, red). Note the presence of multiple basal bodies, based on PCNT signal\\n\\naccumulation, in McIdas-infected cells which co-express the ependymal marker FoxJ1 (arrows).\\n\\n(F-G) The graphs present the percentage of the infected cells that express FoxJ1 over the total number of\\n\\nthe infected cells (F) and the percentage of FoxJ1-positive infected cells which also display\\n\\naccumulation of PCNT signal (G). Data are presented as the median ±IQR of three independent\\n\\nexperiments. Statistical significance was determined using the non-parametric two-tailed Mann–\\n\\nWhitney test (***p < 0.0001).\\n\\nDNA was stained with Dapi (blue). Scale bars, 10 μm.\\n\\nAbbreviations, dd: differentiation day\\n\\nFigure 2.\\n\\nMcIdas reprograms cortical astrocytes into functional ependymal cells.\\n\\n(A) Immunofluorescence of transduced cortical astrocytes at differentiation day 22 with antibodies\\n\\nagainst GFP (green), Glial Fibrillary Acidic Protein (GFAP, grey) which labels astrocytes and acetylated\\n\\na-tubulin (ACT, red), a marker of ciliary axonemes. McIdas expression triggers the formation of\\n\\nmultiple cilia, tiny hair-like structures which appear on the surface of the reprogrammed astrocytes\\n\\n(arrow). Note the downregulation of the GFAP signal in the reprogrammed multiciliated cells. DNA was\\n\\nstained with Dapi (blue). Scale bars, 10 μm\\n\\n(B-C) The graph shows the percentage of infected cells that express GFAP and correspond to an\\n\\nastrocytic population (B) The percentage of the infected cells which possess multiple cilia based on ACT\\n\\nstaining is depicted in the graph (C) Data are presented as the median ±IQR of two independent\\n\\nexperiments. Statistical significance was determined using the non-parametric two-tailed Mann–\\n\\nWhitney test (***p < 0.0001).\\n\\n(D) Live imaging microscopy experiments were performed on cortical astrocytes transduced either with\\n\\nGFP or McIdas lentiviruses. The positions of fluorescent beads included in the live imaging medium,\\n\\nwere marked with colored dots on selected video frames. Τrack of one representative bead is shown in\\n\\nthe right panel. Two independent experiments were performed. Scale bar, 10 μm\\n\\nAbbreviations, dd: differentiation day\\n\\nFigure 3.\\n\\nMcIdas induces ependymal cells regeneration in a mouse model of intracranial hemorrhage\\n\\n34\\n\\n\\n-----\\n\\nhydrocephalus.\\n\\n(A) Mice received unilateral intracranial injection of Lysophosphatidic acid (LPA) to induce\\n\\nhydrocephalus. Coronal brain slices derived from brains electroporated either with a GFP or GFP\\nMcIdas (McIdas) plasmid were cultured for 5 days. Immunofluorescence experiments were performed\\n\\nwith antibodies against GFP (green) and the ependymal marker P73 (red). Inserts show higher\\n\\nmagnification images. Arrows indicate GFP+P73+ cells.\\n\\n(B) The graph depicts the percentage of the electroporated cells that co-express the ependymal marker\\n\\nP73, over the total number of the electroporated cells, at day 5 of culture. Data are presented as the\\n\\nmedian ± IQR of three independent experiments. Statistical significance was determined using the non\\nparametric two-tailed Mann–Whitney test (***p = 0.0002).\\n\\n(C) Coronal brain slices were cultured for 9 days and immunostained against GFP (green), pericentrin\\n\\n(PCNT, red) and acetylated a-tubulin (ACT, grey) to detect mature multiciliated cells. Note the presence\\n\\nof multiple basal bodies and cilia in McIdas electroporated cells.\\n\\n(D-E) The percentage of the electroporated cells that carry multiple basal bodies, based on PCNT\\n\\naccumulation signal, upon 9 days of culture, is presented in the graph (D), (***p < 0.0001). The graph\\n\\nshows the percentage of multiciliated cells based on PCNT and ACT staining (E) upon 9 days of culture,\\n\\n(p=0.0013). Data are presented as the median ±IQR of three independent experiments. Statistical\\n\\nsignificance was determined using the non-parametric two-tailed Mann–Whitney test.\\n\\nDNA was stained with Dapi (blue). Scale bars, 10 μm. .\\n\\nFigure 4.\\n\\nMcIdas ectopic expression promotes in vivo reprogramming into ependymal cells in a genetic\\n\\nmouse model of hydrocephalus.\\n\\n(A) Neonatal subventricular zone electroporation was conducted at GemC1-knockout mice at postnatal\\n\\nday 1 (P1) with plasmids encoding GFP, or GFP-McIdas (McIdas). Mice were sacrificed 4 days post\\n\\nelectroporation and coronal brain sections were immunostained with antibodies against GFP (green) and\\n\\nP73 (red). Arrows point to McIdas-electroporated cells co-expressing P73, which represent cells\\n\\ncommitted to the ependymal lineage. Higher magnification of the boxed region is shown in the right\\n\\npanel.\\n\\n(B) The graph shows the percentage of the electroporated cells expressing P73. Data are presented as the\\n\\nmedian ±IQR of three independent experiments. Statistical significance was determined using the non\\n35\\n\\n\\n-----\\n\\nparametric two-tailed Mann–Whitney test (***p < 0.0001).\\n\\n(C) Coronal brain sections from GemC1-knockout electroporated mice were stained with specific\\n\\nantibodies against GFP (green), Pericentrin (PCNT, red) and acetylated a-tubulin (ACT grey). Multiple\\n\\nbasal bodies and cilia on the surface of McIdas electroporated cells are indicated by an arrow. Higher\\n\\nmagnification of the boxed region is shown in the right panel.\\n\\n(D-E) Quantification of the percentage of the electroporated cells with multiple basal bodies, based on\\n\\nPCNT accumulation (D), (***p < 0.0001), and the percentage of multiciliated cells based on PCNT and\\n\\nACT staining over the total number of the electroporated cells (E), (P=0.0041). Data are presented as the\\n\\nmedian ±IQR of two independent experiments. Statistical significance was determined using the non\\nparametric two-tailed Mann–Whitney test.\\n\\nDNA was stained with Dapi (blue). Scale bars, 10 μm. .\\n\\nAbbreviations, LV: lateral ventricle\\n\\nFigure 5.\\n\\nMcIdas establishes cilia motility in reprogrammed cells in a genetic mouse model of\\n\\nhydrocephalus.\\n\\n(A) Schematic representation of experimental procedures for (B-D). GFP or GFP-McIdas (McIdas)\\n\\nexpressing plasmids were electroporated at the lateral ventricle of postnatal day 1 (P1) GemC1-knockout\\n\\nmouse brains. Coronal brain sections, that contained the lateral ventricles, were obtained from the\\n\\nelectroporated mice at ages P7-P15 and were used in live imaging experiments.\\n\\n(B) Cilia still images from GFP or McIdas electroporated brains at indicated time points in milliseconds.\\n\\nFITC fluorescent cells are presented together with brightfield images. Arrows indicate the positions of\\n\\ncilia at indicated time points. Cilia motility was observed upon McIdas ectopic expression as evidenced\\n\\nby cilia changing positions at indicated time points. Higher magnification of the boxed regions is shown\\n\\nin the right panels.\\n\\n(C) Representative kymograms depicting cilia motility upon GFP or McIdas ectopic expression. Left\\n\\npanel images show the lines used to obtain the kymograms. These images are extracted from the image\\n\\nshown in B, which was utilized to generate the kymograph.(D) The graph shows cilia beat frequency of\\n\\nelectroporated cells quantified from high-speed brightfield cilia motility movies. Data are presented as\\n\\nthe median ±IQR (n=2 for GFP (control) and n=4 for McIdas). Statistical significance was determined\\n\\nusing the non-parametric two-tailed Mann–Whitney test (***p < 0.0001).\\n\\n36\\n\\n\\n-----\\n\\nFigure 6.\\n\\nEctopic expression of McIdas restores the subventricular zone niche cytoarchitecture.\\n\\n(A-B) GemC1-knockout mouse brains were electroporated with plasmids encoding GFP or McIdas-GFP\\n\\n(McIdas) at postnatal day 1 (P1). Mice were sacrificed at ages P7-P9 and lateral ventricular wall whole\\nmount immunofluorescence was conducted with antibodies against GFP (green), β-catenin (red), and\\n\\nVascular Cell adhesion Molecule 1 (VCAM1, grey) (A) or Glial Fibrillary Acidic Protein (GFAP, grey)\\n\\n(B).\\n\\nFormation of pinwheel structures by McIdas-reprogrammed ependymal cells surrounding VCAM1+\\n\\nneural stem cells (arrow, A) or GFAP-expressing neural stem cells (B) was observed. Note the enlarged\\n\\napical domain of McIdas-reprogrammed ependymal cells.\\n\\nHigher magnification of the boxed region is shown in the right panel.\\n\\n(C) The graph shows the percentage of electroporated cells, shown in A, identified in pinwheel\\n\\nstructures. Data are presented as the median ±IQR (n=3 for GFP (control) and n=4 for McIdas).\\n\\nStatistical significance was determined using the non-parametric two-tailed Mann–Whitney test (***p <\\n\\n0.0001.).\\n\\nScale bars, 10 μm.\\n\\nEV Figure Legends\\n\\nFigure EV1.\\n\\nMouse embryonic stem cells programming into ependymal cells.\\n\\n(A) Mouse embryonic stem cells (mESCs) were transfected with GFP, GFP-McIdas (McIdas) or GFP\\nGemC1 (GemC1) expressing vectors and subsequently cultured under differentiating conditions.\\n\\nImmunofluorescence was performed with antibodies against GFP (green), FoxJ1 (red) and Pericentrin\\n\\n(PCNT, magenta) at differentiation day 4. The arrow points to multiple basal bodies in McIdas and\\n\\nGemC1 transfected cells which co-express the ependymal marker FoxJ1. Higher magnification of the\\n\\nboxed regions is shown in the right panels.\\n\\n(B-C) The graphs present the percentage of transfected mESCs that express FoxJ1 over the total number\\n\\nof the transfected cells (B) and the percentage of transfected cells which display multiple basal bodies\\n\\n37\\n\\n\\n-----\\n\\nbased on the accumulation of PCNT signal (C) Data are presented as the median ±IQR of two\\n\\nindependent experiments. Statistical significance was determined using the non-parametric two-tailed\\n\\nMann–Whitney test (***p < 0.0001).\\n\\n(D) GFP (green), PCNT (magenta) and acetylated-a-tubulin (ACT red) immunostaining in transfected\\n\\nmESCs at differentiation day 6. McIdas and GemC1 transfected mESCs present multiple basal bodies\\n\\nand cilia.\\n\\n(E) The graph shows the percentage of transfected cells with multiple basal bodies (PCNT\\n\\naccumulation) and multiple cilia (ACT labeled cilia) over the total number of the transfected cells. Data\\n\\nare presented as the median ±IQR of two independent experiments. Statistical significance was\\n\\ndetermined using the non-parametric two-tailed Mann–Whitney test (***p < 0.0001).\\n\\nDNA was stained with Dapi (blue). Scale bars, 10 μm,\\n\\nAbbreviations, dd: differentiation day\\n\\nFigure EV2.\\n\\nGemC1 drives the commitment of murine cortical astrocytes towards the ependymal cell lineage.\\n\\n(A) Cortical astrocytes were transduced with lentiviruses encoding GFP or GFP-GemC1 (GemC1).\\n\\nImmunofluorescence experiments were performed at day 7 of differentiation with specific antibodies\\n\\nagainst GFP (green) to mark the infected cells, P73 (grey) and S100β (red). Arrow point to a P73+\\n\\ninfected cell, committed to the ependymal lineage.\\n\\n(B-C) Graph depicting the percentage of infected cells which express P73 over the total number of\\n\\ninfected cells (B). The graph shows the percentage of the infected cells which display S100β staining\\n\\naround their cell body (C). Three independent experiments were analyzed. Data are presented as the\\n\\nmedian ±IQR. Statistical significance was determined using the non-parametric two-tailed Mann–\\n\\nWhitney test (***p < 0.0001).\\n\\n(D) Astrocytes were immunostained against GFP (green), FoxJ1 (grey) and pericentrin (PCNT red) at\\n\\ndifferentiation day 14. The arrow points to a GemC1-infected cell which expresses FoxJ1 and possesses\\n\\nmultiple basal bodies.\\n\\n(E-F) Graph presenting the percentage of the infected cells that express FoxJ1 over the total number of\\n\\nthe infected cells (E) The graph shows the percentage of infected cells which express the ependymal\\n\\nmarker FoxJ1 and display accumulation of PCNT signal (F). Data are presented as the median ±IQR\\n\\nfrom three independent experiments. Statistical significance was determined using the non-parametric\\n\\n38\\n\\n\\n-----\\n\\ntwo-tailed Mann–Whitney test (***p < 0.0001).\\n\\nDNA was stained with Dapi (blue). Scale bars, 10 μm\\n\\nAbbreviations, dd: differentiation day\\n\\nFigure EV3.\\n\\nLysophosphatidic Acid (LPA) administration causes hydrocephalus in early postnatal stages in\\n\\nmice.\\n\\n(A) Top views of Postnatal day 7 (P7) wild type mouse brains that either received or not injection of\\n\\nLPA at the lateral ventricle at P5. No differences on the size of the brain were observed macroscopically.\\n\\n(B) Dapi staining on coronal brain sections from P7 mouse brains reveals the dilation of the lateral\\n\\nventricles after LPA injections. Scale bar, 100 μm.\\n\\n(C) Immunofluorescence for Glial Fibrillary Acidic Protein (GFAP, green) which labels astrocytes and\\n\\nacetylated a-tubulin (ACT, red), a marker of ciliary axonemes on coronal sections from P7 mouse brains\\n\\nat the dorsal-lateral and lateral regions of the lateral walls. Arrowheads point to multiple cilia in control\\n\\nependymal cells (ep). Arrows show GFAP positive astrocytes (as) in the lateral wall of LPA-injected\\n\\nbrains, where ciliary disruption is observed. Higher magnification of the boxed regions is shown in the\\n\\nright panels. Scale bars, 10 μm\\n\\nDNA was stained with Dapi or Draq-5 (blue).\\n\\nFigure EV4.\\n\\nEx vivo reprogramming of radial glial cells to ependyma can be achieved through McIdas ectopic\\n\\nexpression.\\n\\n(A) Schematic representation of experimental procedures for (B-E). Postnatal radial glial cells (pRGCs)\\n\\nwere isolated from GemC1-knockout newborn mice, cultured and infected with lentiviruses expressing a\\n\\nGFP-McIdas (McIdas) or a GFP-P73 (P73) fusion protein, while GFP alone was used as a control. Cells\\n\\nwere then cultured under differentiating conditions and analyzed at indicated timepoints.\\n\\n(B) Transduced radial glial cells were co-stained with antibodies against GFP (green) to mark infected\\n\\ncells, FoxJ1 (red) and pericentrin (PCNT, grey) 5 days after the initiation of differentiation. The arrow\\n\\nindicates the accumulation of PCNT signal in FoxJ1 expressing cells upon McIdas ectopic expression.\\n\\n(C) The graph presents the percentage of infected cells which express FoxJ1 and display multiple basal\\n\\nbodies (accumulation of PCNT signal). Three independent experiments were analyzed. Data are\\n\\n39\\n\\n\\n-----\\n\\npresented as the median ±IQR. Statistical significance was determined using the non-parametric two\\ntailed Mann–Whitney test (***p < 0.0001).\\n\\n(D) Radial glial cells infected with GFP, McIdas, or P73 lentiviruses were labeled with antibodies\\n\\nagainst GFP (green), pericentrin (PCNT, grey) and acetylated a-tubulin (ACT, red) to detect mature\\n\\nmulticiliated cells at differentiation day 15 (arrow).\\n\\n(E) The percentage of the infected cells which displayed multiple basal bodies, based on PCNT staining\\n\\nand simultaneously multiple cilia, based on ACT staining, was analyzed. Data are presented as the\\n\\nmedian ±IQR of three independent experiments. Statistical significance was determined using the non\\nparametric two-tailed Mann–Whitney test (***p < 0.0001).\\n\\nDNA was stained with Hoechst (blue). Scale bars, 10 μm,.\\n\\nFigure EV5.\\n\\nMcIdas drives the commitment to the ependymal cell lineage in a genetic mouse model for\\n\\nhydrocephalus.\\n\\n(A) Subventricular zone electroporation was conducted at postnatal day 1 (P1) GemC1-knockout mice\\n\\nwith plasmids encoding GFP, or GFP-McIdas (McIdas). Coronal brain sections were stained with\\n\\nantibodies against GFP (green) to mark the electroporated cells and FoxJ1 (red), a marker of committed\\n\\nependymal cells, 4 days post electroporation. Arrows point to McIdas-electroporated cells expressing\\n\\nFoxJ1. Higher magnification of the boxed region is shown in the right panel.\\n\\n(B) Graph depicting the percentage of electroporated cells that express FoxJ1 over the total number of\\n\\nthe electroporated cells. Two independent experiments were analyzed. Data are presented as the median\\n\\n±IQR. Statistical significance was determined using the non-parametric two-tailed Mann–Whitney test\\n\\n(***p < 0.0001).\\n\\nDNA was stained with Dapi (blue). Scale bars, 10 μm\\n\\n40\\n\\n\\n-----\\n\\n\""
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "md_text"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 2. Markitdown\n",
    "\n",
    "It seems to do a very good job at isolating different sections of the document. It clearly finds the `Figure legends` and `Data availability` sections.\n",
    "\n",
    "Although it extracts data from the PDF too, the extraction is brutally noisy compared to the docx file. The markdown structure is much better depicted.\n",
    "\n",
    "In the markitdown library, the conversion of `.docx` files into structured text or Markdown is handled by its custom conversion engine built on top of `python-docx`.\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "from markitdown import MarkItDown"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "**Latrophilin-2 mediates fluid shear stress mechanotransduction at endothelial junctions**\n",
      "\n",
      "Keiichiro Tanaka1,7,\\*, Minghao Chen1,7, Andrew Prendergast1, Zhenwu Zhuang1, Ali Nasiri2, Divyesh Joshi1, Jared Hintzen1, Minhwan Chung1, Abhishek Kumar1, Arya Mani1, Anthony Koleske3, Jason Crawford4, Stefania Nicoli1 and Martin A. Schwartz1,5,6,\\*\n",
      "\n",
      "1 Yale Cardiovascular Research Center, Section of Cardiovascu­­lar Medicine, Department of Internal Medicine, School of Medicine, Yale University, New Haven, CT 06511, USA\n",
      "\n",
      "2 Department of Internal Medicine\n",
      "\n",
      "3 Department of Molecular Biochemistry and Biophysics\n",
      "\n",
      "4 Department of Chemistry\n",
      "\n",
      "5 Department of Cell Biology\n",
      "\n",
      "6 Department of Biomedical Engineering\n",
      "\n",
      "7 These authors contributed equally.\n",
      "\n",
      "Running title: Latrophilins in shear stress sensing\n",
      "\n",
      "\\*Authors for correspondence:\n",
      "\n",
      "keiichiro.tanaka@yale.edu; martin.schwartz@yale.edu;\n",
      "\n",
      "**Abstract**\n",
      "\n",
      "Endothelial cell responses to fluid shear stress from blood flow are crucial for vascular development, function and disease. A complex of PECAM-1, VE-cadherin, VEGF receptors (VEGFRs) and PlexinD1 located at cell-cell junctions mediates many of these events. But available evidence suggests that another mechanosensor upstream of PECAM-1 initiates signaling. Hypothesizing that GPCR and Gα proteins may serve this role, we performed siRNA screening of Gα subunits and found that Gαi2 and Gαq/11 are required for activation of the junctional complex. We then developed a new activation assay, which showed that these G proteins are activated by flow. We next mapped the Gα residues required for activation and developed an affinity purification method that used this information to identify latrophilin-2 (Lphn-2/ADGRL2) as the upstream GPCR. Latrophilin-2 is required for all PECAM-1 downstream events tested. In both mice and zebrafish, latrophilin-2 is required for flow-dependent angiogenesis and artery remodeling. Furthermore, endothelial specific knockout demonstrates that latrophilin plays a role in flow-dependent artery remodeling. Human genetic data reveal a correlation between the latrophilin-2-encoding *Adgrl2* gene and cardiovascular disease. Together, these results define a pathway that connects latrophilin-dependent G protein activation to subsequent endothelial signaling, vascular physiology and disease.\n",
      "\n",
      "**Keywords**\n",
      "\n",
      "Latrophilin, G protein-coupled receptor, fluid shear stress, vascular development, PECAM-1\n",
      "\n",
      "**Introduction**\n",
      "\n",
      "Fluid shear stress (FSS) from blood flow is critical for early vascular development in mice(Lucitti *et al*, 2007) and zebrafish(Sugden *et al*, 2017), for homeostatic vessel remodeling in adults(Humphrey & Schwartz, 2021; Vogel *et al*, 2020), and in the initiation and progression of atherosclerosis(Baeyens *et al*, 2016; Zhou *et al*, 2014). A complex of proteins at cell-cell junctions consisting of PECAM-1, VE-cadherin, VEGFR2 and 3, and Plexin D1 plays an important role in shear stress responses including vasodilation, flow-dependent vessel remodeling, and atherosclerosis(Conway & Schwartz, 2012; Givens & Tzima, 2016). Stimulation of this junctional complex by flow triggers activation of Src-family kinases (SFKs) within seconds, resulting in ligand-independent activation of VEGF receptors and downstream events including endothelial cell (EC) alignment in the direction of flow(Collins *et al*, 2012). However, the mechanisms by which proteins located at cell-cell junctions can sense forces from shear stress exerted on the apical surface is unclear. Previous work suggests that activation of the junctional complex is not primary but rather requires an upstream event, mediated presumably by another mechanosensor (Conway *et al*, 2013).\n",
      "\n",
      "GPCR/Gα protein signaling is an obvious candidate for initiating signaling on this time scale (seconds). GPCRs are a large family of cell surface receptors that play a crucial role in transducting signals across cell membranes. GPCRs undergo conformational changes in response to ligand binding or other stimuli, after which the activated intracellular domain binds to a heterotrimeric Gα protein complex and catalyzes the exchange of guanosine diphosphate (GDP) for guanosine triphosphate (GTP). Connections between GPCRs and shear stress have been reported(Chachisvilis *et al*, 2006; Dela Paz & Frangos, 2019; Erdogmus *et al*, 2019; Jung *et al*, 2012; Wang *et al*, 2015; Xu *et al*, 2018). However, the literature on Gα proteins and GPCRs in shear stress signaling is highly inconsistent(Chachisvilis *et al.*, 2006; Dela Paz & Frangos, 2019; Jung *et al.*, 2012; Wang *et al.*, 2015). For instance, it has been demonstrated that flow applied to endothelial cells activates Gα proteins independent of an intervening GPCR(Dela Paz *et al*, 2017; Gudi *et al*, 1998). It has also been reported that shear stress responses related to PECAM-1 and cell-cell junction require GPCRs but with different GPCRs implicated in different studies. These studies include reports of ligand-independent activation of a bradykinin receptor(Chachisvilis *et al.*, 2006; Yeh *et al*, 2008) and the sphingosine 1-phosphate receptor S1PR1(Jung *et al.*, 2012). The purinergic receptor P2Y2 was also implicated, in this case via ATP release downstream of activation of the mechanosensitive ion channel Piezo1(Albarran-Juarez *et al*, 2018; Wang *et al.*, 2015), though this pathway has been disputed(Dela Paz & Frangos, 2019). The orphan receptor GPR68 was identified as a shear stress sensor specific to small arteries though the mechanism of activation, including ligand dependence or independence, is unknown(Xu *et al.*, 2018). These studies also vary in the Gα proteins implicated, which included different Gαi isoforms (Gudi *et al*, 1996; Jung *et al.*, 2012) and the Gαq/11 family(Albarran-Juarez *et al.*, 2018; Chachisvilis *et al.*, 2006; Wang *et al.*, 2015).\n",
      "\n",
      "To resolve this issue, we took an unbiased approach starting from the Gα proteins using the dramatic morphological changes in endothelial alignment to flow as the initial readout. Our results showed that Gαi2 and Gαq/11 function in parallel, identified latrophilin-2 (LPHN2) as the key upstream GPCR and demonstrated a specific requirement for LPHN2 in flow-dependent remodeling in zebrafish and mice.\n",
      "\n",
      "**Results**\n",
      "\n",
      "Identification of Gα protein subunits required for flow sensing\n",
      "\n",
      "To investigate the roles of GPCR signaling in endothelial flow responses, we first transfected human umbilical vein endothelial cells (HUVECs) with short interfering RNAs (siRNAs) to suppress Gs, Gi, Gq/11 and G12/13 classes of Gα proteins and assayed EC alignment in flow as our initial readout. Single knockdowns gave either weak (Gi) or no (Gs, G12/13, Gq/11) inhibition of EC alignment in flow (Fig.1A and Appendix fig. S1A-C). However, simultaneous knockdown of Gi and Gq/11 completely inhibited alignment (Fig. 1A), whereas other combinations were ineffective (Appendix fig. S1D, S1E). Activation of SFKs is the earliest flow-responsive event mediated by PECAM-1, followed by VEGFRs and Akt(Tzima *et al*, 2005). These events were similarly blocked by knockdown of Gi plus Gq/11 but not by single knockdowns (Fig. 1B, 1C and Appendix fig. S1F). Therefore, Gi and Gq/11 act in parallel to initiate PECAM-1-dependent flow signaling.\n",
      "\n",
      "Rescue controls with siRNA-resistant Gα variants showed that siRNA-resistant Gq and Gi2 restored cell alignment, whereas Gi1 and Gi3 did not (Fig. 1D and Appendix fig. S2A, S2B). This unexpected result prompted us to search for residues that are similar in Gq and Gi2 but different in Gi1 and Gi3. Two residues (using human Gi2 numbering) fit this pattern: D167 in Gi2 corresponds to N in Gi1/3; and K307 corresponds to Q306 in Gi1/3 (Fig. 1E). Gi1(Q306K), but not Gi1(N166D), rescued EC alignment in flow (Fig. 1F and Appendix fig. S2C). Consistently, Gi2 mutant lacking K307 (K307Q) could not rescue the defects on flow-induced alignment under the knockdown of Gi plus Gq/11 (Appendix fig. S1G and Appendix fig. S2D). Therefore, K307 in Gi2 distinguishes the Gα subunits participate in flow signaling.\n",
      "\n",
      "Flow-induced activation of specific Gα proteins\n",
      "\n",
      "K307 lies in the region that specifies coupling with GPCRs(Bae *et al*, 1999; Flock *et al*, 2017), suggesting that K307 may determine Gα activation. However, testing this hypothesis requires measurement of Gα activation in live cells. Unfortunately, indirect Gα activation assays using second messengers such as cAMP or calcium are unreliable in the context of fluid shear stress where multiple activating/suppressing pathways are stimulated. We therefore developed pulldown assays for Gα based on the specific binding of GTP-loaded Gα subunits to effector proteins, namely GINIP for Gi and GRK2 N-terminal domain for Gq (Appendix fig. S3a)(Gaillard *et al*, 2014; Tesmer *et al*, 2005). To facilitate these assays and distinguish specific Gα isoforms (especially Gi isoforms), we prepared versions containing an internal GluGlu (EE) epitope tag that does not impair function (Appendix fig. S3B and refs(Medina *et al*, 1996; Wilson & Bourne, 1995)). For Gq, to enhance pulldown efficiency, we co-expressed Ric8A, which stabilizes active Gq(Papasergi *et al*, 2015). Active, but not inactive, Gi and Gq bound to these effector proteins, while when the artificial GPCR DREADD was expressed, its ligand clozapine-N-oxide (CNO) also activated Gα proteins (Appendix fig. S3C-S3F). Importantly, this Gα pulldown assay showed not only rapid activation of Gi2 and Gq by FSS (Fig. 1G-J), but also differential responses of Gi1 and gain-of-function Gi1(Q306K) to FSS (Fig. 1K, 1L). Gi activation was unaffected by PECAM-1 knockdown, indicating that this step was upstream or independent of PECAM-1 (Fig. 1M, 1N). Flow-mediated activation of specific Gα subunits thus corresponds to their requirement in EC flow responses.\n",
      "\n",
      "Identification of the upstream GPCR\n",
      "\n",
      "We next sought to identify the GPCR(s) that mediates flow activation of Gi2 and Gq/11. Although affinity purification is the obvious choice, rapid release of Gα proteins from GPCRs after GTP loading limits this approach. Starting with wild-type and its Q306K mutant, we inserted 1) an internal GFP for affinity purification and 2) four alanines (hereafter ins4A) into helix α5, which blocks GTP loading, thereby preventing dissociation from GPCRs even in the presence of GTP(Kaya *et al*, 2016) (Fig. 2A). Both constructs localized mainly to the plasma membrane, as did wild-type Gα (Appendix fig. S4A). We further employed detergent-free nanodisc-forming styrene-maleimide anhydride (SMA) copolymer to stabilize GPCR conformation within its native lipid environment(Lee *et al*, 2016). This approach was validated by coimmunoprecipitation using GFP-Trap® nanobody beads of Gα variants with DREADD GPCR, which showed increased association after receptor activation (Appendix fig. S4B, S4C). Proteomic analysis for GPCRs that associated with Gi1(Q306K) but not wild-type Gi1 in response to FSS (protocol in Appendix fig. S4D), identified S1PR1 and ADGRL3 (Appendix fig. S4E). However, depletion of S1PR1 did not block alignment in FSS (Appendix fig. S4F, S4G). Furthermore, S1PR1 reportedly activates Gi1 and Gi3 as well as Gi2(Lee *et al*, 1996), which is inconsistent with the Gα specificity defined above. We therefore focused on ADGRL family proteins, also called latrophilins (LPHNs).\n",
      "\n",
      "Latrophilins (1,2 and 3 in mammals) are adhesion-type GPCRs that bind to counterreceptors on neighboring cells and regulate neuronal synapses(Sando *et al*, 2019). Although it has been reported that different types of endothelial cells have different latrophilin isoforms, LPHN2 is most widely expressed and is the overwhelming major isoform in HUVECs (Appendix figs S5A)(Maleszewska *et al*, 2016). Consistent with the proteomic results, Gi1(Q306K)(ins4A) associated with endogenous LPHN2, which increased upon FSS (Fig. 2B, C). This result strongly suggests that FSS activates LPHN2 to recruit Gα proteins. Furthermore, Gi1(Q306K), Gi2, and Gq co-immunoprecipitated with LPHN2, whereas Gi1 did not, indicating that LPHN2 specifically bound to the flow-responsive Gα proteins defined above (Fig. 2D). Importantly, LPHN2 knockdown blocked flow-mediated activation of Gi2 (Fig. 2E, 2F). Together, these data provide strong evidence that latrophilin-2 mediates Gα activation essential for endothelial alignment in response to flow.\n",
      "\n",
      "Latrophilins in EC flow signaling *in vitro*.\n",
      "\n",
      "Knockdown of LPHN2, but not LPHN1 or LPHN3, blocked EC alignment in flow (Fig. 3A, 3B; knockdown confirmed in Appendix fig. S5B, S5C). LPHN2 knockdown in human aortic ECs similarly blocked flow-induced alignment (Appendix fig. S5D, S5E). In line with these results, LPHN2 knockdown also blocked Golgi apparatus polarization in flow (Appendix fig. S5F, S5G). LPHN2 localized to cell-cell contacts (Fig.3C and Appendix fig. S6). The flow responsive G1 mutant, Gi1(Q306K), also associated with PECAM-1 following FSS (Fig. 3D). The association between LPHN2-Gi1(Q306K) complex with junctional mechanosensory complex proteins PECAM-1 and VE-Cadherin was detected after 24 hours of flow (Fig. 3D-3G), indicating a persistent interaction. We also assessed the immediate flow-dependent signals. LPHN2 depletion strongly suppressed flow-mediated acute activation of SFKs and Akt (Fig 3H-3J). Thus, LPHN2 colocalizes, physically associates with and is required for signaling through the junctional complex. However, LPHN2 knockdown did not inhibit activation of VEGFR2 by VEGF-A, indicating that LPHN2 functions specifically in response to laminar flow (Appendix fig. S7A, S7B).\n",
      "\n",
      "Among the 3 latrophilin isoforms in human genome, all rescued flow-mediated endothelial alignment after LPHN2 knockdown (Fig. 3K, expression confirmed in Appendix fig. S8A), indicating functional equivalence; the observed differences are therefore due to differential expression. EC alignment was rescued by wild type (WT) LPHN2 but not H1071A, which is defective in coupling to Gα proteins(Nazarko *et al*, 2018) (Fig. 3L, expression confirmed in Appendix fig. S8B). Adhesion-type GPCRs can be activated by internal cleavage by the extracellular GAIN domain which exposures its *Stachel* peptide as a tethered agonist(Vizurraga *et al*, 2020). However, the autoproteolysis-defective mutant fully rescued EC alignment (see Fig.3L). Helix-8 is known to endow some GPCRs with mechanosensitivity to membrane stretch(Erdogmus *et al.*, 2019). However, a LPHN2 helix-8 mutant could still rescue EC alignment in flow (Fig. 3L). Taken together, these findings indicate that latrophilin regulates Gα activation required for endothelial flow responses, but activation requires neither the *Stachel* peptide nor helix-8.\n",
      "\n",
      "Latrophilin-2 regulates flow-induced endothelial cell morphological changes *in vivo*\n",
      "\n",
      "Long-term laminar flow stabilizes EC junctions, promoting a linear, continuous morphology associated with barrier function(Kroon *et al*, 2017) (see Fig.3A). To test whether the role of LPHN2 in this process, we analyzed junctional morphology *in vitro* with and without 16h of flow. In control cells, shear stress shifted junctions from an irregular morphology to linear junctions; in Lphn2 knockdown ECs, junctions were indistinguishable in the absence of flow but failed to linearize in flow (Fig. 4A, 4B), demonstrating that the effect is flow-specific. To further examine contributions of blood flow/latrophilin-2 to ECs *in vivo,* zebrafish embryos are useful as a developmental system that is amenable to manipulation of shear stress using morpholinos against cardiac troponin T type 2a (tnnt2a; ‘*silent heart*’)(Sehnert *et al*, 2002), without global adverse effects. We employed zebrafish embryos expressing Tg(*kdrl:ras-mCherry*) to mark endothelial cells(Sehnert *et al.*, 2002) and designed sgRNAs for CRISPR-Cas9 targeting all latrophilin isoforms (*adgrl1*, *adgrl2a* plus *adgrl2b.1*, and *adgrl3*). Our sgRNAs induced >50% cleavage in the T7 endonuclease assays and reduced mRNA levels through nonsense-mediated decay (Appendix fig. S9). Since Lphn2 is the major endothelial isoform, we performed CRISPR-mediated F(0)/mosaic knockout of *adgrl2a*+*adgrl2b.1* (hereafter Lphn2 mutation), which inhibited elongation and enlargement of ECs in the dorsal aorta (Fig. 4C). Injecting single cell embryos with *silent heart* morpholinos at a dose that completely blocked cardiac contractility similarly blocked the elongation of ECs. Furthermore, combining tnnt2a and Lphn2 mutation had no further effect, supporting a specific role for latrophilins in flow sensing (Fig. 4D, 4E).\n",
      "\n",
      "To address LPHN function in mammals, we crossed Lphn2flox/flox mice with CDH5-CreERT2 mice (Lphn2 ECKO), which were injected with tamoxifen at 8 weeks of age and remained viable 10 weeks later. The descending thoracic aorta (a region of high laminar flow) of Lphn2 ECKO mice had poorly aligned ECs with irregular junctions (Fig. 4F-H). Additionally, Lphn2 ECKO increased vascular permeability in the whole aorta, as assessed by Evans blue extravasation (Fig. 4I, 4J). *In vitro*, flow for 24h increased HUVEC barrier function, which was blocked by LPHN2 knockdown, whereas barrier function in the absence of flow was unaffected (Fig. 4K, 4L). Latrophilins are therefore flow-responsive GPCRs that regulate multiple steps in PECAM1-dependent EC mechanotransduction.\n",
      "\n",
      "Plasma membrane fluidization initiates latrophilin signaling\n",
      "\n",
      "We next considered how FSS activates LPHN2. As mutation of the autoproteolytic GAIN domain, which is speculated to be mechanosensitive, had no effect (Figure 3i), we considered alternatives. FSS induces a rapid increase in plasma membrane fluidity, in some cases associated with decreased plasma membrane cholesterol(Butler *et al*, 2001; Haidekker *et al*, 2000; Yamamoto & Ando, 2013; Yamamoto *et al*, 2020). As GPCRs can respond to such changes(Vizurraga *et al.*, 2020), HUVECs were treated with methyl-β-cyclodextrin (MβCD) to extract plasma membrane cholesterol. Treatment with 5 mM MβCD for 1 min reduced plasma membrane cholesterol to the same extent as flow, assayed by binding of the cholesterol-specific probe D4H-Clover3(Liu *et al*, 2017) (Figure 5A). No discernable effect on cytoskeletal organization was observed under these conditions (Figure 5B). MβCD treatment also induced Gα binding to LPHN2 (Figure 5C) and activated both SFKs and VEGFR2, which were blocked by LPHN2 knockdown (Figure 5D-F). Together with published data demonstrating that cholesterol supplementation blocks flow signaling to these and related pathways(Fancher *et al*, 2018; Tirziu *et al*, 2005; Yamamoto & Ando, 2013, 2015; Yamamoto *et al.*, 2020), these results suggest that membrane fluidization can trigger LPHN2 GPCR activation and downstream signaling (Figure 5G).\n",
      "\n",
      "Latrophilins in vascular remodeling *in vivo*\n",
      "\n",
      "To address roles of LPHN2 in flow-dependent vascular remodeling, we examined zebrafish embryos, where artery lumen diameters are determined by EC shear stress(Sugden *et al.*, 2017). We observed that intersegmental vessels (ISVs) at 48 hours post fertilization had reduced diameters after both latrophilin knockout and blockade of blood flow, with no further effect when combined (Fig. 6A, 6B). At 72 hpf, when intersegmental arteries and veins are distinguishable, the decrease in diameter was observed only in arteries (Fig. 6C). LPHNs are thus required for effects of flow on artery diameter.\n",
      "\n",
      "These results prompted us to examine flow-dependent vascular remodeling in mice. Femoral artery ligation triggers flow-driven arteriogenesis in the thigh and hypoxia-driven angiogenesis in the calf(Cornella *et al*, 2017). Following surgery, LPHN2 ECKO mice showed lower blood flow in the foot even at day 1 and markedly slower recovery than WT littermates (Fig. 6D, 6E). To understand these effects, we performed micro-CT of vascular casts after injection of electron-dense bismuth microparticles that label arteries/arterioles but not capillaries due to particle size. Surgery induced an increase in small arteries in WT mice but essentially none in LPHN2 ECKO mice (Fig. 6F, 6G). LPHN2 ECKO mice also had much lower density of small arteries even in the control leg, consistent with lower flow on day 1. Staining for PECAM to examine the total vasculature in leg muscle sections revealed decreased capillary density as well, even at baseline (Fig. 6H, 6I). LPHN2 ECKO mice thus show low vascular density at baseline and greatly reduced flow-dependent remodeling. As expected from the low vascular density, a treadmill fatigue test (Fig. 6J) LPHN2 ECKO mice showed markedly lower exercise capacity (Fig. 6K, 6L) and impaired oxygen consumption (Fig. 6M). LPHN2 ECKO thus results in vascular defects *in vivo*, at least in part due to defective EC flow sensing. To address whether these defects were related to shear stress sensing, we examined an *in vitro* system in which FSS enhances VEGF-induced sprouting into collagen gels(Galie *et al*, 2014). Enhanced sprouting was completely blocked by LPHN2 depletion with only a weak effect on basal migration (Fig. 6N-6P).\n",
      "\n",
      "Genetic link to human disease.\n",
      "\n",
      "The Cardiovascular Disease Knowledge Portal reported four SNPs near the 5’ end of the *ADGRL2* gene (Appendix fig. S10A) that exhibit genome wide association with hypertension (Appendix fig. S10B). All 4 are in close proximity to each other and are in linkage disequilibrium (D’: 0.9953, R2: 0.97). Two of the SNPs meet the significant threshold (p<5 x 10-8) and rs1195871 is relatively conserved among vertebrates. Of particular interest, the intronic SNP rs186892211 in the *ADGRL2* locus (Appendix fig. S10A) is associated with large-artery atherosclerosis and subsequent ischemic stroke (TOAST classification) with a very high odds ratio (odds ratio 720; p< 3.134e-10, Appendix fig. S10B).\n",
      "\n",
      "**Discussion**\n",
      "\n",
      "These data elucidate a pathway in which fluid shear stress acts through the latrophilin adhesion GPCRs to activate subsequent cellular signaling events and vascular remodeling *in vivo,* thus connecting time scales from seconds to hours to weeks/months. Previous studies showed that laminar flow induces an increase in plasma membrane fluidity associated with depletion of membrane cholesterol (Yamamoto & Ando, 2013, 2015; Yamamoto *et al.*, 2020). Our data demonstrate that artificial cholesterol depletion activates flow-induced signals in a Lphn2-dependent manner, suggesting that latrophilins connect physical changes in the plasma membrane to cellular signaling. Latrophilins mediate activation of the PECAM-1/VE-cadherin/VEGFR pathway through both Gi2 and Gq/11, suggesting parallel or redundant downstream pathways. Consistent with these functional effects, FSS induces physical interactions (direct or indirect) between LPHN2-Gα and PECAM-1. However, LPHNs are not required for activation of VEGFRs by its ligand VEGF.\n",
      "\n",
      "Previous studies identified either Gi or Gq/11 as mediators of flow signaling(Chachisvilis *et al.*, 2006; Jung *et al.*, 2012; Wang *et al.*, 2015). It seems plausible that, depending on EC subtype or experimental conditions, either Gi or Gq/11 predominated, which concealed their functional redundancy. Previous studies also identified other GPCRs as mediators of EC shear stress responses, via direct or indirect mechanisms, as well as making claims for GPCR-independent activation of G proteins(Dela Paz *et al.*, 2017). GPCRs implicated in shear stress signaling include bradykinin receptor B2(Chachisvilis *et al.*, 2006), ATP receptor P2Y2(Albarran-Juarez *et al.*, 2018; Wang *et al.*, 2015), sphingosine phosphate receptor S1PR1(Jung *et al.*, 2012), the apelin receptor APJ(Strohbach *et al*, 2018) and GPR68(Xu *et al.*, 2018). An indirect association of Gq/11 with PECAM1 was also reported, which dissociated upon application of shear stress(dela Paz *et al*, 2014). Differences in EC location or subtype likely account for some of these effects. For example, GPR68 is expressed specifically in small resistance arteries where it controls vasodilation but had no reported effect on development or patterning(Xu *et al.*, 2018). P2Y2 was also reported to affect vasodilation and blood pressure without reported effects on development or patterning. It seems likely that differences in local expression may thus explain some of the apparent multiplicity, though additional variables such as modulation by other inputs and coupling to different downstream effectors may also contribute. Understanding this GPCR-Gα protein signaling network is therefore an important though complex direction for future work. The novel pulldown assay and GPCR-Gα protein affinity purification protocol developed here may facilitate such efforts.\n",
      "\n",
      "It is also noteworthy that latrophilins are required only for activation of signaling events including VEGFR2 by flow, but not for activation of VEGFR2 by its ligand, VEGF. PECAM-1 first appears in vertebrates but has multiple other functions in leukocyte transmigration and activation, platelet reactivity and angiogenesis that are independent of flow(Privratsky *et al*, 2010). Together, these findings argue that latrophilins, in essence, confer flow sensitivity to pre-existing pathways controlled by ligand availability.\n",
      "\n",
      "Lphn2 mutation in zebrafish and endothelial knockout in mice caused multiple vascular defects that implicate FSS mechanotransduction, including reduced EC alignment in the direction of flow, altered junctional morphology and permeability, smaller artery diameters and defects in arterialization in the hindlimb ischemia model. Lower capillary and small artery density in some vascular beds may also be attributable to defective FSS sensing, as FSS regulates angiogenic sprouting (Galie *et al.*, 2014) and stabilizes nascent vessels once blood begins to flow (Campinho *et al*, 2020). We note, however, that flow-independent functional roles for Lphn2 in ECs have been observed (Camillo *et al*, 2021) and likely contribute as well.\n",
      "\n",
      "These results link together many observations to provide a comprehensive model for EC responses to FSS (Appendix fig. S11). These findings are of further interest in light of human genetic data that strongly link the *Adgrl2* gene to vascular disease (Appendix fig. S10). However, the identified polymorphisms reside in regulatory or intronic regions, thus, causal connections to vascular disease are not obvious. Intronic polymorphisms can regulate gene expression but could also be in linkage disequilibrium with causal SNPs that are not yet identified. More detailed population genetic analyses are required to address these issues.\n",
      "\n",
      "These findings open many important questions for future work. Structural insights into latrophilin activation are needed, as are insights into the contribution of the large extracellular domain. The parallel roles of Gi2 and Gq/11 and downstream effectors remain to be understood. Thorough analysis of latrophilin functions in vascular development and physiology remains to be done. Lastly, combined human genetic and structure-function studies are needed to elucidate the contribution of Lphn2 variants to human cardiovascular disease.\n",
      "\n",
      "**Experimental Procedures**\n",
      "\n",
      "**Antibodies**\n",
      "\n",
      "Primary antibodies used in this study:\n",
      "\n",
      "| Epitope | Vendor | Catalog # |\n",
      "| --- | --- | --- |\n",
      "| Phospho-Src | Cell signaling | 6943 |\n",
      "| Src | Cell signaling | 2109 |\n",
      "| Gi | NewEast Biosciences | 26003 |\n",
      "| Gq | NewEast Biosciences | 26060 |\n",
      "| BD Bioscience | 612705 |\n",
      "| Gs | NewEast Biosciences | 26006 |\n",
      "| Phospho-VEGFR2(Tyr1175) | Cell signaling | 2478 |\n",
      "| Phospho-VEGFR3 (Tyr1054/Tyr1059) | Invitrogen | 44-1047G |\n",
      "| VEGFR2 | Cell signaling | 2479 |\n",
      "| Phospho-Akt (Ser473) | Cell signaling | 9271 |\n",
      "| VE-Cadherin | Santa Cruz | sc-6458 |\n",
      "| LPHN2 | Invitrogen | PA5-65359 |\n",
      "| RFP | Antibodies-Online | ABIN129578 |\n",
      "| alpha-smooth muscle actin | Invitrogen | 50-9760-82 |\n",
      "| alpha-smooth muscle actin-Cy3-conjugated | Sigma | C6198 |\n",
      "| ZO-1 | Invitrogen | 61-7300 |\n",
      "| beta-catenin | Cell signaling | 9562 |\n",
      "| GluGlu | BioLegend | MMS-115P |\n",
      "| HA | BioLegend | 901501 |\n",
      "| GFP | Abcam | ab13970 |\n",
      "| Santa Cruz | sc-9996 |\n",
      "| GST | Cell signaling | 2625 |\n",
      "| tubulin | Invitrogen | 62204 |\n",
      "| actin | Santa Cruz | sc-8432 |\n",
      "| PECAM1 | Kind gift from Dr. Peter Newman |  |\n",
      "| BD Bioscience | 557355 |\n",
      "| GM130 | BD Bioscience | 610823 |\n",
      "\n",
      "**Cell Culture**\n",
      "\n",
      "Primary HUVECs were obtained from the Yale Vascular Biology and Therapeutics core facility. Each batch is composed of cells pooled from three donors. Cells were cultured in M199 (Gibco: 11150-059) supplemented with 20% FBS, 1x Penicillin-Streptomycin (Gibco: 15140-122), 60 µg/ml heparin (Sigma: H3393), and endothelial growth cell supplement (hereafter, complete medium). HUVECs used for experiments were between passages 3 and 6.\n",
      "\n",
      "**Shear stress**\n",
      "\n",
      "For generating protein lysates for pulldown assays, HUVECs were seeded on tissue culture-treated plastic slides coated with 10 µg/ml fibronectin for 1 hour at 37˚C and then grown to confluence in HUVEC complete media. For short-term stimulation with shear stress, cells were starved overnight in M199 medium with 2% FBS and 1:10 of ECGS or for 30 minutes in M199 medium containing 0.2 % BSA. These slides were set in parallel flow chambers and shear stress applied as described(Frangos *et al*, 1988).\n",
      "\n",
      "For imaging cells under shear stress, HUVECs were plated in HUVEC complete medium on glass bottom 6-well plates coated with 10 µg/ml fibronectin and shear stress applied on an orbital shaker at 150rpm for 24 hours, leading to uniaxial pulsatile shear in the peripheral region and multiaxial shear in the middle of the well, as previously described(Arshad *et al*, 2021). Cells in the outer region, 0.7-0.9 cm from the center of each well were examined for Golgi orientation and endothelial alignment.\n",
      "\n",
      "**Image analysis**\n",
      "\n",
      "Cell orientation was calculated by taking the masks of the cell nuclei determined by Hoechst images, fitting them as an ellipse, and determining the angle between flow direction and the major axis of the ellipse. Analyzed results were visualized as histograms showing the percent of cells within each 10° of the direction of flow or as quantification of aligned cells with nuclei whose major axis were within 0-30 degrees to flow direction as indicated in the figure legends. Eccentricity for cell alignment was calculated based on first eccentricity of ellipse following the equation below.\n",
      "\n",
      "(a: major axis, b:minor axis)\n",
      "\n",
      "Junctional linearization index was calculated as ratio of perimeter of the cell to perimeter of its convex hull following the equation below:­\n",
      "\n",
      "**Data display**\n",
      "\n",
      "Quantified data are displayed as means ± standard error (SEM) in which the indicated replicates are independent experiments. F test was conducted to check equal variance.\n",
      "\n",
      "**Lentiviral transduction**\n",
      "\n",
      "Lenti-X 293T cells (Clontech, 632180) were cultured for at least 24 hours in DMEM supplemented with 10% FBS and lacking antibiotics, then transfected with lentiviral plasmids encoding the gene of interest and packaging plasmids (Addgene: 12259 and 12260) using Lipofectamine 2000 (Thermo Fisher Scientific: 11668-019) following the manufacturer’s protocols with Opti-MEM medium. Conditioned media from these cultures were collected 48 hours later, sterilized through 0.22µm filters and added to HUVECs together with 8µg/ml of polybrene (Sigma: 107689). After 24 hours, cells were switched to complete medium for 48 hours.\n",
      "\n",
      "**siRNA transfection**\n",
      "\n",
      "HUVECs were cultured in EGMTM-2 Endothelial Cell Growth Medium-2 BulletKitTM\n",
      "\n",
      "(Lonza: CC-3156 and CC-4176) for 24 hours before transfecting with RNAiMax (Thermo Fisher Scientific: 13778-150) with 20nM siRNA in Opti-MEM (Gibco: 31985-070) according to the manufacturer’s instructions. After 6 hours, cells were switched to EGM-2 medium and used for experiments 2-3 days later. Gα protein siRNAs were custom designed based on previous publications(Grzelinski *et al*, 2010; Krumins & Gilman, 2006; Ngai *et al*, 2008). For latrophilin knockdown, ON-TARGET plus Smartpool siRNAs from Dharmacon against human LPHN1 (L-005650-00-0005), LPHN2 (L-005651-00-0005), LPHN3 (L-005652-00-0005) were used.\n",
      "\n",
      "**Western Blotting**\n",
      "\n",
      "HUVECs were washed with PBS and extracted in Laemmli sample buffer. Samples were separated by SDS-PAGE and transferred onto nitrocellulose membranes. Membranes were blocked with 5% milk in TBS-T and probed with primary antibodies at 4˚C for overnight. The targeting proteins were visualized by HRP-conjugated secondary antibodies and subsequent HRP-luminol reaction.\n",
      "\n",
      "**Immunofluorescence**\n",
      "\n",
      "HUVECs were washed with PBS, fixed for 10 minutes with 3.7% PFA in PBS. Following fixation, cells were permeabilized with 0.5% Triton X-100 in PBS for 10 minutes and then incubated with 3% BSA in PBS for 30 minutes for blocking. Cells were washed with PBS after blocking and were incubated Alexa488-Phalloidin and Hoechst for 1 hour, then washed 4 times with PBS and mounted. Images were captured with 20x or 60x objective on a PerkinElmer spinning disk confocal microscope or a Leica Sp8 confocal microscope with the Leica Application Suite (LAS) software. Cell alignment was determined as described previously(Baeyens *et al*, 2014).\n",
      "\n",
      "**Reverse transcription and quantitative PCR**\n",
      "\n",
      "RNA was isolated from HUVECs using RNeasy kit according to the manufacturer’s instructions and quantified using a nanodrop spectrophotometer. Following cDNA synthesis using Bio-Rad iScript kit, RT-PCR was performed as follows. Each PCR reaction contains 42 two-step amplification cycles consisting of: (a) denaturation at 95˚C for 30s, and (b) annealing and extension at 60˚C for 30s. The amplification curve was collected, and the relative transcript level of the target mRNA in each sample was calculated by normalization of Ct values to the reference mRNA (GAPDH). Primer sequences used for RT-PCR were as shown in Table 1.\n",
      "\n",
      "**Gα protein pulldown assay**\n",
      "\n",
      "BL21 *E.coli* cells were transformed with constructs expressing GST-tagged GINIP protein or GST-tagged GRK2 N-terminal domain. Cells were incubated in terrific broth ([http://cshprotocols.cshlp.org/content/2015/9/pdb.rec085894.full](http://cshprotocols.cshlp.org/content/2015/9/pdb.rec085894.full?rss=1)) and protein expression induced by addition of 0.5µg/ml IPTG. Cells were collected after 8 hours, lysed and GST- proteins collected on Glutathione-conjugated beads. Beads were washed 4 times, eluted with 50mM glutathione and proteins desalted on a gel filtration column. Aliquots of 20 µg aliquots were stored frozen and thawed shortly before use.\n",
      "\n",
      "For pulldown assays, HUVECs infected with lentivirus encoding GluGlu-tagged Gi mutant or lentivirus encoding both GluGlu-tagged Gq mutant and Ric8A were lysed in cell lysis buffer composed of 10mM Tris-HCl pH7.5, 150mM NaCl, 1% Triton X-100, 5mM DTT and additional 2µg/ml of GST-GINIP or GST-GRK2N, and the lysates were incubated for 10 minutes at 4˚C with gentle agitation with glutathione beads. Beads were washed three times with cell lysis buffer, isolated proteins were solubilized in SDS sample laemmli buffer, and were analyzed by western blotting.\n",
      "\n",
      "**Mass spectrometry screening for Gα binding receptors**\n",
      "\n",
      "To create a Gα protein variant that interacts more strongly with GPCR, a green fluorescent protein (GFP) tag was inserted for affinity purification, alongside a mutation of four alanine residues into helix α5 (ins4A), as this hinders GTP binding and maintains the association with GPCRs even in the presence of GTP. Additionally, for the extraction of GPCRs while preserving their configuration and lipid environment, a non-detergent nanodisc approach utilizing styrene-maleimide anhydride (SMA) copolymers was employed according to protocols described previously.\n",
      "\n",
      "In the coimmunoprecipitation process, GFP-Trap® nanobody beads were utilized to capture the interactions between GFP-tagged Gα protein variants and activated designer receptors exclusively activated by designer drugs (DREADDs). The resulting complexes were subjected to subsequent proteomic analysis to determine which GPCRs interacted preferentially with the Gi1(Q306K) variant following induction of fluid shear stress. Identified receptors were further analyzed by western blotting as described in the supplementary material.\n",
      "\n",
      "**Measurement of Golgi orientation**\n",
      "\n",
      "Following exposure to laminar shear stress, HUVECs were washed with PBS, fixed for 10 minutes with 3.7% PFA in PBS, permeabilized with 0.5% Triton X-100 in PBS for 10 minutes, blocked with 3% BSA in PBS for 30 minutes and then incubated with GM130 antibody (1:500, BD bioscience) at 4˚C for overnight. After washing, cells were incubated with Hoechst and Alexa647-conjugated secondary antibody at RT for 1 hour to visualize the nuclei and Golgi apparatus, respectively. Images were captured with 20x objective on a Leica Sp8 confocal microscope with the Leica Application Suite (LAS) software.\n",
      "\n",
      "Golgi polarization was calculated as a vector that connects the center of mass of the nucleus to the center of mass of the Golgi. The angle between this vector and the direction of flow was then determined, with 0° defined as against the flow. Oriana V4 software was used to calculate the angle mean and circular SD and to generate a rose plot for data visualization.\n",
      "\n",
      "***In vitro* permeability assay using the FITC-streptavidin/biotinylated-fibronectin**\n",
      "\n",
      "Slides/dishes were coated with biotinylated-Fibronectin at 0.25 mg/mL overnight at 4°C. HUVECs were seeded of 2×106 cells per well on glass-bottom 6-well plates. Cells were subjected to laminar shear stress by orbital shaking at 150 rpm for 24 hours.  Immediately after cessation of flow, FITC-Streptavidin (1:1000, Invitrogen) was added to the cells for 1 minute before  the  fixation  by  the  addition  of  3.7%  (v/v)  paraformaldehyde. Following   subsequent   washing   and   blocking, Alexa Fluor 488 Phalloidin (1:200; Invitrogen) was  added  to  stain  F-actin  filaments. Images were captured from 5 random fields and FITC-streptavidin staining quantified using ImageJ.\n",
      "\n",
      "***In vitro* endothelial 3D invasion assay under shear stress**\n",
      "\n",
      "In order to equalize the VEGF concentration in the collagen gel and endothelial culture medium, VEGF-189 (R&D systems, 8147-VE), a matrix-binding isoform of VEGF, was applied to the bottom of each well at a concentration of 50 ng/ml prior to adding collagen matrices. Rat tail collagen type I (Thermofisher scientific, A1048301) was employed to prepare collagen matrices at the concentration of 1.5 mg/ml with additional NaOH to facilitate the collagen matrix assembly by neutral pH. After thorough mixing, 1 ml of collagen I solution was placed in each well of a 12-well plate (Corning #3513) and polymerized at 37°C and 5% CO2 for 2 hours to overnight.\n",
      "\n",
      "To prepare endothelial cells, confluent HUVEC culture plates were washed with PBS, trypsinized, and counted. After pelleting, cell pellets were resuspended in EGM-2 medium (Lonza, #CC-3162) free of VEGF and heparin to establish a positive gradient of VEGF from cells to the bottom of the collagen gel, and 300,000 cells per well were placed on the collagen gel and incubated for 24 hours. Following incubation, plates were placed on an orbital shaker and ECs were exposed to orbital shaking at 150 rpm for 48 hours as mentioned above in shear stress section. Cells were then fixed in 4% PFA and permeabilized in PBS containing 0.5% Triton X-100. Nonspecific binding was blocked by incubation with PBS containing 3% BSA. Thereafter, cells were subjected to Alexa488-conjugated phalloidin (1:400 dilution; Thermofisher scientific: A12379) and Hoechst (1:1000 dilution; Invitrogen: H1399) for 2 hours to stain actin cytoskeleton and cell nuclei, respectively. All images were captured with the Nikon Sp8 confocal microscope and segregated into peripheral region (with uniaxial pulsatile shear) vs. static controls.\n",
      "\n",
      "**Zebrafish husbandry and handling**\n",
      "\n",
      "Zebrafish were housed and maintained at 28.5ºC in accordance with standard methods approved by the Yale University Institutional Animal Care and Use Committee (#2017-11473)(Lawrence, 2016).\n",
      "\n",
      "**Generation of LPHN/adgrl knockdown zebrafish using CRISPR/Cas9 ribonucleoproteins**\n",
      "\n",
      "Four zebrafish latrophilin paralogs were identified—*adgrl1a, adgrl2a, adgrl2b.1,* and *adgrl3.1*. Fused sgRNAs were generated targeting each paralog (see Table 2) by annealing locus-specific oligonucleotides to a common 5’ universal oligonucleotide and performing *in vitro* RNA transcription (AmpliScribe T7-Flash Kit, Lucigen, ASF3507)(Gagnon *et al*, 2014). An injection mix of 50 ng/µL each sgRNA, an equivalent concentration of Cas9 protein (TrueCut Cas9, Invitrogen, A36497), and either 1 µg/µL *silent heart/tnnt2a(Sehnert et al., 2002)* or standard control morpholino (GeneTools) was prepared. 1 nL of the mixture was injected into *Tg(kdrl:ras-mCherry, fli1a:nls-GFP)* zebrafish at the one cell stage (Parker Hannifin, Picospritzer III). CRISPR reagent efficacy was confirmed by T7 endonuclease assay (see Figure S5). Guides against adgrl2a and 2b.1 were combined to target all LPHN2 isoforms.\n",
      "\n",
      "**Immunostaining and imaging of ECs in the knockdown zebrafish**\n",
      "\n",
      "CRISPR/Cas9 RNP-injected embryos were either fixed at 48 hpf in 4% paraformaldehyde (Santa Cruz Biotechnology, SC-253236) and immunostained for GFP (1:300 chicken anti-GFP, abcam, ab13970), mCherry (1:300 rabbit anti-mCherry, abcam, ab167453), and ZO-1 (1:200 mouse anti-ZO-1, Invitrogen, 61-7300) with species-appropriate secondary antibodies (1:400 Invitrogen anti-chicken/-rabbit/-mouse Alexa 488/546/647, A-11039/A- 10040, A-31571) or imaged live. For fixed imaging, the larvae were processed as previously described(Sugden *et al.*, 2017). For live imaging, larvae were anesthetized by immersion in 600 µM tricaine methylsulfonate (Western Chemical, TRS5), and embedded in 1.5% low-melt agarose (BioRad, 1613106). For staining in detail, larvae were fixed overnight in 4% PFA at +4ºC and then transitioned into 100% methanol via a series of dilutions. Once in methanol, the larvae were stored at -20ºC overnight, then rehydrated the next day via the same dilution series. Larvae were permeabilized using incubation in acetone for 15 minutes at -20ºC and then by incubation in 20 µg/mL proteinase K for 20 minutes at 37ºC. Larvae were refined in 4% PFA for 15 minutes at room temperature and then incubated in blocking buffer (0.5% Triton X-100, 10% goat serum, 1% BSA, 0.01% sodium azide, and 1% DMSO in PBS) for 2 hours at room temperature. Larvae were incubated in primary antibody (1:400 chicken anti-GFP, abcam ab13970) overnight in blocking buffer at +4ºC. Larvae were thoroughly washed in PBS-Tx and incubated overnight at +4ºC in secondary antibody (1:400 goat anti-chicken Alexa488 A-11039, ThermoFisher) in blocking buffer. Following final washes in PBS-Tx, larvae were mounted and imaged. Intersegmental vessel and dorsal aorta diameter were then calculated using ImageJ. All imaging was performed using a Leica SP8 confocal microscope.\n",
      "\n",
      "**Immunostaining and imaging of mouse aorta**\n",
      "\n",
      "Aortae were perfusion-fixed in situ with 4% paraformaldehyde (overnight) prior to staining with primary beta-catenin antibody (Cell signaling: 9562) and fluorophore-conjugated secondary antibody (Invitrogen, donkey anti-Rabbit IgG (H+L) Highly Cross-Adsorbed Secondary Antibody, Alexa Fluor 647, A31573) in in Claudio buffer (1% FBS, 3% BSA, 0.5% Triton X-100, 0.01% Sodium deoxycholate, 0.02% sodium azide in PBS pH7.4)(Franco *et al*, 2013). Cells were stained with Hoechst (Invitrogen: H1399) to label nuclei. Images of stained vessels were acquired using a Leica SP8 confocal microscope with the Leica Application Suite (LAS) software. Endothelial cell alignment in aorta was quantified by measuring the eccentricity of the immunostaining of beta-catenin of the endothelial cells (>150 cells per field of view).\n",
      "\n",
      "**Extravasation of Evans blue dye for aortic vascular permeability**\n",
      "\n",
      "For aortic vascular permeability measurements, Evans blue dye (20 mg/ml in saline, 2.5 µl/g mouse body weight) (Sigma E2129) was injected retro-orbitally to LPHN2 endothelial cell-specific knockout and their control littermates at 8 weeks old. Thirty minutes post injection, animals were euthanized by an overdose of isoflurane and tissue samples including aorta were harvested and washed with PBS to remove extra fatty tissues. Dissected aorta was prepared *en face* and imaged with an Olympus stereo zoom microscope. To quantify blue color associated with extravasation of Evans blue dye, the original RGB images were converted into HSV color model, and the proportion of the blue regions was calculated as the ratio of pixels with hue angles of 120°-240° to the entire aorta.\n",
      "\n",
      "**Immunostaining and imaging of mouse muscle capillaries**\n",
      "\n",
      "Rectus femoris muscle from murine thigh were perfusion-fixed with 4% paraformaldehyde (overnight) and the middle part of the muscle were cryo-sectioned following embedding into OCT compound and stained with primary antibodies for SMA (fluorophore-conjugated) and for CD31 followed by fluorophore-conjugated secondary antibody (Invitrogen, donkey anti-Rabbit IgG (H+L) Highly Cross-Adsorbed Secondary Antibody, Alexa Fluor 647, A31573) in PBS containing 5% BSA. Hoechst (Invitrogen: H1399) was used to nuclear identification. Images of stained vessels were acquired using a Leica SP8 confocal microscope with the Leica Application Suite (LAS) software.\n",
      "\n",
      "**Hindlimb ischemia model**\n",
      "\n",
      "Mice were anesthetized and operated with a mixture of 1.5-2% isoflurane. The front hair of the right thigh was removed with Nair lotion and the local surgical area was disinfected. A 5 mm longitudinal skin incision was made in the right thigh. The distal femoral artery, proximal to the popliteal artery and saphenous artery, was explored and dissected. 6-0 silk sutures (Syneture) were double used for the ligation before excision of the arterial vessel bed between the distal end of the superficial epigastric artery and the trifurcation of the femoral artery into the descending genicular, popliteal, and saphenous branches. The ligated artery was cut between two sutures. The venous structures and accompanying peripheral nerves were kept intact. The overlying skin was closed with a 6-0 prolene (Ethicon) and disinfected. At the end of the procedure, Buprenorphine was administered subcutaneously at 0.05 mg/kg, once pre-emptively, then every 12 h for 72 h. The survival rate for this surgery was 100%.\n",
      "\n",
      "**Laser Doppler perfusion imaging**\n",
      "\n",
      "Hindlimb perfusion was assessed noninvasively in the plantar foot before, 1, 3, 7, 14, and 21 days after hindlimb ischemia by scanning laser-Doppler [model LDI2-IR modified for high resolution; Moor Instruments, Wilmington, DE]. The hindquarters were placed on the top of a heater pad during scanning to minimize variation in body temperature. Doppler perfusion of the plantar foot was assessed within anatomically defined regions of interest (ROIs; Moor Instruments). The ROI for the plantar foot consisted of the hind paw margins. Procedures for minimal preparation, scanning, and ROI selections have been described in detail previously(Zhuang *et al*, 2011). Low or no perfusion is displayed as dark blue, whereas the highest degree of perfusion is displayed as red. These images were quantitatively converted into histograms that represented the amount of blood flow on the x-axis and the number of pixels on the y-axis in the traced area. The average blood flow in each histogram was calculated and the LDI index was determined as the ratio of ischemic to non-ischemic hindlimb blood perfusion.\n",
      "\n",
      "**Volumetric micro-CT angiogram for quantitative arteriogenesis**\n",
      "\n",
      "After the mice were euthanized, the vasculature was flushed with 0.9% normal saline containing heparin (1000 IU/L) for 3 minutes, followed by 4% paraformaldehyde for 5 minutes at 100 mm Hg pressure. Fresh home-made 20% bismuth nanoparticles mixed in 5% gelatin were used as a micro-CT contrast agent and injected over 2 minutes with a syringe pump, as described before(Tirziu *et al.*, 2005). The mice were then immediately chilled on ice for more than 30 min and immersion-fixed in 2% paraformaldehyde overnight. The peripheral vasculature in the ischemic hindlimb and the contralateral normal hindlimb was imaged with a high-resolution micro-CT imaging system (GE eXplore Locus SP), set to a 0.007-mm effective detector pixel size. Microview software (GE Healthcare) was used for initial reconstruction. Advanced workstation with various software was used for 3D reconstruction and segmentation of the whole hindlimb at the 21 µm spatial resolution. Using the knee as the breakpoint, we re-sliced the 3D vasculature into 200 slices both in the thigh and in the calf regions. Quantification was performed by use of a NIH Image. The data were expressed as vascular segment numbers, representing the total number of vessels, of specified diameter, counted in 200 *z*-sections for the thigh region and another 200 *z*-sections for the calf region.\n",
      "\n",
      "**Mouse treadmill fatigue test**\n",
      "\n",
      "Mice were first acclimatized by staying on the motionless treadmill machine for 30 min on day 1 and then trained on the treadmill moving at 12 m/min for 30 min on day 2. Fatigue tests were then conducted on days 3 and 4.  Mice were placed on the treadmill and the shock rod was activated.  Treadmill speeds were increased as follows: 0 m/min, for 12 min to measure basal metabolism, 12 m/min, for1 min, then increasing at a rate of +1 m/min each 1 min until exhaustion.  Exhaustion (endpoint of treadmill cessation) was defined as the point at which mice continuously contacted with the shock rod for 5 continuous seconds or when mice stopped running 3 times within 5 seconds. Before each testing session, Oxymax software (Columbus Instruments, Columbus, OH, USA) and open circuit indirect calorimetry treadmills (Metabolic Modular Treadmill, Columbus Instruments, Columbus, OH) were calibrated and checked for hardware malfunctions according to manufacturer instructions.\n",
      "\n",
      "**Data availability**\n",
      "\n",
      "The datasets produced in this study are available in the following databases:\n",
      "\n",
      "All data: BioStudies S-BIAD928: https://www.ebi.ac.uk/biostudies/studies?query=S-BIAD928\n",
      "\n",
      "**Acknowledgements**\n",
      "\n",
      "This work was supported by NIH grants RO1 HL75092 and R01 HL151469 to MAS, NHLBI 5R01HL130246-05 to SN, and the Uehara memorial foundation postdoctoral fellowship and MEXT | Japan Society for the Promotion of Science (JSPS) Overseas Research Fellowships (2014, receipt number: 805) to KT. We thank Hamit Harun Dag (Istanbul University) for help of synthesizing SMA polymer; Shahid Mansuri (Yale University), Jeremy Herrera (University of Manchester) and David Knight (University of Manchester) for expert advice for mass spectrometry experiments; Didier Trono (École polytechnique fédérale de Lausanne) for lentiviral packaging plasmids pMD2.G (Addgene plasmid # 12259 ; http://n2t.net/addgene:12259 ; RRID:Addgene\\_12259) and psPAX2 (Addgene plasmid # 12260 ; http://n2t.net/addgene:12260 ; RRID:Addgene\\_12260); Bryan Roth (University of North Carolina) for pcDNA5/FRT-HA-hM3D(Gq) (Addgene plasmid # 45547 ; http://n2t.net/addgene:45547 ; RRID:Addgene\\_45547) and pcDNA5/FRT-HA-hM4D(Gi)(Addgene plasmid # 45548 ; http://n2t.net/addgene:45548 ; RRID:Addgene\\_45548); Aziz Moqrich (Institut de Biologie du Dévelopement de Marseille) for GST-GINIP plasmid. We are grateful to Peter Newman (Medical College of Wisconsin) for the kind gift of PECAM-1 antibody and helpful discussions. Schematic models are inspired from BioRender templates (2022).\n",
      "\n",
      "**Disclosure & competing interests statement**\n",
      "\n",
      "The authors declare they have no conflict of interest.\n",
      "\n",
      "**References**\n",
      "\n",
      "Albarran-Juarez J, Iring A, Wang S, Joseph S, Grimm M, Strilic B, Wettschureck N, Althoff TF, Offermanns S (2018) Piezo1 and G(q)/G(11) promote endothelial inflammation depending on flow pattern and integrin activation. *J Exp Med* 215: 2655-2672\n",
      "\n",
      "Arshad M, Ghim M, Mohamied Y, Sherwin SJ, Weinberg PD (2021) Endothelial cells do not align with the mean wall shear stress vector. *J R Soc Interface* 18: 20200772\n",
      "\n",
      "Bae H, Cabrera-Vera TM, Depree KM, Graber SG, Hamm HE (1999) Two amino acids within the alpha4 helix of Galphai1 mediate coupling with 5-hydroxytryptamine1B receptors. *J Biol Chem* 274: 14963-14971\n",
      "\n",
      "Baeyens N, Bandyopadhyay C, Coon BG, Yun S, Schwartz MA (2016) Endothelial fluid shear stress sensing in vascular health and disease. *J Clin Invest* 126: 821-828\n",
      "\n",
      "Baeyens N, Mulligan-Kehoe MJ, Corti F, Simon DD, Ross TD, Rhodes JM, Wang TZ, Mejean CO, Simons M, Humphrey J *et al* (2014) Syndecan 4 is required for endothelial alignment in flow and atheroprotective signaling. *Proc Natl Acad Sci U S A* 111: 17308-17313\n",
      "\n",
      "Butler PJ, Norwich G, Weinbaum S, Chien S (2001) Shear stress induces a time- and position-dependent increase in endothelial cell membrane fluidity. *Am J Physiol Cell Physiol* 280: C962-969\n",
      "\n",
      "Camillo C, Facchinello N, Villari G, Mana G, Gioelli N, Sandri C, Astone M, Tortarolo D, Clapero F, Gays D *et al* (2021) LPHN2 inhibits vascular permeability by differential control of endothelial cell adhesion. *J Cell Biol* 220\n",
      "\n",
      "Campinho P, Vilfan A, Vermot J (2020) Blood Flow Forces in Shaping the Vascular System: A Focus on Endothelial Cell Behavior. *Front Physiol* 11: 552\n",
      "\n",
      "Chachisvilis M, Zhang YL, Frangos JA (2006) G protein-coupled receptors sense fluid shear stress in endothelial cells. *Proc Natl Acad Sci U S A* 103: 15463-15468\n",
      "\n",
      "Collins C, Guilluy C, Welch C, O'Brien ET, Hahn K, Superfine R, Burridge K, Tzima E (2012) Localized tensional forces on PECAM-1 elicit a global mechanotransduction response via the integrin-RhoA pathway. *Curr Biol* 22: 2087-2094\n",
      "\n",
      "Conway D, Schwartz MA (2012) Lessons from the endothelial junctional mechanosensory complex. *F1000 Biol Rep* 4: 1\n",
      "\n",
      "Conway DE, Breckenridge MT, Hinde E, Gratton E, Chen CS, Schwartz MA (2013) Fluid shear stress on endothelial cells modulates mechanical tension across VE-cadherin and PECAM-1. *Curr Biol* 23: 1024-1030\n",
      "\n",
      "Cornella N, Sancho J, Sitges-Serra A (2017) Short and Long-Term Outcomes After Surgical Procedures Lasting for More Than Six Hours. *Sci Rep* 7: 9221\n",
      "\n",
      "Dela Paz NG, Frangos JA (2019) Rapid flow-induced activation of Galphaq/11 is independent of Piezo1 activation. *Am J Physiol Cell Physiol* 316: C741-C752\n",
      "\n",
      "Dela Paz NG, Melchior B, Frangos JA (2017) Shear stress induces Galpha(q/11) activation independently of G protein-coupled receptor activation in endothelial cells. *Am J Physiol Cell Physiol* 312: C428-C437\n",
      "\n",
      "dela Paz NG, Melchior B, Shayo FY, Frangos JA (2014) Heparan sulfates mediate the interaction between platelet endothelial cell adhesion molecule-1 (PECAM-1) and the Galphaq/11 subunits of heterotrimeric G proteins. *J Biol Chem* 289: 7413-7424\n",
      "\n",
      "Erdogmus S, Storch U, Danner L, Becker J, Winter M, Ziegler N, Wirth A, Offermanns S, Hoffmann C, Gudermann T *et al* (2019) Helix 8 is the essential structural motif of mechanosensitive GPCRs. *Nat Commun* 10: 5784\n",
      "\n",
      "Fancher IS, Ahn SJ, Adamos C, Osborn C, Oh MJ, Fang Y, Reardon CA, Getz GS, Phillips SA, Levitan I (2018) Hypercholesterolemia-Induced Loss of Flow-Induced Vasodilation and Lesion Formation in Apolipoprotein E-Deficient Mice Critically Depend on Inwardly Rectifying K(+) Channels. *J Am Heart Assoc* 7\n",
      "\n",
      "Flock T, Hauser AS, Lund N, Gloriam DE, Balaji S, Babu MM (2017) Selectivity determinants of GPCR-G-protein binding. *Nature* 545: 317-322\n",
      "\n",
      "Franco CA, Blanc J, Parlakian A, Blanco R, Aspalter IM, Kazakova N, Diguet N, Mylonas E, Gao-Li J, Vaahtokari A *et al* (2013) SRF selectively controls tip cell invasive behavior in angiogenesis. *Development* 140: 2321-2333\n",
      "\n",
      "Frangos JA, McIntire LV, Eskin SG (1988) Shear stress induced stimulation of mammalian cell metabolism. *Biotechnol Bioeng* 32: 1053-1060\n",
      "\n",
      "Gagnon JA, Valen E, Thyme SB, Huang P, Akhmetova L, Pauli A, Montague TG, Zimmerman S, Richter C, Schier AF (2014) Efficient mutagenesis by Cas9 protein-mediated oligonucleotide insertion and large-scale assessment of single-guide RNAs. *PLoS One* 9: e98186\n",
      "\n",
      "Gaillard S, Lo Re L, Mantilleri A, Hepp R, Urien L, Malapert P, Alonso S, Deage M, Kambrun C, Landry M *et al* (2014) GINIP, a Galphai-interacting protein, functions as a key modulator of peripheral GABAB receptor-mediated analgesia. *Neuron* 84: 123-136\n",
      "\n",
      "Galie PA, Nguyen DH, Choi CK, Cohen DM, Janmey PA, Chen CS (2014) Fluid shear stress threshold regulates angiogenic sprouting. *Proc Natl Acad Sci U S A* 111: 7968-7973\n",
      "\n",
      "Givens C, Tzima E (2016) Endothelial Mechanosignaling: Does One Sensor Fit All? *Antioxid Redox Signal* 25: 373-388\n",
      "\n",
      "Grzelinski M, Pinkenburg O, Buch T, Gold M, Stohr S, Kalwa H, Gudermann T, Aigner A (2010) Critical role of G(alpha)12 and G(alpha)13 for human small cell lung cancer cell proliferation in vitro and tumor growth in vivo. *Clin Cancer Res* 16: 1402-1415\n",
      "\n",
      "Gudi S, Nolan JP, Frangos JA (1998) Modulation of GTPase activity of G proteins by fluid shear stress and phospholipid composition. *Proc Natl Acad Sci U S A* 95: 2515-2519\n",
      "\n",
      "Gudi SR, Clark CB, Frangos JA (1996) Fluid flow rapidly activates G proteins in human endothelial cells. Involvement of G proteins in mechanochemical signal transduction. *Circ Res* 79: 834-839\n",
      "\n",
      "Haidekker MA, L'Heureux N, Frangos JA (2000) Fluid shear stress increases membrane fluidity in endothelial cells: a study with DCVJ fluorescence. *Am J Physiol Heart Circ Physiol* 278: H1401-1406\n",
      "\n",
      "Humphrey JD, Schwartz MA (2021) Vascular Mechanobiology: Homeostasis, Adaptation, and Disease. *Annu Rev Biomed Eng* 23: 1-27\n",
      "\n",
      "Jung B, Obinata H, Galvani S, Mendelson K, Ding BS, Skoura A, Kinzel B, Brinkmann V, Rafii S, Evans T *et al* (2012) Flow-regulated endothelial S1P receptor-1 signaling sustains vascular development. *Dev Cell* 23: 600-610\n",
      "\n",
      "Kaya AI, Lokits AD, Gilbert JA, Iverson TM, Meiler J, Hamm HE (2016) A Conserved Hydrophobic Core in Galphai1 Regulates G Protein Activation and Release from Activated Receptor. *J Biol Chem* 291: 19674-19686\n",
      "\n",
      "Kroon J, Heemskerk N, Kalsbeek MJT, de Waard V, van Rijssel J, van Buul JD (2017) Flow-induced endothelial cell alignment requires the RhoGEF Trio as a scaffold protein to polarize active Rac1 distribution. *Mol Biol Cell* 28: 1745-1753\n",
      "\n",
      "Krumins AM, Gilman AG (2006) Targeted knockdown of G protein subunits selectively prevents receptor-mediated modulation of effectors and reveals complex changes in non-targeted signaling proteins. *J Biol Chem* 281: 10250-10262\n",
      "\n",
      "Lawrence C (2016) New frontiers for zebrafish management. *Methods Cell Biol* 135: 483-508\n",
      "\n",
      "Lee MJ, Evans M, Hla T (1996) The inducible G protein-coupled receptor edg-1 signals via the G(i)/mitogen-activated protein kinase pathway. *J Biol Chem* 271: 11272-11279\n",
      "\n",
      "Lee SC, Knowles TJ, Postis VL, Jamshad M, Parslow RA, Lin YP, Goldman A, Sridhar P, Overduin M, Muench SP *et al* (2016) A method for detergent-free isolation of membrane proteins in their local lipid environment. *Nat Protoc* 11: 1149-1162\n",
      "\n",
      "Liu SL, Sheng R, Jung JH, Wang L, Stec E, O'Connor MJ, Song S, Bikkavilli RK, Winn RA, Lee D *et al* (2017) Orthogonal lipid sensors identify transbilayer asymmetry of plasma membrane cholesterol. *Nat Chem Biol* 13: 268-274\n",
      "\n",
      "Lucitti JL, Jones EA, Huang C, Chen J, Fraser SE, Dickinson ME (2007) Vascular remodeling of the mouse yolk sac requires hemodynamic force. *Development* 134: 3317-3326\n",
      "\n",
      "Maleszewska M, Vanchin B, Harmsen MC, Krenning G (2016) The decrease in histone methyltransferase EZH2 in response to fluid shear stress alters endothelial gene expression and promotes quiescence. *Angiogenesis* 19: 9-24\n",
      "\n",
      "Medina R, Grishina G, Meloni EG, Muth TR, Berlot CH (1996) Localization of the effector-specifying regions of Gi2alpha and Gqalpha. *J Biol Chem* 271: 24720-24727\n",
      "\n",
      "Nazarko O, Kibrom A, Winkler J, Leon K, Stoveken H, Salzman G, Merdas K, Lu Y, Narkhede P, Tall G *et al* (2018) A Comprehensive Mutagenesis Screen of the Adhesion GPCR Latrophilin-1/ADGRL1. *iScience* 3: 264-278\n",
      "\n",
      "Ngai J, Methi T, Andressen KW, Levy FO, Torgersen KM, Vang T, Wettschureck N, Tasken K (2008) The heterotrimeric G-protein alpha-subunit Galphaq regulates TCR-mediated immune responses through an Lck-dependent pathway. *Eur J Immunol* 38: 3208-3218\n",
      "\n",
      "Papasergi MM, Patel BR, Tall GG (2015) The G protein alpha chaperone Ric-8 as a potential therapeutic target. *Mol Pharmacol* 87: 52-63\n",
      "\n",
      "Privratsky JR, Newman DK, Newman PJ (2010) PECAM-1: conflicts of interest in inflammation. *Life Sci* 87: 69-82\n",
      "\n",
      "Sando R, Jiang X, Sudhof TC (2019) Latrophilin GPCRs direct synapse specificity by coincident binding of FLRTs and teneurins. *Science* 363\n",
      "\n",
      "Sehnert AJ, Huq A, Weinstein BM, Walker C, Fishman M, Stainier DY (2002) Cardiac troponin T is essential in sarcomere assembly and cardiac contractility. *Nat Genet* 31: 106-110\n",
      "\n",
      "Strohbach A, Pennewitz M, Glaubitz M, Palankar R, Gross S, Lorenz F, Materzok I, Rong A, Busch MC, Felix SB *et al* (2018) The apelin receptor influences biomechanical and morphological properties of endothelial cells. *J Cell Physiol* 233: 6250-6261\n",
      "\n",
      "Sugden WW, Meissner R, Aegerter-Wilmsen T, Tsaryk R, Leonard EV, Bussmann J, Hamm MJ, Herzog W, Jin Y, Jakobsson L *et al* (2017) Endoglin controls blood vessel diameter through endothelial cell shape changes in response to haemodynamic cues. *Nat Cell Biol* 19: 653-665\n",
      "\n",
      "Tesmer VM, Kawano T, Shankaranarayanan A, Kozasa T, Tesmer JJ (2005) Snapshot of activated G proteins at the membrane: the Galphaq-GRK2-Gbetagamma complex. *Science* 310: 1686-1690\n",
      "\n",
      "Tirziu D, Moodie KL, Zhuang ZW, Singer K, Helisch A, Dunn JF, Li W, Singh J, Simons M (2005) Delayed arteriogenesis in hypercholesterolemic mice. *Circulation* 112: 2501-2509\n",
      "\n",
      "Tzima E, Irani-Tehrani M, Kiosses WB, Dejana E, Schultz DA, Engelhardt B, Cao G, DeLisser H, Schwartz MA (2005) A mechanosensory complex that mediates the endothelial cell response to fluid shear stress. *Nature* 437: 426-431\n",
      "\n",
      "Vizurraga A, Adhikari R, Yeung J, Yu M, Tall GG (2020) Mechanisms of adhesion G protein-coupled receptor activation. *J Biol Chem* 295: 14065-14083\n",
      "\n",
      "Vogel J, Niederer D, Jung G, Troidl K (2020) Exercise-Induced Vascular Adaptations under Artificially Versus Pathologically Reduced Blood Flow: A Focus Review with Special Emphasis on Arteriogenesis. *Cells* 9\n",
      "\n",
      "Wang S, Iring A, Strilic B, Albarran Juarez J, Kaur H, Troidl K, Tonack S, Burbiel JC, Muller CE, Fleming I *et al* (2015) P2Y(2) and Gq/G(1)(1) control blood pressure by mediating endothelial mechanotransduction. *J Clin Invest* 125: 3077-3086\n",
      "\n",
      "Wilson PT, Bourne HR (1995) Fatty acylation of alpha z. Effects of palmitoylation and myristoylation on alpha z signaling. *J Biol Chem* 270: 9667-9675\n",
      "\n",
      "Xu J, Mathur J, Vessieres E, Hammack S, Nonomura K, Favre J, Grimaud L, Petrus M, Francisco A, Li J *et al* (2018) GPR68 Senses Flow and Is Essential for Vascular Physiology. *Cell* 173: 762-775 e716\n",
      "\n",
      "Yamamoto K, Ando J (2013) Endothelial cell and model membranes respond to shear stress by rapidly decreasing the order of their lipid phases. *J Cell Sci* 126: 1227-1234\n",
      "\n",
      "Yamamoto K, Ando J (2015) Vascular endothelial cell membranes differentiate between stretch and shear stress through transitions in their lipid phases. *Am J Physiol Heart Circ Physiol* 309: H1178-1185\n",
      "\n",
      "Yamamoto K, Nogimori Y, Imamura H, Ando J (2020) Shear stress activates mitochondrial oxidative phosphorylation by reducing plasma membrane cholesterol in vascular endothelial cells. *Proc Natl Acad Sci U S A* 117: 33660-33667\n",
      "\n",
      "Yeh JC, Otte LA, Frangos JA (2008) Regulation of G protein-coupled receptor activities by the platelet-endothelial cell adhesion molecule, PECAM-1. *Biochemistry* 47: 9029-9039\n",
      "\n",
      "Zhou J, Li YS, Chien S (2014) Shear stress-initiated signaling and its regulation of endothelial function. *Arterioscler Thromb Vasc Biol* 34: 2191-2198\n",
      "\n",
      "Zhuang ZW, Shi J, Rhodes JM, Tsapakos MJ, Simons M (2011) Challenging the surgical rodent hindlimb ischemia model with the miniinterventional technique. *J Vasc Interv Radiol* 22: 1437-1446\n",
      "\n",
      "**Figure 1. Gα proteins specific for endothelial flow responses**\n",
      "\n",
      "**A**, Flow-induced endothelial alignment after Gα knockdown. HUVECs were subjected to fluid shear stress (FSS) at a rate of 12 dynes/cm2 for 16 hours and nuclear orientation quantified as histograms showing the percentage of cells within each 10° of the flow directions from 0° to 90° (see Methods) \\*\\*\\*\\*: p<0.0001; one-way ANOVA with Tukey’s multiple comparisons test. **B**, Src family kinase activation, quantified in **C**. n=5 for control, Gi knockdown, Gq11 knockdown and n=3 for simultaneous knockdown of Gi and Gq11. Values are means ± SEM. \\*\\*\\*\\*: p<0.0001, \\*\\*\\*: p<0.001; one-way ANOVA with Tukey multiple comparison test. **D**, Rescue of Gq/11 and Gi knockdown by re-expression of siRNA-resistant versions of the indicated proteins. \\*\\*\\*\\*: p<0.0001; one-way ANOVA with Tukey’s multiple comparison test. **E**, Amino acid sequences of Gαi1, Gαi2, and Gαi3 at the mutation sites of Gαi1 gain-of-function mutant. **F**, Rescue of Gq/11 and Gi knockdown with indicated Gα proteins. Each point corresponds to one measurement averaged from >500 cells. N=4. \\*\\*\\*\\*: p<0.0001; one-way ANOVA with Tukey’s multiple comparisons test. **G**, GINIP pulldown assay for activation of Gαi2 by FSS. N=3. Results quantified in **H**. \\*\\*: p=0.0185, *Student*’s t-test. **I,** GRK2N pulldown assay for activation of Gq. N=4, quantified in **J**. \\*: p<0.05, *Student*’s t-test. **K**, GINIP pulldown assay for FSS-induced activation of wild type and Q306K Gαi1. N = 4, quantified in **L**. \\*: p=0.0304, \\*\\*\\*: p=0.0017; *Student*’s t-test. **M**, Gi2 activation after PECAM-1 knockdown. HUVECs expressing GluGlu tagged Gi2 were transfected with scrambled siRNA or PECAM-1 siRNA, exposed to FSS, and Gi2 activation assayed as described above, quantified in **N**. Values are means ± SEM, normalized to input Gα protein levels. \\*\\*\\*: p<0.001; *Student*’s t-test, N=4.\n",
      "\n",
      "**Figure 2. Identifiation of the upstream flow-responsive GPCR**\n",
      "\n",
      "**A**, Strategy for identification of GPCRs that bind the gain-of-function Gi1Q306K mutant but not the wild type Gi1. **B**, Co-immunoprecipitation of Gi1Q306K(ins4A) with endogenous LPHN2 with and without FSS for 2 min. N=3, quantified in **C**. \\*: p=0.0374, *Student*’s t-test. **D**, Co-immunoprecipitation of LPHN2-GFP with the indicated Gα proteins containing internal GluGlu epitope tags. **E**, Gi2 pulldown assay after LPHN2 knockdown. N= 4. Results quantified in **F**. \\*\\*\\*: p<0.001, *Student*’s t-test.\n",
      "\n",
      "**Figure 3. Latrophilins regulate endothelial flow responses**\n",
      "\n",
      "**A**, HUVECs with or without latrophilin-2 knockdown were subjected to FSS for 16 h, then fixed and stained with Hoechst (nuclei), phalloidin (F-actin) and an antibody against VE-Cadherin. Scale bar: 100µm. **B**, Alignment of HUVECs (each bar = 10° increments) after knockdown of latrophilin isoforms was quantified as in Fig. 1 from >2000 cells/experiment, N=3. \\*\\*\\*\\*: p<0.0001; one-way ANOVA with Tukey’s multiple comparisons test. **C**, Localization of LPHN2-mClover3. Scale bar: 100µm. N=6. **D**, Gi1(Q306K) pulldown from ECs ± FSS for 5 min or for 24 hours, probed for PECAM-1. N=3, quantified in **E**. **F**, Gi1(Q306K) pulldown from ECs ± FSS for 5 min or for 24 hours probed for VE-cadherin. N=3, quantified in **G**. **H**, Activation of Src family kinases and Akt by FSS in HUVECs depleted for latrophilin isoforms. N=3-4, quantified in **I** & **J**. \\*: p=0.0184, \\*\\*: p<0.001; one-way ANOVA with Tukey’s multiple comparisons test. **K**, Alignment of LPHN2-depleted HUVECs rescued by re-expression of the indicated latrophilin isoforms. \\*\\*\\*\\*: p<0.0001; one-way ANOVA with Tukey’s multiple comparisons test. Quantification of >2000 cells for each condition, N=3. **L**, LPHN2 knockdown HUVECs were rescued by re-expression of the indicated LPHN2 mutants. \\*\\*\\*\\*: p<0.0001, one-way ANOVA with Tukey’s multiple comparisons test. Quantification of >500 cells for each condition.\n",
      "\n",
      "**Figure 4, Latrophilins in flow-mediated endothelial cell morphology *in vivo***\n",
      "\n",
      "**A**. Representative image of HUVECs transfected with scrambled siRNA or LPHN2 siRNA. Outlines of ECs from VE-cadherin staining are shown to next to each image. Scale Bar: 100µm. **B**, Quantification of junctional linearization index from **A**. n=30 for each condition. \\*\\*: p=0.0019, Statistics is done using two-way ANOVA with Sidak’s multiple comparison test. **C**, Dorsal aortas from 48hpf Tg(*kdrl:ras-mCherry*) WT or LPHN mutant zebrafish embryos stained for ZO-1. EC were outlined as in Methods. LPHN2 mutant indicates *adgrl2a*+*adgrl2b.1* sgRNAs. **D&E**, Quantification of cell size and eccentricity in embryos with latrophilin-2 CRISPR sgRNAs ± *silent heart* morpholinos (MO). N=6 per condition. \\*\\*\\*\\*: p<0.0001, \\*: p=0.0267, \\*\\*: p=0.0015, \\*\\*\\*: p=0.0008; one-way ANOVA with Tukey’s multiple comparisons test. **F**, ECs in the thoracic aorta in 18 weeks old mice at 10 weeks after tamoxifen injections, stained for β-catenin (left) and outlined (right). Data are representative of 6 mice for each condition. Scale Bar: 100µm. **G&H**, quantification of cell eccentricity and junction linearity in mouse aorta. N=4 per condition. \\*\\*: p=0.0013; \\*\\*\\*\\*: p<0.0001; *Student*’s t-test. **I**, Aortas from mice 30 min after Evans blue dye injection, thresholded images on the right. Fractional blue area quantified in **J**. n=3 for each condition. \\*: p=0.0419; *Student*’s t-test. **K,** *in vitro* endothelial permeability assay using FITC-streptavidin and biotin-conjugated fibronectin**.** Scale Bar: 100µm. **L,** Permeability was quantified by measuring FITC-streptavidin area per image field from **K.** n=5, \\*: p=0.0457, one-way ANOVA.\n",
      "\n",
      "**Figure 5. Cholesterol depletion activates Latrophilin-dependent signaling**\n",
      "\n",
      "**A,** ECs were treated with FSS or 5 mM methyl-β-cyclodextrin (mβCD) for 1 min then incubated with D4H-mClover3, rinsed and bound mClover3 quantified as described in Methods. N=27 for each condition. \\*\\*\\*\\*: p<0.0001; One-way ANOVA with Tukey multiple comparisons test. **B,** Phalloidin staining following short acute treatment of methyl-β-cyclodextrin (mβCD). Scale bar: 100µm. **C**, Co-immunoprecipitation of Gi1Q306K(ins4A) with endogenous Latrophilin-2 (LPHN2) after 5mM MβCD for 1min. **D**, Activation of Src family kinases and VEGFR2 after 5mM MβCD for 1min in HUVECs ± LPHN2 knockdown. Results quantified in **E** and **F**, respectively. Values are means ± SEM. N=5. \\*: p=0.0305, \\*\\*\\*: p=0.0002; one-way ANOVA with Tukey multiple comparison test. **G**, Model for junctional endothelial shear stress mechanotransduction.\n",
      "\n",
      "**Figure 6. Latrophilin-2 in flow-induced vascular remodeling**\n",
      "\n",
      "**A**, Representative images of ISVs from 48 hpf embryos with F(0)/mosaic depletions of latrophilin genes or controls. **B**, Quantification of diameters of all ISVs in **A**. n=15 per condition. \\*: p=0.0452, \\*\\*: p=0.0150; one-way ANOVA with Tukey’s multiple comparisons test. **C**, Quantification of diameters of arterial and venous ISVs at 72 hpf of zebrafish. n=6 per condition. \\*\\*: p=0.0013; *Student*’s t-test. **D**, Ratio of blood flow in the right ischemic vs left control foot by laser doppler, quantified in **E**. N=7-8 per condition. \\*#: p=0.0439, \\*: p=0.0135, \\*\\*: p=0.0028, ##: p=0.0056, #: p=0.0163; *Student*’s t-test. **F**, Micro-CT of arterial vasculature after surgery. Representative reconstructed micro-CT images from LPHN2 ECKO and WT littermates on day 22 after surgery, quantified in **G**. n=8 per condition. \\*\\*\\*\\*: p<0.0001, \\*\\*\\*: p=0.0007, \\*: p=0.0167; two-way ANOVA with Tukey’s multiple comparisons test. **H**, Rectus femoris muscle from thighs of 10 weeks old mice of LPHN2 ECKO or wild-type littermates, stained for CD31 and SMA to visualize the entire vasculature and arteries. Scale Bar: 100µm. **I**. Quantification of capillary density in the entire stitched thigh muscle images. n=3 for control and n=6 for Lphn2 ECKO. \\*\\*\\*\\*: p<0.0001; *Student*’s t-test. **J**, Schematic of mouse treadmill fatigue test. **K**, Maximum running during test. n=4 for control and n=5 for Lphn2 ECKO. \\*\\*:p=0.0082; *Student*’s t-test. **L**, Running distance for each mouse. n=4 for control and n=5 for Lphn2 ECKO. \\*\\*: p=0.0064; *Student*’s t-test. **M**, Oxygen consumption during treadmill fatigue test. \\*: p<0.0001; two-way ANOVA with Tukey’s multiple comparisons test. Control: N=5, LPHN2 ECKO: N=4. **N,** Representative 3D reconstructed images at 20° downward angle of ECs plated on top of collagen gels and sprouting under static or flow conditions. ECs were stained with phalloidin; pseudocolors indicate distance from the gel surface as per the color scale on the right. **O**, Histograms of cell area vs. distance from gel surface in **N**. **P**, Mean sprout length. N=4. \\*\\*\\*: p=0.0006; one-way ANOVA with Tukey’s multi comparison test.\n",
      "\n",
      "**Table 1: Primer sequences used for quantitative PCR in this study**\n",
      "\n",
      "| **Gene name** | **Forward (5'-3')** | **Reverse (5'-3')** |\n",
      "| --- | --- | --- |\n",
      "| Human LPHN1 | CAGTACGACTGTGTCCCCTACA | GCACCATGCGCCAGACTG |\n",
      "| Human LPHN2 | GTCCAATATGAATGTGTCCCTTACA | GCACCAAGCACCCGCCTT |\n",
      "| Human LPHN3 | GTGCAGTATGAATGTGTCCCTTACA | GCACCACGCCCCAGATTG |\n",
      "| Human KLF2 | CGGCAAGACCTACACCAAGA | TGGTAGGGCTTCTCACCTGT |\n",
      "| Human GAPDH | TGCACCACCAACTGCTTAGC | GGCATGGACTGTGGTCATGAG |\n",
      "|  |  |  |\n",
      "| zebrafish LPHN1a | CAGTATGAATGTGTGCCATACA | ACACCATGCCCCCGACTG |\n",
      "| zebrafish LPHN2a | GTCCAGTATGAATGTGTTCCCTACA | GCACCACGACCCCGCTTG |\n",
      "| zebrafish LPHN2b.1 | GTCCAGTATGAGTGTGTGCCATACA | TTGCACCAAGCTCCACTC |\n",
      "| zebrafish LPHN3.1 | GTTCAGTATGAGTGTGTGCCATACA | ACACCATGATCCAGCCTG |\n",
      "| zebrafish beta-actin | GATCTGGCATCACACCTTCTAC | TCTTCTCTCTGTTGGCTTTGG |\n",
      "\n",
      "**Table 2: Sequences of sgRNAs and morpholinos used in this study**\n",
      "\n",
      "|  | **Sequence** | **Target** | **Description** |\n",
      "| --- | --- | --- | --- |\n",
      "| **5'** | GCTGTACGAGTATGCTTCATGGG | *adgrl1a* | sgRNA targeting adgrl1a exon 4 |\n",
      "| **5'** | CGACGTATAAACTACCCCATCGG | *adgrl2a* | sgRNA targeting adgrl2a exon 4 |\n",
      "| **5'** | CCGTCTATGATAAACGCTCCGCC | *adgrl2b.1* | sgRNA targeting adgrl2b.1 exon 4 |\n",
      "| **5'** | AGTCCACGGCTGCGATGTATTGG | *adgrl3.1* | sgRNA targeting adgrl3.1 exon 4 |\n",
      "| **5'** | CATGTTTGCTCTGATCTGACACGCA | *tnnt2a* | morpholino targeting tnnt2a translation |\n",
      "| **5'** | CCTCTTACCTCAGTTACAATTTATA | none | control morpholino |\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "md = MarkItDown()\n",
    "result = md.convert(DOCX_FILES[0])\n",
    "print(result.text_content)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\n",
      "\n",
      "2\n",
      "\n",
      "3\n",
      "\n",
      "4\n",
      "\n",
      "5\n",
      "\n",
      "6\n",
      "\n",
      "7\n",
      "\n",
      "8\n",
      "\n",
      "9\n",
      "\n",
      "10\n",
      "\n",
      "11\n",
      "\n",
      "12\n",
      "\n",
      "13\n",
      "\n",
      "14\n",
      "\n",
      "15\n",
      "\n",
      "16\n",
      "\n",
      "17\n",
      "\n",
      "18\n",
      "\n",
      "19\n",
      "\n",
      "20\n",
      "\n",
      "21\n",
      "\n",
      "22\n",
      "\n",
      "23\n",
      "\n",
      "24\n",
      "\n",
      "25\n",
      "\n",
      "26\n",
      "\n",
      "Latrophilin-2 mediates fluid shear stress mechanotransduction at endothelial\n",
      "\n",
      "junctions\n",
      "\n",
      "Keiichiro  Tanaka1,7,*,  Minghao  Chen1,7,  Andrew  Prendergast1,  Zhenwu  Zhuang1,  Ali  Nasiri2,\n",
      "Divyesh  Joshi1,  Jared  Hintzen1,  Minhwan  Chung1,  Abhishek  Kumar1,  Arya  Mani1,  Anthony\n",
      "Koleske3, Jason Crawford4, Stefania Nicoli1 and Martin A. Schwartz1,5,6,*\n",
      "\n",
      "1  Yale  Cardiovascular  Research  Center,  Section  of  Cardiovascular  Medicine,  Department  of\n",
      "\n",
      "Internal Medicine, School of Medicine, Yale University, New Haven, CT 06511, USA\n",
      "2 Department of Internal Medicine\n",
      "3 Department of Molecular Biochemistry and Biophysics\n",
      "4 Department of Chemistry\n",
      "5 Department of Cell Biology\n",
      "6 Department of Biomedical Engineering\n",
      "7 These authors contributed equally.\n",
      "\n",
      "Running title: Latrophilins in shear stress sensing\n",
      "\n",
      "*Authors for correspondence:\n",
      "\n",
      "keiichiro.tanaka@yale.edu; martin.schwartz@yale.edu;\n",
      "\n",
      "\fAbstract\n",
      "\n",
      "Endothelial  cell  responses  to  fluid  shear  stress  from  blood  flow  are  crucial  for  vascular\n",
      "\n",
      "development,  function  and  disease.  A  complex  of  PECAM-1,  VE-cadherin,  VEGF  receptors\n",
      "\n",
      "(VEGFRs)  and  PlexinD1  located  at  cell-cell  junctions  mediates  many  of  these  events.  But\n",
      "\n",
      "available  evidence  suggests  that  another  mechanosensor  upstream  of  PECAM-1  initiates\n",
      "\n",
      "signaling. Hypothesizing that GPCR and Gα proteins may serve this role, we performed siRNA\n",
      "\n",
      "screening  of  Gα  subunits  and  found  that  Gαi2  and  Gαq/11  are  required  for  activation  of  the\n",
      "\n",
      "junctional  complex.  We  then  developed  a  new  activation  assay,  which  showed  that  these  G\n",
      "\n",
      "proteins  are  activated  by  flow.  We  next  mapped  the  Gα  residues  required  for  activation  and\n",
      "\n",
      "developed  an  affinity  purification  method  that  used  this  information  to  identify  latrophilin-2\n",
      "\n",
      "(Lphn-2/ADGRL2)  as  the  upstream  GPCR.    Latrophilin-2  is  required  for  all  PECAM-1\n",
      "\n",
      "downstream  events  tested.  In  both  mice  and  zebrafish,  latrophilin-2  is  required  for  flow-\n",
      "\n",
      "dependent  angiogenesis  and  artery  remodeling.  Furthermore,  endothelial  specific  knockout\n",
      "\n",
      "demonstrates that latrophilin plays a role in flow-dependent artery remodeling.  Human genetic\n",
      "\n",
      "data  reveal  a  correlation  between  the  latrophilin-2-encoding  Adgrl2  gene  and  cardiovascular\n",
      "\n",
      "disease. Together, these results  define a pathway that  connects  latrophilin-dependent G protein\n",
      "\n",
      "activation to subsequent endothelial signaling, vascular physiology and disease.\n",
      "\n",
      "Keywords\n",
      "\n",
      "Latrophilin, G protein-coupled receptor, fluid shear stress, vascular development, PECAM-1\n",
      "\n",
      "27\n",
      "\n",
      "28\n",
      "\n",
      "29\n",
      "\n",
      "30\n",
      "\n",
      "31\n",
      "\n",
      "32\n",
      "\n",
      "33\n",
      "\n",
      "34\n",
      "\n",
      "35\n",
      "\n",
      "36\n",
      "\n",
      "37\n",
      "\n",
      "38\n",
      "\n",
      "39\n",
      "\n",
      "40\n",
      "\n",
      "41\n",
      "\n",
      "42\n",
      "\n",
      "43\n",
      "\n",
      "44\n",
      "\n",
      "45\n",
      "\n",
      "46\n",
      "47\n",
      "\n",
      "\f48\n",
      "\n",
      "49\n",
      "\n",
      "50\n",
      "\n",
      "51\n",
      "\n",
      "52\n",
      "\n",
      "53\n",
      "\n",
      "54\n",
      "\n",
      "55\n",
      "\n",
      "56\n",
      "\n",
      "57\n",
      "\n",
      "58\n",
      "\n",
      "59\n",
      "\n",
      "60\n",
      "\n",
      "61\n",
      "\n",
      "62\n",
      "\n",
      "63\n",
      "\n",
      "64\n",
      "\n",
      "65\n",
      "\n",
      "66\n",
      "\n",
      "67\n",
      "\n",
      "68\n",
      "\n",
      "69\n",
      "\n",
      "70\n",
      "\n",
      "71\n",
      "\n",
      "72\n",
      "\n",
      "73\n",
      "\n",
      "74\n",
      "\n",
      "Introduction\n",
      "\n",
      "Fluid  shear  stress  (FSS)  from  blood  flow  is  critical  for  early  vascular  development  in\n",
      "\n",
      "mice(Lucitti et al, 2007) and zebrafish(Sugden et al, 2017), for homeostatic vessel remodeling in\n",
      "\n",
      "adults(Humphrey & Schwartz, 2021; Vogel et al, 2020), and in the initiation and progression of\n",
      "\n",
      "atherosclerosis(Baeyens  et  al,  2016;  Zhou  et  al,  2014).    A  complex  of  proteins  at  cell-cell\n",
      "\n",
      "junctions  consisting  of  PECAM-1,  VE-cadherin,  VEGFR2  and  3,  and  Plexin  D1  plays  an\n",
      "\n",
      "important  role  in  shear  stress  responses  including  vasodilation,  flow-dependent  vessel\n",
      "\n",
      "remodeling,  and  atherosclerosis(Conway  &  Schwartz,  2012;  Givens  &  Tzima,  2016).\n",
      "\n",
      "Stimulation of this junctional complex by flow triggers activation of Src-family kinases (SFKs)\n",
      "\n",
      "within  seconds,  resulting  in  ligand-independent  activation  of  VEGF  receptors  and  downstream\n",
      "\n",
      "events  including  endothelial  cell  (EC)  alignment  in  the  direction  of  flow(Collins  et  al,  2012).\n",
      "\n",
      "However, the mechanisms by which proteins located at cell-cell junctions can sense forces from\n",
      "\n",
      "shear stress exerted on the apical surface is unclear. Previous work suggests that activation of the\n",
      "\n",
      "junctional complex is not primary but rather requires an upstream event, mediated presumably by\n",
      "\n",
      "another mechanosensor (Conway et al, 2013).\n",
      "\n",
      "GPCR/Gα protein signaling is an obvious candidate for initiating signaling on this time scale\n",
      "\n",
      "(seconds).  GPCRs  are  a  large  family  of  cell  surface  receptors  that  play  a  crucial  role  in\n",
      "\n",
      "transducting  signals  across  cell  membranes.    GPCRs  undergo  conformational  changes  in\n",
      "\n",
      "response to ligand binding or other stimuli, after which the activated intracellular domain binds\n",
      "\n",
      "to  a  heterotrimeric  Gα  protein  complex  and  catalyzes  the  exchange  of  guanosine  diphosphate\n",
      "\n",
      "(GDP)  for  guanosine  triphosphate  (GTP).  Connections  between  GPCRs  and  shear  stress  have\n",
      "\n",
      "been reported(Chachisvilis et al, 2006; Dela Paz & Frangos, 2019; Erdogmus et al, 2019; Jung et\n",
      "\n",
      "al, 2012; Wang et al, 2015; Xu et al, 2018). However, the literature on Gα proteins and GPCRs\n",
      "\n",
      "in  shear  stress  signaling  is  highly  inconsistent(Chachisvilis  et  al.,  2006;  Dela  Paz  &  Frangos,\n",
      "\n",
      "2019;  Jung  et  al.,  2012;  Wang  et  al.,  2015).    For  instance,  it  has  been  demonstrated  that  flow\n",
      "\n",
      "applied to endothelial cells activates Gα proteins independent of an intervening GPCR(Dela Paz\n",
      "\n",
      "et  al,  2017;  Gudi  et  al,  1998).    It  has  also  been  reported  that  shear  stress  responses  related  to\n",
      "\n",
      "\fPECAM-1 and cell-cell junction require GPCRs but with different GPCRs implicated in different\n",
      "\n",
      "studies.    These  studies  include  reports  of  ligand-independent  activation  of  a  bradykinin\n",
      "\n",
      "receptor(Chachisvilis  et  al.,  2006;  Yeh  et  al,  2008)  and  the  sphingosine  1-phosphate  receptor\n",
      "\n",
      "S1PR1(Jung  et  al.,  2012).    The  purinergic  receptor  P2Y2  was  also  implicated,  in  this  case  via\n",
      "\n",
      "ATP  release  downstream  of  activation  of  the  mechanosensitive  ion  channel  Piezo1(Albarran-\n",
      "\n",
      "Juarez  et  al,  2018;  Wang  et  al.,  2015),  though  this  pathway  has  been  disputed(Dela  Paz  &\n",
      "\n",
      "Frangos,  2019).  The  orphan  receptor  GPR68  was  identified  as  a  shear  stress sensor  specific  to\n",
      "\n",
      "small arteries though the mechanism of activation, including ligand dependence or independence,\n",
      "\n",
      "is  unknown(Xu  et  al.,  2018).  These  studies  also  vary  in  the  Gα  proteins  implicated,  which\n",
      "\n",
      "included  different  Gαi  isoforms  (Gudi  et  al,  1996;  Jung  et  al.,  2012)  and  the  Gαq/11\n",
      "\n",
      "family(Albarran-Juarez et al., 2018; Chachisvilis et al., 2006; Wang et al., 2015).\n",
      "\n",
      "To resolve this issue, we took an unbiased approach starting from the Gα proteins using the\n",
      "\n",
      "dramatic  morphological  changes  in  endothelial  alignment  to  flow  as  the  initial  readout.  Our\n",
      "\n",
      "results showed that Gαi2 and Gαq/11 function in parallel, identified latrophilin-2 (LPHN2) as the\n",
      "\n",
      "key  upstream  GPCR  and  demonstrated  a  specific  requirement  for  LPHN2  in  flow-dependent\n",
      "\n",
      "remodeling in zebrafish and mice.\n",
      "\n",
      "75\n",
      "\n",
      "76\n",
      "\n",
      "77\n",
      "\n",
      "78\n",
      "\n",
      "79\n",
      "\n",
      "80\n",
      "\n",
      "81\n",
      "\n",
      "82\n",
      "\n",
      "83\n",
      "\n",
      "84\n",
      "\n",
      "85\n",
      "\n",
      "86\n",
      "\n",
      "87\n",
      "\n",
      "88\n",
      "\n",
      "89\n",
      "\n",
      "90\n",
      "91\n",
      "\n",
      "\f92\n",
      "\n",
      "93\n",
      "\n",
      "94\n",
      "\n",
      "95\n",
      "\n",
      "96\n",
      "\n",
      "97\n",
      "\n",
      "98\n",
      "\n",
      "99\n",
      "\n",
      "100\n",
      "\n",
      "101\n",
      "\n",
      "102\n",
      "\n",
      "103\n",
      "\n",
      "104\n",
      "\n",
      "105\n",
      "\n",
      "106\n",
      "\n",
      "107\n",
      "\n",
      "108\n",
      "\n",
      "109\n",
      "\n",
      "110\n",
      "\n",
      "111\n",
      "\n",
      "112\n",
      "\n",
      "113\n",
      "\n",
      "114\n",
      "\n",
      "115\n",
      "\n",
      "116\n",
      "\n",
      "117\n",
      "\n",
      "118\n",
      "\n",
      "Results\n",
      "\n",
      "Identification of Gα protein subunits required for flow sensing\n",
      "\n",
      "To  investigate  the  roles  of  GPCR  signaling  in  endothelial  flow  responses,  we  first\n",
      "\n",
      "transfected  human  umbilical  vein  endothelial  cells  (HUVECs)  with  short  interfering  RNAs\n",
      "\n",
      "(siRNAs)  to  suppress  Gs,  Gi,  Gq/11  and  G12/13  classes  of  Gα  proteins  and  assayed  EC\n",
      "\n",
      "alignment  in  flow  as  our  initial  readout.  Single  knockdowns  gave  either  weak  (Gi)  or  no  (Gs,\n",
      "\n",
      "G12/13, Gq/11) inhibition of EC alignment in flow (Fig.1A and Appendix fig. S1A-C). However,\n",
      "\n",
      "simultaneous  knockdown  of  Gi  and  Gq/11  completely  inhibited  alignment  (Fig.  1A),  whereas\n",
      "\n",
      "other combinations were ineffective (Appendix fig. S1D, S1E). Activation of SFKs is the earliest\n",
      "\n",
      "flow-responsive event mediated by PECAM-1, followed by VEGFRs and Akt(Tzima et al, 2005).\n",
      "\n",
      "These  events  were  similarly  blocked  by  knockdown  of  Gi  plus  Gq/11  but  not  by  single\n",
      "\n",
      "knockdowns  (Fig.  1B,  1C  and  Appendix  fig.  S1F).  Therefore,  Gi  and  Gq/11  act  in  parallel  to\n",
      "\n",
      "initiate PECAM-1-dependent flow signaling.\n",
      "\n",
      "Rescue controls with siRNA-resistant Gα variants showed that siRNA-resistant Gq and Gi2\n",
      "\n",
      "restored  cell  alignment,  whereas  Gi1  and  Gi3  did  not  (Fig.  1D  and  Appendix  fig.  S2A,  S2B).\n",
      "\n",
      "This  unexpected  result  prompted  us  to  search  for  residues  that  are  similar  in  Gq  and  Gi2  but\n",
      "\n",
      "different in Gi1 and Gi3.  Two residues (using human Gi2 numbering) fit this pattern: D167 in\n",
      "\n",
      "Gi2 corresponds to N in Gi1/3; and K307 corresponds to Q306 in Gi1/3 (Fig. 1E).  Gi1(Q306K),\n",
      "\n",
      "but  not  Gi1(N166D),  rescued  EC  alignment  in  flow  (Fig.  1F  and  Appendix  fig.  S2C).\n",
      "\n",
      "Consistently,  Gi2  mutant  lacking  K307  (K307Q)  could  not  rescue  the  defects  on flow-induced\n",
      "\n",
      "alignment under the knockdown of Gi plus Gq/11 (Appendix fig. S1G and Appendix fig. S2D).\n",
      "\n",
      "Therefore, K307 in Gi2 distinguishes the Gα subunits participate in flow signaling.\n",
      "\n",
      "Flow-induced activation of specific Gα proteins\n",
      "\n",
      "K307  lies  in  the  region  that  specifies  coupling  with  GPCRs(Bae  et  al,  1999;  Flock  et  al,\n",
      "\n",
      "2017),  suggesting  that  K307  may  determine  Gα  activation.    However,  testing  this  hypothesis\n",
      "\n",
      "requires measurement of Gα activation in live cells.  Unfortunately, indirect Gα activation assays\n",
      "\n",
      "\f119\n",
      "\n",
      "120\n",
      "\n",
      "121\n",
      "\n",
      "122\n",
      "\n",
      "123\n",
      "\n",
      "124\n",
      "\n",
      "125\n",
      "\n",
      "126\n",
      "\n",
      "127\n",
      "\n",
      "128\n",
      "\n",
      "129\n",
      "\n",
      "130\n",
      "\n",
      "131\n",
      "\n",
      "132\n",
      "\n",
      "133\n",
      "\n",
      "134\n",
      "\n",
      "135\n",
      "\n",
      "136\n",
      "\n",
      "137\n",
      "\n",
      "138\n",
      "\n",
      "139\n",
      "\n",
      "140\n",
      "\n",
      "141\n",
      "\n",
      "142\n",
      "\n",
      "143\n",
      "\n",
      "144\n",
      "\n",
      "145\n",
      "\n",
      "using second messengers  such as cAMP or calcium are unreliable in the context of fluid shear\n",
      "\n",
      "stress  where  multiple  activating/suppressing  pathways  are  stimulated.  We  therefore  developed\n",
      "\n",
      "pulldown  assays  for  Gα  based  on  the  specific  binding  of  GTP-loaded  Gα  subunits  to  effector\n",
      "\n",
      "proteins,  namely  GINIP  for  Gi  and  GRK2  N-terminal  domain  for  Gq  (Appendix  fig.\n",
      "\n",
      "S3a)(Gaillard et al, 2014; Tesmer et al, 2005).  To facilitate these assays and distinguish specific\n",
      "\n",
      "Gα isoforms (especially Gi isoforms), we prepared versions containing an internal GluGlu (EE)\n",
      "\n",
      "epitope  tag  that  does  not  impair  function  (Appendix  fig.  S3B  and  refs(Medina  et  al,  1996;\n",
      "\n",
      "Wilson  &  Bourne,  1995)).    For  Gq,  to  enhance  pulldown  efficiency,  we  co-expressed  Ric8A,\n",
      "\n",
      "which stabilizes active Gq(Papasergi et al, 2015).  Active, but not inactive, Gi and Gq bound to\n",
      "\n",
      "these  effector  proteins,  while  when  the  artificial  GPCR  DREADD  was  expressed,  its  ligand\n",
      "\n",
      "clozapine-N-oxide (CNO) also activated Gα proteins (Appendix fig. S3C-S3F).  Importantly, this\n",
      "\n",
      "Gα pulldown assay showed not only rapid activation of Gi2 and Gq by FSS (Fig. 1G-J), but also\n",
      "\n",
      "differential  responses  of  Gi1  and  gain-of-function  Gi1(Q306K)  to  FSS  (Fig.  1K,  1L).    Gi\n",
      "\n",
      "activation  was  unaffected  by  PECAM-1  knockdown,  indicating  that  this  step  was  upstream  or\n",
      "\n",
      "independent of PECAM-1 (Fig. 1M, 1N).  Flow-mediated activation of specific Gα subunits thus\n",
      "\n",
      "corresponds to their requirement in EC flow responses.\n",
      "\n",
      "Identification of the upstream GPCR\n",
      "\n",
      "We  next  sought  to  identify  the  GPCR(s)  that  mediates  flow  activation  of  Gi2  and  Gq/11.\n",
      "\n",
      "Although  affinity  purification  is  the  obvious  choice,  rapid  release  of  Gα  proteins  from  GPCRs\n",
      "\n",
      "after  GTP  loading  limits  this  approach.    Starting  with  wild-type  and  its  Q306K  mutant,  we\n",
      "\n",
      "inserted  1)  an  internal  GFP  for  affinity  purification  and  2)  four  alanines  (hereafter  ins4A)  into\n",
      "\n",
      "helix  α5,  which  blocks  GTP  loading,  thereby  preventing  dissociation  from  GPCRs  even  in  the\n",
      "\n",
      "presence  of  GTP(Kaya  et  al,  2016)  (Fig.  2A).  Both  constructs  localized  mainly  to  the  plasma\n",
      "\n",
      "membrane,  as  did  wild-type  Gα  (Appendix  fig.  S4A).    We  further  employed  detergent-free\n",
      "\n",
      "nanodisc-forming  styrene-maleimide  anhydride\n",
      "\n",
      "(SMA)  copolymer\n",
      "\n",
      "to  stabilize  GPCR\n",
      "\n",
      "conformation within its native lipid environment(Lee et al, 2016).  This approach was validated\n",
      "\n",
      "\f146\n",
      "\n",
      "147\n",
      "\n",
      "148\n",
      "\n",
      "149\n",
      "\n",
      "150\n",
      "\n",
      "151\n",
      "\n",
      "152\n",
      "\n",
      "153\n",
      "\n",
      "154\n",
      "\n",
      "155\n",
      "\n",
      "156\n",
      "\n",
      "157\n",
      "\n",
      "158\n",
      "\n",
      "159\n",
      "\n",
      "160\n",
      "\n",
      "161\n",
      "\n",
      "162\n",
      "\n",
      "163\n",
      "\n",
      "164\n",
      "\n",
      "165\n",
      "\n",
      "166\n",
      "\n",
      "167\n",
      "\n",
      "168\n",
      "\n",
      "169\n",
      "\n",
      "170\n",
      "\n",
      "171\n",
      "\n",
      "172\n",
      "\n",
      "by  coimmunoprecipitation  using  GFP-Trap®  nanobody  beads  of  Gα  variants  with  DREADD\n",
      "\n",
      "GPCR, which showed increased association after receptor activation (Appendix fig. S4B, S4C).\n",
      "\n",
      "Proteomic  analysis  for  GPCRs  that  associated  with  Gi1(Q306K)  but  not  wild-type  Gi1  in\n",
      "\n",
      "response to FSS (protocol in Appendix fig. S4D), identified S1PR1 and ADGRL3 (Appendix fig.\n",
      "\n",
      "S4E).  However, depletion of S1PR1 did not block alignment in FSS (Appendix fig. S4F, S4G).\n",
      "\n",
      "Furthermore, S1PR1 reportedly activates Gi1 and Gi3 as well as Gi2(Lee et al, 1996), which is\n",
      "\n",
      "inconsistent  with  the  Gα  specificity  defined  above.    We  therefore  focused  on  ADGRL  family\n",
      "\n",
      "proteins, also called latrophilins (LPHNs).\n",
      "\n",
      "Latrophilins (1,2 and 3 in mammals) are adhesion-type GPCRs that bind to counterreceptors\n",
      "\n",
      "on neighboring  cells  and regulate neuronal synapses(Sando et al,  2019).   Although it has  been\n",
      "\n",
      "reported  that  different  types  of  endothelial  cells  have  different  latrophilin  isoforms,  LPHN2  is\n",
      "\n",
      "most  widely  expressed  and  is  the  overwhelming  major  isoform  in  HUVECs  (Appendix  figs\n",
      "\n",
      "S5A)(Maleszewska  et  al,  2016).    Consistent  with  the  proteomic  results,  Gi1(Q306K)(ins4A)\n",
      "\n",
      "associated  with  endogenous  LPHN2,  which  increased  upon  FSS  (Fig.  2B,  C).  This  result\n",
      "\n",
      "strongly suggests that  FSS activates  LPHN2 to  recruit Gα proteins.  Furthermore, Gi1(Q306K),\n",
      "\n",
      "Gi2, and Gq co-immunoprecipitated with LPHN2, whereas Gi1 did not, indicating that LPHN2\n",
      "\n",
      "specifically  bound  to  the  flow-responsive  Gα  proteins  defined  above  (Fig.  2D).    Importantly,\n",
      "\n",
      "LPHN2 knockdown blocked flow-mediated activation of Gi2 (Fig. 2E, 2F).  Together, these data\n",
      "\n",
      "provide  strong  evidence  that  latrophilin-2  mediates  Gα  activation  essential  for  endothelial\n",
      "\n",
      "alignment in response to flow.\n",
      "\n",
      "Latrophilins in EC flow signaling in vitro.\n",
      "\n",
      "Knockdown of LPHN2, but not LPHN1 or LPHN3, blocked EC alignment in flow (Fig. 3A,\n",
      "\n",
      "3B;  knockdown  confirmed  in  Appendix  fig.  S5B,  S5C).    LPHN2  knockdown  in  human  aortic\n",
      "\n",
      "ECs  similarly  blocked  flow-induced  alignment  (Appendix  fig.  S5D,  S5E).    In  line  with  these\n",
      "\n",
      "results,  LPHN2  knockdown  also  blocked  Golgi  apparatus  polarization  in  flow  (Appendix  fig.\n",
      "\n",
      "S5F,  S5G).  LPHN2  localized  to  cell-cell  contacts  (Fig.3C  and  Appendix  fig.  S6).  The  flow\n",
      "\n",
      "\f173\n",
      "\n",
      "174\n",
      "\n",
      "175\n",
      "\n",
      "176\n",
      "\n",
      "177\n",
      "\n",
      "178\n",
      "\n",
      "179\n",
      "\n",
      "180\n",
      "\n",
      "responsive  G1  mutant,  Gi1(Q306K),  also  associated  with  PECAM-1  following  FSS  (Fig.  3D).\n",
      "\n",
      "The  association  between  LPHN2-Gi1(Q306K)  complex  with  junctional  mechanosensory\n",
      "\n",
      "complex proteins PECAM-1 and VE-Cadherin was detected after 24 hours of flow (Fig. 3D-3G),\n",
      "\n",
      "indicating  a  persistent  interaction.  We  also  assessed  the  immediate  flow-dependent  signals.\n",
      "\n",
      "LPHN2 depletion strongly suppressed flow-mediated acute activation of SFKs and Akt (Fig 3H-\n",
      "\n",
      "3J). Thus,  LPHN2  colocalizes, physically associates  with and is  required for  signaling through\n",
      "\n",
      "the junctional complex. However, LPHN2 knockdown did not inhibit activation of VEGFR2 by\n",
      "\n",
      "VEGF-A,  indicating  that  LPHN2  functions  specifically  in  response  to  laminar  flow  (Appendix\n",
      "\n",
      "181\n",
      "\n",
      "fig. S7A, S7B).\n",
      "\n",
      "182\n",
      "\n",
      "183\n",
      "\n",
      "184\n",
      "\n",
      "185\n",
      "\n",
      "186\n",
      "\n",
      "187\n",
      "\n",
      "188\n",
      "\n",
      "189\n",
      "\n",
      "190\n",
      "\n",
      "191\n",
      "\n",
      "192\n",
      "\n",
      "193\n",
      "\n",
      "Among the 3 latrophilin isoforms in human genome, all rescued flow-mediated endothelial\n",
      "\n",
      "alignment  after  LPHN2  knockdown  (Fig.  3K,  expression  confirmed  in  Appendix  fig.  S8A),\n",
      "\n",
      "indicating  functional  equivalence;  the  observed  differences  are  therefore  due  to  differential\n",
      "\n",
      "expression.    EC  alignment  was  rescued  by  wild  type  (WT)  LPHN2  but  not  H1071A,  which  is\n",
      "\n",
      "defective  in  coupling  to  Gα  proteins(Nazarko  et  al,  2018)  (Fig.  3L,  expression  confirmed  in\n",
      "\n",
      "Appendix  fig.  S8B).    Adhesion-type  GPCRs  can  be  activated  by  internal  cleavage  by  the\n",
      "\n",
      "extracellular GAIN domain which exposures its Stachel peptide as a tethered agonist(Vizurraga\n",
      "\n",
      "et  al,  2020).  However,  the  autoproteolysis-defective  mutant  fully  rescued  EC  alignment  (see\n",
      "\n",
      "Fig.3L).  Helix-8  is  known  to  endow  some  GPCRs  with  mechanosensitivity  to  membrane\n",
      "\n",
      "stretch(Erdogmus  et  al.,  2019).  However,  a  LPHN2  helix-8  mutant  could  still  rescue  EC\n",
      "\n",
      "alignment in flow (Fig. 3L).  Taken together, these findings indicate that latrophilin regulates Gα\n",
      "\n",
      "activation  required  for  endothelial  flow  responses,  but  activation  requires  neither  the  Stachel\n",
      "\n",
      "194\n",
      "\n",
      "peptide nor helix-8.\n",
      "\n",
      "195\n",
      "\n",
      "196\n",
      "\n",
      "197\n",
      "\n",
      "198\n",
      "\n",
      "199\n",
      "\n",
      "Latrophilin-2 regulates flow-induced endothelial cell morphological changes in vivo\n",
      "\n",
      "Long-term laminar flow stabilizes EC junctions, promoting a linear, continuous morphology\n",
      "\n",
      "associated  with  barrier  function(Kroon  et  al,  2017)  (see  Fig.3A).    To  test  whether  the  role  of\n",
      "\n",
      "LPHN2  in  this  process,  we  analyzed  junctional  morphology  in  vitro  with  and  without  16h  of\n",
      "\n",
      "\f200\n",
      "\n",
      "201\n",
      "\n",
      "202\n",
      "\n",
      "203\n",
      "\n",
      "204\n",
      "\n",
      "205\n",
      "\n",
      "206\n",
      "\n",
      "207\n",
      "\n",
      "208\n",
      "\n",
      "209\n",
      "\n",
      "210\n",
      "\n",
      "211\n",
      "\n",
      "212\n",
      "\n",
      "213\n",
      "\n",
      "214\n",
      "\n",
      "215\n",
      "\n",
      "216\n",
      "\n",
      "217\n",
      "\n",
      "218\n",
      "\n",
      "219\n",
      "\n",
      "220\n",
      "\n",
      "221\n",
      "\n",
      "222\n",
      "\n",
      "223\n",
      "\n",
      "224\n",
      "\n",
      "225\n",
      "\n",
      "226\n",
      "\n",
      "flow.  In  control  cells,  shear  stress  shifted  junctions  from  an  irregular  morphology  to  linear\n",
      "\n",
      "junctions; in Lphn2 knockdown ECs, junctions were indistinguishable in the absence of flow but\n",
      "\n",
      "failed to linearize in flow (Fig. 4A, 4B), demonstrating that the effect is flow-specific.  To further\n",
      "\n",
      "examine contributions of blood flow/latrophilin-2 to ECs in vivo, zebrafish embryos are useful as\n",
      "\n",
      "a  developmental  system  that  is  amenable  to  manipulation  of  shear  stress  using  morpholinos\n",
      "\n",
      "against  cardiac  troponin  T  type  2a  (tnnt2a;  ‘silent  heart’)(Sehnert  et  al,  2002),  without  global\n",
      "\n",
      "adverse  effects.    We  employed  zebrafish  embryos  expressing  Tg(kdrl:ras-mCherry)  to  mark\n",
      "\n",
      "endothelial  cells(Sehnert  et  al.,  2002)  and  designed  sgRNAs  for  CRISPR-Cas9  targeting  all\n",
      "\n",
      "latrophilin isoforms  (adgrl1,  adgrl2a  plus  adgrl2b.1,  and  adgrl3).  Our sgRNAs  induced  >50%\n",
      "\n",
      "cleavage in the T7 endonuclease assays and reduced  mRNA levels through nonsense-mediated\n",
      "\n",
      "decay (Appendix fig. S9). Since Lphn2 is the major endothelial isoform, we performed CRISPR-\n",
      "\n",
      "mediated  F(0)/mosaic  knockout  of  adgrl2a+adgrl2b.1  (hereafter  Lphn2  mutation),  which\n",
      "\n",
      "inhibited elongation and enlargement of ECs in the dorsal aorta (Fig. 4C).  Injecting single cell\n",
      "\n",
      "embryos  with  silent  heart  morpholinos  at  a  dose  that  completely  blocked  cardiac  contractility\n",
      "\n",
      "similarly blocked the elongation of ECs. Furthermore, combining tnnt2a and Lphn2 mutation had\n",
      "\n",
      "no further effect, supporting a specific role for latrophilins in flow sensing (Fig. 4D, 4E).\n",
      "\n",
      "To address LPHN function in mammals, we crossed Lphn2flox/flox mice with CDH5-CreERT2\n",
      "\n",
      "mice (Lphn2 ECKO), which were injected with tamoxifen at 8 weeks of age and remained viable\n",
      "\n",
      "10 weeks later.  The descending thoracic aorta (a region of high laminar flow) of Lphn2 ECKO\n",
      "\n",
      "mice had poorly aligned ECs with irregular junctions (Fig. 4F-H).  Additionally, Lphn2 ECKO\n",
      "\n",
      "increased vascular permeability in the whole aorta, as assessed by Evans blue extravasation (Fig.\n",
      "\n",
      "4I, 4J). In vitro, flow for 24h increased HUVEC barrier function, which was blocked by LPHN2\n",
      "\n",
      "knockdown,  whereas  barrier  function  in  the  absence  of  flow  was  unaffected  (Fig.  4K,  4L).\n",
      "\n",
      "Latrophilins  are  therefore  flow-responsive  GPCRs  that  regulate  multiple  steps  in  PECAM1-\n",
      "\n",
      "dependent EC mechanotransduction.\n",
      "\n",
      "Plasma membrane fluidization initiates latrophilin signaling\n",
      "\n",
      "\f227\n",
      "\n",
      "228\n",
      "\n",
      "229\n",
      "\n",
      "230\n",
      "\n",
      "231\n",
      "\n",
      "232\n",
      "\n",
      "233\n",
      "\n",
      "234\n",
      "\n",
      "235\n",
      "\n",
      "236\n",
      "\n",
      "237\n",
      "\n",
      "238\n",
      "\n",
      "239\n",
      "\n",
      "240\n",
      "\n",
      "241\n",
      "\n",
      "We  next  considered how  FSS  activates  LPHN2.  As  mutation  of  the  autoproteolytic  GAIN\n",
      "\n",
      "domain,  which  is  speculated  to  be  mechanosensitive,  had  no  effect  (Figure  3i),  we  considered\n",
      "\n",
      "alternatives. FSS induces a rapid increase in plasma membrane fluidity, in some cases associated\n",
      "\n",
      "with  decreased  plasma  membrane  cholesterol(Butler  et  al,  2001;  Haidekker  et  al,  2000;\n",
      "\n",
      "Yamamoto  &  Ando,  2013;  Yamamoto  et  al,  2020).  As  GPCRs  can  respond  to  such\n",
      "\n",
      "changes(Vizurraga et al., 2020), HUVECs were treated with methyl-β-cyclodextrin (MβCD) to\n",
      "\n",
      "extract plasma membrane cholesterol.  Treatment with 5 mM MβCD for 1 min reduced plasma\n",
      "\n",
      "membrane cholesterol to the same extent as flow, assayed by binding of the cholesterol-specific\n",
      "\n",
      "probe  D4H-Clover3(Liu  et  al,  2017)  (Figure  5A).    No  discernable  effect  on  cytoskeletal\n",
      "\n",
      "organization was observed under these conditions (Figure 5B). MβCD treatment also induced Gα\n",
      "\n",
      "binding to LPHN2 (Figure 5C) and activated both SFKs and VEGFR2, which were blocked by\n",
      "\n",
      "LPHN2 knockdown (Figure 5D-F). Together with published data demonstrating that cholesterol\n",
      "\n",
      "supplementation blocks flow signaling to these and related pathways(Fancher et al, 2018; Tirziu\n",
      "\n",
      "et al, 2005; Yamamoto & Ando, 2013, 2015; Yamamoto et al., 2020), these results suggest that\n",
      "\n",
      "membrane fluidization  can trigger  LPHN2  GPCR activation and downstream signaling (Figure\n",
      "\n",
      "242\n",
      "\n",
      "5G).\n",
      "\n",
      "243\n",
      "\n",
      "244\n",
      "\n",
      "245\n",
      "\n",
      "246\n",
      "\n",
      "247\n",
      "\n",
      "248\n",
      "\n",
      "249\n",
      "\n",
      "250\n",
      "\n",
      "251\n",
      "\n",
      "252\n",
      "\n",
      "253\n",
      "\n",
      "Latrophilins in vascular remodeling in vivo\n",
      "\n",
      "To address roles of LPHN2 in flow-dependent vascular remodeling, we examined zebrafish\n",
      "\n",
      "embryos, where artery lumen diameters are determined by EC shear stress(Sugden et al., 2017).\n",
      "\n",
      "We  observed  that  intersegmental  vessels  (ISVs)  at  48  hours  post  fertilization  had  reduced\n",
      "\n",
      "diameters  after  both  latrophilin  knockout  and  blockade  of  blood  flow,  with  no  further  effect\n",
      "\n",
      "when  combined  (Fig.  6A,  6B).  At  72  hpf,  when  intersegmental  arteries  and  veins  are\n",
      "\n",
      "distinguishable, the decrease in diameter was observed only in arteries (Fig. 6C). LPHNs are thus\n",
      "\n",
      "required for effects of flow on artery diameter.\n",
      "\n",
      "These  results  prompted  us  to  examine  flow-dependent  vascular  remodeling  in  mice.\n",
      "\n",
      "Femoral  artery  ligation  triggers  flow-driven  arteriogenesis  in  the  thigh  and  hypoxia-driven\n",
      "\n",
      "\f254\n",
      "\n",
      "angiogenesis  in  the  calf\n",
      "\n",
      "(Cornella  et  al,  2017)\n",
      "\n",
      ".  Following  surgery,  LPHN2  ECKO  mice  showed\n",
      "\n",
      "255\n",
      "\n",
      "256\n",
      "\n",
      "257\n",
      "\n",
      "258\n",
      "\n",
      "259\n",
      "\n",
      "260\n",
      "\n",
      "261\n",
      "\n",
      "262\n",
      "\n",
      "263\n",
      "\n",
      "264\n",
      "\n",
      "265\n",
      "\n",
      "266\n",
      "\n",
      "267\n",
      "\n",
      "268\n",
      "\n",
      "269\n",
      "\n",
      "lower blood flow in the foot  even at  day 1  and markedly  slower recovery than WT littermates\n",
      "\n",
      "(Fig.  6D,  6E).  To  understand  these  effects,  we  performed  micro-CT  of  vascular  casts  after\n",
      "\n",
      "injection of electron-dense bismuth microparticles that label arteries/arterioles but not capillaries\n",
      "\n",
      "due  to  particle  size.  Surgery  induced  an  increase  in  small  arteries  in  WT  mice  but  essentially\n",
      "\n",
      "none in LPHN2 ECKO mice (Fig. 6F, 6G). LPHN2 ECKO mice also had much lower density of\n",
      "\n",
      "small arteries even in the control leg, consistent with lower flow on day 1. Staining for PECAM\n",
      "\n",
      "to  examine  the  total  vasculature  in  leg  muscle  sections  revealed  decreased  capillary  density  as\n",
      "\n",
      "well,  even  at  baseline  (Fig.  6H,  6I).    LPHN2  ECKO  mice  thus  show  low  vascular  density  at\n",
      "\n",
      "baseline  and  greatly  reduced  flow-dependent  remodeling.  As  expected  from  the  low  vascular\n",
      "\n",
      "density, a treadmill fatigue test (Fig. 6J) LPHN2 ECKO mice showed markedly lower exercise\n",
      "\n",
      "capacity (Fig. 6K, 6L) and impaired oxygen consumption (Fig. 6M). LPHN2 ECKO thus results\n",
      "\n",
      "in vascular defects in vivo, at least in part due to defective EC flow sensing. To address whether\n",
      "\n",
      "these defects were related to shear stress sensing, we examined an in vitro system in which FSS\n",
      "\n",
      "enhances VEGF-induced sprouting into collagen gels(Galie et al, 2014). Enhanced sprouting was\n",
      "\n",
      "completely blocked by LPHN2 depletion with only a weak effect on basal migration (Fig.  6N-\n",
      "\n",
      "270\n",
      "\n",
      "6P).\n",
      "\n",
      "271\n",
      "\n",
      "272\n",
      "\n",
      "273\n",
      "\n",
      "274\n",
      "\n",
      "275\n",
      "\n",
      "276\n",
      "\n",
      "277\n",
      "\n",
      "278\n",
      "\n",
      "279\n",
      "\n",
      "280\n",
      "\n",
      "Genetic link to human disease.\n",
      "\n",
      "The  Cardiovascular  Disease  Knowledge  Portal  reported  four  SNPs  near  the  5’  end  of  the\n",
      "\n",
      "ADGRL2  gene  (Appendix  fig.  S10A)  that  exhibit  genome  wide  association  with  hypertension\n",
      "\n",
      "(Appendix  fig.  S10B).  All  4  are  in  close  proximity  to  each  other  and  are  in  linkage\n",
      "disequilibrium (D’: 0.9953, R2: 0.97). Two of the SNPs meet the significant threshold (p<5 x 10-\n",
      "8)  and  rs1195871  is  relatively  conserved  among  vertebrates.  Of  particular  interest,  the  intronic\n",
      "\n",
      "SNP  rs186892211  in  the  ADGRL2  locus  (Appendix  fig.  S10A)  is  associated  with  large-artery\n",
      "\n",
      "atherosclerosis  and  subsequent  ischemic  stroke  (TOAST  classification)  with  a  very  high  odds\n",
      "\n",
      "ratio (odds ratio 720; p< 3.134e-10, Appendix fig. S10B).\n",
      "\n",
      "\f281\n",
      "\n",
      "Discussion\n",
      "\n",
      "282\n",
      "\n",
      "283\n",
      "\n",
      "284\n",
      "\n",
      "285\n",
      "\n",
      "286\n",
      "\n",
      "287\n",
      "\n",
      "288\n",
      "\n",
      "289\n",
      "\n",
      "290\n",
      "\n",
      "291\n",
      "\n",
      "292\n",
      "\n",
      "293\n",
      "\n",
      "294\n",
      "\n",
      "295\n",
      "\n",
      "296\n",
      "\n",
      "297\n",
      "\n",
      "298\n",
      "\n",
      "299\n",
      "\n",
      "300\n",
      "\n",
      "301\n",
      "\n",
      "302\n",
      "\n",
      "303\n",
      "\n",
      "304\n",
      "\n",
      "305\n",
      "\n",
      "306\n",
      "\n",
      "307\n",
      "\n",
      "These  data  elucidate  a  pathway  in  which  fluid  shear  stress  acts  through  the  latrophilin\n",
      "\n",
      "adhesion GPCRs to activate subsequent cellular signaling events and vascular remodeling in vivo,\n",
      "\n",
      "thus  connecting  time  scales  from  seconds  to  hours  to  weeks/months.  Previous  studies  showed\n",
      "\n",
      "that laminar flow induces an increase in plasma membrane fluidity associated with depletion of\n",
      "\n",
      "membrane  cholesterol  (Yamamoto  &  Ando,  2013,  2015;  Yamamoto  et  al.,  2020).    Our  data\n",
      "\n",
      "demonstrate  that  artificial  cholesterol  depletion  activates  flow-induced  signals  in  a  Lphn2-\n",
      "\n",
      "dependent  manner,  suggesting  that  latrophilins  connect  physical  changes  in  the  plasma\n",
      "\n",
      "membrane  to  cellular  signaling.  Latrophilins  mediate  activation  of  the  PECAM-1/VE-\n",
      "\n",
      "cadherin/VEGFR  pathway  through  both  Gi2  and  Gq/11,  suggesting  parallel  or  redundant\n",
      "\n",
      "downstream  pathways.    Consistent  with  these  functional  effects,  FSS  induces  physical\n",
      "\n",
      "interactions (direct or indirect) between LPHN2-Gα and PECAM-1.  However, LPHNs are not\n",
      "\n",
      "required for activation of VEGFRs by its ligand VEGF.\n",
      "\n",
      "Previous studies identified either Gi or Gq/11 as mediators of flow signaling(Chachisvilis et\n",
      "\n",
      "al.,  2006;  Jung  et  al.,  2012;  Wang  et  al.,  2015).    It  seems  plausible  that,  depending  on  EC\n",
      "\n",
      "subtype  or  experimental  conditions,  either  Gi  or  Gq/11  predominated,  which  concealed  their\n",
      "\n",
      "functional  redundancy.  Previous  studies  also  identified  other  GPCRs  as  mediators  of  EC  shear\n",
      "\n",
      "stress  responses,  via  direct  or  indirect  mechanisms,  as  well  as  making  claims  for  GPCR-\n",
      "\n",
      "independent  activation  of  G  proteins(Dela  Paz  et  al.,  2017).  GPCRs  implicated  in  shear  stress\n",
      "\n",
      "signaling\n",
      "\n",
      "include  bradykinin\n",
      "\n",
      "receptor  B2(Chachisvilis  et  al.,  2006),  ATP\n",
      "\n",
      "receptor\n",
      "\n",
      "P2Y2(Albarran-Juarez  et  al.,  2018;  Wang  et  al.,  2015),  sphingosine  phosphate  receptor\n",
      "\n",
      "S1PR1(Jung et al., 2012), the apelin receptor APJ(Strohbach et al, 2018) and GPR68(Xu et al.,\n",
      "\n",
      "2018).  An  indirect  association  of  Gq/11  with  PECAM1  was  also  reported,  which  dissociated\n",
      "\n",
      "upon  application  of  shear  stress(dela  Paz  et  al,  2014).  Differences  in  EC  location  or  subtype\n",
      "\n",
      "likely account for some of these effects. For example, GPR68 is expressed specifically in small\n",
      "\n",
      "resistance  arteries  where  it  controls  vasodilation  but  had  no  reported  effect  on  development  or\n",
      "\n",
      "patterning(Xu  et  al.,  2018).  P2Y2  was  also  reported  to  affect  vasodilation  and  blood  pressure\n",
      "\n",
      "\f308\n",
      "\n",
      "309\n",
      "\n",
      "310\n",
      "\n",
      "311\n",
      "\n",
      "312\n",
      "\n",
      "313\n",
      "\n",
      "314\n",
      "\n",
      "315\n",
      "\n",
      "316\n",
      "\n",
      "317\n",
      "\n",
      "318\n",
      "\n",
      "319\n",
      "\n",
      "320\n",
      "\n",
      "321\n",
      "\n",
      "322\n",
      "\n",
      "323\n",
      "\n",
      "324\n",
      "\n",
      "325\n",
      "\n",
      "326\n",
      "\n",
      "327\n",
      "\n",
      "328\n",
      "\n",
      "329\n",
      "\n",
      "330\n",
      "\n",
      "331\n",
      "\n",
      "332\n",
      "\n",
      "333\n",
      "\n",
      "334\n",
      "\n",
      "without reported effects on development or patterning.  It seems likely that differences in local\n",
      "\n",
      "expression may thus explain some of the apparent multiplicity, though additional variables such\n",
      "\n",
      "as  modulation  by  other  inputs  and  coupling  to  different  downstream  effectors  may  also\n",
      "\n",
      "contribute.  Understanding  this  GPCR-Gα  protein  signaling  network  is  therefore  an  important\n",
      "\n",
      "though  complex  direction  for  future  work.    The  novel  pulldown  assay  and  GPCR-Gα  protein\n",
      "\n",
      "affinity purification protocol developed here may facilitate such efforts.\n",
      "\n",
      "It  is  also  noteworthy  that  latrophilins  are  required  only  for  activation  of  signaling  events\n",
      "\n",
      "including VEGFR2 by flow, but not for activation of VEGFR2 by its ligand, VEGF. PECAM-1\n",
      "\n",
      "first  appears  in  vertebrates  but  has  multiple  other  functions  in  leukocyte  transmigration  and\n",
      "\n",
      "activation,  platelet  reactivity  and  angiogenesis  that  are  independent  of  flow(Privratsky  et  al,\n",
      "\n",
      "2010). Together, these findings argue that latrophilins, in essence, confer flow sensitivity to pre-\n",
      "\n",
      "existing pathways controlled by ligand availability.\n",
      "\n",
      "Lphn2  mutation  in  zebrafish  and  endothelial  knockout  in  mice  caused  multiple  vascular\n",
      "\n",
      "defects  that  implicate  FSS  mechanotransduction,  including  reduced  EC  alignment  in  the\n",
      "\n",
      "direction of flow, altered junctional morphology and permeability, smaller artery diameters and\n",
      "\n",
      "defects  in  arterialization  in  the  hindlimb  ischemia  model.  Lower  capillary  and  small  artery\n",
      "\n",
      "density in some vascular beds may also be attributable to defective FSS sensing, as FSS regulates\n",
      "\n",
      "angiogenic sprouting (Galie et al., 2014) and stabilizes nascent vessels once blood begins to flow\n",
      "\n",
      "(Campinho et al, 2020). We note, however, that flow-independent functional roles for Lphn2 in\n",
      "\n",
      "ECs have been observed (Camillo et al, 2021) and likely contribute as well.\n",
      "\n",
      "These  results  link  together  many  observations  to  provide  a  comprehensive  model  for  EC\n",
      "\n",
      "responses  to  FSS  (Appendix  fig.  S11).  These  findings  are  of  further  interest  in  light  of  human\n",
      "\n",
      "genetic data that strongly link the Adgrl2 gene to vascular disease (Appendix fig. S10).  However,\n",
      "\n",
      "the identified polymorphisms reside in regulatory or intronic regions, thus, causal connections to\n",
      "\n",
      "vascular disease are not obvious. Intronic polymorphisms can regulate gene expression but could\n",
      "\n",
      "also  be  in  linkage  disequilibrium  with  causal  SNPs  that  are  not  yet  identified.  More  detailed\n",
      "\n",
      "population genetic analyses are required to address these issues.\n",
      "\n",
      "\fThese  findings  open  many  important  questions  for  future  work.    Structural  insights  into\n",
      "\n",
      "latrophilin  activation  are  needed,  as  are  insights  into  the  contribution  of  the  large  extracellular\n",
      "\n",
      "domain.   The parallel roles of Gi2 and Gq/11 and downstream effectors remain to be understood.\n",
      "\n",
      "Thorough analysis of latrophilin functions in vascular development and physiology remains to be\n",
      "\n",
      "done.  Lastly, combined human genetic and structure-function studies are needed to elucidate the\n",
      "\n",
      "contribution of Lphn2 variants to human cardiovascular disease.\n",
      "\n",
      "335\n",
      "\n",
      "336\n",
      "\n",
      "337\n",
      "\n",
      "338\n",
      "\n",
      "339\n",
      "\n",
      "340\n",
      "341\n",
      "\n",
      "\fExperimental Procedures\n",
      "\n",
      "342\n",
      "\n",
      "343\n",
      "\n",
      "344\n",
      "\n",
      "Antibodies\n",
      "\n",
      "345\n",
      "\n",
      "Primary antibodies used in this study:\n",
      "\n",
      "Epitope\n",
      "\n",
      "Phospho-Src\n",
      "Src\n",
      "Gi\n",
      "\n",
      "Gq\n",
      "\n",
      "Gs\n",
      "Phospho-VEGFR2(Tyr1175)\n",
      "Phospho-VEGFR3 (Tyr1054/Tyr1059)\n",
      "VEGFR2\n",
      "Phospho-Akt (Ser473)\n",
      "VE-Cadherin\n",
      "LPHN2\n",
      "RFP\n",
      "alpha-smooth muscle actin\n",
      "alpha-smooth muscle actin-Cy3-conjugated\n",
      "ZO-1\n",
      "beta-catenin\n",
      "GluGlu\n",
      "HA\n",
      "\n",
      "GFP\n",
      "\n",
      "GST\n",
      "tubulin\n",
      "actin\n",
      "\n",
      "PECAM1\n",
      "\n",
      "GM130\n",
      "\n",
      "346\n",
      "\n",
      "347\n",
      "\n",
      "348\n",
      "\n",
      "Vendor\n",
      "\n",
      "Cell signaling\n",
      "Cell signaling\n",
      "NewEast Biosciences\n",
      "NewEast Biosciences\n",
      "BD Bioscience\n",
      "NewEast Biosciences\n",
      "Cell signaling\n",
      "Invitrogen\n",
      "Cell signaling\n",
      "Cell signaling\n",
      "Santa Cruz\n",
      "Invitrogen\n",
      "Antibodies-Online\n",
      "Invitrogen\n",
      "Sigma\n",
      "Invitrogen\n",
      "Cell signaling\n",
      "BioLegend\n",
      "BioLegend\n",
      "Abcam\n",
      "Santa Cruz\n",
      "Cell signaling\n",
      "Invitrogen\n",
      "Santa Cruz\n",
      "Kind gift from Dr. Peter\n",
      "Newman\n",
      "BD Bioscience\n",
      "BD Bioscience\n",
      "\n",
      "Catalog #\n",
      "6943\n",
      "2109\n",
      "26003\n",
      "26060\n",
      "612705\n",
      "26006\n",
      "2478\n",
      "44-1047G\n",
      "2479\n",
      "9271\n",
      "sc-6458\n",
      "PA5-65359\n",
      "ABIN129578\n",
      "50-9760-82\n",
      "C6198\n",
      "61-7300\n",
      "9562\n",
      "MMS-115P\n",
      "901501\n",
      "ab13970\n",
      "sc-9996\n",
      "2625\n",
      "62204\n",
      "sc-8432\n",
      "\n",
      "557355\n",
      "610823\n",
      "\n",
      "\f349\n",
      "\n",
      "Cell Culture\n",
      "\n",
      "Primary  HUVECs  were  obtained  from  the  Yale  Vascular  Biology  and  Therapeutics  core\n",
      "\n",
      "facility.  Each batch is composed of cells pooled from three donors.  Cells were cultured in M199\n",
      "\n",
      "(Gibco:  11150-059)  supplemented  with  20%  FBS,  1x  Penicillin-Streptomycin  (Gibco:  15140-\n",
      "\n",
      "122),  60  µg/ml  heparin  (Sigma:  H3393),  and  endothelial  growth  cell  supplement  (hereafter,\n",
      "\n",
      "complete medium).  HUVECs used for experiments were between passages 3 and 6.\n",
      "\n",
      "350\n",
      "\n",
      "351\n",
      "\n",
      "352\n",
      "\n",
      "353\n",
      "\n",
      "354\n",
      "\n",
      "355\n",
      "\n",
      "356\n",
      "\n",
      "Shear stress\n",
      "\n",
      "For generating protein lysates for pulldown assays, HUVECs were seeded on tissue culture-\n",
      "\n",
      "treated  plastic  slides  coated  with  10  µg/ml  fibronectin  for  1  hour  at  37˚C  and  then  grown  to\n",
      "\n",
      "confluence in HUVEC complete media.  For short-term stimulation with shear stress, cells were\n",
      "\n",
      "starved overnight in M199 medium with 2% FBS and 1:10 of ECGS or for 30 minutes in M199\n",
      "\n",
      "medium containing 0.2 % BSA.  These slides were set in parallel flow chambers and shear stress\n",
      "\n",
      "applied as described(Frangos et al, 1988).\n",
      "\n",
      "For imaging cells under shear stress, HUVECs were plated in HUVEC complete medium on\n",
      "\n",
      "glass bottom 6-well plates coated with 10 µg/ml fibronectin and shear stress applied on an orbital\n",
      "\n",
      "shaker  at  150rpm  for  24  hours,  leading  to  uniaxial  pulsatile  shear  in  the  peripheral  region  and\n",
      "\n",
      "multiaxial shear in the middle of the well, as previously described(Arshad et al, 2021).  Cells in\n",
      "\n",
      "the outer region, 0.7-0.9  cm from the center of each well were  examined  for Golgi  orientation\n",
      "\n",
      "and endothelial alignment.\n",
      "\n",
      "357\n",
      "\n",
      "358\n",
      "\n",
      "359\n",
      "\n",
      "360\n",
      "\n",
      "361\n",
      "\n",
      "362\n",
      "\n",
      "363\n",
      "\n",
      "364\n",
      "\n",
      "365\n",
      "\n",
      "366\n",
      "\n",
      "367\n",
      "\n",
      "368\n",
      "\n",
      "369\n",
      "\n",
      "370\n",
      "\n",
      "Image analysis\n",
      "\n",
      "371\n",
      "\n",
      "372\n",
      "\n",
      "373\n",
      "\n",
      "374\n",
      "\n",
      "375\n",
      "\n",
      "Cell orientation was calculated by taking the masks of the cell nuclei determined by Hoechst\n",
      "\n",
      "images,  fitting  them  as  an  ellipse,  and  determining  the  angle  between  flow  direction  and  the\n",
      "\n",
      "major axis of the ellipse.  Analyzed results were visualized as histograms showing the percent of\n",
      "\n",
      "cells  within  each  10°  of  the  direction  of  flow  or  as  quantification  of  aligned  cells  with  nuclei\n",
      "\n",
      "whose major axis were within 0-30 degrees to flow direction as indicated in the figure legends.\n",
      "\n",
      "\f376\n",
      "\n",
      "Eccentricity for cell alignment was calculated based on first eccentricity of ellipse following the\n",
      "\n",
      "377\n",
      "\n",
      "equation below.\n",
      "\n",
      "378\n",
      "\n",
      "379\n",
      "\n",
      "380\n",
      "\n",
      "381\n",
      "\n",
      "382\n",
      "\n",
      "383\n",
      "\n",
      "384\n",
      "\n",
      "𝐸𝑐𝑐𝑒𝑛𝑡𝑟𝑖𝑐𝑖𝑡𝑦  =\n",
      "\n",
      "√𝑎2 − 𝑏2\n",
      "𝑎\n",
      "\n",
      "(a: major axis, b:minor axis)\n",
      "\n",
      "Junctional linearization index was calculated as ratio of perimeter of the cell to perimeter of\n",
      "\n",
      "its convex hull following the equation below:\n",
      "\n",
      "𝐽𝑢𝑛𝑐𝑡𝑖𝑜𝑛𝑎𝑙 𝑙𝑖𝑛𝑒𝑎𝑟𝑖𝑧𝑎𝑡𝑖𝑜𝑛 𝑖𝑛𝑑𝑒𝑥  =\n",
      "\n",
      "𝑒𝑥𝑎𝑐𝑡 𝑐𝑜𝑛𝑡𝑜𝑢𝑟 𝑜𝑓 𝑡ℎ𝑒 𝑐𝑒𝑙𝑙\n",
      "𝑐𝑜𝑛𝑣𝑒𝑥 𝑝𝑒𝑟𝑖𝑚𝑒𝑡𝑒𝑟\n",
      "\n",
      "385\n",
      "\n",
      "Data display\n",
      "\n",
      "Quantified  data  are  displayed  as  means  ±  standard  error  (SEM)  in  which  the  indicated\n",
      "\n",
      "replicates are independent experiments.  F test was conducted to check equal variance.\n",
      "\n",
      "386\n",
      "\n",
      "387\n",
      "\n",
      "388\n",
      "\n",
      "389\n",
      "\n",
      "Lentiviral transduction\n",
      "\n",
      "390\n",
      "\n",
      "391\n",
      "\n",
      "392\n",
      "\n",
      "393\n",
      "\n",
      "394\n",
      "\n",
      "395\n",
      "\n",
      "396\n",
      "\n",
      "Lenti-X  293T  cells  (Clontech,  632180)  were  cultured  for  at  least  24  hours  in  DMEM\n",
      "\n",
      "supplemented  with  10%  FBS  and  lacking  antibiotics,  then  transfected  with  lentiviral  plasmids\n",
      "\n",
      "encoding  the  gene  of  interest  and  packaging  plasmids  (Addgene:  12259  and  12260)  using\n",
      "\n",
      "Lipofectamine  2000  (Thermo  Fisher  Scientific:  11668-019)  following  the  manufacturer’s\n",
      "\n",
      "protocols  with  Opti-MEM  medium.    Conditioned  media  from  these  cultures  were  collected  48\n",
      "\n",
      "hours  later,  sterilized  through  0.22µm  filters  and  added  to  HUVECs  together  with  8µg/ml  of\n",
      "\n",
      "polybrene  (Sigma:  107689).    After  24  hours,  cells  were  switched  to  complete  medium  for  48\n",
      "\n",
      "397\n",
      "\n",
      "hours.\n",
      "\n",
      "398\n",
      "\n",
      "\f399\n",
      "\n",
      "siRNA transfection\n",
      "\n",
      "HUVECs were cultured in EGMTM-2 Endothelial Cell Growth Medium-2 BulletKitTM\n",
      "\n",
      " (Lonza:  CC-3156  and  CC-4176)  for  24  hours  before  transfecting  with  RNAiMax  (Thermo\n",
      "\n",
      "Fisher Scientific: 13778-150) with 20nM siRNA in Opti-MEM (Gibco: 31985-070) according to\n",
      "\n",
      "the manufacturer’s instructions.  After 6 hours, cells were switched to EGM-2 medium and used\n",
      "\n",
      "for  experiments  2-3  days  later.    Gα  protein  siRNAs  were  custom  designed  based  on  previous\n",
      "\n",
      "publications(Grzelinski et al, 2010; Krumins & Gilman, 2006; Ngai et al, 2008).  For latrophilin\n",
      "\n",
      "knockdown, ON-TARGET plus Smartpool siRNAs from Dharmacon against human LPHN1 (L-\n",
      "\n",
      "005650-00-0005), LPHN2 (L-005651-00-0005), LPHN3 (L-005652-00-0005) were used.\n",
      "\n",
      "400\n",
      "\n",
      "401\n",
      "\n",
      "402\n",
      "\n",
      "403\n",
      "\n",
      "404\n",
      "\n",
      "405\n",
      "\n",
      "406\n",
      "\n",
      "407\n",
      "\n",
      "408\n",
      "\n",
      "409\n",
      "\n",
      "Western Blotting\n",
      "\n",
      "410\n",
      "\n",
      "411\n",
      "\n",
      "412\n",
      "\n",
      "413\n",
      "\n",
      "HUVECs  were washed with  PBS and extracted in  Laemmli sample buffer.   Samples were\n",
      "\n",
      "separated  by  SDS-PAGE  and  transferred  onto  nitrocellulose  membranes.    Membranes  were\n",
      "\n",
      "blocked with 5% milk in TBS-T and probed with primary antibodies at 4˚C for overnight.  The\n",
      "\n",
      "targeting  proteins  were  visualized  by  HRP-conjugated  secondary  antibodies  and  subsequent\n",
      "\n",
      "414\n",
      "\n",
      "HRP-luminol reaction.\n",
      "\n",
      "415\n",
      "\n",
      "416\n",
      "\n",
      "Immunofluorescence\n",
      "\n",
      "HUVECs were washed with PBS, fixed for 10 minutes with 3.7% PFA in PBS.  Following\n",
      "\n",
      "fixation,  cells  were  permeabilized  with  0.5%  Triton  X-100  in  PBS  for  10  minutes  and  then\n",
      "\n",
      "incubated with 3% BSA in PBS for 30 minutes for blocking.   Cells were washed with PBS after\n",
      "\n",
      "blocking and were incubated Alexa488-Phalloidin and Hoechst for 1 hour, then washed 4 times\n",
      "\n",
      "with  PBS  and  mounted.    Images  were  captured  with  20x  or  60x  objective  on  a  PerkinElmer\n",
      "\n",
      "spinning  disk  confocal  microscope  or  a  Leica  Sp8  confocal  microscope  with  the  Leica\n",
      "\n",
      "Application  Suite\n",
      "\n",
      "(LAS)\n",
      "\n",
      "software.  Cell  alignment  was  determined  as  described\n",
      "\n",
      "previously(Baeyens et al, 2014).\n",
      "\n",
      "417\n",
      "\n",
      "418\n",
      "\n",
      "419\n",
      "\n",
      "420\n",
      "\n",
      "421\n",
      "\n",
      "422\n",
      "\n",
      "423\n",
      "\n",
      "424\n",
      "\n",
      "425\n",
      "\n",
      "\f426\n",
      "\n",
      "427\n",
      "\n",
      "428\n",
      "\n",
      "429\n",
      "\n",
      "430\n",
      "\n",
      "431\n",
      "\n",
      "432\n",
      "\n",
      "433\n",
      "\n",
      "434\n",
      "\n",
      "435\n",
      "\n",
      "436\n",
      "\n",
      "437\n",
      "\n",
      "438\n",
      "\n",
      "439\n",
      "\n",
      "440\n",
      "\n",
      "441\n",
      "\n",
      "442\n",
      "\n",
      "443\n",
      "\n",
      "444\n",
      "\n",
      "445\n",
      "\n",
      "446\n",
      "\n",
      "447\n",
      "\n",
      "448\n",
      "\n",
      "449\n",
      "\n",
      "Reverse transcription and quantitative PCR\n",
      "\n",
      "RNA  was  isolated  from  HUVECs  using  RNeasy  kit  according  to  the  manufacturer’s\n",
      "\n",
      "instructions  and  quantified  using  a  nanodrop  spectrophotometer.        Following  cDNA  synthesis\n",
      "\n",
      "using Bio-Rad iScript kit, RT-PCR was performed as follows.  Each PCR reaction contains 42\n",
      "\n",
      "two-step amplification cycles consisting of: (a) denaturation at 95˚C for 30s, and (b) annealing\n",
      "\n",
      "and extension at 60˚C for 30s.  The amplification curve was collected, and the relative transcript\n",
      "\n",
      "level  of  the  target  mRNA  in  each  sample  was  calculated  by  normalization  of  Ct  values  to  the\n",
      "\n",
      "reference mRNA (GAPDH).  Primer sequences used for RT-PCR were as shown in Table 1.\n",
      "\n",
      "Gα protein pulldown assay\n",
      "\n",
      "BL21 E.coli  cells  were  transformed  with  constructs expressing GST-tagged GINIP protein\n",
      "\n",
      "or  GST-tagged  GRK2  N-terminal  domain.\n",
      "\n",
      "  Cells  were\n",
      "\n",
      "incubated\n",
      "\n",
      "in\n",
      "\n",
      "terrific  broth\n",
      "\n",
      "(http://cshprotocols.cshlp.org/content/2015/9/pdb.rec085894.full) and protein expression induced\n",
      "\n",
      "by  addition  of  0.5µg/ml  IPTG.    Cells  were  collected  after  8  hours,  lysed  and  GST-  proteins\n",
      "\n",
      "collected  on  Glutathione-conjugated  beads.  Beads  were  washed  4  times,  eluted  with  50mM\n",
      "\n",
      "glutathione  and  proteins  desalted  on  a  gel  filtration  column.  Aliquots  of  20  µg  aliquots  were\n",
      "\n",
      "stored frozen and thawed shortly before use.\n",
      "\n",
      "For pulldown assays, HUVECs infected with lentivirus encoding GluGlu-tagged Gi mutant\n",
      "\n",
      "or lentivirus encoding both GluGlu-tagged Gq mutant and Ric8A were lysed in cell lysis buffer\n",
      "\n",
      "composed of 10mM Tris-HCl pH7.5, 150mM NaCl, 1% Triton X-100, 5mM DTT and additional\n",
      "\n",
      "2µg/ml  of  GST-GINIP  or  GST-GRK2N,  and  the  lysates  were  incubated  for  10 minutes  at  4˚C\n",
      "\n",
      "with  gentle  agitation  with  glutathione  beads.    Beads  were  washed  three  times  with  cell  lysis\n",
      "\n",
      "buffer, isolated proteins were solubilized in SDS sample laemmli buffer, and were analyzed by\n",
      "\n",
      "450\n",
      "\n",
      "western blotting.\n",
      "\n",
      "451\n",
      "\n",
      "452\n",
      "\n",
      "\f453\n",
      "\n",
      "454\n",
      "\n",
      "455\n",
      "\n",
      "456\n",
      "\n",
      "457\n",
      "\n",
      "458\n",
      "\n",
      "459\n",
      "\n",
      "Mass spectrometry screening for Gα binding receptors\n",
      "\n",
      "To create a Gα protein variant that interacts more strongly with GPCR, a green fluorescent\n",
      "\n",
      "protein  (GFP)  tag  was  inserted  for  affinity  purification,  alongside  a  mutation  of  four  alanine\n",
      "\n",
      "residues into helix  α5 (ins4A), as this  hinders GTP binding and maintains  the association with\n",
      "\n",
      "GPCRs  even  in  the  presence  of  GTP.    Additionally,  for  the  extraction  of  GPCRs  while\n",
      "\n",
      "preserving  their  configuration  and  lipid  environment,  a  non-detergent  nanodisc  approach\n",
      "\n",
      "utilizing styrene-maleimide anhydride (SMA) copolymers was employed according to protocols\n",
      "\n",
      "460\n",
      "\n",
      "described previously.\n",
      "\n",
      "461\n",
      "\n",
      "462\n",
      "\n",
      "463\n",
      "\n",
      "464\n",
      "\n",
      "465\n",
      "\n",
      "466\n",
      "\n",
      "467\n",
      "\n",
      "468\n",
      "\n",
      "469\n",
      "\n",
      "470\n",
      "\n",
      "471\n",
      "\n",
      "472\n",
      "\n",
      "473\n",
      "\n",
      "474\n",
      "\n",
      "475\n",
      "\n",
      "476\n",
      "\n",
      "477\n",
      "\n",
      "478\n",
      "\n",
      "In the coimmunoprecipitation process, GFP-Trap® nanobody beads were utilized to capture\n",
      "\n",
      "the  interactions  between  GFP-tagged  Gα  protein  variants  and  activated  designer  receptors\n",
      "\n",
      "exclusively activated by designer drugs (DREADDs). The resulting complexes were subjected to\n",
      "\n",
      "subsequent  proteomic  analysis  to  determine  which  GPCRs  interacted  preferentially  with  the\n",
      "\n",
      "Gi1(Q306K)  variant  following  induction  of  fluid  shear  stress.  Identified receptors  were  further\n",
      "\n",
      "analyzed by western blotting as described in the supplementary material.\n",
      "\n",
      "Measurement of Golgi orientation\n",
      "\n",
      "Following exposure to laminar shear stress, HUVECs were washed with PBS, fixed for 10\n",
      "\n",
      "minutes with 3.7% PFA in PBS, permeabilized with 0.5% Triton X-100 in PBS for 10 minutes,\n",
      "\n",
      "blocked with 3% BSA in PBS for 30 minutes and then incubated with GM130 antibody (1:500,\n",
      "\n",
      "BD  bioscience)  at  4˚C  for  overnight.  After  washing,  cells  were  incubated  with  Hoechst  and\n",
      "\n",
      "Alexa647-conjugated  secondary  antibody  at  RT  for  1  hour  to  visualize  the  nuclei  and  Golgi\n",
      "\n",
      "apparatus,  respectively.    Images  were  captured  with  20x  objective  on  a  Leica  Sp8  confocal\n",
      "\n",
      "microscope with the Leica Application Suite (LAS) software.\n",
      "\n",
      "Golgi polarization was calculated as a vector that connects the center of mass of the nucleus\n",
      "\n",
      "to the center of mass of the Golgi. The angle between this vector and the direction of flow was\n",
      "\n",
      "\f479\n",
      "\n",
      "480\n",
      "\n",
      "481\n",
      "\n",
      "482\n",
      "\n",
      "483\n",
      "\n",
      "484\n",
      "\n",
      "485\n",
      "\n",
      "486\n",
      "\n",
      "487\n",
      "\n",
      "488\n",
      "\n",
      "489\n",
      "\n",
      "490\n",
      "\n",
      "491\n",
      "\n",
      "492\n",
      "\n",
      "493\n",
      "\n",
      "494\n",
      "\n",
      "495\n",
      "\n",
      "496\n",
      "\n",
      "497\n",
      "\n",
      "498\n",
      "\n",
      "then determined, with 0° defined as against the flow. Oriana V4 software was used to calculate\n",
      "\n",
      "the angle mean and circular SD and to generate a rose plot for data visualization.\n",
      "\n",
      "In vitro permeability assay using the FITC-streptavidin/biotinylated-fibronectin\n",
      "\n",
      "Slides/dishes  were coated  with  biotinylated-Fibronectin  at 0.25 mg/mL overnight at  4°C.\n",
      "\n",
      "HUVECs  were  seeded  of  2×106  cells  per  well  on  glass-bottom  6-well  plates.\n",
      "\n",
      "Cells were subjected to laminar  shear  stress  by  orbital  shaking  at  150\n",
      "\n",
      "rpm\n",
      "\n",
      "for  24\n",
      "\n",
      "hours.  Immediately after cessation of flow, FITC-Streptavidin (1:1000, Invitrogen) was added to\n",
      "\n",
      "the cells for 1 minute before  the  fixation  by  the  addition  of  3.7%  (v/v)  paraformaldehyde.\n",
      "\n",
      "Following    subsequent    washing    and    blocking,  Alexa  Fluor  488  Phalloidin  (1:200;\n",
      "\n",
      "Invitrogen) was  added  to  stain  F-actin  filaments. Images were captured from 5 random fields\n",
      "\n",
      "and FITC-streptavidin staining quantified using ImageJ.\n",
      "\n",
      "In vitro endothelial 3D invasion assay under shear stress\n",
      "\n",
      "In  order  to  equalize  the  VEGF  concentration  in  the  collagen  gel  and  endothelial  culture\n",
      "\n",
      "medium, VEGF-189 (R&D systems, 8147-VE), a matrix-binding isoform of VEGF, was applied\n",
      "\n",
      "to the bottom of each well at a concentration of 50 ng/ml prior to adding collagen matrices. Rat\n",
      "\n",
      "tail  collagen  type  I  (Thermofisher  scientific,  A1048301)  was  employed  to  prepare  collagen\n",
      "\n",
      "matrices at the concentration of 1.5 mg/ml with additional NaOH to facilitate the collagen matrix\n",
      "\n",
      "assembly by neutral pH. After thorough mixing, 1 ml of collagen I solution was placed in each\n",
      "\n",
      "499\n",
      "\n",
      "well  of  a  12-well  plate  (Corning  #3513)  and  polymerized  at  37°C  and  5%  CO2 for  2  hours  to\n",
      "\n",
      "500\n",
      "\n",
      "overnight.\n",
      "\n",
      "501\n",
      "\n",
      "502\n",
      "\n",
      "503\n",
      "\n",
      "504\n",
      "\n",
      "505\n",
      "\n",
      "To  prepare  endothelial  cells,  confluent  HUVEC  culture  plates  were  washed  with  PBS,\n",
      "\n",
      "trypsinized,  and  counted.    After  pelleting,  cell  pellets  were  resuspended  in  EGM-2  medium\n",
      "\n",
      "(Lonza,  #CC-3162)  free  of  VEGF  and  heparin  to  establish  a  positive  gradient  of  VEGF  from\n",
      "\n",
      "cells to the bottom of the collagen gel, and 300,000 cells per well were placed on the collagen gel\n",
      "\n",
      "and  incubated  for  24  hours.  Following  incubation,  plates were  placed  on  an  orbital  shaker  and\n",
      "\n",
      "\f506\n",
      "\n",
      "507\n",
      "\n",
      "508\n",
      "\n",
      "509\n",
      "\n",
      "510\n",
      "\n",
      "511\n",
      "\n",
      "512\n",
      "\n",
      "ECs were exposed to orbital shaking at 150 rpm for 48 hours as mentioned above in shear stress\n",
      "\n",
      "section. Cells were then fixed in 4% PFA and permeabilized in PBS containing 0.5% Triton X-\n",
      "\n",
      "100. Nonspecific binding was blocked by incubation with PBS containing 3% BSA. Thereafter,\n",
      "\n",
      "cells were subjected to Alexa488-conjugated phalloidin (1:400 dilution; Thermofisher scientific:\n",
      "\n",
      "A12379)  and  Hoechst  (1:1000  dilution;  Invitrogen:  H1399)  for  2  hours  to  stain  actin\n",
      "\n",
      "cytoskeleton and cell nuclei, respectively. All images were captured with the Nikon Sp8 confocal\n",
      "\n",
      "microscope  and  segregated  into  peripheral  region  (with  uniaxial  pulsatile  shear)  vs.  static\n",
      "\n",
      "513\n",
      "\n",
      "controls.\n",
      "\n",
      "514\n",
      "\n",
      "515\n",
      "\n",
      "516\n",
      "\n",
      "517\n",
      "\n",
      "Zebrafish husbandry and handling\n",
      "\n",
      "Zebrafish  were  housed  and  maintained  at  28.5ºC  in  accordance  with  standard  methods\n",
      "\n",
      "approved  by  the  Yale  University  Institutional  Animal  Care  and  Use  Committee  (#2017-\n",
      "\n",
      "518\n",
      "\n",
      "11473)(Lawrence, 2016).\n",
      "\n",
      "519\n",
      "\n",
      "520\n",
      "\n",
      "521\n",
      "\n",
      "522\n",
      "\n",
      "523\n",
      "\n",
      "524\n",
      "\n",
      "525\n",
      "\n",
      "526\n",
      "\n",
      "527\n",
      "\n",
      "528\n",
      "\n",
      "529\n",
      "\n",
      "530\n",
      "\n",
      "531\n",
      "\n",
      "532\n",
      "\n",
      "Generation of LPHN/adgrl knockdown zebrafish using CRISPR/Cas9 ribonucleoproteins\n",
      "\n",
      "Four  zebrafish  latrophilin  paralogs  were  identified—adgrl1a,  adgrl2a,  adgrl2b.1,  and\n",
      "\n",
      "adgrl3.1. Fused sgRNAs were generated targeting each paralog (see Table 2) by annealing locus-\n",
      "\n",
      "specific oligonucleotides to a common 5’ universal oligonucleotide and performing in vitro RNA\n",
      "\n",
      "transcription (AmpliScribe T7-Flash Kit, Lucigen, ASF3507)(Gagnon et al, 2014). An injection\n",
      "\n",
      "mix  of  50  ng/µL  each  sgRNA,  an  equivalent  concentration  of  Cas9  protein  (TrueCut  Cas9,\n",
      "\n",
      "Invitrogen,  A36497),  and  either  1  µg/µL  silent  heart/tnnt2a(Sehnert  et  al.,  2002)  or  standard\n",
      "\n",
      "control  morpholino  (GeneTools)  was  prepared.  1  nL  of  the  mixture  was  injected  into\n",
      "\n",
      "Tg(kdrl:ras-mCherry,  fli1a:nls-GFP)  zebrafish  at  the  one  cell  stage  (Parker  Hannifin,\n",
      "\n",
      "Picospritzer III). CRISPR reagent efficacy was confirmed by T7 endonuclease assay (see Figure\n",
      "\n",
      "S5).  Guides against adgrl2a and 2b.1 were combined to target all LPHN2 isoforms.\n",
      "\n",
      "Immunostaining and imaging of ECs in the knockdown zebrafish\n",
      "\n",
      "\f533\n",
      "\n",
      "534\n",
      "\n",
      "535\n",
      "\n",
      "536\n",
      "\n",
      "537\n",
      "\n",
      "538\n",
      "\n",
      "539\n",
      "\n",
      "540\n",
      "\n",
      "541\n",
      "\n",
      "542\n",
      "\n",
      "543\n",
      "\n",
      "544\n",
      "\n",
      "545\n",
      "\n",
      "546\n",
      "\n",
      "547\n",
      "\n",
      "548\n",
      "\n",
      "549\n",
      "\n",
      "550\n",
      "\n",
      "551\n",
      "\n",
      "552\n",
      "\n",
      "553\n",
      "\n",
      "554\n",
      "\n",
      "555\n",
      "\n",
      "556\n",
      "\n",
      "557\n",
      "\n",
      "558\n",
      "\n",
      "559\n",
      "\n",
      "CRISPR/Cas9  RNP-injected  embryos  were  either  fixed  at  48  hpf  in  4%  paraformaldehyde\n",
      "\n",
      "(Santa Cruz Biotechnology, SC-253236) and immunostained for GFP (1:300 chicken anti-GFP,\n",
      "\n",
      "abcam,  ab13970),  mCherry  (1:300  rabbit  anti-mCherry,  abcam,  ab167453),  and  ZO-1  (1:200\n",
      "\n",
      "mouse  anti-ZO-1,  Invitrogen,  61-7300)  with  species-appropriate  secondary  antibodies  (1:400\n",
      "\n",
      "Invitrogen  anti-chicken/-rabbit/-mouse  Alexa  488/546/647,  A-11039/A-  10040,  A-31571)  or\n",
      "\n",
      "imaged live. For fixed imaging, the larvae were processed as previously described(Sugden et al.,\n",
      "\n",
      "2017).  For  live  imaging,  larvae  were  anesthetized  by  immersion  in  600  µM  tricaine\n",
      "\n",
      "methylsulfonate (Western Chemical, TRS5), and embedded in 1.5% low-melt agarose (BioRad,\n",
      "\n",
      "1613106).  For  staining  in  detail,  larvae  were  fixed  overnight  in  4%  PFA  at  +4ºC  and  then\n",
      "\n",
      "transitioned  into  100%  methanol  via  a  series  of  dilutions.  Once  in  methanol,  the  larvae  were\n",
      "\n",
      "stored at -20ºC overnight, then rehydrated the next day via the same dilution series. Larvae were\n",
      "\n",
      "permeabilized using incubation in acetone for 15 minutes at -20ºC and then by incubation in 20\n",
      "\n",
      "µg/mL proteinase K for 20 minutes at 37ºC. Larvae were refined in 4% PFA for 15 minutes at\n",
      "\n",
      "room temperature and then incubated in blocking buffer (0.5% Triton X-100, 10% goat serum,\n",
      "\n",
      "1% BSA, 0.01% sodium azide, and 1% DMSO in PBS) for 2 hours at room temperature. Larvae\n",
      "\n",
      "were  incubated  in  primary  antibody  (1:400  chicken  anti-GFP,  abcam  ab13970)  overnight  in\n",
      "\n",
      "blocking buffer at +4ºC. Larvae were thoroughly washed in PBS-Tx and incubated overnight at\n",
      "\n",
      "+4ºC  in  secondary  antibody  (1:400  goat  anti-chicken  Alexa488  A-11039,  ThermoFisher)  in\n",
      "\n",
      "blocking  buffer.  Following  final  washes  in  PBS-Tx,  larvae  were  mounted  and  imaged.\n",
      "\n",
      "Intersegmental vessel and dorsal aorta diameter were then calculated using ImageJ. All imaging\n",
      "\n",
      "was performed using a Leica SP8 confocal microscope.\n",
      "\n",
      "Immunostaining and imaging of mouse aorta\n",
      "\n",
      "Aortae were perfusion-fixed in situ with 4% paraformaldehyde (overnight) prior to staining\n",
      "\n",
      "with primary beta-catenin antibody (Cell signaling: 9562) and fluorophore-conjugated secondary\n",
      "\n",
      "antibody  (Invitrogen,  donkey  anti-Rabbit  IgG  (H+L)  Highly  Cross-Adsorbed  Secondary\n",
      "\n",
      "Antibody, Alexa Fluor 647, A31573) in in Claudio buffer (1% FBS, 3% BSA, 0.5% Triton X-\n",
      "\n",
      "\f560\n",
      "\n",
      "561\n",
      "\n",
      "562\n",
      "\n",
      "563\n",
      "\n",
      "564\n",
      "\n",
      "565\n",
      "\n",
      "566\n",
      "\n",
      "567\n",
      "\n",
      "568\n",
      "\n",
      "569\n",
      "\n",
      "570\n",
      "\n",
      "571\n",
      "\n",
      "572\n",
      "\n",
      "573\n",
      "\n",
      "574\n",
      "\n",
      "575\n",
      "\n",
      "576\n",
      "\n",
      "577\n",
      "\n",
      "578\n",
      "\n",
      "579\n",
      "\n",
      "580\n",
      "\n",
      "581\n",
      "\n",
      "582\n",
      "\n",
      "583\n",
      "\n",
      "584\n",
      "\n",
      "100,  0.01%  Sodium  deoxycholate,  0.02%  sodium  azide  in  PBS  pH7.4)(Franco  et  al,  2013).\n",
      "\n",
      "Cells were stained with Hoechst (Invitrogen: H1399) to  label nuclei. Images of stained vessels\n",
      "\n",
      "were  acquired using a Leica SP8 confocal  microscope  with the Leica Application Suite (LAS)\n",
      "\n",
      "software. Endothelial cell alignment in aorta was quantified by measuring the eccentricity of the\n",
      "\n",
      "immunostaining of beta-catenin of the endothelial cells (>150 cells per field of view).\n",
      "\n",
      "Extravasation of Evans blue dye for aortic vascular permeability\n",
      "\n",
      "For aortic vascular permeability measurements, Evans blue dye (20 mg/ml in saline, 2.5 µl/g\n",
      "\n",
      "mouse  body  weight)  (Sigma  E2129)  was  injected  retro-orbitally  to  LPHN2  endothelial  cell-\n",
      "\n",
      "specific  knockout  and  their  control  littermates  at  8  weeks  old.    Thirty  minutes  post  injection,\n",
      "\n",
      "animals  were euthanized by an overdose  of isoflurane  and tissue samples  including aorta were\n",
      "\n",
      "harvested and washed with PBS to remove extra fatty tissues.  Dissected aorta was prepared en\n",
      "\n",
      "face  and  imaged  with  an  Olympus  stereo  zoom  microscope.  To  quantify  blue  color  associated\n",
      "\n",
      "with extravasation of Evans blue dye, the original RGB images were converted into HSV color\n",
      "\n",
      "model,  and  the  proportion  of  the  blue  regions  was  calculated  as  the  ratio  of  pixels  with  hue\n",
      "\n",
      "angles of 120°-240° to the entire aorta.\n",
      "\n",
      "Immunostaining and imaging of mouse muscle capillaries\n",
      "\n",
      "Rectus femoris muscle from murine thigh were perfusion-fixed with 4% paraformaldehyde\n",
      "\n",
      "(overnight)  and  the  middle  part  of  the  muscle  were  cryo-sectioned  following  embedding  into\n",
      "\n",
      "OCT compound and stained with primary antibodies for SMA (fluorophore-conjugated) and for\n",
      "\n",
      "CD31  followed  by  fluorophore-conjugated  secondary  antibody  (Invitrogen,  donkey  anti-Rabbit\n",
      "\n",
      "IgG  (H+L)  Highly  Cross-Adsorbed  Secondary  Antibody,  Alexa  Fluor  647,  A31573)  in  PBS\n",
      "\n",
      "containing 5% BSA.  Hoechst (Invitrogen: H1399) was used to nuclear identification. Images of\n",
      "\n",
      "stained vessels were acquired using a Leica SP8 confocal microscope with the Leica Application\n",
      "\n",
      "585\n",
      "\n",
      "Suite (LAS) software.\n",
      "\n",
      "586\n",
      "\n",
      "\fHindlimb ischemia model\n",
      "\n",
      "Mice were anesthetized and operated with a mixture of 1.5-2% isoflurane. The front hair of\n",
      "\n",
      "the right thigh was removed with Nair lotion and the local surgical area was disinfected. A 5 mm\n",
      "\n",
      "longitudinal skin incision was made in the right thigh. The distal femoral artery, proximal to the\n",
      "\n",
      "popliteal  artery  and  saphenous  artery,  was  explored  and  dissected.  6-0  silk  sutures  (Syneture)\n",
      "\n",
      "were double used for the ligation before excision of the arterial vessel bed between the distal end\n",
      "\n",
      "of the superficial epigastric artery and the trifurcation of the femoral artery into the descending\n",
      "\n",
      "genicular,  popliteal,  and  saphenous  branches.  The  ligated  artery  was  cut  between  two  sutures.\n",
      "\n",
      "The venous structures and accompanying peripheral nerves were kept intact. The overlying skin\n",
      "\n",
      "was  closed  with  a  6-0  prolene  (Ethicon)  and  disinfected.  At  the  end  of  the  procedure,\n",
      "\n",
      "Buprenorphine was administered subcutaneously at 0.05 mg/kg, once pre-emptively, then every\n",
      "\n",
      "12 h for 72 h. The survival rate for this surgery was 100%.\n",
      "\n",
      "Laser Doppler perfusion imaging\n",
      "\n",
      "Hindlimb perfusion was assessed noninvasively in the plantar foot before, 1, 3, 7, 14, and 21\n",
      "\n",
      "days  after  hindlimb  ischemia  by  scanning  laser-Doppler  [model  LDI2-IR  modified  for  high\n",
      "\n",
      "resolution;  Moor  Instruments,  Wilmington,  DE].  The  hindquarters  were  placed  on  the  top  of  a\n",
      "\n",
      "heater pad during scanning to minimize variation in body temperature. Doppler perfusion of the\n",
      "\n",
      "plantar  foot  was  assessed  within  anatomically  defined  regions  of  interest  (ROIs;  Moor\n",
      "\n",
      "Instruments).  The  ROI  for  the  plantar  foot  consisted  of  the  hind  paw  margins.  Procedures  for\n",
      "\n",
      "minimal  preparation,  scanning,  and  ROI  selections  have  been  described\n",
      "\n",
      "in  detail\n",
      "\n",
      "previously(Zhuang  et  al,  2011).  Low  or  no  perfusion  is  displayed  as  dark  blue,  whereas  the\n",
      "\n",
      "highest degree of perfusion is displayed as red. These images were quantitatively converted into\n",
      "\n",
      "histograms that represented the amount of blood flow on the x-axis and the number of pixels on\n",
      "\n",
      "the y-axis in the traced area. The average blood flow in each histogram was calculated and the\n",
      "\n",
      "LDI index was determined as the ratio of ischemic to non-ischemic hindlimb blood perfusion.\n",
      "\n",
      "587\n",
      "\n",
      "588\n",
      "\n",
      "589\n",
      "\n",
      "590\n",
      "\n",
      "591\n",
      "\n",
      "592\n",
      "\n",
      "593\n",
      "\n",
      "594\n",
      "\n",
      "595\n",
      "\n",
      "596\n",
      "\n",
      "597\n",
      "\n",
      "598\n",
      "\n",
      "599\n",
      "\n",
      "600\n",
      "\n",
      "601\n",
      "\n",
      "602\n",
      "\n",
      "603\n",
      "\n",
      "604\n",
      "\n",
      "605\n",
      "\n",
      "606\n",
      "\n",
      "607\n",
      "\n",
      "608\n",
      "\n",
      "609\n",
      "\n",
      "610\n",
      "\n",
      "611\n",
      "\n",
      "612\n",
      "\n",
      "613\n",
      "\n",
      "\f614\n",
      "\n",
      "615\n",
      "\n",
      "616\n",
      "\n",
      "617\n",
      "\n",
      "618\n",
      "\n",
      "619\n",
      "\n",
      "620\n",
      "\n",
      "621\n",
      "\n",
      "622\n",
      "\n",
      "623\n",
      "\n",
      "624\n",
      "\n",
      "625\n",
      "\n",
      "626\n",
      "\n",
      "627\n",
      "\n",
      "628\n",
      "\n",
      "629\n",
      "\n",
      "630\n",
      "\n",
      "631\n",
      "\n",
      "632\n",
      "\n",
      "633\n",
      "\n",
      "634\n",
      "\n",
      "635\n",
      "\n",
      "636\n",
      "\n",
      "637\n",
      "\n",
      "638\n",
      "\n",
      "639\n",
      "\n",
      "640\n",
      "\n",
      "Volumetric micro-CT angiogram for quantitative arteriogenesis\n",
      "\n",
      "After  the  mice  were  euthanized,  the  vasculature  was  flushed  with  0.9%  normal  saline\n",
      "\n",
      "containing heparin (1000 IU/L) for 3 minutes, followed by 4% paraformaldehyde for 5 minutes\n",
      "\n",
      "at 100 mm Hg pressure. Fresh home-made 20% bismuth nanoparticles mixed in 5% gelatin were\n",
      "\n",
      "used as a micro-CT contrast agent and injected over 2 minutes with a syringe pump, as described\n",
      "\n",
      "before(Tirziu et al., 2005). The mice were then immediately chilled on ice for more than 30 min\n",
      "\n",
      "and  immersion-fixed  in  2%  paraformaldehyde  overnight.  The  peripheral  vasculature  in  the\n",
      "\n",
      "ischemic  hindlimb  and  the  contralateral  normal  hindlimb  was  imaged  with  a  high-resolution\n",
      "\n",
      "micro-CT  imaging  system  (GE  eXplore  Locus  SP),  set  to  a  0.007-mm  effective  detector  pixel\n",
      "\n",
      "size.  Microview  software  (GE  Healthcare)  was  used  for  initial  reconstruction.  Advanced\n",
      "\n",
      "workstation with various software was used for 3D reconstruction and segmentation of the whole\n",
      "\n",
      "hindlimb at the 21 µm spatial resolution. Using the knee as the breakpoint, we re-sliced the 3D\n",
      "\n",
      "vasculature  into  200  slices  both  in  the  thigh  and  in  the  calf  regions.  Quantification  was\n",
      "\n",
      "performed  by  use  of  a  NIH  Image.  The  data  were  expressed  as  vascular  segment  numbers,\n",
      "\n",
      "representing the total number of vessels, of specified diameter, counted in 200 z-sections for the\n",
      "\n",
      "thigh region and another 200 z-sections for the calf region.\n",
      "\n",
      "Mouse treadmill fatigue test\n",
      "\n",
      "Mice were first acclimatized by staying on the motionless treadmill machine for 30 min on\n",
      "\n",
      "day 1 and then trained on the treadmill moving at 12 m/min for 30 min on day 2. Fatigue tests\n",
      "\n",
      "were then conducted on days 3 and 4.  Mice were placed on the treadmill and the shock rod was\n",
      "\n",
      "activated.   Treadmill  speeds  were  increased  as  follows:  0  m/min,  for  12  min  to  measure  basal\n",
      "\n",
      "metabolism,  12  m/min,  for1  min,  then  increasing  at  a  rate  of  +1  m/min  each  1  min  until\n",
      "\n",
      "exhaustion.  Exhaustion (endpoint of treadmill cessation) was defined as the point at which mice\n",
      "\n",
      "continuously  contacted  with  the  shock  rod  for  5  continuous  seconds  or  when  mice  stopped\n",
      "\n",
      "running  3  times  within  5  seconds.    Before  each  testing  session,  Oxymax  software  (Columbus\n",
      "\n",
      "Instruments,  Columbus,  OH,  USA)  and  open  circuit  indirect  calorimetry  treadmills  (Metabolic\n",
      "\n",
      "\fModular  Treadmill,  Columbus  Instruments,  Columbus,  OH)  were  calibrated  and  checked  for\n",
      "\n",
      "hardware malfunctions according to manufacturer instructions.\n",
      "\n",
      "641\n",
      "\n",
      "642\n",
      "\n",
      "643\n",
      "\n",
      "644\n",
      "\n",
      "Data availability\n",
      "\n",
      "645\n",
      "\n",
      "646\n",
      "\n",
      "The datasets produced in this study are available in the following databases:\n",
      "\n",
      "All  data:  BioStudies  S-BIAD928:  https://www.ebi.ac.uk/biostudies/studies?query=S-\n",
      "\n",
      "647\n",
      "\n",
      "BIAD928\n",
      "\n",
      "648\n",
      "\n",
      "649\n",
      "\n",
      "Acknowledgements\n",
      "\n",
      "650\n",
      "\n",
      "651\n",
      "\n",
      "652\n",
      "\n",
      "653\n",
      "\n",
      "654\n",
      "\n",
      "655\n",
      "\n",
      "656\n",
      "\n",
      "657\n",
      "\n",
      "658\n",
      "\n",
      "659\n",
      "\n",
      "660\n",
      "\n",
      "661\n",
      "\n",
      "662\n",
      "\n",
      "663\n",
      "\n",
      "664\n",
      "\n",
      "665\n",
      "\n",
      "This  work  was  supported  by  NIH  grants  RO1  HL75092  and  R01  HL151469  to  MAS,\n",
      "\n",
      "NHLBI 5R01HL130246-05 to SN, and the Uehara memorial foundation postdoctoral fellowship\n",
      "\n",
      "and MEXT | Japan Society for the Promotion of Science (JSPS) Overseas Research Fellowships\n",
      "\n",
      "(2014, receipt number: 805) to KT.  We thank Hamit Harun Dag (Istanbul University) for help of\n",
      "\n",
      "synthesizing  SMA  polymer;  Shahid  Mansuri  (Yale  University),  Jeremy  Herrera  (University  of\n",
      "\n",
      "Manchester)  and  David  Knight  (University  of  Manchester)  for  expert  advice  for  mass\n",
      "\n",
      "spectrometry  experiments;  Didier  Trono  (École  polytechnique  fédérale  de  Lausanne)  for\n",
      "\n",
      "lentiviral\n",
      "\n",
      "packaging\n",
      "\n",
      "plasmids\n",
      "\n",
      "pMD2.G\n",
      "\n",
      "(Addgene\n",
      "\n",
      "plasmid\n",
      "\n",
      "#\n",
      "\n",
      "12259\n",
      "\n",
      ";\n",
      "\n",
      "http://n2t.net/addgene:12259 ; RRID:Addgene_12259) and psPAX2 (Addgene plasmid # 12260 ;\n",
      "\n",
      "http://n2t.net/addgene:12260  ;  RRID:Addgene_12260);    Bryan  Roth  (University  of  North\n",
      "\n",
      "Carolina)\n",
      "\n",
      "for\n",
      "\n",
      "pcDNA5/FRT-HA-hM3D(Gq)\n",
      "\n",
      "(Addgene\n",
      "\n",
      "plasmid\n",
      "\n",
      "#\n",
      "\n",
      "45547\n",
      "\n",
      ";\n",
      "\n",
      "http://n2t.net/addgene:45547\n",
      "\n",
      ";\n",
      "\n",
      "RRID:Addgene_45547)\n",
      "\n",
      "and\n",
      "\n",
      "pcDNA5/FRT-HA-\n",
      "\n",
      "hM4D(Gi)(Addgene  plasmid  #  45548  ;  http://n2t.net/addgene:45548  ;  RRID:Addgene_45548);\n",
      "\n",
      "Aziz Moqrich (Institut de Biologie du Dévelopement de Marseille) for GST-GINIP plasmid.  We\n",
      "\n",
      "are  grateful  to  Peter  Newman  (Medical  College  of  Wisconsin)  for  the  kind  gift  of  PECAM-1\n",
      "\n",
      "antibody  and  helpful  discussions.    Schematic  models  are  inspired  from  BioRender  templates\n",
      "\n",
      "666\n",
      "\n",
      "(2022).\n",
      "\n",
      "667\n",
      "\n",
      "\fDisclosure & competing interests statement\n",
      "\n",
      "The authors declare they have no conflict of interest.\n",
      "\n",
      "668\n",
      "\n",
      "669\n",
      "\n",
      "670\n",
      "671\n",
      "\n",
      "\f672\n",
      "\n",
      "References\n",
      "\n",
      "673\n",
      "674\n",
      "675\n",
      "676\n",
      "677\n",
      "678\n",
      "679\n",
      "680\n",
      "681\n",
      "682\n",
      "683\n",
      "684\n",
      "685\n",
      "686\n",
      "687\n",
      "688\n",
      "689\n",
      "690\n",
      "691\n",
      "692\n",
      "693\n",
      "694\n",
      "695\n",
      "696\n",
      "697\n",
      "698\n",
      "699\n",
      "700\n",
      "701\n",
      "702\n",
      "703\n",
      "704\n",
      "705\n",
      "706\n",
      "707\n",
      "708\n",
      "709\n",
      "710\n",
      "711\n",
      "\n",
      "Albarran-Juarez J, Iring A, Wang S, Joseph S, Grimm M, Strilic B, Wettschureck N, Althoff TF,\n",
      "Offermanns S (2018) Piezo1 and G(q)/G(11) promote endothelial inflammation depending on\n",
      "flow pattern and integrin activation. J Exp Med 215: 2655-2672\n",
      "Arshad M, Ghim M, Mohamied Y, Sherwin SJ, Weinberg PD (2021) Endothelial cells do not align\n",
      "with the mean wall shear stress vector. J R Soc Interface 18: 20200772\n",
      "Bae H, Cabrera-Vera TM, Depree KM, Graber SG, Hamm HE (1999) Two amino acids within the\n",
      "alpha4 helix of Galphai1 mediate coupling with 5-hydroxytryptamine1B receptors. J Biol Chem\n",
      "274: 14963-14971\n",
      "Baeyens N, Bandyopadhyay C, Coon BG, Yun S, Schwartz MA (2016) Endothelial fluid shear\n",
      "stress sensing in vascular health and disease. J Clin Invest 126: 821-828\n",
      "Baeyens N, Mulligan-Kehoe MJ, Corti F, Simon DD, Ross TD, Rhodes JM, Wang TZ, Mejean CO,\n",
      "Simons M, Humphrey J et al (2014) Syndecan 4 is required for endothelial alignment in flow and\n",
      "atheroprotective signaling. Proc Natl Acad Sci U S A 111: 17308-17313\n",
      "Butler PJ, Norwich G, Weinbaum S, Chien S (2001) Shear stress induces a time- and position-\n",
      "dependent increase in endothelial cell membrane fluidity. Am J Physiol Cell Physiol 280: C962-\n",
      "969\n",
      "Camillo C, Facchinello N, Villari G, Mana G, Gioelli N, Sandri C, Astone M, Tortarolo D, Clapero F,\n",
      "Gays D et al (2021) LPHN2 inhibits vascular permeability by differential control of endothelial\n",
      "cell adhesion. J Cell Biol 220\n",
      "Campinho P, Vilfan A, Vermot J (2020) Blood Flow Forces in Shaping the Vascular System: A\n",
      "Focus on Endothelial Cell Behavior. Front Physiol 11: 552\n",
      "Chachisvilis M, Zhang YL, Frangos JA (2006) G protein-coupled receptors sense fluid shear stress\n",
      "in endothelial cells. Proc Natl Acad Sci U S A 103: 15463-15468\n",
      "Collins C, Guilluy C, Welch C, O'Brien ET, Hahn K, Superfine R, Burridge K, Tzima E (2012)\n",
      "Localized tensional forces on PECAM-1 elicit a global mechanotransduction response via the\n",
      "integrin-RhoA pathway. Curr Biol 22: 2087-2094\n",
      "Conway D, Schwartz MA (2012) Lessons from the endothelial junctional mechanosensory\n",
      "complex. F1000 Biol Rep 4: 1\n",
      "Conway DE, Breckenridge MT, Hinde E, Gratton E, Chen CS, Schwartz MA (2013) Fluid shear\n",
      "stress on endothelial cells modulates mechanical tension across VE-cadherin and PECAM-1. Curr\n",
      "Biol 23: 1024-1030\n",
      "Cornella N, Sancho J, Sitges-Serra A (2017) Short and Long-Term Outcomes After Surgical\n",
      "Procedures Lasting for More Than Six Hours. Sci Rep 7: 9221\n",
      "Dela Paz NG, Frangos JA (2019) Rapid flow-induced activation of Galphaq/11 is independent of\n",
      "Piezo1 activation. Am J Physiol Cell Physiol 316: C741-C752\n",
      "Dela Paz NG, Melchior B, Frangos JA (2017) Shear stress induces Galpha(q/11) activation\n",
      "independently of G protein-coupled receptor activation in endothelial cells. Am J Physiol Cell\n",
      "Physiol 312: C428-C437\n",
      "\n",
      "\f712\n",
      "713\n",
      "714\n",
      "715\n",
      "716\n",
      "717\n",
      "718\n",
      "719\n",
      "720\n",
      "721\n",
      "722\n",
      "723\n",
      "724\n",
      "725\n",
      "726\n",
      "727\n",
      "728\n",
      "729\n",
      "730\n",
      "731\n",
      "732\n",
      "733\n",
      "734\n",
      "735\n",
      "736\n",
      "737\n",
      "738\n",
      "739\n",
      "740\n",
      "741\n",
      "742\n",
      "743\n",
      "744\n",
      "745\n",
      "746\n",
      "747\n",
      "748\n",
      "749\n",
      "750\n",
      "751\n",
      "752\n",
      "753\n",
      "754\n",
      "\n",
      "dela Paz NG, Melchior B, Shayo FY, Frangos JA (2014) Heparan sulfates mediate the interaction\n",
      "between platelet endothelial cell adhesion molecule-1 (PECAM-1) and the Galphaq/11 subunits\n",
      "of heterotrimeric G proteins. J Biol Chem 289: 7413-7424\n",
      "Erdogmus S, Storch U, Danner L, Becker J, Winter M, Ziegler N, Wirth A, Offermanns S,\n",
      "Hoffmann C, Gudermann T et al (2019) Helix 8 is the essential structural motif of\n",
      "mechanosensitive GPCRs. Nat Commun 10: 5784\n",
      "Fancher IS, Ahn SJ, Adamos C, Osborn C, Oh MJ, Fang Y, Reardon CA, Getz GS, Phillips SA,\n",
      "Levitan I (2018) Hypercholesterolemia-Induced Loss of Flow-Induced Vasodilation and Lesion\n",
      "Formation in Apolipoprotein E-Deficient Mice Critically Depend on Inwardly Rectifying K(+)\n",
      "Channels. J Am Heart Assoc 7\n",
      "Flock T, Hauser AS, Lund N, Gloriam DE, Balaji S, Babu MM (2017) Selectivity determinants of\n",
      "GPCR-G-protein binding. Nature 545: 317-322\n",
      "Franco CA, Blanc J, Parlakian A, Blanco R, Aspalter IM, Kazakova N, Diguet N, Mylonas E, Gao-Li J,\n",
      "Vaahtokari A et al (2013) SRF selectively controls tip cell invasive behavior in angiogenesis.\n",
      "Development 140: 2321-2333\n",
      "Frangos JA, McIntire LV, Eskin SG (1988) Shear stress induced stimulation of mammalian cell\n",
      "metabolism. Biotechnol Bioeng 32: 1053-1060\n",
      "Gagnon JA, Valen E, Thyme SB, Huang P, Akhmetova L, Pauli A, Montague TG, Zimmerman S,\n",
      "Richter C, Schier AF (2014) Efficient mutagenesis by Cas9 protein-mediated oligonucleotide\n",
      "insertion and large-scale assessment of single-guide RNAs. PLoS One 9: e98186\n",
      "Gaillard S, Lo Re L, Mantilleri A, Hepp R, Urien L, Malapert P, Alonso S, Deage M, Kambrun C,\n",
      "Landry M et al (2014) GINIP, a Galphai-interacting protein, functions as a key modulator of\n",
      "peripheral GABAB receptor-mediated analgesia. Neuron 84: 123-136\n",
      "Galie PA, Nguyen DH, Choi CK, Cohen DM, Janmey PA, Chen CS (2014) Fluid shear stress\n",
      "threshold regulates angiogenic sprouting. Proc Natl Acad Sci U S A 111: 7968-7973\n",
      "Givens C, Tzima E (2016) Endothelial Mechanosignaling: Does One Sensor Fit All? Antioxid\n",
      "Redox Signal 25: 373-388\n",
      "Grzelinski M, Pinkenburg O, Buch T, Gold M, Stohr S, Kalwa H, Gudermann T, Aigner A (2010)\n",
      "Critical role of G(alpha)12 and G(alpha)13 for human small cell lung cancer cell proliferation in\n",
      "vitro and tumor growth in vivo. Clin Cancer Res 16: 1402-1415\n",
      "Gudi S, Nolan JP, Frangos JA (1998) Modulation of GTPase activity of G proteins by fluid shear\n",
      "stress and phospholipid composition. Proc Natl Acad Sci U S A 95: 2515-2519\n",
      "Gudi SR, Clark CB, Frangos JA (1996) Fluid flow rapidly activates G proteins in human\n",
      "endothelial cells. Involvement of G proteins in mechanochemical signal transduction. Circ Res\n",
      "79: 834-839\n",
      "Haidekker MA, L'Heureux N, Frangos JA (2000) Fluid shear stress increases membrane fluidity in\n",
      "endothelial cells: a study with DCVJ fluorescence. Am J Physiol Heart Circ Physiol 278: H1401-\n",
      "1406\n",
      "Humphrey JD, Schwartz MA (2021) Vascular Mechanobiology: Homeostasis, Adaptation, and\n",
      "Disease. Annu Rev Biomed Eng 23: 1-27\n",
      "Jung B, Obinata H, Galvani S, Mendelson K, Ding BS, Skoura A, Kinzel B, Brinkmann V, Rafii S,\n",
      "Evans T et al (2012) Flow-regulated endothelial S1P receptor-1 signaling sustains vascular\n",
      "development. Dev Cell 23: 600-610\n",
      "\n",
      "\f755\n",
      "756\n",
      "757\n",
      "758\n",
      "759\n",
      "760\n",
      "761\n",
      "762\n",
      "763\n",
      "764\n",
      "765\n",
      "766\n",
      "767\n",
      "768\n",
      "769\n",
      "770\n",
      "771\n",
      "772\n",
      "773\n",
      "774\n",
      "775\n",
      "776\n",
      "777\n",
      "778\n",
      "779\n",
      "780\n",
      "781\n",
      "782\n",
      "783\n",
      "784\n",
      "785\n",
      "786\n",
      "787\n",
      "788\n",
      "789\n",
      "790\n",
      "791\n",
      "792\n",
      "793\n",
      "794\n",
      "795\n",
      "796\n",
      "\n",
      "Kaya AI, Lokits AD, Gilbert JA, Iverson TM, Meiler J, Hamm HE (2016) A Conserved Hydrophobic\n",
      "Core in Galphai1 Regulates G Protein Activation and Release from Activated Receptor. J Biol\n",
      "Chem 291: 19674-19686\n",
      "Kroon J, Heemskerk N, Kalsbeek MJT, de Waard V, van Rijssel J, van Buul JD (2017) Flow-\n",
      "induced endothelial cell alignment requires the RhoGEF Trio as a scaffold protein to polarize\n",
      "active Rac1 distribution. Mol Biol Cell 28: 1745-1753\n",
      "Krumins AM, Gilman AG (2006) Targeted knockdown of G protein subunits selectively prevents\n",
      "receptor-mediated modulation of effectors and reveals complex changes in non-targeted\n",
      "signaling proteins. J Biol Chem 281: 10250-10262\n",
      "Lawrence C (2016) New frontiers for zebrafish management. Methods Cell Biol 135: 483-508\n",
      "Lee MJ, Evans M, Hla T (1996) The inducible G protein-coupled receptor edg-1 signals via the\n",
      "G(i)/mitogen-activated protein kinase pathway. J Biol Chem 271: 11272-11279\n",
      "Lee SC, Knowles TJ, Postis VL, Jamshad M, Parslow RA, Lin YP, Goldman A, Sridhar P, Overduin\n",
      "M, Muench SP et al (2016) A method for detergent-free isolation of membrane proteins in their\n",
      "local lipid environment. Nat Protoc 11: 1149-1162\n",
      "Liu SL, Sheng R, Jung JH, Wang L, Stec E, O'Connor MJ, Song S, Bikkavilli RK, Winn RA, Lee D et al\n",
      "(2017) Orthogonal lipid sensors identify transbilayer asymmetry of plasma membrane\n",
      "cholesterol. Nat Chem Biol 13: 268-274\n",
      "Lucitti JL, Jones EA, Huang C, Chen J, Fraser SE, Dickinson ME (2007) Vascular remodeling of the\n",
      "mouse yolk sac requires hemodynamic force. Development 134: 3317-3326\n",
      "Maleszewska M, Vanchin B, Harmsen MC, Krenning G (2016) The decrease in histone\n",
      "methyltransferase EZH2 in response to fluid shear stress alters endothelial gene expression and\n",
      "promotes quiescence. Angiogenesis 19: 9-24\n",
      "Medina R, Grishina G, Meloni EG, Muth TR, Berlot CH (1996) Localization of the effector-\n",
      "specifying regions of Gi2alpha and Gqalpha. J Biol Chem 271: 24720-24727\n",
      "Nazarko O, Kibrom A, Winkler J, Leon K, Stoveken H, Salzman G, Merdas K, Lu Y, Narkhede P,\n",
      "Tall G et al (2018) A Comprehensive Mutagenesis Screen of the Adhesion GPCR Latrophilin-\n",
      "1/ADGRL1. iScience 3: 264-278\n",
      "Ngai J, Methi T, Andressen KW, Levy FO, Torgersen KM, Vang T, Wettschureck N, Tasken K\n",
      "(2008) The heterotrimeric G-protein alpha-subunit Galphaq regulates TCR-mediated immune\n",
      "responses through an Lck-dependent pathway. Eur J Immunol 38: 3208-3218\n",
      "Papasergi MM, Patel BR, Tall GG (2015) The G protein alpha chaperone Ric-8 as a potential\n",
      "therapeutic target. Mol Pharmacol 87: 52-63\n",
      "Privratsky JR, Newman DK, Newman PJ (2010) PECAM-1: conflicts of interest in inflammation.\n",
      "Life Sci 87: 69-82\n",
      "Sando R, Jiang X, Sudhof TC (2019) Latrophilin GPCRs direct synapse specificity by coincident\n",
      "binding of FLRTs and teneurins. Science 363\n",
      "Sehnert AJ, Huq A, Weinstein BM, Walker C, Fishman M, Stainier DY (2002) Cardiac troponin T is\n",
      "essential in sarcomere assembly and cardiac contractility. Nat Genet 31: 106-110\n",
      "Strohbach A, Pennewitz M, Glaubitz M, Palankar R, Gross S, Lorenz F, Materzok I, Rong A, Busch\n",
      "MC, Felix SB et al (2018) The apelin receptor influences biomechanical and morphological\n",
      "properties of endothelial cells. J Cell Physiol 233: 6250-6261\n",
      "\n",
      "\fSugden WW, Meissner R, Aegerter-Wilmsen T, Tsaryk R, Leonard EV, Bussmann J, Hamm MJ,\n",
      "Herzog W, Jin Y, Jakobsson L et al (2017) Endoglin controls blood vessel diameter through\n",
      "endothelial cell shape changes in response to haemodynamic cues. Nat Cell Biol 19: 653-665\n",
      "Tesmer VM, Kawano T, Shankaranarayanan A, Kozasa T, Tesmer JJ (2005) Snapshot of activated\n",
      "G proteins at the membrane: the Galphaq-GRK2-Gbetagamma complex. Science 310: 1686-\n",
      "1690\n",
      "Tirziu D, Moodie KL, Zhuang ZW, Singer K, Helisch A, Dunn JF, Li W, Singh J, Simons M (2005)\n",
      "Delayed arteriogenesis in hypercholesterolemic mice. Circulation 112: 2501-2509\n",
      "Tzima E, Irani-Tehrani M, Kiosses WB, Dejana E, Schultz DA, Engelhardt B, Cao G, DeLisser H,\n",
      "Schwartz MA (2005) A mechanosensory complex that mediates the endothelial cell response to\n",
      "fluid shear stress. Nature 437: 426-431\n",
      "Vizurraga A, Adhikari R, Yeung J, Yu M, Tall GG (2020) Mechanisms of adhesion G protein-\n",
      "coupled receptor activation. J Biol Chem 295: 14065-14083\n",
      "Vogel J, Niederer D, Jung G, Troidl K (2020) Exercise-Induced Vascular Adaptations under\n",
      "Artificially Versus Pathologically Reduced Blood Flow: A Focus Review with Special Emphasis on\n",
      "Arteriogenesis. Cells 9\n",
      "Wang S, Iring A, Strilic B, Albarran Juarez J, Kaur H, Troidl K, Tonack S, Burbiel JC, Muller CE,\n",
      "Fleming I et al (2015) P2Y(2) and Gq/G(1)(1) control blood pressure by mediating endothelial\n",
      "mechanotransduction. J Clin Invest 125: 3077-3086\n",
      "Wilson PT, Bourne HR (1995) Fatty acylation of alpha z. Effects of palmitoylation and\n",
      "myristoylation on alpha z signaling. J Biol Chem 270: 9667-9675\n",
      "Xu J, Mathur J, Vessieres E, Hammack S, Nonomura K, Favre J, Grimaud L, Petrus M, Francisco A,\n",
      "Li J et al (2018) GPR68 Senses Flow and Is Essential for Vascular Physiology. Cell 173: 762-775\n",
      "e716\n",
      "Yamamoto K, Ando J (2013) Endothelial cell and model membranes respond to shear stress by\n",
      "rapidly decreasing the order of their lipid phases. J Cell Sci 126: 1227-1234\n",
      "Yamamoto K, Ando J (2015) Vascular endothelial cell membranes differentiate between stretch\n",
      "and shear stress through transitions in their lipid phases. Am J Physiol Heart Circ Physiol 309:\n",
      "H1178-1185\n",
      "Yamamoto K, Nogimori Y, Imamura H, Ando J (2020) Shear stress activates mitochondrial\n",
      "oxidative phosphorylation by reducing plasma membrane cholesterol in vascular endothelial\n",
      "cells. Proc Natl Acad Sci U S A 117: 33660-33667\n",
      "Yeh JC, Otte LA, Frangos JA (2008) Regulation of G protein-coupled receptor activities by the\n",
      "platelet-endothelial cell adhesion molecule, PECAM-1. Biochemistry 47: 9029-9039\n",
      "Zhou J, Li YS, Chien S (2014) Shear stress-initiated signaling and its regulation of endothelial\n",
      "function. Arterioscler Thromb Vasc Biol 34: 2191-2198\n",
      "Zhuang ZW, Shi J, Rhodes JM, Tsapakos MJ, Simons M (2011) Challenging the surgical rodent\n",
      "hindlimb ischemia model with the miniinterventional technique. J Vasc Interv Radiol 22: 1437-\n",
      "1446\n",
      "\n",
      "797\n",
      "798\n",
      "799\n",
      "800\n",
      "801\n",
      "802\n",
      "803\n",
      "804\n",
      "805\n",
      "806\n",
      "807\n",
      "808\n",
      "809\n",
      "810\n",
      "811\n",
      "812\n",
      "813\n",
      "814\n",
      "815\n",
      "816\n",
      "817\n",
      "818\n",
      "819\n",
      "820\n",
      "821\n",
      "822\n",
      "823\n",
      "824\n",
      "825\n",
      "826\n",
      "827\n",
      "828\n",
      "829\n",
      "830\n",
      "831\n",
      "832\n",
      "833\n",
      "834\n",
      "835\n",
      "\n",
      "836\n",
      "837\n",
      "\n",
      "\f838\n",
      "\n",
      "839\n",
      "\n",
      "840\n",
      "\n",
      "841\n",
      "\n",
      "842\n",
      "\n",
      "843\n",
      "\n",
      "844\n",
      "\n",
      "845\n",
      "\n",
      "846\n",
      "\n",
      "847\n",
      "\n",
      "848\n",
      "\n",
      "849\n",
      "\n",
      "850\n",
      "\n",
      "851\n",
      "\n",
      "852\n",
      "\n",
      "853\n",
      "\n",
      "854\n",
      "\n",
      "855\n",
      "\n",
      "856\n",
      "\n",
      "857\n",
      "\n",
      "858\n",
      "\n",
      "859\n",
      "\n",
      "860\n",
      "\n",
      "861\n",
      "\n",
      "862\n",
      "\n",
      "863\n",
      "\n",
      "864\n",
      "\n",
      "Figure 1. Gα proteins specific for endothelial flow responses\n",
      "\n",
      "A, Flow-induced endothelial alignment after Gα knockdown.  HUVECs were subjected to fluid\n",
      "shear  stress (FSS) at a  rate of 12 dynes/cm2  for  16 hours and  nuclear orientation  quantified  as\n",
      "\n",
      "histograms showing the percentage of cells within each 10° of the flow directions from 0° to 90°\n",
      "\n",
      "(see  Methods)  ****:  p<0.0001;  one-way  ANOVA  with  Tukey’s  multiple  comparisons  test.  B,\n",
      "\n",
      "Src family kinase activation, quantified in C. n=5 for control, Gi knockdown, Gq11 knockdown\n",
      "\n",
      "and  n=3  for  simultaneous  knockdown  of  Gi  and  Gq11.    Values  are  means  ±  SEM.  ****:\n",
      "\n",
      "p<0.0001, ***: p<0.001; one-way ANOVA with Tukey multiple comparison test.  D, Rescue of\n",
      "\n",
      "Gq/11 and Gi knockdown by re-expression of siRNA-resistant versions of the indicated proteins.\n",
      "\n",
      "****:  p<0.0001;  one-way  ANOVA  with  Tukey’s  multiple  comparison  test.  E,  Amino  acid\n",
      "\n",
      "sequences  of  Gαi1,  Gαi2,  and  Gαi3  at  the  mutation  sites  of  Gαi1  gain-of-function  mutant.    F,\n",
      "\n",
      "Rescue of Gq/11 and Gi knockdown with indicated Gα proteins. Each point corresponds to one\n",
      "\n",
      "measurement averaged from >500 cells. N=4. ****: p<0.0001; one-way ANOVA with Tukey’s\n",
      "\n",
      "multiple  comparisons  test.    G,  GINIP  pulldown  assay  for  activation  of  Gαi2  by  FSS.    N=3.\n",
      "\n",
      "Results quantified in H.  **: p=0.0185, Student’s t-test.  I, GRK2N pulldown assay for activation\n",
      "\n",
      "of  Gq.    N=4,  quantified  in  J.  *:  p<0.05,  Student’s  t-test.  K,  GINIP  pulldown  assay  for  FSS-\n",
      "\n",
      "induced  activation  of  wild  type  and  Q306K  Gαi1.    N  =  4,  quantified  in  L.  *:  p=0.0304,  ***:\n",
      "\n",
      "p=0.0017; Student’s t-test. M, Gi2 activation after PECAM-1 knockdown. HUVECs expressing\n",
      "\n",
      "GluGlu  tagged  Gi2  were  transfected  with  scrambled  siRNA  or  PECAM-1  siRNA,  exposed  to\n",
      "\n",
      "FSS, and Gi2 activation assayed as described above, quantified in N. Values are means ± SEM,\n",
      "\n",
      "normalized to input Gα protein levels. ***: p<0.001; Student’s t-test, N=4.\n",
      "\n",
      "Figure 2. Identifiation of the upstream flow-responsive GPCR\n",
      "\n",
      "A, Strategy for identification of GPCRs that bind the gain-of-function Gi1Q306K mutant but not\n",
      "\n",
      "the wild type Gi1.  B, Co-immunoprecipitation of Gi1Q306K(ins4A) with endogenous LPHN2\n",
      "\n",
      "with  and  without  FSS  for  2  min.  N=3,  quantified  in  C.  *:  p=0.0374,  Student’s  t-test.  D,  Co-\n",
      "\n",
      "immunoprecipitation of LPHN2-GFP with the indicated Gα proteins containing internal GluGlu\n",
      "\n",
      "\f865\n",
      "\n",
      "866\n",
      "\n",
      "867\n",
      "\n",
      "868\n",
      "\n",
      "869\n",
      "\n",
      "870\n",
      "\n",
      "871\n",
      "\n",
      "872\n",
      "\n",
      "873\n",
      "\n",
      "874\n",
      "\n",
      "875\n",
      "\n",
      "876\n",
      "\n",
      "877\n",
      "\n",
      "878\n",
      "\n",
      "879\n",
      "\n",
      "880\n",
      "\n",
      "881\n",
      "\n",
      "882\n",
      "\n",
      "883\n",
      "\n",
      "epitope tags.  E, Gi2 pulldown assay after LPHN2 knockdown.  N= 4. Results quantified in F.\n",
      "\n",
      "***: p<0.001, Student’s t-test.\n",
      "\n",
      "Figure 3. Latrophilins regulate endothelial flow responses\n",
      "\n",
      "A, HUVECs with or without latrophilin-2 knockdown were subjected to FSS for 16 h, then fixed\n",
      "\n",
      "and  stained  with  Hoechst  (nuclei),  phalloidin  (F-actin)  and  an  antibody  against  VE-Cadherin.\n",
      "\n",
      "Scale bar: 100µm.  B, Alignment of HUVECs (each bar = 10° increments) after knockdown of\n",
      "\n",
      "latrophilin  isoforms  was  quantified  as  in  Fig.  1  from  >2000  cells/experiment,  N=3.    ****:\n",
      "\n",
      "p<0.0001;  one-way  ANOVA  with  Tukey’s  multiple  comparisons  test.  C,  Localization  of\n",
      "\n",
      "LPHN2-mClover3. Scale bar: 100µm. N=6. D, Gi1(Q306K) pulldown from ECs ± FSS for 5 min\n",
      "\n",
      "or for 24 hours, probed for PECAM-1. N=3, quantified in E. F, Gi1(Q306K) pulldown from ECs\n",
      "\n",
      "± FSS for 5 min or for 24 hours probed for VE-cadherin. N=3, quantified in G. H, Activation of\n",
      "\n",
      "Src  family  kinases  and  Akt  by  FSS  in  HUVECs  depleted  for  latrophilin  isoforms.  N=3-4,\n",
      "\n",
      "quantified  in  I  &  J.  *:  p=0.0184,  **:  p<0.001;  one-way  ANOVA  with  Tukey’s  multiple\n",
      "\n",
      "comparisons  test.  K,  Alignment  of  LPHN2-depleted  HUVECs  rescued  by  re-expression  of  the\n",
      "\n",
      "indicated  latrophilin  isoforms.  ****:  p<0.0001;  one-way  ANOVA  with  Tukey’s  multiple\n",
      "\n",
      "comparisons test. Quantification of >2000 cells for each condition, N=3. L, LPHN2 knockdown\n",
      "\n",
      "HUVECs were rescued by re-expression of the indicated LPHN2 mutants. ****: p<0.0001, one-\n",
      "\n",
      "way  ANOVA  with  Tukey’s  multiple  comparisons  test.  Quantification  of  >500  cells  for  each\n",
      "\n",
      "884\n",
      "\n",
      "condition.\n",
      "\n",
      "885\n",
      "\n",
      "886\n",
      "\n",
      "887\n",
      "\n",
      "888\n",
      "\n",
      "889\n",
      "\n",
      "890\n",
      "\n",
      "891\n",
      "\n",
      "Figure 4, Latrophilins in flow-mediated endothelial cell morphology in vivo\n",
      "\n",
      "A.  Representative  image  of  HUVECs  transfected  with  scrambled  siRNA  or  LPHN2  siRNA.\n",
      "\n",
      "Outlines of ECs from VE-cadherin staining are shown to next to each image. Scale Bar: 100µm.\n",
      "\n",
      "B,  Quantification  of  junctional  linearization  index  from  A.  n=30  for  each  condition.    **:\n",
      "\n",
      "p=0.0019, Statistics is done using two-way ANOVA with Sidak’s multiple comparison test. C,\n",
      "\n",
      "Dorsal aortas from 48hpf Tg(kdrl:ras-mCherry) WT or LPHN mutant zebrafish embryos stained\n",
      "\n",
      "\f892\n",
      "\n",
      "893\n",
      "\n",
      "894\n",
      "\n",
      "895\n",
      "\n",
      "896\n",
      "\n",
      "897\n",
      "\n",
      "898\n",
      "\n",
      "899\n",
      "\n",
      "900\n",
      "\n",
      "901\n",
      "\n",
      "902\n",
      "\n",
      "903\n",
      "\n",
      "for ZO-1. EC were outlined as in Methods. LPHN2 mutant indicates adgrl2a+adgrl2b.1 sgRNAs.\n",
      "\n",
      "D&E,  Quantification  of  cell  size  and  eccentricity  in  embryos  with  latrophilin-2  CRISPR\n",
      "\n",
      "sgRNAs ± silent heart morpholinos (MO). N=6 per condition.  ****: p<0.0001, *: p=0.0267, **:\n",
      "\n",
      "p=0.0015, ***: p=0.0008; one-way ANOVA with Tukey’s multiple comparisons test.  F, ECs in\n",
      "\n",
      "the  thoracic  aorta  in  18  weeks  old  mice  at  10  weeks  after  tamoxifen  injections,  stained  for  β-\n",
      "\n",
      "catenin  (left)  and  outlined  (right).    Data  are  representative  of  6  mice  for  each  condition.  Scale\n",
      "\n",
      "Bar: 100µm. G&H, quantification of cell eccentricity and junction linearity in mouse aorta.  N=4\n",
      "\n",
      "per condition. **: p=0.0013; ****: p<0.0001; Student’s t-test. I, Aortas from mice 30 min after\n",
      "\n",
      "Evans blue dye injection, thresholded images on the right. Fractional blue area quantified in J.\n",
      "\n",
      "n=3 for each condition.  *: p=0.0419; Student’s t-test. K, in vitro endothelial permeability assay\n",
      "\n",
      "using  FITC-streptavidin  and  biotin-conjugated  fibronectin.  Scale  Bar:  100µm.  L,  Permeability\n",
      "\n",
      "was quantified by measuring FITC-streptavidin area per image field from K. n=5, *: p=0.0457,\n",
      "\n",
      "904\n",
      "\n",
      "one-way ANOVA.\n",
      "\n",
      "905\n",
      "\n",
      "906\n",
      "\n",
      "907\n",
      "\n",
      "908\n",
      "\n",
      "909\n",
      "\n",
      "910\n",
      "\n",
      "911\n",
      "\n",
      "912\n",
      "\n",
      "913\n",
      "\n",
      "914\n",
      "\n",
      "915\n",
      "\n",
      "Figure 5. Cholesterol depletion activates Latrophilin-dependent signaling\n",
      "\n",
      "A, ECs were treated with FSS or 5 mM methyl-β-cyclodextrin (mβCD) for 1 min then incubated\n",
      "\n",
      "with D4H-mClover3, rinsed and bound mClover3 quantified as described in Methods. N=27 for\n",
      "\n",
      "each condition.   ****:  p<0.0001; One-way ANOVA  with  Tukey multiple comparisons test.  B,\n",
      "\n",
      "Phalloidin staining following short acute treatment of methyl-β-cyclodextrin (mβCD).  Scale bar:\n",
      "\n",
      "100µm.  C,  Co-immunoprecipitation  of  Gi1Q306K(ins4A)  with  endogenous  Latrophilin-2\n",
      "\n",
      "(LPHN2)  after  5mM  MβCD  for  1min.  D,  Activation  of  Src  family  kinases  and  VEGFR2  after\n",
      "\n",
      "5mM  MβCD  for  1min  in  HUVECs  ±  LPHN2  knockdown.  Results  quantified  in  E  and  F,\n",
      "\n",
      "respectively.  Values  are  means  ±  SEM.  N=5.  *:  p=0.0305,  ***:  p=0.0002;  one-way  ANOVA\n",
      "\n",
      "with  Tukey  multiple  comparison  test.    G,  Model  for  junctional  endothelial  shear  stress\n",
      "\n",
      "916\n",
      "\n",
      "mechanotransduction.\n",
      "\n",
      "917\n",
      "\n",
      "918\n",
      "\n",
      "Figure 6. Latrophilin-2 in flow-induced vascular remodeling\n",
      "\n",
      "\fA,  Representative  images  of  ISVs  from  48  hpf  embryos  with  F(0)/mosaic  depletions  of\n",
      "\n",
      "latrophilin genes or controls.  B, Quantification of diameters of all ISVs in A. n=15 per condition.\n",
      "\n",
      "*:  p=0.0452,  **:  p=0.0150;  one-way  ANOVA  with  Tukey’s  multiple  comparisons  test.  C,\n",
      "\n",
      "Quantification of diameters of arterial and venous ISVs at 72 hpf of zebrafish. n=6 per condition.\n",
      "\n",
      "**: p=0.0013; Student’s t-test. D, Ratio of blood flow in the right ischemic vs left control foot by\n",
      "\n",
      "laser doppler, quantified in E.  N=7-8 per condition. *#: p=0.0439, *: p=0.0135, **: p=0.0028,\n",
      "\n",
      "##:  p=0.0056,  #:  p=0.0163;  Student’s  t-test.  F,  Micro-CT  of  arterial  vasculature  after  surgery.\n",
      "\n",
      "Representative reconstructed micro-CT images from LPHN2 ECKO and WT littermates on day\n",
      "\n",
      "22  after  surgery,  quantified  in  G.  n=8  per  condition.  ****:  p<0.0001,  ***:  p=0.0007,  *:\n",
      "\n",
      "p=0.0167; two-way ANOVA with Tukey’s multiple comparisons test.  H, Rectus femoris muscle\n",
      "\n",
      "from thighs of 10 weeks old mice of LPHN2 ECKO or wild-type littermates, stained for CD31\n",
      "\n",
      "and SMA to visualize the entire vasculature and arteries. Scale Bar: 100µm. I. Quantification of\n",
      "\n",
      "capillary density in the entire stitched thigh muscle images.  n=3 for control and n=6 for Lphn2\n",
      "\n",
      "ECKO.  ****:  p<0.0001;  Student’s  t-test.  J,  Schematic  of  mouse  treadmill  fatigue  test.    K,\n",
      "\n",
      "Maximum  running  during  test.  n=4  for  control  and  n=5  for  Lphn2  ECKO.    **:p=0.0082;\n",
      "\n",
      "Student’s t-test.  L, Running distance for each mouse.  n=4 for control and n=5 for Lphn2 ECKO.\n",
      "\n",
      "**:  p=0.0064;  Student’s  t-test.  M,  Oxygen  consumption  during  treadmill  fatigue  test.    *:\n",
      "\n",
      "p<0.0001;   two-way  ANOVA  with  Tukey’s  multiple comparisons  test.   Control:  N=5,  LPHN2\n",
      "\n",
      "ECKO: N=4.  N, Representative 3D reconstructed images at 20° downward angle of ECs plated\n",
      "\n",
      "on  top  of  collagen  gels  and  sprouting  under  static  or  flow  conditions.    ECs  were  stained  with\n",
      "\n",
      "phalloidin; pseudocolors indicate distance from the gel surface as per the color scale on the right.\n",
      "\n",
      "O, Histograms of cell area vs. distance from gel surface in N.  P, Mean sprout length.  N=4.  ***:\n",
      "\n",
      "p=0.0006; one-way ANOVA with Tukey’s multi comparison test.\n",
      "\n",
      "919\n",
      "\n",
      "920\n",
      "\n",
      "921\n",
      "\n",
      "922\n",
      "\n",
      "923\n",
      "\n",
      "924\n",
      "\n",
      "925\n",
      "\n",
      "926\n",
      "\n",
      "927\n",
      "\n",
      "928\n",
      "\n",
      "929\n",
      "\n",
      "930\n",
      "\n",
      "931\n",
      "\n",
      "932\n",
      "\n",
      "933\n",
      "\n",
      "934\n",
      "\n",
      "935\n",
      "\n",
      "936\n",
      "\n",
      "937\n",
      "\n",
      "938\n",
      "\n",
      "939\n",
      "\n",
      "940\n",
      "\n",
      "941\n",
      "\n",
      "942\n",
      "943\n",
      "\n",
      "\f944\n",
      "\n",
      "Table 1: Primer sequences used for quantitative PCR in this study\n",
      "\n",
      "Gene name\n",
      "\n",
      "Forward (5'-3')\n",
      "\n",
      "Reverse (5'-3')\n",
      "\n",
      "Human LPHN1\n",
      "\n",
      "CAGTACGACTGTGTCCCCTACA\n",
      "\n",
      "GCACCATGCGCCAGACTG\n",
      "\n",
      "Human LPHN2\n",
      "\n",
      "GTCCAATATGAATGTGTCCCTTACA\n",
      "\n",
      "GCACCAAGCACCCGCCTT\n",
      "\n",
      "Human LPHN3\n",
      "\n",
      "GTGCAGTATGAATGTGTCCCTTACA\n",
      "\n",
      "GCACCACGCCCCAGATTG\n",
      "\n",
      "Human KLF2\n",
      "\n",
      "CGGCAAGACCTACACCAAGA\n",
      "\n",
      "TGGTAGGGCTTCTCACCTGT\n",
      "\n",
      "Human GAPDH\n",
      "\n",
      "TGCACCACCAACTGCTTAGC\n",
      "\n",
      "GGCATGGACTGTGGTCATGAG\n",
      "\n",
      "zebrafish LPHN1a\n",
      "\n",
      "CAGTATGAATGTGTGCCATACA\n",
      "\n",
      "ACACCATGCCCCCGACTG\n",
      "\n",
      "zebrafish LPHN2a\n",
      "\n",
      "GTCCAGTATGAATGTGTTCCCTACA\n",
      "\n",
      "GCACCACGACCCCGCTTG\n",
      "\n",
      "zebrafish LPHN2b.1\n",
      "\n",
      "GTCCAGTATGAGTGTGTGCCATACA\n",
      "\n",
      "TTGCACCAAGCTCCACTC\n",
      "\n",
      "zebrafish LPHN3.1\n",
      "\n",
      "GTTCAGTATGAGTGTGTGCCATACA\n",
      "\n",
      "ACACCATGATCCAGCCTG\n",
      "\n",
      "zebrafish beta-actin\n",
      "\n",
      "GATCTGGCATCACACCTTCTAC\n",
      "\n",
      "TCTTCTCTCTGTTGGCTTTGG\n",
      "\n",
      "Table 2: Sequences of sgRNAs and morpholinos used in this study\n",
      "\n",
      "Sequence\n",
      "\n",
      "Target\n",
      "\n",
      "Description\n",
      "\n",
      "5'  GCTGTACGAGTATGCTTCATGGG\n",
      "\n",
      "adgrl1a\n",
      "\n",
      "sgRNA targeting adgrl1a exon 4\n",
      "\n",
      "5'  CGACGTATAAACTACCCCATCGG\n",
      "\n",
      "adgrl2a\n",
      "\n",
      "sgRNA targeting adgrl2a exon 4\n",
      "\n",
      "5'  CCGTCTATGATAAACGCTCCGCC\n",
      "\n",
      "adgrl2b.1\n",
      "\n",
      "sgRNA targeting adgrl2b.1 exon 4\n",
      "\n",
      "5'  AGTCCACGGCTGCGATGTATTGG\n",
      "\n",
      "adgrl3.1\n",
      "\n",
      "sgRNA targeting adgrl3.1 exon 4\n",
      "\n",
      "5'  CATGTTTGCTCTGATCTGACACGCA\n",
      "\n",
      "tnnt2a\n",
      "\n",
      "morpholino targeting tnnt2a translation\n",
      "\n",
      "5'  CCTCTTACCTCAGTTACAATTTATA\n",
      "\n",
      "none\n",
      "\n",
      "control morpholino\n",
      "\n",
      "945\n",
      "\n",
      "946\n",
      "\n",
      "947\n",
      "\n",
      "948\n",
      "\n",
      "949\n",
      "\n",
      "\fns\n",
      "\n",
      "B\n",
      "\n",
      "40\n",
      "\n",
      "30\n",
      "\n",
      "20\n",
      "\n",
      "10\n",
      "\n",
      "A\n",
      "\n",
      ")\n",
      "\n",
      "%\n",
      "\n",
      "(\n",
      "y\n",
      "c\n",
      "n\n",
      "e\n",
      "u\n",
      "q\n",
      "e\n",
      "r\n",
      "f\n",
      "e\n",
      "v\n",
      "i\n",
      "t\n",
      "a\n",
      "e\n",
      "R\n",
      "\n",
      "l\n",
      "\n",
      "0\n",
      "\n",
      "Flow\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "siRNA\n",
      "\n",
      "Scrambled\n",
      "\n",
      "Gi\n",
      "\n",
      "-\n",
      "+\n",
      "Gq/11\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "Gi+Gq/11\n",
      "\n",
      "D\n",
      "\n",
      "Scrambled\n",
      "\n",
      "Gi\n",
      "knockdown\n",
      "\n",
      "Gq/11\n",
      "knockdown\n",
      "\n",
      "Gi&Gq/11\n",
      "knockdown\n",
      "\n",
      "C\n",
      "\n",
      "0 0.5 2\n",
      "\n",
      "0 0.5 2\n",
      "\n",
      "0 0.5 2\n",
      "\n",
      "0 0.5 2\n",
      "\n",
      "c\n",
      "r\n",
      "S\n",
      "/\n",
      "c\n",
      "r\n",
      "S\n",
      "p\n",
      "\n",
      "Shear\n",
      "Stress\n",
      "(min.)\n",
      "\n",
      "P-Src\n",
      "\n",
      "Src\n",
      "\n",
      "Gi\n",
      "\n",
      "Gq/11\n",
      "\n",
      "actin\n",
      "\n",
      "l\n",
      "\n",
      ")\n",
      "c\n",
      "i\n",
      "t\n",
      "a\n",
      "t\n",
      "s\n",
      "d\n",
      "e\n",
      "b\n",
      "m\n",
      "a\n",
      "r\n",
      "c\n",
      "S\n",
      "o\n",
      "t\n",
      "d\n",
      "e\n",
      "z\n",
      "\n",
      "i\n",
      "l\n",
      "\n",
      "a\n",
      "m\n",
      "r\n",
      "o\n",
      "n\n",
      "(\n",
      "\n",
      "5\n",
      "\n",
      "4\n",
      "\n",
      "3\n",
      "\n",
      "2\n",
      "\n",
      "1\n",
      "\n",
      "0\n",
      "\n",
      "ns\n",
      "\n",
      "0 0.5 2\n",
      "Scrambled Gi KD\n",
      "\n",
      "0 0.5 2 0 0.5 2\n",
      "Gq/11 KD\n",
      "\n",
      "0 0.5 2\n",
      "Gi&Gq/11\n",
      "KD\n",
      "\n",
      "ns\n",
      "\n",
      "ns\n",
      "\n",
      "ns\n",
      "\n",
      "30\n",
      "\n",
      "20\n",
      "\n",
      "10\n",
      "\n",
      ")\n",
      "\n",
      "%\n",
      "\n",
      "(\n",
      "y\n",
      "c\n",
      "n\n",
      "e\n",
      "u\n",
      "q\n",
      "e\n",
      "r\n",
      "f\n",
      "e\n",
      "v\n",
      "i\n",
      "t\n",
      "a\n",
      "e\n",
      "R\n",
      "\n",
      "l\n",
      "\n",
      "0\n",
      "Flow -\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "Scrambled\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "+Gq\n",
      "\n",
      "-\n",
      "+\n",
      "+Gi1\n",
      "\n",
      "-\n",
      "+\n",
      "+Gi2\n",
      "\n",
      "Gi & Gq/11 knockdown\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "+Gi3\n",
      "\n",
      "E\n",
      "\n",
      "Gq 166 RVADPAYLPTQ 176\n",
      "162 RIAQSDYIPTQ 172\n",
      "Gi2\n",
      "161 RISQSNYIPTQ 171\n",
      "Gi3\n",
      "161 RIAQPNYIPTQ 171\n",
      "Gi1\n",
      "Gi1(N166D)\n",
      "161 -----D----- 171\n",
      "\n",
      "Gq 307 EYILKMFVDLN 317\n",
      "302 SYIQSKFEDLN 312\n",
      "Gi2\n",
      "301 AYIQCQFEDLN 311\n",
      "Gi3\n",
      "301 AYIQCQFEDLN 311\n",
      "Gi1\n",
      "Gi1(Q306K)\n",
      "301 -----K----- 311\n",
      "\n",
      "F\n",
      "\n",
      "ns\n",
      "\n",
      "ns\n",
      "\n",
      "ns\n",
      "\n",
      "60\n",
      "\n",
      "40\n",
      "\n",
      "20\n",
      "\n",
      ")\n",
      "\n",
      "%\n",
      "\n",
      "(\n",
      ")\n",
      "s\n",
      "e\n",
      "e\n",
      "r\n",
      "g\n",
      "e\n",
      "d\n",
      "0\n",
      "3\n",
      "-\n",
      "0\n",
      "(\n",
      "\n",
      "s\n",
      "\n",
      "l\n",
      "l\n",
      "\n",
      "e\n",
      "c\n",
      "d\n",
      "e\n",
      "n\n",
      "g\n",
      "\n",
      "i\n",
      "l\n",
      "\n",
      "A\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "\n",
      "0\n",
      "F l o\n",
      "\n",
      "w\n",
      "\n",
      "S cra m bled\n",
      "\n",
      "G\n",
      "\n",
      "12 dynes/cm2 laminar\n",
      "30s\n",
      "\n",
      "2 min\n",
      "\n",
      "0\n",
      "\n",
      "GST-pulldown\n",
      "IB: GluGlu\n",
      "Lysate\n",
      "IB: GluGlu\n",
      "Lysate\n",
      "IB: actin\n",
      "GST-pulldown\n",
      "IB: GST\n",
      "\n",
      "I\n",
      "\n",
      "12 dynes/cm2 laminar\n",
      "\n",
      "1 min\n",
      "\n",
      "2 min\n",
      "\n",
      "0\n",
      "\n",
      "GST-pulldown\n",
      "IB: GluGlu\n",
      "Lysate\n",
      "IB: GluGlu\n",
      "GST-pulldown\n",
      "IB: GST\n",
      "Lysate\n",
      "IB: tubulin\n",
      "\n",
      "H\n",
      "\n",
      "5\n",
      "\n",
      "4\n",
      "\n",
      "3\n",
      "\n",
      "2\n",
      "\n",
      "1\n",
      "\n",
      "0\n",
      "\n",
      "0 30s 2min\n",
      "\n",
      "n\n",
      "w\n",
      "o\n",
      "d\n",
      "\n",
      "l\n",
      "l\n",
      "\n",
      "u\n",
      "p\n",
      "2\n",
      "G\n",
      "\n",
      "i\n",
      "\n",
      ")\n",
      "c\n",
      "i\n",
      "t\n",
      "a\n",
      "t\n",
      "s\n",
      "o\n",
      "t\n",
      "d\n",
      "e\n",
      "z\n",
      "\n",
      "i\n",
      "l\n",
      "\n",
      "a\n",
      "m\n",
      "r\n",
      "o\n",
      "n\n",
      "(\n",
      "\n",
      "J\n",
      "\n",
      "4\n",
      "\n",
      "3\n",
      "\n",
      "2\n",
      "\n",
      "1\n",
      "\n",
      "0\n",
      "\n",
      "0\n",
      "\n",
      "m i n\n",
      "\n",
      "1\n",
      "\n",
      "m i n\n",
      "\n",
      "2\n",
      "\n",
      "n\n",
      "w\n",
      "o\n",
      "d\n",
      "\n",
      "l\n",
      "l\n",
      "\n",
      "u\n",
      "p\n",
      "q\n",
      "G\n",
      "\n",
      ")\n",
      "c\n",
      "i\n",
      "t\n",
      "a\n",
      "t\n",
      "s\n",
      "o\n",
      "t\n",
      "d\n",
      "e\n",
      "z\n",
      "\n",
      "i\n",
      "l\n",
      "\n",
      "a\n",
      "m\n",
      "r\n",
      "o\n",
      "n\n",
      "(\n",
      "\n",
      "Gi1EE\n",
      "(Q306K) Gi2EE\n",
      "+\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "\n",
      "Gi1EE\n",
      "+\n",
      "-\n",
      "\n",
      "K\n",
      "\n",
      "12 dynes/cm2\n",
      "laminar 2min\n",
      "\n",
      "GST-pulldown\n",
      "IB:GluGlu\n",
      "Low exposure\n",
      "GST-pulldown\n",
      "IB:GluGlu\n",
      "High exposure\n",
      "Lysate\n",
      "IB:GluGlu\n",
      "Low exposure\n",
      "Lysate\n",
      "IB:GluGlu\n",
      "High exposure\n",
      "Lysate\n",
      "IB:actin\n",
      "\n",
      "GST-pulldown\n",
      "IB:GST\n",
      "\n",
      "-\n",
      "\n",
      "ns\n",
      "\n",
      "L\n",
      "\n",
      "y\n",
      "t\n",
      "i\n",
      "v\n",
      "i\n",
      "t\n",
      "c\n",
      "a\n",
      "e\n",
      "v\n",
      "i\n",
      "t\n",
      "a\n",
      "e\n",
      "R\n",
      "\n",
      "l\n",
      "\n",
      "8\n",
      "\n",
      "6\n",
      "\n",
      "4\n",
      "\n",
      "2\n",
      "\n",
      "0\n",
      "Flow\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "Gi1EE Gi1EE\n",
      "(Q306K)\n",
      "\n",
      "Gi2EE\n",
      "\n",
      "-\n",
      "+\n",
      "+Gi1\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "+\n",
      "+Gi2 +Gi1\n",
      "(N166D)\n",
      "\n",
      "- +\n",
      "+Gi1\n",
      "(Q306K)\n",
      "\n",
      "- +\n",
      "+Gi1\n",
      "(NDQK)\n",
      "\n",
      "Gi & Gq/11 knockdown\n",
      "\n",
      "S cra m bled\n",
      "\n",
      "PECAM-1\n",
      "KD\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "M\n",
      "\n",
      "12 dynes/cm2\n",
      "laminar 2min\n",
      "GST-pulldown\n",
      "IB: GluGlu\n",
      "Lysate\n",
      "IB: GluGlu\n",
      "Lysate\n",
      "IB: PECAM-1\n",
      "Lysate\n",
      "IB: actin\n",
      "GST-pulldown\n",
      "IB: GST\n",
      "\n",
      "N\n",
      "\n",
      "y\n",
      "t\n",
      "i\n",
      "v\n",
      "i\n",
      "t\n",
      "c\n",
      "a\n",
      "e\n",
      "v\n",
      "i\n",
      "t\n",
      "\n",
      "l\n",
      "\n",
      "a\n",
      "e\n",
      "R\n",
      "\n",
      ")\n",
      "\n",
      "w\n",
      "o\n",
      "\n",
      "l\n",
      "f\n",
      "\n",
      "l\n",
      "\n",
      "d\n",
      "e\n",
      "b\n",
      "m\n",
      "a\n",
      "r\n",
      "c\n",
      "S\n",
      "o\n",
      "\n",
      "t\n",
      "\n",
      "d\n",
      "e\n",
      "z\n",
      "\n",
      "i\n",
      "l\n",
      "\n",
      "a\n",
      "m\n",
      "r\n",
      "o\n",
      "n\n",
      "(\n",
      "\n",
      "2.0\n",
      "\n",
      "1.5\n",
      "\n",
      "1.0\n",
      "\n",
      "0.5\n",
      "\n",
      "0.0\n",
      "F l o\n",
      "\n",
      "w\n",
      "\n",
      "c r a\n",
      "\n",
      "S\n",
      "\n",
      "-\n",
      "b l e\n",
      "\n",
      "m\n",
      "\n",
      "+\n",
      "\n",
      "d\n",
      "\n",
      "C\n",
      "\n",
      "E\n",
      "\n",
      "P\n",
      "\n",
      "+\n",
      "M - 1\n",
      "D\n",
      "\n",
      "-\n",
      "\n",
      "A\n",
      "\n",
      "K\n",
      "\n",
      "\fA\n",
      "\n",
      "Pi\n",
      "\n",
      "B\n",
      "\n",
      "C\n",
      "\n",
      "Laminar flow\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "GPCR\n",
      "\n",
      "GFP-Trap\n",
      "IB: LPHN2\n",
      "\n",
      "Membrane\n",
      "IB: LPHN2\n",
      "\n",
      "GFP-Trap\n",
      "IB: GFP\n",
      "(Gi1(Q/K)(ins4A)-GFP)\n",
      "\n",
      "Mass\n",
      "Spectrometry/\n",
      "Protein\n",
      "identification\n",
      "\n",
      "D\n",
      "\n",
      "GFP-Trap\n",
      "IB: GluGlu\n",
      "\n",
      "Lysate\n",
      "IB: GluGlu\n",
      "\n",
      "GFP-Trap\n",
      "IB: GFP\n",
      "\n",
      "Lysate\n",
      "IB: Tubulin\n",
      "\n",
      "Scrambled LPHN2KD\n",
      "\n",
      "F\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      ")\n",
      "\n",
      "w\n",
      "o\n",
      "l\n",
      "f\n",
      "\n",
      "y\n",
      "t\n",
      "i\n",
      "v\n",
      "i\n",
      "t\n",
      "c\n",
      "a\n",
      "e\n",
      "v\n",
      "i\n",
      "t\n",
      "a\n",
      "e\n",
      "R\n",
      "\n",
      "l\n",
      "\n",
      "l\n",
      "\n",
      "o\n",
      "r\n",
      "t\n",
      "n\n",
      "o\n",
      "c\n",
      "o\n",
      "t\n",
      "d\n",
      "e\n",
      "z\n",
      "\n",
      "i\n",
      "l\n",
      "\n",
      "a\n",
      "m\n",
      "r\n",
      "o\n",
      "n\n",
      "(\n",
      "\n",
      "1.5\n",
      "\n",
      "1.0\n",
      "\n",
      "0.5\n",
      "\n",
      "0.0\n",
      "F l o\n",
      "\n",
      "w\n",
      "\n",
      "GPCR\n",
      "\n",
      "ins4A\n",
      "mutation\n",
      "\n",
      "GTP\n",
      "\n",
      "SMA\n",
      "nanodisc\n",
      "\n",
      "E\n",
      "\n",
      "GFP\n",
      "\n",
      "12 dynes/cm2\n",
      "Flow 2min\n",
      "GST-pulldown\n",
      "IB:GluGlu\n",
      "\n",
      "Lysate\n",
      "IB:GluGlu\n",
      "\n",
      "GST-pulldown\n",
      "IB:GST\n",
      "\n",
      "Lysate\n",
      "IB:Tubulin\n",
      "\n",
      ")\n",
      "d\n",
      "e\n",
      "z\n",
      "\n",
      "i\n",
      "l\n",
      "\n",
      "a\n",
      "m\n",
      "r\n",
      "o\n",
      "n\n",
      "(\n",
      "2\n",
      "N\n",
      "H\n",
      "P\n",
      "L\n",
      "\n",
      "1.5\n",
      "\n",
      "1.0\n",
      "\n",
      "0.5\n",
      "\n",
      "0.0\n",
      "\n",
      "Static Flow\n",
      "\n",
      "LPHN2-GFP\n",
      "\n",
      "Gi1 E E\n",
      "\n",
      "Gi1 E E\n",
      "(Q 306 K)\n",
      "\n",
      "Gi2 E E\n",
      "\n",
      "G q E E\n",
      "\n",
      "ns\n",
      "\n",
      "-+\n",
      "\n",
      "-\n",
      "Scr.\n",
      "\n",
      "+\n",
      "LPHN2\n",
      "KD\n",
      "\n",
      "\fA\n",
      "\n",
      "l\n",
      "\n",
      "d\n",
      "e\n",
      "b\n",
      "m\n",
      "a\n",
      "r\n",
      "c\n",
      "S\n",
      "\n",
      "D\n",
      "K\n",
      "2\n",
      "N\n",
      "H\n",
      "P\n",
      "L\n",
      "\n",
      "D\n",
      "\n",
      "Hoechst\n",
      "\n",
      "static\n",
      "\n",
      "Flow direction\n",
      "\n",
      "B\n",
      "\n",
      "25\n",
      "\n",
      "20\n",
      "\n",
      "15\n",
      "\n",
      "10\n",
      "\n",
      "5\n",
      "\n",
      ")\n",
      "\n",
      "%\n",
      "\n",
      "(\n",
      "y\n",
      "c\n",
      "n\n",
      "e\n",
      "u\n",
      "q\n",
      "e\n",
      "r\n",
      "f\n",
      "e\n",
      "v\n",
      "i\n",
      "t\n",
      "a\n",
      "e\n",
      "R\n",
      "\n",
      "l\n",
      "\n",
      "ns\n",
      "\n",
      "C\n",
      "\n",
      "I\n",
      "\n",
      "Phalloidin\n",
      "F\n",
      "\n",
      "24 hours flo w\n",
      "\n",
      "5 min flo w\n",
      "\n",
      "0\n",
      "Flow -\n",
      "+\n",
      "siRNA Scr.\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "+\n",
      "LPHN1 LPHN2 LPHN3\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "\n",
      "-\n",
      "\n",
      "VE-Cadherin\n",
      "\n",
      "24 hours flo w\n",
      "\n",
      "5 min flo w\n",
      "\n",
      "static\n",
      "\n",
      "ns\n",
      "\n",
      "ns\n",
      "\n",
      "3\n",
      "\n",
      "2\n",
      "\n",
      "1\n",
      "\n",
      "S cra m bled\n",
      "\n",
      "L P H N 1 K D\n",
      "\n",
      "L P H N 2 K D\n",
      "\n",
      "L P H N 3 K D\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "J\n",
      "\n",
      "-\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "0\n",
      "F l o w\n",
      "+\n",
      "+\n",
      "s i R N A Scr. LPHN1 LPHN2 LPHN3\n",
      "10\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "\n",
      "-\n",
      "\n",
      "ns\n",
      "\n",
      "H\n",
      "\n",
      "12 dynes/cm2\n",
      "laminar flow 2min\n",
      "Lysate\n",
      "IB:P-Src\n",
      "Lysate\n",
      "IB:Src\n",
      "\n",
      "Lysate\n",
      "IB:P-Akt\n",
      "\n",
      "Lysate\n",
      "IB:tubulin\n",
      "\n",
      "GFP-Trap\n",
      "IB: PECAM-1\n",
      "\n",
      "Membrane\n",
      "IB: PECAM-1\n",
      "\n",
      "GFP-Trap\n",
      "IB: GFP\n",
      "1.5\n",
      "\n",
      "E\n",
      "\n",
      "GFP-Trap\n",
      "IB: VECad\n",
      "\n",
      "Membrane\n",
      "IB: VECad\n",
      "\n",
      "GFP-Trap\n",
      "IB: GFP\n",
      "G\n",
      "\n",
      "1.5\n",
      "\n",
      "d\n",
      "a\n",
      "C\n",
      "E\n",
      "V\n",
      "\n",
      "t\n",
      "u\n",
      "p\n",
      "n\n",
      "\n",
      "i\n",
      "\n",
      "1.0\n",
      "\n",
      "/\n",
      "s\n",
      "n\n",
      "w\n",
      "o\n",
      "d\n",
      "\n",
      "l\n",
      "l\n",
      "\n",
      "u\n",
      "p\n",
      "\n",
      "0.5\n",
      "\n",
      "0.0\n",
      "\n",
      "t\n",
      "u\n",
      "p\n",
      "n\n",
      "\n",
      "i\n",
      "\n",
      "1.0\n",
      "\n",
      "1\n",
      "M\n",
      "A\n",
      "C\n",
      "E\n",
      "P\n",
      "\n",
      "0.5\n",
      "\n",
      "0.0\n",
      "\n",
      "/\n",
      "s\n",
      "n\n",
      "w\n",
      "o\n",
      "d\n",
      "\n",
      "l\n",
      "l\n",
      "\n",
      "u\n",
      "p\n",
      "\n",
      "K\n",
      "\n",
      "static\n",
      "5 min flo w\n",
      "24 hours flo w\n",
      "\n",
      "30\n",
      "\n",
      "ns\n",
      "\n",
      "static\n",
      "5 min flo w\n",
      "24 hours flo w\n",
      "\n",
      "20\n",
      "\n",
      "10\n",
      "\n",
      "0\n",
      "F l o\n",
      "\n",
      "w\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "Scr.\n",
      "\n",
      "-\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "+\n",
      "\n",
      "+\n",
      "-\n",
      "+LPHN1 +LPHN2 +LPHN3\n",
      "LPHN2 knockdown\n",
      "\n",
      "-\n",
      "\n",
      ")\n",
      "\n",
      "%\n",
      "\n",
      "(\n",
      "y\n",
      "c\n",
      "n\n",
      "e\n",
      "u\n",
      "q\n",
      "e\n",
      "r\n",
      "f\n",
      "e\n",
      "v\n",
      "i\n",
      "t\n",
      "a\n",
      "e\n",
      "R\n",
      "\n",
      "l\n",
      "\n",
      "L\n",
      "\n",
      "ns\n",
      "\n",
      "w\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "Scr.\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "\n",
      "25\n",
      "\n",
      "20\n",
      "\n",
      "15\n",
      "\n",
      "10\n",
      "\n",
      "5\n",
      "\n",
      "0\n",
      "F l o\n",
      "\n",
      ")\n",
      "s\n",
      "e\n",
      "g\n",
      "a\n",
      "t\n",
      "n\n",
      "e\n",
      "c\n",
      "r\n",
      "e\n",
      "p\n",
      "(\n",
      "y\n",
      "c\n",
      "n\n",
      "e\n",
      "u\n",
      "q\n",
      "e\n",
      "r\n",
      "f\n",
      "e\n",
      "v\n",
      "i\n",
      "t\n",
      "\n",
      "l\n",
      "\n",
      "a\n",
      "e\n",
      "R\n",
      "\n",
      "-\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "0\n",
      "F l o w\n",
      "+\n",
      "+\n",
      "s i R N A Scr. LPHN1 LPHN2 LPHN3\n",
      "ns\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "\n",
      "-\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "+LPHN2\n",
      "\n",
      "-\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "+\n",
      "+LPHN2\n",
      "(GPCR\n",
      "mut.)\n",
      "\n",
      "+LPHN2\n",
      "(GAIN\n",
      "mut.)\n",
      "LPHN2 knockdown\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "+LPHN2\n",
      "(Helix-8\n",
      "mut.)\n",
      "\n",
      "l\n",
      "\n",
      ")\n",
      "c\n",
      "i\n",
      "t\n",
      "a\n",
      "t\n",
      "s\n",
      "d\n",
      "e\n",
      "b\n",
      "m\n",
      "a\n",
      "r\n",
      "c\n",
      "S\n",
      "o\n",
      "t\n",
      "d\n",
      "e\n",
      "z\n",
      "\n",
      "i\n",
      "l\n",
      "\n",
      "a\n",
      "m\n",
      "r\n",
      "o\n",
      "n\n",
      "(\n",
      "\n",
      "c\n",
      "r\n",
      "S\n",
      "/\n",
      "c\n",
      "r\n",
      "S\n",
      "p\n",
      "\n",
      ")\n",
      "c\n",
      "i\n",
      "t\n",
      "\n",
      "l\n",
      "\n",
      "a\n",
      "t\n",
      "s\n",
      "d\n",
      "e\n",
      "b\n",
      "m\n",
      "a\n",
      "r\n",
      "c\n",
      "S\n",
      "o\n",
      "\n",
      "t\n",
      "\n",
      "d\n",
      "e\n",
      "z\n",
      "\n",
      "i\n",
      "l\n",
      "\n",
      "a\n",
      "m\n",
      "r\n",
      "o\n",
      "n\n",
      "(\n",
      "\n",
      "n\n",
      "\n",
      "i\n",
      "l\n",
      "\n",
      "u\n",
      "b\n",
      "u\n",
      "\n",
      "t\n",
      "/\n",
      "t\n",
      "k\n",
      "A\n",
      "p\n",
      "\n",
      "8\n",
      "\n",
      "6\n",
      "\n",
      "4\n",
      "\n",
      "2\n",
      "\n",
      "\fns\n",
      "\n",
      "ns\n",
      "\n",
      "I\n",
      "\n",
      "wild type\n",
      "\n",
      "Lphn2 ECKO\n",
      "\n",
      "Bright field\n",
      "\n",
      "Blue area\n",
      "\n",
      "Bright field\n",
      "\n",
      "Blue area\n",
      "\n",
      "J\n",
      "\n",
      "A\n",
      "\n",
      "static\n",
      "\n",
      "flow\n",
      "\n",
      "kdrl:ras-mCherry\n",
      "\n",
      "ZO-1\n",
      "\n",
      "Cell shape\n",
      "\n",
      "kdrl:ras-mCherry\n",
      "\n",
      "ZO-1\n",
      "\n",
      "Cell shape\n",
      "\n",
      "hap\n",
      "\n",
      "l\n",
      "\n",
      "o\n",
      "r\n",
      "t\n",
      "n\n",
      "o\n",
      "C\n",
      "\n",
      "D\n",
      "K\n",
      "2\n",
      "N\n",
      "H\n",
      "P\n",
      "L\n",
      "\n",
      "C\n",
      "\n",
      "l\n",
      "\n",
      "o\n",
      "r\n",
      "t\n",
      "n\n",
      "o\n",
      "C\n",
      "\n",
      "t\n",
      "\n",
      "t\n",
      "\n",
      "n\n",
      "a\n",
      "u\n",
      "m\n",
      "2\n",
      "n\n",
      "h\n",
      "p\n",
      "L\n",
      "\n",
      "F\n",
      "\n",
      "l\n",
      "\n",
      "o\n",
      "r\n",
      "t\n",
      "n\n",
      "o\n",
      "C\n",
      "\n",
      "O\n",
      "K\n",
      "C\n",
      "E\n",
      "2\n",
      "n\n",
      "h\n",
      "p\n",
      "L\n",
      "\n",
      "1.0\n",
      "\n",
      "0.9\n",
      "\n",
      "0.8\n",
      "\n",
      "0.7\n",
      "\n",
      "B\n",
      "\n",
      "x\n",
      "e\n",
      "d\n",
      "n\n",
      "\n",
      "i\n",
      "\n",
      "n\n",
      "o\n",
      "i\n",
      "t\n",
      "a\n",
      "z\n",
      "\n",
      "i\n",
      "l\n",
      "r\n",
      "a\n",
      "e\n",
      "n\n",
      "\n",
      "i\n",
      "l\n",
      "\n",
      "l\n",
      "\n",
      "a\n",
      "n\n",
      "o\n",
      "i\n",
      "t\n",
      "c\n",
      "n\n",
      "u\n",
      "\n",
      "j\n",
      "\n",
      "Control\n",
      "\n",
      "Lphn2\n",
      "mut\n",
      "Control\n",
      "Lphn2\n",
      "mut\n",
      "\n",
      "D\n",
      "\n",
      "-\n",
      "\n",
      "O\n",
      "M\n",
      "\n",
      "t\n",
      "n\n",
      "e\n",
      "\n",
      "l\n",
      "i\n",
      "\n",
      "s\n",
      "+\n",
      "\n",
      "t\n",
      "r\n",
      "a\n",
      "e\n",
      "h\n",
      "\n",
      "0.6\n",
      "F l o\n",
      "\n",
      "w\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "-\n",
      "\n",
      "+\n",
      "\n",
      "Scrambled LPHN2KD\n",
      "\n",
      "n\n",
      "s\n",
      "\n",
      "n\n",
      "s\n",
      "\n",
      "0\n",
      "\n",
      "500\n",
      "\n",
      "1000\n",
      "Cell size\n",
      "\n",
      "1500\n",
      "\n",
      "2000\n",
      "\n",
      "E\n",
      "\n",
      "Control\n",
      "\n",
      "-\n",
      "\n",
      "Lphn2\n",
      "mut\n",
      "Control\n",
      "Lphn2\n",
      "mut\n",
      "\n",
      "O\n",
      "M\n",
      "\n",
      "t\n",
      "n\n",
      "e\n",
      "\n",
      "l\n",
      "i\n",
      "\n",
      "s\n",
      "+\n",
      "\n",
      "t\n",
      "r\n",
      "a\n",
      "e\n",
      "h\n",
      "\n",
      "0.5\n",
      "\n",
      "0.6\n",
      "\n",
      "0.7\n",
      "\n",
      "0.8\n",
      "\n",
      "0.9\n",
      "\n",
      "1.0\n",
      "\n",
      "Eccentricity\n",
      "\n",
      "G\n",
      "\n",
      "H\n",
      "\n",
      "Control\n",
      "\n",
      "Lphn2\n",
      "ECKO\n",
      "\n",
      "0.80\n",
      "\n",
      "0.85\n",
      "\n",
      "0.90\n",
      "Eccentricity\n",
      "\n",
      "0.95\n",
      "\n",
      "1.00\n",
      "\n",
      "Control\n",
      "\n",
      "Lphn2\n",
      "ECKO\n",
      "\n",
      "0.5\n",
      "\n",
      "0.6\n",
      "\n",
      "0.7\n",
      "\n",
      "0.8\n",
      "Junctional linearization index\n",
      "\n",
      "0.9\n",
      "\n",
      "1.0\n",
      "\n",
      "1.1\n",
      "\n",
      "a\n",
      "e\n",
      "r\n",
      "a\n",
      "a\n",
      "t\n",
      "r\n",
      "o\n",
      "a\n",
      "\n",
      "l\n",
      "\n",
      "a\n",
      "t\n",
      "o\n",
      "T\n",
      "\n",
      "/\n",
      "a\n",
      "e\n",
      "r\n",
      "a\n",
      "e\n",
      "u\n",
      "B\n",
      "\n",
      "l\n",
      "\n",
      "0.5\n",
      "\n",
      "0.4\n",
      "\n",
      "0.3\n",
      "\n",
      "0.2\n",
      "\n",
      "0.1\n",
      "\n",
      "0.0\n",
      "\n",
      "Lphn2 E C K O\n",
      "C ontrol\n",
      "\n",
      "Static\n",
      "\n",
      "24 hours laminar flow\n",
      "\n",
      "F-actin\n",
      "\n",
      "FITC-avidin\n",
      "\n",
      "F-actin\n",
      "\n",
      "FITC-avidin\n",
      "\n",
      "Scrambled static\n",
      "\n",
      "Scrambled flow\n",
      "\n",
      "LPHN2KD static\n",
      "\n",
      "LPHN2KD flow\n",
      "\n",
      "n\n",
      "s\n",
      "\n",
      "0\n",
      "\n",
      "5000\n",
      "\n",
      "10000\n",
      "\n",
      "15000\n",
      "\n",
      "20000\n",
      "\n",
      "FITC-avidin area (pixel)\n",
      "\n",
      "K\n",
      "\n",
      "l\n",
      "\n",
      "d\n",
      "e\n",
      "b\n",
      "m\n",
      "a\n",
      "r\n",
      "c\n",
      "S\n",
      "\n",
      "D\n",
      "K\n",
      "2\n",
      "N\n",
      "H\n",
      "P\n",
      "L\n",
      "\n",
      "L\n",
      "\n",
      "\fHoechst\n",
      "\n",
      "Phalloidin\n",
      "\n",
      "VE-cadherin\n",
      "\n",
      "Merge\n",
      "\n",
      "C\n",
      "\n",
      "MβCD\n",
      "\n",
      "−\n",
      "\n",
      "+\n",
      "\n",
      "B\n",
      "\n",
      "c\n",
      "i\n",
      "t\n",
      "\n",
      "t\n",
      "\n",
      "a\n",
      "S\n",
      "\n",
      "D\n",
      "C\n",
      "β\n",
      "M\n",
      "M\n",
      "m\n",
      "5\n",
      "\n",
      ".\n",
      "\n",
      "i\n",
      "\n",
      "n\n",
      "m\n",
      "1\n",
      "\n",
      "ns\n",
      "\n",
      "A\n",
      "\n",
      "y\n",
      "t\n",
      "i\n",
      "s\n",
      "n\n",
      "e\n",
      "t\n",
      "n\n",
      "\n",
      "i\n",
      "\n",
      "l\n",
      "\n",
      "3\n",
      "r\n",
      "e\n",
      "v\n",
      "o\n",
      "C\n",
      "m\n",
      "H\n",
      "4\n",
      "D\n",
      "\n",
      "-\n",
      "\n",
      ")\n",
      "l\n",
      "o\n",
      "r\n",
      "t\n",
      "n\n",
      "o\n",
      "C\n",
      "o\n",
      "t\n",
      "d\n",
      "e\n",
      "z\n",
      "\n",
      "i\n",
      "l\n",
      "\n",
      "a\n",
      "m\n",
      "r\n",
      "o\n",
      "n\n",
      "(\n",
      "\n",
      "2.0\n",
      "\n",
      "1.5\n",
      "\n",
      "1.0\n",
      "\n",
      "0.5\n",
      "\n",
      "0.0\n",
      "\n",
      "Scrambled\n",
      "\n",
      "LPHN2KD\n",
      "\n",
      "−\n",
      "\n",
      "+\n",
      "\n",
      "−\n",
      "\n",
      "+\n",
      "\n",
      "G\n",
      "\n",
      "D\n",
      "\n",
      "MβCD\n",
      "Lysate\n",
      "IB: pSrc(Y416)\n",
      "\n",
      "Lysate\n",
      "IB: Src\n",
      "\n",
      "Lysate\n",
      "IB: pVEGFR2\n",
      "(Y1054)\n",
      "\n",
      "Lysate\n",
      "IB: VEGFR2\n",
      "\n",
      "Lysate\n",
      "IB: Tubulin\n",
      "\n",
      "Flo w\n",
      "\n",
      "M M b C D 1 min\n",
      "\n",
      "C ontrol\n",
      "5 m\n",
      "\n",
      "2.0\n",
      "\n",
      "1.5\n",
      "\n",
      "1.0\n",
      "\n",
      "0.5\n",
      "\n",
      "F\n",
      "\n",
      "ns\n",
      "\n",
      "2\n",
      "R\n",
      "F\n",
      "G\n",
      "E\n",
      "V\n",
      "\n",
      "/\n",
      ")\n",
      "4\n",
      "5\n",
      "0\n",
      "1\n",
      "Y\n",
      "(\n",
      "2\n",
      "R\n",
      "F\n",
      "G\n",
      "E\n",
      "V\n",
      "p\n",
      "\n",
      "E\n",
      "\n",
      ")\n",
      "t\n",
      "n\n",
      "e\n",
      "m\n",
      "a\n",
      "e\n",
      "r\n",
      "t\n",
      "\n",
      "t\n",
      "\n",
      "c\n",
      "r\n",
      "S\n",
      "/\n",
      "c\n",
      "r\n",
      "S\n",
      "p\n",
      "\n",
      "l\n",
      "\n",
      "o\n",
      "n\n",
      "d\n",
      "e\n",
      "b\n",
      "m\n",
      "a\n",
      "r\n",
      "c\n",
      "S\n",
      "o\n",
      "t\n",
      "d\n",
      "e\n",
      "z\n",
      "\n",
      "i\n",
      "l\n",
      "\n",
      "a\n",
      "m\n",
      "r\n",
      "o\n",
      "n\n",
      "(\n",
      "\n",
      "l\n",
      "\n",
      ")\n",
      "c\n",
      "i\n",
      "t\n",
      "a\n",
      "t\n",
      "s\n",
      "d\n",
      "e\n",
      "b\n",
      "m\n",
      "a\n",
      "r\n",
      "c\n",
      "s\n",
      "o\n",
      "t\n",
      "d\n",
      "e\n",
      "z\n",
      "\n",
      "i\n",
      "l\n",
      "\n",
      "a\n",
      "m\n",
      "r\n",
      "o\n",
      "n\n",
      "(\n",
      "\n",
      "ns\n",
      "\n",
      "5\n",
      "\n",
      "4\n",
      "\n",
      "3\n",
      "\n",
      "2\n",
      "\n",
      "1\n",
      "\n",
      "0.0\n",
      "MβCD\n",
      "\n",
      "−\n",
      "\n",
      "+ −\n",
      "Scrambled LPHN2KD\n",
      "\n",
      "+\n",
      "\n",
      "0\n",
      "MβCD\n",
      "\n",
      "−\n",
      "\n",
      "+\n",
      "\n",
      "−\n",
      "\n",
      "+\n",
      "\n",
      "Scrambled LPHN2KD\n",
      "\n",
      "GFP-Trap\n",
      "IB:LPHN2\n",
      "\n",
      "Membrane\n",
      "IB:LPHN2\n",
      "\n",
      "GFP-Trap\n",
      "IB:GFP\n",
      "(Gi1(Q/K)\n",
      "(ins4A)-GFP)\n",
      "\n",
      "Fluid\n",
      "Shear\n",
      "Stress\n",
      "\n",
      "Cholesterol\n",
      "decrease\n",
      "\n",
      "Membrane\n",
      "fluidization\n",
      "\n",
      "Latrophilin\n",
      "\n",
      "Gi2, Gq/11\n",
      "\n",
      "PECAM-1\n",
      "\n",
      "VE-cadherin\n",
      "\n",
      "VEGFRs\n",
      "\n",
      "EC alignment\n",
      "Vasodilation\n",
      "Vascular remodeling\n",
      "\n",
      "\fA\n",
      "\n",
      "e\n",
      "p\n",
      "y\n",
      "t\n",
      "-\n",
      "d\n",
      "l\n",
      "I\n",
      "\n",
      "W\n",
      "\n",
      ")\n",
      "0\n",
      "F\n",
      "(\n",
      "\n",
      "t\n",
      "n\n",
      "a\n",
      "t\n",
      "u\n",
      "M\n",
      "3\n",
      ",\n",
      "b\n",
      "2\n",
      ",\n",
      "a\n",
      "2\n",
      ",\n",
      "1\n",
      "n\n",
      "h\n",
      "p\n",
      "L\n",
      "\n",
      "D\n",
      "\n",
      "DA\n",
      "\n",
      "DA\n",
      "\n",
      "l\n",
      "\n",
      "o\n",
      "r\n",
      "t\n",
      "n\n",
      "o\n",
      "C\n",
      "\n",
      "2\n",
      "n\n",
      "h\n",
      "p\n",
      "L\n",
      "\n",
      "O\n",
      "K\n",
      "C\n",
      "E\n",
      "\n",
      "l\n",
      "\n",
      "o\n",
      "r\n",
      "t\n",
      "n\n",
      "o\n",
      "C\n",
      "\n",
      "2\n",
      "n\n",
      "h\n",
      "p\n",
      "L\n",
      "\n",
      "O\n",
      "K\n",
      "C\n",
      "E\n",
      "\n",
      "F\n",
      "\n",
      "Tg(Kdrl:Ras-mCherry); Tg(Fli1a: nls-eGFP)\n",
      "\n",
      "48hpf\n",
      "\n",
      "B\n",
      "\n",
      "Control\n",
      "\n",
      "48hpf\n",
      "\n",
      "ISV\n",
      "\n",
      "ISV\n",
      "\n",
      "-\n",
      "\n",
      "Lphn\n",
      "mut\n",
      "Control\n",
      "Lphn\n",
      "mut\n",
      "\n",
      "t\n",
      "n\n",
      "e\n",
      "\n",
      "l\n",
      "i\n",
      "\n",
      "s\n",
      "+\n",
      "\n",
      "t\n",
      "r\n",
      "a\n",
      "e\n",
      "h\n",
      "\n",
      "n\n",
      "s\n",
      "\n",
      "0\n",
      "\n",
      "5\n",
      "ISV diameter (µm)\n",
      "\n",
      "10\n",
      "\n",
      "C\n",
      "\n",
      "72hpf\n",
      "\n",
      "Control\n",
      "\n",
      "Lphn\n",
      "mut\n",
      "\n",
      "0\n",
      "\n",
      "Control\n",
      "\n",
      "Lphn\n",
      "mut\n",
      "\n",
      "2\n",
      "\n",
      "4\n",
      "arterial ISV diameter (µm)\n",
      "\n",
      "6\n",
      "\n",
      "8\n",
      "\n",
      "n\n",
      "s\n",
      "\n",
      "CD31\n",
      "\n",
      "SMA\n",
      "\n",
      "Hoechst/CD31/SMA\n",
      "\n",
      "2\n",
      "\n",
      "m\n",
      "m\n",
      "\n",
      "r\n",
      "e\n",
      "p\n",
      "s\n",
      "e\n",
      "i\n",
      "r\n",
      "a\n",
      "\n",
      "l\n",
      "l\n",
      "i\n",
      "\n",
      "p\n",
      "a\n",
      "c\n",
      "\n",
      "H\n",
      "\n",
      "l\n",
      "\n",
      "o\n",
      "r\n",
      "t\n",
      "n\n",
      "o\n",
      "C\n",
      "\n",
      "O\n",
      "K\n",
      "C\n",
      "E\n",
      "2\n",
      "n\n",
      "h\n",
      "p\n",
      "L\n",
      "\n",
      "I\n",
      "\n",
      "1500\n",
      "\n",
      "1000\n",
      "\n",
      "500\n",
      "\n",
      "0\n",
      "\n",
      "Lphn2 E C K O\n",
      "C ontrol\n",
      "\n",
      "0\n",
      "\n",
      "5\n",
      "\n",
      "10\n",
      "\n",
      "15\n",
      "\n",
      "venous ISV diameter (µm)\n",
      "\n",
      "J\n",
      "\n",
      "Pre\n",
      "\n",
      "Day 1\n",
      "\n",
      "Day 3\n",
      "\n",
      "E\n",
      "1.0\n",
      "\n",
      "0.8\n",
      "\n",
      "Control\n",
      "Mutant\n",
      "\n",
      "x\n",
      "e\n",
      "d\n",
      "n\n",
      "\n",
      "i\n",
      "\n",
      "0.6\n",
      "\n",
      "Day 7\n",
      "\n",
      "Day 14\n",
      "\n",
      "Day 21\n",
      "\n",
      "i\n",
      "\n",
      "a\n",
      "m\n",
      "e\n",
      "h\n",
      "c\n",
      "s\n",
      "I\n",
      "\n",
      "0.4\n",
      "\n",
      "#\n",
      "\n",
      "##\n",
      "\n",
      "K\n",
      "\n",
      "i\n",
      "\n",
      ")\n",
      "n\n",
      "m\n",
      "m\n",
      "\n",
      "/\n",
      "\n",
      "0.2\n",
      "\n",
      "0.0\n",
      "\n",
      "#\n",
      "\n",
      "Pre\n",
      "\n",
      "Day 1 Day 3 Day 7 Day 14 Day 21\n",
      "\n",
      "Control\n",
      "\n",
      "unligated\n",
      "\n",
      "ischemia\n",
      "\n",
      "Lphn2 ECKO mutant\n",
      "\n",
      "unligated\n",
      "\n",
      "ischemia\n",
      "\n",
      "Start speed: 12 m / min\n",
      "Increases by 1m / min\n",
      "every minute\n",
      "\n",
      "L\n",
      "\n",
      "800\n",
      "\n",
      ")\n",
      "\n",
      "m\n",
      "\n",
      "i\n",
      "\n",
      "(\n",
      "e\n",
      "c\n",
      "n\n",
      "a\n",
      "t\n",
      "s\n",
      "d\n",
      "g\n",
      "n\n",
      "n\n",
      "n\n",
      "u\n",
      "R\n",
      "\n",
      "i\n",
      "\n",
      "600\n",
      "\n",
      "400\n",
      "\n",
      "200\n",
      "\n",
      "0\n",
      "\n",
      "Lphn2 E C K O\n",
      "C ontrol\n",
      "\n",
      "50\n",
      "\n",
      "40\n",
      "\n",
      "30\n",
      "\n",
      "20\n",
      "\n",
      "10\n",
      "\n",
      "0\n",
      "\n",
      "Lphn2 E C K O\n",
      "C ontrol\n",
      "\n",
      "(\n",
      "\n",
      "d\n",
      "e\n",
      "e\n",
      "p\n",
      "s\n",
      "m\n",
      "u\n",
      "m\n",
      "x\n",
      "a\n",
      "m\n",
      "\n",
      "i\n",
      "\n",
      "M\n",
      "\n",
      "5500\n",
      "\n",
      "G\n",
      "\n",
      "5000\n",
      "\n",
      "4000\n",
      "\n",
      "g\n",
      "n\n",
      "i\n",
      "t\n",
      "n\n",
      "u\n",
      "o\n",
      "c\n",
      "\n",
      "3000\n",
      "\n",
      "l\n",
      "\n",
      "e\n",
      "s\n",
      "s\n",
      "e\n",
      "v\n",
      "\n",
      "2000\n",
      "\n",
      "1000\n",
      "\n",
      "0\n",
      "\n",
      "Control - unligated\n",
      "Control - ischemia\n",
      "Lphn2 mutant - unligated\n",
      "Lphn2 mutant - ischemia\n",
      "\n",
      "ns\n",
      "\n",
      "ns\n",
      "\n",
      "ns\n",
      "\n",
      "ns\n",
      "\n",
      "ns\n",
      "\n",
      "ns\n",
      "\n",
      "ns\n",
      "\n",
      "ns\n",
      "\n",
      "ns\n",
      "\n",
      "ns\n",
      "\n",
      "ns\n",
      "\n",
      "ns\n",
      "\n",
      "ns\n",
      "\n",
      "ns\n",
      "\n",
      "ns\n",
      "\n",
      "ns ns\n",
      "\n",
      "ns ns\n",
      "\n",
      "ns\n",
      "\n",
      "ns\n",
      "\n",
      "ns\n",
      "\n",
      "<48\n",
      "\n",
      "<72\n",
      "\n",
      "<96\n",
      "\n",
      "<120\n",
      "\n",
      "<144\n",
      "\n",
      "<168\n",
      "\n",
      "<192\n",
      "\n",
      "<216\n",
      "\n",
      "<240\n",
      "\n",
      ">240\n",
      "\n",
      "Vessel size\n",
      "\n",
      "N\n",
      "\n",
      "c\n",
      "i\n",
      "t\n",
      "a\n",
      "t\n",
      "S\n",
      "\n",
      "w\n",
      "o\n",
      "F\n",
      "\n",
      "l\n",
      "\n",
      "l\n",
      "\n",
      "o\n",
      "r\n",
      "t\n",
      "n\n",
      "o\n",
      "C\n",
      "\n",
      "D\n",
      "K\n",
      "2\n",
      "N\n",
      "H\n",
      "P\n",
      "L\n",
      "\n",
      "c\n",
      "i\n",
      "t\n",
      "a\n",
      "t\n",
      "S\n",
      "\n",
      "w\n",
      "o\n",
      "F\n",
      "\n",
      "l\n",
      "\n",
      ")\n",
      "r\n",
      "h\n",
      "/\n",
      "g\n",
      "k\n",
      "/\n",
      "l\n",
      "\n",
      "m\n",
      "\n",
      "(\n",
      "2\n",
      "O\n",
      "V\n",
      "\n",
      "5000\n",
      "\n",
      "4500\n",
      "\n",
      "4000\n",
      "\n",
      "3500\n",
      "\n",
      "0\n",
      "\n",
      "Control\n",
      "Lphn2 ECKO\n",
      "\n",
      "10\n",
      "\n",
      "20\n",
      "\n",
      "30\n",
      "\n",
      "Time (min)\n",
      "\n",
      "O\n",
      "\n",
      "4000000\n",
      "\n",
      ")\n",
      "2\n",
      "\n",
      "m\n",
      "µ\n",
      "(\n",
      "a\n",
      "e\n",
      "r\n",
      "a\n",
      "\n",
      "l\n",
      "l\n",
      "\n",
      "e\n",
      "C\n",
      "\n",
      "3000000\n",
      "\n",
      "2000000\n",
      "\n",
      "1000000\n",
      "\n",
      "0µm\n",
      "\n",
      "Control static\n",
      "\n",
      "Control flow\n",
      "\n",
      "LPHN2KD static\n",
      "\n",
      "LPHN2KD flow\n",
      "\n",
      "0\n",
      "\n",
      "0\n",
      "\n",
      "50\n",
      "Distance from gel surface (µm)\n",
      "\n",
      "100\n",
      "\n",
      "P\n",
      "\n",
      "Control static\n",
      "\n",
      "120µm\n",
      "\n",
      "Control flow\n",
      "\n",
      "LPHN2KD static\n",
      "\n",
      "LPHN2KD flow\n",
      "\n",
      "n\n",
      "s\n",
      "\n",
      "0\n",
      "\n",
      "10\n",
      "\n",
      "20\n",
      "\n",
      "30\n",
      "\n",
      "Average sprout length ( m)\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "md = MarkItDown()\n",
    "result = md.convert(PDF_FILES[0])\n",
    "print(result.text_content)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 3. Docling\n",
    "\n",
    "Generates the extraction to different formats including markdown and html. However, the structure of the DOCX file is not kept since there are no headers or titles. Only paragraphs objects.\n",
    "\n",
    "DOCLING nuses the `python-docx` library as main engine for the conversion."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The Nedd4L ubiquitin ligase is activated by FCHO2-generated membrane curvature\n",
      "\n",
      "Yasuhisa Sakamoto1, Akiyoshi Uezu1, Koji Kikuchi1, Jangmi Kang2, Eiko Fujii2, Toshiro Moroishi3, Shiro Suetsugu4, and Hiroyuki Nakanishi1,2*\n",
      "\n",
      "1Department of Molecular Pharmacology, Faculty of Life Sciences, Kumamoto University, 1–1–1 Honjyo, Kumamoto 860-8556, Japan\n",
      "\n",
      "2Faculty of Clinical Nutrition and Dietetics, Konan Women’s University, 6–2–23 Morikita-machi, Kobe 658-0001, Japan\n",
      "\n",
      "3Department of Molecular and Medical Pharmacology, Faculty of Life Sciences, Kumamoto University, 1–1–1 Honjyo, Kumamoto 860-8556, Japan\n",
      "\n",
      "4Division of Biological Science, Graduate School of Science and Technology, Nara Institute of Science and Technology, 8916-5 Takayama-cho, Ikoma 630-0192, Japan\n",
      "\n",
      "*Corresponding author. Tel: +81-96-373-5074; Fax: +81-96-373-5078; E-mail: hnakanis@gpo.kumamoto-u.ac.jp\n",
      "\n",
      "Abstract\n",
      "\n",
      "The C2-WW-HECT domain ubiquitin ligase Nedd4L regulates membrane sorting during endocytosis through the ubiquitination of cargo molecules such as the epithelial sodium channel (ENaC). Nedd4L is catalytically autoinhibited by an intramolecular interaction between its C2 and HECT domains, but the protein’s activation mechanism is poorly understood. Here, we show that Nedd4L activation is linked to membrane shape by FCHO2, a Bin-Amphiphysin-Rsv (BAR) domain protein that regulates endocytosis. FCHO2 was required for the Nedd4L-mediated ubiquitination and endocytosis of ENaC, with Nedd4L co-localizing with FCHO2 at clathrin-coated pits. In cells, Nedd4L was specifically recruited to, and activated by, the FCHO2 BAR domain. Furthermore, we reconstituted FCHO2-induced recruitment and activation of Nedd4L in vitro. Both the recruitment and activation were mediated by membrane curvature rather than protein-protein interactions. The Nedd4L C2 domain recognized a specific degree of membrane curvature that was generated by the FCHO2 BAR domain, with this curvature directly activating Nedd4L by relieving its autoinhibition. Thus, we show for the first time a specific function (i.e., recruitment and activation of an enzyme regulating cargo sorting) of membrane curvature by a BAR domain protein.\n",
      "\n",
      "Keywords: Nedd4L; FCHO2; clathrin; endocytosis; membrane curvature.\n",
      "\n",
      "Introduction\n",
      "\n",
      "Posttranslational modification of proteins by covalent attachment of ubiquitin (Ub) is catalyzed by three enzymes: a Ub-activating enzyme (E1), a Ub-conjugating enzyme (E2), and a Ub-protein ligase (E3) that determines substrate specificity (Hershko & Ciechanover, 1998). E3 Ub ligases are classified into two categories based on their Ub transfer mechanisms: RING finger/U-box E3 and HECT-type E3. The Nedd4 family belongs to HECT-type E3 Ub ligases and consists of nine members, including Nedd4/Nedd4-1, Nedd4L/Nedd4-2, Itch, Smurf1, Smurf2, WWP1, WWP2, NEDL1, and NEDL2 (Ingham et al, 2004; Rotin & Kumar, 2009). They are characterized by a common modular organization, with an N-terminal C2 domain, two to four WW domains, and a C-terminal catalytic HECT domain. While the C2 domain was originally identified in protein kinase C (PKC) as a Ca2+-dependent phosphatidylserine (PS)-binding domain (Nishizuka, 1992), it shows significant diversity in the binding partners, including intracellular proteins and other phospholipids, such as phosphatidylinositol (4,5)-bisphosphate [PI(4,5)P2] (Lemmon, 2008). The WW domains recognize proline-rich motifs, such as PPxY (PY motif, where x is any residue), of substrates or adaptor proteins. The HECT domain catalyzes the isopeptide bond formation between the Ub C terminus and the substrate lysine residues. At least some of the Nedd4 family members, including Nedd4L, are catalytically autoinhibited by an intramolecular interaction between the C2 and HECT domains (Wang et al, 2010; Wiesner et al, 2007; Zhu et al, 2017). However, the activation mechanism of the Nedd4 family is poorly understood.\n",
      "\n",
      "The Nedd4 family is implicated in a wide variety of cellular processes by regulating membrane trafficking and degradation of components of various signaling pathways (Ingham et al, 2004; Rotin & Kumar, 2009). Among a diversity of substrates, the best-characterized is the epithelial sodium channel (ENaC). This channel is composed of three subunits, α-, β-, and γENaC, each of which possesses two transmembrane segments and intracellular N- and C-termini with a large extracellular loop (Butterworth, 2010; Rotin & Staub, 2011; Rossier, 2014). Each subunit has a PY motif at the cytosolic C-terminus. Nedd4L Ub ligase specifically binds to the PY motif and mediates the ubiquitination of lysine residues on the N-terminus of at least α- and γENaC at the plasma membrane. Ub serves as an internalization signal for recognition by adaptor proteins, such as epsin and Eps15, for clathrin-mediated endocytosis (CME) (Traub, 2009). Consequently, ENaC is constitutively internalized through CME. Heterozygous mutations of the PY motif of β- or γENaC lead to impaired ubiquitination and endocytosis, resulting in persistence of the channels at the cell surface and increased Na+ absorption in Liddle syndrome, an autosomal dominant form of severe hypertension.\n",
      "\n",
      "The process of CME begins at the cytoplasmic surface of the plasma membrane with the assembly of clathrin with adaptor proteins and transmembrane cargo molecules (Doherty & McMahon, 2009; Traub, 2009). In parallel with this assembly, membrane curvature is generated to form hemispherical clathrin-coated pits (CCPs). This process leads to deep membrane invaginations. Subsequently, the neck of invaginated pits is constricted and severed to separate the newly formed clathrin-coated vesicles (CCVs) from the plasma membrane. These membrane curvatures are considered to be generated, sensed, and/or maintained by Bin-Amphiphysin-Rsv (BAR) domain superfamily proteins. BAR domains form a crescent-shaped dimer characterized by a specific degree of intrinsic curvature. They bind to negatively charged phospholipids, such as PS, and force membranes to bend according to their intrinsic curvatures, thereby inducing the formation of membrane tubules with specific diameters (curvatures). For example, the diameters of membrane tubules generated by the amphiphysin BAR domain, FCHO2 BAR domain, and FBP17 BAR domain are 20–50 nm (sharper curvature) (Peter et al, 2004; Takei et al, 1999; Yin et al, 2009), 50–80 nm (intermediate curvature) (Henne et al, 2007), and ~150 nm (Takano et al, 2008) (shallower curvature), respectively.\n",
      "\n",
      "The FCHO2 BAR domain has been proposed to trigger the initial invagination for CCP formation (Henne et al, 2010). FCHO2 binds to the flat surface of the plasma membrane at sites where CCPs will form. FCHO2 then generates the initial membrane curvature and recruits the clathrin machinery for CCP formation. Here, we report the role of FCHO2 in Nedd4L-mediated ENaC endocytosis and show that FCHO2 BAR domain-generated membrane curvature induces the recruitment and activation of Nedd4L.\n",
      "\n",
      "Results\n",
      "\n",
      "FCHO2 is required for Nedd4L-mediated ubiquitination and endocytosis of ENaC\n",
      "\n",
      "To study the role of FCHO2 in ENaC endocytosis, we established a HeLa cell line stably expressing all three subunits, α-, β-, and γENaC (αβγENaC-HeLa cells) (Fig EV1A-C). A FLAG tag was introduced into the extracellular region of αENaC in a position previously shown not to affect channel activity (Firsov et al, 1996). Upon FCHO2 knockdown by small interfering RNA (siRNA) methods, ENaC internalization was reduced, as was upon Nedd4L knockdown (Zhou et al, 2007) (Fig 1 A and B). Transferrin receptor (TfR) internalization was reduced upon FCHO2 knockdown but not upon Nedd4L knockdown (Fig 1C). The phenotype of FCHO2 knockdown in ENaC internalization was rescued by an siRNA-resistant (sr) form of FCHO2 (Fig 1D and Appendix Fig S1). Furthermore, αENaC ubiquitination was reduced upon FCHO2 or Nedd4L knockdown but not upon FBP17 knockdown (Fig 1E and Fig EV2). FCHO2 or Nedd4L knockdown increased αENaC expression at the cell surface (Fig 1E and F). These results suggest that FCHO2 is required for Nedd4L-mediated ubiquitination and endocytosis of ENaC.\n",
      "\n",
      "Recruitment and activation of Nedd4L by FCHO2 in cells\n",
      "\n",
      "Immunofluorescence microscopy revealed that in αβγENaC-HeLa cells, cell-surface αENaC co-localized with FCHO2 at clathrin-coated structures (Fig 2A). However, its co-localization with Nedd4L was not detected (Appendix Fig S2A), because exogenous expression of Nedd4L-green fluorescent protein (GFP) stimulated ENaC internalization, resulting in the disappearance of αENaC from the cell surface. Cell-surface αENaC was observed in cells lacking Nedd4L-GFP expression (non-transfected cells). When a catalytically inactive Nedd4L mutant, C922A, was expressed, it co-localized with cell-surface αENaC and endogenous FCHO2 (Fig 2B). The mutant also formed FCHO2-negative spots. In wild-type HeLa cells, Nedd4L accumulated at clathrin-coated structures where FCHO2 localized (Fig 2C). Additionally, Nedd4L formed clathrin-negative spots, suggesting that they may be involved in clathrin-independent endocytosis.\n",
      "\n",
      "Using total internal reflection fluorescence (TIRF) microscopy, we investigated whether Nedd4L is recruited to CCPs in wild-type HeLa cells. Similar to FCHO2 described previously (Henne et al, 2010; Lehmann et al, 2019; Zaccai et al, 2022), Nedd4L showed dynamic co-localization with clathrin during pit formation (Fig 3). When both fluorescent protein-tagged FCHO2 and Nedd4L were expressed, the accumulation of Nedd4L at FCHO2-positive spots was not clearly observed (Appendix Fig S2B). The reason behind this remains unknown, but it has been shown that clathrin coat formation, measured in time-lapse images of cells expressing fluorescent proteins, is affected by several factors, such as overexpression level, choice of the proteins, and choice of the fluorescent tag (Aguet et al, 2013). These factors may contribute to the failure to detect the accumulation of Nedd4L at FCHO2-positive spots.\n",
      "\n",
      "While Nedd4L shows a punctate staining pattern at the cell surface (see Figs 2C, 3B and 3C), it exhibited diffuse distribution throughout the cytoplasm (Fig 4A and B). In COS7 cells, co-expression of the FCHO2 BAR domain [amino acid (aa) 1-302] recruited Nedd4L to membrane tubules (Fig 4A). The Nedd4L C2 domain (Myc-Nedd4L C2, aa 1-160) was also concentrated to membrane tubules, whereas C2 domain-deleted Nedd4L (Myc-Nedd4L ΔC2) was not. In contrast to FCHO2, the FBP17 BAR domain did not recruit Nedd4L to membrane tubules. Additionally, the BAR domain protein amphiphysin1 did not recruit Nedd4L to membrane tubules. Thus, Nedd4L is selectively recruited through the C2 domain to membrane tubules generated by the FCHO2 BAR domain. Similar results were obtained in HEK293 cells (Fig 4B). In these cells, membrane tubules generated by amphiphysin1 were hardly detected. It showed a punctate staining pattern and did not co-localized with Nedd4L.\n",
      "\n",
      "We also investigated whether the FCHO2 BAR domain not only recruits but also activates Nedd4L in cells. For this purpose, we performed an in vivo ubiquitination assay using HEK293 cells, which formed membrane tubules upon FCHO2 expression more efficiently than did COS7 cells: 91 ± 2% of HEK293 cells vs. 32 ± 2% of COS7 cells [the mean ± standard error of the mean (S.E.M.) of four independent experiments]. As the catalytic activity of E3 Ub ligases is typically reflected in their autoubiquitination (Wiesner et al, 2007), we determined activity by measuring the ubiquitination level of Nedd4L itself. Expression of the FCHO2 BAR domain in cells remarkably enhanced Nedd4L autoubiquitination, whereas FBP17 or amphiphysin1 did not (Fig 5A). Tubulation-deficient FCHO2 mutants (K146E+R152E and L136E) (Henne et al, 2010) did not enhance Nedd4L activity (Fig 5A–C and Fig EV3). A catalytically inactive Nedd4L mutant, C922A, which was recruited to FCHO2-generated membrane tubules, showed little enhancement of ubiquitination upon FCHO2 expression (Fig 5D and E), indicating that Nedd4L ubiquitination is self-induced. Using αENaC as a substrate, we further confirmed that Nedd4L is activated upon FCHO2 expression in cells (Fig 5F).\n",
      "\n",
      "Preference of Nedd4L for FCHO2-generated membrane curvature\n",
      "\n",
      "The aforementioned results prompted us to investigate whether FCHO2 and Nedd4L interact through membrane tubules rather than through stereotyped protein–protein interactions. Membrane tubules generated by BAR domains in cells have specific diameters (curvatures) (Henne et al, 2007; Peter et al, 2004; Takano et al, 2008; Takei et al, 1999; Yin et al, 2009). Nedd4L may preferentially bind to the specific degree of membrane curvature generated by FCHO2. To investigate this possibility, we first examined the biochemical properties of Nedd4L. Using a co-sedimentation assay with liposomes containing various percentages of PS, we found that the C2 domain of Nedd4L is a Ca2+-dependent PS-binding domain (Fig 6A and B). Submicromolar Ca2+ concentrations, which are comparable to intracellular levels in unstimulated cells, were sufficient for the Nedd4L C2 domain to bind to PS-containing liposomes. The C2 domain alone and full-length Nedd4L showed different Ca2+ requirements. The higher Ca2+ requirement of full-length Nedd4L may be due to the intramolecular interaction between the C2 and HECT domains. Increasing PS percentages in synthetic liposomes enhanced the liposome binding of the Nedd4L C2 domain, and PS requirements varied depending on Ca2+ concentration.\n",
      "\n",
      "We next examined whether Nedd4L preferentially binds to a specific degree of membrane curvature. For this purpose, liposomes of different sizes (curvatures) were prepared by stepwise extrusion through filter membranes with different pore sizes (0.8, 0.4, 0.2, 0.1, 0.05, and 0.03 µm) and sonication. Sonication or extrusion through a 0.03 µm pore-size filter was used to prepare the smallest liposomes. Transmission electron microscopy revealed that the actual liposome size was similar to the pore size for 0.2, 0.1, 0.05, and 0.03 µm pore-size liposomes, whereas it was significantly smaller for 0.8 and 0.4 µm pore-size liposomes (Fig 7A). Nedd4L showed the strongest binding to 0.05 µm pore-size liposomes (Fig 7B).\n",
      "\n",
      "Synaptotagmin I (SytI) tandem C2 domains can both generate and sense membrane curvature in vitro by penetrating their hydrophobic residues into the lipid bilayer (Martens et al, 2007). As a sensor, the binding affinity is considered to be highest for membranes that have a degree of curvature consistent with that generated by SytI itself. We investigated whether, in addition to sensing membrane curvature, the Nedd4L C2 domain can also generate membrane curvature in vitro and, if so, what diameter of membrane tubules is generated. As expected, transmission electron microscopy revealed that the C2 domain deformed liposomes into extensive tubulation (Fig 7C). It should be noted that Nedd4L does not generate membrane tubules in cells (see Fig 4A and B). The tubule diameter generated by Nedd4L in vitro was mainly 40–80 nm, which is consistent not only with the diameter of 0.05 µm pore-size liposomes, to which Nedd4L preferentially binds, but also with the diameter of membrane tubules generated by FCHO2 (Fig 7D). These results suggest that Nedd4L shows the highest binding affinity for the degree of membrane curvature generated by FCHO2. It is therefore likely that FCHO2-induced recruitment and activation of Nedd4L are mediated by the degree of membrane curvature.\n",
      "\n",
      "Roles of conserved hydrophobic residues in the Nedd4L C2 domain\n",
      "\n",
      "We compared the C2 domain sequence of Nedd4L to those of other Nedd4 family members, SytI, and PKCγ (Fig 8A). SytI C2 domains possess hydrophobic residues (M173, F234, V304, and I367) for penetration into the lipid bilayer to generate and sense membrane curvature (Martens et al, 2007). The Nedd4L C2 domain also has three hydrophobic residues (I37, F38, and L99), which are conserved among Nedd4 family members except for WWP2. We mutated these residues to alanine to produce six mutants, I37A, F38A, L99A, I37A+F38A, I37A+L99A, and I37A+F38A+L99A (triple-A). Any single mutation did not inhibit Nedd4L recruitment to FCHO2-generated membrane tubules (Fig 8B). However, double and triple mutations severely impaired this recruitment. Furthermore, the I37A+F38A and triple-A mutants showed little liposome binding and lost curvature generation (Fig 8C and D). Thus, these hydrophobic residues (I37, F38, and L99) are critical for both membrane binding and curvature sensing and generation. These residues are likely inserted into the lipid bilayer, thereby strengthening the C2 domain interaction with membranes and enabling the sensing and generation of membrane curvature. It has been shown that I37 and F38 are directly involved in Ca2+ binding (Escobedo et al, 2014). The residues may be inserted into the lipid bilayer while binding Ca2+.\n",
      "\n",
      "In the C2 domain of Smurf2, another Nedd4 family member, conserved hydrophobic residues (F29 and F30) are responsible for interactions with the HECT domain (Fig 8A) (Wiesner et al, 2007). Mutation at these residues impairs the interaction between the C2 and HECT domains, resulting in enhanced Smurf2 autoubiquitination by relieving autoinhibition. Consistent with these previous findings, we found that the Nedd4L I37A+F38A mutant, which was not recruited to FCHO2-generated membrane tubules, enhanced autoubiquitination, irrespective of FCHO2 expression (Fig 5D and E). These results suggest that I37 and F38 are critical for interactions with the HECT domain as well as for membrane binding and curvature sensing. Interestingly, the Nedd4L I37A+F38A mutant, as well as the C922A mutant, showed little αENaC ubiquitination (Fig EV4), indicating that membrane binding of Nedd4L is critical for ENaC ubiquitination.\n",
      "\n",
      "In vitro activation of Nedd4L by FCHO2-generated membrane curvature\n",
      "\n",
      "To examine the properties of Nedd4L catalytic activity, we carried out an in vitro ubiquitination assay using an αENaC peptide as a substrate. The αENaC peptide possesses a conserved PY motif for interaction with the Nedd4L WW domain, and it is fused with monomeric streptavidin (mSA) (Lim et al, 2013) to bind to liposomes containing biotinylated phospholipids (biotinylated liposomes) (Sakamoto et al, 2017) (Fig 9A and B). We found that Ca2+ and PS were required for Nedd4L catalytic activity (Fig 9C and D), and that 0.05 µm pore-size liposomes were most effective in stimulating Ca2+- and PS-dependent Nedd4L activity (Fig 9E). Thus, the properties of Nedd4L catalytic activity are similar to those of its liposome binding. Furthermore, we found that Nedd4L did not ubiquitinate PY motif-mutated (Y644A in αENaC) mSA-αENaC (Appendix Fig S3). These findings are consistent with defects in ENaC ubiquitination observed in Liddle syndrome.\n",
      "\n",
      "We next examined whether membrane localization of mSA-αENaC affects the membrane binding and catalytic activity of Nedd4L. Nedd4L bound only slightly to mSA-αENaC on PS-free (0% PS) liposomes, but the liposome binding was synergistically increased by elevating the PS percentage in liposomes (Fig 9F). By contrast, when free biotin competitively inhibited the binding of mSA-αENaC to biotinylated liposomes, the PS-dependent liposome binding of Nedd4L was significantly reduced, resulting in remarkable inhibition of mSA-αENaC ubiquitination (Fig 9D and F). Thus, membrane localization of the PY motif is critical for membrane binding and activation of Nedd4L. It is, therefore, likely that the PY motifs on membranes, such as ENaC, play a role in enhancing Nedd4L catalytic activity by potentiating membrane binding through interactions with the WW domain.\n",
      "\n",
      "We then reconstituted in vitro FCHO2-induced recruitment and activation of Nedd4L. When liposomes (20% PS) were used, the FCHO2 BAR domain enhanced the liposome binding of Nedd4L (Fig 10A) and mSA-αENaC ubiquitination (Fig 10B). This enhancement was lost when the strength of interaction of Nedd4L with liposomes was increased by elevating the PS percentage in liposomes (~50% PS) (Fig 10C and Fig EV5).　In contrast to FCHO2, neither amphiphysin1 nor FBP17 enhanced Nedd4L activity (Fig 10D and Fig EV6A). Similarly, tubulation-deficient FCHO2 mutant (L136E) did not enhance Nedd4L activity (Fig 10E and Fig EV6A). These results are in good agreement with those obtained from in vivo experiments on FCHO2-induced recruitment and activation of Nedd4L. Electron microscopic analysis revealed that mSA-αENaC-associated liposomes were deformed by membrane-sculpting proteins into tubules with diameters similar to those of mSA-αENaC-free liposomes (Fig EV6B and see also Fig 7D). The diameter of membrane tubules generated by FCHO2 is consistent with that of 0.05 µm pore-size liposomes which are most effective in stimulating Nedd4L activity.\n",
      "\n",
      "To confirm whether FCHO2-induced recruitment and activation of Nedd4L are mediated by liposomes (i.e., membrane curvature) rather than by protein–protein interactions, we conducted two sets of experiments. Initially, we employed synthetic liposomes where 20% PS was substituted with 5% PI(4,5)P2. The FCHO2 BAR domain bound to PI(4,5)P2 as well as PS with high affinity, whereas the Nedd4L C2 domain did not (Fig 11A and B). The FCHO2 BAR domain is known to induce the tubulation of liposomes [5% PI(4,5)P2] (Henne et al, 2010). If Nedd4L could interact through protein–protein interactions with the FCHO2 BAR domain binding to liposomes [5% PI(4,5)P2], the BAR domain would facilitate the co-precipitation of Nedd4L in a co-sedimentation assay, thereby stimulating mSA-αENaC ubiquitination. However, the FCHO2 BAR domain did not demonstrate such interactions (Fig 11C and D). Therefore, it is unlikely that Nedd4L interacts with the FCHO2 BAR domain on liposomes through protein–protein interactions. Secondly, we employed 0.05 µm pore-size liposomes that are most effective in stimulating Nedd4L activity. The diameter of 0.05 µm pore-size liposomes is consistent with that of membrane tubules generated by the FCHO2 BAR domain. The BAR domain is, thus, expected not to induce the tubulation of 0.05 µm pore-size liposomes. The FCHO2 BAR domain showed similar binding activity toward 0.8 and 0.05 µm pore-size liposomes (Fig 12A). When 0.8 µm pore-size liposomes were used, the FCHO2 BAR domain enhanced Nedd4L activity (Fig 12B). However, in the case of 0.05 µm pore-size liposomes, curvature-stimulated Nedd4L activity was not potentiated by the addition of the FCHO2 BAR domain. If Nedd4L had interacted with the BAR domain, we would have expected potentiation of Nedd4L activity. These results strongly suggest that FCHO2-induced activation of Nedd4L is mediated by membrane curvature and not by protein–protein interactions.\n",
      "\n",
      "Discussion\n",
      "\n",
      "Nedd4L represents the ancestral Ub ligase with strong similarity to yeast Rsp5 which plays a key role in the trafficking, sorting, and degradation of a large number of proteins (Yang & Kumar, 2010). Consistently, Nedd4L regulates a growing number of substrates, including membrane receptors, transporters, and ion channels. Although Nedd4L exists in a catalytically autoinhibited state due to an intramolecular interaction between the N-terminal C2 and C-terminal HECT domains, little is known about its activation mechanism. In this study, we clearly demonstrated that Nedd4L is activated by FCHO2 F-BAR domain-induced membrane curvature. It has been shown that the activity of synptojanin-1, a phosphoinositide phosphatase, is modulated by membrane curvature (Chang-Ileto et al, 2011). Taken together, it is conceivable that membrane curvature serves as a signal for the activation of various enzymes involved in membrane trafficking.\n",
      "\n",
      "Unique Properties of the Nedd4L C2 Domain\n",
      "\n",
      "Although both sytI and Nedd4L C2 domains are Ca2+-dependent PS-binding domains that can generate and sense membrane curvature, their properties show a number of significant differences. An interesting difference exists in the degrees of membrane curvatures generated by sytI and Nedd4L: the diameters of their membrane tubules are about 15–20 nm and 40–80 nm, respectively (Hui et al, 2009; Martens et al, 2007). This difference is not simply due to that between tandem and single C2 domains, because it has been shown that the sytI C2B domain alone generates membrane tubules with a similar diameter (Hui et al, 2009). SytI and Nedd4L C2 domains must therefore have respective unique structures that determine the specific degrees of membrane curvatures that they sense and generate.\n",
      "\n",
      "Another important difference between sytI and Nedd4L C2 domains exists in their Ca2+ requirements. SytI C2 domains require Ca2+ concentrations in the submillimolar range to generate membrane curvature (Hui et al, 2009). In contrast, submicromolar Ca2+ concentrations, which are comparable to intracellular levels in unstimulated cells, are sufficient for the Nedd4L C2 domain to sense membrane curvature. This is consistent with the observation that Nedd4L is recruited and activated in serum-starved cells. The different Ca2+ requirements between sytI and Nedd4L C2 domains are reflected in their functions. SytI C2 domains induce membrane curvature in response to an increase in intracellular Ca2+ concentration, which is essential for Ca2+-triggered membrane fusion (Hui et al, 2009; Martens et al, 2007). Thus, sytI C2 domains serve as a Ca2+ sensor and a curvature generator. In contrast, the Nedd4L C2 domain recognizes a specific degree of curvature without an increase in intracellular Ca2+ concentration. Therefore, the Nedd4L C2 domain serves as a curvature sensor to regulate the localization and activity of the E3 Ub ligase.\n",
      "\n",
      "A striking feature that distinguishes the Nedd4L C2 domain from other C2 domain proteins (except for at least a subset of Nedd4 family members) is that the Nedd4L C2 domain interacts not only with membranes but also with the intramolecular catalytic domain for its autoinhibition. Mutational experiments have revealed that I37 and F38 residues in the C2 domain are critical for interactions with both membranes and the HECT domain, suggesting that these two interactions are competitive. This idea is supported by the observation that the Ca2+ EC50 value of full-length Nedd4L is much higher than that of the C2 domain. The HECT domain in full-length Nedd4L may competitively inhibit the interaction of the C2 domain with membranes. It is likely that membranes displace the HECT domain from the C2 domain when the strength of the C2 domain interaction with membranes is increased by the preferred membrane curvature. This displacement relieves the autoinhibition, resulting in Nedd4L activation. Thus, the membrane binding of Nedd4L is coupled with its activation.\n",
      "\n",
      "Stimulatory and inhibitory effects of the FCHO2 BAR domain on the liposome binding and catalytic activity of Nedd4L\n",
      "\n",
      "The FCHO2 BAR domain enhances both the liposome binding and catalytic activity of Nedd4L when the interaction strength of Nedd4L with liposomes is weak (20% PS) (Fig 10A and B). Conversely, the BAR domain inhibits the binding and activity of Nedd4L when the interaction of Nedd4L with liposomes is increased by elevating the PS percentage in liposomes (~50% PS) (Fig 10C and Fig EV5). Thus, the FCHO2 BAR domain exerts both stimulatory and inhibitory effects on Nedd4L, depending on the PS percentage in liposomes. However, given that 20% PS is comparable to the concentration in the inner leaflet of the plasma membrane, it is plausible that FCHO2 exhibits only a stimulatory effect on Nedd4L in cells. The mechanism underlying the stimulatory and inhibitory effects in vitro is as follows: when using liposomes (20% PS), Nedd4L hardly binds to liposomes through PS (Fig. 9F and 10A). The addition of the FCHO2 BAR domain enhances the strength of Nedd4L’s interaction with PS by inducing membrane curvature. Consequently, Nedd4L gains a new binding to liposomes through PS, which augments its catalytic activity. In contrast, when using liposomes (~50% PS), Nedd4L binds to liposome through PS. The addition of the FCHO2 BAR domain hinders the PS-mediated liposome binding of Nedd4L, because FCHO2 and Nedd4L are PS-binding proteins and compete for PS binding in liposomes. Consequently, the BAR domain inhibits the catalytic activity of Nedd4L. This is consistent with the results obtained with 0.05 µm pore-size liposomes (Fig 12B). These 0.05 µm pore-size liposomes are comparable to liposomes with a higher PS percentage, as the interaction of Nedd4L with liposomes through PS is increased by membrane curvature. Therefore, when using 0.05 µm pore-size liposomes, the FCHO2 BAR domain inhibits the catalytic activity of Nedd4L.\n",
      "\n",
      "Based on a model in which a subset of BAR domain proteins, such as FCHO2 and FBP17, drive liposome tubulation in vitro (Frost et al, 2008; Shimada et al, 2007), the FCHO2 BAR domain is considered to polymerize on the liposome surface into spiral dense coats by lateral and tip-to-tip interactions. When using liposomes (~50% PS), it is conceivable that the FCHO2 BAR domain densely packs the liposomes, leaving little free space for Nedd4L binding. Conversely, when using liposomes (20% PS), it is likely that the liposomes are loosely packed, providing ample surface space for Nedd4L binding.\n",
      "\n",
      "Model for ENaC Endocytosis\n",
      "\n",
      "Based on the present study, we propose a model for ENaC endocytosis through ubiquitination (Fig 13). First, FCHO2 binds to the plasma membrane and generates a specific degree of membrane curvature. FCHO2 interacts with clathrin adaptor proteins, such as Eps15, and recruits the clathrin machinery for CCP formation (Uezu et al, 2011; Henne et al, 2010). Eps15 has Ub-interacting motifs (Di Fiore et al, 2003). Nedd4L is subsequently recruited to and activated by FCHO2-generated curved membrane at the rim of nascent CCPs that ENaC enters. ENaC is then ubiquitinated and captured by Eps15 which FCHO2 interacts with. Thus, ubiquitinated ENaC is spatially restricted to CCPs and separated from unmodified ENaC. Overall, our findings suggest that in the early stage of CME, a membrane deformation induced by FCHO2 can program, through both the recruitment and activation of Nedd4L, the gathering of a subclass of cargo molecules in CCPs.\n",
      "\n",
      "Methods\n",
      "\n",
      "Construction\n",
      "\n",
      "Human αENaC, βENaC and γENaC cDNAs were kindly supplied by Dr. H. Kai (Kumamoto University, Kumamoto, Japan) (Sugahara et al, 2009). FLAG-αENaC was constructed by replacing the sequence [amino acid (aa) 169–174, TLVAGS] in the extracellular region with the FLAG sequence (DYKDDDK) as described previously (Firsov et al, 1996). This introduction of the epitope tag was carried out by polymerase chain reaction (PCR). FLAG-αENaC, βENaC, and γENaC were subcloned into pTRE-Tight-Hygro, pCAGI-Puro (Miyahara et al, 2000), and pCAGI-Bst (Umeda et al, 2006), respectively. pTRE-Tight-Hygro was constructed by inserting the hygromycin-resistance gene into pTRE-Tight (Clontech). FLAG-αENaC was also subcloned into pCAGI-Hygro, which was constructed by replacing the puromycin-resistance gene of pCAGI-Puro with the hygromycin-resistance gene. Human Nedd4L (NM\\_015277, isoform 3) cDNA was obtained by reverse transcription (RT)-PCR and cloned in pCMV-Myc (Miyahara et al, 2000). Nedd4L was also cloned in pCAGI-puro-EGFP and pGEX-6P-hexahistidine (His6). pGEX-6P-His6 was constructed from pGEX-6P (GE Healthcare) to express an N-terminal GST-tagged, C-terminal His6-tagged protein. pCAGI-puro-EGFP was constructed by subcloning EGFP cDNA into pCAGI-puro to express a C-terminal GFP-tagged protein. The Nedd4L C2 domain (C2, aa 1–160) and C2 domain-deleted mutant (ΔC2, aa 150–955) were subcloned into pCMV-Myc and/or pGEX-6P. Mouse FCHO2 (full length, aa 1–809; BAR domain, aa 1–302) and human FBP17 (BAR domain, aa 1–300) were subcloned into pEGFP-C1 and/or pGEX-6P (Uezu et al, 2011; Tsujita et al, 2006). FCHO2 (aa 1–327) were subcloned into pGEX-6P. Human amphiphysin1 cDNA was obtained by RT-PCR and cloned in pEGFP-N3. Amphiphysin1 (BAR domain, aa 1–377) was then subcloned into pGEX-6P. mTagRFP-T-Clathrin-15 was a gift from Michael Davidson (Addgene plasmid # 58005 ; http://n2t.net/addgene:58005 ; RRID:Addgene\\_58005) (Shaner et al, 2008). Human hemagglutinin (HA)-tagged Ub cDNA in pCGN was kindly donated by Dr. M. Nakao (Kumamoto University, Kumamoto, Japan). Ub cDNA was subcloned into pET3a. Human UbcH7 cDNA was obtained by RT-PCR and cloned in pGEX-6P. pET21d-Ube1 was a gift from Cynthia Wolberger (Addgene plasmid # 34965 ; http://n2t.net/addgene:34965 ; RRID:Addgene\\_34965) (Berndsen & Wolberger, 2011). pRSET-mSA was a gift from Sheldon Park (Addgene plasmid # 39860 ; http://n2t.net/addgene:39860 ; RRID:Addgene\\_39860) (Lim et al, 2013). mSA-αENaC constructed in pRSET-mSA to express a fusion protein of the N-terminal (aa 7–40) and C-terminal (aa 625–659) intracellular regions of αENaC (see Fig. 9A). Mutagenesis was performed by PCR using mutated primers and a site-directed mutagenesis kit (Stratagene).\n",
      "\n",
      "Protein expression and purification\n",
      "\n",
      "Expression and purification of GST-tagged proteins were performed using standard protocols. His6-tagged proteins were purified with Talon metal affinity resin (Clontech) using the manufacturer’s protocol. The GST tag was cleaved off by PreScission protease (GE Healthcare). The GST tag of BAR domains did not inhibit liposome binding or tubulation, whereas the GST tag of Nedd4L inhibited these functions. As FCHO2 (BAR domain, aa 1–302) was easily aggregated by the removal of the GST tag, FCHO2 (aa 1–327) was used instead as a GST-cleaved form of the FCHO2 BAR domain. Full-length Nedd4L (N-terminal GST-tagged, C-terminal His6-tagged) was purified by glutathione–Sepharose (GE Healthcare) column chromatography. The GST tag was cleaved off and the Nedd4L sample was then subjected to Talon metal affinity column chromatography. After the eluate was passed through glutathione–Sepharose beads, the sample was dialyzed against buffer A (20 mM Tris-HCl at pH 7.5, 100 mM KCl, 0.5 mM EDTA, 1 mM DTT, and 20 % sucrose). The Nedd4L sample was stored at −80°C until use. Ub was expressed in E. coli strain BL21 (DE3) pJY2 and purified as previously described (Pickart & Raasi, 2005). His6-tagged mSA-αENaC was expressed in E. coli strain BL21 (DE3) pLysS and purified as previously described (Lim et al, 2013) with slight modifications. Briefly, the inclusion body was isolated and solubilized in buffer B (50 mM Tris-HCl at pH 8.0 and 6 M guanidine hydrochloride). The sample was then subjected to Talon metal affinity column chromatography. The eluate was added drop by drop to ice-cold refolding buffer (50 mM Tris-HCl at pH 8.0, 150 mM NaCl, 0.15 mg/ml D-biotin, 0.2 mg/ml oxidized glutathione, and 1 mg/ml reduced glutathione) under rapid stirring. The sample was then concentrated using a Centriprep centrifugal filter unit (Millipore) and centrifuged to remove aggregates. The supernatant was dialyzed against buffer C (20 mM Tris-HCl at pH 8.0). The mSA-αENaC sample was further purified by Mono Q anion exchange column chromatography as follows. The sample was applied to a Mono Q 5/50 GL column (GE Healthcare) equilibrated with buffer C. Elution was performed with a 50 ml linear gradient of NaCl (0–200 mM) in buffer C. Fractions of 1 ml each were collected. The active fractions (fraction 18–22) were collected and dialyzed against buffer D (10 mM Tris-HCl at pH 7.5, 100 mM KCl, and 0.5 mM EDTA). The mSA-αENaC sample was stored at −80°C until use.\n",
      "\n",
      "Antibodies\n",
      "\n",
      "Rabbit anti-Nedd4L antibody was raised against GST-Nedd4L C2 (aa 1–160). Rabbit anti-FCHO2 antibody was obtained as previously described (Uezu et al, 2011). The following antibodies were purchased from commercial sources: mouse anti-Myc (9E10) (American Type Culture Collection); mouse anti-FLAG (M2) and mouse anti-α-tubulin (clone DM1A) (Sigma-Aldrich); mouse anti-clathrin heavy chain (BD Biosciences); rabbit anti-GFP (MBL Co.); rabbit anti-αENaC (Calbiochem); rabbit anti-βENaC (Proteintech); rabbit anti-γENaC (Abcam); mouse anti-transferrin receptor H68.4 (Santa Cruz); mouse anti-Ub antibody P4D1 (Santa Cruz); and secondary antibodies conjugated with Alexa Fluor 488, 594, and 647 (Invitrogen).\n",
      "\n",
      "Cell culture and transfection\n",
      "\n",
      "HeLa, HEK293, and COS7 cells were cultured in Dulbecco’s modified Eagle medium (DMEM) supplemented with 10% fetal calf serum. HEK293 cells were grown on poly-L-lysine-coated dishes. Transfection was performed by using Lipofectamine 2000 (Invitrogen) according to the manufacturer’s protocol.\n",
      "\n",
      "Generation of stable HeLa cells expressing all three ENaC subunits\n",
      "\n",
      "HeLa cells were transfected with the pTet-On Advanced vector (Clontech) to develop stable Tet-On cell lines for a doxycycline (Dox)-inducible expression system according to the manufacturer’s protocol. Tet-On cells were selected in 0.5 mg/ml G418. Cloned Tet-On cells were then transfected with pCAGI-Bst-γENaC and selected in 4 µg/ml blasticidin (in the presence of G418). Blasticidin-resistant cells were transfected with pCAGI-Puro-βENaC and selected in 1 µg/ml puromycin (in the presence of G418 and blasticidin). Puromycin-resistant cells were finally transfected with pTRE-Tight-Hygro-FLAG-αENaC and cultured in 0.2 mg/ml hygromycin (in the presence of G418, blasticidin, and puromycin). After selection, cells were cloned and maintained in 0.5 mg/ml G418, 0.2 mg/ml hygromycin, 1 µg/ml puromycin, and 2.5 µg/ml blasticidin. FLAG-tagged αENaC expression was induced by treating cells with 1 µg/ml Dox in the presence of 40 µM amiloride overnight. Amiloride was added to prevent cellular Na+ overload.\n",
      "\n",
      "Immunofluorescence microscopy\n",
      "\n",
      "Cells were fixed with 3.7% formaldehyde in phosphate-buffered saline (PBS) at room temperature for 15 min, and then permeabilized with 0.2% Triton X-100 in PBS for 10 min. After blocking with 1% bovine serum albumin (BSA) in PBS, cells were sequentially incubated with primary antibodies and fluorescence-conjugated secondary antibodies at room temperature for 1 h, respectively. Cells were then analyzed with either a fluorescence microscope (BX51; Olympus) or a confocal microscopy system (BX50, Fluoview FV300, and Fluoview FV1200; Olympus). A 60 × oil immersion objective (NA = 1.40, Olympus) was used. All comparable images were acquired under identical parameters. To visualize cell-surface FLAG-αENaC in αβγENaC-HeLa cells, cells were incubated with 5 µg/ml anti-FLAG M2 antibody at 4°C for 1 h to prevent internalization. Cells were then fixed, permeabilized, and stained with a secondary antibody. Cells were also sequentially incubated with primary (anti-FCHO2) and secondary antibodies. To quantify co-localization, the Pearson's correlation coefficient was calculated from three cells using the Fiji (ImageJ) Coloc2 plug-in.\n",
      "\n",
      "RNA interference\n",
      "\n",
      "Stealth™ double-stranded RNAs were purchased from Invitrogen. siRNA sequences of human FCHO2, Nedd4L, and FBP17 were 5′-CCA CAG AUC UUA GAG UGG AUU AUA A-3′ (corresponding to nucleotides 2048–2072 relative to the start codon), 5′-GAA GAG UUG CUG GUC UGG CCG UAU U-3′ (corresponding to nucleotides 1778–1802 relative to the start codon), and 5′-CCC ACU UCA UAU GUC GAA GUC UGU U-3′for human FBP17 (corresponding to nucleotides 1804–1828 relative to the start codon), respectively. A double-stranded RNA targeting luciferase (GL-2, 5′-CGU ACG CGG AAU ACU UCG AAA UGU C-3′) was used as a control. HeLa cells were transfected with 20 nM siRNA using Lipofectamine 2000 according to the manufacturer’s protocol. After 24 h, a second transfection was performed, and cells were cultured for 3 days and subjected to various experiments. For rescue experiments, cells were transfected with the intended plasmid at 36 h after the second transfection. After 18 h, cells were fixed and analyzed.\n",
      "\n",
      "Quantitative real-time PCR analysis\n",
      "\n",
      "Total RNA was isolated using ISOGEN (Nippon Gene, 319-90211), and cDNA was synthesized using ReverTra Ace qPCR RT Master Mix with gDNA Remover (Toyobo, FSQ-301), following manufacturer’s instruction. Real time PCR was carried out using Thunderbird Next SYBR (Toyobo, QPX-201) on StepOnePlus Real-Time PCR Systems (Applied Biosystems). The sequences of qPCR primers are 5′-TCT CCC ATC GCT TCA ACG AG-3′ and 5′-GTT GCA CCC AGC TTG AGA GA-3′ for human FBP17; 5′-GTC TCC TCT GAC TTC AAC AGC G-3′ and 5′-ACC ACC CTG TTG CTG TAG CCA A-3′ for human GAPDH. The expression of target genes was normalized to GAPDH mRNA levels.\n",
      "\n",
      "Endocytosis assay\n",
      "\n",
      "αβγENaC-HeLa cells at approximately 90% confluence in a 10-cm dish were starved with serum-free medium (DMEM containing 1% BSA, 1 µg/ml Dox, and 40 µM amiloride) at 37°C for 4 h. For ENaC endocytosis, cells were then incubated with 5 µg/ml anti-FLAG M2 in 2 ml of DMEM containing 1% BSA at 4°C for 1 h to prevent internalization. After three washes with ice-cold PBS containing 1 mM MgCl2 and 0.1 mM CaCl2 (PBS-CM), endocytosis was allowed by incubating cells in prewarmed DMEM containing 1% BSA at 37°C for the indicated periods of time. Endocytosis was stopped by placing cells on ice and washing them with ice-cold PBS-CM. Antibody bound to the cell surface but not internalized was removed by acid stripping with 0.2 M acetic acid and 0.5 M NaCl solution (pH 2.5). After washing with ice-cold PBS-CM, internalized ENaC was detected by immunoblotting (IB) or immunofluorescence microscopy. For IB, cells were scraped in 1 ml of lysis buffer A (50 mM Tris-HCl at pH 8.0, 150 mM NaCl, 5 mM EDTA, 1% Triton X-100, 1 mM PMSF, 20 µg/ml leupeptin, and 1 µg/ml pepstatin A). Cells were solubilized at 4°C for 1 h on a rotating wheel, and centrifuged at 20,000 × g at 4°C for 15 min. Protein G-Sepharose beads (GE Healthcare) were added to the supernatant and incubated at 4°C for 3 h. The beads were then thoroughly washed with lysis buffer A and boiled in sodium dodecyl sulfate (SDS) sample buffer. The sample was subjected to SDS-polyacrylamide gel electrophoresis (PAGE), followed by IB with anti-ENaC subunit antibodies. Internalization was expressed as a percentage of the initial amount of cell-surface ENaC subunit determined by incubating cells with anti-FLAG antibody at 4°C for 1 h and lysing them without incubation at 37°C or acid stripping. For immunofluorescence microscopy, cells were transfected with control or FCHO2 siRNA and then with GFP-siRNA-resistant (sr) FCHO2 mutant. After cells were labeled with anti-FLAG antibody on 4°C, they were incubated at 37°C for 10 min to allow ENaC endocytosis. After acid stripping, cells were fixed with 3.7% formaldehyde and permeabilized with 0.2% Triton X-100. The antibody-labeled, internalized αENaC was visualized with a fluorescence-conjugated secondary antibody. The percentage of cells with internalized αENaC among cells expressing GFP-sr-FCHO2 was determined using MetaMorph imaging System software. All images were acquired under identical parameters. For TfR endocytosis, a cell surface biotinylation assay was carried out as previously described (Roberts et al, 2001). Briefly, the cell surface was labeled with Sulfo-NHS-SS-Biotin (Thermo Fisher Scientific) at 4°C. After cells were incubated with ice-cold quenching buffer (50 mM NH4Cl in PBS-CM) at 4°C, they were incubated with serum free medium for 10 min at 37°C. Biotin was removed from the cell surface by incubation with ice-cold MesNa buffer (50 mM MesNa, 100 mM NaCl, and 100 mM Tris-HCl, pH 8.8). Cells were then incubated with iodoacetamide at 4°C. Cell lysates were then incubated with NeutrAvidin beads (Thermo Fisher Scientific). The beads were boiled in SDS sample buffer. Proteins were analyzed by IB with TfR antibody.\n",
      "\n",
      "ENaC expression and ubiquitination at the cell surface\n",
      "\n",
      "αβγENaC-HeLa cells at approximately 90% confluence in a 10-cm dish were starved with serum-free medium (DMEM containing 1% BSA, 1 µg/ml Dox, and 40 µM amiloride) at 37°C for 4 h, washed three times with ice-cold PBS, and incubated at 4°C for 45 min with 2 ml of 1 mg/ml EZ-link Sulfo-NHS-SS-Biotin in PBS with gentle shaking. Subsequently, cells were extensively washed with ice-cold Tris-buffered saline, and then scraped into 1 ml of lysis buffer B (50 mM Tris-HCl at pH 7.5, 150 mM NaCl, 5 mM EDTA, 1% Triton X-100, 10 mM N-ethylmaleimide, 1 mM PMSF, 10 µg/ml leupeptin, and 1 µg/ml pepstatin A) containing 40 µM MG132. Cells were solubilized at 4°C for 1 h on a rotating wheel, and centrifuged at 20,000 × g at 4°C for 15 min. The supernatant (4 mg of protein) was incubated at 4°C for 3 h with 10 µg of anti-FLAG M2 antibody. Protein G-Sepharose beads (50 µl of 50% slurry) were then added to the sample, which was incubated at 4°C for 3 h. After beads were thoroughly washed with lysis buffer B containing 40 µM MG132, FLAG-tagged proteins were eluted from the beads with 200 µg/ml FLAG peptide (Sigma-Aldrich) in 0.5 ml of lysis buffer B. After centrifugation, the supernatant was incubated with 25 µl of NeutrAvidin beads at 4°C overnight. After the beads were thoroughly washed with lysis buffer B, bound proteins were eluted by incubating the beads at 4°C for 1 h with 150 µl of lysis buffer B containing 200 mM DTT. After centrifugation, 5 × SDS sample buffer was added to the supernatant, which was then analyzed by IB with anti-FLAG and anti-Ub antibodies. The chemiluminescence intensity was quantified by scanning using ImageJ software (NIH).\n",
      "\n",
      "TIRF Microscopy\n",
      "\n",
      "For live observation with TIRF microscopy, HeLa cells were grown on a glass-bottomed dish as described previously (Shimada et al, 2007). Cells were observed at 10–12 h after transfection as described previously (Henne et al, 2010). Images were acquired for 5 min at 1 s intervals with a TIRF microscopy system (Olympus), a 100 × oil immersion objective NA = 1.45 (Olympus) and MetaMorph Imaging System Software. All comparable images were acquired under identical parameters.\n",
      "\n",
      "Liposome preparation\n",
      "\n",
      "Phospholipids were purchased from commercial sources: phosphatidylcholine (PC, 1,2-dioleoyl-sn-glycero-3-phosphocholine), phosphatidylethanolamine (PE, 1,2-dioleoyl-sn-glycero-3-phosphoethanolamine), PS (1,2-dioleoyl-sn-glycero-3-phospho-l-serine), rhodamine-PE [18:1 L-α-PE-N-(lissamine rhodamine B sulfonyl)], PEG2000-biotin-PE (1,2-distearoyl-sn-glycero-3-phosphoethanolamine-N-[biotinyl (polyethyleneglycol)-2000]), porcine brain PS, brain PI(4,5)P2, and porcine brain total lipid extract were obtained from Avanti Polar Lipids; cholesterol and Folch lipids [bovine brain lipids (Folch Fraction 1)] were obtained from Sigma-Aldrich; X-biotin-PE [N-((6-(biotinoyl)amino)hexanoyl)-1,2-dihexadecanoylsn-glycero-3-phosphoethanolamine] was obtained from Thermo Fisher Scientific; and fluorescein-PE [N-(fluorescein-5-thiocarbamoyl)-1,2-dihexadecanoyl-sn-glycero-3-phosphoethanolamine] was obtained from AAT Bioquest Inc.\n",
      "\n",
      "Liposomes were prepared using brain lipids (Folch lipids, ~50% PS of total lipids), brain lipids (20% PS of total lipids, a mixture of 89.5% porcine brain total lipid extract and 10.5% porcine brain PS), or a synthetic lipid mixture [70% PC, 20% PE, 10% cholesterol (w/w), and varying percentages of PS or PI(4,5)P2 (with a corresponding reduction in PC)]. Lipid concentrations were monitored by including 1% rhodamine-PE (w/w). To prepare mSA-αENaC–associated liposomes, 2% PEG2000-biotin-PE (w/w) was included. Lipids were dried under nitrogen gas, desiccated for 2 h, and resuspended at 1 mg/ml in lipid buffer (50 mM HEPES-NaOH at pH 7.2, 100 mM KCl, 2 mM MgCl2, and 5 mM EGTA), followed by hydration at 37°C for 1 h. To make liposomes of different sizes, they were subjected to five freeze–thaw cycles and centrifuged at 100,000 × g for 15 min. Precipitated liposomes were then resuspended in the same buffer to make a final concentration of 1 mg/ml. They were subjected to extrusion through polycarbonate filter membranes (Avanti Polar Lipids): liposomes were sequentially passed 9 times through a 0.8-µm filter, 21 times through a 0.4-µm filter, 21 times through a 0.2-µm filter, 21 times through a 0.1-µm filter, 21 times through a 0.05-µm filter, and 21 times through a 0.03-µm filter. Instead of extrusion through a 0.03-µm filter, liposomes with the smallest size were also prepared by sonication three times for 5 s with a Branson Sonifier 25 at power level 1. Liposome diameters were examined by transmission electron microscopy.\n",
      "\n",
      "Co-sedimentation assay\n",
      "\n",
      "A co-sedimentation assay was performed as described (Uezu et al, 2011) with slight modifications. Briefly, 5–10 µg of protein was incubated at 25°C for 10 min with 0.5 mg/ml liposomes in 100 µl of lipid buffer containing various concentrations of CaCl2 and centrifuged at 165,000 × g at 25°C for 15 min. Equal amounts of the supernatants and pellets were subjected to SDS-PAGE, followed by Coomassie brilliant blue (CBB) staining. Proteins were quantified by scanning using ImageJ (NIH). The assay with liposomes of different sizes was performed using NeutrAvidin (Thermo Fisher Scientific Inc) as described (Sakamoto et al, 2017). Briefly, brain-lipid (Folch-lipid) liposomes (~50% PS) were supplemented with 1% X-biotin-PE and 1% rhodamine-PE, subjected to extrusion through filter membranes, and then sonicated. Purified Nedd4L protein (10 µg) was incubated with 0.2 mg/ml liposomes in 100 µl of lipid buffer containing 2mM CaCl2 (0.1 µM Ca2+). The sample was then incubated with 2 µg of NeutrAvidin and ultracentrifuged. Both total and supernatant fluorescence were measured. The proportion of precipitated liposomes of each size was ~100%.\n",
      "\n",
      "The assay with mSA-αENaC–associated liposomes was performed as follows. Liposomes were prepared from a synthetic lipid mixture [various percentages of PS or 5% PI(4,5)P2] or brain lipids supplemented with 2% PEG2000-biotin-PE and 1% rhodamine-PE. The liposomes (10 µg) were incubated with purified mSA-αENaC protein (35 pmol) at 4°C for 10 min in 25 µl of lipid buffer. The sample was then incubated at 25°C for 10 min with 0.07 µM (8 µg/ml) Nedd4L and/or 1.4 µM BAR domains in 50 µl of ubiquitination buffer (50 mM HEPES-NaOH at pH 7.2, 100 mM KCl, 2 mM MgCl2, 5 mM EGTA, and 0.1 mM DTT) containing 4 mM CaCl2 (0.7 µM Ca2+). Subsequently, the sample was subjected to ultracentrifugation. Equal amounts of the total samples and pellets were subjected to SDS-PAGE, followed by CBB staining and IB. The chemiluminescence intensity was quantified by scanning using ImageJ (NIH).\n",
      "\n",
      "The assay with mSA-αENaC-associated liposomes (0.05 µm pore-size) was done as follows. Liposomes were prepared from brain lipids (20% PS) supplemented with 2% PEG2000-biotin-PE, 1% rhodamine-PE, and 0.5% fluorescein-PE (w/w) and extruded through membrane filters. To precipitate liposomes, the sample (50 μl) was first incubated with 1 µg of anti-fluorescein goat antibody (Novus Biologicals) at room temperature for 1 h and then incubated with 0.5 µg of protein A/G/L (BioVision) at room temperature for 1 h. Subsequently, the sample was centrifuged at 165,000 × g at 25°C for 15 min. Equal amounts of the supernatants and pellets were subjected to SDS-PAGE, followed by CBB staining and IB. The percentage of precipitated liposomes was ~100%.\n",
      "\n",
      "Calculation of free Ca2+ concentrations\n",
      "\n",
      "At 1.0, 2.0, 3.0, 3.5, 4.0, 4.5, and 4.9 mM CaCl2 in lipid buffer, free Ca2+ concentrations were calculated to be 0.04, 0.1, 0.3, 0.4, 0.7, 1.6, and 8.1 µM, respectively, as determined by the Ca-Mg-ATP-EGTA Calculator v1.0 using constants from the NIST database #46 v8. At 1.0, 2.0, 3.0, 3.5, 4.0, 4.5, and 4.9 mM CaCl2 in ubiquitination buffer, free Ca2+ concentrations were calculated to be 0.04, 0.1, 0.2, 0.4, 0.7, 1.4, and 6.7 µM, respectively.\n",
      "\n",
      "In vitro tubulation assay\n",
      "\n",
      "An in vitro tubulation assay was performed as described (Uezu et al, 2011; Tsujita et al, 2006) with slight modifications. Briefly, liposomes were prepared from brain lipids (Folch lipids) supplemented with 1 % rhodamine-PE. The liposomes (0.2 mg/ml liposomes) were incubated with the Nedd4L C2 domain (4.5 µM) or BAR domains (1.4 µM) at 25°C for 10 min in lipid buffer containing 3 mM CaCl2 (0.3 µM Ca2+). The sample was examined by fluorescence or electron microscopy. The tubulation assay with mSA-αENaC-associated liposomes was performed using the sample mixture utilized in the co-sedimentation assay in which mSA-αENaC-associated liposomes (brain-lipid liposomes) were used.\n",
      "\n",
      "Negative-staining electron microscopy\n",
      "\n",
      "Negative-staining transmission electron microscopy was performed as described (Shimada et al, 2007). Box and whisker plots were used to represent the distribution of liposome diameters.\n",
      "\n",
      "In vivo ubiquitination assay\n",
      "\n",
      "HEK293 cells were used, because they formed membrane tubules upon FCHO2 expression more efficiently than did COS7 cells. HEK293 cells at approximately 90% confluence on a 6-well plate were transfected with 0.8 µg of pCMV-Myc-Nedd4L, 1.6 µg of pEGFP-BAR domain, and 1.6 µg of pCGN-HA-Ub. In some experiments, cells were transfected with various doses (0–1.5 µg) of pCMV-Myc-Nedd4L, 1.5 µg of pEGFP-FCHO2 BAR domain, 1 µg of pCGN-HA-Ub, and 0.5 µg of pCAGI-Hygro-FLAG-αENaC. Total transfected cDNA amount was held constant using empty pCMV-Myc vector. Amiloride (40 µM) was added to the medium when αENaC cDNA was transfected. At 21 h after transfection, cells were washed with PBS and incubated with serum-free DMEM containing 1% BSA. After 3 h incubation, 10 µM MG132 was added to the medium. After further 4 h incubation, cells were solubilized in 150 µl of lysis buffer C (50 mM Tris-HCl at pH 7.5, 150 mM NaCl, 5 mM EDTA, 10 mM N-ethylmaleimide, 1 mM PMSF, 10 µg/ml leupeptin, and 1µg/ml pepstatin A) containing 1% SDS. After boiling, the sample was sonicated and then centrifuged at 20,000 × g for 15 min. The supernatant was diluted 10-fold with lysis buffer C containing 1% Triton X-100 and then incubated at 4°C for 3 h with either anti-Myc or anti-FLAG M2 antibody. Protein G-Sepharose beads were added to the sample, which was then incubated at 4°C for 3 h. After the beads were thoroughly washed with lysis buffer C containing 1% Triton X-100, bound proteins were eluted by boiling in SDS sample buffer for 5 min. These proteins were then subjected to SDS-PAGE, followed by IB.\n",
      "\n",
      "In vitro ubiquitination assay\n",
      "\n",
      "mSA-αENaC-associated liposomes were made by incubating liposomes (10 µg) with mSA-αENaC (35 pmol) in 25 µl of lipid buffer, as described for the co-sedimentation assay. They were then incubated at 25°C for 10 min with 0.07 µM Nedd4L, 0.04 µM E1 (Ube1), 0.7 µM E2 (UbcH7), 7 µM Ub, and various concentrations of BAR domains in 50 µl of ubiquitination buffer containing 4 mM CaCl2 (0.7 µM Ca2+) and 2 mM ATP. Where indicated, various concentrations of CaCl2 were used. The ubiquitination reaction was started by adding ATP and stopped by adding 10 µl of 6 × SDS sample. The sample was then boiled, subjected to SDS-PAGE, and immunoblotted with anti-Ub and anti-αENaC antibodies. The chemiluminescence intensity was quantified by scanning using ImageJ (NIH).\n",
      "\n",
      "Other procedures\n",
      "\n",
      "Stripping immunoblots for re-probing was carried out with the WB Stripping Solution Strong (Nacalai Tesque) according to the manufacturer’s protocol. Protein concentrations were quantified with BSA as a reference protein by the Bradford method (Bio-Rad) or the BCA method (Thermo Fisher Scientific). SDS-PAGE was performed as described (Laemmli, 1970).\n",
      "\n",
      "Statistical analysis\n",
      "\n",
      "All experiments were performed at least three times (biological replication) and representative results are shown. Data are shown as the mean ± S.E.M. Student’s tests, one-way analysis of variance test, and two-way analysis of variance test were used to evaluate statistical significances between different treatment groups.\n",
      "\n",
      "Data availability\n",
      "\n",
      "This study includes no data deposited in external repositories.\n",
      "\n",
      "Acknowledgments\n",
      "\n",
      "We thank Drs. H. Kai and M. Nakao for kindly providing us with ENaC and Ub cDNAs, respectively. We are also grateful to Dr. T. Takenawa (Kobe University, Kobe, Japan) for helpful discussions. This work was partly carried out at the Institute of Molecular Embryology and Genetics, the Gene Technology Centre, and Research Facilities of the School of Medicine, Kumamoto University. This study was supported by JSPS KAKENHI Grant Numbers 19K06643 (to H.N.), 19K06544 (to Y.S.), and Setsuro Fujii Memorial, The Osaka Foundation for Promotion of Fundamental Medical Research (to Y.S.) and by grants from the Mochida Memorial Foundation, the Takeda Science Foundation, and the Uehara Memorial Foundation (to K.K.).\n",
      "\n",
      "Disclosure and Competing Interests Statement\n",
      "\n",
      "The authors declare no competing interests.\n",
      "\n",
      "References\n",
      "\n",
      "Aguet F, Antonescu CN, Mettlen M, Schmid S & Danuser G (2013) Advances in analysis of low signal-to-noise images link dynamin and AP2 to the function of an endocytic checkpoint. Dev Cell 26: 279–291\n",
      "\n",
      "Berndsen CE & Wolberger C (2011) A spectrophotometric assay for conjugation of ubiquitin and ubiquitin-like proteins. Anal Biochem 418: 102–110\n",
      "\n",
      "Butterworth MB (2010) Regulation of the epithelial sodium channel (ENaC) by membrane trafficking. Biochim Biophys Acta 1802: 1166–77\n",
      "\n",
      "Chang-Ileto B, Frere SG, Chan RB, Voronov SV, Roux A, & Di Paolo G (2011) Synaptojanin 1-mediated PI(4,5)P2 hydrolysis is modulated by membrane curvature and facilitates membrane fission. Dev. Cell 20: 206–218\n",
      "\n",
      "Doherty GJ & McMahon HT (2009) Mechanisms of endocytosis. Annu Rev Biochem 78: 857–902\n",
      "\n",
      "Di Fiore PP, Polo S & Hofmann K (2003) When ubiquitin meets ubiquitin receptors: a signalling connection. Nat Rev Mol Cell Biol 4: 491–497\n",
      "\n",
      "Escobedo A, Gomes T, Aragon E, Martin-Malpartida P, Ruiz L, & Macias MJ (2014) Structural basis of the activation and degradation mechaisms of the E3 ubiquitin lagase Nedd4L. Structure 22: 1446–1457\n",
      "\n",
      "Firsov D, Schild L, Gautschi I, Mérillat AM, Schneeberger E & Rossier BC (1996) Cell surface expression of the epithelial Na channel and a mutant causing Liddle syndrome: a quantitative approach. Proc Natl Acad Sci U S A 93: 15370–15375\n",
      "\n",
      "Frost A, Perera R, Roux A, Spasov K, Destaing O, Egelman EH, De Camilli P & Unger VM (2008) Structural basis of membrane invagination by F-BAR domains. Cell 132: 807–817\n",
      "\n",
      "Henne WM, Boucrot E, Meinecke M, Evergren E, Vallis Y, Mittal R & McMahon HT (2010) FCHo proteins are nucleators of clathrin-mediated endocytosis. Science 328: 1281–4\n",
      "\n",
      "Henne WM, Kent HM, Ford MGJ, Hegde BG, Daumke O, Butler PJG, Mittal R, Langen R, Evans PR & McMahon HT (2007) Structure and Analysis of FCHo2 F-BAR Domain: A Dimerizing and Membrane Recruitment Module that Effects Membrane Curvature. Structure 15: 839–852\n",
      "\n",
      "Hershko A & Ciechanover A (1998) THE UBIQUITIN SYSTEM. Annu Rev Biochem 67: 425–479\n",
      "\n",
      "Hughey RP, Mueller GM, Bruns JB, Kinlough CL, Poland PA, Harkleroad KL, Carattino MD & Kleyman TR (2003) Maturation of the epithelial Na+ channel involves proteolytic processing of the alpha- and gamma-subunits. J Biol Chem 278: 37073–37082\n",
      "\n",
      "Hui E, Johnson CP, Yao J, Dunning FM & Chapman ER (2009) Synaptotagmin-mediated bending of the target membrane is a critical step in Ca(2+)-regulated fusion. Cell 138: 709–21\n",
      "\n",
      "Ingham RJ, Gish G & Pawson T (2004) The Nedd4 family of E3 ubiquitin ligases: functional diversity within a common modular architecture. Oncogene 23: 1972–84\n",
      "\n",
      "Laemmli UK (1970) Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature 227: 680–685\n",
      "\n",
      "Lehmann M, Lukonin I, Noe F, Schmoranzer J, Clementi C, Loerke D & Haucke V (2019) Nanoscale coupling of endocytic pit growth and stability. Sci Adv 5: eaax5775\n",
      "\n",
      "Lemmon M (2008) Membrane recognition by phospholipid-binding domains. Nat Rev Mol Cell Biol 9: 99–111\n",
      "\n",
      "Lim KH, Huang H, Pralle A & Park S (2013) Stable, high-affinity streptavidin monomer for protein labeling and monovalent biotin detection. Biotechnol Bioeng 110: 57–67\n",
      "\n",
      "Martens S, Kozlov MM & McMahon HT (2007) How synaptotagmin promotes membrane fusion. Science 316: 1205–8\n",
      "\n",
      "Miyahara M, Nakanishi H, Takahashi K, Satoh-Horikawa K, Tachibana K & Takai Y (2000) Interaction of nectin with afadin is necessary for its clustering at cell-cell contact sites but not for its cis dimerization or trans interaction. J Biol Chem 275: 613–618\n",
      "\n",
      "Nishizuka Y (1992) Intracellular signaling by hydrolysis of phospholipids and activation of protein kinase C. Science 258: 607–14\n",
      "\n",
      "Peter BJ, Kent HM, Mills IG, Vallis Y, Butler PJG, Evans PR & McMahon HT (2004) BAR domains as sensors of membrane curvature: the amphiphysin BAR structure. Science 303: 495–9\n",
      "\n",
      "Pickart CM & Raasi S (2005) Controlled synthesis of polyubiquitin chains. Methods Enzym 399: 21–36\n",
      "\n",
      "Roberts M, Barry S, Woods A, van der Sluijs P, & Norman J (2001) PDGF-regulated rab4-dependent recycling of avb3 integrin from early endosomes is necessary for cell adhesion and spreading. Curr Biol 11: 1392–1402\n",
      "\n",
      "Rossier BC (2014) Epithelial sodium channel (ENaC) and the control of blood pressure. Curr Opin Pharmacol 15: 33–46\n",
      "\n",
      "Rotin D & Kumar S (2009) Physiological functions of the HECT family of ubiquitin ligases. Nat Rev Mol Cell Biol 10: 398–409\n",
      "\n",
      "Rotin D & Staub O (2011) Role of the ubiquitin system in regulating ion transport. Pflugers Arch 461: 1–21\n",
      "\n",
      "Saffarian S, Coccuci E, & Kirchausen T (2009) Distinct dynamics of endocytic clathrin-coated pits and coated palques. PLoS Biol 7: e1000191\n",
      "\n",
      "Sakamoto Y, Kikuchi K, Umeda K & Nakanishi H (2017) Effects of various spacers between biotin and the phospholipid headgroup on immobilization and sedimentation of biotinylated phospholipid-containing liposomes facilitated by avidin-biotin interactions. J Biochem 162: 221–226\n",
      "\n",
      "Shaner NC, Lin MZ, McKeown MR, Steinbach PA, Hazelwood KL, Davidson MW & Tsien RY (2008) Improving the photostability of bright monomeric orange and red fluorescent proteins. Nat Methods 5: 545–551\n",
      "\n",
      "Shimada A, Niwa H, Tsujita K, Suetsugu S, Nitta K, Hanawa-Suetsugu K, Akasaka R, Nishino Y, Toyama M, Chen L, et al (2007) Curved EFC/F-BAR-domain dimers are joined end to end into a filament for membrane invagination in endocytosis. Cell 129: 761–772\n",
      "\n",
      "Sugahara T, Koga T, Ueno-Shuto K, Shuto T, Watanabe E, Maekawa A, Kitamura K, Tomita K, Mizuno A, Sato T, et al (2009) Calreticulin positively regulates the expression and function of epithelial sodium channel. Exp Cell Res 315: 3294–3300\n",
      "\n",
      "Takano K, Toyooka K & Suetsugu S (2008) EFC/F-BAR proteins and the N-WASP–WIP complex induce membrane curvature-dependent actin polymerization. EMBO J 27: 2817–2828\n",
      "\n",
      "Takei K, Slepnev VI, Haucke V & De Camilli P (1999) Functional partnership between amphiphysin and dynamin in clathrin-mediated endocytosis. Nat Cell Biol 1: 33–39\n",
      "\n",
      "Traub LM (2009) Tickets to ride: selecting cargo for clathrin-regulated internalization. Nat Rev Mol Cell Biol 10: 583–96\n",
      "\n",
      "Tsujita K, Suetsugu S, Sasaki N, Furutani M, Oikawa T & Takenawa T (2006) Coordination between the actin cytoskeleton and membrane deformation by a novel membrane tubulation domain of PCH proteins is involved in endocytosis. J Cell Biol 172: 269–279\n",
      "\n",
      "Uezu A, Umeda K, Tsujita K, Suetsugu S, Takenawa T & Nakanishi H (2011) Characterization of the EFC/F-BAR domain protein, FCHO2. Genes Cells 16: 868–878\n",
      "\n",
      "Umeda K, Ikenouchi J, Katahira-Tayama S, Furuse K, Sasaki H, Nakayama M, Matsui T, Tsukita S, Furuse M & Tsukita S (2006) ZO-1 and ZO-2 independently determine where claudins are polymerized in tight-junction strand formation. Cell 126: 741–754\n",
      "\n",
      "Wang J, Peng Q, Lin Q, Childress C, Carey D & Yang W (2010) Calcium activates Nedd4 E3 ubiquitin ligases by releasing the C2 domain-mediated auto-inhibition. J Biol Chem 285: 12279–12288\n",
      "\n",
      "Wiesner S, Ogunjimi AA, Wang H-R, Rotin D, Sicheri F, Wrana JL & Forman-Kay JD (2007) Autoinhibition of the HECT-type ubiquitin ligase Smurf2 through its C2 domain. Cell 130: 651–862\n",
      "\n",
      "Yang B & Kumar S (2010) Nedd4 and Nedd4-2: closely related ubiquitin-protein ligases with distinct physiological functions. Cell Death Differ 17: 68–77\n",
      "\n",
      "Yin Y, Arkhipov A & Schulten K (2009) Simulations of Membrane Tubulation by Lattices of Amphiphysin N-BAR Domains. Structure 17: 882–892\n",
      "\n",
      "Zaccai N, Kadlecova Z, Dickson VK, Korobchevskaya K & Traub L (2022) FCHO control AP2's initiating role in endocytosis through a Ptd-Ins(4,5)P2-dependent switch. Sci Adv 8: eabn2018\n",
      "\n",
      "Zhou R, Patel S V. & Snyder PM (2007) Nedd4-2 catalyzes ubiquitination and degradation of cell surface ENaC. J Biol Chem 282: 20207–20212\n",
      "\n",
      "Zhu K, Shan Z, Chen X, Cai Y, Cui L, Yao W, Wang Z, Shi P, Tian C, Lou J, et al (2017) Allosteric auto‐inhibition and activation of the Nedd4 family E3 ligase Itch. EMBO Rep 18: 1618–1630\n",
      "\n",
      "Figure legends\n",
      "\n",
      "Figure 1. FCHO2 is required for Nedd4L-mediated ubiquitination and endocytosis of ENaC.\n",
      "\n",
      "A. Knockdown of FCHO2 or Nedd4L by siRNA. αβγENaC-HeLa cells treated with each siRNA were analyzed by immunoblotting (IB).\n",
      "\n",
      "B. Inhibition of ENaC endocytosis by knockdown of FCHO2 or Nedd4L. After cells were transfected with each siRNA, cell-surface ENaC was labeled by incubating at 4°C with anti-FLAG antibody. Endocytosis was started by incubating cells at 37°C for the indicated periods of time and stopped by chilling cells on ice. Antibody bound to the cell surface but not internalized was removed by acid stripping. After cells were solubilized, the antibody-labeled internalized ENaC subunits were precipitated and analyzed by IB (upper panel) and quantified (bottom and right panels). Internalization was expressed as a percentage of the initial amount of cell-surface αENaC determined by lysing cells without incubation at 37°C or acid stripping. Data are shown as the mean ± S.E.M. of three independent experiments. **P < 0.01; ***P < 0.001 (Student's t-test).\n",
      "\n",
      "C. Inhibition of transferrin receptor (TfR) endocytosis by FCHO2 knockdown. After cells were transfected with each siRNA, cell-surface proteins were biotinylated at 4°C. Cells were then incubated at 37°C for 10 min. Biotin was removed from the cell surface, and internalized biotinylated proteins were precipitated with avidin beads. Samples were subjected to IB with anti-TfR antibody and quantified. Internalization was expressed as a proportion of the initial amount of TfR on the cell surface. Data are shown as the mean ± S.E.M. of three independent experiments. *P < 0.05 (Student's t-test).\n",
      "\n",
      "D. Restoration of αENaC endocytosis by FCHO2 re-expression in knockdown cells. After cells were transfected with the indicated siRNA and GFP construct, cell-surface αENaC was labeled with anti-FLAG antibody at 4°C. Cells were then incubated at 37°C for 10 min. After acid stripping, cells were subjected to immunofluorescence microscopy (left panels) and quantified (right panel). The antibody-labeled, internalized αENaC was visualized with a fluorescence-conjugated secondary antibody. The border of each cell is delineated by a solid line. Arrows indicate cells expressing GFP-tagged proteins. Scale bars, 10 μm. The proportion of cells with internalized αENaC among cells expressing control GFP or GFP-siRNA-resistant (sr) -FCHO2 is shown. Data are shown as the mean ± S.E.M. of five independent images (31-71 cells expressing GFP-tagged proteins per image). ***P < 0.001 (two-way analysis of variance with post hoc test).\n",
      "\n",
      "E, F. Effects of FCHO2, Nedd4L, and FBP17 knockdown on ENaC ubiquitination and expression at the cell surface. After cells were transfected with each siRNA, cell-surface αENaC was biotinylated and sequentially precipitated with anti-FLAG antibody and then avidin beads. Samples were subjected to IB with anti-FLAG and anti-Ub antibodies (E) and quantified (F). E, ENaC ubiquitination, Asterisks indicate the same band of ubiquitinated FLAG-αENaC. F, ENaC expression at the cell surface. Data are shown as the mean ± S.E.M. of three independent experiments. *P < 0.05 (Student's t-test).\n",
      "\n",
      "Figure 2. Localization of cell-surface ENaC, FCHO2, and Nedd4L at clathrin-coated structures.\n",
      "\n",
      "A, B. Localization of cell-surface ENaC. mTagRFP-clathrin light chain (CLC) (A) or Nedd4L C922A (catalytic inactive mutant)-GFP (B) was expressed in αβγENaC-HeLa cells. Cell-surface αENaC was labeled with anti-FLAG antibody at 4°C. Cells were then fixed and stained with anti-FCHO2 antibody. Samples were subjected to immunofluorescence microscopy (left panel). Insets represent enlarged images of the dashed boxes. Scale bar, 10 μm. Co-localization was analyzed (right panel). The mean values of Pearson’s correlation coefficient from three cells and the S.E.M. are shown. A. Co-localization of cell-surface ENaC with FCHO2 at clathrin-coated structures. B. Co-localization of cell-surface ENaC with Nedd4L C922A mutant and FCHO2.\n",
      "\n",
      "C. Co-localization of Nedd4L with FCHO2 at clathrin-coated structures. Nedd4L-GFP and mTagRFP-CLC were co-expressed in wild-type HeLa cells. Cells were then fixed and stained with anti-FCHO2 antibody (left panel). Insets represent enlarged images of the dashed boxes. Scale bar, 10 μm. Co-localization was analyzed (right panel). The mean values of Pearson’s correlation coefficient from three cells and the S.E.M. are shown.\n",
      "\n",
      "Figure 3. Recruitment of Nedd4L to CCPs. DsRed-CLC were co-expressed with Nedd4L-GFP in wild-type HeLa cells. Images were acquired at 1 s intervals through TIRF microscopy. Upper panel, TIRF images of HeLa cells. Insets represent enlarged images of the dashed boxes. Bottom panel, selected snapshots (every 10 sec) from a time series were obtained from a single CCP indicated by the arrowhead. The time point at which Nedd4L started to accumulate was set 0. Scale bar, 2 μm.\n",
      "\n",
      "Figure 4. Recruitment of Nedd4L to membrane tubules generated by FCHO2 in cells.\n",
      "\n",
      "GFP-BAR domains were co-expressed with Myc-Nedd4L proteins in COS7 cells (A) and HEK293 cells (B). Cells were serum-starved and subjected to immunofluorescence microscopy. Insets represent enlarged images of the dashed boxes. Scale bar, 10 μm.\n",
      "\n",
      "Figure 5. Activation of Nedd4L by membrane tubules generated by FCHO2 in cells.\n",
      "\n",
      "A–E. Nedd4L autoubiquitination. An in vivo ubiquitination assay was performed with various BAR domains and FCHO2 mutants (A, B) and Nedd4L mutants (D). GFP-BAR domains were co-expressed with Myc-Nedd4L proteins and HA-tagged Ub in HEK293 cells. Cell lysates were subjected to immunoprecipitation (IP) and analyzed by IB (A, B, D). WT, wild type. Cells were also subjected to immunofluorescence microscopy (C, E). Scale bar, 10 μm.\n",
      "\n",
      "F. αENaC ubiquitination. An in vivo ubiquitination assay was performed with various concentrations of Myc-Nedd4L using FALAG-αENaC as a substrate in the presence or absence of GFP-FCHO2 BAR domain. Cell lysates were subjected to IP. Samples were analyzed by IB.\n",
      "\n",
      "Figure 6. The Nedd4L C2 domain is a Ca2+-dependent PS binding domain.\n",
      "\n",
      "A co-sedimentation assay was performed with the C2 domain or full-length Nedd4L using brain-lipid liposomes (~50% PS) at various Ca2+ concentrations (A) and with the C2 domain using synthetic liposomes containing various percentages of PS at 0.3 μM or 1.0 mM Ca2+ (B). The supernatants (S) and pellets (P) were subjected to sodium dodecyl sulfate (SDS)-polyacrylamide gel electrophoresis (PAGE), followed by Coomassie brilliant blue (CBB) staining (upper panels). Bottom panels, quantitative analysis. (−) for [Ca2+] indicates EGTA alone. Data are shown as the mean ± S.E.M. of three independent experiments.\n",
      "\n",
      "A. Ca2+ dependency.\n",
      "\n",
      "B. PS dependency.\n",
      "\n",
      "Figure 7. Preference of Nedd4L for FCHO2-generated membrane curvature.\n",
      "\n",
      "A. Distribution of liposome diameters. Brain-lipid liposomes (~50% PS) of different diameters were prepared by extrusion through the indicated pore sizes and sonication. Left panel, electron microscopic images. Scale bars, 200 nm. Right panel, distribution of liposome diameters shown by boxplots (number of observations per pore size = 25). The center line inside the box corresponds to the median, the bounds of the box encompass data points between the first and third quartiles, and the whiskers extend to the minimum and maximum values including outliers. So, sonication.\n",
      "\n",
      "B-D. Sensing and generation of membrane curvature by Nedd4L. Co-sedimentation (B) and in vitro tubulation (C, D) assays were performed using brain-lipid liposomes (~50% PS) at 0.1 μM (B) and 0.3 μM Ca2+ (C, D). B, Preference of Nedd4L for a specific degree of membrane curvature. Liposomes were supplemented with X-biotin-PE and rhodamine-PE. Full-length Nedd4L was incubated with liposomes of different sizes, which were prepared by extrusion through the indicated pore sizes. The sample was then incubated with avidin beads and ultracentrifuged. The proportion of precipitated liposomes of each size was calculated to be ~100% by measuring both total and supernatant fluorescence. The supernatants (S) and pellets (P) were subjected to SDS- PAGE, followed by CBB staining (upper panel). Bottom panel, quantitative analysis. So, sonication. Data are shown as the mean ± S.E.M. of five independent experiments. *P < 0.05; **P < 0.01 (one-way analysis of variance with Tukey’s post hoc test). C, Curvature generation by Nedd4L. The assay was performed with the Nedd4L C2 domain. Samples were examined by electron microscopy. Scale bar, 100 nm. D, The degree of membrane curvature generated by Nedd4L is consistent with that generated by FCHO2. The assay was performed with the indicated protein. Left panel, electron microscopic image. Scale bar, 100 nm. Right panel, distribution of tubule diameters shown by boxplots (number of observations per protein = 14–33). The center line inside the box corresponds to the median, the bounds of the box encompass data points between the first and third quartiles, and the whiskers extend to the minimum and maximum values including outliers.\n",
      "\n",
      "Figure 8. Conserved hydrophobic residues of Nedd4L are responsible for sensing and generation of membrane curvature.\n",
      "\n",
      "A. Sequence alignment of C2 domains. Conserved residues are highlighted with the following color code: yellow, aspartates for Ca2+ binding; blue, very hydrophobic residues probably for penetration into the lipid bilayer; and brown, others. Dots show the mutation sites. Hs, Homo sapiens; Sc, Saccharomyces cerevisiae; and Rt, Rattus norvegicus.\n",
      "\n",
      "B. Inability of Nedd4L C2 domain mutants to sense membrane curvature in cells. GFP-FCHO2 BAR domain and Myc-Nedd4L C2 domain mutants were co-expressed in COS7 cells. Insets are enlarged images of dashed boxes. Scale bar, 10 μm.\n",
      "\n",
      "C, D. Inability of Nedd4L C2 domain mutants to bind to membranes and to generate membrane curvature in vitro. Co-sedimentation (C) and in vitro tubulation (D) assays were performed using brain-lipid liposomes (~50% PS) at 0.3 μM Ca2+. C, Membrane binding. The assay was performed with Nedd4L C2 domain mutants using various concentrations of liposomes. The supernatants (S) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining (upper panel). Bottom panel, quantitative analysis. Data are shown as the mean ± S.E.M. of three independent experiments. D, Curvature generation. The assay was performed with Nedd4L C2 domain mutants. Samples were examined by fluorescence microscopy. Scale bar, 10 μm.\n",
      "\n",
      "Figure 9. Properties of the catalytic activity and membrane binding of Nedd4L.\n",
      "\n",
      "Liposomes supplemented with biotinylated phospholipids were incubated with mSA-αENaC (A, B). Samples were mixed with Nedd4L. Mixtures were subjected to ubiquitination (C-E) and co-sedimentation (F) assays. These assays were performed with brain-lipid liposomes (~50% PS) at various Ca2+ concentrations (C), with synthetic liposomes (various percentages of PS) at 0.7 μM Ca2+ (D, F), and with brain-lipid liposomes (20% PS) at 0.7 μM Ca2+ (E). Ubiquitination and liposome binding were quantified by scanning and expressed as ratios to maximum levels. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.\n",
      "\n",
      "A, B. Schematic diagrams of mSA-αENaC. A, Structure. B. Binding to biotinylated liposomes.\n",
      "\n",
      "C. Ca2+-dependent Nedd4L activity. Ubiquitination was detected by IB (upper panel) and the intensity was quantified by scanning (bottom panel). Data are shown as the mean ± S.E.M. of three independent experiments.\n",
      "\n",
      "D, Membrane localization of mSA-αENaC enhances the PS-dependent catalytic activity of Nedd4L. Assays were performed in the presence or absence of 40 μM biotin. Ubiquitination was detected by IB (left panel). The intensities were quantified by scanning (right panel). Data are shown as the mean ± S.E.M. of three independent experiments.\n",
      "\n",
      "E. Nedd4L activity is predominantly stimulated by the specific degree of membrane curvature. The assay was performed using liposomes of different sizes which were prepared by extrusion through the indicated pore sizes. Ubiquitination was detected by IB (left panel) and quantified (right panel). Data are shown as the mean ± S.E.M. of three independent experiments. ***P < 0.001 (one-way analysis of variance with Tukey’s post hoc test).\n",
      "\n",
      "F. Membrane localization of mSA-αENaC enhances the PS-dependent membrane binding of Nedd4L. Assays were performed in the presence or absence of 40 μM biotin. Liposome binding was detected by IB (left panel). The intensities were quantified by scanning (right panel). T, total sample. P, pellet. Data are shown as the mean ± S.E.M. of three independent experiments.\n",
      "\n",
      "Figure 10. Recruitment and activation of Nedd4L by FCHO2-generated membrane curvature in vitro.\n",
      "\n",
      "mSA-αENaC-associated brain-lipid liposomes (20% PS) were mixed with Nedd4L in the presence or absence of BAR domains. Mixtures were subjected to co-sedimentation (A) and ubiquitination (B-E) assays. These assays were performed at 0.7 μM Ca2+ with brain-lipid liposomes (20%PS) (A, B, D, and E) and with the indicated (20% PS or ~50% PS) brain-lipid liposomes (C). Ubiquitination and liposome binding were quantified by scanning and expressed as ratios to maximum levels. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.\n",
      "\n",
      "A. FCHO2-induced recruitment of Nedd4L. The assay was performed with the FCHO2 BAR domain (1.4 μM, 50 μg/ml) in the presence or absence of biotin. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining (FCHO2 BAR) and IB (Nedd4L and mSA-αENaC).\n",
      "\n",
      "B, C. FCHO2-induced activation of Nedd4L. B, Effects of various doses of the FCHO2 BAR domain. The assay was performed with various concentrations of the FCHO2 BAR domain. Ubiquitination was detected by IB (left panel) and the intensity was quantified by scanning (right panel). C, Effects of the PS percentage in liposomes. The assay was performed with the FCHO2 BAR domain (1.4 μM). Samples were analyzed by IB. Data are shown as the mean ± S.E.M. of three independent experiments.\n",
      "\n",
      "D, E. Specificity of BAR domains and effect of an FCHO2 mutant. The assay was performed with various BAR domains (D) and an FCHO2 mutant (E) (1.4 μM each). Samples were analyzed by IB.\n",
      "\n",
      "Figure 11. Effects of PI(4,5)P2 on FCHO2-induced recruitment and activation of Nedd4L.\n",
      "\n",
      "A, B. Binding to PI(4,5)P2. A co-sedimentation assay was performed at 0.7 μM Ca2+ with the FCHO2 BAR or Nedd4L C2 domain using mSA-αENaC-associated synthetic liposomes containing 20% PS, 5% PI(4,5)P2 (A), or various percentages of PI(4,5)P2 (B). The supernatants (S) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining (A and upper panel in B). Bottom panel in B, quantitative analysis. Control, synthetic liposomes [0% PS and 0% PI(4,5)P2]. Data are shown as the mean ± S.E.M. of three independent experiments.\n",
      "\n",
      "C, D. Effects on FCHO2-induced recruitment and activation of Nedd4L. Co-sedimentation (C) and ubiquitination (D) assays were performed at 0.7 μM Ca2+ with mSA-αENaC-associated synthetic liposomes containing 20% PS or 5% PI(4,5)P2 in the presence or absence of the FCHO2 BAR domain. C, Nedd4L recruitment. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by IB with anti-Nedd4L antibody (upper panel) and CBB staining (lower panel). D, Nedd4L activation. Samples were analyzed by IB. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.\n",
      "\n",
      "Figure 12. Effects of liposome size on FCHO2-induced activation of Nedd4L. Co-sedimentation (A) and ubiquitination (B) assays were performed at 0.7 μM Ca2+ with the FCHO2 BAR domain using brain-lipid liposomes (20% PS, 0.8 µm or 0.05 µm pore-size) that were associated with mSA-αENaC.\n",
      "\n",
      "A. Binding of the FCHO2 BAR domain to 0.8 µm and 0.05 µm pore-size liposomes. The co-sedimentation assay was performed using synthetic (0% PS) or brain-lipid liposomes (20% PS) containing rhodamine-PE and fluorescein-PE. Liposomes were precipitated with anti-fluorescein antibody. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining. The proportion of precipitated liposomes of each size was calculated to be ~100% by measuring both total and supernatant fluorescence.\n",
      "\n",
      "B. Effects of liposome size on FCHO2-induced activation of Nedd4L. Samples were analyzed by IB. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.\n",
      "\n",
      "Figure 13. Model of ENaC endocytosis.\n",
      "\n",
      "FCHO2 binds to the plasma membrane and generates a specific degree of membrane curvature according to the intrinsic curvature of its BAR domain. Nedd4L exists in a catalytically autoinhibited state due to an intramolecular interaction between the N-terminal C2 and C-terminal HECT domains. Nedd4L is recruited to FCHO2-generated membrane curvature at the rim of nascent CCPs that ENaC enters. Membrane binding of the C2 domain causes catalytic activation by relieving autoinhibition. ENaC is ubiquitinated and captured by adaptor proteins with Ub-interacting motifs, such as Eps15.\n",
      "\n",
      "Extended View Figure Legends\n",
      "\n",
      "Figure EV1. Expression of ENaC subunits in αβγENaC-HeLa cells.\n",
      "\n",
      "A, B. Doxycycline (Dox)-induced expression of αENaC. αβγENaC-HeLa cells were cultured overnight in the presence or absence of Dox. The lysates of parental HeLa and αβγENaC-HeLa cells were subjected to IB. A, 7.5% gel. B, Gradient gel (5-20%). Anti-γENaC antibody cross-reacted with two bands of endogenous proteins in parental HeLa cells, the lower band of which overlapped with γENaC (arrow) in αβγENaC-HeLa cells. Upon Dox-induced αENaC expression, an additional 70-kDa γENaC band (asterisk) was detected. It has been shown that co-expression of all three subunits induces ENaC maturation, including proteolytic cleavage of α- and γENaC (Hughey et al, 2003). The 70-kDa γENaC band is likely a cleavage product comprising the C-terminal region. The 20-kDa αENaC band (arrowhead) is likely a cleavage product comprising the N-terminal region.\n",
      "\n",
      "C. Association of α-, β-, and γENaC. When αENaC was immunoprecipitated with anti-FLAG antibody from αβγENaC-HeLa cells treated with Dox, β- and γENaC were co-precipitated. Asterisk, 70-kDa γENaC band.\n",
      "\n",
      "Figure EV2. Knockdown of FBP17 by siRNA.\n",
      "\n",
      "αβγENaC-HeLa cells were treated with each siRNA, and FBP17 mRNA levels were quantified using real-time PCR. The expression of FBP17 was normalized to GAPDH mRNA levels. Data are shown as the mean ± S.E.M. of three independent experiments. ***P < 0.001 (Student's t-test).\n",
      "\n",
      "Figure EV3. Inability of FCHO2 mutants to generate membrane tubules.\n",
      "\n",
      "GFP-FCHO2 BAR domain [wild type (WT) or mutant] was expressed in COS7 cells. Cells were then subjected to immunofluorescence microscopy. Scale bar, 10 μm.\n",
      "\n",
      "Figure EV4. Inability of Nedd4L mutants to ubiquitinate αENaC. An in vivo ubiquitination assay was performed with various Nedd4L constructs (each 0.5 µg) using FLAG-αENaC as a substrate in the presence or absence of GFP-FCHO2 BAR domain. Cell lysates were subjected to IP. Samples were analyzed by IB.\n",
      "\n",
      "Figure EV5. Inhibition of the liposome binding of Nedd4L by the FCHO2 BAR domain.\n",
      "\n",
      "A co-sedimentation assay was performed at 0.7 μM Ca2+ with control liposomes (0% PS) or brain-lipid liposomes (~50% PS) in the presence or absence of the FCHO2 BAR domain. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by IB (upper panel) and CBB staining (lower panel).\n",
      "\n",
      "Figure EV6. Membrane binding and curvature generation of BAR domains and an FCHO2 mutant. Co-sedimentation (A) and in vitro tubulation (B) assays were performed at 0.7 μM Ca2+ with the indicated BAR domains using brain-lipid liposomes (20% PS) that were associated with mSA-αENaC.\n",
      "\n",
      "A. Membrane binding. The supernatants (S) and pellets (P) were subjected to SDS-PAGE followed by CBB staining.\n",
      "\n",
      "B. Curvature generation. Left panel, electron microscopic image. Scale bar, 100 nm. Right panel, distribution of tubule diameters shown by boxplots (number of observations per protein = 22–26). The center line inside the box corresponds to the median, the bounds of the box encompass data points between the first and third quartiles, and the whiskers extend to the minimum and maximum values including outliers.\n"
     ]
    }
   ],
   "source": [
    "from docling.document_converter import DocumentConverter\n",
    "\n",
    "converter = DocumentConverter()\n",
    "docling_docx = converter.convert(DOCX_FILES[1])\n",
    "print(docling_docx.document.export_to_markdown())  # output: \"## Docling Technical Report[...]\"\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<!DOCTYPE html>\n",
      "<html lang=\"en\">\n",
      "<head>\n",
      "    <link rel=\"icon\" type=\"image/png\"\n",
      "    href=\"https://ds4sd.github.io/docling/assets/logo.png\"/>\n",
      "    <meta charset=\"UTF-8\">\n",
      "    <title>\n",
      "    Powered by Docling\n",
      "    </title>\n",
      "    <style>\n",
      "    html {\n",
      "    background-color: LightGray;\n",
      "    }\n",
      "    body {\n",
      "    margin: 0 auto;\n",
      "    width:800px;\n",
      "    padding: 30px;\n",
      "    background-color: White;\n",
      "    font-family: Arial, sans-serif;\n",
      "    box-shadow: 10px 10px 10px grey;\n",
      "    }\n",
      "    figure{\n",
      "    display: block;\n",
      "    width: 100%;\n",
      "    margin: 0px;\n",
      "    margin-top: 10px;\n",
      "    margin-bottom: 10px;\n",
      "    }\n",
      "    img {\n",
      "    display: block;\n",
      "    margin: auto;\n",
      "    margin-top: 10px;\n",
      "    margin-bottom: 10px;\n",
      "    max-width: 640px;\n",
      "    max-height: 640px;\n",
      "    }\n",
      "    table {\n",
      "    min-width:500px;\n",
      "    background-color: White;\n",
      "    border-collapse: collapse;\n",
      "    cell-padding: 5px;\n",
      "    margin: auto;\n",
      "    margin-top: 10px;\n",
      "    margin-bottom: 10px;\n",
      "    }\n",
      "    th, td {\n",
      "    border: 1px solid black;\n",
      "    padding: 8px;\n",
      "    }\n",
      "    th {\n",
      "    font-weight: bold;\n",
      "    }\n",
      "    table tr:nth-child(even) td{\n",
      "    background-color: LightGray;\n",
      "    }\n",
      "    </style>\n",
      "    </head>\n",
      "<p>The Nedd4L ubiquitin ligase is activated by FCHO2-generated membrane curvature</p>\n",
      "<p></p>\n",
      "<p>Yasuhisa Sakamoto1, Akiyoshi Uezu1, Koji Kikuchi1, Jangmi Kang2, Eiko Fujii2, Toshiro Moroishi3, Shiro Suetsugu4, and Hiroyuki Nakanishi1,2*</p>\n",
      "<p></p>\n",
      "<p>1Department of Molecular Pharmacology, Faculty of Life Sciences, Kumamoto University, 1–1–1 Honjyo, Kumamoto 860-8556, Japan</p>\n",
      "<p>2Faculty of Clinical Nutrition and Dietetics, Konan Women’s University, 6–2–23 Morikita-machi, Kobe 658-0001, Japan</p>\n",
      "<p>3Department of Molecular and Medical Pharmacology, Faculty of Life Sciences, Kumamoto University, 1–1–1 Honjyo, Kumamoto 860-8556, Japan</p>\n",
      "<p>4Division of Biological Science, Graduate School of Science and Technology, Nara Institute of Science and Technology, 8916-5 Takayama-cho, Ikoma 630-0192, Japan</p>\n",
      "<p>*Corresponding author. Tel: +81-96-373-5074; Fax: +81-96-373-5078; E-mail: hnakanis@gpo.kumamoto-u.ac.jp</p>\n",
      "<p></p>\n",
      "<p></p>\n",
      "<p></p>\n",
      "<p>Abstract</p>\n",
      "<p></p>\n",
      "<p>The C2-WW-HECT domain ubiquitin ligase Nedd4L regulates membrane sorting during endocytosis through the ubiquitination of cargo molecules such as the epithelial sodium channel (ENaC). Nedd4L is catalytically autoinhibited by an intramolecular interaction between its C2 and HECT domains, but the protein’s activation mechanism is poorly understood. Here, we show that Nedd4L activation is linked to membrane shape by FCHO2, a Bin-Amphiphysin-Rsv (BAR) domain protein that regulates endocytosis. FCHO2 was required for the Nedd4L-mediated ubiquitination and endocytosis of ENaC, with Nedd4L co-localizing with FCHO2 at clathrin-coated pits. In cells, Nedd4L was specifically recruited to, and activated by, the FCHO2 BAR domain. Furthermore, we reconstituted FCHO2-induced recruitment and activation of Nedd4L in vitro. Both the recruitment and activation were mediated by membrane curvature rather than protein-protein interactions. The Nedd4L C2 domain recognized a specific degree of membrane curvature that was generated by the FCHO2 BAR domain, with this curvature directly activating Nedd4L by relieving its autoinhibition. Thus, we show for the first time a specific function (i.e., recruitment and activation of an enzyme regulating cargo sorting) of membrane curvature by a BAR domain protein.</p>\n",
      "<p></p>\n",
      "<p>Keywords: Nedd4L; FCHO2; clathrin; endocytosis; membrane curvature.</p>\n",
      "<p></p>\n",
      "<p>Introduction</p>\n",
      "<p></p>\n",
      "<p>Posttranslational modification of proteins by covalent attachment of ubiquitin (Ub) is catalyzed by three enzymes: a Ub-activating enzyme (E1), a Ub-conjugating enzyme (E2), and a Ub-protein ligase (E3) that determines substrate specificity (Hershko & Ciechanover, 1998). E3 Ub ligases are classified into two categories based on their Ub transfer mechanisms: RING finger/U-box E3 and HECT-type E3. The Nedd4 family belongs to HECT-type E3 Ub ligases and consists of nine members, including Nedd4/Nedd4-1, Nedd4L/Nedd4-2, Itch, Smurf1, Smurf2, WWP1, WWP2, NEDL1, and NEDL2 (Ingham et al, 2004; Rotin & Kumar, 2009). They are characterized by a common modular organization, with an N-terminal C2 domain, two to four WW domains, and a C-terminal catalytic HECT domain. While the C2 domain was originally identified in protein kinase C (PKC) as a Ca2+-dependent phosphatidylserine (PS)-binding domain (Nishizuka, 1992), it shows significant diversity in the binding partners, including intracellular proteins and other phospholipids, such as phosphatidylinositol (4,5)-bisphosphate [PI(4,5)P2] (Lemmon, 2008). The WW domains recognize proline-rich motifs, such as PPxY (PY motif, where x is any residue), of substrates or adaptor proteins. The HECT domain catalyzes the isopeptide bond formation between the Ub C terminus and the substrate lysine residues. At least some of the Nedd4 family members, including Nedd4L, are catalytically autoinhibited by an intramolecular interaction between the C2 and HECT domains (Wang et al, 2010; Wiesner et al, 2007; Zhu et al, 2017). However, the activation mechanism of the Nedd4 family is poorly understood.</p>\n",
      "<p>The Nedd4 family is implicated in a wide variety of cellular processes by regulating membrane trafficking and degradation of components of various signaling pathways (Ingham et al, 2004; Rotin & Kumar, 2009). Among a diversity of substrates, the best-characterized is the epithelial sodium channel (ENaC). This channel is composed of three subunits, α-, β-, and γENaC, each of which possesses two transmembrane segments and intracellular N- and C-termini with a large extracellular loop (Butterworth, 2010; Rotin & Staub, 2011; Rossier, 2014). Each subunit has a PY motif at the cytosolic C-terminus. Nedd4L Ub ligase specifically binds to the PY motif and mediates the ubiquitination of lysine residues on the N-terminus of at least α- and γENaC at the plasma membrane. Ub serves as an internalization signal for recognition by adaptor proteins, such as epsin and Eps15, for clathrin-mediated endocytosis (CME) (Traub, 2009). Consequently, ENaC is constitutively internalized through CME. Heterozygous mutations of the PY motif of β- or γENaC lead to impaired ubiquitination and endocytosis, resulting in persistence of the channels at the cell surface and increased Na+ absorption in Liddle syndrome, an autosomal dominant form of severe hypertension.</p>\n",
      "<p>The process of CME begins at the cytoplasmic surface of the plasma membrane with the assembly of clathrin with adaptor proteins and transmembrane cargo molecules (Doherty & McMahon, 2009; Traub, 2009). In parallel with this assembly, membrane curvature is generated to form hemispherical clathrin-coated pits (CCPs). This process leads to deep membrane invaginations. Subsequently, the neck of invaginated pits is constricted and severed to separate the newly formed clathrin-coated vesicles (CCVs) from the plasma membrane. These membrane curvatures are considered to be generated, sensed, and/or maintained by Bin-Amphiphysin-Rsv (BAR) domain superfamily proteins. BAR domains form a crescent-shaped dimer characterized by a specific degree of intrinsic curvature. They bind to negatively charged phospholipids, such as PS, and force membranes to bend according to their intrinsic curvatures, thereby inducing the formation of membrane tubules with specific diameters (curvatures). For example, the diameters of membrane tubules generated by the amphiphysin BAR domain, FCHO2 BAR domain, and FBP17 BAR domain are 20–50 nm (sharper curvature) (Peter et al, 2004; Takei et al, 1999; Yin et al, 2009), 50–80 nm (intermediate curvature) (Henne et al, 2007), and ~150 nm (Takano et al, 2008) (shallower curvature), respectively.</p>\n",
      "<p>The FCHO2 BAR domain has been proposed to trigger the initial invagination for CCP formation (Henne et al, 2010). FCHO2 binds to the flat surface of the plasma membrane at sites where CCPs will form. FCHO2 then generates the initial membrane curvature and recruits the clathrin machinery for CCP formation. Here, we report the role of FCHO2 in Nedd4L-mediated ENaC endocytosis and show that FCHO2 BAR domain-generated membrane curvature induces the recruitment and activation of Nedd4L.</p>\n",
      "<p></p>\n",
      "<p>Results</p>\n",
      "<p></p>\n",
      "<p>FCHO2 is required for Nedd4L-mediated ubiquitination and endocytosis of ENaC</p>\n",
      "<p>To study the role of FCHO2 in ENaC endocytosis, we established a HeLa cell line stably expressing all three subunits, α-, β-, and γENaC (αβγENaC-HeLa cells) (Fig EV1A-C). A FLAG tag was introduced into the extracellular region of αENaC in a position previously shown not to affect channel activity (Firsov et al, 1996). Upon FCHO2 knockdown by small interfering RNA (siRNA) methods, ENaC internalization was reduced, as was upon Nedd4L knockdown (Zhou et al, 2007) (Fig 1 A and B). Transferrin receptor (TfR) internalization was reduced upon FCHO2 knockdown but not upon Nedd4L knockdown (Fig 1C). The phenotype of FCHO2 knockdown in ENaC internalization was rescued by an siRNA-resistant (sr) form of FCHO2 (Fig 1D and Appendix Fig S1). Furthermore, αENaC ubiquitination was reduced upon FCHO2 or Nedd4L knockdown but not upon FBP17 knockdown (Fig 1E and Fig EV2). FCHO2 or Nedd4L knockdown increased αENaC expression at the cell surface (Fig 1E and F). These results suggest that FCHO2 is required for Nedd4L-mediated ubiquitination and endocytosis of ENaC.</p>\n",
      "<p></p>\n",
      "<p>Recruitment and activation of Nedd4L by FCHO2 in cells</p>\n",
      "<p>Immunofluorescence microscopy revealed that in αβγENaC-HeLa cells, cell-surface αENaC co-localized with FCHO2 at clathrin-coated structures (Fig 2A). However, its co-localization with Nedd4L was not detected (Appendix Fig S2A), because exogenous expression of Nedd4L-green fluorescent protein (GFP) stimulated ENaC internalization, resulting in the disappearance of αENaC from the cell surface. Cell-surface αENaC was observed in cells lacking Nedd4L-GFP expression (non-transfected cells). When a catalytically inactive Nedd4L mutant, C922A, was expressed, it co-localized with cell-surface αENaC and endogenous FCHO2 (Fig 2B). The mutant also formed FCHO2-negative spots. In wild-type HeLa cells, Nedd4L accumulated at clathrin-coated structures where FCHO2 localized (Fig 2C). Additionally, Nedd4L formed clathrin-negative spots, suggesting that they may be involved in clathrin-independent endocytosis.</p>\n",
      "<p>Using total internal reflection fluorescence (TIRF) microscopy, we investigated whether Nedd4L is recruited to CCPs in wild-type HeLa cells. Similar to FCHO2 described previously (Henne et al, 2010; Lehmann et al, 2019; Zaccai et al, 2022), Nedd4L showed dynamic co-localization with clathrin during pit formation (Fig 3). When both fluorescent protein-tagged FCHO2 and Nedd4L were expressed, the accumulation of Nedd4L at FCHO2-positive spots was not clearly observed (Appendix Fig S2B). The reason behind this remains unknown, but it has been shown that clathrin coat formation, measured in time-lapse images of cells expressing fluorescent proteins, is affected by several factors, such as overexpression level, choice of the proteins, and choice of the fluorescent tag (Aguet et al, 2013). These factors may contribute to the failure to detect the accumulation of Nedd4L at FCHO2-positive spots.</p>\n",
      "<p>While Nedd4L shows a punctate staining pattern at the cell surface (see Figs 2C, 3B and 3C), it exhibited diffuse distribution throughout the cytoplasm (Fig 4A and B). In COS7 cells, co-expression of the FCHO2 BAR domain [amino acid (aa) 1-302] recruited Nedd4L to membrane tubules (Fig 4A). The Nedd4L C2 domain (Myc-Nedd4L C2, aa 1-160) was also concentrated to membrane tubules, whereas C2 domain-deleted Nedd4L (Myc-Nedd4L ΔC2) was not. In contrast to FCHO2, the FBP17 BAR domain did not recruit Nedd4L to membrane tubules. Additionally, the BAR domain protein amphiphysin1 did not recruit Nedd4L to membrane tubules. Thus, Nedd4L is selectively recruited through the C2 domain to membrane tubules generated by the FCHO2 BAR domain. Similar results were obtained in HEK293 cells (Fig 4B). In these cells, membrane tubules generated by amphiphysin1 were hardly detected. It showed a punctate staining pattern and did not co-localized with Nedd4L.</p>\n",
      "<p>We also investigated whether the FCHO2 BAR domain not only recruits but also activates Nedd4L in cells. For this purpose, we performed an in vivo ubiquitination assay using HEK293 cells, which formed membrane tubules upon FCHO2 expression more efficiently than did COS7 cells: 91 ± 2% of HEK293 cells vs. 32 ± 2% of COS7 cells [the mean ± standard error of the mean (S.E.M.) of four independent experiments]. As the catalytic activity of E3 Ub ligases is typically reflected in their autoubiquitination (Wiesner et al, 2007), we determined activity by measuring the ubiquitination level of Nedd4L itself. Expression of the FCHO2 BAR domain in cells remarkably enhanced Nedd4L autoubiquitination, whereas FBP17 or amphiphysin1 did not (Fig 5A). Tubulation-deficient FCHO2 mutants (K146E+R152E and L136E) (Henne et al, 2010) did not enhance Nedd4L activity (Fig 5A–C and Fig EV3). A catalytically inactive Nedd4L mutant, C922A, which was recruited to FCHO2-generated membrane tubules, showed little enhancement of ubiquitination upon FCHO2 expression (Fig 5D and E), indicating that Nedd4L ubiquitination is self-induced. Using αENaC as a substrate, we further confirmed that Nedd4L is activated upon FCHO2 expression in cells (Fig 5F).</p>\n",
      "<p></p>\n",
      "<p>Preference of Nedd4L for FCHO2-generated membrane curvature</p>\n",
      "<p>The aforementioned results prompted us to investigate whether FCHO2 and Nedd4L interact through membrane tubules rather than through stereotyped protein–protein interactions. Membrane tubules generated by BAR domains in cells have specific diameters (curvatures) (Henne et al, 2007; Peter et al, 2004; Takano et al, 2008; Takei et al, 1999; Yin et al, 2009). Nedd4L may preferentially bind to the specific degree of membrane curvature generated by FCHO2. To investigate this possibility, we first examined the biochemical properties of Nedd4L. Using a co-sedimentation assay with liposomes containing various percentages of PS, we found that the C2 domain of Nedd4L is a Ca2+-dependent PS-binding domain (Fig 6A and B). Submicromolar Ca2+ concentrations, which are comparable to intracellular levels in unstimulated cells, were sufficient for the Nedd4L C2 domain to bind to PS-containing liposomes. The C2 domain alone and full-length Nedd4L showed different Ca2+ requirements. The higher Ca2+ requirement of full-length Nedd4L may be due to the intramolecular interaction between the C2 and HECT domains. Increasing PS percentages in synthetic liposomes enhanced the liposome binding of the Nedd4L C2 domain, and PS requirements varied depending on Ca2+ concentration.</p>\n",
      "<p>We next examined whether Nedd4L preferentially binds to a specific degree of membrane curvature. For this purpose, liposomes of different sizes (curvatures) were prepared by stepwise extrusion through filter membranes with different pore sizes (0.8, 0.4, 0.2, 0.1, 0.05, and 0.03 µm) and sonication. Sonication or extrusion through a 0.03 µm pore-size filter was used to prepare the smallest liposomes. Transmission electron microscopy revealed that the actual liposome size was similar to the pore size for 0.2, 0.1, 0.05, and 0.03 µm pore-size liposomes, whereas it was significantly smaller for 0.8 and 0.4 µm pore-size liposomes (Fig 7A). Nedd4L showed the strongest binding to 0.05 µm pore-size liposomes (Fig 7B).</p>\n",
      "<p>Synaptotagmin I (SytI) tandem C2 domains can both generate and sense membrane curvature in vitro by penetrating their hydrophobic residues into the lipid bilayer (Martens et al, 2007). As a sensor, the binding affinity is considered to be highest for membranes that have a degree of curvature consistent with that generated by SytI itself. We investigated whether, in addition to sensing membrane curvature, the Nedd4L C2 domain can also generate membrane curvature in vitro and, if so, what diameter of membrane tubules is generated. As expected, transmission electron microscopy revealed that the C2 domain deformed liposomes into extensive tubulation (Fig 7C). It should be noted that Nedd4L does not generate membrane tubules in cells (see Fig 4A and B). The tubule diameter generated by Nedd4L in vitro was mainly 40–80 nm, which is consistent not only with the diameter of 0.05 µm pore-size liposomes, to which Nedd4L preferentially binds, but also with the diameter of membrane tubules generated by FCHO2 (Fig 7D). These results suggest that Nedd4L shows the highest binding affinity for the degree of membrane curvature generated by FCHO2. It is therefore likely that FCHO2-induced recruitment and activation of Nedd4L are mediated by the degree of membrane curvature.</p>\n",
      "<p></p>\n",
      "<p>Roles of conserved hydrophobic residues in the Nedd4L C2 domain</p>\n",
      "<p>We compared the C2 domain sequence of Nedd4L to those of other Nedd4 family members, SytI, and PKCγ (Fig 8A). SytI C2 domains possess hydrophobic residues (M173, F234, V304, and I367) for penetration into the lipid bilayer to generate and sense membrane curvature (Martens et al, 2007). The Nedd4L C2 domain also has three hydrophobic residues (I37, F38, and L99), which are conserved among Nedd4 family members except for WWP2. We mutated these residues to alanine to produce six mutants, I37A, F38A, L99A, I37A+F38A, I37A+L99A, and I37A+F38A+L99A (triple-A). Any single mutation did not inhibit Nedd4L recruitment to FCHO2-generated membrane tubules (Fig 8B). However, double and triple mutations severely impaired this recruitment. Furthermore, the I37A+F38A and triple-A mutants showed little liposome binding and lost curvature generation (Fig 8C and D). Thus, these hydrophobic residues (I37, F38, and L99) are critical for both membrane binding and curvature sensing and generation. These residues are likely inserted into the lipid bilayer, thereby strengthening the C2 domain interaction with membranes and enabling the sensing and generation of membrane curvature. It has been shown that I37 and F38 are directly involved in Ca2+ binding (Escobedo et al, 2014). The residues may be inserted into the lipid bilayer while binding Ca2+.</p>\n",
      "<p>In the C2 domain of Smurf2, another Nedd4 family member, conserved hydrophobic residues (F29 and F30) are responsible for interactions with the HECT domain (Fig 8A) (Wiesner et al, 2007). Mutation at these residues impairs the interaction between the C2 and HECT domains, resulting in enhanced Smurf2 autoubiquitination by relieving autoinhibition. Consistent with these previous findings, we found that the Nedd4L I37A+F38A mutant, which was not recruited to FCHO2-generated membrane tubules, enhanced autoubiquitination, irrespective of FCHO2 expression (Fig 5D and E). These results suggest that I37 and F38 are critical for interactions with the HECT domain as well as for membrane binding and curvature sensing. Interestingly, the Nedd4L I37A+F38A mutant, as well as the C922A mutant, showed little αENaC ubiquitination (Fig EV4), indicating that membrane binding of Nedd4L is critical for ENaC ubiquitination.</p>\n",
      "<p></p>\n",
      "<p>In vitro activation of Nedd4L by FCHO2-generated membrane curvature</p>\n",
      "<p>To examine the properties of Nedd4L catalytic activity, we carried out an in vitro ubiquitination assay using an αENaC peptide as a substrate. The αENaC peptide possesses a conserved PY motif for interaction with the Nedd4L WW domain, and it is fused with monomeric streptavidin (mSA) (Lim et al, 2013) to bind to liposomes containing biotinylated phospholipids (biotinylated liposomes) (Sakamoto et al, 2017) (Fig 9A and B). We found that Ca2+ and PS were required for Nedd4L catalytic activity (Fig 9C and D), and that 0.05 µm pore-size liposomes were most effective in stimulating Ca2+- and PS-dependent Nedd4L activity (Fig 9E). Thus, the properties of Nedd4L catalytic activity are similar to those of its liposome binding. Furthermore, we found that Nedd4L did not ubiquitinate PY motif-mutated (Y644A in αENaC) mSA-αENaC (Appendix Fig S3). These findings are consistent with defects in ENaC ubiquitination observed in Liddle syndrome.</p>\n",
      "<p>We next examined whether membrane localization of mSA-αENaC affects the membrane binding and catalytic activity of Nedd4L. Nedd4L bound only slightly to mSA-αENaC on PS-free (0% PS) liposomes, but the liposome binding was synergistically increased by elevating the PS percentage in liposomes (Fig 9F). By contrast, when free biotin competitively inhibited the binding of mSA-αENaC to biotinylated liposomes, the PS-dependent liposome binding of Nedd4L was significantly reduced, resulting in remarkable inhibition of mSA-αENaC ubiquitination (Fig 9D and F). Thus, membrane localization of the PY motif is critical for membrane binding and activation of Nedd4L. It is, therefore, likely that the PY motifs on membranes, such as ENaC, play a role in enhancing Nedd4L catalytic activity by potentiating membrane binding through interactions with the WW domain.</p>\n",
      "<p>We then reconstituted in vitro FCHO2-induced recruitment and activation of Nedd4L. When liposomes (20% PS) were used, the FCHO2 BAR domain enhanced the liposome binding of Nedd4L (Fig 10A) and mSA-αENaC ubiquitination (Fig 10B). This enhancement was lost when the strength of interaction of Nedd4L with liposomes was increased by elevating the PS percentage in liposomes (~50% PS) (Fig 10C and Fig EV5).　In contrast to FCHO2, neither amphiphysin1 nor FBP17 enhanced Nedd4L activity (Fig 10D and Fig EV6A). Similarly, tubulation-deficient FCHO2 mutant (L136E) did not enhance Nedd4L activity (Fig 10E and Fig EV6A). These results are in good agreement with those obtained from in vivo experiments on FCHO2-induced recruitment and activation of Nedd4L. Electron microscopic analysis revealed that mSA-αENaC-associated liposomes were deformed by membrane-sculpting proteins into tubules with diameters similar to those of mSA-αENaC-free liposomes (Fig EV6B and see also Fig 7D). The diameter of membrane tubules generated by FCHO2 is consistent with that of 0.05 µm pore-size liposomes which are most effective in stimulating Nedd4L activity.</p>\n",
      "<p>To confirm whether FCHO2-induced recruitment and activation of Nedd4L are mediated by liposomes (i.e., membrane curvature) rather than by protein–protein interactions, we conducted two sets of experiments. Initially, we employed synthetic liposomes where 20% PS was substituted with 5% PI(4,5)P2. The FCHO2 BAR domain bound to PI(4,5)P2 as well as PS with high affinity, whereas the Nedd4L C2 domain did not (Fig 11A and B). The FCHO2 BAR domain is known to induce the tubulation of liposomes [5% PI(4,5)P2] (Henne et al, 2010). If Nedd4L could interact through protein–protein interactions with the FCHO2 BAR domain binding to liposomes [5% PI(4,5)P2], the BAR domain would facilitate the co-precipitation of Nedd4L in a co-sedimentation assay, thereby stimulating mSA-αENaC ubiquitination. However, the FCHO2 BAR domain did not demonstrate such interactions (Fig 11C and D). Therefore, it is unlikely that Nedd4L interacts with the FCHO2 BAR domain on liposomes through protein–protein interactions. Secondly, we employed 0.05 µm pore-size liposomes that are most effective in stimulating Nedd4L activity. The diameter of 0.05 µm pore-size liposomes is consistent with that of membrane tubules generated by the FCHO2 BAR domain. The BAR domain is, thus, expected not to induce the tubulation of 0.05 µm pore-size liposomes. The FCHO2 BAR domain showed similar binding activity toward 0.8 and 0.05 µm pore-size liposomes (Fig 12A). When 0.8 µm pore-size liposomes were used, the FCHO2 BAR domain enhanced Nedd4L activity (Fig 12B). However, in the case of 0.05 µm pore-size liposomes, curvature-stimulated Nedd4L activity was not potentiated by the addition of the FCHO2 BAR domain. If Nedd4L had interacted with the BAR domain, we would have expected potentiation of Nedd4L activity. These results strongly suggest that FCHO2-induced activation of Nedd4L is mediated by membrane curvature and not by protein–protein interactions.</p>\n",
      "<p></p>\n",
      "<p>Discussion</p>\n",
      "<p></p>\n",
      "<p>Nedd4L represents the ancestral Ub ligase with strong similarity to yeast Rsp5 which plays a key role in the trafficking, sorting, and degradation of a large number of proteins (Yang & Kumar, 2010). Consistently, Nedd4L regulates a growing number of substrates, including membrane receptors, transporters, and ion channels. Although Nedd4L exists in a catalytically autoinhibited state due to an intramolecular interaction between the N-terminal C2 and C-terminal HECT domains, little is known about its activation mechanism. In this study, we clearly demonstrated that Nedd4L is activated by FCHO2 F-BAR domain-induced membrane curvature. It has been shown that the activity of synptojanin-1, a phosphoinositide phosphatase, is modulated by membrane curvature (Chang-Ileto et al, 2011). Taken together, it is conceivable that membrane curvature serves as a signal for the activation of various enzymes involved in membrane trafficking.</p>\n",
      "<p></p>\n",
      "<p>Unique Properties of the Nedd4L C2 Domain</p>\n",
      "<p>Although both sytI and Nedd4L C2 domains are Ca2+-dependent PS-binding domains that can generate and sense membrane curvature, their properties show a number of significant differences. An interesting difference exists in the degrees of membrane curvatures generated by sytI and Nedd4L: the diameters of their membrane tubules are about 15–20 nm and 40–80 nm, respectively (Hui et al, 2009; Martens et al, 2007). This difference is not simply due to that between tandem and single C2 domains, because it has been shown that the sytI C2B domain alone generates membrane tubules with a similar diameter (Hui et al, 2009). SytI and Nedd4L C2 domains must therefore have respective unique structures that determine the specific degrees of membrane curvatures that they sense and generate.</p>\n",
      "<p>Another important difference between sytI and Nedd4L C2 domains exists in their Ca2+ requirements. SytI C2 domains require Ca2+ concentrations in the submillimolar range to generate membrane curvature (Hui et al, 2009). In contrast, submicromolar Ca2+ concentrations, which are comparable to intracellular levels in unstimulated cells, are sufficient for the Nedd4L C2 domain to sense membrane curvature. This is consistent with the observation that Nedd4L is recruited and activated in serum-starved cells. The different Ca2+ requirements between sytI and Nedd4L C2 domains are reflected in their functions. SytI C2 domains induce membrane curvature in response to an increase in intracellular Ca2+ concentration, which is essential for Ca2+-triggered membrane fusion (Hui et al, 2009; Martens et al, 2007). Thus, sytI C2 domains serve as a Ca2+ sensor and a curvature generator. In contrast, the Nedd4L C2 domain recognizes a specific degree of curvature without an increase in intracellular Ca2+ concentration. Therefore, the Nedd4L C2 domain serves as a curvature sensor to regulate the localization and activity of the E3 Ub ligase.</p>\n",
      "<p>A striking feature that distinguishes the Nedd4L C2 domain from other C2 domain proteins (except for at least a subset of Nedd4 family members) is that the Nedd4L C2 domain interacts not only with membranes but also with the intramolecular catalytic domain for its autoinhibition. Mutational experiments have revealed that I37 and F38 residues in the C2 domain are critical for interactions with both membranes and the HECT domain, suggesting that these two interactions are competitive. This idea is supported by the observation that the Ca2+ EC50 value of full-length Nedd4L is much higher than that of the C2 domain. The HECT domain in full-length Nedd4L may competitively inhibit the interaction of the C2 domain with membranes. It is likely that membranes displace the HECT domain from the C2 domain when the strength of the C2 domain interaction with membranes is increased by the preferred membrane curvature. This displacement relieves the autoinhibition, resulting in Nedd4L activation. Thus, the membrane binding of Nedd4L is coupled with its activation.</p>\n",
      "<p></p>\n",
      "<p>Stimulatory and inhibitory effects of the FCHO2 BAR domain on the liposome binding and catalytic activity of Nedd4L</p>\n",
      "<p>The FCHO2 BAR domain enhances both the liposome binding and catalytic activity of Nedd4L when the interaction strength of Nedd4L with liposomes is weak (20% PS) (Fig 10A and B). Conversely, the BAR domain inhibits the binding and activity of Nedd4L when the interaction of Nedd4L with liposomes is increased by elevating the PS percentage in liposomes (~50% PS) (Fig 10C and Fig EV5). Thus, the FCHO2 BAR domain exerts both stimulatory and inhibitory effects on Nedd4L, depending on the PS percentage in liposomes. However, given that 20% PS is comparable to the concentration in the inner leaflet of the plasma membrane, it is plausible that FCHO2 exhibits only a stimulatory effect on Nedd4L in cells. The mechanism underlying the stimulatory and inhibitory effects in vitro is as follows: when using liposomes (20% PS), Nedd4L hardly binds to liposomes through PS (Fig. 9F and 10A). The addition of the FCHO2 BAR domain enhances the strength of Nedd4L’s interaction with PS by inducing membrane curvature. Consequently, Nedd4L gains a new binding to liposomes through PS, which augments its catalytic activity. In contrast, when using liposomes (~50% PS), Nedd4L binds to liposome through PS. The addition of the FCHO2 BAR domain hinders the PS-mediated liposome binding of Nedd4L, because FCHO2 and Nedd4L are PS-binding proteins and compete for PS binding in liposomes. Consequently, the BAR domain inhibits the catalytic activity of Nedd4L. This is consistent with the results obtained with 0.05 µm pore-size liposomes (Fig 12B). These 0.05 µm pore-size liposomes are comparable to liposomes with a higher PS percentage, as the interaction of Nedd4L with liposomes through PS is increased by membrane curvature. Therefore, when using 0.05 µm pore-size liposomes, the FCHO2 BAR domain inhibits the catalytic activity of Nedd4L.</p>\n",
      "<p>Based on a model in which a subset of BAR domain proteins, such as FCHO2 and FBP17, drive liposome tubulation in vitro (Frost et al, 2008; Shimada et al, 2007), the FCHO2 BAR domain is considered to polymerize on the liposome surface into spiral dense coats by lateral and tip-to-tip interactions. When using liposomes (~50% PS), it is conceivable that the FCHO2 BAR domain densely packs the liposomes, leaving little free space for Nedd4L binding. Conversely, when using liposomes (20% PS), it is likely that the liposomes are loosely packed, providing ample surface space for Nedd4L binding.</p>\n",
      "<p></p>\n",
      "<p>Model for ENaC Endocytosis</p>\n",
      "<p>Based on the present study, we propose a model for ENaC endocytosis through ubiquitination (Fig 13). First, FCHO2 binds to the plasma membrane and generates a specific degree of membrane curvature. FCHO2 interacts with clathrin adaptor proteins, such as Eps15, and recruits the clathrin machinery for CCP formation (Uezu et al, 2011; Henne et al, 2010). Eps15 has Ub-interacting motifs (Di Fiore et al, 2003). Nedd4L is subsequently recruited to and activated by FCHO2-generated curved membrane at the rim of nascent CCPs that ENaC enters. ENaC is then ubiquitinated and captured by Eps15 which FCHO2 interacts with. Thus, ubiquitinated ENaC is spatially restricted to CCPs and separated from unmodified ENaC. Overall, our findings suggest that in the early stage of CME, a membrane deformation induced by FCHO2 can program, through both the recruitment and activation of Nedd4L, the gathering of a subclass of cargo molecules in CCPs.</p>\n",
      "<p></p>\n",
      "<p>Methods</p>\n",
      "<p></p>\n",
      "<p>Construction</p>\n",
      "<p>Human αENaC, βENaC and γENaC cDNAs were kindly supplied by Dr. H. Kai (Kumamoto University, Kumamoto, Japan) (Sugahara et al, 2009). FLAG-αENaC was constructed by replacing the sequence [amino acid (aa) 169–174, TLVAGS] in the extracellular region with the FLAG sequence (DYKDDDK) as described previously (Firsov et al, 1996). This introduction of the epitope tag was carried out by polymerase chain reaction (PCR). FLAG-αENaC, βENaC, and γENaC were subcloned into pTRE-Tight-Hygro, pCAGI-Puro (Miyahara et al, 2000), and pCAGI-Bst (Umeda et al, 2006), respectively. pTRE-Tight-Hygro was constructed by inserting the hygromycin-resistance gene into pTRE-Tight (Clontech). FLAG-αENaC was also subcloned into pCAGI-Hygro, which was constructed by replacing the puromycin-resistance gene of pCAGI-Puro with the hygromycin-resistance gene. Human Nedd4L (NM_015277, isoform 3) cDNA was obtained by reverse transcription (RT)-PCR and cloned in pCMV-Myc (Miyahara et al, 2000). Nedd4L was also cloned in pCAGI-puro-EGFP and pGEX-6P-hexahistidine (His6). pGEX-6P-His6 was constructed from pGEX-6P (GE Healthcare) to express an N-terminal GST-tagged, C-terminal His6-tagged protein. pCAGI-puro-EGFP was constructed by subcloning EGFP cDNA into pCAGI-puro to express a C-terminal GFP-tagged protein. The Nedd4L C2 domain (C2, aa 1–160) and C2 domain-deleted mutant (ΔC2, aa 150–955) were subcloned into pCMV-Myc and/or pGEX-6P. Mouse FCHO2 (full length, aa 1–809; BAR domain, aa 1–302) and human FBP17 (BAR domain, aa 1–300) were subcloned into pEGFP-C1 and/or pGEX-6P (Uezu et al, 2011; Tsujita et al, 2006). FCHO2 (aa 1–327) were subcloned into pGEX-6P. Human amphiphysin1 cDNA was obtained by RT-PCR and cloned in pEGFP-N3. Amphiphysin1 (BAR domain, aa 1–377) was then subcloned into pGEX-6P. mTagRFP-T-Clathrin-15 was a gift from Michael Davidson (Addgene plasmid # 58005 ; http://n2t.net/addgene:58005 ; RRID:Addgene_58005) (Shaner et al, 2008). Human hemagglutinin (HA)-tagged Ub cDNA in pCGN was kindly donated by Dr. M. Nakao (Kumamoto University, Kumamoto, Japan). Ub cDNA was subcloned into pET3a. Human UbcH7 cDNA was obtained by RT-PCR and cloned in pGEX-6P. pET21d-Ube1 was a gift from Cynthia Wolberger (Addgene plasmid # 34965 ; http://n2t.net/addgene:34965 ; RRID:Addgene_34965) (Berndsen & Wolberger, 2011). pRSET-mSA was a gift from Sheldon Park (Addgene plasmid # 39860 ; http://n2t.net/addgene:39860 ; RRID:Addgene_39860) (Lim et al, 2013). mSA-αENaC constructed in pRSET-mSA to express a fusion protein of the N-terminal (aa 7–40) and C-terminal (aa 625–659) intracellular regions of αENaC (see Fig. 9A). Mutagenesis was performed by PCR using mutated primers and a site-directed mutagenesis kit (Stratagene).</p>\n",
      "<p></p>\n",
      "<p>Protein expression and purification</p>\n",
      "<p>Expression and purification of GST-tagged proteins were performed using standard protocols. His6-tagged proteins were purified with Talon metal affinity resin (Clontech) using the manufacturer’s protocol. The GST tag was cleaved off by PreScission protease (GE Healthcare). The GST tag of BAR domains did not inhibit liposome binding or tubulation, whereas the GST tag of Nedd4L inhibited these functions. As FCHO2 (BAR domain, aa 1–302) was easily aggregated by the removal of the GST tag, FCHO2 (aa 1–327) was used instead as a GST-cleaved form of the FCHO2 BAR domain. Full-length Nedd4L (N-terminal GST-tagged, C-terminal His6-tagged) was purified by glutathione–Sepharose (GE Healthcare) column chromatography. The GST tag was cleaved off and the Nedd4L sample was then subjected to Talon metal affinity column chromatography. After the eluate was passed through glutathione–Sepharose beads, the sample was dialyzed against buffer A (20 mM Tris-HCl at pH 7.5, 100 mM KCl, 0.5 mM EDTA, 1 mM DTT, and 20 % sucrose). The Nedd4L sample was stored at −80°C until use. Ub was expressed in E. coli strain BL21 (DE3) pJY2 and purified as previously described (Pickart & Raasi, 2005). His6-tagged mSA-αENaC was expressed in E. coli strain BL21 (DE3) pLysS and purified as previously described (Lim et al, 2013) with slight modifications. Briefly, the inclusion body was isolated and solubilized in buffer B (50 mM Tris-HCl at pH 8.0 and 6 M guanidine hydrochloride). The sample was then subjected to Talon metal affinity column chromatography. The eluate was added drop by drop to ice-cold refolding buffer (50 mM Tris-HCl at pH 8.0, 150 mM NaCl, 0.15 mg/ml D-biotin, 0.2 mg/ml oxidized glutathione, and 1 mg/ml reduced glutathione) under rapid stirring. The sample was then concentrated using a Centriprep centrifugal filter unit (Millipore) and centrifuged to remove aggregates. The supernatant was dialyzed against buffer C (20 mM Tris-HCl at pH 8.0). The mSA-αENaC sample was further purified by Mono Q anion exchange column chromatography as follows. The sample was applied to a Mono Q 5/50 GL column (GE Healthcare) equilibrated with buffer C. Elution was performed with a 50 ml linear gradient of NaCl (0–200 mM) in buffer C. Fractions of 1 ml each were collected. The active fractions (fraction 18–22) were collected and dialyzed against buffer D (10 mM Tris-HCl at pH 7.5, 100 mM KCl, and 0.5 mM EDTA). The mSA-αENaC sample was stored at −80°C until use.</p>\n",
      "<p></p>\n",
      "<p>Antibodies</p>\n",
      "<p>Rabbit anti-Nedd4L antibody was raised against GST-Nedd4L C2 (aa 1–160). Rabbit anti-FCHO2 antibody was obtained as previously described (Uezu et al, 2011). The following antibodies were purchased from commercial sources: mouse anti-Myc (9E10) (American Type Culture Collection); mouse anti-FLAG (M2) and mouse anti-α-tubulin (clone DM1A) (Sigma-Aldrich); mouse anti-clathrin heavy chain (BD Biosciences); rabbit anti-GFP (MBL Co.); rabbit anti-αENaC (Calbiochem); rabbit anti-βENaC (Proteintech); rabbit anti-γENaC (Abcam); mouse anti-transferrin receptor H68.4 (Santa Cruz); mouse anti-Ub antibody P4D1 (Santa Cruz); and secondary antibodies conjugated with Alexa Fluor 488, 594, and 647 (Invitrogen).</p>\n",
      "<p></p>\n",
      "<p>Cell culture and transfection</p>\n",
      "<p>HeLa, HEK293, and COS7 cells were cultured in Dulbecco’s modified Eagle medium (DMEM) supplemented with 10% fetal calf serum. HEK293 cells were grown on poly-L-lysine-coated dishes. Transfection was performed by using Lipofectamine 2000 (Invitrogen) according to the manufacturer’s protocol.</p>\n",
      "<p></p>\n",
      "<p>Generation of stable HeLa cells expressing all three ENaC subunits</p>\n",
      "<p>HeLa cells were transfected with the pTet-On Advanced vector (Clontech) to develop stable Tet-On cell lines for a doxycycline (Dox)-inducible expression system according to the manufacturer’s protocol. Tet-On cells were selected in 0.5 mg/ml G418. Cloned Tet-On cells were then transfected with pCAGI-Bst-γENaC and selected in 4 µg/ml blasticidin (in the presence of G418). Blasticidin-resistant cells were transfected with pCAGI-Puro-βENaC and selected in 1 µg/ml puromycin (in the presence of G418 and blasticidin). Puromycin-resistant cells were finally transfected with pTRE-Tight-Hygro-FLAG-αENaC and cultured in 0.2 mg/ml hygromycin (in the presence of G418, blasticidin, and puromycin). After selection, cells were cloned and maintained in 0.5 mg/ml G418, 0.2 mg/ml hygromycin, 1 µg/ml puromycin, and 2.5 µg/ml blasticidin. FLAG-tagged αENaC expression was induced by treating cells with 1 µg/ml Dox in the presence of 40 µM amiloride overnight. Amiloride was added to prevent cellular Na+ overload.</p>\n",
      "<p></p>\n",
      "<p>Immunofluorescence microscopy</p>\n",
      "<p>Cells were fixed with 3.7% formaldehyde in phosphate-buffered saline (PBS) at room temperature for 15 min, and then permeabilized with 0.2% Triton X-100 in PBS for 10 min. After blocking with 1% bovine serum albumin (BSA) in PBS, cells were sequentially incubated with primary antibodies and fluorescence-conjugated secondary antibodies at room temperature for 1 h, respectively. Cells were then analyzed with either a fluorescence microscope (BX51; Olympus) or a confocal microscopy system (BX50, Fluoview FV300, and Fluoview FV1200; Olympus). A 60 × oil immersion objective (NA = 1.40, Olympus) was used. All comparable images were acquired under identical parameters. To visualize cell-surface FLAG-αENaC in αβγENaC-HeLa cells, cells were incubated with 5 µg/ml anti-FLAG M2 antibody at 4°C for 1 h to prevent internalization. Cells were then fixed, permeabilized, and stained with a secondary antibody. Cells were also sequentially incubated with primary (anti-FCHO2) and secondary antibodies. To quantify co-localization, the Pearson's correlation coefficient was calculated from three cells using the Fiji (ImageJ) Coloc2 plug-in.</p>\n",
      "<p></p>\n",
      "<p>RNA interference</p>\n",
      "<p>Stealth™ double-stranded RNAs were purchased from Invitrogen. siRNA sequences of human FCHO2, Nedd4L, and FBP17 were 5′-CCA CAG AUC UUA GAG UGG AUU AUA A-3′ (corresponding to nucleotides 2048–2072 relative to the start codon), 5′-GAA GAG UUG CUG GUC UGG CCG UAU U-3′ (corresponding to nucleotides 1778–1802 relative to the start codon), and 5′-CCC ACU UCA UAU GUC GAA GUC UGU U-3′for human FBP17 (corresponding to nucleotides 1804–1828 relative to the start codon), respectively. A double-stranded RNA targeting luciferase (GL-2, 5′-CGU ACG CGG AAU ACU UCG AAA UGU C-3′) was used as a control. HeLa cells were transfected with 20 nM siRNA using Lipofectamine 2000 according to the manufacturer’s protocol. After 24 h, a second transfection was performed, and cells were cultured for 3 days and subjected to various experiments. For rescue experiments, cells were transfected with the intended plasmid at 36 h after the second transfection. After 18 h, cells were fixed and analyzed.</p>\n",
      "<p></p>\n",
      "<p>Quantitative real-time PCR analysis</p>\n",
      "<p>Total RNA was isolated using ISOGEN (Nippon Gene, 319-90211), and cDNA was synthesized using ReverTra Ace qPCR RT Master Mix with gDNA Remover (Toyobo, FSQ-301), following manufacturer’s instruction. Real time PCR was carried out using Thunderbird Next SYBR (Toyobo, QPX-201) on StepOnePlus Real-Time PCR Systems (Applied Biosystems). The sequences of qPCR primers are 5′-TCT CCC ATC GCT TCA ACG AG-3′ and 5′-GTT GCA CCC AGC TTG AGA GA-3′ for human FBP17; 5′-GTC TCC TCT GAC TTC AAC AGC G-3′ and 5′-ACC ACC CTG TTG CTG TAG CCA A-3′ for human GAPDH. The expression of target genes was normalized to GAPDH mRNA levels.</p>\n",
      "<p></p>\n",
      "<p>Endocytosis assay</p>\n",
      "<p>αβγENaC-HeLa cells at approximately 90% confluence in a 10-cm dish were starved with serum-free medium (DMEM containing 1% BSA, 1 µg/ml Dox, and 40 µM amiloride) at 37°C for 4 h. For ENaC endocytosis, cells were then incubated with 5 µg/ml anti-FLAG M2 in 2 ml of DMEM containing 1% BSA at 4°C for 1 h to prevent internalization. After three washes with ice-cold PBS containing 1 mM MgCl2 and 0.1 mM CaCl2 (PBS-CM), endocytosis was allowed by incubating cells in prewarmed DMEM containing 1% BSA at 37°C for the indicated periods of time. Endocytosis was stopped by placing cells on ice and washing them with ice-cold PBS-CM. Antibody bound to the cell surface but not internalized was removed by acid stripping with 0.2 M acetic acid and 0.5 M NaCl solution (pH 2.5). After washing with ice-cold PBS-CM, internalized ENaC was detected by immunoblotting (IB) or immunofluorescence microscopy. For IB, cells were scraped in 1 ml of lysis buffer A (50 mM Tris-HCl at pH 8.0, 150 mM NaCl, 5 mM EDTA, 1% Triton X-100, 1 mM PMSF, 20 µg/ml leupeptin, and 1 µg/ml pepstatin A). Cells were solubilized at 4°C for 1 h on a rotating wheel, and centrifuged at 20,000 × g at 4°C for 15 min. Protein G-Sepharose beads (GE Healthcare) were added to the supernatant and incubated at 4°C for 3 h. The beads were then thoroughly washed with lysis buffer A and boiled in sodium dodecyl sulfate (SDS) sample buffer. The sample was subjected to SDS-polyacrylamide gel electrophoresis (PAGE), followed by IB with anti-ENaC subunit antibodies. Internalization was expressed as a percentage of the initial amount of cell-surface ENaC subunit determined by incubating cells with anti-FLAG antibody at 4°C for 1 h and lysing them without incubation at 37°C or acid stripping. For immunofluorescence microscopy, cells were transfected with control or FCHO2 siRNA and then with GFP-siRNA-resistant (sr) FCHO2 mutant. After cells were labeled with anti-FLAG antibody on 4°C, they were incubated at 37°C for 10 min to allow ENaC endocytosis. After acid stripping, cells were fixed with 3.7% formaldehyde and permeabilized with 0.2% Triton X-100. The antibody-labeled, internalized αENaC was visualized with a fluorescence-conjugated secondary antibody. The percentage of cells with internalized αENaC among cells expressing GFP-sr-FCHO2 was determined using MetaMorph imaging System software. All images were acquired under identical parameters. For TfR endocytosis, a cell surface biotinylation assay was carried out as previously described (Roberts et al, 2001). Briefly, the cell surface was labeled with Sulfo-NHS-SS-Biotin (Thermo Fisher Scientific) at 4°C. After cells were incubated with ice-cold quenching buffer (50 mM NH4Cl in PBS-CM) at 4°C, they were incubated with serum free medium for 10 min at 37°C. Biotin was removed from the cell surface by incubation with ice-cold MesNa buffer (50 mM MesNa, 100 mM NaCl, and 100 mM Tris-HCl, pH 8.8). Cells were then incubated with iodoacetamide at 4°C. Cell lysates were then incubated with NeutrAvidin beads (Thermo Fisher Scientific). The beads were boiled in SDS sample buffer. Proteins were analyzed by IB with TfR antibody.</p>\n",
      "<p></p>\n",
      "<p>ENaC expression and ubiquitination at the cell surface</p>\n",
      "<p>αβγENaC-HeLa cells at approximately 90% confluence in a 10-cm dish were starved with serum-free medium (DMEM containing 1% BSA, 1 µg/ml Dox, and 40 µM amiloride) at 37°C for 4 h, washed three times with ice-cold PBS, and incubated at 4°C for 45 min with 2 ml of 1 mg/ml EZ-link Sulfo-NHS-SS-Biotin in PBS with gentle shaking. Subsequently, cells were extensively washed with ice-cold Tris-buffered saline, and then scraped into 1 ml of lysis buffer B (50 mM Tris-HCl at pH 7.5, 150 mM NaCl, 5 mM EDTA, 1% Triton X-100, 10 mM N-ethylmaleimide, 1 mM PMSF, 10 µg/ml leupeptin, and 1 µg/ml pepstatin A) containing 40 µM MG132. Cells were solubilized at 4°C for 1 h on a rotating wheel, and centrifuged at 20,000 × g at 4°C for 15 min. The supernatant (4 mg of protein) was incubated at 4°C for 3 h with 10 µg of anti-FLAG M2 antibody. Protein G-Sepharose beads (50 µl of 50% slurry) were then added to the sample, which was incubated at 4°C for 3 h. After beads were thoroughly washed with lysis buffer B containing 40 µM MG132, FLAG-tagged proteins were eluted from the beads with 200 µg/ml FLAG peptide (Sigma-Aldrich) in 0.5 ml of lysis buffer B. After centrifugation, the supernatant was incubated with 25 µl of NeutrAvidin beads at 4°C overnight. After the beads were thoroughly washed with lysis buffer B, bound proteins were eluted by incubating the beads at 4°C for 1 h with 150 µl of lysis buffer B containing 200 mM DTT. After centrifugation, 5 × SDS sample buffer was added to the supernatant, which was then analyzed by IB with anti-FLAG and anti-Ub antibodies. The chemiluminescence intensity was quantified by scanning using ImageJ software (NIH).</p>\n",
      "<p></p>\n",
      "<p>TIRF Microscopy</p>\n",
      "<p>For live observation with TIRF microscopy, HeLa cells were grown on a glass-bottomed dish as described previously (Shimada et al, 2007). Cells were observed at 10–12 h after transfection as described previously (Henne et al, 2010). Images were acquired for 5 min at 1 s intervals with a TIRF microscopy system (Olympus), a 100 × oil immersion objective NA = 1.45 (Olympus) and MetaMorph Imaging System Software. All comparable images were acquired under identical parameters.</p>\n",
      "<p></p>\n",
      "<p>Liposome preparation</p>\n",
      "<p>Phospholipids were purchased from commercial sources: phosphatidylcholine (PC, 1,2-dioleoyl-sn-glycero-3-phosphocholine), phosphatidylethanolamine (PE, 1,2-dioleoyl-sn-glycero-3-phosphoethanolamine), PS (1,2-dioleoyl-sn-glycero-3-phospho-l-serine), rhodamine-PE [18:1 L-α-PE-N-(lissamine rhodamine B sulfonyl)], PEG2000-biotin-PE (1,2-distearoyl-sn-glycero-3-phosphoethanolamine-N-[biotinyl (polyethyleneglycol)-2000]), porcine brain PS, brain PI(4,5)P2, and porcine brain total lipid extract were obtained from Avanti Polar Lipids; cholesterol and Folch lipids [bovine brain lipids (Folch Fraction 1)] were obtained from Sigma-Aldrich; X-biotin-PE [N-((6-(biotinoyl)amino)hexanoyl)-1,2-dihexadecanoylsn-glycero-3-phosphoethanolamine] was obtained from Thermo Fisher Scientific; and fluorescein-PE [N-(fluorescein-5-thiocarbamoyl)-1,2-dihexadecanoyl-sn-glycero-3-phosphoethanolamine] was obtained from AAT Bioquest Inc.</p>\n",
      "<p>Liposomes were prepared using brain lipids (Folch lipids, ~50% PS of total lipids), brain lipids (20% PS of total lipids, a mixture of 89.5% porcine brain total lipid extract and 10.5% porcine brain PS), or a synthetic lipid mixture [70% PC, 20% PE, 10% cholesterol (w/w), and varying percentages of PS or PI(4,5)P2 (with a corresponding reduction in PC)]. Lipid concentrations were monitored by including 1% rhodamine-PE (w/w). To prepare mSA-αENaC–associated liposomes, 2% PEG2000-biotin-PE (w/w) was included. Lipids were dried under nitrogen gas, desiccated for 2 h, and resuspended at 1 mg/ml in lipid buffer (50 mM HEPES-NaOH at pH 7.2, 100 mM KCl, 2 mM MgCl2, and 5 mM EGTA), followed by hydration at 37°C for 1 h. To make liposomes of different sizes, they were subjected to five freeze–thaw cycles and centrifuged at 100,000 × g for 15 min. Precipitated liposomes were then resuspended in the same buffer to make a final concentration of 1 mg/ml. They were subjected to extrusion through polycarbonate filter membranes (Avanti Polar Lipids): liposomes were sequentially passed 9 times through a 0.8-µm filter, 21 times through a 0.4-µm filter, 21 times through a 0.2-µm filter, 21 times through a 0.1-µm filter, 21 times through a 0.05-µm filter, and 21 times through a 0.03-µm filter. Instead of extrusion through a 0.03-µm filter, liposomes with the smallest size were also prepared by sonication three times for 5 s with a Branson Sonifier 25 at power level 1. Liposome diameters were examined by transmission electron microscopy.</p>\n",
      "<p></p>\n",
      "<p>Co-sedimentation assay</p>\n",
      "<p>A co-sedimentation assay was performed as described (Uezu et al, 2011) with slight modifications. Briefly, 5–10 µg of protein was incubated at 25°C for 10 min with 0.5 mg/ml liposomes in 100 µl of lipid buffer containing various concentrations of CaCl2 and centrifuged at 165,000 × g at 25°C for 15 min. Equal amounts of the supernatants and pellets were subjected to SDS-PAGE, followed by Coomassie brilliant blue (CBB) staining. Proteins were quantified by scanning using ImageJ (NIH). The assay with liposomes of different sizes was performed using NeutrAvidin (Thermo Fisher Scientific Inc) as described (Sakamoto et al, 2017). Briefly, brain-lipid (Folch-lipid) liposomes (~50% PS) were supplemented with 1% X-biotin-PE and 1% rhodamine-PE, subjected to extrusion through filter membranes, and then sonicated. Purified Nedd4L protein (10 µg) was incubated with 0.2 mg/ml liposomes in 100 µl of lipid buffer containing 2mM CaCl2 (0.1 µM Ca2+). The sample was then incubated with 2 µg of NeutrAvidin and ultracentrifuged. Both total and supernatant fluorescence were measured. The proportion of precipitated liposomes of each size was ~100%.</p>\n",
      "<p>The assay with mSA-αENaC–associated liposomes was performed as follows. Liposomes were prepared from a synthetic lipid mixture [various percentages of PS or 5% PI(4,5)P2] or brain lipids supplemented with 2% PEG2000-biotin-PE and 1% rhodamine-PE. The liposomes (10 µg) were incubated with purified mSA-αENaC protein (35 pmol) at 4°C for 10 min in 25 µl of lipid buffer. The sample was then incubated at 25°C for 10 min with 0.07 µM (8 µg/ml) Nedd4L and/or 1.4 µM BAR domains in 50 µl of ubiquitination buffer (50 mM HEPES-NaOH at pH 7.2, 100 mM KCl, 2 mM MgCl2, 5 mM EGTA, and 0.1 mM DTT) containing 4 mM CaCl2 (0.7 µM Ca2+). Subsequently, the sample was subjected to ultracentrifugation. Equal amounts of the total samples and pellets were subjected to SDS-PAGE, followed by CBB staining and IB. The chemiluminescence intensity was quantified by scanning using ImageJ (NIH).</p>\n",
      "<p>The assay with mSA-αENaC-associated liposomes (0.05 µm pore-size) was done as follows. Liposomes were prepared from brain lipids (20% PS) supplemented with 2% PEG2000-biotin-PE, 1% rhodamine-PE, and 0.5% fluorescein-PE (w/w) and extruded through membrane filters. To precipitate liposomes, the sample (50 μl) was first incubated with 1 µg of anti-fluorescein goat antibody (Novus Biologicals) at room temperature for 1 h and then incubated with 0.5 µg of protein A/G/L (BioVision) at room temperature for 1 h. Subsequently, the sample was centrifuged at 165,000 × g at 25°C for 15 min. Equal amounts of the supernatants and pellets were subjected to SDS-PAGE, followed by CBB staining and IB. The percentage of precipitated liposomes was ~100%.</p>\n",
      "<p></p>\n",
      "<p>Calculation of free Ca2+ concentrations</p>\n",
      "<p>At 1.0, 2.0, 3.0, 3.5, 4.0, 4.5, and 4.9 mM CaCl2 in lipid buffer, free Ca2+ concentrations were calculated to be 0.04, 0.1, 0.3, 0.4, 0.7, 1.6, and 8.1 µM, respectively, as determined by the Ca-Mg-ATP-EGTA Calculator v1.0 using constants from the NIST database #46 v8. At 1.0, 2.0, 3.0, 3.5, 4.0, 4.5, and 4.9 mM CaCl2 in ubiquitination buffer, free Ca2+ concentrations were calculated to be 0.04, 0.1, 0.2, 0.4, 0.7, 1.4, and 6.7 µM, respectively.</p>\n",
      "<p></p>\n",
      "<p>In vitro tubulation assay</p>\n",
      "<p>An in vitro tubulation assay was performed as described (Uezu et al, 2011; Tsujita et al, 2006) with slight modifications. Briefly, liposomes were prepared from brain lipids (Folch lipids) supplemented with 1 % rhodamine-PE. The liposomes (0.2 mg/ml liposomes) were incubated with the Nedd4L C2 domain (4.5 µM) or BAR domains (1.4 µM) at 25°C for 10 min in lipid buffer containing 3 mM CaCl2 (0.3 µM Ca2+). The sample was examined by fluorescence or electron microscopy. The tubulation assay with mSA-αENaC-associated liposomes was performed using the sample mixture utilized in the co-sedimentation assay in which mSA-αENaC-associated liposomes (brain-lipid liposomes) were used.</p>\n",
      "<p></p>\n",
      "<p>Negative-staining electron microscopy</p>\n",
      "<p>Negative-staining transmission electron microscopy was performed as described (Shimada et al, 2007). Box and whisker plots were used to represent the distribution of liposome diameters.</p>\n",
      "<p></p>\n",
      "<p>In vivo ubiquitination assay</p>\n",
      "<p>HEK293 cells were used, because they formed membrane tubules upon FCHO2 expression more efficiently than did COS7 cells. HEK293 cells at approximately 90% confluence on a 6-well plate were transfected with 0.8 µg of pCMV-Myc-Nedd4L, 1.6 µg of pEGFP-BAR domain, and 1.6 µg of pCGN-HA-Ub. In some experiments, cells were transfected with various doses (0–1.5 µg) of pCMV-Myc-Nedd4L, 1.5 µg of pEGFP-FCHO2 BAR domain, 1 µg of pCGN-HA-Ub, and 0.5 µg of pCAGI-Hygro-FLAG-αENaC. Total transfected cDNA amount was held constant using empty pCMV-Myc vector. Amiloride (40 µM) was added to the medium when αENaC cDNA was transfected. At 21 h after transfection, cells were washed with PBS and incubated with serum-free DMEM containing 1% BSA. After 3 h incubation, 10 µM MG132 was added to the medium. After further 4 h incubation, cells were solubilized in 150 µl of lysis buffer C (50 mM Tris-HCl at pH 7.5, 150 mM NaCl, 5 mM EDTA, 10 mM N-ethylmaleimide, 1 mM PMSF, 10 µg/ml leupeptin, and 1µg/ml pepstatin A) containing 1% SDS. After boiling, the sample was sonicated and then centrifuged at 20,000 × g for 15 min. The supernatant was diluted 10-fold with lysis buffer C containing 1% Triton X-100 and then incubated at 4°C for 3 h with either anti-Myc or anti-FLAG M2 antibody. Protein G-Sepharose beads were added to the sample, which was then incubated at 4°C for 3 h. After the beads were thoroughly washed with lysis buffer C containing 1% Triton X-100, bound proteins were eluted by boiling in SDS sample buffer for 5 min. These proteins were then subjected to SDS-PAGE, followed by IB.</p>\n",
      "<p></p>\n",
      "<p>In vitro ubiquitination assay</p>\n",
      "<p>mSA-αENaC-associated liposomes were made by incubating liposomes (10 µg) with mSA-αENaC (35 pmol) in 25 µl of lipid buffer, as described for the co-sedimentation assay. They were then incubated at 25°C for 10 min with 0.07 µM Nedd4L, 0.04 µM E1 (Ube1), 0.7 µM E2 (UbcH7), 7 µM Ub, and various concentrations of BAR domains in 50 µl of ubiquitination buffer containing 4 mM CaCl2 (0.7 µM Ca2+) and 2 mM ATP. Where indicated, various concentrations of CaCl2 were used. The ubiquitination reaction was started by adding ATP and stopped by adding 10 µl of 6 × SDS sample. The sample was then boiled, subjected to SDS-PAGE, and immunoblotted with anti-Ub and anti-αENaC antibodies. The chemiluminescence intensity was quantified by scanning using ImageJ (NIH).</p>\n",
      "<p></p>\n",
      "<p>Other procedures</p>\n",
      "<p>Stripping immunoblots for re-probing was carried out with the WB Stripping Solution Strong (Nacalai Tesque) according to the manufacturer’s protocol. Protein concentrations were quantified with BSA as a reference protein by the Bradford method (Bio-Rad) or the BCA method (Thermo Fisher Scientific). SDS-PAGE was performed as described (Laemmli, 1970).</p>\n",
      "<p></p>\n",
      "<p>Statistical analysis</p>\n",
      "<p>All experiments were performed at least three times (biological replication) and representative results are shown. Data are shown as the mean ± S.E.M. Student’s tests, one-way analysis of variance test, and two-way analysis of variance test were used to evaluate statistical significances between different treatment groups.</p>\n",
      "<p></p>\n",
      "<p>Data availability</p>\n",
      "<p>This study includes no data deposited in external repositories.</p>\n",
      "<p></p>\n",
      "<p>Acknowledgments</p>\n",
      "<p></p>\n",
      "<p>We thank Drs. H. Kai and M. Nakao for kindly providing us with ENaC and Ub cDNAs, respectively. We are also grateful to Dr. T. Takenawa (Kobe University, Kobe, Japan) for helpful discussions. This work was partly carried out at the Institute of Molecular Embryology and Genetics, the Gene Technology Centre, and Research Facilities of the School of Medicine, Kumamoto University. This study was supported by JSPS KAKENHI Grant Numbers 19K06643 (to H.N.), 19K06544 (to Y.S.), and Setsuro Fujii Memorial, The Osaka Foundation for Promotion of Fundamental Medical Research (to Y.S.) and by grants from the Mochida Memorial Foundation, the Takeda Science Foundation, and the Uehara Memorial Foundation (to K.K.).</p>\n",
      "<p></p>\n",
      "<p>Disclosure and Competing Interests Statement</p>\n",
      "<p>The authors declare no competing interests.</p>\n",
      "<p></p>\n",
      "<p></p>\n",
      "<p>References</p>\n",
      "<p></p>\n",
      "<p>Aguet F, Antonescu CN, Mettlen M, Schmid S & Danuser G (2013) Advances in analysis of low signal-to-noise images link dynamin and AP2 to the function of an endocytic checkpoint. Dev Cell 26: 279–291</p>\n",
      "<p>Berndsen CE & Wolberger C (2011) A spectrophotometric assay for conjugation of ubiquitin and ubiquitin-like proteins. Anal Biochem 418: 102–110</p>\n",
      "<p>Butterworth MB (2010) Regulation of the epithelial sodium channel (ENaC) by membrane trafficking. Biochim Biophys Acta 1802: 1166–77</p>\n",
      "<p>Chang-Ileto B, Frere SG, Chan RB, Voronov SV, Roux A, & Di Paolo G (2011) Synaptojanin 1-mediated PI(4,5)P2 hydrolysis is modulated by membrane curvature and facilitates membrane fission. Dev. Cell 20: 206–218</p>\n",
      "<p>Doherty GJ & McMahon HT (2009) Mechanisms of endocytosis. Annu Rev Biochem 78: 857–902</p>\n",
      "<p>Di Fiore PP, Polo S & Hofmann K (2003) When ubiquitin meets ubiquitin receptors: a signalling connection. Nat Rev Mol Cell Biol 4: 491–497</p>\n",
      "<p>Escobedo A, Gomes T, Aragon E, Martin-Malpartida P, Ruiz L, & Macias MJ (2014) Structural basis of the activation and degradation mechaisms of the E3 ubiquitin lagase Nedd4L. Structure 22: 1446–1457</p>\n",
      "<p>Firsov D, Schild L, Gautschi I, Mérillat AM, Schneeberger E & Rossier BC (1996) Cell surface expression of the epithelial Na channel and a mutant causing Liddle syndrome: a quantitative approach. Proc Natl Acad Sci U S A 93: 15370–15375</p>\n",
      "<p>Frost A, Perera R, Roux A, Spasov K, Destaing O, Egelman EH, De Camilli P & Unger VM (2008) Structural basis of membrane invagination by F-BAR domains. Cell 132: 807–817</p>\n",
      "<p>Henne WM, Boucrot E, Meinecke M, Evergren E, Vallis Y, Mittal R & McMahon HT (2010) FCHo proteins are nucleators of clathrin-mediated endocytosis. Science 328: 1281–4</p>\n",
      "<p>Henne WM, Kent HM, Ford MGJ, Hegde BG, Daumke O, Butler PJG, Mittal R, Langen R, Evans PR & McMahon HT (2007) Structure and Analysis of FCHo2 F-BAR Domain: A Dimerizing and Membrane Recruitment Module that Effects Membrane Curvature. Structure 15: 839–852</p>\n",
      "<p>Hershko A & Ciechanover A (1998) THE UBIQUITIN SYSTEM. Annu Rev Biochem 67: 425–479</p>\n",
      "<p>Hughey RP, Mueller GM, Bruns JB, Kinlough CL, Poland PA, Harkleroad KL, Carattino MD & Kleyman TR (2003) Maturation of the epithelial Na+ channel involves proteolytic processing of the alpha- and gamma-subunits. J Biol Chem 278: 37073–37082</p>\n",
      "<p>Hui E, Johnson CP, Yao J, Dunning FM & Chapman ER (2009) Synaptotagmin-mediated bending of the target membrane is a critical step in Ca(2+)-regulated fusion. Cell 138: 709–21</p>\n",
      "<p>Ingham RJ, Gish G & Pawson T (2004) The Nedd4 family of E3 ubiquitin ligases: functional diversity within a common modular architecture. Oncogene 23: 1972–84</p>\n",
      "<p>Laemmli UK (1970) Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature 227: 680–685</p>\n",
      "<p>Lehmann M, Lukonin I, Noe F, Schmoranzer J, Clementi C, Loerke D & Haucke V (2019) Nanoscale coupling of endocytic pit growth and stability. Sci Adv 5: eaax5775</p>\n",
      "<p>Lemmon M (2008) Membrane recognition by phospholipid-binding domains. Nat Rev Mol Cell Biol 9: 99–111</p>\n",
      "<p>Lim KH, Huang H, Pralle A & Park S (2013) Stable, high-affinity streptavidin monomer for protein labeling and monovalent biotin detection. Biotechnol Bioeng 110: 57–67</p>\n",
      "<p>Martens S, Kozlov MM & McMahon HT (2007) How synaptotagmin promotes membrane fusion. Science 316: 1205–8</p>\n",
      "<p>Miyahara M, Nakanishi H, Takahashi K, Satoh-Horikawa K, Tachibana K & Takai Y (2000) Interaction of nectin with afadin is necessary for its clustering at cell-cell contact sites but not for its cis dimerization or trans interaction. J Biol Chem 275: 613–618</p>\n",
      "<p>Nishizuka Y (1992) Intracellular signaling by hydrolysis of phospholipids and activation of protein kinase C. Science 258: 607–14</p>\n",
      "<p>Peter BJ, Kent HM, Mills IG, Vallis Y, Butler PJG, Evans PR & McMahon HT (2004) BAR domains as sensors of membrane curvature: the amphiphysin BAR structure. Science 303: 495–9</p>\n",
      "<p>Pickart CM & Raasi S (2005) Controlled synthesis of polyubiquitin chains. Methods Enzym 399: 21–36</p>\n",
      "<p>Roberts M, Barry S, Woods A, van der Sluijs P, & Norman J (2001) PDGF-regulated rab4-dependent recycling of avb3 integrin from early endosomes is necessary for cell adhesion and spreading. Curr Biol 11: 1392–1402</p>\n",
      "<p>Rossier BC (2014) Epithelial sodium channel (ENaC) and the control of blood pressure. Curr Opin Pharmacol 15: 33–46</p>\n",
      "<p>Rotin D & Kumar S (2009) Physiological functions of the HECT family of ubiquitin ligases. Nat Rev Mol Cell Biol 10: 398–409</p>\n",
      "<p>Rotin D & Staub O (2011) Role of the ubiquitin system in regulating ion transport. Pflugers Arch 461: 1–21</p>\n",
      "<p>Saffarian S, Coccuci E, & Kirchausen T (2009) Distinct dynamics of endocytic clathrin-coated pits and coated palques. PLoS Biol 7: e1000191</p>\n",
      "<p>Sakamoto Y, Kikuchi K, Umeda K & Nakanishi H (2017) Effects of various spacers between biotin and the phospholipid headgroup on immobilization and sedimentation of biotinylated phospholipid-containing liposomes facilitated by avidin-biotin interactions. J Biochem 162: 221–226</p>\n",
      "<p>Shaner NC, Lin MZ, McKeown MR, Steinbach PA, Hazelwood KL, Davidson MW & Tsien RY (2008) Improving the photostability of bright monomeric orange and red fluorescent proteins. Nat Methods 5: 545–551</p>\n",
      "<p>Shimada A, Niwa H, Tsujita K, Suetsugu S, Nitta K, Hanawa-Suetsugu K, Akasaka R, Nishino Y, Toyama M, Chen L, et al (2007) Curved EFC/F-BAR-domain dimers are joined end to end into a filament for membrane invagination in endocytosis. Cell 129: 761–772</p>\n",
      "<p>Sugahara T, Koga T, Ueno-Shuto K, Shuto T, Watanabe E, Maekawa A, Kitamura K, Tomita K, Mizuno A, Sato T, et al (2009) Calreticulin positively regulates the expression and function of epithelial sodium channel. Exp Cell Res 315: 3294–3300</p>\n",
      "<p>Takano K, Toyooka K & Suetsugu S (2008) EFC/F-BAR proteins and the N-WASP–WIP complex induce membrane curvature-dependent actin polymerization. EMBO J 27: 2817–2828</p>\n",
      "<p>Takei K, Slepnev VI, Haucke V & De Camilli P (1999) Functional partnership between amphiphysin and dynamin in clathrin-mediated endocytosis. Nat Cell Biol 1: 33–39</p>\n",
      "<p>Traub LM (2009) Tickets to ride: selecting cargo for clathrin-regulated internalization. Nat Rev Mol Cell Biol 10: 583–96</p>\n",
      "<p>Tsujita K, Suetsugu S, Sasaki N, Furutani M, Oikawa T & Takenawa T (2006) Coordination between the actin cytoskeleton and membrane deformation by a novel membrane tubulation domain of PCH proteins is involved in endocytosis. J Cell Biol 172: 269–279</p>\n",
      "<p>Uezu A, Umeda K, Tsujita K, Suetsugu S, Takenawa T & Nakanishi H (2011) Characterization of the EFC/F-BAR domain protein, FCHO2. Genes Cells 16: 868–878</p>\n",
      "<p>Umeda K, Ikenouchi J, Katahira-Tayama S, Furuse K, Sasaki H, Nakayama M, Matsui T, Tsukita S, Furuse M & Tsukita S (2006) ZO-1 and ZO-2 independently determine where claudins are polymerized in tight-junction strand formation. Cell 126: 741–754</p>\n",
      "<p>Wang J, Peng Q, Lin Q, Childress C, Carey D & Yang W (2010) Calcium activates Nedd4 E3 ubiquitin ligases by releasing the C2 domain-mediated auto-inhibition. J Biol Chem 285: 12279–12288</p>\n",
      "<p>Wiesner S, Ogunjimi AA, Wang H-R, Rotin D, Sicheri F, Wrana JL & Forman-Kay JD (2007) Autoinhibition of the HECT-type ubiquitin ligase Smurf2 through its C2 domain. Cell 130: 651–862</p>\n",
      "<p>Yang B & Kumar S (2010) Nedd4 and Nedd4-2: closely related ubiquitin-protein ligases with distinct physiological functions. Cell Death Differ 17: 68–77</p>\n",
      "<p>Yin Y, Arkhipov A & Schulten K (2009) Simulations of Membrane Tubulation by Lattices of Amphiphysin N-BAR Domains. Structure 17: 882–892</p>\n",
      "<p>Zaccai N, Kadlecova Z, Dickson VK, Korobchevskaya K & Traub L (2022) FCHO control AP2's initiating role in endocytosis through a Ptd-Ins(4,5)P2-dependent switch. Sci Adv 8: eabn2018</p>\n",
      "<p>Zhou R, Patel S V. & Snyder PM (2007) Nedd4-2 catalyzes ubiquitination and degradation of cell surface ENaC. J Biol Chem 282: 20207–20212</p>\n",
      "<p>Zhu K, Shan Z, Chen X, Cai Y, Cui L, Yao W, Wang Z, Shi P, Tian C, Lou J, et al (2017) Allosteric auto‐inhibition and activation of the Nedd4 family E3 ligase Itch. EMBO Rep 18: 1618–1630</p>\n",
      "<p></p>\n",
      "<p></p>\n",
      "<p></p>\n",
      "<p>Figure legends</p>\n",
      "<p></p>\n",
      "<p>Figure 1. FCHO2 is required for Nedd4L-mediated ubiquitination and endocytosis of ENaC.</p>\n",
      "<p>A. Knockdown of FCHO2 or Nedd4L by siRNA. αβγENaC-HeLa cells treated with each siRNA were analyzed by immunoblotting (IB).</p>\n",
      "<p>B. Inhibition of ENaC endocytosis by knockdown of FCHO2 or Nedd4L. After cells were transfected with each siRNA, cell-surface ENaC was labeled by incubating at 4°C with anti-FLAG antibody. Endocytosis was started by incubating cells at 37°C for the indicated periods of time and stopped by chilling cells on ice. Antibody bound to the cell surface but not internalized was removed by acid stripping. After cells were solubilized, the antibody-labeled internalized ENaC subunits were precipitated and analyzed by IB (upper panel) and quantified (bottom and right panels). Internalization was expressed as a percentage of the initial amount of cell-surface αENaC determined by lysing cells without incubation at 37°C or acid stripping. Data are shown as the mean ± S.E.M. of three independent experiments. **P < 0.01; ***P < 0.001 (Student's t-test).</p>\n",
      "<p>C. Inhibition of transferrin receptor (TfR) endocytosis by FCHO2 knockdown. After cells were transfected with each siRNA, cell-surface proteins were biotinylated at 4°C. Cells were then incubated at 37°C for 10 min. Biotin was removed from the cell surface, and internalized biotinylated proteins were precipitated with avidin beads. Samples were subjected to IB with anti-TfR antibody and quantified. Internalization was expressed as a proportion of the initial amount of TfR on the cell surface. Data are shown as the mean ± S.E.M. of three independent experiments. *P < 0.05 (Student's t-test).</p>\n",
      "<p>D. Restoration of αENaC endocytosis by FCHO2 re-expression in knockdown cells. After cells were transfected with the indicated siRNA and GFP construct, cell-surface αENaC was labeled with anti-FLAG antibody at 4°C. Cells were then incubated at 37°C for 10 min. After acid stripping, cells were subjected to immunofluorescence microscopy (left panels) and quantified (right panel). The antibody-labeled, internalized αENaC was visualized with a fluorescence-conjugated secondary antibody. The border of each cell is delineated by a solid line. Arrows indicate cells expressing GFP-tagged proteins. Scale bars, 10 μm. The proportion of cells with internalized αENaC among cells expressing control GFP or GFP-siRNA-resistant (sr) -FCHO2 is shown. Data are shown as the mean ± S.E.M. of five independent images (31-71 cells expressing GFP-tagged proteins per image). ***P < 0.001 (two-way analysis of variance with post hoc test).</p>\n",
      "<p>E, F. Effects of FCHO2, Nedd4L, and FBP17 knockdown on ENaC ubiquitination and expression at the cell surface. After cells were transfected with each siRNA, cell-surface αENaC was biotinylated and sequentially precipitated with anti-FLAG antibody and then avidin beads. Samples were subjected to IB with anti-FLAG and anti-Ub antibodies (E) and quantified (F). E, ENaC ubiquitination, Asterisks indicate the same band of ubiquitinated FLAG-αENaC. F, ENaC expression at the cell surface. Data are shown as the mean ± S.E.M. of three independent experiments. *P < 0.05 (Student's t-test).</p>\n",
      "<p></p>\n",
      "<p>Figure 2. Localization of cell-surface ENaC, FCHO2, and Nedd4L at clathrin-coated structures.</p>\n",
      "<p>A, B. Localization of cell-surface ENaC. mTagRFP-clathrin light chain (CLC) (A) or Nedd4L C922A (catalytic inactive mutant)-GFP (B) was expressed in αβγENaC-HeLa cells. Cell-surface αENaC was labeled with anti-FLAG antibody at 4°C. Cells were then fixed and stained with anti-FCHO2 antibody. Samples were subjected to immunofluorescence microscopy (left panel). Insets represent enlarged images of the dashed boxes. Scale bar, 10 μm. Co-localization was analyzed (right panel). The mean values of Pearson’s correlation coefficient from three cells and the S.E.M. are shown. A. Co-localization of cell-surface ENaC with FCHO2 at clathrin-coated structures. B. Co-localization of cell-surface ENaC with Nedd4L C922A mutant and FCHO2.</p>\n",
      "<p>C. Co-localization of Nedd4L with FCHO2 at clathrin-coated structures. Nedd4L-GFP and mTagRFP-CLC were co-expressed in wild-type HeLa cells. Cells were then fixed and stained with anti-FCHO2 antibody (left panel). Insets represent enlarged images of the dashed boxes. Scale bar, 10 μm. Co-localization was analyzed (right panel). The mean values of Pearson’s correlation coefficient from three cells and the S.E.M. are shown.</p>\n",
      "<p></p>\n",
      "<p>Figure 3. Recruitment of Nedd4L to CCPs. DsRed-CLC were co-expressed with Nedd4L-GFP in wild-type HeLa cells. Images were acquired at 1 s intervals through TIRF microscopy. Upper panel, TIRF images of HeLa cells. Insets represent enlarged images of the dashed boxes. Bottom panel, selected snapshots (every 10 sec) from a time series were obtained from a single CCP indicated by the arrowhead. The time point at which Nedd4L started to accumulate was set 0. Scale bar, 2 μm.</p>\n",
      "<p></p>\n",
      "<p>Figure 4. Recruitment of Nedd4L to membrane tubules generated by FCHO2 in cells.</p>\n",
      "<p>GFP-BAR domains were co-expressed with Myc-Nedd4L proteins in COS7 cells (A) and HEK293 cells (B). Cells were serum-starved and subjected to immunofluorescence microscopy. Insets represent enlarged images of the dashed boxes. Scale bar, 10 μm.</p>\n",
      "<p></p>\n",
      "<p>Figure 5. Activation of Nedd4L by membrane tubules generated by FCHO2 in cells.</p>\n",
      "<p>A–E. Nedd4L autoubiquitination. An in vivo ubiquitination assay was performed with various BAR domains and FCHO2 mutants (A, B) and Nedd4L mutants (D). GFP-BAR domains were co-expressed with Myc-Nedd4L proteins and HA-tagged Ub in HEK293 cells. Cell lysates were subjected to immunoprecipitation (IP) and analyzed by IB (A, B, D). WT, wild type. Cells were also subjected to immunofluorescence microscopy (C, E). Scale bar, 10 μm.</p>\n",
      "<p>F. αENaC ubiquitination. An in vivo ubiquitination assay was performed with various concentrations of Myc-Nedd4L using FALAG-αENaC as a substrate in the presence or absence of GFP-FCHO2 BAR domain. Cell lysates were subjected to IP. Samples were analyzed by IB.</p>\n",
      "<p></p>\n",
      "<p>Figure 6. The Nedd4L C2 domain is a Ca2+-dependent PS binding domain.</p>\n",
      "<p>A co-sedimentation assay was performed with the C2 domain or full-length Nedd4L using brain-lipid liposomes (~50% PS) at various Ca2+ concentrations (A) and with the C2 domain using synthetic liposomes containing various percentages of PS at 0.3 μM or 1.0 mM Ca2+ (B). The supernatants (S) and pellets (P) were subjected to sodium dodecyl sulfate (SDS)-polyacrylamide gel electrophoresis (PAGE), followed by Coomassie brilliant blue (CBB) staining (upper panels). Bottom panels, quantitative analysis. (−) for [Ca2+] indicates EGTA alone. Data are shown as the mean ± S.E.M. of three independent experiments.</p>\n",
      "<p>A. Ca2+ dependency.</p>\n",
      "<p>B. PS dependency.</p>\n",
      "<p></p>\n",
      "<p>Figure 7. Preference of Nedd4L for FCHO2-generated membrane curvature.</p>\n",
      "<p>A. Distribution of liposome diameters. Brain-lipid liposomes (~50% PS) of different diameters were prepared by extrusion through the indicated pore sizes and sonication. Left panel, electron microscopic images. Scale bars, 200 nm. Right panel, distribution of liposome diameters shown by boxplots (number of observations per pore size = 25). The center line inside the box corresponds to the median, the bounds of the box encompass data points between the first and third quartiles, and the whiskers extend to the minimum and maximum values including outliers. So, sonication.</p>\n",
      "<p>B-D. Sensing and generation of membrane curvature by Nedd4L. Co-sedimentation (B) and in vitro tubulation (C, D) assays were performed using brain-lipid liposomes (~50% PS) at 0.1 μM (B) and 0.3 μM Ca2+ (C, D). B, Preference of Nedd4L for a specific degree of membrane curvature. Liposomes were supplemented with X-biotin-PE and rhodamine-PE. Full-length Nedd4L was incubated with liposomes of different sizes, which were prepared by extrusion through the indicated pore sizes. The sample was then incubated with avidin beads and ultracentrifuged. The proportion of precipitated liposomes of each size was calculated to be ~100% by measuring both total and supernatant fluorescence. The supernatants (S) and pellets (P) were subjected to SDS- PAGE, followed by CBB staining (upper panel). Bottom panel, quantitative analysis. So, sonication. Data are shown as the mean ± S.E.M. of five independent experiments. *P < 0.05; **P < 0.01 (one-way analysis of variance with Tukey’s post hoc test). C, Curvature generation by Nedd4L. The assay was performed with the Nedd4L C2 domain. Samples were examined by electron microscopy. Scale bar, 100 nm. D, The degree of membrane curvature generated by Nedd4L is consistent with that generated by FCHO2. The assay was performed with the indicated protein. Left panel, electron microscopic image. Scale bar, 100 nm. Right panel, distribution of tubule diameters shown by boxplots (number of observations per protein = 14–33). The center line inside the box corresponds to the median, the bounds of the box encompass data points between the first and third quartiles, and the whiskers extend to the minimum and maximum values including outliers.</p>\n",
      "<p></p>\n",
      "<p>Figure 8. Conserved hydrophobic residues of Nedd4L are responsible for sensing and generation of membrane curvature.</p>\n",
      "<p>A. Sequence alignment of C2 domains. Conserved residues are highlighted with the following color code: yellow, aspartates for Ca2+ binding; blue, very hydrophobic residues probably for penetration into the lipid bilayer; and brown, others. Dots show the mutation sites. Hs, Homo sapiens; Sc, Saccharomyces cerevisiae; and Rt, Rattus norvegicus.</p>\n",
      "<p>B. Inability of Nedd4L C2 domain mutants to sense membrane curvature in cells. GFP-FCHO2 BAR domain and Myc-Nedd4L C2 domain mutants were co-expressed in COS7 cells. Insets are enlarged images of dashed boxes. Scale bar, 10 μm.</p>\n",
      "<p>C, D. Inability of Nedd4L C2 domain mutants to bind to membranes and to generate membrane curvature in vitro. Co-sedimentation (C) and in vitro tubulation (D) assays were performed using brain-lipid liposomes (~50% PS) at 0.3 μM Ca2+. C, Membrane binding. The assay was performed with Nedd4L C2 domain mutants using various concentrations of liposomes. The supernatants (S) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining (upper panel). Bottom panel, quantitative analysis. Data are shown as the mean ± S.E.M. of three independent experiments. D, Curvature generation. The assay was performed with Nedd4L C2 domain mutants. Samples were examined by fluorescence microscopy. Scale bar, 10 μm.</p>\n",
      "<p></p>\n",
      "<p>Figure 9. Properties of the catalytic activity and membrane binding of Nedd4L.</p>\n",
      "<p>Liposomes supplemented with biotinylated phospholipids were incubated with mSA-αENaC (A, B). Samples were mixed with Nedd4L. Mixtures were subjected to ubiquitination (C-E) and co-sedimentation (F) assays. These assays were performed with brain-lipid liposomes (~50% PS) at various Ca2+ concentrations (C), with synthetic liposomes (various percentages of PS) at 0.7 μM Ca2+ (D, F), and with brain-lipid liposomes (20% PS) at 0.7 μM Ca2+ (E). Ubiquitination and liposome binding were quantified by scanning and expressed as ratios to maximum levels. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.</p>\n",
      "<p>A, B. Schematic diagrams of mSA-αENaC. A, Structure. B. Binding to biotinylated liposomes.</p>\n",
      "<p>C. Ca2+-dependent Nedd4L activity. Ubiquitination was detected by IB (upper panel) and the intensity was quantified by scanning (bottom panel). Data are shown as the mean ± S.E.M. of three independent experiments.</p>\n",
      "<p>D, Membrane localization of mSA-αENaC enhances the PS-dependent catalytic activity of Nedd4L. Assays were performed in the presence or absence of 40 μM biotin. Ubiquitination was detected by IB (left panel). The intensities were quantified by scanning (right panel). Data are shown as the mean ± S.E.M. of three independent experiments.</p>\n",
      "<p>E. Nedd4L activity is predominantly stimulated by the specific degree of membrane curvature. The assay was performed using liposomes of different sizes which were prepared by extrusion through the indicated pore sizes. Ubiquitination was detected by IB (left panel) and quantified (right panel). Data are shown as the mean ± S.E.M. of three independent experiments. ***P < 0.001 (one-way analysis of variance with Tukey’s post hoc test).</p>\n",
      "<p>F. Membrane localization of mSA-αENaC enhances the PS-dependent membrane binding of Nedd4L. Assays were performed in the presence or absence of 40 μM biotin. Liposome binding was detected by IB (left panel). The intensities were quantified by scanning (right panel). T, total sample. P, pellet. Data are shown as the mean ± S.E.M. of three independent experiments.</p>\n",
      "<p></p>\n",
      "<p></p>\n",
      "<p>Figure 10. Recruitment and activation of Nedd4L by FCHO2-generated membrane curvature in vitro.</p>\n",
      "<p>mSA-αENaC-associated brain-lipid liposomes (20% PS) were mixed with Nedd4L in the presence or absence of BAR domains. Mixtures were subjected to co-sedimentation (A) and ubiquitination (B-E) assays. These assays were performed at 0.7 μM Ca2+ with brain-lipid liposomes (20%PS) (A, B, D, and E) and with the indicated (20% PS or ~50% PS) brain-lipid liposomes (C). Ubiquitination and liposome binding were quantified by scanning and expressed as ratios to maximum levels. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.</p>\n",
      "<p>A. FCHO2-induced recruitment of Nedd4L. The assay was performed with the FCHO2 BAR domain (1.4 μM, 50 μg/ml) in the presence or absence of biotin. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining (FCHO2 BAR) and IB (Nedd4L and mSA-αENaC).</p>\n",
      "<p>B, C. FCHO2-induced activation of Nedd4L. B, Effects of various doses of the FCHO2 BAR domain. The assay was performed with various concentrations of the FCHO2 BAR domain. Ubiquitination was detected by IB (left panel) and the intensity was quantified by scanning (right panel). C, Effects of the PS percentage in liposomes. The assay was performed with the FCHO2 BAR domain (1.4 μM). Samples were analyzed by IB. Data are shown as the mean ± S.E.M. of three independent experiments.</p>\n",
      "<p>D, E. Specificity of BAR domains and effect of an FCHO2 mutant. The assay was performed with various BAR domains (D) and an FCHO2 mutant (E) (1.4 μM each). Samples were analyzed by IB.</p>\n",
      "<p></p>\n",
      "<p>Figure 11. Effects of PI(4,5)P2 on FCHO2-induced recruitment and activation of Nedd4L.</p>\n",
      "<p>A, B. Binding to PI(4,5)P2. A co-sedimentation assay was performed at 0.7 μM Ca2+ with the FCHO2 BAR or Nedd4L C2 domain using mSA-αENaC-associated synthetic liposomes containing 20% PS, 5% PI(4,5)P2 (A), or various percentages of PI(4,5)P2 (B). The supernatants (S) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining (A and upper panel in B). Bottom panel in B, quantitative analysis. Control, synthetic liposomes [0% PS and 0% PI(4,5)P2]. Data are shown as the mean ± S.E.M. of three independent experiments.</p>\n",
      "<p>C, D. Effects on FCHO2-induced recruitment and activation of Nedd4L. Co-sedimentation (C) and ubiquitination (D) assays were performed at 0.7 μM Ca2+ with mSA-αENaC-associated synthetic liposomes containing 20% PS or 5% PI(4,5)P2 in the presence or absence of the FCHO2 BAR domain. C, Nedd4L recruitment. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by IB with anti-Nedd4L antibody (upper panel) and CBB staining (lower panel). D, Nedd4L activation. Samples were analyzed by IB. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.</p>\n",
      "<p></p>\n",
      "<p>Figure 12. Effects of liposome size on FCHO2-induced activation of Nedd4L. Co-sedimentation (A) and ubiquitination (B) assays were performed at 0.7 μM Ca2+ with the FCHO2 BAR domain using brain-lipid liposomes (20% PS, 0.8 µm or 0.05 µm pore-size) that were associated with mSA-αENaC.</p>\n",
      "<p>A. Binding of the FCHO2 BAR domain to 0.8 µm and 0.05 µm pore-size liposomes. The co-sedimentation assay was performed using synthetic (0% PS) or brain-lipid liposomes (20% PS) containing rhodamine-PE and fluorescein-PE. Liposomes were precipitated with anti-fluorescein antibody. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining. The proportion of precipitated liposomes of each size was calculated to be ~100% by measuring both total and supernatant fluorescence.</p>\n",
      "<p>B. Effects of liposome size on FCHO2-induced activation of Nedd4L. Samples were analyzed by IB. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.</p>\n",
      "<p></p>\n",
      "<p>Figure 13. Model of ENaC endocytosis.</p>\n",
      "<p>FCHO2 binds to the plasma membrane and generates a specific degree of membrane curvature according to the intrinsic curvature of its BAR domain. Nedd4L exists in a catalytically autoinhibited state due to an intramolecular interaction between the N-terminal C2 and C-terminal HECT domains. Nedd4L is recruited to FCHO2-generated membrane curvature at the rim of nascent CCPs that ENaC enters. Membrane binding of the C2 domain causes catalytic activation by relieving autoinhibition. ENaC is ubiquitinated and captured by adaptor proteins with Ub-interacting motifs, such as Eps15.</p>\n",
      "<p></p>\n",
      "<p>Extended View Figure Legends</p>\n",
      "<p></p>\n",
      "<p>Figure EV1. Expression of ENaC subunits in αβγENaC-HeLa cells.</p>\n",
      "<p>A, B. Doxycycline (Dox)-induced expression of αENaC. αβγENaC-HeLa cells were cultured overnight in the presence or absence of Dox. The lysates of parental HeLa and αβγENaC-HeLa cells were subjected to IB. A, 7.5% gel. B, Gradient gel (5-20%). Anti-γENaC antibody cross-reacted with two bands of endogenous proteins in parental HeLa cells, the lower band of which overlapped with γENaC (arrow) in αβγENaC-HeLa cells. Upon Dox-induced αENaC expression, an additional 70-kDa γENaC band (asterisk) was detected. It has been shown that co-expression of all three subunits induces ENaC maturation, including proteolytic cleavage of α- and γENaC (Hughey et al, 2003). The 70-kDa γENaC band is likely a cleavage product comprising the C-terminal region. The 20-kDa αENaC band (arrowhead) is likely a cleavage product comprising the N-terminal region.</p>\n",
      "<p>C. Association of α-, β-, and γENaC. When αENaC was immunoprecipitated with anti-FLAG antibody from αβγENaC-HeLa cells treated with Dox, β- and γENaC were co-precipitated. Asterisk, 70-kDa γENaC band.</p>\n",
      "<p></p>\n",
      "<p>Figure EV2. Knockdown of FBP17 by siRNA.</p>\n",
      "<p>αβγENaC-HeLa cells were treated with each siRNA, and FBP17 mRNA levels were quantified using real-time PCR. The expression of FBP17 was normalized to GAPDH mRNA levels. Data are shown as the mean ± S.E.M. of three independent experiments. ***P < 0.001 (Student's t-test).</p>\n",
      "<p></p>\n",
      "<p>Figure EV3. Inability of FCHO2 mutants to generate membrane tubules.</p>\n",
      "<p>GFP-FCHO2 BAR domain [wild type (WT) or mutant] was expressed in COS7 cells. Cells were then subjected to immunofluorescence microscopy. Scale bar, 10 μm.</p>\n",
      "<p></p>\n",
      "<p>Figure EV4. Inability of Nedd4L mutants to ubiquitinate αENaC. An in vivo ubiquitination assay was performed with various Nedd4L constructs (each 0.5 µg) using FLAG-αENaC as a substrate in the presence or absence of GFP-FCHO2 BAR domain. Cell lysates were subjected to IP. Samples were analyzed by IB.</p>\n",
      "<p></p>\n",
      "<p>Figure EV5. Inhibition of the liposome binding of Nedd4L by the FCHO2 BAR domain.</p>\n",
      "<p>A co-sedimentation assay was performed at 0.7 μM Ca2+ with control liposomes (0% PS) or brain-lipid liposomes (~50% PS) in the presence or absence of the FCHO2 BAR domain. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by IB (upper panel) and CBB staining (lower panel).</p>\n",
      "<p></p>\n",
      "<p>Figure EV6. Membrane binding and curvature generation of BAR domains and an FCHO2 mutant. Co-sedimentation (A) and in vitro tubulation (B) assays were performed at 0.7 μM Ca2+ with the indicated BAR domains using brain-lipid liposomes (20% PS) that were associated with mSA-αENaC.</p>\n",
      "<p>A. Membrane binding. The supernatants (S) and pellets (P) were subjected to SDS-PAGE followed by CBB staining.</p>\n",
      "<p>B. Curvature generation. Left panel, electron microscopic image. Scale bar, 100 nm. Right panel, distribution of tubule diameters shown by boxplots (number of observations per protein = 22–26). The center line inside the box corresponds to the median, the bounds of the box encompass data points between the first and third quartiles, and the whiskers extend to the minimum and maximum values including outliers.</p>\n",
      "</html>\n"
     ]
    }
   ],
   "source": [
    "from docling.document_converter import DocumentConverter\n",
    "\n",
    "converter = DocumentConverter()\n",
    "docling_docx = converter.convert(DOCX_FILES[1])\n",
    "print(docling_docx.document.export_to_html())  # output: \"## Docling Technical Report[...]\"\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Extracting sections and headers\n",
    "\n",
    "#### Native from docling"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "self_ref='#/texts/0' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='The Nedd4L ubiquitin ligase is activated by FCHO2-generated membrane curvature' text='The Nedd4L ubiquitin ligase is activated by FCHO2-generated membrane curvature'\n",
      "self_ref='#/texts/1' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/2' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Yasuhisa Sakamoto1, Akiyoshi Uezu1, Koji Kikuchi1, Jangmi Kang2, Eiko Fujii2, Toshiro Moroishi3, Shiro Suetsugu4, and Hiroyuki Nakanishi1,2*' text='Yasuhisa Sakamoto1, Akiyoshi Uezu1, Koji Kikuchi1, Jangmi Kang2, Eiko Fujii2, Toshiro Moroishi3, Shiro Suetsugu4, and Hiroyuki Nakanishi1,2*'\n",
      "self_ref='#/texts/3' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/4' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='1Department of Molecular Pharmacology, Faculty of Life Sciences, Kumamoto University, 1–1–1 Honjyo, Kumamoto 860-8556, Japan' text='1Department of Molecular Pharmacology, Faculty of Life Sciences, Kumamoto University, 1–1–1 Honjyo, Kumamoto 860-8556, Japan'\n",
      "self_ref='#/texts/5' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='2Faculty of Clinical Nutrition and Dietetics, Konan Women’s University, 6–2–23 Morikita-machi, Kobe 658-0001, Japan' text='2Faculty of Clinical Nutrition and Dietetics, Konan Women’s University, 6–2–23 Morikita-machi, Kobe 658-0001, Japan'\n",
      "self_ref='#/texts/6' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='3Department of Molecular and Medical Pharmacology, Faculty of Life Sciences, Kumamoto University, 1–1–1 Honjyo, Kumamoto 860-8556, Japan' text='3Department of Molecular and Medical Pharmacology, Faculty of Life Sciences, Kumamoto University, 1–1–1 Honjyo, Kumamoto 860-8556, Japan'\n",
      "self_ref='#/texts/7' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='4Division of Biological Science, Graduate School of Science and Technology, Nara Institute of Science and Technology, 8916-5 Takayama-cho, Ikoma 630-0192, Japan' text='4Division of Biological Science, Graduate School of Science and Technology, Nara Institute of Science and Technology, 8916-5 Takayama-cho, Ikoma 630-0192, Japan'\n",
      "self_ref='#/texts/8' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='*Corresponding author. Tel: +81-96-373-5074; Fax: +81-96-373-5078; E-mail: hnakanis@gpo.kumamoto-u.ac.jp' text='*Corresponding author. Tel: +81-96-373-5074; Fax: +81-96-373-5078; E-mail: hnakanis@gpo.kumamoto-u.ac.jp'\n",
      "self_ref='#/texts/9' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/10' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/11' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/12' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Abstract' text='Abstract'\n",
      "self_ref='#/texts/13' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/14' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='The C2-WW-HECT domain ubiquitin ligase Nedd4L regulates membrane sorting during endocytosis through the ubiquitination of cargo molecules such as the epithelial sodium channel (ENaC). Nedd4L is catalytically autoinhibited by an intramolecular interaction between its C2 and HECT domains, but the protein’s activation mechanism is poorly understood. Here, we show that Nedd4L activation is linked to membrane shape by FCHO2, a Bin-Amphiphysin-Rsv (BAR) domain protein that regulates endocytosis. FCHO2 was required for the Nedd4L-mediated ubiquitination and endocytosis of ENaC, with Nedd4L co-localizing with FCHO2 at clathrin-coated pits. In cells, Nedd4L was specifically recruited to, and activated by, the FCHO2 BAR domain. Furthermore, we reconstituted FCHO2-induced recruitment and activation of Nedd4L in vitro. Both the recruitment and activation were mediated by membrane curvature rather than protein-protein interactions. The Nedd4L C2 domain recognized a specific degree of membrane curvature that was generated by the FCHO2 BAR domain, with this curvature directly activating Nedd4L by relieving its autoinhibition. Thus, we show for the first time a specific function (i.e., recruitment and activation of an enzyme regulating cargo sorting) of membrane curvature by a BAR domain protein.' text='The C2-WW-HECT domain ubiquitin ligase Nedd4L regulates membrane sorting during endocytosis through the ubiquitination of cargo molecules such as the epithelial sodium channel (ENaC). Nedd4L is catalytically autoinhibited by an intramolecular interaction between its C2 and HECT domains, but the protein’s activation mechanism is poorly understood. Here, we show that Nedd4L activation is linked to membrane shape by FCHO2, a Bin-Amphiphysin-Rsv (BAR) domain protein that regulates endocytosis. FCHO2 was required for the Nedd4L-mediated ubiquitination and endocytosis of ENaC, with Nedd4L co-localizing with FCHO2 at clathrin-coated pits. In cells, Nedd4L was specifically recruited to, and activated by, the FCHO2 BAR domain. Furthermore, we reconstituted FCHO2-induced recruitment and activation of Nedd4L in vitro. Both the recruitment and activation were mediated by membrane curvature rather than protein-protein interactions. The Nedd4L C2 domain recognized a specific degree of membrane curvature that was generated by the FCHO2 BAR domain, with this curvature directly activating Nedd4L by relieving its autoinhibition. Thus, we show for the first time a specific function (i.e., recruitment and activation of an enzyme regulating cargo sorting) of membrane curvature by a BAR domain protein.'\n",
      "self_ref='#/texts/15' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/16' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Keywords: Nedd4L; FCHO2; clathrin; endocytosis; membrane curvature.' text='Keywords: Nedd4L; FCHO2; clathrin; endocytosis; membrane curvature.'\n",
      "self_ref='#/texts/17' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/18' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Introduction' text='Introduction'\n",
      "self_ref='#/texts/19' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/20' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Posttranslational modification of proteins by covalent attachment of ubiquitin (Ub) is catalyzed by three enzymes: a Ub-activating enzyme (E1), a Ub-conjugating enzyme (E2), and a Ub-protein ligase (E3) that determines substrate specificity (Hershko & Ciechanover, 1998). E3 Ub ligases are classified into two categories based on their Ub transfer mechanisms: RING finger/U-box E3 and HECT-type E3. The Nedd4 family belongs to HECT-type E3 Ub ligases and consists of nine members, including Nedd4/Nedd4-1, Nedd4L/Nedd4-2, Itch, Smurf1, Smurf2, WWP1, WWP2, NEDL1, and NEDL2 (Ingham et al, 2004; Rotin & Kumar, 2009). They are characterized by a common modular organization, with an N-terminal C2 domain, two to four WW domains, and a C-terminal catalytic HECT domain. While the C2 domain was originally identified in protein kinase C (PKC) as a Ca2+-dependent phosphatidylserine (PS)-binding domain (Nishizuka, 1992), it shows significant diversity in the binding partners, including intracellular proteins and other phospholipids, such as phosphatidylinositol (4,5)-bisphosphate [PI(4,5)P2] (Lemmon, 2008). The WW domains recognize proline-rich motifs, such as PPxY (PY motif, where x is any residue), of substrates or adaptor proteins. The HECT domain catalyzes the isopeptide bond formation between the Ub C terminus and the substrate lysine residues. At least some of the Nedd4 family members, including Nedd4L, are catalytically autoinhibited by an intramolecular interaction between the C2 and HECT domains (Wang et al, 2010; Wiesner et al, 2007; Zhu et al, 2017). However, the activation mechanism of the Nedd4 family is poorly understood.' text='Posttranslational modification of proteins by covalent attachment of ubiquitin (Ub) is catalyzed by three enzymes: a Ub-activating enzyme (E1), a Ub-conjugating enzyme (E2), and a Ub-protein ligase (E3) that determines substrate specificity (Hershko & Ciechanover, 1998). E3 Ub ligases are classified into two categories based on their Ub transfer mechanisms: RING finger/U-box E3 and HECT-type E3. The Nedd4 family belongs to HECT-type E3 Ub ligases and consists of nine members, including Nedd4/Nedd4-1, Nedd4L/Nedd4-2, Itch, Smurf1, Smurf2, WWP1, WWP2, NEDL1, and NEDL2 (Ingham et al, 2004; Rotin & Kumar, 2009). They are characterized by a common modular organization, with an N-terminal C2 domain, two to four WW domains, and a C-terminal catalytic HECT domain. While the C2 domain was originally identified in protein kinase C (PKC) as a Ca2+-dependent phosphatidylserine (PS)-binding domain (Nishizuka, 1992), it shows significant diversity in the binding partners, including intracellular proteins and other phospholipids, such as phosphatidylinositol (4,5)-bisphosphate [PI(4,5)P2] (Lemmon, 2008). The WW domains recognize proline-rich motifs, such as PPxY (PY motif, where x is any residue), of substrates or adaptor proteins. The HECT domain catalyzes the isopeptide bond formation between the Ub C terminus and the substrate lysine residues. At least some of the Nedd4 family members, including Nedd4L, are catalytically autoinhibited by an intramolecular interaction between the C2 and HECT domains (Wang et al, 2010; Wiesner et al, 2007; Zhu et al, 2017). However, the activation mechanism of the Nedd4 family is poorly understood.'\n",
      "self_ref='#/texts/21' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='The Nedd4 family is implicated in a wide variety of cellular processes by regulating membrane trafficking and degradation of components of various signaling pathways (Ingham et al, 2004; Rotin & Kumar, 2009). Among a diversity of substrates, the best-characterized is the epithelial sodium channel (ENaC). This channel is composed of three subunits, α-, β-, and γENaC, each of which possesses two transmembrane segments and intracellular N- and C-termini with a large extracellular loop (Butterworth, 2010; Rotin & Staub, 2011; Rossier, 2014). Each subunit has a PY motif at the cytosolic C-terminus. Nedd4L Ub ligase specifically binds to the PY motif and mediates the ubiquitination of lysine residues on the N-terminus of at least α- and γENaC at the plasma membrane. Ub serves as an internalization signal for recognition by adaptor proteins, such as epsin and Eps15, for clathrin-mediated endocytosis (CME) (Traub, 2009). Consequently, ENaC is constitutively internalized through CME. Heterozygous mutations of the PY motif of β- or γENaC lead to impaired ubiquitination and endocytosis, resulting in persistence of the channels at the cell surface and increased Na+ absorption in Liddle syndrome, an autosomal dominant form of severe hypertension.' text='The Nedd4 family is implicated in a wide variety of cellular processes by regulating membrane trafficking and degradation of components of various signaling pathways (Ingham et al, 2004; Rotin & Kumar, 2009). Among a diversity of substrates, the best-characterized is the epithelial sodium channel (ENaC). This channel is composed of three subunits, α-, β-, and γENaC, each of which possesses two transmembrane segments and intracellular N- and C-termini with a large extracellular loop (Butterworth, 2010; Rotin & Staub, 2011; Rossier, 2014). Each subunit has a PY motif at the cytosolic C-terminus. Nedd4L Ub ligase specifically binds to the PY motif and mediates the ubiquitination of lysine residues on the N-terminus of at least α- and γENaC at the plasma membrane. Ub serves as an internalization signal for recognition by adaptor proteins, such as epsin and Eps15, for clathrin-mediated endocytosis (CME) (Traub, 2009). Consequently, ENaC is constitutively internalized through CME. Heterozygous mutations of the PY motif of β- or γENaC lead to impaired ubiquitination and endocytosis, resulting in persistence of the channels at the cell surface and increased Na+ absorption in Liddle syndrome, an autosomal dominant form of severe hypertension.'\n",
      "self_ref='#/texts/22' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='The process of CME begins at the cytoplasmic surface of the plasma membrane with the assembly of clathrin with adaptor proteins and transmembrane cargo molecules (Doherty & McMahon, 2009; Traub, 2009). In parallel with this assembly, membrane curvature is generated to form hemispherical clathrin-coated pits (CCPs). This process leads to deep membrane invaginations. Subsequently, the neck of invaginated pits is constricted and severed to separate the newly formed clathrin-coated vesicles (CCVs) from the plasma membrane. These membrane curvatures are considered to be generated, sensed, and/or maintained by Bin-Amphiphysin-Rsv (BAR) domain superfamily proteins. BAR domains form a crescent-shaped dimer characterized by a specific degree of intrinsic curvature. They bind to negatively charged phospholipids, such as PS, and force membranes to bend according to their intrinsic curvatures, thereby inducing the formation of membrane tubules with specific diameters (curvatures). For example, the diameters of membrane tubules generated by the amphiphysin BAR domain, FCHO2 BAR domain, and FBP17 BAR domain are 20–50 nm (sharper curvature) (Peter et al, 2004; Takei et al, 1999; Yin et al, 2009), 50–80 nm (intermediate curvature) (Henne et al, 2007), and ~150 nm (Takano et al, 2008) (shallower curvature), respectively.' text='The process of CME begins at the cytoplasmic surface of the plasma membrane with the assembly of clathrin with adaptor proteins and transmembrane cargo molecules (Doherty & McMahon, 2009; Traub, 2009). In parallel with this assembly, membrane curvature is generated to form hemispherical clathrin-coated pits (CCPs). This process leads to deep membrane invaginations. Subsequently, the neck of invaginated pits is constricted and severed to separate the newly formed clathrin-coated vesicles (CCVs) from the plasma membrane. These membrane curvatures are considered to be generated, sensed, and/or maintained by Bin-Amphiphysin-Rsv (BAR) domain superfamily proteins. BAR domains form a crescent-shaped dimer characterized by a specific degree of intrinsic curvature. They bind to negatively charged phospholipids, such as PS, and force membranes to bend according to their intrinsic curvatures, thereby inducing the formation of membrane tubules with specific diameters (curvatures). For example, the diameters of membrane tubules generated by the amphiphysin BAR domain, FCHO2 BAR domain, and FBP17 BAR domain are 20–50 nm (sharper curvature) (Peter et al, 2004; Takei et al, 1999; Yin et al, 2009), 50–80 nm (intermediate curvature) (Henne et al, 2007), and ~150 nm (Takano et al, 2008) (shallower curvature), respectively.'\n",
      "self_ref='#/texts/23' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='The FCHO2 BAR domain has been proposed to trigger the initial invagination for CCP formation (Henne et al, 2010). FCHO2 binds to the flat surface of the plasma membrane at sites where CCPs will form. FCHO2 then generates the initial membrane curvature and recruits the clathrin machinery for CCP formation. Here, we report the role of FCHO2 in Nedd4L-mediated ENaC endocytosis and show that FCHO2 BAR domain-generated membrane curvature induces the recruitment and activation of Nedd4L.' text='The FCHO2 BAR domain has been proposed to trigger the initial invagination for CCP formation (Henne et al, 2010). FCHO2 binds to the flat surface of the plasma membrane at sites where CCPs will form. FCHO2 then generates the initial membrane curvature and recruits the clathrin machinery for CCP formation. Here, we report the role of FCHO2 in Nedd4L-mediated ENaC endocytosis and show that FCHO2 BAR domain-generated membrane curvature induces the recruitment and activation of Nedd4L.'\n",
      "self_ref='#/texts/24' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/25' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Results' text='Results'\n",
      "self_ref='#/texts/26' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/27' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='FCHO2 is required for Nedd4L-mediated ubiquitination and endocytosis of ENaC' text='FCHO2 is required for Nedd4L-mediated ubiquitination and endocytosis of ENaC'\n",
      "self_ref='#/texts/28' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='To study the role of FCHO2 in ENaC endocytosis, we established a HeLa cell line stably expressing all three subunits, α-, β-, and γENaC (αβγENaC-HeLa cells) (Fig EV1A-C). A FLAG tag was introduced into the extracellular region of αENaC in a position previously shown not to affect channel activity (Firsov et al, 1996). Upon FCHO2 knockdown by small interfering RNA (siRNA) methods, ENaC internalization was reduced, as was upon Nedd4L knockdown (Zhou et al, 2007) (Fig 1 A and B). Transferrin receptor (TfR) internalization was reduced upon FCHO2 knockdown but not upon Nedd4L knockdown (Fig 1C). The phenotype of FCHO2 knockdown in ENaC internalization was rescued by an siRNA-resistant (sr) form of FCHO2 (Fig 1D and Appendix Fig S1). Furthermore, αENaC ubiquitination was reduced upon FCHO2 or Nedd4L knockdown but not upon FBP17 knockdown (Fig 1E and Fig EV2). FCHO2 or Nedd4L knockdown increased αENaC expression at the cell surface (Fig 1E and F). These results suggest that FCHO2 is required for Nedd4L-mediated ubiquitination and endocytosis of ENaC.' text='To study the role of FCHO2 in ENaC endocytosis, we established a HeLa cell line stably expressing all three subunits, α-, β-, and γENaC (αβγENaC-HeLa cells) (Fig EV1A-C). A FLAG tag was introduced into the extracellular region of αENaC in a position previously shown not to affect channel activity (Firsov et al, 1996). Upon FCHO2 knockdown by small interfering RNA (siRNA) methods, ENaC internalization was reduced, as was upon Nedd4L knockdown (Zhou et al, 2007) (Fig 1 A and B). Transferrin receptor (TfR) internalization was reduced upon FCHO2 knockdown but not upon Nedd4L knockdown (Fig 1C). The phenotype of FCHO2 knockdown in ENaC internalization was rescued by an siRNA-resistant (sr) form of FCHO2 (Fig 1D and Appendix Fig S1). Furthermore, αENaC ubiquitination was reduced upon FCHO2 or Nedd4L knockdown but not upon FBP17 knockdown (Fig 1E and Fig EV2). FCHO2 or Nedd4L knockdown increased αENaC expression at the cell surface (Fig 1E and F). These results suggest that FCHO2 is required for Nedd4L-mediated ubiquitination and endocytosis of ENaC.'\n",
      "self_ref='#/texts/29' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/30' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Recruitment and activation of Nedd4L by FCHO2 in cells' text='Recruitment and activation of Nedd4L by FCHO2 in cells'\n",
      "self_ref='#/texts/31' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Immunofluorescence microscopy revealed that in αβγENaC-HeLa cells, cell-surface αENaC co-localized with FCHO2 at clathrin-coated structures (Fig 2A). However, its co-localization with Nedd4L was not detected (Appendix Fig S2A), because exogenous expression of Nedd4L-green fluorescent protein (GFP) stimulated ENaC internalization, resulting in the disappearance of αENaC from the cell surface. Cell-surface αENaC was observed in cells lacking Nedd4L-GFP expression (non-transfected cells). When a catalytically inactive Nedd4L mutant, C922A, was expressed, it co-localized with cell-surface αENaC and endogenous FCHO2 (Fig 2B). The mutant also formed FCHO2-negative spots. In wild-type HeLa cells, Nedd4L accumulated at clathrin-coated structures where FCHO2 localized (Fig 2C). Additionally, Nedd4L formed clathrin-negative spots, suggesting that they may be involved in clathrin-independent endocytosis.' text='Immunofluorescence microscopy revealed that in αβγENaC-HeLa cells, cell-surface αENaC co-localized with FCHO2 at clathrin-coated structures (Fig 2A). However, its co-localization with Nedd4L was not detected (Appendix Fig S2A), because exogenous expression of Nedd4L-green fluorescent protein (GFP) stimulated ENaC internalization, resulting in the disappearance of αENaC from the cell surface. Cell-surface αENaC was observed in cells lacking Nedd4L-GFP expression (non-transfected cells). When a catalytically inactive Nedd4L mutant, C922A, was expressed, it co-localized with cell-surface αENaC and endogenous FCHO2 (Fig 2B). The mutant also formed FCHO2-negative spots. In wild-type HeLa cells, Nedd4L accumulated at clathrin-coated structures where FCHO2 localized (Fig 2C). Additionally, Nedd4L formed clathrin-negative spots, suggesting that they may be involved in clathrin-independent endocytosis.'\n",
      "self_ref='#/texts/32' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Using total internal reflection fluorescence (TIRF) microscopy, we investigated whether Nedd4L is recruited to CCPs in wild-type HeLa cells. Similar to FCHO2 described previously (Henne et al, 2010; Lehmann et al, 2019; Zaccai et al, 2022), Nedd4L showed dynamic co-localization with clathrin during pit formation (Fig 3). When both fluorescent protein-tagged FCHO2 and Nedd4L were expressed, the accumulation of Nedd4L at FCHO2-positive spots was not clearly observed (Appendix Fig S2B). The reason behind this remains unknown, but it has been shown that clathrin coat formation, measured in time-lapse images of cells expressing fluorescent proteins, is affected by several factors, such as overexpression level, choice of the proteins, and choice of the fluorescent tag (Aguet et al, 2013). These factors may contribute to the failure to detect the accumulation of Nedd4L at FCHO2-positive spots.' text='Using total internal reflection fluorescence (TIRF) microscopy, we investigated whether Nedd4L is recruited to CCPs in wild-type HeLa cells. Similar to FCHO2 described previously (Henne et al, 2010; Lehmann et al, 2019; Zaccai et al, 2022), Nedd4L showed dynamic co-localization with clathrin during pit formation (Fig 3). When both fluorescent protein-tagged FCHO2 and Nedd4L were expressed, the accumulation of Nedd4L at FCHO2-positive spots was not clearly observed (Appendix Fig S2B). The reason behind this remains unknown, but it has been shown that clathrin coat formation, measured in time-lapse images of cells expressing fluorescent proteins, is affected by several factors, such as overexpression level, choice of the proteins, and choice of the fluorescent tag (Aguet et al, 2013). These factors may contribute to the failure to detect the accumulation of Nedd4L at FCHO2-positive spots.'\n",
      "self_ref='#/texts/33' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='While Nedd4L shows a punctate staining pattern at the cell surface (see Figs 2C, 3B and 3C), it exhibited diffuse distribution throughout the cytoplasm (Fig 4A and B). In COS7 cells, co-expression of the FCHO2 BAR domain [amino acid (aa) 1-302] recruited Nedd4L to membrane tubules (Fig 4A). The Nedd4L C2 domain (Myc-Nedd4L C2, aa 1-160) was also concentrated to membrane tubules, whereas C2 domain-deleted Nedd4L (Myc-Nedd4L ΔC2) was not. In contrast to FCHO2, the FBP17 BAR domain did not recruit Nedd4L to membrane tubules. Additionally, the BAR domain protein amphiphysin1 did not recruit Nedd4L to membrane tubules. Thus, Nedd4L is selectively recruited through the C2 domain to membrane tubules generated by the FCHO2 BAR domain. Similar results were obtained in HEK293 cells (Fig 4B). In these cells, membrane tubules generated by amphiphysin1 were hardly detected. It showed a punctate staining pattern and did not co-localized with Nedd4L.' text='While Nedd4L shows a punctate staining pattern at the cell surface (see Figs 2C, 3B and 3C), it exhibited diffuse distribution throughout the cytoplasm (Fig 4A and B). In COS7 cells, co-expression of the FCHO2 BAR domain [amino acid (aa) 1-302] recruited Nedd4L to membrane tubules (Fig 4A). The Nedd4L C2 domain (Myc-Nedd4L C2, aa 1-160) was also concentrated to membrane tubules, whereas C2 domain-deleted Nedd4L (Myc-Nedd4L ΔC2) was not. In contrast to FCHO2, the FBP17 BAR domain did not recruit Nedd4L to membrane tubules. Additionally, the BAR domain protein amphiphysin1 did not recruit Nedd4L to membrane tubules. Thus, Nedd4L is selectively recruited through the C2 domain to membrane tubules generated by the FCHO2 BAR domain. Similar results were obtained in HEK293 cells (Fig 4B). In these cells, membrane tubules generated by amphiphysin1 were hardly detected. It showed a punctate staining pattern and did not co-localized with Nedd4L.'\n",
      "self_ref='#/texts/34' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='We also investigated whether the FCHO2 BAR domain not only recruits but also activates Nedd4L in cells. For this purpose, we performed an in vivo ubiquitination assay using HEK293 cells, which formed membrane tubules upon FCHO2 expression more efficiently than did COS7 cells: 91 ± 2% of HEK293 cells vs. 32 ± 2% of COS7 cells [the mean ± standard error of the mean (S.E.M.) of four independent experiments]. As the catalytic activity of E3 Ub ligases is typically reflected in their autoubiquitination (Wiesner et al, 2007), we determined activity by measuring the ubiquitination level of Nedd4L itself. Expression of the FCHO2 BAR domain in cells remarkably enhanced Nedd4L autoubiquitination, whereas FBP17 or amphiphysin1 did not (Fig 5A). Tubulation-deficient FCHO2 mutants (K146E+R152E and L136E) (Henne et al, 2010) did not enhance Nedd4L activity (Fig 5A–C and Fig EV3). A catalytically inactive Nedd4L mutant, C922A, which was recruited to FCHO2-generated membrane tubules, showed little enhancement of ubiquitination upon FCHO2 expression (Fig 5D and E), indicating that Nedd4L ubiquitination is self-induced. Using αENaC as a substrate, we further confirmed that Nedd4L is activated upon FCHO2 expression in cells (Fig 5F).' text='We also investigated whether the FCHO2 BAR domain not only recruits but also activates Nedd4L in cells. For this purpose, we performed an in vivo ubiquitination assay using HEK293 cells, which formed membrane tubules upon FCHO2 expression more efficiently than did COS7 cells: 91 ± 2% of HEK293 cells vs. 32 ± 2% of COS7 cells [the mean ± standard error of the mean (S.E.M.) of four independent experiments]. As the catalytic activity of E3 Ub ligases is typically reflected in their autoubiquitination (Wiesner et al, 2007), we determined activity by measuring the ubiquitination level of Nedd4L itself. Expression of the FCHO2 BAR domain in cells remarkably enhanced Nedd4L autoubiquitination, whereas FBP17 or amphiphysin1 did not (Fig 5A). Tubulation-deficient FCHO2 mutants (K146E+R152E and L136E) (Henne et al, 2010) did not enhance Nedd4L activity (Fig 5A–C and Fig EV3). A catalytically inactive Nedd4L mutant, C922A, which was recruited to FCHO2-generated membrane tubules, showed little enhancement of ubiquitination upon FCHO2 expression (Fig 5D and E), indicating that Nedd4L ubiquitination is self-induced. Using αENaC as a substrate, we further confirmed that Nedd4L is activated upon FCHO2 expression in cells (Fig 5F).'\n",
      "self_ref='#/texts/35' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/36' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Preference of Nedd4L for FCHO2-generated membrane curvature' text='Preference of Nedd4L for FCHO2-generated membrane curvature'\n",
      "self_ref='#/texts/37' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='The aforementioned results prompted us to investigate whether FCHO2 and Nedd4L interact through membrane tubules rather than through stereotyped protein–protein interactions. Membrane tubules generated by BAR domains in cells have specific diameters (curvatures) (Henne et al, 2007; Peter et al, 2004; Takano et al, 2008; Takei et al, 1999; Yin et al, 2009). Nedd4L may preferentially bind to the specific degree of membrane curvature generated by FCHO2. To investigate this possibility, we first examined the biochemical properties of Nedd4L. Using a co-sedimentation assay with liposomes containing various percentages of PS, we found that the C2 domain of Nedd4L is a Ca2+-dependent PS-binding domain (Fig 6A and B). Submicromolar Ca2+ concentrations, which are comparable to intracellular levels in unstimulated cells, were sufficient for the Nedd4L C2 domain to bind to PS-containing liposomes. The C2 domain alone and full-length Nedd4L showed different Ca2+ requirements. The higher Ca2+ requirement of full-length Nedd4L may be due to the intramolecular interaction between the C2 and HECT domains. Increasing PS percentages in synthetic liposomes enhanced the liposome binding of the Nedd4L C2 domain, and PS requirements varied depending on Ca2+ concentration.' text='The aforementioned results prompted us to investigate whether FCHO2 and Nedd4L interact through membrane tubules rather than through stereotyped protein–protein interactions. Membrane tubules generated by BAR domains in cells have specific diameters (curvatures) (Henne et al, 2007; Peter et al, 2004; Takano et al, 2008; Takei et al, 1999; Yin et al, 2009). Nedd4L may preferentially bind to the specific degree of membrane curvature generated by FCHO2. To investigate this possibility, we first examined the biochemical properties of Nedd4L. Using a co-sedimentation assay with liposomes containing various percentages of PS, we found that the C2 domain of Nedd4L is a Ca2+-dependent PS-binding domain (Fig 6A and B). Submicromolar Ca2+ concentrations, which are comparable to intracellular levels in unstimulated cells, were sufficient for the Nedd4L C2 domain to bind to PS-containing liposomes. The C2 domain alone and full-length Nedd4L showed different Ca2+ requirements. The higher Ca2+ requirement of full-length Nedd4L may be due to the intramolecular interaction between the C2 and HECT domains. Increasing PS percentages in synthetic liposomes enhanced the liposome binding of the Nedd4L C2 domain, and PS requirements varied depending on Ca2+ concentration.'\n",
      "self_ref='#/texts/38' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='We next examined whether Nedd4L preferentially binds to a specific degree of membrane curvature. For this purpose, liposomes of different sizes (curvatures) were prepared by stepwise extrusion through filter membranes with different pore sizes (0.8, 0.4, 0.2, 0.1, 0.05, and 0.03 µm) and sonication. Sonication or extrusion through a 0.03 µm pore-size filter was used to prepare the smallest liposomes. Transmission electron microscopy revealed that the actual liposome size was similar to the pore size for 0.2, 0.1, 0.05, and 0.03 µm pore-size liposomes, whereas it was significantly smaller for 0.8 and 0.4 µm pore-size liposomes (Fig 7A). Nedd4L showed the strongest binding to 0.05 µm pore-size liposomes (Fig 7B).' text='We next examined whether Nedd4L preferentially binds to a specific degree of membrane curvature. For this purpose, liposomes of different sizes (curvatures) were prepared by stepwise extrusion through filter membranes with different pore sizes (0.8, 0.4, 0.2, 0.1, 0.05, and 0.03 µm) and sonication. Sonication or extrusion through a 0.03 µm pore-size filter was used to prepare the smallest liposomes. Transmission electron microscopy revealed that the actual liposome size was similar to the pore size for 0.2, 0.1, 0.05, and 0.03 µm pore-size liposomes, whereas it was significantly smaller for 0.8 and 0.4 µm pore-size liposomes (Fig 7A). Nedd4L showed the strongest binding to 0.05 µm pore-size liposomes (Fig 7B).'\n",
      "self_ref='#/texts/39' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Synaptotagmin I (SytI) tandem C2 domains can both generate and sense membrane curvature in vitro by penetrating their hydrophobic residues into the lipid bilayer (Martens et al, 2007). As a sensor, the binding affinity is considered to be highest for membranes that have a degree of curvature consistent with that generated by SytI itself. We investigated whether, in addition to sensing membrane curvature, the Nedd4L C2 domain can also generate membrane curvature in vitro and, if so, what diameter of membrane tubules is generated. As expected, transmission electron microscopy revealed that the C2 domain deformed liposomes into extensive tubulation (Fig 7C). It should be noted that Nedd4L does not generate membrane tubules in cells (see Fig 4A and B). The tubule diameter generated by Nedd4L in vitro was mainly 40–80 nm, which is consistent not only with the diameter of 0.05 µm pore-size liposomes, to which Nedd4L preferentially binds, but also with the diameter of membrane tubules generated by FCHO2 (Fig 7D). These results suggest that Nedd4L shows the highest binding affinity for the degree of membrane curvature generated by FCHO2. It is therefore likely that FCHO2-induced recruitment and activation of Nedd4L are mediated by the degree of membrane curvature.' text='Synaptotagmin I (SytI) tandem C2 domains can both generate and sense membrane curvature in vitro by penetrating their hydrophobic residues into the lipid bilayer (Martens et al, 2007). As a sensor, the binding affinity is considered to be highest for membranes that have a degree of curvature consistent with that generated by SytI itself. We investigated whether, in addition to sensing membrane curvature, the Nedd4L C2 domain can also generate membrane curvature in vitro and, if so, what diameter of membrane tubules is generated. As expected, transmission electron microscopy revealed that the C2 domain deformed liposomes into extensive tubulation (Fig 7C). It should be noted that Nedd4L does not generate membrane tubules in cells (see Fig 4A and B). The tubule diameter generated by Nedd4L in vitro was mainly 40–80 nm, which is consistent not only with the diameter of 0.05 µm pore-size liposomes, to which Nedd4L preferentially binds, but also with the diameter of membrane tubules generated by FCHO2 (Fig 7D). These results suggest that Nedd4L shows the highest binding affinity for the degree of membrane curvature generated by FCHO2. It is therefore likely that FCHO2-induced recruitment and activation of Nedd4L are mediated by the degree of membrane curvature.'\n",
      "self_ref='#/texts/40' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/41' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Roles of conserved hydrophobic residues in the Nedd4L C2 domain' text='Roles of conserved hydrophobic residues in the Nedd4L C2 domain'\n",
      "self_ref='#/texts/42' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='We compared the C2 domain sequence of Nedd4L to those of other Nedd4 family members, SytI, and PKCγ (Fig 8A). SytI C2 domains possess hydrophobic residues (M173, F234, V304, and I367) for penetration into the lipid bilayer to generate and sense membrane curvature (Martens et al, 2007). The Nedd4L C2 domain also has three hydrophobic residues (I37, F38, and L99), which are conserved among Nedd4 family members except for WWP2. We mutated these residues to alanine to produce six mutants, I37A, F38A, L99A, I37A+F38A, I37A+L99A, and I37A+F38A+L99A (triple-A). Any single mutation did not inhibit Nedd4L recruitment to FCHO2-generated membrane tubules (Fig 8B). However, double and triple mutations severely impaired this recruitment. Furthermore, the I37A+F38A and triple-A mutants showed little liposome binding and lost curvature generation (Fig 8C and D). Thus, these hydrophobic residues (I37, F38, and L99) are critical for both membrane binding and curvature sensing and generation. These residues are likely inserted into the lipid bilayer, thereby strengthening the C2 domain interaction with membranes and enabling the sensing and generation of membrane curvature. It has been shown that I37 and F38 are directly involved in Ca2+ binding (Escobedo et al, 2014). The residues may be inserted into the lipid bilayer while binding Ca2+.' text='We compared the C2 domain sequence of Nedd4L to those of other Nedd4 family members, SytI, and PKCγ (Fig 8A). SytI C2 domains possess hydrophobic residues (M173, F234, V304, and I367) for penetration into the lipid bilayer to generate and sense membrane curvature (Martens et al, 2007). The Nedd4L C2 domain also has three hydrophobic residues (I37, F38, and L99), which are conserved among Nedd4 family members except for WWP2. We mutated these residues to alanine to produce six mutants, I37A, F38A, L99A, I37A+F38A, I37A+L99A, and I37A+F38A+L99A (triple-A). Any single mutation did not inhibit Nedd4L recruitment to FCHO2-generated membrane tubules (Fig 8B). However, double and triple mutations severely impaired this recruitment. Furthermore, the I37A+F38A and triple-A mutants showed little liposome binding and lost curvature generation (Fig 8C and D). Thus, these hydrophobic residues (I37, F38, and L99) are critical for both membrane binding and curvature sensing and generation. These residues are likely inserted into the lipid bilayer, thereby strengthening the C2 domain interaction with membranes and enabling the sensing and generation of membrane curvature. It has been shown that I37 and F38 are directly involved in Ca2+ binding (Escobedo et al, 2014). The residues may be inserted into the lipid bilayer while binding Ca2+.'\n",
      "self_ref='#/texts/43' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='In the C2 domain of Smurf2, another Nedd4 family member, conserved hydrophobic residues (F29 and F30) are responsible for interactions with the HECT domain (Fig 8A) (Wiesner et al, 2007). Mutation at these residues impairs the interaction between the C2 and HECT domains, resulting in enhanced Smurf2 autoubiquitination by relieving autoinhibition. Consistent with these previous findings, we found that the Nedd4L I37A+F38A mutant, which was not recruited to FCHO2-generated membrane tubules, enhanced autoubiquitination, irrespective of FCHO2 expression (Fig 5D and E). These results suggest that I37 and F38 are critical for interactions with the HECT domain as well as for membrane binding and curvature sensing. Interestingly, the Nedd4L I37A+F38A mutant, as well as the C922A mutant, showed little αENaC ubiquitination (Fig EV4), indicating that membrane binding of Nedd4L is critical for ENaC ubiquitination.' text='In the C2 domain of Smurf2, another Nedd4 family member, conserved hydrophobic residues (F29 and F30) are responsible for interactions with the HECT domain (Fig 8A) (Wiesner et al, 2007). Mutation at these residues impairs the interaction between the C2 and HECT domains, resulting in enhanced Smurf2 autoubiquitination by relieving autoinhibition. Consistent with these previous findings, we found that the Nedd4L I37A+F38A mutant, which was not recruited to FCHO2-generated membrane tubules, enhanced autoubiquitination, irrespective of FCHO2 expression (Fig 5D and E). These results suggest that I37 and F38 are critical for interactions with the HECT domain as well as for membrane binding and curvature sensing. Interestingly, the Nedd4L I37A+F38A mutant, as well as the C922A mutant, showed little αENaC ubiquitination (Fig EV4), indicating that membrane binding of Nedd4L is critical for ENaC ubiquitination.'\n",
      "self_ref='#/texts/44' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/45' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='In vitro activation of Nedd4L by FCHO2-generated membrane curvature' text='In vitro activation of Nedd4L by FCHO2-generated membrane curvature'\n",
      "self_ref='#/texts/46' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='To examine the properties of Nedd4L catalytic activity, we carried out an in vitro ubiquitination assay using an αENaC peptide as a substrate. The αENaC peptide possesses a conserved PY motif for interaction with the Nedd4L WW domain, and it is fused with monomeric streptavidin (mSA) (Lim et al, 2013) to bind to liposomes containing biotinylated phospholipids (biotinylated liposomes) (Sakamoto et al, 2017) (Fig 9A and B). We found that Ca2+ and PS were required for Nedd4L catalytic activity (Fig 9C and D), and that 0.05 µm pore-size liposomes were most effective in stimulating Ca2+- and PS-dependent Nedd4L activity (Fig 9E). Thus, the properties of Nedd4L catalytic activity are similar to those of its liposome binding. Furthermore, we found that Nedd4L did not ubiquitinate PY motif-mutated (Y644A in αENaC) mSA-αENaC (Appendix Fig S3). These findings are consistent with defects in ENaC ubiquitination observed in Liddle syndrome.' text='To examine the properties of Nedd4L catalytic activity, we carried out an in vitro ubiquitination assay using an αENaC peptide as a substrate. The αENaC peptide possesses a conserved PY motif for interaction with the Nedd4L WW domain, and it is fused with monomeric streptavidin (mSA) (Lim et al, 2013) to bind to liposomes containing biotinylated phospholipids (biotinylated liposomes) (Sakamoto et al, 2017) (Fig 9A and B). We found that Ca2+ and PS were required for Nedd4L catalytic activity (Fig 9C and D), and that 0.05 µm pore-size liposomes were most effective in stimulating Ca2+- and PS-dependent Nedd4L activity (Fig 9E). Thus, the properties of Nedd4L catalytic activity are similar to those of its liposome binding. Furthermore, we found that Nedd4L did not ubiquitinate PY motif-mutated (Y644A in αENaC) mSA-αENaC (Appendix Fig S3). These findings are consistent with defects in ENaC ubiquitination observed in Liddle syndrome.'\n",
      "self_ref='#/texts/47' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='We next examined whether membrane localization of mSA-αENaC affects the membrane binding and catalytic activity of Nedd4L. Nedd4L bound only slightly to mSA-αENaC on PS-free (0% PS) liposomes, but the liposome binding was synergistically increased by elevating the PS percentage in liposomes (Fig 9F). By contrast, when free biotin competitively inhibited the binding of mSA-αENaC to biotinylated liposomes, the PS-dependent liposome binding of Nedd4L was significantly reduced, resulting in remarkable inhibition of mSA-αENaC ubiquitination (Fig 9D and F). Thus, membrane localization of the PY motif is critical for membrane binding and activation of Nedd4L. It is, therefore, likely that the PY motifs on membranes, such as ENaC, play a role in enhancing Nedd4L catalytic activity by potentiating membrane binding through interactions with the WW domain.' text='We next examined whether membrane localization of mSA-αENaC affects the membrane binding and catalytic activity of Nedd4L. Nedd4L bound only slightly to mSA-αENaC on PS-free (0% PS) liposomes, but the liposome binding was synergistically increased by elevating the PS percentage in liposomes (Fig 9F). By contrast, when free biotin competitively inhibited the binding of mSA-αENaC to biotinylated liposomes, the PS-dependent liposome binding of Nedd4L was significantly reduced, resulting in remarkable inhibition of mSA-αENaC ubiquitination (Fig 9D and F). Thus, membrane localization of the PY motif is critical for membrane binding and activation of Nedd4L. It is, therefore, likely that the PY motifs on membranes, such as ENaC, play a role in enhancing Nedd4L catalytic activity by potentiating membrane binding through interactions with the WW domain.'\n",
      "self_ref='#/texts/48' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='We then reconstituted in vitro FCHO2-induced recruitment and activation of Nedd4L. When liposomes (20% PS) were used, the FCHO2 BAR domain enhanced the liposome binding of Nedd4L (Fig 10A) and mSA-αENaC ubiquitination (Fig 10B). This enhancement was lost when the strength of interaction of Nedd4L with liposomes was increased by elevating the PS percentage in liposomes (~50% PS) (Fig 10C and Fig EV5).\\u3000In contrast to FCHO2, neither amphiphysin1 nor FBP17 enhanced Nedd4L activity (Fig 10D and Fig EV6A). Similarly, tubulation-deficient FCHO2 mutant (L136E) did not enhance Nedd4L activity (Fig 10E and Fig EV6A). These results are in good agreement with those obtained from in vivo experiments on FCHO2-induced recruitment and activation of Nedd4L. Electron microscopic analysis revealed that mSA-αENaC-associated liposomes were deformed by membrane-sculpting proteins into tubules with diameters similar to those of mSA-αENaC-free liposomes (Fig EV6B and see also Fig 7D). The diameter of membrane tubules generated by FCHO2 is consistent with that of 0.05 µm pore-size liposomes which are most effective in stimulating Nedd4L activity.' text='We then reconstituted in vitro FCHO2-induced recruitment and activation of Nedd4L. When liposomes (20% PS) were used, the FCHO2 BAR domain enhanced the liposome binding of Nedd4L (Fig 10A) and mSA-αENaC ubiquitination (Fig 10B). This enhancement was lost when the strength of interaction of Nedd4L with liposomes was increased by elevating the PS percentage in liposomes (~50% PS) (Fig 10C and Fig EV5).\\u3000In contrast to FCHO2, neither amphiphysin1 nor FBP17 enhanced Nedd4L activity (Fig 10D and Fig EV6A). Similarly, tubulation-deficient FCHO2 mutant (L136E) did not enhance Nedd4L activity (Fig 10E and Fig EV6A). These results are in good agreement with those obtained from in vivo experiments on FCHO2-induced recruitment and activation of Nedd4L. Electron microscopic analysis revealed that mSA-αENaC-associated liposomes were deformed by membrane-sculpting proteins into tubules with diameters similar to those of mSA-αENaC-free liposomes (Fig EV6B and see also Fig 7D). The diameter of membrane tubules generated by FCHO2 is consistent with that of 0.05 µm pore-size liposomes which are most effective in stimulating Nedd4L activity.'\n",
      "self_ref='#/texts/49' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='To confirm whether FCHO2-induced recruitment and activation of Nedd4L are mediated by liposomes (i.e., membrane curvature) rather than by protein–protein interactions, we conducted two sets of experiments. Initially, we employed synthetic liposomes where 20% PS was substituted with 5% PI(4,5)P2. The FCHO2 BAR domain bound to PI(4,5)P2 as well as PS with high affinity, whereas the Nedd4L C2 domain did not (Fig 11A and B). The FCHO2 BAR domain is known to induce the tubulation of liposomes [5% PI(4,5)P2] (Henne et al, 2010). If Nedd4L could interact through protein–protein interactions with the FCHO2 BAR domain binding to liposomes [5% PI(4,5)P2], the BAR domain would facilitate the co-precipitation of Nedd4L in a co-sedimentation assay, thereby stimulating mSA-αENaC ubiquitination. However, the FCHO2 BAR domain did not demonstrate such interactions (Fig 11C and D). Therefore, it is unlikely that Nedd4L interacts with the FCHO2 BAR domain on liposomes through protein–protein interactions. Secondly, we employed 0.05 µm pore-size liposomes that are most effective in stimulating Nedd4L activity. The diameter of 0.05 µm pore-size liposomes is consistent with that of membrane tubules generated by the FCHO2 BAR domain. The BAR domain is, thus, expected not to induce the tubulation of 0.05 µm pore-size liposomes. The FCHO2 BAR domain showed similar binding activity toward 0.8 and 0.05 µm pore-size liposomes (Fig 12A). When 0.8 µm pore-size liposomes were used, the FCHO2 BAR domain enhanced Nedd4L activity (Fig 12B). However, in the case of 0.05 µm pore-size liposomes, curvature-stimulated Nedd4L activity was not potentiated by the addition of the FCHO2 BAR domain. If Nedd4L had interacted with the BAR domain, we would have expected potentiation of Nedd4L activity. These results strongly suggest that FCHO2-induced activation of Nedd4L is mediated by membrane curvature and not by protein–protein interactions.' text='To confirm whether FCHO2-induced recruitment and activation of Nedd4L are mediated by liposomes (i.e., membrane curvature) rather than by protein–protein interactions, we conducted two sets of experiments. Initially, we employed synthetic liposomes where 20% PS was substituted with 5% PI(4,5)P2. The FCHO2 BAR domain bound to PI(4,5)P2 as well as PS with high affinity, whereas the Nedd4L C2 domain did not (Fig 11A and B). The FCHO2 BAR domain is known to induce the tubulation of liposomes [5% PI(4,5)P2] (Henne et al, 2010). If Nedd4L could interact through protein–protein interactions with the FCHO2 BAR domain binding to liposomes [5% PI(4,5)P2], the BAR domain would facilitate the co-precipitation of Nedd4L in a co-sedimentation assay, thereby stimulating mSA-αENaC ubiquitination. However, the FCHO2 BAR domain did not demonstrate such interactions (Fig 11C and D). Therefore, it is unlikely that Nedd4L interacts with the FCHO2 BAR domain on liposomes through protein–protein interactions. Secondly, we employed 0.05 µm pore-size liposomes that are most effective in stimulating Nedd4L activity. The diameter of 0.05 µm pore-size liposomes is consistent with that of membrane tubules generated by the FCHO2 BAR domain. The BAR domain is, thus, expected not to induce the tubulation of 0.05 µm pore-size liposomes. The FCHO2 BAR domain showed similar binding activity toward 0.8 and 0.05 µm pore-size liposomes (Fig 12A). When 0.8 µm pore-size liposomes were used, the FCHO2 BAR domain enhanced Nedd4L activity (Fig 12B). However, in the case of 0.05 µm pore-size liposomes, curvature-stimulated Nedd4L activity was not potentiated by the addition of the FCHO2 BAR domain. If Nedd4L had interacted with the BAR domain, we would have expected potentiation of Nedd4L activity. These results strongly suggest that FCHO2-induced activation of Nedd4L is mediated by membrane curvature and not by protein–protein interactions.'\n",
      "self_ref='#/texts/50' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/51' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Discussion' text='Discussion'\n",
      "self_ref='#/texts/52' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/53' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Nedd4L represents the ancestral Ub ligase with strong similarity to yeast Rsp5 which plays a key role in the trafficking, sorting, and degradation of a large number of proteins (Yang & Kumar, 2010). Consistently, Nedd4L regulates a growing number of substrates, including membrane receptors, transporters, and ion channels. Although Nedd4L exists in a catalytically autoinhibited state due to an intramolecular interaction between the N-terminal C2 and C-terminal HECT domains, little is known about its activation mechanism. In this study, we clearly demonstrated that Nedd4L is activated by FCHO2 F-BAR domain-induced membrane curvature. It has been shown that the activity of synptojanin-1, a phosphoinositide phosphatase, is modulated by membrane curvature (Chang-Ileto et al, 2011). Taken together, it is conceivable that membrane curvature serves as a signal for the activation of various enzymes involved in membrane trafficking.' text='Nedd4L represents the ancestral Ub ligase with strong similarity to yeast Rsp5 which plays a key role in the trafficking, sorting, and degradation of a large number of proteins (Yang & Kumar, 2010). Consistently, Nedd4L regulates a growing number of substrates, including membrane receptors, transporters, and ion channels. Although Nedd4L exists in a catalytically autoinhibited state due to an intramolecular interaction between the N-terminal C2 and C-terminal HECT domains, little is known about its activation mechanism. In this study, we clearly demonstrated that Nedd4L is activated by FCHO2 F-BAR domain-induced membrane curvature. It has been shown that the activity of synptojanin-1, a phosphoinositide phosphatase, is modulated by membrane curvature (Chang-Ileto et al, 2011). Taken together, it is conceivable that membrane curvature serves as a signal for the activation of various enzymes involved in membrane trafficking.'\n",
      "self_ref='#/texts/54' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/55' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Unique Properties of the Nedd4L C2 Domain' text='Unique Properties of the Nedd4L C2 Domain'\n",
      "self_ref='#/texts/56' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Although both sytI and Nedd4L C2 domains are Ca2+-dependent PS-binding domains that can generate and sense membrane curvature, their properties show a number of significant differences. An interesting difference exists in the degrees of membrane curvatures generated by sytI and Nedd4L: the diameters of their membrane tubules are about 15–20 nm and 40–80 nm, respectively (Hui et al, 2009; Martens et al, 2007). This difference is not simply due to that between tandem and single C2 domains, because it has been shown that the sytI C2B domain alone generates membrane tubules with a similar diameter (Hui et al, 2009). SytI and Nedd4L C2 domains must therefore have respective unique structures that determine the specific degrees of membrane curvatures that they sense and generate.' text='Although both sytI and Nedd4L C2 domains are Ca2+-dependent PS-binding domains that can generate and sense membrane curvature, their properties show a number of significant differences. An interesting difference exists in the degrees of membrane curvatures generated by sytI and Nedd4L: the diameters of their membrane tubules are about 15–20 nm and 40–80 nm, respectively (Hui et al, 2009; Martens et al, 2007). This difference is not simply due to that between tandem and single C2 domains, because it has been shown that the sytI C2B domain alone generates membrane tubules with a similar diameter (Hui et al, 2009). SytI and Nedd4L C2 domains must therefore have respective unique structures that determine the specific degrees of membrane curvatures that they sense and generate.'\n",
      "self_ref='#/texts/57' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Another important difference between sytI and Nedd4L C2 domains exists in their Ca2+ requirements. SytI C2 domains require Ca2+ concentrations in the submillimolar range to generate membrane curvature (Hui et al, 2009). In contrast, submicromolar Ca2+ concentrations, which are comparable to intracellular levels in unstimulated cells, are sufficient for the Nedd4L C2 domain to sense membrane curvature. This is consistent with the observation that Nedd4L is recruited and activated in serum-starved cells. The different Ca2+ requirements between sytI and Nedd4L C2 domains are reflected in their functions. SytI C2 domains induce membrane curvature in response to an increase in intracellular Ca2+ concentration, which is essential for Ca2+-triggered membrane fusion (Hui et al, 2009; Martens et al, 2007). Thus, sytI C2 domains serve as a Ca2+ sensor and a curvature generator. In contrast, the Nedd4L C2 domain recognizes a specific degree of curvature without an increase in intracellular Ca2+ concentration. Therefore, the Nedd4L C2 domain serves as a curvature sensor to regulate the localization and activity of the E3 Ub ligase.' text='Another important difference between sytI and Nedd4L C2 domains exists in their Ca2+ requirements. SytI C2 domains require Ca2+ concentrations in the submillimolar range to generate membrane curvature (Hui et al, 2009). In contrast, submicromolar Ca2+ concentrations, which are comparable to intracellular levels in unstimulated cells, are sufficient for the Nedd4L C2 domain to sense membrane curvature. This is consistent with the observation that Nedd4L is recruited and activated in serum-starved cells. The different Ca2+ requirements between sytI and Nedd4L C2 domains are reflected in their functions. SytI C2 domains induce membrane curvature in response to an increase in intracellular Ca2+ concentration, which is essential for Ca2+-triggered membrane fusion (Hui et al, 2009; Martens et al, 2007). Thus, sytI C2 domains serve as a Ca2+ sensor and a curvature generator. In contrast, the Nedd4L C2 domain recognizes a specific degree of curvature without an increase in intracellular Ca2+ concentration. Therefore, the Nedd4L C2 domain serves as a curvature sensor to regulate the localization and activity of the E3 Ub ligase.'\n",
      "self_ref='#/texts/58' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='A striking feature that distinguishes the Nedd4L C2 domain from other C2 domain proteins (except for at least a subset of Nedd4 family members) is that the Nedd4L C2 domain interacts not only with membranes but also with the intramolecular catalytic domain for its autoinhibition. Mutational experiments have revealed that I37 and F38 residues in the C2 domain are critical for interactions with both membranes and the HECT domain, suggesting that these two interactions are competitive. This idea is supported by the observation that the Ca2+ EC50 value of full-length Nedd4L is much higher than that of the C2 domain. The HECT domain in full-length Nedd4L may competitively inhibit the interaction of the C2 domain with membranes. It is likely that membranes displace the HECT domain from the C2 domain when the strength of the C2 domain interaction with membranes is increased by the preferred membrane curvature. This displacement relieves the autoinhibition, resulting in Nedd4L activation. Thus, the membrane binding of Nedd4L is coupled with its activation.' text='A striking feature that distinguishes the Nedd4L C2 domain from other C2 domain proteins (except for at least a subset of Nedd4 family members) is that the Nedd4L C2 domain interacts not only with membranes but also with the intramolecular catalytic domain for its autoinhibition. Mutational experiments have revealed that I37 and F38 residues in the C2 domain are critical for interactions with both membranes and the HECT domain, suggesting that these two interactions are competitive. This idea is supported by the observation that the Ca2+ EC50 value of full-length Nedd4L is much higher than that of the C2 domain. The HECT domain in full-length Nedd4L may competitively inhibit the interaction of the C2 domain with membranes. It is likely that membranes displace the HECT domain from the C2 domain when the strength of the C2 domain interaction with membranes is increased by the preferred membrane curvature. This displacement relieves the autoinhibition, resulting in Nedd4L activation. Thus, the membrane binding of Nedd4L is coupled with its activation.'\n",
      "self_ref='#/texts/59' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/60' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Stimulatory and inhibitory effects of the FCHO2 BAR domain on the liposome binding and catalytic activity of Nedd4L' text='Stimulatory and inhibitory effects of the FCHO2 BAR domain on the liposome binding and catalytic activity of Nedd4L'\n",
      "self_ref='#/texts/61' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='The FCHO2 BAR domain enhances both the liposome binding and catalytic activity of Nedd4L when the interaction strength of Nedd4L with liposomes is weak (20% PS) (Fig 10A and B). Conversely, the BAR domain inhibits the binding and activity of Nedd4L when the interaction of Nedd4L with liposomes is increased by elevating the PS percentage in liposomes (~50% PS) (Fig 10C and Fig EV5). Thus, the FCHO2 BAR domain exerts both stimulatory and inhibitory effects on Nedd4L, depending on the PS percentage in liposomes. However, given that 20% PS is comparable to the concentration in the inner leaflet of the plasma membrane, it is plausible that FCHO2 exhibits only a stimulatory effect on Nedd4L in cells. The mechanism underlying the stimulatory and inhibitory effects in vitro is as follows: when using liposomes (20% PS), Nedd4L hardly binds to liposomes through PS (Fig. 9F and 10A). The addition of the FCHO2 BAR domain enhances the strength of Nedd4L’s interaction with PS by inducing membrane curvature. Consequently, Nedd4L gains a new binding to liposomes through PS, which augments its catalytic activity. In contrast, when using liposomes (~50% PS), Nedd4L binds to liposome through PS. The addition of the FCHO2 BAR domain hinders the PS-mediated liposome binding of Nedd4L, because FCHO2 and Nedd4L are PS-binding proteins and compete for PS binding in liposomes. Consequently, the BAR domain inhibits the catalytic activity of Nedd4L. This is consistent with the results obtained with 0.05 µm pore-size liposomes (Fig 12B). These 0.05 µm pore-size liposomes are comparable to liposomes with a higher PS percentage, as the interaction of Nedd4L with liposomes through PS is increased by membrane curvature. Therefore, when using 0.05 µm pore-size liposomes, the FCHO2 BAR domain inhibits the catalytic activity of Nedd4L.' text='The FCHO2 BAR domain enhances both the liposome binding and catalytic activity of Nedd4L when the interaction strength of Nedd4L with liposomes is weak (20% PS) (Fig 10A and B). Conversely, the BAR domain inhibits the binding and activity of Nedd4L when the interaction of Nedd4L with liposomes is increased by elevating the PS percentage in liposomes (~50% PS) (Fig 10C and Fig EV5). Thus, the FCHO2 BAR domain exerts both stimulatory and inhibitory effects on Nedd4L, depending on the PS percentage in liposomes. However, given that 20% PS is comparable to the concentration in the inner leaflet of the plasma membrane, it is plausible that FCHO2 exhibits only a stimulatory effect on Nedd4L in cells. The mechanism underlying the stimulatory and inhibitory effects in vitro is as follows: when using liposomes (20% PS), Nedd4L hardly binds to liposomes through PS (Fig. 9F and 10A). The addition of the FCHO2 BAR domain enhances the strength of Nedd4L’s interaction with PS by inducing membrane curvature. Consequently, Nedd4L gains a new binding to liposomes through PS, which augments its catalytic activity. In contrast, when using liposomes (~50% PS), Nedd4L binds to liposome through PS. The addition of the FCHO2 BAR domain hinders the PS-mediated liposome binding of Nedd4L, because FCHO2 and Nedd4L are PS-binding proteins and compete for PS binding in liposomes. Consequently, the BAR domain inhibits the catalytic activity of Nedd4L. This is consistent with the results obtained with 0.05 µm pore-size liposomes (Fig 12B). These 0.05 µm pore-size liposomes are comparable to liposomes with a higher PS percentage, as the interaction of Nedd4L with liposomes through PS is increased by membrane curvature. Therefore, when using 0.05 µm pore-size liposomes, the FCHO2 BAR domain inhibits the catalytic activity of Nedd4L.'\n",
      "self_ref='#/texts/62' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Based on a model in which a subset of BAR domain proteins, such as FCHO2 and FBP17, drive liposome tubulation in vitro (Frost et al, 2008; Shimada et al, 2007), the FCHO2 BAR domain is considered to polymerize on the liposome surface into spiral dense coats by lateral and tip-to-tip interactions. When using liposomes (~50% PS), it is conceivable that the FCHO2 BAR domain densely packs the liposomes, leaving little free space for Nedd4L binding. Conversely, when using liposomes (20% PS), it is likely that the liposomes are loosely packed, providing ample surface space for Nedd4L binding.' text='Based on a model in which a subset of BAR domain proteins, such as FCHO2 and FBP17, drive liposome tubulation in vitro (Frost et al, 2008; Shimada et al, 2007), the FCHO2 BAR domain is considered to polymerize on the liposome surface into spiral dense coats by lateral and tip-to-tip interactions. When using liposomes (~50% PS), it is conceivable that the FCHO2 BAR domain densely packs the liposomes, leaving little free space for Nedd4L binding. Conversely, when using liposomes (20% PS), it is likely that the liposomes are loosely packed, providing ample surface space for Nedd4L binding.'\n",
      "self_ref='#/texts/63' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/64' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Model for ENaC Endocytosis' text='Model for ENaC Endocytosis'\n",
      "self_ref='#/texts/65' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Based on the present study, we propose a model for ENaC endocytosis through ubiquitination (Fig 13). First, FCHO2 binds to the plasma membrane and generates a specific degree of membrane curvature. FCHO2 interacts with clathrin adaptor proteins, such as Eps15, and recruits the clathrin machinery for CCP formation (Uezu et al, 2011; Henne et al, 2010). Eps15 has Ub-interacting motifs (Di Fiore et al, 2003). Nedd4L is subsequently recruited to and activated by FCHO2-generated curved membrane at the rim of nascent CCPs that ENaC enters. ENaC is then ubiquitinated and captured by Eps15 which FCHO2 interacts with. Thus, ubiquitinated ENaC is spatially restricted to CCPs and separated from unmodified ENaC. Overall, our findings suggest that in the early stage of CME, a membrane deformation induced by FCHO2 can program, through both the recruitment and activation of Nedd4L, the gathering of a subclass of cargo molecules in CCPs.' text='Based on the present study, we propose a model for ENaC endocytosis through ubiquitination (Fig 13). First, FCHO2 binds to the plasma membrane and generates a specific degree of membrane curvature. FCHO2 interacts with clathrin adaptor proteins, such as Eps15, and recruits the clathrin machinery for CCP formation (Uezu et al, 2011; Henne et al, 2010). Eps15 has Ub-interacting motifs (Di Fiore et al, 2003). Nedd4L is subsequently recruited to and activated by FCHO2-generated curved membrane at the rim of nascent CCPs that ENaC enters. ENaC is then ubiquitinated and captured by Eps15 which FCHO2 interacts with. Thus, ubiquitinated ENaC is spatially restricted to CCPs and separated from unmodified ENaC. Overall, our findings suggest that in the early stage of CME, a membrane deformation induced by FCHO2 can program, through both the recruitment and activation of Nedd4L, the gathering of a subclass of cargo molecules in CCPs.'\n",
      "self_ref='#/texts/66' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/67' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Methods' text='Methods'\n",
      "self_ref='#/texts/68' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/69' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Construction' text='Construction'\n",
      "self_ref='#/texts/70' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Human αENaC, βENaC and γENaC cDNAs were kindly supplied by Dr. H. Kai (Kumamoto University, Kumamoto, Japan) (Sugahara et al, 2009). FLAG-αENaC was constructed by replacing the sequence [amino acid (aa) 169–174, TLVAGS] in the extracellular region with the FLAG sequence (DYKDDDK) as described previously (Firsov et al, 1996). This introduction of the epitope tag was carried out by polymerase chain reaction (PCR). FLAG-αENaC, βENaC, and γENaC were subcloned into pTRE-Tight-Hygro, pCAGI-Puro (Miyahara et al, 2000), and pCAGI-Bst (Umeda et al, 2006), respectively. pTRE-Tight-Hygro was constructed by inserting the hygromycin-resistance gene into pTRE-Tight (Clontech). FLAG-αENaC was also subcloned into pCAGI-Hygro, which was constructed by replacing the puromycin-resistance gene of pCAGI-Puro with the hygromycin-resistance gene. Human Nedd4L (NM_015277, isoform 3) cDNA was obtained by reverse transcription (RT)-PCR and cloned in pCMV-Myc (Miyahara et al, 2000). Nedd4L was also cloned in pCAGI-puro-EGFP and pGEX-6P-hexahistidine (His6). pGEX-6P-His6 was constructed from pGEX-6P (GE Healthcare) to express an N-terminal GST-tagged, C-terminal His6-tagged protein. pCAGI-puro-EGFP was constructed by subcloning EGFP cDNA into pCAGI-puro to express a C-terminal GFP-tagged protein. The Nedd4L C2 domain (C2, aa 1–160) and C2 domain-deleted mutant (ΔC2, aa 150–955) were subcloned into pCMV-Myc and/or pGEX-6P. Mouse FCHO2 (full length, aa 1–809; BAR domain, aa 1–302) and human FBP17 (BAR domain, aa 1–300) were subcloned into pEGFP-C1 and/or pGEX-6P (Uezu et al, 2011; Tsujita et al, 2006). FCHO2 (aa 1–327) were subcloned into pGEX-6P. Human amphiphysin1 cDNA was obtained by RT-PCR and cloned in pEGFP-N3. Amphiphysin1 (BAR domain, aa 1–377) was then subcloned into pGEX-6P. mTagRFP-T-Clathrin-15 was a gift from Michael Davidson (Addgene plasmid # 58005 ; http://n2t.net/addgene:58005 ; RRID:Addgene_58005) (Shaner et al, 2008). Human hemagglutinin (HA)-tagged Ub cDNA in pCGN was kindly donated by Dr. M. Nakao (Kumamoto University, Kumamoto, Japan). Ub cDNA was subcloned into pET3a. Human UbcH7 cDNA was obtained by RT-PCR and cloned in pGEX-6P. pET21d-Ube1 was a gift from Cynthia Wolberger (Addgene plasmid # 34965 ; http://n2t.net/addgene:34965 ; RRID:Addgene_34965) (Berndsen & Wolberger, 2011). pRSET-mSA was a gift from Sheldon Park (Addgene plasmid # 39860 ; http://n2t.net/addgene:39860 ; RRID:Addgene_39860) (Lim et al, 2013). mSA-αENaC constructed in pRSET-mSA to express a fusion protein of the N-terminal (aa 7–40) and C-terminal (aa 625–659) intracellular regions of αENaC (see Fig. 9A). Mutagenesis was performed by PCR using mutated primers and a site-directed mutagenesis kit (Stratagene).' text='Human αENaC, βENaC and γENaC cDNAs were kindly supplied by Dr. H. Kai (Kumamoto University, Kumamoto, Japan) (Sugahara et al, 2009). FLAG-αENaC was constructed by replacing the sequence [amino acid (aa) 169–174, TLVAGS] in the extracellular region with the FLAG sequence (DYKDDDK) as described previously (Firsov et al, 1996). This introduction of the epitope tag was carried out by polymerase chain reaction (PCR). FLAG-αENaC, βENaC, and γENaC were subcloned into pTRE-Tight-Hygro, pCAGI-Puro (Miyahara et al, 2000), and pCAGI-Bst (Umeda et al, 2006), respectively. pTRE-Tight-Hygro was constructed by inserting the hygromycin-resistance gene into pTRE-Tight (Clontech). FLAG-αENaC was also subcloned into pCAGI-Hygro, which was constructed by replacing the puromycin-resistance gene of pCAGI-Puro with the hygromycin-resistance gene. Human Nedd4L (NM_015277, isoform 3) cDNA was obtained by reverse transcription (RT)-PCR and cloned in pCMV-Myc (Miyahara et al, 2000). Nedd4L was also cloned in pCAGI-puro-EGFP and pGEX-6P-hexahistidine (His6). pGEX-6P-His6 was constructed from pGEX-6P (GE Healthcare) to express an N-terminal GST-tagged, C-terminal His6-tagged protein. pCAGI-puro-EGFP was constructed by subcloning EGFP cDNA into pCAGI-puro to express a C-terminal GFP-tagged protein. The Nedd4L C2 domain (C2, aa 1–160) and C2 domain-deleted mutant (ΔC2, aa 150–955) were subcloned into pCMV-Myc and/or pGEX-6P. Mouse FCHO2 (full length, aa 1–809; BAR domain, aa 1–302) and human FBP17 (BAR domain, aa 1–300) were subcloned into pEGFP-C1 and/or pGEX-6P (Uezu et al, 2011; Tsujita et al, 2006). FCHO2 (aa 1–327) were subcloned into pGEX-6P. Human amphiphysin1 cDNA was obtained by RT-PCR and cloned in pEGFP-N3. Amphiphysin1 (BAR domain, aa 1–377) was then subcloned into pGEX-6P. mTagRFP-T-Clathrin-15 was a gift from Michael Davidson (Addgene plasmid # 58005 ; http://n2t.net/addgene:58005 ; RRID:Addgene_58005) (Shaner et al, 2008). Human hemagglutinin (HA)-tagged Ub cDNA in pCGN was kindly donated by Dr. M. Nakao (Kumamoto University, Kumamoto, Japan). Ub cDNA was subcloned into pET3a. Human UbcH7 cDNA was obtained by RT-PCR and cloned in pGEX-6P. pET21d-Ube1 was a gift from Cynthia Wolberger (Addgene plasmid # 34965 ; http://n2t.net/addgene:34965 ; RRID:Addgene_34965) (Berndsen & Wolberger, 2011). pRSET-mSA was a gift from Sheldon Park (Addgene plasmid # 39860 ; http://n2t.net/addgene:39860 ; RRID:Addgene_39860) (Lim et al, 2013). mSA-αENaC constructed in pRSET-mSA to express a fusion protein of the N-terminal (aa 7–40) and C-terminal (aa 625–659) intracellular regions of αENaC (see Fig. 9A). Mutagenesis was performed by PCR using mutated primers and a site-directed mutagenesis kit (Stratagene).'\n",
      "self_ref='#/texts/71' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/72' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Protein expression and purification' text='Protein expression and purification'\n",
      "self_ref='#/texts/73' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Expression and purification of GST-tagged proteins were performed using standard protocols. His6-tagged proteins were purified with Talon metal affinity resin (Clontech) using the manufacturer’s protocol. The GST tag was cleaved off by PreScission protease (GE Healthcare). The GST tag of BAR domains did not inhibit liposome binding or tubulation, whereas the GST tag of Nedd4L inhibited these functions. As FCHO2 (BAR domain, aa 1–302) was easily aggregated by the removal of the GST tag, FCHO2 (aa 1–327) was used instead as a GST-cleaved form of the FCHO2 BAR domain. Full-length Nedd4L (N-terminal GST-tagged, C-terminal His6-tagged) was purified by glutathione–Sepharose (GE Healthcare) column chromatography. The GST tag was cleaved off and the Nedd4L sample was then subjected to Talon metal affinity column chromatography. After the eluate was passed through glutathione–Sepharose beads, the sample was dialyzed against buffer A (20 mM Tris-HCl at pH 7.5, 100 mM KCl, 0.5 mM EDTA, 1 mM DTT, and 20 % sucrose). The Nedd4L sample was stored at −80°C until use. Ub was expressed in E. coli strain BL21 (DE3) pJY2 and purified as previously described (Pickart & Raasi, 2005). His6-tagged mSA-αENaC was expressed in E. coli strain BL21 (DE3) pLysS and purified as previously described (Lim et al, 2013) with slight modifications. Briefly, the inclusion body was isolated and solubilized in buffer B (50 mM Tris-HCl at pH 8.0 and 6 M guanidine hydrochloride). The sample was then subjected to Talon metal affinity column chromatography. The eluate was added drop by drop to ice-cold refolding buffer (50 mM Tris-HCl at pH 8.0, 150 mM NaCl, 0.15 mg/ml D-biotin, 0.2 mg/ml oxidized glutathione, and 1 mg/ml reduced glutathione) under rapid stirring. The sample was then concentrated using a Centriprep centrifugal filter unit (Millipore) and centrifuged to remove aggregates. The supernatant was dialyzed against buffer C (20 mM Tris-HCl at pH 8.0). The mSA-αENaC sample was further purified by Mono Q anion exchange column chromatography as follows. The sample was applied to a Mono Q 5/50 GL column (GE Healthcare) equilibrated with buffer C. Elution was performed with a 50 ml linear gradient of NaCl (0–200 mM) in buffer C. Fractions of 1 ml each were collected. The active fractions (fraction 18–22) were collected and dialyzed against buffer D (10 mM Tris-HCl at pH 7.5, 100 mM KCl, and 0.5 mM EDTA). The mSA-αENaC sample was stored at −80°C until use.' text='Expression and purification of GST-tagged proteins were performed using standard protocols. His6-tagged proteins were purified with Talon metal affinity resin (Clontech) using the manufacturer’s protocol. The GST tag was cleaved off by PreScission protease (GE Healthcare). The GST tag of BAR domains did not inhibit liposome binding or tubulation, whereas the GST tag of Nedd4L inhibited these functions. As FCHO2 (BAR domain, aa 1–302) was easily aggregated by the removal of the GST tag, FCHO2 (aa 1–327) was used instead as a GST-cleaved form of the FCHO2 BAR domain. Full-length Nedd4L (N-terminal GST-tagged, C-terminal His6-tagged) was purified by glutathione–Sepharose (GE Healthcare) column chromatography. The GST tag was cleaved off and the Nedd4L sample was then subjected to Talon metal affinity column chromatography. After the eluate was passed through glutathione–Sepharose beads, the sample was dialyzed against buffer A (20 mM Tris-HCl at pH 7.5, 100 mM KCl, 0.5 mM EDTA, 1 mM DTT, and 20 % sucrose). The Nedd4L sample was stored at −80°C until use. Ub was expressed in E. coli strain BL21 (DE3) pJY2 and purified as previously described (Pickart & Raasi, 2005). His6-tagged mSA-αENaC was expressed in E. coli strain BL21 (DE3) pLysS and purified as previously described (Lim et al, 2013) with slight modifications. Briefly, the inclusion body was isolated and solubilized in buffer B (50 mM Tris-HCl at pH 8.0 and 6 M guanidine hydrochloride). The sample was then subjected to Talon metal affinity column chromatography. The eluate was added drop by drop to ice-cold refolding buffer (50 mM Tris-HCl at pH 8.0, 150 mM NaCl, 0.15 mg/ml D-biotin, 0.2 mg/ml oxidized glutathione, and 1 mg/ml reduced glutathione) under rapid stirring. The sample was then concentrated using a Centriprep centrifugal filter unit (Millipore) and centrifuged to remove aggregates. The supernatant was dialyzed against buffer C (20 mM Tris-HCl at pH 8.0). The mSA-αENaC sample was further purified by Mono Q anion exchange column chromatography as follows. The sample was applied to a Mono Q 5/50 GL column (GE Healthcare) equilibrated with buffer C. Elution was performed with a 50 ml linear gradient of NaCl (0–200 mM) in buffer C. Fractions of 1 ml each were collected. The active fractions (fraction 18–22) were collected and dialyzed against buffer D (10 mM Tris-HCl at pH 7.5, 100 mM KCl, and 0.5 mM EDTA). The mSA-αENaC sample was stored at −80°C until use.'\n",
      "self_ref='#/texts/74' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/75' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Antibodies' text='Antibodies'\n",
      "self_ref='#/texts/76' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Rabbit anti-Nedd4L antibody was raised against GST-Nedd4L C2 (aa 1–160). Rabbit anti-FCHO2 antibody was obtained as previously described (Uezu et al, 2011). The following antibodies were purchased from commercial sources: mouse anti-Myc (9E10) (American Type Culture Collection); mouse anti-FLAG (M2) and mouse anti-α-tubulin (clone DM1A) (Sigma-Aldrich); mouse anti-clathrin heavy chain (BD Biosciences); rabbit anti-GFP (MBL Co.); rabbit anti-αENaC (Calbiochem); rabbit anti-βENaC (Proteintech); rabbit anti-γENaC (Abcam); mouse anti-transferrin receptor H68.4 (Santa Cruz); mouse anti-Ub antibody P4D1 (Santa Cruz); and secondary antibodies conjugated with Alexa Fluor 488, 594, and 647 (Invitrogen).' text='Rabbit anti-Nedd4L antibody was raised against GST-Nedd4L C2 (aa 1–160). Rabbit anti-FCHO2 antibody was obtained as previously described (Uezu et al, 2011). The following antibodies were purchased from commercial sources: mouse anti-Myc (9E10) (American Type Culture Collection); mouse anti-FLAG (M2) and mouse anti-α-tubulin (clone DM1A) (Sigma-Aldrich); mouse anti-clathrin heavy chain (BD Biosciences); rabbit anti-GFP (MBL Co.); rabbit anti-αENaC (Calbiochem); rabbit anti-βENaC (Proteintech); rabbit anti-γENaC (Abcam); mouse anti-transferrin receptor H68.4 (Santa Cruz); mouse anti-Ub antibody P4D1 (Santa Cruz); and secondary antibodies conjugated with Alexa Fluor 488, 594, and 647 (Invitrogen).'\n",
      "self_ref='#/texts/77' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/78' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Cell culture and transfection' text='Cell culture and transfection'\n",
      "self_ref='#/texts/79' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='HeLa, HEK293, and COS7 cells were cultured in Dulbecco’s modified Eagle medium (DMEM) supplemented with 10% fetal calf serum. HEK293 cells were grown on poly-L-lysine-coated dishes. Transfection was performed by using Lipofectamine 2000 (Invitrogen) according to the manufacturer’s protocol.' text='HeLa, HEK293, and COS7 cells were cultured in Dulbecco’s modified Eagle medium (DMEM) supplemented with 10% fetal calf serum. HEK293 cells were grown on poly-L-lysine-coated dishes. Transfection was performed by using Lipofectamine 2000 (Invitrogen) according to the manufacturer’s protocol.'\n",
      "self_ref='#/texts/80' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/81' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Generation of stable HeLa cells expressing all three ENaC subunits' text='Generation of stable HeLa cells expressing all three ENaC subunits'\n",
      "self_ref='#/texts/82' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='HeLa cells were transfected with the pTet-On Advanced vector (Clontech) to develop stable Tet-On cell lines for a doxycycline (Dox)-inducible expression system according to the manufacturer’s protocol. Tet-On cells were selected in 0.5 mg/ml G418. Cloned Tet-On cells were then transfected with pCAGI-Bst-γENaC and selected in 4 µg/ml blasticidin (in the presence of G418). Blasticidin-resistant cells were transfected with pCAGI-Puro-βENaC and selected in 1 µg/ml puromycin (in the presence of G418 and blasticidin). Puromycin-resistant cells were finally transfected with pTRE-Tight-Hygro-FLAG-αENaC and cultured in 0.2 mg/ml hygromycin (in the presence of G418, blasticidin, and puromycin). After selection, cells were cloned and maintained in 0.5 mg/ml G418, 0.2 mg/ml hygromycin, 1 µg/ml puromycin, and 2.5 µg/ml blasticidin. FLAG-tagged αENaC expression was induced by treating cells with 1 µg/ml Dox in the presence of 40 µM amiloride overnight. Amiloride was added to prevent cellular Na+ overload.' text='HeLa cells were transfected with the pTet-On Advanced vector (Clontech) to develop stable Tet-On cell lines for a doxycycline (Dox)-inducible expression system according to the manufacturer’s protocol. Tet-On cells were selected in 0.5 mg/ml G418. Cloned Tet-On cells were then transfected with pCAGI-Bst-γENaC and selected in 4 µg/ml blasticidin (in the presence of G418). Blasticidin-resistant cells were transfected with pCAGI-Puro-βENaC and selected in 1 µg/ml puromycin (in the presence of G418 and blasticidin). Puromycin-resistant cells were finally transfected with pTRE-Tight-Hygro-FLAG-αENaC and cultured in 0.2 mg/ml hygromycin (in the presence of G418, blasticidin, and puromycin). After selection, cells were cloned and maintained in 0.5 mg/ml G418, 0.2 mg/ml hygromycin, 1 µg/ml puromycin, and 2.5 µg/ml blasticidin. FLAG-tagged αENaC expression was induced by treating cells with 1 µg/ml Dox in the presence of 40 µM amiloride overnight. Amiloride was added to prevent cellular Na+ overload.'\n",
      "self_ref='#/texts/83' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/84' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Immunofluorescence microscopy' text='Immunofluorescence microscopy'\n",
      "self_ref='#/texts/85' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig=\"Cells were fixed with 3.7% formaldehyde in phosphate-buffered saline (PBS) at room temperature for 15 min, and then permeabilized with 0.2% Triton X-100 in PBS for 10 min. After blocking with 1% bovine serum albumin (BSA) in PBS, cells were sequentially incubated with primary antibodies and fluorescence-conjugated secondary antibodies at room temperature for 1 h, respectively. Cells were then analyzed with either a fluorescence microscope (BX51; Olympus) or a confocal microscopy system (BX50, Fluoview FV300, and Fluoview FV1200; Olympus). A 60 × oil immersion objective (NA = 1.40, Olympus) was used. All comparable images were acquired under identical parameters. To visualize cell-surface FLAG-αENaC in αβγENaC-HeLa cells, cells were incubated with 5 µg/ml anti-FLAG M2 antibody at 4°C for 1 h to prevent internalization. Cells were then fixed, permeabilized, and stained with a secondary antibody. Cells were also sequentially incubated with primary (anti-FCHO2) and secondary antibodies. To quantify co-localization, the Pearson's correlation coefficient was calculated from three cells using the Fiji (ImageJ) Coloc2 plug-in.\" text=\"Cells were fixed with 3.7% formaldehyde in phosphate-buffered saline (PBS) at room temperature for 15 min, and then permeabilized with 0.2% Triton X-100 in PBS for 10 min. After blocking with 1% bovine serum albumin (BSA) in PBS, cells were sequentially incubated with primary antibodies and fluorescence-conjugated secondary antibodies at room temperature for 1 h, respectively. Cells were then analyzed with either a fluorescence microscope (BX51; Olympus) or a confocal microscopy system (BX50, Fluoview FV300, and Fluoview FV1200; Olympus). A 60 × oil immersion objective (NA = 1.40, Olympus) was used. All comparable images were acquired under identical parameters. To visualize cell-surface FLAG-αENaC in αβγENaC-HeLa cells, cells were incubated with 5 µg/ml anti-FLAG M2 antibody at 4°C for 1 h to prevent internalization. Cells were then fixed, permeabilized, and stained with a secondary antibody. Cells were also sequentially incubated with primary (anti-FCHO2) and secondary antibodies. To quantify co-localization, the Pearson's correlation coefficient was calculated from three cells using the Fiji (ImageJ) Coloc2 plug-in.\"\n",
      "self_ref='#/texts/86' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/87' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='RNA interference' text='RNA interference'\n",
      "self_ref='#/texts/88' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Stealth™ double-stranded RNAs were purchased from Invitrogen. siRNA sequences of human FCHO2, Nedd4L, and FBP17 were 5′-CCA CAG AUC UUA GAG UGG AUU AUA A-3′ (corresponding to nucleotides 2048–2072 relative to the start codon), 5′-GAA GAG UUG CUG GUC UGG CCG UAU U-3′ (corresponding to nucleotides 1778–1802 relative to the start codon), and 5′-CCC ACU UCA UAU GUC GAA GUC UGU U-3′for human FBP17 (corresponding to nucleotides 1804–1828 relative to the start codon), respectively. A double-stranded RNA targeting luciferase (GL-2, 5′-CGU ACG CGG AAU ACU UCG AAA UGU C-3′) was used as a control. HeLa cells were transfected with 20 nM siRNA using Lipofectamine 2000 according to the manufacturer’s protocol. After 24 h, a second transfection was performed, and cells were cultured for 3 days and subjected to various experiments. For rescue experiments, cells were transfected with the intended plasmid at 36 h after the second transfection. After 18 h, cells were fixed and analyzed.' text='Stealth™ double-stranded RNAs were purchased from Invitrogen. siRNA sequences of human FCHO2, Nedd4L, and FBP17 were 5′-CCA CAG AUC UUA GAG UGG AUU AUA A-3′ (corresponding to nucleotides 2048–2072 relative to the start codon), 5′-GAA GAG UUG CUG GUC UGG CCG UAU U-3′ (corresponding to nucleotides 1778–1802 relative to the start codon), and 5′-CCC ACU UCA UAU GUC GAA GUC UGU U-3′for human FBP17 (corresponding to nucleotides 1804–1828 relative to the start codon), respectively. A double-stranded RNA targeting luciferase (GL-2, 5′-CGU ACG CGG AAU ACU UCG AAA UGU C-3′) was used as a control. HeLa cells were transfected with 20 nM siRNA using Lipofectamine 2000 according to the manufacturer’s protocol. After 24 h, a second transfection was performed, and cells were cultured for 3 days and subjected to various experiments. For rescue experiments, cells were transfected with the intended plasmid at 36 h after the second transfection. After 18 h, cells were fixed and analyzed.'\n",
      "self_ref='#/texts/89' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/90' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Quantitative real-time PCR analysis' text='Quantitative real-time PCR analysis'\n",
      "self_ref='#/texts/91' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Total RNA was isolated using ISOGEN (Nippon Gene, 319-90211), and cDNA was synthesized using ReverTra Ace qPCR RT Master Mix with gDNA Remover (Toyobo, FSQ-301), following manufacturer’s instruction. Real time PCR was carried out using Thunderbird Next SYBR (Toyobo, QPX-201) on StepOnePlus Real-Time PCR Systems (Applied Biosystems). The sequences of qPCR primers are 5′-TCT CCC ATC GCT TCA ACG AG-3′ and 5′-GTT GCA CCC AGC TTG AGA GA-3′ for human FBP17; 5′-GTC TCC TCT GAC TTC AAC AGC G-3′ and\\xa05′-ACC ACC CTG TTG CTG TAG CCA A-3′ for\\xa0human GAPDH. The expression of target genes was normalized to GAPDH mRNA levels.' text='Total RNA was isolated using ISOGEN (Nippon Gene, 319-90211), and cDNA was synthesized using ReverTra Ace qPCR RT Master Mix with gDNA Remover (Toyobo, FSQ-301), following manufacturer’s instruction. Real time PCR was carried out using Thunderbird Next SYBR (Toyobo, QPX-201) on StepOnePlus Real-Time PCR Systems (Applied Biosystems). The sequences of qPCR primers are 5′-TCT CCC ATC GCT TCA ACG AG-3′ and 5′-GTT GCA CCC AGC TTG AGA GA-3′ for human FBP17; 5′-GTC TCC TCT GAC TTC AAC AGC G-3′ and\\xa05′-ACC ACC CTG TTG CTG TAG CCA A-3′ for\\xa0human GAPDH. The expression of target genes was normalized to GAPDH mRNA levels.'\n",
      "self_ref='#/texts/92' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/93' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Endocytosis assay' text='Endocytosis assay'\n",
      "self_ref='#/texts/94' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='αβγENaC-HeLa cells at approximately 90% confluence in a 10-cm dish were starved with serum-free medium (DMEM containing 1% BSA, 1 µg/ml Dox, and 40 µM amiloride) at 37°C for 4 h. For ENaC endocytosis, cells were then incubated with 5 µg/ml anti-FLAG M2 in 2 ml of DMEM containing 1% BSA at 4°C for 1 h to prevent internalization. After three washes with ice-cold PBS containing 1 mM MgCl2 and 0.1 mM CaCl2 (PBS-CM), endocytosis was allowed by incubating cells in prewarmed DMEM containing 1% BSA at 37°C for the indicated periods of time. Endocytosis was stopped by placing cells on ice and washing them with ice-cold PBS-CM. Antibody bound to the cell surface but not internalized was removed by acid stripping with 0.2 M acetic acid and 0.5 M NaCl solution (pH 2.5). After washing with ice-cold PBS-CM, internalized ENaC was detected by immunoblotting (IB) or immunofluorescence microscopy. For IB, cells were scraped in 1 ml of lysis buffer A (50 mM Tris-HCl at pH 8.0, 150 mM NaCl, 5 mM EDTA, 1% Triton X-100, 1 mM PMSF, 20 µg/ml leupeptin, and 1 µg/ml pepstatin A). Cells were solubilized at 4°C for 1 h on a rotating wheel, and centrifuged at 20,000 × g at 4°C for 15 min. Protein G-Sepharose beads (GE Healthcare) were added to the supernatant and incubated at 4°C for 3 h. The beads were then thoroughly washed with lysis buffer A and boiled in sodium dodecyl sulfate (SDS) sample buffer. The sample was subjected to SDS-polyacrylamide gel electrophoresis (PAGE), followed by IB with anti-ENaC subunit antibodies. Internalization was expressed as a percentage of the initial amount of cell-surface ENaC subunit determined by incubating cells with anti-FLAG antibody at 4°C for 1 h and lysing them without incubation at 37°C or acid stripping. For immunofluorescence microscopy, cells were transfected with control or FCHO2 siRNA and then with GFP-siRNA-resistant (sr) FCHO2 mutant. After cells were labeled with anti-FLAG antibody on 4°C, they were incubated at 37°C for 10 min to allow ENaC endocytosis. After acid stripping, cells were fixed with 3.7% formaldehyde and permeabilized with 0.2% Triton X-100. The antibody-labeled, internalized αENaC was visualized with a fluorescence-conjugated secondary antibody. The percentage of cells with internalized αENaC among cells expressing GFP-sr-FCHO2 was determined using MetaMorph imaging System software. All images were acquired under identical parameters. For TfR endocytosis, a cell surface biotinylation assay was carried out as previously described (Roberts et al, 2001). Briefly, the cell surface was labeled with Sulfo-NHS-SS-Biotin (Thermo Fisher Scientific) at 4°C. After cells were incubated with ice-cold quenching buffer (50 mM NH4Cl in PBS-CM) at 4°C, they were incubated with serum free medium for 10 min at 37°C. Biotin was removed from the cell surface by incubation with ice-cold MesNa buffer (50 mM MesNa, 100 mM NaCl, and 100 mM Tris-HCl, pH 8.8). Cells were then incubated with iodoacetamide at 4°C. Cell lysates were then incubated with NeutrAvidin beads (Thermo Fisher Scientific). The beads were boiled in SDS sample buffer. Proteins were analyzed by IB with TfR antibody.' text='αβγENaC-HeLa cells at approximately 90% confluence in a 10-cm dish were starved with serum-free medium (DMEM containing 1% BSA, 1 µg/ml Dox, and 40 µM amiloride) at 37°C for 4 h. For ENaC endocytosis, cells were then incubated with 5 µg/ml anti-FLAG M2 in 2 ml of DMEM containing 1% BSA at 4°C for 1 h to prevent internalization. After three washes with ice-cold PBS containing 1 mM MgCl2 and 0.1 mM CaCl2 (PBS-CM), endocytosis was allowed by incubating cells in prewarmed DMEM containing 1% BSA at 37°C for the indicated periods of time. Endocytosis was stopped by placing cells on ice and washing them with ice-cold PBS-CM. Antibody bound to the cell surface but not internalized was removed by acid stripping with 0.2 M acetic acid and 0.5 M NaCl solution (pH 2.5). After washing with ice-cold PBS-CM, internalized ENaC was detected by immunoblotting (IB) or immunofluorescence microscopy. For IB, cells were scraped in 1 ml of lysis buffer A (50 mM Tris-HCl at pH 8.0, 150 mM NaCl, 5 mM EDTA, 1% Triton X-100, 1 mM PMSF, 20 µg/ml leupeptin, and 1 µg/ml pepstatin A). Cells were solubilized at 4°C for 1 h on a rotating wheel, and centrifuged at 20,000 × g at 4°C for 15 min. Protein G-Sepharose beads (GE Healthcare) were added to the supernatant and incubated at 4°C for 3 h. The beads were then thoroughly washed with lysis buffer A and boiled in sodium dodecyl sulfate (SDS) sample buffer. The sample was subjected to SDS-polyacrylamide gel electrophoresis (PAGE), followed by IB with anti-ENaC subunit antibodies. Internalization was expressed as a percentage of the initial amount of cell-surface ENaC subunit determined by incubating cells with anti-FLAG antibody at 4°C for 1 h and lysing them without incubation at 37°C or acid stripping. For immunofluorescence microscopy, cells were transfected with control or FCHO2 siRNA and then with GFP-siRNA-resistant (sr) FCHO2 mutant. After cells were labeled with anti-FLAG antibody on 4°C, they were incubated at 37°C for 10 min to allow ENaC endocytosis. After acid stripping, cells were fixed with 3.7% formaldehyde and permeabilized with 0.2% Triton X-100. The antibody-labeled, internalized αENaC was visualized with a fluorescence-conjugated secondary antibody. The percentage of cells with internalized αENaC among cells expressing GFP-sr-FCHO2 was determined using MetaMorph imaging System software. All images were acquired under identical parameters. For TfR endocytosis, a cell surface biotinylation assay was carried out as previously described (Roberts et al, 2001). Briefly, the cell surface was labeled with Sulfo-NHS-SS-Biotin (Thermo Fisher Scientific) at 4°C. After cells were incubated with ice-cold quenching buffer (50 mM NH4Cl in PBS-CM) at 4°C, they were incubated with serum free medium for 10 min at 37°C. Biotin was removed from the cell surface by incubation with ice-cold MesNa buffer (50 mM MesNa, 100 mM NaCl, and 100 mM Tris-HCl, pH 8.8). Cells were then incubated with iodoacetamide at 4°C. Cell lysates were then incubated with NeutrAvidin beads (Thermo Fisher Scientific). The beads were boiled in SDS sample buffer. Proteins were analyzed by IB with TfR antibody.'\n",
      "self_ref='#/texts/95' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/96' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='ENaC expression and ubiquitination at the cell surface' text='ENaC expression and ubiquitination at the cell surface'\n",
      "self_ref='#/texts/97' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='αβγENaC-HeLa cells at approximately 90% confluence in a 10-cm dish were starved with serum-free medium (DMEM containing 1% BSA, 1 µg/ml Dox, and 40 µM amiloride) at 37°C for 4 h, washed three times with ice-cold PBS, and incubated at 4°C for 45 min with 2 ml of 1 mg/ml EZ-link Sulfo-NHS-SS-Biotin in PBS with gentle shaking. Subsequently, cells were extensively washed with ice-cold Tris-buffered saline, and then scraped into 1 ml of lysis buffer B (50 mM Tris-HCl at pH 7.5, 150 mM NaCl, 5 mM EDTA, 1% Triton X-100, 10 mM N-ethylmaleimide, 1 mM PMSF, 10 µg/ml leupeptin, and 1 µg/ml pepstatin A) containing 40 µM MG132. Cells were solubilized at 4°C for 1 h on a rotating wheel, and centrifuged at 20,000 × g at 4°C for 15 min. The supernatant (4 mg of protein) was incubated at 4°C for 3 h with 10 µg of anti-FLAG M2 antibody. Protein G-Sepharose beads (50 µl of 50% slurry) were then added to the sample, which was incubated at 4°C for 3 h. After beads were thoroughly washed with lysis buffer B containing 40 µM MG132, FLAG-tagged proteins were eluted from the beads with 200 µg/ml FLAG peptide (Sigma-Aldrich) in 0.5 ml of lysis buffer B. After centrifugation, the supernatant was incubated with 25 µl of NeutrAvidin beads at 4°C overnight. After the beads were thoroughly washed with lysis buffer B, bound proteins were eluted by incubating the beads at 4°C for 1 h with 150 µl of lysis buffer B containing 200 mM DTT. After centrifugation, 5 × SDS sample buffer was added to the supernatant, which was then analyzed by IB with anti-FLAG and anti-Ub antibodies. The chemiluminescence intensity was quantified by scanning using ImageJ software (NIH).' text='αβγENaC-HeLa cells at approximately 90% confluence in a 10-cm dish were starved with serum-free medium (DMEM containing 1% BSA, 1 µg/ml Dox, and 40 µM amiloride) at 37°C for 4 h, washed three times with ice-cold PBS, and incubated at 4°C for 45 min with 2 ml of 1 mg/ml EZ-link Sulfo-NHS-SS-Biotin in PBS with gentle shaking. Subsequently, cells were extensively washed with ice-cold Tris-buffered saline, and then scraped into 1 ml of lysis buffer B (50 mM Tris-HCl at pH 7.5, 150 mM NaCl, 5 mM EDTA, 1% Triton X-100, 10 mM N-ethylmaleimide, 1 mM PMSF, 10 µg/ml leupeptin, and 1 µg/ml pepstatin A) containing 40 µM MG132. Cells were solubilized at 4°C for 1 h on a rotating wheel, and centrifuged at 20,000 × g at 4°C for 15 min. The supernatant (4 mg of protein) was incubated at 4°C for 3 h with 10 µg of anti-FLAG M2 antibody. Protein G-Sepharose beads (50 µl of 50% slurry) were then added to the sample, which was incubated at 4°C for 3 h. After beads were thoroughly washed with lysis buffer B containing 40 µM MG132, FLAG-tagged proteins were eluted from the beads with 200 µg/ml FLAG peptide (Sigma-Aldrich) in 0.5 ml of lysis buffer B. After centrifugation, the supernatant was incubated with 25 µl of NeutrAvidin beads at 4°C overnight. After the beads were thoroughly washed with lysis buffer B, bound proteins were eluted by incubating the beads at 4°C for 1 h with 150 µl of lysis buffer B containing 200 mM DTT. After centrifugation, 5 × SDS sample buffer was added to the supernatant, which was then analyzed by IB with anti-FLAG and anti-Ub antibodies. The chemiluminescence intensity was quantified by scanning using ImageJ software (NIH).'\n",
      "self_ref='#/texts/98' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/99' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='TIRF Microscopy' text='TIRF Microscopy'\n",
      "self_ref='#/texts/100' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='For live observation with TIRF microscopy, HeLa cells were grown on a glass-bottomed dish as described previously (Shimada et al, 2007). Cells were observed at 10–12 h after transfection as described previously (Henne et al, 2010). Images were acquired for 5 min at 1 s intervals with a TIRF microscopy system (Olympus), a 100 × oil immersion objective NA = 1.45 (Olympus) and MetaMorph Imaging System Software. All comparable images were acquired under identical parameters.' text='For live observation with TIRF microscopy, HeLa cells were grown on a glass-bottomed dish as described previously (Shimada et al, 2007). Cells were observed at 10–12 h after transfection as described previously (Henne et al, 2010). Images were acquired for 5 min at 1 s intervals with a TIRF microscopy system (Olympus), a 100 × oil immersion objective NA = 1.45 (Olympus) and MetaMorph Imaging System Software. All comparable images were acquired under identical parameters.'\n",
      "self_ref='#/texts/101' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/102' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Liposome preparation' text='Liposome preparation'\n",
      "self_ref='#/texts/103' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Phospholipids were purchased from commercial sources: phosphatidylcholine (PC, 1,2-dioleoyl-sn-glycero-3-phosphocholine), phosphatidylethanolamine (PE, 1,2-dioleoyl-sn-glycero-3-phosphoethanolamine), PS (1,2-dioleoyl-sn-glycero-3-phospho-l-serine), rhodamine-PE [18:1 L-α-PE-N-(lissamine rhodamine B sulfonyl)], PEG2000-biotin-PE (1,2-distearoyl-sn-glycero-3-phosphoethanolamine-N-[biotinyl (polyethyleneglycol)-2000]), porcine brain PS, brain PI(4,5)P2, and porcine brain total lipid extract were obtained from Avanti Polar Lipids; cholesterol and Folch lipids [bovine brain lipids (Folch Fraction 1)] were obtained from Sigma-Aldrich; X-biotin-PE [N-((6-(biotinoyl)amino)hexanoyl)-1,2-dihexadecanoylsn-glycero-3-phosphoethanolamine] was obtained from Thermo Fisher Scientific; and fluorescein-PE [N-(fluorescein-5-thiocarbamoyl)-1,2-dihexadecanoyl-sn-glycero-3-phosphoethanolamine] was obtained from AAT Bioquest Inc.' text='Phospholipids were purchased from commercial sources: phosphatidylcholine (PC, 1,2-dioleoyl-sn-glycero-3-phosphocholine), phosphatidylethanolamine (PE, 1,2-dioleoyl-sn-glycero-3-phosphoethanolamine), PS (1,2-dioleoyl-sn-glycero-3-phospho-l-serine), rhodamine-PE [18:1 L-α-PE-N-(lissamine rhodamine B sulfonyl)], PEG2000-biotin-PE (1,2-distearoyl-sn-glycero-3-phosphoethanolamine-N-[biotinyl (polyethyleneglycol)-2000]), porcine brain PS, brain PI(4,5)P2, and porcine brain total lipid extract were obtained from Avanti Polar Lipids; cholesterol and Folch lipids [bovine brain lipids (Folch Fraction 1)] were obtained from Sigma-Aldrich; X-biotin-PE [N-((6-(biotinoyl)amino)hexanoyl)-1,2-dihexadecanoylsn-glycero-3-phosphoethanolamine] was obtained from Thermo Fisher Scientific; and fluorescein-PE [N-(fluorescein-5-thiocarbamoyl)-1,2-dihexadecanoyl-sn-glycero-3-phosphoethanolamine] was obtained from AAT Bioquest Inc.'\n",
      "self_ref='#/texts/104' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Liposomes were prepared using brain lipids (Folch lipids, ~50% PS of total lipids), brain lipids (20% PS of total lipids, a mixture of 89.5% porcine brain total lipid extract and 10.5% porcine brain PS), or a synthetic lipid mixture [70% PC, 20% PE, 10% cholesterol (w/w), and varying percentages of PS or PI(4,5)P2 (with a corresponding reduction in PC)]. Lipid concentrations were monitored by including 1% rhodamine-PE (w/w). To prepare mSA-αENaC–associated liposomes, 2% PEG2000-biotin-PE (w/w) was included. Lipids were dried under nitrogen gas, desiccated for 2 h, and resuspended at 1 mg/ml in lipid buffer (50 mM HEPES-NaOH at pH 7.2, 100 mM KCl, 2 mM MgCl2, and 5 mM EGTA), followed by hydration at 37°C for 1 h. To make liposomes of different sizes, they were subjected to five freeze–thaw cycles and centrifuged at 100,000 × g for 15 min. Precipitated liposomes were then resuspended in the same buffer to make a final concentration of 1 mg/ml. They were subjected to extrusion through polycarbonate filter membranes (Avanti Polar Lipids): liposomes were sequentially passed 9 times through a 0.8-µm filter, 21 times through a 0.4-µm filter, 21 times through a 0.2-µm filter, 21 times through a 0.1-µm filter, 21 times through a 0.05-µm filter, and 21 times through a 0.03-µm filter. Instead of extrusion through a 0.03-µm filter, liposomes with the smallest size were also prepared by sonication three times for 5 s with a Branson Sonifier 25 at power level 1. Liposome diameters were examined by transmission electron microscopy.' text='Liposomes were prepared using brain lipids (Folch lipids, ~50% PS of total lipids), brain lipids (20% PS of total lipids, a mixture of 89.5% porcine brain total lipid extract and 10.5% porcine brain PS), or a synthetic lipid mixture [70% PC, 20% PE, 10% cholesterol (w/w), and varying percentages of PS or PI(4,5)P2 (with a corresponding reduction in PC)]. Lipid concentrations were monitored by including 1% rhodamine-PE (w/w). To prepare mSA-αENaC–associated liposomes, 2% PEG2000-biotin-PE (w/w) was included. Lipids were dried under nitrogen gas, desiccated for 2 h, and resuspended at 1 mg/ml in lipid buffer (50 mM HEPES-NaOH at pH 7.2, 100 mM KCl, 2 mM MgCl2, and 5 mM EGTA), followed by hydration at 37°C for 1 h. To make liposomes of different sizes, they were subjected to five freeze–thaw cycles and centrifuged at 100,000 × g for 15 min. Precipitated liposomes were then resuspended in the same buffer to make a final concentration of 1 mg/ml. They were subjected to extrusion through polycarbonate filter membranes (Avanti Polar Lipids): liposomes were sequentially passed 9 times through a 0.8-µm filter, 21 times through a 0.4-µm filter, 21 times through a 0.2-µm filter, 21 times through a 0.1-µm filter, 21 times through a 0.05-µm filter, and 21 times through a 0.03-µm filter. Instead of extrusion through a 0.03-µm filter, liposomes with the smallest size were also prepared by sonication three times for 5 s with a Branson Sonifier 25 at power level 1. Liposome diameters were examined by transmission electron microscopy.'\n",
      "self_ref='#/texts/105' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/106' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Co-sedimentation assay' text='Co-sedimentation assay'\n",
      "self_ref='#/texts/107' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='A co-sedimentation assay was performed as described (Uezu et al, 2011) with slight modifications. Briefly, 5–10 µg of protein was incubated at 25°C for 10 min with 0.5 mg/ml liposomes in 100 µl of lipid buffer containing various concentrations of CaCl2 and centrifuged at 165,000 × g at 25°C for 15 min. Equal amounts of the supernatants and pellets were subjected to SDS-PAGE, followed by Coomassie brilliant blue (CBB) staining. Proteins were quantified by scanning using ImageJ (NIH). The assay with liposomes of different sizes was performed using NeutrAvidin (Thermo Fisher Scientific Inc) as described (Sakamoto et al, 2017). Briefly, brain-lipid (Folch-lipid) liposomes (~50% PS) were supplemented with 1% X-biotin-PE and 1% rhodamine-PE, subjected to extrusion through filter membranes, and then sonicated. Purified Nedd4L protein (10 µg) was incubated with 0.2 mg/ml liposomes in 100 µl of lipid buffer containing 2mM CaCl2 (0.1 µM Ca2+). The sample was then incubated with 2 µg of NeutrAvidin and ultracentrifuged. Both total and supernatant fluorescence were measured. The proportion of precipitated liposomes of each size was ~100%.' text='A co-sedimentation assay was performed as described (Uezu et al, 2011) with slight modifications. Briefly, 5–10 µg of protein was incubated at 25°C for 10 min with 0.5 mg/ml liposomes in 100 µl of lipid buffer containing various concentrations of CaCl2 and centrifuged at 165,000 × g at 25°C for 15 min. Equal amounts of the supernatants and pellets were subjected to SDS-PAGE, followed by Coomassie brilliant blue (CBB) staining. Proteins were quantified by scanning using ImageJ (NIH). The assay with liposomes of different sizes was performed using NeutrAvidin (Thermo Fisher Scientific Inc) as described (Sakamoto et al, 2017). Briefly, brain-lipid (Folch-lipid) liposomes (~50% PS) were supplemented with 1% X-biotin-PE and 1% rhodamine-PE, subjected to extrusion through filter membranes, and then sonicated. Purified Nedd4L protein (10 µg) was incubated with 0.2 mg/ml liposomes in 100 µl of lipid buffer containing 2mM CaCl2 (0.1 µM Ca2+). The sample was then incubated with 2 µg of NeutrAvidin and ultracentrifuged. Both total and supernatant fluorescence were measured. The proportion of precipitated liposomes of each size was ~100%.'\n",
      "self_ref='#/texts/108' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='The assay with mSA-αENaC–associated liposomes was performed as follows. Liposomes were prepared from a synthetic lipid mixture [various percentages of PS or 5% PI(4,5)P2] or brain lipids supplemented with 2% PEG2000-biotin-PE and 1% rhodamine-PE. The liposomes (10 µg) were incubated with purified mSA-αENaC protein (35 pmol) at 4°C for 10 min in 25 µl of lipid buffer. The sample was then incubated at 25°C for 10 min with 0.07 µM (8 µg/ml) Nedd4L and/or 1.4 µM BAR domains in 50 µl of ubiquitination buffer (50 mM HEPES-NaOH at pH 7.2, 100 mM KCl, 2 mM MgCl2, 5 mM EGTA, and 0.1 mM DTT) containing 4 mM CaCl2 (0.7 µM Ca2+). Subsequently, the sample was subjected to ultracentrifugation. Equal amounts of the total samples and pellets were subjected to SDS-PAGE, followed by CBB staining and IB. The chemiluminescence intensity was quantified by scanning using ImageJ (NIH).' text='The assay with mSA-αENaC–associated liposomes was performed as follows. Liposomes were prepared from a synthetic lipid mixture [various percentages of PS or 5% PI(4,5)P2] or brain lipids supplemented with 2% PEG2000-biotin-PE and 1% rhodamine-PE. The liposomes (10 µg) were incubated with purified mSA-αENaC protein (35 pmol) at 4°C for 10 min in 25 µl of lipid buffer. The sample was then incubated at 25°C for 10 min with 0.07 µM (8 µg/ml) Nedd4L and/or 1.4 µM BAR domains in 50 µl of ubiquitination buffer (50 mM HEPES-NaOH at pH 7.2, 100 mM KCl, 2 mM MgCl2, 5 mM EGTA, and 0.1 mM DTT) containing 4 mM CaCl2 (0.7 µM Ca2+). Subsequently, the sample was subjected to ultracentrifugation. Equal amounts of the total samples and pellets were subjected to SDS-PAGE, followed by CBB staining and IB. The chemiluminescence intensity was quantified by scanning using ImageJ (NIH).'\n",
      "self_ref='#/texts/109' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='The assay with mSA-αENaC-associated liposomes (0.05 µm pore-size) was done as follows. Liposomes were prepared from brain lipids (20% PS) supplemented with 2% PEG2000-biotin-PE, 1% rhodamine-PE, and 0.5% fluorescein-PE (w/w) and extruded through membrane filters. To precipitate liposomes, the sample (50 μl) was first incubated with 1 µg of anti-fluorescein goat antibody (Novus Biologicals) at room temperature for 1 h and then incubated with 0.5 µg of protein A/G/L (BioVision) at room temperature for 1 h. Subsequently, the sample was centrifuged at 165,000 × g at 25°C for 15 min. Equal amounts of the supernatants and pellets were subjected to SDS-PAGE, followed by CBB staining and IB. The percentage of precipitated liposomes was ~100%.' text='The assay with mSA-αENaC-associated liposomes (0.05 µm pore-size) was done as follows. Liposomes were prepared from brain lipids (20% PS) supplemented with 2% PEG2000-biotin-PE, 1% rhodamine-PE, and 0.5% fluorescein-PE (w/w) and extruded through membrane filters. To precipitate liposomes, the sample (50 μl) was first incubated with 1 µg of anti-fluorescein goat antibody (Novus Biologicals) at room temperature for 1 h and then incubated with 0.5 µg of protein A/G/L (BioVision) at room temperature for 1 h. Subsequently, the sample was centrifuged at 165,000 × g at 25°C for 15 min. Equal amounts of the supernatants and pellets were subjected to SDS-PAGE, followed by CBB staining and IB. The percentage of precipitated liposomes was ~100%.'\n",
      "self_ref='#/texts/110' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/111' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Calculation of free Ca2+ concentrations' text='Calculation of free Ca2+ concentrations'\n",
      "self_ref='#/texts/112' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='At 1.0, 2.0, 3.0, 3.5, 4.0, 4.5, and 4.9 mM CaCl2 in lipid buffer, free Ca2+ concentrations were calculated to be 0.04, 0.1, 0.3, 0.4, 0.7, 1.6, and 8.1 µM, respectively, as determined by the Ca-Mg-ATP-EGTA Calculator v1.0 using constants from the NIST database #46 v8. At 1.0, 2.0, 3.0, 3.5, 4.0, 4.5, and 4.9 mM CaCl2 in ubiquitination buffer, free Ca2+ concentrations were calculated to be 0.04, 0.1, 0.2, 0.4, 0.7, 1.4, and 6.7 µM, respectively.' text='At 1.0, 2.0, 3.0, 3.5, 4.0, 4.5, and 4.9 mM CaCl2 in lipid buffer, free Ca2+ concentrations were calculated to be 0.04, 0.1, 0.3, 0.4, 0.7, 1.6, and 8.1 µM, respectively, as determined by the Ca-Mg-ATP-EGTA Calculator v1.0 using constants from the NIST database #46 v8. At 1.0, 2.0, 3.0, 3.5, 4.0, 4.5, and 4.9 mM CaCl2 in ubiquitination buffer, free Ca2+ concentrations were calculated to be 0.04, 0.1, 0.2, 0.4, 0.7, 1.4, and 6.7 µM, respectively.'\n",
      "self_ref='#/texts/113' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/114' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='In vitro tubulation assay' text='In vitro tubulation assay'\n",
      "self_ref='#/texts/115' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='An in vitro tubulation assay was performed as described (Uezu et al, 2011; Tsujita et al, 2006) with slight modifications. Briefly, liposomes were prepared from brain lipids (Folch lipids) supplemented with 1 % rhodamine-PE. The liposomes (0.2 mg/ml liposomes) were incubated with the Nedd4L C2 domain (4.5 µM) or BAR domains (1.4 µM) at 25°C for 10 min in lipid buffer containing 3 mM CaCl2 (0.3 µM Ca2+). The sample was examined by fluorescence or electron microscopy. The tubulation assay with mSA-αENaC-associated liposomes was performed using the sample mixture utilized in the co-sedimentation assay in which mSA-αENaC-associated liposomes (brain-lipid liposomes) were used.' text='An in vitro tubulation assay was performed as described (Uezu et al, 2011; Tsujita et al, 2006) with slight modifications. Briefly, liposomes were prepared from brain lipids (Folch lipids) supplemented with 1 % rhodamine-PE. The liposomes (0.2 mg/ml liposomes) were incubated with the Nedd4L C2 domain (4.5 µM) or BAR domains (1.4 µM) at 25°C for 10 min in lipid buffer containing 3 mM CaCl2 (0.3 µM Ca2+). The sample was examined by fluorescence or electron microscopy. The tubulation assay with mSA-αENaC-associated liposomes was performed using the sample mixture utilized in the co-sedimentation assay in which mSA-αENaC-associated liposomes (brain-lipid liposomes) were used.'\n",
      "self_ref='#/texts/116' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/117' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Negative-staining electron microscopy' text='Negative-staining electron microscopy'\n",
      "self_ref='#/texts/118' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Negative-staining transmission electron microscopy was performed as described (Shimada et al, 2007). Box and whisker plots were used to represent the distribution of liposome diameters.' text='Negative-staining transmission electron microscopy was performed as described (Shimada et al, 2007). Box and whisker plots were used to represent the distribution of liposome diameters.'\n",
      "self_ref='#/texts/119' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/120' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='In vivo ubiquitination assay' text='In vivo ubiquitination assay'\n",
      "self_ref='#/texts/121' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='HEK293 cells were used, because they formed membrane tubules upon FCHO2 expression more efficiently than did COS7 cells. HEK293 cells at approximately 90% confluence on a 6-well plate were transfected with 0.8 µg of pCMV-Myc-Nedd4L, 1.6 µg of pEGFP-BAR domain, and 1.6 µg of pCGN-HA-Ub. In some experiments, cells were transfected with various doses (0–1.5 µg) of pCMV-Myc-Nedd4L, 1.5 µg of pEGFP-FCHO2 BAR domain, 1 µg of pCGN-HA-Ub, and 0.5 µg of pCAGI-Hygro-FLAG-αENaC. Total transfected cDNA amount was held constant using empty pCMV-Myc vector. Amiloride (40 µM) was added to the medium when αENaC cDNA was transfected. At 21 h after transfection, cells were washed with PBS and incubated with serum-free DMEM containing 1% BSA. After 3 h incubation, 10 µM MG132 was added to the medium. After further 4 h incubation, cells were solubilized in 150 µl of lysis buffer C (50 mM Tris-HCl at pH 7.5, 150 mM NaCl, 5 mM EDTA, 10 mM N-ethylmaleimide, 1 mM PMSF, 10 µg/ml leupeptin, and 1µg/ml pepstatin A) containing 1% SDS. After boiling, the sample was sonicated and then centrifuged at 20,000 × g for 15 min. The supernatant was diluted 10-fold with lysis buffer C containing 1% Triton X-100 and then incubated at 4°C for 3 h with either anti-Myc or anti-FLAG M2 antibody. Protein G-Sepharose beads were added to the sample, which was then incubated at 4°C for 3 h. After the beads were thoroughly washed with lysis buffer C containing 1% Triton X-100, bound proteins were eluted by boiling in SDS sample buffer for 5 min. These proteins were then subjected to SDS-PAGE, followed by IB.' text='HEK293 cells were used, because they formed membrane tubules upon FCHO2 expression more efficiently than did COS7 cells. HEK293 cells at approximately 90% confluence on a 6-well plate were transfected with 0.8 µg of pCMV-Myc-Nedd4L, 1.6 µg of pEGFP-BAR domain, and 1.6 µg of pCGN-HA-Ub. In some experiments, cells were transfected with various doses (0–1.5 µg) of pCMV-Myc-Nedd4L, 1.5 µg of pEGFP-FCHO2 BAR domain, 1 µg of pCGN-HA-Ub, and 0.5 µg of pCAGI-Hygro-FLAG-αENaC. Total transfected cDNA amount was held constant using empty pCMV-Myc vector. Amiloride (40 µM) was added to the medium when αENaC cDNA was transfected. At 21 h after transfection, cells were washed with PBS and incubated with serum-free DMEM containing 1% BSA. After 3 h incubation, 10 µM MG132 was added to the medium. After further 4 h incubation, cells were solubilized in 150 µl of lysis buffer C (50 mM Tris-HCl at pH 7.5, 150 mM NaCl, 5 mM EDTA, 10 mM N-ethylmaleimide, 1 mM PMSF, 10 µg/ml leupeptin, and 1µg/ml pepstatin A) containing 1% SDS. After boiling, the sample was sonicated and then centrifuged at 20,000 × g for 15 min. The supernatant was diluted 10-fold with lysis buffer C containing 1% Triton X-100 and then incubated at 4°C for 3 h with either anti-Myc or anti-FLAG M2 antibody. Protein G-Sepharose beads were added to the sample, which was then incubated at 4°C for 3 h. After the beads were thoroughly washed with lysis buffer C containing 1% Triton X-100, bound proteins were eluted by boiling in SDS sample buffer for 5 min. These proteins were then subjected to SDS-PAGE, followed by IB.'\n",
      "self_ref='#/texts/122' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/123' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='In vitro ubiquitination assay' text='In vitro ubiquitination assay'\n",
      "self_ref='#/texts/124' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='mSA-αENaC-associated liposomes were made by incubating liposomes (10 µg) with mSA-αENaC (35 pmol) in 25 µl of lipid buffer, as described for the co-sedimentation assay. They were then incubated at 25°C for 10 min with 0.07 µM Nedd4L, 0.04 µM E1 (Ube1), 0.7 µM E2 (UbcH7), 7 µM Ub, and various concentrations of BAR domains in 50 µl of ubiquitination buffer containing 4 mM CaCl2 (0.7 µM Ca2+) and 2 mM ATP. Where indicated, various concentrations of CaCl2 were used. The ubiquitination reaction was started by adding ATP and stopped by adding 10 µl of 6 × SDS sample. The sample was then boiled, subjected to SDS-PAGE, and immunoblotted with anti-Ub and anti-αENaC antibodies. The chemiluminescence intensity was quantified by scanning using ImageJ (NIH).' text='mSA-αENaC-associated liposomes were made by incubating liposomes (10 µg) with mSA-αENaC (35 pmol) in 25 µl of lipid buffer, as described for the co-sedimentation assay. They were then incubated at 25°C for 10 min with 0.07 µM Nedd4L, 0.04 µM E1 (Ube1), 0.7 µM E2 (UbcH7), 7 µM Ub, and various concentrations of BAR domains in 50 µl of ubiquitination buffer containing 4 mM CaCl2 (0.7 µM Ca2+) and 2 mM ATP. Where indicated, various concentrations of CaCl2 were used. The ubiquitination reaction was started by adding ATP and stopped by adding 10 µl of 6 × SDS sample. The sample was then boiled, subjected to SDS-PAGE, and immunoblotted with anti-Ub and anti-αENaC antibodies. The chemiluminescence intensity was quantified by scanning using ImageJ (NIH).'\n",
      "self_ref='#/texts/125' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/126' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Other procedures' text='Other procedures'\n",
      "self_ref='#/texts/127' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Stripping immunoblots for re-probing was carried out with the WB Stripping Solution Strong (Nacalai Tesque) according to the manufacturer’s protocol. Protein concentrations were quantified with BSA as a reference protein by the Bradford method (Bio-Rad) or the BCA method (Thermo Fisher Scientific). SDS-PAGE was performed as described (Laemmli, 1970).' text='Stripping immunoblots for re-probing was carried out with the WB Stripping Solution Strong (Nacalai Tesque) according to the manufacturer’s protocol. Protein concentrations were quantified with BSA as a reference protein by the Bradford method (Bio-Rad) or the BCA method (Thermo Fisher Scientific). SDS-PAGE was performed as described (Laemmli, 1970).'\n",
      "self_ref='#/texts/128' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/129' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Statistical analysis' text='Statistical analysis'\n",
      "self_ref='#/texts/130' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='All experiments were performed at least three times (biological replication) and representative results are shown. Data are shown as the mean ± S.E.M. Student’s tests, one-way analysis of variance test, and two-way analysis of variance test were used to evaluate statistical significances between different treatment groups.' text='All experiments were performed at least three times (biological replication) and representative results are shown. Data are shown as the mean ± S.E.M. Student’s tests, one-way analysis of variance test, and two-way analysis of variance test were used to evaluate statistical significances between different treatment groups.'\n",
      "self_ref='#/texts/131' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/132' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Data availability' text='Data availability'\n",
      "self_ref='#/texts/133' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='This study includes no data deposited in external repositories.' text='This study includes no data deposited in external repositories.'\n",
      "self_ref='#/texts/134' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/135' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Acknowledgments' text='Acknowledgments'\n",
      "self_ref='#/texts/136' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/137' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='We thank Drs. H. Kai and M. Nakao for kindly providing us with ENaC and Ub cDNAs, respectively. We are also grateful to Dr. T. Takenawa (Kobe University, Kobe, Japan) for helpful discussions. This work was partly carried out at the Institute of Molecular Embryology and Genetics, the Gene Technology Centre, and Research Facilities of the School of Medicine, Kumamoto University. This study was supported by JSPS KAKENHI Grant Numbers 19K06643 (to H.N.), 19K06544 (to Y.S.), and Setsuro Fujii Memorial, The Osaka Foundation for Promotion of Fundamental Medical Research (to Y.S.) and by grants from the Mochida Memorial Foundation, the Takeda Science Foundation, and the Uehara Memorial Foundation (to K.K.).' text='We thank Drs. H. Kai and M. Nakao for kindly providing us with ENaC and Ub cDNAs, respectively. We are also grateful to Dr. T. Takenawa (Kobe University, Kobe, Japan) for helpful discussions. This work was partly carried out at the Institute of Molecular Embryology and Genetics, the Gene Technology Centre, and Research Facilities of the School of Medicine, Kumamoto University. This study was supported by JSPS KAKENHI Grant Numbers 19K06643 (to H.N.), 19K06544 (to Y.S.), and Setsuro Fujii Memorial, The Osaka Foundation for Promotion of Fundamental Medical Research (to Y.S.) and by grants from the Mochida Memorial Foundation, the Takeda Science Foundation, and the Uehara Memorial Foundation (to K.K.).'\n",
      "self_ref='#/texts/138' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/139' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Disclosure and Competing Interests Statement' text='Disclosure and Competing Interests Statement'\n",
      "self_ref='#/texts/140' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='The authors declare no competing interests.' text='The authors declare no competing interests.'\n",
      "self_ref='#/texts/141' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/142' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/143' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='References' text='References'\n",
      "self_ref='#/texts/144' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/145' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Aguet F, Antonescu CN, Mettlen M, Schmid S & Danuser G (2013) Advances in analysis of low signal-to-noise images link dynamin and AP2 to the function of an endocytic checkpoint. Dev Cell 26: 279–291' text='Aguet F, Antonescu CN, Mettlen M, Schmid S & Danuser G (2013) Advances in analysis of low signal-to-noise images link dynamin and AP2 to the function of an endocytic checkpoint. Dev Cell 26: 279–291'\n",
      "self_ref='#/texts/146' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Berndsen CE & Wolberger C (2011) A spectrophotometric assay for conjugation of ubiquitin and ubiquitin-like proteins. Anal Biochem 418: 102–110' text='Berndsen CE & Wolberger C (2011) A spectrophotometric assay for conjugation of ubiquitin and ubiquitin-like proteins. Anal Biochem 418: 102–110'\n",
      "self_ref='#/texts/147' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Butterworth MB (2010) Regulation of the epithelial sodium channel (ENaC) by membrane trafficking. Biochim Biophys Acta 1802: 1166–77' text='Butterworth MB (2010) Regulation of the epithelial sodium channel (ENaC) by membrane trafficking. Biochim Biophys Acta 1802: 1166–77'\n",
      "self_ref='#/texts/148' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Chang-Ileto B, Frere SG, Chan RB, Voronov SV, Roux A, & Di Paolo G (2011) Synaptojanin 1-mediated PI(4,5)P2 hydrolysis is modulated by membrane curvature and facilitates membrane fission. Dev. Cell 20: 206–218' text='Chang-Ileto B, Frere SG, Chan RB, Voronov SV, Roux A, & Di Paolo G (2011) Synaptojanin 1-mediated PI(4,5)P2 hydrolysis is modulated by membrane curvature and facilitates membrane fission. Dev. Cell 20: 206–218'\n",
      "self_ref='#/texts/149' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Doherty GJ & McMahon HT (2009) Mechanisms of endocytosis. Annu Rev Biochem 78: 857–902' text='Doherty GJ & McMahon HT (2009) Mechanisms of endocytosis. Annu Rev Biochem 78: 857–902'\n",
      "self_ref='#/texts/150' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Di Fiore PP, Polo S & Hofmann K (2003) When ubiquitin meets ubiquitin receptors: a signalling connection. Nat Rev Mol Cell Biol 4: 491–497' text='Di Fiore PP, Polo S & Hofmann K (2003) When ubiquitin meets ubiquitin receptors: a signalling connection. Nat Rev Mol Cell Biol 4: 491–497'\n",
      "self_ref='#/texts/151' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Escobedo A, Gomes T, Aragon E, Martin-Malpartida P, Ruiz L, & Macias MJ (2014) Structural basis of the activation and degradation mechaisms of the E3 ubiquitin lagase Nedd4L. Structure 22: 1446–1457' text='Escobedo A, Gomes T, Aragon E, Martin-Malpartida P, Ruiz L, & Macias MJ (2014) Structural basis of the activation and degradation mechaisms of the E3 ubiquitin lagase Nedd4L. Structure 22: 1446–1457'\n",
      "self_ref='#/texts/152' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Firsov D, Schild L, Gautschi I, Mérillat AM, Schneeberger E & Rossier BC (1996) Cell surface expression of the epithelial Na channel and a mutant causing Liddle syndrome: a quantitative approach. Proc Natl Acad Sci U S A 93: 15370–15375' text='Firsov D, Schild L, Gautschi I, Mérillat AM, Schneeberger E & Rossier BC (1996) Cell surface expression of the epithelial Na channel and a mutant causing Liddle syndrome: a quantitative approach. Proc Natl Acad Sci U S A 93: 15370–15375'\n",
      "self_ref='#/texts/153' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Frost A, Perera R, Roux A, Spasov K, Destaing O, Egelman EH, De Camilli P & Unger VM (2008) Structural basis of membrane invagination by F-BAR domains. Cell 132: 807–817' text='Frost A, Perera R, Roux A, Spasov K, Destaing O, Egelman EH, De Camilli P & Unger VM (2008) Structural basis of membrane invagination by F-BAR domains. Cell 132: 807–817'\n",
      "self_ref='#/texts/154' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Henne WM, Boucrot E, Meinecke M, Evergren E, Vallis Y, Mittal R & McMahon HT (2010) FCHo proteins are nucleators of clathrin-mediated endocytosis. Science 328: 1281–4' text='Henne WM, Boucrot E, Meinecke M, Evergren E, Vallis Y, Mittal R & McMahon HT (2010) FCHo proteins are nucleators of clathrin-mediated endocytosis. Science 328: 1281–4'\n",
      "self_ref='#/texts/155' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Henne WM, Kent HM, Ford MGJ, Hegde BG, Daumke O, Butler PJG, Mittal R, Langen R, Evans PR & McMahon HT (2007) Structure and Analysis of FCHo2 F-BAR Domain: A Dimerizing and Membrane Recruitment Module that Effects Membrane Curvature. Structure 15: 839–852' text='Henne WM, Kent HM, Ford MGJ, Hegde BG, Daumke O, Butler PJG, Mittal R, Langen R, Evans PR & McMahon HT (2007) Structure and Analysis of FCHo2 F-BAR Domain: A Dimerizing and Membrane Recruitment Module that Effects Membrane Curvature. Structure 15: 839–852'\n",
      "self_ref='#/texts/156' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Hershko A & Ciechanover A (1998) THE UBIQUITIN SYSTEM. Annu Rev Biochem 67: 425–479' text='Hershko A & Ciechanover A (1998) THE UBIQUITIN SYSTEM. Annu Rev Biochem 67: 425–479'\n",
      "self_ref='#/texts/157' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Hughey RP, Mueller GM, Bruns JB, Kinlough CL, Poland PA, Harkleroad KL, Carattino MD & Kleyman TR (2003) Maturation of the epithelial Na+ channel involves proteolytic processing of the alpha- and gamma-subunits. J Biol Chem 278: 37073–37082' text='Hughey RP, Mueller GM, Bruns JB, Kinlough CL, Poland PA, Harkleroad KL, Carattino MD & Kleyman TR (2003) Maturation of the epithelial Na+ channel involves proteolytic processing of the alpha- and gamma-subunits. J Biol Chem 278: 37073–37082'\n",
      "self_ref='#/texts/158' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Hui E, Johnson CP, Yao J, Dunning FM & Chapman ER (2009) Synaptotagmin-mediated bending of the target membrane is a critical step in Ca(2+)-regulated fusion. Cell 138: 709–21' text='Hui E, Johnson CP, Yao J, Dunning FM & Chapman ER (2009) Synaptotagmin-mediated bending of the target membrane is a critical step in Ca(2+)-regulated fusion. Cell 138: 709–21'\n",
      "self_ref='#/texts/159' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Ingham RJ, Gish G & Pawson T (2004) The Nedd4 family of E3 ubiquitin ligases: functional diversity within a common modular architecture. Oncogene 23: 1972–84' text='Ingham RJ, Gish G & Pawson T (2004) The Nedd4 family of E3 ubiquitin ligases: functional diversity within a common modular architecture. Oncogene 23: 1972–84'\n",
      "self_ref='#/texts/160' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Laemmli UK (1970) Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature 227: 680–685' text='Laemmli UK (1970) Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature 227: 680–685'\n",
      "self_ref='#/texts/161' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Lehmann M, Lukonin I, Noe F, Schmoranzer J, Clementi C, Loerke D & Haucke V (2019) Nanoscale coupling of endocytic pit growth and stability. Sci Adv 5: eaax5775' text='Lehmann M, Lukonin I, Noe F, Schmoranzer J, Clementi C, Loerke D & Haucke V (2019) Nanoscale coupling of endocytic pit growth and stability. Sci Adv 5: eaax5775'\n",
      "self_ref='#/texts/162' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Lemmon M (2008) Membrane recognition by phospholipid-binding domains. Nat Rev Mol Cell Biol 9: 99–111' text='Lemmon M (2008) Membrane recognition by phospholipid-binding domains. Nat Rev Mol Cell Biol 9: 99–111'\n",
      "self_ref='#/texts/163' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Lim KH, Huang H, Pralle A & Park S (2013) Stable, high-affinity streptavidin monomer for protein labeling and monovalent biotin detection. Biotechnol Bioeng 110: 57–67' text='Lim KH, Huang H, Pralle A & Park S (2013) Stable, high-affinity streptavidin monomer for protein labeling and monovalent biotin detection. Biotechnol Bioeng 110: 57–67'\n",
      "self_ref='#/texts/164' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Martens S, Kozlov MM & McMahon HT (2007) How synaptotagmin promotes membrane fusion. Science 316: 1205–8' text='Martens S, Kozlov MM & McMahon HT (2007) How synaptotagmin promotes membrane fusion. Science 316: 1205–8'\n",
      "self_ref='#/texts/165' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Miyahara M, Nakanishi H, Takahashi K, Satoh-Horikawa K, Tachibana K & Takai Y (2000) Interaction of nectin with afadin is necessary for its clustering at cell-cell contact sites but not for its cis dimerization or trans interaction. J Biol Chem 275: 613–618' text='Miyahara M, Nakanishi H, Takahashi K, Satoh-Horikawa K, Tachibana K & Takai Y (2000) Interaction of nectin with afadin is necessary for its clustering at cell-cell contact sites but not for its cis dimerization or trans interaction. J Biol Chem 275: 613–618'\n",
      "self_ref='#/texts/166' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Nishizuka Y (1992) Intracellular signaling by hydrolysis of phospholipids and activation of protein kinase C. Science 258: 607–14' text='Nishizuka Y (1992) Intracellular signaling by hydrolysis of phospholipids and activation of protein kinase C. Science 258: 607–14'\n",
      "self_ref='#/texts/167' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Peter BJ, Kent HM, Mills IG, Vallis Y, Butler PJG, Evans PR & McMahon HT (2004) BAR domains as sensors of membrane curvature: the amphiphysin BAR structure. Science 303: 495–9' text='Peter BJ, Kent HM, Mills IG, Vallis Y, Butler PJG, Evans PR & McMahon HT (2004) BAR domains as sensors of membrane curvature: the amphiphysin BAR structure. Science 303: 495–9'\n",
      "self_ref='#/texts/168' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Pickart CM & Raasi S (2005) Controlled synthesis of polyubiquitin chains. Methods Enzym 399: 21–36' text='Pickart CM & Raasi S (2005) Controlled synthesis of polyubiquitin chains. Methods Enzym 399: 21–36'\n",
      "self_ref='#/texts/169' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Roberts M, Barry S, Woods A, van der Sluijs P, & Norman J (2001) PDGF-regulated rab4-dependent recycling of avb3 integrin from early endosomes is necessary for cell adhesion and spreading. Curr Biol 11: 1392–1402' text='Roberts M, Barry S, Woods A, van der Sluijs P, & Norman J (2001) PDGF-regulated rab4-dependent recycling of avb3 integrin from early endosomes is necessary for cell adhesion and spreading. Curr Biol 11: 1392–1402'\n",
      "self_ref='#/texts/170' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Rossier BC (2014) Epithelial sodium channel (ENaC) and the control of blood pressure. Curr Opin Pharmacol 15: 33–46' text='Rossier BC (2014) Epithelial sodium channel (ENaC) and the control of blood pressure. Curr Opin Pharmacol 15: 33–46'\n",
      "self_ref='#/texts/171' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Rotin D & Kumar S (2009) Physiological functions of the HECT family of ubiquitin ligases. Nat Rev Mol Cell Biol 10: 398–409' text='Rotin D & Kumar S (2009) Physiological functions of the HECT family of ubiquitin ligases. Nat Rev Mol Cell Biol 10: 398–409'\n",
      "self_ref='#/texts/172' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Rotin D & Staub O (2011) Role of the ubiquitin system in regulating ion transport. Pflugers Arch 461: 1–21' text='Rotin D & Staub O (2011) Role of the ubiquitin system in regulating ion transport. Pflugers Arch 461: 1–21'\n",
      "self_ref='#/texts/173' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Saffarian S, Coccuci E, & Kirchausen T (2009) Distinct dynamics of endocytic clathrin-coated pits and coated palques. PLoS Biol 7: e1000191' text='Saffarian S, Coccuci E, & Kirchausen T (2009) Distinct dynamics of endocytic clathrin-coated pits and coated palques. PLoS Biol 7: e1000191'\n",
      "self_ref='#/texts/174' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Sakamoto Y, Kikuchi K, Umeda K & Nakanishi H (2017) Effects of various spacers between biotin and the phospholipid headgroup on immobilization and sedimentation of biotinylated phospholipid-containing liposomes facilitated by avidin-biotin interactions. J Biochem 162: 221–226' text='Sakamoto Y, Kikuchi K, Umeda K & Nakanishi H (2017) Effects of various spacers between biotin and the phospholipid headgroup on immobilization and sedimentation of biotinylated phospholipid-containing liposomes facilitated by avidin-biotin interactions. J Biochem 162: 221–226'\n",
      "self_ref='#/texts/175' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Shaner NC, Lin MZ, McKeown MR, Steinbach PA, Hazelwood KL, Davidson MW & Tsien RY (2008) Improving the photostability of bright monomeric orange and red fluorescent proteins. Nat Methods 5: 545–551' text='Shaner NC, Lin MZ, McKeown MR, Steinbach PA, Hazelwood KL, Davidson MW & Tsien RY (2008) Improving the photostability of bright monomeric orange and red fluorescent proteins. Nat Methods 5: 545–551'\n",
      "self_ref='#/texts/176' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Shimada A, Niwa H, Tsujita K, Suetsugu S, Nitta K, Hanawa-Suetsugu K, Akasaka R, Nishino Y, Toyama M, Chen L, et al (2007) Curved EFC/F-BAR-domain dimers are joined end to end into a filament for membrane invagination in endocytosis. Cell 129: 761–772' text='Shimada A, Niwa H, Tsujita K, Suetsugu S, Nitta K, Hanawa-Suetsugu K, Akasaka R, Nishino Y, Toyama M, Chen L, et al (2007) Curved EFC/F-BAR-domain dimers are joined end to end into a filament for membrane invagination in endocytosis. Cell 129: 761–772'\n",
      "self_ref='#/texts/177' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Sugahara T, Koga T, Ueno-Shuto K, Shuto T, Watanabe E, Maekawa A, Kitamura K, Tomita K, Mizuno A, Sato T, et al (2009) Calreticulin positively regulates the expression and function of epithelial sodium channel. Exp Cell Res 315: 3294–3300' text='Sugahara T, Koga T, Ueno-Shuto K, Shuto T, Watanabe E, Maekawa A, Kitamura K, Tomita K, Mizuno A, Sato T, et al (2009) Calreticulin positively regulates the expression and function of epithelial sodium channel. Exp Cell Res 315: 3294–3300'\n",
      "self_ref='#/texts/178' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Takano K, Toyooka K & Suetsugu S (2008) EFC/F-BAR proteins and the N-WASP–WIP complex induce membrane curvature-dependent actin polymerization. EMBO J 27: 2817–2828' text='Takano K, Toyooka K & Suetsugu S (2008) EFC/F-BAR proteins and the N-WASP–WIP complex induce membrane curvature-dependent actin polymerization. EMBO J 27: 2817–2828'\n",
      "self_ref='#/texts/179' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Takei K, Slepnev VI, Haucke V & De Camilli P (1999) Functional partnership between amphiphysin and dynamin in clathrin-mediated endocytosis. Nat Cell Biol 1: 33–39' text='Takei K, Slepnev VI, Haucke V & De Camilli P (1999) Functional partnership between amphiphysin and dynamin in clathrin-mediated endocytosis. Nat Cell Biol 1: 33–39'\n",
      "self_ref='#/texts/180' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Traub LM (2009) Tickets to ride: selecting cargo for clathrin-regulated internalization. Nat Rev Mol Cell Biol 10: 583–96' text='Traub LM (2009) Tickets to ride: selecting cargo for clathrin-regulated internalization. Nat Rev Mol Cell Biol 10: 583–96'\n",
      "self_ref='#/texts/181' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Tsujita K, Suetsugu S, Sasaki N, Furutani M, Oikawa T & Takenawa T (2006) Coordination between the actin cytoskeleton and membrane deformation by a novel membrane tubulation domain of PCH proteins is involved in endocytosis. J Cell Biol 172: 269–279' text='Tsujita K, Suetsugu S, Sasaki N, Furutani M, Oikawa T & Takenawa T (2006) Coordination between the actin cytoskeleton and membrane deformation by a novel membrane tubulation domain of PCH proteins is involved in endocytosis. J Cell Biol 172: 269–279'\n",
      "self_ref='#/texts/182' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Uezu A, Umeda K, Tsujita K, Suetsugu S, Takenawa T & Nakanishi H (2011) Characterization of the EFC/F-BAR domain protein, FCHO2. Genes Cells 16: 868–878' text='Uezu A, Umeda K, Tsujita K, Suetsugu S, Takenawa T & Nakanishi H (2011) Characterization of the EFC/F-BAR domain protein, FCHO2. Genes Cells 16: 868–878'\n",
      "self_ref='#/texts/183' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Umeda K, Ikenouchi J, Katahira-Tayama S, Furuse K, Sasaki H, Nakayama M, Matsui T, Tsukita S, Furuse M & Tsukita S (2006) ZO-1 and ZO-2 independently determine where claudins are polymerized in tight-junction strand formation. Cell 126: 741–754' text='Umeda K, Ikenouchi J, Katahira-Tayama S, Furuse K, Sasaki H, Nakayama M, Matsui T, Tsukita S, Furuse M & Tsukita S (2006) ZO-1 and ZO-2 independently determine where claudins are polymerized in tight-junction strand formation. Cell 126: 741–754'\n",
      "self_ref='#/texts/184' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Wang J, Peng Q, Lin Q, Childress C, Carey D & Yang W (2010) Calcium activates Nedd4 E3 ubiquitin ligases by releasing the C2 domain-mediated auto-inhibition. J Biol Chem 285: 12279–12288' text='Wang J, Peng Q, Lin Q, Childress C, Carey D & Yang W (2010) Calcium activates Nedd4 E3 ubiquitin ligases by releasing the C2 domain-mediated auto-inhibition. J Biol Chem 285: 12279–12288'\n",
      "self_ref='#/texts/185' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Wiesner S, Ogunjimi AA, Wang H-R, Rotin D, Sicheri F, Wrana JL & Forman-Kay JD (2007) Autoinhibition of the HECT-type ubiquitin ligase Smurf2 through its C2 domain. Cell 130: 651–862' text='Wiesner S, Ogunjimi AA, Wang H-R, Rotin D, Sicheri F, Wrana JL & Forman-Kay JD (2007) Autoinhibition of the HECT-type ubiquitin ligase Smurf2 through its C2 domain. Cell 130: 651–862'\n",
      "self_ref='#/texts/186' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Yang B & Kumar S (2010) Nedd4 and Nedd4-2: closely related ubiquitin-protein ligases with distinct physiological functions. Cell Death Differ 17: 68–77' text='Yang B & Kumar S (2010) Nedd4 and Nedd4-2: closely related ubiquitin-protein ligases with distinct physiological functions. Cell Death Differ 17: 68–77'\n",
      "self_ref='#/texts/187' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Yin Y, Arkhipov A & Schulten K (2009) Simulations of Membrane Tubulation by Lattices of Amphiphysin N-BAR Domains. Structure 17: 882–892' text='Yin Y, Arkhipov A & Schulten K (2009) Simulations of Membrane Tubulation by Lattices of Amphiphysin N-BAR Domains. Structure 17: 882–892'\n",
      "self_ref='#/texts/188' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig=\"Zaccai N, Kadlecova Z, Dickson VK, Korobchevskaya K & Traub L (2022) FCHO control AP2's initiating role in endocytosis through a Ptd-Ins(4,5)P2-dependent switch. Sci Adv 8: eabn2018\" text=\"Zaccai N, Kadlecova Z, Dickson VK, Korobchevskaya K & Traub L (2022) FCHO control AP2's initiating role in endocytosis through a Ptd-Ins(4,5)P2-dependent switch. Sci Adv 8: eabn2018\"\n",
      "self_ref='#/texts/189' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Zhou R, Patel S V. & Snyder PM (2007) Nedd4-2 catalyzes ubiquitination and degradation of cell surface ENaC. J Biol Chem 282: 20207–20212' text='Zhou R, Patel S V. & Snyder PM (2007) Nedd4-2 catalyzes ubiquitination and degradation of cell surface ENaC. J Biol Chem 282: 20207–20212'\n",
      "self_ref='#/texts/190' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Zhu K, Shan Z, Chen X, Cai Y, Cui L, Yao W, Wang Z, Shi P, Tian C, Lou J, et al (2017) Allosteric auto‐inhibition and activation of the Nedd4 family E3 ligase Itch. EMBO Rep 18: 1618–1630' text='Zhu K, Shan Z, Chen X, Cai Y, Cui L, Yao W, Wang Z, Shi P, Tian C, Lou J, et al (2017) Allosteric auto‐inhibition and activation of the Nedd4 family E3 ligase Itch. EMBO Rep 18: 1618–1630'\n",
      "self_ref='#/texts/191' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/192' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/193' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/194' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Figure legends' text='Figure legends'\n",
      "self_ref='#/texts/195' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/196' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Figure 1. FCHO2 is required for Nedd4L-mediated ubiquitination and endocytosis of ENaC.' text='Figure 1. FCHO2 is required for Nedd4L-mediated ubiquitination and endocytosis of ENaC.'\n",
      "self_ref='#/texts/197' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='A. Knockdown of FCHO2 or Nedd4L by siRNA. αβγENaC-HeLa cells treated with each siRNA were analyzed by immunoblotting (IB).' text='A. Knockdown of FCHO2 or Nedd4L by siRNA. αβγENaC-HeLa cells treated with each siRNA were analyzed by immunoblotting (IB).'\n",
      "self_ref='#/texts/198' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig=\"B. Inhibition of ENaC endocytosis by knockdown of FCHO2 or Nedd4L. After cells were transfected with each siRNA, cell-surface ENaC was labeled by incubating at 4°C with anti-FLAG antibody. Endocytosis was started by incubating cells at 37°C for the indicated periods of time and stopped by chilling cells on ice. Antibody bound to the cell surface but not internalized was removed by acid stripping. After cells were solubilized, the antibody-labeled internalized ENaC subunits were precipitated and analyzed by IB (upper panel) and quantified (bottom and right panels). Internalization was expressed as a percentage of the initial amount of cell-surface αENaC determined by lysing cells without incubation at 37°C or acid stripping. Data are shown as the mean ± S.E.M. of three independent experiments. **P < 0.01; ***P < 0.001 (Student's t-test).\" text=\"B. Inhibition of ENaC endocytosis by knockdown of FCHO2 or Nedd4L. After cells were transfected with each siRNA, cell-surface ENaC was labeled by incubating at 4°C with anti-FLAG antibody. Endocytosis was started by incubating cells at 37°C for the indicated periods of time and stopped by chilling cells on ice. Antibody bound to the cell surface but not internalized was removed by acid stripping. After cells were solubilized, the antibody-labeled internalized ENaC subunits were precipitated and analyzed by IB (upper panel) and quantified (bottom and right panels). Internalization was expressed as a percentage of the initial amount of cell-surface αENaC determined by lysing cells without incubation at 37°C or acid stripping. Data are shown as the mean ± S.E.M. of three independent experiments. **P < 0.01; ***P < 0.001 (Student's t-test).\"\n",
      "self_ref='#/texts/199' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig=\"C. Inhibition of transferrin receptor (TfR) endocytosis by FCHO2 knockdown. After cells were transfected with each siRNA, cell-surface proteins were biotinylated at 4°C. Cells were then incubated at 37°C for 10 min. Biotin was removed from the cell surface, and internalized biotinylated proteins were precipitated with avidin beads. Samples were subjected to IB with anti-TfR antibody and quantified. Internalization was expressed as a proportion of the initial amount of TfR on the cell surface. Data are shown as the mean ± S.E.M. of three independent experiments. *P < 0.05 (Student's t-test).\" text=\"C. Inhibition of transferrin receptor (TfR) endocytosis by FCHO2 knockdown. After cells were transfected with each siRNA, cell-surface proteins were biotinylated at 4°C. Cells were then incubated at 37°C for 10 min. Biotin was removed from the cell surface, and internalized biotinylated proteins were precipitated with avidin beads. Samples were subjected to IB with anti-TfR antibody and quantified. Internalization was expressed as a proportion of the initial amount of TfR on the cell surface. Data are shown as the mean ± S.E.M. of three independent experiments. *P < 0.05 (Student's t-test).\"\n",
      "self_ref='#/texts/200' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='D. Restoration of αENaC endocytosis by FCHO2 re-expression in knockdown cells. After cells were transfected with the indicated siRNA and GFP construct, cell-surface αENaC was labeled with anti-FLAG antibody at 4°C. Cells were then incubated at 37°C for 10 min. After acid stripping, cells were subjected to immunofluorescence microscopy (left panels) and quantified (right panel). The antibody-labeled, internalized αENaC was visualized with a fluorescence-conjugated secondary antibody. The border of each cell is delineated by a solid line. Arrows indicate cells expressing GFP-tagged proteins. Scale bars, 10 μm. The proportion of cells with internalized αENaC among cells expressing control GFP or GFP-siRNA-resistant (sr) -FCHO2 is shown. Data are shown as the mean ± S.E.M. of five independent images (31-71 cells expressing GFP-tagged proteins per image). ***P < 0.001 (two-way analysis of variance with post hoc test).' text='D. Restoration of αENaC endocytosis by FCHO2 re-expression in knockdown cells. After cells were transfected with the indicated siRNA and GFP construct, cell-surface αENaC was labeled with anti-FLAG antibody at 4°C. Cells were then incubated at 37°C for 10 min. After acid stripping, cells were subjected to immunofluorescence microscopy (left panels) and quantified (right panel). The antibody-labeled, internalized αENaC was visualized with a fluorescence-conjugated secondary antibody. The border of each cell is delineated by a solid line. Arrows indicate cells expressing GFP-tagged proteins. Scale bars, 10 μm. The proportion of cells with internalized αENaC among cells expressing control GFP or GFP-siRNA-resistant (sr) -FCHO2 is shown. Data are shown as the mean ± S.E.M. of five independent images (31-71 cells expressing GFP-tagged proteins per image). ***P < 0.001 (two-way analysis of variance with post hoc test).'\n",
      "self_ref='#/texts/201' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig=\"E, F. Effects of FCHO2, Nedd4L, and FBP17 knockdown on ENaC ubiquitination and expression at the cell surface. After cells were transfected with each siRNA, cell-surface αENaC was biotinylated and sequentially precipitated with anti-FLAG antibody and then avidin beads. Samples were subjected to IB with anti-FLAG and anti-Ub antibodies (E) and quantified (F). E, ENaC ubiquitination, Asterisks indicate the same band of ubiquitinated FLAG-αENaC. F, ENaC expression at the cell surface. Data are shown as the mean ± S.E.M. of three independent experiments. *P < 0.05 (Student's t-test).\" text=\"E, F. Effects of FCHO2, Nedd4L, and FBP17 knockdown on ENaC ubiquitination and expression at the cell surface. After cells were transfected with each siRNA, cell-surface αENaC was biotinylated and sequentially precipitated with anti-FLAG antibody and then avidin beads. Samples were subjected to IB with anti-FLAG and anti-Ub antibodies (E) and quantified (F). E, ENaC ubiquitination, Asterisks indicate the same band of ubiquitinated FLAG-αENaC. F, ENaC expression at the cell surface. Data are shown as the mean ± S.E.M. of three independent experiments. *P < 0.05 (Student's t-test).\"\n",
      "self_ref='#/texts/202' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/203' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Figure 2. Localization of cell-surface ENaC, FCHO2, and Nedd4L at clathrin-coated structures.' text='Figure 2. Localization of cell-surface ENaC, FCHO2, and Nedd4L at clathrin-coated structures.'\n",
      "self_ref='#/texts/204' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='A, B. Localization of cell-surface ENaC. mTagRFP-clathrin light chain (CLC) (A) or Nedd4L C922A (catalytic inactive mutant)-GFP (B) was expressed in αβγENaC-HeLa cells. Cell-surface αENaC was labeled with anti-FLAG antibody at 4°C. Cells were then fixed and stained with anti-FCHO2 antibody. Samples were subjected to immunofluorescence microscopy (left panel). Insets represent enlarged images of the dashed boxes. Scale bar, 10 μm. Co-localization was analyzed (right panel). The mean values of Pearson’s correlation coefficient from three cells and the S.E.M. are shown. A. Co-localization of cell-surface ENaC with FCHO2 at clathrin-coated structures. B. Co-localization of cell-surface ENaC with Nedd4L C922A mutant and FCHO2.' text='A, B. Localization of cell-surface ENaC. mTagRFP-clathrin light chain (CLC) (A) or Nedd4L C922A (catalytic inactive mutant)-GFP (B) was expressed in αβγENaC-HeLa cells. Cell-surface αENaC was labeled with anti-FLAG antibody at 4°C. Cells were then fixed and stained with anti-FCHO2 antibody. Samples were subjected to immunofluorescence microscopy (left panel). Insets represent enlarged images of the dashed boxes. Scale bar, 10 μm. Co-localization was analyzed (right panel). The mean values of Pearson’s correlation coefficient from three cells and the S.E.M. are shown. A. Co-localization of cell-surface ENaC with FCHO2 at clathrin-coated structures. B. Co-localization of cell-surface ENaC with Nedd4L C922A mutant and FCHO2.'\n",
      "self_ref='#/texts/205' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='C. Co-localization of Nedd4L with FCHO2 at clathrin-coated structures. Nedd4L-GFP and mTagRFP-CLC were co-expressed in wild-type HeLa cells. Cells were then fixed and stained with anti-FCHO2 antibody (left panel). Insets represent enlarged images of the dashed boxes. Scale bar, 10 μm. Co-localization was analyzed (right panel). The mean values of Pearson’s correlation coefficient from three cells and the S.E.M. are shown.' text='C. Co-localization of Nedd4L with FCHO2 at clathrin-coated structures. Nedd4L-GFP and mTagRFP-CLC were co-expressed in wild-type HeLa cells. Cells were then fixed and stained with anti-FCHO2 antibody (left panel). Insets represent enlarged images of the dashed boxes. Scale bar, 10 μm. Co-localization was analyzed (right panel). The mean values of Pearson’s correlation coefficient from three cells and the S.E.M. are shown.'\n",
      "self_ref='#/texts/206' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/207' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Figure 3. Recruitment of Nedd4L to CCPs. DsRed-CLC were co-expressed with Nedd4L-GFP in wild-type HeLa cells. Images were acquired at 1 s intervals through TIRF microscopy. Upper panel, TIRF images of HeLa cells. Insets represent enlarged images of the dashed boxes. Bottom panel, selected snapshots (every 10 sec) from a time series were obtained from a single CCP indicated by the arrowhead. The time point at which Nedd4L started to accumulate was set 0. Scale bar, 2 μm.' text='Figure 3. Recruitment of Nedd4L to CCPs. DsRed-CLC were co-expressed with Nedd4L-GFP in wild-type HeLa cells. Images were acquired at 1 s intervals through TIRF microscopy. Upper panel, TIRF images of HeLa cells. Insets represent enlarged images of the dashed boxes. Bottom panel, selected snapshots (every 10 sec) from a time series were obtained from a single CCP indicated by the arrowhead. The time point at which Nedd4L started to accumulate was set 0. Scale bar, 2 μm.'\n",
      "self_ref='#/texts/208' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/209' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Figure 4. Recruitment of Nedd4L to membrane tubules generated by FCHO2 in cells.' text='Figure 4. Recruitment of Nedd4L to membrane tubules generated by FCHO2 in cells.'\n",
      "self_ref='#/texts/210' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='GFP-BAR domains were co-expressed with Myc-Nedd4L proteins in COS7 cells (A) and HEK293 cells (B). Cells were serum-starved and subjected to immunofluorescence microscopy. Insets represent enlarged images of the dashed boxes. Scale bar, 10 μm.' text='GFP-BAR domains were co-expressed with Myc-Nedd4L proteins in COS7 cells (A) and HEK293 cells (B). Cells were serum-starved and subjected to immunofluorescence microscopy. Insets represent enlarged images of the dashed boxes. Scale bar, 10 μm.'\n",
      "self_ref='#/texts/211' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/212' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Figure 5. Activation of Nedd4L by membrane tubules generated by FCHO2 in cells.' text='Figure 5. Activation of Nedd4L by membrane tubules generated by FCHO2 in cells.'\n",
      "self_ref='#/texts/213' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='A–E. Nedd4L autoubiquitination. An in vivo ubiquitination assay was performed with various BAR domains and FCHO2 mutants (A, B) and Nedd4L mutants (D). GFP-BAR domains were co-expressed with Myc-Nedd4L proteins and HA-tagged Ub in HEK293 cells. Cell lysates were subjected to immunoprecipitation (IP) and analyzed by IB (A, B, D). WT, wild type. Cells were also subjected to immunofluorescence microscopy (C, E). Scale bar, 10 μm.' text='A–E. Nedd4L autoubiquitination. An in vivo ubiquitination assay was performed with various BAR domains and FCHO2 mutants (A, B) and Nedd4L mutants (D). GFP-BAR domains were co-expressed with Myc-Nedd4L proteins and HA-tagged Ub in HEK293 cells. Cell lysates were subjected to immunoprecipitation (IP) and analyzed by IB (A, B, D). WT, wild type. Cells were also subjected to immunofluorescence microscopy (C, E). Scale bar, 10 μm.'\n",
      "self_ref='#/texts/214' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='F. αENaC ubiquitination. An in vivo ubiquitination assay was performed with various concentrations of Myc-Nedd4L using FALAG-αENaC as a substrate in the presence or absence of GFP-FCHO2 BAR domain. Cell lysates were subjected to IP. Samples were analyzed by IB.' text='F. αENaC ubiquitination. An in vivo ubiquitination assay was performed with various concentrations of Myc-Nedd4L using FALAG-αENaC as a substrate in the presence or absence of GFP-FCHO2 BAR domain. Cell lysates were subjected to IP. Samples were analyzed by IB.'\n",
      "self_ref='#/texts/215' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/216' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Figure 6. The Nedd4L C2 domain is a Ca2+-dependent PS binding domain.' text='Figure 6. The Nedd4L C2 domain is a Ca2+-dependent PS binding domain.'\n",
      "self_ref='#/texts/217' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='A co-sedimentation assay was performed with the C2 domain or full-length Nedd4L using brain-lipid liposomes (~50% PS) at various Ca2+ concentrations (A) and with the C2 domain using synthetic liposomes containing various percentages of PS at 0.3 μM or 1.0 mM Ca2+ (B). The supernatants (S) and pellets (P) were subjected to sodium dodecyl sulfate (SDS)-polyacrylamide gel electrophoresis (PAGE), followed by Coomassie brilliant blue (CBB) staining (upper panels). Bottom panels, quantitative analysis. (−) for [Ca2+] indicates EGTA alone. Data are shown as the mean ± S.E.M. of three independent experiments.' text='A co-sedimentation assay was performed with the C2 domain or full-length Nedd4L using brain-lipid liposomes (~50% PS) at various Ca2+ concentrations (A) and with the C2 domain using synthetic liposomes containing various percentages of PS at 0.3 μM or 1.0 mM Ca2+ (B). The supernatants (S) and pellets (P) were subjected to sodium dodecyl sulfate (SDS)-polyacrylamide gel electrophoresis (PAGE), followed by Coomassie brilliant blue (CBB) staining (upper panels). Bottom panels, quantitative analysis. (−) for [Ca2+] indicates EGTA alone. Data are shown as the mean ± S.E.M. of three independent experiments.'\n",
      "self_ref='#/texts/218' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='A. Ca2+ dependency.' text='A. Ca2+ dependency.'\n",
      "self_ref='#/texts/219' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='B. PS dependency.' text='B. PS dependency.'\n",
      "self_ref='#/texts/220' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/221' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Figure 7. Preference of Nedd4L for FCHO2-generated membrane curvature.' text='Figure 7. Preference of Nedd4L for FCHO2-generated membrane curvature.'\n",
      "self_ref='#/texts/222' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='A. Distribution of liposome diameters. Brain-lipid liposomes (~50% PS) of different diameters were prepared by extrusion through the indicated pore sizes and sonication. Left panel, electron microscopic images. Scale bars, 200 nm. Right panel, distribution of liposome diameters shown by boxplots (number of observations per pore size = 25). The center line inside the box corresponds to the median, the bounds of the box encompass data points between the first and third quartiles, and the whiskers extend to the minimum and maximum values including outliers. So, sonication.' text='A. Distribution of liposome diameters. Brain-lipid liposomes (~50% PS) of different diameters were prepared by extrusion through the indicated pore sizes and sonication. Left panel, electron microscopic images. Scale bars, 200 nm. Right panel, distribution of liposome diameters shown by boxplots (number of observations per pore size = 25). The center line inside the box corresponds to the median, the bounds of the box encompass data points between the first and third quartiles, and the whiskers extend to the minimum and maximum values including outliers. So, sonication.'\n",
      "self_ref='#/texts/223' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='B-D. Sensing and generation of membrane curvature by Nedd4L. Co-sedimentation (B) and in vitro tubulation (C, D) assays were performed using brain-lipid liposomes (~50% PS) at 0.1 μM (B) and 0.3 μM Ca2+ (C, D). B, Preference of Nedd4L for a specific degree of membrane curvature. Liposomes were supplemented with X-biotin-PE and rhodamine-PE. Full-length Nedd4L was incubated with liposomes of different sizes, which were prepared by extrusion through the indicated pore sizes. The sample was then incubated with avidin beads and ultracentrifuged. The proportion of precipitated liposomes of each size was calculated to be ~100% by measuring both total and supernatant fluorescence. The supernatants (S) and pellets (P) were subjected to SDS- PAGE, followed by CBB staining (upper panel). Bottom panel, quantitative analysis. So, sonication. Data are shown as the mean ± S.E.M. of five independent experiments. *P < 0.05; **P < 0.01 (one-way analysis of variance with Tukey’s post hoc test). C, Curvature generation by Nedd4L. The assay was performed with the Nedd4L C2 domain. Samples were examined by electron microscopy. Scale bar, 100 nm. D, The degree of membrane curvature generated by Nedd4L is consistent with that generated by FCHO2. The assay was performed with the indicated protein. Left panel, electron microscopic image. Scale bar, 100 nm. Right panel, distribution of tubule diameters shown by boxplots (number of observations per protein = 14–33). The center line inside the box corresponds to the median, the bounds of the box encompass data points between the first and third quartiles, and the whiskers extend to the minimum and maximum values including outliers.' text='B-D. Sensing and generation of membrane curvature by Nedd4L. Co-sedimentation (B) and in vitro tubulation (C, D) assays were performed using brain-lipid liposomes (~50% PS) at 0.1 μM (B) and 0.3 μM Ca2+ (C, D). B, Preference of Nedd4L for a specific degree of membrane curvature. Liposomes were supplemented with X-biotin-PE and rhodamine-PE. Full-length Nedd4L was incubated with liposomes of different sizes, which were prepared by extrusion through the indicated pore sizes. The sample was then incubated with avidin beads and ultracentrifuged. The proportion of precipitated liposomes of each size was calculated to be ~100% by measuring both total and supernatant fluorescence. The supernatants (S) and pellets (P) were subjected to SDS- PAGE, followed by CBB staining (upper panel). Bottom panel, quantitative analysis. So, sonication. Data are shown as the mean ± S.E.M. of five independent experiments. *P < 0.05; **P < 0.01 (one-way analysis of variance with Tukey’s post hoc test). C, Curvature generation by Nedd4L. The assay was performed with the Nedd4L C2 domain. Samples were examined by electron microscopy. Scale bar, 100 nm. D, The degree of membrane curvature generated by Nedd4L is consistent with that generated by FCHO2. The assay was performed with the indicated protein. Left panel, electron microscopic image. Scale bar, 100 nm. Right panel, distribution of tubule diameters shown by boxplots (number of observations per protein = 14–33). The center line inside the box corresponds to the median, the bounds of the box encompass data points between the first and third quartiles, and the whiskers extend to the minimum and maximum values including outliers.'\n",
      "self_ref='#/texts/224' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/225' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Figure 8. Conserved hydrophobic residues of Nedd4L are responsible for sensing and generation of membrane curvature.' text='Figure 8. Conserved hydrophobic residues of Nedd4L are responsible for sensing and generation of membrane curvature.'\n",
      "self_ref='#/texts/226' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='A. Sequence alignment of C2 domains. Conserved residues are highlighted with the following color code: yellow, aspartates for Ca2+ binding; blue, very hydrophobic residues probably for penetration into the lipid bilayer; and brown, others. Dots show the mutation sites. Hs, Homo sapiens; Sc, Saccharomyces cerevisiae; and Rt, Rattus norvegicus.' text='A. Sequence alignment of C2 domains. Conserved residues are highlighted with the following color code: yellow, aspartates for Ca2+ binding; blue, very hydrophobic residues probably for penetration into the lipid bilayer; and brown, others. Dots show the mutation sites. Hs, Homo sapiens; Sc, Saccharomyces cerevisiae; and Rt, Rattus norvegicus.'\n",
      "self_ref='#/texts/227' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='B. Inability of Nedd4L C2 domain mutants to sense membrane curvature in cells. GFP-FCHO2 BAR domain and Myc-Nedd4L C2 domain mutants were co-expressed in COS7 cells. Insets are enlarged images of dashed boxes. Scale bar, 10 μm.' text='B. Inability of Nedd4L C2 domain mutants to sense membrane curvature in cells. GFP-FCHO2 BAR domain and Myc-Nedd4L C2 domain mutants were co-expressed in COS7 cells. Insets are enlarged images of dashed boxes. Scale bar, 10 μm.'\n",
      "self_ref='#/texts/228' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='C, D. Inability of Nedd4L C2 domain mutants to bind to membranes and to generate membrane curvature in vitro. Co-sedimentation (C) and in vitro tubulation (D) assays were performed using brain-lipid liposomes (~50% PS) at 0.3 μM Ca2+. C, Membrane binding. The assay was performed with Nedd4L C2 domain mutants using various concentrations of liposomes. The supernatants (S) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining (upper panel). Bottom panel, quantitative analysis. Data are shown as the mean ± S.E.M. of three independent experiments. D, Curvature generation. The assay was performed with Nedd4L C2 domain mutants. Samples were examined by fluorescence microscopy. Scale bar, 10 μm.' text='C, D. Inability of Nedd4L C2 domain mutants to bind to membranes and to generate membrane curvature in vitro. Co-sedimentation (C) and in vitro tubulation (D) assays were performed using brain-lipid liposomes (~50% PS) at 0.3 μM Ca2+. C, Membrane binding. The assay was performed with Nedd4L C2 domain mutants using various concentrations of liposomes. The supernatants (S) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining (upper panel). Bottom panel, quantitative analysis. Data are shown as the mean ± S.E.M. of three independent experiments. D, Curvature generation. The assay was performed with Nedd4L C2 domain mutants. Samples were examined by fluorescence microscopy. Scale bar, 10 μm.'\n",
      "self_ref='#/texts/229' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/230' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Figure 9. Properties of the catalytic activity and membrane binding of Nedd4L.' text='Figure 9. Properties of the catalytic activity and membrane binding of Nedd4L.'\n",
      "self_ref='#/texts/231' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Liposomes supplemented with biotinylated phospholipids were incubated with mSA-αENaC (A, B). Samples were mixed with Nedd4L. Mixtures were subjected to ubiquitination (C-E) and co-sedimentation (F) assays. These assays were performed with brain-lipid liposomes (~50% PS) at various Ca2+ concentrations (C), with synthetic liposomes (various percentages of PS) at 0.7 μM Ca2+ (D, F), and with brain-lipid liposomes (20% PS) at 0.7 μM Ca2+ (E). Ubiquitination and liposome binding were quantified by scanning and expressed as ratios to maximum levels. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.' text='Liposomes supplemented with biotinylated phospholipids were incubated with mSA-αENaC (A, B). Samples were mixed with Nedd4L. Mixtures were subjected to ubiquitination (C-E) and co-sedimentation (F) assays. These assays were performed with brain-lipid liposomes (~50% PS) at various Ca2+ concentrations (C), with synthetic liposomes (various percentages of PS) at 0.7 μM Ca2+ (D, F), and with brain-lipid liposomes (20% PS) at 0.7 μM Ca2+ (E). Ubiquitination and liposome binding were quantified by scanning and expressed as ratios to maximum levels. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.'\n",
      "self_ref='#/texts/232' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='A, B. Schematic diagrams of mSA-αENaC. A, Structure. B. Binding to biotinylated liposomes.' text='A, B. Schematic diagrams of mSA-αENaC. A, Structure. B. Binding to biotinylated liposomes.'\n",
      "self_ref='#/texts/233' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='C. Ca2+-dependent Nedd4L activity. Ubiquitination was detected by IB (upper panel) and the intensity was quantified by scanning (bottom panel). Data are shown as the mean ± S.E.M. of three independent experiments.' text='C. Ca2+-dependent Nedd4L activity. Ubiquitination was detected by IB (upper panel) and the intensity was quantified by scanning (bottom panel). Data are shown as the mean ± S.E.M. of three independent experiments.'\n",
      "self_ref='#/texts/234' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='D, Membrane localization of mSA-αENaC enhances the PS-dependent catalytic activity of Nedd4L. Assays were performed in the presence or absence of 40 μM biotin. Ubiquitination was detected by IB (left panel). The intensities were quantified by scanning (right panel). Data are shown as the mean ± S.E.M. of three independent experiments.' text='D, Membrane localization of mSA-αENaC enhances the PS-dependent catalytic activity of Nedd4L. Assays were performed in the presence or absence of 40 μM biotin. Ubiquitination was detected by IB (left panel). The intensities were quantified by scanning (right panel). Data are shown as the mean ± S.E.M. of three independent experiments.'\n",
      "self_ref='#/texts/235' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='E. Nedd4L activity is predominantly stimulated by the specific degree of membrane curvature. The assay was performed using liposomes of different sizes which were prepared by extrusion through the indicated pore sizes. Ubiquitination was detected by IB (left panel) and quantified (right panel). Data are shown as the mean ± S.E.M. of three independent experiments. ***P < 0.001 (one-way analysis of variance with Tukey’s post hoc test).' text='E. Nedd4L activity is predominantly stimulated by the specific degree of membrane curvature. The assay was performed using liposomes of different sizes which were prepared by extrusion through the indicated pore sizes. Ubiquitination was detected by IB (left panel) and quantified (right panel). Data are shown as the mean ± S.E.M. of three independent experiments. ***P < 0.001 (one-way analysis of variance with Tukey’s post hoc test).'\n",
      "self_ref='#/texts/236' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='F. Membrane localization of mSA-αENaC enhances the PS-dependent membrane binding of Nedd4L. Assays were performed in the presence or absence of 40 μM biotin. Liposome binding was detected by IB (left panel). The intensities were quantified by scanning (right panel). T, total sample. P, pellet. Data are shown as the mean ± S.E.M. of three independent experiments.' text='F. Membrane localization of mSA-αENaC enhances the PS-dependent membrane binding of Nedd4L. Assays were performed in the presence or absence of 40 μM biotin. Liposome binding was detected by IB (left panel). The intensities were quantified by scanning (right panel). T, total sample. P, pellet. Data are shown as the mean ± S.E.M. of three independent experiments.'\n",
      "self_ref='#/texts/237' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/238' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/239' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Figure 10. Recruitment and activation of Nedd4L by FCHO2-generated membrane curvature in vitro.' text='Figure 10. Recruitment and activation of Nedd4L by FCHO2-generated membrane curvature in vitro.'\n",
      "self_ref='#/texts/240' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='mSA-αENaC-associated brain-lipid liposomes (20% PS) were mixed with Nedd4L in the presence or absence of BAR domains. Mixtures were subjected to co-sedimentation (A) and ubiquitination (B-E) assays. These assays were performed at 0.7 μM Ca2+ with brain-lipid liposomes (20%PS) (A, B, D, and E) and with the indicated (20% PS or ~50% PS) brain-lipid liposomes (C). Ubiquitination and liposome binding were quantified by scanning and expressed as ratios to maximum levels. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.' text='mSA-αENaC-associated brain-lipid liposomes (20% PS) were mixed with Nedd4L in the presence or absence of BAR domains. Mixtures were subjected to co-sedimentation (A) and ubiquitination (B-E) assays. These assays were performed at 0.7 μM Ca2+ with brain-lipid liposomes (20%PS) (A, B, D, and E) and with the indicated (20% PS or ~50% PS) brain-lipid liposomes (C). Ubiquitination and liposome binding were quantified by scanning and expressed as ratios to maximum levels. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.'\n",
      "self_ref='#/texts/241' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='A. FCHO2-induced recruitment of Nedd4L. The assay was performed with the FCHO2 BAR domain (1.4 μM, 50 μg/ml) in the presence or absence of biotin. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining (FCHO2 BAR) and IB (Nedd4L and mSA-αENaC).' text='A. FCHO2-induced recruitment of Nedd4L. The assay was performed with the FCHO2 BAR domain (1.4 μM, 50 μg/ml) in the presence or absence of biotin. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining (FCHO2 BAR) and IB (Nedd4L and mSA-αENaC).'\n",
      "self_ref='#/texts/242' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='B, C. FCHO2-induced activation of Nedd4L. B, Effects of various doses of the FCHO2 BAR domain. The assay was performed with various concentrations of the FCHO2 BAR domain. Ubiquitination was detected by IB (left panel) and the intensity was quantified by scanning (right panel). C, Effects of the PS percentage in liposomes. The assay was performed with the FCHO2 BAR domain (1.4 μM). Samples were analyzed by IB. Data are shown as the mean ± S.E.M. of three independent experiments.' text='B, C. FCHO2-induced activation of Nedd4L. B, Effects of various doses of the FCHO2 BAR domain. The assay was performed with various concentrations of the FCHO2 BAR domain. Ubiquitination was detected by IB (left panel) and the intensity was quantified by scanning (right panel). C, Effects of the PS percentage in liposomes. The assay was performed with the FCHO2 BAR domain (1.4 μM). Samples were analyzed by IB. Data are shown as the mean ± S.E.M. of three independent experiments.'\n",
      "self_ref='#/texts/243' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='D, E. Specificity of BAR domains and effect of an FCHO2 mutant. The assay was performed with various BAR domains (D) and an FCHO2 mutant (E) (1.4 μM each). Samples were analyzed by IB.' text='D, E. Specificity of BAR domains and effect of an FCHO2 mutant. The assay was performed with various BAR domains (D) and an FCHO2 mutant (E) (1.4 μM each). Samples were analyzed by IB.'\n",
      "self_ref='#/texts/244' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/245' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Figure 11. Effects of PI(4,5)P2 on FCHO2-induced recruitment and activation of Nedd4L.' text='Figure 11. Effects of PI(4,5)P2 on FCHO2-induced recruitment and activation of Nedd4L.'\n",
      "self_ref='#/texts/246' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='A, B. Binding to PI(4,5)P2. A co-sedimentation assay was performed at 0.7 μM Ca2+ with the FCHO2 BAR or Nedd4L C2 domain using mSA-αENaC-associated synthetic liposomes containing 20% PS, 5% PI(4,5)P2 (A), or various percentages of PI(4,5)P2 (B). The supernatants (S) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining (A and upper panel in B). Bottom panel in B, quantitative analysis. Control, synthetic liposomes [0% PS and 0% PI(4,5)P2]. Data are shown as the mean ± S.E.M. of three independent experiments.' text='A, B. Binding to PI(4,5)P2. A co-sedimentation assay was performed at 0.7 μM Ca2+ with the FCHO2 BAR or Nedd4L C2 domain using mSA-αENaC-associated synthetic liposomes containing 20% PS, 5% PI(4,5)P2 (A), or various percentages of PI(4,5)P2 (B). The supernatants (S) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining (A and upper panel in B). Bottom panel in B, quantitative analysis. Control, synthetic liposomes [0% PS and 0% PI(4,5)P2]. Data are shown as the mean ± S.E.M. of three independent experiments.'\n",
      "self_ref='#/texts/247' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='C, D. Effects on FCHO2-induced recruitment and activation of Nedd4L. Co-sedimentation (C) and ubiquitination (D) assays were performed at 0.7 μM Ca2+ with mSA-αENaC-associated synthetic liposomes containing 20% PS or 5% PI(4,5)P2 in the presence or absence of the FCHO2 BAR domain. C, Nedd4L recruitment. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by IB with anti-Nedd4L antibody (upper panel) and CBB staining (lower panel). D, Nedd4L activation. Samples were analyzed by IB. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.' text='C, D. Effects on FCHO2-induced recruitment and activation of Nedd4L. Co-sedimentation (C) and ubiquitination (D) assays were performed at 0.7 μM Ca2+ with mSA-αENaC-associated synthetic liposomes containing 20% PS or 5% PI(4,5)P2 in the presence or absence of the FCHO2 BAR domain. C, Nedd4L recruitment. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by IB with anti-Nedd4L antibody (upper panel) and CBB staining (lower panel). D, Nedd4L activation. Samples were analyzed by IB. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.'\n",
      "self_ref='#/texts/248' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/249' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Figure 12. Effects of liposome size on FCHO2-induced activation of Nedd4L. Co-sedimentation (A) and ubiquitination (B) assays were performed at 0.7 μM Ca2+ with the FCHO2 BAR domain using brain-lipid liposomes (20% PS, 0.8 µm or 0.05 µm pore-size) that were associated with mSA-αENaC.' text='Figure 12. Effects of liposome size on FCHO2-induced activation of Nedd4L. Co-sedimentation (A) and ubiquitination (B) assays were performed at 0.7 μM Ca2+ with the FCHO2 BAR domain using brain-lipid liposomes (20% PS, 0.8 µm or 0.05 µm pore-size) that were associated with mSA-αENaC.'\n",
      "self_ref='#/texts/250' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='A. Binding of the FCHO2 BAR domain to 0.8 µm and 0.05 µm pore-size liposomes. The co-sedimentation assay was performed using synthetic (0% PS) or brain-lipid liposomes (20% PS) containing rhodamine-PE and fluorescein-PE. Liposomes were precipitated with anti-fluorescein antibody. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining. The proportion of precipitated liposomes of each size was calculated to be ~100% by measuring both total and supernatant fluorescence.' text='A. Binding of the FCHO2 BAR domain to 0.8 µm and 0.05 µm pore-size liposomes. The co-sedimentation assay was performed using synthetic (0% PS) or brain-lipid liposomes (20% PS) containing rhodamine-PE and fluorescein-PE. Liposomes were precipitated with anti-fluorescein antibody. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining. The proportion of precipitated liposomes of each size was calculated to be ~100% by measuring both total and supernatant fluorescence.'\n",
      "self_ref='#/texts/251' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='B. Effects of liposome size on FCHO2-induced activation of Nedd4L. Samples were analyzed by IB. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.' text='B. Effects of liposome size on FCHO2-induced activation of Nedd4L. Samples were analyzed by IB. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.'\n",
      "self_ref='#/texts/252' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/253' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Figure 13. Model of ENaC endocytosis.' text='Figure 13. Model of ENaC endocytosis.'\n",
      "self_ref='#/texts/254' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='FCHO2 binds to the plasma membrane and generates a specific degree of membrane curvature according to the intrinsic curvature of its BAR domain. Nedd4L exists in a catalytically autoinhibited state due to an intramolecular interaction between the N-terminal C2 and C-terminal HECT domains. Nedd4L is recruited to FCHO2-generated membrane curvature at the rim of nascent CCPs that ENaC enters. Membrane binding of the C2 domain causes catalytic activation by relieving autoinhibition. ENaC is ubiquitinated and captured by adaptor proteins with Ub-interacting motifs, such as Eps15.' text='FCHO2 binds to the plasma membrane and generates a specific degree of membrane curvature according to the intrinsic curvature of its BAR domain. Nedd4L exists in a catalytically autoinhibited state due to an intramolecular interaction between the N-terminal C2 and C-terminal HECT domains. Nedd4L is recruited to FCHO2-generated membrane curvature at the rim of nascent CCPs that ENaC enters. Membrane binding of the C2 domain causes catalytic activation by relieving autoinhibition. ENaC is ubiquitinated and captured by adaptor proteins with Ub-interacting motifs, such as Eps15.'\n",
      "self_ref='#/texts/255' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/256' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Extended View Figure Legends' text='Extended View Figure Legends'\n",
      "self_ref='#/texts/257' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/258' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Figure EV1. Expression of ENaC subunits in αβγENaC-HeLa cells.' text='Figure EV1. Expression of ENaC subunits in αβγENaC-HeLa cells.'\n",
      "self_ref='#/texts/259' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='A, B. Doxycycline (Dox)-induced expression of αENaC. αβγENaC-HeLa cells were cultured overnight in the presence or absence of Dox. The lysates of parental HeLa and αβγENaC-HeLa cells were subjected to IB. A, 7.5% gel. B, Gradient gel (5-20%). Anti-γENaC antibody cross-reacted with two bands of endogenous proteins in parental HeLa cells, the lower band of which overlapped with γENaC (arrow) in αβγENaC-HeLa cells. Upon Dox-induced αENaC expression, an additional 70-kDa γENaC band (asterisk) was detected. It has been shown that co-expression of all three subunits induces ENaC maturation, including proteolytic cleavage of α- and γENaC (Hughey et al, 2003). The 70-kDa γENaC band is likely a cleavage product comprising the C-terminal region. The 20-kDa αENaC band (arrowhead) is likely a cleavage product comprising the N-terminal region.' text='A, B. Doxycycline (Dox)-induced expression of αENaC. αβγENaC-HeLa cells were cultured overnight in the presence or absence of Dox. The lysates of parental HeLa and αβγENaC-HeLa cells were subjected to IB. A, 7.5% gel. B, Gradient gel (5-20%). Anti-γENaC antibody cross-reacted with two bands of endogenous proteins in parental HeLa cells, the lower band of which overlapped with γENaC (arrow) in αβγENaC-HeLa cells. Upon Dox-induced αENaC expression, an additional 70-kDa γENaC band (asterisk) was detected. It has been shown that co-expression of all three subunits induces ENaC maturation, including proteolytic cleavage of α- and γENaC (Hughey et al, 2003). The 70-kDa γENaC band is likely a cleavage product comprising the C-terminal region. The 20-kDa αENaC band (arrowhead) is likely a cleavage product comprising the N-terminal region.'\n",
      "self_ref='#/texts/260' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='C. Association of α-, β-, and γENaC. When αENaC was immunoprecipitated with anti-FLAG antibody from αβγENaC-HeLa cells treated with Dox, β- and γENaC were co-precipitated. Asterisk, 70-kDa γENaC band.' text='C. Association of α-, β-, and γENaC. When αENaC was immunoprecipitated with anti-FLAG antibody from αβγENaC-HeLa cells treated with Dox, β- and γENaC were co-precipitated. Asterisk, 70-kDa γENaC band.'\n",
      "self_ref='#/texts/261' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/262' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Figure EV2. Knockdown of FBP17 by siRNA.' text='Figure EV2. Knockdown of FBP17 by siRNA.'\n",
      "self_ref='#/texts/263' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig=\"αβγENaC-HeLa cells were treated with each siRNA, and FBP17 mRNA levels were quantified using real-time PCR. The expression of FBP17 was normalized to GAPDH mRNA levels. Data are shown as the mean ± S.E.M. of three independent experiments. ***P < 0.001 (Student's t-test).\" text=\"αβγENaC-HeLa cells were treated with each siRNA, and FBP17 mRNA levels were quantified using real-time PCR. The expression of FBP17 was normalized to GAPDH mRNA levels. Data are shown as the mean ± S.E.M. of three independent experiments. ***P < 0.001 (Student's t-test).\"\n",
      "self_ref='#/texts/264' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/265' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Figure EV3. Inability of FCHO2 mutants to generate membrane tubules.' text='Figure EV3. Inability of FCHO2 mutants to generate membrane tubules.'\n",
      "self_ref='#/texts/266' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='GFP-FCHO2 BAR domain [wild type (WT) or mutant] was expressed in COS7 cells. Cells were then subjected to immunofluorescence microscopy. Scale bar, 10 μm.' text='GFP-FCHO2 BAR domain [wild type (WT) or mutant] was expressed in COS7 cells. Cells were then subjected to immunofluorescence microscopy. Scale bar, 10 μm.'\n",
      "self_ref='#/texts/267' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/268' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Figure EV4. Inability of Nedd4L mutants to ubiquitinate αENaC. An in vivo ubiquitination assay was performed with various Nedd4L constructs (each 0.5 µg) using FLAG-αENaC as a substrate in the presence or absence of GFP-FCHO2 BAR domain. Cell lysates were subjected to IP. Samples were analyzed by IB.' text='Figure EV4. Inability of Nedd4L mutants to ubiquitinate αENaC. An in vivo ubiquitination assay was performed with various Nedd4L constructs (each 0.5 µg) using FLAG-αENaC as a substrate in the presence or absence of GFP-FCHO2 BAR domain. Cell lysates were subjected to IP. Samples were analyzed by IB.'\n",
      "self_ref='#/texts/269' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/270' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Figure EV5. Inhibition of the liposome binding of Nedd4L by the FCHO2 BAR domain.' text='Figure EV5. Inhibition of the liposome binding of Nedd4L by the FCHO2 BAR domain.'\n",
      "self_ref='#/texts/271' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='A co-sedimentation assay was performed at 0.7 μM Ca2+ with control liposomes (0% PS) or brain-lipid liposomes (~50% PS) in the presence or absence of the FCHO2 BAR domain. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by IB (upper panel) and CBB staining (lower panel).' text='A co-sedimentation assay was performed at 0.7 μM Ca2+ with control liposomes (0% PS) or brain-lipid liposomes (~50% PS) in the presence or absence of the FCHO2 BAR domain. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by IB (upper panel) and CBB staining (lower panel).'\n",
      "self_ref='#/texts/272' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='' text=''\n",
      "self_ref='#/texts/273' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='Figure EV6. Membrane binding and curvature generation of BAR domains and an FCHO2 mutant. Co-sedimentation (A) and in vitro tubulation (B) assays were performed at 0.7 μM Ca2+ with the indicated BAR domains using brain-lipid liposomes (20% PS) that were associated with mSA-αENaC.' text='Figure EV6. Membrane binding and curvature generation of BAR domains and an FCHO2 mutant. Co-sedimentation (A) and in vitro tubulation (B) assays were performed at 0.7 μM Ca2+ with the indicated BAR domains using brain-lipid liposomes (20% PS) that were associated with mSA-αENaC.'\n",
      "self_ref='#/texts/274' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='A. Membrane binding. The supernatants (S) and pellets (P) were subjected to SDS-PAGE followed by CBB staining.' text='A. Membrane binding. The supernatants (S) and pellets (P) were subjected to SDS-PAGE followed by CBB staining.'\n",
      "self_ref='#/texts/275' parent=RefItem(cref='#/body') children=[] label=<DocItemLabel.PARAGRAPH: 'paragraph'> prov=[] orig='B. Curvature generation. Left panel, electron microscopic image. Scale bar, 100 nm. Right panel, distribution of tubule diameters shown by boxplots (number of observations per protein = 22–26). The center line inside the box corresponds to the median, the bounds of the box encompass data points between the first and third quartiles, and the whiskers extend to the minimum and maximum values including outliers.' text='B. Curvature generation. Left panel, electron microscopic image. Scale bar, 100 nm. Right panel, distribution of tubule diameters shown by boxplots (number of observations per protein = 22–26). The center line inside the box corresponds to the median, the bounds of the box encompass data points between the first and third quartiles, and the whiskers extend to the minimum and maximum values including outliers.'\n",
      "Section Headers:\n",
      "\n",
      "Sections:\n"
     ]
    }
   ],
   "source": [
    "from docling.document_converter import DocumentConverter\n",
    "from docling_core.types.doc.labels import DocItemLabel\n",
    "\n",
    "# Initialize the converter\n",
    "converter = DocumentConverter()\n",
    "\n",
    "# Convert your DOCX file\n",
    "docling_docx = converter.convert(DOCX_FILES[1])\n",
    "\n",
    "# Initialize lists to store headers and sections\n",
    "section_headers = []\n",
    "sections = []\n",
    "\n",
    "# Iterate over the document items\n",
    "for item, _ in docling_docx.document.iterate_items():\n",
    "    \n",
    "    # Check if the item is a section header\n",
    "    print(item)\n",
    "    if item.label == DocItemLabel.SECTION_HEADER:\n",
    "        section_headers.append(item.text)\n",
    "        sections.append({\n",
    "            'header': item.text,\n",
    "            'content': []\n",
    "        })\n",
    "    # Check if the item is a paragraph and there's a preceding section\n",
    "    elif item.label == DocItemLabel.PARAGRAPH and sections:\n",
    "        sections[-1]['content'].append(item.text)\n",
    "\n",
    "# Display the extracted section headers\n",
    "print(\"Section Headers:\")\n",
    "for header in section_headers:\n",
    "    print(header)\n",
    "\n",
    "# Display the sections with their content\n",
    "print(\"\\nSections:\")\n",
    "for section in sections:\n",
    "    print(f\"Header: {section['header']}\")\n",
    "    print(\"Content:\")\n",
    "    for paragraph in section['content']:\n",
    "        print(paragraph)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Using beautifulsoup to parse the html export"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "metadata": {},
   "outputs": [],
   "source": [
    "from bs4 import BeautifulSoup\n",
    "\n",
    "converter = DocumentConverter()\n",
    "docling_docx = converter.convert(DOCX_FILES[1])\n",
    "\n",
    "# Example HTML content from docling's `export_to_html()` method\n",
    "html_content = docling_docx.document.export_to_html()\n",
    "\n",
    "def extract_sections_from_html(html):\n",
    "    # Parse the HTML content\n",
    "    soup = BeautifulSoup(html, \"html.parser\")\n",
    "    \n",
    "    sections = []\n",
    "    current_section = None\n",
    "\n",
    "    # Iterate over all HTML elements\n",
    "    for element in soup.find_all([\"h1\", \"h2\", \"h3\", \"h4\", \"h5\", \"h6\", \"p\"]):  # Include headers and paragraphs\n",
    "        if element.name in [\"h1\", \"h2\", \"h3\", \"h4\", \"h5\", \"h6\"]:  # Check if it's a header\n",
    "            # Save the previous section if one exists\n",
    "            if current_section:\n",
    "                sections.append(current_section)\n",
    "            # Start a new section\n",
    "            current_section = {\"section_header\": element.text.strip(), \"section_content\": \"\"}\n",
    "        elif element.name == \"p\" and current_section:  # Add paragraph content to the current section\n",
    "            current_section[\"section_content\"] += element.text.strip() + \" \"\n",
    "\n",
    "    # Append the last section\n",
    "    if current_section:\n",
    "        sections.append(current_section)\n",
    "\n",
    "    return sections\n",
    "\n",
    "# Extract sections\n",
    "sections = extract_sections_from_html(html_content)\n",
    "\n",
    "# Print the extracted sections\n",
    "for section in sections:\n",
    "    print(f\"Header: {section['section_header']}\")\n",
    "    print(f\"Content: {section['section_content']}\\n\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<!DOCTYPE html>\n",
       "\n",
       "<html lang=\"en\">\n",
       "<head>\n",
       "<link href=\"https://ds4sd.github.io/docling/assets/logo.png\" rel=\"icon\" type=\"image/png\"/>\n",
       "<meta charset=\"utf-8\"/>\n",
       "<title>\n",
       "    Powered by Docling\n",
       "    </title>\n",
       "<style>\n",
       "    html {\n",
       "    background-color: LightGray;\n",
       "    }\n",
       "    body {\n",
       "    margin: 0 auto;\n",
       "    width:800px;\n",
       "    padding: 30px;\n",
       "    background-color: White;\n",
       "    font-family: Arial, sans-serif;\n",
       "    box-shadow: 10px 10px 10px grey;\n",
       "    }\n",
       "    figure{\n",
       "    display: block;\n",
       "    width: 100%;\n",
       "    margin: 0px;\n",
       "    margin-top: 10px;\n",
       "    margin-bottom: 10px;\n",
       "    }\n",
       "    img {\n",
       "    display: block;\n",
       "    margin: auto;\n",
       "    margin-top: 10px;\n",
       "    margin-bottom: 10px;\n",
       "    max-width: 640px;\n",
       "    max-height: 640px;\n",
       "    }\n",
       "    table {\n",
       "    min-width:500px;\n",
       "    background-color: White;\n",
       "    border-collapse: collapse;\n",
       "    cell-padding: 5px;\n",
       "    margin: auto;\n",
       "    margin-top: 10px;\n",
       "    margin-bottom: 10px;\n",
       "    }\n",
       "    th, td {\n",
       "    border: 1px solid black;\n",
       "    padding: 8px;\n",
       "    }\n",
       "    th {\n",
       "    font-weight: bold;\n",
       "    }\n",
       "    table tr:nth-child(even) td{\n",
       "    background-color: LightGray;\n",
       "    }\n",
       "    </style>\n",
       "</head>\n",
       "<p>The Nedd4L ubiquitin ligase is activated by FCHO2-generated membrane curvature</p>\n",
       "<p></p>\n",
       "<p>Yasuhisa Sakamoto1, Akiyoshi Uezu1, Koji Kikuchi1, Jangmi Kang2, Eiko Fujii2, Toshiro Moroishi3, Shiro Suetsugu4, and Hiroyuki Nakanishi1,2*</p>\n",
       "<p></p>\n",
       "<p>1Department of Molecular Pharmacology, Faculty of Life Sciences, Kumamoto University, 1–1–1 Honjyo, Kumamoto 860-8556, Japan</p>\n",
       "<p>2Faculty of Clinical Nutrition and Dietetics, Konan Women’s University, 6–2–23 Morikita-machi, Kobe 658-0001, Japan</p>\n",
       "<p>3Department of Molecular and Medical Pharmacology, Faculty of Life Sciences, Kumamoto University, 1–1–1 Honjyo, Kumamoto 860-8556, Japan</p>\n",
       "<p>4Division of Biological Science, Graduate School of Science and Technology, Nara Institute of Science and Technology, 8916-5 Takayama-cho, Ikoma 630-0192, Japan</p>\n",
       "<p>*Corresponding author. Tel: +81-96-373-5074; Fax: +81-96-373-5078; E-mail: hnakanis@gpo.kumamoto-u.ac.jp</p>\n",
       "<p></p>\n",
       "<p></p>\n",
       "<p></p>\n",
       "<p>Abstract</p>\n",
       "<p></p>\n",
       "<p>The C2-WW-HECT domain ubiquitin ligase Nedd4L regulates membrane sorting during endocytosis through the ubiquitination of cargo molecules such as the epithelial sodium channel (ENaC). Nedd4L is catalytically autoinhibited by an intramolecular interaction between its C2 and HECT domains, but the protein’s activation mechanism is poorly understood. Here, we show that Nedd4L activation is linked to membrane shape by FCHO2, a Bin-Amphiphysin-Rsv (BAR) domain protein that regulates endocytosis. FCHO2 was required for the Nedd4L-mediated ubiquitination and endocytosis of ENaC, with Nedd4L co-localizing with FCHO2 at clathrin-coated pits. In cells, Nedd4L was specifically recruited to, and activated by, the FCHO2 BAR domain. Furthermore, we reconstituted FCHO2-induced recruitment and activation of Nedd4L in vitro. Both the recruitment and activation were mediated by membrane curvature rather than protein-protein interactions. The Nedd4L C2 domain recognized a specific degree of membrane curvature that was generated by the FCHO2 BAR domain, with this curvature directly activating Nedd4L by relieving its autoinhibition. Thus, we show for the first time a specific function (i.e., recruitment and activation of an enzyme regulating cargo sorting) of membrane curvature by a BAR domain protein.</p>\n",
       "<p></p>\n",
       "<p>Keywords: Nedd4L; FCHO2; clathrin; endocytosis; membrane curvature.</p>\n",
       "<p></p>\n",
       "<p>Introduction</p>\n",
       "<p></p>\n",
       "<p>Posttranslational modification of proteins by covalent attachment of ubiquitin (Ub) is catalyzed by three enzymes: a Ub-activating enzyme (E1), a Ub-conjugating enzyme (E2), and a Ub-protein ligase (E3) that determines substrate specificity (Hershko &amp; Ciechanover, 1998). E3 Ub ligases are classified into two categories based on their Ub transfer mechanisms: RING finger/U-box E3 and HECT-type E3. The Nedd4 family belongs to HECT-type E3 Ub ligases and consists of nine members, including Nedd4/Nedd4-1, Nedd4L/Nedd4-2, Itch, Smurf1, Smurf2, WWP1, WWP2, NEDL1, and NEDL2 (Ingham et al, 2004; Rotin &amp; Kumar, 2009). They are characterized by a common modular organization, with an N-terminal C2 domain, two to four WW domains, and a C-terminal catalytic HECT domain. While the C2 domain was originally identified in protein kinase C (PKC) as a Ca2+-dependent phosphatidylserine (PS)-binding domain (Nishizuka, 1992), it shows significant diversity in the binding partners, including intracellular proteins and other phospholipids, such as phosphatidylinositol (4,5)-bisphosphate [PI(4,5)P2] (Lemmon, 2008). The WW domains recognize proline-rich motifs, such as PPxY (PY motif, where x is any residue), of substrates or adaptor proteins. The HECT domain catalyzes the isopeptide bond formation between the Ub C terminus and the substrate lysine residues. At least some of the Nedd4 family members, including Nedd4L, are catalytically autoinhibited by an intramolecular interaction between the C2 and HECT domains (Wang et al, 2010; Wiesner et al, 2007; Zhu et al, 2017). However, the activation mechanism of the Nedd4 family is poorly understood.</p>\n",
       "<p>The Nedd4 family is implicated in a wide variety of cellular processes by regulating membrane trafficking and degradation of components of various signaling pathways (Ingham et al, 2004; Rotin &amp; Kumar, 2009). Among a diversity of substrates, the best-characterized is the epithelial sodium channel (ENaC). This channel is composed of three subunits, α-, β-, and γENaC, each of which possesses two transmembrane segments and intracellular N- and C-termini with a large extracellular loop (Butterworth, 2010; Rotin &amp; Staub, 2011; Rossier, 2014). Each subunit has a PY motif at the cytosolic C-terminus. Nedd4L Ub ligase specifically binds to the PY motif and mediates the ubiquitination of lysine residues on the N-terminus of at least α- and γENaC at the plasma membrane. Ub serves as an internalization signal for recognition by adaptor proteins, such as epsin and Eps15, for clathrin-mediated endocytosis (CME) (Traub, 2009). Consequently, ENaC is constitutively internalized through CME. Heterozygous mutations of the PY motif of β- or γENaC lead to impaired ubiquitination and endocytosis, resulting in persistence of the channels at the cell surface and increased Na+ absorption in Liddle syndrome, an autosomal dominant form of severe hypertension.</p>\n",
       "<p>The process of CME begins at the cytoplasmic surface of the plasma membrane with the assembly of clathrin with adaptor proteins and transmembrane cargo molecules (Doherty &amp; McMahon, 2009; Traub, 2009). In parallel with this assembly, membrane curvature is generated to form hemispherical clathrin-coated pits (CCPs). This process leads to deep membrane invaginations. Subsequently, the neck of invaginated pits is constricted and severed to separate the newly formed clathrin-coated vesicles (CCVs) from the plasma membrane. These membrane curvatures are considered to be generated, sensed, and/or maintained by Bin-Amphiphysin-Rsv (BAR) domain superfamily proteins. BAR domains form a crescent-shaped dimer characterized by a specific degree of intrinsic curvature. They bind to negatively charged phospholipids, such as PS, and force membranes to bend according to their intrinsic curvatures, thereby inducing the formation of membrane tubules with specific diameters (curvatures). For example, the diameters of membrane tubules generated by the amphiphysin BAR domain, FCHO2 BAR domain, and FBP17 BAR domain are 20–50 nm (sharper curvature) (Peter et al, 2004; Takei et al, 1999; Yin et al, 2009), 50–80 nm (intermediate curvature) (Henne et al, 2007), and ~150 nm (Takano et al, 2008) (shallower curvature), respectively.</p>\n",
       "<p>The FCHO2 BAR domain has been proposed to trigger the initial invagination for CCP formation (Henne et al, 2010). FCHO2 binds to the flat surface of the plasma membrane at sites where CCPs will form. FCHO2 then generates the initial membrane curvature and recruits the clathrin machinery for CCP formation. Here, we report the role of FCHO2 in Nedd4L-mediated ENaC endocytosis and show that FCHO2 BAR domain-generated membrane curvature induces the recruitment and activation of Nedd4L.</p>\n",
       "<p></p>\n",
       "<p>Results</p>\n",
       "<p></p>\n",
       "<p>FCHO2 is required for Nedd4L-mediated ubiquitination and endocytosis of ENaC</p>\n",
       "<p>To study the role of FCHO2 in ENaC endocytosis, we established a HeLa cell line stably expressing all three subunits, α-, β-, and γENaC (αβγENaC-HeLa cells) (Fig EV1A-C). A FLAG tag was introduced into the extracellular region of αENaC in a position previously shown not to affect channel activity (Firsov et al, 1996). Upon FCHO2 knockdown by small interfering RNA (siRNA) methods, ENaC internalization was reduced, as was upon Nedd4L knockdown (Zhou et al, 2007) (Fig 1 A and B). Transferrin receptor (TfR) internalization was reduced upon FCHO2 knockdown but not upon Nedd4L knockdown (Fig 1C). The phenotype of FCHO2 knockdown in ENaC internalization was rescued by an siRNA-resistant (sr) form of FCHO2 (Fig 1D and Appendix Fig S1). Furthermore, αENaC ubiquitination was reduced upon FCHO2 or Nedd4L knockdown but not upon FBP17 knockdown (Fig 1E and Fig EV2). FCHO2 or Nedd4L knockdown increased αENaC expression at the cell surface (Fig 1E and F). These results suggest that FCHO2 is required for Nedd4L-mediated ubiquitination and endocytosis of ENaC.</p>\n",
       "<p></p>\n",
       "<p>Recruitment and activation of Nedd4L by FCHO2 in cells</p>\n",
       "<p>Immunofluorescence microscopy revealed that in αβγENaC-HeLa cells, cell-surface αENaC co-localized with FCHO2 at clathrin-coated structures (Fig 2A). However, its co-localization with Nedd4L was not detected (Appendix Fig S2A), because exogenous expression of Nedd4L-green fluorescent protein (GFP) stimulated ENaC internalization, resulting in the disappearance of αENaC from the cell surface. Cell-surface αENaC was observed in cells lacking Nedd4L-GFP expression (non-transfected cells). When a catalytically inactive Nedd4L mutant, C922A, was expressed, it co-localized with cell-surface αENaC and endogenous FCHO2 (Fig 2B). The mutant also formed FCHO2-negative spots. In wild-type HeLa cells, Nedd4L accumulated at clathrin-coated structures where FCHO2 localized (Fig 2C). Additionally, Nedd4L formed clathrin-negative spots, suggesting that they may be involved in clathrin-independent endocytosis.</p>\n",
       "<p>Using total internal reflection fluorescence (TIRF) microscopy, we investigated whether Nedd4L is recruited to CCPs in wild-type HeLa cells. Similar to FCHO2 described previously (Henne et al, 2010; Lehmann et al, 2019; Zaccai et al, 2022), Nedd4L showed dynamic co-localization with clathrin during pit formation (Fig 3). When both fluorescent protein-tagged FCHO2 and Nedd4L were expressed, the accumulation of Nedd4L at FCHO2-positive spots was not clearly observed (Appendix Fig S2B). The reason behind this remains unknown, but it has been shown that clathrin coat formation, measured in time-lapse images of cells expressing fluorescent proteins, is affected by several factors, such as overexpression level, choice of the proteins, and choice of the fluorescent tag (Aguet et al, 2013). These factors may contribute to the failure to detect the accumulation of Nedd4L at FCHO2-positive spots.</p>\n",
       "<p>While Nedd4L shows a punctate staining pattern at the cell surface (see Figs 2C, 3B and 3C), it exhibited diffuse distribution throughout the cytoplasm (Fig 4A and B). In COS7 cells, co-expression of the FCHO2 BAR domain [amino acid (aa) 1-302] recruited Nedd4L to membrane tubules (Fig 4A). The Nedd4L C2 domain (Myc-Nedd4L C2, aa 1-160) was also concentrated to membrane tubules, whereas C2 domain-deleted Nedd4L (Myc-Nedd4L ΔC2) was not. In contrast to FCHO2, the FBP17 BAR domain did not recruit Nedd4L to membrane tubules. Additionally, the BAR domain protein amphiphysin1 did not recruit Nedd4L to membrane tubules. Thus, Nedd4L is selectively recruited through the C2 domain to membrane tubules generated by the FCHO2 BAR domain. Similar results were obtained in HEK293 cells (Fig 4B). In these cells, membrane tubules generated by amphiphysin1 were hardly detected. It showed a punctate staining pattern and did not co-localized with Nedd4L.</p>\n",
       "<p>We also investigated whether the FCHO2 BAR domain not only recruits but also activates Nedd4L in cells. For this purpose, we performed an in vivo ubiquitination assay using HEK293 cells, which formed membrane tubules upon FCHO2 expression more efficiently than did COS7 cells: 91 ± 2% of HEK293 cells vs. 32 ± 2% of COS7 cells [the mean ± standard error of the mean (S.E.M.) of four independent experiments]. As the catalytic activity of E3 Ub ligases is typically reflected in their autoubiquitination (Wiesner et al, 2007), we determined activity by measuring the ubiquitination level of Nedd4L itself. Expression of the FCHO2 BAR domain in cells remarkably enhanced Nedd4L autoubiquitination, whereas FBP17 or amphiphysin1 did not (Fig 5A). Tubulation-deficient FCHO2 mutants (K146E+R152E and L136E) (Henne et al, 2010) did not enhance Nedd4L activity (Fig 5A–C and Fig EV3). A catalytically inactive Nedd4L mutant, C922A, which was recruited to FCHO2-generated membrane tubules, showed little enhancement of ubiquitination upon FCHO2 expression (Fig 5D and E), indicating that Nedd4L ubiquitination is self-induced. Using αENaC as a substrate, we further confirmed that Nedd4L is activated upon FCHO2 expression in cells (Fig 5F).</p>\n",
       "<p></p>\n",
       "<p>Preference of Nedd4L for FCHO2-generated membrane curvature</p>\n",
       "<p>The aforementioned results prompted us to investigate whether FCHO2 and Nedd4L interact through membrane tubules rather than through stereotyped protein–protein interactions. Membrane tubules generated by BAR domains in cells have specific diameters (curvatures) (Henne et al, 2007; Peter et al, 2004; Takano et al, 2008; Takei et al, 1999; Yin et al, 2009). Nedd4L may preferentially bind to the specific degree of membrane curvature generated by FCHO2. To investigate this possibility, we first examined the biochemical properties of Nedd4L. Using a co-sedimentation assay with liposomes containing various percentages of PS, we found that the C2 domain of Nedd4L is a Ca2+-dependent PS-binding domain (Fig 6A and B). Submicromolar Ca2+ concentrations, which are comparable to intracellular levels in unstimulated cells, were sufficient for the Nedd4L C2 domain to bind to PS-containing liposomes. The C2 domain alone and full-length Nedd4L showed different Ca2+ requirements. The higher Ca2+ requirement of full-length Nedd4L may be due to the intramolecular interaction between the C2 and HECT domains. Increasing PS percentages in synthetic liposomes enhanced the liposome binding of the Nedd4L C2 domain, and PS requirements varied depending on Ca2+ concentration.</p>\n",
       "<p>We next examined whether Nedd4L preferentially binds to a specific degree of membrane curvature. For this purpose, liposomes of different sizes (curvatures) were prepared by stepwise extrusion through filter membranes with different pore sizes (0.8, 0.4, 0.2, 0.1, 0.05, and 0.03 µm) and sonication. Sonication or extrusion through a 0.03 µm pore-size filter was used to prepare the smallest liposomes. Transmission electron microscopy revealed that the actual liposome size was similar to the pore size for 0.2, 0.1, 0.05, and 0.03 µm pore-size liposomes, whereas it was significantly smaller for 0.8 and 0.4 µm pore-size liposomes (Fig 7A). Nedd4L showed the strongest binding to 0.05 µm pore-size liposomes (Fig 7B).</p>\n",
       "<p>Synaptotagmin I (SytI) tandem C2 domains can both generate and sense membrane curvature in vitro by penetrating their hydrophobic residues into the lipid bilayer (Martens et al, 2007). As a sensor, the binding affinity is considered to be highest for membranes that have a degree of curvature consistent with that generated by SytI itself. We investigated whether, in addition to sensing membrane curvature, the Nedd4L C2 domain can also generate membrane curvature in vitro and, if so, what diameter of membrane tubules is generated. As expected, transmission electron microscopy revealed that the C2 domain deformed liposomes into extensive tubulation (Fig 7C). It should be noted that Nedd4L does not generate membrane tubules in cells (see Fig 4A and B). The tubule diameter generated by Nedd4L in vitro was mainly 40–80 nm, which is consistent not only with the diameter of 0.05 µm pore-size liposomes, to which Nedd4L preferentially binds, but also with the diameter of membrane tubules generated by FCHO2 (Fig 7D). These results suggest that Nedd4L shows the highest binding affinity for the degree of membrane curvature generated by FCHO2. It is therefore likely that FCHO2-induced recruitment and activation of Nedd4L are mediated by the degree of membrane curvature.</p>\n",
       "<p></p>\n",
       "<p>Roles of conserved hydrophobic residues in the Nedd4L C2 domain</p>\n",
       "<p>We compared the C2 domain sequence of Nedd4L to those of other Nedd4 family members, SytI, and PKCγ (Fig 8A). SytI C2 domains possess hydrophobic residues (M173, F234, V304, and I367) for penetration into the lipid bilayer to generate and sense membrane curvature (Martens et al, 2007). The Nedd4L C2 domain also has three hydrophobic residues (I37, F38, and L99), which are conserved among Nedd4 family members except for WWP2. We mutated these residues to alanine to produce six mutants, I37A, F38A, L99A, I37A+F38A, I37A+L99A, and I37A+F38A+L99A (triple-A). Any single mutation did not inhibit Nedd4L recruitment to FCHO2-generated membrane tubules (Fig 8B). However, double and triple mutations severely impaired this recruitment. Furthermore, the I37A+F38A and triple-A mutants showed little liposome binding and lost curvature generation (Fig 8C and D). Thus, these hydrophobic residues (I37, F38, and L99) are critical for both membrane binding and curvature sensing and generation. These residues are likely inserted into the lipid bilayer, thereby strengthening the C2 domain interaction with membranes and enabling the sensing and generation of membrane curvature. It has been shown that I37 and F38 are directly involved in Ca2+ binding (Escobedo et al, 2014). The residues may be inserted into the lipid bilayer while binding Ca2+.</p>\n",
       "<p>In the C2 domain of Smurf2, another Nedd4 family member, conserved hydrophobic residues (F29 and F30) are responsible for interactions with the HECT domain (Fig 8A) (Wiesner et al, 2007). Mutation at these residues impairs the interaction between the C2 and HECT domains, resulting in enhanced Smurf2 autoubiquitination by relieving autoinhibition. Consistent with these previous findings, we found that the Nedd4L I37A+F38A mutant, which was not recruited to FCHO2-generated membrane tubules, enhanced autoubiquitination, irrespective of FCHO2 expression (Fig 5D and E). These results suggest that I37 and F38 are critical for interactions with the HECT domain as well as for membrane binding and curvature sensing. Interestingly, the Nedd4L I37A+F38A mutant, as well as the C922A mutant, showed little αENaC ubiquitination (Fig EV4), indicating that membrane binding of Nedd4L is critical for ENaC ubiquitination.</p>\n",
       "<p></p>\n",
       "<p>In vitro activation of Nedd4L by FCHO2-generated membrane curvature</p>\n",
       "<p>To examine the properties of Nedd4L catalytic activity, we carried out an in vitro ubiquitination assay using an αENaC peptide as a substrate. The αENaC peptide possesses a conserved PY motif for interaction with the Nedd4L WW domain, and it is fused with monomeric streptavidin (mSA) (Lim et al, 2013) to bind to liposomes containing biotinylated phospholipids (biotinylated liposomes) (Sakamoto et al, 2017) (Fig 9A and B). We found that Ca2+ and PS were required for Nedd4L catalytic activity (Fig 9C and D), and that 0.05 µm pore-size liposomes were most effective in stimulating Ca2+- and PS-dependent Nedd4L activity (Fig 9E). Thus, the properties of Nedd4L catalytic activity are similar to those of its liposome binding. Furthermore, we found that Nedd4L did not ubiquitinate PY motif-mutated (Y644A in αENaC) mSA-αENaC (Appendix Fig S3). These findings are consistent with defects in ENaC ubiquitination observed in Liddle syndrome.</p>\n",
       "<p>We next examined whether membrane localization of mSA-αENaC affects the membrane binding and catalytic activity of Nedd4L. Nedd4L bound only slightly to mSA-αENaC on PS-free (0% PS) liposomes, but the liposome binding was synergistically increased by elevating the PS percentage in liposomes (Fig 9F). By contrast, when free biotin competitively inhibited the binding of mSA-αENaC to biotinylated liposomes, the PS-dependent liposome binding of Nedd4L was significantly reduced, resulting in remarkable inhibition of mSA-αENaC ubiquitination (Fig 9D and F). Thus, membrane localization of the PY motif is critical for membrane binding and activation of Nedd4L. It is, therefore, likely that the PY motifs on membranes, such as ENaC, play a role in enhancing Nedd4L catalytic activity by potentiating membrane binding through interactions with the WW domain.</p>\n",
       "<p>We then reconstituted in vitro FCHO2-induced recruitment and activation of Nedd4L. When liposomes (20% PS) were used, the FCHO2 BAR domain enhanced the liposome binding of Nedd4L (Fig 10A) and mSA-αENaC ubiquitination (Fig 10B). This enhancement was lost when the strength of interaction of Nedd4L with liposomes was increased by elevating the PS percentage in liposomes (~50% PS) (Fig 10C and Fig EV5).　In contrast to FCHO2, neither amphiphysin1 nor FBP17 enhanced Nedd4L activity (Fig 10D and Fig EV6A). Similarly, tubulation-deficient FCHO2 mutant (L136E) did not enhance Nedd4L activity (Fig 10E and Fig EV6A). These results are in good agreement with those obtained from in vivo experiments on FCHO2-induced recruitment and activation of Nedd4L. Electron microscopic analysis revealed that mSA-αENaC-associated liposomes were deformed by membrane-sculpting proteins into tubules with diameters similar to those of mSA-αENaC-free liposomes (Fig EV6B and see also Fig 7D). The diameter of membrane tubules generated by FCHO2 is consistent with that of 0.05 µm pore-size liposomes which are most effective in stimulating Nedd4L activity.</p>\n",
       "<p>To confirm whether FCHO2-induced recruitment and activation of Nedd4L are mediated by liposomes (i.e., membrane curvature) rather than by protein–protein interactions, we conducted two sets of experiments. Initially, we employed synthetic liposomes where 20% PS was substituted with 5% PI(4,5)P2. The FCHO2 BAR domain bound to PI(4,5)P2 as well as PS with high affinity, whereas the Nedd4L C2 domain did not (Fig 11A and B). The FCHO2 BAR domain is known to induce the tubulation of liposomes [5% PI(4,5)P2] (Henne et al, 2010). If Nedd4L could interact through protein–protein interactions with the FCHO2 BAR domain binding to liposomes [5% PI(4,5)P2], the BAR domain would facilitate the co-precipitation of Nedd4L in a co-sedimentation assay, thereby stimulating mSA-αENaC ubiquitination. However, the FCHO2 BAR domain did not demonstrate such interactions (Fig 11C and D). Therefore, it is unlikely that Nedd4L interacts with the FCHO2 BAR domain on liposomes through protein–protein interactions. Secondly, we employed 0.05 µm pore-size liposomes that are most effective in stimulating Nedd4L activity. The diameter of 0.05 µm pore-size liposomes is consistent with that of membrane tubules generated by the FCHO2 BAR domain. The BAR domain is, thus, expected not to induce the tubulation of 0.05 µm pore-size liposomes. The FCHO2 BAR domain showed similar binding activity toward 0.8 and 0.05 µm pore-size liposomes (Fig 12A). When 0.8 µm pore-size liposomes were used, the FCHO2 BAR domain enhanced Nedd4L activity (Fig 12B). However, in the case of 0.05 µm pore-size liposomes, curvature-stimulated Nedd4L activity was not potentiated by the addition of the FCHO2 BAR domain. If Nedd4L had interacted with the BAR domain, we would have expected potentiation of Nedd4L activity. These results strongly suggest that FCHO2-induced activation of Nedd4L is mediated by membrane curvature and not by protein–protein interactions.</p>\n",
       "<p></p>\n",
       "<p>Discussion</p>\n",
       "<p></p>\n",
       "<p>Nedd4L represents the ancestral Ub ligase with strong similarity to yeast Rsp5 which plays a key role in the trafficking, sorting, and degradation of a large number of proteins (Yang &amp; Kumar, 2010). Consistently, Nedd4L regulates a growing number of substrates, including membrane receptors, transporters, and ion channels. Although Nedd4L exists in a catalytically autoinhibited state due to an intramolecular interaction between the N-terminal C2 and C-terminal HECT domains, little is known about its activation mechanism. In this study, we clearly demonstrated that Nedd4L is activated by FCHO2 F-BAR domain-induced membrane curvature. It has been shown that the activity of synptojanin-1, a phosphoinositide phosphatase, is modulated by membrane curvature (Chang-Ileto et al, 2011). Taken together, it is conceivable that membrane curvature serves as a signal for the activation of various enzymes involved in membrane trafficking.</p>\n",
       "<p></p>\n",
       "<p>Unique Properties of the Nedd4L C2 Domain</p>\n",
       "<p>Although both sytI and Nedd4L C2 domains are Ca2+-dependent PS-binding domains that can generate and sense membrane curvature, their properties show a number of significant differences. An interesting difference exists in the degrees of membrane curvatures generated by sytI and Nedd4L: the diameters of their membrane tubules are about 15–20 nm and 40–80 nm, respectively (Hui et al, 2009; Martens et al, 2007). This difference is not simply due to that between tandem and single C2 domains, because it has been shown that the sytI C2B domain alone generates membrane tubules with a similar diameter (Hui et al, 2009). SytI and Nedd4L C2 domains must therefore have respective unique structures that determine the specific degrees of membrane curvatures that they sense and generate.</p>\n",
       "<p>Another important difference between sytI and Nedd4L C2 domains exists in their Ca2+ requirements. SytI C2 domains require Ca2+ concentrations in the submillimolar range to generate membrane curvature (Hui et al, 2009). In contrast, submicromolar Ca2+ concentrations, which are comparable to intracellular levels in unstimulated cells, are sufficient for the Nedd4L C2 domain to sense membrane curvature. This is consistent with the observation that Nedd4L is recruited and activated in serum-starved cells. The different Ca2+ requirements between sytI and Nedd4L C2 domains are reflected in their functions. SytI C2 domains induce membrane curvature in response to an increase in intracellular Ca2+ concentration, which is essential for Ca2+-triggered membrane fusion (Hui et al, 2009; Martens et al, 2007). Thus, sytI C2 domains serve as a Ca2+ sensor and a curvature generator. In contrast, the Nedd4L C2 domain recognizes a specific degree of curvature without an increase in intracellular Ca2+ concentration. Therefore, the Nedd4L C2 domain serves as a curvature sensor to regulate the localization and activity of the E3 Ub ligase.</p>\n",
       "<p>A striking feature that distinguishes the Nedd4L C2 domain from other C2 domain proteins (except for at least a subset of Nedd4 family members) is that the Nedd4L C2 domain interacts not only with membranes but also with the intramolecular catalytic domain for its autoinhibition. Mutational experiments have revealed that I37 and F38 residues in the C2 domain are critical for interactions with both membranes and the HECT domain, suggesting that these two interactions are competitive. This idea is supported by the observation that the Ca2+ EC50 value of full-length Nedd4L is much higher than that of the C2 domain. The HECT domain in full-length Nedd4L may competitively inhibit the interaction of the C2 domain with membranes. It is likely that membranes displace the HECT domain from the C2 domain when the strength of the C2 domain interaction with membranes is increased by the preferred membrane curvature. This displacement relieves the autoinhibition, resulting in Nedd4L activation. Thus, the membrane binding of Nedd4L is coupled with its activation.</p>\n",
       "<p></p>\n",
       "<p>Stimulatory and inhibitory effects of the FCHO2 BAR domain on the liposome binding and catalytic activity of Nedd4L</p>\n",
       "<p>The FCHO2 BAR domain enhances both the liposome binding and catalytic activity of Nedd4L when the interaction strength of Nedd4L with liposomes is weak (20% PS) (Fig 10A and B). Conversely, the BAR domain inhibits the binding and activity of Nedd4L when the interaction of Nedd4L with liposomes is increased by elevating the PS percentage in liposomes (~50% PS) (Fig 10C and Fig EV5). Thus, the FCHO2 BAR domain exerts both stimulatory and inhibitory effects on Nedd4L, depending on the PS percentage in liposomes. However, given that 20% PS is comparable to the concentration in the inner leaflet of the plasma membrane, it is plausible that FCHO2 exhibits only a stimulatory effect on Nedd4L in cells. The mechanism underlying the stimulatory and inhibitory effects in vitro is as follows: when using liposomes (20% PS), Nedd4L hardly binds to liposomes through PS (Fig. 9F and 10A). The addition of the FCHO2 BAR domain enhances the strength of Nedd4L’s interaction with PS by inducing membrane curvature. Consequently, Nedd4L gains a new binding to liposomes through PS, which augments its catalytic activity. In contrast, when using liposomes (~50% PS), Nedd4L binds to liposome through PS. The addition of the FCHO2 BAR domain hinders the PS-mediated liposome binding of Nedd4L, because FCHO2 and Nedd4L are PS-binding proteins and compete for PS binding in liposomes. Consequently, the BAR domain inhibits the catalytic activity of Nedd4L. This is consistent with the results obtained with 0.05 µm pore-size liposomes (Fig 12B). These 0.05 µm pore-size liposomes are comparable to liposomes with a higher PS percentage, as the interaction of Nedd4L with liposomes through PS is increased by membrane curvature. Therefore, when using 0.05 µm pore-size liposomes, the FCHO2 BAR domain inhibits the catalytic activity of Nedd4L.</p>\n",
       "<p>Based on a model in which a subset of BAR domain proteins, such as FCHO2 and FBP17, drive liposome tubulation in vitro (Frost et al, 2008; Shimada et al, 2007), the FCHO2 BAR domain is considered to polymerize on the liposome surface into spiral dense coats by lateral and tip-to-tip interactions. When using liposomes (~50% PS), it is conceivable that the FCHO2 BAR domain densely packs the liposomes, leaving little free space for Nedd4L binding. Conversely, when using liposomes (20% PS), it is likely that the liposomes are loosely packed, providing ample surface space for Nedd4L binding.</p>\n",
       "<p></p>\n",
       "<p>Model for ENaC Endocytosis</p>\n",
       "<p>Based on the present study, we propose a model for ENaC endocytosis through ubiquitination (Fig 13). First, FCHO2 binds to the plasma membrane and generates a specific degree of membrane curvature. FCHO2 interacts with clathrin adaptor proteins, such as Eps15, and recruits the clathrin machinery for CCP formation (Uezu et al, 2011; Henne et al, 2010). Eps15 has Ub-interacting motifs (Di Fiore et al, 2003). Nedd4L is subsequently recruited to and activated by FCHO2-generated curved membrane at the rim of nascent CCPs that ENaC enters. ENaC is then ubiquitinated and captured by Eps15 which FCHO2 interacts with. Thus, ubiquitinated ENaC is spatially restricted to CCPs and separated from unmodified ENaC. Overall, our findings suggest that in the early stage of CME, a membrane deformation induced by FCHO2 can program, through both the recruitment and activation of Nedd4L, the gathering of a subclass of cargo molecules in CCPs.</p>\n",
       "<p></p>\n",
       "<p>Methods</p>\n",
       "<p></p>\n",
       "<p>Construction</p>\n",
       "<p>Human αENaC, βENaC and γENaC cDNAs were kindly supplied by Dr. H. Kai (Kumamoto University, Kumamoto, Japan) (Sugahara et al, 2009). FLAG-αENaC was constructed by replacing the sequence [amino acid (aa) 169–174, TLVAGS] in the extracellular region with the FLAG sequence (DYKDDDK) as described previously (Firsov et al, 1996). This introduction of the epitope tag was carried out by polymerase chain reaction (PCR). FLAG-αENaC, βENaC, and γENaC were subcloned into pTRE-Tight-Hygro, pCAGI-Puro (Miyahara et al, 2000), and pCAGI-Bst (Umeda et al, 2006), respectively. pTRE-Tight-Hygro was constructed by inserting the hygromycin-resistance gene into pTRE-Tight (Clontech). FLAG-αENaC was also subcloned into pCAGI-Hygro, which was constructed by replacing the puromycin-resistance gene of pCAGI-Puro with the hygromycin-resistance gene. Human Nedd4L (NM_015277, isoform 3) cDNA was obtained by reverse transcription (RT)-PCR and cloned in pCMV-Myc (Miyahara et al, 2000). Nedd4L was also cloned in pCAGI-puro-EGFP and pGEX-6P-hexahistidine (His6). pGEX-6P-His6 was constructed from pGEX-6P (GE Healthcare) to express an N-terminal GST-tagged, C-terminal His6-tagged protein. pCAGI-puro-EGFP was constructed by subcloning EGFP cDNA into pCAGI-puro to express a C-terminal GFP-tagged protein. The Nedd4L C2 domain (C2, aa 1–160) and C2 domain-deleted mutant (ΔC2, aa 150–955) were subcloned into pCMV-Myc and/or pGEX-6P. Mouse FCHO2 (full length, aa 1–809; BAR domain, aa 1–302) and human FBP17 (BAR domain, aa 1–300) were subcloned into pEGFP-C1 and/or pGEX-6P (Uezu et al, 2011; Tsujita et al, 2006). FCHO2 (aa 1–327) were subcloned into pGEX-6P. Human amphiphysin1 cDNA was obtained by RT-PCR and cloned in pEGFP-N3. Amphiphysin1 (BAR domain, aa 1–377) was then subcloned into pGEX-6P. mTagRFP-T-Clathrin-15 was a gift from Michael Davidson (Addgene plasmid # 58005 ; http://n2t.net/addgene:58005 ; RRID:Addgene_58005) (Shaner et al, 2008). Human hemagglutinin (HA)-tagged Ub cDNA in pCGN was kindly donated by Dr. M. Nakao (Kumamoto University, Kumamoto, Japan). Ub cDNA was subcloned into pET3a. Human UbcH7 cDNA was obtained by RT-PCR and cloned in pGEX-6P. pET21d-Ube1 was a gift from Cynthia Wolberger (Addgene plasmid # 34965 ; http://n2t.net/addgene:34965 ; RRID:Addgene_34965) (Berndsen &amp; Wolberger, 2011). pRSET-mSA was a gift from Sheldon Park (Addgene plasmid # 39860 ; http://n2t.net/addgene:39860 ; RRID:Addgene_39860) (Lim et al, 2013). mSA-αENaC constructed in pRSET-mSA to express a fusion protein of the N-terminal (aa 7–40) and C-terminal (aa 625–659) intracellular regions of αENaC (see Fig. 9A). Mutagenesis was performed by PCR using mutated primers and a site-directed mutagenesis kit (Stratagene).</p>\n",
       "<p></p>\n",
       "<p>Protein expression and purification</p>\n",
       "<p>Expression and purification of GST-tagged proteins were performed using standard protocols. His6-tagged proteins were purified with Talon metal affinity resin (Clontech) using the manufacturer’s protocol. The GST tag was cleaved off by PreScission protease (GE Healthcare). The GST tag of BAR domains did not inhibit liposome binding or tubulation, whereas the GST tag of Nedd4L inhibited these functions. As FCHO2 (BAR domain, aa 1–302) was easily aggregated by the removal of the GST tag, FCHO2 (aa 1–327) was used instead as a GST-cleaved form of the FCHO2 BAR domain. Full-length Nedd4L (N-terminal GST-tagged, C-terminal His6-tagged) was purified by glutathione–Sepharose (GE Healthcare) column chromatography. The GST tag was cleaved off and the Nedd4L sample was then subjected to Talon metal affinity column chromatography. After the eluate was passed through glutathione–Sepharose beads, the sample was dialyzed against buffer A (20 mM Tris-HCl at pH 7.5, 100 mM KCl, 0.5 mM EDTA, 1 mM DTT, and 20 % sucrose). The Nedd4L sample was stored at −80°C until use. Ub was expressed in E. coli strain BL21 (DE3) pJY2 and purified as previously described (Pickart &amp; Raasi, 2005). His6-tagged mSA-αENaC was expressed in E. coli strain BL21 (DE3) pLysS and purified as previously described (Lim et al, 2013) with slight modifications. Briefly, the inclusion body was isolated and solubilized in buffer B (50 mM Tris-HCl at pH 8.0 and 6 M guanidine hydrochloride). The sample was then subjected to Talon metal affinity column chromatography. The eluate was added drop by drop to ice-cold refolding buffer (50 mM Tris-HCl at pH 8.0, 150 mM NaCl, 0.15 mg/ml D-biotin, 0.2 mg/ml oxidized glutathione, and 1 mg/ml reduced glutathione) under rapid stirring. The sample was then concentrated using a Centriprep centrifugal filter unit (Millipore) and centrifuged to remove aggregates. The supernatant was dialyzed against buffer C (20 mM Tris-HCl at pH 8.0). The mSA-αENaC sample was further purified by Mono Q anion exchange column chromatography as follows. The sample was applied to a Mono Q 5/50 GL column (GE Healthcare) equilibrated with buffer C. Elution was performed with a 50 ml linear gradient of NaCl (0–200 mM) in buffer C. Fractions of 1 ml each were collected. The active fractions (fraction 18–22) were collected and dialyzed against buffer D (10 mM Tris-HCl at pH 7.5, 100 mM KCl, and 0.5 mM EDTA). The mSA-αENaC sample was stored at −80°C until use.</p>\n",
       "<p></p>\n",
       "<p>Antibodies</p>\n",
       "<p>Rabbit anti-Nedd4L antibody was raised against GST-Nedd4L C2 (aa 1–160). Rabbit anti-FCHO2 antibody was obtained as previously described (Uezu et al, 2011). The following antibodies were purchased from commercial sources: mouse anti-Myc (9E10) (American Type Culture Collection); mouse anti-FLAG (M2) and mouse anti-α-tubulin (clone DM1A) (Sigma-Aldrich); mouse anti-clathrin heavy chain (BD Biosciences); rabbit anti-GFP (MBL Co.); rabbit anti-αENaC (Calbiochem); rabbit anti-βENaC (Proteintech); rabbit anti-γENaC (Abcam); mouse anti-transferrin receptor H68.4 (Santa Cruz); mouse anti-Ub antibody P4D1 (Santa Cruz); and secondary antibodies conjugated with Alexa Fluor 488, 594, and 647 (Invitrogen).</p>\n",
       "<p></p>\n",
       "<p>Cell culture and transfection</p>\n",
       "<p>HeLa, HEK293, and COS7 cells were cultured in Dulbecco’s modified Eagle medium (DMEM) supplemented with 10% fetal calf serum. HEK293 cells were grown on poly-L-lysine-coated dishes. Transfection was performed by using Lipofectamine 2000 (Invitrogen) according to the manufacturer’s protocol.</p>\n",
       "<p></p>\n",
       "<p>Generation of stable HeLa cells expressing all three ENaC subunits</p>\n",
       "<p>HeLa cells were transfected with the pTet-On Advanced vector (Clontech) to develop stable Tet-On cell lines for a doxycycline (Dox)-inducible expression system according to the manufacturer’s protocol. Tet-On cells were selected in 0.5 mg/ml G418. Cloned Tet-On cells were then transfected with pCAGI-Bst-γENaC and selected in 4 µg/ml blasticidin (in the presence of G418). Blasticidin-resistant cells were transfected with pCAGI-Puro-βENaC and selected in 1 µg/ml puromycin (in the presence of G418 and blasticidin). Puromycin-resistant cells were finally transfected with pTRE-Tight-Hygro-FLAG-αENaC and cultured in 0.2 mg/ml hygromycin (in the presence of G418, blasticidin, and puromycin). After selection, cells were cloned and maintained in 0.5 mg/ml G418, 0.2 mg/ml hygromycin, 1 µg/ml puromycin, and 2.5 µg/ml blasticidin. FLAG-tagged αENaC expression was induced by treating cells with 1 µg/ml Dox in the presence of 40 µM amiloride overnight. Amiloride was added to prevent cellular Na+ overload.</p>\n",
       "<p></p>\n",
       "<p>Immunofluorescence microscopy</p>\n",
       "<p>Cells were fixed with 3.7% formaldehyde in phosphate-buffered saline (PBS) at room temperature for 15 min, and then permeabilized with 0.2% Triton X-100 in PBS for 10 min. After blocking with 1% bovine serum albumin (BSA) in PBS, cells were sequentially incubated with primary antibodies and fluorescence-conjugated secondary antibodies at room temperature for 1 h, respectively. Cells were then analyzed with either a fluorescence microscope (BX51; Olympus) or a confocal microscopy system (BX50, Fluoview FV300, and Fluoview FV1200; Olympus). A 60 × oil immersion objective (NA = 1.40, Olympus) was used. All comparable images were acquired under identical parameters. To visualize cell-surface FLAG-αENaC in αβγENaC-HeLa cells, cells were incubated with 5 µg/ml anti-FLAG M2 antibody at 4°C for 1 h to prevent internalization. Cells were then fixed, permeabilized, and stained with a secondary antibody. Cells were also sequentially incubated with primary (anti-FCHO2) and secondary antibodies. To quantify co-localization, the Pearson's correlation coefficient was calculated from three cells using the Fiji (ImageJ) Coloc2 plug-in.</p>\n",
       "<p></p>\n",
       "<p>RNA interference</p>\n",
       "<p>Stealth™ double-stranded RNAs were purchased from Invitrogen. siRNA sequences of human FCHO2, Nedd4L, and FBP17 were 5′-CCA CAG AUC UUA GAG UGG AUU AUA A-3′ (corresponding to nucleotides 2048–2072 relative to the start codon), 5′-GAA GAG UUG CUG GUC UGG CCG UAU U-3′ (corresponding to nucleotides 1778–1802 relative to the start codon), and 5′-CCC ACU UCA UAU GUC GAA GUC UGU U-3′for human FBP17 (corresponding to nucleotides 1804–1828 relative to the start codon), respectively. A double-stranded RNA targeting luciferase (GL-2, 5′-CGU ACG CGG AAU ACU UCG AAA UGU C-3′) was used as a control. HeLa cells were transfected with 20 nM siRNA using Lipofectamine 2000 according to the manufacturer’s protocol. After 24 h, a second transfection was performed, and cells were cultured for 3 days and subjected to various experiments. For rescue experiments, cells were transfected with the intended plasmid at 36 h after the second transfection. After 18 h, cells were fixed and analyzed.</p>\n",
       "<p></p>\n",
       "<p>Quantitative real-time PCR analysis</p>\n",
       "<p>Total RNA was isolated using ISOGEN (Nippon Gene, 319-90211), and cDNA was synthesized using ReverTra Ace qPCR RT Master Mix with gDNA Remover (Toyobo, FSQ-301), following manufacturer’s instruction. Real time PCR was carried out using Thunderbird Next SYBR (Toyobo, QPX-201) on StepOnePlus Real-Time PCR Systems (Applied Biosystems). The sequences of qPCR primers are 5′-TCT CCC ATC GCT TCA ACG AG-3′ and 5′-GTT GCA CCC AGC TTG AGA GA-3′ for human FBP17; 5′-GTC TCC TCT GAC TTC AAC AGC G-3′ and 5′-ACC ACC CTG TTG CTG TAG CCA A-3′ for human GAPDH. The expression of target genes was normalized to GAPDH mRNA levels.</p>\n",
       "<p></p>\n",
       "<p>Endocytosis assay</p>\n",
       "<p>αβγENaC-HeLa cells at approximately 90% confluence in a 10-cm dish were starved with serum-free medium (DMEM containing 1% BSA, 1 µg/ml Dox, and 40 µM amiloride) at 37°C for 4 h. For ENaC endocytosis, cells were then incubated with 5 µg/ml anti-FLAG M2 in 2 ml of DMEM containing 1% BSA at 4°C for 1 h to prevent internalization. After three washes with ice-cold PBS containing 1 mM MgCl2 and 0.1 mM CaCl2 (PBS-CM), endocytosis was allowed by incubating cells in prewarmed DMEM containing 1% BSA at 37°C for the indicated periods of time. Endocytosis was stopped by placing cells on ice and washing them with ice-cold PBS-CM. Antibody bound to the cell surface but not internalized was removed by acid stripping with 0.2 M acetic acid and 0.5 M NaCl solution (pH 2.5). After washing with ice-cold PBS-CM, internalized ENaC was detected by immunoblotting (IB) or immunofluorescence microscopy. For IB, cells were scraped in 1 ml of lysis buffer A (50 mM Tris-HCl at pH 8.0, 150 mM NaCl, 5 mM EDTA, 1% Triton X-100, 1 mM PMSF, 20 µg/ml leupeptin, and 1 µg/ml pepstatin A). Cells were solubilized at 4°C for 1 h on a rotating wheel, and centrifuged at 20,000 × g at 4°C for 15 min. Protein G-Sepharose beads (GE Healthcare) were added to the supernatant and incubated at 4°C for 3 h. The beads were then thoroughly washed with lysis buffer A and boiled in sodium dodecyl sulfate (SDS) sample buffer. The sample was subjected to SDS-polyacrylamide gel electrophoresis (PAGE), followed by IB with anti-ENaC subunit antibodies. Internalization was expressed as a percentage of the initial amount of cell-surface ENaC subunit determined by incubating cells with anti-FLAG antibody at 4°C for 1 h and lysing them without incubation at 37°C or acid stripping. For immunofluorescence microscopy, cells were transfected with control or FCHO2 siRNA and then with GFP-siRNA-resistant (sr) FCHO2 mutant. After cells were labeled with anti-FLAG antibody on 4°C, they were incubated at 37°C for 10 min to allow ENaC endocytosis. After acid stripping, cells were fixed with 3.7% formaldehyde and permeabilized with 0.2% Triton X-100. The antibody-labeled, internalized αENaC was visualized with a fluorescence-conjugated secondary antibody. The percentage of cells with internalized αENaC among cells expressing GFP-sr-FCHO2 was determined using MetaMorph imaging System software. All images were acquired under identical parameters. For TfR endocytosis, a cell surface biotinylation assay was carried out as previously described (Roberts et al, 2001). Briefly, the cell surface was labeled with Sulfo-NHS-SS-Biotin (Thermo Fisher Scientific) at 4°C. After cells were incubated with ice-cold quenching buffer (50 mM NH4Cl in PBS-CM) at 4°C, they were incubated with serum free medium for 10 min at 37°C. Biotin was removed from the cell surface by incubation with ice-cold MesNa buffer (50 mM MesNa, 100 mM NaCl, and 100 mM Tris-HCl, pH 8.8). Cells were then incubated with iodoacetamide at 4°C. Cell lysates were then incubated with NeutrAvidin beads (Thermo Fisher Scientific). The beads were boiled in SDS sample buffer. Proteins were analyzed by IB with TfR antibody.</p>\n",
       "<p></p>\n",
       "<p>ENaC expression and ubiquitination at the cell surface</p>\n",
       "<p>αβγENaC-HeLa cells at approximately 90% confluence in a 10-cm dish were starved with serum-free medium (DMEM containing 1% BSA, 1 µg/ml Dox, and 40 µM amiloride) at 37°C for 4 h, washed three times with ice-cold PBS, and incubated at 4°C for 45 min with 2 ml of 1 mg/ml EZ-link Sulfo-NHS-SS-Biotin in PBS with gentle shaking. Subsequently, cells were extensively washed with ice-cold Tris-buffered saline, and then scraped into 1 ml of lysis buffer B (50 mM Tris-HCl at pH 7.5, 150 mM NaCl, 5 mM EDTA, 1% Triton X-100, 10 mM N-ethylmaleimide, 1 mM PMSF, 10 µg/ml leupeptin, and 1 µg/ml pepstatin A) containing 40 µM MG132. Cells were solubilized at 4°C for 1 h on a rotating wheel, and centrifuged at 20,000 × g at 4°C for 15 min. The supernatant (4 mg of protein) was incubated at 4°C for 3 h with 10 µg of anti-FLAG M2 antibody. Protein G-Sepharose beads (50 µl of 50% slurry) were then added to the sample, which was incubated at 4°C for 3 h. After beads were thoroughly washed with lysis buffer B containing 40 µM MG132, FLAG-tagged proteins were eluted from the beads with 200 µg/ml FLAG peptide (Sigma-Aldrich) in 0.5 ml of lysis buffer B. After centrifugation, the supernatant was incubated with 25 µl of NeutrAvidin beads at 4°C overnight. After the beads were thoroughly washed with lysis buffer B, bound proteins were eluted by incubating the beads at 4°C for 1 h with 150 µl of lysis buffer B containing 200 mM DTT. After centrifugation, 5 × SDS sample buffer was added to the supernatant, which was then analyzed by IB with anti-FLAG and anti-Ub antibodies. The chemiluminescence intensity was quantified by scanning using ImageJ software (NIH).</p>\n",
       "<p></p>\n",
       "<p>TIRF Microscopy</p>\n",
       "<p>For live observation with TIRF microscopy, HeLa cells were grown on a glass-bottomed dish as described previously (Shimada et al, 2007). Cells were observed at 10–12 h after transfection as described previously (Henne et al, 2010). Images were acquired for 5 min at 1 s intervals with a TIRF microscopy system (Olympus), a 100 × oil immersion objective NA = 1.45 (Olympus) and MetaMorph Imaging System Software. All comparable images were acquired under identical parameters.</p>\n",
       "<p></p>\n",
       "<p>Liposome preparation</p>\n",
       "<p>Phospholipids were purchased from commercial sources: phosphatidylcholine (PC, 1,2-dioleoyl-sn-glycero-3-phosphocholine), phosphatidylethanolamine (PE, 1,2-dioleoyl-sn-glycero-3-phosphoethanolamine), PS (1,2-dioleoyl-sn-glycero-3-phospho-l-serine), rhodamine-PE [18:1 L-α-PE-N-(lissamine rhodamine B sulfonyl)], PEG2000-biotin-PE (1,2-distearoyl-sn-glycero-3-phosphoethanolamine-N-[biotinyl (polyethyleneglycol)-2000]), porcine brain PS, brain PI(4,5)P2, and porcine brain total lipid extract were obtained from Avanti Polar Lipids; cholesterol and Folch lipids [bovine brain lipids (Folch Fraction 1)] were obtained from Sigma-Aldrich; X-biotin-PE [N-((6-(biotinoyl)amino)hexanoyl)-1,2-dihexadecanoylsn-glycero-3-phosphoethanolamine] was obtained from Thermo Fisher Scientific; and fluorescein-PE [N-(fluorescein-5-thiocarbamoyl)-1,2-dihexadecanoyl-sn-glycero-3-phosphoethanolamine] was obtained from AAT Bioquest Inc.</p>\n",
       "<p>Liposomes were prepared using brain lipids (Folch lipids, ~50% PS of total lipids), brain lipids (20% PS of total lipids, a mixture of 89.5% porcine brain total lipid extract and 10.5% porcine brain PS), or a synthetic lipid mixture [70% PC, 20% PE, 10% cholesterol (w/w), and varying percentages of PS or PI(4,5)P2 (with a corresponding reduction in PC)]. Lipid concentrations were monitored by including 1% rhodamine-PE (w/w). To prepare mSA-αENaC–associated liposomes, 2% PEG2000-biotin-PE (w/w) was included. Lipids were dried under nitrogen gas, desiccated for 2 h, and resuspended at 1 mg/ml in lipid buffer (50 mM HEPES-NaOH at pH 7.2, 100 mM KCl, 2 mM MgCl2, and 5 mM EGTA), followed by hydration at 37°C for 1 h. To make liposomes of different sizes, they were subjected to five freeze–thaw cycles and centrifuged at 100,000 × g for 15 min. Precipitated liposomes were then resuspended in the same buffer to make a final concentration of 1 mg/ml. They were subjected to extrusion through polycarbonate filter membranes (Avanti Polar Lipids): liposomes were sequentially passed 9 times through a 0.8-µm filter, 21 times through a 0.4-µm filter, 21 times through a 0.2-µm filter, 21 times through a 0.1-µm filter, 21 times through a 0.05-µm filter, and 21 times through a 0.03-µm filter. Instead of extrusion through a 0.03-µm filter, liposomes with the smallest size were also prepared by sonication three times for 5 s with a Branson Sonifier 25 at power level 1. Liposome diameters were examined by transmission electron microscopy.</p>\n",
       "<p></p>\n",
       "<p>Co-sedimentation assay</p>\n",
       "<p>A co-sedimentation assay was performed as described (Uezu et al, 2011) with slight modifications. Briefly, 5–10 µg of protein was incubated at 25°C for 10 min with 0.5 mg/ml liposomes in 100 µl of lipid buffer containing various concentrations of CaCl2 and centrifuged at 165,000 × g at 25°C for 15 min. Equal amounts of the supernatants and pellets were subjected to SDS-PAGE, followed by Coomassie brilliant blue (CBB) staining. Proteins were quantified by scanning using ImageJ (NIH). The assay with liposomes of different sizes was performed using NeutrAvidin (Thermo Fisher Scientific Inc) as described (Sakamoto et al, 2017). Briefly, brain-lipid (Folch-lipid) liposomes (~50% PS) were supplemented with 1% X-biotin-PE and 1% rhodamine-PE, subjected to extrusion through filter membranes, and then sonicated. Purified Nedd4L protein (10 µg) was incubated with 0.2 mg/ml liposomes in 100 µl of lipid buffer containing 2mM CaCl2 (0.1 µM Ca2+). The sample was then incubated with 2 µg of NeutrAvidin and ultracentrifuged. Both total and supernatant fluorescence were measured. The proportion of precipitated liposomes of each size was ~100%.</p>\n",
       "<p>The assay with mSA-αENaC–associated liposomes was performed as follows. Liposomes were prepared from a synthetic lipid mixture [various percentages of PS or 5% PI(4,5)P2] or brain lipids supplemented with 2% PEG2000-biotin-PE and 1% rhodamine-PE. The liposomes (10 µg) were incubated with purified mSA-αENaC protein (35 pmol) at 4°C for 10 min in 25 µl of lipid buffer. The sample was then incubated at 25°C for 10 min with 0.07 µM (8 µg/ml) Nedd4L and/or 1.4 µM BAR domains in 50 µl of ubiquitination buffer (50 mM HEPES-NaOH at pH 7.2, 100 mM KCl, 2 mM MgCl2, 5 mM EGTA, and 0.1 mM DTT) containing 4 mM CaCl2 (0.7 µM Ca2+). Subsequently, the sample was subjected to ultracentrifugation. Equal amounts of the total samples and pellets were subjected to SDS-PAGE, followed by CBB staining and IB. The chemiluminescence intensity was quantified by scanning using ImageJ (NIH).</p>\n",
       "<p>The assay with mSA-αENaC-associated liposomes (0.05 µm pore-size) was done as follows. Liposomes were prepared from brain lipids (20% PS) supplemented with 2% PEG2000-biotin-PE, 1% rhodamine-PE, and 0.5% fluorescein-PE (w/w) and extruded through membrane filters. To precipitate liposomes, the sample (50 μl) was first incubated with 1 µg of anti-fluorescein goat antibody (Novus Biologicals) at room temperature for 1 h and then incubated with 0.5 µg of protein A/G/L (BioVision) at room temperature for 1 h. Subsequently, the sample was centrifuged at 165,000 × g at 25°C for 15 min. Equal amounts of the supernatants and pellets were subjected to SDS-PAGE, followed by CBB staining and IB. The percentage of precipitated liposomes was ~100%.</p>\n",
       "<p></p>\n",
       "<p>Calculation of free Ca2+ concentrations</p>\n",
       "<p>At 1.0, 2.0, 3.0, 3.5, 4.0, 4.5, and 4.9 mM CaCl2 in lipid buffer, free Ca2+ concentrations were calculated to be 0.04, 0.1, 0.3, 0.4, 0.7, 1.6, and 8.1 µM, respectively, as determined by the Ca-Mg-ATP-EGTA Calculator v1.0 using constants from the NIST database #46 v8. At 1.0, 2.0, 3.0, 3.5, 4.0, 4.5, and 4.9 mM CaCl2 in ubiquitination buffer, free Ca2+ concentrations were calculated to be 0.04, 0.1, 0.2, 0.4, 0.7, 1.4, and 6.7 µM, respectively.</p>\n",
       "<p></p>\n",
       "<p>In vitro tubulation assay</p>\n",
       "<p>An in vitro tubulation assay was performed as described (Uezu et al, 2011; Tsujita et al, 2006) with slight modifications. Briefly, liposomes were prepared from brain lipids (Folch lipids) supplemented with 1 % rhodamine-PE. The liposomes (0.2 mg/ml liposomes) were incubated with the Nedd4L C2 domain (4.5 µM) or BAR domains (1.4 µM) at 25°C for 10 min in lipid buffer containing 3 mM CaCl2 (0.3 µM Ca2+). The sample was examined by fluorescence or electron microscopy. The tubulation assay with mSA-αENaC-associated liposomes was performed using the sample mixture utilized in the co-sedimentation assay in which mSA-αENaC-associated liposomes (brain-lipid liposomes) were used.</p>\n",
       "<p></p>\n",
       "<p>Negative-staining electron microscopy</p>\n",
       "<p>Negative-staining transmission electron microscopy was performed as described (Shimada et al, 2007). Box and whisker plots were used to represent the distribution of liposome diameters.</p>\n",
       "<p></p>\n",
       "<p>In vivo ubiquitination assay</p>\n",
       "<p>HEK293 cells were used, because they formed membrane tubules upon FCHO2 expression more efficiently than did COS7 cells. HEK293 cells at approximately 90% confluence on a 6-well plate were transfected with 0.8 µg of pCMV-Myc-Nedd4L, 1.6 µg of pEGFP-BAR domain, and 1.6 µg of pCGN-HA-Ub. In some experiments, cells were transfected with various doses (0–1.5 µg) of pCMV-Myc-Nedd4L, 1.5 µg of pEGFP-FCHO2 BAR domain, 1 µg of pCGN-HA-Ub, and 0.5 µg of pCAGI-Hygro-FLAG-αENaC. Total transfected cDNA amount was held constant using empty pCMV-Myc vector. Amiloride (40 µM) was added to the medium when αENaC cDNA was transfected. At 21 h after transfection, cells were washed with PBS and incubated with serum-free DMEM containing 1% BSA. After 3 h incubation, 10 µM MG132 was added to the medium. After further 4 h incubation, cells were solubilized in 150 µl of lysis buffer C (50 mM Tris-HCl at pH 7.5, 150 mM NaCl, 5 mM EDTA, 10 mM N-ethylmaleimide, 1 mM PMSF, 10 µg/ml leupeptin, and 1µg/ml pepstatin A) containing 1% SDS. After boiling, the sample was sonicated and then centrifuged at 20,000 × g for 15 min. The supernatant was diluted 10-fold with lysis buffer C containing 1% Triton X-100 and then incubated at 4°C for 3 h with either anti-Myc or anti-FLAG M2 antibody. Protein G-Sepharose beads were added to the sample, which was then incubated at 4°C for 3 h. After the beads were thoroughly washed with lysis buffer C containing 1% Triton X-100, bound proteins were eluted by boiling in SDS sample buffer for 5 min. These proteins were then subjected to SDS-PAGE, followed by IB.</p>\n",
       "<p></p>\n",
       "<p>In vitro ubiquitination assay</p>\n",
       "<p>mSA-αENaC-associated liposomes were made by incubating liposomes (10 µg) with mSA-αENaC (35 pmol) in 25 µl of lipid buffer, as described for the co-sedimentation assay. They were then incubated at 25°C for 10 min with 0.07 µM Nedd4L, 0.04 µM E1 (Ube1), 0.7 µM E2 (UbcH7), 7 µM Ub, and various concentrations of BAR domains in 50 µl of ubiquitination buffer containing 4 mM CaCl2 (0.7 µM Ca2+) and 2 mM ATP. Where indicated, various concentrations of CaCl2 were used. The ubiquitination reaction was started by adding ATP and stopped by adding 10 µl of 6 × SDS sample. The sample was then boiled, subjected to SDS-PAGE, and immunoblotted with anti-Ub and anti-αENaC antibodies. The chemiluminescence intensity was quantified by scanning using ImageJ (NIH).</p>\n",
       "<p></p>\n",
       "<p>Other procedures</p>\n",
       "<p>Stripping immunoblots for re-probing was carried out with the WB Stripping Solution Strong (Nacalai Tesque) according to the manufacturer’s protocol. Protein concentrations were quantified with BSA as a reference protein by the Bradford method (Bio-Rad) or the BCA method (Thermo Fisher Scientific). SDS-PAGE was performed as described (Laemmli, 1970).</p>\n",
       "<p></p>\n",
       "<p>Statistical analysis</p>\n",
       "<p>All experiments were performed at least three times (biological replication) and representative results are shown. Data are shown as the mean ± S.E.M. Student’s tests, one-way analysis of variance test, and two-way analysis of variance test were used to evaluate statistical significances between different treatment groups.</p>\n",
       "<p></p>\n",
       "<p>Data availability</p>\n",
       "<p>This study includes no data deposited in external repositories.</p>\n",
       "<p></p>\n",
       "<p>Acknowledgments</p>\n",
       "<p></p>\n",
       "<p>We thank Drs. H. Kai and M. Nakao for kindly providing us with ENaC and Ub cDNAs, respectively. We are also grateful to Dr. T. Takenawa (Kobe University, Kobe, Japan) for helpful discussions. This work was partly carried out at the Institute of Molecular Embryology and Genetics, the Gene Technology Centre, and Research Facilities of the School of Medicine, Kumamoto University. This study was supported by JSPS KAKENHI Grant Numbers 19K06643 (to H.N.), 19K06544 (to Y.S.), and Setsuro Fujii Memorial, The Osaka Foundation for Promotion of Fundamental Medical Research (to Y.S.) and by grants from the Mochida Memorial Foundation, the Takeda Science Foundation, and the Uehara Memorial Foundation (to K.K.).</p>\n",
       "<p></p>\n",
       "<p>Disclosure and Competing Interests Statement</p>\n",
       "<p>The authors declare no competing interests.</p>\n",
       "<p></p>\n",
       "<p></p>\n",
       "<p>References</p>\n",
       "<p></p>\n",
       "<p>Aguet F, Antonescu CN, Mettlen M, Schmid S &amp; Danuser G (2013) Advances in analysis of low signal-to-noise images link dynamin and AP2 to the function of an endocytic checkpoint. Dev Cell 26: 279–291</p>\n",
       "<p>Berndsen CE &amp; Wolberger C (2011) A spectrophotometric assay for conjugation of ubiquitin and ubiquitin-like proteins. Anal Biochem 418: 102–110</p>\n",
       "<p>Butterworth MB (2010) Regulation of the epithelial sodium channel (ENaC) by membrane trafficking. Biochim Biophys Acta 1802: 1166–77</p>\n",
       "<p>Chang-Ileto B, Frere SG, Chan RB, Voronov SV, Roux A, &amp; Di Paolo G (2011) Synaptojanin 1-mediated PI(4,5)P2 hydrolysis is modulated by membrane curvature and facilitates membrane fission. Dev. Cell 20: 206–218</p>\n",
       "<p>Doherty GJ &amp; McMahon HT (2009) Mechanisms of endocytosis. Annu Rev Biochem 78: 857–902</p>\n",
       "<p>Di Fiore PP, Polo S &amp; Hofmann K (2003) When ubiquitin meets ubiquitin receptors: a signalling connection. Nat Rev Mol Cell Biol 4: 491–497</p>\n",
       "<p>Escobedo A, Gomes T, Aragon E, Martin-Malpartida P, Ruiz L, &amp; Macias MJ (2014) Structural basis of the activation and degradation mechaisms of the E3 ubiquitin lagase Nedd4L. Structure 22: 1446–1457</p>\n",
       "<p>Firsov D, Schild L, Gautschi I, Mérillat AM, Schneeberger E &amp; Rossier BC (1996) Cell surface expression of the epithelial Na channel and a mutant causing Liddle syndrome: a quantitative approach. Proc Natl Acad Sci U S A 93: 15370–15375</p>\n",
       "<p>Frost A, Perera R, Roux A, Spasov K, Destaing O, Egelman EH, De Camilli P &amp; Unger VM (2008) Structural basis of membrane invagination by F-BAR domains. Cell 132: 807–817</p>\n",
       "<p>Henne WM, Boucrot E, Meinecke M, Evergren E, Vallis Y, Mittal R &amp; McMahon HT (2010) FCHo proteins are nucleators of clathrin-mediated endocytosis. Science 328: 1281–4</p>\n",
       "<p>Henne WM, Kent HM, Ford MGJ, Hegde BG, Daumke O, Butler PJG, Mittal R, Langen R, Evans PR &amp; McMahon HT (2007) Structure and Analysis of FCHo2 F-BAR Domain: A Dimerizing and Membrane Recruitment Module that Effects Membrane Curvature. Structure 15: 839–852</p>\n",
       "<p>Hershko A &amp; Ciechanover A (1998) THE UBIQUITIN SYSTEM. Annu Rev Biochem 67: 425–479</p>\n",
       "<p>Hughey RP, Mueller GM, Bruns JB, Kinlough CL, Poland PA, Harkleroad KL, Carattino MD &amp; Kleyman TR (2003) Maturation of the epithelial Na+ channel involves proteolytic processing of the alpha- and gamma-subunits. J Biol Chem 278: 37073–37082</p>\n",
       "<p>Hui E, Johnson CP, Yao J, Dunning FM &amp; Chapman ER (2009) Synaptotagmin-mediated bending of the target membrane is a critical step in Ca(2+)-regulated fusion. Cell 138: 709–21</p>\n",
       "<p>Ingham RJ, Gish G &amp; Pawson T (2004) The Nedd4 family of E3 ubiquitin ligases: functional diversity within a common modular architecture. Oncogene 23: 1972–84</p>\n",
       "<p>Laemmli UK (1970) Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature 227: 680–685</p>\n",
       "<p>Lehmann M, Lukonin I, Noe F, Schmoranzer J, Clementi C, Loerke D &amp; Haucke V (2019) Nanoscale coupling of endocytic pit growth and stability. Sci Adv 5: eaax5775</p>\n",
       "<p>Lemmon M (2008) Membrane recognition by phospholipid-binding domains. Nat Rev Mol Cell Biol 9: 99–111</p>\n",
       "<p>Lim KH, Huang H, Pralle A &amp; Park S (2013) Stable, high-affinity streptavidin monomer for protein labeling and monovalent biotin detection. Biotechnol Bioeng 110: 57–67</p>\n",
       "<p>Martens S, Kozlov MM &amp; McMahon HT (2007) How synaptotagmin promotes membrane fusion. Science 316: 1205–8</p>\n",
       "<p>Miyahara M, Nakanishi H, Takahashi K, Satoh-Horikawa K, Tachibana K &amp; Takai Y (2000) Interaction of nectin with afadin is necessary for its clustering at cell-cell contact sites but not for its cis dimerization or trans interaction. J Biol Chem 275: 613–618</p>\n",
       "<p>Nishizuka Y (1992) Intracellular signaling by hydrolysis of phospholipids and activation of protein kinase C. Science 258: 607–14</p>\n",
       "<p>Peter BJ, Kent HM, Mills IG, Vallis Y, Butler PJG, Evans PR &amp; McMahon HT (2004) BAR domains as sensors of membrane curvature: the amphiphysin BAR structure. Science 303: 495–9</p>\n",
       "<p>Pickart CM &amp; Raasi S (2005) Controlled synthesis of polyubiquitin chains. Methods Enzym 399: 21–36</p>\n",
       "<p>Roberts M, Barry S, Woods A, van der Sluijs P, &amp; Norman J (2001) PDGF-regulated rab4-dependent recycling of avb3 integrin from early endosomes is necessary for cell adhesion and spreading. Curr Biol 11: 1392–1402</p>\n",
       "<p>Rossier BC (2014) Epithelial sodium channel (ENaC) and the control of blood pressure. Curr Opin Pharmacol 15: 33–46</p>\n",
       "<p>Rotin D &amp; Kumar S (2009) Physiological functions of the HECT family of ubiquitin ligases. Nat Rev Mol Cell Biol 10: 398–409</p>\n",
       "<p>Rotin D &amp; Staub O (2011) Role of the ubiquitin system in regulating ion transport. Pflugers Arch 461: 1–21</p>\n",
       "<p>Saffarian S, Coccuci E, &amp; Kirchausen T (2009) Distinct dynamics of endocytic clathrin-coated pits and coated palques. PLoS Biol 7: e1000191</p>\n",
       "<p>Sakamoto Y, Kikuchi K, Umeda K &amp; Nakanishi H (2017) Effects of various spacers between biotin and the phospholipid headgroup on immobilization and sedimentation of biotinylated phospholipid-containing liposomes facilitated by avidin-biotin interactions. J Biochem 162: 221–226</p>\n",
       "<p>Shaner NC, Lin MZ, McKeown MR, Steinbach PA, Hazelwood KL, Davidson MW &amp; Tsien RY (2008) Improving the photostability of bright monomeric orange and red fluorescent proteins. Nat Methods 5: 545–551</p>\n",
       "<p>Shimada A, Niwa H, Tsujita K, Suetsugu S, Nitta K, Hanawa-Suetsugu K, Akasaka R, Nishino Y, Toyama M, Chen L, et al (2007) Curved EFC/F-BAR-domain dimers are joined end to end into a filament for membrane invagination in endocytosis. Cell 129: 761–772</p>\n",
       "<p>Sugahara T, Koga T, Ueno-Shuto K, Shuto T, Watanabe E, Maekawa A, Kitamura K, Tomita K, Mizuno A, Sato T, et al (2009) Calreticulin positively regulates the expression and function of epithelial sodium channel. Exp Cell Res 315: 3294–3300</p>\n",
       "<p>Takano K, Toyooka K &amp; Suetsugu S (2008) EFC/F-BAR proteins and the N-WASP–WIP complex induce membrane curvature-dependent actin polymerization. EMBO J 27: 2817–2828</p>\n",
       "<p>Takei K, Slepnev VI, Haucke V &amp; De Camilli P (1999) Functional partnership between amphiphysin and dynamin in clathrin-mediated endocytosis. Nat Cell Biol 1: 33–39</p>\n",
       "<p>Traub LM (2009) Tickets to ride: selecting cargo for clathrin-regulated internalization. Nat Rev Mol Cell Biol 10: 583–96</p>\n",
       "<p>Tsujita K, Suetsugu S, Sasaki N, Furutani M, Oikawa T &amp; Takenawa T (2006) Coordination between the actin cytoskeleton and membrane deformation by a novel membrane tubulation domain of PCH proteins is involved in endocytosis. J Cell Biol 172: 269–279</p>\n",
       "<p>Uezu A, Umeda K, Tsujita K, Suetsugu S, Takenawa T &amp; Nakanishi H (2011) Characterization of the EFC/F-BAR domain protein, FCHO2. Genes Cells 16: 868–878</p>\n",
       "<p>Umeda K, Ikenouchi J, Katahira-Tayama S, Furuse K, Sasaki H, Nakayama M, Matsui T, Tsukita S, Furuse M &amp; Tsukita S (2006) ZO-1 and ZO-2 independently determine where claudins are polymerized in tight-junction strand formation. Cell 126: 741–754</p>\n",
       "<p>Wang J, Peng Q, Lin Q, Childress C, Carey D &amp; Yang W (2010) Calcium activates Nedd4 E3 ubiquitin ligases by releasing the C2 domain-mediated auto-inhibition. J Biol Chem 285: 12279–12288</p>\n",
       "<p>Wiesner S, Ogunjimi AA, Wang H-R, Rotin D, Sicheri F, Wrana JL &amp; Forman-Kay JD (2007) Autoinhibition of the HECT-type ubiquitin ligase Smurf2 through its C2 domain. Cell 130: 651–862</p>\n",
       "<p>Yang B &amp; Kumar S (2010) Nedd4 and Nedd4-2: closely related ubiquitin-protein ligases with distinct physiological functions. Cell Death Differ 17: 68–77</p>\n",
       "<p>Yin Y, Arkhipov A &amp; Schulten K (2009) Simulations of Membrane Tubulation by Lattices of Amphiphysin N-BAR Domains. Structure 17: 882–892</p>\n",
       "<p>Zaccai N, Kadlecova Z, Dickson VK, Korobchevskaya K &amp; Traub L (2022) FCHO control AP2's initiating role in endocytosis through a Ptd-Ins(4,5)P2-dependent switch. Sci Adv 8: eabn2018</p>\n",
       "<p>Zhou R, Patel S V. &amp; Snyder PM (2007) Nedd4-2 catalyzes ubiquitination and degradation of cell surface ENaC. J Biol Chem 282: 20207–20212</p>\n",
       "<p>Zhu K, Shan Z, Chen X, Cai Y, Cui L, Yao W, Wang Z, Shi P, Tian C, Lou J, et al (2017) Allosteric auto‐inhibition and activation of the Nedd4 family E3 ligase Itch. EMBO Rep 18: 1618–1630</p>\n",
       "<p></p>\n",
       "<p></p>\n",
       "<p></p>\n",
       "<p>Figure legends</p>\n",
       "<p></p>\n",
       "<p>Figure 1. FCHO2 is required for Nedd4L-mediated ubiquitination and endocytosis of ENaC.</p>\n",
       "<p>A. Knockdown of FCHO2 or Nedd4L by siRNA. αβγENaC-HeLa cells treated with each siRNA were analyzed by immunoblotting (IB).</p>\n",
       "<p>B. Inhibition of ENaC endocytosis by knockdown of FCHO2 or Nedd4L. After cells were transfected with each siRNA, cell-surface ENaC was labeled by incubating at 4°C with anti-FLAG antibody. Endocytosis was started by incubating cells at 37°C for the indicated periods of time and stopped by chilling cells on ice. Antibody bound to the cell surface but not internalized was removed by acid stripping. After cells were solubilized, the antibody-labeled internalized ENaC subunits were precipitated and analyzed by IB (upper panel) and quantified (bottom and right panels). Internalization was expressed as a percentage of the initial amount of cell-surface αENaC determined by lysing cells without incubation at 37°C or acid stripping. Data are shown as the mean ± S.E.M. of three independent experiments. **P &lt; 0.01; ***P &lt; 0.001 (Student's t-test).</p>\n",
       "<p>C. Inhibition of transferrin receptor (TfR) endocytosis by FCHO2 knockdown. After cells were transfected with each siRNA, cell-surface proteins were biotinylated at 4°C. Cells were then incubated at 37°C for 10 min. Biotin was removed from the cell surface, and internalized biotinylated proteins were precipitated with avidin beads. Samples were subjected to IB with anti-TfR antibody and quantified. Internalization was expressed as a proportion of the initial amount of TfR on the cell surface. Data are shown as the mean ± S.E.M. of three independent experiments. *P &lt; 0.05 (Student's t-test).</p>\n",
       "<p>D. Restoration of αENaC endocytosis by FCHO2 re-expression in knockdown cells. After cells were transfected with the indicated siRNA and GFP construct, cell-surface αENaC was labeled with anti-FLAG antibody at 4°C. Cells were then incubated at 37°C for 10 min. After acid stripping, cells were subjected to immunofluorescence microscopy (left panels) and quantified (right panel). The antibody-labeled, internalized αENaC was visualized with a fluorescence-conjugated secondary antibody. The border of each cell is delineated by a solid line. Arrows indicate cells expressing GFP-tagged proteins. Scale bars, 10 μm. The proportion of cells with internalized αENaC among cells expressing control GFP or GFP-siRNA-resistant (sr) -FCHO2 is shown. Data are shown as the mean ± S.E.M. of five independent images (31-71 cells expressing GFP-tagged proteins per image). ***P &lt; 0.001 (two-way analysis of variance with post hoc test).</p>\n",
       "<p>E, F. Effects of FCHO2, Nedd4L, and FBP17 knockdown on ENaC ubiquitination and expression at the cell surface. After cells were transfected with each siRNA, cell-surface αENaC was biotinylated and sequentially precipitated with anti-FLAG antibody and then avidin beads. Samples were subjected to IB with anti-FLAG and anti-Ub antibodies (E) and quantified (F). E, ENaC ubiquitination, Asterisks indicate the same band of ubiquitinated FLAG-αENaC. F, ENaC expression at the cell surface. Data are shown as the mean ± S.E.M. of three independent experiments. *P &lt; 0.05 (Student's t-test).</p>\n",
       "<p></p>\n",
       "<p>Figure 2. Localization of cell-surface ENaC, FCHO2, and Nedd4L at clathrin-coated structures.</p>\n",
       "<p>A, B. Localization of cell-surface ENaC. mTagRFP-clathrin light chain (CLC) (A) or Nedd4L C922A (catalytic inactive mutant)-GFP (B) was expressed in αβγENaC-HeLa cells. Cell-surface αENaC was labeled with anti-FLAG antibody at 4°C. Cells were then fixed and stained with anti-FCHO2 antibody. Samples were subjected to immunofluorescence microscopy (left panel). Insets represent enlarged images of the dashed boxes. Scale bar, 10 μm. Co-localization was analyzed (right panel). The mean values of Pearson’s correlation coefficient from three cells and the S.E.M. are shown. A. Co-localization of cell-surface ENaC with FCHO2 at clathrin-coated structures. B. Co-localization of cell-surface ENaC with Nedd4L C922A mutant and FCHO2.</p>\n",
       "<p>C. Co-localization of Nedd4L with FCHO2 at clathrin-coated structures. Nedd4L-GFP and mTagRFP-CLC were co-expressed in wild-type HeLa cells. Cells were then fixed and stained with anti-FCHO2 antibody (left panel). Insets represent enlarged images of the dashed boxes. Scale bar, 10 μm. Co-localization was analyzed (right panel). The mean values of Pearson’s correlation coefficient from three cells and the S.E.M. are shown.</p>\n",
       "<p></p>\n",
       "<p>Figure 3. Recruitment of Nedd4L to CCPs. DsRed-CLC were co-expressed with Nedd4L-GFP in wild-type HeLa cells. Images were acquired at 1 s intervals through TIRF microscopy. Upper panel, TIRF images of HeLa cells. Insets represent enlarged images of the dashed boxes. Bottom panel, selected snapshots (every 10 sec) from a time series were obtained from a single CCP indicated by the arrowhead. The time point at which Nedd4L started to accumulate was set 0. Scale bar, 2 μm.</p>\n",
       "<p></p>\n",
       "<p>Figure 4. Recruitment of Nedd4L to membrane tubules generated by FCHO2 in cells.</p>\n",
       "<p>GFP-BAR domains were co-expressed with Myc-Nedd4L proteins in COS7 cells (A) and HEK293 cells (B). Cells were serum-starved and subjected to immunofluorescence microscopy. Insets represent enlarged images of the dashed boxes. Scale bar, 10 μm.</p>\n",
       "<p></p>\n",
       "<p>Figure 5. Activation of Nedd4L by membrane tubules generated by FCHO2 in cells.</p>\n",
       "<p>A–E. Nedd4L autoubiquitination. An in vivo ubiquitination assay was performed with various BAR domains and FCHO2 mutants (A, B) and Nedd4L mutants (D). GFP-BAR domains were co-expressed with Myc-Nedd4L proteins and HA-tagged Ub in HEK293 cells. Cell lysates were subjected to immunoprecipitation (IP) and analyzed by IB (A, B, D). WT, wild type. Cells were also subjected to immunofluorescence microscopy (C, E). Scale bar, 10 μm.</p>\n",
       "<p>F. αENaC ubiquitination. An in vivo ubiquitination assay was performed with various concentrations of Myc-Nedd4L using FALAG-αENaC as a substrate in the presence or absence of GFP-FCHO2 BAR domain. Cell lysates were subjected to IP. Samples were analyzed by IB.</p>\n",
       "<p></p>\n",
       "<p>Figure 6. The Nedd4L C2 domain is a Ca2+-dependent PS binding domain.</p>\n",
       "<p>A co-sedimentation assay was performed with the C2 domain or full-length Nedd4L using brain-lipid liposomes (~50% PS) at various Ca2+ concentrations (A) and with the C2 domain using synthetic liposomes containing various percentages of PS at 0.3 μM or 1.0 mM Ca2+ (B). The supernatants (S) and pellets (P) were subjected to sodium dodecyl sulfate (SDS)-polyacrylamide gel electrophoresis (PAGE), followed by Coomassie brilliant blue (CBB) staining (upper panels). Bottom panels, quantitative analysis. (−) for [Ca2+] indicates EGTA alone. Data are shown as the mean ± S.E.M. of three independent experiments.</p>\n",
       "<p>A. Ca2+ dependency.</p>\n",
       "<p>B. PS dependency.</p>\n",
       "<p></p>\n",
       "<p>Figure 7. Preference of Nedd4L for FCHO2-generated membrane curvature.</p>\n",
       "<p>A. Distribution of liposome diameters. Brain-lipid liposomes (~50% PS) of different diameters were prepared by extrusion through the indicated pore sizes and sonication. Left panel, electron microscopic images. Scale bars, 200 nm. Right panel, distribution of liposome diameters shown by boxplots (number of observations per pore size = 25). The center line inside the box corresponds to the median, the bounds of the box encompass data points between the first and third quartiles, and the whiskers extend to the minimum and maximum values including outliers. So, sonication.</p>\n",
       "<p>B-D. Sensing and generation of membrane curvature by Nedd4L. Co-sedimentation (B) and in vitro tubulation (C, D) assays were performed using brain-lipid liposomes (~50% PS) at 0.1 μM (B) and 0.3 μM Ca2+ (C, D). B, Preference of Nedd4L for a specific degree of membrane curvature. Liposomes were supplemented with X-biotin-PE and rhodamine-PE. Full-length Nedd4L was incubated with liposomes of different sizes, which were prepared by extrusion through the indicated pore sizes. The sample was then incubated with avidin beads and ultracentrifuged. The proportion of precipitated liposomes of each size was calculated to be ~100% by measuring both total and supernatant fluorescence. The supernatants (S) and pellets (P) were subjected to SDS- PAGE, followed by CBB staining (upper panel). Bottom panel, quantitative analysis. So, sonication. Data are shown as the mean ± S.E.M. of five independent experiments. *P &lt; 0.05; **P &lt; 0.01 (one-way analysis of variance with Tukey’s post hoc test). C, Curvature generation by Nedd4L. The assay was performed with the Nedd4L C2 domain. Samples were examined by electron microscopy. Scale bar, 100 nm. D, The degree of membrane curvature generated by Nedd4L is consistent with that generated by FCHO2. The assay was performed with the indicated protein. Left panel, electron microscopic image. Scale bar, 100 nm. Right panel, distribution of tubule diameters shown by boxplots (number of observations per protein = 14–33). The center line inside the box corresponds to the median, the bounds of the box encompass data points between the first and third quartiles, and the whiskers extend to the minimum and maximum values including outliers.</p>\n",
       "<p></p>\n",
       "<p>Figure 8. Conserved hydrophobic residues of Nedd4L are responsible for sensing and generation of membrane curvature.</p>\n",
       "<p>A. Sequence alignment of C2 domains. Conserved residues are highlighted with the following color code: yellow, aspartates for Ca2+ binding; blue, very hydrophobic residues probably for penetration into the lipid bilayer; and brown, others. Dots show the mutation sites. Hs, Homo sapiens; Sc, Saccharomyces cerevisiae; and Rt, Rattus norvegicus.</p>\n",
       "<p>B. Inability of Nedd4L C2 domain mutants to sense membrane curvature in cells. GFP-FCHO2 BAR domain and Myc-Nedd4L C2 domain mutants were co-expressed in COS7 cells. Insets are enlarged images of dashed boxes. Scale bar, 10 μm.</p>\n",
       "<p>C, D. Inability of Nedd4L C2 domain mutants to bind to membranes and to generate membrane curvature in vitro. Co-sedimentation (C) and in vitro tubulation (D) assays were performed using brain-lipid liposomes (~50% PS) at 0.3 μM Ca2+. C, Membrane binding. The assay was performed with Nedd4L C2 domain mutants using various concentrations of liposomes. The supernatants (S) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining (upper panel). Bottom panel, quantitative analysis. Data are shown as the mean ± S.E.M. of three independent experiments. D, Curvature generation. The assay was performed with Nedd4L C2 domain mutants. Samples were examined by fluorescence microscopy. Scale bar, 10 μm.</p>\n",
       "<p></p>\n",
       "<p>Figure 9. Properties of the catalytic activity and membrane binding of Nedd4L.</p>\n",
       "<p>Liposomes supplemented with biotinylated phospholipids were incubated with mSA-αENaC (A, B). Samples were mixed with Nedd4L. Mixtures were subjected to ubiquitination (C-E) and co-sedimentation (F) assays. These assays were performed with brain-lipid liposomes (~50% PS) at various Ca2+ concentrations (C), with synthetic liposomes (various percentages of PS) at 0.7 μM Ca2+ (D, F), and with brain-lipid liposomes (20% PS) at 0.7 μM Ca2+ (E). Ubiquitination and liposome binding were quantified by scanning and expressed as ratios to maximum levels. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.</p>\n",
       "<p>A, B. Schematic diagrams of mSA-αENaC. A, Structure. B. Binding to biotinylated liposomes.</p>\n",
       "<p>C. Ca2+-dependent Nedd4L activity. Ubiquitination was detected by IB (upper panel) and the intensity was quantified by scanning (bottom panel). Data are shown as the mean ± S.E.M. of three independent experiments.</p>\n",
       "<p>D, Membrane localization of mSA-αENaC enhances the PS-dependent catalytic activity of Nedd4L. Assays were performed in the presence or absence of 40 μM biotin. Ubiquitination was detected by IB (left panel). The intensities were quantified by scanning (right panel). Data are shown as the mean ± S.E.M. of three independent experiments.</p>\n",
       "<p>E. Nedd4L activity is predominantly stimulated by the specific degree of membrane curvature. The assay was performed using liposomes of different sizes which were prepared by extrusion through the indicated pore sizes. Ubiquitination was detected by IB (left panel) and quantified (right panel). Data are shown as the mean ± S.E.M. of three independent experiments. ***P &lt; 0.001 (one-way analysis of variance with Tukey’s post hoc test).</p>\n",
       "<p>F. Membrane localization of mSA-αENaC enhances the PS-dependent membrane binding of Nedd4L. Assays were performed in the presence or absence of 40 μM biotin. Liposome binding was detected by IB (left panel). The intensities were quantified by scanning (right panel). T, total sample. P, pellet. Data are shown as the mean ± S.E.M. of three independent experiments.</p>\n",
       "<p></p>\n",
       "<p></p>\n",
       "<p>Figure 10. Recruitment and activation of Nedd4L by FCHO2-generated membrane curvature in vitro.</p>\n",
       "<p>mSA-αENaC-associated brain-lipid liposomes (20% PS) were mixed with Nedd4L in the presence or absence of BAR domains. Mixtures were subjected to co-sedimentation (A) and ubiquitination (B-E) assays. These assays were performed at 0.7 μM Ca2+ with brain-lipid liposomes (20%PS) (A, B, D, and E) and with the indicated (20% PS or ~50% PS) brain-lipid liposomes (C). Ubiquitination and liposome binding were quantified by scanning and expressed as ratios to maximum levels. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.</p>\n",
       "<p>A. FCHO2-induced recruitment of Nedd4L. The assay was performed with the FCHO2 BAR domain (1.4 μM, 50 μg/ml) in the presence or absence of biotin. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining (FCHO2 BAR) and IB (Nedd4L and mSA-αENaC).</p>\n",
       "<p>B, C. FCHO2-induced activation of Nedd4L. B, Effects of various doses of the FCHO2 BAR domain. The assay was performed with various concentrations of the FCHO2 BAR domain. Ubiquitination was detected by IB (left panel) and the intensity was quantified by scanning (right panel). C, Effects of the PS percentage in liposomes. The assay was performed with the FCHO2 BAR domain (1.4 μM). Samples were analyzed by IB. Data are shown as the mean ± S.E.M. of three independent experiments.</p>\n",
       "<p>D, E. Specificity of BAR domains and effect of an FCHO2 mutant. The assay was performed with various BAR domains (D) and an FCHO2 mutant (E) (1.4 μM each). Samples were analyzed by IB.</p>\n",
       "<p></p>\n",
       "<p>Figure 11. Effects of PI(4,5)P2 on FCHO2-induced recruitment and activation of Nedd4L.</p>\n",
       "<p>A, B. Binding to PI(4,5)P2. A co-sedimentation assay was performed at 0.7 μM Ca2+ with the FCHO2 BAR or Nedd4L C2 domain using mSA-αENaC-associated synthetic liposomes containing 20% PS, 5% PI(4,5)P2 (A), or various percentages of PI(4,5)P2 (B). The supernatants (S) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining (A and upper panel in B). Bottom panel in B, quantitative analysis. Control, synthetic liposomes [0% PS and 0% PI(4,5)P2]. Data are shown as the mean ± S.E.M. of three independent experiments.</p>\n",
       "<p>C, D. Effects on FCHO2-induced recruitment and activation of Nedd4L. Co-sedimentation (C) and ubiquitination (D) assays were performed at 0.7 μM Ca2+ with mSA-αENaC-associated synthetic liposomes containing 20% PS or 5% PI(4,5)P2 in the presence or absence of the FCHO2 BAR domain. C, Nedd4L recruitment. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by IB with anti-Nedd4L antibody (upper panel) and CBB staining (lower panel). D, Nedd4L activation. Samples were analyzed by IB. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.</p>\n",
       "<p></p>\n",
       "<p>Figure 12. Effects of liposome size on FCHO2-induced activation of Nedd4L. Co-sedimentation (A) and ubiquitination (B) assays were performed at 0.7 μM Ca2+ with the FCHO2 BAR domain using brain-lipid liposomes (20% PS, 0.8 µm or 0.05 µm pore-size) that were associated with mSA-αENaC.</p>\n",
       "<p>A. Binding of the FCHO2 BAR domain to 0.8 µm and 0.05 µm pore-size liposomes. The co-sedimentation assay was performed using synthetic (0% PS) or brain-lipid liposomes (20% PS) containing rhodamine-PE and fluorescein-PE. Liposomes were precipitated with anti-fluorescein antibody. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining. The proportion of precipitated liposomes of each size was calculated to be ~100% by measuring both total and supernatant fluorescence.</p>\n",
       "<p>B. Effects of liposome size on FCHO2-induced activation of Nedd4L. Samples were analyzed by IB. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.</p>\n",
       "<p></p>\n",
       "<p>Figure 13. Model of ENaC endocytosis.</p>\n",
       "<p>FCHO2 binds to the plasma membrane and generates a specific degree of membrane curvature according to the intrinsic curvature of its BAR domain. Nedd4L exists in a catalytically autoinhibited state due to an intramolecular interaction between the N-terminal C2 and C-terminal HECT domains. Nedd4L is recruited to FCHO2-generated membrane curvature at the rim of nascent CCPs that ENaC enters. Membrane binding of the C2 domain causes catalytic activation by relieving autoinhibition. ENaC is ubiquitinated and captured by adaptor proteins with Ub-interacting motifs, such as Eps15.</p>\n",
       "<p></p>\n",
       "<p>Extended View Figure Legends</p>\n",
       "<p></p>\n",
       "<p>Figure EV1. Expression of ENaC subunits in αβγENaC-HeLa cells.</p>\n",
       "<p>A, B. Doxycycline (Dox)-induced expression of αENaC. αβγENaC-HeLa cells were cultured overnight in the presence or absence of Dox. The lysates of parental HeLa and αβγENaC-HeLa cells were subjected to IB. A, 7.5% gel. B, Gradient gel (5-20%). Anti-γENaC antibody cross-reacted with two bands of endogenous proteins in parental HeLa cells, the lower band of which overlapped with γENaC (arrow) in αβγENaC-HeLa cells. Upon Dox-induced αENaC expression, an additional 70-kDa γENaC band (asterisk) was detected. It has been shown that co-expression of all three subunits induces ENaC maturation, including proteolytic cleavage of α- and γENaC (Hughey et al, 2003). The 70-kDa γENaC band is likely a cleavage product comprising the C-terminal region. The 20-kDa αENaC band (arrowhead) is likely a cleavage product comprising the N-terminal region.</p>\n",
       "<p>C. Association of α-, β-, and γENaC. When αENaC was immunoprecipitated with anti-FLAG antibody from αβγENaC-HeLa cells treated with Dox, β- and γENaC were co-precipitated. Asterisk, 70-kDa γENaC band.</p>\n",
       "<p></p>\n",
       "<p>Figure EV2. Knockdown of FBP17 by siRNA.</p>\n",
       "<p>αβγENaC-HeLa cells were treated with each siRNA, and FBP17 mRNA levels were quantified using real-time PCR. The expression of FBP17 was normalized to GAPDH mRNA levels. Data are shown as the mean ± S.E.M. of three independent experiments. ***P &lt; 0.001 (Student's t-test).</p>\n",
       "<p></p>\n",
       "<p>Figure EV3. Inability of FCHO2 mutants to generate membrane tubules.</p>\n",
       "<p>GFP-FCHO2 BAR domain [wild type (WT) or mutant] was expressed in COS7 cells. Cells were then subjected to immunofluorescence microscopy. Scale bar, 10 μm.</p>\n",
       "<p></p>\n",
       "<p>Figure EV4. Inability of Nedd4L mutants to ubiquitinate αENaC. An in vivo ubiquitination assay was performed with various Nedd4L constructs (each 0.5 µg) using FLAG-αENaC as a substrate in the presence or absence of GFP-FCHO2 BAR domain. Cell lysates were subjected to IP. Samples were analyzed by IB.</p>\n",
       "<p></p>\n",
       "<p>Figure EV5. Inhibition of the liposome binding of Nedd4L by the FCHO2 BAR domain.</p>\n",
       "<p>A co-sedimentation assay was performed at 0.7 μM Ca2+ with control liposomes (0% PS) or brain-lipid liposomes (~50% PS) in the presence or absence of the FCHO2 BAR domain. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by IB (upper panel) and CBB staining (lower panel).</p>\n",
       "<p></p>\n",
       "<p>Figure EV6. Membrane binding and curvature generation of BAR domains and an FCHO2 mutant. Co-sedimentation (A) and in vitro tubulation (B) assays were performed at 0.7 μM Ca2+ with the indicated BAR domains using brain-lipid liposomes (20% PS) that were associated with mSA-αENaC.</p>\n",
       "<p>A. Membrane binding. The supernatants (S) and pellets (P) were subjected to SDS-PAGE followed by CBB staining.</p>\n",
       "<p>B. Curvature generation. Left panel, electron microscopic image. Scale bar, 100 nm. Right panel, distribution of tubule diameters shown by boxplots (number of observations per protein = 22–26). The center line inside the box corresponds to the median, the bounds of the box encompass data points between the first and third quartiles, and the whiskers extend to the minimum and maximum values including outliers.</p>\n",
       "</html>"
      ]
     },
     "execution_count": 67,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "BeautifulSoup(html_content, \"html.parser\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Langchain integration"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Token indices sequence length is longer than the specified maximum sequence length for this model (528 > 512). Running this sequence through the model will result in indexing errors\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/0', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/1', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/2', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/3', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/4', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/5', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/6', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/7', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/8', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/9', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/10', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/11', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/12', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/13', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='The Nedd4L ubiquitin ligase is activated by FCHO2-generated membrane curvature\\n\\nYasuhisa Sakamoto1, Akiyoshi Uezu1, Koji Kikuchi1, Jangmi Kang2, Eiko Fujii2, Toshiro Moroishi3, Shiro Suetsugu4, and Hiroyuki Nakanishi1,2*\\n\\n1Department of Molecular Pharmacology, Faculty of Life Sciences, Kumamoto University, 1–1–1 Honjyo, Kumamoto 860-8556, Japan\\n2Faculty of Clinical Nutrition and Dietetics, Konan Women’s University, 6–2–23 Morikita-machi, Kobe 658-0001, Japan\\n3Department of Molecular and Medical Pharmacology, Faculty of Life Sciences, Kumamoto University, 1–1–1 Honjyo, Kumamoto 860-8556, Japan\\n4Division of Biological Science, Graduate School of Science and Technology, Nara Institute of Science and Technology, 8916-5 Takayama-cho, Ikoma 630-0192, Japan\\n*Corresponding author. Tel: +81-96-373-5074; Fax: +81-96-373-5078; E-mail: hnakanis@gpo.kumamoto-u.ac.jp\\n\\n\\n\\nAbstract\\n'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/14', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/15', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/16', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/17', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/18', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/19', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='The C2-WW-HECT domain ubiquitin ligase Nedd4L regulates membrane sorting during endocytosis through the ubiquitination of cargo molecules such as the epithelial sodium channel (ENaC). Nedd4L is catalytically autoinhibited by an intramolecular interaction between its C2 and HECT domains, but the protein’s activation mechanism is poorly understood. Here, we show that Nedd4L activation is linked to membrane shape by FCHO2, a Bin-Amphiphysin-Rsv (BAR) domain protein that regulates endocytosis. FCHO2 was required for the Nedd4L-mediated ubiquitination and endocytosis of ENaC, with Nedd4L co-localizing with FCHO2 at clathrin-coated pits. In cells, Nedd4L was specifically recruited to, and activated by, the FCHO2 BAR domain. Furthermore, we reconstituted FCHO2-induced recruitment and activation of Nedd4L in vitro. Both the recruitment and activation were mediated by membrane curvature rather than protein-protein interactions. The Nedd4L C2 domain recognized a specific degree of membrane curvature that was generated by the FCHO2 BAR domain, with this curvature directly activating Nedd4L by relieving its autoinhibition. Thus, we show for the first time a specific function (i.e., recruitment and activation of an enzyme regulating cargo sorting) of membrane curvature by a BAR domain protein.\\n\\nKeywords: Nedd4L; FCHO2; clathrin; endocytosis; membrane curvature.\\n\\nIntroduction\\n'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/20', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='Posttranslational modification of proteins by covalent attachment of ubiquitin (Ub) is catalyzed by three enzymes: a Ub-activating enzyme (E1), a Ub-conjugating enzyme (E2), and a Ub-protein ligase (E3) that determines substrate specificity (Hershko & Ciechanover, 1998). E3 Ub ligases are classified into two categories based on their Ub transfer mechanisms: RING finger/U-box E3 and HECT-type E3. The Nedd4 family belongs to HECT-type E3 Ub ligases and consists of nine members, including Nedd4/Nedd4-1, Nedd4L/Nedd4-2, Itch, Smurf1, Smurf2, WWP1, WWP2, NEDL1, and NEDL2 (Ingham et al, 2004; Rotin & Kumar, 2009). They are characterized by a common modular organization, with an N-terminal C2 domain, two to four WW domains, and a C-terminal catalytic HECT domain. While the C2 domain was originally identified in protein kinase C (PKC) as a Ca2+-dependent phosphatidylserine (PS)-binding domain (Nishizuka, 1992), it shows significant diversity in the binding partners, including intracellular proteins and other phospholipids, such as phosphatidylinositol (4,5)-bisphosphate [PI(4,5)P2] (Lemmon, 2008). The WW domains recognize proline-rich motifs, such as PPxY (PY motif, where x is any residue), of substrates or adaptor proteins. The HECT domain catalyzes the isopeptide bond formation between the Ub C terminus and the substrate lysine residues. At least some of the Nedd4 family members, including Nedd4L, are catalytically autoinhibited by an intramolecular interaction between the C2 and HECT domains (Wang et al, 2010; Wiesner et al, 2007; Zhu et al, 2017). However, the activation mechanism of the Nedd4 family is poorly understood.'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/21', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='The Nedd4 family is implicated in a wide variety of cellular processes by regulating membrane trafficking and degradation of components of various signaling pathways (Ingham et al, 2004; Rotin & Kumar, 2009). Among a diversity of substrates, the best-characterized is the epithelial sodium channel (ENaC). This channel is composed of three subunits, α-, β-, and γENaC, each of which possesses two transmembrane segments and intracellular N- and C-termini with a large extracellular loop (Butterworth, 2010; Rotin & Staub, 2011; Rossier, 2014). Each subunit has a PY motif at the cytosolic C-terminus. Nedd4L Ub ligase specifically binds to the PY motif and mediates the ubiquitination of lysine residues on the N-terminus of at least α- and γENaC at the plasma membrane. Ub serves as an internalization signal for recognition by adaptor proteins, such as epsin and Eps15, for clathrin-mediated endocytosis (CME) (Traub, 2009). Consequently, ENaC is constitutively internalized through CME. Heterozygous mutations of the PY motif of β- or γENaC lead to impaired ubiquitination and endocytosis, resulting in persistence of the channels at the cell surface and increased Na+ absorption in Liddle syndrome, an autosomal dominant form of severe hypertension.'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/22', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/23', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/24', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/25', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/26', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/27', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='The process of CME begins at the cytoplasmic surface of the plasma membrane with the assembly of clathrin with adaptor proteins and transmembrane cargo molecules (Doherty & McMahon, 2009; Traub, 2009). In parallel with this assembly, membrane curvature is generated to form hemispherical clathrin-coated pits (CCPs). This process leads to deep membrane invaginations. Subsequently, the neck of invaginated pits is constricted and severed to separate the newly formed clathrin-coated vesicles (CCVs) from the plasma membrane. These membrane curvatures are considered to be generated, sensed, and/or maintained by Bin-Amphiphysin-Rsv (BAR) domain superfamily proteins. BAR domains form a crescent-shaped dimer characterized by a specific degree of intrinsic curvature. They bind to negatively charged phospholipids, such as PS, and force membranes to bend according to their intrinsic curvatures, thereby inducing the formation of membrane tubules with specific diameters (curvatures). For example, the diameters of membrane tubules generated by the amphiphysin BAR domain, FCHO2 BAR domain, and FBP17 BAR domain are 20–50 nm (sharper curvature) (Peter et al, 2004; Takei et al, 1999; Yin et al, 2009), 50–80 nm (intermediate curvature) (Henne et al, 2007), and ~150 nm (Takano et al, 2008) (shallower curvature), respectively.\\nThe FCHO2 BAR domain has been proposed to trigger the initial invagination for CCP formation (Henne et al, 2010). FCHO2 binds to the flat surface of the plasma membrane at sites where CCPs will form. FCHO2 then generates the initial membrane curvature and recruits the clathrin machinery for CCP formation. Here, we report the role of FCHO2 in Nedd4L-mediated ENaC endocytosis and show that FCHO2 BAR domain-generated membrane curvature induces the recruitment and activation of Nedd4L.\\n\\nResults\\n\\nFCHO2 is required for Nedd4L-mediated ubiquitination and endocytosis of ENaC'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/28', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/29', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/30', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='To study the role of FCHO2 in ENaC endocytosis, we established a HeLa cell line stably expressing all three subunits, α-, β-, and γENaC (αβγENaC-HeLa cells) (Fig EV1A-C). A FLAG tag was introduced into the extracellular region of αENaC in a position previously shown not to affect channel activity (Firsov et al, 1996). Upon FCHO2 knockdown by small interfering RNA (siRNA) methods, ENaC internalization was reduced, as was upon Nedd4L knockdown (Zhou et al, 2007) (Fig 1 A and B). Transferrin receptor (TfR) internalization was reduced upon FCHO2 knockdown but not upon Nedd4L knockdown (Fig 1C). The phenotype of FCHO2 knockdown in ENaC internalization was rescued by an siRNA-resistant (sr) form of FCHO2 (Fig 1D and Appendix Fig S1). Furthermore, αENaC ubiquitination was reduced upon FCHO2 or Nedd4L knockdown but not upon FBP17 knockdown (Fig 1E and Fig EV2). FCHO2 or Nedd4L knockdown increased αENaC expression at the cell surface (Fig 1E and F). These results suggest that FCHO2 is required for Nedd4L-mediated ubiquitination and endocytosis of ENaC.\\n\\nRecruitment and activation of Nedd4L by FCHO2 in cells'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/31', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/32', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='Immunofluorescence microscopy revealed that in αβγENaC-HeLa cells, cell-surface αENaC co-localized with FCHO2 at clathrin-coated structures (Fig 2A). However, its co-localization with Nedd4L was not detected (Appendix Fig S2A), because exogenous expression of Nedd4L-green fluorescent protein (GFP) stimulated ENaC internalization, resulting in the disappearance of αENaC from the cell surface. Cell-surface αENaC was observed in cells lacking Nedd4L-GFP expression (non-transfected cells). When a catalytically inactive Nedd4L mutant, C922A, was expressed, it co-localized with cell-surface αENaC and endogenous FCHO2 (Fig 2B). The mutant also formed FCHO2-negative spots. In wild-type HeLa cells, Nedd4L accumulated at clathrin-coated structures where FCHO2 localized (Fig 2C). Additionally, Nedd4L formed clathrin-negative spots, suggesting that they may be involved in clathrin-independent endocytosis.\\nUsing total internal reflection fluorescence (TIRF) microscopy, we investigated whether Nedd4L is recruited to CCPs in wild-type HeLa cells. Similar to FCHO2 described previously (Henne et al, 2010; Lehmann et al, 2019; Zaccai et al, 2022), Nedd4L showed dynamic co-localization with clathrin during pit formation (Fig 3). When both fluorescent protein-tagged FCHO2 and Nedd4L were expressed, the accumulation of Nedd4L at FCHO2-positive spots was not clearly observed (Appendix Fig S2B). The reason behind this remains unknown, but it has been shown that clathrin coat formation, measured in time-lapse images of cells expressing fluorescent proteins, is affected by several factors, such as overexpression level, choice of the proteins, and choice of the fluorescent tag (Aguet et al, 2013). These factors may contribute to the failure to detect the accumulation of Nedd4L at FCHO2-positive spots.'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/33', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='While Nedd4L shows a punctate staining pattern at the cell surface (see Figs 2C, 3B and 3C), it exhibited diffuse distribution throughout the cytoplasm (Fig 4A and B). In COS7 cells, co-expression of the FCHO2 BAR domain [amino acid (aa) 1-302] recruited Nedd4L to membrane tubules (Fig 4A). The Nedd4L C2 domain (Myc-Nedd4L C2, aa 1-160) was also concentrated to membrane tubules, whereas C2 domain-deleted Nedd4L (Myc-Nedd4L ΔC2) was not. In contrast to FCHO2, the FBP17 BAR domain did not recruit Nedd4L to membrane tubules. Additionally, the BAR domain protein amphiphysin1 did not recruit Nedd4L to membrane tubules. Thus, Nedd4L is selectively recruited through the C2 domain to membrane tubules generated by the FCHO2 BAR domain. Similar results were obtained in HEK293 cells (Fig 4B). In these cells, membrane tubules generated by amphiphysin1 were hardly detected. It showed a punctate staining pattern and did not co-localized with Nedd4L.'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/34', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/35', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/36', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='We also investigated whether the FCHO2 BAR domain not only recruits but also activates Nedd4L in cells. For this purpose, we performed an in vivo ubiquitination assay using HEK293 cells, which formed membrane tubules upon FCHO2 expression more efficiently than did COS7 cells: 91 ± 2% of HEK293 cells vs. 32 ± 2% of COS7 cells [the mean ± standard error of the mean (S.E.M.) of four independent experiments]. As the catalytic activity of E3 Ub ligases is typically reflected in their autoubiquitination (Wiesner et al, 2007), we determined activity by measuring the ubiquitination level of Nedd4L itself. Expression of the FCHO2 BAR domain in cells remarkably enhanced Nedd4L autoubiquitination, whereas FBP17 or amphiphysin1 did not (Fig 5A). Tubulation-deficient FCHO2 mutants (K146E+R152E and L136E) (Henne et al, 2010) did not enhance Nedd4L activity (Fig 5A–C and Fig EV3). A catalytically inactive Nedd4L mutant, C922A, which was recruited to FCHO2-generated membrane tubules, showed little enhancement of ubiquitination upon FCHO2 expression (Fig 5D and E), indicating that Nedd4L ubiquitination is self-induced. Using αENaC as a substrate, we further confirmed that Nedd4L is activated upon FCHO2 expression in cells (Fig 5F).\\n\\nPreference of Nedd4L for FCHO2-generated membrane curvature'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/37', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/38', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='The aforementioned results prompted us to investigate whether FCHO2 and Nedd4L interact through membrane tubules rather than through stereotyped protein–protein interactions. Membrane tubules generated by BAR domains in cells have specific diameters (curvatures) (Henne et al, 2007; Peter et al, 2004; Takano et al, 2008; Takei et al, 1999; Yin et al, 2009). Nedd4L may preferentially bind to the specific degree of membrane curvature generated by FCHO2. To investigate this possibility, we first examined the biochemical properties of Nedd4L. Using a co-sedimentation assay with liposomes containing various percentages of PS, we found that the C2 domain of Nedd4L is a Ca2+-dependent PS-binding domain (Fig 6A and B). Submicromolar Ca2+ concentrations, which are comparable to intracellular levels in unstimulated cells, were sufficient for the Nedd4L C2 domain to bind to PS-containing liposomes. The C2 domain alone and full-length Nedd4L showed different Ca2+ requirements. The higher Ca2+ requirement of full-length Nedd4L may be due to the intramolecular interaction between the C2 and HECT domains. Increasing PS percentages in synthetic liposomes enhanced the liposome binding of the Nedd4L C2 domain, and PS requirements varied depending on Ca2+ concentration.\\nWe next examined whether Nedd4L preferentially binds to a specific degree of membrane curvature. For this purpose, liposomes of different sizes (curvatures) were prepared by stepwise extrusion through filter membranes with different pore sizes (0.8, 0.4, 0.2, 0.1, 0.05, and 0.03 µm) and sonication. Sonication or extrusion through a 0.03 µm pore-size filter was used to prepare the smallest liposomes. Transmission electron microscopy revealed that the actual liposome size was similar to the pore size for 0.2, 0.1, 0.05, and 0.03 µm pore-size liposomes, whereas it was significantly smaller for 0.8 and 0.4 µm pore-size liposomes (Fig 7A). Nedd4L showed the strongest binding to 0.05 µm pore-size liposomes (Fig 7B).'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/39', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/40', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/41', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='Synaptotagmin I (SytI) tandem C2 domains can both generate and sense membrane curvature in vitro by penetrating their hydrophobic residues into the lipid bilayer (Martens et al, 2007). As a sensor, the binding affinity is considered to be highest for membranes that have a degree of curvature consistent with that generated by SytI itself. We investigated whether, in addition to sensing membrane curvature, the Nedd4L C2 domain can also generate membrane curvature in vitro and, if so, what diameter of membrane tubules is generated. As expected, transmission electron microscopy revealed that the C2 domain deformed liposomes into extensive tubulation (Fig 7C). It should be noted that Nedd4L does not generate membrane tubules in cells (see Fig 4A and B). The tubule diameter generated by Nedd4L in vitro was mainly 40–80 nm, which is consistent not only with the diameter of 0.05 µm pore-size liposomes, to which Nedd4L preferentially binds, but also with the diameter of membrane tubules generated by FCHO2 (Fig 7D). These results suggest that Nedd4L shows the highest binding affinity for the degree of membrane curvature generated by FCHO2. It is therefore likely that FCHO2-induced recruitment and activation of Nedd4L are mediated by the degree of membrane curvature.\\n\\nRoles of conserved hydrophobic residues in the Nedd4L C2 domain'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/42', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='We compared the C2 domain sequence of Nedd4L to those of other Nedd4 family members, SytI, and PKCγ (Fig 8A). SytI C2 domains possess hydrophobic residues (M173, F234, V304, and I367) for penetration into the lipid bilayer to generate and sense membrane curvature (Martens et al, 2007). The Nedd4L C2 domain also has three hydrophobic residues (I37, F38, and L99), which are conserved among Nedd4 family members except for WWP2. We mutated these residues to alanine to produce six mutants, I37A, F38A, L99A, I37A+F38A, I37A+L99A, and I37A+F38A+L99A (triple-A). Any single mutation did not inhibit Nedd4L recruitment to FCHO2-generated membrane tubules (Fig 8B). However, double and triple mutations severely impaired this recruitment. Furthermore, the I37A+F38A and triple-A mutants showed little liposome binding and lost curvature generation (Fig 8C and D). Thus, these hydrophobic residues (I37, F38, and L99) are critical for both membrane binding and curvature sensing and generation. These residues are likely inserted into the lipid bilayer, thereby strengthening the C2 domain interaction with membranes and enabling the sensing and generation of membrane curvature. It has been shown that I37 and F38 are directly involved in Ca2+ binding (Escobedo et al, 2014). The residues may be inserted into the lipid bilayer while binding Ca2+.'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/43', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/44', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/45', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='In the C2 domain of Smurf2, another Nedd4 family member, conserved hydrophobic residues (F29 and F30) are responsible for interactions with the HECT domain (Fig 8A) (Wiesner et al, 2007). Mutation at these residues impairs the interaction between the C2 and HECT domains, resulting in enhanced Smurf2 autoubiquitination by relieving autoinhibition. Consistent with these previous findings, we found that the Nedd4L I37A+F38A mutant, which was not recruited to FCHO2-generated membrane tubules, enhanced autoubiquitination, irrespective of FCHO2 expression (Fig 5D and E). These results suggest that I37 and F38 are critical for interactions with the HECT domain as well as for membrane binding and curvature sensing. Interestingly, the Nedd4L I37A+F38A mutant, as well as the C922A mutant, showed little αENaC ubiquitination (Fig EV4), indicating that membrane binding of Nedd4L is critical for ENaC ubiquitination.\\n\\nIn vitro activation of Nedd4L by FCHO2-generated membrane curvature'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/46', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/47', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='To examine the properties of Nedd4L catalytic activity, we carried out an in vitro ubiquitination assay using an αENaC peptide as a substrate. The αENaC peptide possesses a conserved PY motif for interaction with the Nedd4L WW domain, and it is fused with monomeric streptavidin (mSA) (Lim et al, 2013) to bind to liposomes containing biotinylated phospholipids (biotinylated liposomes) (Sakamoto et al, 2017) (Fig 9A and B). We found that Ca2+ and PS were required for Nedd4L catalytic activity (Fig 9C and D), and that 0.05 µm pore-size liposomes were most effective in stimulating Ca2+- and PS-dependent Nedd4L activity (Fig 9E). Thus, the properties of Nedd4L catalytic activity are similar to those of its liposome binding. Furthermore, we found that Nedd4L did not ubiquitinate PY motif-mutated (Y644A in αENaC) mSA-αENaC (Appendix Fig S3). These findings are consistent with defects in ENaC ubiquitination observed in Liddle syndrome.\\nWe next examined whether membrane localization of mSA-αENaC affects the membrane binding and catalytic activity of Nedd4L. Nedd4L bound only slightly to mSA-αENaC on PS-free (0% PS) liposomes, but the liposome binding was synergistically increased by elevating the PS percentage in liposomes (Fig 9F). By contrast, when free biotin competitively inhibited the binding of mSA-αENaC to biotinylated liposomes, the PS-dependent liposome binding of Nedd4L was significantly reduced, resulting in remarkable inhibition of mSA-αENaC ubiquitination (Fig 9D and F). Thus, membrane localization of the PY motif is critical for membrane binding and activation of Nedd4L. It is, therefore, likely that the PY motifs on membranes, such as ENaC, play a role in enhancing Nedd4L catalytic activity by potentiating membrane binding through interactions with the WW domain.'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/48', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='We then reconstituted in vitro FCHO2-induced recruitment and activation of Nedd4L. When liposomes (20% PS) were used, the FCHO2 BAR domain enhanced the liposome binding of Nedd4L (Fig 10A) and mSA-αENaC ubiquitination (Fig 10B). This enhancement was lost when the strength of interaction of Nedd4L with liposomes was increased by elevating the PS percentage in liposomes (~50% PS) (Fig 10C and Fig EV5).\\u3000In contrast to FCHO2, neither amphiphysin1 nor FBP17 enhanced Nedd4L activity (Fig 10D and Fig EV6A). Similarly, tubulation-deficient FCHO2 mutant (L136E) did not enhance Nedd4L activity (Fig 10E and Fig EV6A). These results are in good agreement with those obtained from in vivo experiments on FCHO2-induced recruitment and activation of Nedd4L. Electron microscopic analysis revealed that mSA-αENaC-associated liposomes were deformed by membrane-sculpting proteins into tubules with diameters similar to those of mSA-αENaC-free liposomes (Fig EV6B and see also Fig 7D). The diameter of membrane tubules generated by FCHO2 is consistent with that of 0.05 µm pore-size liposomes which are most effective in stimulating Nedd4L activity.'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/49', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='To confirm whether FCHO2-induced recruitment and activation of Nedd4L are mediated by liposomes (i.e., membrane curvature) rather than by protein–protein interactions, we conducted two sets of experiments. Initially, we employed synthetic liposomes where 20% PS was substituted with 5% PI(4,5)P2. The FCHO2 BAR domain bound to PI(4,5)P2 as well as PS with high affinity, whereas the Nedd4L C2 domain did not (Fig 11A and B). The FCHO2 BAR domain is known to induce the tubulation of liposomes [5% PI(4,5)P2] (Henne et al, 2010). If Nedd4L could interact through protein–protein interactions with the FCHO2 BAR domain binding to liposomes [5% PI(4,5)P2], the BAR domain would facilitate the co-precipitation of Nedd4L in a co-sedimentation assay, thereby stimulating mSA-αENaC ubiquitination. However, the FCHO2 BAR domain did not demonstrate such interactions (Fig 11C and D). Therefore, it is unlikely that Nedd4L interacts with the FCHO2 BAR domain on liposomes through protein–protein interactions. Secondly, we employed 0.05 µm pore-size liposomes that are most effective in stimulating Nedd4L activity. The diameter of 0.05 µm pore-size liposomes is consistent with that of membrane tubules generated by the FCHO2 BAR domain. The BAR domain is, thus, expected not to induce the tubulation of 0.05 µm pore-size liposomes. The FCHO2 BAR domain showed similar binding activity toward 0.8 and 0.05 µm pore-size liposomes (Fig 12A). When 0.8 µm pore-size liposomes were used, the FCHO2 BAR domain enhanced Nedd4L activity (Fig 12B). However, in the case of 0.05 µm pore-size liposomes, curvature-stimulated Nedd4L activity was not potentiated by the addition of the FCHO2 BAR domain. If Nedd4L had interacted with the BAR domain, we would have expected potentiation of Nedd4L activity. These results strongly suggest that FCHO2-induced activation of'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/49', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/50', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/51', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/52', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/53', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/54', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/55', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/56', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='Nedd4L is mediated by membrane curvature and not by protein–protein interactions.\\n\\nDiscussion\\n\\nNedd4L represents the ancestral Ub ligase with strong similarity to yeast Rsp5 which plays a key role in the trafficking, sorting, and degradation of a large number of proteins (Yang & Kumar, 2010). Consistently, Nedd4L regulates a growing number of substrates, including membrane receptors, transporters, and ion channels. Although Nedd4L exists in a catalytically autoinhibited state due to an intramolecular interaction between the N-terminal C2 and C-terminal HECT domains, little is known about its activation mechanism. In this study, we clearly demonstrated that Nedd4L is activated by FCHO2 F-BAR domain-induced membrane curvature. It has been shown that the activity of synptojanin-1, a phosphoinositide phosphatase, is modulated by membrane curvature (Chang-Ileto et al, 2011). Taken together, it is conceivable that membrane curvature serves as a signal for the activation of various enzymes involved in membrane trafficking.\\n\\nUnique Properties of the Nedd4L C2 Domain\\nAlthough both sytI and Nedd4L C2 domains are Ca2+-dependent PS-binding domains that can generate and sense membrane curvature, their properties show a number of significant differences. An interesting difference exists in the degrees of membrane curvatures generated by sytI and Nedd4L: the diameters of their membrane tubules are about 15–20 nm and 40–80 nm, respectively (Hui et al, 2009; Martens et al, 2007). This difference is not simply due to that between tandem and single C2 domains, because it has been shown that the sytI C2B domain alone generates membrane tubules with a similar diameter (Hui et al, 2009). SytI and Nedd4L C2 domains must therefore have respective unique structures that determine the specific degrees of membrane curvatures that they sense and generate.'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/57', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/58', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/59', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/60', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='Another important difference between sytI and Nedd4L C2 domains exists in their Ca2+ requirements. SytI C2 domains require Ca2+ concentrations in the submillimolar range to generate membrane curvature (Hui et al, 2009). In contrast, submicromolar Ca2+ concentrations, which are comparable to intracellular levels in unstimulated cells, are sufficient for the Nedd4L C2 domain to sense membrane curvature. This is consistent with the observation that Nedd4L is recruited and activated in serum-starved cells. The different Ca2+ requirements between sytI and Nedd4L C2 domains are reflected in their functions. SytI C2 domains induce membrane curvature in response to an increase in intracellular Ca2+ concentration, which is essential for Ca2+-triggered membrane fusion (Hui et al, 2009; Martens et al, 2007). Thus, sytI C2 domains serve as a Ca2+ sensor and a curvature generator. In contrast, the Nedd4L C2 domain recognizes a specific degree of curvature without an increase in intracellular Ca2+ concentration. Therefore, the Nedd4L C2 domain serves as a curvature sensor to regulate the localization and activity of the E3 Ub ligase.\\nA striking feature that distinguishes the Nedd4L C2 domain from other C2 domain proteins (except for at least a subset of Nedd4 family members) is that the Nedd4L C2 domain interacts not only with membranes but also with the intramolecular catalytic domain for its autoinhibition. Mutational experiments have revealed that I37 and F38 residues in the C2 domain are critical for interactions with both membranes and the HECT domain, suggesting that these two interactions are competitive. This idea is supported by the observation that the Ca2+ EC50 value of full-length Nedd4L is much higher than that of the C2 domain. The HECT domain in full-length Nedd4L may competitively inhibit the interaction of the C2 domain with membranes. It is likely that membranes displace the HECT domain from the C2 domain when the strength of the C2 domain interaction with membranes is increased by the preferred membrane curvature. This displacement relieves the autoinhibition, resulting in Nedd4L activation. Thus, the membrane binding of Nedd4L is coupled with its activation.\\n\\nStimulatory and inhibitory effects of the FCHO2 BAR domain on the liposome binding and catalytic activity of Nedd4L'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/61', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='The FCHO2 BAR domain enhances both the liposome binding and catalytic activity of Nedd4L when the interaction strength of Nedd4L with liposomes is weak (20% PS) (Fig 10A and B). Conversely, the BAR domain inhibits the binding and activity of Nedd4L when the interaction of Nedd4L with liposomes is increased by elevating the PS percentage in liposomes (~50% PS) (Fig 10C and Fig EV5). Thus, the FCHO2 BAR domain exerts both stimulatory and inhibitory effects on Nedd4L, depending on the PS percentage in liposomes. However, given that 20% PS is comparable to the concentration in the inner leaflet of the plasma membrane, it is plausible that FCHO2 exhibits only a stimulatory effect on Nedd4L in cells. The mechanism underlying the stimulatory and inhibitory effects in vitro is as follows: when using liposomes (20% PS), Nedd4L hardly binds to liposomes through PS (Fig. 9F and 10A). The addition of the FCHO2 BAR domain enhances the strength of Nedd4L’s interaction with PS by inducing membrane curvature. Consequently, Nedd4L gains a new binding to liposomes through PS, which augments its catalytic activity. In contrast, when using liposomes (~50% PS), Nedd4L binds to liposome through PS. The addition of the FCHO2 BAR domain hinders the PS-mediated liposome binding of Nedd4L, because FCHO2 and Nedd4L are PS-binding proteins and compete for PS binding in liposomes. Consequently, the BAR domain inhibits the catalytic activity of Nedd4L. This is consistent with the results obtained with 0.05 µm pore-size liposomes (Fig 12B). These 0.05 µm pore-size liposomes are comparable to liposomes with a higher PS percentage, as the interaction of Nedd4L with liposomes through PS is increased by membrane curvature. Therefore, when using 0.05 µm pore-size liposomes, the FCHO2 BAR domain inhibits the catalytic activity of Nedd4L.'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/62', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/63', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/64', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/65', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/66', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/67', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/68', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/69', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='Based on a model in which a subset of BAR domain proteins, such as FCHO2 and FBP17, drive liposome tubulation in vitro (Frost et al, 2008; Shimada et al, 2007), the FCHO2 BAR domain is considered to polymerize on the liposome surface into spiral dense coats by lateral and tip-to-tip interactions. When using liposomes (~50% PS), it is conceivable that the FCHO2 BAR domain densely packs the liposomes, leaving little free space for Nedd4L binding. Conversely, when using liposomes (20% PS), it is likely that the liposomes are loosely packed, providing ample surface space for Nedd4L binding.\\n\\nModel for ENaC Endocytosis\\nBased on the present study, we propose a model for ENaC endocytosis through ubiquitination (Fig 13). First, FCHO2 binds to the plasma membrane and generates a specific degree of membrane curvature. FCHO2 interacts with clathrin adaptor proteins, such as Eps15, and recruits the clathrin machinery for CCP formation (Uezu et al, 2011; Henne et al, 2010). Eps15 has Ub-interacting motifs (Di Fiore et al, 2003). Nedd4L is subsequently recruited to and activated by FCHO2-generated curved membrane at the rim of nascent CCPs that ENaC enters. ENaC is then ubiquitinated and captured by Eps15 which FCHO2 interacts with. Thus, ubiquitinated ENaC is spatially restricted to CCPs and separated from unmodified ENaC. Overall, our findings suggest that in the early stage of CME, a membrane deformation induced by FCHO2 can program, through both the recruitment and activation of Nedd4L, the gathering of a subclass of cargo molecules in CCPs.\\n\\nMethods\\n\\nConstruction'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/70', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='Human αENaC, βENaC and γENaC cDNAs were kindly supplied by Dr. H. Kai (Kumamoto University, Kumamoto, Japan) (Sugahara et al, 2009). FLAG-αENaC was constructed by replacing the sequence [amino acid (aa) 169–174, TLVAGS] in the extracellular region with the FLAG sequence (DYKDDDK) as described previously (Firsov et al, 1996). This introduction of the epitope tag was carried out by polymerase chain reaction (PCR). FLAG-αENaC, βENaC, and γENaC were subcloned into pTRE-Tight-Hygro, pCAGI-Puro (Miyahara et al, 2000), and pCAGI-Bst (Umeda et al, 2006), respectively. pTRE-Tight-Hygro was constructed by inserting the hygromycin-resistance gene into pTRE-Tight (Clontech). FLAG-αENaC was also subcloned into pCAGI-Hygro, which was constructed by replacing the puromycin-resistance gene of pCAGI-Puro with the hygromycin-resistance gene. Human Nedd4L (NM_015277, isoform 3) cDNA was obtained by reverse transcription (RT)-PCR and cloned in pCMV-Myc (Miyahara et al, 2000). Nedd4L was also cloned in pCAGI-puro-EGFP and pGEX-6P-hexahistidine (His6). pGEX-6P-His6 was constructed from pGEX-6P (GE Healthcare) to express an N-terminal GST-tagged, C-terminal His6-tagged protein. pCAGI-puro-EGFP was constructed by subcloning EGFP cDNA into pCAGI-puro to express a C-terminal GFP-tagged protein. The Nedd4L C2 domain (C2, aa 1–160) and C2 domain-deleted mutant (ΔC2, aa 150–955) were subcloned into pCMV-Myc and/or pGEX-6P. Mouse FCHO2 (full length, aa 1–809; BAR domain, aa 1–302) and human FBP17 (BAR domain, aa'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/70', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/71', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/72', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='1–300) were subcloned into pEGFP-C1 and/or pGEX-6P (Uezu et al, 2011; Tsujita et al, 2006). FCHO2 (aa 1–327) were subcloned into pGEX-6P. Human amphiphysin1 cDNA was obtained by RT-PCR and cloned in pEGFP-N3. Amphiphysin1 (BAR domain, aa 1–377) was then subcloned into pGEX-6P. mTagRFP-T-Clathrin-15 was a gift from Michael Davidson (Addgene plasmid # 58005 ; http://n2t.net/addgene:58005 ; RRID:Addgene_58005) (Shaner et al, 2008). Human hemagglutinin (HA)-tagged Ub cDNA in pCGN was kindly donated by Dr. M. Nakao (Kumamoto University, Kumamoto, Japan). Ub cDNA was subcloned into pET3a. Human UbcH7 cDNA was obtained by RT-PCR and cloned in pGEX-6P. pET21d-Ube1 was a gift from Cynthia Wolberger (Addgene plasmid # 34965 ; http://n2t.net/addgene:34965 ; RRID:Addgene_34965) (Berndsen & Wolberger, 2011). pRSET-mSA was a gift from Sheldon Park (Addgene plasmid # 39860 ; http://n2t.net/addgene:39860 ; RRID:Addgene_39860) (Lim et al, 2013). mSA-αENaC constructed in pRSET-mSA to express a fusion protein of the N-terminal (aa 7–40) and C-terminal (aa 625–659) intracellular regions of αENaC (see Fig. 9A). Mutagenesis was performed by PCR using mutated primers and a site-directed mutagenesis kit (Stratagene).\\n\\nProtein expression and purification'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/73', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='Expression and purification of GST-tagged proteins were performed using standard protocols. His6-tagged proteins were purified with Talon metal affinity resin (Clontech) using the manufacturer’s protocol. The GST tag was cleaved off by PreScission protease (GE Healthcare). The GST tag of BAR domains did not inhibit liposome binding or tubulation, whereas the GST tag of Nedd4L inhibited these functions. As FCHO2 (BAR domain, aa 1–302) was easily aggregated by the removal of the GST tag, FCHO2 (aa 1–327) was used instead as a GST-cleaved form of the FCHO2 BAR domain. Full-length Nedd4L (N-terminal GST-tagged, C-terminal His6-tagged) was purified by glutathione–Sepharose (GE Healthcare) column chromatography. The GST tag was cleaved off and the Nedd4L sample was then subjected to Talon metal affinity column chromatography. After the eluate was passed through glutathione–Sepharose beads, the sample was dialyzed against buffer A (20 mM Tris-HCl at pH 7.5, 100 mM KCl, 0.5 mM EDTA, 1 mM DTT, and 20 % sucrose). The Nedd4L sample was stored at −80°C until use. Ub was expressed in E. coli strain BL21 (DE3) pJY2 and purified as previously described (Pickart & Raasi, 2005). His6-tagged mSA-αENaC was expressed in E. coli strain BL21 (DE3) pLysS and purified as previously described (Lim et al, 2013) with slight modifications. Briefly, the inclusion body was isolated and solubilized in buffer B (50 mM Tris-HCl at pH 8.0 and 6 M guanidine hydrochloride). The sample was then subjected to Talon metal affinity column chromatography. The eluate was added drop by drop to ice-cold refolding buffer (50 mM Tris-HCl at pH 8.0, 150 mM NaCl, 0.15 mg/ml D-biotin, 0.2 mg/ml oxidized glutathione, and 1 mg/ml reduced'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/73', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/74', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/75', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/76', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/77', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/78', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='glutathione) under rapid stirring. The sample was then concentrated using a Centriprep centrifugal filter unit (Millipore) and centrifuged to remove aggregates. The supernatant was dialyzed against buffer C (20 mM Tris-HCl at pH 8.0). The mSA-αENaC sample was further purified by Mono Q anion exchange column chromatography as follows. The sample was applied to a Mono Q 5/50 GL column (GE Healthcare) equilibrated with buffer C. Elution was performed with a 50 ml linear gradient of NaCl (0–200 mM) in buffer C. Fractions of 1 ml each were collected. The active fractions (fraction 18–22) were collected and dialyzed against buffer D (10 mM Tris-HCl at pH 7.5, 100 mM KCl, and 0.5 mM EDTA). The mSA-αENaC sample was stored at −80°C until use.\\n\\nAntibodies\\nRabbit anti-Nedd4L antibody was raised against GST-Nedd4L C2 (aa 1–160). Rabbit anti-FCHO2 antibody was obtained as previously described (Uezu et al, 2011). The following antibodies were purchased from commercial sources: mouse anti-Myc (9E10) (American Type Culture Collection); mouse anti-FLAG (M2) and mouse anti-α-tubulin (clone DM1A) (Sigma-Aldrich); mouse anti-clathrin heavy chain (BD Biosciences); rabbit anti-GFP (MBL Co.); rabbit anti-αENaC (Calbiochem); rabbit anti-βENaC (Proteintech); rabbit anti-γENaC (Abcam); mouse anti-transferrin receptor H68.4 (Santa Cruz); mouse anti-Ub antibody P4D1 (Santa Cruz); and secondary antibodies conjugated with Alexa Fluor 488, 594, and 647 (Invitrogen).\\n\\nCell culture and transfection'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/79', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/80', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/81', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/82', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/83', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/84', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='HeLa, HEK293, and COS7 cells were cultured in Dulbecco’s modified Eagle medium (DMEM) supplemented with 10% fetal calf serum. HEK293 cells were grown on poly-L-lysine-coated dishes. Transfection was performed by using Lipofectamine 2000 (Invitrogen) according to the manufacturer’s protocol.\\n\\nGeneration of stable HeLa cells expressing all three ENaC subunits\\nHeLa cells were transfected with the pTet-On Advanced vector (Clontech) to develop stable Tet-On cell lines for a doxycycline (Dox)-inducible expression system according to the manufacturer’s protocol. Tet-On cells were selected in 0.5 mg/ml G418. Cloned Tet-On cells were then transfected with pCAGI-Bst-γENaC and selected in 4 µg/ml blasticidin (in the presence of G418). Blasticidin-resistant cells were transfected with pCAGI-Puro-βENaC and selected in 1 µg/ml puromycin (in the presence of G418 and blasticidin). Puromycin-resistant cells were finally transfected with pTRE-Tight-Hygro-FLAG-αENaC and cultured in 0.2 mg/ml hygromycin (in the presence of G418, blasticidin, and puromycin). After selection, cells were cloned and maintained in 0.5 mg/ml G418, 0.2 mg/ml hygromycin, 1 µg/ml puromycin, and 2.5 µg/ml blasticidin. FLAG-tagged αENaC expression was induced by treating cells with 1 µg/ml Dox in the presence of 40 µM amiloride overnight. Amiloride was added to prevent cellular Na+ overload.\\n\\nImmunofluorescence microscopy'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/85', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/86', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/87', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content=\"Cells were fixed with 3.7% formaldehyde in phosphate-buffered saline (PBS) at room temperature for 15 min, and then permeabilized with 0.2% Triton X-100 in PBS for 10 min. After blocking with 1% bovine serum albumin (BSA) in PBS, cells were sequentially incubated with primary antibodies and fluorescence-conjugated secondary antibodies at room temperature for 1 h, respectively. Cells were then analyzed with either a fluorescence microscope (BX51; Olympus) or a confocal microscopy system (BX50, Fluoview FV300, and Fluoview FV1200; Olympus). A 60 × oil immersion objective (NA = 1.40, Olympus) was used. All comparable images were acquired under identical parameters. To visualize cell-surface FLAG-αENaC in αβγENaC-HeLa cells, cells were incubated with 5 µg/ml anti-FLAG M2 antibody at 4°C for 1 h to prevent internalization. Cells were then fixed, permeabilized, and stained with a secondary antibody. Cells were also sequentially incubated with primary (anti-FCHO2) and secondary antibodies. To quantify co-localization, the Pearson's correlation coefficient was calculated from three cells using the Fiji (ImageJ) Coloc2 plug-in.\\n\\nRNA interference\"),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/88', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/89', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/90', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='Stealth™ double-stranded RNAs were purchased from Invitrogen. siRNA sequences of human FCHO2, Nedd4L, and FBP17 were 5′-CCA CAG AUC UUA GAG UGG AUU AUA A-3′ (corresponding to nucleotides 2048–2072 relative to the start codon), 5′-GAA GAG UUG CUG GUC UGG CCG UAU U-3′ (corresponding to nucleotides 1778–1802 relative to the start codon), and 5′-CCC ACU UCA UAU GUC GAA GUC UGU U-3′for human FBP17 (corresponding to nucleotides 1804–1828 relative to the start codon), respectively. A double-stranded RNA targeting luciferase (GL-2, 5′-CGU ACG CGG AAU ACU UCG AAA UGU C-3′) was used as a control. HeLa cells were transfected with 20 nM siRNA using Lipofectamine 2000 according to the manufacturer’s protocol. After 24 h, a second transfection was performed, and cells were cultured for 3 days and subjected to various experiments. For rescue experiments, cells were transfected with the intended plasmid at 36 h after the second transfection. After 18 h, cells were fixed and analyzed.\\n\\nQuantitative real-time PCR analysis'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/91', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/92', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/93', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='Total RNA was isolated using ISOGEN (Nippon Gene, 319-90211), and cDNA was synthesized using ReverTra Ace qPCR RT Master Mix with gDNA Remover (Toyobo, FSQ-301), following manufacturer’s instruction. Real time PCR was carried out using Thunderbird Next SYBR (Toyobo, QPX-201) on StepOnePlus Real-Time PCR Systems (Applied Biosystems). The sequences of qPCR primers are 5′-TCT CCC ATC GCT TCA ACG AG-3′ and 5′-GTT GCA CCC AGC TTG AGA GA-3′ for human FBP17; 5′-GTC TCC TCT GAC TTC AAC AGC G-3′ and\\xa05′-ACC ACC CTG TTG CTG TAG CCA A-3′ for\\xa0human GAPDH. The expression of target genes was normalized to GAPDH mRNA levels.\\n\\nEndocytosis assay'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/94', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='αβγENaC-HeLa cells at approximately 90% confluence in a 10-cm dish were starved with serum-free medium (DMEM containing 1% BSA, 1 µg/ml Dox, and 40 µM amiloride) at 37°C for 4 h. For ENaC endocytosis, cells were then incubated with 5 µg/ml anti-FLAG M2 in 2 ml of DMEM containing 1% BSA at 4°C for 1 h to prevent internalization. After three washes with ice-cold PBS containing 1 mM MgCl2 and 0.1 mM CaCl2 (PBS-CM), endocytosis was allowed by incubating cells in prewarmed DMEM containing 1% BSA at 37°C for the indicated periods of time. Endocytosis was stopped by placing cells on ice and washing them with ice-cold PBS-CM. Antibody bound to the cell surface but not internalized was removed by acid stripping with 0.2 M acetic acid and 0.5 M NaCl solution (pH 2.5). After washing with ice-cold PBS-CM, internalized ENaC was detected by immunoblotting (IB) or immunofluorescence microscopy. For IB, cells were scraped in 1 ml of lysis buffer A (50 mM Tris-HCl at pH 8.0, 150 mM NaCl, 5 mM EDTA, 1% Triton X-100, 1 mM PMSF, 20 µg/ml leupeptin, and 1 µg/ml pepstatin A). Cells were solubilized at 4°C for 1 h on a rotating wheel, and centrifuged at 20,000 × g at 4°C for 15 min. Protein G-Sepharose beads (GE Healthcare) were added to the supernatant and incubated at 4°C for 3 h. The beads were then thoroughly washed with lysis buffer A and boiled in sodium dodecyl sulfate (SDS) sample buffer. The sample was subjected to SDS-polyacrylamide gel electrophoresis (PAGE), followed by IB with anti-ENaC subunit antibodies. Internalization was expressed as a percentage of the initial amount of cell-surface ENaC subunit determined by incubating cells with anti-FLAG antibody at 4°C for 1 h and lysing them without incubation at'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/94', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/95', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/96', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='37°C or acid stripping. For immunofluorescence microscopy, cells were transfected with control or FCHO2 siRNA and then with GFP-siRNA-resistant (sr) FCHO2 mutant. After cells were labeled with anti-FLAG antibody on 4°C, they were incubated at 37°C for 10 min to allow ENaC endocytosis. After acid stripping, cells were fixed with 3.7% formaldehyde and permeabilized with 0.2% Triton X-100. The antibody-labeled, internalized αENaC was visualized with a fluorescence-conjugated secondary antibody. The percentage of cells with internalized αENaC among cells expressing GFP-sr-FCHO2 was determined using MetaMorph imaging System software. All images were acquired under identical parameters. For TfR endocytosis, a cell surface biotinylation assay was carried out as previously described (Roberts et al, 2001). Briefly, the cell surface was labeled with Sulfo-NHS-SS-Biotin (Thermo Fisher Scientific) at 4°C. After cells were incubated with ice-cold quenching buffer (50 mM NH4Cl in PBS-CM) at 4°C, they were incubated with serum free medium for 10 min at 37°C. Biotin was removed from the cell surface by incubation with ice-cold MesNa buffer (50 mM MesNa, 100 mM NaCl, and 100 mM Tris-HCl, pH 8.8). Cells were then incubated with iodoacetamide at 4°C. Cell lysates were then incubated with NeutrAvidin beads (Thermo Fisher Scientific). The beads were boiled in SDS sample buffer. Proteins were analyzed by IB with TfR antibody.\\n\\nENaC expression and ubiquitination at the cell surface'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/97', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/98', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/99', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='αβγENaC-HeLa cells at approximately 90% confluence in a 10-cm dish were starved with serum-free medium (DMEM containing 1% BSA, 1 µg/ml Dox, and 40 µM amiloride) at 37°C for 4 h, washed three times with ice-cold PBS, and incubated at 4°C for 45 min with 2 ml of 1 mg/ml EZ-link Sulfo-NHS-SS-Biotin in PBS with gentle shaking. Subsequently, cells were extensively washed with ice-cold Tris-buffered saline, and then scraped into 1 ml of lysis buffer B (50 mM Tris-HCl at pH 7.5, 150 mM NaCl, 5 mM EDTA, 1% Triton X-100, 10 mM N-ethylmaleimide, 1 mM PMSF, 10 µg/ml leupeptin, and 1 µg/ml pepstatin A) containing 40 µM MG132. Cells were solubilized at 4°C for 1 h on a rotating wheel, and centrifuged at 20,000 × g at 4°C for 15 min. The supernatant (4 mg of protein) was incubated at 4°C for 3 h with 10 µg of anti-FLAG M2 antibody. Protein G-Sepharose beads (50 µl of 50% slurry) were then added to the sample, which was incubated at 4°C for 3 h. After beads were thoroughly washed with lysis buffer B containing 40 µM MG132, FLAG-tagged proteins were eluted from the beads with 200 µg/ml FLAG peptide (Sigma-Aldrich) in 0.5 ml of lysis buffer B. After centrifugation, the supernatant was incubated with 25 µl of NeutrAvidin beads at 4°C overnight. After the beads were thoroughly washed with lysis buffer B, bound proteins were eluted by incubating the beads at 4°C for 1 h with 150 µl of lysis buffer B containing 200 mM DTT. After centrifugation, 5 × SDS sample buffer was added to the supernatant, which was then analyzed by IB with anti-FLAG and anti-Ub antibodies. The chemiluminescence intensity was quantified by scanning using ImageJ software (NIH).\\n\\nTIRF Microscopy'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/100', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/101', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/102', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/103', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='For live observation with TIRF microscopy, HeLa cells were grown on a glass-bottomed dish as described previously (Shimada et al, 2007). Cells were observed at 10–12 h after transfection as described previously (Henne et al, 2010). Images were acquired for 5 min at 1 s intervals with a TIRF microscopy system (Olympus), a 100 × oil immersion objective NA = 1.45 (Olympus) and MetaMorph Imaging System Software. All comparable images were acquired under identical parameters.\\n\\nLiposome preparation\\nPhospholipids were purchased from commercial sources: phosphatidylcholine (PC, 1,2-dioleoyl-sn-glycero-3-phosphocholine), phosphatidylethanolamine (PE, 1,2-dioleoyl-sn-glycero-3-phosphoethanolamine), PS (1,2-dioleoyl-sn-glycero-3-phospho-l-serine), rhodamine-PE [18:1 L-α-PE-N-(lissamine rhodamine B sulfonyl)], PEG2000-biotin-PE (1,2-distearoyl-sn-glycero-3-phosphoethanolamine-N-[biotinyl (polyethyleneglycol)-2000]), porcine brain PS, brain PI(4,5)P2, and porcine brain total lipid extract were obtained from Avanti Polar Lipids; cholesterol and Folch lipids [bovine brain lipids (Folch Fraction 1)] were obtained from Sigma-Aldrich; X-biotin-PE [N-((6-(biotinoyl)amino)hexanoyl)-1,2-dihexadecanoylsn-glycero-3-phosphoethanolamine] was obtained from Thermo Fisher Scientific; and fluorescein-PE [N-(fluorescein-5-thiocarbamoyl)-1,2-dihexadecanoyl-sn-glycero-3-phosphoethanolamine] was obtained from AAT Bioquest Inc.'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/104', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/105', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/106', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='Liposomes were prepared using brain lipids (Folch lipids, ~50% PS of total lipids), brain lipids (20% PS of total lipids, a mixture of 89.5% porcine brain total lipid extract and 10.5% porcine brain PS), or a synthetic lipid mixture [70% PC, 20% PE, 10% cholesterol (w/w), and varying percentages of PS or PI(4,5)P2 (with a corresponding reduction in PC)]. Lipid concentrations were monitored by including 1% rhodamine-PE (w/w). To prepare mSA-αENaC–associated liposomes, 2% PEG2000-biotin-PE (w/w) was included. Lipids were dried under nitrogen gas, desiccated for 2 h, and resuspended at 1 mg/ml in lipid buffer (50 mM HEPES-NaOH at pH 7.2, 100 mM KCl, 2 mM MgCl2, and 5 mM EGTA), followed by hydration at 37°C for 1 h. To make liposomes of different sizes, they were subjected to five freeze–thaw cycles and centrifuged at 100,000 × g for 15 min. Precipitated liposomes were then resuspended in the same buffer to make a final concentration of 1 mg/ml. They were subjected to extrusion through polycarbonate filter membranes (Avanti Polar Lipids): liposomes were sequentially passed 9 times through a 0.8-µm filter, 21 times through a 0.4-µm filter, 21 times through a 0.2-µm filter, 21 times through a 0.1-µm filter, 21 times through a 0.05-µm filter, and 21 times through a 0.03-µm filter. Instead of extrusion through a 0.03-µm filter, liposomes with the smallest size were also prepared by sonication three times for 5 s with a Branson Sonifier 25 at power level 1. Liposome diameters were examined by transmission electron microscopy.\\n\\nCo-sedimentation assay'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/107', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='A co-sedimentation assay was performed as described (Uezu et al, 2011) with slight modifications. Briefly, 5–10 µg of protein was incubated at 25°C for 10 min with 0.5 mg/ml liposomes in 100 µl of lipid buffer containing various concentrations of CaCl2 and centrifuged at 165,000 × g at 25°C for 15 min. Equal amounts of the supernatants and pellets were subjected to SDS-PAGE, followed by Coomassie brilliant blue (CBB) staining. Proteins were quantified by scanning using ImageJ (NIH). The assay with liposomes of different sizes was performed using NeutrAvidin (Thermo Fisher Scientific Inc) as described (Sakamoto et al, 2017). Briefly, brain-lipid (Folch-lipid) liposomes (~50% PS) were supplemented with 1% X-biotin-PE and 1% rhodamine-PE, subjected to extrusion through filter membranes, and then sonicated. Purified Nedd4L protein (10 µg) was incubated with 0.2 mg/ml liposomes in 100 µl of lipid buffer containing 2mM CaCl2 (0.1 µM Ca2+). The sample was then incubated with 2 µg of NeutrAvidin and ultracentrifuged. Both total and supernatant fluorescence were measured. The proportion of precipitated liposomes of each size was ~100%.'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/108', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='The assay with mSA-αENaC–associated liposomes was performed as follows. Liposomes were prepared from a synthetic lipid mixture [various percentages of PS or 5% PI(4,5)P2] or brain lipids supplemented with 2% PEG2000-biotin-PE and 1% rhodamine-PE. The liposomes (10 µg) were incubated with purified mSA-αENaC protein (35 pmol) at 4°C for 10 min in 25 µl of lipid buffer. The sample was then incubated at 25°C for 10 min with 0.07 µM (8 µg/ml) Nedd4L and/or 1.4 µM BAR domains in 50 µl of ubiquitination buffer (50 mM HEPES-NaOH at pH 7.2, 100 mM KCl, 2 mM MgCl2, 5 mM EGTA, and 0.1 mM DTT) containing 4 mM CaCl2 (0.7 µM Ca2+). Subsequently, the sample was subjected to ultracentrifugation. Equal amounts of the total samples and pellets were subjected to SDS-PAGE, followed by CBB staining and IB. The chemiluminescence intensity was quantified by scanning using ImageJ (NIH).'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/109', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/110', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/111', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/112', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/113', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/114', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='The assay with mSA-αENaC-associated liposomes (0.05 µm pore-size) was done as follows. Liposomes were prepared from brain lipids (20% PS) supplemented with 2% PEG2000-biotin-PE, 1% rhodamine-PE, and 0.5% fluorescein-PE (w/w) and extruded through membrane filters. To precipitate liposomes, the sample (50 μl) was first incubated with 1 µg of anti-fluorescein goat antibody (Novus Biologicals) at room temperature for 1 h and then incubated with 0.5 µg of protein A/G/L (BioVision) at room temperature for 1 h. Subsequently, the sample was centrifuged at 165,000 × g at 25°C for 15 min. Equal amounts of the supernatants and pellets were subjected to SDS-PAGE, followed by CBB staining and IB. The percentage of precipitated liposomes was ~100%.\\n\\nCalculation of free Ca2+ concentrations\\nAt 1.0, 2.0, 3.0, 3.5, 4.0, 4.5, and 4.9 mM CaCl2 in lipid buffer, free Ca2+ concentrations were calculated to be 0.04, 0.1, 0.3, 0.4, 0.7, 1.6, and 8.1 µM, respectively, as determined by the Ca-Mg-ATP-EGTA Calculator v1.0 using constants from the NIST database #46 v8. At 1.0, 2.0, 3.0, 3.5, 4.0, 4.5, and 4.9 mM CaCl2 in ubiquitination buffer, free Ca2+ concentrations were calculated to be 0.04, 0.1, 0.2, 0.4, 0.7, 1.4, and 6.7 µM, respectively.\\n\\nIn vitro tubulation assay'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/115', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/116', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/117', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/118', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/119', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/120', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='An in vitro tubulation assay was performed as described (Uezu et al, 2011; Tsujita et al, 2006) with slight modifications. Briefly, liposomes were prepared from brain lipids (Folch lipids) supplemented with 1 % rhodamine-PE. The liposomes (0.2 mg/ml liposomes) were incubated with the Nedd4L C2 domain (4.5 µM) or BAR domains (1.4 µM) at 25°C for 10 min in lipid buffer containing 3 mM CaCl2 (0.3 µM Ca2+). The sample was examined by fluorescence or electron microscopy. The tubulation assay with mSA-αENaC-associated liposomes was performed using the sample mixture utilized in the co-sedimentation assay in which mSA-αENaC-associated liposomes (brain-lipid liposomes) were used.\\n\\nNegative-staining electron microscopy\\nNegative-staining transmission electron microscopy was performed as described (Shimada et al, 2007). Box and whisker plots were used to represent the distribution of liposome diameters.\\n\\nIn vivo ubiquitination assay'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/121', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/122', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='HEK293 cells were used, because they formed membrane tubules upon FCHO2 expression more efficiently than did COS7 cells. HEK293 cells at approximately 90% confluence on a 6-well plate were transfected with 0.8 µg of pCMV-Myc-Nedd4L, 1.6 µg of pEGFP-BAR domain, and 1.6 µg of pCGN-HA-Ub. In some experiments, cells were transfected with various doses (0–1.5 µg) of pCMV-Myc-Nedd4L, 1.5 µg of pEGFP-FCHO2 BAR domain, 1 µg of pCGN-HA-Ub, and 0.5 µg of pCAGI-Hygro-FLAG-αENaC. Total transfected cDNA amount was held constant using empty pCMV-Myc vector. Amiloride (40 µM) was added to the medium when αENaC cDNA was transfected. At 21 h after transfection, cells were washed with PBS and incubated with serum-free DMEM containing 1% BSA. After 3 h incubation, 10 µM MG132 was added to the medium. After further 4 h incubation, cells were solubilized in 150 µl of lysis buffer C (50 mM Tris-HCl at pH 7.5, 150 mM NaCl, 5 mM EDTA, 10 mM N-ethylmaleimide, 1 mM PMSF, 10 µg/ml leupeptin, and 1µg/ml pepstatin A) containing 1% SDS. After boiling, the sample was sonicated and then centrifuged at 20,000 × g for 15 min. The supernatant was diluted 10-fold with lysis buffer C containing 1% Triton X-100 and then incubated at 4°C for 3 h with either anti-Myc or anti-FLAG M2 antibody. Protein G-Sepharose beads were added to the sample, which was then incubated at 4°C for 3 h. After the beads were thoroughly washed with lysis buffer C containing 1% Triton X-100, bound proteins were eluted by boiling in SDS sample buffer for 5 min. These proteins were then subjected to SDS-PAGE, followed by IB.\\n'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/123', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/124', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/125', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/126', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/127', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/128', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/129', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/130', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/131', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/132', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/133', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/134', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/135', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/136', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='In vitro ubiquitination assay\\nmSA-αENaC-associated liposomes were made by incubating liposomes (10 µg) with mSA-αENaC (35 pmol) in 25 µl of lipid buffer, as described for the co-sedimentation assay. They were then incubated at 25°C for 10 min with 0.07 µM Nedd4L, 0.04 µM E1 (Ube1), 0.7 µM E2 (UbcH7), 7 µM Ub, and various concentrations of BAR domains in 50 µl of ubiquitination buffer containing 4 mM CaCl2 (0.7 µM Ca2+) and 2 mM ATP. Where indicated, various concentrations of CaCl2 were used. The ubiquitination reaction was started by adding ATP and stopped by adding 10 µl of 6 × SDS sample. The sample was then boiled, subjected to SDS-PAGE, and immunoblotted with anti-Ub and anti-αENaC antibodies. The chemiluminescence intensity was quantified by scanning using ImageJ (NIH).\\n\\nOther procedures\\nStripping immunoblots for re-probing was carried out with the WB Stripping Solution Strong (Nacalai Tesque) according to the manufacturer’s protocol. Protein concentrations were quantified with BSA as a reference protein by the Bradford method (Bio-Rad) or the BCA method (Thermo Fisher Scientific). SDS-PAGE was performed as described (Laemmli, 1970).\\n\\nStatistical analysis\\nAll experiments were performed at least three times (biological replication) and representative results are shown. Data are shown as the mean ± S.E.M. Student’s tests, one-way analysis of variance test, and two-way analysis of variance test were used to evaluate statistical significances between different treatment groups.\\n\\nData availability\\nThis study includes no data deposited in external repositories.\\n\\nAcknowledgments\\n'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/137', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/138', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/139', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/140', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/141', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/142', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/143', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/144', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/145', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/146', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/147', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/148', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/149', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/150', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='We thank Drs. H. Kai and M. Nakao for kindly providing us with ENaC and Ub cDNAs, respectively. We are also grateful to Dr. T. Takenawa (Kobe University, Kobe, Japan) for helpful discussions. This work was partly carried out at the Institute of Molecular Embryology and Genetics, the Gene Technology Centre, and Research Facilities of the School of Medicine, Kumamoto University. This study was supported by JSPS KAKENHI Grant Numbers 19K06643 (to H.N.), 19K06544 (to Y.S.), and Setsuro Fujii Memorial, The Osaka Foundation for Promotion of Fundamental Medical Research (to Y.S.) and by grants from the Mochida Memorial Foundation, the Takeda Science Foundation, and the Uehara Memorial Foundation (to K.K.).\\n\\nDisclosure and Competing Interests Statement\\nThe authors declare no competing interests.\\n\\n\\nReferences\\n\\nAguet F, Antonescu CN, Mettlen M, Schmid S & Danuser G (2013) Advances in analysis of low signal-to-noise images link dynamin and AP2 to the function of an endocytic checkpoint. Dev Cell 26: 279–291\\nBerndsen CE & Wolberger C (2011) A spectrophotometric assay for conjugation of ubiquitin and ubiquitin-like proteins. Anal Biochem 418: 102–110\\nButterworth MB (2010) Regulation of the epithelial sodium channel (ENaC) by membrane trafficking. Biochim Biophys Acta 1802: 1166–77\\nChang-Ileto B, Frere SG, Chan RB, Voronov SV, Roux A, & Di Paolo G (2011) Synaptojanin 1-mediated PI(4,5)P2 hydrolysis is modulated by membrane curvature and facilitates membrane fission. Dev. Cell 20: 206–218\\nDoherty GJ & McMahon HT (2009) Mechanisms of endocytosis. Annu Rev Biochem 78: 857–902\\nDi Fiore PP, Polo S & Hofmann K (2003) When ubiquitin meets ubiquitin receptors: a signalling connection. Nat Rev Mol Cell Biol 4: 491–497'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/151', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/152', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/153', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/154', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/155', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/156', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/157', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/158', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='Escobedo A, Gomes T, Aragon E, Martin-Malpartida P, Ruiz L, & Macias MJ (2014) Structural basis of the activation and degradation mechaisms of the E3 ubiquitin lagase Nedd4L. Structure 22: 1446–1457\\nFirsov D, Schild L, Gautschi I, Mérillat AM, Schneeberger E & Rossier BC (1996) Cell surface expression of the epithelial Na channel and a mutant causing Liddle syndrome: a quantitative approach. Proc Natl Acad Sci U S A 93: 15370–15375\\nFrost A, Perera R, Roux A, Spasov K, Destaing O, Egelman EH, De Camilli P & Unger VM (2008) Structural basis of membrane invagination by F-BAR domains. Cell 132: 807–817\\nHenne WM, Boucrot E, Meinecke M, Evergren E, Vallis Y, Mittal R & McMahon HT (2010) FCHo proteins are nucleators of clathrin-mediated endocytosis. Science 328: 1281–4\\nHenne WM, Kent HM, Ford MGJ, Hegde BG, Daumke O, Butler PJG, Mittal R, Langen R, Evans PR & McMahon HT (2007) Structure and Analysis of FCHo2 F-BAR Domain: A Dimerizing and Membrane Recruitment Module that Effects Membrane Curvature. Structure 15: 839–852\\nHershko A & Ciechanover A (1998) THE UBIQUITIN SYSTEM. Annu Rev Biochem 67: 425–479\\nHughey RP, Mueller GM, Bruns JB, Kinlough CL, Poland PA, Harkleroad KL, Carattino MD & Kleyman TR (2003) Maturation of the epithelial Na+ channel involves proteolytic processing of the alpha- and gamma-subunits. J Biol Chem 278: 37073–37082\\nHui E, Johnson CP, Yao J, Dunning FM & Chapman ER (2009) Synaptotagmin-mediated bending of the target membrane is a critical step in Ca(2+)-regulated fusion. Cell 138: 709–21'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/159', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/160', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/161', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/162', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/163', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/164', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/165', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/166', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/167', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/168', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/169', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='Ingham RJ, Gish G & Pawson T (2004) The Nedd4 family of E3 ubiquitin ligases: functional diversity within a common modular architecture. Oncogene 23: 1972–84\\nLaemmli UK (1970) Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature 227: 680–685\\nLehmann M, Lukonin I, Noe F, Schmoranzer J, Clementi C, Loerke D & Haucke V (2019) Nanoscale coupling of endocytic pit growth and stability. Sci Adv 5: eaax5775\\nLemmon M (2008) Membrane recognition by phospholipid-binding domains. Nat Rev Mol Cell Biol 9: 99–111\\nLim KH, Huang H, Pralle A & Park S (2013) Stable, high-affinity streptavidin monomer for protein labeling and monovalent biotin detection. Biotechnol Bioeng 110: 57–67\\nMartens S, Kozlov MM & McMahon HT (2007) How synaptotagmin promotes membrane fusion. Science 316: 1205–8\\nMiyahara M, Nakanishi H, Takahashi K, Satoh-Horikawa K, Tachibana K & Takai Y (2000) Interaction of nectin with afadin is necessary for its clustering at cell-cell contact sites but not for its cis dimerization or trans interaction. J Biol Chem 275: 613–618\\nNishizuka Y (1992) Intracellular signaling by hydrolysis of phospholipids and activation of protein kinase C. Science 258: 607–14\\nPeter BJ, Kent HM, Mills IG, Vallis Y, Butler PJG, Evans PR & McMahon HT (2004) BAR domains as sensors of membrane curvature: the amphiphysin BAR structure. Science 303: 495–9\\nPickart CM & Raasi S (2005) Controlled synthesis of polyubiquitin chains. Methods Enzym 399: 21–36\\nRoberts M, Barry S, Woods A, van der Sluijs P, & Norman J (2001) PDGF-regulated rab4-dependent recycling of avb3 integrin from early endosomes is necessary for cell adhesion and spreading. Curr Biol 11: 1392–1402'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/170', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/171', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/172', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/173', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/174', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/175', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/176', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/177', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/178', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='Rossier BC (2014) Epithelial sodium channel (ENaC) and the control of blood pressure. Curr Opin Pharmacol 15: 33–46\\nRotin D & Kumar S (2009) Physiological functions of the HECT family of ubiquitin ligases. Nat Rev Mol Cell Biol 10: 398–409\\nRotin D & Staub O (2011) Role of the ubiquitin system in regulating ion transport. Pflugers Arch 461: 1–21\\nSaffarian S, Coccuci E, & Kirchausen T (2009) Distinct dynamics of endocytic clathrin-coated pits and coated palques. PLoS Biol 7: e1000191\\nSakamoto Y, Kikuchi K, Umeda K & Nakanishi H (2017) Effects of various spacers between biotin and the phospholipid headgroup on immobilization and sedimentation of biotinylated phospholipid-containing liposomes facilitated by avidin-biotin interactions. J Biochem 162: 221–226\\nShaner NC, Lin MZ, McKeown MR, Steinbach PA, Hazelwood KL, Davidson MW & Tsien RY (2008) Improving the photostability of bright monomeric orange and red fluorescent proteins. Nat Methods 5: 545–551\\nShimada A, Niwa H, Tsujita K, Suetsugu S, Nitta K, Hanawa-Suetsugu K, Akasaka R, Nishino Y, Toyama M, Chen L, et al (2007) Curved EFC/F-BAR-domain dimers are joined end to end into a filament for membrane invagination in endocytosis. Cell 129: 761–772\\nSugahara T, Koga T, Ueno-Shuto K, Shuto T, Watanabe E, Maekawa A, Kitamura K, Tomita K, Mizuno A, Sato T, et al (2009) Calreticulin positively regulates the expression and function of epithelial sodium channel. Exp Cell Res 315: 3294–3300\\nTakano K, Toyooka K & Suetsugu S (2008) EFC/F-BAR proteins and the N-WASP–WIP complex induce membrane curvature-dependent actin polymerization. EMBO J 27: 2817–2828'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/179', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/180', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/181', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/182', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/183', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/184', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/185', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/186', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='Takei K, Slepnev VI, Haucke V & De Camilli P (1999) Functional partnership between amphiphysin and dynamin in clathrin-mediated endocytosis. Nat Cell Biol 1: 33–39\\nTraub LM (2009) Tickets to ride: selecting cargo for clathrin-regulated internalization. Nat Rev Mol Cell Biol 10: 583–96\\nTsujita K, Suetsugu S, Sasaki N, Furutani M, Oikawa T & Takenawa T (2006) Coordination between the actin cytoskeleton and membrane deformation by a novel membrane tubulation domain of PCH proteins is involved in endocytosis. J Cell Biol 172: 269–279\\nUezu A, Umeda K, Tsujita K, Suetsugu S, Takenawa T & Nakanishi H (2011) Characterization of the EFC/F-BAR domain protein, FCHO2. Genes Cells 16: 868–878\\nUmeda K, Ikenouchi J, Katahira-Tayama S, Furuse K, Sasaki H, Nakayama M, Matsui T, Tsukita S, Furuse M & Tsukita S (2006) ZO-1 and ZO-2 independently determine where claudins are polymerized in tight-junction strand formation. Cell 126: 741–754\\nWang J, Peng Q, Lin Q, Childress C, Carey D & Yang W (2010) Calcium activates Nedd4 E3 ubiquitin ligases by releasing the C2 domain-mediated auto-inhibition. J Biol Chem 285: 12279–12288\\nWiesner S, Ogunjimi AA, Wang H-R, Rotin D, Sicheri F, Wrana JL & Forman-Kay JD (2007) Autoinhibition of the HECT-type ubiquitin ligase Smurf2 through its C2 domain. Cell 130: 651–862\\nYang B & Kumar S (2010) Nedd4 and Nedd4-2: closely related ubiquitin-protein ligases with distinct physiological functions. Cell Death Differ 17: 68–77'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/187', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/188', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/189', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/190', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/191', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/192', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/193', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/194', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/195', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/196', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/197', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content=\"Yin Y, Arkhipov A & Schulten K (2009) Simulations of Membrane Tubulation by Lattices of Amphiphysin N-BAR Domains. Structure 17: 882–892\\nZaccai N, Kadlecova Z, Dickson VK, Korobchevskaya K & Traub L (2022) FCHO control AP2's initiating role in endocytosis through a Ptd-Ins(4,5)P2-dependent switch. Sci Adv 8: eabn2018\\nZhou R, Patel S V. & Snyder PM (2007) Nedd4-2 catalyzes ubiquitination and degradation of cell surface ENaC. J Biol Chem 282: 20207–20212\\nZhu K, Shan Z, Chen X, Cai Y, Cui L, Yao W, Wang Z, Shi P, Tian C, Lou J, et al (2017) Allosteric auto‐inhibition and activation of the Nedd4 family E3 ligase Itch. EMBO Rep 18: 1618–1630\\n\\n\\n\\nFigure legends\\n\\nFigure 1. FCHO2 is required for Nedd4L-mediated ubiquitination and endocytosis of ENaC.\\nA. Knockdown of FCHO2 or Nedd4L by siRNA. αβγENaC-HeLa cells treated with each siRNA were analyzed by immunoblotting (IB).\"),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/198', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/199', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content=\"B. Inhibition of ENaC endocytosis by knockdown of FCHO2 or Nedd4L. After cells were transfected with each siRNA, cell-surface ENaC was labeled by incubating at 4°C with anti-FLAG antibody. Endocytosis was started by incubating cells at 37°C for the indicated periods of time and stopped by chilling cells on ice. Antibody bound to the cell surface but not internalized was removed by acid stripping. After cells were solubilized, the antibody-labeled internalized ENaC subunits were precipitated and analyzed by IB (upper panel) and quantified (bottom and right panels). Internalization was expressed as a percentage of the initial amount of cell-surface αENaC determined by lysing cells without incubation at 37°C or acid stripping. Data are shown as the mean ± S.E.M. of three independent experiments. **P < 0.01; ***P < 0.001 (Student's t-test).\\nC. Inhibition of transferrin receptor (TfR) endocytosis by FCHO2 knockdown. After cells were transfected with each siRNA, cell-surface proteins were biotinylated at 4°C. Cells were then incubated at 37°C for 10 min. Biotin was removed from the cell surface, and internalized biotinylated proteins were precipitated with avidin beads. Samples were subjected to IB with anti-TfR antibody and quantified. Internalization was expressed as a proportion of the initial amount of TfR on the cell surface. Data are shown as the mean ± S.E.M. of three independent experiments. *P < 0.05 (Student's t-test).\"),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/200', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/201', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/202', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/203', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content=\"D. Restoration of αENaC endocytosis by FCHO2 re-expression in knockdown cells. After cells were transfected with the indicated siRNA and GFP construct, cell-surface αENaC was labeled with anti-FLAG antibody at 4°C. Cells were then incubated at 37°C for 10 min. After acid stripping, cells were subjected to immunofluorescence microscopy (left panels) and quantified (right panel). The antibody-labeled, internalized αENaC was visualized with a fluorescence-conjugated secondary antibody. The border of each cell is delineated by a solid line. Arrows indicate cells expressing GFP-tagged proteins. Scale bars, 10 μm. The proportion of cells with internalized αENaC among cells expressing control GFP or GFP-siRNA-resistant (sr) -FCHO2 is shown. Data are shown as the mean ± S.E.M. of five independent images (31-71 cells expressing GFP-tagged proteins per image). ***P < 0.001 (two-way analysis of variance with post hoc test).\\nE, F. Effects of FCHO2, Nedd4L, and FBP17 knockdown on ENaC ubiquitination and expression at the cell surface. After cells were transfected with each siRNA, cell-surface αENaC was biotinylated and sequentially precipitated with anti-FLAG antibody and then avidin beads. Samples were subjected to IB with anti-FLAG and anti-Ub antibodies (E) and quantified (F). E, ENaC ubiquitination, Asterisks indicate the same band of ubiquitinated FLAG-αENaC. F, ENaC expression at the cell surface. Data are shown as the mean ± S.E.M. of three independent experiments. *P < 0.05 (Student's t-test).\\n\\nFigure 2. Localization of cell-surface ENaC, FCHO2, and Nedd4L at clathrin-coated structures.\"),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/204', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/205', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/206', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/207', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/208', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/209', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='A, B. Localization of cell-surface ENaC. mTagRFP-clathrin light chain (CLC) (A) or Nedd4L C922A (catalytic inactive mutant)-GFP (B) was expressed in αβγENaC-HeLa cells. Cell-surface αENaC was labeled with anti-FLAG antibody at 4°C. Cells were then fixed and stained with anti-FCHO2 antibody. Samples were subjected to immunofluorescence microscopy (left panel). Insets represent enlarged images of the dashed boxes. Scale bar, 10 μm. Co-localization was analyzed (right panel). The mean values of Pearson’s correlation coefficient from three cells and the S.E.M. are shown. A. Co-localization of cell-surface ENaC with FCHO2 at clathrin-coated structures. B. Co-localization of cell-surface ENaC with Nedd4L C922A mutant and FCHO2.\\nC. Co-localization of Nedd4L with FCHO2 at clathrin-coated structures. Nedd4L-GFP and mTagRFP-CLC were co-expressed in wild-type HeLa cells. Cells were then fixed and stained with anti-FCHO2 antibody (left panel). Insets represent enlarged images of the dashed boxes. Scale bar, 10 μm. Co-localization was analyzed (right panel). The mean values of Pearson’s correlation coefficient from three cells and the S.E.M. are shown.\\n\\nFigure 3. Recruitment of Nedd4L to CCPs. DsRed-CLC were co-expressed with Nedd4L-GFP in wild-type HeLa cells. Images were acquired at 1 s intervals through TIRF microscopy. Upper panel, TIRF images of HeLa cells. Insets represent enlarged images of the dashed boxes. Bottom panel, selected snapshots (every 10 sec) from a time series were obtained from a single CCP indicated by the arrowhead. The time point at which Nedd4L started to accumulate was set 0. Scale bar, 2 μm.\\n\\nFigure 4. Recruitment of Nedd4L to membrane tubules generated by FCHO2 in cells.'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/210', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/211', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/212', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/213', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/214', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/215', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/216', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/217', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/218', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='GFP-BAR domains were co-expressed with Myc-Nedd4L proteins in COS7 cells (A) and HEK293 cells (B). Cells were serum-starved and subjected to immunofluorescence microscopy. Insets represent enlarged images of the dashed boxes. Scale bar, 10 μm.\\n\\nFigure 5. Activation of Nedd4L by membrane tubules generated by FCHO2 in cells.\\nA–E. Nedd4L autoubiquitination. An in vivo ubiquitination assay was performed with various BAR domains and FCHO2 mutants (A, B) and Nedd4L mutants (D). GFP-BAR domains were co-expressed with Myc-Nedd4L proteins and HA-tagged Ub in HEK293 cells. Cell lysates were subjected to immunoprecipitation (IP) and analyzed by IB (A, B, D). WT, wild type. Cells were also subjected to immunofluorescence microscopy (C, E). Scale bar, 10 μm.\\nF. αENaC ubiquitination. An in vivo ubiquitination assay was performed with various concentrations of Myc-Nedd4L using FALAG-αENaC as a substrate in the presence or absence of GFP-FCHO2 BAR domain. Cell lysates were subjected to IP. Samples were analyzed by IB.\\n\\nFigure 6. The Nedd4L C2 domain is a Ca2+-dependent PS binding domain.\\nA co-sedimentation assay was performed with the C2 domain or full-length Nedd4L using brain-lipid liposomes (~50% PS) at various Ca2+ concentrations (A) and with the C2 domain using synthetic liposomes containing various percentages of PS at 0.3 μM or 1.0 mM Ca2+ (B). The supernatants (S) and pellets (P) were subjected to sodium dodecyl sulfate (SDS)-polyacrylamide gel electrophoresis (PAGE), followed by Coomassie brilliant blue (CBB) staining (upper panels). Bottom panels, quantitative analysis. (−) for [Ca2+] indicates EGTA alone. Data are shown as the mean ± S.E.M. of three independent experiments.\\nA. Ca2+ dependency.'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/219', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/220', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/221', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/222', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='B. PS dependency.\\n\\nFigure 7. Preference of Nedd4L for FCHO2-generated membrane curvature.\\nA. Distribution of liposome diameters. Brain-lipid liposomes (~50% PS) of different diameters were prepared by extrusion through the indicated pore sizes and sonication. Left panel, electron microscopic images. Scale bars, 200 nm. Right panel, distribution of liposome diameters shown by boxplots (number of observations per pore size = 25). The center line inside the box corresponds to the median, the bounds of the box encompass data points between the first and third quartiles, and the whiskers extend to the minimum and maximum values including outliers. So, sonication.'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/223', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/224', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/225', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='B-D. Sensing and generation of membrane curvature by Nedd4L. Co-sedimentation (B) and in vitro tubulation (C, D) assays were performed using brain-lipid liposomes (~50% PS) at 0.1 μM (B) and 0.3 μM Ca2+ (C, D). B, Preference of Nedd4L for a specific degree of membrane curvature. Liposomes were supplemented with X-biotin-PE and rhodamine-PE. Full-length Nedd4L was incubated with liposomes of different sizes, which were prepared by extrusion through the indicated pore sizes. The sample was then incubated with avidin beads and ultracentrifuged. The proportion of precipitated liposomes of each size was calculated to be ~100% by measuring both total and supernatant fluorescence. The supernatants (S) and pellets (P) were subjected to SDS- PAGE, followed by CBB staining (upper panel). Bottom panel, quantitative analysis. So, sonication. Data are shown as the mean ± S.E.M. of five independent experiments. *P < 0.05; **P < 0.01 (one-way analysis of variance with Tukey’s post hoc test). C, Curvature generation by Nedd4L. The assay was performed with the Nedd4L C2 domain. Samples were examined by electron microscopy. Scale bar, 100 nm. D, The degree of membrane curvature generated by Nedd4L is consistent with that generated by FCHO2. The assay was performed with the indicated protein. Left panel, electron microscopic image. Scale bar, 100 nm. Right panel, distribution of tubule diameters shown by boxplots (number of observations per protein = 14–33). The center line inside the box corresponds to the median, the bounds of the box encompass data points between the first and third quartiles, and the whiskers extend to the minimum and maximum values including outliers.\\n\\nFigure 8. Conserved hydrophobic residues of Nedd4L are responsible for sensing and generation of membrane curvature.'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/226', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/227', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/228', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/229', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/230', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='A. Sequence alignment of C2 domains. Conserved residues are highlighted with the following color code: yellow, aspartates for Ca2+ binding; blue, very hydrophobic residues probably for penetration into the lipid bilayer; and brown, others. Dots show the mutation sites. Hs, Homo sapiens; Sc, Saccharomyces cerevisiae; and Rt, Rattus norvegicus.\\nB. Inability of Nedd4L C2 domain mutants to sense membrane curvature in cells. GFP-FCHO2 BAR domain and Myc-Nedd4L C2 domain mutants were co-expressed in COS7 cells. Insets are enlarged images of dashed boxes. Scale bar, 10 μm.\\nC, D. Inability of Nedd4L C2 domain mutants to bind to membranes and to generate membrane curvature in vitro. Co-sedimentation (C) and in vitro tubulation (D) assays were performed using brain-lipid liposomes (~50% PS) at 0.3 μM Ca2+. C, Membrane binding. The assay was performed with Nedd4L C2 domain mutants using various concentrations of liposomes. The supernatants (S) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining (upper panel). Bottom panel, quantitative analysis. Data are shown as the mean ± S.E.M. of three independent experiments. D, Curvature generation. The assay was performed with Nedd4L C2 domain mutants. Samples were examined by fluorescence microscopy. Scale bar, 10 μm.\\n\\nFigure 9. Properties of the catalytic activity and membrane binding of Nedd4L.'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/231', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/232', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/233', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/234', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/235', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='Liposomes supplemented with biotinylated phospholipids were incubated with mSA-αENaC (A, B). Samples were mixed with Nedd4L. Mixtures were subjected to ubiquitination (C-E) and co-sedimentation (F) assays. These assays were performed with brain-lipid liposomes (~50% PS) at various Ca2+ concentrations (C), with synthetic liposomes (various percentages of PS) at 0.7 μM Ca2+ (D, F), and with brain-lipid liposomes (20% PS) at 0.7 μM Ca2+ (E). Ubiquitination and liposome binding were quantified by scanning and expressed as ratios to maximum levels. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.\\nA, B. Schematic diagrams of mSA-αENaC. A, Structure. B. Binding to biotinylated liposomes.\\nC. Ca2+-dependent Nedd4L activity. Ubiquitination was detected by IB (upper panel) and the intensity was quantified by scanning (bottom panel). Data are shown as the mean ± S.E.M. of three independent experiments.\\nD, Membrane localization of mSA-αENaC enhances the PS-dependent catalytic activity of Nedd4L. Assays were performed in the presence or absence of 40 μM biotin. Ubiquitination was detected by IB (left panel). The intensities were quantified by scanning (right panel). Data are shown as the mean ± S.E.M. of three independent experiments.\\nE. Nedd4L activity is predominantly stimulated by the specific degree of membrane curvature. The assay was performed using liposomes of different sizes which were prepared by extrusion through the indicated pore sizes. Ubiquitination was detected by IB (left panel) and quantified (right panel). Data are shown as the mean ± S.E.M. of three independent experiments. ***P < 0.001 (one-way analysis of variance with Tukey’s post hoc test).'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/236', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/237', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/238', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/239', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/240', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/241', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='F. Membrane localization of mSA-αENaC enhances the PS-dependent membrane binding of Nedd4L. Assays were performed in the presence or absence of 40 μM biotin. Liposome binding was detected by IB (left panel). The intensities were quantified by scanning (right panel). T, total sample. P, pellet. Data are shown as the mean ± S.E.M. of three independent experiments.\\n\\n\\nFigure 10. Recruitment and activation of Nedd4L by FCHO2-generated membrane curvature in vitro.\\nmSA-αENaC-associated brain-lipid liposomes (20% PS) were mixed with Nedd4L in the presence or absence of BAR domains. Mixtures were subjected to co-sedimentation (A) and ubiquitination (B-E) assays. These assays were performed at 0.7 μM Ca2+ with brain-lipid liposomes (20%PS) (A, B, D, and E) and with the indicated (20% PS or ~50% PS) brain-lipid liposomes (C). Ubiquitination and liposome binding were quantified by scanning and expressed as ratios to maximum levels. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.\\nA. FCHO2-induced recruitment of Nedd4L. The assay was performed with the FCHO2 BAR domain (1.4 μM, 50 μg/ml) in the presence or absence of biotin. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining (FCHO2 BAR) and IB (Nedd4L and mSA-αENaC).'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/242', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/243', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/244', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/245', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/246', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='B, C. FCHO2-induced activation of Nedd4L. B, Effects of various doses of the FCHO2 BAR domain. The assay was performed with various concentrations of the FCHO2 BAR domain. Ubiquitination was detected by IB (left panel) and the intensity was quantified by scanning (right panel). C, Effects of the PS percentage in liposomes. The assay was performed with the FCHO2 BAR domain (1.4 μM). Samples were analyzed by IB. Data are shown as the mean ± S.E.M. of three independent experiments.\\nD, E. Specificity of BAR domains and effect of an FCHO2 mutant. The assay was performed with various BAR domains (D) and an FCHO2 mutant (E) (1.4 μM each). Samples were analyzed by IB.\\n\\nFigure 11. Effects of PI(4,5)P2 on FCHO2-induced recruitment and activation of Nedd4L.\\nA, B. Binding to PI(4,5)P2. A co-sedimentation assay was performed at 0.7 μM Ca2+ with the FCHO2 BAR or Nedd4L C2 domain using mSA-αENaC-associated synthetic liposomes containing 20% PS, 5% PI(4,5)P2 (A), or various percentages of PI(4,5)P2 (B). The supernatants (S) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining (A and upper panel in B). Bottom panel in B, quantitative analysis. Control, synthetic liposomes [0% PS and 0% PI(4,5)P2]. Data are shown as the mean ± S.E.M. of three independent experiments.'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/247', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/248', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/249', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/250', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/251', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/252', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/253', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='C, D. Effects on FCHO2-induced recruitment and activation of Nedd4L. Co-sedimentation (C) and ubiquitination (D) assays were performed at 0.7 μM Ca2+ with mSA-αENaC-associated synthetic liposomes containing 20% PS or 5% PI(4,5)P2 in the presence or absence of the FCHO2 BAR domain. C, Nedd4L recruitment. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by IB with anti-Nedd4L antibody (upper panel) and CBB staining (lower panel). D, Nedd4L activation. Samples were analyzed by IB. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.\\n\\nFigure 12. Effects of liposome size on FCHO2-induced activation of Nedd4L. Co-sedimentation (A) and ubiquitination (B) assays were performed at 0.7 μM Ca2+ with the FCHO2 BAR domain using brain-lipid liposomes (20% PS, 0.8 µm or 0.05 µm pore-size) that were associated with mSA-αENaC.\\nA. Binding of the FCHO2 BAR domain to 0.8 µm and 0.05 µm pore-size liposomes. The co-sedimentation assay was performed using synthetic (0% PS) or brain-lipid liposomes (20% PS) containing rhodamine-PE and fluorescein-PE. Liposomes were precipitated with anti-fluorescein antibody. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining. The proportion of precipitated liposomes of each size was calculated to be ~100% by measuring both total and supernatant fluorescence.\\nB. Effects of liposome size on FCHO2-induced activation of Nedd4L. Samples were analyzed by IB. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.\\n\\nFigure 13. Model of ENaC endocytosis.'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/254', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/255', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/256', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/257', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/258', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/259', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/260', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/261', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/262', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='FCHO2 binds to the plasma membrane and generates a specific degree of membrane curvature according to the intrinsic curvature of its BAR domain. Nedd4L exists in a catalytically autoinhibited state due to an intramolecular interaction between the N-terminal C2 and C-terminal HECT domains. Nedd4L is recruited to FCHO2-generated membrane curvature at the rim of nascent CCPs that ENaC enters. Membrane binding of the C2 domain causes catalytic activation by relieving autoinhibition. ENaC is ubiquitinated and captured by adaptor proteins with Ub-interacting motifs, such as Eps15.\\n\\nExtended View Figure Legends\\n\\nFigure EV1. Expression of ENaC subunits in αβγENaC-HeLa cells.\\nA, B. Doxycycline (Dox)-induced expression of αENaC. αβγENaC-HeLa cells were cultured overnight in the presence or absence of Dox. The lysates of parental HeLa and αβγENaC-HeLa cells were subjected to IB. A, 7.5% gel. B, Gradient gel (5-20%). Anti-γENaC antibody cross-reacted with two bands of endogenous proteins in parental HeLa cells, the lower band of which overlapped with γENaC (arrow) in αβγENaC-HeLa cells. Upon Dox-induced αENaC expression, an additional 70-kDa γENaC band (asterisk) was detected. It has been shown that co-expression of all three subunits induces ENaC maturation, including proteolytic cleavage of α- and γENaC (Hughey et al, 2003). The 70-kDa γENaC band is likely a cleavage product comprising the C-terminal region. The 20-kDa αENaC band (arrowhead) is likely a cleavage product comprising the N-terminal region.\\nC. Association of α-, β-, and γENaC. When αENaC was immunoprecipitated with anti-FLAG antibody from αβγENaC-HeLa cells treated with Dox, β- and γENaC were co-precipitated. Asterisk, 70-kDa γENaC band.\\n\\nFigure EV2. Knockdown of FBP17 by siRNA.'),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/263', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/264', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/265', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/266', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/267', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/268', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/269', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/270', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/271', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/272', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/273', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}, {'self_ref': '#/texts/274', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content=\"αβγENaC-HeLa cells were treated with each siRNA, and FBP17 mRNA levels were quantified using real-time PCR. The expression of FBP17 was normalized to GAPDH mRNA levels. Data are shown as the mean ± S.E.M. of three independent experiments. ***P < 0.001 (Student's t-test).\\n\\nFigure EV3. Inability of FCHO2 mutants to generate membrane tubules.\\nGFP-FCHO2 BAR domain [wild type (WT) or mutant] was expressed in COS7 cells. Cells were then subjected to immunofluorescence microscopy. Scale bar, 10 μm.\\n\\nFigure EV4. Inability of Nedd4L mutants to ubiquitinate αENaC. An in vivo ubiquitination assay was performed with various Nedd4L constructs (each 0.5 µg) using FLAG-αENaC as a substrate in the presence or absence of GFP-FCHO2 BAR domain. Cell lysates were subjected to IP. Samples were analyzed by IB.\\n\\nFigure EV5. Inhibition of the liposome binding of Nedd4L by the FCHO2 BAR domain.\\nA co-sedimentation assay was performed at 0.7 μM Ca2+ with control liposomes (0% PS) or brain-lipid liposomes (~50% PS) in the presence or absence of the FCHO2 BAR domain. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by IB (upper panel) and CBB staining (lower panel).\\n\\nFigure EV6. Membrane binding and curvature generation of BAR domains and an FCHO2 mutant. Co-sedimentation (A) and in vitro tubulation (B) assays were performed at 0.7 μM Ca2+ with the indicated BAR domains using brain-lipid liposomes (20% PS) that were associated with mSA-αENaC.\\nA. Membrane binding. The supernatants (S) and pellets (P) were subjected to SDS-PAGE followed by CBB staining.\"),\n",
       " Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx', 'dl_meta': {'schema_name': 'docling_core.transforms.chunker.DocMeta', 'version': '1.0.0', 'doc_items': [{'self_ref': '#/texts/275', 'parent': {'$ref': '#/body'}, 'children': [], 'label': 'paragraph', 'prov': []}], 'origin': {'mimetype': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'binary_hash': 14961976322973087418, 'filename': 'EMBOJ-2023-114687.docx'}}}, page_content='B. Curvature generation. Left panel, electron microscopic image. Scale bar, 100 nm. Right panel, distribution of tubule diameters shown by boxplots (number of observations per protein = 22–26). The center line inside the box corresponds to the median, the bounds of the box encompass data points between the first and third quartiles, and the whiskers extend to the minimum and maximum values including outliers.')]"
      ]
     },
     "execution_count": 46,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from langchain_docling import DoclingLoader\n",
    "\n",
    "loader = DoclingLoader(file_path=DOCX_FILES[1])\n",
    "docs = loader.load()\n",
    "docs\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 4. Everything to Markdown\n",
    "\n",
    "Uses the pandoc and xml engines to extract the data from DOCX and DOC files\n",
    "\n",
    "Can be used together with LLM to convert to different formats\n",
    "\n",
    "No clear how we could use this to directly extract the sections of the document."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### PANDOC ENGINE"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 72,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "**Title: Targeting the transcription factor YY1 is synthetic lethal with\n",
      "loss of the histone demethylase KDM5C**\n",
      "\n",
      "**Qian Zheng<sup>1#</sup>, Pengfei Li<sup>2#</sup>, Yulong\n",
      "Qiang<sup>1#</sup>, Jiachen Fan<sup>1</sup>, Yuzhu Xing<sup>1</sup>,\n",
      "Ying Zhang<sup>1</sup>, Fan Yang<sup>1</sup>\\*, Feng Li<sup>1,3</sup>\\*\n",
      "and Jie Xiong<sup>4</sup>\\***\n",
      "\n",
      "<sup>1</sup>Department of Medical Genetics, TaiKang Medical School\n",
      "(School of Basic Medical Sciences), Wuhan University, Wuhan 430071,\n",
      "China\n",
      "\n",
      "<sup>2</sup>Inner Mongolia Key Laboratory of Molecular Pathology, Inner\n",
      "Mongolia Medical University, Huhhot, Inner Mongolia, 010059, China\n",
      "\n",
      "<sup>3</sup>Hubei Provincial Key Laboratory of Allergy and Immunology,\n",
      "Wuhan University, Wuhan 430071, China\n",
      "\n",
      "<sup>4</sup>Department of Immunology, TaiKang Medical School (School of\n",
      "Basic Medical Sciences), Wuhan University, Wuhan 430071, China\n",
      "\n",
      "<sup>\\#</sup>These authors contribute equally to this paper.\n",
      "\n",
      "Correspondence to:\n",
      "\n",
      "**Jie Xiong**\n",
      "\n",
      "E-mail: <jiexiong@whu.edu.cn>，Tel: +86 27 68759222\n",
      "\n",
      "Department of Immunology, TaiKang Medical School (School of Basic\n",
      "Medical Sciences), Wuhan University, Wuhan 430071, China\n",
      "\n",
      "**Feng Li**\n",
      "\n",
      "E-mail: fli222@whu.edu.cn, Tel: +86 27 68759222\n",
      "\n",
      "> Department of Medical Genetics, TaiKang Medical School (School of\n",
      "> Basic Medical Sciences), Wuhan University, Wuhan 430071, China.\n",
      "\n",
      "> Hubei Provincial Key Laboratory of Allergy and Immunology, Wuhan\n",
      "> University, Wuhan 430071, China\n",
      "\n",
      "> **Fan Yang**\n",
      "\n",
      "E-mail: yangfan2022@whu.edu.cn, Tel: +86 27 68759222\n",
      "\n",
      "> Department of Medical Genetics, TaiKang Medical School (School of\n",
      "> Basic Medical Sciences), Wuhan University, Wuhan 430071, China.\n",
      "\n",
      "**\n",
      "**\n",
      "\n",
      "**Abstract**\n",
      "\n",
      "An understanding of the enzymatic and scaffolding functions of\n",
      "epigenetic modifiers is important for the development of epigenetic\n",
      "therapies for cancer. The H3K4me2/3 histone demethylase KDM5C has been\n",
      "shown to regulate transcription. The diverse roles of KDM5C are likely\n",
      "determined by its interacting partners, which are still largely unknown.\n",
      "In this study, we screen for KDM5C-binding proteins and show that YY1\n",
      "interacts with KDM5C. A synergistic antitumor effect is exerted when\n",
      "both KDM5C and YY1 are depleted, and targeting YY1 appears to be a\n",
      "vulnerability in KDM5C-deficient cancer cells. Mechanistically, KDM5C\n",
      "promotes global YY1 chromatin recruitment, especially at promoters.\n",
      "Moreover, an intact KDM5C JmjC domain but not KDM5C histone demethylase\n",
      "activity is required for KDM5C-mediated YY1 chromatin binding.\n",
      "Transcriptional profiling reveals that dual inhibition of KDM5C and YY1\n",
      "increases transcriptional repression of cell cycle- and\n",
      "apoptosis-related genes. In summary, our work demonstrates a synthetic\n",
      "lethal interaction between YY1 and KDM5C and suggests combination\n",
      "therapies for cancer treatments.\n",
      "\n",
      "**Key word:** KDM5C; YY1; chromatin recruitment; promoter; cancer\n",
      "therapy**\n",
      "**\n",
      "\n",
      "**Introduction**\n",
      "\n",
      "Epigenetic regulators change phenotype acquisition or gene expression\n",
      "without altering the DNA sequence. By conferring a fitness advantage,\n",
      "mutational or non-mutational epigenetic reprogramming has been\n",
      "implicated in the evolution of tumors during malignant progression and\n",
      "metastasis. In addition, numerous studies have shown that aberrant\n",
      "epigenetic alterations contribute to cancer therapeutic resistance(Hogg\n",
      "*et al*, 2020; Miranda Furtado *et al*, 2019; Zafar *et al*, 2021).\n",
      "Therefore, targeting epigenetic regulators is an attractive strategy for\n",
      "cancer therapy.\n",
      "\n",
      "The histone demethylase KDM5C is a member of the SMC homolog family. By\n",
      "specifically removing one or two methyl groups from Lys4 in histone H3,\n",
      "the dynamic balance in histone H3K4 methylation is maintained. KDM5C\n",
      "carries a composite catalytic region in which JmjN and JmjC domains are\n",
      "located. JmjC is critical for catalyzing the demethylation of H3K4me2/3,\n",
      "while the JmjN domain interacts with the JmjC domain for the complete\n",
      "catalytic function of KDM5C. H514 is a highly conserved amino acid that\n",
      "is located in the catalytic core of the JmjC domain, and its mutation\n",
      "eliminates KDM5C demethylase activity. KDM5C can suppress transcription\n",
      "by reducing the level of H3K4me2/3 via its demethylase activity;\n",
      "however, whether modulation of H3K4 methylation status is the only\n",
      "mechanism or the predominant mechanism by which KDM5C regulates gene\n",
      "expression is unclear.\n",
      "\n",
      "KDM5C is frequently mutated or aberrantly expressed in various cancer\n",
      "types. Several studies have indicated that KDM5C is involved in\n",
      "regulating different biological processes in tumor cells through its\n",
      "histone demethylase activity and functions as a tumor suppressor. For\n",
      "example, the high-risk HPV E6 protein degrades KDM5C in cervical cancer\n",
      "cells, which results in super enhancers activation of the\n",
      "proto-oncogenes c-MET and EGFR by changing the H3K4me3/H3K4me1\n",
      "ratio(Chen *et al*, 2018). In clear cell renal cell carcinoma cells,\n",
      "KDM5C inhibits the expression of genes related to glycogen metabolism by\n",
      "reducing H3K4me3 levels at the promoters of these genes, while\n",
      "inactivating KDM5C mutation confers cell resistance to ferroptosis(Zheng\n",
      "*et al*, 2021). However, the therapeutic strategy for KDM5C-deficient\n",
      "tumors is unknown. Intriguingly, KDM5C has also been shown to promote\n",
      "tumorigenesis, although the exact mechanism is unclear; in general,\n",
      "these observations suggest that KDM5C potentially regulates gene\n",
      "expression via a distinct mechanism.\n",
      "\n",
      "To better understand the mechanisms underlying how KDM5C regulates gene\n",
      "expression, we screened and thus identified Yin Yang 1 (YY1) as a\n",
      "KDM5C-interacting partner. Targeting YY1 in KDM5C-deficient tumor cells\n",
      "effectively suppressed cell proliferation and tumorigenesis. Chromatin\n",
      "Immunoprecipitation followed by next generation sequencing (ChIP-seq)\n",
      "analysis with multiple engineered cell lines revealed that KDM5C is\n",
      "essential for YY1 chromatin recruitment, which is partially dependent on\n",
      "an intact JmjC domain in KDM5C. Simultaneous loss of KDM5C and YY1 led\n",
      "to significantly enhanced transcriptional inhibition of a number of\n",
      "essential cell fate genes, elevated the apoptosis rate and led to cell\n",
      "cycle arrest. Our study suggests a novel scaffolding function of\n",
      "epigenetic regulators and provides a rationale for the development of\n",
      "new combination treatments.\n",
      "\n",
      "**\n",
      "**\n",
      "\n",
      "**Results**\n",
      "\n",
      "**YY1 is identified as a partner of KDM5C**\n",
      "\n",
      "Previous studies have shown that KDM5C plays diverse roles in cancer\n",
      "progression. Indeed, a systemic The Cancer Genome Atlas (TCGA) clinical\n",
      "data-based analysis clearly revealed that high KDM5C expression was\n",
      "positively or negatively correlated with cancer patient survival,\n",
      "depending on the cancer type (Figure **1A**), reinforcing the idea that\n",
      "KDM5C may exhibit a distinct regulatory effect on gene expression,\n",
      "prompting us to screen the interacting partners of KDM5C. Tandem\n",
      "affinity purification (TAP) was subsequently performed. We generated a\n",
      "HEK293T cell line that stably expressed SBP- and Flag-tagged KDM5C, and\n",
      "the lysates of these cells were incubated with streptavidin-coated\n",
      "beads. The bead-bound proteins were subsequently eluted with buffer\n",
      "containing biotin. Next, the captured proteins were incubated with\n",
      "anti-Flag beads, and the proteins bound to these beads were eluted and\n",
      "analyzed by mass spectrometry (MS). The results of the MS analysis\n",
      "indicated that a number of proteins potentially interact with KDM5C,\n",
      "including YY1 (Figure **1B**), the overexpression of which has been\n",
      "observed in the vast majority of cancers(Gao *et al*, 2019; van Leenders\n",
      "*et al*, 2007). Given the pivotal role played by YY1 in the\n",
      "transcriptional regulation of numerous genes(Meliala *et al*, 2020), we\n",
      "focused on the interaction between KDM5C and YY1. To confirm their\n",
      "interaction, mutual coimmunoprecipitation (co-IP) assays were performed\n",
      "in HEK293T cells, and the results clearly showed that KDM5C bound to YY1\n",
      "(Figure **1C**). Co-IP with endogenous proteins was also performed and\n",
      "the results confirmed that KDM5C interacts with YY1 in mammalian cells\n",
      "(Figure **1D**). In addition, we also performed the semi-endogenous\n",
      "co-IP with YY1 and KDM5C-Flag in 769-P renal cancer cell lines. As shown\n",
      "in Figure **1E**, KDM5C clearly bound to YY1. To delineate the\n",
      "interaction, a series of KDM5C truncation mutants were constructed and\n",
      "used for further analysis (Figure **1F**). We found that multiple\n",
      "regions, including the JmjN and JmjC domains, which are critical for\n",
      "H3K4me2/3 removal, were also involved in the interaction between KDM5C\n",
      "and YY1 (Figure **1G**).\n",
      "\n",
      "**YY1 depletion significantly inhibits the proliferation and\n",
      "tumorigenicity of KDM5C-deficient tumor**\n",
      "\n",
      "Many studies have shown that YY1 functions as an oncogene because its\n",
      "overexpression is observed in multiple cancers(Li *et al*, 2022; Qiao\n",
      "*et al*, 2019; Wang *et al*, 2022). YY1 tethers cofactors and promotes\n",
      "or represses transcription, depending on its interacting partner, which\n",
      "includes c-myc and p53(Shrivastava *et al*, 1993; Sui *et al*, 2004),\n",
      "prompting us to investigate the role of the YY1 KDM5C interaction in\n",
      "tumorigenesis. To achieve this goal, we first selected human kidney\n",
      "proximal tubular epithelial cell line (HK-2), and several renal cancer\n",
      "cell lines (ACHN, 769-P and RCC4) with different KDM5C and YY1 protein\n",
      "expression levels, as described in our previous studies(Chen *et al.*,\n",
      "2018; Zheng *et al.*, 2021) (Figure **2A**). Interestingly, knocking\n",
      "down YY1 in KDM5C-deficient RCC4 tumor cells significantly reduced the\n",
      "colony formation ability of the cells, which was not observed in the\n",
      "KDM5C-proficient tumor cell lines ACHN and 769-P (Figure **2B** and\n",
      "**EV1B-D**). Similar results were also obtained in a CCK8-based tumor\n",
      "cell proliferation assay (Figure **2C** and **EV1E-F**). Notably, tumor\n",
      "growth was markedly inhibited when both of KDM5C and YY1 were depleted,\n",
      "whereas the loss of either KDM5C or YY1 only negligibly influenced ACHN\n",
      "cell growth (Figure **2D**). These results strongly indicate that KDM5C\n",
      "might be important for YY1-related tumorigenesis, suggesting that YY1\n",
      "may be a vulnerable target in KDM5C-deficient tumor cells.\n",
      "\n",
      "**KDM5C colocalizes with YY1 and contributes to YY1** **genome-wide\n",
      "chromatin recruitment**\n",
      "\n",
      "The stimulatory effect of YY1 on tumor cell growth was at least\n",
      "partially dependent on the presence of KDM5C. Given that YY1 always\n",
      "promotes tumorigenesis by increasing oncogene transcription, it is\n",
      "unlikely that YY1 tethers KDM5C because KDM5C has H3K4me2/3 demethylase\n",
      "activity, which is associated with repressed gene expression. A recent\n",
      "study revealed that KDM5B, which is a member of the KDM5 demethylase\n",
      "family, recruited a methyltransferase to suppress antitumor immunity,\n",
      "prompting us to explore whether KDM5C has a scaffolding function similar\n",
      "to that of KDM5B(Zhang *et al*, 2021). We generated a stable\n",
      "KDM5C-depleted ACHN cell lines (Figure **EV1B**), and ChIP-seq assays\n",
      "were subsequently performed. As shown in Figure **3A**, the intensity of\n",
      "the YY1 binding peak was reduced upon loss of KDM5C, implying that KDM5C\n",
      "might be a general regulator of YY1 activity by mediating its chromatin\n",
      "recruitment. Next, we asked to what extent KDM5C co-localizes with YY1.\n",
      "To evaluate the genomic co-occupancy of KDM5C and YY1, we performed the\n",
      "cleavage under targets and tagmentation (CUT&Tag) assay in the ACHN\n",
      "renal cell line, and the annotation of the YY1 and KDM5C peaks revealed\n",
      "an obvious overlap (n=3645) between YY1 and KDM5C binding events (Figure\n",
      "**3B**). Further analyzed revealed a significant overlap, with 7,957\n",
      "genes being concurrently bound by both YY1 and KDM5C (Hypergeometric\n",
      "test, P=4.87e-17). This overlap accounts for 56.32% of the total genes\n",
      "bound by KDM5C and 66.71% of the total genes bound by YY1. Moreover, a\n",
      "heatmap and a metaplot analysis revealed that the enrichment of YY1 was\n",
      "reduced after KDM5C was depleted in the peaks (n=3645) where YY1 and\n",
      "KDM5C were directly related (Figure **3C**). Reduced enrichment of YY1\n",
      "was also observed in HK-2 cells with KDM5C depletion (Figure **EV1A**\n",
      "and **EV2A**). These results revealed that the intensity of the YY1\n",
      "binding peaks was partially dependent on the KDM5C protein level in both\n",
      "nontumoral cell and tumor cells, suggesting that KDM5C mediated YY1\n",
      "chromatin recruitment may represent a universal mechanism of action. The\n",
      "localization of the KDM5C-dependent YY1 binding peaks was subsequently\n",
      "analyzed, and the results revealed that most of the peaks were enriched\n",
      "in promoters (Figure **3D**), implying that KDM5C may regulate gene\n",
      "expression via a distinct mechanism.\n",
      "\n",
      "**An intact catalytic domain but not the demethylation activity of KDM5C\n",
      "is required for YY1 recruitment**\n",
      "\n",
      "Since our results suggested that KDM5C functions as a scaffold and that\n",
      "KDM5C is essential for YY1 binding to a promoter, we asked, whether the\n",
      "demethylase activity of KDM5C is involved in YY1 recruitment. We\n",
      "generated ACHN cell lines that stably expressed KDM5C-WT or its\n",
      "enzymatically inactive mutant H514A and performed ChIP-seq to evaluate\n",
      "the effects of KDM5C or its mutants on YY1 recruitment (Figure\n",
      "**EV1B**). As shown in Figure **4A,** ectopic KDM5C WT but not the\n",
      "KDM5C-H514A mutant enhanced YY1 chromatin binding. A similar trend was\n",
      "also observed in HK-2 cells (Figure **EV2A**). Because the primary\n",
      "function of KDM5C is to catalyze the removal of methyl groups from\n",
      "H3K4me2/3, we evaluated whether H3K4me3 abundance plays a role in YY1\n",
      "recruitment.\n",
      "\n",
      "We first performed ChIP-seq of H3K4me3 in ACHN shKDM5C cells and\n",
      "analyzed the change in the signal intensity of this mark. Increased\n",
      "H3K4me3 enrichment was clearly observed in the KDM5C/YY1 overlapping\n",
      "region (Figure **4B**), but the opposite trend of YY1 recruitment was\n",
      "observed in KDM5C-depleted cells (Figure **3C**). To further understand\n",
      "the relationship between YY1 recruitment and H3K4me3, we focused on the\n",
      "overlapping regions of YY1 and H3K4me3 (n=11087), which were classified\n",
      "into KDM5C demethylase-dependent peaks (n=9943) and KDM5C\n",
      "demethylase-independent peaks (n=1144), as shown in Figure **4C**.\n",
      "Reduced YY1 ChIP-seq signals were observed in almost all of the\n",
      "overlapping YY1 and H3K4me3 regions (Figure **4D-E**), regardless of the\n",
      "effect of KDM5C on H3K4me3 intensity. We therefore hypothesized that an\n",
      "intact KDM5C JmjC domain, rather than KDM5C histone demethylase\n",
      "activity, is essential for YY1 recruitment.\n",
      "\n",
      "To further explore the mechanism underlying the effects of KDM5C, we\n",
      "evaluated the interaction between KDM5C-H514A and YY1 and found that the\n",
      "mutation did not influence their interaction (Figure **EV3**),\n",
      "indicating that the effect of KDM5C-H514A on YY1 chromatin recruitment\n",
      "was unlikely to be caused by disrupted protein binding activity. Because\n",
      "the catalytic composite JmjN/C domain binds the substrate H3 tail\n",
      "harboring methylated H3K4 and since a previous study revealed that none\n",
      "of the KDM5C PHD domains exhibited strong binding affinity for\n",
      "unmethylated or methylated histones(Iwase *et al*, 2007), we speculated\n",
      "that the replacement of the histidine residue at position 514 with an\n",
      "alanine residue destroyed the integrity of the JmjN/C domain and\n",
      "influenced the chromatin-binding activity of KDM5C. To test this\n",
      "hypothesis, we expressed KDM5C-WT or KDM5C-H514A in HEK293T cells and\n",
      "isolated chromatin-bound proteins. Interestingly, although the protein\n",
      "levels in whole-cell lysates were indistinguishable, H154A exhibited\n",
      "obviously reduced chromatin-binding capacity (Figure **4F**), which\n",
      "might help explain why KDM5C-H514A failed to recruit YY1 to chromatin.\n",
      "In addition, tumor cell proliferation was markedly inhibited only when\n",
      "both of KDM5C and YY1 were depleted in the KDM5C-proficient tumor cells\n",
      "ACHN and 769-P (Figure **4G-H** and **EV1B-C**). The inhibitory effect\n",
      "was blunted when KDM5C-WT was re-expressed; in contrast, KDM5C-H514A\n",
      "only moderately restored the growth of ACHN and 769-P double knockdown\n",
      "cells. A similar trend was also observed in the colony formation assay\n",
      "(Figure **4J**). In KDM5C-deficient RCC4, cells overexpression of\n",
      "KDM5C-WT largely restored cell sensitivity to YY1 depletion (Figure\n",
      "**4I** and **EV1D**). These result collectively suggest that an intact\n",
      "catalytic domain of KDM5C is required for YY1 recruitment.\n",
      "\n",
      "**KDM5C cooperates with YY1 to regulate essential cell fate gene\n",
      "expression**\n",
      "\n",
      "To systematically evaluate the effect of KDM5C-mediated YY1 recruitment\n",
      "on gene expression, we performed RNA-seq with shKDM5C, shYY1 or\n",
      "double-knockdown ACHN cells. As shown in Figure **5A-B**, the loss of\n",
      "KDM5C or YY1 led to the upregulated or downregulated expression of\n",
      "several genes. In contrast, the simultaneous loss of KDM5C and YY1\n",
      "resulted in altered transcription of thousands of genes (1511 genes\n",
      "whose expression was downregulated versus 797 genes whose expression was\n",
      "upregulated, \\|log<sub>2</sub>FC\\|≥1.5, p value&lt;0.05, Figure **5C**),\n",
      "indicating that YY1 recruitment by KDM5C likely contributed to gene\n",
      "expression activation. A similar pattern was also observed via RNA-seq\n",
      "with HK-2 nontumoral cells (Figure **EV4**). Compared with the loss of\n",
      "KDM5C or YY1 alone, double knockdown caused transcriptional repression\n",
      "of a sharply increased number of genes (1099 newly identified\n",
      "downregulated genes in total), as shown in Figure **5D**. Given that\n",
      "depletion of both YY1 and KDM5C led to significant suppression of tumor\n",
      "cell proliferation (Figure **4**), we next investigated the gene\n",
      "expression patterns by analyzing the RNA-seq results and found that the\n",
      "expression of several cell cycle- and cell apoptosis-related genes,\n",
      "including JAM3, KNTC1, SMC2, SMC4, LMNB1 and FOXM1, was markedly\n",
      "downregulated in the double-knockdown cell group (Figure **5E**). These\n",
      "results were verified by qPCR in ACHN cells (Figure **5F)**.\n",
      "\n",
      "We compared the RNA-seq results with our CUT&Tag data, and found that\n",
      "almost half of the downregulated genes (49.96%) in the KDM5C/YY1\n",
      "double-knockdown group were annotated in the genomic regions where YY1\n",
      "and KDM5C overlapped, as shown in Figure **EV5B**. We then analyzed the\n",
      "YY1 peaks at the promoters of individual genes via the IGV browser. As\n",
      "shown in Figure **EV5C-D**, KDM5C depletion reduced theYY1 peak\n",
      "intensity at the promoters of the indicated genes, and the presence of\n",
      "KDM5C increased the amount of YY1 recruited to these promoters.\n",
      "KDM5C-H514A marginally increased the YY1 intensity in the indicated\n",
      "regions. Moreover, ChIP-qPCR was performed in HK-2 cells and the result\n",
      "revealed that YY1 enrichment at the promoter region of the indicated\n",
      "genes was increased after KDM5C-WT was expressed, whereas KDM5C-H514A\n",
      "failed to increase YY1 binding (Figure **EV2C**). These results strongly\n",
      "suggest that KDM5C increases the expression of essential cell-fate genes\n",
      "by recruiting YY1 to their promoters.\n",
      "\n",
      "**Simultaneous loss of KDM5C and YY1 significantly promotes cell cycle\n",
      "arrest and cell apoptosis**\n",
      "\n",
      "Previous studies have shown that *SMC2/4, LMNB1,* and *KNTC1* are\n",
      "involved in cell cycle arrest and apoptosis regulation. To confirm the\n",
      "synergistic effect of KDM5C and YY1 on tumorigenicity and cancer cell\n",
      "proliferation, we analyzed the cell cycle progression and apoptosis rate\n",
      "of tumor cells in which KDM5C, YY1 or both had been depleted. We first\n",
      "measured the ratio of renal cancer cells in each phase of the cell\n",
      "cycle, and the results indicated that the loss of both KDM5C and YY1\n",
      "caused an increased ratio of 769-P tumor cells arrested in the G2/M\n",
      "phase. Similar results were also observed in KDM5C-deficient RCC4 cells\n",
      "after YY1 depletion (Figure **6A**). Moreover, double knockdown of KDM5C\n",
      "and YY1 but not loss of either gene individually greatly increased the\n",
      "percentage of apoptotic cells in 769-P tumors. In addition, the absence\n",
      "of YY1 in RCC4 cells induced a significant increase in the percentage of\n",
      "apoptosis cells (Figure **6B**). These results confirmed the inhibitory\n",
      "effect of YY1 depletion on KDM5C-deficient cells (Figure **4**). Taken\n",
      "together, our study revealed a synergistic regulatory effect of KDM5C\n",
      "and YY1 on cancer cell proliferation and cell cycle progression.\n",
      "\n",
      "**\n",
      "**\n",
      "\n",
      "**Discussion**\n",
      "\n",
      "As a member of the GLI-Kruppel family of zinc finger DNA-binding\n",
      "proteins, YY1 has been proven to regulate genes either by directly\n",
      "binding to a promoter or indirectly by associating with\n",
      "chromatin-remodeling proteins or histone modifiers (Gordon *et al*,\n",
      "2006; Khachigian, 2018). YY1 is frequently overexpressed in many cancer\n",
      "types and reshapes the oncogenic gene expression network(Bonavida &\n",
      "Kaufhold, 2015; Sarvagalla *et al*, 2019). Although YY1 can directly\n",
      "bind to a promoter DNA sequence, the mechanism underlying its\n",
      "recruitment to chromatin is still not fully understood. In addition,\n",
      "although targeting YY1 might be an attractive therapeutic strategy, a\n",
      "robust biomarker to analyze its effect is lacking. In this study, we\n",
      "found that YY1 is a KDM5C-associated protein, and the JmjC domain of\n",
      "KDM5C was essential for the KDM5C–YY1 interaction. Moreover, we\n",
      "demonstrated that KDM5C mediates the chromatin recruitment of YY1, which\n",
      "is important for YY1 binding to promoter regions and the transcription\n",
      "of cell fate-related genes. Importantly, targeting YY1 effectively\n",
      "inhibited the proliferation and tumorigenicity of tumor cells with low\n",
      "KDM5C expression. Considering these findings, we propose the following\n",
      "model: YY1 chromatin recruitment is at least partially dependent on the\n",
      "presence of KDM5C, and in tumor cells with high KDM5C expression, a\n",
      "small amount of YY1 can be recruited to chromatin even after KDM5C is\n",
      "knocked down. In contrast, depleting YY1 in low KDM5C-expressing tumor\n",
      "cells completely eliminated chromatin-bound YY1 and promoted tumor\n",
      "regression (Figure **7**).\n",
      "\n",
      "As a specific H3K4 histone demethylase, KDM5C plays dual roles as both a\n",
      "proto-oncogene and tumor suppressor(Chen *et al.*, 2018; Ji *et al*,\n",
      "2015; Shen *et al*, 2016; Shen *et al*, 2021). KDM5C interacts with\n",
      "specific transcription factors, such as MYC(Outchkourov *et al*, 2013;\n",
      "Secombe *et al*, 2007), and downregulates gene expression by reducing\n",
      "H3K4me2/3 levels at enhancer or promoter regions. However, studies,\n",
      "including our previous reports, have shown that KDM5C also upregulates\n",
      "the expression of certain genes, but the relevant molecular mechanism of\n",
      "KDM5C effect on gene expression is unclear(Outchkourov *et al.*, 2013;\n",
      "Scandaglia *et al*, 2017; Zheng *et al.*, 2021). A recent study showed\n",
      "that the histone demethylase KDM5B recruited the H3K9 methyltransferase\n",
      "SETDB1 to suppress antitumor immunity in a KDM5B demethylase-independent\n",
      "manner, implying that the scaffolding function of this epigenetic\n",
      "regulator was important in certain contexts, a finding that has been\n",
      "appreciated to a lesser degree in prior studies(Zhang *et al.*, 2021).\n",
      "Our finding that KDM5C recruited YY1 to chromatin in a histone\n",
      "demethylase-independent manner reinforces the evidence suggesting that\n",
      "the scaffolding function of epigenetic regulators might be critical for\n",
      "tumorigenesis. Indeed, epigenetic inhibitors targeting the catalytic\n",
      "domain often show limited efficacy and intolerable toxicity, and our\n",
      "findings strongly suggest that targeting the scaffolding function should\n",
      "be considered in the development of new cancer prevention strategies and\n",
      "therapies.\n",
      "\n",
      "Abnormalities in enzymes that regulate epigenetic modifications are\n",
      "important drivers of tumor occurrence and cancer progression(Dawson,\n",
      "2017). Therefore, epigenome-based therapies might be developed to block\n",
      "the dysregulated expression of proto-oncogenes in tumor cells and\n",
      "increase antitumor responses(Miranda Furtado *et al.*, 2019; Nebbioso\n",
      "*et al*, 2018). KDM5C has been shown to be frequently mutated or\n",
      "aberrantly expressed in various cancers. The degree of YY1 chromatin\n",
      "recruitment was decreased in KDM5C-deficient tumor cells, and further\n",
      "inhibition of YY1 expression might completely suppress YY1-directed\n",
      "transcription and significantly impair the proliferation of tumor cells.\n",
      "Furthermore, simultaneously targeting KDM5C and YY1 effectively\n",
      "inhibited tumor growth in KDM5C-proficient tumor cells. In recent years,\n",
      "the induction of synthetic lethality has been applied to the\n",
      "identification of novel drug targets, opening new avenues to\n",
      "tumor-targeted therapy(Nijman & Friend, 2013). Considering the\n",
      "definition of synthetic lethality and the results presented here, we\n",
      "propose that KDM5C and YY1 form a synthetically lethal gene pair,\n",
      "providing a rationale for the clinical treatment of patients with\n",
      "KDM5C-deficient cancer by targeting YY1.\n",
      "\n",
      "In summary, our work provides mechanistic insight into YY1 chromatin\n",
      "recruitment and suggests an important role for the scaffolding function\n",
      "of the histone demethylase KDM5C. Because proteolysis-targeting chimeras\n",
      "(PROTACs) are used to target and degrade proteins with important\n",
      "scaffolding functions and serve as a potential therapeutic strategy in\n",
      "cancer, future work focusing on the design and synthesis of KDM5C PROTAC\n",
      "drugs is important, and further studies in this research direction are\n",
      "warranted.\n",
      "\n",
      "**Methods**\n",
      "\n",
      "**Vectors and plasmids**\n",
      "\n",
      "The pHAGE vector carries a puromycin resistance cassette, endowing the\n",
      "transfected cells with resistance to puromycin, hence enabling us to\n",
      "select and generate stable cells. The overexpression vectors used in our\n",
      "context (KDM5C or YY1 with HA or Flag tags and their truncated\n",
      "constructs) were constructed using pHAGE for mammalian cell expression.\n",
      "The KDM5C enzymatically inactive mutant H514A was generated via\n",
      "site-directed mutagenesis PCR from the pHAGE-KDM5C plasmid. shRNAs\n",
      "targeting human KDM5C and YY1 were constructed using pLKO.1. The\n",
      "construction of pLKO.1 is detailed on the *addgene* website. All of the\n",
      "constructs were validated by DNA sequencing. The targeted sequences of\n",
      "KDM5C and YY1 genes are as follows:\n",
      "\n",
      "shCtrl: 5’-CAACAAGATGAAGAGCACCAA-3’;\n",
      "\n",
      "shKDM5C-1: 5’-AGTACCTGCGGTATCGGTATA-3’;\n",
      "\n",
      "shKDM5C-2: 5’-GCCACACTTGAG GCCATAATC-3’;\n",
      "\n",
      "shYY1-1: 5’-GACGACGACTACATTGAACAA-3’;\n",
      "\n",
      "shYY1-2: 5’-GCCTCTCCTTTGTATATTATT-3’.\n",
      "\n",
      "**Cell culture**\n",
      "\n",
      "The human kidney proximal tubular epithelial cell line HK-2 and human\n",
      "renal cancer cell lines (ACHN, 769-P and RCC4) were obtained from the\n",
      "American Type Culture Collection (ATCC). HEK293T cells were obtained\n",
      "from the Institute of Cellular Resources, Shanghai Institutes for\n",
      "Biological Sciences, Chinese Academy of Sciences. HK-2, ACHN, RCC4, and\n",
      "HEK293T cells were cultured in high-glucose Dulbecco’s modified Eagle’s\n",
      "medium (DMEM, HyClone) while 769-P cells were cultured in Roswell Park\n",
      "Memorial Institute-RPMI medium(1640, HyClone) and both media were\n",
      "supplemented with 10% fetal bovine serum (FBS, Cell-box, AUS-01S-02) in\n",
      "a humidified atmosphere containing 5% CO2 at 37 ℃. The cell lines were\n",
      "authenticated by STR profiling, and mycoplasma testing was performed\n",
      "monthly with a MycoBlue Mycoplasma Detector (Vazyme, D101-01).\n",
      "\n",
      "For transient protein expression, the cells were transfected with the\n",
      "indicated constructs via NEOFECT DNA transfection reagent (NeoFect,\n",
      "TF20121201) according to the manufacturer’s instructions. After 48 h of\n",
      "transfection, the cells were harvested for the following assays. To\n",
      "generate stable overexpression or knockout cell lines, appropriate\n",
      "constructs and two packaging plasmids psPAX2 and pMD2.G were\n",
      "cotransfected into HEK293T cells. After 60 h, the lentivirus particles\n",
      "from the culture medium were collected and filtered with 0.45 µm filter,\n",
      "and then added to the host cells along with 8 μg/mL polybrene (Solarbio,\n",
      "H8761). Infected cells were then selected in puromycin (Solarbio,\n",
      "P8230)-containing medium. The protein levels of the target genes were\n",
      "determined via western blotting with the corresponding antibodies. To\n",
      "generate the double-knockdown cells, we selected a GFP-labeled KDM5C\n",
      "knockdown plasmid (pLKO.1-GFP) to infect cells depleted of YY1, and then\n",
      "screened the cells for puromycin resistance and green fluorescence. To\n",
      "restore KDM5C or H514A in knockdown cells, we inserted a hygromycin\n",
      "resistance gene into pHAGE-KDM5C/H514A-puro, which could be used to\n",
      "screen cells expressing KDM5C/H514A. The cells involved in the cell\n",
      "cycle and apoptosis assays did not contain fluorescence, which might\n",
      "have affected the results. We confirmed protein expression via Western\n",
      "blotting.\n",
      "\n",
      "**RNA extraction and qRT-PCR**\n",
      "\n",
      "Total RNA was extracted using the RNA isolater Total RNA Extraction\n",
      "Reagent (Vazyme, R401-01) according to the manufacturer’s instructions.\n",
      "Reverse transcription reactions were performed by using ReverTra Ace\n",
      "qPCR RT Master Mix with gDNA Remover (TOYOBO, FSQ-301). RT-PCR was\n",
      "performed with ChamQ SYBR qPCR Master Mix (Vazyme, Q311-02) on a CFX\n",
      "Connect<sup>TM</sup> Real-Time System (Bio-Rad). The housekeeping gene\n",
      "expression levels of β-actin or GAPDH were used for normalization. Each\n",
      "reaction was performed in triplicates. For quantification of gene\n",
      "expression, the 2<sup>-△△</sup>Ct method was used. The primer sequences\n",
      "of indicated genes were as follows:\n",
      "\n",
      "h-JAM3-RT\n",
      "\n",
      "Forward : 5’-GAGACTCAGCCCTTTATCGC-3’;\n",
      "\n",
      "Reverse : 5’- TGTTGCCATCTTGCCTACTG-3’;\n",
      "\n",
      "h-KNTC1-RT\n",
      "\n",
      "Forward : 5’-CTCACGAAGTTGCACAGG-3’;\n",
      "\n",
      "Reverse : 5’-GCCTCCAGCTCTTGTTTG-3’;\n",
      "\n",
      "h-SMC2-RT\n",
      "\n",
      "Forward : 5’-GTTTTGGATGGTCTGGAGTT-3’;\n",
      "\n",
      "Reverse : 5’-ATTGGAGCAGGTTTGAAGAG-3’;\n",
      "\n",
      "h-SMC4-RT\n",
      "\n",
      "Forward : 5’-TCGACCACCTAAGAAAAGTTG-3’;\n",
      "\n",
      "Reverse : 5’-CCATGAAGTAAAGGGGAGTG-3’;\n",
      "\n",
      "h-LMNB1-RT\n",
      "\n",
      "Forward : 5’-CTGTCTCCAAGCCCTTCTT-3’;\n",
      "\n",
      "Reverse : 5’-ACTCGCCTCTGATTCTTCC-3’;\n",
      "\n",
      "h-FOXM1-RT\n",
      "\n",
      "Forward : 5’-GCTGCCCCTTCCTGTTC-3’;\n",
      "\n",
      "Reverse : 5’-TGGACTCTGCCACTTCCTT-3’;\n",
      "\n",
      "h-GAPDH-RT\n",
      "\n",
      "Forward : 5’-ACATCAAGAAGGTGGTGAAG-3’;\n",
      "\n",
      "Reverse : 5’-CTGTTGCTGTAGCCAAATTC-3’;\n",
      "\n",
      "**Western blotting**\n",
      "\n",
      "The cell pellets were lysed with RIPA buffer (Beyotime, P0013B)\n",
      "containing a protease inhibitor mixture cocktail (MedChemExpress,\n",
      "HY-K0010). Then, the cells were centrifuged at 4 ℃ at 13,000 rpm for 10\n",
      "min to remove insoluble cellular components. Total protein\n",
      "concentrations were measured with a BCA protein assay kit (Beyotime,\n",
      "P0012). Proteins (20–100 μg) were separated via 8-15% sodium dodecyl\n",
      "sulphate-polyacrylamide gel electrophoresis (SDS-PAGE) and transferred\n",
      "to nitrocellulose membranes (NC, PALL, 66485) in Tris-glycine buffer.\n",
      "The blots were blocked with 5% nonfat milk then incubated with diluted\n",
      "primary antibodies at 4 ℃ overnight. Then, appropriate horseradish\n",
      "peroxidase (HRP)-conjugated secondary antibodies were incubated with the\n",
      "blots for 1 h at room temperature. The signals were visualized with an\n",
      "enhanced chemiluminescence detection kit (ECL, Vazyme, E411-05) and\n",
      "exposured to X-ray film (Sigma, America). The primary antibodies used\n",
      "were purchased from commercial sources: anti-KDM5C (Cell Signaling\n",
      "Technology, D29B9; BETHYL, A301-034A; Proteintech, 14426-1-AP), anti-YY1\n",
      "(Cell Signaling Technology, D5D9Z), anti-Flag (Proteintech, 20543-1-AP),\n",
      "anti-HA (Proteintech, 66006-2-Ig), anti-GAPDH (Proteintech, 60004-1-Ig),\n",
      "anti-β-actin (Proteintech, 66009-1-Ig), anti-H3 (Cell Signaling\n",
      "Technology, 2650) and anti-H3K4me3 (Cell Signaling Technology, 9751).\n",
      "\n",
      "**Coimmunoprecipitation (co-IP)**\n",
      "\n",
      "For the immunoprecipitation assay, appropriately constructed plasmids\n",
      "were transfected into the HEK293T and 769-P cell lines. After 48 h of\n",
      "transfection, the cells were harvested and then lysed in IP lysis buffer\n",
      "(Beyotime, P0013) with EDTA-free protease inhibitor cocktail\n",
      "(MedChemExpress, HY-K0010) for 30 min on ice, sonicated, and then\n",
      "insoluble proteins were removed by centrifugation at 4 ℃ at 12000 rpm\n",
      "for 10 min. An aliquot of the cleared cell lysate was collected as an\n",
      "input control. The remaining lysates were subsequently incubated with\n",
      "anti-HA or anti-Flag magnetic beads (MedChemExpress, HY-K0201, HY-K0207)\n",
      "overnight at 4 ℃ with gentle shaking.\n",
      "\n",
      "For endogenous protein interactions, the cell lysates were processed as\n",
      "described above. One tenth of each lysates was used as the input, half\n",
      "of the remainder was incubated with KDM5C antibody, and the other half\n",
      "was incubated with homologous IgG for 6 hours at 4 ℃. The lysates were\n",
      "then mixed with ProteinA/G agarose and incubated overnight at 4 ℃.\n",
      "\n",
      "Magnetic beads or agarose beads with a complex of antigen and antibody\n",
      "were washed four times in TBS-T buffer (100 nM Tris-HCl, 0.9% NaCl, 0.1%\n",
      "Tween-20). The proteins pulled down by the specified antibodies were\n",
      "eluted with 2×SDS loading buffer through boiling the beads, and then\n",
      "verified by Western blotting.\n",
      "\n",
      "**Tandem affinity purofication and mass spectrometry (TAP-MS)**\n",
      "\n",
      "HEK293T cells overexpressing SBP-Flag-KDM5C were harvested for Tap-MS.\n",
      "Cell pellets were lysed with IP lysis buffer (Beyotime, P0013), and the\n",
      "supernatant was subsequently incubated with strep beads overnight at 4\n",
      "℃. After three washes, SBP-linked protein were elutied with biotin\n",
      "solution, and then incubated with anti-Flag magnetic beads\n",
      "(MedChemExpress, HY-K0207). After enrichment of the target proteins, we\n",
      "washed the beads to remove nonspecific proteins and then performed\n",
      "SDS-PAGE on the products of the two groups. All of the lanes of the gel\n",
      "were collected for mass spectrometry analysis.\n",
      "\n",
      "**Proliferation ability assay（CCK8）**\n",
      "\n",
      "The cells were seeded in a 96-well plate at 30-50% confluence according\n",
      "to the cell growth rate. When the cells just attached at the initial\n",
      "point, the OD450 nm was read every 24 h via a cell counting kit 8 (CCK8,\n",
      "MedChemExpress, HY-K0301) according to the manufacturer’s instruction.\n",
      "The CCK8 solution was diluted with complete medium, incubated with the\n",
      "cells for 1-2 h, and then read with a microplate reader (BioTek Epoch,\n",
      "America) at 450 nm. The cell viability assay was performed in triplicate\n",
      "and repeated three times. Cell growth curves were drawn and analyzed via\n",
      "GraphPad Prism software V9.4.1.\n",
      "\n",
      "**Colony formation assay**\n",
      "\n",
      "The cells were seeded into a 6-well plate at the same cell density. Each\n",
      "well contained 1000 cells, which were cultivated in a cell incubator.\n",
      "Fresh cell culture medium was added every three days. After four weeks\n",
      "(ACHN and 769-P) or eight weeks (RCC4), the cell culture medium was\n",
      "removed. The cells were gently washed with PBS, fixed with 4%\n",
      "paraformaldehyde for 20 min at 37 ℃, and stained with 0.1% crystal\n",
      "violet in PBS. The assay was performed in triplicate and repeated two or\n",
      "three times.\n",
      "\n",
      "**Cell cycle assay**\n",
      "\n",
      "The cells were gently collected at the exponential growth phase, washed\n",
      "with cold PBS, and then resuspended in 70% ethanol for more than 30 min\n",
      "at 4 ℃. Then, the cells were washed with cold PBS, incubated with\n",
      "propidium iodide(PI) solution (Byotime, C1052) and protected from light\n",
      "at 37 ℃ for 30 min. Red fluorescence was detected via flow cytometry\n",
      "(LSRFortessaX20, BD) at Ex=488 nm, and then analyzed with Flowjo. The\n",
      "assay was repeated two or three times.\n",
      "\n",
      "**Cell apoptosis assay**\n",
      "\n",
      "In accordance with the instructions of the Annexin V-FITC/PI apoptosis\n",
      "kit (Multi Sciences, AP101), we collected 1-10×10^<sup>5</sup> cells and\n",
      "resuspended them in 500μl of 1×Binding buffer, and add 5 μl of Annexin\n",
      "V-FITC and 10 μl of PI to each sample. The samples were mixed gently and\n",
      "protected from light for 5 min, then Annexin V-FITC (Ex = 488 nm; Em =\n",
      "530 nm) and PI (Ex = 535 nm; Em = 615 nm) were detected via flow\n",
      "cytometry (LSRFortessaX20, BD) with. The results were analyzed via\n",
      "Flowjo.\n",
      "\n",
      "**Isolation of chromatin-bound proteins from subcellular fractions**\n",
      "\n",
      "After the transfection of HEK293T cells with empty vector, Flag-KDM5C or\n",
      "Flag-H514A, we collected the cells and lysed them with E1 buffer (50 nM\n",
      "HEPES-KOH, 140 nM NaCl, 1mM EDTA, 10% glycerol, 0.5% NP-40, 0.25% Triton\n",
      "X-100, 1mM DTT, Protease inhibitor cocktail). The cells were\n",
      "subsequently centrifuged 1,100× g at 4 ℃ for 2 min after which the\n",
      "supernatant was obtained as the cytoplasmic fraction. Next, we\n",
      "resuspended the pellet in E2 buffer (10nM Tris-HCl, 200mM NaCl, 1mM\n",
      "EDTA, 0.5mM EGTA, Protease inhibitor cocktail). After centrifugation,\n",
      "the supernatant was obtained as the nuclear fraction. We resuspended the\n",
      "pellet in E3 buffer (500 mM Tris-HCl, 500mM NaCl, protease inhibitor\n",
      "cocktail) and sonicated it on ice for 5 min, 2 s ON/45 s OFF. The\n",
      "supernatant was centrifuged at 4 ℃ for 10 min at 16,000× g, and\n",
      "transferred to fresh microcentrifuge tubes. The protein concentrations\n",
      "were analyzed via Western blotting.\n",
      "\n",
      "**Chromatin immunoprecipitation followed by next-generation sequencing\n",
      "(ChIP-seq)**\n",
      "\n",
      "The cells were plated in 10 cm dishes until they reached confluence and\n",
      "were then fixed with 37% fresh formaldehyde to a final concentration of\n",
      "1% for 13 min at room temperature with rotation, after which the\n",
      "reaction was quenched with 0.125 mM glycine for 10 min. The final number\n",
      "of Chip-seq cells in each group was 4×10<sup>7</sup>.\n",
      "\n",
      "The cells were washed three times with ice-cold PBS and then harvested\n",
      "bwith a cell scraper and lysed in lysis buffer 1 (50mM HEPES-KOH, 140mM\n",
      "NaCl, 1mM EDTA, 10% glycerin, 0.5% NP-40, 0.25% TritonX-100), lysis\n",
      "buffer 2 (pH 8.0 10mM Tris-HCl, 200 mM NaCl 1mM EDTA, 0.5mM EGTA), and\n",
      "lysis buffer 3 (pH 8.0 10mM Tris-HCl, 100mM NaCl, 1mM EDTA, 0.5mM EGTA,\n",
      "0.2% N-lauroyl sarcosine sodium salt). The chromatin was sonicated in\n",
      "lysis buffer 3 to ensure that the size of the chromatin fraction was\n",
      "between 200-500bp. 80 μL of each sample was transferred to a clean tube\n",
      "as the input DNA. The chromatin sample was diluted with ChIP dilution\n",
      "buffer (20mM Tris-HCl, 150mM NaCl, 1mM EDTA, 1% TritonX-100). 6 μL of\n",
      "antibody was added to each tube and the mixturewas incubated at 4 ℃\n",
      "overnight. To each tube, 80μL of protein G Sepharose 4 Fast Flow (GE\n",
      "Healthcare, 17061805) was added, and the immune complexes were collected\n",
      "by incubation at 4 ℃ for at least 6 h with gentle rotation. The bound\n",
      "complexes were washed once with each of the following buffers: low-salt\n",
      "wash buffer (0.1% SDS, 1% TritonX-100, 2 mM EDTA, 20 mM Tris-HCl pH 8.0,\n",
      "150 mM NaCl), high-salt wash buffer (0.1% SDS, 1% Triton X-100, 2 mM\n",
      "EDTA, 20 mM Tris-HCl pH 8.0, 500 mM NaCl), LiCl wash buffer (0.25 M\n",
      "LiCl, 1% NP-40, 1mM EDTA, 10 mM Tris-HCl pH 8.0) and TE buffer (10 mM\n",
      "Tris-HCl pH 8.0, 1mM EDTA). The beads were incubated at 65 ℃ with TE\n",
      "buffer containing proteinase K, RNase and SDS overnight, and the DNA was\n",
      "extracted using phenol/chloroform. ChIP-seq was performed by Beijing\n",
      "Novogene Biotech Co., Ltd.\n",
      "\n",
      "**ChIP-qPCR**\n",
      "\n",
      "The qPCR templates were obtained from ChIP. For method details, please\n",
      "refer to “**qRT-PCR**”. We calculated the fold enrichment of those genes\n",
      "via YY1-ChIP, and IgG was used as a negative control. The calculation\n",
      "formula is as followed:\n",
      "\n",
      "ΔCt \\[normalized ChIP\\] = (Ct \\[ChIP\\] - (Ct \\[Input\\] -Log2 (Input\n",
      "Dilution Factor))\n",
      "\n",
      "ΔΔCt \\[ChIP/NIS\\] = ΔCt \\[normalized ChIP\\] - ΔCt \\[ IgG \\]\n",
      "\n",
      "Fold Enrichment = 2^ (-ΔΔCt \\[ChIP/NIS\\])\n",
      "\n",
      "Primer sequences used in ChIP-qPCR are as followed:\n",
      "\n",
      "JAM3\n",
      "\n",
      "Forward : 5’-TGGTAACTGGGGCGGGTCGCA-3’;\n",
      "\n",
      "Reverse : 5’-AGCTTCGGCCCGCTCTGCTTC-3’;\n",
      "\n",
      "KNTC1\n",
      "\n",
      "Forward : 5’-CAACACCGCCTTACTTCC-3’;\n",
      "\n",
      "Reverse : 5’-CCGGGACTCTGAACTATGGC-3’;\n",
      "\n",
      "SMC2\n",
      "\n",
      "Forward : 5’-GGAGCTACCATTATCGAGAC-3’;\n",
      "\n",
      "Reverse : 5’-CTTGACGCACCCCAAAAG-3’;\n",
      "\n",
      "SMC4\n",
      "\n",
      "Forward : 5’-AAAAGAATCCCTCGCTCTTC-3’;\n",
      "\n",
      "Reverse : 5’-TGACAGCAAGAGAAATCTCC-3’;\n",
      "\n",
      "LMNB1\n",
      "\n",
      "Forward : 5’-TTGCCCAAGGGCCAGATTTTA-3’;\n",
      "\n",
      "Reverse : 5’-AATGCAAGACGAGGGTGACT-3’;\n",
      "\n",
      "FOXM1\n",
      "\n",
      "Forward : 5’-ATTTTCCCACAGTGAACGAT\n",
      "\n",
      "Reverse : 5’-AACCTTGTCTGCCATTGTATC\n",
      "\n",
      "**Cleavage Under Targets and Tagmentation assay**\n",
      "\n",
      "The Cleavage Under Targets and Tagmentation (CUT&Tag) assay was\n",
      "conducted according to the manual instructions of Hyperactive Universal\n",
      "CUT&Tag Assay Kit for Illumina Pro (Vazyme, TD904). A ChIP-grade\n",
      "anti-KDM5C antibody (Bethyl, A301-034A) and anti-YY1 (CST, D5D9Z) were\n",
      "used for immunoprecipitation. Rabbit IgG (ABclonal, AC042) was used as a\n",
      "negative control. The library was constructed with the TruePrep Index\n",
      "Kit V2 for Illumina (Vazyme, TD202).\n",
      "\n",
      "**ChIP-seq and CUT&Tag data analysis**\n",
      "\n",
      "The raw data (raw reads) in fastq format were first processed via fastp\n",
      "software to obtain clean data, from which reads containing adapters were\n",
      "removed. The reference genome hg38 and gene model annotation files were\n",
      "downloaded directly from the genome website. The index of the reference\n",
      "genome was built using Bowtie2, and the clean reads were aligned to the\n",
      "reference genome with Bowtie2. After the reads were mapped to the\n",
      "reference genome, MACS2 peak-calling software was used to identify\n",
      "regions enriched with IP over the background. A q-value threshold of\n",
      "0.05 was used for all of the datasets. Only peaks with counts greater\n",
      "than 200 were retained in the CUT&Tag data. Bedtools reldist and\n",
      "bedtools intersect were used to compare two Chip-seq or CUT&Tag datasets\n",
      "in the global or promoter region. Motif analysis was performed via\n",
      "Homer. Chipseeker was used to analyze genomic distribution.\n",
      "\n",
      "**RNA-seq and data analysis**\n",
      "\n",
      "Total RNA was extracted from the corresponding cells. The concentration\n",
      "was measured by Qubit 3.0 fluorometer (Life Technologies) and RNA purity\n",
      "was checked via the kaiaoK5500®Spectrophotometer (Kaiao, Beijing,\n",
      "China). Sample integrity was assessed using an Agilent 2100 RNA Nano\n",
      "6000 assay kit (Agilent Technologies). RNA-seq was performed by Annoroad\n",
      "Gene Technology Corporation (Beijing).\n",
      "\n",
      "Briefly, sequencing libraries were constructed using the NEBNext Ultra\n",
      "RNA Library Prep Kit for Illumina (NEB, USA). Clustering of the\n",
      "index-coded samples was performed on a cBot cluster generation system\n",
      "using the HiSeq PE Cluster Kit v4-cBot-HS (Illumina) according to the\n",
      "manufacturer’s instructions. After cluster generation, the libraries\n",
      "were sequenced on an Illumina platform and 150 bp paired-end reads were\n",
      "generated.\n",
      "\n",
      "For the data analysis, an index of the reference genome was built using\n",
      "HISAT2 v2.0.5 and paired-end clean reads were aligned to the reference\n",
      "genome via HISAT2 v2.0.5. FeatureCounts v1.5.0-p3 was used to count the\n",
      "numbers of reads mapped to each gene. Differential expression analysis\n",
      "of the two groups was performed using the DESeq2 R package. Heatmaps\n",
      "were generated via pheatmap (v1.0.12) package in R.\n",
      "\n",
      "**Xenograft assay**\n",
      "\n",
      "All animal experiments were approved by the Animal Research Ethics Board\n",
      "of Wuhan University (WAEF-2022-0113) and confirmed by the animal\n",
      "experimental guidelines. Four-week-old female BALB/c nude mice were\n",
      "purchased from Beijing Vital River Laboratory Animal Technology (China)\n",
      "and were housed under specific pathogen-free conditions in Animal\n",
      "Experiment Center-Animal Biosafety Level-III Laboratory of Wuhan\n",
      "University. The animals were randomly divided into four groups of 6 mice\n",
      "each. The mice were injected with 100 μL of cells suspended in Matrigel\n",
      "Basement Membrane Matrix (Corning, 356234) at a population of 3 ×\n",
      "10<sup>6</sup> ACHN indicated cells into the left or right dorsal flank\n",
      "subcutaneously.\n",
      "\n",
      "The tumor size was estimated once a week via a digital Vernier caliper.\n",
      "The tumor volume was calculated with the following formula: tumor volume\n",
      "(mm3) =length (mm)×width<sup>2</sup> (mm)× 0.5. Tumors were havested 8\n",
      "weeks after injection.\n",
      "\n",
      "**Statistical analysis**\n",
      "\n",
      "The quantitative experiment was repeated three times or more\n",
      "independently. The graphs were generated with GraphPad Prism software.\n",
      "Two-tailed Student’s t-test, one-way ANOVA and the Wilcoxon rank-sum\n",
      "test were used to assess the significance of the experiments. *P* values\n",
      "&lt; 0.05 were considered to indicate the significant differences.\n",
      "\n",
      "**\n",
      "**\n",
      "\n",
      "**Data Availability**\n",
      "\n",
      "The sequencing data are available in GEO Data Sets with the accession\n",
      "number GSE270718\n",
      "(https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE270718),\n",
      "GSE270719 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE270719)\n",
      "and GSE275838\n",
      "(https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE275838). All\n",
      "other raw data support the findings of this study are available from the\n",
      "corresponding author upon reasonable request.\n",
      "\n",
      "**\n",
      "**\n",
      "\n",
      "**Acknowledgments**\n",
      "\n",
      "This work was supported by Natural Science Foundation of China (grant\n",
      "numbers 82203491 and 82173137), Natural Science Foundation of Hubei\n",
      "Province of China (grant number 2022CFA008), China Postdoctoral Science\n",
      "Foundation (grant number 2022TQ0237), Natural Science Foundation of\n",
      "Inner Mongolia (grant number 2024QN08005) and Translational Medicine and\n",
      "Interdisciplinary Research Joint Fund of Zhongnan Hospital of Wuhan\n",
      "University (grant number ZNJC202212).\n",
      "\n",
      "**\n",
      "**\n",
      "\n",
      "**Disclosure Statement & Competing Interests**\n",
      "\n",
      "These authors declare that they have no conflict of interest.\n",
      "\n",
      "**\n",
      "**\n",
      "\n",
      "**References**\n",
      "\n",
      "Bonavida B, Kaufhold S (2015) Prognostic significance of YY1 protein\n",
      "expression and mRNA levels by bioinformatics analysis in human cancers:\n",
      "a therapeutic target. *Pharmacol Ther* 150: 149-168\n",
      "\n",
      "Chen X, Loo JX, Shi X, Xiong W, Guo Y, Ke H, Yang M, Jiang Y, Xia S,\n",
      "Zhao M *et al* (2018) E6 Protein Expressed by High-Risk HPV Activates\n",
      "Super-Enhancers of the EGFR and c-MET Oncogenes by Destabilizing the\n",
      "Histone Demethylase KDM5C. *Cancer Res* 78: 1418-1430\n",
      "\n",
      "Dawson MA (2017) The cancer epigenome: Concepts, challenges, and\n",
      "therapeutic opportunities. *Science* 355: 1147-1152\n",
      "\n",
      "Gao P, Li L, Yang L, Gui D, Zhang J, Han J, Wang J, Wang N, Lu J, Chen S\n",
      "*et al* (2019) Yin Yang 1 protein ameliorates diabetic nephropathy\n",
      "pathology through transcriptional repression of TGFbeta1. *Sci Transl\n",
      "Med* 11\n",
      "\n",
      "Gordon S, Akopyan G, Garban H, Bonavida B (2006) Transcription factor\n",
      "YY1: structure, function, and therapeutic implications in cancer\n",
      "biology. *Oncogene* 25: 1125-1142\n",
      "\n",
      "Hanahan D (2022) Hallmarks of Cancer: New Dimensions. *Cancer Discov*\n",
      "12: 31-46\n",
      "\n",
      "Hogg SJ, Beavis PA, Dawson MA, Johnstone RW (2020) Targeting the\n",
      "epigenetic regulation of antitumour immunity. *Nat Rev Drug Discov* 19:\n",
      "776-800\n",
      "\n",
      "Iwase S, Lan F, Bayliss P, de la Torre-Ubieta L, Huarte M, Qi HH,\n",
      "Whetstine JR, Bonni A, Roberts TM, Shi Y (2007) The X-linked mental\n",
      "retardation gene SMCX/JARID1C defines a family of histone H3 lysine 4\n",
      "demethylases. *Cell* 128: 1077-1088\n",
      "\n",
      "Ji X, Jin S, Qu X, Li K, Wang H, He H, Guo F, Dong L (2015)\n",
      "Lysine-specific demethylase 5C promotes hepatocellular carcinoma cell\n",
      "invasion through inhibition BMP7 expression. *BMC Cancer* 15: 801\n",
      "\n",
      "Khachigian LM (2018) The Yin and Yang of YY1 in tumor growth and\n",
      "suppression. *Int J Cancer* 143: 460-465\n",
      "\n",
      "Li Y, Li J, Li Z, Wei M, Zhao H, Miyagishi M, Wu S, Kasim V (2022)\n",
      "Homeostasis Imbalance of YY2 and YY1 Promotes Tumor Growth by\n",
      "Manipulating Ferroptosis. *Adv Sci (Weinh)* 9: e2104836\n",
      "\n",
      "Meliala ITS, Hosea R, Kasim V, Wu S (2020) The biological implications\n",
      "of Yin Yang 1 in the hallmarks of cancer. *Theranostics* 10: 4183-4200\n",
      "\n",
      "Miranda Furtado CL, Dos Santos Luciano MC, Silva Santos RD, Furtado GP,\n",
      "Moraes MO, Pessoa C (2019) Epidrugs: targeting epigenetic marks in\n",
      "cancer treatment. *Epigenetics* 14: 1164-1176\n",
      "\n",
      "Nebbioso A, Tambaro FP, Dell'Aversana C, Altucci L (2018) Cancer\n",
      "epigenetics: Moving forward. *PLoS Genet* 14: e1007362\n",
      "\n",
      "Nijman SM, Friend SH (2013) Cancer. Potential of the synthetic lethality\n",
      "principle. *Science* 342: 809-811\n",
      "\n",
      "Outchkourov N, MuinO J, Kaufmann K, Van Ijcken WJ, Koerkamp MJ, Van LD,\n",
      "P. GD, Holstege FP, Grosveld F, Timmers HT (2013) Balancing of Histone\n",
      "H3K4 Methylation States by the Kdm5c/SMCX Histone Demethylase Modulates\n",
      "Promoter and Enhancer Function. *Cell Reports* 3: 1071-1079\n",
      "\n",
      "Qiao K, Ning S, Wan L, Wu H, Wang Q, Zhang X, Xu S, Pang D (2019)\n",
      "LINC00673 is activated by YY1 and promotes the proliferation of breast\n",
      "cancer cells via the miR-515-5p/MARK4/Hippo signaling pathway. *J Exp\n",
      "Clin Cancer Res* 38: 418\n",
      "\n",
      "Sarvagalla S, Kolapalli SP, Vallabhapurapu S (2019) The Two Sides of YY1\n",
      "in Cancer: A Friend and a Foe. *Front Oncol* 9: 1230\n",
      "\n",
      "Scandaglia M, Lopez-Atalaya JP, Medrano-Fernandez A, Lopez-Cascales MT,\n",
      "Del Blanco B, Lipinski M, Benito E, Olivares R, Iwase S, Shi Y *et al*\n",
      "(2017) Loss of Kdm5c Causes Spurious Transcription and Prevents the\n",
      "Fine-Tuning of Activity-Regulated Enhancers in Neurons. *Cell Rep* 21:\n",
      "47-59\n",
      "\n",
      "Secombe J, Li L, Carlos L, Eisenman RN (2007) The Trithorax group\n",
      "protein Lid is a trimethyl histone H3K4 demethylase required for\n",
      "dMyc-induced cell growth. *Genes Dev* 21: 537-551\n",
      "\n",
      "Shen H, Xu W, Guo R, Rong B, Gu L, Wang Z, He C, Zheng L, Hu X, Hu Z *et\n",
      "al* (2016) Suppression of Enhancer Overactivation by a RACK7-Histone\n",
      "Demethylase Complex. *Cell* 165: 331-342\n",
      "\n",
      "Shen HF, Zhang WJ, Huang Y, He YH, Hu GS, Wang L, Peng BL, Yi J, Li TT,\n",
      "Rong R *et al* (2021) The Dual Function of KDM5C in Both Gene\n",
      "Transcriptional Activation and Repression Promotes Breast Cancer Cell\n",
      "Growth and Tumorigenesis. *Adv Sci (Weinh)* 8: 2004635\n",
      "\n",
      "Shrivastava A, Saleque S, Kalpana GV, Artandi S, Goff SP, Calame K\n",
      "(1993) Inhibition of transcriptional regulator Yin-Yang-1 by association\n",
      "with c-Myc. *Science* 262: 1889-1892\n",
      "\n",
      "Sui G, Affar el B, Shi Y, Brignone C, Wall NR, Yin P, Donohoe M, Luke\n",
      "MP, Calvo D, Grossman SR *et al* (2004) Yin Yang 1 is a negative\n",
      "regulator of p53. *Cell* 117: 859-872\n",
      "\n",
      "van Leenders GJ, Dukers D, Hessels D, van den Kieboom SW, Hulsbergen CA,\n",
      "Witjes JA, Otte AP, Meijer CJ, Raaphorst FM (2007) Polycomb-group\n",
      "oncogenes EZH2, BMI1, and RING1 are overexpressed in prostate cancer\n",
      "with adverse pathologic and clinical features. *Eur Urol* 52: 455-463\n",
      "\n",
      "Wang W, Qiao S, Li G, Cheng J, Yang C, Zhong C, Stovall DB, Shi J, Teng\n",
      "C, Li D *et al* (2022) A histidine cluster determines\n",
      "YY1-compartmentalized coactivators and chromatin elements in\n",
      "phase-separated enhancer clusters. *Nucleic Acids Res* 50: 4917-4937\n",
      "\n",
      "Zafar A, Wang W, Liu G, Wang X, Xian W, McKeon F, Foster J, Zhou J,\n",
      "Zhang R (2021) Molecular targeting therapies for neuroblastoma: Progress\n",
      "and challenges. *Med Res Rev* 41: 961-1021\n",
      "\n",
      "Zhang SM, Cai WL, Liu X, Thakral D, Luo J, Chan LH, McGeary MK, Song E,\n",
      "Blenman KRM, Micevic G *et al* (2021) KDM5B promotes immune evasion by\n",
      "recruiting SETDB1 to silence retroelements. *Nature* 598: 682-687\n",
      "\n",
      "Zheng Q, Li P, Zhou X, Qiang Y, Fan J, Lin Y, Chen Y, Guo J, Wang F, Xue\n",
      "H *et al* (2021) Deficiency of the X-inactivation escaping gene KDM5C in\n",
      "clear cell renal cell carcinoma promotes tumorigenicity by reprogramming\n",
      "glycogen metabolism and inhibiting ferroptosis. *Theranostics* 11:\n",
      "8674-8691\n",
      "\n",
      "**\n",
      "**\n",
      "\n",
      "**Figure Legends**\n",
      "\n",
      "**Figure 1 YY1 was identified as a KDM5C binding protein**\n",
      "\n",
      "A. Forest plot of the hazard ratio (HR) showing the association between\n",
      "high expression levels of KDM5C and overall survival in multiple\n",
      "cancers. For each type of cancer, the HR and 95% confidence interval\n",
      "(CI) are plotted as red dots and horizontal lines on the basis of a Cox\n",
      "proportional hazards model; when the HR=1, the study object had no\n",
      "effect on overall survival; when the HR&gt;1, the study object was\n",
      "determined to be a risk factor; and when the HR&lt;1, the study object\n",
      "was identified as a protective factor.\n",
      "\n",
      "B. Strategy for identifying KDM5C-binding partners in HEK293T cells via\n",
      "TAP-MS.\n",
      "\n",
      "C-D. Co-IP followed by Western blotting was used to detect the\n",
      "interaction between KDM5C and YY1 exogenously (C) and endogenously (D)\n",
      "in HEK 293 cells.\n",
      "\n",
      "E. Semi-endogenous co-IP with KDM5C-Flag and YY1 in the renal cancer\n",
      "cell line 769-P.\n",
      "\n",
      "F. Schematic representation of KDM5C domains and corresponding\n",
      "constructs.\n",
      "\n",
      "G. Semi-endogenous co-IP with KDM5C truncation mutants and endogenous\n",
      "YY1 in HEK293T cells.\n",
      "\n",
      "**Figure 2 YY1 is a vulnerable target in KDM5C-deficient tumor cells**\n",
      "\n",
      "A. Western blotting showing KDM5C expression in HK-2 human kidney\n",
      "proximal tubular epithelial cells and renal cancer cell lines (ACHN,\n",
      "769-P and RCC4).\n",
      "\n",
      "B. Effect of YY1 knockdown on the survival of the indicated tumor cell\n",
      "lines as measured by a colony formation assay (n=3 biological\n",
      "replicates).\n",
      "\n",
      "C. Proliferation of the indicated cell lines after YY1 was depleted, as\n",
      "measured by a CCK8 assay. The data are shown as the mean ± SD of three\n",
      "independent experiments after analysis by one-way ANOVA. In ACHN cells,\n",
      "*P*=0.116 (shYY1-1) and *P*=0.1222 (shYY1-2); in 769-P cells, *P*=0.7353\n",
      "(shYY1-1) and *P*=0.9113 (shYY1-2); and in RCC4 cells, *P*=0.0007\n",
      "(shYY1-1) and *P*=2E-05 (shYY1-2).\n",
      "\n",
      "D. Xenograft tumors excised from nude mice (n = 6/group) that had been\n",
      "subcutaneously injected with the indicated tumor cells. Statistical\n",
      "significance was determined by one-way ANOVA. *P*=1E-05\n",
      "\n",
      "**Figure 3 KDM5C is essential for YY1 genome-wide chromatin\n",
      "recruitment**\n",
      "\n",
      "A. Violin plot showing the alteration in YY1 binding peaks after KDM5C\n",
      "depletion. YY1 binding peaks in both ACHN parental and shKDM5C cells\n",
      "were counted (n=17,030). The data are presented as mean ± SD.\n",
      "Statistical significance was determined by the Mann-Whitney U test, *P*=\n",
      "2.22E-16.\n",
      "\n",
      "B. Venn diagram showing the overlapping peaks of YY1 and KDM5C in the\n",
      "co-occupied regions. Representative data from two independent CUT-Tag\n",
      "experiments.\n",
      "\n",
      "C. Heatmap and metaplot showing the chromatin enrichment of YY1 at\n",
      "overlapping YY1 and KDM5C peak regions (n=3645) in ACHN parental and\n",
      "shKDM5C cells.\n",
      "\n",
      "D. Pie chart showing the annotated genomic distribution of altered YY1\n",
      "peaks on different DNA elements in shKDM5C or KDM5C overexpressing\n",
      "cells.\n",
      "\n",
      "**Figure 4 An intact JmjC domain but not KDM5C H3K4 demethylation\n",
      "activity is required for KDM5C-mediated YY1 recruitment**\n",
      "\n",
      "A. Heatmap and metaplot showing the distribution of YY1 peaks in the\n",
      "overlapping regions of YY1 and KDM5C (n=3645) in ACHN cells expressing\n",
      "the KDM5C-WT or KDM5C-H514A mutant.\n",
      "\n",
      "B. Loss of KDMC5 resulted in increased H3K4me3 signal intensity in the\n",
      "YY1 and KDM5C overlapping region (n=3645) in ACHN parental and shKDM5C\n",
      "cells.\n",
      "\n",
      "C. Venn diagram showing the number of YY1 peaks in the H3K4me3/YY1\n",
      "overlapping region (n=11087) with or without modulation by KDM5C.\n",
      "\n",
      "D-E. Heatmaps and metaplots revealed reduced YY1 enrichment at KDM5C\n",
      "demethylase dependent regions (n=9943) or KDM5C demethylase independent\n",
      "regions (n=1144) in ACHN parental and shKDM5C cells.\n",
      "\n",
      "F. KDM5C protein levels in whole-cell lysates or chromatin fractions\n",
      "from HEK293T cells transfected with KDM5C-WT or KDM5C-H514A mutant\n",
      "construct.\n",
      "\n",
      "G-I. Effect of the re-expression of the KDM5C-WT or KDM5C-H514A mutant\n",
      "on the proliferation of ACHN (G), 769-P (H) or RCC4 (I) cells after the\n",
      "depletion KDM5C, YY1 or both, as determined by the CCK8 assay. Each\n",
      "experiment was repeated three times with similar results. “Dsh”\n",
      "indicates cells in which KDM5C and YY1 were both knocked down. The data\n",
      "are shown as the mean ± SD of three independent experiments after\n",
      "analysis via Student’s t test. In ACHN cells, *P*=5.37E-08 (Dsh) and\n",
      "*P*=3.48E-07 (Dsh+KDM5C); in 769-P cells, *P*=4.45E-10 (Dsh) and\n",
      "*P*=7.49E-10 (Dsh+KDM5C); in ACHN cells, *P*=5.86E-09 (Dsh) and\n",
      "*P*=2.72E-07 (Dsh+KDM5C).\n",
      "\n",
      "J. Colony formation assay with the indicated cell lines(n=3). “Dsh”\n",
      "indicates cells with KDM5C and YY1 simultaneously knocked down.\n",
      "\n",
      "**Figure 5 KDM5C and YY1 coregulate the expression of a number of\n",
      "essential cell-fate genes**\n",
      "\n",
      "A-C. Volcano plots showing the change in the number of transcripts in\n",
      "shKDM5C, shYY1 or double shKDM5C ACHN cells, as determined by RNA-seq\n",
      "analysis (\\|log<sub>2</sub> FC\\|&gt;1.5, p value &lt;0.05). “Double sh”\n",
      "indicates cells with KDM5C and YY1 simultaneously knocked down.\n",
      "\n",
      "D. Venn diagrams showing the number of downregulated genes in shKDM5C,\n",
      "shYY1 or shKDM5C/YY1 double-knockdown cells.\n",
      "\n",
      "E. Heatmap showing the normalized gene expression profile in shKDM5C,\n",
      "shYY1 or double sh ACHN cells. “Double sh” indicates cells with KDM5C\n",
      "and YY1 simultaneously knocked down.\n",
      "\n",
      "F. qPCR was used to assess the transcription of select genes in the\n",
      "indicated ACHN cells. Each experiment was repeated three times with\n",
      "similar results. “Double sh” indicates cells with KDM5C and YY1 double\n",
      "knocked down. The data are shown as the mean ± SD of three independent\n",
      "experiments after analysis by two tailed Student’s T-test. *P*=2.86E-05\n",
      "(JAM3), *P*=4.06E-06 (KNTC1), *P*=1.31E-04(SMC2), *P*=9.18E-06(SMC4),\n",
      "*P*=1.91E-05(LMNB1) and *P*=1.47E-04 (FOXM1).\n",
      "\n",
      "**Figure 6 Simultaneous targeting of KDM5C and YY1 effectively induces\n",
      "cell cycle arrest in the G2/M phase and apoptosis.**\n",
      "\n",
      "A. Cell cycle analysis with the indicated cell lines via flow cytometry.\n",
      "The percentage of cells in each phase of the cell cycle is shown on the\n",
      "Y-axis. Each experiment was repeated three times with similar results.\n",
      "“Double sh” indicates cells with simultaneous knockdown of KDM5C and\n",
      "YY1.\n",
      "\n",
      "B. Flow cytometry–based Annexin-V/PI staining showing cells in the\n",
      "indicated lineages undergoing apoptosis. The quantitative results\n",
      "indicate the ratio of apoptotic cells. Each experiment was repeated\n",
      "three times with similar results. “Double sh” indicates cells with\n",
      "simultaneous knockdown of KDM5C and YY1. The data are shown as the mean\n",
      "± SD of three independent experiments after analysis by two tailed\n",
      "Student’s T-test. *P*=5.3E-05 (RCC4) and *P*=1.1E-06 (769-P).\n",
      "\n",
      "**Figure 7 A mechanistic hypothesis model for KDM5C-mediated YY1\n",
      "recruitment.**\n",
      "\n",
      "YY1 and KDM5C constitute a pair of synthetic lethal genes. KDM5C\n",
      "recruits YY1 to the promoter regions of cell cycle- or apoptosis-related\n",
      "genes. Targeting YY1 increases the vulnerability of KDM5C-deficient\n",
      "cancer cells.\n",
      "\n",
      "**\n",
      "**\n",
      "\n",
      "**Expanded View Figure Legends**\n",
      "\n",
      "**Figure EV1 Validation of the constructed cell lines used in this\n",
      "study**\n",
      "\n",
      "A-D. KDM5C and YY1 protein levels in the indicated cells were measured\n",
      "via Western blotting. “Dsh” indicates cells in which KDM5C and YY1 were\n",
      "both knocked down.\n",
      "\n",
      "E-F. Proliferation rate of the indicated cell lines upon KDM5C\n",
      "depletion, as determined via a CCK8 assay. The data are shown as the\n",
      "mean ± SD of three independent experiments after analysis via one-way\n",
      "ANOVA. In 769-P cells, *P*=0.2701 (shKDM5C -1) and *P*=0.4753\n",
      "(shKDM5C-2); in ACHN cells, *P*=0.4308 (shKDM5C-1) and *P*=0.3869\n",
      "(shYY1-2).\n",
      "\n",
      "**Figure EV2 ChIP-seq experiments targeting YY1 in HK-2 cells**\n",
      "\n",
      "A. Heatmap and metaplot showing the chromatin enrichment of YY1 in HK-2\n",
      "cells in the presence or absence of KDM5C.\n",
      "\n",
      "B-C. ChIP-PCR assay showing the enrichment of YY1 at the promoters of\n",
      "the indicated genes in HK-2 cells after the depletion of YY1, KDM5C or\n",
      "both (B), or in cells expressing KDM5C-WT or KDM5C-H514A mutant (C).\n",
      "Each experiment was repeated three times with similar results.\n",
      "Statistical significate was determined by one-way ANOVA. In B,\n",
      "*P*=1.24E-06 (JAM3), *P*=2.27E-06 (KNTC1), *P*=2.82E-08 (SMC2),\n",
      "*P*=2.41E-05 (SMC4) and *P*=3.96E-05 (FOXM1), respectively. In C,\n",
      "*P*=4.53E-06 (JAM3), *P*=4.46E-06 (KNTC1), *P*=5.53E-06(SMC2),\n",
      "*P*=6.82E-06 (SMC4), *P*=1.05E-06 (FOXM1) and *P*=0.0002 (LMNB1),\n",
      "respectively.\n",
      "\n",
      "**Figure EV3 Evaluation of the interaction between YY1 and KDM5C-WT or\n",
      "KDM5C-H514A mutant.**\n",
      "\n",
      "Co-IP followed by Western blotting showing the interaction between YY1\n",
      "and KDM5C-WT or KDM5C-H514A mutant in HEK293T cells.\n",
      "\n",
      "**Figure EV4 RNA-seq in HK-2 derived cells.**\n",
      "\n",
      "A-C. Volcano plots showing the change in the number of transcripts in\n",
      "shKDM5C, shYY1 or double-knockdown HK-2 cells, as determined by RNA-seq\n",
      "analysis (\\|log<sub>2</sub> FC\\|&gt;1.5, p value &lt;0.05).\n",
      "\n",
      "D. Heatmap showing the normalized gene expression profile in shKDM5C,\n",
      "shYY1 or double sh ACHN cells. “Double sh” indicates cells in which\n",
      "KDM5C and YY1 were both knocked down.\n",
      "\n",
      "**Figure EV5 Effects of KDM5C on YY1 enrichment at promoter regions of\n",
      "specific genes**\n",
      "\n",
      "A-B. The connections between the ChIP-seq and RNA-seq data were\n",
      "analyzed, and the genomic regions of the downregulated genes in the\n",
      "KDM5C/YY1 double-knockdown group were annotated.\n",
      "\n",
      "C-D. ChIP-seq snapshots reflecting changes in YY1 enrichment at the\n",
      "promoter regions of the JAM3, LNMB1, FOXM1, KNTC1, SMC2 and SMC4 genes\n",
      "in the indicated cell lines. The promoters are indicated by H3K4me3\n",
      "peaks.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "from wisup_e2m import DocxParser\n",
    "\n",
    "parser = DocxParser(engine=\"pandoc\") # docx engines: pandoc, xml\n",
    "docx_data = parser.parse(DOCX_FILES[5])\n",
    "print(docx_data.text)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### XML ENGINE"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Latrophilin-2 mediates fluid shear stress mechanotransduction at endothelial junctions'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Keiichiro Tanaka1,7,*, Minghao Chen1,7, Andrew Prendergast1, Zhenwu Zhuang1, Ali Nasiri2, Divyesh Joshi1, Jared Hintzen1, Minhwan Chung1, Abhishek Kumar1, Arya Mani1, Anthony Koleske3, Jason Crawford4, Stefania Nicoli1 and Martin A. Schwartz1,5,6,*'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='1 Yale Cardiovascular Research Center, Section of Cardiovascular Medicine, Department of Internal Medicine, School of Medicine, Yale University, New Haven, CT 06511, USA'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='2 Department of Internal Medicine'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='3 Department of Molecular Biochemistry and Biophysics'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='4 Department of Chemistry'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='5 Department of Cell Biology'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='6 Department of Biomedical Engineering '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='7 These authors contributed equally.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Running title: Latrophilins in shear stress sensing'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='*Authors for correspondence:'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='keiichiro.tanaka@yale.edu; martin.schwartz@yale.edu;'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Abstract'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Endothelial cell responses to fluid shear stress from blood flow are crucial for vascular development, function and disease. A complex of PECAM-1, VE-cadherin, VEGF receptors (VEGFRs) and PlexinD1 located at cell-cell junctions mediates many of these events. But available evidence suggests that another mechanosensor upstream of PECAM-1 initiates signaling. Hypothesizing that GPCR and Gα proteins may serve this role, we performed siRNA screening of Gα subunits and found that Gαi2 and Gαq/11 are required for activation of the junctional complex. We then developed a new activation assay, which showed that these G proteins are activated by flow. We next mapped the Gα residues required for activation and developed an affinity purification method that used this information to identify latrophilin-2 (Lphn-2/ADGRL2) as the upstream GPCR.  Latrophilin-2 is required for all PECAM-1 downstream events tested. In both mice and zebrafish, latrophilin-2 is required for flow-dependent angiogenesis and artery remodeling. Furthermore, endothelial specific knockout demonstrates that latrophilin plays a role in flow-dependent artery remodeling.  Human genetic data reveal a correlation between the latrophilin-2-encoding Adgrl2 gene and cardiovascular disease. Together, these results define a pathway that connects latrophilin-dependent G protein activation to subsequent endothelial signaling, vascular physiology and disease. '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Keywords'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Latrophilin, G protein-coupled receptor, fluid shear stress, vascular development, PECAM-1'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Introduction'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Fluid shear stress (FSS) from blood flow is critical for early vascular development in mice(Lucitti et al, 2007) and zebrafish(Sugden et al, 2017), for homeostatic vessel remodeling in adults(Humphrey & Schwartz, 2021; Vogel et al, 2020), and in the initiation and progression of atherosclerosis(Baeyens et al, 2016; Zhou et al, 2014).  A complex of proteins at cell-cell junctions consisting of PECAM-1, VE-cadherin, VEGFR2 and 3, and Plexin D1 plays an important role in shear stress responses including vasodilation, flow-dependent vessel remodeling, and atherosclerosis(Conway & Schwartz, 2012; Givens & Tzima, 2016). Stimulation of this junctional complex by flow triggers activation of Src-family kinases (SFKs) within seconds, resulting in ligand-independent activation of VEGF receptors and downstream events including endothelial cell (EC) alignment in the direction of flow(Collins et al, 2012). However, the mechanisms by which proteins located at cell-cell junctions can sense forces from shear stress exerted on the apical surface is unclear. Previous work suggests that activation of the junctional complex is not primary but rather requires an upstream event, mediated presumably by another mechanosensor (Conway et al, 2013). '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='GPCR/Gα protein signaling is an obvious candidate for initiating signaling on this time scale (seconds). GPCRs are a large family of cell surface receptors that play a crucial role in transducting signals across cell membranes.  GPCRs undergo conformational changes in response to ligand binding or other stimuli, after which the activated intracellular domain binds to a heterotrimeric Gα protein complex and catalyzes the exchange of guanosine diphosphate (GDP) for guanosine triphosphate (GTP). Connections between GPCRs and shear stress have been reported(Chachisvilis et al, 2006; Dela Paz & Frangos, 2019; Erdogmus et al, 2019; Jung et al, 2012; Wang et al, 2015; Xu et al, 2018). However, the literature on Gα proteins and GPCRs in shear stress signaling is highly inconsistent(Chachisvilis et al., 2006; Dela Paz & Frangos, 2019; Jung et al., 2012; Wang et al., 2015).  For instance, it has been demonstrated that flow applied to endothelial cells activates Gα proteins independent of an intervening GPCR(Dela Paz et al, 2017; Gudi et al, 1998).  It has also been reported that shear stress responses related to PECAM-1 and cell-cell junction require GPCRs but with different GPCRs implicated in different studies.  These studies include reports of ligand-independent activation of a bradykinin receptor(Chachisvilis et al., 2006; Yeh et al, 2008) and the sphingosine 1-phosphate receptor S1PR1(Jung et al., 2012).  The purinergic receptor P2Y2 was also implicated, in this case via ATP release downstream of activation of the mechanosensitive ion channel Piezo1(Albarran-Juarez et al, 2018; Wang et al., 2015), though this pathway has been disputed(Dela Paz & Frangos, 2019). The orphan receptor GPR68 was identified as a shear stress sensor specific to small arteries though the mechanism of activation, including ligand dependence or independence, is unknown(Xu et al., 2018). These studies also vary in the Gα proteins implicated, which included different Gαi isoforms (Gudi et al, 1996; Jung et al., 2012) and the Gαq/11 family(Albarran-Juarez et al., 2018; Chachisvilis et al., 2006; Wang et al., 2015).  '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='To resolve this issue, we took an unbiased approach starting from the Gα proteins using the dramatic morphological changes in endothelial alignment to flow as the initial readout. Our results showed that Gαi2 and Gαq/11 function in parallel, identified latrophilin-2 (LPHN2) as the key upstream GPCR and demonstrated a specific requirement for LPHN2 in flow-dependent remodeling in zebrafish and mice.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Results'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Identification of Gα protein subunits required for flow sensing'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='To investigate the roles of GPCR signaling in endothelial flow responses, we first transfected human umbilical vein endothelial cells (HUVECs) with short interfering RNAs (siRNAs) to suppress Gs, Gi, Gq/11 and G12/13 classes of Gα proteins and assayed EC alignment in flow as our initial readout. Single knockdowns gave either weak (Gi) or no (Gs, G12/13, Gq/11) inhibition of EC alignment in flow (Fig.1A and Appendix fig. S1A-C). However, simultaneous knockdown of Gi and Gq/11 completely inhibited alignment (Fig. 1A), whereas other combinations were ineffective (Appendix fig. S1D, S1E). Activation of SFKs is the earliest flow-responsive event mediated by PECAM-1, followed by VEGFRs and Akt(Tzima et al, 2005). These events were similarly blocked by knockdown of Gi plus Gq/11 but not by single knockdowns (Fig. 1B, 1C and Appendix fig. S1F). Therefore, Gi and Gq/11 act in parallel to initiate PECAM-1-dependent flow signaling.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Rescue controls with siRNA-resistant Gα variants showed that siRNA-resistant Gq and Gi2 restored cell alignment, whereas Gi1 and Gi3 did not (Fig. 1D and Appendix fig. S2A, S2B).  This unexpected result prompted us to search for residues that are similar in Gq and Gi2 but different in Gi1 and Gi3.  Two residues (using human Gi2 numbering) fit this pattern: D167 in Gi2 corresponds to N in Gi1/3; and K307 corresponds to Q306 in Gi1/3 (Fig. 1E).  Gi1(Q306K), but not Gi1(N166D), rescued EC alignment in flow (Fig. 1F and Appendix fig. S2C).  Consistently, Gi2 mutant lacking K307 (K307Q) could not rescue the defects on flow-induced alignment under the knockdown of Gi plus Gq/11 (Appendix fig. S1G and Appendix fig. S2D).  Therefore, K307 in Gi2 distinguishes the Gα subunits participate in flow signaling.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Flow-induced activation of specific Gα proteins'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='K307 lies in the region that specifies coupling with GPCRs(Bae et al, 1999; Flock et al, 2017), suggesting that K307 may determine Gα activation.  However, testing this hypothesis requires measurement of Gα activation in live cells.  Unfortunately, indirect Gα activation assays using second messengers such as cAMP or calcium are unreliable in the context of fluid shear stress where multiple activating/suppressing pathways are stimulated. We therefore developed pulldown assays for Gα based on the specific binding of GTP-loaded Gα subunits to effector proteins, namely GINIP for Gi and GRK2 N-terminal domain for Gq (Appendix fig. S3a)(Gaillard et al, 2014; Tesmer et al, 2005).  To facilitate these assays and distinguish specific Gα isoforms (especially Gi isoforms), we prepared versions containing an internal GluGlu (EE) epitope tag that does not impair function (Appendix fig. S3B and refs(Medina et al, 1996; Wilson & Bourne, 1995)).  For Gq, to enhance pulldown efficiency, we co-expressed Ric8A, which stabilizes active Gq(Papasergi et al, 2015).  Active, but not inactive, Gi and Gq bound to these effector proteins, while when the artificial GPCR DREADD was expressed, its ligand clozapine-N-oxide (CNO) also activated Gα proteins (Appendix fig. S3C-S3F).  Importantly, this Gα pulldown assay showed not only rapid activation of Gi2 and Gq by FSS (Fig. 1G-J), but also differential responses of Gi1 and gain-of-function Gi1(Q306K) to FSS (Fig. 1K, 1L).  Gi activation was unaffected by PECAM-1 knockdown, indicating that this step was upstream or independent of PECAM-1 (Fig. 1M, 1N).  Flow-mediated activation of specific Gα subunits thus corresponds to their requirement in EC flow responses.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Identification of the upstream GPCR'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='We next sought to identify the GPCR(s) that mediates flow activation of Gi2 and Gq/11. Although affinity purification is the obvious choice, rapid release of Gα proteins from GPCRs after GTP loading limits this approach.  Starting with wild-type and its Q306K mutant, we inserted 1) an internal GFP for affinity purification and 2) four alanines (hereafter ins4A) into helix α5, which blocks GTP loading, thereby preventing dissociation from GPCRs even in the presence of GTP(Kaya et al, 2016) (Fig. 2A). Both constructs localized mainly to the plasma membrane, as did wild-type Gα (Appendix fig. S4A).  We further employed detergent-free nanodisc-forming styrene-maleimide anhydride (SMA) copolymer to stabilize GPCR conformation within its native lipid environment(Lee et al, 2016).  This approach was validated by coimmunoprecipitation using GFP-Trap® nanobody beads of Gα variants with DREADD GPCR, which showed increased association after receptor activation (Appendix fig. S4B, S4C).  Proteomic analysis for GPCRs that associated with Gi1(Q306K) but not wild-type Gi1 in response to FSS (protocol in Appendix fig. S4D), identified S1PR1 and ADGRL3 (Appendix fig. S4E).  However, depletion of S1PR1 did not block alignment in FSS (Appendix fig. S4F, S4G).  Furthermore, S1PR1 reportedly activates Gi1 and Gi3 as well as Gi2(Lee et al, 1996), which is inconsistent with the Gα specificity defined above.  We therefore focused on ADGRL family proteins, also called latrophilins (LPHNs).'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Latrophilins (1,2 and 3 in mammals) are adhesion-type GPCRs that bind to counterreceptors on neighboring cells and regulate neuronal synapses(Sando et al, 2019).  Although it has been reported that different types of endothelial cells have different latrophilin isoforms, LPHN2 is most widely expressed and is the overwhelming major isoform in HUVECs (Appendix figs S5A)(Maleszewska et al, 2016).  Consistent with the proteomic results, Gi1(Q306K)(ins4A) associated with endogenous LPHN2, which increased upon FSS (Fig. 2B, C). This result strongly suggests that FSS activates LPHN2 to recruit Gα proteins. Furthermore, Gi1(Q306K), Gi2, and Gq co-immunoprecipitated with LPHN2, whereas Gi1 did not, indicating that LPHN2 specifically bound to the flow-responsive Gα proteins defined above (Fig. 2D).  Importantly, LPHN2 knockdown blocked flow-mediated activation of Gi2 (Fig. 2E, 2F).  Together, these data provide strong evidence that latrophilin-2 mediates Gα activation essential for endothelial alignment in response to flow.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Latrophilins in EC flow signaling in vitro.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Knockdown of LPHN2, but not LPHN1 or LPHN3, blocked EC alignment in flow (Fig. 3A, 3B; knockdown confirmed in Appendix fig. S5B, S5C).  LPHN2 knockdown in human aortic ECs similarly blocked flow-induced alignment (Appendix fig. S5D, S5E).  In line with these results, LPHN2 knockdown also blocked Golgi apparatus polarization in flow (Appendix fig. S5F, S5G). LPHN2 localized to cell-cell contacts (Fig.3C and Appendix fig. S6). The flow responsive G1 mutant, Gi1(Q306K), also associated with PECAM-1 following FSS (Fig. 3D). The association between LPHN2-Gi1(Q306K) complex with junctional mechanosensory complex proteins PECAM-1 and VE-Cadherin was detected after 24 hours of flow (Fig. 3D-3G), indicating a persistent interaction. We also assessed the immediate flow-dependent signals. LPHN2 depletion strongly suppressed flow-mediated acute activation of SFKs and Akt (Fig 3H-3J). Thus, LPHN2 colocalizes, physically associates with and is required for signaling through the junctional complex. However, LPHN2 knockdown did not inhibit activation of VEGFR2 by VEGF-A, indicating that LPHN2 functions specifically in response to laminar flow (Appendix fig. S7A, S7B).  '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Among the 3 latrophilin isoforms in human genome, all rescued flow-mediated endothelial alignment after LPHN2 knockdown (Fig. 3K, expression confirmed in Appendix fig. S8A), indicating functional equivalence; the observed differences are therefore due to differential expression.  EC alignment was rescued by wild type (WT) LPHN2 but not H1071A, which is defective in coupling to Gα proteins(Nazarko et al, 2018) (Fig. 3L, expression confirmed in Appendix fig. S8B).  Adhesion-type GPCRs can be activated by internal cleavage by the extracellular GAIN domain which exposures its Stachel peptide as a tethered agonist(Vizurraga et al, 2020). However, the autoproteolysis-defective mutant fully rescued EC alignment (see Fig.3L). Helix-8 is known to endow some GPCRs with mechanosensitivity to membrane stretch(Erdogmus et al., 2019). However, a LPHN2 helix-8 mutant could still rescue EC alignment in flow (Fig. 3L).  Taken together, these findings indicate that latrophilin regulates Gα activation required for endothelial flow responses, but activation requires neither the Stachel peptide nor helix-8.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Latrophilin-2 regulates flow-induced endothelial cell morphological changes in vivo'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Long-term laminar flow stabilizes EC junctions, promoting a linear, continuous morphology associated with barrier function(Kroon et al, 2017) (see Fig.3A).  To test whether the role of LPHN2 in this process, we analyzed junctional morphology in vitro with and without 16h of flow. In control cells, shear stress shifted junctions from an irregular morphology to linear junctions; in Lphn2 knockdown ECs, junctions were indistinguishable in the absence of flow but failed to linearize in flow (Fig. 4A, 4B), demonstrating that the effect is flow-specific.  To further examine contributions of blood flow/latrophilin-2 to ECs in vivo, zebrafish embryos are useful as a developmental system that is amenable to manipulation of shear stress using morpholinos against cardiac troponin T type 2a (tnnt2a; ‘silent heart’)(Sehnert et al, 2002), without global adverse effects.  We employed zebrafish embryos expressing Tg(kdrl:ras-mCherry) to mark endothelial cells(Sehnert et al., 2002) and designed sgRNAs for CRISPR-Cas9 targeting all latrophilin isoforms (adgrl1, adgrl2a plus adgrl2b.1, and adgrl3). Our sgRNAs induced >50% cleavage in the T7 endonuclease assays and reduced mRNA levels through nonsense-mediated decay (Appendix fig. S9). Since Lphn2 is the major endothelial isoform, we performed CRISPR-mediated F(0)/mosaic knockout of adgrl2a+adgrl2b.1 (hereafter Lphn2 mutation), which inhibited elongation and enlargement of ECs in the dorsal aorta (Fig. 4C).  Injecting single cell embryos with silent heart morpholinos at a dose that completely blocked cardiac contractility similarly blocked the elongation of ECs. Furthermore, combining tnnt2a and Lphn2 mutation had no further effect, supporting a specific role for latrophilins in flow sensing (Fig. 4D, 4E).  '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='To address LPHN function in mammals, we crossed Lphn2flox/flox mice with CDH5-CreERT2 mice (Lphn2 ECKO), which were injected with tamoxifen at 8 weeks of age and remained viable 10 weeks later.  The descending thoracic aorta (a region of high laminar flow) of Lphn2 ECKO mice had poorly aligned ECs with irregular junctions (Fig. 4F-H).  Additionally, Lphn2 ECKO increased vascular permeability in the whole aorta, as assessed by Evans blue extravasation (Fig. 4I, 4J). In vitro, flow for 24h increased HUVEC barrier function, which was blocked by LPHN2 knockdown, whereas barrier function in the absence of flow was unaffected (Fig. 4K, 4L). Latrophilins are therefore flow-responsive GPCRs that regulate multiple steps in PECAM1-dependent EC mechanotransduction. '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Plasma membrane fluidization initiates latrophilin signaling'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='We next considered how FSS activates LPHN2. As mutation of the autoproteolytic GAIN domain, which is speculated to be mechanosensitive, had no effect (Figure 3i), we considered alternatives. FSS induces a rapid increase in plasma membrane fluidity, in some cases associated with decreased plasma membrane cholesterol(Butler et al, 2001; Haidekker et al, 2000; Yamamoto & Ando, 2013; Yamamoto et al, 2020). As GPCRs can respond to such changes(Vizurraga et al., 2020), HUVECs were treated with methyl-β-cyclodextrin (MβCD) to extract plasma membrane cholesterol.  Treatment with 5 mM MβCD for 1 min reduced plasma membrane cholesterol to the same extent as flow, assayed by binding of the cholesterol-specific probe D4H-Clover3(Liu et al, 2017) (Figure 5A).  No discernable effect on cytoskeletal organization was observed under these conditions (Figure 5B). MβCD treatment also induced Gα binding to LPHN2 (Figure 5C) and activated both SFKs and VEGFR2, which were blocked by LPHN2 knockdown (Figure 5D-F). Together with published data demonstrating that cholesterol supplementation blocks flow signaling to these and related pathways(Fancher et al, 2018; Tirziu et al, 2005; Yamamoto & Ando, 2013, 2015; Yamamoto et al., 2020), these results suggest that membrane fluidization can trigger LPHN2 GPCR activation and downstream signaling (Figure 5G).'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Latrophilins in vascular remodeling in vivo'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='To address roles of LPHN2 in flow-dependent vascular remodeling, we examined zebrafish embryos, where artery lumen diameters are determined by EC shear stress(Sugden et al., 2017). We observed that intersegmental vessels (ISVs) at 48 hours post fertilization had reduced diameters after both latrophilin knockout and blockade of blood flow, with no further effect when combined (Fig. 6A, 6B). At 72 hpf, when intersegmental arteries and veins are distinguishable, the decrease in diameter was observed only in arteries (Fig. 6C). LPHNs are thus required for effects of flow on artery diameter.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='These results prompted us to examine flow-dependent vascular remodeling in mice.  Femoral artery ligation triggers flow-driven arteriogenesis in the thigh and hypoxia-driven angiogenesis in the calf(Cornella et al, 2017). Following surgery, LPHN2 ECKO mice showed lower blood flow in the foot even at day 1 and markedly slower recovery than WT littermates (Fig. 6D, 6E). To understand these effects, we performed micro-CT of vascular casts after injection of electron-dense bismuth microparticles that label arteries/arterioles but not capillaries due to particle size. Surgery induced an increase in small arteries in WT mice but essentially none in LPHN2 ECKO mice (Fig. 6F, 6G). LPHN2 ECKO mice also had much lower density of small arteries even in the control leg, consistent with lower flow on day 1. Staining for PECAM to examine the total vasculature in leg muscle sections revealed decreased capillary density as well, even at baseline (Fig. 6H, 6I).  LPHN2 ECKO mice thus show low vascular density at baseline and greatly reduced flow-dependent remodeling. As expected from the low vascular density, a treadmill fatigue test (Fig. 6J) LPHN2 ECKO mice showed markedly lower exercise capacity (Fig. 6K, 6L) and impaired oxygen consumption (Fig. 6M). LPHN2 ECKO thus results in vascular defects in vivo, at least in part due to defective EC flow sensing. To address whether these defects were related to shear stress sensing, we examined an in vitro system in which FSS enhances VEGF-induced sprouting into collagen gels(Galie et al, 2014). Enhanced sprouting was completely blocked by LPHN2 depletion with only a weak effect on basal migration (Fig. 6N-6P).'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Genetic link to human disease.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='The Cardiovascular Disease Knowledge Portal reported four SNPs near the 5’ end of the ADGRL2 gene (Appendix fig. S10A) that exhibit genome wide association with hypertension (Appendix fig. S10B). All 4 are in close proximity to each other and are in linkage disequilibrium (D’: 0.9953, R2: 0.97). Two of the SNPs meet the significant threshold (p<5 x 10-8) and rs1195871 is relatively conserved among vertebrates. Of particular interest, the intronic SNP rs186892211 in the ADGRL2 locus (Appendix fig. S10A) is associated with large-artery atherosclerosis and subsequent ischemic stroke (TOAST classification) with a very high odds ratio (odds ratio 720; p< 3.134e-10, Appendix fig. S10B). '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Discussion'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='These data elucidate a pathway in which fluid shear stress acts through the latrophilin adhesion GPCRs to activate subsequent cellular signaling events and vascular remodeling in vivo, thus connecting time scales from seconds to hours to weeks/months. Previous studies showed that laminar flow induces an increase in plasma membrane fluidity associated with depletion of membrane cholesterol (Yamamoto & Ando, 2013, 2015; Yamamoto et al., 2020).  Our data demonstrate that artificial cholesterol depletion activates flow-induced signals in a Lphn2-dependent manner, suggesting that latrophilins connect physical changes in the plasma membrane to cellular signaling. Latrophilins mediate activation of the PECAM-1/VE-cadherin/VEGFR pathway through both Gi2 and Gq/11, suggesting parallel or redundant downstream pathways.  Consistent with these functional effects, FSS induces physical interactions (direct or indirect) between LPHN2-Gα and PECAM-1.  However, LPHNs are not required for activation of VEGFRs by its ligand VEGF.  '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Previous studies identified either Gi or Gq/11 as mediators of flow signaling(Chachisvilis et al., 2006; Jung et al., 2012; Wang et al., 2015).  It seems plausible that, depending on EC subtype or experimental conditions, either Gi or Gq/11 predominated, which concealed their functional redundancy. Previous studies also identified other GPCRs as mediators of EC shear stress responses, via direct or indirect mechanisms, as well as making claims for GPCR-independent activation of G proteins(Dela Paz et al., 2017). GPCRs implicated in shear stress signaling include bradykinin receptor B2(Chachisvilis et al., 2006), ATP receptor P2Y2(Albarran-Juarez et al., 2018; Wang et al., 2015), sphingosine phosphate receptor S1PR1(Jung et al., 2012), the apelin receptor APJ(Strohbach et al, 2018) and GPR68(Xu et al., 2018). An indirect association of Gq/11 with PECAM1 was also reported, which dissociated upon application of shear stress(dela Paz et al, 2014). Differences in EC location or subtype likely account for some of these effects. For example, GPR68 is expressed specifically in small resistance arteries where it controls vasodilation but had no reported effect on development or patterning(Xu et al., 2018). P2Y2 was also reported to affect vasodilation and blood pressure without reported effects on development or patterning.  It seems likely that differences in local expression may thus explain some of the apparent multiplicity, though additional variables such as modulation by other inputs and coupling to different downstream effectors may also contribute. Understanding this GPCR-Gα protein signaling network is therefore an important though complex direction for future work.  The novel pulldown assay and GPCR-Gα protein affinity purification protocol developed here may facilitate such efforts.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='It is also noteworthy that latrophilins are required only for activation of signaling events including VEGFR2 by flow, but not for activation of VEGFR2 by its ligand, VEGF. PECAM-1 first appears in vertebrates but has multiple other functions in leukocyte transmigration and activation, platelet reactivity and angiogenesis that are independent of flow(Privratsky et al, 2010). Together, these findings argue that latrophilins, in essence, confer flow sensitivity to pre-existing pathways controlled by ligand availability.  '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Lphn2 mutation in zebrafish and endothelial knockout in mice caused multiple vascular defects that implicate FSS mechanotransduction, including reduced EC alignment in the direction of flow, altered junctional morphology and permeability, smaller artery diameters and defects in arterialization in the hindlimb ischemia model. Lower capillary and small artery density in some vascular beds may also be attributable to defective FSS sensing, as FSS regulates angiogenic sprouting (Galie et al., 2014) and stabilizes nascent vessels once blood begins to flow (Campinho et al, 2020). We note, however, that flow-independent functional roles for Lphn2 in ECs have been observed (Camillo et al, 2021) and likely contribute as well.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='These results link together many observations to provide a comprehensive model for EC responses to FSS (Appendix fig. S11). These findings are of further interest in light of human genetic data that strongly link the Adgrl2 gene to vascular disease (Appendix fig. S10).  However, the identified polymorphisms reside in regulatory or intronic regions, thus, causal connections to vascular disease are not obvious. Intronic polymorphisms can regulate gene expression but could also be in linkage disequilibrium with causal SNPs that are not yet identified. More detailed population genetic analyses are required to address these issues.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='These findings open many important questions for future work.  Structural insights into latrophilin activation are needed, as are insights into the contribution of the large extracellular domain.   The parallel roles of Gi2 and Gq/11 and downstream effectors remain to be understood. Thorough analysis of latrophilin functions in vascular development and physiology remains to be done.  Lastly, combined human genetic and structure-function studies are needed to elucidate the contribution of Lphn2 variants to human cardiovascular disease. '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Experimental Procedures'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Antibodies'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Primary antibodies used in this study:'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}tbl' | ele.text=None\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Cell Culture'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Primary HUVECs were obtained from the Yale Vascular Biology and Therapeutics core facility.  Each batch is composed of cells pooled from three donors.  Cells were cultured in M199 (Gibco: 11150-059) supplemented with 20% FBS, 1x Penicillin-Streptomycin (Gibco: 15140-122), 60 µg/ml heparin (Sigma: H3393), and endothelial growth cell supplement (hereafter, complete medium).  HUVECs used for experiments were between passages 3 and 6.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Shear stress'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='For generating protein lysates for pulldown assays, HUVECs were seeded on tissue culture-treated plastic slides coated with 10 µg/ml fibronectin for 1 hour at 37˚C and then grown to confluence in HUVEC complete media.  For short-term stimulation with shear stress, cells were starved overnight in M199 medium with 2% FBS and 1:10 of ECGS or for 30 minutes in M199 medium containing 0.2 % BSA.  These slides were set in parallel flow chambers and shear stress applied as described(Frangos et al, 1988).'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='For imaging cells under shear stress, HUVECs were plated in HUVEC complete medium on glass bottom 6-well plates coated with 10 µg/ml fibronectin and shear stress applied on an orbital shaker at 150rpm for 24 hours, leading to uniaxial pulsatile shear in the peripheral region and multiaxial shear in the middle of the well, as previously described(Arshad et al, 2021).  Cells in the outer region, 0.7-0.9 cm from the center of each well were examined for Golgi orientation and endothelial alignment.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Image analysis'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Cell orientation was calculated by taking the masks of the cell nuclei determined by Hoechst images, fitting them as an ellipse, and determining the angle between flow direction and the major axis of the ellipse.  Analyzed results were visualized as histograms showing the percent of cells within each 10° of the direction of flow or as quantification of aligned cells with nuclei whose major axis were within 0-30 degrees to flow direction as indicated in the figure legends.  Eccentricity for cell alignment was calculated based on first eccentricity of ellipse following the equation below.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='(a: major axis, b:minor axis)'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Junctional linearization index was calculated as ratio of perimeter of the cell to perimeter of its convex hull following the equation below:'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Data display'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Quantified data are displayed as means ± standard error (SEM) in which the indicated replicates are independent experiments.  F test was conducted to check equal variance.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Lentiviral transduction'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Lenti-X 293T cells (Clontech, 632180) were cultured for at least 24 hours in DMEM supplemented with 10% FBS and lacking antibiotics, then transfected with lentiviral plasmids encoding the gene of interest and packaging plasmids (Addgene: 12259 and 12260) using Lipofectamine 2000 (Thermo Fisher Scientific: 11668-019) following the manufacturer’s protocols with Opti-MEM medium.  Conditioned media from these cultures were collected 48 hours later, sterilized through 0.22µm filters and added to HUVECs together with 8µg/ml of polybrene (Sigma: 107689).  After 24 hours, cells were switched to complete medium for 48 hours.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='siRNA transfection'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='HUVECs were cultured in EGMTM-2 Endothelial Cell Growth Medium-2 BulletKitTM'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=' (Lonza: CC-3156 and CC-4176) for 24 hours before transfecting with RNAiMax (Thermo Fisher Scientific: 13778-150) with 20nM siRNA in Opti-MEM (Gibco: 31985-070) according to the manufacturer’s instructions.  After 6 hours, cells were switched to EGM-2 medium and used for experiments 2-3 days later.  Gα protein siRNAs were custom designed based on previous publications(Grzelinski et al, 2010; Krumins & Gilman, 2006; Ngai et al, 2008).  For latrophilin knockdown, ON-TARGET plus Smartpool siRNAs from Dharmacon against human LPHN1 (L-005650-00-0005), LPHN2 (L-005651-00-0005), LPHN3 (L-005652-00-0005) were used.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Western Blotting'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='HUVECs were washed with PBS and extracted in Laemmli sample buffer.  Samples were separated by SDS-PAGE and transferred onto nitrocellulose membranes.  Membranes were blocked with 5% milk in TBS-T and probed with primary antibodies at 4˚C for overnight.  The targeting proteins were visualized by HRP-conjugated secondary antibodies and subsequent HRP-luminol reaction.  '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Immunofluorescence'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='HUVECs were washed with PBS, fixed for 10 minutes with 3.7% PFA in PBS.  Following fixation, cells were permeabilized with 0.5% Triton X-100 in PBS for 10 minutes and then incubated with 3% BSA in PBS for 30 minutes for blocking.   Cells were washed with PBS after blocking and were incubated Alexa488-Phalloidin and Hoechst for 1 hour, then washed 4 times with PBS and mounted.  Images were captured with 20x or 60x objective on a PerkinElmer spinning disk confocal microscope or a Leica Sp8 confocal microscope with the Leica Application Suite (LAS) software. Cell alignment was determined as described previously(Baeyens et al, 2014).\\xa0'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Reverse transcription and quantitative PCR'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='RNA was isolated from HUVECs using RNeasy kit according to the manufacturer’s instructions and quantified using a nanodrop spectrophotometer.    Following cDNA synthesis using Bio-Rad iScript kit, RT-PCR was performed as follows.  Each PCR reaction contains 42 two-step amplification cycles consisting of: (a) denaturation at 95˚C for 30s, and (b) annealing and extension at 60˚C for 30s.  The amplification curve was collected, and the relative transcript level of the target mRNA in each sample was calculated by normalization of Ct values to the reference mRNA (GAPDH).  Primer sequences used for RT-PCR were as shown in Table 1.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Gα protein pulldown assay'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='BL21 E.coli cells were transformed with constructs expressing GST-tagged GINIP protein or GST-tagged GRK2 N-terminal domain.  Cells were incubated in terrific broth (http://cshprotocols.cshlp.org/content/2015/9/pdb.rec085894.full) and protein expression induced by addition of 0.5µg/ml IPTG.  Cells were collected after 8 hours, lysed and GST- proteins collected on Glutathione-conjugated beads. Beads were washed 4 times, eluted with 50mM glutathione and proteins desalted on a gel filtration column. Aliquots of 20 µg aliquots were stored frozen and thawed shortly before use.  '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='For pulldown assays, HUVECs infected with lentivirus encoding GluGlu-tagged Gi mutant or lentivirus encoding both GluGlu-tagged Gq mutant and Ric8A were lysed in cell lysis buffer composed of 10mM Tris-HCl pH7.5, 150mM NaCl, 1% Triton X-100, 5mM DTT and additional 2µg/ml of GST-GINIP or GST-GRK2N, and the lysates were incubated for 10 minutes at 4˚C with gentle agitation with glutathione beads.  Beads were washed three times with cell lysis buffer, isolated proteins were solubilized in SDS sample laemmli buffer, and were analyzed by western blotting.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Mass spectrometry screening for Gα binding receptors'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='To create a Gα protein variant that interacts more strongly with GPCR, a green fluorescent protein (GFP) tag was inserted for affinity purification, alongside a mutation of four alanine residues into helix α5 (ins4A), as this hinders GTP binding and maintains the association with GPCRs even in the presence of GTP.  Additionally, for the extraction of GPCRs while preserving their configuration and lipid environment, a non-detergent nanodisc approach utilizing styrene-maleimide anhydride (SMA) copolymers was employed according to protocols described previously.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='In the coimmunoprecipitation process, GFP-Trap® nanobody beads were utilized to capture the interactions between GFP-tagged Gα protein variants and activated designer receptors exclusively activated by designer drugs (DREADDs). The resulting complexes were subjected to subsequent proteomic analysis to determine which GPCRs interacted preferentially with the Gi1(Q306K) variant following induction of fluid shear stress. Identified receptors were further analyzed by western blotting as described in the supplementary material.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Measurement of Golgi orientation'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Following exposure to laminar shear stress, HUVECs were washed with PBS, fixed for 10 minutes with 3.7% PFA in PBS, permeabilized with 0.5% Triton X-100 in PBS for 10 minutes, blocked with 3% BSA in PBS for 30 minutes and then incubated with GM130 antibody (1:500, BD bioscience) at 4˚C for overnight. After washing, cells were incubated with Hoechst and Alexa647-conjugated secondary antibody at RT for 1 hour to visualize the nuclei and Golgi apparatus, respectively.  Images were captured with 20x objective on a Leica Sp8 confocal microscope with the Leica Application Suite (LAS) software.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Golgi polarization was calculated as a vector that connects the center of mass of the nucleus to the center of mass of the Golgi. The angle between this vector and the direction of flow was then determined, with 0° defined as against the flow. Oriana V4 software was used to calculate the angle mean and circular SD and to generate a rose plot for data visualization.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='In vitro permeability assay using the FITC-streptavidin/biotinylated-fibronectin'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Slides/dishes were\\xa0coated with biotinylated-Fibronectin at\\xa00.25\\xa0mg/mL\\xa0overnight\\xa0at 4°C. HUVECs were seeded of 2×106 cells per well on glass-bottom 6-well plates. Cells\\xa0were\\xa0subjected\\xa0to\\xa0laminar shear stress by orbital shaking at 150 rpm for 24 hours.\\xa0 Immediately\\xa0after\\xa0cessation\\xa0of\\xa0flow, FITC-Streptavidin (1:1000, Invitrogen) was added to the cells for 1 minute before\\xa0 the\\xa0 fixation\\xa0 by\\xa0 the\\xa0 addition\\xa0 of\\xa0 3.7%\\xa0 (v/v)\\xa0 paraformaldehyde. Following\\xa0\\xa0 subsequent\\xa0\\xa0 washing\\xa0\\xa0 and\\xa0\\xa0 blocking, Alexa Fluor 488 Phalloidin (1:200; Invitrogen) was\\xa0 added\\xa0 to\\xa0 stain\\xa0 F-actin\\xa0 filaments. Images were captured from 5 random fields and FITC-streptavidin staining quantified using ImageJ.\\xa0'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='In vitro endothelial 3D invasion assay under shear stress'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='In order to equalize the VEGF concentration in the collagen gel and endothelial culture medium, VEGF-189 (R&D systems, 8147-VE), a matrix-binding isoform of VEGF, was applied to the bottom of each well at a concentration of 50 ng/ml prior to adding collagen matrices. Rat tail collagen type I (Thermofisher scientific, A1048301) was employed to prepare collagen matrices at the concentration of 1.5 mg/ml with additional NaOH to facilitate the collagen matrix assembly by neutral pH. After thorough mixing, 1 ml of collagen I solution was placed in each well of a 12-well plate (Corning #3513) and polymerized at 37°C and 5% CO2 for 2 hours to overnight. '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='To prepare endothelial cells, confluent HUVEC culture plates were washed with PBS, trypsinized, and counted.  After pelleting, cell pellets were resuspended in EGM-2 medium (Lonza, #CC-3162) free of VEGF and heparin to establish a positive gradient of VEGF from cells to the bottom of the collagen gel, and 300,000 cells per well were placed on the collagen gel and incubated for 24 hours. Following incubation, plates were placed on an orbital shaker and ECs were exposed to orbital shaking at 150 rpm for 48 hours as mentioned above in shear stress section. Cells were then fixed in 4% PFA and permeabilized in PBS containing 0.5% Triton X-100. Nonspecific binding was blocked by incubation with PBS containing 3% BSA. Thereafter, cells were subjected to Alexa488-conjugated phalloidin (1:400 dilution; Thermofisher scientific: A12379) and Hoechst (1:1000 dilution; Invitrogen: H1399) for 2 hours to stain actin cytoskeleton and cell nuclei, respectively. All images were captured with the Nikon Sp8 confocal microscope and segregated into peripheral region (with uniaxial pulsatile shear) vs. static controls.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Zebrafish husbandry and handling'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Zebrafish were housed and maintained at 28.5ºC in accordance with standard methods approved by the Yale University Institutional Animal Care and Use Committee (#2017-11473)(Lawrence, 2016).'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Generation of LPHN/adgrl knockdown zebrafish using CRISPR/Cas9 ribonucleoproteins'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Four zebrafish latrophilin paralogs were identified—adgrl1a, adgrl2a, adgrl2b.1, and adgrl3.1. Fused sgRNAs were generated targeting each paralog (see Table 2) by annealing locus-specific oligonucleotides to a common 5’ universal oligonucleotide and performing in vitro RNA transcription (AmpliScribe T7-Flash Kit, Lucigen, ASF3507)(Gagnon et al, 2014). An injection mix of 50 ng/µL each sgRNA, an equivalent concentration of Cas9 protein (TrueCut Cas9, Invitrogen, A36497), and either 1 µg/µL silent heart/tnnt2a(Sehnert et al., 2002) or standard control morpholino (GeneTools) was prepared. 1 nL of the mixture was injected into Tg(kdrl:ras-mCherry, fli1a:nls-GFP) zebrafish at the one cell stage (Parker Hannifin, Picospritzer III). CRISPR reagent efficacy was confirmed by T7 endonuclease assay (see Figure S5).  Guides against adgrl2a and 2b.1 were combined to target all LPHN2 isoforms.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Immunostaining and imaging of ECs in the knockdown zebrafish'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='CRISPR/Cas9 RNP-injected embryos were either fixed at 48 hpf in 4% paraformaldehyde (Santa Cruz Biotechnology, SC-253236) and immunostained for GFP (1:300 chicken anti-GFP, abcam, ab13970), mCherry (1:300 rabbit anti-mCherry, abcam, ab167453), and ZO-1 (1:200 mouse anti-ZO-1, Invitrogen, 61-7300) with species-appropriate secondary antibodies (1:400 Invitrogen anti-chicken/-rabbit/-mouse Alexa 488/546/647, A-11039/A- 10040, A-31571) or imaged live. For fixed imaging, the larvae were processed as previously described(Sugden et al., 2017). For live imaging, larvae were anesthetized by immersion in 600 µM tricaine methylsulfonate (Western Chemical, TRS5), and embedded in 1.5% low-melt agarose (BioRad, 1613106). For staining in detail, larvae were fixed overnight in 4% PFA at +4ºC and then transitioned into 100% methanol via a series of dilutions. Once in methanol, the larvae were stored at -20ºC overnight, then rehydrated the next day via the same dilution series. Larvae were permeabilized using incubation in acetone for 15 minutes at -20ºC and then by incubation in 20 µg/mL proteinase K for 20 minutes at 37ºC. Larvae were refined in 4% PFA for 15 minutes at room temperature and then incubated in blocking buffer (0.5% Triton X-100, 10% goat serum, 1% BSA, 0.01% sodium azide, and 1% DMSO in PBS) for 2 hours at room temperature. Larvae were incubated in primary antibody (1:400 chicken anti-GFP, abcam ab13970) overnight in blocking buffer at +4ºC. Larvae were thoroughly washed in PBS-Tx and incubated overnight at +4ºC in secondary antibody (1:400 goat anti-chicken Alexa488 A-11039, ThermoFisher) in blocking buffer. Following final washes in PBS-Tx, larvae were mounted and imaged. Intersegmental vessel and dorsal aorta diameter were then calculated using ImageJ. All imaging was performed using a Leica SP8 confocal microscope.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Immunostaining and imaging of mouse aorta'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Aortae were perfusion-fixed in situ with 4% paraformaldehyde (overnight) prior to staining with primary beta-catenin antibody (Cell signaling: 9562) and fluorophore-conjugated secondary antibody (Invitrogen, donkey anti-Rabbit IgG (H+L) Highly Cross-Adsorbed Secondary Antibody, Alexa Fluor 647, A31573) in in Claudio buffer (1% FBS, 3% BSA, 0.5% Triton X-100, 0.01% Sodium deoxycholate, 0.02% sodium azide in PBS pH7.4)(Franco et al, 2013).  Cells were stained with Hoechst (Invitrogen: H1399) to label nuclei. Images of stained vessels were acquired using a Leica SP8 confocal microscope with the Leica Application Suite (LAS) software. Endothelial cell alignment in aorta was quantified by measuring the eccentricity of the immunostaining of beta-catenin of the endothelial cells (>150 cells per field of view). '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Extravasation of Evans blue dye for aortic vascular permeability'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='For aortic vascular permeability measurements, Evans blue dye (20 mg/ml in saline, 2.5 µl/g mouse body weight) (Sigma E2129) was injected retro-orbitally to LPHN2 endothelial cell-specific knockout and their control littermates at 8 weeks old.  Thirty minutes post injection, animals were euthanized by an overdose of isoflurane and tissue samples including aorta were harvested and washed with PBS to remove extra fatty tissues.  Dissected aorta was prepared en face and imaged with an Olympus stereo zoom microscope. To quantify blue color associated with extravasation of Evans blue dye, the original RGB images were converted into HSV color model, and the proportion of the blue regions was calculated as the ratio of pixels with hue angles of 120°-240° to the entire aorta. '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Immunostaining and imaging of mouse muscle capillaries'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Rectus femoris muscle from murine thigh were perfusion-fixed with 4% paraformaldehyde (overnight) and the middle part of the muscle were cryo-sectioned following embedding into OCT compound and stained with primary antibodies for SMA (fluorophore-conjugated) and for CD31 followed by fluorophore-conjugated secondary antibody (Invitrogen, donkey anti-Rabbit IgG (H+L) Highly Cross-Adsorbed Secondary Antibody, Alexa Fluor 647, A31573) in PBS containing 5% BSA.  Hoechst (Invitrogen: H1399) was used to nuclear identification. Images of stained vessels were acquired using a Leica SP8 confocal microscope with the Leica Application Suite (LAS) software. '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Hindlimb ischemia model'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Mice were anesthetized and operated with a mixture of 1.5-2% isoflurane. The front hair of the right thigh was removed with Nair lotion and the local surgical area was disinfected. A 5 mm longitudinal skin incision was made in the right thigh. The distal femoral artery, proximal to the popliteal artery and saphenous artery, was explored and dissected. 6-0 silk sutures (Syneture) were double used for the ligation before excision of the arterial vessel bed between the distal end of the superficial epigastric artery and the trifurcation of the femoral artery into the descending genicular, popliteal, and saphenous branches. The ligated artery was cut between two sutures. The venous structures and accompanying peripheral nerves were kept intact. The overlying skin was closed with a 6-0 prolene (Ethicon) and disinfected. At the end of the procedure, Buprenorphine was administered subcutaneously at 0.05 mg/kg, once pre-emptively, then every 12 h for 72 h. The survival rate for this surgery was 100%.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='\\xa0'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Laser Doppler perfusion imaging'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Hindlimb perfusion was assessed noninvasively in the plantar foot before, 1, 3, 7, 14, and 21 days after hindlimb ischemia by scanning laser-Doppler [model LDI2-IR modified for high resolution; Moor Instruments, Wilmington, DE]. The hindquarters were placed on the top of a heater pad during scanning to minimize variation in body temperature. Doppler perfusion of the plantar foot was assessed within anatomically defined regions of interest (ROIs; Moor Instruments). The ROI for the plantar foot consisted of the hind paw margins. Procedures for minimal preparation, scanning, and ROI selections have been described in detail previously(Zhuang et al, 2011). Low or no perfusion is displayed as dark blue, whereas the highest degree of perfusion is displayed as red. These images were quantitatively converted into histograms that represented the amount of blood flow on the x-axis and the number of pixels on the y-axis in the traced area. The average blood flow in each histogram was calculated and the LDI index was determined as the ratio of ischemic to non-ischemic hindlimb blood perfusion.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='\\xa0'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Volumetric micro-CT angiogram for quantitative arteriogenesis'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='After the mice were euthanized, the vasculature was flushed with 0.9% normal saline containing heparin (1000 IU/L) for 3 minutes, followed by 4% paraformaldehyde for 5 minutes at 100 mm Hg pressure. Fresh home-made 20% bismuth nanoparticles mixed in 5% gelatin were used as a micro-CT contrast agent and injected over 2 minutes with a syringe pump, as described before(Tirziu et al., 2005). The mice were then immediately chilled on ice for more than 30 min and immersion-fixed in 2% paraformaldehyde overnight. The peripheral vasculature in the ischemic hindlimb and the contralateral normal hindlimb was imaged with a high-resolution micro-CT imaging system (GE eXplore Locus SP), set to a 0.007-mm effective detector pixel size. Microview software (GE Healthcare) was used for initial reconstruction. Advanced workstation with various software was used for 3D reconstruction and segmentation of the whole hindlimb at the 21 µm spatial resolution. Using the knee as the breakpoint, we re-sliced the 3D vasculature into 200 slices both in the thigh and in the calf regions. Quantification was performed by use of a NIH Image. The data were expressed as vascular segment numbers, representing the total number of vessels, of specified diameter, counted in 200\\xa0z-sections for the thigh region and another 200\\xa0z-sections for the calf region.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Mouse treadmill fatigue test '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Mice were first acclimatized by staying on the motionless treadmill machine for 30 min on day 1 and then trained on the treadmill moving at 12 m/min for 30 min on day 2. Fatigue tests were then conducted on days 3 and 4. \\xa0Mice were placed on the treadmill and the shock rod was activated. \\xa0Treadmill speeds were increased as follows: 0 m/min, for 12 min to measure basal metabolism, 12 m/min, for1 min, then increasing at a rate of +1 m/min each 1 min until exhaustion. \\xa0Exhaustion (endpoint of treadmill cessation) was defined as the point at which mice continuously contacted with the shock rod for 5 continuous seconds or when mice stopped running 3 times within 5 seconds.  Before each testing session, Oxymax software (Columbus Instruments, Columbus, OH, USA) and open circuit indirect calorimetry treadmills (Metabolic Modular Treadmill, Columbus Instruments, Columbus, OH) were calibrated and checked for hardware malfunctions according to manufacturer instructions. '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Data availability'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='The datasets produced in this study are available in the following databases:'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='All data: BioStudies S-BIAD928: https://www.ebi.ac.uk/biostudies/studies?query=S-BIAD928'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Acknowledgements'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='This work was supported by NIH grants RO1 HL75092 and R01 HL151469 to MAS, NHLBI 5R01HL130246-05 to SN, and the Uehara memorial foundation postdoctoral fellowship and MEXT | Japan Society for the Promotion of Science (JSPS) Overseas Research Fellowships (2014, receipt number: 805) to KT.  We thank Hamit Harun Dag (Istanbul University) for help of synthesizing SMA polymer; Shahid Mansuri (Yale University), Jeremy Herrera (University of Manchester) and David Knight (University of Manchester) for expert advice for mass spectrometry experiments; Didier Trono (École polytechnique fédérale de Lausanne) for lentiviral packaging plasmids pMD2.G (Addgene plasmid # 12259 ; http://n2t.net/addgene:12259 ; RRID:Addgene_12259) and psPAX2 (Addgene plasmid # 12260 ; http://n2t.net/addgene:12260 ; RRID:Addgene_12260);  Bryan Roth (University of North Carolina) for pcDNA5/FRT-HA-hM3D(Gq) (Addgene plasmid # 45547 ; http://n2t.net/addgene:45547 ; RRID:Addgene_45547) and pcDNA5/FRT-HA-hM4D(Gi)(Addgene plasmid # 45548 ; http://n2t.net/addgene:45548 ; RRID:Addgene_45548); Aziz Moqrich (Institut de Biologie du Dévelopement de Marseille) for GST-GINIP plasmid.  We are grateful to Peter Newman (Medical College of Wisconsin) for the kind gift of PECAM-1 antibody and helpful discussions.  Schematic models are inspired from BioRender templates (2022).'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Disclosure & competing interests statement'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='The authors declare they have no conflict of interest.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='References'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Albarran-Juarez J, Iring A, Wang S, Joseph S, Grimm M, Strilic B, Wettschureck N, Althoff TF, Offermanns S (2018) Piezo1 and G(q)/G(11) promote endothelial inflammation depending on flow pattern and integrin activation. J Exp Med 215: 2655-2672'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Arshad M, Ghim M, Mohamied Y, Sherwin SJ, Weinberg PD (2021) Endothelial cells do not align with the mean wall shear stress vector. J R Soc Interface 18: 20200772'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Bae H, Cabrera-Vera TM, Depree KM, Graber SG, Hamm HE (1999) Two amino acids within the alpha4 helix of Galphai1 mediate coupling with 5-hydroxytryptamine1B receptors. J Biol Chem 274: 14963-14971'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Baeyens N, Bandyopadhyay C, Coon BG, Yun S, Schwartz MA (2016) Endothelial fluid shear stress sensing in vascular health and disease. J Clin Invest 126: 821-828'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Baeyens N, Mulligan-Kehoe MJ, Corti F, Simon DD, Ross TD, Rhodes JM, Wang TZ, Mejean CO, Simons M, Humphrey J et al (2014) Syndecan 4 is required for endothelial alignment in flow and atheroprotective signaling. Proc Natl Acad Sci U S A 111: 17308-17313'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Butler PJ, Norwich G, Weinbaum S, Chien S (2001) Shear stress induces a time- and position-dependent increase in endothelial cell membrane fluidity. Am J Physiol Cell Physiol 280: C962-969'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Camillo C, Facchinello N, Villari G, Mana G, Gioelli N, Sandri C, Astone M, Tortarolo D, Clapero F, Gays D et al (2021) LPHN2 inhibits vascular permeability by differential control of endothelial cell adhesion. J Cell Biol 220'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Campinho P, Vilfan A, Vermot J (2020) Blood Flow Forces in Shaping the Vascular System: A Focus on Endothelial Cell Behavior. Front Physiol 11: 552'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Chachisvilis M, Zhang YL, Frangos JA (2006) G protein-coupled receptors sense fluid shear stress in endothelial cells. Proc Natl Acad Sci U S A 103: 15463-15468'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=\"Collins C, Guilluy C, Welch C, O'Brien ET, Hahn K, Superfine R, Burridge K, Tzima E (2012) Localized tensional forces on PECAM-1 elicit a global mechanotransduction response via the integrin-RhoA pathway. Curr Biol 22: 2087-2094\"\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Conway D, Schwartz MA (2012) Lessons from the endothelial junctional mechanosensory complex. F1000 Biol Rep 4: 1'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Conway DE, Breckenridge MT, Hinde E, Gratton E, Chen CS, Schwartz MA (2013) Fluid shear stress on endothelial cells modulates mechanical tension across VE-cadherin and PECAM-1. Curr Biol 23: 1024-1030'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Cornella N, Sancho J, Sitges-Serra A (2017) Short and Long-Term Outcomes After Surgical Procedures Lasting for More Than Six Hours. Sci Rep 7: 9221'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Dela Paz NG, Frangos JA (2019) Rapid flow-induced activation of Galphaq/11 is independent of Piezo1 activation. Am J Physiol Cell Physiol 316: C741-C752'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Dela Paz NG, Melchior B, Frangos JA (2017) Shear stress induces Galpha(q/11) activation independently of G protein-coupled receptor activation in endothelial cells. Am J Physiol Cell Physiol 312: C428-C437'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='dela Paz NG, Melchior B, Shayo FY, Frangos JA (2014) Heparan sulfates mediate the interaction between platelet endothelial cell adhesion molecule-1 (PECAM-1) and the Galphaq/11 subunits of heterotrimeric G proteins. J Biol Chem 289: 7413-7424'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Erdogmus S, Storch U, Danner L, Becker J, Winter M, Ziegler N, Wirth A, Offermanns S, Hoffmann C, Gudermann T et al (2019) Helix 8 is the essential structural motif of mechanosensitive GPCRs. Nat Commun 10: 5784'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Fancher IS, Ahn SJ, Adamos C, Osborn C, Oh MJ, Fang Y, Reardon CA, Getz GS, Phillips SA, Levitan I (2018) Hypercholesterolemia-Induced Loss of Flow-Induced Vasodilation and Lesion Formation in Apolipoprotein E-Deficient Mice Critically Depend on Inwardly Rectifying K(+) Channels. J Am Heart Assoc 7'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Flock T, Hauser AS, Lund N, Gloriam DE, Balaji S, Babu MM (2017) Selectivity determinants of GPCR-G-protein binding. Nature 545: 317-322'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Franco CA, Blanc J, Parlakian A, Blanco R, Aspalter IM, Kazakova N, Diguet N, Mylonas E, Gao-Li J, Vaahtokari A et al (2013) SRF selectively controls tip cell invasive behavior in angiogenesis. Development 140: 2321-2333'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Frangos JA, McIntire LV, Eskin SG (1988) Shear stress induced stimulation of mammalian cell metabolism. Biotechnol Bioeng 32: 1053-1060'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Gagnon JA, Valen E, Thyme SB, Huang P, Akhmetova L, Pauli A, Montague TG, Zimmerman S, Richter C, Schier AF (2014) Efficient mutagenesis by Cas9 protein-mediated oligonucleotide insertion and large-scale assessment of single-guide RNAs. PLoS One 9: e98186'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Gaillard S, Lo Re L, Mantilleri A, Hepp R, Urien L, Malapert P, Alonso S, Deage M, Kambrun C, Landry M et al (2014) GINIP, a Galphai-interacting protein, functions as a key modulator of peripheral GABAB receptor-mediated analgesia. Neuron 84: 123-136'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Galie PA, Nguyen DH, Choi CK, Cohen DM, Janmey PA, Chen CS (2014) Fluid shear stress threshold regulates angiogenic sprouting. Proc Natl Acad Sci U S A 111: 7968-7973'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Givens C, Tzima E (2016) Endothelial Mechanosignaling: Does One Sensor Fit All? Antioxid Redox Signal 25: 373-388'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Grzelinski M, Pinkenburg O, Buch T, Gold M, Stohr S, Kalwa H, Gudermann T, Aigner A (2010) Critical role of G(alpha)12 and G(alpha)13 for human small cell lung cancer cell proliferation in vitro and tumor growth in vivo. Clin Cancer Res 16: 1402-1415'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Gudi S, Nolan JP, Frangos JA (1998) Modulation of GTPase activity of G proteins by fluid shear stress and phospholipid composition. Proc Natl Acad Sci U S A 95: 2515-2519'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Gudi SR, Clark CB, Frangos JA (1996) Fluid flow rapidly activates G proteins in human endothelial cells. Involvement of G proteins in mechanochemical signal transduction. Circ Res 79: 834-839'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=\"Haidekker MA, L'Heureux N, Frangos JA (2000) Fluid shear stress increases membrane fluidity in endothelial cells: a study with DCVJ fluorescence. Am J Physiol Heart Circ Physiol 278: H1401-1406\"\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Humphrey JD, Schwartz MA (2021) Vascular Mechanobiology: Homeostasis, Adaptation, and Disease. Annu Rev Biomed Eng 23: 1-27'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Jung B, Obinata H, Galvani S, Mendelson K, Ding BS, Skoura A, Kinzel B, Brinkmann V, Rafii S, Evans T et al (2012) Flow-regulated endothelial S1P receptor-1 signaling sustains vascular development. Dev Cell 23: 600-610'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Kaya AI, Lokits AD, Gilbert JA, Iverson TM, Meiler J, Hamm HE (2016) A Conserved Hydrophobic Core in Galphai1 Regulates G Protein Activation and Release from Activated Receptor. J Biol Chem 291: 19674-19686'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Kroon J, Heemskerk N, Kalsbeek MJT, de Waard V, van Rijssel J, van Buul JD (2017) Flow-induced endothelial cell alignment requires the RhoGEF Trio as a scaffold protein to polarize active Rac1 distribution. Mol Biol Cell 28: 1745-1753'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Krumins AM, Gilman AG (2006) Targeted knockdown of G protein subunits selectively prevents receptor-mediated modulation of effectors and reveals complex changes in non-targeted signaling proteins. J Biol Chem 281: 10250-10262'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Lawrence C (2016) New frontiers for zebrafish management. Methods Cell Biol 135: 483-508'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Lee MJ, Evans M, Hla T (1996) The inducible G protein-coupled receptor edg-1 signals via the G(i)/mitogen-activated protein kinase pathway. J Biol Chem 271: 11272-11279'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Lee SC, Knowles TJ, Postis VL, Jamshad M, Parslow RA, Lin YP, Goldman A, Sridhar P, Overduin M, Muench SP et al (2016) A method for detergent-free isolation of membrane proteins in their local lipid environment. Nat Protoc 11: 1149-1162'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=\"Liu SL, Sheng R, Jung JH, Wang L, Stec E, O'Connor MJ, Song S, Bikkavilli RK, Winn RA, Lee D et al (2017) Orthogonal lipid sensors identify transbilayer asymmetry of plasma membrane cholesterol. Nat Chem Biol 13: 268-274\"\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Lucitti JL, Jones EA, Huang C, Chen J, Fraser SE, Dickinson ME (2007) Vascular remodeling of the mouse yolk sac requires hemodynamic force. Development 134: 3317-3326'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Maleszewska M, Vanchin B, Harmsen MC, Krenning G (2016) The decrease in histone methyltransferase EZH2 in response to fluid shear stress alters endothelial gene expression and promotes quiescence. Angiogenesis 19: 9-24'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Medina R, Grishina G, Meloni EG, Muth TR, Berlot CH (1996) Localization of the effector-specifying regions of Gi2alpha and Gqalpha. J Biol Chem 271: 24720-24727'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Nazarko O, Kibrom A, Winkler J, Leon K, Stoveken H, Salzman G, Merdas K, Lu Y, Narkhede P, Tall G et al (2018) A Comprehensive Mutagenesis Screen of the Adhesion GPCR Latrophilin-1/ADGRL1. iScience 3: 264-278'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Ngai J, Methi T, Andressen KW, Levy FO, Torgersen KM, Vang T, Wettschureck N, Tasken K (2008) The heterotrimeric G-protein alpha-subunit Galphaq regulates TCR-mediated immune responses through an Lck-dependent pathway. Eur J Immunol 38: 3208-3218'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Papasergi MM, Patel BR, Tall GG (2015) The G protein alpha chaperone Ric-8 as a potential therapeutic target. Mol Pharmacol 87: 52-63'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Privratsky JR, Newman DK, Newman PJ (2010) PECAM-1: conflicts of interest in inflammation. Life Sci 87: 69-82'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Sando R, Jiang X, Sudhof TC (2019) Latrophilin GPCRs direct synapse specificity by coincident binding of FLRTs and teneurins. Science 363'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Sehnert AJ, Huq A, Weinstein BM, Walker C, Fishman M, Stainier DY (2002) Cardiac troponin T is essential in sarcomere assembly and cardiac contractility. Nat Genet 31: 106-110'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Strohbach A, Pennewitz M, Glaubitz M, Palankar R, Gross S, Lorenz F, Materzok I, Rong A, Busch MC, Felix SB et al (2018) The apelin receptor influences biomechanical and morphological properties of endothelial cells. J Cell Physiol 233: 6250-6261'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Sugden WW, Meissner R, Aegerter-Wilmsen T, Tsaryk R, Leonard EV, Bussmann J, Hamm MJ, Herzog W, Jin Y, Jakobsson L et al (2017) Endoglin controls blood vessel diameter through endothelial cell shape changes in response to haemodynamic cues. Nat Cell Biol 19: 653-665'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Tesmer VM, Kawano T, Shankaranarayanan A, Kozasa T, Tesmer JJ (2005) Snapshot of activated G proteins at the membrane: the Galphaq-GRK2-Gbetagamma complex. Science 310: 1686-1690'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Tirziu D, Moodie KL, Zhuang ZW, Singer K, Helisch A, Dunn JF, Li W, Singh J, Simons M (2005) Delayed arteriogenesis in hypercholesterolemic mice. Circulation 112: 2501-2509'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Tzima E, Irani-Tehrani M, Kiosses WB, Dejana E, Schultz DA, Engelhardt B, Cao G, DeLisser H, Schwartz MA (2005) A mechanosensory complex that mediates the endothelial cell response to fluid shear stress. Nature 437: 426-431'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Vizurraga A, Adhikari R, Yeung J, Yu M, Tall GG (2020) Mechanisms of adhesion G protein-coupled receptor activation. J Biol Chem 295: 14065-14083'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Vogel J, Niederer D, Jung G, Troidl K (2020) Exercise-Induced Vascular Adaptations under Artificially Versus Pathologically Reduced Blood Flow: A Focus Review with Special Emphasis on Arteriogenesis. Cells 9'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Wang S, Iring A, Strilic B, Albarran Juarez J, Kaur H, Troidl K, Tonack S, Burbiel JC, Muller CE, Fleming I et al (2015) P2Y(2) and Gq/G(1)(1) control blood pressure by mediating endothelial mechanotransduction. J Clin Invest 125: 3077-3086'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Wilson PT, Bourne HR (1995) Fatty acylation of alpha z. Effects of palmitoylation and myristoylation on alpha z signaling. J Biol Chem 270: 9667-9675'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Xu J, Mathur J, Vessieres E, Hammack S, Nonomura K, Favre J, Grimaud L, Petrus M, Francisco A, Li J et al (2018) GPR68 Senses Flow and Is Essential for Vascular Physiology. Cell 173: 762-775 e716'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Yamamoto K, Ando J (2013) Endothelial cell and model membranes respond to shear stress by rapidly decreasing the order of their lipid phases. J Cell Sci 126: 1227-1234'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Yamamoto K, Ando J (2015) Vascular endothelial cell membranes differentiate between stretch and shear stress through transitions in their lipid phases. Am J Physiol Heart Circ Physiol 309: H1178-1185'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Yamamoto K, Nogimori Y, Imamura H, Ando J (2020) Shear stress activates mitochondrial oxidative phosphorylation by reducing plasma membrane cholesterol in vascular endothelial cells. Proc Natl Acad Sci U S A 117: 33660-33667'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Yeh JC, Otte LA, Frangos JA (2008) Regulation of G protein-coupled receptor activities by the platelet-endothelial cell adhesion molecule, PECAM-1. Biochemistry 47: 9029-9039'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Zhou J, Li YS, Chien S (2014) Shear stress-initiated signaling and its regulation of endothelial function. Arterioscler Thromb Vasc Biol 34: 2191-2198'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Zhuang ZW, Shi J, Rhodes JM, Tsapakos MJ, Simons M (2011) Challenging the surgical rodent hindlimb ischemia model with the miniinterventional technique. J Vasc Interv Radiol 22: 1437-1446'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Figure 1. Gα proteins specific for endothelial flow responses'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='A, Flow-induced endothelial alignment after Gα knockdown.  HUVECs were subjected to fluid shear stress (FSS) at a rate of 12 dynes/cm2 for 16 hours and nuclear orientation quantified as histograms showing the percentage of cells within each 10° of the flow directions from 0° to 90° (see Methods) ****: p<0.0001; one-way ANOVA with Tukey’s multiple comparisons test. B, Src family kinase activation, quantified in C. n=5 for control, Gi knockdown, Gq11 knockdown and n=3 for simultaneous knockdown of Gi and Gq11.  Values are means ± SEM. ****: p<0.0001, ***: p<0.001; one-way ANOVA with Tukey multiple comparison test.  D, Rescue of Gq/11 and Gi knockdown by re-expression of siRNA-resistant versions of the indicated proteins. ****: p<0.0001; one-way ANOVA with Tukey’s multiple comparison test. E, Amino acid sequences of Gαi1, Gαi2, and Gαi3 at the mutation sites of Gαi1 gain-of-function mutant.  F, Rescue of Gq/11 and Gi knockdown with indicated Gα proteins. Each point corresponds to one measurement averaged from >500 cells. N=4. ****: p<0.0001; one-way ANOVA with Tukey’s multiple comparisons test.  G, GINIP pulldown assay for activation of Gαi2 by FSS.  N=3. Results quantified in H.  **: p=0.0185, Student’s t-test.  I, GRK2N pulldown assay for activation of Gq.  N=4, quantified in J. *: p<0.05, Student’s t-test. K, GINIP pulldown assay for FSS-induced activation of wild type and Q306K Gαi1.  N = 4, quantified in L. *: p=0.0304, ***: p=0.0017; Student’s t-test. M, Gi2 activation after PECAM-1 knockdown. HUVECs expressing GluGlu tagged Gi2 were transfected with scrambled siRNA or PECAM-1 siRNA, exposed to FSS, and Gi2 activation assayed as described above, quantified in N. Values are means ± SEM, normalized to input Gα protein levels. ***: p<0.001; Student’s t-test, N=4.  '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Figure 2. Identifiation of the upstream flow-responsive GPCR'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='A, Strategy for identification of GPCRs that bind the gain-of-function Gi1Q306K mutant but not the wild type Gi1.  B, Co-immunoprecipitation of Gi1Q306K(ins4A) with endogenous LPHN2 with and without FSS for 2 min. N=3, quantified in C. *: p=0.0374, Student’s t-test. D, Co-immunoprecipitation of LPHN2-GFP with the indicated Gα proteins containing internal GluGlu epitope tags.  E, Gi2 pulldown assay after LPHN2 knockdown.  N= 4. Results quantified in F.  ***: p<0.001, Student’s t-test.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Figure 3. Latrophilins regulate endothelial flow responses'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='A, HUVECs with or without latrophilin-2 knockdown were subjected to FSS for 16 h, then fixed and stained with Hoechst (nuclei), phalloidin (F-actin) and an antibody against VE-Cadherin. Scale bar: 100µm. B, Alignment of HUVECs (each bar = 10° increments) after knockdown of latrophilin isoforms was quantified as in Fig. 1 from >2000 cells/experiment, N=3.  ****: p<0.0001; one-way ANOVA with Tukey’s multiple comparisons test. C, Localization of LPHN2-mClover3. Scale bar: 100µm. N=6. D, Gi1(Q306K) pulldown from ECs ± FSS for 5 min or for 24 hours, probed for PECAM-1. N=3, quantified in E. F, Gi1(Q306K) pulldown from ECs ± FSS for 5 min or for 24 hours probed for VE-cadherin. N=3, quantified in G. H, Activation of Src family kinases and Akt by FSS in HUVECs depleted for latrophilin isoforms. N=3-4, quantified in I & J. *: p=0.0184, **: p<0.001; one-way ANOVA with Tukey’s multiple comparisons test. K, Alignment of LPHN2-depleted HUVECs rescued by re-expression of the indicated latrophilin isoforms. ****: p<0.0001; one-way ANOVA with Tukey’s multiple comparisons test. Quantification of >2000 cells for each condition, N=3. L, LPHN2 knockdown HUVECs were rescued by re-expression of the indicated LPHN2 mutants. ****: p<0.0001, one-way ANOVA with Tukey’s multiple comparisons test. Quantification of >500 cells for each condition.  '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Figure 4, Latrophilins in flow-mediated endothelial cell morphology in vivo'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='A. Representative image of HUVECs transfected with scrambled siRNA or LPHN2 siRNA.  Outlines of ECs from VE-cadherin staining are shown to next to each image. Scale Bar: 100µm. B, Quantification of junctional linearization index from A. n=30 for each condition.  **: p=0.0019, Statistics is done using two-way ANOVA with Sidak’s multiple comparison test. C, Dorsal aortas from 48hpf Tg(kdrl:ras-mCherry) WT or LPHN mutant zebrafish embryos stained for ZO-1. EC were outlined as in Methods. LPHN2 mutant indicates adgrl2a+adgrl2b.1 sgRNAs.  D&E, Quantification of cell size and eccentricity in embryos with latrophilin-2 CRISPR sgRNAs ± silent heart morpholinos (MO). N=6 per condition.  ****: p<0.0001, *: p=0.0267, **: p=0.0015, ***: p=0.0008; one-way ANOVA with Tukey’s multiple comparisons test.  F, ECs in the thoracic aorta in 18 weeks old mice at 10 weeks after tamoxifen injections, stained for β-catenin (left) and outlined (right).  Data are representative of 6 mice for each condition. Scale Bar: 100µm. G&H, quantification of cell eccentricity and junction linearity in mouse aorta.  N=4 per condition. **: p=0.0013; ****: p<0.0001; Student’s t-test. I, Aortas from mice 30 min after Evans blue dye injection, thresholded images on the right. Fractional blue area quantified in J. n=3 for each condition.  *: p=0.0419; Student’s t-test. K, in vitro endothelial permeability assay using FITC-streptavidin and biotin-conjugated fibronectin. Scale Bar: 100µm. L, Permeability was quantified by measuring FITC-streptavidin area per image field from K. n=5, *: p=0.0457, one-way ANOVA.  '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Figure 5. Cholesterol depletion activates Latrophilin-dependent signaling'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='A, ECs were treated with FSS or 5 mM methyl-β-cyclodextrin (mβCD) for 1 min then incubated with D4H-mClover3, rinsed and bound mClover3 quantified as described in Methods. N=27 for each condition.  ****: p<0.0001; One-way ANOVA with Tukey multiple comparisons test. B, Phalloidin staining following short acute treatment of methyl-β-cyclodextrin (mβCD).  Scale bar: 100µm. C, Co-immunoprecipitation of Gi1Q306K(ins4A) with endogenous Latrophilin-2 (LPHN2) after 5mM MβCD for 1min. D, Activation of Src family kinases and VEGFR2 after 5mM MβCD for 1min in HUVECs ± LPHN2 knockdown. Results quantified in E and F, respectively. Values are means ± SEM. N=5. *: p=0.0305, ***: p=0.0002; one-way ANOVA with Tukey multiple comparison test.  G, Model for junctional endothelial shear stress mechanotransduction. '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Figure 6. Latrophilin-2 in flow-induced vascular remodeling '\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='A, Representative images of ISVs from 48 hpf embryos with F(0)/mosaic depletions of latrophilin genes or controls.  B, Quantification of diameters of all ISVs in A. n=15 per condition. *: p=0.0452, **: p=0.0150; one-way ANOVA with Tukey’s multiple comparisons test. C, Quantification of diameters of arterial and venous ISVs at 72 hpf of zebrafish. n=6 per condition. **: p=0.0013; Student’s t-test. D, Ratio of blood flow in the right ischemic vs left control foot by laser doppler, quantified in E.  N=7-8 per condition. *#: p=0.0439, *: p=0.0135, **: p=0.0028, ##: p=0.0056, #: p=0.0163; Student’s t-test. F, Micro-CT of arterial vasculature after surgery.  Representative reconstructed micro-CT images from LPHN2 ECKO and WT littermates on day 22 after surgery, quantified in G. n=8 per condition. ****: p<0.0001, ***: p=0.0007, *: p=0.0167; two-way ANOVA with Tukey’s multiple comparisons test.  H, Rectus femoris muscle from thighs of 10 weeks old mice of LPHN2 ECKO or wild-type littermates, stained for CD31 and SMA to visualize the entire vasculature and arteries. Scale Bar: 100µm. I. Quantification of capillary density in the entire stitched thigh muscle images.  n=3 for control and n=6 for Lphn2 ECKO. ****: p<0.0001; Student’s t-test. J, Schematic of mouse treadmill fatigue test.  K, Maximum running during test. n=4 for control and n=5 for Lphn2 ECKO.  **:p=0.0082; Student’s t-test.  L, Running distance for each mouse.  n=4 for control and n=5 for Lphn2 ECKO.  **: p=0.0064; Student’s t-test. M, Oxygen consumption during treadmill fatigue test.  *: p<0.0001;  two-way ANOVA with Tukey’s multiple comparisons test.  Control: N=5, LPHN2 ECKO: N=4.  N, Representative 3D reconstructed images at 20° downward angle of ECs plated on top of collagen gels and sprouting under static or flow conditions.  ECs were stained with phalloidin; pseudocolors indicate distance from the gel surface as per the color scale on the right.  O, Histograms of cell area vs. distance from gel surface in N.  P, Mean sprout length.  N=4.  ***: p=0.0006; one-way ANOVA with Tukey’s multi comparison test.'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Table 1: Primer sequences used for quantitative PCR in this study'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}tbl' | ele.text=None\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text='Table 2: Sequences of sgRNAs and morpholinos used in this study'\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}tbl' | ele.text=None\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}p' | ele.text=''\n",
      "ele.tag='{http://schemas.openxmlformats.org/wordprocessingml/2006/main}sectPr' | ele.text=None\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Latrophilin-2 mediates fluid shear stress mechanotransduction at endothelial junctions\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Keiichiro Tanaka1,7,*, Minghao Chen1,7, Andrew Prendergast1, Zhenwu Zhuang1, Ali Nasiri2, Divyesh Joshi1, Jared Hintzen1, Minhwan Chung1, Abhishek Kumar1, Arya Mani1, Anthony Koleske3, Jason Crawford4, Stefania Nicoli1 and Martin A. Schwartz1,5,6,*\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "1 Yale Cardiovascular Research Center, Section of Cardiovascular Medicine, Department of Internal Medicine, School of Medicine, Yale University, New Haven, CT 06511, USA\n",
      "\n",
      "\n",
      "2 Department of Internal Medicine\n",
      "\n",
      "\n",
      "3 Department of Molecular Biochemistry and Biophysics\n",
      "\n",
      "\n",
      "4 Department of Chemistry\n",
      "\n",
      "\n",
      "5 Department of Cell Biology\n",
      "\n",
      "\n",
      "6 Department of Biomedical Engineering \n",
      "\n",
      "\n",
      "7 These authors contributed equally.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Running title: Latrophilins in shear stress sensing\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "*Authors for correspondence:\n",
      "\n",
      "\n",
      "keiichiro.tanaka@yale.edu; martin.schwartz@yale.edu;\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Abstract\n",
      "\n",
      "\n",
      "Endothelial cell responses to fluid shear stress from blood flow are crucial for vascular development, function and disease. A complex of PECAM-1, VE-cadherin, VEGF receptors (VEGFRs) and PlexinD1 located at cell-cell junctions mediates many of these events. But available evidence suggests that another mechanosensor upstream of PECAM-1 initiates signaling. Hypothesizing that GPCR and Gα proteins may serve this role, we performed siRNA screening of Gα subunits and found that Gαi2 and Gαq/11 are required for activation of the junctional complex. We then developed a new activation assay, which showed that these G proteins are activated by flow. We next mapped the Gα residues required for activation and developed an affinity purification method that used this information to identify latrophilin-2 (Lphn-2/ADGRL2) as the upstream GPCR.  Latrophilin-2 is required for all PECAM-1 downstream events tested. In both mice and zebrafish, latrophilin-2 is required for flow-dependent angiogenesis and artery remodeling. Furthermore, endothelial specific knockout demonstrates that latrophilin plays a role in flow-dependent artery remodeling.  Human genetic data reveal a correlation between the latrophilin-2-encoding Adgrl2 gene and cardiovascular disease. Together, these results define a pathway that connects latrophilin-dependent G protein activation to subsequent endothelial signaling, vascular physiology and disease. \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Keywords\n",
      "\n",
      "\n",
      "Latrophilin, G protein-coupled receptor, fluid shear stress, vascular development, PECAM-1\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Introduction\n",
      "\n",
      "\n",
      "Fluid shear stress (FSS) from blood flow is critical for early vascular development in mice(Lucitti et al, 2007) and zebrafish(Sugden et al, 2017), for homeostatic vessel remodeling in adults(Humphrey & Schwartz, 2021; Vogel et al, 2020), and in the initiation and progression of atherosclerosis(Baeyens et al, 2016; Zhou et al, 2014).  A complex of proteins at cell-cell junctions consisting of PECAM-1, VE-cadherin, VEGFR2 and 3, and Plexin D1 plays an important role in shear stress responses including vasodilation, flow-dependent vessel remodeling, and atherosclerosis(Conway & Schwartz, 2012; Givens & Tzima, 2016). Stimulation of this junctional complex by flow triggers activation of Src-family kinases (SFKs) within seconds, resulting in ligand-independent activation of VEGF receptors and downstream events including endothelial cell (EC) alignment in the direction of flow(Collins et al, 2012). However, the mechanisms by which proteins located at cell-cell junctions can sense forces from shear stress exerted on the apical surface is unclear. Previous work suggests that activation of the junctional complex is not primary but rather requires an upstream event, mediated presumably by another mechanosensor (Conway et al, 2013). \n",
      "\n",
      "\n",
      "GPCR/Gα protein signaling is an obvious candidate for initiating signaling on this time scale (seconds). GPCRs are a large family of cell surface receptors that play a crucial role in transducting signals across cell membranes.  GPCRs undergo conformational changes in response to ligand binding or other stimuli, after which the activated intracellular domain binds to a heterotrimeric Gα protein complex and catalyzes the exchange of guanosine diphosphate (GDP) for guanosine triphosphate (GTP). Connections between GPCRs and shear stress have been reported(Chachisvilis et al, 2006; Dela Paz & Frangos, 2019; Erdogmus et al, 2019; Jung et al, 2012; Wang et al, 2015; Xu et al, 2018). However, the literature on Gα proteins and GPCRs in shear stress signaling is highly inconsistent(Chachisvilis et al., 2006; Dela Paz & Frangos, 2019; Jung et al., 2012; Wang et al., 2015).  For instance, it has been demonstrated that flow applied to endothelial cells activates Gα proteins independent of an intervening GPCR(Dela Paz et al, 2017; Gudi et al, 1998).  It has also been reported that shear stress responses related to PECAM-1 and cell-cell junction require GPCRs but with different GPCRs implicated in different studies.  These studies include reports of ligand-independent activation of a bradykinin receptor(Chachisvilis et al., 2006; Yeh et al, 2008) and the sphingosine 1-phosphate receptor S1PR1(Jung et al., 2012).  The purinergic receptor P2Y2 was also implicated, in this case via ATP release downstream of activation of the mechanosensitive ion channel Piezo1(Albarran-Juarez et al, 2018; Wang et al., 2015), though this pathway has been disputed(Dela Paz & Frangos, 2019). The orphan receptor GPR68 was identified as a shear stress sensor specific to small arteries though the mechanism of activation, including ligand dependence or independence, is unknown(Xu et al., 2018). These studies also vary in the Gα proteins implicated, which included different Gαi isoforms (Gudi et al, 1996; Jung et al., 2012) and the Gαq/11 family(Albarran-Juarez et al., 2018; Chachisvilis et al., 2006; Wang et al., 2015).  \n",
      "\n",
      "\n",
      "To resolve this issue, we took an unbiased approach starting from the Gα proteins using the dramatic morphological changes in endothelial alignment to flow as the initial readout. Our results showed that Gαi2 and Gαq/11 function in parallel, identified latrophilin-2 (LPHN2) as the key upstream GPCR and demonstrated a specific requirement for LPHN2 in flow-dependent remodeling in zebrafish and mice.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Results\n",
      "\n",
      "\n",
      "Identification of Gα protein subunits required for flow sensing\n",
      "\n",
      "\n",
      "To investigate the roles of GPCR signaling in endothelial flow responses, we first transfected human umbilical vein endothelial cells (HUVECs) with short interfering RNAs (siRNAs) to suppress Gs, Gi, Gq/11 and G12/13 classes of Gα proteins and assayed EC alignment in flow as our initial readout. Single knockdowns gave either weak (Gi) or no (Gs, G12/13, Gq/11) inhibition of EC alignment in flow (Fig.1A and Appendix fig. S1A-C). However, simultaneous knockdown of Gi and Gq/11 completely inhibited alignment (Fig. 1A), whereas other combinations were ineffective (Appendix fig. S1D, S1E). Activation of SFKs is the earliest flow-responsive event mediated by PECAM-1, followed by VEGFRs and Akt(Tzima et al, 2005). These events were similarly blocked by knockdown of Gi plus Gq/11 but not by single knockdowns (Fig. 1B, 1C and Appendix fig. S1F). Therefore, Gi and Gq/11 act in parallel to initiate PECAM-1-dependent flow signaling.\n",
      "\n",
      "\n",
      "Rescue controls with siRNA-resistant Gα variants showed that siRNA-resistant Gq and Gi2 restored cell alignment, whereas Gi1 and Gi3 did not (Fig. 1D and Appendix fig. S2A, S2B).  This unexpected result prompted us to search for residues that are similar in Gq and Gi2 but different in Gi1 and Gi3.  Two residues (using human Gi2 numbering) fit this pattern: D167 in Gi2 corresponds to N in Gi1/3; and K307 corresponds to Q306 in Gi1/3 (Fig. 1E).  Gi1(Q306K), but not Gi1(N166D), rescued EC alignment in flow (Fig. 1F and Appendix fig. S2C).  Consistently, Gi2 mutant lacking K307 (K307Q) could not rescue the defects on flow-induced alignment under the knockdown of Gi plus Gq/11 (Appendix fig. S1G and Appendix fig. S2D).  Therefore, K307 in Gi2 distinguishes the Gα subunits participate in flow signaling.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Flow-induced activation of specific Gα proteins\n",
      "\n",
      "\n",
      "K307 lies in the region that specifies coupling with GPCRs(Bae et al, 1999; Flock et al, 2017), suggesting that K307 may determine Gα activation.  However, testing this hypothesis requires measurement of Gα activation in live cells.  Unfortunately, indirect Gα activation assays using second messengers such as cAMP or calcium are unreliable in the context of fluid shear stress where multiple activating/suppressing pathways are stimulated. We therefore developed pulldown assays for Gα based on the specific binding of GTP-loaded Gα subunits to effector proteins, namely GINIP for Gi and GRK2 N-terminal domain for Gq (Appendix fig. S3a)(Gaillard et al, 2014; Tesmer et al, 2005).  To facilitate these assays and distinguish specific Gα isoforms (especially Gi isoforms), we prepared versions containing an internal GluGlu (EE) epitope tag that does not impair function (Appendix fig. S3B and refs(Medina et al, 1996; Wilson & Bourne, 1995)).  For Gq, to enhance pulldown efficiency, we co-expressed Ric8A, which stabilizes active Gq(Papasergi et al, 2015).  Active, but not inactive, Gi and Gq bound to these effector proteins, while when the artificial GPCR DREADD was expressed, its ligand clozapine-N-oxide (CNO) also activated Gα proteins (Appendix fig. S3C-S3F).  Importantly, this Gα pulldown assay showed not only rapid activation of Gi2 and Gq by FSS (Fig. 1G-J), but also differential responses of Gi1 and gain-of-function Gi1(Q306K) to FSS (Fig. 1K, 1L).  Gi activation was unaffected by PECAM-1 knockdown, indicating that this step was upstream or independent of PECAM-1 (Fig. 1M, 1N).  Flow-mediated activation of specific Gα subunits thus corresponds to their requirement in EC flow responses.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Identification of the upstream GPCR\n",
      "\n",
      "\n",
      "We next sought to identify the GPCR(s) that mediates flow activation of Gi2 and Gq/11. Although affinity purification is the obvious choice, rapid release of Gα proteins from GPCRs after GTP loading limits this approach.  Starting with wild-type and its Q306K mutant, we inserted 1) an internal GFP for affinity purification and 2) four alanines (hereafter ins4A) into helix α5, which blocks GTP loading, thereby preventing dissociation from GPCRs even in the presence of GTP(Kaya et al, 2016) (Fig. 2A). Both constructs localized mainly to the plasma membrane, as did wild-type Gα (Appendix fig. S4A).  We further employed detergent-free nanodisc-forming styrene-maleimide anhydride (SMA) copolymer to stabilize GPCR conformation within its native lipid environment(Lee et al, 2016).  This approach was validated by coimmunoprecipitation using GFP-Trap® nanobody beads of Gα variants with DREADD GPCR, which showed increased association after receptor activation (Appendix fig. S4B, S4C).  Proteomic analysis for GPCRs that associated with Gi1(Q306K) but not wild-type Gi1 in response to FSS (protocol in Appendix fig. S4D), identified S1PR1 and ADGRL3 (Appendix fig. S4E).  However, depletion of S1PR1 did not block alignment in FSS (Appendix fig. S4F, S4G).  Furthermore, S1PR1 reportedly activates Gi1 and Gi3 as well as Gi2(Lee et al, 1996), which is inconsistent with the Gα specificity defined above.  We therefore focused on ADGRL family proteins, also called latrophilins (LPHNs).\n",
      "\n",
      "\n",
      "Latrophilins (1,2 and 3 in mammals) are adhesion-type GPCRs that bind to counterreceptors on neighboring cells and regulate neuronal synapses(Sando et al, 2019).  Although it has been reported that different types of endothelial cells have different latrophilin isoforms, LPHN2 is most widely expressed and is the overwhelming major isoform in HUVECs (Appendix figs S5A)(Maleszewska et al, 2016).  Consistent with the proteomic results, Gi1(Q306K)(ins4A) associated with endogenous LPHN2, which increased upon FSS (Fig. 2B, C). This result strongly suggests that FSS activates LPHN2 to recruit Gα proteins. Furthermore, Gi1(Q306K), Gi2, and Gq co-immunoprecipitated with LPHN2, whereas Gi1 did not, indicating that LPHN2 specifically bound to the flow-responsive Gα proteins defined above (Fig. 2D).  Importantly, LPHN2 knockdown blocked flow-mediated activation of Gi2 (Fig. 2E, 2F).  Together, these data provide strong evidence that latrophilin-2 mediates Gα activation essential for endothelial alignment in response to flow.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Latrophilins in EC flow signaling in vitro.\n",
      "\n",
      "\n",
      "Knockdown of LPHN2, but not LPHN1 or LPHN3, blocked EC alignment in flow (Fig. 3A, 3B; knockdown confirmed in Appendix fig. S5B, S5C).  LPHN2 knockdown in human aortic ECs similarly blocked flow-induced alignment (Appendix fig. S5D, S5E).  In line with these results, LPHN2 knockdown also blocked Golgi apparatus polarization in flow (Appendix fig. S5F, S5G). LPHN2 localized to cell-cell contacts (Fig.3C and Appendix fig. S6). The flow responsive G1 mutant, Gi1(Q306K), also associated with PECAM-1 following FSS (Fig. 3D). The association between LPHN2-Gi1(Q306K) complex with junctional mechanosensory complex proteins PECAM-1 and VE-Cadherin was detected after 24 hours of flow (Fig. 3D-3G), indicating a persistent interaction. We also assessed the immediate flow-dependent signals. LPHN2 depletion strongly suppressed flow-mediated acute activation of SFKs and Akt (Fig 3H-3J). Thus, LPHN2 colocalizes, physically associates with and is required for signaling through the junctional complex. However, LPHN2 knockdown did not inhibit activation of VEGFR2 by VEGF-A, indicating that LPHN2 functions specifically in response to laminar flow (Appendix fig. S7A, S7B).  \n",
      "\n",
      "\n",
      "Among the 3 latrophilin isoforms in human genome, all rescued flow-mediated endothelial alignment after LPHN2 knockdown (Fig. 3K, expression confirmed in Appendix fig. S8A), indicating functional equivalence; the observed differences are therefore due to differential expression.  EC alignment was rescued by wild type (WT) LPHN2 but not H1071A, which is defective in coupling to Gα proteins(Nazarko et al, 2018) (Fig. 3L, expression confirmed in Appendix fig. S8B).  Adhesion-type GPCRs can be activated by internal cleavage by the extracellular GAIN domain which exposures its Stachel peptide as a tethered agonist(Vizurraga et al, 2020). However, the autoproteolysis-defective mutant fully rescued EC alignment (see Fig.3L). Helix-8 is known to endow some GPCRs with mechanosensitivity to membrane stretch(Erdogmus et al., 2019). However, a LPHN2 helix-8 mutant could still rescue EC alignment in flow (Fig. 3L).  Taken together, these findings indicate that latrophilin regulates Gα activation required for endothelial flow responses, but activation requires neither the Stachel peptide nor helix-8.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Latrophilin-2 regulates flow-induced endothelial cell morphological changes in vivo\n",
      "\n",
      "\n",
      "Long-term laminar flow stabilizes EC junctions, promoting a linear, continuous morphology associated with barrier function(Kroon et al, 2017) (see Fig.3A).  To test whether the role of LPHN2 in this process, we analyzed junctional morphology in vitro with and without 16h of flow. In control cells, shear stress shifted junctions from an irregular morphology to linear junctions; in Lphn2 knockdown ECs, junctions were indistinguishable in the absence of flow but failed to linearize in flow (Fig. 4A, 4B), demonstrating that the effect is flow-specific.  To further examine contributions of blood flow/latrophilin-2 to ECs in vivo, zebrafish embryos are useful as a developmental system that is amenable to manipulation of shear stress using morpholinos against cardiac troponin T type 2a (tnnt2a; ‘silent heart’)(Sehnert et al, 2002), without global adverse effects.  We employed zebrafish embryos expressing Tg(kdrl:ras-mCherry) to mark endothelial cells(Sehnert et al., 2002) and designed sgRNAs for CRISPR-Cas9 targeting all latrophilin isoforms (adgrl1, adgrl2a plus adgrl2b.1, and adgrl3). Our sgRNAs induced >50% cleavage in the T7 endonuclease assays and reduced mRNA levels through nonsense-mediated decay (Appendix fig. S9). Since Lphn2 is the major endothelial isoform, we performed CRISPR-mediated F(0)/mosaic knockout of adgrl2a+adgrl2b.1 (hereafter Lphn2 mutation), which inhibited elongation and enlargement of ECs in the dorsal aorta (Fig. 4C).  Injecting single cell embryos with silent heart morpholinos at a dose that completely blocked cardiac contractility similarly blocked the elongation of ECs. Furthermore, combining tnnt2a and Lphn2 mutation had no further effect, supporting a specific role for latrophilins in flow sensing (Fig. 4D, 4E).  \n",
      "\n",
      "\n",
      "To address LPHN function in mammals, we crossed Lphn2flox/flox mice with CDH5-CreERT2 mice (Lphn2 ECKO), which were injected with tamoxifen at 8 weeks of age and remained viable 10 weeks later.  The descending thoracic aorta (a region of high laminar flow) of Lphn2 ECKO mice had poorly aligned ECs with irregular junctions (Fig. 4F-H).  Additionally, Lphn2 ECKO increased vascular permeability in the whole aorta, as assessed by Evans blue extravasation (Fig. 4I, 4J). In vitro, flow for 24h increased HUVEC barrier function, which was blocked by LPHN2 knockdown, whereas barrier function in the absence of flow was unaffected (Fig. 4K, 4L). Latrophilins are therefore flow-responsive GPCRs that regulate multiple steps in PECAM1-dependent EC mechanotransduction. \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Plasma membrane fluidization initiates latrophilin signaling\n",
      "\n",
      "\n",
      "We next considered how FSS activates LPHN2. As mutation of the autoproteolytic GAIN domain, which is speculated to be mechanosensitive, had no effect (Figure 3i), we considered alternatives. FSS induces a rapid increase in plasma membrane fluidity, in some cases associated with decreased plasma membrane cholesterol(Butler et al, 2001; Haidekker et al, 2000; Yamamoto & Ando, 2013; Yamamoto et al, 2020). As GPCRs can respond to such changes(Vizurraga et al., 2020), HUVECs were treated with methyl-β-cyclodextrin (MβCD) to extract plasma membrane cholesterol.  Treatment with 5 mM MβCD for 1 min reduced plasma membrane cholesterol to the same extent as flow, assayed by binding of the cholesterol-specific probe D4H-Clover3(Liu et al, 2017) (Figure 5A).  No discernable effect on cytoskeletal organization was observed under these conditions (Figure 5B). MβCD treatment also induced Gα binding to LPHN2 (Figure 5C) and activated both SFKs and VEGFR2, which were blocked by LPHN2 knockdown (Figure 5D-F). Together with published data demonstrating that cholesterol supplementation blocks flow signaling to these and related pathways(Fancher et al, 2018; Tirziu et al, 2005; Yamamoto & Ando, 2013, 2015; Yamamoto et al., 2020), these results suggest that membrane fluidization can trigger LPHN2 GPCR activation and downstream signaling (Figure 5G).\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Latrophilins in vascular remodeling in vivo\n",
      "\n",
      "\n",
      "To address roles of LPHN2 in flow-dependent vascular remodeling, we examined zebrafish embryos, where artery lumen diameters are determined by EC shear stress(Sugden et al., 2017). We observed that intersegmental vessels (ISVs) at 48 hours post fertilization had reduced diameters after both latrophilin knockout and blockade of blood flow, with no further effect when combined (Fig. 6A, 6B). At 72 hpf, when intersegmental arteries and veins are distinguishable, the decrease in diameter was observed only in arteries (Fig. 6C). LPHNs are thus required for effects of flow on artery diameter.\n",
      "\n",
      "\n",
      "These results prompted us to examine flow-dependent vascular remodeling in mice.  Femoral artery ligation triggers flow-driven arteriogenesis in the thigh and hypoxia-driven angiogenesis in the calf(Cornella et al, 2017). Following surgery, LPHN2 ECKO mice showed lower blood flow in the foot even at day 1 and markedly slower recovery than WT littermates (Fig. 6D, 6E). To understand these effects, we performed micro-CT of vascular casts after injection of electron-dense bismuth microparticles that label arteries/arterioles but not capillaries due to particle size. Surgery induced an increase in small arteries in WT mice but essentially none in LPHN2 ECKO mice (Fig. 6F, 6G). LPHN2 ECKO mice also had much lower density of small arteries even in the control leg, consistent with lower flow on day 1. Staining for PECAM to examine the total vasculature in leg muscle sections revealed decreased capillary density as well, even at baseline (Fig. 6H, 6I).  LPHN2 ECKO mice thus show low vascular density at baseline and greatly reduced flow-dependent remodeling. As expected from the low vascular density, a treadmill fatigue test (Fig. 6J) LPHN2 ECKO mice showed markedly lower exercise capacity (Fig. 6K, 6L) and impaired oxygen consumption (Fig. 6M). LPHN2 ECKO thus results in vascular defects in vivo, at least in part due to defective EC flow sensing. To address whether these defects were related to shear stress sensing, we examined an in vitro system in which FSS enhances VEGF-induced sprouting into collagen gels(Galie et al, 2014). Enhanced sprouting was completely blocked by LPHN2 depletion with only a weak effect on basal migration (Fig. 6N-6P).\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Genetic link to human disease.\n",
      "\n",
      "\n",
      "The Cardiovascular Disease Knowledge Portal reported four SNPs near the 5’ end of the ADGRL2 gene (Appendix fig. S10A) that exhibit genome wide association with hypertension (Appendix fig. S10B). All 4 are in close proximity to each other and are in linkage disequilibrium (D’: 0.9953, R2: 0.97). Two of the SNPs meet the significant threshold (p<5 x 10-8) and rs1195871 is relatively conserved among vertebrates. Of particular interest, the intronic SNP rs186892211 in the ADGRL2 locus (Appendix fig. S10A) is associated with large-artery atherosclerosis and subsequent ischemic stroke (TOAST classification) with a very high odds ratio (odds ratio 720; p< 3.134e-10, Appendix fig. S10B). \n",
      "\n",
      "\n",
      "Discussion\n",
      "\n",
      "\n",
      "These data elucidate a pathway in which fluid shear stress acts through the latrophilin adhesion GPCRs to activate subsequent cellular signaling events and vascular remodeling in vivo, thus connecting time scales from seconds to hours to weeks/months. Previous studies showed that laminar flow induces an increase in plasma membrane fluidity associated with depletion of membrane cholesterol (Yamamoto & Ando, 2013, 2015; Yamamoto et al., 2020).  Our data demonstrate that artificial cholesterol depletion activates flow-induced signals in a Lphn2-dependent manner, suggesting that latrophilins connect physical changes in the plasma membrane to cellular signaling. Latrophilins mediate activation of the PECAM-1/VE-cadherin/VEGFR pathway through both Gi2 and Gq/11, suggesting parallel or redundant downstream pathways.  Consistent with these functional effects, FSS induces physical interactions (direct or indirect) between LPHN2-Gα and PECAM-1.  However, LPHNs are not required for activation of VEGFRs by its ligand VEGF.  \n",
      "\n",
      "\n",
      "Previous studies identified either Gi or Gq/11 as mediators of flow signaling(Chachisvilis et al., 2006; Jung et al., 2012; Wang et al., 2015).  It seems plausible that, depending on EC subtype or experimental conditions, either Gi or Gq/11 predominated, which concealed their functional redundancy. Previous studies also identified other GPCRs as mediators of EC shear stress responses, via direct or indirect mechanisms, as well as making claims for GPCR-independent activation of G proteins(Dela Paz et al., 2017). GPCRs implicated in shear stress signaling include bradykinin receptor B2(Chachisvilis et al., 2006), ATP receptor P2Y2(Albarran-Juarez et al., 2018; Wang et al., 2015), sphingosine phosphate receptor S1PR1(Jung et al., 2012), the apelin receptor APJ(Strohbach et al, 2018) and GPR68(Xu et al., 2018). An indirect association of Gq/11 with PECAM1 was also reported, which dissociated upon application of shear stress(dela Paz et al, 2014). Differences in EC location or subtype likely account for some of these effects. For example, GPR68 is expressed specifically in small resistance arteries where it controls vasodilation but had no reported effect on development or patterning(Xu et al., 2018). P2Y2 was also reported to affect vasodilation and blood pressure without reported effects on development or patterning.  It seems likely that differences in local expression may thus explain some of the apparent multiplicity, though additional variables such as modulation by other inputs and coupling to different downstream effectors may also contribute. Understanding this GPCR-Gα protein signaling network is therefore an important though complex direction for future work.  The novel pulldown assay and GPCR-Gα protein affinity purification protocol developed here may facilitate such efforts.\n",
      "\n",
      "\n",
      "It is also noteworthy that latrophilins are required only for activation of signaling events including VEGFR2 by flow, but not for activation of VEGFR2 by its ligand, VEGF. PECAM-1 first appears in vertebrates but has multiple other functions in leukocyte transmigration and activation, platelet reactivity and angiogenesis that are independent of flow(Privratsky et al, 2010). Together, these findings argue that latrophilins, in essence, confer flow sensitivity to pre-existing pathways controlled by ligand availability.  \n",
      "\n",
      "\n",
      "Lphn2 mutation in zebrafish and endothelial knockout in mice caused multiple vascular defects that implicate FSS mechanotransduction, including reduced EC alignment in the direction of flow, altered junctional morphology and permeability, smaller artery diameters and defects in arterialization in the hindlimb ischemia model. Lower capillary and small artery density in some vascular beds may also be attributable to defective FSS sensing, as FSS regulates angiogenic sprouting (Galie et al., 2014) and stabilizes nascent vessels once blood begins to flow (Campinho et al, 2020). We note, however, that flow-independent functional roles for Lphn2 in ECs have been observed (Camillo et al, 2021) and likely contribute as well.\n",
      "\n",
      "\n",
      "These results link together many observations to provide a comprehensive model for EC responses to FSS (Appendix fig. S11). These findings are of further interest in light of human genetic data that strongly link the Adgrl2 gene to vascular disease (Appendix fig. S10).  However, the identified polymorphisms reside in regulatory or intronic regions, thus, causal connections to vascular disease are not obvious. Intronic polymorphisms can regulate gene expression but could also be in linkage disequilibrium with causal SNPs that are not yet identified. More detailed population genetic analyses are required to address these issues.\n",
      "\n",
      "\n",
      "These findings open many important questions for future work.  Structural insights into latrophilin activation are needed, as are insights into the contribution of the large extracellular domain.   The parallel roles of Gi2 and Gq/11 and downstream effectors remain to be understood. Thorough analysis of latrophilin functions in vascular development and physiology remains to be done.  Lastly, combined human genetic and structure-function studies are needed to elucidate the contribution of Lphn2 variants to human cardiovascular disease. \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Experimental Procedures\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Antibodies\n",
      "\n",
      "\n",
      "Primary antibodies used in this study:\n",
      "\n",
      "\n",
      "|Epitope | Vendor | Catalog #|\n",
      "|--- | --- | ---|\n",
      "|Phospho-Src | Cell signaling | 6943|\n",
      "|Src | Cell signaling | 2109|\n",
      "|Gi | NewEast Biosciences | 26003|\n",
      "|Gq | NewEast Biosciences | 26060|\n",
      "| | BD Bioscience | 612705|\n",
      "|Gs | NewEast Biosciences | 26006|\n",
      "|Phospho-VEGFR2(Tyr1175) | Cell signaling | 2478|\n",
      "|Phospho-VEGFR3 (Tyr1054/Tyr1059) | Invitrogen | 44-1047G|\n",
      "|VEGFR2 | Cell signaling | 2479|\n",
      "|Phospho-Akt (Ser473) | Cell signaling | 9271|\n",
      "|VE-Cadherin | Santa Cruz | sc-6458|\n",
      "|LPHN2 | Invitrogen | PA5-65359|\n",
      "|RFP | Antibodies-Online | ABIN129578|\n",
      "|alpha-smooth muscle actin | Invitrogen | 50-9760-82|\n",
      "|alpha-smooth muscle actin-Cy3-conjugated | Sigma | C6198|\n",
      "|ZO-1 | Invitrogen | 61-7300|\n",
      "|beta-catenin | Cell signaling | 9562|\n",
      "|GluGlu | BioLegend | MMS-115P|\n",
      "|HA | BioLegend | 901501|\n",
      "|GFP | Abcam | ab13970|\n",
      "| | Santa Cruz | sc-9996|\n",
      "|GST | Cell signaling | 2625|\n",
      "|tubulin | Invitrogen | 62204|\n",
      "|actin | Santa Cruz | sc-8432|\n",
      "|PECAM1 | Kind gift from Dr. Peter Newman | |\n",
      "| | BD Bioscience | 557355|\n",
      "|GM130 | BD Bioscience | 610823|\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Cell Culture\n",
      "\n",
      "\n",
      "Primary HUVECs were obtained from the Yale Vascular Biology and Therapeutics core facility.  Each batch is composed of cells pooled from three donors.  Cells were cultured in M199 (Gibco: 11150-059) supplemented with 20% FBS, 1x Penicillin-Streptomycin (Gibco: 15140-122), 60 µg/ml heparin (Sigma: H3393), and endothelial growth cell supplement (hereafter, complete medium).  HUVECs used for experiments were between passages 3 and 6.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Shear stress\n",
      "\n",
      "\n",
      "For generating protein lysates for pulldown assays, HUVECs were seeded on tissue culture-treated plastic slides coated with 10 µg/ml fibronectin for 1 hour at 37˚C and then grown to confluence in HUVEC complete media.  For short-term stimulation with shear stress, cells were starved overnight in M199 medium with 2% FBS and 1:10 of ECGS or for 30 minutes in M199 medium containing 0.2 % BSA.  These slides were set in parallel flow chambers and shear stress applied as described(Frangos et al, 1988).\n",
      "\n",
      "\n",
      "For imaging cells under shear stress, HUVECs were plated in HUVEC complete medium on glass bottom 6-well plates coated with 10 µg/ml fibronectin and shear stress applied on an orbital shaker at 150rpm for 24 hours, leading to uniaxial pulsatile shear in the peripheral region and multiaxial shear in the middle of the well, as previously described(Arshad et al, 2021).  Cells in the outer region, 0.7-0.9 cm from the center of each well were examined for Golgi orientation and endothelial alignment.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Image analysis\n",
      "\n",
      "\n",
      "Cell orientation was calculated by taking the masks of the cell nuclei determined by Hoechst images, fitting them as an ellipse, and determining the angle between flow direction and the major axis of the ellipse.  Analyzed results were visualized as histograms showing the percent of cells within each 10° of the direction of flow or as quantification of aligned cells with nuclei whose major axis were within 0-30 degrees to flow direction as indicated in the figure legends.  Eccentricity for cell alignment was calculated based on first eccentricity of ellipse following the equation below.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "(a: major axis, b:minor axis)\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Junctional linearization index was calculated as ratio of perimeter of the cell to perimeter of its convex hull following the equation below:\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Data display\n",
      "\n",
      "\n",
      "Quantified data are displayed as means ± standard error (SEM) in which the indicated replicates are independent experiments.  F test was conducted to check equal variance.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Lentiviral transduction\n",
      "\n",
      "\n",
      "Lenti-X 293T cells (Clontech, 632180) were cultured for at least 24 hours in DMEM supplemented with 10% FBS and lacking antibiotics, then transfected with lentiviral plasmids encoding the gene of interest and packaging plasmids (Addgene: 12259 and 12260) using Lipofectamine 2000 (Thermo Fisher Scientific: 11668-019) following the manufacturer’s protocols with Opti-MEM medium.  Conditioned media from these cultures were collected 48 hours later, sterilized through 0.22µm filters and added to HUVECs together with 8µg/ml of polybrene (Sigma: 107689).  After 24 hours, cells were switched to complete medium for 48 hours.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "siRNA transfection\n",
      "\n",
      "\n",
      "HUVECs were cultured in EGMTM-2 Endothelial Cell Growth Medium-2 BulletKitTM\n",
      "\n",
      "\n",
      " (Lonza: CC-3156 and CC-4176) for 24 hours before transfecting with RNAiMax (Thermo Fisher Scientific: 13778-150) with 20nM siRNA in Opti-MEM (Gibco: 31985-070) according to the manufacturer’s instructions.  After 6 hours, cells were switched to EGM-2 medium and used for experiments 2-3 days later.  Gα protein siRNAs were custom designed based on previous publications(Grzelinski et al, 2010; Krumins & Gilman, 2006; Ngai et al, 2008).  For latrophilin knockdown, ON-TARGET plus Smartpool siRNAs from Dharmacon against human LPHN1 (L-005650-00-0005), LPHN2 (L-005651-00-0005), LPHN3 (L-005652-00-0005) were used.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Western Blotting\n",
      "\n",
      "\n",
      "HUVECs were washed with PBS and extracted in Laemmli sample buffer.  Samples were separated by SDS-PAGE and transferred onto nitrocellulose membranes.  Membranes were blocked with 5% milk in TBS-T and probed with primary antibodies at 4˚C for overnight.  The targeting proteins were visualized by HRP-conjugated secondary antibodies and subsequent HRP-luminol reaction.  \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Immunofluorescence\n",
      "\n",
      "\n",
      "HUVECs were washed with PBS, fixed for 10 minutes with 3.7% PFA in PBS.  Following fixation, cells were permeabilized with 0.5% Triton X-100 in PBS for 10 minutes and then incubated with 3% BSA in PBS for 30 minutes for blocking.   Cells were washed with PBS after blocking and were incubated Alexa488-Phalloidin and Hoechst for 1 hour, then washed 4 times with PBS and mounted.  Images were captured with 20x or 60x objective on a PerkinElmer spinning disk confocal microscope or a Leica Sp8 confocal microscope with the Leica Application Suite (LAS) software. Cell alignment was determined as described previously(Baeyens et al, 2014). \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Reverse transcription and quantitative PCR\n",
      "\n",
      "\n",
      "RNA was isolated from HUVECs using RNeasy kit according to the manufacturer’s instructions and quantified using a nanodrop spectrophotometer.    Following cDNA synthesis using Bio-Rad iScript kit, RT-PCR was performed as follows.  Each PCR reaction contains 42 two-step amplification cycles consisting of: (a) denaturation at 95˚C for 30s, and (b) annealing and extension at 60˚C for 30s.  The amplification curve was collected, and the relative transcript level of the target mRNA in each sample was calculated by normalization of Ct values to the reference mRNA (GAPDH).  Primer sequences used for RT-PCR were as shown in Table 1.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Gα protein pulldown assay\n",
      "\n",
      "\n",
      "BL21 E.coli cells were transformed with constructs expressing GST-tagged GINIP protein or GST-tagged GRK2 N-terminal domain.  Cells were incubated in terrific broth (http://cshprotocols.cshlp.org/content/2015/9/pdb.rec085894.full) and protein expression induced by addition of 0.5µg/ml IPTG.  Cells were collected after 8 hours, lysed and GST- proteins collected on Glutathione-conjugated beads. Beads were washed 4 times, eluted with 50mM glutathione and proteins desalted on a gel filtration column. Aliquots of 20 µg aliquots were stored frozen and thawed shortly before use.  \n",
      "\n",
      "\n",
      "For pulldown assays, HUVECs infected with lentivirus encoding GluGlu-tagged Gi mutant or lentivirus encoding both GluGlu-tagged Gq mutant and Ric8A were lysed in cell lysis buffer composed of 10mM Tris-HCl pH7.5, 150mM NaCl, 1% Triton X-100, 5mM DTT and additional 2µg/ml of GST-GINIP or GST-GRK2N, and the lysates were incubated for 10 minutes at 4˚C with gentle agitation with glutathione beads.  Beads were washed three times with cell lysis buffer, isolated proteins were solubilized in SDS sample laemmli buffer, and were analyzed by western blotting.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Mass spectrometry screening for Gα binding receptors\n",
      "\n",
      "\n",
      "To create a Gα protein variant that interacts more strongly with GPCR, a green fluorescent protein (GFP) tag was inserted for affinity purification, alongside a mutation of four alanine residues into helix α5 (ins4A), as this hinders GTP binding and maintains the association with GPCRs even in the presence of GTP.  Additionally, for the extraction of GPCRs while preserving their configuration and lipid environment, a non-detergent nanodisc approach utilizing styrene-maleimide anhydride (SMA) copolymers was employed according to protocols described previously.\n",
      "\n",
      "\n",
      "In the coimmunoprecipitation process, GFP-Trap® nanobody beads were utilized to capture the interactions between GFP-tagged Gα protein variants and activated designer receptors exclusively activated by designer drugs (DREADDs). The resulting complexes were subjected to subsequent proteomic analysis to determine which GPCRs interacted preferentially with the Gi1(Q306K) variant following induction of fluid shear stress. Identified receptors were further analyzed by western blotting as described in the supplementary material.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Measurement of Golgi orientation\n",
      "\n",
      "\n",
      "Following exposure to laminar shear stress, HUVECs were washed with PBS, fixed for 10 minutes with 3.7% PFA in PBS, permeabilized with 0.5% Triton X-100 in PBS for 10 minutes, blocked with 3% BSA in PBS for 30 minutes and then incubated with GM130 antibody (1:500, BD bioscience) at 4˚C for overnight. After washing, cells were incubated with Hoechst and Alexa647-conjugated secondary antibody at RT for 1 hour to visualize the nuclei and Golgi apparatus, respectively.  Images were captured with 20x objective on a Leica Sp8 confocal microscope with the Leica Application Suite (LAS) software.\n",
      "\n",
      "\n",
      "Golgi polarization was calculated as a vector that connects the center of mass of the nucleus to the center of mass of the Golgi. The angle between this vector and the direction of flow was then determined, with 0° defined as against the flow. Oriana V4 software was used to calculate the angle mean and circular SD and to generate a rose plot for data visualization.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "In vitro permeability assay using the FITC-streptavidin/biotinylated-fibronectin\n",
      "\n",
      "\n",
      "Slides/dishes were coated with biotinylated-Fibronectin at 0.25 mg/mL overnight at 4°C. HUVECs were seeded of 2×106 cells per well on glass-bottom 6-well plates. Cells were subjected to laminar shear stress by orbital shaking at 150 rpm for 24 hours.  Immediately after cessation of flow, FITC-Streptavidin (1:1000, Invitrogen) was added to the cells for 1 minute before  the  fixation  by  the  addition  of  3.7%  (v/v)  paraformaldehyde. Following   subsequent   washing   and   blocking, Alexa Fluor 488 Phalloidin (1:200; Invitrogen) was  added  to  stain  F-actin  filaments. Images were captured from 5 random fields and FITC-streptavidin staining quantified using ImageJ. \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "In vitro endothelial 3D invasion assay under shear stress\n",
      "\n",
      "\n",
      "In order to equalize the VEGF concentration in the collagen gel and endothelial culture medium, VEGF-189 (R&D systems, 8147-VE), a matrix-binding isoform of VEGF, was applied to the bottom of each well at a concentration of 50 ng/ml prior to adding collagen matrices. Rat tail collagen type I (Thermofisher scientific, A1048301) was employed to prepare collagen matrices at the concentration of 1.5 mg/ml with additional NaOH to facilitate the collagen matrix assembly by neutral pH. After thorough mixing, 1 ml of collagen I solution was placed in each well of a 12-well plate (Corning #3513) and polymerized at 37°C and 5% CO2 for 2 hours to overnight. \n",
      "\n",
      "\n",
      "To prepare endothelial cells, confluent HUVEC culture plates were washed with PBS, trypsinized, and counted.  After pelleting, cell pellets were resuspended in EGM-2 medium (Lonza, #CC-3162) free of VEGF and heparin to establish a positive gradient of VEGF from cells to the bottom of the collagen gel, and 300,000 cells per well were placed on the collagen gel and incubated for 24 hours. Following incubation, plates were placed on an orbital shaker and ECs were exposed to orbital shaking at 150 rpm for 48 hours as mentioned above in shear stress section. Cells were then fixed in 4% PFA and permeabilized in PBS containing 0.5% Triton X-100. Nonspecific binding was blocked by incubation with PBS containing 3% BSA. Thereafter, cells were subjected to Alexa488-conjugated phalloidin (1:400 dilution; Thermofisher scientific: A12379) and Hoechst (1:1000 dilution; Invitrogen: H1399) for 2 hours to stain actin cytoskeleton and cell nuclei, respectively. All images were captured with the Nikon Sp8 confocal microscope and segregated into peripheral region (with uniaxial pulsatile shear) vs. static controls.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Zebrafish husbandry and handling\n",
      "\n",
      "\n",
      "Zebrafish were housed and maintained at 28.5ºC in accordance with standard methods approved by the Yale University Institutional Animal Care and Use Committee (#2017-11473)(Lawrence, 2016).\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Generation of LPHN/adgrl knockdown zebrafish using CRISPR/Cas9 ribonucleoproteins\n",
      "\n",
      "\n",
      "Four zebrafish latrophilin paralogs were identified—adgrl1a, adgrl2a, adgrl2b.1, and adgrl3.1. Fused sgRNAs were generated targeting each paralog (see Table 2) by annealing locus-specific oligonucleotides to a common 5’ universal oligonucleotide and performing in vitro RNA transcription (AmpliScribe T7-Flash Kit, Lucigen, ASF3507)(Gagnon et al, 2014). An injection mix of 50 ng/µL each sgRNA, an equivalent concentration of Cas9 protein (TrueCut Cas9, Invitrogen, A36497), and either 1 µg/µL silent heart/tnnt2a(Sehnert et al., 2002) or standard control morpholino (GeneTools) was prepared. 1 nL of the mixture was injected into Tg(kdrl:ras-mCherry, fli1a:nls-GFP) zebrafish at the one cell stage (Parker Hannifin, Picospritzer III). CRISPR reagent efficacy was confirmed by T7 endonuclease assay (see Figure S5).  Guides against adgrl2a and 2b.1 were combined to target all LPHN2 isoforms.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Immunostaining and imaging of ECs in the knockdown zebrafish\n",
      "\n",
      "\n",
      "CRISPR/Cas9 RNP-injected embryos were either fixed at 48 hpf in 4% paraformaldehyde (Santa Cruz Biotechnology, SC-253236) and immunostained for GFP (1:300 chicken anti-GFP, abcam, ab13970), mCherry (1:300 rabbit anti-mCherry, abcam, ab167453), and ZO-1 (1:200 mouse anti-ZO-1, Invitrogen, 61-7300) with species-appropriate secondary antibodies (1:400 Invitrogen anti-chicken/-rabbit/-mouse Alexa 488/546/647, A-11039/A- 10040, A-31571) or imaged live. For fixed imaging, the larvae were processed as previously described(Sugden et al., 2017). For live imaging, larvae were anesthetized by immersion in 600 µM tricaine methylsulfonate (Western Chemical, TRS5), and embedded in 1.5% low-melt agarose (BioRad, 1613106). For staining in detail, larvae were fixed overnight in 4% PFA at +4ºC and then transitioned into 100% methanol via a series of dilutions. Once in methanol, the larvae were stored at -20ºC overnight, then rehydrated the next day via the same dilution series. Larvae were permeabilized using incubation in acetone for 15 minutes at -20ºC and then by incubation in 20 µg/mL proteinase K for 20 minutes at 37ºC. Larvae were refined in 4% PFA for 15 minutes at room temperature and then incubated in blocking buffer (0.5% Triton X-100, 10% goat serum, 1% BSA, 0.01% sodium azide, and 1% DMSO in PBS) for 2 hours at room temperature. Larvae were incubated in primary antibody (1:400 chicken anti-GFP, abcam ab13970) overnight in blocking buffer at +4ºC. Larvae were thoroughly washed in PBS-Tx and incubated overnight at +4ºC in secondary antibody (1:400 goat anti-chicken Alexa488 A-11039, ThermoFisher) in blocking buffer. Following final washes in PBS-Tx, larvae were mounted and imaged. Intersegmental vessel and dorsal aorta diameter were then calculated using ImageJ. All imaging was performed using a Leica SP8 confocal microscope.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Immunostaining and imaging of mouse aorta\n",
      "\n",
      "\n",
      "Aortae were perfusion-fixed in situ with 4% paraformaldehyde (overnight) prior to staining with primary beta-catenin antibody (Cell signaling: 9562) and fluorophore-conjugated secondary antibody (Invitrogen, donkey anti-Rabbit IgG (H+L) Highly Cross-Adsorbed Secondary Antibody, Alexa Fluor 647, A31573) in in Claudio buffer (1% FBS, 3% BSA, 0.5% Triton X-100, 0.01% Sodium deoxycholate, 0.02% sodium azide in PBS pH7.4)(Franco et al, 2013).  Cells were stained with Hoechst (Invitrogen: H1399) to label nuclei. Images of stained vessels were acquired using a Leica SP8 confocal microscope with the Leica Application Suite (LAS) software. Endothelial cell alignment in aorta was quantified by measuring the eccentricity of the immunostaining of beta-catenin of the endothelial cells (>150 cells per field of view). \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Extravasation of Evans blue dye for aortic vascular permeability\n",
      "\n",
      "\n",
      "For aortic vascular permeability measurements, Evans blue dye (20 mg/ml in saline, 2.5 µl/g mouse body weight) (Sigma E2129) was injected retro-orbitally to LPHN2 endothelial cell-specific knockout and their control littermates at 8 weeks old.  Thirty minutes post injection, animals were euthanized by an overdose of isoflurane and tissue samples including aorta were harvested and washed with PBS to remove extra fatty tissues.  Dissected aorta was prepared en face and imaged with an Olympus stereo zoom microscope. To quantify blue color associated with extravasation of Evans blue dye, the original RGB images were converted into HSV color model, and the proportion of the blue regions was calculated as the ratio of pixels with hue angles of 120°-240° to the entire aorta. \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Immunostaining and imaging of mouse muscle capillaries\n",
      "\n",
      "\n",
      "Rectus femoris muscle from murine thigh were perfusion-fixed with 4% paraformaldehyde (overnight) and the middle part of the muscle were cryo-sectioned following embedding into OCT compound and stained with primary antibodies for SMA (fluorophore-conjugated) and for CD31 followed by fluorophore-conjugated secondary antibody (Invitrogen, donkey anti-Rabbit IgG (H+L) Highly Cross-Adsorbed Secondary Antibody, Alexa Fluor 647, A31573) in PBS containing 5% BSA.  Hoechst (Invitrogen: H1399) was used to nuclear identification. Images of stained vessels were acquired using a Leica SP8 confocal microscope with the Leica Application Suite (LAS) software. \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Hindlimb ischemia model\n",
      "\n",
      "\n",
      "Mice were anesthetized and operated with a mixture of 1.5-2% isoflurane. The front hair of the right thigh was removed with Nair lotion and the local surgical area was disinfected. A 5 mm longitudinal skin incision was made in the right thigh. The distal femoral artery, proximal to the popliteal artery and saphenous artery, was explored and dissected. 6-0 silk sutures (Syneture) were double used for the ligation before excision of the arterial vessel bed between the distal end of the superficial epigastric artery and the trifurcation of the femoral artery into the descending genicular, popliteal, and saphenous branches. The ligated artery was cut between two sutures. The venous structures and accompanying peripheral nerves were kept intact. The overlying skin was closed with a 6-0 prolene (Ethicon) and disinfected. At the end of the procedure, Buprenorphine was administered subcutaneously at 0.05 mg/kg, once pre-emptively, then every 12 h for 72 h. The survival rate for this surgery was 100%.\n",
      "\n",
      "\n",
      " \n",
      "\n",
      "\n",
      "Laser Doppler perfusion imaging\n",
      "\n",
      "\n",
      "Hindlimb perfusion was assessed noninvasively in the plantar foot before, 1, 3, 7, 14, and 21 days after hindlimb ischemia by scanning laser-Doppler [model LDI2-IR modified for high resolution; Moor Instruments, Wilmington, DE]. The hindquarters were placed on the top of a heater pad during scanning to minimize variation in body temperature. Doppler perfusion of the plantar foot was assessed within anatomically defined regions of interest (ROIs; Moor Instruments). The ROI for the plantar foot consisted of the hind paw margins. Procedures for minimal preparation, scanning, and ROI selections have been described in detail previously(Zhuang et al, 2011). Low or no perfusion is displayed as dark blue, whereas the highest degree of perfusion is displayed as red. These images were quantitatively converted into histograms that represented the amount of blood flow on the x-axis and the number of pixels on the y-axis in the traced area. The average blood flow in each histogram was calculated and the LDI index was determined as the ratio of ischemic to non-ischemic hindlimb blood perfusion.\n",
      "\n",
      "\n",
      " \n",
      "\n",
      "\n",
      "Volumetric micro-CT angiogram for quantitative arteriogenesis\n",
      "\n",
      "\n",
      "After the mice were euthanized, the vasculature was flushed with 0.9% normal saline containing heparin (1000 IU/L) for 3 minutes, followed by 4% paraformaldehyde for 5 minutes at 100 mm Hg pressure. Fresh home-made 20% bismuth nanoparticles mixed in 5% gelatin were used as a micro-CT contrast agent and injected over 2 minutes with a syringe pump, as described before(Tirziu et al., 2005). The mice were then immediately chilled on ice for more than 30 min and immersion-fixed in 2% paraformaldehyde overnight. The peripheral vasculature in the ischemic hindlimb and the contralateral normal hindlimb was imaged with a high-resolution micro-CT imaging system (GE eXplore Locus SP), set to a 0.007-mm effective detector pixel size. Microview software (GE Healthcare) was used for initial reconstruction. Advanced workstation with various software was used for 3D reconstruction and segmentation of the whole hindlimb at the 21 µm spatial resolution. Using the knee as the breakpoint, we re-sliced the 3D vasculature into 200 slices both in the thigh and in the calf regions. Quantification was performed by use of a NIH Image. The data were expressed as vascular segment numbers, representing the total number of vessels, of specified diameter, counted in 200 z-sections for the thigh region and another 200 z-sections for the calf region.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Mouse treadmill fatigue test \n",
      "\n",
      "\n",
      "Mice were first acclimatized by staying on the motionless treadmill machine for 30 min on day 1 and then trained on the treadmill moving at 12 m/min for 30 min on day 2. Fatigue tests were then conducted on days 3 and 4.  Mice were placed on the treadmill and the shock rod was activated.  Treadmill speeds were increased as follows: 0 m/min, for 12 min to measure basal metabolism, 12 m/min, for1 min, then increasing at a rate of +1 m/min each 1 min until exhaustion.  Exhaustion (endpoint of treadmill cessation) was defined as the point at which mice continuously contacted with the shock rod for 5 continuous seconds or when mice stopped running 3 times within 5 seconds.  Before each testing session, Oxymax software (Columbus Instruments, Columbus, OH, USA) and open circuit indirect calorimetry treadmills (Metabolic Modular Treadmill, Columbus Instruments, Columbus, OH) were calibrated and checked for hardware malfunctions according to manufacturer instructions. \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Data availability\n",
      "\n",
      "\n",
      "The datasets produced in this study are available in the following databases:\n",
      "\n",
      "\n",
      "All data: BioStudies S-BIAD928: https://www.ebi.ac.uk/biostudies/studies?query=S-BIAD928\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Acknowledgements\n",
      "\n",
      "\n",
      "This work was supported by NIH grants RO1 HL75092 and R01 HL151469 to MAS, NHLBI 5R01HL130246-05 to SN, and the Uehara memorial foundation postdoctoral fellowship and MEXT | Japan Society for the Promotion of Science (JSPS) Overseas Research Fellowships (2014, receipt number: 805) to KT.  We thank Hamit Harun Dag (Istanbul University) for help of synthesizing SMA polymer; Shahid Mansuri (Yale University), Jeremy Herrera (University of Manchester) and David Knight (University of Manchester) for expert advice for mass spectrometry experiments; Didier Trono (École polytechnique fédérale de Lausanne) for lentiviral packaging plasmids pMD2.G (Addgene plasmid # 12259 ; http://n2t.net/addgene:12259 ; RRID:Addgene_12259) and psPAX2 (Addgene plasmid # 12260 ; http://n2t.net/addgene:12260 ; RRID:Addgene_12260);  Bryan Roth (University of North Carolina) for pcDNA5/FRT-HA-hM3D(Gq) (Addgene plasmid # 45547 ; http://n2t.net/addgene:45547 ; RRID:Addgene_45547) and pcDNA5/FRT-HA-hM4D(Gi)(Addgene plasmid # 45548 ; http://n2t.net/addgene:45548 ; RRID:Addgene_45548); Aziz Moqrich (Institut de Biologie du Dévelopement de Marseille) for GST-GINIP plasmid.  We are grateful to Peter Newman (Medical College of Wisconsin) for the kind gift of PECAM-1 antibody and helpful discussions.  Schematic models are inspired from BioRender templates (2022).\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Disclosure & competing interests statement\n",
      "\n",
      "\n",
      "The authors declare they have no conflict of interest.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "References\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Albarran-Juarez J, Iring A, Wang S, Joseph S, Grimm M, Strilic B, Wettschureck N, Althoff TF, Offermanns S (2018) Piezo1 and G(q)/G(11) promote endothelial inflammation depending on flow pattern and integrin activation. J Exp Med 215: 2655-2672\n",
      "\n",
      "\n",
      "Arshad M, Ghim M, Mohamied Y, Sherwin SJ, Weinberg PD (2021) Endothelial cells do not align with the mean wall shear stress vector. J R Soc Interface 18: 20200772\n",
      "\n",
      "\n",
      "Bae H, Cabrera-Vera TM, Depree KM, Graber SG, Hamm HE (1999) Two amino acids within the alpha4 helix of Galphai1 mediate coupling with 5-hydroxytryptamine1B receptors. J Biol Chem 274: 14963-14971\n",
      "\n",
      "\n",
      "Baeyens N, Bandyopadhyay C, Coon BG, Yun S, Schwartz MA (2016) Endothelial fluid shear stress sensing in vascular health and disease. J Clin Invest 126: 821-828\n",
      "\n",
      "\n",
      "Baeyens N, Mulligan-Kehoe MJ, Corti F, Simon DD, Ross TD, Rhodes JM, Wang TZ, Mejean CO, Simons M, Humphrey J et al (2014) Syndecan 4 is required for endothelial alignment in flow and atheroprotective signaling. Proc Natl Acad Sci U S A 111: 17308-17313\n",
      "\n",
      "\n",
      "Butler PJ, Norwich G, Weinbaum S, Chien S (2001) Shear stress induces a time- and position-dependent increase in endothelial cell membrane fluidity. Am J Physiol Cell Physiol 280: C962-969\n",
      "\n",
      "\n",
      "Camillo C, Facchinello N, Villari G, Mana G, Gioelli N, Sandri C, Astone M, Tortarolo D, Clapero F, Gays D et al (2021) LPHN2 inhibits vascular permeability by differential control of endothelial cell adhesion. J Cell Biol 220\n",
      "\n",
      "\n",
      "Campinho P, Vilfan A, Vermot J (2020) Blood Flow Forces in Shaping the Vascular System: A Focus on Endothelial Cell Behavior. Front Physiol 11: 552\n",
      "\n",
      "\n",
      "Chachisvilis M, Zhang YL, Frangos JA (2006) G protein-coupled receptors sense fluid shear stress in endothelial cells. Proc Natl Acad Sci U S A 103: 15463-15468\n",
      "\n",
      "\n",
      "Collins C, Guilluy C, Welch C, O'Brien ET, Hahn K, Superfine R, Burridge K, Tzima E (2012) Localized tensional forces on PECAM-1 elicit a global mechanotransduction response via the integrin-RhoA pathway. Curr Biol 22: 2087-2094\n",
      "\n",
      "\n",
      "Conway D, Schwartz MA (2012) Lessons from the endothelial junctional mechanosensory complex. F1000 Biol Rep 4: 1\n",
      "\n",
      "\n",
      "Conway DE, Breckenridge MT, Hinde E, Gratton E, Chen CS, Schwartz MA (2013) Fluid shear stress on endothelial cells modulates mechanical tension across VE-cadherin and PECAM-1. Curr Biol 23: 1024-1030\n",
      "\n",
      "\n",
      "Cornella N, Sancho J, Sitges-Serra A (2017) Short and Long-Term Outcomes After Surgical Procedures Lasting for More Than Six Hours. Sci Rep 7: 9221\n",
      "\n",
      "\n",
      "Dela Paz NG, Frangos JA (2019) Rapid flow-induced activation of Galphaq/11 is independent of Piezo1 activation. Am J Physiol Cell Physiol 316: C741-C752\n",
      "\n",
      "\n",
      "Dela Paz NG, Melchior B, Frangos JA (2017) Shear stress induces Galpha(q/11) activation independently of G protein-coupled receptor activation in endothelial cells. Am J Physiol Cell Physiol 312: C428-C437\n",
      "\n",
      "\n",
      "dela Paz NG, Melchior B, Shayo FY, Frangos JA (2014) Heparan sulfates mediate the interaction between platelet endothelial cell adhesion molecule-1 (PECAM-1) and the Galphaq/11 subunits of heterotrimeric G proteins. J Biol Chem 289: 7413-7424\n",
      "\n",
      "\n",
      "Erdogmus S, Storch U, Danner L, Becker J, Winter M, Ziegler N, Wirth A, Offermanns S, Hoffmann C, Gudermann T et al (2019) Helix 8 is the essential structural motif of mechanosensitive GPCRs. Nat Commun 10: 5784\n",
      "\n",
      "\n",
      "Fancher IS, Ahn SJ, Adamos C, Osborn C, Oh MJ, Fang Y, Reardon CA, Getz GS, Phillips SA, Levitan I (2018) Hypercholesterolemia-Induced Loss of Flow-Induced Vasodilation and Lesion Formation in Apolipoprotein E-Deficient Mice Critically Depend on Inwardly Rectifying K(+) Channels. J Am Heart Assoc 7\n",
      "\n",
      "\n",
      "Flock T, Hauser AS, Lund N, Gloriam DE, Balaji S, Babu MM (2017) Selectivity determinants of GPCR-G-protein binding. Nature 545: 317-322\n",
      "\n",
      "\n",
      "Franco CA, Blanc J, Parlakian A, Blanco R, Aspalter IM, Kazakova N, Diguet N, Mylonas E, Gao-Li J, Vaahtokari A et al (2013) SRF selectively controls tip cell invasive behavior in angiogenesis. Development 140: 2321-2333\n",
      "\n",
      "\n",
      "Frangos JA, McIntire LV, Eskin SG (1988) Shear stress induced stimulation of mammalian cell metabolism. Biotechnol Bioeng 32: 1053-1060\n",
      "\n",
      "\n",
      "Gagnon JA, Valen E, Thyme SB, Huang P, Akhmetova L, Pauli A, Montague TG, Zimmerman S, Richter C, Schier AF (2014) Efficient mutagenesis by Cas9 protein-mediated oligonucleotide insertion and large-scale assessment of single-guide RNAs. PLoS One 9: e98186\n",
      "\n",
      "\n",
      "Gaillard S, Lo Re L, Mantilleri A, Hepp R, Urien L, Malapert P, Alonso S, Deage M, Kambrun C, Landry M et al (2014) GINIP, a Galphai-interacting protein, functions as a key modulator of peripheral GABAB receptor-mediated analgesia. Neuron 84: 123-136\n",
      "\n",
      "\n",
      "Galie PA, Nguyen DH, Choi CK, Cohen DM, Janmey PA, Chen CS (2014) Fluid shear stress threshold regulates angiogenic sprouting. Proc Natl Acad Sci U S A 111: 7968-7973\n",
      "\n",
      "\n",
      "Givens C, Tzima E (2016) Endothelial Mechanosignaling: Does One Sensor Fit All? Antioxid Redox Signal 25: 373-388\n",
      "\n",
      "\n",
      "Grzelinski M, Pinkenburg O, Buch T, Gold M, Stohr S, Kalwa H, Gudermann T, Aigner A (2010) Critical role of G(alpha)12 and G(alpha)13 for human small cell lung cancer cell proliferation in vitro and tumor growth in vivo. Clin Cancer Res 16: 1402-1415\n",
      "\n",
      "\n",
      "Gudi S, Nolan JP, Frangos JA (1998) Modulation of GTPase activity of G proteins by fluid shear stress and phospholipid composition. Proc Natl Acad Sci U S A 95: 2515-2519\n",
      "\n",
      "\n",
      "Gudi SR, Clark CB, Frangos JA (1996) Fluid flow rapidly activates G proteins in human endothelial cells. Involvement of G proteins in mechanochemical signal transduction. Circ Res 79: 834-839\n",
      "\n",
      "\n",
      "Haidekker MA, L'Heureux N, Frangos JA (2000) Fluid shear stress increases membrane fluidity in endothelial cells: a study with DCVJ fluorescence. Am J Physiol Heart Circ Physiol 278: H1401-1406\n",
      "\n",
      "\n",
      "Humphrey JD, Schwartz MA (2021) Vascular Mechanobiology: Homeostasis, Adaptation, and Disease. Annu Rev Biomed Eng 23: 1-27\n",
      "\n",
      "\n",
      "Jung B, Obinata H, Galvani S, Mendelson K, Ding BS, Skoura A, Kinzel B, Brinkmann V, Rafii S, Evans T et al (2012) Flow-regulated endothelial S1P receptor-1 signaling sustains vascular development. Dev Cell 23: 600-610\n",
      "\n",
      "\n",
      "Kaya AI, Lokits AD, Gilbert JA, Iverson TM, Meiler J, Hamm HE (2016) A Conserved Hydrophobic Core in Galphai1 Regulates G Protein Activation and Release from Activated Receptor. J Biol Chem 291: 19674-19686\n",
      "\n",
      "\n",
      "Kroon J, Heemskerk N, Kalsbeek MJT, de Waard V, van Rijssel J, van Buul JD (2017) Flow-induced endothelial cell alignment requires the RhoGEF Trio as a scaffold protein to polarize active Rac1 distribution. Mol Biol Cell 28: 1745-1753\n",
      "\n",
      "\n",
      "Krumins AM, Gilman AG (2006) Targeted knockdown of G protein subunits selectively prevents receptor-mediated modulation of effectors and reveals complex changes in non-targeted signaling proteins. J Biol Chem 281: 10250-10262\n",
      "\n",
      "\n",
      "Lawrence C (2016) New frontiers for zebrafish management. Methods Cell Biol 135: 483-508\n",
      "\n",
      "\n",
      "Lee MJ, Evans M, Hla T (1996) The inducible G protein-coupled receptor edg-1 signals via the G(i)/mitogen-activated protein kinase pathway. J Biol Chem 271: 11272-11279\n",
      "\n",
      "\n",
      "Lee SC, Knowles TJ, Postis VL, Jamshad M, Parslow RA, Lin YP, Goldman A, Sridhar P, Overduin M, Muench SP et al (2016) A method for detergent-free isolation of membrane proteins in their local lipid environment. Nat Protoc 11: 1149-1162\n",
      "\n",
      "\n",
      "Liu SL, Sheng R, Jung JH, Wang L, Stec E, O'Connor MJ, Song S, Bikkavilli RK, Winn RA, Lee D et al (2017) Orthogonal lipid sensors identify transbilayer asymmetry of plasma membrane cholesterol. Nat Chem Biol 13: 268-274\n",
      "\n",
      "\n",
      "Lucitti JL, Jones EA, Huang C, Chen J, Fraser SE, Dickinson ME (2007) Vascular remodeling of the mouse yolk sac requires hemodynamic force. Development 134: 3317-3326\n",
      "\n",
      "\n",
      "Maleszewska M, Vanchin B, Harmsen MC, Krenning G (2016) The decrease in histone methyltransferase EZH2 in response to fluid shear stress alters endothelial gene expression and promotes quiescence. Angiogenesis 19: 9-24\n",
      "\n",
      "\n",
      "Medina R, Grishina G, Meloni EG, Muth TR, Berlot CH (1996) Localization of the effector-specifying regions of Gi2alpha and Gqalpha. J Biol Chem 271: 24720-24727\n",
      "\n",
      "\n",
      "Nazarko O, Kibrom A, Winkler J, Leon K, Stoveken H, Salzman G, Merdas K, Lu Y, Narkhede P, Tall G et al (2018) A Comprehensive Mutagenesis Screen of the Adhesion GPCR Latrophilin-1/ADGRL1. iScience 3: 264-278\n",
      "\n",
      "\n",
      "Ngai J, Methi T, Andressen KW, Levy FO, Torgersen KM, Vang T, Wettschureck N, Tasken K (2008) The heterotrimeric G-protein alpha-subunit Galphaq regulates TCR-mediated immune responses through an Lck-dependent pathway. Eur J Immunol 38: 3208-3218\n",
      "\n",
      "\n",
      "Papasergi MM, Patel BR, Tall GG (2015) The G protein alpha chaperone Ric-8 as a potential therapeutic target. Mol Pharmacol 87: 52-63\n",
      "\n",
      "\n",
      "Privratsky JR, Newman DK, Newman PJ (2010) PECAM-1: conflicts of interest in inflammation. Life Sci 87: 69-82\n",
      "\n",
      "\n",
      "Sando R, Jiang X, Sudhof TC (2019) Latrophilin GPCRs direct synapse specificity by coincident binding of FLRTs and teneurins. Science 363\n",
      "\n",
      "\n",
      "Sehnert AJ, Huq A, Weinstein BM, Walker C, Fishman M, Stainier DY (2002) Cardiac troponin T is essential in sarcomere assembly and cardiac contractility. Nat Genet 31: 106-110\n",
      "\n",
      "\n",
      "Strohbach A, Pennewitz M, Glaubitz M, Palankar R, Gross S, Lorenz F, Materzok I, Rong A, Busch MC, Felix SB et al (2018) The apelin receptor influences biomechanical and morphological properties of endothelial cells. J Cell Physiol 233: 6250-6261\n",
      "\n",
      "\n",
      "Sugden WW, Meissner R, Aegerter-Wilmsen T, Tsaryk R, Leonard EV, Bussmann J, Hamm MJ, Herzog W, Jin Y, Jakobsson L et al (2017) Endoglin controls blood vessel diameter through endothelial cell shape changes in response to haemodynamic cues. Nat Cell Biol 19: 653-665\n",
      "\n",
      "\n",
      "Tesmer VM, Kawano T, Shankaranarayanan A, Kozasa T, Tesmer JJ (2005) Snapshot of activated G proteins at the membrane: the Galphaq-GRK2-Gbetagamma complex. Science 310: 1686-1690\n",
      "\n",
      "\n",
      "Tirziu D, Moodie KL, Zhuang ZW, Singer K, Helisch A, Dunn JF, Li W, Singh J, Simons M (2005) Delayed arteriogenesis in hypercholesterolemic mice. Circulation 112: 2501-2509\n",
      "\n",
      "\n",
      "Tzima E, Irani-Tehrani M, Kiosses WB, Dejana E, Schultz DA, Engelhardt B, Cao G, DeLisser H, Schwartz MA (2005) A mechanosensory complex that mediates the endothelial cell response to fluid shear stress. Nature 437: 426-431\n",
      "\n",
      "\n",
      "Vizurraga A, Adhikari R, Yeung J, Yu M, Tall GG (2020) Mechanisms of adhesion G protein-coupled receptor activation. J Biol Chem 295: 14065-14083\n",
      "\n",
      "\n",
      "Vogel J, Niederer D, Jung G, Troidl K (2020) Exercise-Induced Vascular Adaptations under Artificially Versus Pathologically Reduced Blood Flow: A Focus Review with Special Emphasis on Arteriogenesis. Cells 9\n",
      "\n",
      "\n",
      "Wang S, Iring A, Strilic B, Albarran Juarez J, Kaur H, Troidl K, Tonack S, Burbiel JC, Muller CE, Fleming I et al (2015) P2Y(2) and Gq/G(1)(1) control blood pressure by mediating endothelial mechanotransduction. J Clin Invest 125: 3077-3086\n",
      "\n",
      "\n",
      "Wilson PT, Bourne HR (1995) Fatty acylation of alpha z. Effects of palmitoylation and myristoylation on alpha z signaling. J Biol Chem 270: 9667-9675\n",
      "\n",
      "\n",
      "Xu J, Mathur J, Vessieres E, Hammack S, Nonomura K, Favre J, Grimaud L, Petrus M, Francisco A, Li J et al (2018) GPR68 Senses Flow and Is Essential for Vascular Physiology. Cell 173: 762-775 e716\n",
      "\n",
      "\n",
      "Yamamoto K, Ando J (2013) Endothelial cell and model membranes respond to shear stress by rapidly decreasing the order of their lipid phases. J Cell Sci 126: 1227-1234\n",
      "\n",
      "\n",
      "Yamamoto K, Ando J (2015) Vascular endothelial cell membranes differentiate between stretch and shear stress through transitions in their lipid phases. Am J Physiol Heart Circ Physiol 309: H1178-1185\n",
      "\n",
      "\n",
      "Yamamoto K, Nogimori Y, Imamura H, Ando J (2020) Shear stress activates mitochondrial oxidative phosphorylation by reducing plasma membrane cholesterol in vascular endothelial cells. Proc Natl Acad Sci U S A 117: 33660-33667\n",
      "\n",
      "\n",
      "Yeh JC, Otte LA, Frangos JA (2008) Regulation of G protein-coupled receptor activities by the platelet-endothelial cell adhesion molecule, PECAM-1. Biochemistry 47: 9029-9039\n",
      "\n",
      "\n",
      "Zhou J, Li YS, Chien S (2014) Shear stress-initiated signaling and its regulation of endothelial function. Arterioscler Thromb Vasc Biol 34: 2191-2198\n",
      "\n",
      "\n",
      "Zhuang ZW, Shi J, Rhodes JM, Tsapakos MJ, Simons M (2011) Challenging the surgical rodent hindlimb ischemia model with the miniinterventional technique. J Vasc Interv Radiol 22: 1437-1446\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Figure 1. Gα proteins specific for endothelial flow responses\n",
      "\n",
      "\n",
      "A, Flow-induced endothelial alignment after Gα knockdown.  HUVECs were subjected to fluid shear stress (FSS) at a rate of 12 dynes/cm2 for 16 hours and nuclear orientation quantified as histograms showing the percentage of cells within each 10° of the flow directions from 0° to 90° (see Methods) ****: p<0.0001; one-way ANOVA with Tukey’s multiple comparisons test. B, Src family kinase activation, quantified in C. n=5 for control, Gi knockdown, Gq11 knockdown and n=3 for simultaneous knockdown of Gi and Gq11.  Values are means ± SEM. ****: p<0.0001, ***: p<0.001; one-way ANOVA with Tukey multiple comparison test.  D, Rescue of Gq/11 and Gi knockdown by re-expression of siRNA-resistant versions of the indicated proteins. ****: p<0.0001; one-way ANOVA with Tukey’s multiple comparison test. E, Amino acid sequences of Gαi1, Gαi2, and Gαi3 at the mutation sites of Gαi1 gain-of-function mutant.  F, Rescue of Gq/11 and Gi knockdown with indicated Gα proteins. Each point corresponds to one measurement averaged from >500 cells. N=4. ****: p<0.0001; one-way ANOVA with Tukey’s multiple comparisons test.  G, GINIP pulldown assay for activation of Gαi2 by FSS.  N=3. Results quantified in H.  **: p=0.0185, Student’s t-test.  I, GRK2N pulldown assay for activation of Gq.  N=4, quantified in J. *: p<0.05, Student’s t-test. K, GINIP pulldown assay for FSS-induced activation of wild type and Q306K Gαi1.  N = 4, quantified in L. *: p=0.0304, ***: p=0.0017; Student’s t-test. M, Gi2 activation after PECAM-1 knockdown. HUVECs expressing GluGlu tagged Gi2 were transfected with scrambled siRNA or PECAM-1 siRNA, exposed to FSS, and Gi2 activation assayed as described above, quantified in N. Values are means ± SEM, normalized to input Gα protein levels. ***: p<0.001; Student’s t-test, N=4.  \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Figure 2. Identifiation of the upstream flow-responsive GPCR\n",
      "\n",
      "\n",
      "A, Strategy for identification of GPCRs that bind the gain-of-function Gi1Q306K mutant but not the wild type Gi1.  B, Co-immunoprecipitation of Gi1Q306K(ins4A) with endogenous LPHN2 with and without FSS for 2 min. N=3, quantified in C. *: p=0.0374, Student’s t-test. D, Co-immunoprecipitation of LPHN2-GFP with the indicated Gα proteins containing internal GluGlu epitope tags.  E, Gi2 pulldown assay after LPHN2 knockdown.  N= 4. Results quantified in F.  ***: p<0.001, Student’s t-test.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Figure 3. Latrophilins regulate endothelial flow responses\n",
      "\n",
      "\n",
      "A, HUVECs with or without latrophilin-2 knockdown were subjected to FSS for 16 h, then fixed and stained with Hoechst (nuclei), phalloidin (F-actin) and an antibody against VE-Cadherin. Scale bar: 100µm. B, Alignment of HUVECs (each bar = 10° increments) after knockdown of latrophilin isoforms was quantified as in Fig. 1 from >2000 cells/experiment, N=3.  ****: p<0.0001; one-way ANOVA with Tukey’s multiple comparisons test. C, Localization of LPHN2-mClover3. Scale bar: 100µm. N=6. D, Gi1(Q306K) pulldown from ECs ± FSS for 5 min or for 24 hours, probed for PECAM-1. N=3, quantified in E. F, Gi1(Q306K) pulldown from ECs ± FSS for 5 min or for 24 hours probed for VE-cadherin. N=3, quantified in G. H, Activation of Src family kinases and Akt by FSS in HUVECs depleted for latrophilin isoforms. N=3-4, quantified in I & J. *: p=0.0184, **: p<0.001; one-way ANOVA with Tukey’s multiple comparisons test. K, Alignment of LPHN2-depleted HUVECs rescued by re-expression of the indicated latrophilin isoforms. ****: p<0.0001; one-way ANOVA with Tukey’s multiple comparisons test. Quantification of >2000 cells for each condition, N=3. L, LPHN2 knockdown HUVECs were rescued by re-expression of the indicated LPHN2 mutants. ****: p<0.0001, one-way ANOVA with Tukey’s multiple comparisons test. Quantification of >500 cells for each condition.  \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Figure 4, Latrophilins in flow-mediated endothelial cell morphology in vivo\n",
      "\n",
      "\n",
      "A. Representative image of HUVECs transfected with scrambled siRNA or LPHN2 siRNA.  Outlines of ECs from VE-cadherin staining are shown to next to each image. Scale Bar: 100µm. B, Quantification of junctional linearization index from A. n=30 for each condition.  **: p=0.0019, Statistics is done using two-way ANOVA with Sidak’s multiple comparison test. C, Dorsal aortas from 48hpf Tg(kdrl:ras-mCherry) WT or LPHN mutant zebrafish embryos stained for ZO-1. EC were outlined as in Methods. LPHN2 mutant indicates adgrl2a+adgrl2b.1 sgRNAs.  D&E, Quantification of cell size and eccentricity in embryos with latrophilin-2 CRISPR sgRNAs ± silent heart morpholinos (MO). N=6 per condition.  ****: p<0.0001, *: p=0.0267, **: p=0.0015, ***: p=0.0008; one-way ANOVA with Tukey’s multiple comparisons test.  F, ECs in the thoracic aorta in 18 weeks old mice at 10 weeks after tamoxifen injections, stained for β-catenin (left) and outlined (right).  Data are representative of 6 mice for each condition. Scale Bar: 100µm. G&H, quantification of cell eccentricity and junction linearity in mouse aorta.  N=4 per condition. **: p=0.0013; ****: p<0.0001; Student’s t-test. I, Aortas from mice 30 min after Evans blue dye injection, thresholded images on the right. Fractional blue area quantified in J. n=3 for each condition.  *: p=0.0419; Student’s t-test. K, in vitro endothelial permeability assay using FITC-streptavidin and biotin-conjugated fibronectin. Scale Bar: 100µm. L, Permeability was quantified by measuring FITC-streptavidin area per image field from K. n=5, *: p=0.0457, one-way ANOVA.  \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Figure 5. Cholesterol depletion activates Latrophilin-dependent signaling\n",
      "\n",
      "\n",
      "A, ECs were treated with FSS or 5 mM methyl-β-cyclodextrin (mβCD) for 1 min then incubated with D4H-mClover3, rinsed and bound mClover3 quantified as described in Methods. N=27 for each condition.  ****: p<0.0001; One-way ANOVA with Tukey multiple comparisons test. B, Phalloidin staining following short acute treatment of methyl-β-cyclodextrin (mβCD).  Scale bar: 100µm. C, Co-immunoprecipitation of Gi1Q306K(ins4A) with endogenous Latrophilin-2 (LPHN2) after 5mM MβCD for 1min. D, Activation of Src family kinases and VEGFR2 after 5mM MβCD for 1min in HUVECs ± LPHN2 knockdown. Results quantified in E and F, respectively. Values are means ± SEM. N=5. *: p=0.0305, ***: p=0.0002; one-way ANOVA with Tukey multiple comparison test.  G, Model for junctional endothelial shear stress mechanotransduction. \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Figure 6. Latrophilin-2 in flow-induced vascular remodeling \n",
      "\n",
      "\n",
      "A, Representative images of ISVs from 48 hpf embryos with F(0)/mosaic depletions of latrophilin genes or controls.  B, Quantification of diameters of all ISVs in A. n=15 per condition. *: p=0.0452, **: p=0.0150; one-way ANOVA with Tukey’s multiple comparisons test. C, Quantification of diameters of arterial and venous ISVs at 72 hpf of zebrafish. n=6 per condition. **: p=0.0013; Student’s t-test. D, Ratio of blood flow in the right ischemic vs left control foot by laser doppler, quantified in E.  N=7-8 per condition. *#: p=0.0439, *: p=0.0135, **: p=0.0028, ##: p=0.0056, #: p=0.0163; Student’s t-test. F, Micro-CT of arterial vasculature after surgery.  Representative reconstructed micro-CT images from LPHN2 ECKO and WT littermates on day 22 after surgery, quantified in G. n=8 per condition. ****: p<0.0001, ***: p=0.0007, *: p=0.0167; two-way ANOVA with Tukey’s multiple comparisons test.  H, Rectus femoris muscle from thighs of 10 weeks old mice of LPHN2 ECKO or wild-type littermates, stained for CD31 and SMA to visualize the entire vasculature and arteries. Scale Bar: 100µm. I. Quantification of capillary density in the entire stitched thigh muscle images.  n=3 for control and n=6 for Lphn2 ECKO. ****: p<0.0001; Student’s t-test. J, Schematic of mouse treadmill fatigue test.  K, Maximum running during test. n=4 for control and n=5 for Lphn2 ECKO.  **:p=0.0082; Student’s t-test.  L, Running distance for each mouse.  n=4 for control and n=5 for Lphn2 ECKO.  **: p=0.0064; Student’s t-test. M, Oxygen consumption during treadmill fatigue test.  *: p<0.0001;  two-way ANOVA with Tukey’s multiple comparisons test.  Control: N=5, LPHN2 ECKO: N=4.  N, Representative 3D reconstructed images at 20° downward angle of ECs plated on top of collagen gels and sprouting under static or flow conditions.  ECs were stained with phalloidin; pseudocolors indicate distance from the gel surface as per the color scale on the right.  O, Histograms of cell area vs. distance from gel surface in N.  P, Mean sprout length.  N=4.  ***: p=0.0006; one-way ANOVA with Tukey’s multi comparison test.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Table 1: Primer sequences used for quantitative PCR in this study\n",
      "\n",
      "\n",
      "|Gene name | Forward (5'-3') | Reverse (5'-3')|\n",
      "|--- | --- | ---|\n",
      "|Human LPHN1 | CAGTACGACTGTGTCCCCTACA | GCACCATGCGCCAGACTG|\n",
      "|Human LPHN2 | GTCCAATATGAATGTGTCCCTTACA | GCACCAAGCACCCGCCTT|\n",
      "|Human LPHN3 | GTGCAGTATGAATGTGTCCCTTACA | GCACCACGCCCCAGATTG|\n",
      "|Human KLF2 | CGGCAAGACCTACACCAAGA | TGGTAGGGCTTCTCACCTGT|\n",
      "|Human GAPDH | TGCACCACCAACTGCTTAGC | GGCATGGACTGTGGTCATGAG|\n",
      "| |  | |\n",
      "|zebrafish LPHN1a | CAGTATGAATGTGTGCCATACA | ACACCATGCCCCCGACTG|\n",
      "|zebrafish LPHN2a | GTCCAGTATGAATGTGTTCCCTACA | GCACCACGACCCCGCTTG|\n",
      "|zebrafish LPHN2b.1 | GTCCAGTATGAGTGTGTGCCATACA | TTGCACCAAGCTCCACTC|\n",
      "|zebrafish LPHN3.1 | GTTCAGTATGAGTGTGTGCCATACA | ACACCATGATCCAGCCTG|\n",
      "|zebrafish beta-actin | GATCTGGCATCACACCTTCTAC | TCTTCTCTCTGTTGGCTTTGG|\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Table 2: Sequences of sgRNAs and morpholinos used in this study\n",
      "\n",
      "\n",
      "|  | Sequence | Target | Description|\n",
      "|--- | --- | --- | ---|\n",
      "|5' | GCTGTACGAGTATGCTTCATGGG | adgrl1a | sgRNA targeting adgrl1a exon 4|\n",
      "|5' | CGACGTATAAACTACCCCATCGG | adgrl2a | sgRNA targeting adgrl2a exon 4|\n",
      "|5' | CCGTCTATGATAAACGCTCCGCC | adgrl2b.1 | sgRNA targeting adgrl2b.1 exon 4|\n",
      "|5' | AGTCCACGGCTGCGATGTATTGG | adgrl3.1 | sgRNA targeting adgrl3.1 exon 4|\n",
      "|5' | CATGTTTGCTCTGATCTGACACGCA | tnnt2a | morpholino targeting tnnt2a translation|\n",
      "|5' | CCTCTTACCTCAGTTACAATTTATA | none | control morpholino|\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "parser = DocxParser(engine=\"xml\") # docx engines: pandoc, xml\n",
    "docx_data = parser.parse(DOCX_FILES[0])\n",
    "print(docx_data.text)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 5. Unstructured\n",
    "\n",
    "It uses the `python-docx` engine and it is able to actually get proper titles or headings. It is not always perfect, but it is a better position  than others. It also has integration with LangChain."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 80,
   "metadata": {},
   "outputs": [],
   "source": [
    "from unstructured.partition.docx import partition_docx\n",
    "\n",
    "import ssl\n",
    "import urllib.request\n",
    "\n",
    "#######\n",
    "### ONLY FOR TESTING PURPOSES! ! ! !  NEVER IN PROD\n",
    "ssl._create_default_https_context = ssl._create_unverified_context\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Partition the DOCX file into elements\n",
    "elements = partition_docx(filename=DOCX_FILES[5])\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Narrative Text: Title: Targeting the transcription factor YY1 is synthetic lethal with loss of the histone demethylase KDM5C\n",
      "Text: Qian Zheng1#, Pengfei Li2#, Yulong Qiang1#, Jiachen Fan1, Yuzhu Xing1, Ying Zhang1, Fan Yang1*, Feng Li1,3* and Jie Xiong4* \n",
      "Text: 1Department of Medical Genetics, TaiKang Medical School (School of Basic Medical Sciences), Wuhan University, Wuhan 430071, China\n",
      "Text: 2Inner Mongolia Key Laboratory of Molecular Pathology, Inner Mongolia Medical University, Huhhot, Inner Mongolia, 010059, China\n",
      "Text: 3Hubei Provincial Key Laboratory of Allergy and Immunology, Wuhan University, Wuhan 430071, China\n",
      "Text: 4Department of Immunology, TaiKang Medical School (School of Basic Medical Sciences), Wuhan University, Wuhan 430071, China\n",
      "Narrative Text: #These authors contribute equally to this paper.   \n",
      "Title: Correspondence to:\n",
      "Title: Jie Xiong\n",
      "Title: E-mail: jiexiong@whu.edu.cn，Tel: +86 27 68759222\n",
      "Text: Department of Immunology, TaiKang Medical School (School of Basic Medical Sciences), Wuhan University, Wuhan 430071, China\n",
      "Title: Feng Li\n",
      "Text: E-mail: fli222@whu.edu.cn, Tel: +86 27 68759222\n",
      "Title: Department of Medical Genetics, TaiKang Medical School (School of Basic Medical \n",
      "Text: \n",
      "Title: Sciences), Wuhan University, Wuhan 430071, China.\n",
      "Text: Hubei Provincial Key Laboratory of Allergy and Immunology, Wuhan University, Wuhan 430071, China \n",
      "Title: Fan Yang\n",
      "Text: E-mail: yangfan2022@whu.edu.cn, Tel: +86 27 68759222\n",
      "Text: Department of Medical Genetics, TaiKang Medical School (School of Basic Medical Sciences), Wuhan University, Wuhan 430071, China.\n",
      "Text: \n",
      "Title: Abstract\n",
      "Narrative Text: An understanding of the enzymatic and scaffolding functions of epigenetic modifiers is important for the development of epigenetic therapies for cancer. The H3K4me2/3 histone demethylase KDM5C has been shown to regulate transcription. The diverse roles of KDM5C are likely determined by its interacting partners, which are still largely unknown. In this study, we screen for KDM5C-binding proteins and show that YY1 interacts with KDM5C. A synergistic antitumor effect is exerted when both KDM5C and YY1 are depleted, and targeting YY1 appears to be a vulnerability in KDM5C-deficient cancer cells. Mechanistically, KDM5C promotes global YY1 chromatin recruitment, especially at promoters. Moreover, an intact KDM5C JmjC domain but not KDM5C histone demethylase activity is required for KDM5C-mediated YY1 chromatin binding. Transcriptional profiling reveals that dual inhibition of KDM5C and YY1 increases transcriptional repression of cell cycle- and apoptosis-related genes. In summary, our work demonstrates a synthetic lethal interaction between YY1 and KDM5C and suggests combination therapies for cancer treatments.\n",
      "Title: Key word: KDM5C; YY1; chromatin recruitment; promoter; cancer therapy\n",
      "Text: \n",
      "Title: Introduction\n",
      "Narrative Text: Epigenetic regulators change phenotype acquisition or gene expression without altering the DNA sequence. By conferring a fitness advantage, mutational or non-mutational epigenetic reprogramming has been implicated in the evolution of tumors during malignant progression and metastasis. In addition, numerous studies have shown that aberrant epigenetic alterations contribute to cancer therapeutic resistance(Hogg et al, 2020; Miranda Furtado et al, 2019; Zafar et al, 2021). Therefore, targeting epigenetic regulators is an attractive strategy for cancer therapy.\n",
      "Narrative Text: The histone demethylase KDM5C is a member of the SMC homolog family. By specifically removing one or two methyl groups from Lys4 in histone H3, the dynamic balance in histone H3K4 methylation is maintained. KDM5C carries a composite catalytic region in which JmjN and JmjC domains are located. JmjC is critical for catalyzing the demethylation of H3K4me2/3, while the JmjN domain interacts with the JmjC domain for the complete catalytic function of KDM5C. H514 is a highly conserved amino acid that is located in the catalytic core of the JmjC domain, and its mutation eliminates KDM5C demethylase activity. KDM5C can suppress transcription by reducing the level of H3K4me2/3 via its demethylase activity; however, whether modulation of H3K4 methylation status is the only mechanism or the predominant mechanism by which KDM5C regulates gene expression is unclear.\n",
      "Narrative Text: KDM5C is frequently mutated or aberrantly expressed in various cancer types. Several studies have indicated that KDM5C is involved in regulating different biological processes in tumor cells through its histone demethylase activity and functions as a tumor suppressor. \n",
      "Text: \n",
      "Narrative Text: For example, the high-risk HPV E6 protein degrades KDM5C in cervical cancer cells, which results in super enhancers activation of the proto-oncogenes c-MET and EGFR by changing the H3K4me3/H3K4me1 ratio(Chen et al, 2018). In clear cell renal cell carcinoma cells, KDM5C inhibits the expression of genes related to glycogen metabolism by reducing H3K4me3 levels at the promoters of these genes, while inactivating KDM5C mutation confers cell resistance to ferroptosis(Zheng et al, 2021). However, the therapeutic strategy for KDM5C-deficient tumors is unknown. Intriguingly, KDM5C has also been shown to promote tumorigenesis, although the exact mechanism is unclear; in general, these observations suggest that KDM5C potentially regulates gene expression via a distinct mechanism.\n",
      "Narrative Text: To better understand the mechanisms underlying how KDM5C regulates gene expression, we screened and thus identified Yin Yang 1 (YY1) as a KDM5C-interacting partner. Targeting YY1 in KDM5C-deficient tumor cells effectively suppressed cell proliferation and tumorigenesis. Chromatin Immunoprecipitation followed by next generation sequencing (ChIP-seq) analysis with multiple engineered cell lines revealed that KDM5C is essential for YY1 chromatin recruitment, which is partially dependent on an intact JmjC domain in KDM5C. Simultaneous loss of KDM5C and YY1 led to significantly enhanced transcriptional inhibition of a number of essential cell fate genes, elevated the apoptosis rate and led to cell cycle arrest. Our study suggests a novel scaffolding function of epigenetic regulators and provides a rationale for the development of new combination treatments.\n",
      "Text: \n",
      "Title: Results\n",
      "Narrative Text: YY1 is identified as a partner of KDM5C \n",
      "Narrative Text: Previous studies have shown that KDM5C plays diverse roles in cancer progression. Indeed, a systemic The Cancer Genome Atlas (TCGA) clinical data-based analysis clearly revealed that high KDM5C expression was positively or negatively correlated with cancer patient survival, depending on the cancer type (Figure 1A), reinforcing the idea that KDM5C may exhibit a distinct regulatory effect on gene expression, prompting us to screen the interacting partners of KDM5C. Tandem affinity purification (TAP) was subsequently performed. We generated a HEK293T cell line that stably expressed SBP- and Flag-tagged KDM5C, and the lysates of these cells were incubated with streptavidin-coated beads. The bead-bound proteins were subsequently eluted with buffer containing biotin. Next, the captured proteins were incubated with anti-Flag beads, and the proteins bound to these beads were eluted and analyzed by mass spectrometry (MS). The results of the MS analysis indicated that a number of proteins potentially interact with KDM5C, including YY1 (Figure 1B), the overexpression of which has been observed in the vast majority of cancers(Gao et al, 2019; van Leenders et al, 2007). Given the pivotal role played by YY1 in the transcriptional regulation of numerous genes(Meliala et al, 2020), we focused on the interaction between KDM5C and YY1. To confirm their interaction, mutual coimmunoprecipitation (co-IP) assays were performed in HEK293T cells, and the results clearly showed that KDM5C bound to YY1 (Figure 1C). Co-IP with endogenous proteins was also performed and the results confirmed that KDM5C interacts with YY1 in mammalian cells (Figure 1D). In addition, we also performed the semi-endogenous co-IP \n",
      "Text: \n",
      "Narrative Text: with YY1 and KDM5C-Flag in 769-P renal cancer cell lines. As shown in Figure 1E, KDM5C clearly bound to YY1. To delineate the interaction, a series of KDM5C truncation mutants were constructed and used for further analysis (Figure 1F). We found that multiple regions, including the JmjN and JmjC domains, which are critical for H3K4me2/3 removal, were also involved in the interaction between KDM5C and YY1 (Figure 1G).\n",
      "Narrative Text: YY1 depletion significantly inhibits the proliferation and tumorigenicity of KDM5C-deficient tumor\n",
      "Narrative Text: Many studies have shown that YY1 functions as an oncogene because its overexpression is observed in multiple cancers(Li et al, 2022; Qiao et al, 2019; Wang et al, 2022). YY1 tethers cofactors and promotes or represses transcription, depending on its interacting partner, which includes c-myc and p53(Shrivastava et al, 1993; Sui et al, 2004), prompting us to investigate the role of the YY1 KDM5C interaction in tumorigenesis. To achieve this goal, we first selected human kidney proximal tubular epithelial cell line (HK-2), and several renal cancer cell lines (ACHN, 769-P and RCC4) with different KDM5C and YY1 protein expression levels, as described in our previous studies(Chen et al., 2018; Zheng et al., 2021) (Figure 2A). Interestingly, knocking down YY1 in KDM5C-deficient RCC4 tumor cells significantly reduced the colony formation ability of the cells, which was not observed in the KDM5C-proficient tumor cell lines ACHN and 769-P (Figure 2B and EV1B-D). Similar results were also obtained in a CCK8-based tumor cell proliferation assay (Figure 2C and EV1E-F). Notably, tumor growth was markedly inhibited when both of KDM5C and YY1 were depleted, whereas the loss of either KDM5C or YY1 only negligibly influenced ACHN cell growth (Figure 2D). These results strongly indicate that \n",
      "Text: \n",
      "Narrative Text: KDM5C might be important for YY1-related tumorigenesis, suggesting that YY1 may be a vulnerable target in KDM5C-deficient tumor cells.\n",
      "Narrative Text: KDM5C colocalizes with YY1 and contributes to YY1 genome-wide chromatin recruitment\n",
      "Narrative Text: The stimulatory effect of YY1 on tumor cell growth was at least partially dependent on the presence of KDM5C. Given that YY1 always promotes tumorigenesis by increasing oncogene transcription, it is unlikely that YY1 tethers KDM5C because KDM5C has H3K4me2/3 demethylase activity, which is associated with repressed gene expression. A recent study revealed that KDM5B, which is a member of the KDM5 demethylase family, recruited a methyltransferase to suppress antitumor immunity, prompting us to explore whether KDM5C has a scaffolding function similar to that of KDM5B(Zhang et al, 2021). We generated a stable KDM5C-depleted ACHN cell lines (Figure EV1B), and ChIP-seq assays were subsequently performed. As shown in Figure 3A, the intensity of the YY1 binding peak was reduced upon loss of KDM5C, implying that KDM5C might be a general regulator of YY1 activity by mediating its chromatin recruitment. Next, we asked to what extent KDM5C co-localizes with YY1. To evaluate the genomic co-occupancy of KDM5C and YY1, we performed the cleavage under targets and tagmentation (CUT&Tag) assay in the ACHN renal cell line, and the annotation of the YY1 and KDM5C peaks revealed an obvious overlap (n=3645) between YY1 and KDM5C binding events (Figure 3B). Further analyzed revealed a significant overlap, with 7,957 genes being concurrently bound by both YY1 and KDM5C (Hypergeometric test, P=4.87e-17). This overlap accounts for 56.32% of the total genes bound by KDM5C and 66.71% of the total genes bound by YY1. \n",
      "Text: \n",
      "Narrative Text: Moreover, a heatmap and a metaplot analysis revealed that the enrichment of YY1 was reduced after KDM5C was depleted in the peaks (n=3645) where YY1 and KDM5C were directly related (Figure 3C). Reduced enrichment of YY1 was also observed in HK-2 cells with KDM5C depletion (Figure EV1A and EV2A). These results revealed that the intensity of the YY1 binding peaks was partially dependent on the KDM5C protein level in both nontumoral cell and tumor cells, suggesting that KDM5C mediated YY1 chromatin recruitment may represent a universal mechanism of action. The localization of the KDM5C-dependent YY1 binding peaks was subsequently analyzed, and the results revealed that most of the peaks were enriched in promoters (Figure 3D), implying that KDM5C may regulate gene expression via a distinct mechanism.\n",
      "Narrative Text: An intact catalytic domain but not the demethylation activity of KDM5C is required for YY1 recruitment\n",
      "Narrative Text: Since our results suggested that KDM5C functions as a scaffold and that KDM5C is essential for YY1 binding to a promoter, we asked, whether the demethylase activity of KDM5C is involved in YY1 recruitment. We generated ACHN cell lines that stably expressed KDM5C-WT or its enzymatically inactive mutant H514A and performed ChIP-seq to evaluate the effects of KDM5C or its mutants on YY1 recruitment (Figure EV1B). As shown in Figure 4A, ectopic KDM5C WT but not the KDM5C-H514A mutant enhanced YY1 chromatin binding. A similar trend was also observed in HK-2 cells (Figure EV2A). Because the primary function of KDM5C is to catalyze the removal of methyl groups from H3K4me2/3, we evaluated whether H3K4me3 abundance plays a role in YY1 recruitment.\n",
      "Narrative Text: We first performed ChIP-seq of H3K4me3 in ACHN shKDM5C cells and analyzed the \n",
      "Text: \n",
      "Narrative Text: change in the signal intensity of this mark. Increased H3K4me3 enrichment was clearly observed in the KDM5C/YY1 overlapping region (Figure 4B), but the opposite trend of YY1 recruitment was observed in KDM5C-depleted cells (Figure 3C). To further understand the relationship between YY1 recruitment and H3K4me3, we focused on the overlapping regions of YY1 and H3K4me3 (n=11087), which were classified into KDM5C demethylase-dependent peaks (n=9943) and KDM5C demethylase-independent peaks (n=1144), as shown in Figure 4C. Reduced YY1 ChIP-seq signals were observed in almost all of the overlapping YY1 and H3K4me3 regions (Figure 4D-E), regardless of the effect of KDM5C on H3K4me3 intensity. We therefore hypothesized that an intact KDM5C JmjC domain, rather than KDM5C histone demethylase activity, is essential for YY1 recruitment.\n",
      "Narrative Text: To further explore the mechanism underlying the effects of KDM5C, we evaluated the interaction between KDM5C-H514A and YY1 and found that the mutation did not influence their interaction (Figure EV3), indicating that the effect of KDM5C-H514A on YY1 chromatin recruitment was unlikely to be caused by disrupted protein binding activity. Because the catalytic composite JmjN/C domain binds the substrate H3 tail harboring methylated H3K4 and since a previous study revealed that none of the KDM5C PHD domains exhibited strong binding affinity for unmethylated or methylated histones(Iwase et al, 2007), we speculated that the replacement of the histidine residue at position 514 with an alanine residue destroyed the integrity of the JmjN/C domain and influenced the chromatin-binding activity of KDM5C. To test this hypothesis, we expressed KDM5C-WT or KDM5C-H514A in HEK293T cells and isolated chromatin-bound proteins. Interestingly, although the protein levels in whole-cell lysates were indistinguishable, H154A exhibited obviously reduced \n",
      "Text: \n",
      "Narrative Text: chromatin-binding capacity (Figure 4F), which might help explain why KDM5C-H514A failed to recruit YY1 to chromatin. In addition, tumor cell proliferation was markedly inhibited only when both of KDM5C and YY1 were depleted in the KDM5C-proficient tumor cells ACHN and 769-P (Figure 4G-H and EV1B-C). The inhibitory effect was blunted when KDM5C-WT was re-expressed; in contrast, KDM5C-H514A only moderately restored the growth of ACHN and 769-P double knockdown cells. A similar trend was also observed in the colony formation assay (Figure 4J). In KDM5C-deficient RCC4, cells overexpression of KDM5C-WT largely restored cell sensitivity to YY1 depletion (Figure 4I and EV1D). These result collectively suggest that an intact catalytic domain of KDM5C is required for YY1 recruitment.\n",
      "Narrative Text: KDM5C cooperates with YY1 to regulate essential cell fate gene expression\n",
      "Narrative Text: To systematically evaluate the effect of KDM5C-mediated YY1 recruitment on gene expression, we performed RNA-seq with shKDM5C, shYY1 or double-knockdown ACHN cells. As shown in Figure 5A-B, the loss of KDM5C or YY1 led to the upregulated or downregulated expression of several genes. In contrast, the simultaneous loss of KDM5C and YY1 resulted in altered transcription of thousands of genes (1511 genes whose expression was downregulated versus 797 genes whose expression was upregulated, |log2FC|≥1.5, p value<0.05, Figure 5C), indicating that YY1 recruitment by KDM5C likely contributed to gene expression activation. A similar pattern was also observed via RNA-seq with HK-2 nontumoral cells (Figure EV4). Compared with the loss of KDM5C or YY1 alone, double knockdown caused transcriptional repression of a sharply increased number of genes (1099 newly identified downregulated genes in total), as shown in Figure 5D. \n",
      "Text: \n",
      "Narrative Text: Given that depletion of both YY1 and KDM5C led to significant suppression of tumor cell proliferation (Figure 4), we next investigated the gene expression patterns by analyzing the RNA-seq results and found that the expression of several cell cycle- and cell apoptosis-related genes, including JAM3, KNTC1, SMC2, SMC4, LMNB1 and FOXM1, was markedly downregulated in the double-knockdown cell group (Figure 5E). These results were verified by qPCR in ACHN cells (Figure 5F).\n",
      "Narrative Text: We compared the RNA-seq results with our CUT&Tag data, and found that almost half of the downregulated genes (49.96%) in the KDM5C/YY1 double-knockdown group were annotated in the genomic regions where YY1 and KDM5C overlapped, as shown in Figure EV5B. We then analyzed the YY1 peaks at the promoters of individual genes via the IGV browser. As shown in Figure EV5C-D, KDM5C depletion reduced theYY1 peak intensity at the promoters of the indicated genes, and the presence of KDM5C increased the amount of YY1 recruited to these promoters. KDM5C-H514A marginally increased the YY1 intensity in the indicated regions. Moreover, ChIP-qPCR was performed in HK-2 cells and the result revealed that YY1 enrichment at the promoter region of the indicated genes was increased after KDM5C-WT was expressed, whereas KDM5C-H514A failed to increase YY1 binding (Figure EV2C). These results strongly suggest that KDM5C increases the expression of essential cell-fate genes by recruiting YY1 to their promoters. \n",
      "Narrative Text: Simultaneous loss of KDM5C and YY1 significantly promotes cell cycle arrest and cell apoptosis\n",
      "Narrative Text: Previous studies have shown that SMC2/4, LMNB1, and KNTC1 are involved in cell cycle arrest and apoptosis regulation. To confirm the synergistic effect of KDM5C and YY1 \n",
      "Text: \n",
      "Narrative Text: on tumorigenicity and cancer cell proliferation, we analyzed the cell cycle progression and apoptosis rate of tumor cells in which KDM5C, YY1 or both had been depleted. We first measured the ratio of renal cancer cells in each phase of the cell cycle, and the results indicated that the loss of both KDM5C and YY1 caused an increased ratio of 769-P tumor cells arrested in the G2/M phase. Similar results were also observed in KDM5C-deficient RCC4 cells after YY1 depletion (Figure 6A). Moreover, double knockdown of KDM5C and YY1 but not loss of either gene individually greatly increased the percentage of apoptotic cells in 769-P tumors. In addition, the absence of YY1 in RCC4 cells induced a significant increase in the percentage of apoptosis cells (Figure 6B). These results confirmed the inhibitory effect of YY1 depletion on KDM5C-deficient cells (Figure 4). Taken together, our study revealed a synergistic regulatory effect of KDM5C and YY1 on cancer cell proliferation and cell cycle progression.\n",
      "Text: \n",
      "Title: Discussion\n",
      "Narrative Text: As a member of the GLI-Kruppel family of zinc finger DNA-binding proteins, YY1 has been proven to regulate genes either by directly binding to a promoter or indirectly by associating with chromatin-remodeling proteins or histone modifiers (Gordon et al, 2006; Khachigian, 2018). YY1 is frequently overexpressed in many cancer types and reshapes the oncogenic gene expression network(Bonavida & Kaufhold, 2015; Sarvagalla et al, 2019). Although YY1 can directly bind to a promoter DNA sequence, the mechanism underlying its recruitment to chromatin is still not fully understood. In addition, although targeting YY1 might be an attractive therapeutic strategy, a robust biomarker to analyze its effect is lacking. In this study, we found that YY1 is a KDM5C-associated protein, and the JmjC domain of KDM5C was essential for the KDM5C–YY1 interaction. Moreover, we demonstrated that KDM5C mediates the chromatin recruitment of YY1, which is important for YY1 binding to promoter regions and the transcription of cell fate-related genes. Importantly, targeting YY1 effectively inhibited the proliferation and tumorigenicity of tumor cells with low KDM5C expression. Considering these findings, we propose the following model: YY1 chromatin recruitment is at least partially dependent on the presence of KDM5C, and in tumor cells with high KDM5C expression, a small amount of YY1 can be recruited to chromatin even after KDM5C is knocked down. In contrast, depleting YY1 in low KDM5C-expressing tumor cells completely eliminated chromatin-bound YY1 and promoted tumor regression (Figure 7).\n",
      "Narrative Text: As a specific H3K4 histone demethylase, KDM5C plays dual roles as both a proto-oncogene and tumor suppressor(Chen et al., 2018; Ji et al, 2015; Shen et al, 2016; Shen \n",
      "Text: \n",
      "Narrative Text: et al, 2021). KDM5C interacts with specific transcription factors, such as MYC(Outchkourov et al, 2013; Secombe et al, 2007), and downregulates gene expression by reducing H3K4me2/3 levels at enhancer or promoter regions. However, studies, including our previous reports, have shown that KDM5C also upregulates the expression of certain genes, but the relevant molecular mechanism of KDM5C effect on gene expression is unclear(Outchkourov et al., 2013; Scandaglia et al, 2017; Zheng et al., 2021). A recent study showed that the histone demethylase KDM5B recruited the H3K9 methyltransferase SETDB1 to suppress antitumor immunity in a KDM5B demethylase-independent manner, implying that the scaffolding function of this epigenetic regulator was important in certain contexts, a finding that has been appreciated to a lesser degree in prior studies(Zhang et al., 2021). Our finding that KDM5C recruited YY1 to chromatin in a histone demethylase-independent manner reinforces the evidence suggesting that the scaffolding function of epigenetic regulators might be critical for tumorigenesis. Indeed, epigenetic inhibitors targeting the catalytic domain often show limited efficacy and intolerable toxicity, and our findings strongly suggest that targeting the scaffolding function should be considered in the development of new cancer prevention strategies and therapies.\n",
      "Narrative Text: Abnormalities in enzymes that regulate epigenetic modifications are important drivers of tumor occurrence and cancer progression(Dawson, 2017). Therefore, epigenome-based therapies might be developed to block the dysregulated expression of proto-oncogenes in tumor cells and increase antitumor responses(Miranda Furtado et al., 2019; Nebbioso et al, 2018). KDM5C has been shown to be frequently mutated or aberrantly expressed in various cancers. The degree of YY1 chromatin recruitment was decreased in \n",
      "Text: \n",
      "Narrative Text: KDM5C-deficient tumor cells, and further inhibition of YY1 expression might completely suppress YY1-directed transcription and significantly impair the proliferation of tumor cells. Furthermore, simultaneously targeting KDM5C and YY1 effectively inhibited tumor growth in KDM5C-proficient tumor cells. In recent years, the induction of synthetic lethality has been applied to the identification of novel drug targets, opening new avenues to tumor-targeted therapy(Nijman & Friend, 2013). Considering the definition of synthetic lethality and the results presented here, we propose that KDM5C and YY1 form a synthetically lethal gene pair, providing a rationale for the clinical treatment of patients with KDM5C-deficient cancer by targeting YY1.\n",
      "Narrative Text: In summary, our work provides mechanistic insight into YY1 chromatin recruitment and suggests an important role for the scaffolding function of the histone demethylase KDM5C. Because proteolysis-targeting chimeras (PROTACs) are used to target and degrade proteins with important scaffolding functions and serve as a potential therapeutic strategy in cancer, future work focusing on the design and synthesis of KDM5C PROTAC drugs is important, and further studies in this research direction are warranted.\n",
      "Text: \n",
      "Title: Methods\n",
      "Title: Vectors and plasmids\n",
      "Narrative Text: The pHAGE vector carries a puromycin resistance cassette, endowing the transfected cells with resistance to puromycin, hence enabling us to select and generate stable cells. The overexpression vectors used in our context (KDM5C or YY1 with HA or Flag tags and their truncated constructs) were constructed using pHAGE for mammalian cell expression. The KDM5C enzymatically inactive mutant H514A was generated via site-directed mutagenesis PCR from the pHAGE-KDM5C plasmid. shRNAs targeting human KDM5C and YY1 were constructed using pLKO.1. The construction of pLKO.1 is detailed on the addgene website. All of the constructs were validated by DNA sequencing. The targeted sequences of KDM5C and YY1 genes are as follows:\n",
      "Title: shCtrl: 5’-CAACAAGATGAAGAGCACCAA-3’; \n",
      "Title: shKDM5C-1: 5’-AGTACCTGCGGTATCGGTATA-3’; \n",
      "Title: shKDM5C-2: 5’-GCCACACTTGAG GCCATAATC-3’; \n",
      "Title: shYY1-1: 5’-GACGACGACTACATTGAACAA-3’; \n",
      "Title: shYY1-2: 5’-GCCTCTCCTTTGTATATTATT-3’. \n",
      "Title: Cell culture\n",
      "Narrative Text: The human kidney proximal tubular epithelial cell line HK-2 and human renal cancer cell lines (ACHN, 769-P and RCC4) were obtained from the American Type Culture Collection (ATCC). HEK293T cells were obtained from the Institute of Cellular Resources, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences. HK-2, ACHN, \n",
      "Text: \n",
      "Narrative Text: RCC4, and HEK293T cells were cultured in high-glucose Dulbecco’s modified Eagle’s medium (DMEM, HyClone) while 769-P cells were cultured in Roswell Park Memorial Institute-RPMI medium(1640, HyClone) and both media were supplemented with 10% fetal bovine serum (FBS, Cell-box, AUS-01S-02) in a humidified atmosphere containing 5% CO2 at 37 ℃. The cell lines were authenticated by STR profiling, and mycoplasma testing was performed monthly with a MycoBlue Mycoplasma Detector (Vazyme, D101-01).\n",
      "Narrative Text: For transient protein expression, the cells were transfected with the indicated constructs via NEOFECT DNA transfection reagent (NeoFect, TF20121201) according to the manufacturer’s instructions. After 48 h of transfection, the cells were harvested for the following assays. To generate stable overexpression or knockout cell lines, appropriate constructs and two packaging plasmids psPAX2 and pMD2.G were cotransfected into HEK293T cells. After 60 h, the lentivirus particles from the culture medium were collected and filtered with 0.45 µm filter, and then added to the host cells along with 8 μg/mL polybrene (Solarbio, H8761). Infected cells were then selected in puromycin (Solarbio, P8230)-containing medium. The protein levels of the target genes were determined via western blotting with the corresponding antibodies. To generate the double-knockdown cells, we selected a GFP-labeled KDM5C knockdown plasmid (pLKO.1-GFP) to infect cells depleted of YY1, and then screened the cells for puromycin resistance and green fluorescence. To restore KDM5C or H514A in knockdown cells, we inserted a hygromycin resistance gene into pHAGE-KDM5C/H514A-puro, which could be used to screen cells expressing KDM5C/H514A. The cells involved in the cell cycle and apoptosis assays did not contain fluorescence, which might have affected the results. We confirmed protein \n",
      "Text: \n",
      "Title: expression via Western blotting. \n",
      "Title: RNA extraction and qRT-PCR\n",
      "Narrative Text: Total RNA was extracted using the RNA isolater Total RNA Extraction Reagent (Vazyme, R401-01) according to the manufacturer’s instructions. Reverse transcription reactions were performed by using ReverTra Ace qPCR RT Master Mix with gDNA Remover (TOYOBO, FSQ-301). RT-PCR was performed with ChamQ SYBR qPCR Master Mix (Vazyme, Q311-02) on a CFX ConnectTM Real-Time System (Bio-Rad). The housekeeping gene expression levels of β-actin or GAPDH were used for normalization. Each reaction was performed in triplicates. For quantification of gene expression, the 2-△△Ct method was used. The primer sequences of indicated genes were as follows:\n",
      "Title: h-JAM3-RT \n",
      "Title: Forward : 5’-GAGACTCAGCCCTTTATCGC-3’;\n",
      "Title: Reverse : 5’- TGTTGCCATCTTGCCTACTG-3’; \n",
      "Title: h-KNTC1-RT\n",
      "Title: Forward : 5’-CTCACGAAGTTGCACAGG-3’;\n",
      "Title: Reverse : 5’-GCCTCCAGCTCTTGTTTG-3’;\n",
      "Title: h-SMC2-RT\n",
      "Title: Forward : 5’-GTTTTGGATGGTCTGGAGTT-3’;\n",
      "Title: Reverse : 5’-ATTGGAGCAGGTTTGAAGAG-3’;\n",
      "Title: h-SMC4-RT\n",
      "Title: Forward : 5’-TCGACCACCTAAGAAAAGTTG-3’;\n",
      "Text: \n",
      "Title: Reverse : 5’-CCATGAAGTAAAGGGGAGTG-3’;\n",
      "Title: h-LMNB1-RT\n",
      "Title: Forward : 5’-CTGTCTCCAAGCCCTTCTT-3’;\n",
      "Title: Reverse : 5’-ACTCGCCTCTGATTCTTCC-3’;\n",
      "Title: h-FOXM1-RT\n",
      "Title: Forward : 5’-GCTGCCCCTTCCTGTTC-3’;\n",
      "Title: Reverse : 5’-TGGACTCTGCCACTTCCTT-3’;\n",
      "Title: h-GAPDH-RT \n",
      "Title: Forward : 5’-ACATCAAGAAGGTGGTGAAG-3’;  \n",
      "Title: Reverse : 5’-CTGTTGCTGTAGCCAAATTC-3’;\n",
      "Title: Western blotting\n",
      "Narrative Text: The cell pellets were lysed with RIPA buffer (Beyotime, P0013B) containing a protease inhibitor mixture cocktail (MedChemExpress, HY-K0010). Then, the cells were centrifuged at 4 ℃ at 13,000 rpm for 10 min to remove insoluble cellular components. Total protein concentrations were measured with a BCA protein assay kit (Beyotime, P0012). Proteins (20–100 μg) were separated via 8-15% sodium dodecyl sulphate-polyacrylamide gel electrophoresis (SDS-PAGE) and transferred to nitrocellulose membranes (NC, PALL, 66485) in Tris-glycine buffer. The blots were blocked with 5% nonfat milk then incubated with diluted primary antibodies at 4 ℃ overnight. Then, appropriate horseradish peroxidase (HRP)-conjugated secondary antibodies were incubated with the blots for 1 h at room temperature. The signals were visualized with an enhanced chemiluminescence \n",
      "Text: \n",
      "Narrative Text: detection kit (ECL, Vazyme, E411-05) and exposured to X-ray film (Sigma, America). The primary antibodies used were purchased from commercial sources: anti-KDM5C (Cell Signaling Technology, D29B9; BETHYL, A301-034A; Proteintech, 14426-1-AP), anti-YY1 (Cell Signaling Technology, D5D9Z), anti-Flag (Proteintech, 20543-1-AP), anti-HA (Proteintech, 66006-2-Ig), anti-GAPDH (Proteintech, 60004-1-Ig), anti-β-actin (Proteintech, 66009-1-Ig), anti-H3 (Cell Signaling Technology, 2650) and anti-H3K4me3 (Cell Signaling Technology, 9751).\n",
      "Title: Coimmunoprecipitation (co-IP)\n",
      "Narrative Text: For the immunoprecipitation assay, appropriately constructed plasmids were transfected into the HEK293T and 769-P cell lines. After 48 h of transfection, the cells were harvested and then lysed in IP lysis buffer (Beyotime, P0013) with EDTA-free protease inhibitor cocktail (MedChemExpress, HY-K0010) for 30 min on ice, sonicated, and then insoluble proteins were removed by centrifugation at 4 ℃ at 12000 rpm for 10 min. An aliquot of the cleared cell lysate was collected as an input control. The remaining lysates were subsequently incubated with anti-HA or anti-Flag magnetic beads (MedChemExpress, HY-K0201, HY-K0207) overnight at 4 ℃ with gentle shaking. \n",
      "Narrative Text: For endogenous protein interactions, the cell lysates were processed as described above. One tenth of each lysates was used as the input, half of the remainder was incubated with KDM5C antibody, and the other half was incubated with homologous IgG for 6 hours at 4 ℃. The lysates were then mixed with ProteinA/G agarose and incubated overnight at 4 ℃.\n",
      "Text: \n",
      "Narrative Text: Magnetic beads or agarose beads with a complex of antigen and antibody were washed four times in TBS-T buffer (100 nM Tris-HCl, 0.9% NaCl, 0.1% Tween-20). The proteins pulled down by the specified antibodies were eluted with 2×SDS loading buffer through boiling the beads, and then verified by Western blotting.\n",
      "Title: Tandem affinity purofication and mass spectrometry (TAP-MS)\n",
      "Narrative Text: HEK293T cells overexpressing SBP-Flag-KDM5C were harvested for Tap-MS. Cell pellets were lysed with IP lysis buffer (Beyotime, P0013), and the supernatant was subsequently incubated with strep beads overnight at 4 ℃. After three washes, SBP-linked protein were elutied with biotin solution, and then incubated with anti-Flag magnetic beads (MedChemExpress, HY-K0207). After enrichment of the target proteins, we washed the beads to remove nonspecific proteins and then performed SDS-PAGE on the products of the two groups. All of the lanes of the gel were collected for mass spectrometry analysis.\n",
      "Title: Proliferation ability assay（CCK8）\n",
      "Narrative Text: The cells were seeded in a 96-well plate at 30-50% confluence according to the cell growth rate. When the cells just attached at the initial point, the OD450 nm was read every 24 h via a cell counting kit 8 (CCK8, MedChemExpress, HY-K0301) according to the manufacturer’s instruction. The CCK8 solution was diluted with complete medium, incubated with the cells for 1-2 h, and then read with a microplate reader (BioTek Epoch, America) at 450 nm. The cell viability assay was performed in triplicate and repeated three times. Cell growth curves were drawn and analyzed via GraphPad Prism software V9.4.1.\n",
      "Title: Colony formation assay\n",
      "Narrative Text: The cells were seeded into a 6-well plate at the same cell density. Each well contained 1000 cells, which were cultivated in a cell incubator. Fresh cell culture medium was added every three days. After four weeks (ACHN and 769-P) or eight weeks (RCC4), the cell culture medium was removed. The cells were gently washed with PBS, fixed with 4% paraformaldehyde for 20 min at 37 ℃, and stained with 0.1% crystal violet in PBS. The assay was performed in triplicate and repeated two or three times. \n",
      "Title: Cell cycle assay\n",
      "Narrative Text: The cells were gently collected at the exponential growth phase, washed with cold PBS, and then resuspended in 70% ethanol for more than 30 min at 4 ℃. Then, the cells were washed with cold PBS, incubated with propidium iodide(PI) solution (Byotime, C1052) and protected from light at 37 ℃ for 30 min. Red fluorescence was detected via flow cytometry (LSRFortessaX20, BD) at Ex=488 nm, and then analyzed with Flowjo. The assay was repeated two or three times.\n",
      "Title: Cell apoptosis assay\n",
      "Narrative Text: In accordance with the instructions of the Annexin V-FITC/PI apoptosis kit (Multi Sciences, AP101), we collected 1-10×10^5 cells and resuspended them in 500μl of 1×Binding buffer, and add 5 μl of Annexin V-FITC and 10 μl of PI to each sample. The samples were mixed gently and protected from light for 5 min, then Annexin V-FITC (Ex = \n",
      "Text: \n",
      "Narrative Text: 488 nm; Em = 530 nm) and PI (Ex = 535 nm; Em = 615 nm) were detected via flow cytometry (LSRFortessaX20, BD) with. The results were analyzed via Flowjo.\n",
      "Title: Isolation of chromatin-bound proteins from subcellular fractions\n",
      "Narrative Text: After the transfection of HEK293T cells with empty vector, Flag-KDM5C or Flag-H514A, we collected the cells and lysed them with E1 buffer (50 nM HEPES-KOH, 140 nM NaCl, 1mM EDTA, 10% glycerol, 0.5% NP-40, 0.25% Triton X-100, 1mM DTT, Protease inhibitor cocktail). The cells were subsequently centrifuged 1,100× g at 4 ℃ for 2 min after which the supernatant was obtained as the cytoplasmic fraction. Next, we resuspended the pellet in E2 buffer (10nM Tris-HCl, 200mM NaCl, 1mM EDTA, 0.5mM EGTA, Protease inhibitor cocktail). After centrifugation, the supernatant was obtained as the nuclear fraction. We resuspended the pellet in E3 buffer (500 mM Tris-HCl, 500mM NaCl, protease inhibitor cocktail) and sonicated it on ice for 5 min, 2 s ON/45 s OFF. The supernatant was centrifuged at 4 ℃ for 10 min at 16,000× g, and transferred to fresh microcentrifuge tubes. The protein concentrations were analyzed via Western blotting.\n",
      "Narrative Text: Chromatin immunoprecipitation followed by next-generation sequencing (ChIP-seq)\n",
      "Narrative Text: The cells were plated in 10 cm dishes until they reached confluence and were then fixed with 37% fresh formaldehyde to a final concentration of 1% for 13 min at room temperature with rotation, after which the reaction was quenched with 0.125 mM glycine for 10 min. The final number of Chip-seq cells in each group was 4×107.\n",
      "Narrative Text: The cells were washed three times with ice-cold PBS and then harvested bwith a cell \n",
      "Text: \n",
      "Narrative Text: scraper and lysed in lysis buffer 1 (50mM HEPES-KOH, 140mM NaCl, 1mM EDTA, 10% glycerin, 0.5% NP-40, 0.25% TritonX-100), lysis buffer 2 (pH 8.0 10mM Tris-HCl, 200 mM NaCl 1mM EDTA, 0.5mM EGTA), and lysis buffer 3 (pH 8.0 10mM Tris-HCl, 100mM NaCl, 1mM EDTA, 0.5mM EGTA, 0.2% N-lauroyl sarcosine sodium salt). The chromatin was sonicated in lysis buffer 3 to ensure that the size of the chromatin fraction was between 200-500bp. 80 μL of each sample was transferred to a clean tube as the input DNA. The chromatin sample was diluted with ChIP dilution buffer (20mM Tris-HCl, 150mM NaCl, 1mM EDTA, 1% TritonX-100). 6 μL of antibody was added to each tube and the mixturewas incubated at 4 ℃ overnight. To each tube, 80μL of protein G Sepharose 4 Fast Flow (GE Healthcare, 17061805) was added, and the immune complexes were collected by incubation at 4 ℃ for at least 6 h with gentle rotation. The bound complexes were washed once with each of the following buffers: low-salt wash buffer (0.1% SDS, 1% TritonX-100, 2 mM EDTA, 20 mM Tris-HCl pH 8.0, 150 mM NaCl), high-salt wash buffer (0.1% SDS, 1% Triton X-100, 2 mM EDTA, 20 mM Tris-HCl pH 8.0, 500 mM NaCl), LiCl wash buffer (0.25 M LiCl, 1% NP-40, 1mM EDTA, 10 mM Tris-HCl pH 8.0) and TE buffer (10 mM Tris-HCl pH 8.0, 1mM EDTA). The beads were incubated at 65 ℃ with TE buffer containing proteinase K, RNase and SDS overnight, and the DNA was extracted using phenol/chloroform. ChIP-seq was performed by Beijing Novogene Biotech Co., Ltd.\n",
      "Title: ChIP-qPCR\n",
      "Narrative Text: The qPCR templates were obtained from ChIP. For method details, please refer to “qRT-PCR”. We calculated the fold enrichment of those genes via YY1-ChIP, and IgG was \n",
      "Text: \n",
      "Narrative Text: used as a negative control. The calculation formula is as followed:\n",
      "Text: ΔCt [normalized ChIP] = (Ct [ChIP] - (Ct [Input] -Log2 (Input Dilution Factor))\n",
      "Narrative Text: ΔΔCt [ChIP/NIS] = ΔCt [normalized ChIP] - ΔCt [ IgG ]\n",
      "Title: Fold Enrichment = 2^ (-ΔΔCt [ChIP/NIS])\n",
      "Narrative Text: Primer sequences used in ChIP-qPCR are as followed:\n",
      "Title: JAM3\n",
      "Title: Forward : 5’-TGGTAACTGGGGCGGGTCGCA-3’;\n",
      "Title: Reverse : 5’-AGCTTCGGCCCGCTCTGCTTC-3’;\n",
      "Title: KNTC1\n",
      "Title: Forward : 5’-CAACACCGCCTTACTTCC-3’;\n",
      "Title: Reverse : 5’-CCGGGACTCTGAACTATGGC-3’;\n",
      "Title: SMC2\n",
      "Title: Forward : 5’-GGAGCTACCATTATCGAGAC-3’;\n",
      "Title: Reverse : 5’-CTTGACGCACCCCAAAAG-3’;\n",
      "Title: SMC4\n",
      "Title: Forward : 5’-AAAAGAATCCCTCGCTCTTC-3’;\n",
      "Title: Reverse : 5’-TGACAGCAAGAGAAATCTCC-3’;\n",
      "Title: LMNB1\n",
      "Title: Forward : 5’-TTGCCCAAGGGCCAGATTTTA-3’;\n",
      "Title: Reverse : 5’-AATGCAAGACGAGGGTGACT-3’;\n",
      "Title: FOXM1\n",
      "Title: Forward : 5’-ATTTTCCCACAGTGAACGAT\n",
      "Text: \n",
      "Title: Reverse : 5’-AACCTTGTCTGCCATTGTATC\n",
      "Title: Cleavage Under Targets and Tagmentation assay\n",
      "Narrative Text: The Cleavage Under Targets and Tagmentation (CUT&Tag) assay was conducted according to the manual instructions of Hyperactive Universal CUT&Tag Assay Kit for Illumina Pro (Vazyme, TD904). A ChIP-grade anti-KDM5C antibody (Bethyl, A301-034A) and anti-YY1 (CST, D5D9Z) were used for immunoprecipitation. Rabbit IgG (ABclonal, AC042) was used as a negative control. The library was constructed with the TruePrep Index Kit V2 for Illumina (Vazyme, TD202). \n",
      "Title: ChIP-seq and CUT&Tag data analysis\n",
      "Narrative Text: The raw data (raw reads) in fastq format were first processed via fastp software to obtain clean data, from which reads containing adapters were removed. The reference genome hg38 and gene model annotation files were downloaded directly from the genome website. The index of the reference genome was built using Bowtie2, and the clean reads were aligned to the reference genome with Bowtie2. After the reads were mapped to the reference genome, MACS2 peak-calling software was used to identify regions enriched with IP over the background. A q-value threshold of 0.05 was used for all of the datasets. Only peaks with counts greater than 200 were retained in the CUT&Tag data. Bedtools reldist and bedtools intersect were used to compare two Chip-seq or CUT&Tag datasets in the global or promoter region. Motif analysis was performed via Homer. Chipseeker was used to analyze genomic distribution.\n",
      "Title: RNA-seq and data analysis\n",
      "Narrative Text: Total RNA was extracted from the corresponding cells. The concentration was measured by Qubit 3.0 fluorometer (Life Technologies) and RNA purity was checked via the kaiaoK5500®Spectrophotometer (Kaiao, Beijing, China). Sample integrity was assessed using an Agilent 2100 RNA Nano 6000 assay kit (Agilent Technologies). RNA-seq was performed by Annoroad Gene Technology Corporation (Beijing).\n",
      "Narrative Text: Briefly, sequencing libraries were constructed using the NEBNext Ultra RNA Library Prep Kit for Illumina (NEB, USA). Clustering of the index-coded samples was performed on a cBot cluster generation system using the HiSeq PE Cluster Kit v4-cBot-HS (Illumina) according to the manufacturer’s instructions. After cluster generation, the libraries were sequenced on an Illumina platform and 150 bp paired-end reads were generated.\n",
      "Narrative Text: For the data analysis, an index of the reference genome was built using HISAT2 v2.0.5 and paired-end clean reads were aligned to the reference genome via HISAT2 v2.0.5. FeatureCounts v1.5.0-p3 was used to count the numbers of reads mapped to each gene. Differential expression analysis of the two groups was performed using the DESeq2 R package. Heatmaps were generated via pheatmap (v1.0.12) package in R.\n",
      "Narrative Text: Xenograft assay\n",
      "Narrative Text: All animal experiments were approved by the Animal Research Ethics Board of Wuhan University (WAEF-2022-0113) and confirmed by the animal experimental guidelines. Four-week-old female BALB/c nude mice were purchased from Beijing Vital River Laboratory \n",
      "Text: \n",
      "Narrative Text: Animal Technology (China) and were housed under specific pathogen-free conditions in Animal Experiment Center-Animal Biosafety Level-III Laboratory of Wuhan University. The animals were randomly divided into four groups of 6 mice each. The mice were injected with 100 μL of cells suspended in Matrigel Basement Membrane Matrix (Corning, 356234) at a population of 3 × 106 ACHN indicated cells into the left or right dorsal flank subcutaneously.\n",
      "Narrative Text: The tumor size was estimated once a week via a digital Vernier caliper. The tumor volume was calculated with the following formula: tumor volume (mm3) =length (mm)×width2 (mm)× 0.5. Tumors were havested 8 weeks after injection.\n",
      "Title: Statistical analysis\n",
      "Narrative Text: The quantitative experiment was repeated three times or more independently. The graphs were generated with GraphPad Prism software. Two-tailed Student’s t-test, one-way ANOVA and the Wilcoxon rank-sum test were used to assess the significance of the experiments. P values < 0.05 were considered to indicate the significant differences. \n",
      "Text: \n",
      "Title: Data Availability\n",
      "Narrative Text: The sequencing data are available in GEO Data Sets with the accession number GSE270718 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE270718), GSE270719 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE270719) and GSE275838 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE275838). All other raw data support the findings of this study are available from the corresponding author upon reasonable request.\n",
      "Text: \n",
      "Title: Acknowledgments\n",
      "Text: This work was supported by Natural Science Foundation of China (grant numbers 82203491 and 82173137), Natural Science Foundation of Hubei Province of China (grant number 2022CFA008), China Postdoctoral Science Foundation (grant number 2022TQ0237), Natural Science Foundation of Inner Mongolia (grant number 2024QN08005) and Translational Medicine and Interdisciplinary Research Joint Fund of Zhongnan Hospital of Wuhan University (grant number ZNJC202212). \n",
      "Text: \n",
      "Title: Disclosure Statement & Competing Interests \n",
      "Narrative Text: These authors declare that they have no conflict of interest.\n",
      "Text: \n",
      "Title: References\n",
      "Narrative Text: Bonavida B, Kaufhold S (2015) Prognostic significance of YY1 protein expression and mRNA levels by bioinformatics analysis in human cancers: a therapeutic target. Pharmacol Ther 150: 149-168\n",
      "Narrative Text: Chen X, Loo JX, Shi X, Xiong W, Guo Y, Ke H, Yang M, Jiang Y, Xia S, Zhao M et al (2018) E6 Protein Expressed by High-Risk HPV Activates Super-Enhancers of the EGFR and c-MET Oncogenes by Destabilizing the Histone Demethylase KDM5C. Cancer Res 78: 1418-1430\n",
      "Narrative Text: Dawson MA (2017) The cancer epigenome: Concepts, challenges, and therapeutic opportunities. Science 355: 1147-1152\n",
      "Narrative Text: Gao P, Li L, Yang L, Gui D, Zhang J, Han J, Wang J, Wang N, Lu J, Chen S et al (2019) Yin Yang 1 protein ameliorates diabetic nephropathy pathology through transcriptional repression of TGFbeta1. Sci Transl Med 11\n",
      "Narrative Text: Gordon S, Akopyan G, Garban H, Bonavida B (2006) Transcription factor YY1: structure, function, and therapeutic implications in cancer biology. Oncogene 25: 1125-1142\n",
      "Narrative Text: Hanahan D (2022) Hallmarks of Cancer: New Dimensions. Cancer Discov 12: 31-46\n",
      "Narrative Text: Hogg SJ, Beavis PA, Dawson MA, Johnstone RW (2020) Targeting the epigenetic regulation of antitumour immunity. Nat Rev Drug Discov 19: 776-800\n",
      "Narrative Text: Iwase S, Lan F, Bayliss P, de la Torre-Ubieta L, Huarte M, Qi HH, Whetstine JR, Bonni A, Roberts TM, Shi Y (2007) The X-linked mental retardation gene SMCX/JARID1C defines a family of histone H3 lysine 4 demethylases. Cell 128: 1077-1088\n",
      "Narrative Text: Ji X, Jin S, Qu X, Li K, Wang H, He H, Guo F, Dong L (2015) Lysine-specific demethylase 5C promotes hepatocellular carcinoma cell invasion through inhibition BMP7 expression. BMC Cancer 15: 801\n",
      "Narrative Text: Khachigian LM (2018) The Yin and Yang of YY1 in tumor growth and suppression. Int J Cancer 143: 460-465\n",
      "Narrative Text: Li Y, Li J, Li Z, Wei M, Zhao H, Miyagishi M, Wu S, Kasim V (2022) Homeostasis Imbalance of YY2 and YY1 Promotes Tumor Growth by Manipulating Ferroptosis. Adv Sci (Weinh) 9: e2104836\n",
      "Narrative Text: Meliala ITS, Hosea R, Kasim V, Wu S (2020) The biological implications of Yin Yang 1 in the hallmarks of cancer. Theranostics 10: 4183-4200\n",
      "Narrative Text: Miranda Furtado CL, Dos Santos Luciano MC, Silva Santos RD, Furtado GP, Moraes MO, Pessoa C (2019) Epidrugs: targeting epigenetic marks in cancer treatment. Epigenetics 14: 1164-1176\n",
      "Narrative Text: Nebbioso A, Tambaro FP, Dell'Aversana C, Altucci L (2018) Cancer epigenetics: Moving forward. PLoS Genet 14: e1007362\n",
      "Narrative Text: Nijman SM, Friend SH (2013) Cancer. Potential of the synthetic lethality principle. Science 342: 809-811\n",
      "Narrative Text: Outchkourov N, MuinO J, Kaufmann K, Van Ijcken WJ, Koerkamp MJ, Van LD, P. GD, Holstege FP, Grosveld F, Timmers HT (2013) Balancing of Histone H3K4 Methylation States by the Kdm5c/SMCX Histone Demethylase Modulates Promoter and Enhancer Function. Cell Reports 3: 1071-1079\n",
      "Narrative Text: Qiao K, Ning S, Wan L, Wu H, Wang Q, Zhang X, Xu S, Pang D (2019) LINC00673 is activated by YY1 and promotes the proliferation of breast cancer cells via the miR-515-5p/MARK4/Hippo signaling pathway. J Exp Clin Cancer Res 38: 418\n",
      "Text: Sarvagalla S, Kolapalli SP, Vallabhapurapu S (2019) The Two Sides of YY1 in Cancer: A Friend and \n",
      "Text: \n",
      "Title: a Foe. Front Oncol 9: 1230\n",
      "Narrative Text: Scandaglia M, Lopez-Atalaya JP, Medrano-Fernandez A, Lopez-Cascales MT, Del Blanco B, Lipinski M, Benito E, Olivares R, Iwase S, Shi Y et al (2017) Loss of Kdm5c Causes Spurious Transcription and Prevents the Fine-Tuning of Activity-Regulated Enhancers in Neurons. Cell Rep 21: 47-59\n",
      "Narrative Text: Secombe J, Li L, Carlos L, Eisenman RN (2007) The Trithorax group protein Lid is a trimethyl histone H3K4 demethylase required for dMyc-induced cell growth. Genes Dev 21: 537-551\n",
      "Narrative Text: Shen H, Xu W, Guo R, Rong B, Gu L, Wang Z, He C, Zheng L, Hu X, Hu Z et al (2016) Suppression of Enhancer Overactivation by a RACK7-Histone Demethylase Complex. Cell 165: 331-342\n",
      "Narrative Text: Shen HF, Zhang WJ, Huang Y, He YH, Hu GS, Wang L, Peng BL, Yi J, Li TT, Rong R et al (2021) The Dual Function of KDM5C in Both Gene Transcriptional Activation and Repression Promotes Breast Cancer Cell Growth and Tumorigenesis. Adv Sci (Weinh) 8: 2004635\n",
      "Narrative Text: Shrivastava A, Saleque S, Kalpana GV, Artandi S, Goff SP, Calame K (1993) Inhibition of transcriptional regulator Yin-Yang-1 by association with c-Myc. Science 262: 1889-1892\n",
      "Narrative Text: Sui G, Affar el B, Shi Y, Brignone C, Wall NR, Yin P, Donohoe M, Luke MP, Calvo D, Grossman SR et al (2004) Yin Yang 1 is a negative regulator of p53. Cell 117: 859-872\n",
      "Narrative Text: van Leenders GJ, Dukers D, Hessels D, van den Kieboom SW, Hulsbergen CA, Witjes JA, Otte AP, Meijer CJ, Raaphorst FM (2007) Polycomb-group oncogenes EZH2, BMI1, and RING1 are overexpressed in prostate cancer with adverse pathologic and clinical features. Eur Urol 52: 455-463\n",
      "Narrative Text: Wang W, Qiao S, Li G, Cheng J, Yang C, Zhong C, Stovall DB, Shi J, Teng C, Li D et al (2022) A histidine cluster determines YY1-compartmentalized coactivators and chromatin elements in phase-separated enhancer clusters. Nucleic Acids Res 50: 4917-4937\n",
      "Narrative Text: Zafar A, Wang W, Liu G, Wang X, Xian W, McKeon F, Foster J, Zhou J, Zhang R (2021) Molecular targeting therapies for neuroblastoma: Progress and challenges. Med Res Rev 41: 961-1021\n",
      "Narrative Text: Zhang SM, Cai WL, Liu X, Thakral D, Luo J, Chan LH, McGeary MK, Song E, Blenman KRM, Micevic G et al (2021) KDM5B promotes immune evasion by recruiting SETDB1 to silence retroelements. Nature 598: 682-687\n",
      "Narrative Text: Zheng Q, Li P, Zhou X, Qiang Y, Fan J, Lin Y, Chen Y, Guo J, Wang F, Xue H et al (2021) Deficiency of the X-inactivation escaping gene KDM5C in clear cell renal cell carcinoma promotes tumorigenicity by reprogramming glycogen metabolism and inhibiting ferroptosis. Theranostics 11: 8674-8691\n",
      "Text: \n",
      "Title: Figure Legends\n",
      "Narrative Text: Figure 1 YY1 was identified as a KDM5C binding protein\n",
      "Narrative Text: A. Forest plot of the hazard ratio (HR) showing the association between high expression levels of KDM5C and overall survival in multiple cancers. For each type of cancer, the HR and 95% confidence interval (CI) are plotted as red dots and horizontal lines on the basis of a Cox proportional hazards model; when the HR=1, the study object had no effect on overall survival; when the HR>1, the study object was determined to be a risk factor; and when the HR<1, the study object was identified as a protective factor.\n",
      "Narrative Text: B. Strategy for identifying KDM5C-binding partners in HEK293T cells via TAP-MS.\n",
      "Narrative Text: C-D. Co-IP followed by Western blotting was used to detect the interaction between KDM5C and YY1 exogenously (C) and endogenously (D) in HEK 293 cells.\n",
      "Text: E. Semi-endogenous co-IP with KDM5C-Flag and YY1 in the renal cancer cell line 769-P.\n",
      "Narrative Text: F. Schematic representation of KDM5C domains and corresponding constructs.\n",
      "Text: G. Semi-endogenous co-IP with KDM5C truncation mutants and endogenous YY1 in HEK293T cells.\n",
      "Narrative Text: Figure 2 YY1 is a vulnerable target in KDM5C-deficient tumor cells\n",
      "Narrative Text: A. Western blotting showing KDM5C expression in HK-2 human kidney proximal tubular epithelial cells and renal cancer cell lines (ACHN, 769-P and RCC4).\n",
      "Narrative Text: B. Effect of YY1 knockdown on the survival of the indicated tumor cell lines as measured by a colony formation assay (n=3 biological replicates).\n",
      "Narrative Text: C. Proliferation of the indicated cell lines after YY1 was depleted, as measured by a CCK8 \n",
      "Text: \n",
      "Narrative Text: assay. The data are shown as the mean ± SD of three independent experiments after analysis by one-way ANOVA. In ACHN cells, P=0.116 (shYY1-1) and P=0.1222 (shYY1-2); in 769-P cells, P=0.7353 (shYY1-1) and P=0.9113 (shYY1-2); and in RCC4 cells, P=0.0007 (shYY1-1) and P=2E-05 (shYY1-2).\n",
      "Narrative Text: D. Xenograft tumors excised from nude mice (n = 6/group) that had been subcutaneously injected with the indicated tumor cells. Statistical significance was determined by one-way ANOVA. P=1E-05\n",
      "Narrative Text: Figure 3 KDM5C is essential for YY1 genome-wide chromatin recruitment\n",
      "Narrative Text: A. Violin plot showing the alteration in YY1 binding peaks after KDM5C depletion. YY1 binding peaks in both ACHN parental and shKDM5C cells were counted (n=17,030). The data are presented as mean ± SD. Statistical significance was determined by the Mann-Whitney U test, P= 2.22E-16.\n",
      "Narrative Text: B. Venn diagram showing the overlapping peaks of YY1 and KDM5C in the co-occupied regions. Representative data from two independent CUT-Tag experiments.\n",
      "Narrative Text: C. Heatmap and metaplot showing the chromatin enrichment of YY1 at overlapping YY1 and KDM5C peak regions (n=3645) in ACHN parental and shKDM5C cells.\n",
      "Narrative Text: D. Pie chart showing the annotated genomic distribution of altered YY1 peaks on different DNA elements in shKDM5C or KDM5C overexpressing cells.\n",
      "Narrative Text: Figure 4 An intact JmjC domain but not KDM5C H3K4 demethylation activity is required for KDM5C-mediated YY1 recruitment\n",
      "Narrative Text: A. Heatmap and metaplot showing the distribution of YY1 peaks in the overlapping regions \n",
      "Text: \n",
      "Narrative Text: of YY1 and KDM5C (n=3645) in ACHN cells expressing the KDM5C-WT or KDM5C-H514A mutant.\n",
      "Narrative Text: B. Loss of KDMC5 resulted in increased H3K4me3 signal intensity in the YY1 and KDM5C overlapping region (n=3645) in ACHN parental and shKDM5C cells.\n",
      "Narrative Text: C. Venn diagram showing the number of YY1 peaks in the H3K4me3/YY1 overlapping region (n=11087) with or without modulation by KDM5C.\n",
      "Narrative Text: D-E. Heatmaps and metaplots revealed reduced YY1 enrichment at KDM5C demethylase dependent regions (n=9943) or KDM5C demethylase independent regions (n=1144) in ACHN parental and shKDM5C cells.\n",
      "Narrative Text: F. KDM5C protein levels in whole-cell lysates or chromatin fractions from HEK293T cells transfected with KDM5C-WT or KDM5C-H514A mutant construct.\n",
      "Narrative Text: G-I. Effect of the re-expression of the KDM5C-WT or KDM5C-H514A mutant on the proliferation of ACHN (G), 769-P (H) or RCC4 (I) cells after the depletion KDM5C, YY1 or both, as determined by the CCK8 assay. Each experiment was repeated three times with similar results. “Dsh” indicates cells in which KDM5C and YY1 were both knocked down. The data are shown as the mean ± SD of three independent experiments after analysis via Student’s t test. In ACHN cells, P=5.37E-08 (Dsh) and P=3.48E-07 (Dsh+KDM5C); in 769-P cells, P=4.45E-10 (Dsh) and P=7.49E-10 (Dsh+KDM5C); in ACHN cells, P=5.86E-09 (Dsh) and P=2.72E-07 (Dsh+KDM5C).\n",
      "Narrative Text: J. Colony formation assay with the indicated cell lines(n=3). “Dsh” indicates cells with KDM5C and YY1 simultaneously knocked down.\n",
      "Text: \n",
      "Narrative Text: Figure 5 KDM5C and YY1 coregulate the expression of a number of essential cell-fate genes\n",
      "Narrative Text: A-C. Volcano plots showing the change in the number of transcripts in shKDM5C, shYY1 or double shKDM5C ACHN cells, as determined by RNA-seq analysis (|log2 FC|>1.5, p value <0.05). “Double sh” indicates cells with KDM5C and YY1 simultaneously knocked down.\n",
      "Narrative Text: D. Venn diagrams showing the number of downregulated genes in shKDM5C, shYY1 or shKDM5C/YY1 double-knockdown cells.\n",
      "Narrative Text: E. Heatmap showing the normalized gene expression profile in shKDM5C, shYY1 or double sh ACHN cells. “Double sh” indicates cells with KDM5C and YY1 simultaneously knocked down.\n",
      "Narrative Text: F. qPCR was used to assess the transcription of select genes in the indicated ACHN cells. Each experiment was repeated three times with similar results. “Double sh” indicates cells with KDM5C and YY1 double knocked down. The data are shown as the mean ± SD of three independent experiments after analysis by two tailed Student’s T-test. P=2.86E-05 (JAM3), P=4.06E-06 (KNTC1), P=1.31E-04(SMC2), P=9.18E-06(SMC4), P=1.91E-05(LMNB1) and P=1.47E-04 (FOXM1).\n",
      "Narrative Text: Figure 6 Simultaneous targeting of KDM5C and YY1 effectively induces cell cycle arrest in the G2/M phase and apoptosis.\n",
      "Narrative Text: A. Cell cycle analysis with the indicated cell lines via flow cytometry. The percentage of cells in each phase of the cell cycle is shown on the Y-axis. Each experiment was repeated \n",
      "Text: \n",
      "Narrative Text: three times with similar results. “Double sh” indicates cells with simultaneous knockdown of KDM5C and YY1.\n",
      "Narrative Text: B. Flow cytometry–based Annexin-V/PI staining showing cells in the indicated lineages undergoing apoptosis. The quantitative results indicate the ratio of apoptotic cells. Each experiment was repeated three times with similar results. “Double sh” indicates cells with simultaneous knockdown of KDM5C and YY1. The data are shown as the mean ± SD of three independent experiments after analysis by two tailed Student’s T-test. P=5.3E-05 (RCC4) and P=1.1E-06 (769-P).\n",
      "Title: Figure 7 A mechanistic hypothesis model for KDM5C-mediated YY1 recruitment. \n",
      "Narrative Text: YY1 and KDM5C constitute a pair of synthetic lethal genes. KDM5C recruits YY1 to the promoter regions of cell cycle- or apoptosis-related genes. Targeting YY1 increases the vulnerability of KDM5C-deficient cancer cells.\n",
      "Text: \n",
      "Title: Expanded View Figure Legends\n",
      "Narrative Text: Figure EV1 Validation of the constructed cell lines used in this study\n",
      "Narrative Text: A-D. KDM5C and YY1 protein levels in the indicated cells were measured via Western blotting. “Dsh” indicates cells in which KDM5C and YY1 were both knocked down.\n",
      "Narrative Text: E-F. Proliferation rate of the indicated cell lines upon KDM5C depletion, as determined via a CCK8 assay. The data are shown as the mean ± SD of three independent experiments after analysis via one-way ANOVA. In 769-P cells, P=0.2701 (shKDM5C -1) and P=0.4753 (shKDM5C-2); in ACHN cells, P=0.4308 (shKDM5C-1) and P=0.3869 (shYY1-2).\n",
      "Narrative Text: Figure EV2 ChIP-seq experiments targeting YY1 in HK-2 cells \n",
      "Narrative Text: A. Heatmap and metaplot showing the chromatin enrichment of YY1 in HK-2 cells in the presence or absence of KDM5C.\n",
      "Narrative Text: B-C. ChIP-PCR assay showing the enrichment of YY1 at the promoters of the indicated genes in HK-2 cells after the depletion of YY1, KDM5C or both (B), or in cells expressing KDM5C-WT or KDM5C-H514A mutant (C). Each experiment was repeated three times with similar results. Statistical significate was determined by one-way ANOVA. In B, P=1.24E-06 (JAM3), P=2.27E-06 (KNTC1), P=2.82E-08 (SMC2), P=2.41E-05 (SMC4) and P=3.96E-05 (FOXM1), respectively. In C, P=4.53E-06 (JAM3), P=4.46E-06 (KNTC1), P=5.53E-06(SMC2), P=6.82E-06 (SMC4), P=1.05E-06 (FOXM1) and P=0.0002 (LMNB1), respectively.\n",
      "Title: Figure EV3 Evaluation of the interaction between YY1 and KDM5C-WT or KDM5C-\n",
      "Text: \n",
      "Title: H514A mutant.\n",
      "Narrative Text: Co-IP followed by Western blotting showing the interaction between YY1 and KDM5C-WT or KDM5C-H514A mutant in HEK293T cells.\n",
      "Narrative Text: Figure EV4 RNA-seq in HK-2 derived cells.\n",
      "Narrative Text: A-C. Volcano plots showing the change in the number of transcripts in shKDM5C, shYY1 or double-knockdown HK-2 cells, as determined by RNA-seq analysis (|log2 FC|>1.5, p value <0.05). \n",
      "Narrative Text: D. Heatmap showing the normalized gene expression profile in shKDM5C, shYY1 or double sh ACHN cells. “Double sh” indicates cells in which KDM5C and YY1 were both knocked down.\n",
      "Text: Figure EV5 Effects of KDM5C on YY1 enrichment at promoter regions of specific genes\n",
      "Narrative Text: A-B. The connections between the ChIP-seq and RNA-seq data were analyzed, and the genomic regions of the downregulated genes in the KDM5C/YY1 double-knockdown group were annotated.\n",
      "Narrative Text: C-D. ChIP-seq snapshots reflecting changes in YY1 enrichment at the promoter regions of the JAM3, LNMB1, FOXM1, KNTC1, SMC2 and SMC4 genes in the indicated cell lines. The promoters are indicated by H3K4me3 peaks.\n"
     ]
    }
   ],
   "source": [
    "from unstructured.documents.elements import NarrativeText, Text, Title, ListItem, EmailAddress, PageBreak\n",
    "# Process elements based on their type\n",
    "for element in elements:\n",
    "    if isinstance(element, Title):\n",
    "        # Handle titles\n",
    "        print(f\"Title: {element.text}\")\n",
    "    elif isinstance(element, NarrativeText):\n",
    "        # Handle narrative text (paragraphs)\n",
    "        print(f\"Narrative Text: {element.text}\")\n",
    "    elif isinstance(element, Text):\n",
    "        # Handle general text\n",
    "        print(f\"Text: {element.text}\")\n",
    "    elif isinstance(element, ListItem):\n",
    "        # Handle list items\n",
    "        print(f\"List Item: {element.text}\")\n",
    "    elif isinstance(element, EmailAddress):\n",
    "        # Handle email addresses\n",
    "        print(f\"Email Address: {element.text}\")\n",
    "    elif isinstance(element, PageBreak):\n",
    "        # Handle page breaks\n",
    "        print(\"Page Break\")\n",
    "    else:\n",
    "        # Handle unknown types\n",
    "        print(f\"Unknown Element: {type(element)}\")\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## LangChain integration"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Title: Targeting the transcription factor YY1 is synthetic lethal with loss of the histone demethylase KDM5C\n",
      "\n",
      "Qian Zheng1#, Pengfei Li2#, Yulong Qiang1#, Jiachen Fan1, Yuzhu Xing1, Ying Zhang1, Fan Yang1*, Feng Li1,3* and Jie Xiong4* \n",
      "\n",
      "1Department of Medical Genetics, TaiKang Medical School (School of Basic Medical Sciences), Wuhan University, Wuhan 430071, China\n",
      "\n",
      "2Inner Mongolia Key Laboratory of Molecular Pathology, Inner Mongolia Medical University, Huhhot, Inner Mongolia, 010059, China\n",
      "\n",
      "3Hubei Provincial Key Laboratory of Allergy and Immunology, Wuhan University, Wuhan 430071, China\n",
      "\n",
      "4Department of Immunology, TaiKang Medical School (School of Basic Medical Sciences), Wuhan University, Wuhan 430071, China\n",
      "\n",
      "#These authors contribute equally to this paper.   \n",
      "\n",
      "Correspondence to:\n",
      "\n",
      "Jie Xiong\n",
      "\n",
      "E-mail: jiexiong@whu.edu.cn，Tel: +86 27 68759222\n",
      "\n",
      "Department of Immunology, TaiKang Medical School (School of Basic Medical Sciences), Wuhan University, Wuhan 430071, China\n",
      "\n",
      "Feng Li\n",
      "\n",
      "E-mail: fli222@whu.edu.cn, Tel: +86 27 68759222\n",
      "\n",
      "Department of Medical Genetics, TaiKang Medical School (School of Basic Medical \n",
      "\n",
      "\n",
      "\n",
      "Sciences), Wuhan University, Wuhan 430071, China.\n",
      "\n",
      "Hubei Provincial Key Laboratory of Allergy and Immunology, Wuhan University, Wuhan 430071, China \n",
      "\n",
      "Fan Yang\n",
      "\n",
      "E-mail: yangfan2022@whu.edu.cn, Tel: +86 27 68759222\n",
      "\n",
      "Department of Medical Genetics, TaiKang Medical School (School of Basic Medical Sciences), Wuhan University, Wuhan 430071, China.\n",
      "\n",
      "\n",
      "\n",
      "Abstract\n",
      "\n",
      "An understanding of the enzymatic and scaffolding functions of epigenetic modifiers is important for the development of epigenetic therapies for cancer. The H3K4me2/3 histone demethylase KDM5C has been shown to regulate transcription. The diverse roles of KDM5C are likely determined by its interacting partners, which are still largely unknown. In this study, we screen for KDM5C-binding proteins and show that YY1 interacts with KDM5C. A synergistic antitumor effect is exerted when both KDM5C and YY1 are depleted, and targeting YY1 appears to be a vulnerability in KDM5C-deficient cancer cells. Mechanistically, KDM5C promotes global YY1 chromatin recruitment, especially at promoters. Moreover, an intact KDM5C JmjC domain but not KDM5C histone demethylase activity is required for KDM5C-mediated YY1 chromatin binding. Transcriptional profiling reveals that dual inhibition of KDM5C and YY1 increases transcriptional repression of cell cycle- and apoptosis-related genes. In summary, our work demonstrates a synthetic lethal interaction between YY1 and KDM5C and suggests combination therapies for cancer treatments.\n",
      "\n",
      "Key word: KDM5C; YY1; chromatin recruitment; promoter; cancer therapy\n",
      "\n",
      "\n",
      "\n",
      "Introduction\n",
      "\n",
      "Epigenetic regulators change phenotype acquisition or gene expression without altering the DNA sequence. By conferring a fitness advantage, mutational or non-mutational epigenetic reprogramming has been implicated in the evolution of tumors during malignant progression and metastasis. In addition, numerous studies have shown that aberrant epigenetic alterations contribute to cancer therapeutic resistance(Hogg et al, 2020; Miranda Furtado et al, 2019; Zafar et al, 2021). Therefore, targeting epigenetic regulators is an attractive strategy for cancer therapy.\n",
      "\n",
      "The histone demethylase KDM5C is a member of the SMC homolog family. By specifically removing one or two methyl groups from Lys4 in histone H3, the dynamic balance in histone H3K4 methylation is maintained. KDM5C carries a composite catalytic region in which JmjN and JmjC domains are located. JmjC is critical for catalyzing the demethylation of H3K4me2/3, while the JmjN domain interacts with the JmjC domain for the complete catalytic function of KDM5C. H514 is a highly conserved amino acid that is located in the catalytic core of the JmjC domain, and its mutation eliminates KDM5C demethylase activity. KDM5C can suppress transcription by reducing the level of H3K4me2/3 via its demethylase activity; however, whether modulation of H3K4 methylation status is the only mechanism or the predominant mechanism by which KDM5C regulates gene expression is unclear.\n",
      "\n",
      "KDM5C is frequently mutated or aberrantly expressed in various cancer types. Several studies have indicated that KDM5C is involved in regulating different biological processes in tumor cells through its histone demethylase activity and functions as a tumor suppressor. \n",
      "\n",
      "\n",
      "\n",
      "For example, the high-risk HPV E6 protein degrades KDM5C in cervical cancer cells, which results in super enhancers activation of the proto-oncogenes c-MET and EGFR by changing the H3K4me3/H3K4me1 ratio(Chen et al, 2018). In clear cell renal cell carcinoma cells, KDM5C inhibits the expression of genes related to glycogen metabolism by reducing H3K4me3 levels at the promoters of these genes, while inactivating KDM5C mutation confers cell resistance to ferroptosis(Zheng et al, 2021). However, the therapeutic strategy for KDM5C-deficient tumors is unknown. Intriguingly, KDM5C has also been shown to promote tumorigenesis, although the exact mechanism is unclear; in general, these observations suggest that KDM5C potentially regulates gene expression via a distinct mechanism.\n",
      "\n",
      "To better understand the mechanisms underlying how KDM5C regulates gene expression, we screened and thus identified Yin Yang 1 (YY1) as a KDM5C-interacting partner. Targeting YY1 in KDM5C-deficient tumor cells effectively suppressed cell proliferation and tumorigenesis. Chromatin Immunoprecipitation followed by next generation sequencing (ChIP-seq) analysis with multiple engineered cell lines revealed that KDM5C is essential for YY1 chromatin recruitment, which is partially dependent on an intact JmjC domain in KDM5C. Simultaneous loss of KDM5C and YY1 led to significantly enhanced transcriptional inhibition of a number of essential cell fate genes, elevated the apoptosis rate and led to cell cycle arrest. Our study suggests a novel scaffolding function of epigenetic regulators and provides a rationale for the development of new combination treatments.\n",
      "\n",
      "\n",
      "\n",
      "Results\n",
      "\n",
      "YY1 is identified as a partner of KDM5C \n",
      "\n",
      "Previous studies have shown that KDM5C plays diverse roles in cancer progression. Indeed, a systemic The Cancer Genome Atlas (TCGA) clinical data-based analysis clearly revealed that high KDM5C expression was positively or negatively correlated with cancer patient survival, depending on the cancer type (Figure 1A), reinforcing the idea that KDM5C may exhibit a distinct regulatory effect on gene expression, prompting us to screen the interacting partners of KDM5C. Tandem affinity purification (TAP) was subsequently performed. We generated a HEK293T cell line that stably expressed SBP- and Flag-tagged KDM5C, and the lysates of these cells were incubated with streptavidin-coated beads. The bead-bound proteins were subsequently eluted with buffer containing biotin. Next, the captured proteins were incubated with anti-Flag beads, and the proteins bound to these beads were eluted and analyzed by mass spectrometry (MS). The results of the MS analysis indicated that a number of proteins potentially interact with KDM5C, including YY1 (Figure 1B), the overexpression of which has been observed in the vast majority of cancers(Gao et al, 2019; van Leenders et al, 2007). Given the pivotal role played by YY1 in the transcriptional regulation of numerous genes(Meliala et al, 2020), we focused on the interaction between KDM5C and YY1. To confirm their interaction, mutual coimmunoprecipitation (co-IP) assays were performed in HEK293T cells, and the results clearly showed that KDM5C bound to YY1 (Figure 1C). Co-IP with endogenous proteins was also performed and the results confirmed that KDM5C interacts with YY1 in mammalian cells (Figure 1D). In addition, we also performed the semi-endogenous co-IP \n",
      "\n",
      "\n",
      "\n",
      "with YY1 and KDM5C-Flag in 769-P renal cancer cell lines. As shown in Figure 1E, KDM5C clearly bound to YY1. To delineate the interaction, a series of KDM5C truncation mutants were constructed and used for further analysis (Figure 1F). We found that multiple regions, including the JmjN and JmjC domains, which are critical for H3K4me2/3 removal, were also involved in the interaction between KDM5C and YY1 (Figure 1G).\n",
      "\n",
      "YY1 depletion significantly inhibits the proliferation and tumorigenicity of KDM5C-deficient tumor\n",
      "\n",
      "Many studies have shown that YY1 functions as an oncogene because its overexpression is observed in multiple cancers(Li et al, 2022; Qiao et al, 2019; Wang et al, 2022). YY1 tethers cofactors and promotes or represses transcription, depending on its interacting partner, which includes c-myc and p53(Shrivastava et al, 1993; Sui et al, 2004), prompting us to investigate the role of the YY1 KDM5C interaction in tumorigenesis. To achieve this goal, we first selected human kidney proximal tubular epithelial cell line (HK-2), and several renal cancer cell lines (ACHN, 769-P and RCC4) with different KDM5C and YY1 protein expression levels, as described in our previous studies(Chen et al., 2018; Zheng et al., 2021) (Figure 2A). Interestingly, knocking down YY1 in KDM5C-deficient RCC4 tumor cells significantly reduced the colony formation ability of the cells, which was not observed in the KDM5C-proficient tumor cell lines ACHN and 769-P (Figure 2B and EV1B-D). Similar results were also obtained in a CCK8-based tumor cell proliferation assay (Figure 2C and EV1E-F). Notably, tumor growth was markedly inhibited when both of KDM5C and YY1 were depleted, whereas the loss of either KDM5C or YY1 only negligibly influenced ACHN cell growth (Figure 2D). These results strongly indicate that \n",
      "\n",
      "\n",
      "\n",
      "KDM5C might be important for YY1-related tumorigenesis, suggesting that YY1 may be a vulnerable target in KDM5C-deficient tumor cells.\n",
      "\n",
      "KDM5C colocalizes with YY1 and contributes to YY1 genome-wide chromatin recruitment\n",
      "\n",
      "The stimulatory effect of YY1 on tumor cell growth was at least partially dependent on the presence of KDM5C. Given that YY1 always promotes tumorigenesis by increasing oncogene transcription, it is unlikely that YY1 tethers KDM5C because KDM5C has H3K4me2/3 demethylase activity, which is associated with repressed gene expression. A recent study revealed that KDM5B, which is a member of the KDM5 demethylase family, recruited a methyltransferase to suppress antitumor immunity, prompting us to explore whether KDM5C has a scaffolding function similar to that of KDM5B(Zhang et al, 2021). We generated a stable KDM5C-depleted ACHN cell lines (Figure EV1B), and ChIP-seq assays were subsequently performed. As shown in Figure 3A, the intensity of the YY1 binding peak was reduced upon loss of KDM5C, implying that KDM5C might be a general regulator of YY1 activity by mediating its chromatin recruitment. Next, we asked to what extent KDM5C co-localizes with YY1. To evaluate the genomic co-occupancy of KDM5C and YY1, we performed the cleavage under targets and tagmentation (CUT&Tag) assay in the ACHN renal cell line, and the annotation of the YY1 and KDM5C peaks revealed an obvious overlap (n=3645) between YY1 and KDM5C binding events (Figure 3B). Further analyzed revealed a significant overlap, with 7,957 genes being concurrently bound by both YY1 and KDM5C (Hypergeometric test, P=4.87e-17). This overlap accounts for 56.32% of the total genes bound by KDM5C and 66.71% of the total genes bound by YY1. \n",
      "\n",
      "\n",
      "\n",
      "Moreover, a heatmap and a metaplot analysis revealed that the enrichment of YY1 was reduced after KDM5C was depleted in the peaks (n=3645) where YY1 and KDM5C were directly related (Figure 3C). Reduced enrichment of YY1 was also observed in HK-2 cells with KDM5C depletion (Figure EV1A and EV2A). These results revealed that the intensity of the YY1 binding peaks was partially dependent on the KDM5C protein level in both nontumoral cell and tumor cells, suggesting that KDM5C mediated YY1 chromatin recruitment may represent a universal mechanism of action. The localization of the KDM5C-dependent YY1 binding peaks was subsequently analyzed, and the results revealed that most of the peaks were enriched in promoters (Figure 3D), implying that KDM5C may regulate gene expression via a distinct mechanism.\n",
      "\n",
      "An intact catalytic domain but not the demethylation activity of KDM5C is required for YY1 recruitment\n",
      "\n",
      "Since our results suggested that KDM5C functions as a scaffold and that KDM5C is essential for YY1 binding to a promoter, we asked, whether the demethylase activity of KDM5C is involved in YY1 recruitment. We generated ACHN cell lines that stably expressed KDM5C-WT or its enzymatically inactive mutant H514A and performed ChIP-seq to evaluate the effects of KDM5C or its mutants on YY1 recruitment (Figure EV1B). As shown in Figure 4A, ectopic KDM5C WT but not the KDM5C-H514A mutant enhanced YY1 chromatin binding. A similar trend was also observed in HK-2 cells (Figure EV2A). Because the primary function of KDM5C is to catalyze the removal of methyl groups from H3K4me2/3, we evaluated whether H3K4me3 abundance plays a role in YY1 recruitment.\n",
      "\n",
      "We first performed ChIP-seq of H3K4me3 in ACHN shKDM5C cells and analyzed the \n",
      "\n",
      "\n",
      "\n",
      "change in the signal intensity of this mark. Increased H3K4me3 enrichment was clearly observed in the KDM5C/YY1 overlapping region (Figure 4B), but the opposite trend of YY1 recruitment was observed in KDM5C-depleted cells (Figure 3C). To further understand the relationship between YY1 recruitment and H3K4me3, we focused on the overlapping regions of YY1 and H3K4me3 (n=11087), which were classified into KDM5C demethylase-dependent peaks (n=9943) and KDM5C demethylase-independent peaks (n=1144), as shown in Figure 4C. Reduced YY1 ChIP-seq signals were observed in almost all of the overlapping YY1 and H3K4me3 regions (Figure 4D-E), regardless of the effect of KDM5C on H3K4me3 intensity. We therefore hypothesized that an intact KDM5C JmjC domain, rather than KDM5C histone demethylase activity, is essential for YY1 recruitment.\n",
      "\n",
      "To further explore the mechanism underlying the effects of KDM5C, we evaluated the interaction between KDM5C-H514A and YY1 and found that the mutation did not influence their interaction (Figure EV3), indicating that the effect of KDM5C-H514A on YY1 chromatin recruitment was unlikely to be caused by disrupted protein binding activity. Because the catalytic composite JmjN/C domain binds the substrate H3 tail harboring methylated H3K4 and since a previous study revealed that none of the KDM5C PHD domains exhibited strong binding affinity for unmethylated or methylated histones(Iwase et al, 2007), we speculated that the replacement of the histidine residue at position 514 with an alanine residue destroyed the integrity of the JmjN/C domain and influenced the chromatin-binding activity of KDM5C. To test this hypothesis, we expressed KDM5C-WT or KDM5C-H514A in HEK293T cells and isolated chromatin-bound proteins. Interestingly, although the protein levels in whole-cell lysates were indistinguishable, H154A exhibited obviously reduced \n",
      "\n",
      "\n",
      "\n",
      "chromatin-binding capacity (Figure 4F), which might help explain why KDM5C-H514A failed to recruit YY1 to chromatin. In addition, tumor cell proliferation was markedly inhibited only when both of KDM5C and YY1 were depleted in the KDM5C-proficient tumor cells ACHN and 769-P (Figure 4G-H and EV1B-C). The inhibitory effect was blunted when KDM5C-WT was re-expressed; in contrast, KDM5C-H514A only moderately restored the growth of ACHN and 769-P double knockdown cells. A similar trend was also observed in the colony formation assay (Figure 4J). In KDM5C-deficient RCC4, cells overexpression of KDM5C-WT largely restored cell sensitivity to YY1 depletion (Figure 4I and EV1D). These result collectively suggest that an intact catalytic domain of KDM5C is required for YY1 recruitment.\n",
      "\n",
      "KDM5C cooperates with YY1 to regulate essential cell fate gene expression\n",
      "\n",
      "To systematically evaluate the effect of KDM5C-mediated YY1 recruitment on gene expression, we performed RNA-seq with shKDM5C, shYY1 or double-knockdown ACHN cells. As shown in Figure 5A-B, the loss of KDM5C or YY1 led to the upregulated or downregulated expression of several genes. In contrast, the simultaneous loss of KDM5C and YY1 resulted in altered transcription of thousands of genes (1511 genes whose expression was downregulated versus 797 genes whose expression was upregulated, |log2FC|≥1.5, p value<0.05, Figure 5C), indicating that YY1 recruitment by KDM5C likely contributed to gene expression activation. A similar pattern was also observed via RNA-seq with HK-2 nontumoral cells (Figure EV4). Compared with the loss of KDM5C or YY1 alone, double knockdown caused transcriptional repression of a sharply increased number of genes (1099 newly identified downregulated genes in total), as shown in Figure 5D. \n",
      "\n",
      "\n",
      "\n",
      "Given that depletion of both YY1 and KDM5C led to significant suppression of tumor cell proliferation (Figure 4), we next investigated the gene expression patterns by analyzing the RNA-seq results and found that the expression of several cell cycle- and cell apoptosis-related genes, including JAM3, KNTC1, SMC2, SMC4, LMNB1 and FOXM1, was markedly downregulated in the double-knockdown cell group (Figure 5E). These results were verified by qPCR in ACHN cells (Figure 5F).\n",
      "\n",
      "We compared the RNA-seq results with our CUT&Tag data, and found that almost half of the downregulated genes (49.96%) in the KDM5C/YY1 double-knockdown group were annotated in the genomic regions where YY1 and KDM5C overlapped, as shown in Figure EV5B. We then analyzed the YY1 peaks at the promoters of individual genes via the IGV browser. As shown in Figure EV5C-D, KDM5C depletion reduced theYY1 peak intensity at the promoters of the indicated genes, and the presence of KDM5C increased the amount of YY1 recruited to these promoters. KDM5C-H514A marginally increased the YY1 intensity in the indicated regions. Moreover, ChIP-qPCR was performed in HK-2 cells and the result revealed that YY1 enrichment at the promoter region of the indicated genes was increased after KDM5C-WT was expressed, whereas KDM5C-H514A failed to increase YY1 binding (Figure EV2C). These results strongly suggest that KDM5C increases the expression of essential cell-fate genes by recruiting YY1 to their promoters. \n",
      "\n",
      "Simultaneous loss of KDM5C and YY1 significantly promotes cell cycle arrest and cell apoptosis\n",
      "\n",
      "Previous studies have shown that SMC2/4, LMNB1, and KNTC1 are involved in cell cycle arrest and apoptosis regulation. To confirm the synergistic effect of KDM5C and YY1 \n",
      "\n",
      "\n",
      "\n",
      "on tumorigenicity and cancer cell proliferation, we analyzed the cell cycle progression and apoptosis rate of tumor cells in which KDM5C, YY1 or both had been depleted. We first measured the ratio of renal cancer cells in each phase of the cell cycle, and the results indicated that the loss of both KDM5C and YY1 caused an increased ratio of 769-P tumor cells arrested in the G2/M phase. Similar results were also observed in KDM5C-deficient RCC4 cells after YY1 depletion (Figure 6A). Moreover, double knockdown of KDM5C and YY1 but not loss of either gene individually greatly increased the percentage of apoptotic cells in 769-P tumors. In addition, the absence of YY1 in RCC4 cells induced a significant increase in the percentage of apoptosis cells (Figure 6B). These results confirmed the inhibitory effect of YY1 depletion on KDM5C-deficient cells (Figure 4). Taken together, our study revealed a synergistic regulatory effect of KDM5C and YY1 on cancer cell proliferation and cell cycle progression.\n",
      "\n",
      "\n",
      "\n",
      "Discussion\n",
      "\n",
      "As a member of the GLI-Kruppel family of zinc finger DNA-binding proteins, YY1 has been proven to regulate genes either by directly binding to a promoter or indirectly by associating with chromatin-remodeling proteins or histone modifiers (Gordon et al, 2006; Khachigian, 2018). YY1 is frequently overexpressed in many cancer types and reshapes the oncogenic gene expression network(Bonavida & Kaufhold, 2015; Sarvagalla et al, 2019). Although YY1 can directly bind to a promoter DNA sequence, the mechanism underlying its recruitment to chromatin is still not fully understood. In addition, although targeting YY1 might be an attractive therapeutic strategy, a robust biomarker to analyze its effect is lacking. In this study, we found that YY1 is a KDM5C-associated protein, and the JmjC domain of KDM5C was essential for the KDM5C–YY1 interaction. Moreover, we demonstrated that KDM5C mediates the chromatin recruitment of YY1, which is important for YY1 binding to promoter regions and the transcription of cell fate-related genes. Importantly, targeting YY1 effectively inhibited the proliferation and tumorigenicity of tumor cells with low KDM5C expression. Considering these findings, we propose the following model: YY1 chromatin recruitment is at least partially dependent on the presence of KDM5C, and in tumor cells with high KDM5C expression, a small amount of YY1 can be recruited to chromatin even after KDM5C is knocked down. In contrast, depleting YY1 in low KDM5C-expressing tumor cells completely eliminated chromatin-bound YY1 and promoted tumor regression (Figure 7).\n",
      "\n",
      "As a specific H3K4 histone demethylase, KDM5C plays dual roles as both a proto-oncogene and tumor suppressor(Chen et al., 2018; Ji et al, 2015; Shen et al, 2016; Shen \n",
      "\n",
      "\n",
      "\n",
      "et al, 2021). KDM5C interacts with specific transcription factors, such as MYC(Outchkourov et al, 2013; Secombe et al, 2007), and downregulates gene expression by reducing H3K4me2/3 levels at enhancer or promoter regions. However, studies, including our previous reports, have shown that KDM5C also upregulates the expression of certain genes, but the relevant molecular mechanism of KDM5C effect on gene expression is unclear(Outchkourov et al., 2013; Scandaglia et al, 2017; Zheng et al., 2021). A recent study showed that the histone demethylase KDM5B recruited the H3K9 methyltransferase SETDB1 to suppress antitumor immunity in a KDM5B demethylase-independent manner, implying that the scaffolding function of this epigenetic regulator was important in certain contexts, a finding that has been appreciated to a lesser degree in prior studies(Zhang et al., 2021). Our finding that KDM5C recruited YY1 to chromatin in a histone demethylase-independent manner reinforces the evidence suggesting that the scaffolding function of epigenetic regulators might be critical for tumorigenesis. Indeed, epigenetic inhibitors targeting the catalytic domain often show limited efficacy and intolerable toxicity, and our findings strongly suggest that targeting the scaffolding function should be considered in the development of new cancer prevention strategies and therapies.\n",
      "\n",
      "Abnormalities in enzymes that regulate epigenetic modifications are important drivers of tumor occurrence and cancer progression(Dawson, 2017). Therefore, epigenome-based therapies might be developed to block the dysregulated expression of proto-oncogenes in tumor cells and increase antitumor responses(Miranda Furtado et al., 2019; Nebbioso et al, 2018). KDM5C has been shown to be frequently mutated or aberrantly expressed in various cancers. The degree of YY1 chromatin recruitment was decreased in \n",
      "\n",
      "\n",
      "\n",
      "KDM5C-deficient tumor cells, and further inhibition of YY1 expression might completely suppress YY1-directed transcription and significantly impair the proliferation of tumor cells. Furthermore, simultaneously targeting KDM5C and YY1 effectively inhibited tumor growth in KDM5C-proficient tumor cells. In recent years, the induction of synthetic lethality has been applied to the identification of novel drug targets, opening new avenues to tumor-targeted therapy(Nijman & Friend, 2013). Considering the definition of synthetic lethality and the results presented here, we propose that KDM5C and YY1 form a synthetically lethal gene pair, providing a rationale for the clinical treatment of patients with KDM5C-deficient cancer by targeting YY1.\n",
      "\n",
      "In summary, our work provides mechanistic insight into YY1 chromatin recruitment and suggests an important role for the scaffolding function of the histone demethylase KDM5C. Because proteolysis-targeting chimeras (PROTACs) are used to target and degrade proteins with important scaffolding functions and serve as a potential therapeutic strategy in cancer, future work focusing on the design and synthesis of KDM5C PROTAC drugs is important, and further studies in this research direction are warranted.\n",
      "\n",
      "\n",
      "\n",
      "Methods\n",
      "\n",
      "Vectors and plasmids\n",
      "\n",
      "The pHAGE vector carries a puromycin resistance cassette, endowing the transfected cells with resistance to puromycin, hence enabling us to select and generate stable cells. The overexpression vectors used in our context (KDM5C or YY1 with HA or Flag tags and their truncated constructs) were constructed using pHAGE for mammalian cell expression. The KDM5C enzymatically inactive mutant H514A was generated via site-directed mutagenesis PCR from the pHAGE-KDM5C plasmid. shRNAs targeting human KDM5C and YY1 were constructed using pLKO.1. The construction of pLKO.1 is detailed on the addgene website. All of the constructs were validated by DNA sequencing. The targeted sequences of KDM5C and YY1 genes are as follows:\n",
      "\n",
      "shCtrl: 5’-CAACAAGATGAAGAGCACCAA-3’; \n",
      "\n",
      "shKDM5C-1: 5’-AGTACCTGCGGTATCGGTATA-3’; \n",
      "\n",
      "shKDM5C-2: 5’-GCCACACTTGAG GCCATAATC-3’; \n",
      "\n",
      "shYY1-1: 5’-GACGACGACTACATTGAACAA-3’; \n",
      "\n",
      "shYY1-2: 5’-GCCTCTCCTTTGTATATTATT-3’. \n",
      "\n",
      "Cell culture\n",
      "\n",
      "The human kidney proximal tubular epithelial cell line HK-2 and human renal cancer cell lines (ACHN, 769-P and RCC4) were obtained from the American Type Culture Collection (ATCC). HEK293T cells were obtained from the Institute of Cellular Resources, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences. HK-2, ACHN, \n",
      "\n",
      "\n",
      "\n",
      "RCC4, and HEK293T cells were cultured in high-glucose Dulbecco’s modified Eagle’s medium (DMEM, HyClone) while 769-P cells were cultured in Roswell Park Memorial Institute-RPMI medium(1640, HyClone) and both media were supplemented with 10% fetal bovine serum (FBS, Cell-box, AUS-01S-02) in a humidified atmosphere containing 5% CO2 at 37 ℃. The cell lines were authenticated by STR profiling, and mycoplasma testing was performed monthly with a MycoBlue Mycoplasma Detector (Vazyme, D101-01).\n",
      "\n",
      "For transient protein expression, the cells were transfected with the indicated constructs via NEOFECT DNA transfection reagent (NeoFect, TF20121201) according to the manufacturer’s instructions. After 48 h of transfection, the cells were harvested for the following assays. To generate stable overexpression or knockout cell lines, appropriate constructs and two packaging plasmids psPAX2 and pMD2.G were cotransfected into HEK293T cells. After 60 h, the lentivirus particles from the culture medium were collected and filtered with 0.45 µm filter, and then added to the host cells along with 8 μg/mL polybrene (Solarbio, H8761). Infected cells were then selected in puromycin (Solarbio, P8230)-containing medium. The protein levels of the target genes were determined via western blotting with the corresponding antibodies. To generate the double-knockdown cells, we selected a GFP-labeled KDM5C knockdown plasmid (pLKO.1-GFP) to infect cells depleted of YY1, and then screened the cells for puromycin resistance and green fluorescence. To restore KDM5C or H514A in knockdown cells, we inserted a hygromycin resistance gene into pHAGE-KDM5C/H514A-puro, which could be used to screen cells expressing KDM5C/H514A. The cells involved in the cell cycle and apoptosis assays did not contain fluorescence, which might have affected the results. We confirmed protein \n",
      "\n",
      "\n",
      "\n",
      "expression via Western blotting. \n",
      "\n",
      "RNA extraction and qRT-PCR\n",
      "\n",
      "Total RNA was extracted using the RNA isolater Total RNA Extraction Reagent (Vazyme, R401-01) according to the manufacturer’s instructions. Reverse transcription reactions were performed by using ReverTra Ace qPCR RT Master Mix with gDNA Remover (TOYOBO, FSQ-301). RT-PCR was performed with ChamQ SYBR qPCR Master Mix (Vazyme, Q311-02) on a CFX ConnectTM Real-Time System (Bio-Rad). The housekeeping gene expression levels of β-actin or GAPDH were used for normalization. Each reaction was performed in triplicates. For quantification of gene expression, the 2-△△Ct method was used. The primer sequences of indicated genes were as follows:\n",
      "\n",
      "h-JAM3-RT \n",
      "\n",
      "Forward : 5’-GAGACTCAGCCCTTTATCGC-3’;\n",
      "\n",
      "Reverse : 5’- TGTTGCCATCTTGCCTACTG-3’; \n",
      "\n",
      "h-KNTC1-RT\n",
      "\n",
      "Forward : 5’-CTCACGAAGTTGCACAGG-3’;\n",
      "\n",
      "Reverse : 5’-GCCTCCAGCTCTTGTTTG-3’;\n",
      "\n",
      "h-SMC2-RT\n",
      "\n",
      "Forward : 5’-GTTTTGGATGGTCTGGAGTT-3’;\n",
      "\n",
      "Reverse : 5’-ATTGGAGCAGGTTTGAAGAG-3’;\n",
      "\n",
      "h-SMC4-RT\n",
      "\n",
      "Forward : 5’-TCGACCACCTAAGAAAAGTTG-3’;\n",
      "\n",
      "\n",
      "\n",
      "Reverse : 5’-CCATGAAGTAAAGGGGAGTG-3’;\n",
      "\n",
      "h-LMNB1-RT\n",
      "\n",
      "Forward : 5’-CTGTCTCCAAGCCCTTCTT-3’;\n",
      "\n",
      "Reverse : 5’-ACTCGCCTCTGATTCTTCC-3’;\n",
      "\n",
      "h-FOXM1-RT\n",
      "\n",
      "Forward : 5’-GCTGCCCCTTCCTGTTC-3’;\n",
      "\n",
      "Reverse : 5’-TGGACTCTGCCACTTCCTT-3’;\n",
      "\n",
      "h-GAPDH-RT \n",
      "\n",
      "Forward : 5’-ACATCAAGAAGGTGGTGAAG-3’;  \n",
      "\n",
      "Reverse : 5’-CTGTTGCTGTAGCCAAATTC-3’;\n",
      "\n",
      "Western blotting\n",
      "\n",
      "The cell pellets were lysed with RIPA buffer (Beyotime, P0013B) containing a protease inhibitor mixture cocktail (MedChemExpress, HY-K0010). Then, the cells were centrifuged at 4 ℃ at 13,000 rpm for 10 min to remove insoluble cellular components. Total protein concentrations were measured with a BCA protein assay kit (Beyotime, P0012). Proteins (20–100 μg) were separated via 8-15% sodium dodecyl sulphate-polyacrylamide gel electrophoresis (SDS-PAGE) and transferred to nitrocellulose membranes (NC, PALL, 66485) in Tris-glycine buffer. The blots were blocked with 5% nonfat milk then incubated with diluted primary antibodies at 4 ℃ overnight. Then, appropriate horseradish peroxidase (HRP)-conjugated secondary antibodies were incubated with the blots for 1 h at room temperature. The signals were visualized with an enhanced chemiluminescence \n",
      "\n",
      "\n",
      "\n",
      "detection kit (ECL, Vazyme, E411-05) and exposured to X-ray film (Sigma, America). The primary antibodies used were purchased from commercial sources: anti-KDM5C (Cell Signaling Technology, D29B9; BETHYL, A301-034A; Proteintech, 14426-1-AP), anti-YY1 (Cell Signaling Technology, D5D9Z), anti-Flag (Proteintech, 20543-1-AP), anti-HA (Proteintech, 66006-2-Ig), anti-GAPDH (Proteintech, 60004-1-Ig), anti-β-actin (Proteintech, 66009-1-Ig), anti-H3 (Cell Signaling Technology, 2650) and anti-H3K4me3 (Cell Signaling Technology, 9751).\n",
      "\n",
      "Coimmunoprecipitation (co-IP)\n",
      "\n",
      "For the immunoprecipitation assay, appropriately constructed plasmids were transfected into the HEK293T and 769-P cell lines. After 48 h of transfection, the cells were harvested and then lysed in IP lysis buffer (Beyotime, P0013) with EDTA-free protease inhibitor cocktail (MedChemExpress, HY-K0010) for 30 min on ice, sonicated, and then insoluble proteins were removed by centrifugation at 4 ℃ at 12000 rpm for 10 min. An aliquot of the cleared cell lysate was collected as an input control. The remaining lysates were subsequently incubated with anti-HA or anti-Flag magnetic beads (MedChemExpress, HY-K0201, HY-K0207) overnight at 4 ℃ with gentle shaking. \n",
      "\n",
      "For endogenous protein interactions, the cell lysates were processed as described above. One tenth of each lysates was used as the input, half of the remainder was incubated with KDM5C antibody, and the other half was incubated with homologous IgG for 6 hours at 4 ℃. The lysates were then mixed with ProteinA/G agarose and incubated overnight at 4 ℃.\n",
      "\n",
      "\n",
      "\n",
      "Magnetic beads or agarose beads with a complex of antigen and antibody were washed four times in TBS-T buffer (100 nM Tris-HCl, 0.9% NaCl, 0.1% Tween-20). The proteins pulled down by the specified antibodies were eluted with 2×SDS loading buffer through boiling the beads, and then verified by Western blotting.\n",
      "\n",
      "Tandem affinity purofication and mass spectrometry (TAP-MS)\n",
      "\n",
      "HEK293T cells overexpressing SBP-Flag-KDM5C were harvested for Tap-MS. Cell pellets were lysed with IP lysis buffer (Beyotime, P0013), and the supernatant was subsequently incubated with strep beads overnight at 4 ℃. After three washes, SBP-linked protein were elutied with biotin solution, and then incubated with anti-Flag magnetic beads (MedChemExpress, HY-K0207). After enrichment of the target proteins, we washed the beads to remove nonspecific proteins and then performed SDS-PAGE on the products of the two groups. All of the lanes of the gel were collected for mass spectrometry analysis.\n",
      "\n",
      "Proliferation ability assay（CCK8）\n",
      "\n",
      "The cells were seeded in a 96-well plate at 30-50% confluence according to the cell growth rate. When the cells just attached at the initial point, the OD450 nm was read every 24 h via a cell counting kit 8 (CCK8, MedChemExpress, HY-K0301) according to the manufacturer’s instruction. The CCK8 solution was diluted with complete medium, incubated with the cells for 1-2 h, and then read with a microplate reader (BioTek Epoch, America) at 450 nm. The cell viability assay was performed in triplicate and repeated three times. Cell growth curves were drawn and analyzed via GraphPad Prism software V9.4.1.\n",
      "\n",
      "Colony formation assay\n",
      "\n",
      "The cells were seeded into a 6-well plate at the same cell density. Each well contained 1000 cells, which were cultivated in a cell incubator. Fresh cell culture medium was added every three days. After four weeks (ACHN and 769-P) or eight weeks (RCC4), the cell culture medium was removed. The cells were gently washed with PBS, fixed with 4% paraformaldehyde for 20 min at 37 ℃, and stained with 0.1% crystal violet in PBS. The assay was performed in triplicate and repeated two or three times. \n",
      "\n",
      "Cell cycle assay\n",
      "\n",
      "The cells were gently collected at the exponential growth phase, washed with cold PBS, and then resuspended in 70% ethanol for more than 30 min at 4 ℃. Then, the cells were washed with cold PBS, incubated with propidium iodide(PI) solution (Byotime, C1052) and protected from light at 37 ℃ for 30 min. Red fluorescence was detected via flow cytometry (LSRFortessaX20, BD) at Ex=488 nm, and then analyzed with Flowjo. The assay was repeated two or three times.\n",
      "\n",
      "Cell apoptosis assay\n",
      "\n",
      "In accordance with the instructions of the Annexin V-FITC/PI apoptosis kit (Multi Sciences, AP101), we collected 1-10×10^5 cells and resuspended them in 500μl of 1×Binding buffer, and add 5 μl of Annexin V-FITC and 10 μl of PI to each sample. The samples were mixed gently and protected from light for 5 min, then Annexin V-FITC (Ex = \n",
      "\n",
      "\n",
      "\n",
      "488 nm; Em = 530 nm) and PI (Ex = 535 nm; Em = 615 nm) were detected via flow cytometry (LSRFortessaX20, BD) with. The results were analyzed via Flowjo.\n",
      "\n",
      "Isolation of chromatin-bound proteins from subcellular fractions\n",
      "\n",
      "After the transfection of HEK293T cells with empty vector, Flag-KDM5C or Flag-H514A, we collected the cells and lysed them with E1 buffer (50 nM HEPES-KOH, 140 nM NaCl, 1mM EDTA, 10% glycerol, 0.5% NP-40, 0.25% Triton X-100, 1mM DTT, Protease inhibitor cocktail). The cells were subsequently centrifuged 1,100× g at 4 ℃ for 2 min after which the supernatant was obtained as the cytoplasmic fraction. Next, we resuspended the pellet in E2 buffer (10nM Tris-HCl, 200mM NaCl, 1mM EDTA, 0.5mM EGTA, Protease inhibitor cocktail). After centrifugation, the supernatant was obtained as the nuclear fraction. We resuspended the pellet in E3 buffer (500 mM Tris-HCl, 500mM NaCl, protease inhibitor cocktail) and sonicated it on ice for 5 min, 2 s ON/45 s OFF. The supernatant was centrifuged at 4 ℃ for 10 min at 16,000× g, and transferred to fresh microcentrifuge tubes. The protein concentrations were analyzed via Western blotting.\n",
      "\n",
      "Chromatin immunoprecipitation followed by next-generation sequencing (ChIP-seq)\n",
      "\n",
      "The cells were plated in 10 cm dishes until they reached confluence and were then fixed with 37% fresh formaldehyde to a final concentration of 1% for 13 min at room temperature with rotation, after which the reaction was quenched with 0.125 mM glycine for 10 min. The final number of Chip-seq cells in each group was 4×107.\n",
      "\n",
      "The cells were washed three times with ice-cold PBS and then harvested bwith a cell \n",
      "\n",
      "\n",
      "\n",
      "scraper and lysed in lysis buffer 1 (50mM HEPES-KOH, 140mM NaCl, 1mM EDTA, 10% glycerin, 0.5% NP-40, 0.25% TritonX-100), lysis buffer 2 (pH 8.0 10mM Tris-HCl, 200 mM NaCl 1mM EDTA, 0.5mM EGTA), and lysis buffer 3 (pH 8.0 10mM Tris-HCl, 100mM NaCl, 1mM EDTA, 0.5mM EGTA, 0.2% N-lauroyl sarcosine sodium salt). The chromatin was sonicated in lysis buffer 3 to ensure that the size of the chromatin fraction was between 200-500bp. 80 μL of each sample was transferred to a clean tube as the input DNA. The chromatin sample was diluted with ChIP dilution buffer (20mM Tris-HCl, 150mM NaCl, 1mM EDTA, 1% TritonX-100). 6 μL of antibody was added to each tube and the mixturewas incubated at 4 ℃ overnight. To each tube, 80μL of protein G Sepharose 4 Fast Flow (GE Healthcare, 17061805) was added, and the immune complexes were collected by incubation at 4 ℃ for at least 6 h with gentle rotation. The bound complexes were washed once with each of the following buffers: low-salt wash buffer (0.1% SDS, 1% TritonX-100, 2 mM EDTA, 20 mM Tris-HCl pH 8.0, 150 mM NaCl), high-salt wash buffer (0.1% SDS, 1% Triton X-100, 2 mM EDTA, 20 mM Tris-HCl pH 8.0, 500 mM NaCl), LiCl wash buffer (0.25 M LiCl, 1% NP-40, 1mM EDTA, 10 mM Tris-HCl pH 8.0) and TE buffer (10 mM Tris-HCl pH 8.0, 1mM EDTA). The beads were incubated at 65 ℃ with TE buffer containing proteinase K, RNase and SDS overnight, and the DNA was extracted using phenol/chloroform. ChIP-seq was performed by Beijing Novogene Biotech Co., Ltd.\n",
      "\n",
      "ChIP-qPCR\n",
      "\n",
      "The qPCR templates were obtained from ChIP. For method details, please refer to “qRT-PCR”. We calculated the fold enrichment of those genes via YY1-ChIP, and IgG was \n",
      "\n",
      "\n",
      "\n",
      "used as a negative control. The calculation formula is as followed:\n",
      "\n",
      "ΔCt [normalized ChIP] = (Ct [ChIP] - (Ct [Input] -Log2 (Input Dilution Factor))\n",
      "\n",
      "ΔΔCt [ChIP/NIS] = ΔCt [normalized ChIP] - ΔCt [ IgG ]\n",
      "\n",
      "Fold Enrichment = 2^ (-ΔΔCt [ChIP/NIS])\n",
      "\n",
      "Primer sequences used in ChIP-qPCR are as followed:\n",
      "\n",
      "JAM3\n",
      "\n",
      "Forward : 5’-TGGTAACTGGGGCGGGTCGCA-3’;\n",
      "\n",
      "Reverse : 5’-AGCTTCGGCCCGCTCTGCTTC-3’;\n",
      "\n",
      "KNTC1\n",
      "\n",
      "Forward : 5’-CAACACCGCCTTACTTCC-3’;\n",
      "\n",
      "Reverse : 5’-CCGGGACTCTGAACTATGGC-3’;\n",
      "\n",
      "SMC2\n",
      "\n",
      "Forward : 5’-GGAGCTACCATTATCGAGAC-3’;\n",
      "\n",
      "Reverse : 5’-CTTGACGCACCCCAAAAG-3’;\n",
      "\n",
      "SMC4\n",
      "\n",
      "Forward : 5’-AAAAGAATCCCTCGCTCTTC-3’;\n",
      "\n",
      "Reverse : 5’-TGACAGCAAGAGAAATCTCC-3’;\n",
      "\n",
      "LMNB1\n",
      "\n",
      "Forward : 5’-TTGCCCAAGGGCCAGATTTTA-3’;\n",
      "\n",
      "Reverse : 5’-AATGCAAGACGAGGGTGACT-3’;\n",
      "\n",
      "FOXM1\n",
      "\n",
      "Forward : 5’-ATTTTCCCACAGTGAACGAT\n",
      "\n",
      "\n",
      "\n",
      "Reverse : 5’-AACCTTGTCTGCCATTGTATC\n",
      "\n",
      "Cleavage Under Targets and Tagmentation assay\n",
      "\n",
      "The Cleavage Under Targets and Tagmentation (CUT&Tag) assay was conducted according to the manual instructions of Hyperactive Universal CUT&Tag Assay Kit for Illumina Pro (Vazyme, TD904). A ChIP-grade anti-KDM5C antibody (Bethyl, A301-034A) and anti-YY1 (CST, D5D9Z) were used for immunoprecipitation. Rabbit IgG (ABclonal, AC042) was used as a negative control. The library was constructed with the TruePrep Index Kit V2 for Illumina (Vazyme, TD202). \n",
      "\n",
      "ChIP-seq and CUT&Tag data analysis\n",
      "\n",
      "The raw data (raw reads) in fastq format were first processed via fastp software to obtain clean data, from which reads containing adapters were removed. The reference genome hg38 and gene model annotation files were downloaded directly from the genome website. The index of the reference genome was built using Bowtie2, and the clean reads were aligned to the reference genome with Bowtie2. After the reads were mapped to the reference genome, MACS2 peak-calling software was used to identify regions enriched with IP over the background. A q-value threshold of 0.05 was used for all of the datasets. Only peaks with counts greater than 200 were retained in the CUT&Tag data. Bedtools reldist and bedtools intersect were used to compare two Chip-seq or CUT&Tag datasets in the global or promoter region. Motif analysis was performed via Homer. Chipseeker was used to analyze genomic distribution.\n",
      "\n",
      "RNA-seq and data analysis\n",
      "\n",
      "Total RNA was extracted from the corresponding cells. The concentration was measured by Qubit 3.0 fluorometer (Life Technologies) and RNA purity was checked via the kaiaoK5500®Spectrophotometer (Kaiao, Beijing, China). Sample integrity was assessed using an Agilent 2100 RNA Nano 6000 assay kit (Agilent Technologies). RNA-seq was performed by Annoroad Gene Technology Corporation (Beijing).\n",
      "\n",
      "Briefly, sequencing libraries were constructed using the NEBNext Ultra RNA Library Prep Kit for Illumina (NEB, USA). Clustering of the index-coded samples was performed on a cBot cluster generation system using the HiSeq PE Cluster Kit v4-cBot-HS (Illumina) according to the manufacturer’s instructions. After cluster generation, the libraries were sequenced on an Illumina platform and 150 bp paired-end reads were generated.\n",
      "\n",
      "For the data analysis, an index of the reference genome was built using HISAT2 v2.0.5 and paired-end clean reads were aligned to the reference genome via HISAT2 v2.0.5. FeatureCounts v1.5.0-p3 was used to count the numbers of reads mapped to each gene. Differential expression analysis of the two groups was performed using the DESeq2 R package. Heatmaps were generated via pheatmap (v1.0.12) package in R.\n",
      "\n",
      "Xenograft assay\n",
      "\n",
      "All animal experiments were approved by the Animal Research Ethics Board of Wuhan University (WAEF-2022-0113) and confirmed by the animal experimental guidelines. Four-week-old female BALB/c nude mice were purchased from Beijing Vital River Laboratory \n",
      "\n",
      "\n",
      "\n",
      "Animal Technology (China) and were housed under specific pathogen-free conditions in Animal Experiment Center-Animal Biosafety Level-III Laboratory of Wuhan University. The animals were randomly divided into four groups of 6 mice each. The mice were injected with 100 μL of cells suspended in Matrigel Basement Membrane Matrix (Corning, 356234) at a population of 3 × 106 ACHN indicated cells into the left or right dorsal flank subcutaneously.\n",
      "\n",
      "The tumor size was estimated once a week via a digital Vernier caliper. The tumor volume was calculated with the following formula: tumor volume (mm3) =length (mm)×width2 (mm)× 0.5. Tumors were havested 8 weeks after injection.\n",
      "\n",
      "Statistical analysis\n",
      "\n",
      "The quantitative experiment was repeated three times or more independently. The graphs were generated with GraphPad Prism software. Two-tailed Student’s t-test, one-way ANOVA and the Wilcoxon rank-sum test were used to assess the significance of the experiments. P values < 0.05 were considered to indicate the significant differences. \n",
      "\n",
      "\n",
      "\n",
      "Data Availability\n",
      "\n",
      "The sequencing data are available in GEO Data Sets with the accession number GSE270718 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE270718), GSE270719 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE270719) and GSE275838 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE275838). All other raw data support the findings of this study are available from the corresponding author upon reasonable request.\n",
      "\n",
      "\n",
      "\n",
      "Acknowledgments\n",
      "\n",
      "This work was supported by Natural Science Foundation of China (grant numbers 82203491 and 82173137), Natural Science Foundation of Hubei Province of China (grant number 2022CFA008), China Postdoctoral Science Foundation (grant number 2022TQ0237), Natural Science Foundation of Inner Mongolia (grant number 2024QN08005) and Translational Medicine and Interdisciplinary Research Joint Fund of Zhongnan Hospital of Wuhan University (grant number ZNJC202212). \n",
      "\n",
      "\n",
      "\n",
      "Disclosure Statement & Competing Interests \n",
      "\n",
      "These authors declare that they have no conflict of interest.\n",
      "\n",
      "\n",
      "\n",
      "References\n",
      "\n",
      "Bonavida B, Kaufhold S (2015) Prognostic significance of YY1 protein expression and mRNA levels by bioinformatics analysis in human cancers: a therapeutic target. Pharmacol Ther 150: 149-168\n",
      "\n",
      "Chen X, Loo JX, Shi X, Xiong W, Guo Y, Ke H, Yang M, Jiang Y, Xia S, Zhao M et al (2018) E6 Protein Expressed by High-Risk HPV Activates Super-Enhancers of the EGFR and c-MET Oncogenes by Destabilizing the Histone Demethylase KDM5C. Cancer Res 78: 1418-1430\n",
      "\n",
      "Dawson MA (2017) The cancer epigenome: Concepts, challenges, and therapeutic opportunities. Science 355: 1147-1152\n",
      "\n",
      "Gao P, Li L, Yang L, Gui D, Zhang J, Han J, Wang J, Wang N, Lu J, Chen S et al (2019) Yin Yang 1 protein ameliorates diabetic nephropathy pathology through transcriptional repression of TGFbeta1. Sci Transl Med 11\n",
      "\n",
      "Gordon S, Akopyan G, Garban H, Bonavida B (2006) Transcription factor YY1: structure, function, and therapeutic implications in cancer biology. Oncogene 25: 1125-1142\n",
      "\n",
      "Hanahan D (2022) Hallmarks of Cancer: New Dimensions. Cancer Discov 12: 31-46\n",
      "\n",
      "Hogg SJ, Beavis PA, Dawson MA, Johnstone RW (2020) Targeting the epigenetic regulation of antitumour immunity. Nat Rev Drug Discov 19: 776-800\n",
      "\n",
      "Iwase S, Lan F, Bayliss P, de la Torre-Ubieta L, Huarte M, Qi HH, Whetstine JR, Bonni A, Roberts TM, Shi Y (2007) The X-linked mental retardation gene SMCX/JARID1C defines a family of histone H3 lysine 4 demethylases. Cell 128: 1077-1088\n",
      "\n",
      "Ji X, Jin S, Qu X, Li K, Wang H, He H, Guo F, Dong L (2015) Lysine-specific demethylase 5C promotes hepatocellular carcinoma cell invasion through inhibition BMP7 expression. BMC Cancer 15: 801\n",
      "\n",
      "Khachigian LM (2018) The Yin and Yang of YY1 in tumor growth and suppression. Int J Cancer 143: 460-465\n",
      "\n",
      "Li Y, Li J, Li Z, Wei M, Zhao H, Miyagishi M, Wu S, Kasim V (2022) Homeostasis Imbalance of YY2 and YY1 Promotes Tumor Growth by Manipulating Ferroptosis. Adv Sci (Weinh) 9: e2104836\n",
      "\n",
      "Meliala ITS, Hosea R, Kasim V, Wu S (2020) The biological implications of Yin Yang 1 in the hallmarks of cancer. Theranostics 10: 4183-4200\n",
      "\n",
      "Miranda Furtado CL, Dos Santos Luciano MC, Silva Santos RD, Furtado GP, Moraes MO, Pessoa C (2019) Epidrugs: targeting epigenetic marks in cancer treatment. Epigenetics 14: 1164-1176\n",
      "\n",
      "Nebbioso A, Tambaro FP, Dell'Aversana C, Altucci L (2018) Cancer epigenetics: Moving forward. PLoS Genet 14: e1007362\n",
      "\n",
      "Nijman SM, Friend SH (2013) Cancer. Potential of the synthetic lethality principle. Science 342: 809-811\n",
      "\n",
      "Outchkourov N, MuinO J, Kaufmann K, Van Ijcken WJ, Koerkamp MJ, Van LD, P. GD, Holstege FP, Grosveld F, Timmers HT (2013) Balancing of Histone H3K4 Methylation States by the Kdm5c/SMCX Histone Demethylase Modulates Promoter and Enhancer Function. Cell Reports 3: 1071-1079\n",
      "\n",
      "Qiao K, Ning S, Wan L, Wu H, Wang Q, Zhang X, Xu S, Pang D (2019) LINC00673 is activated by YY1 and promotes the proliferation of breast cancer cells via the miR-515-5p/MARK4/Hippo signaling pathway. J Exp Clin Cancer Res 38: 418\n",
      "\n",
      "Sarvagalla S, Kolapalli SP, Vallabhapurapu S (2019) The Two Sides of YY1 in Cancer: A Friend and \n",
      "\n",
      "\n",
      "\n",
      "a Foe. Front Oncol 9: 1230\n",
      "\n",
      "Scandaglia M, Lopez-Atalaya JP, Medrano-Fernandez A, Lopez-Cascales MT, Del Blanco B, Lipinski M, Benito E, Olivares R, Iwase S, Shi Y et al (2017) Loss of Kdm5c Causes Spurious Transcription and Prevents the Fine-Tuning of Activity-Regulated Enhancers in Neurons. Cell Rep 21: 47-59\n",
      "\n",
      "Secombe J, Li L, Carlos L, Eisenman RN (2007) The Trithorax group protein Lid is a trimethyl histone H3K4 demethylase required for dMyc-induced cell growth. Genes Dev 21: 537-551\n",
      "\n",
      "Shen H, Xu W, Guo R, Rong B, Gu L, Wang Z, He C, Zheng L, Hu X, Hu Z et al (2016) Suppression of Enhancer Overactivation by a RACK7-Histone Demethylase Complex. Cell 165: 331-342\n",
      "\n",
      "Shen HF, Zhang WJ, Huang Y, He YH, Hu GS, Wang L, Peng BL, Yi J, Li TT, Rong R et al (2021) The Dual Function of KDM5C in Both Gene Transcriptional Activation and Repression Promotes Breast Cancer Cell Growth and Tumorigenesis. Adv Sci (Weinh) 8: 2004635\n",
      "\n",
      "Shrivastava A, Saleque S, Kalpana GV, Artandi S, Goff SP, Calame K (1993) Inhibition of transcriptional regulator Yin-Yang-1 by association with c-Myc. Science 262: 1889-1892\n",
      "\n",
      "Sui G, Affar el B, Shi Y, Brignone C, Wall NR, Yin P, Donohoe M, Luke MP, Calvo D, Grossman SR et al (2004) Yin Yang 1 is a negative regulator of p53. Cell 117: 859-872\n",
      "\n",
      "van Leenders GJ, Dukers D, Hessels D, van den Kieboom SW, Hulsbergen CA, Witjes JA, Otte AP, Meijer CJ, Raaphorst FM (2007) Polycomb-group oncogenes EZH2, BMI1, and RING1 are overexpressed in prostate cancer with adverse pathologic and clinical features. Eur Urol 52: 455-463\n",
      "\n",
      "Wang W, Qiao S, Li G, Cheng J, Yang C, Zhong C, Stovall DB, Shi J, Teng C, Li D et al (2022) A histidine cluster determines YY1-compartmentalized coactivators and chromatin elements in phase-separated enhancer clusters. Nucleic Acids Res 50: 4917-4937\n",
      "\n",
      "Zafar A, Wang W, Liu G, Wang X, Xian W, McKeon F, Foster J, Zhou J, Zhang R (2021) Molecular targeting therapies for neuroblastoma: Progress and challenges. Med Res Rev 41: 961-1021\n",
      "\n",
      "Zhang SM, Cai WL, Liu X, Thakral D, Luo J, Chan LH, McGeary MK, Song E, Blenman KRM, Micevic G et al (2021) KDM5B promotes immune evasion by recruiting SETDB1 to silence retroelements. Nature 598: 682-687\n",
      "\n",
      "Zheng Q, Li P, Zhou X, Qiang Y, Fan J, Lin Y, Chen Y, Guo J, Wang F, Xue H et al (2021) Deficiency of the X-inactivation escaping gene KDM5C in clear cell renal cell carcinoma promotes tumorigenicity by reprogramming glycogen metabolism and inhibiting ferroptosis. Theranostics 11: 8674-8691\n",
      "\n",
      "\n",
      "\n",
      "Figure Legends\n",
      "\n",
      "Figure 1 YY1 was identified as a KDM5C binding protein\n",
      "\n",
      "A. Forest plot of the hazard ratio (HR) showing the association between high expression levels of KDM5C and overall survival in multiple cancers. For each type of cancer, the HR and 95% confidence interval (CI) are plotted as red dots and horizontal lines on the basis of a Cox proportional hazards model; when the HR=1, the study object had no effect on overall survival; when the HR>1, the study object was determined to be a risk factor; and when the HR<1, the study object was identified as a protective factor.\n",
      "\n",
      "B. Strategy for identifying KDM5C-binding partners in HEK293T cells via TAP-MS.\n",
      "\n",
      "C-D. Co-IP followed by Western blotting was used to detect the interaction between KDM5C and YY1 exogenously (C) and endogenously (D) in HEK 293 cells.\n",
      "\n",
      "E. Semi-endogenous co-IP with KDM5C-Flag and YY1 in the renal cancer cell line 769-P.\n",
      "\n",
      "F. Schematic representation of KDM5C domains and corresponding constructs.\n",
      "\n",
      "G. Semi-endogenous co-IP with KDM5C truncation mutants and endogenous YY1 in HEK293T cells.\n",
      "\n",
      "Figure 2 YY1 is a vulnerable target in KDM5C-deficient tumor cells\n",
      "\n",
      "A. Western blotting showing KDM5C expression in HK-2 human kidney proximal tubular epithelial cells and renal cancer cell lines (ACHN, 769-P and RCC4).\n",
      "\n",
      "B. Effect of YY1 knockdown on the survival of the indicated tumor cell lines as measured by a colony formation assay (n=3 biological replicates).\n",
      "\n",
      "C. Proliferation of the indicated cell lines after YY1 was depleted, as measured by a CCK8 \n",
      "\n",
      "\n",
      "\n",
      "assay. The data are shown as the mean ± SD of three independent experiments after analysis by one-way ANOVA. In ACHN cells, P=0.116 (shYY1-1) and P=0.1222 (shYY1-2); in 769-P cells, P=0.7353 (shYY1-1) and P=0.9113 (shYY1-2); and in RCC4 cells, P=0.0007 (shYY1-1) and P=2E-05 (shYY1-2).\n",
      "\n",
      "D. Xenograft tumors excised from nude mice (n = 6/group) that had been subcutaneously injected with the indicated tumor cells. Statistical significance was determined by one-way ANOVA. P=1E-05\n",
      "\n",
      "Figure 3 KDM5C is essential for YY1 genome-wide chromatin recruitment\n",
      "\n",
      "A. Violin plot showing the alteration in YY1 binding peaks after KDM5C depletion. YY1 binding peaks in both ACHN parental and shKDM5C cells were counted (n=17,030). The data are presented as mean ± SD. Statistical significance was determined by the Mann-Whitney U test, P= 2.22E-16.\n",
      "\n",
      "B. Venn diagram showing the overlapping peaks of YY1 and KDM5C in the co-occupied regions. Representative data from two independent CUT-Tag experiments.\n",
      "\n",
      "C. Heatmap and metaplot showing the chromatin enrichment of YY1 at overlapping YY1 and KDM5C peak regions (n=3645) in ACHN parental and shKDM5C cells.\n",
      "\n",
      "D. Pie chart showing the annotated genomic distribution of altered YY1 peaks on different DNA elements in shKDM5C or KDM5C overexpressing cells.\n",
      "\n",
      "Figure 4 An intact JmjC domain but not KDM5C H3K4 demethylation activity is required for KDM5C-mediated YY1 recruitment\n",
      "\n",
      "A. Heatmap and metaplot showing the distribution of YY1 peaks in the overlapping regions \n",
      "\n",
      "\n",
      "\n",
      "of YY1 and KDM5C (n=3645) in ACHN cells expressing the KDM5C-WT or KDM5C-H514A mutant.\n",
      "\n",
      "B. Loss of KDMC5 resulted in increased H3K4me3 signal intensity in the YY1 and KDM5C overlapping region (n=3645) in ACHN parental and shKDM5C cells.\n",
      "\n",
      "C. Venn diagram showing the number of YY1 peaks in the H3K4me3/YY1 overlapping region (n=11087) with or without modulation by KDM5C.\n",
      "\n",
      "D-E. Heatmaps and metaplots revealed reduced YY1 enrichment at KDM5C demethylase dependent regions (n=9943) or KDM5C demethylase independent regions (n=1144) in ACHN parental and shKDM5C cells.\n",
      "\n",
      "F. KDM5C protein levels in whole-cell lysates or chromatin fractions from HEK293T cells transfected with KDM5C-WT or KDM5C-H514A mutant construct.\n",
      "\n",
      "G-I. Effect of the re-expression of the KDM5C-WT or KDM5C-H514A mutant on the proliferation of ACHN (G), 769-P (H) or RCC4 (I) cells after the depletion KDM5C, YY1 or both, as determined by the CCK8 assay. Each experiment was repeated three times with similar results. “Dsh” indicates cells in which KDM5C and YY1 were both knocked down. The data are shown as the mean ± SD of three independent experiments after analysis via Student’s t test. In ACHN cells, P=5.37E-08 (Dsh) and P=3.48E-07 (Dsh+KDM5C); in 769-P cells, P=4.45E-10 (Dsh) and P=7.49E-10 (Dsh+KDM5C); in ACHN cells, P=5.86E-09 (Dsh) and P=2.72E-07 (Dsh+KDM5C).\n",
      "\n",
      "J. Colony formation assay with the indicated cell lines(n=3). “Dsh” indicates cells with KDM5C and YY1 simultaneously knocked down.\n",
      "\n",
      "\n",
      "\n",
      "Figure 5 KDM5C and YY1 coregulate the expression of a number of essential cell-fate genes\n",
      "\n",
      "A-C. Volcano plots showing the change in the number of transcripts in shKDM5C, shYY1 or double shKDM5C ACHN cells, as determined by RNA-seq analysis (|log2 FC|>1.5, p value <0.05). “Double sh” indicates cells with KDM5C and YY1 simultaneously knocked down.\n",
      "\n",
      "D. Venn diagrams showing the number of downregulated genes in shKDM5C, shYY1 or shKDM5C/YY1 double-knockdown cells.\n",
      "\n",
      "E. Heatmap showing the normalized gene expression profile in shKDM5C, shYY1 or double sh ACHN cells. “Double sh” indicates cells with KDM5C and YY1 simultaneously knocked down.\n",
      "\n",
      "F. qPCR was used to assess the transcription of select genes in the indicated ACHN cells. Each experiment was repeated three times with similar results. “Double sh” indicates cells with KDM5C and YY1 double knocked down. The data are shown as the mean ± SD of three independent experiments after analysis by two tailed Student’s T-test. P=2.86E-05 (JAM3), P=4.06E-06 (KNTC1), P=1.31E-04(SMC2), P=9.18E-06(SMC4), P=1.91E-05(LMNB1) and P=1.47E-04 (FOXM1).\n",
      "\n",
      "Figure 6 Simultaneous targeting of KDM5C and YY1 effectively induces cell cycle arrest in the G2/M phase and apoptosis.\n",
      "\n",
      "A. Cell cycle analysis with the indicated cell lines via flow cytometry. The percentage of cells in each phase of the cell cycle is shown on the Y-axis. Each experiment was repeated \n",
      "\n",
      "\n",
      "\n",
      "three times with similar results. “Double sh” indicates cells with simultaneous knockdown of KDM5C and YY1.\n",
      "\n",
      "B. Flow cytometry–based Annexin-V/PI staining showing cells in the indicated lineages undergoing apoptosis. The quantitative results indicate the ratio of apoptotic cells. Each experiment was repeated three times with similar results. “Double sh” indicates cells with simultaneous knockdown of KDM5C and YY1. The data are shown as the mean ± SD of three independent experiments after analysis by two tailed Student’s T-test. P=5.3E-05 (RCC4) and P=1.1E-06 (769-P).\n",
      "\n",
      "Figure 7 A mechanistic hypothesis model for KDM5C-mediated YY1 recruitment. \n",
      "\n",
      "YY1 and KDM5C constitute a pair of synthetic lethal genes. KDM5C recruits YY1 to the promoter regions of cell cycle- or apoptosis-related genes. Targeting YY1 increases the vulnerability of KDM5C-deficient cancer cells.\n",
      "\n",
      "\n",
      "\n",
      "Expanded View Figure Legends\n",
      "\n",
      "Figure EV1 Validation of the constructed cell lines used in this study\n",
      "\n",
      "A-D. KDM5C and YY1 protein levels in the indicated cells were measured via Western blotting. “Dsh” indicates cells in which KDM5C and YY1 were both knocked down.\n",
      "\n",
      "E-F. Proliferation rate of the indicated cell lines upon KDM5C depletion, as determined via a CCK8 assay. The data are shown as the mean ± SD of three independent experiments after analysis via one-way ANOVA. In 769-P cells, P=0.2701 (shKDM5C -1) and P=0.4753 (shKDM5C-2); in ACHN cells, P=0.4308 (shKDM5C-1) and P=0.3869 (shYY1-2).\n",
      "\n",
      "Figure EV2 ChIP-seq experiments targeting YY1 in HK-2 cells \n",
      "\n",
      "A. Heatmap and metaplot showing the chromatin enrichment of YY1 in HK-2 cells in the presence or absence of KDM5C.\n",
      "\n",
      "B-C. ChIP-PCR assay showing the enrichment of YY1 at the promoters of the indicated genes in HK-2 cells after the depletion of YY1, KDM5C or both (B), or in cells expressing KDM5C-WT or KDM5C-H514A mutant (C). Each experiment was repeated three times with similar results. Statistical significate was determined by one-way ANOVA. In B, P=1.24E-06 (JAM3), P=2.27E-06 (KNTC1), P=2.82E-08 (SMC2), P=2.41E-05 (SMC4) and P=3.96E-05 (FOXM1), respectively. In C, P=4.53E-06 (JAM3), P=4.46E-06 (KNTC1), P=5.53E-06(SMC2), P=6.82E-06 (SMC4), P=1.05E-06 (FOXM1) and P=0.0002 (LMNB1), respectively.\n",
      "\n",
      "Figure EV3 Evaluation of the interaction between YY1 and KDM5C-WT or KDM5C-\n",
      "\n",
      "\n",
      "\n",
      "H514A mutant.\n",
      "\n",
      "Co-IP followed by Western blotting showing the interaction between YY1 and KDM5C-WT or KDM5C-H514A mutant in HEK293T cells.\n",
      "\n",
      "Figure EV4 RNA-seq in HK-2 derived cells.\n",
      "\n",
      "A-C. Volcano plots showing the change in the number of transcripts in shKDM5C, shYY1 or double-knockdown HK-2 cells, as determined by RNA-seq analysis (|log2 FC|>1.5, p value <0.05). \n",
      "\n",
      "D. Heatmap showing the normalized gene expression profile in shKDM5C, shYY1 or double sh ACHN cells. “Double sh” indicates cells in which KDM5C and YY1 were both knocked down.\n",
      "\n",
      "Figure EV5 Effects of KDM5C on YY1 enrichment at promoter regions of specific genes\n",
      "\n",
      "A-B. The connections between the ChIP-seq and RNA-seq data were analyzed, and the genomic regions of the downregulated genes in the KDM5C/YY1 double-knockdown group were annotated.\n",
      "\n",
      "C-D. ChIP-seq snapshots reflecting changes in YY1 enrichment at the promoter regions of the JAM3, LNMB1, FOXM1, KNTC1, SMC2 and SMC4 genes in the indicated cell lines. The promoters are indicated by H3K4me3 peaks.\n",
      "Metadata: {'source': './data/docx/EMBOR-2024-58727V1.docx'}\n"
     ]
    }
   ],
   "source": [
    "from langchain_community.document_loaders.word_document import UnstructuredWordDocumentLoader\n",
    "\n",
    "# Initialize the loader\n",
    "docx_loader = UnstructuredWordDocumentLoader(DOCX_FILES[5])\n",
    "\n",
    "# Load the document into LangChain\n",
    "documents = docx_loader.load()\n",
    "\n",
    "# Process each document (LangChain uses the \"Document\" abstraction)\n",
    "for document in documents:\n",
    "    # The content of the document is in the \"page_content\" attribute\n",
    "    content = document.page_content\n",
    "\n",
    "    # Titles, headings, paragraphs can be parsed further using the Unstructured elements\n",
    "    # For now, print the raw content:\n",
    "    print(content)\n",
    "\n",
    "    # Optionally, use metadata if it's available\n",
    "    metadata = document.metadata\n",
    "    print(f\"Metadata: {metadata}\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".venv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
