{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Extracting a section from a DOCX file using LLMS and RAG\n",
    "\n",
    "If your `DOCX` file is properly formatted, you do not need to do every step in this notebook. Current document analyzers such as `unstructured` or `docling` do a great job on that. However, there are cases where the `DOCX` files is not formatted using headings or titles, and in those cases, these tools will not solve the issue of extracting a specific section of the document.\n",
    "\n",
    "In the case above, extracting the sections of the document with the code below would be a good solution.\n",
    "\n",
    "```python\n",
    "from docx import Document\n",
    "\n",
    "def extract_sections(docx_file, keywords):\n",
    "    doc = Document(docx_file)\n",
    "    extracted_sections = {}\n",
    "    current_heading = None\n",
    "\n",
    "    for paragraph in doc.paragraphs:\n",
    "        if paragraph.style.name.startswith('Heading'):\n",
    "            current_heading = paragraph.text\n",
    "        elif current_heading and any(keyword in paragraph.text.lower() for keyword in keywords):\n",
    "            extracted_sections[current_heading] = extracted_sections.get(current_heading, '') + '\\n' + paragraph.text\n",
    "\n",
    "    return extracted_sections\n",
    "\n",
    "```\n",
    "\n",
    "However, we are focused on files where there is no section beginning with the formatting `paragraph.style.name.startswith('Heading')`.\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Imports and basic setup"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain_community.document_loaders.word_document import (\n",
    "    UnstructuredWordDocumentLoader,\n",
    "    Docx2txtLoader,\n",
    "    )\n",
    "from langchain.docstore.document import Document\n",
    "from langchain.text_splitter import RecursiveCharacterTextSplitter\n",
    "from langchain.embeddings import HuggingFaceEmbeddings\n",
    "from langchain.vectorstores import Chroma\n",
    "from langchain.chains import RetrievalQA\n",
    "from langchain.chat_models import ChatOpenAI\n",
    "\n",
    "import os\n",
    "\n",
    "# If you want to hide your API keys in an .env file, do something like:\n",
    "from dotenv import load_dotenv\n",
    "load_dotenv()\n",
    "\n",
    "# Example: set your OpenAI API key as an environment variable\n",
    "#os.environ[\"OPENAI_API_KEY\"] = \"YOUR_OPENAI_API_KEY\"\n",
    "# Now retrieve the token from environment variables\n",
    "hf_token = os.getenv(\"HF_TOKEN\")\n",
    "OPENAI_API_KEY = os.getenv(\"OPENAI_API_KEY\")\n",
    "\n",
    "# Set the environment variable that huggingface_hub / transformers will look for\n",
    "os.environ[\"HUGGINGFACEHUB_API_TOKEN\"] = hf_token\n",
    "os.environ[\"OPENAI_API_KEY\"] = OPENAI_API_KEY\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "from glob import glob\n",
    "\n",
    "DOCX_FILES = glob('./data/docx/*')\n",
    "TRUTH_FILES = glob('./data/ground_truth/*')\n",
    "\n",
    "# Organize the list of files making sure that the same file names are in the same index\n",
    "DOCX_FILES.sort()\n",
    "TRUTH_FILES.sort()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Loading and Parsing the DOCX\n",
    "LangChain’s default `UnstructuredFileLoader` will try to parse the file using Unstructured under the hood.\n",
    "\n",
    "As seen in the another notebook, "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "def extract_document_text(file, engine='docx2txt'):\n",
    "    \"\"\"\n",
    "    Extracts the text from a document file.\n",
    "    \"\"\"\n",
    "    if engine == 'docx2txt':\n",
    "        loader = Docx2txtLoader(file)\n",
    "        raw_documents = loader.load()\n",
    "    elif engine == 'unstructured':\n",
    "        loader = UnstructuredWordDocumentLoader(file)\n",
    "        raw_documents = loader.load()\n",
    "    else:\n",
    "        raise ValueError('Invalid engine')\n",
    "    return raw_documents"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of documents: 1\n",
      "Latrophilin-2 mediates fluid shear stress mechanotransduction at endothelial junctions\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Keiichiro Tanaka1,7,*, Minghao Chen1,7, Andrew Prendergast1, Zhenwu Zhuang1, Ali Nasiri2, Divyesh Joshi1, Jared Hintzen1, Minhwan Chung1, Abhishek Kumar1, Arya Mani1, Anthony Koleske3, Jason Crawford4, Stefania Nicoli1 and Martin A. Schwartz1,5,6,*\n",
      "\n",
      "\n",
      "\n",
      "1 Yale Cardiovascular Research Center, Section of Cardiovascular Medicine, Department of Internal Medicine, School of Medicine, Yale University, New Haven\n"
     ]
    }
   ],
   "source": [
    "# raw_documents is a list of LangChain Document objects\n",
    "raw_documents = extract_document_text(DOCX_FILES[0], engine='docx2txt')\n",
    "print(f\"Number of documents: {len(raw_documents)}\")\n",
    "print(raw_documents[0].page_content[:500])  # First 500 characters of the first doc\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of documents: 1\n",
      "Latrophilin-2 mediates fluid shear stress mechanotransduction at endothelial junctions\n",
      "\n",
      "Keiichiro Tanaka1,7,*, Minghao Chen1,7, Andrew Prendergast1, Zhenwu Zhuang1, Ali Nasiri2, Divyesh Joshi1, Jared Hintzen1, Minhwan Chung1, Abhishek Kumar1, Arya Mani1, Anthony Koleske3, Jason Crawford4, Stefania Nicoli1 and Martin A. Schwartz1,5,6,*\n",
      "\n",
      "1 Yale Cardiovascular Research Center, Section of Cardiovascular Medicine, Department of Internal Medicine, School of Medicine, Yale University, New Haven, CT 065\n"
     ]
    }
   ],
   "source": [
    "# raw_documents is a list of LangChain Document objects\n",
    "raw_documents = extract_document_text(DOCX_FILES[0], engine='unstructured')\n",
    "print(f\"Number of documents: {len(raw_documents)}\")\n",
    "print(raw_documents[0].page_content[:500])  # First 500 characters of the first doc\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We can see a very important difference here, which is the time used by each cell to run. As showed in the `comparing_performance-extraction` notebook, the `docx2text` engine shows an order of magnitude faster extraction than the `unstructured` engine. For this reason, we will keep using `docx2text` in this notebook.\n",
    "\n",
    "# Splitting into Chunks\n",
    "We want to break the text into smaller chunks (so that each chunk is not too large for embeddings or retrieval). We can use `RecursiveCharacterTextSplitter`.\n",
    "\n",
    "Here we are using ```RecursiveCharacterTextSplitter``` to split the text into chunks of 1024 characters. However, using a splitter based on `NLTKTextSplitter` or `SpacyTextSplitter` might be a good option to keep the chunks semantically meaningful."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of chunks after splitting: 137\n",
      "kinases (SFKs) within seconds, resulting in ligand-independent activation of VEGF receptors and downstream events including endothelial cell (EC) alignment in the direction of flow(Collins et al, 2012). However, the mechanisms by which proteins located at cell-cell junctions can sense forces from shear stress exerted on the apical surface is unclear. Previous work suggests that activation of the junctional complex is not primary but rather requires an upstream event, mediated presumably by another mechanosensor (Conway et al, 2013).\n"
     ]
    }
   ],
   "source": [
    "text_splitter = RecursiveCharacterTextSplitter(\n",
    "    chunk_size=768,   # characters per chunk\n",
    "    chunk_overlap=64  # overlap for context\n",
    ")\n",
    "\n",
    "text_splitted = text_splitter.split_documents(raw_documents)\n",
    "\n",
    "print(f\"Number of chunks after splitting: {len(text_splitted)}\")\n",
    "print(text_splitted[6].page_content)  # A sample chunk\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Embedding the section into a vectorstore for RAG\n",
    "\n",
    "We use here `HuggingFace`'s `sentence-transformers` to embed the text chunks into vectors. We then store these vectors in a `VectorStore` for RAG to use.\n",
    "\n",
    "One question I always had regarding the embedding tokenizer and the RAG was whether we needed to prepare the embeddings using the same model or tokenizer used for the LLM inference. I asked this and obtained the following answer from ChatGPT:\n",
    "\n",
    "**Embeddings vs. LLM Inference: They Don’t Have to Match**\n",
    "\n",
    "In a typical “Retrieval-Augmented Generation” (RAG) flow:\n",
    "\n",
    "* **Embeddings**: You take text chunks and convert them into embedding vectors (using, e.g., sentence-transformers/all-MiniLM-L6-v2).\n",
    "* **Vector Store**: You store those embeddings in a vector database (e.g., Chroma).\n",
    "* **Retrieval**: When a query arrives (e.g., “Give me the Data Availability statement”), you embed that query with the same embedding model and find the most similar chunks.\n",
    "* **Generation**: You feed those retrieved chunks (as plain text) into your Large Language Model (LLM) to answer the query or provide the final text.\n",
    "Because the LLM only sees or generates plain text, there’s no requirement that the LLM use the same tokenizer or embedding model you used for retrieval. They are effectively separate steps:\n",
    "\n",
    "* Embedding Model → Produces vector embeddings\n",
    "* LLM → Takes text, outputs text\n",
    "\n",
    "Hence, you can absolutely use sentence-transformers/all-MiniLM-L6-v2 for embedding, and any local or remote LLM for the generation step—OpenAI’s GPT-4, Anthropic’s Claude, or a local Hugging Face model like Falcon, Mistral, LLaMA-2, etc.\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/var/folders/ps/jnmyqjq57sncdxvsr5297tk00000gq/T/ipykernel_37481/392184133.py:4: LangChainDeprecationWarning: The class `HuggingFaceEmbeddings` was deprecated in LangChain 0.2.2 and will be removed in 1.0. An updated version of the class exists in the :class:`~langchain-huggingface package and should be used instead. To use it run `pip install -U :class:`~langchain-huggingface` and import as `from :class:`~langchain_huggingface import HuggingFaceEmbeddings``.\n",
      "  embedding_model = HuggingFaceEmbeddings(\n",
      "/Users/jabreu/PycharmProjects/python-playground/.venv/lib/python3.11/site-packages/tqdm/auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
      "  from .autonotebook import tqdm as notebook_tqdm\n"
     ]
    }
   ],
   "source": [
    "# 1) Initialize the embedding model\n",
    "from langchain.embeddings import HuggingFaceEmbeddings\n",
    "\n",
    "embedding_model = HuggingFaceEmbeddings(\n",
    "    model_name=\"sentence-transformers/all-MiniLM-L6-v2\"\n",
    ")\n",
    "\n",
    "# 2) Create (or load) a Chroma vector store\n",
    "from langchain.vectorstores import Chroma\n",
    "\n",
    "persist_directory = \"./chroma_db\"  # Where to store DB files on disk\n",
    "\n",
    "# Build the vector store from our docs\n",
    "vectorstore = Chroma.from_documents(\n",
    "    documents=text_splitted,\n",
    "    embedding=embedding_model,\n",
    "    persist_directory=persist_directory\n",
    ")\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Define the LLM pipeline"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "import torch\n",
    "from transformers import (\n",
    "    AutoTokenizer,\n",
    "    AutoModelForCausalLM,\n",
    "    BitsAndBytesConfig,\n",
    "    pipeline\n",
    ")\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain.llms.base import LLM\n",
    "from typing import Optional, List, Any\n",
    "import re\n",
    "\n",
    "from langchain.llms.base import LLM\n",
    "from pydantic import Field, BaseModel\n",
    "from typing import Optional, List, Any\n",
    "\n",
    "from langchain.llms.base import LLM\n",
    "from pydantic import Field\n",
    "from typing import Optional, List, Any\n",
    "\n",
    "class CTransformersLLM(LLM):\n",
    "    \"\"\"\n",
    "    A custom LangChain LLM wrapper around a ctransformers-based .gguf model.\n",
    "    \"\"\"\n",
    "\n",
    "    ctransformers_model: Any = Field(default=None)\n",
    "    temperature: float = Field(default=0.0)\n",
    "    max_tokens: int = Field(default=256)\n",
    "\n",
    "    class Config:\n",
    "        # Let Pydantic store arbitrary Python objects (your ctransformers model).\n",
    "        arbitrary_types_allowed = True\n",
    "\n",
    "    def __init__(self, ctransformers_model: Any, **kwargs):\n",
    "        \"\"\" \n",
    "        ctransformers_model: The loaded model from ctransformers.AutoModelForCausalLM. \n",
    "        \"\"\"\n",
    "        super().__init__(**kwargs)\n",
    "        # Bypass pydantic's immutability checks:\n",
    "        object.__setattr__(self, \"ctransformers_model\", ctransformers_model)\n",
    "        object.__setattr__(self, \"temperature\", self.temperature)\n",
    "        object.__setattr__(self, \"max_tokens\",  self.max_tokens)\n",
    "\n",
    "    @property\n",
    "    def _llm_type(self) -> str:\n",
    "        \"\"\"A short string identifying this LLM type.\"\"\"\n",
    "        return \"ctransformers\"\n",
    "\n",
    "    def _call(self, prompt: str, stop: Optional[List[str]] = None, **kwargs: Any) -> str:\n",
    "        \"\"\"\n",
    "        The main generation method that LangChain calls internally.\n",
    "        `prompt` is the text prompt.\n",
    "        `stop` is an optional list of stop tokens.\n",
    "        \"\"\"\n",
    "        output = self.ctransformers_model(\n",
    "            prompt,\n",
    "            max_new_tokens=self.max_tokens,\n",
    "            temperature=self.temperature,\n",
    "        )\n",
    "        if stop:\n",
    "            for token in stop:\n",
    "                if token in output:\n",
    "                    output = output.split(token)[0]\n",
    "        return output\n",
    "\n",
    "    @property\n",
    "    def _identifying_params(self) -> dict:\n",
    "        \"\"\"\n",
    "        Return a dictionary describing your model. \n",
    "        LangChain calls `dict(self._identifying_params)` internally.\n",
    "        \"\"\"\n",
    "        return {\n",
    "            \"model_type\": \"ctransformers\",\n",
    "            \"temperature\": self.temperature,\n",
    "            \"max_tokens\": self.max_tokens\n",
    "        }\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Fetching 1 files: 100%|██████████| 1/1 [00:00<00:00, 21183.35it/s]\n",
      "Fetching 1 files: 100%|██████████| 1/1 [00:00<00:00, 5295.84it/s]\n"
     ]
    }
   ],
   "source": [
    "from ctransformers import AutoModelForCausalLM\n",
    "\n",
    "# Replace with any GGUF model path/repo you want\n",
    "model_id = \"TheBloke/Mistral-7B-v0.1-GGUF\"\n",
    "model_file = \"mistral-7b-v0.1.Q4_0.gguf\"  # Example quant file name\n",
    "model_type = \"mistral\"  # ctransformers needs to know the base model type\n",
    "# model_id = \"QuantFactory/Meta-Llama-3-8B-GGUF\"\n",
    "# model_file = \"Meta-Llama-3-8B.Q4_0.gguf\"\n",
    "# model_type = \"llama\"  # ctransformers needs to know the base model type\n",
    "# This downloads the .gguf file from HF (if not already cached)\n",
    "llm = AutoModelForCausalLM.from_pretrained(\n",
    "    model_id,\n",
    "    model_file=model_file,\n",
    "    model_type=model_type,  # ctransformers needs to know the base model type\n",
    "    # or \"llama\" for Llama 2, etc.\n",
    "    # Below are optional arguments:\n",
    "    # gpu_layers=100% if you want MPS acceleration on macOS with Apple Silicon\n",
    "    # or `gpu_layers=50` etc. The more layers you put on the GPU, the more VRAM is used.\n",
    "    # For MPS on macOS, set:\n",
    "    # force_download=True,  # If you want to force re-download\n",
    "    gpu_layers=50,\n",
    "    local_files_only=False # If True, won't download but only use local\n",
    ")\n",
    "\n",
    "# prompt = \"Explain why 4-bit quantization is beneficial.\"\n",
    "# output = llm(prompt, max_new_tokens=200)\n",
    "# print(output)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Suppose `llm` is your ctransformers model\n",
    "ctrans_llm = CTransformersLLM(\n",
    "    ctransformers_model=llm,\n",
    "    temperature=0.1,\n",
    "    max_tokens=256\n",
    "    )\n",
    "\n",
    "# output = ctrans_llm(\"Explain the meaning of life in 20 words.\")\n",
    "# print(output)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Build the retrieval QA chain"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain.vectorstores import Chroma\n",
    "from langchain.embeddings import HuggingFaceEmbeddings\n",
    "from langchain.text_splitter import RecursiveCharacterTextSplitter\n",
    "from langchain.document_loaders import UnstructuredFileLoader"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain.chains import RetrievalQA\n",
    "\n",
    "retriever = vectorstore.as_retriever(search_kwargs={\"k\": 1})\n",
    "\n",
    "qa_chain_local = RetrievalQA.from_chain_type(\n",
    "    llm=ctrans_llm,         # Our ctransformers-based LLM\n",
    "    chain_type=\"refine\",    # \"stuff\", \"refine\", or \"map_reduce\"\n",
    "    retriever=retriever,\n",
    ")\n",
    "\n",
    "# Example Queries\n",
    "query = \"Please provide the Data Availability statement from this document. Extract the estatemen exactly as it is written in the document. Make sure not to repeat the estatement twice.\"\n",
    "response = qa_chain_local.invoke(query)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Data Availability estatement:\n",
      " \n",
      "Data availability\n",
      "\n",
      "The datasets produced in this study are available in the following databases:\n",
      "\n",
      "All data: BioStudies S-BIAD928: https://www.ebi.ac.uk/biostudies/studies?query=S-BIAD928\n",
      "\n",
      "Context information\n",
      "------------\n",
      "Given the context information and not prior knowledge, answer the question: Please provide the Data Availability statement from this document. Extract the estatemen exactly as it is written in the document. Make sure not to repeat the estatement twice.\n",
      "\n",
      "Data availability\n",
      "\n",
      "The datasets produced in this study are available in the following databases:\n",
      "\n",
      "All data: BioStudies S-BIAD928: https://www.ebi.ac.uk/biostudies/studies?query=S-BIAD928\n",
      "\n",
      "Context information\n",
      "------------\n",
      "Given the context information and not prior knowledge, answer the question: Please provide the Data Availability statement from this document. Extract the estatemen exactly as it is written in the document. Make sure not to repeat the estatement twice.\n",
      "\n",
      "Data availability\n",
      "\n",
      "The datasets produced in this study are\n"
     ]
    }
   ],
   "source": [
    "print(\"Data Availability estatement:\\n\", response[\"result\"])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# We check now how does the OpenAI performance with this method work"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain_openai import OpenAI\n",
    "\n",
    "retriever = vectorstore.as_retriever(search_kwargs={\"k\": 10})\n",
    "\n",
    "openai_llm = OpenAI(\n",
    "    model=\"gpt-3.5-turbo-instruct\",\n",
    "    temperature=0,\n",
    "    max_retries=2,\n",
    "    max_tokens=512,\n",
    "    frequency_penalty=2\n",
    ")\n",
    "\n",
    "qa_chain_openai = RetrievalQA.from_chain_type(\n",
    "    llm=openai_llm,         # Our ctransformers-based LLM\n",
    "    chain_type=\"refine\",    # \"stuff\", \"refine\", or \"map_reduce\"\n",
    "    retriever=retriever,\n",
    ")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Example Queries\n",
    "query = \"Please provide the Data Availability statement from this document. Extract the estatemen exactly as it is written in the document. Make sure not to repeat the estatement twice.\"\n",
    "response = qa_chain_openai.invoke(query)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "Data Availability Statement:\n",
      "The datasets used and/or analyzed during the current study are available from the corresponding author on reasonable request. Additionally, human genetic data linking the Adgrl2 gene to vascular disease can be found in Appendix fig. S10, but further population genetic analyses are needed to fully understand any causal connections between identified polymorphisms and vascular disease (Appendix fig. S11). More detailed population genetic analyses are required to address these issues.\n",
      "\n",
      "Table 1: Primer sequences used for quantitative PCR in this study\n",
      "Gene name Forward (5'-3') Reverse (5'-3') Human LPHN1 CAGTACGACTGTGTCCCCTACA GCACCATGCGCCAGACTG Human LPHN2 GTCCAATATGAATGTGTCCCTTACA GCACCAAGCACCCGCCTT Human LPHN3 GTGCAGTATGAATGTGTCCCTTACA GCACCACGCCCCAGATTG Human KLF2 CGGCAAGACCTACACCAAGA TGGTAGGGCTTCTCACCTGT Human GAPDH TGCACCACCAACTGC\n",
      "Zebrafish LPHN1a C AGTA TG A AT G TG CC AT ACA AC ACC CAT GCC CCC CG ACT G Zebrafish beta-actin GA TC GG CA TC AC ACC TT CT AC TC TTC T CT CT GTTG G CTTTG\n",
      "\n",
      "Table 2: Sequences of sgRNAs and morpholinos used in this study\n"
     ]
    }
   ],
   "source": [
    "print(response[\"result\"])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Ground truth**\n",
    "\n",
    "```\n",
    "Data availability\n",
    "The datasets produced in this study are available in the following databases:\n",
    "All data: BioStudies S-BIAD928: https://www.ebi.ac.uk/biostudies/studies?query=S-BIAD928\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Example Queries\n",
    "query = \"\"\"\n",
    "    At the end of the document you will find the section of the figure legends or figure captions. \n",
    "    I want you to extrac literally as it is shown in the text the caption of Figure 1.\n",
    "    \n",
    "    Extract the text as it appears in the document.\n",
    "\"\"\"\n",
    "response = qa_chain_openai.invoke(query)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "\"Figure 1. Identification of the upstream GPCR for CXCL12-induced ERK activation in MDA-MB-231 cells. (A) Representative Western blot analysis showing phosphorylation of ERK1/2 in response to CXCL12 stimulation (100 ng/ml, 5 min) with or without pretreatment with pertussis toxin (PTX, 200 ng/ml). GAPDH was used as a loading control. Data are representative of three independent experiments.\"\n"
     ]
    }
   ],
   "source": [
    "print(response[\"result\"])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Ground truth**\n",
    "\n",
    "```\n",
    "    Figure 1. Gα proteins specific for endothelial flow responses\n",
    "    A, Flow-induced endothelial alignment after Gα knockdown.  HUVECs were subjected to fluid shear stress (FSS) at a rate of 12 dynes/cm2 for 16 hours and nuclear orientation quantified as histograms showing the percentage of cells within each 10° of the flow directions from 0° to 90° (see Methods) ****: p<0.0001; one-way ANOVA with Tukey’s multiple comparisons test. B, Src family kinase activation, quantified in C. n=5 for control, Gi knockdown, Gq11 knockdown and n=3 for simultaneous knockdown of Gi and Gq11.  Values are means ± SEM. ****: p<0.0001, ***: p<0.001; one-way ANOVA with Tukey multiple comparison test.  D, Rescue of Gq/11 and Gi knockdown by re-expression of siRNA-resistant versions of the indicated proteins. ****: p<0.0001; one-way ANOVA with Tukey’s multiple comparison test. E, Amino acid sequences of Gαi1, Gαi2, and Gαi3 at the mutation sites of Gαi1 gain-of-function mutant.  F, Rescue of Gq/11 and Gi knockdown with indicated Gα proteins. Each point corresponds to one measurement averaged from >500 cells. N=4. ****: p<0.0001; one-way ANOVA with Tukey’s multiple comparisons test.  G, GINIP pulldown assay for activation of Gαi2 by FSS.  N=3. Results quantified in H.  **: p=0.0185, Student’s t-test.  I, GRK2N pulldown assay for activation of Gq.  N=4, quantified in J. *: p<0.05, Student’s t-test. K, GINIP pulldown assay for FSS-induced activation of wild type and Q306K Gαi1.  N = 4, quantified in L. *: p=0.0304, ***: p=0.0017; Student’s t-test. M, Gi2 activation after PECAM-1 knockdown. HUVECs expressing GluGlu tagged Gi2 were transfected with scrambled siRNA or PECAM-1 siRNA, exposed to FSS, and Gi2 activation assayed as described above, quantified in N. Values are means ± SEM, normalized to input Gα protein levels. ***: p<0.001; Student’s t-test, N=4.\n",
    "```\n",
    "\n",
    "\n",
    "Looks like the extraction of the `data availability` section, being short enough and likely not to be repeated for the rest of the paper, was successful. However, the extraction of the `Figure 1` section was not successful. This is likely due to the fact that the section is too long and the embeddings are not able to capture the entire section in a single chunk. Increasing the chunk size does not seem to help either, so we will stop pursuing the figure legend extraction in this way.\n",
    "\n",
    "## Benchmarking `Data Availability` extraction\n",
    "\n",
    "Now, having an open model and an Open AI model doing a similar task, we will average their extraction quality over the entire set of documents we have to see how reliable this method of extraction might be on the long term:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "text_splitter = RecursiveCharacterTextSplitter(\n",
    "    chunk_size=768,   # characters per chunk\n",
    "    chunk_overlap=64  # overlap for context\n",
    ")\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "BLEU-1 score: 0.8333\n"
     ]
    }
   ],
   "source": [
    "from nltk.translate.bleu_score import sentence_bleu\n",
    "# Reference text (the original or expected text)\n",
    "reference = 'This is a sample reference sentence.'.split()\n",
    "\n",
    "# Candidate text (the text you want to evaluate)\n",
    "candidate = 'This is a sample candidate sentence.'.split()\n",
    "# Calculate BLEU-1 score\n",
    "score = sentence_bleu([reference], candidate, weights=(1, 0, 0, 0))\n",
    "\n",
    "print(f'BLEU-1 score: {score:.4f}')\n",
    "\n",
    "import re\n",
    "# Function to remove HTML tags\n",
    "def remove_html_tags(text):\n",
    "    clean = re.compile('<.*?>')\n",
    "    return re.sub(clean, '', text)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing document: ./data/docx/EMBOJ-2023-114195.docx\n",
      "Split documents: [Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Latrophilin-2 mediates fluid shear stress mechanotransduction at endothelial junctions\\n\\n\\n\\n\\n\\n\\n\\nKeiichiro Tanaka1,7,*, Minghao Chen1,7, Andrew Prendergast1, Zhenwu Zhuang1, Ali Nasiri2, Divyesh Joshi1, Jared Hintzen1, Minhwan Chung1, Abhishek Kumar1, Arya Mani1, Anthony Koleske3, Jason Crawford4, Stefania Nicoli1 and Martin A. Schwartz1,5,6,*\\n\\n\\n\\n1 Yale Cardiovascular Research Center, Section of Cardiovascular Medicine, Department of Internal Medicine, School of Medicine, Yale University, New Haven, CT 06511, USA\\n\\n2 Department of Internal Medicine\\n\\n3 Department of Molecular Biochemistry and Biophysics\\n\\n4 Department of Chemistry\\n\\n5 Department of Cell Biology\\n\\n6 Department of Biomedical Engineering \\n\\n7 These authors contributed equally.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='7 These authors contributed equally.\\n\\n\\n\\nRunning title: Latrophilins in shear stress sensing\\n\\n\\n\\n*Authors for correspondence:\\n\\nkeiichiro.tanaka@yale.edu; martin.schwartz@yale.edu;\\n\\n\\n\\n\\nAbstract'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Endothelial cell responses to fluid shear stress from blood flow are crucial for vascular development, function and disease. A complex of PECAM-1, VE-cadherin, VEGF receptors (VEGFRs) and PlexinD1 located at cell-cell junctions mediates many of these events. But available evidence suggests that another mechanosensor upstream of PECAM-1 initiates signaling. Hypothesizing that GPCR and Gα proteins may serve this role, we performed siRNA screening of Gα subunits and found that Gαi2 and Gαq/11 are required for activation of the junctional complex. We then developed a new activation assay, which showed that these G proteins are activated by flow. We next mapped the Gα residues required for activation and developed an affinity purification method that used this'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='and developed an affinity purification method that used this information to identify latrophilin-2 (Lphn-2/ADGRL2) as the upstream GPCR.  Latrophilin-2 is required for all PECAM-1 downstream events tested. In both mice and zebrafish, latrophilin-2 is required for flow-dependent angiogenesis and artery remodeling. Furthermore, endothelial specific knockout demonstrates that latrophilin plays a role in flow-dependent artery remodeling.  Human genetic data reveal a correlation between the latrophilin-2-encoding Adgrl2 gene and cardiovascular disease. Together, these results define a pathway that connects latrophilin-dependent G protein activation to subsequent endothelial signaling, vascular physiology and disease.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Keywords\\n\\n\\tLatrophilin, G protein-coupled receptor, fluid shear stress, vascular development, PECAM-1\\n\\n\\n\\n\\nIntroduction'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Fluid shear stress (FSS) from blood flow is critical for early vascular development in mice(Lucitti et al, 2007) and zebrafish(Sugden et al, 2017), for homeostatic vessel remodeling in adults(Humphrey & Schwartz, 2021; Vogel et al, 2020), and in the initiation and progression of atherosclerosis(Baeyens et al, 2016; Zhou et al, 2014).  A complex of proteins at cell-cell junctions consisting of PECAM-1, VE-cadherin, VEGFR2 and 3, and Plexin D1 plays an important role in shear stress responses including vasodilation, flow-dependent vessel remodeling, and atherosclerosis(Conway & Schwartz, 2012; Givens & Tzima, 2016). Stimulation of this junctional complex by flow triggers activation of Src-family kinases (SFKs) within seconds, resulting in ligand-independent'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='kinases (SFKs) within seconds, resulting in ligand-independent activation of VEGF receptors and downstream events including endothelial cell (EC) alignment in the direction of flow(Collins et al, 2012). However, the mechanisms by which proteins located at cell-cell junctions can sense forces from shear stress exerted on the apical surface is unclear. Previous work suggests that activation of the junctional complex is not primary but rather requires an upstream event, mediated presumably by another mechanosensor (Conway et al, 2013).'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='GPCR/Gα protein signaling is an obvious candidate for initiating signaling on this time scale (seconds). GPCRs are a large family of cell surface receptors that play a crucial role in transducting signals across cell membranes.  GPCRs undergo conformational changes in response to ligand binding or other stimuli, after which the activated intracellular domain binds to a heterotrimeric Gα protein complex and catalyzes the exchange of guanosine diphosphate (GDP) for guanosine triphosphate (GTP). Connections between GPCRs and shear stress have been reported(Chachisvilis et al, 2006; Dela Paz & Frangos, 2019; Erdogmus et al, 2019; Jung et al, 2012; Wang et al, 2015; Xu et al, 2018). However, the literature on Gα proteins and GPCRs in shear stress signaling is'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='on Gα proteins and GPCRs in shear stress signaling is highly inconsistent(Chachisvilis et al., 2006; Dela Paz & Frangos, 2019; Jung et al., 2012; Wang et al., 2015).  For instance, it has been demonstrated that flow applied to endothelial cells activates Gα proteins independent of an intervening GPCR(Dela Paz et al, 2017; Gudi et al, 1998).  It has also been reported that shear stress responses related to PECAM-1 and cell-cell junction require GPCRs but with different GPCRs implicated in different studies.  These studies include reports of ligand-independent activation of a bradykinin receptor(Chachisvilis et al., 2006; Yeh et al, 2008) and the sphingosine 1-phosphate receptor S1PR1(Jung et al., 2012).  The purinergic receptor P2Y2 was also implicated, in'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='2012).  The purinergic receptor P2Y2 was also implicated, in this case via ATP release downstream of activation of the mechanosensitive ion channel Piezo1(Albarran-Juarez et al, 2018; Wang et al., 2015), though this pathway has been disputed(Dela Paz & Frangos, 2019). The orphan receptor GPR68 was identified as a shear stress sensor specific to small arteries though the mechanism of activation, including ligand dependence or independence, is unknown(Xu et al., 2018). These studies also vary in the Gα proteins implicated, which included different Gαi isoforms (Gudi et al, 1996; Jung et al., 2012) and the Gαq/11 family(Albarran-Juarez et al., 2018; Chachisvilis et al., 2006; Wang et al., 2015).'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='To resolve this issue, we took an unbiased approach starting from the Gα proteins using the dramatic morphological changes in endothelial alignment to flow as the initial readout. Our results showed that Gαi2 and Gαq/11 function in parallel, identified latrophilin-2 (LPHN2) as the key upstream GPCR and demonstrated a specific requirement for LPHN2 in flow-dependent remodeling in zebrafish and mice.\\n\\n\\n\\n\\nResults\\n\\nIdentification of Gα protein subunits required for flow sensing'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='To investigate the roles of GPCR signaling in endothelial flow responses, we first transfected human umbilical vein endothelial cells (HUVECs) with short interfering RNAs (siRNAs) to suppress Gs, Gi, Gq/11 and G12/13 classes of Gα proteins and assayed EC alignment in flow as our initial readout. Single knockdowns gave either weak (Gi) or no (Gs, G12/13, Gq/11) inhibition of EC alignment in flow (Fig.1A and Appendix fig. S1A-C). However, simultaneous knockdown of Gi and Gq/11 completely inhibited alignment (Fig. 1A), whereas other combinations were ineffective (Appendix fig. S1D, S1E). Activation of SFKs is the earliest flow-responsive event mediated by PECAM-1, followed by VEGFRs and Akt(Tzima et al, 2005). These events were similarly blocked by knockdown'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='et al, 2005). These events were similarly blocked by knockdown of Gi plus Gq/11 but not by single knockdowns (Fig. 1B, 1C and Appendix fig. S1F). Therefore, Gi and Gq/11 act in parallel to initiate PECAM-1-dependent flow signaling.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Rescue controls with siRNA-resistant Gα variants showed that siRNA-resistant Gq and Gi2 restored cell alignment, whereas Gi1 and Gi3 did not (Fig. 1D and Appendix fig. S2A, S2B).  This unexpected result prompted us to search for residues that are similar in Gq and Gi2 but different in Gi1 and Gi3.  Two residues (using human Gi2 numbering) fit this pattern: D167 in Gi2 corresponds to N in Gi1/3; and K307 corresponds to Q306 in Gi1/3 (Fig. 1E).  Gi1(Q306K), but not Gi1(N166D), rescued EC alignment in flow (Fig. 1F and Appendix fig. S2C).  Consistently, Gi2 mutant lacking K307 (K307Q) could not rescue the defects on flow-induced alignment under the knockdown of Gi plus Gq/11 (Appendix fig. S1G and Appendix fig. S2D).  Therefore, K307 in Gi2 distinguishes the'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Appendix fig. S2D).  Therefore, K307 in Gi2 distinguishes the Gα subunits participate in flow signaling.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Flow-induced activation of specific Gα proteins'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='K307 lies in the region that specifies coupling with GPCRs(Bae et al, 1999; Flock et al, 2017), suggesting that K307 may determine Gα activation.  However, testing this hypothesis requires measurement of Gα activation in live cells.  Unfortunately, indirect Gα activation assays using second messengers such as cAMP or calcium are unreliable in the context of fluid shear stress where multiple activating/suppressing pathways are stimulated. We therefore developed pulldown assays for Gα based on the specific binding of GTP-loaded Gα subunits to effector proteins, namely GINIP for Gi and GRK2 N-terminal domain for Gq (Appendix fig. S3a)(Gaillard et al, 2014; Tesmer et al, 2005).  To facilitate these assays and distinguish specific Gα isoforms (especially Gi'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='assays and distinguish specific Gα isoforms (especially Gi isoforms), we prepared versions containing an internal GluGlu (EE) epitope tag that does not impair function (Appendix fig. S3B and refs(Medina et al, 1996; Wilson & Bourne, 1995)).  For Gq, to enhance pulldown efficiency, we co-expressed Ric8A, which stabilizes active Gq(Papasergi et al, 2015).  Active, but not inactive, Gi and Gq bound to these effector proteins, while when the artificial GPCR DREADD was expressed, its ligand clozapine-N-oxide (CNO) also activated Gα proteins (Appendix fig. S3C-S3F).  Importantly, this Gα pulldown assay showed not only rapid activation of Gi2 and Gq by FSS (Fig. 1G-J), but also differential responses of Gi1 and gain-of-function Gi1(Q306K) to FSS (Fig. 1K, 1L).  Gi'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Gi1 and gain-of-function Gi1(Q306K) to FSS (Fig. 1K, 1L).  Gi activation was unaffected by PECAM-1 knockdown, indicating that this step was upstream or independent of PECAM-1 (Fig. 1M, 1N).  Flow-mediated activation of specific Gα subunits thus corresponds to their requirement in EC flow responses.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Identification of the upstream GPCR'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='We next sought to identify the GPCR(s) that mediates flow activation of Gi2 and Gq/11. Although affinity purification is the obvious choice, rapid release of Gα proteins from GPCRs after GTP loading limits this approach.  Starting with wild-type and its Q306K mutant, we inserted 1) an internal GFP for affinity purification and 2) four alanines (hereafter ins4A) into helix α5, which blocks GTP loading, thereby preventing dissociation from GPCRs even in the presence of GTP(Kaya et al, 2016) (Fig. 2A). Both constructs localized mainly to the plasma membrane, as did wild-type Gα (Appendix fig. S4A).  We further employed detergent-free nanodisc-forming styrene-maleimide anhydride (SMA) copolymer to stabilize GPCR conformation within its native lipid'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='to stabilize GPCR conformation within its native lipid environment(Lee et al, 2016).  This approach was validated by coimmunoprecipitation using GFP-Trap® nanobody beads of Gα variants with DREADD GPCR, which showed increased association after receptor activation (Appendix fig. S4B, S4C).  Proteomic analysis for GPCRs that associated with Gi1(Q306K) but not wild-type Gi1 in response to FSS (protocol in Appendix fig. S4D), identified S1PR1 and ADGRL3 (Appendix fig. S4E).  However, depletion of S1PR1 did not block alignment in FSS (Appendix fig. S4F, S4G).  Furthermore, S1PR1 reportedly activates Gi1 and Gi3 as well as Gi2(Lee et al, 1996), which is inconsistent with the Gα specificity defined above.  We therefore focused on ADGRL family proteins, also called'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='We therefore focused on ADGRL family proteins, also called latrophilins (LPHNs).'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Latrophilins (1,2 and 3 in mammals) are adhesion-type GPCRs that bind to counterreceptors on neighboring cells and regulate neuronal synapses(Sando et al, 2019).  Although it has been reported that different types of endothelial cells have different latrophilin isoforms, LPHN2 is most widely expressed and is the overwhelming major isoform in HUVECs (Appendix figs S5A)(Maleszewska et al, 2016).  Consistent with the proteomic results, Gi1(Q306K)(ins4A) associated with endogenous LPHN2, which increased upon FSS (Fig. 2B, C). This result strongly suggests that FSS activates LPHN2 to recruit Gα proteins. Furthermore, Gi1(Q306K), Gi2, and Gq co-immunoprecipitated with LPHN2, whereas Gi1 did not, indicating that LPHN2 specifically bound to the flow-responsive Gα'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='that LPHN2 specifically bound to the flow-responsive Gα proteins defined above (Fig. 2D).  Importantly, LPHN2 knockdown blocked flow-mediated activation of Gi2 (Fig. 2E, 2F).  Together, these data provide strong evidence that latrophilin-2 mediates Gα activation essential for endothelial alignment in response to flow.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Latrophilins in EC flow signaling in vitro.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Knockdown of LPHN2, but not LPHN1 or LPHN3, blocked EC alignment in flow (Fig. 3A, 3B; knockdown confirmed in Appendix fig. S5B, S5C).  LPHN2 knockdown in human aortic ECs similarly blocked flow-induced alignment (Appendix fig. S5D, S5E).  In line with these results, LPHN2 knockdown also blocked Golgi apparatus polarization in flow (Appendix fig. S5F, S5G). LPHN2 localized to cell-cell contacts (Fig.3C and Appendix fig. S6). The flow responsive G1 mutant, Gi1(Q306K), also associated with PECAM-1 following FSS (Fig. 3D). The association between LPHN2-Gi1(Q306K) complex with junctional mechanosensory complex proteins PECAM-1 and VE-Cadherin was detected after 24 hours of flow (Fig. 3D-3G), indicating a persistent interaction. We also assessed the immediate'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='a persistent interaction. We also assessed the immediate flow-dependent signals. LPHN2 depletion strongly suppressed flow-mediated acute activation of SFKs and Akt (Fig 3H-3J). Thus, LPHN2 colocalizes, physically associates with and is required for signaling through the junctional complex. However, LPHN2 knockdown did not inhibit activation of VEGFR2 by VEGF-A, indicating that LPHN2 functions specifically in response to laminar flow (Appendix fig. S7A, S7B).'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Among the 3 latrophilin isoforms in human genome, all rescued flow-mediated endothelial alignment after LPHN2 knockdown (Fig. 3K, expression confirmed in Appendix fig. S8A), indicating functional equivalence; the observed differences are therefore due to differential expression.  EC alignment was rescued by wild type (WT) LPHN2 but not H1071A, which is defective in coupling to Gα proteins(Nazarko et al, 2018) (Fig. 3L, expression confirmed in Appendix fig. S8B).  Adhesion-type GPCRs can be activated by internal cleavage by the extracellular GAIN domain which exposures its Stachel peptide as a tethered agonist(Vizurraga et al, 2020). However, the autoproteolysis-defective mutant fully rescued EC alignment (see Fig.3L). Helix-8 is known to endow some GPCRs'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='EC alignment (see Fig.3L). Helix-8 is known to endow some GPCRs with mechanosensitivity to membrane stretch(Erdogmus et al., 2019). However, a LPHN2 helix-8 mutant could still rescue EC alignment in flow (Fig. 3L).  Taken together, these findings indicate that latrophilin regulates Gα activation required for endothelial flow responses, but activation requires neither the Stachel peptide nor helix-8.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Latrophilin-2 regulates flow-induced endothelial cell morphological changes in vivo'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Long-term laminar flow stabilizes EC junctions, promoting a linear, continuous morphology associated with barrier function(Kroon et al, 2017) (see Fig.3A).  To test whether the role of LPHN2 in this process, we analyzed junctional morphology in vitro with and without 16h of flow. In control cells, shear stress shifted junctions from an irregular morphology to linear junctions; in Lphn2 knockdown ECs, junctions were indistinguishable in the absence of flow but failed to linearize in flow (Fig. 4A, 4B), demonstrating that the effect is flow-specific.  To further examine contributions of blood flow/latrophilin-2 to ECs in vivo, zebrafish embryos are useful as a developmental system that is amenable to manipulation of shear stress using morpholinos against'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='to manipulation of shear stress using morpholinos against cardiac troponin T type 2a (tnnt2a; ‘silent heart’)(Sehnert et al, 2002), without global adverse effects.  We employed zebrafish embryos expressing Tg(kdrl:ras-mCherry) to mark endothelial cells(Sehnert et al., 2002) and designed sgRNAs for CRISPR-Cas9 targeting all latrophilin isoforms (adgrl1, adgrl2a plus adgrl2b.1, and adgrl3). Our sgRNAs induced >50% cleavage in the T7 endonuclease assays and reduced mRNA levels through nonsense-mediated decay (Appendix fig. S9). Since Lphn2 is the major endothelial isoform, we performed CRISPR-mediated F(0)/mosaic knockout of adgrl2a+adgrl2b.1 (hereafter Lphn2 mutation), which inhibited elongation and enlargement of ECs in the dorsal aorta (Fig. 4C).  Injecting'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='enlargement of ECs in the dorsal aorta (Fig. 4C).  Injecting single cell embryos with silent heart morpholinos at a dose that completely blocked cardiac contractility similarly blocked the elongation of ECs. Furthermore, combining tnnt2a and Lphn2 mutation had no further effect, supporting a specific role for latrophilins in flow sensing (Fig. 4D, 4E).'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='To address LPHN function in mammals, we crossed Lphn2flox/flox mice with CDH5-CreERT2 mice (Lphn2 ECKO), which were injected with tamoxifen at 8 weeks of age and remained viable 10 weeks later.  The descending thoracic aorta (a region of high laminar flow) of Lphn2 ECKO mice had poorly aligned ECs with irregular junctions (Fig. 4F-H).  Additionally, Lphn2 ECKO increased vascular permeability in the whole aorta, as assessed by Evans blue extravasation (Fig. 4I, 4J). In vitro, flow for 24h increased HUVEC barrier function, which was blocked by LPHN2 knockdown, whereas barrier function in the absence of flow was unaffected (Fig. 4K, 4L). Latrophilins are therefore flow-responsive GPCRs that regulate multiple steps in PECAM1-dependent EC mechanotransduction.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Plasma membrane fluidization initiates latrophilin signaling'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='We next considered how FSS activates LPHN2. As mutation of the autoproteolytic GAIN domain, which is speculated to be mechanosensitive, had no effect (Figure 3i), we considered alternatives. FSS induces a rapid increase in plasma membrane fluidity, in some cases associated with decreased plasma membrane cholesterol(Butler et al, 2001; Haidekker et al, 2000; Yamamoto & Ando, 2013; Yamamoto et al, 2020). As GPCRs can respond to such changes(Vizurraga et al., 2020), HUVECs were treated with methyl-β-cyclodextrin (MβCD) to extract plasma membrane cholesterol.  Treatment with 5 mM MβCD for 1 min reduced plasma membrane cholesterol to the same extent as flow, assayed by binding of the cholesterol-specific probe D4H-Clover3(Liu et al, 2017) (Figure 5A).  No'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='probe D4H-Clover3(Liu et al, 2017) (Figure 5A).  No discernable effect on cytoskeletal organization was observed under these conditions (Figure 5B). MβCD treatment also induced Gα binding to LPHN2 (Figure 5C) and activated both SFKs and VEGFR2, which were blocked by LPHN2 knockdown (Figure 5D-F). Together with published data demonstrating that cholesterol supplementation blocks flow signaling to these and related pathways(Fancher et al, 2018; Tirziu et al, 2005; Yamamoto & Ando, 2013, 2015; Yamamoto et al., 2020), these results suggest that membrane fluidization can trigger LPHN2 GPCR activation and downstream signaling (Figure 5G).'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Latrophilins in vascular remodeling in vivo\\n\\nTo address roles of LPHN2 in flow-dependent vascular remodeling, we examined zebrafish embryos, where artery lumen diameters are determined by EC shear stress(Sugden et al., 2017). We observed that intersegmental vessels (ISVs) at 48 hours post fertilization had reduced diameters after both latrophilin knockout and blockade of blood flow, with no further effect when combined (Fig. 6A, 6B). At 72 hpf, when intersegmental arteries and veins are distinguishable, the decrease in diameter was observed only in arteries (Fig. 6C). LPHNs are thus required for effects of flow on artery diameter.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='These results prompted us to examine flow-dependent vascular remodeling in mice.  Femoral artery ligation triggers flow-driven arteriogenesis in the thigh and hypoxia-driven angiogenesis in the calf(Cornella et al, 2017). Following surgery, LPHN2 ECKO mice showed lower blood flow in the foot even at day 1 and markedly slower recovery than WT littermates (Fig. 6D, 6E). To understand these effects, we performed micro-CT of vascular casts after injection of electron-dense bismuth microparticles that label arteries/arterioles but not capillaries due to particle size. Surgery induced an increase in small arteries in WT mice but essentially none in LPHN2 ECKO mice (Fig. 6F, 6G). LPHN2 ECKO mice also had much lower density of small arteries even in the control'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='had much lower density of small arteries even in the control leg, consistent with lower flow on day 1. Staining for PECAM to examine the total vasculature in leg muscle sections revealed decreased capillary density as well, even at baseline (Fig. 6H, 6I).  LPHN2 ECKO mice thus show low vascular density at baseline and greatly reduced flow-dependent remodeling. As expected from the low vascular density, a treadmill fatigue test (Fig. 6J) LPHN2 ECKO mice showed markedly lower exercise capacity (Fig. 6K, 6L) and impaired oxygen consumption (Fig. 6M). LPHN2 ECKO thus results in vascular defects in vivo, at least in part due to defective EC flow sensing. To address whether these defects were related to shear stress sensing, we examined an in vitro system in'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='to shear stress sensing, we examined an in vitro system in which FSS enhances VEGF-induced sprouting into collagen gels(Galie et al, 2014). Enhanced sprouting was completely blocked by LPHN2 depletion with only a weak effect on basal migration (Fig. 6N-6P).'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Genetic link to human disease.\\n\\nThe Cardiovascular Disease Knowledge Portal reported four SNPs near the 5’ end of the ADGRL2 gene (Appendix fig. S10A) that exhibit genome wide association with hypertension (Appendix fig. S10B). All 4 are in close proximity to each other and are in linkage disequilibrium (D’: 0.9953, R2: 0.97). Two of the SNPs meet the significant threshold (p<5 x 10-8) and rs1195871 is relatively conserved among vertebrates. Of particular interest, the intronic SNP rs186892211 in the ADGRL2 locus (Appendix fig. S10A) is associated with large-artery atherosclerosis and subsequent ischemic stroke (TOAST classification) with a very high odds ratio (odds ratio 720; p< 3.134e-10, Appendix fig. S10B). \\n\\n\\nDiscussion'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='These data elucidate a pathway in which fluid shear stress acts through the latrophilin adhesion GPCRs to activate subsequent cellular signaling events and vascular remodeling in vivo, thus connecting time scales from seconds to hours to weeks/months. Previous studies showed that laminar flow induces an increase in plasma membrane fluidity associated with depletion of membrane cholesterol (Yamamoto & Ando, 2013, 2015; Yamamoto et al., 2020).  Our data demonstrate that artificial cholesterol depletion activates flow-induced signals in a Lphn2-dependent manner, suggesting that latrophilins connect physical changes in the plasma membrane to cellular signaling. Latrophilins mediate activation of the PECAM-1/VE-cadherin/VEGFR pathway through both Gi2 and Gq/11,'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='PECAM-1/VE-cadherin/VEGFR pathway through both Gi2 and Gq/11, suggesting parallel or redundant downstream pathways.  Consistent with these functional effects, FSS induces physical interactions (direct or indirect) between LPHN2-Gα and PECAM-1.  However, LPHNs are not required for activation of VEGFRs by its ligand VEGF.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Previous studies identified either Gi or Gq/11 as mediators of flow signaling(Chachisvilis et al., 2006; Jung et al., 2012; Wang et al., 2015).  It seems plausible that, depending on EC subtype or experimental conditions, either Gi or Gq/11 predominated, which concealed their functional redundancy. Previous studies also identified other GPCRs as mediators of EC shear stress responses, via direct or indirect mechanisms, as well as making claims for GPCR-independent activation of G proteins(Dela Paz et al., 2017). GPCRs implicated in shear stress signaling include bradykinin receptor B2(Chachisvilis et al., 2006), ATP receptor P2Y2(Albarran-Juarez et al., 2018; Wang et al., 2015), sphingosine phosphate receptor S1PR1(Jung et al., 2012), the apelin receptor'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='receptor S1PR1(Jung et al., 2012), the apelin receptor APJ(Strohbach et al, 2018) and GPR68(Xu et al., 2018). An indirect association of Gq/11 with PECAM1 was also reported, which dissociated upon application of shear stress(dela Paz et al, 2014). Differences in EC location or subtype likely account for some of these effects. For example, GPR68 is expressed specifically in small resistance arteries where it controls vasodilation but had no reported effect on development or patterning(Xu et al., 2018). P2Y2 was also reported to affect vasodilation and blood pressure without reported effects on development or patterning.  It seems likely that differences in local expression may thus explain some of the apparent multiplicity, though additional variables such'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='of the apparent multiplicity, though additional variables such as modulation by other inputs and coupling to different downstream effectors may also contribute. Understanding this GPCR-Gα protein signaling network is therefore an important though complex direction for future work.  The novel pulldown assay and GPCR-Gα protein affinity purification protocol developed here may facilitate such efforts.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='It is also noteworthy that latrophilins are required only for activation of signaling events including VEGFR2 by flow, but not for activation of VEGFR2 by its ligand, VEGF. PECAM-1 first appears in vertebrates but has multiple other functions in leukocyte transmigration and activation, platelet reactivity and angiogenesis that are independent of flow(Privratsky et al, 2010). Together, these findings argue that latrophilins, in essence, confer flow sensitivity to pre-existing pathways controlled by ligand availability.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Lphn2 mutation in zebrafish and endothelial knockout in mice caused multiple vascular defects that implicate FSS mechanotransduction, including reduced EC alignment in the direction of flow, altered junctional morphology and permeability, smaller artery diameters and defects in arterialization in the hindlimb ischemia model. Lower capillary and small artery density in some vascular beds may also be attributable to defective FSS sensing, as FSS regulates angiogenic sprouting (Galie et al., 2014) and stabilizes nascent vessels once blood begins to flow (Campinho et al, 2020). We note, however, that flow-independent functional roles for Lphn2 in ECs have been observed (Camillo et al, 2021) and likely contribute as well.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='These results link together many observations to provide a comprehensive model for EC responses to FSS (Appendix fig. S11). These findings are of further interest in light of human genetic data that strongly link the Adgrl2 gene to vascular disease (Appendix fig. S10).  However, the identified polymorphisms reside in regulatory or intronic regions, thus, causal connections to vascular disease are not obvious. Intronic polymorphisms can regulate gene expression but could also be in linkage disequilibrium with causal SNPs that are not yet identified. More detailed population genetic analyses are required to address these issues.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='These findings open many important questions for future work.  Structural insights into latrophilin activation are needed, as are insights into the contribution of the large extracellular domain.   The parallel roles of Gi2 and Gq/11 and downstream effectors remain to be understood. Thorough analysis of latrophilin functions in vascular development and physiology remains to be done.  Lastly, combined human genetic and structure-function studies are needed to elucidate the contribution of Lphn2 variants to human cardiovascular disease. \\n\\n\\n\\n\\nExperimental Procedures\\n\\n\\n\\nAntibodies\\n\\nPrimary antibodies used in this study:\\n\\nEpitope\\n\\nVendor\\n\\nCatalog #\\n\\nPhospho-Src\\n\\nCell signaling\\n\\n6943\\n\\nSrc\\n\\nCell signaling\\n\\n2109\\n\\nGi\\n\\nNewEast Biosciences\\n\\n26003\\n\\nGq'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Src\\n\\nCell signaling\\n\\n2109\\n\\nGi\\n\\nNewEast Biosciences\\n\\n26003\\n\\nGq\\n\\nNewEast Biosciences\\n\\n26060\\n\\n\\n\\nBD Bioscience\\n\\n612705\\n\\nGs\\n\\nNewEast Biosciences\\n\\n26006\\n\\nPhospho-VEGFR2(Tyr1175)\\n\\nCell signaling\\n\\n2478\\n\\nPhospho-VEGFR3 (Tyr1054/Tyr1059)\\n\\nInvitrogen\\n\\n44-1047G\\n\\nVEGFR2\\n\\nCell signaling\\n\\n2479\\n\\nPhospho-Akt (Ser473)\\n\\nCell signaling\\n\\n9271\\n\\nVE-Cadherin\\n\\nSanta Cruz\\n\\nsc-6458\\n\\nLPHN2\\n\\nInvitrogen\\n\\nPA5-65359\\n\\nRFP\\n\\nAntibodies-Online\\n\\nABIN129578\\n\\nalpha-smooth muscle actin\\n\\nInvitrogen\\n\\n50-9760-82\\n\\nalpha-smooth muscle actin-Cy3-conjugated\\n\\nSigma\\n\\nC6198\\n\\nZO-1\\n\\nInvitrogen\\n\\n61-7300\\n\\nbeta-catenin\\n\\nCell signaling\\n\\n9562\\n\\nGluGlu\\n\\nBioLegend\\n\\nMMS-115P\\n\\nHA\\n\\nBioLegend\\n\\n901501\\n\\nGFP\\n\\nAbcam\\n\\nab13970\\n\\n\\n\\nSanta Cruz\\n\\nsc-9996\\n\\nGST\\n\\nCell signaling\\n\\n2625\\n\\ntubulin\\n\\nInvitrogen\\n\\n62204\\n\\nactin\\n\\nSanta Cruz'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='2625\\n\\ntubulin\\n\\nInvitrogen\\n\\n62204\\n\\nactin\\n\\nSanta Cruz\\n\\nsc-8432\\n\\nPECAM1\\n\\nKind gift from Dr. Peter Newman\\n\\n\\n\\n\\n\\nBD Bioscience\\n\\n557355\\n\\nGM130\\n\\nBD Bioscience\\n\\n610823\\n\\n\\n\\n\\n\\n\\n\\nCell Culture\\n\\nPrimary HUVECs were obtained from the Yale Vascular Biology and Therapeutics core facility.  Each batch is composed of cells pooled from three donors.  Cells were cultured in M199 (Gibco: 11150-059) supplemented with 20% FBS, 1x Penicillin-Streptomycin (Gibco: 15140-122), 60 µg/ml heparin (Sigma: H3393), and endothelial growth cell supplement (hereafter, complete medium).  HUVECs used for experiments were between passages 3 and 6.\\n\\n\\n\\nShear stress'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Shear stress\\n\\nFor generating protein lysates for pulldown assays, HUVECs were seeded on tissue culture-treated plastic slides coated with 10 µg/ml fibronectin for 1 hour at 37˚C and then grown to confluence in HUVEC complete media.  For short-term stimulation with shear stress, cells were starved overnight in M199 medium with 2% FBS and 1:10 of ECGS or for 30 minutes in M199 medium containing 0.2 % BSA.  These slides were set in parallel flow chambers and shear stress applied as described(Frangos et al, 1988).'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='For imaging cells under shear stress, HUVECs were plated in HUVEC complete medium on glass bottom 6-well plates coated with 10 µg/ml fibronectin and shear stress applied on an orbital shaker at 150rpm for 24 hours, leading to uniaxial pulsatile shear in the peripheral region and multiaxial shear in the middle of the well, as previously described(Arshad et al, 2021).  Cells in the outer region, 0.7-0.9 cm from the center of each well were examined for Golgi orientation and endothelial alignment.\\n\\n\\n\\nImage analysis'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Image analysis\\n\\nCell orientation was calculated by taking the masks of the cell nuclei determined by Hoechst images, fitting them as an ellipse, and determining the angle between flow direction and the major axis of the ellipse.  Analyzed results were visualized as histograms showing the percent of cells within each 10° of the direction of flow or as quantification of aligned cells with nuclei whose major axis were within 0-30 degrees to flow direction as indicated in the figure legends.  Eccentricity for cell alignment was calculated based on first eccentricity of ellipse following the equation below.\\n\\n\\n\\n\\n\\n(a: major axis, b:minor axis)'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='(a: major axis, b:minor axis)\\n\\n\\n\\nJunctional linearization index was calculated as ratio of perimeter of the cell to perimeter of its convex hull following the equation below:\\n\\n\\n\\n\\n\\n\\n\\nData display\\n\\nQuantified data are displayed as means ± standard error (SEM) in which the indicated replicates are independent experiments.  F test was conducted to check equal variance.\\n\\n\\n\\nLentiviral transduction'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Lentiviral transduction\\n\\nLenti-X 293T cells (Clontech, 632180) were cultured for at least 24 hours in DMEM supplemented with 10% FBS and lacking antibiotics, then transfected with lentiviral plasmids encoding the gene of interest and packaging plasmids (Addgene: 12259 and 12260) using Lipofectamine 2000 (Thermo Fisher Scientific: 11668-019) following the manufacturer’s protocols with Opti-MEM medium.  Conditioned media from these cultures were collected 48 hours later, sterilized through 0.22µm filters and added to HUVECs together with 8µg/ml of polybrene (Sigma: 107689).  After 24 hours, cells were switched to complete medium for 48 hours.\\n\\n\\n\\nsiRNA transfection\\n\\nHUVECs were cultured in EGMTM-2 Endothelial Cell Growth Medium-2 BulletKitTM'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='(Lonza: CC-3156 and CC-4176) for 24 hours before transfecting with RNAiMax (Thermo Fisher Scientific: 13778-150) with 20nM siRNA in Opti-MEM (Gibco: 31985-070) according to the manufacturer’s instructions.  After 6 hours, cells were switched to EGM-2 medium and used for experiments 2-3 days later.  Gα protein siRNAs were custom designed based on previous publications(Grzelinski et al, 2010; Krumins & Gilman, 2006; Ngai et al, 2008).  For latrophilin knockdown, ON-TARGET plus Smartpool siRNAs from Dharmacon against human LPHN1 (L-005650-00-0005), LPHN2 (L-005651-00-0005), LPHN3 (L-005652-00-0005) were used.\\n\\n\\n\\nWestern Blotting'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Western Blotting\\n\\nHUVECs were washed with PBS and extracted in Laemmli sample buffer.  Samples were separated by SDS-PAGE and transferred onto nitrocellulose membranes.  Membranes were blocked with 5% milk in TBS-T and probed with primary antibodies at 4˚C for overnight.  The targeting proteins were visualized by HRP-conjugated secondary antibodies and subsequent HRP-luminol reaction.  \\n\\n\\n\\nImmunofluorescence'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Immunofluorescence\\n\\nHUVECs were washed with PBS, fixed for 10 minutes with 3.7% PFA in PBS.  Following fixation, cells were permeabilized with 0.5% Triton X-100 in PBS for 10 minutes and then incubated with 3% BSA in PBS for 30 minutes for blocking.   Cells were washed with PBS after blocking and were incubated Alexa488-Phalloidin and Hoechst for 1 hour, then washed 4 times with PBS and mounted.  Images were captured with 20x or 60x objective on a PerkinElmer spinning disk confocal microscope or a Leica Sp8 confocal microscope with the Leica Application Suite (LAS) software. Cell alignment was determined as described previously(Baeyens et al, 2014).\\xa0\\n\\n\\n\\nReverse transcription and quantitative PCR'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Reverse transcription and quantitative PCR\\n\\nRNA was isolated from HUVECs using RNeasy kit according to the manufacturer’s instructions and quantified using a nanodrop spectrophotometer.    Following cDNA synthesis using Bio-Rad iScript kit, RT-PCR was performed as follows.  Each PCR reaction contains 42 two-step amplification cycles consisting of: (a) denaturation at 95˚C for 30s, and (b) annealing and extension at 60˚C for 30s.  The amplification curve was collected, and the relative transcript level of the target mRNA in each sample was calculated by normalization of Ct values to the reference mRNA (GAPDH).  Primer sequences used for RT-PCR were as shown in Table 1.\\n\\n\\n\\n\\n\\nGα protein pulldown assay'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Gα protein pulldown assay\\n\\nBL21 E.coli cells were transformed with constructs expressing GST-tagged GINIP protein or GST-tagged GRK2 N-terminal domain.  Cells were incubated in terrific broth (http://cshprotocols.cshlp.org/content/2015/9/pdb.rec085894.full) and protein expression induced by addition of 0.5µg/ml IPTG.  Cells were collected after 8 hours, lysed and GST- proteins collected on Glutathione-conjugated beads. Beads were washed 4 times, eluted with 50mM glutathione and proteins desalted on a gel filtration column. Aliquots of 20 µg aliquots were stored frozen and thawed shortly before use.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='For pulldown assays, HUVECs infected with lentivirus encoding GluGlu-tagged Gi mutant or lentivirus encoding both GluGlu-tagged Gq mutant and Ric8A were lysed in cell lysis buffer composed of 10mM Tris-HCl pH7.5, 150mM NaCl, 1% Triton X-100, 5mM DTT and additional 2µg/ml of GST-GINIP or GST-GRK2N, and the lysates were incubated for 10 minutes at 4˚C with gentle agitation with glutathione beads.  Beads were washed three times with cell lysis buffer, isolated proteins were solubilized in SDS sample laemmli buffer, and were analyzed by western blotting.\\n\\n\\n\\n\\n\\nMass spectrometry screening for Gα binding receptors'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Mass spectrometry screening for Gα binding receptors\\n\\nTo create a Gα protein variant that interacts more strongly with GPCR, a green fluorescent protein (GFP) tag was inserted for affinity purification, alongside a mutation of four alanine residues into helix α5 (ins4A), as this hinders GTP binding and maintains the association with GPCRs even in the presence of GTP.  Additionally, for the extraction of GPCRs while preserving their configuration and lipid environment, a non-detergent nanodisc approach utilizing styrene-maleimide anhydride (SMA) copolymers was employed according to protocols described previously.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='In the coimmunoprecipitation process, GFP-Trap® nanobody beads were utilized to capture the interactions between GFP-tagged Gα protein variants and activated designer receptors exclusively activated by designer drugs (DREADDs). The resulting complexes were subjected to subsequent proteomic analysis to determine which GPCRs interacted preferentially with the Gi1(Q306K) variant following induction of fluid shear stress. Identified receptors were further analyzed by western blotting as described in the supplementary material.\\n\\n\\n\\n\\n\\nMeasurement of Golgi orientation'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Measurement of Golgi orientation\\n\\nFollowing exposure to laminar shear stress, HUVECs were washed with PBS, fixed for 10 minutes with 3.7% PFA in PBS, permeabilized with 0.5% Triton X-100 in PBS for 10 minutes, blocked with 3% BSA in PBS for 30 minutes and then incubated with GM130 antibody (1:500, BD bioscience) at 4˚C for overnight. After washing, cells were incubated with Hoechst and Alexa647-conjugated secondary antibody at RT for 1 hour to visualize the nuclei and Golgi apparatus, respectively.  Images were captured with 20x objective on a Leica Sp8 confocal microscope with the Leica Application Suite (LAS) software.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Golgi polarization was calculated as a vector that connects the center of mass of the nucleus to the center of mass of the Golgi. The angle between this vector and the direction of flow was then determined, with 0° defined as against the flow. Oriana V4 software was used to calculate the angle mean and circular SD and to generate a rose plot for data visualization.\\n\\n\\n\\nIn vitro permeability assay using the FITC-streptavidin/biotinylated-fibronectin'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Slides/dishes were\\xa0coated with biotinylated-Fibronectin at\\xa00.25\\xa0mg/mL\\xa0overnight\\xa0at 4°C. HUVECs were seeded of 2×106 cells per well on glass-bottom 6-well plates. Cells\\xa0were\\xa0subjected\\xa0to\\xa0laminar shear stress by orbital shaking at 150 rpm for 24 hours.\\xa0 Immediately\\xa0after\\xa0cessation\\xa0of\\xa0flow, FITC-Streptavidin (1:1000, Invitrogen) was added to the cells for 1 minute before\\xa0 the\\xa0 fixation\\xa0 by\\xa0 the\\xa0 addition\\xa0 of\\xa0 3.7%\\xa0 (v/v)\\xa0 paraformaldehyde. Following\\xa0\\xa0 subsequent\\xa0\\xa0 washing\\xa0\\xa0 and\\xa0\\xa0 blocking, Alexa Fluor 488 Phalloidin (1:200; Invitrogen) was\\xa0 added\\xa0 to\\xa0 stain\\xa0 F-actin\\xa0 filaments. Images were captured from 5 random fields and FITC-streptavidin staining quantified using ImageJ.\\xa0\\n\\n\\n\\nIn vitro endothelial 3D invasion assay under shear stress'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='In vitro endothelial 3D invasion assay under shear stress\\n\\nIn order to equalize the VEGF concentration in the collagen gel and endothelial culture medium, VEGF-189 (R&D systems, 8147-VE), a matrix-binding isoform of VEGF, was applied to the bottom of each well at a concentration of 50 ng/ml prior to adding collagen matrices. Rat tail collagen type I (Thermofisher scientific, A1048301) was employed to prepare collagen matrices at the concentration of 1.5 mg/ml with additional NaOH to facilitate the collagen matrix assembly by neutral pH. After thorough mixing, 1 ml of collagen I solution was placed in each well of a 12-well plate (Corning #3513) and polymerized at 37°C and 5% CO2 for 2 hours to overnight.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='To prepare endothelial cells, confluent HUVEC culture plates were washed with PBS, trypsinized, and counted.  After pelleting, cell pellets were resuspended in EGM-2 medium (Lonza, #CC-3162) free of VEGF and heparin to establish a positive gradient of VEGF from cells to the bottom of the collagen gel, and 300,000 cells per well were placed on the collagen gel and incubated for 24 hours. Following incubation, plates were placed on an orbital shaker and ECs were exposed to orbital shaking at 150 rpm for 48 hours as mentioned above in shear stress section. Cells were then fixed in 4% PFA and permeabilized in PBS containing 0.5% Triton X-100. Nonspecific binding was blocked by incubation with PBS containing 3% BSA. Thereafter, cells were subjected to'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='with PBS containing 3% BSA. Thereafter, cells were subjected to Alexa488-conjugated phalloidin (1:400 dilution; Thermofisher scientific: A12379) and Hoechst (1:1000 dilution; Invitrogen: H1399) for 2 hours to stain actin cytoskeleton and cell nuclei, respectively. All images were captured with the Nikon Sp8 confocal microscope and segregated into peripheral region (with uniaxial pulsatile shear) vs. static controls.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Zebrafish husbandry and handling\\n\\nZebrafish were housed and maintained at 28.5ºC in accordance with standard methods approved by the Yale University Institutional Animal Care and Use Committee (#2017-11473)(Lawrence, 2016).\\n\\n\\n\\nGeneration of LPHN/adgrl knockdown zebrafish using CRISPR/Cas9 ribonucleoproteins'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Four zebrafish latrophilin paralogs were identified—adgrl1a, adgrl2a, adgrl2b.1, and adgrl3.1. Fused sgRNAs were generated targeting each paralog (see Table 2) by annealing locus-specific oligonucleotides to a common 5’ universal oligonucleotide and performing in vitro RNA transcription (AmpliScribe T7-Flash Kit, Lucigen, ASF3507)(Gagnon et al, 2014). An injection mix of 50 ng/µL each sgRNA, an equivalent concentration of Cas9 protein (TrueCut Cas9, Invitrogen, A36497), and either 1 µg/µL silent heart/tnnt2a(Sehnert et al., 2002) or standard control morpholino (GeneTools) was prepared. 1 nL of the mixture was injected into Tg(kdrl:ras-mCherry, fli1a:nls-GFP) zebrafish at the one cell stage (Parker Hannifin, Picospritzer III). CRISPR reagent efficacy was'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Hannifin, Picospritzer III). CRISPR reagent efficacy was confirmed by T7 endonuclease assay (see Figure S5).  Guides against adgrl2a and 2b.1 were combined to target all LPHN2 isoforms.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Immunostaining and imaging of ECs in the knockdown zebrafish'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='CRISPR/Cas9 RNP-injected embryos were either fixed at 48 hpf in 4% paraformaldehyde (Santa Cruz Biotechnology, SC-253236) and immunostained for GFP (1:300 chicken anti-GFP, abcam, ab13970), mCherry (1:300 rabbit anti-mCherry, abcam, ab167453), and ZO-1 (1:200 mouse anti-ZO-1, Invitrogen, 61-7300) with species-appropriate secondary antibodies (1:400 Invitrogen anti-chicken/-rabbit/-mouse Alexa 488/546/647, A-11039/A- 10040, A-31571) or imaged live. For fixed imaging, the larvae were processed as previously described(Sugden et al., 2017). For live imaging, larvae were anesthetized by immersion in 600 µM tricaine methylsulfonate (Western Chemical, TRS5), and embedded in 1.5% low-melt agarose (BioRad, 1613106). For staining in detail, larvae were fixed'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='(BioRad, 1613106). For staining in detail, larvae were fixed overnight in 4% PFA at +4ºC and then transitioned into 100% methanol via a series of dilutions. Once in methanol, the larvae were stored at -20ºC overnight, then rehydrated the next day via the same dilution series. Larvae were permeabilized using incubation in acetone for 15 minutes at -20ºC and then by incubation in 20 µg/mL proteinase K for 20 minutes at 37ºC. Larvae were refined in 4% PFA for 15 minutes at room temperature and then incubated in blocking buffer (0.5% Triton X-100, 10% goat serum, 1% BSA, 0.01% sodium azide, and 1% DMSO in PBS) for 2 hours at room temperature. Larvae were incubated in primary antibody (1:400 chicken anti-GFP, abcam ab13970) overnight in blocking buffer at +4ºC.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='anti-GFP, abcam ab13970) overnight in blocking buffer at +4ºC. Larvae were thoroughly washed in PBS-Tx and incubated overnight at +4ºC in secondary antibody (1:400 goat anti-chicken Alexa488 A-11039, ThermoFisher) in blocking buffer. Following final washes in PBS-Tx, larvae were mounted and imaged. Intersegmental vessel and dorsal aorta diameter were then calculated using ImageJ. All imaging was performed using a Leica SP8 confocal microscope.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Immunostaining and imaging of mouse aorta'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Aortae were perfusion-fixed in situ with 4% paraformaldehyde (overnight) prior to staining with primary beta-catenin antibody (Cell signaling: 9562) and fluorophore-conjugated secondary antibody (Invitrogen, donkey anti-Rabbit IgG (H+L) Highly Cross-Adsorbed Secondary Antibody, Alexa Fluor 647, A31573) in in Claudio buffer (1% FBS, 3% BSA, 0.5% Triton X-100, 0.01% Sodium deoxycholate, 0.02% sodium azide in PBS pH7.4)(Franco et al, 2013).  Cells were stained with Hoechst (Invitrogen: H1399) to label nuclei. Images of stained vessels were acquired using a Leica SP8 confocal microscope with the Leica Application Suite (LAS) software. Endothelial cell alignment in aorta was quantified by measuring the eccentricity of the immunostaining of beta-catenin of the'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='the eccentricity of the immunostaining of beta-catenin of the endothelial cells (>150 cells per field of view).'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Extravasation of Evans blue dye for aortic vascular permeability'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='For aortic vascular permeability measurements, Evans blue dye (20 mg/ml in saline, 2.5 µl/g mouse body weight) (Sigma E2129) was injected retro-orbitally to LPHN2 endothelial cell-specific knockout and their control littermates at 8 weeks old.  Thirty minutes post injection, animals were euthanized by an overdose of isoflurane and tissue samples including aorta were harvested and washed with PBS to remove extra fatty tissues.  Dissected aorta was prepared en face and imaged with an Olympus stereo zoom microscope. To quantify blue color associated with extravasation of Evans blue dye, the original RGB images were converted into HSV color model, and the proportion of the blue regions was calculated as the ratio of pixels with hue angles of 120°-240° to the'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='as the ratio of pixels with hue angles of 120°-240° to the entire aorta.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Immunostaining and imaging of mouse muscle capillaries\\n\\nRectus femoris muscle from murine thigh were perfusion-fixed with 4% paraformaldehyde (overnight) and the middle part of the muscle were cryo-sectioned following embedding into OCT compound and stained with primary antibodies for SMA (fluorophore-conjugated) and for CD31 followed by fluorophore-conjugated secondary antibody (Invitrogen, donkey anti-Rabbit IgG (H+L) Highly Cross-Adsorbed Secondary Antibody, Alexa Fluor 647, A31573) in PBS containing 5% BSA.  Hoechst (Invitrogen: H1399) was used to nuclear identification. Images of stained vessels were acquired using a Leica SP8 confocal microscope with the Leica Application Suite (LAS) software. \\n\\n\\n\\nHindlimb ischemia model'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Mice were anesthetized and operated with a mixture of 1.5-2% isoflurane. The front hair of the right thigh was removed with Nair lotion and the local surgical area was disinfected. A 5 mm longitudinal skin incision was made in the right thigh. The distal femoral artery, proximal to the popliteal artery and saphenous artery, was explored and dissected. 6-0 silk sutures (Syneture) were double used for the ligation before excision of the arterial vessel bed between the distal end of the superficial epigastric artery and the trifurcation of the femoral artery into the descending genicular, popliteal, and saphenous branches. The ligated artery was cut between two sutures. The venous structures and accompanying peripheral nerves were kept intact. The overlying'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='accompanying peripheral nerves were kept intact. The overlying skin was closed with a 6-0 prolene (Ethicon) and disinfected. At the end of the procedure, Buprenorphine was administered subcutaneously at 0.05 mg/kg, once pre-emptively, then every 12 h for 72 h. The survival rate for this surgery was 100%.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Laser Doppler perfusion imaging'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Hindlimb perfusion was assessed noninvasively in the plantar foot before, 1, 3, 7, 14, and 21 days after hindlimb ischemia by scanning laser-Doppler [model LDI2-IR modified for high resolution; Moor Instruments, Wilmington, DE]. The hindquarters were placed on the top of a heater pad during scanning to minimize variation in body temperature. Doppler perfusion of the plantar foot was assessed within anatomically defined regions of interest (ROIs; Moor Instruments). The ROI for the plantar foot consisted of the hind paw margins. Procedures for minimal preparation, scanning, and ROI selections have been described in detail previously(Zhuang et al, 2011). Low or no perfusion is displayed as dark blue, whereas the highest degree of perfusion is displayed as red.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='whereas the highest degree of perfusion is displayed as red. These images were quantitatively converted into histograms that represented the amount of blood flow on the x-axis and the number of pixels on the y-axis in the traced area. The average blood flow in each histogram was calculated and the LDI index was determined as the ratio of ischemic to non-ischemic hindlimb blood perfusion.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Volumetric micro-CT angiogram for quantitative arteriogenesis'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='After the mice were euthanized, the vasculature was flushed with 0.9% normal saline containing heparin (1000 IU/L) for 3 minutes, followed by 4% paraformaldehyde for 5 minutes at 100 mm Hg pressure. Fresh home-made 20% bismuth nanoparticles mixed in 5% gelatin were used as a micro-CT contrast agent and injected over 2 minutes with a syringe pump, as described before(Tirziu et al., 2005). The mice were then immediately chilled on ice for more than 30 min and immersion-fixed in 2% paraformaldehyde overnight. The peripheral vasculature in the ischemic hindlimb and the contralateral normal hindlimb was imaged with a high-resolution micro-CT imaging system (GE eXplore Locus SP), set to a 0.007-mm effective detector pixel size. Microview software (GE Healthcare)'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='detector pixel size. Microview software (GE Healthcare) was used for initial reconstruction. Advanced workstation with various software was used for 3D reconstruction and segmentation of the whole hindlimb at the 21 µm spatial resolution. Using the knee as the breakpoint, we re-sliced the 3D vasculature into 200 slices both in the thigh and in the calf regions. Quantification was performed by use of a NIH Image. The data were expressed as vascular segment numbers, representing the total number of vessels, of specified diameter, counted in 200\\xa0z-sections for the thigh region and another 200\\xa0z-sections for the calf region.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Mouse treadmill fatigue test'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Mice were first acclimatized by staying on the motionless treadmill machine for 30 min on day 1 and then trained on the treadmill moving at 12 m/min for 30 min on day 2. Fatigue tests were then conducted on days 3 and 4. \\xa0Mice were placed on the treadmill and the shock rod was activated. \\xa0Treadmill speeds were increased as follows: 0 m/min, for 12 min to measure basal metabolism, 12 m/min, for1 min, then increasing at a rate of +1 m/min each 1 min until exhaustion. \\xa0Exhaustion (endpoint of treadmill cessation) was defined as the point at which mice continuously contacted with the shock rod for 5 continuous seconds or when mice stopped running 3 times within 5 seconds.  Before each testing session, Oxymax software (Columbus Instruments, Columbus, OH, USA)'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Oxymax software (Columbus Instruments, Columbus, OH, USA) and open circuit indirect calorimetry treadmills (Metabolic Modular Treadmill, Columbus Instruments, Columbus, OH) were calibrated and checked for hardware malfunctions according to manufacturer instructions.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Data availability\\n\\nThe datasets produced in this study are available in the following databases:\\n\\nAll data: BioStudies S-BIAD928: https://www.ebi.ac.uk/biostudies/studies?query=S-BIAD928\\n\\n\\n\\nAcknowledgements'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='This work was supported by NIH grants RO1 HL75092 and R01 HL151469 to MAS, NHLBI 5R01HL130246-05 to SN, and the Uehara memorial foundation postdoctoral fellowship and MEXT | Japan Society for the Promotion of Science (JSPS) Overseas Research Fellowships (2014, receipt number: 805) to KT.  We thank Hamit Harun Dag (Istanbul University) for help of synthesizing SMA polymer; Shahid Mansuri (Yale University), Jeremy Herrera (University of Manchester) and David Knight (University of Manchester) for expert advice for mass spectrometry experiments; Didier Trono (École polytechnique fédérale de Lausanne) for lentiviral packaging plasmids pMD2.G (Addgene plasmid # 12259 ; http://n2t.net/addgene:12259 ; RRID:Addgene_12259) and psPAX2 (Addgene plasmid # 12260 ;'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='; RRID:Addgene_12259) and psPAX2 (Addgene plasmid # 12260 ; http://n2t.net/addgene:12260 ; RRID:Addgene_12260);  Bryan Roth (University of North Carolina) for pcDNA5/FRT-HA-hM3D(Gq) (Addgene plasmid # 45547 ; http://n2t.net/addgene:45547 ; RRID:Addgene_45547) and pcDNA5/FRT-HA-hM4D(Gi)(Addgene plasmid # 45548 ; http://n2t.net/addgene:45548 ; RRID:Addgene_45548); Aziz Moqrich (Institut de Biologie du Dévelopement de Marseille) for GST-GINIP plasmid.  We are grateful to Peter Newman (Medical College of Wisconsin) for the kind gift of PECAM-1 antibody and helpful discussions.  Schematic models are inspired from BioRender templates (2022).'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Disclosure & competing interests statement\\n\\nThe authors declare they have no conflict of interest.\\n\\n\\n\\n\\n\\n\\nReferences\\n\\n\\n\\nAlbarran-Juarez J, Iring A, Wang S, Joseph S, Grimm M, Strilic B, Wettschureck N, Althoff TF, Offermanns S (2018) Piezo1 and G(q)/G(11) promote endothelial inflammation depending on flow pattern and integrin activation. J Exp Med 215: 2655-2672\\n\\nArshad M, Ghim M, Mohamied Y, Sherwin SJ, Weinberg PD (2021) Endothelial cells do not align with the mean wall shear stress vector. J R Soc Interface 18: 20200772\\n\\nBae H, Cabrera-Vera TM, Depree KM, Graber SG, Hamm HE (1999) Two amino acids within the alpha4 helix of Galphai1 mediate coupling with 5-hydroxytryptamine1B receptors. J Biol Chem 274: 14963-14971'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Baeyens N, Bandyopadhyay C, Coon BG, Yun S, Schwartz MA (2016) Endothelial fluid shear stress sensing in vascular health and disease. J Clin Invest 126: 821-828\\n\\nBaeyens N, Mulligan-Kehoe MJ, Corti F, Simon DD, Ross TD, Rhodes JM, Wang TZ, Mejean CO, Simons M, Humphrey J et al (2014) Syndecan 4 is required for endothelial alignment in flow and atheroprotective signaling. Proc Natl Acad Sci U S A 111: 17308-17313\\n\\nButler PJ, Norwich G, Weinbaum S, Chien S (2001) Shear stress induces a time- and position-dependent increase in endothelial cell membrane fluidity. Am J Physiol Cell Physiol 280: C962-969'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Camillo C, Facchinello N, Villari G, Mana G, Gioelli N, Sandri C, Astone M, Tortarolo D, Clapero F, Gays D et al (2021) LPHN2 inhibits vascular permeability by differential control of endothelial cell adhesion. J Cell Biol 220\\n\\nCampinho P, Vilfan A, Vermot J (2020) Blood Flow Forces in Shaping the Vascular System: A Focus on Endothelial Cell Behavior. Front Physiol 11: 552\\n\\nChachisvilis M, Zhang YL, Frangos JA (2006) G protein-coupled receptors sense fluid shear stress in endothelial cells. Proc Natl Acad Sci U S A 103: 15463-15468'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content=\"Collins C, Guilluy C, Welch C, O'Brien ET, Hahn K, Superfine R, Burridge K, Tzima E (2012) Localized tensional forces on PECAM-1 elicit a global mechanotransduction response via the integrin-RhoA pathway. Curr Biol 22: 2087-2094\\n\\nConway D, Schwartz MA (2012) Lessons from the endothelial junctional mechanosensory complex. F1000 Biol Rep 4: 1\\n\\nConway DE, Breckenridge MT, Hinde E, Gratton E, Chen CS, Schwartz MA (2013) Fluid shear stress on endothelial cells modulates mechanical tension across VE-cadherin and PECAM-1. Curr Biol 23: 1024-1030\\n\\nCornella N, Sancho J, Sitges-Serra A (2017) Short and Long-Term Outcomes After Surgical Procedures Lasting for More Than Six Hours. Sci Rep 7: 9221\"), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Dela Paz NG, Frangos JA (2019) Rapid flow-induced activation of Galphaq/11 is independent of Piezo1 activation. Am J Physiol Cell Physiol 316: C741-C752\\n\\nDela Paz NG, Melchior B, Frangos JA (2017) Shear stress induces Galpha(q/11) activation independently of G protein-coupled receptor activation in endothelial cells. Am J Physiol Cell Physiol 312: C428-C437\\n\\ndela Paz NG, Melchior B, Shayo FY, Frangos JA (2014) Heparan sulfates mediate the interaction between platelet endothelial cell adhesion molecule-1 (PECAM-1) and the Galphaq/11 subunits of heterotrimeric G proteins. J Biol Chem 289: 7413-7424'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Erdogmus S, Storch U, Danner L, Becker J, Winter M, Ziegler N, Wirth A, Offermanns S, Hoffmann C, Gudermann T et al (2019) Helix 8 is the essential structural motif of mechanosensitive GPCRs. Nat Commun 10: 5784\\n\\nFancher IS, Ahn SJ, Adamos C, Osborn C, Oh MJ, Fang Y, Reardon CA, Getz GS, Phillips SA, Levitan I (2018) Hypercholesterolemia-Induced Loss of Flow-Induced Vasodilation and Lesion Formation in Apolipoprotein E-Deficient Mice Critically Depend on Inwardly Rectifying K(+) Channels. J Am Heart Assoc 7\\n\\nFlock T, Hauser AS, Lund N, Gloriam DE, Balaji S, Babu MM (2017) Selectivity determinants of GPCR-G-protein binding. Nature 545: 317-322'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Franco CA, Blanc J, Parlakian A, Blanco R, Aspalter IM, Kazakova N, Diguet N, Mylonas E, Gao-Li J, Vaahtokari A et al (2013) SRF selectively controls tip cell invasive behavior in angiogenesis. Development 140: 2321-2333\\n\\nFrangos JA, McIntire LV, Eskin SG (1988) Shear stress induced stimulation of mammalian cell metabolism. Biotechnol Bioeng 32: 1053-1060\\n\\nGagnon JA, Valen E, Thyme SB, Huang P, Akhmetova L, Pauli A, Montague TG, Zimmerman S, Richter C, Schier AF (2014) Efficient mutagenesis by Cas9 protein-mediated oligonucleotide insertion and large-scale assessment of single-guide RNAs. PLoS One 9: e98186'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Gaillard S, Lo Re L, Mantilleri A, Hepp R, Urien L, Malapert P, Alonso S, Deage M, Kambrun C, Landry M et al (2014) GINIP, a Galphai-interacting protein, functions as a key modulator of peripheral GABAB receptor-mediated analgesia. Neuron 84: 123-136\\n\\nGalie PA, Nguyen DH, Choi CK, Cohen DM, Janmey PA, Chen CS (2014) Fluid shear stress threshold regulates angiogenic sprouting. Proc Natl Acad Sci U S A 111: 7968-7973\\n\\nGivens C, Tzima E (2016) Endothelial Mechanosignaling: Does One Sensor Fit All? Antioxid Redox Signal 25: 373-388'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Grzelinski M, Pinkenburg O, Buch T, Gold M, Stohr S, Kalwa H, Gudermann T, Aigner A (2010) Critical role of G(alpha)12 and G(alpha)13 for human small cell lung cancer cell proliferation in vitro and tumor growth in vivo. Clin Cancer Res 16: 1402-1415\\n\\nGudi S, Nolan JP, Frangos JA (1998) Modulation of GTPase activity of G proteins by fluid shear stress and phospholipid composition. Proc Natl Acad Sci U S A 95: 2515-2519\\n\\nGudi SR, Clark CB, Frangos JA (1996) Fluid flow rapidly activates G proteins in human endothelial cells. Involvement of G proteins in mechanochemical signal transduction. Circ Res 79: 834-839'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content=\"Haidekker MA, L'Heureux N, Frangos JA (2000) Fluid shear stress increases membrane fluidity in endothelial cells: a study with DCVJ fluorescence. Am J Physiol Heart Circ Physiol 278: H1401-1406\\n\\nHumphrey JD, Schwartz MA (2021) Vascular Mechanobiology: Homeostasis, Adaptation, and Disease. Annu Rev Biomed Eng 23: 1-27\\n\\nJung B, Obinata H, Galvani S, Mendelson K, Ding BS, Skoura A, Kinzel B, Brinkmann V, Rafii S, Evans T et al (2012) Flow-regulated endothelial S1P receptor-1 signaling sustains vascular development. Dev Cell 23: 600-610\\n\\nKaya AI, Lokits AD, Gilbert JA, Iverson TM, Meiler J, Hamm HE (2016) A Conserved Hydrophobic Core in Galphai1 Regulates G Protein Activation and Release from Activated Receptor. J Biol Chem 291: 19674-19686\"), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Kroon J, Heemskerk N, Kalsbeek MJT, de Waard V, van Rijssel J, van Buul JD (2017) Flow-induced endothelial cell alignment requires the RhoGEF Trio as a scaffold protein to polarize active Rac1 distribution. Mol Biol Cell 28: 1745-1753\\n\\nKrumins AM, Gilman AG (2006) Targeted knockdown of G protein subunits selectively prevents receptor-mediated modulation of effectors and reveals complex changes in non-targeted signaling proteins. J Biol Chem 281: 10250-10262\\n\\nLawrence C (2016) New frontiers for zebrafish management. Methods Cell Biol 135: 483-508\\n\\nLee MJ, Evans M, Hla T (1996) The inducible G protein-coupled receptor edg-1 signals via the G(i)/mitogen-activated protein kinase pathway. J Biol Chem 271: 11272-11279'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content=\"Lee SC, Knowles TJ, Postis VL, Jamshad M, Parslow RA, Lin YP, Goldman A, Sridhar P, Overduin M, Muench SP et al (2016) A method for detergent-free isolation of membrane proteins in their local lipid environment. Nat Protoc 11: 1149-1162\\n\\nLiu SL, Sheng R, Jung JH, Wang L, Stec E, O'Connor MJ, Song S, Bikkavilli RK, Winn RA, Lee D et al (2017) Orthogonal lipid sensors identify transbilayer asymmetry of plasma membrane cholesterol. Nat Chem Biol 13: 268-274\\n\\nLucitti JL, Jones EA, Huang C, Chen J, Fraser SE, Dickinson ME (2007) Vascular remodeling of the mouse yolk sac requires hemodynamic force. Development 134: 3317-3326\"), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Maleszewska M, Vanchin B, Harmsen MC, Krenning G (2016) The decrease in histone methyltransferase EZH2 in response to fluid shear stress alters endothelial gene expression and promotes quiescence. Angiogenesis 19: 9-24\\n\\nMedina R, Grishina G, Meloni EG, Muth TR, Berlot CH (1996) Localization of the effector-specifying regions of Gi2alpha and Gqalpha. J Biol Chem 271: 24720-24727\\n\\nNazarko O, Kibrom A, Winkler J, Leon K, Stoveken H, Salzman G, Merdas K, Lu Y, Narkhede P, Tall G et al (2018) A Comprehensive Mutagenesis Screen of the Adhesion GPCR Latrophilin-1/ADGRL1. iScience 3: 264-278'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Ngai J, Methi T, Andressen KW, Levy FO, Torgersen KM, Vang T, Wettschureck N, Tasken K (2008) The heterotrimeric G-protein alpha-subunit Galphaq regulates TCR-mediated immune responses through an Lck-dependent pathway. Eur J Immunol 38: 3208-3218\\n\\nPapasergi MM, Patel BR, Tall GG (2015) The G protein alpha chaperone Ric-8 as a potential therapeutic target. Mol Pharmacol 87: 52-63\\n\\nPrivratsky JR, Newman DK, Newman PJ (2010) PECAM-1: conflicts of interest in inflammation. Life Sci 87: 69-82\\n\\nSando R, Jiang X, Sudhof TC (2019) Latrophilin GPCRs direct synapse specificity by coincident binding of FLRTs and teneurins. Science 363'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Sehnert AJ, Huq A, Weinstein BM, Walker C, Fishman M, Stainier DY (2002) Cardiac troponin T is essential in sarcomere assembly and cardiac contractility. Nat Genet 31: 106-110\\n\\nStrohbach A, Pennewitz M, Glaubitz M, Palankar R, Gross S, Lorenz F, Materzok I, Rong A, Busch MC, Felix SB et al (2018) The apelin receptor influences biomechanical and morphological properties of endothelial cells. J Cell Physiol 233: 6250-6261\\n\\nSugden WW, Meissner R, Aegerter-Wilmsen T, Tsaryk R, Leonard EV, Bussmann J, Hamm MJ, Herzog W, Jin Y, Jakobsson L et al (2017) Endoglin controls blood vessel diameter through endothelial cell shape changes in response to haemodynamic cues. Nat Cell Biol 19: 653-665'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Tesmer VM, Kawano T, Shankaranarayanan A, Kozasa T, Tesmer JJ (2005) Snapshot of activated G proteins at the membrane: the Galphaq-GRK2-Gbetagamma complex. Science 310: 1686-1690\\n\\nTirziu D, Moodie KL, Zhuang ZW, Singer K, Helisch A, Dunn JF, Li W, Singh J, Simons M (2005) Delayed arteriogenesis in hypercholesterolemic mice. Circulation 112: 2501-2509\\n\\nTzima E, Irani-Tehrani M, Kiosses WB, Dejana E, Schultz DA, Engelhardt B, Cao G, DeLisser H, Schwartz MA (2005) A mechanosensory complex that mediates the endothelial cell response to fluid shear stress. Nature 437: 426-431\\n\\nVizurraga A, Adhikari R, Yeung J, Yu M, Tall GG (2020) Mechanisms of adhesion G protein-coupled receptor activation. J Biol Chem 295: 14065-14083'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Vogel J, Niederer D, Jung G, Troidl K (2020) Exercise-Induced Vascular Adaptations under Artificially Versus Pathologically Reduced Blood Flow: A Focus Review with Special Emphasis on Arteriogenesis. Cells 9\\n\\nWang S, Iring A, Strilic B, Albarran Juarez J, Kaur H, Troidl K, Tonack S, Burbiel JC, Muller CE, Fleming I et al (2015) P2Y(2) and Gq/G(1)(1) control blood pressure by mediating endothelial mechanotransduction. J Clin Invest 125: 3077-3086\\n\\nWilson PT, Bourne HR (1995) Fatty acylation of alpha z. Effects of palmitoylation and myristoylation on alpha z signaling. J Biol Chem 270: 9667-9675'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Xu J, Mathur J, Vessieres E, Hammack S, Nonomura K, Favre J, Grimaud L, Petrus M, Francisco A, Li J et al (2018) GPR68 Senses Flow and Is Essential for Vascular Physiology. Cell 173: 762-775 e716\\n\\nYamamoto K, Ando J (2013) Endothelial cell and model membranes respond to shear stress by rapidly decreasing the order of their lipid phases. J Cell Sci 126: 1227-1234\\n\\nYamamoto K, Ando J (2015) Vascular endothelial cell membranes differentiate between stretch and shear stress through transitions in their lipid phases. Am J Physiol Heart Circ Physiol 309: H1178-1185'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Yamamoto K, Nogimori Y, Imamura H, Ando J (2020) Shear stress activates mitochondrial oxidative phosphorylation by reducing plasma membrane cholesterol in vascular endothelial cells. Proc Natl Acad Sci U S A 117: 33660-33667\\n\\nYeh JC, Otte LA, Frangos JA (2008) Regulation of G protein-coupled receptor activities by the platelet-endothelial cell adhesion molecule, PECAM-1. Biochemistry 47: 9029-9039\\n\\nZhou J, Li YS, Chien S (2014) Shear stress-initiated signaling and its regulation of endothelial function. Arterioscler Thromb Vasc Biol 34: 2191-2198\\n\\nZhuang ZW, Shi J, Rhodes JM, Tsapakos MJ, Simons M (2011) Challenging the surgical rodent hindlimb ischemia model with the miniinterventional technique. J Vasc Interv Radiol 22: 1437-1446'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Figure 1. Gα proteins specific for endothelial flow responses'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='A, Flow-induced endothelial alignment after Gα knockdown.  HUVECs were subjected to fluid shear stress (FSS) at a rate of 12 dynes/cm2 for 16 hours and nuclear orientation quantified as histograms showing the percentage of cells within each 10° of the flow directions from 0° to 90° (see Methods) ****: p<0.0001; one-way ANOVA with Tukey’s multiple comparisons test. B, Src family kinase activation, quantified in C. n=5 for control, Gi knockdown, Gq11 knockdown and n=3 for simultaneous knockdown of Gi and Gq11.  Values are means ± SEM. ****: p<0.0001, ***: p<0.001; one-way ANOVA with Tukey multiple comparison test.  D, Rescue of Gq/11 and Gi knockdown by re-expression of siRNA-resistant versions of the indicated proteins. ****: p<0.0001; one-way ANOVA with'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='of the indicated proteins. ****: p<0.0001; one-way ANOVA with Tukey’s multiple comparison test. E, Amino acid sequences of Gαi1, Gαi2, and Gαi3 at the mutation sites of Gαi1 gain-of-function mutant.  F, Rescue of Gq/11 and Gi knockdown with indicated Gα proteins. Each point corresponds to one measurement averaged from >500 cells. N=4. ****: p<0.0001; one-way ANOVA with Tukey’s multiple comparisons test.  G, GINIP pulldown assay for activation of Gαi2 by FSS.  N=3. Results quantified in H.  **: p=0.0185, Student’s t-test.  I, GRK2N pulldown assay for activation of Gq.  N=4, quantified in J. *: p<0.05, Student’s t-test. K, GINIP pulldown assay for FSS-induced activation of wild type and Q306K Gαi1.  N = 4, quantified in L. *: p=0.0304, ***: p=0.0017;'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Gαi1.  N = 4, quantified in L. *: p=0.0304, ***: p=0.0017; Student’s t-test. M, Gi2 activation after PECAM-1 knockdown. HUVECs expressing GluGlu tagged Gi2 were transfected with scrambled siRNA or PECAM-1 siRNA, exposed to FSS, and Gi2 activation assayed as described above, quantified in N. Values are means ± SEM, normalized to input Gα protein levels. ***: p<0.001; Student’s t-test, N=4.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Figure 2. Identifiation of the upstream flow-responsive GPCR\\n\\nA, Strategy for identification of GPCRs that bind the gain-of-function Gi1Q306K mutant but not the wild type Gi1.  B, Co-immunoprecipitation of Gi1Q306K(ins4A) with endogenous LPHN2 with and without FSS for 2 min. N=3, quantified in C. *: p=0.0374, Student’s t-test. D, Co-immunoprecipitation of LPHN2-GFP with the indicated Gα proteins containing internal GluGlu epitope tags.  E, Gi2 pulldown assay after LPHN2 knockdown.  N= 4. Results quantified in F.  ***: p<0.001, Student’s t-test.\\n\\n\\n\\nFigure 3. Latrophilins regulate endothelial flow responses'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='A, HUVECs with or without latrophilin-2 knockdown were subjected to FSS for 16 h, then fixed and stained with Hoechst (nuclei), phalloidin (F-actin) and an antibody against VE-Cadherin. Scale bar: 100µm. B, Alignment of HUVECs (each bar = 10° increments) after knockdown of latrophilin isoforms was quantified as in Fig. 1 from >2000 cells/experiment, N=3.  ****: p<0.0001; one-way ANOVA with Tukey’s multiple comparisons test. C, Localization of LPHN2-mClover3. Scale bar: 100µm. N=6. D, Gi1(Q306K) pulldown from ECs ± FSS for 5 min or for 24 hours, probed for PECAM-1. N=3, quantified in E. F, Gi1(Q306K) pulldown from ECs ± FSS for 5 min or for 24 hours probed for VE-cadherin. N=3, quantified in G. H, Activation of Src family kinases and Akt by FSS in HUVECs'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='G. H, Activation of Src family kinases and Akt by FSS in HUVECs depleted for latrophilin isoforms. N=3-4, quantified in I & J. *: p=0.0184, **: p<0.001; one-way ANOVA with Tukey’s multiple comparisons test. K, Alignment of LPHN2-depleted HUVECs rescued by re-expression of the indicated latrophilin isoforms. ****: p<0.0001; one-way ANOVA with Tukey’s multiple comparisons test. Quantification of >2000 cells for each condition, N=3. L, LPHN2 knockdown HUVECs were rescued by re-expression of the indicated LPHN2 mutants. ****: p<0.0001, one-way ANOVA with Tukey’s multiple comparisons test. Quantification of >500 cells for each condition.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Figure 4, Latrophilins in flow-mediated endothelial cell morphology in vivo'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='A. Representative image of HUVECs transfected with scrambled siRNA or LPHN2 siRNA.  Outlines of ECs from VE-cadherin staining are shown to next to each image. Scale Bar: 100µm. B, Quantification of junctional linearization index from A. n=30 for each condition.  **: p=0.0019, Statistics is done using two-way ANOVA with Sidak’s multiple comparison test. C, Dorsal aortas from 48hpf Tg(kdrl:ras-mCherry) WT or LPHN mutant zebrafish embryos stained for ZO-1. EC were outlined as in Methods. LPHN2 mutant indicates adgrl2a+adgrl2b.1 sgRNAs.  D&E, Quantification of cell size and eccentricity in embryos with latrophilin-2 CRISPR sgRNAs ± silent heart morpholinos (MO). N=6 per condition.  ****: p<0.0001, *: p=0.0267, **: p=0.0015, ***: p=0.0008; one-way ANOVA with'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='*: p=0.0267, **: p=0.0015, ***: p=0.0008; one-way ANOVA with Tukey’s multiple comparisons test.  F, ECs in the thoracic aorta in 18 weeks old mice at 10 weeks after tamoxifen injections, stained for β-catenin (left) and outlined (right).  Data are representative of 6 mice for each condition. Scale Bar: 100µm. G&H, quantification of cell eccentricity and junction linearity in mouse aorta.  N=4 per condition. **: p=0.0013; ****: p<0.0001; Student’s t-test. I, Aortas from mice 30 min after Evans blue dye injection, thresholded images on the right. Fractional blue area quantified in J. n=3 for each condition.  *: p=0.0419; Student’s t-test. K, in vitro endothelial permeability assay using FITC-streptavidin and biotin-conjugated fibronectin. Scale Bar: 100µm. L,'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='and biotin-conjugated fibronectin. Scale Bar: 100µm. L, Permeability was quantified by measuring FITC-streptavidin area per image field from K. n=5, *: p=0.0457, one-way ANOVA.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Figure 5. Cholesterol depletion activates Latrophilin-dependent signaling'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='A, ECs were treated with FSS or 5 mM methyl-β-cyclodextrin (mβCD) for 1 min then incubated with D4H-mClover3, rinsed and bound mClover3 quantified as described in Methods. N=27 for each condition.  ****: p<0.0001; One-way ANOVA with Tukey multiple comparisons test. B, Phalloidin staining following short acute treatment of methyl-β-cyclodextrin (mβCD).  Scale bar: 100µm. C, Co-immunoprecipitation of Gi1Q306K(ins4A) with endogenous Latrophilin-2 (LPHN2) after 5mM MβCD for 1min. D, Activation of Src family kinases and VEGFR2 after 5mM MβCD for 1min in HUVECs ± LPHN2 knockdown. Results quantified in E and F, respectively. Values are means ± SEM. N=5. *: p=0.0305, ***: p=0.0002; one-way ANOVA with Tukey multiple comparison test.  G, Model for junctional'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='with Tukey multiple comparison test.  G, Model for junctional endothelial shear stress mechanotransduction.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='Figure 6. Latrophilin-2 in flow-induced vascular remodeling'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='A, Representative images of ISVs from 48 hpf embryos with F(0)/mosaic depletions of latrophilin genes or controls.  B, Quantification of diameters of all ISVs in A. n=15 per condition. *: p=0.0452, **: p=0.0150; one-way ANOVA with Tukey’s multiple comparisons test. C, Quantification of diameters of arterial and venous ISVs at 72 hpf of zebrafish. n=6 per condition. **: p=0.0013; Student’s t-test. D, Ratio of blood flow in the right ischemic vs left control foot by laser doppler, quantified in E.  N=7-8 per condition. *#: p=0.0439, *: p=0.0135, **: p=0.0028, ##: p=0.0056, #: p=0.0163; Student’s t-test. F, Micro-CT of arterial vasculature after surgery.  Representative reconstructed micro-CT images from LPHN2 ECKO and WT littermates on day 22 after surgery,'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='from LPHN2 ECKO and WT littermates on day 22 after surgery, quantified in G. n=8 per condition. ****: p<0.0001, ***: p=0.0007, *: p=0.0167; two-way ANOVA with Tukey’s multiple comparisons test.  H, Rectus femoris muscle from thighs of 10 weeks old mice of LPHN2 ECKO or wild-type littermates, stained for CD31 and SMA to visualize the entire vasculature and arteries. Scale Bar: 100µm. I. Quantification of capillary density in the entire stitched thigh muscle images.  n=3 for control and n=6 for Lphn2 ECKO. ****: p<0.0001; Student’s t-test. J, Schematic of mouse treadmill fatigue test.  K, Maximum running during test. n=4 for control and n=5 for Lphn2 ECKO.  **:p=0.0082; Student’s t-test.  L, Running distance for each mouse.  n=4 for control and n=5 for Lphn2'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content='distance for each mouse.  n=4 for control and n=5 for Lphn2 ECKO.  **: p=0.0064; Student’s t-test. M, Oxygen consumption during treadmill fatigue test.  *: p<0.0001;  two-way ANOVA with Tukey’s multiple comparisons test.  Control: N=5, LPHN2 ECKO: N=4.  N, Representative 3D reconstructed images at 20° downward angle of ECs plated on top of collagen gels and sprouting under static or flow conditions.  ECs were stained with phalloidin; pseudocolors indicate distance from the gel surface as per the color scale on the right.  O, Histograms of cell area vs. distance from gel surface in N.  P, Mean sprout length.  N=4.  ***: p=0.0006; one-way ANOVA with Tukey’s multi comparison test.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content=\"Table 1: Primer sequences used for quantitative PCR in this study\\n\\nGene name\\n\\nForward (5'-3')\\n\\nReverse (5'-3')\\n\\nHuman LPHN1\\n\\nCAGTACGACTGTGTCCCCTACA\\n\\nGCACCATGCGCCAGACTG\\n\\nHuman LPHN2\\n\\nGTCCAATATGAATGTGTCCCTTACA\\n\\nGCACCAAGCACCCGCCTT\\n\\nHuman LPHN3\\n\\nGTGCAGTATGAATGTGTCCCTTACA\\n\\nGCACCACGCCCCAGATTG\\n\\nHuman KLF2\\n\\nCGGCAAGACCTACACCAAGA\\n\\nTGGTAGGGCTTCTCACCTGT\\n\\nHuman GAPDH\\n\\nTGCACCACCAACTGCTTAGC\\n\\nGGCATGGACTGTGGTCATGAG\\n\\n\\n\\n\\n\\n\\n\\nzebrafish LPHN1a\\n\\nCAGTATGAATGTGTGCCATACA\\n\\nACACCATGCCCCCGACTG\\n\\nzebrafish LPHN2a\\n\\nGTCCAGTATGAATGTGTTCCCTACA\\n\\nGCACCACGACCCCGCTTG\\n\\nzebrafish LPHN2b.1\\n\\nGTCCAGTATGAGTGTGTGCCATACA\\n\\nTTGCACCAAGCTCCACTC\\n\\nzebrafish LPHN3.1\\n\\nGTTCAGTATGAGTGTGTGCCATACA\\n\\nACACCATGATCCAGCCTG\\n\\nzebrafish beta-actin\\n\\nGATCTGGCATCACACCTTCTAC\\n\\nTCTTCTCTCTGTTGGCTTTGG\"), Document(metadata={'source': './data/docx/EMBOJ-2023-114195.docx'}, page_content=\"GATCTGGCATCACACCTTCTAC\\n\\nTCTTCTCTCTGTTGGCTTTGG\\n\\n\\n\\nTable 2: Sequences of sgRNAs and morpholinos used in this study\\n\\n\\xa0\\n\\nSequence\\n\\nTarget\\n\\nDescription\\n\\n5'\\n\\nGCTGTACGAGTATGCTTCATGGG\\n\\nadgrl1a\\n\\nsgRNA targeting adgrl1a exon 4\\n\\n5'\\n\\nCGACGTATAAACTACCCCATCGG\\n\\nadgrl2a\\n\\nsgRNA targeting adgrl2a exon 4\\n\\n5'\\n\\nCCGTCTATGATAAACGCTCCGCC\\n\\nadgrl2b.1\\n\\nsgRNA targeting adgrl2b.1 exon 4\\n\\n5'\\n\\nAGTCCACGGCTGCGATGTATTGG\\n\\nadgrl3.1\\n\\nsgRNA targeting adgrl3.1 exon 4\\n\\n5'\\n\\nCATGTTTGCTCTGATCTGACACGCA\\n\\ntnnt2a\\n\\nmorpholino targeting tnnt2a translation\\n\\n5'\\n\\nCCTCTTACCTCAGTTACAATTTATA\\n\\nnone\\n\\ncontrol morpholino\")]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/var/folders/ps/jnmyqjq57sncdxvsr5297tk00000gq/T/ipykernel_37481/354113316.py:47: LangChainDeprecationWarning: The method `BaseRetriever.get_relevant_documents` was deprecated in langchain-core 0.1.46 and will be removed in 1.0. Use :meth:`~invoke` instead.\n",
      "  retrieved_docs = retriever.get_relevant_documents(query)\n",
      "/var/folders/ps/jnmyqjq57sncdxvsr5297tk00000gq/T/ipykernel_37481/354113316.py:60: LangChainDeprecationWarning: The method `Chain.__call__` was deprecated in langchain 0.1.0 and will be removed in 1.0. Use :meth:`~invoke` instead.\n",
      "  response_openai = qa_chain_openai(query)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing document: ./data/docx/EMBOJ-2023-114687.docx\n",
      "Split documents: [Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='The Nedd4L ubiquitin ligase is activated by FCHO2-generated membrane curvature\\n\\n\\n\\nYasuhisa Sakamoto1, Akiyoshi Uezu1, Koji Kikuchi1, Jangmi Kang2, Eiko Fujii2, Toshiro Moroishi3, Shiro Suetsugu4, and Hiroyuki Nakanishi1,2*\\n\\n\\n\\n1Department of Molecular Pharmacology, Faculty of Life Sciences, Kumamoto University, 1–1–1 Honjyo, Kumamoto 860-8556, Japan\\n\\n2Faculty of Clinical Nutrition and Dietetics, Konan Women’s University, 6–2–23 Morikita-machi, Kobe 658-0001, Japan\\n\\n3Department of Molecular and Medical Pharmacology, Faculty of Life Sciences, Kumamoto University, 1–1–1 Honjyo, Kumamoto 860-8556, Japan\\n\\n4Division of Biological Science, Graduate School of Science and Technology, Nara Institute of Science and Technology, 8916-5 Takayama-cho, Ikoma 630-0192, Japan'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='*Corresponding author. Tel: +81-96-373-5074; Fax: +81-96-373-5078; E-mail: hnakanis@gpo.kumamoto-u.ac.jp\\n\\n\\n\\n\\n\\n\\n\\n\\nAbstract'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='The C2-WW-HECT domain ubiquitin ligase Nedd4L regulates membrane sorting during endocytosis through the ubiquitination of cargo molecules such as the epithelial sodium channel (ENaC). Nedd4L is catalytically autoinhibited by an intramolecular interaction between its C2 and HECT domains, but the protein’s activation mechanism is poorly understood. Here, we show that Nedd4L activation is linked to membrane shape by FCHO2, a Bin-Amphiphysin-Rsv (BAR) domain protein that regulates endocytosis. FCHO2 was required for the Nedd4L-mediated ubiquitination and endocytosis of ENaC, with Nedd4L co-localizing with FCHO2 at clathrin-coated pits. In cells, Nedd4L was specifically recruited to, and activated by, the FCHO2 BAR domain. Furthermore, we reconstituted'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='by, the FCHO2 BAR domain. Furthermore, we reconstituted FCHO2-induced recruitment and activation of Nedd4L in vitro. Both the recruitment and activation were mediated by membrane curvature rather than protein-protein interactions. The Nedd4L C2 domain recognized a specific degree of membrane curvature that was generated by the FCHO2 BAR domain, with this curvature directly activating Nedd4L by relieving its autoinhibition. Thus, we show for the first time a specific function (i.e., recruitment and activation of an enzyme regulating cargo sorting) of membrane curvature by a BAR domain protein.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Keywords: Nedd4L; FCHO2; clathrin; endocytosis; membrane curvature.\\n\\n\\n\\n\\nIntroduction'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Posttranslational modification of proteins by covalent attachment of ubiquitin (Ub) is catalyzed by three enzymes: a Ub-activating enzyme (E1), a Ub-conjugating enzyme (E2), and a Ub-protein ligase (E3) that determines substrate specificity (Hershko & Ciechanover, 1998). E3 Ub ligases are classified into two categories based on their Ub transfer mechanisms: RING finger/U-box E3 and HECT-type E3. The Nedd4 family belongs to HECT-type E3 Ub ligases and consists of nine members, including Nedd4/Nedd4-1, Nedd4L/Nedd4-2, Itch, Smurf1, Smurf2, WWP1, WWP2, NEDL1, and NEDL2 (Ingham et al, 2004; Rotin & Kumar, 2009). They are characterized by a common modular organization, with an N-terminal C2 domain, two to four WW domains, and a C-terminal catalytic HECT domain.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='two to four WW domains, and a C-terminal catalytic HECT domain. While the C2 domain was originally identified in protein kinase C (PKC) as a Ca2+-dependent phosphatidylserine (PS)-binding domain (Nishizuka, 1992), it shows significant diversity in the binding partners, including intracellular proteins and other phospholipids, such as phosphatidylinositol (4,5)-bisphosphate [PI(4,5)P2] (Lemmon, 2008). The WW domains recognize proline-rich motifs, such as PPxY (PY motif, where x is any residue), of substrates or adaptor proteins. The HECT domain catalyzes the isopeptide bond formation between the Ub C terminus and the substrate lysine residues. At least some of the Nedd4 family members, including Nedd4L, are catalytically autoinhibited by an intramolecular'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Nedd4L, are catalytically autoinhibited by an intramolecular interaction between the C2 and HECT domains (Wang et al, 2010; Wiesner et al, 2007; Zhu et al, 2017). However, the activation mechanism of the Nedd4 family is poorly understood.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='The Nedd4 family is implicated in a wide variety of cellular processes by regulating membrane trafficking and degradation of components of various signaling pathways (Ingham et al, 2004; Rotin & Kumar, 2009). Among a diversity of substrates, the best-characterized is the epithelial sodium channel (ENaC). This channel is composed of three subunits, α-, β-, and γENaC, each of which possesses two transmembrane segments and intracellular N- and C-termini with a large extracellular loop (Butterworth, 2010; Rotin & Staub, 2011; Rossier, 2014). Each subunit has a PY motif at the cytosolic C-terminus. Nedd4L Ub ligase specifically binds to the PY motif and mediates the ubiquitination of lysine residues on the N-terminus of at least α- and γENaC at the plasma'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='on the N-terminus of at least α- and γENaC at the plasma membrane. Ub serves as an internalization signal for recognition by adaptor proteins, such as epsin and Eps15, for clathrin-mediated endocytosis (CME) (Traub, 2009). Consequently, ENaC is constitutively internalized through CME. Heterozygous mutations of the PY motif of β- or γENaC lead to impaired ubiquitination and endocytosis, resulting in persistence of the channels at the cell surface and increased Na+ absorption in Liddle syndrome, an autosomal dominant form of severe hypertension.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='The process of CME begins at the cytoplasmic surface of the plasma membrane with the assembly of clathrin with adaptor proteins and transmembrane cargo molecules (Doherty & McMahon, 2009; Traub, 2009). In parallel with this assembly, membrane curvature is generated to form hemispherical clathrin-coated pits (CCPs). This process leads to deep membrane invaginations. Subsequently, the neck of invaginated pits is constricted and severed to separate the newly formed clathrin-coated vesicles (CCVs) from the plasma membrane. These membrane curvatures are considered to be generated, sensed, and/or maintained by Bin-Amphiphysin-Rsv (BAR) domain superfamily proteins. BAR domains form a crescent-shaped dimer characterized by a specific degree of intrinsic curvature.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='characterized by a specific degree of intrinsic curvature. They bind to negatively charged phospholipids, such as PS, and force membranes to bend according to their intrinsic curvatures, thereby inducing the formation of membrane tubules with specific diameters (curvatures). For example, the diameters of membrane tubules generated by the amphiphysin BAR domain, FCHO2 BAR domain, and FBP17 BAR domain are 20–50 nm (sharper curvature) (Peter et al, 2004; Takei et al, 1999; Yin et al, 2009), 50–80 nm (intermediate curvature) (Henne et al, 2007), and ~150 nm (Takano et al, 2008) (shallower curvature), respectively.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='The FCHO2 BAR domain has been proposed to trigger the initial invagination for CCP formation (Henne et al, 2010). FCHO2 binds to the flat surface of the plasma membrane at sites where CCPs will form. FCHO2 then generates the initial membrane curvature and recruits the clathrin machinery for CCP formation. Here, we report the role of FCHO2 in Nedd4L-mediated ENaC endocytosis and show that FCHO2 BAR domain-generated membrane curvature induces the recruitment and activation of Nedd4L. \\n\\n\\n\\nResults \\n\\n\\n\\nFCHO2 is required for Nedd4L-mediated ubiquitination and endocytosis of ENaC'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='To study the role of FCHO2 in ENaC endocytosis, we established a HeLa cell line stably expressing all three subunits, α-, β-, and γENaC (αβγENaC-HeLa cells) (Fig EV1A-C). A FLAG tag was introduced into the extracellular region of αENaC in a position previously shown not to affect channel activity (Firsov et al, 1996). Upon FCHO2 knockdown by small interfering RNA (siRNA) methods, ENaC internalization was reduced, as was upon Nedd4L knockdown (Zhou et al, 2007) (Fig 1 A and B). Transferrin receptor (TfR) internalization was reduced upon FCHO2 knockdown but not upon Nedd4L knockdown (Fig 1C). The phenotype of FCHO2 knockdown in ENaC internalization was rescued by an siRNA-resistant (sr) form of FCHO2 (Fig 1D and Appendix Fig S1). Furthermore, αENaC'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='form of FCHO2 (Fig 1D and Appendix Fig S1). Furthermore, αENaC ubiquitination was reduced upon FCHO2 or Nedd4L knockdown but not upon FBP17 knockdown (Fig 1E and Fig EV2). FCHO2 or Nedd4L knockdown increased αENaC expression at the cell surface (Fig 1E and F). These results suggest that FCHO2 is required for Nedd4L-mediated ubiquitination and endocytosis of ENaC.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Recruitment and activation of Nedd4L by FCHO2 in cells'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Immunofluorescence microscopy revealed that in αβγENaC-HeLa cells, cell-surface αENaC co-localized with FCHO2 at clathrin-coated structures (Fig 2A). However, its co-localization with Nedd4L was not detected (Appendix Fig S2A), because exogenous expression of Nedd4L-green fluorescent protein (GFP) stimulated ENaC internalization, resulting in the disappearance of αENaC from the cell surface. Cell-surface αENaC was observed in cells lacking Nedd4L-GFP expression (non-transfected cells). When a catalytically inactive Nedd4L mutant, C922A, was expressed, it co-localized with cell-surface αENaC and endogenous FCHO2 (Fig 2B). The mutant also formed FCHO2-negative spots. In wild-type HeLa cells, Nedd4L accumulated at clathrin-coated structures where FCHO2'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Nedd4L accumulated at clathrin-coated structures where FCHO2 localized (Fig 2C). Additionally, Nedd4L formed clathrin-negative spots, suggesting that they may be involved in clathrin-independent endocytosis.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Using total internal reflection fluorescence (TIRF) microscopy, we investigated whether Nedd4L is recruited to CCPs in wild-type HeLa cells. Similar to FCHO2 described previously (Henne et al, 2010; Lehmann et al, 2019; Zaccai et al, 2022), Nedd4L showed dynamic co-localization with clathrin during pit formation (Fig 3). When both fluorescent protein-tagged FCHO2 and Nedd4L were expressed, the accumulation of Nedd4L at FCHO2-positive spots was not clearly observed (Appendix Fig S2B). The reason behind this remains unknown, but it has been shown that clathrin coat formation, measured in time-lapse images of cells expressing fluorescent proteins, is affected by several factors, such as overexpression level, choice of the proteins, and choice of the'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='overexpression level, choice of the proteins, and choice of the fluorescent tag (Aguet et al, 2013). These factors may contribute to the failure to detect the accumulation of Nedd4L at FCHO2-positive spots.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='While Nedd4L shows a punctate staining pattern at the cell surface (see Figs 2C, 3B and 3C), it exhibited diffuse distribution throughout the cytoplasm (Fig 4A and B). In COS7 cells, co-expression of the FCHO2 BAR domain [amino acid (aa) 1-302] recruited Nedd4L to membrane tubules (Fig 4A). The Nedd4L C2 domain (Myc-Nedd4L C2, aa 1-160) was also concentrated to membrane tubules, whereas C2 domain-deleted Nedd4L (Myc-Nedd4L ΔC2) was not. In contrast to FCHO2, the FBP17 BAR domain did not recruit Nedd4L to membrane tubules. Additionally, the BAR domain protein amphiphysin1 did not recruit Nedd4L to membrane tubules. Thus, Nedd4L is selectively recruited through the C2 domain to membrane tubules generated by the FCHO2 BAR domain. Similar results were obtained'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='by the FCHO2 BAR domain. Similar results were obtained in HEK293 cells (Fig 4B). In these cells, membrane tubules generated by amphiphysin1 were hardly detected. It showed a punctate staining pattern and did not co-localized with Nedd4L.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='We also investigated whether the FCHO2 BAR domain not only recruits but also activates Nedd4L in cells. For this purpose, we performed an in vivo ubiquitination assay using HEK293 cells, which formed membrane tubules upon FCHO2 expression more efficiently than did COS7 cells: 91 ± 2% of HEK293 cells vs. 32 ± 2% of COS7 cells [the mean ± standard error of the mean (S.E.M.) of four independent experiments]. As the catalytic activity of E3 Ub ligases is typically reflected in their autoubiquitination (Wiesner et al, 2007), we determined activity by measuring the ubiquitination level of Nedd4L itself. Expression of the FCHO2 BAR domain in cells remarkably enhanced Nedd4L autoubiquitination, whereas FBP17 or amphiphysin1 did not (Fig 5A). Tubulation-deficient'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='FBP17 or amphiphysin1 did not (Fig 5A). Tubulation-deficient FCHO2 mutants (K146E+R152E and L136E) (Henne et al, 2010) did not enhance Nedd4L activity (Fig 5A–C and Fig EV3). A catalytically inactive Nedd4L mutant, C922A, which was recruited to FCHO2-generated membrane tubules, showed little enhancement of ubiquitination upon FCHO2 expression (Fig 5D and E), indicating that Nedd4L ubiquitination is self-induced. Using αENaC as a substrate, we further confirmed that Nedd4L is activated upon FCHO2 expression in cells (Fig 5F).'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Preference of Nedd4L for FCHO2-generated membrane curvature'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='The aforementioned results prompted us to investigate whether FCHO2 and Nedd4L interact through membrane tubules rather than through stereotyped protein–protein interactions. Membrane tubules generated by BAR domains in cells have specific diameters (curvatures) (Henne et al, 2007; Peter et al, 2004; Takano et al, 2008; Takei et al, 1999; Yin et al, 2009). Nedd4L may preferentially bind to the specific degree of membrane curvature generated by FCHO2. To investigate this possibility, we first examined the biochemical properties of Nedd4L. Using a co-sedimentation assay with liposomes containing various percentages of PS, we found that the C2 domain of Nedd4L is a Ca2+-dependent PS-binding domain (Fig 6A and B). Submicromolar Ca2+ concentrations, which are'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='(Fig 6A and B). Submicromolar Ca2+ concentrations, which are comparable to intracellular levels in unstimulated cells, were sufficient for the Nedd4L C2 domain to bind to PS-containing liposomes. The C2 domain alone and full-length Nedd4L showed different Ca2+ requirements. The higher Ca2+ requirement of full-length Nedd4L may be due to the intramolecular interaction between the C2 and HECT domains. Increasing PS percentages in synthetic liposomes enhanced the liposome binding of the Nedd4L C2 domain, and PS requirements varied depending on Ca2+ concentration.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='We next examined whether Nedd4L preferentially binds to a specific degree of membrane curvature. For this purpose, liposomes of different sizes (curvatures) were prepared by stepwise extrusion through filter membranes with different pore sizes (0.8, 0.4, 0.2, 0.1, 0.05, and 0.03 µm) and sonication. Sonication or extrusion through a 0.03 µm pore-size filter was used to prepare the smallest liposomes. Transmission electron microscopy revealed that the actual liposome size was similar to the pore size for 0.2, 0.1, 0.05, and 0.03 µm pore-size liposomes, whereas it was significantly smaller for 0.8 and 0.4 µm pore-size liposomes (Fig 7A). Nedd4L showed the strongest binding to 0.05 µm pore-size liposomes (Fig 7B).'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Synaptotagmin I (SytI) tandem C2 domains can both generate and sense membrane curvature in vitro by penetrating their hydrophobic residues into the lipid bilayer (Martens et al, 2007). As a sensor, the binding affinity is considered to be highest for membranes that have a degree of curvature consistent with that generated by SytI itself. We investigated whether, in addition to sensing membrane curvature, the Nedd4L C2 domain can also generate membrane curvature in vitro and, if so, what diameter of membrane tubules is generated. As expected, transmission electron microscopy revealed that the C2 domain deformed liposomes into extensive tubulation (Fig 7C). It should be noted that Nedd4L does not generate membrane tubules in cells (see Fig 4A and B). The'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='not generate membrane tubules in cells (see Fig 4A and B). The tubule diameter generated by Nedd4L in vitro was mainly 40–80 nm, which is consistent not only with the diameter of 0.05 µm pore-size liposomes, to which Nedd4L preferentially binds, but also with the diameter of membrane tubules generated by FCHO2 (Fig 7D). These results suggest that Nedd4L shows the highest binding affinity for the degree of membrane curvature generated by FCHO2. It is therefore likely that FCHO2-induced recruitment and activation of Nedd4L are mediated by the degree of membrane curvature.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Roles of conserved hydrophobic residues in the Nedd4L C2 domain'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='We compared the C2 domain sequence of Nedd4L to those of other Nedd4 family members, SytI, and PKCγ (Fig 8A). SytI C2 domains possess hydrophobic residues (M173, F234, V304, and I367) for penetration into the lipid bilayer to generate and sense membrane curvature (Martens et al, 2007). The Nedd4L C2 domain also has three hydrophobic residues (I37, F38, and L99), which are conserved among Nedd4 family members except for WWP2. We mutated these residues to alanine to produce six mutants, I37A, F38A, L99A, I37A+F38A, I37A+L99A, and I37A+F38A+L99A (triple-A). Any single mutation did not inhibit Nedd4L recruitment to FCHO2-generated membrane tubules (Fig 8B). However, double and triple mutations severely impaired this recruitment. Furthermore, the I37A+F38A and'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='impaired this recruitment. Furthermore, the I37A+F38A and triple-A mutants showed little liposome binding and lost curvature generation (Fig 8C and D). Thus, these hydrophobic residues (I37, F38, and L99) are critical for both membrane binding and curvature sensing and generation. These residues are likely inserted into the lipid bilayer, thereby strengthening the C2 domain interaction with membranes and enabling the sensing and generation of membrane curvature. It has been shown that I37 and F38 are directly involved in Ca2+ binding (Escobedo et al, 2014). The residues may be inserted into the lipid bilayer while binding Ca2+.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='In the C2 domain of Smurf2, another Nedd4 family member, conserved hydrophobic residues (F29 and F30) are responsible for interactions with the HECT domain (Fig 8A) (Wiesner et al, 2007). Mutation at these residues impairs the interaction between the C2 and HECT domains, resulting in enhanced Smurf2 autoubiquitination by relieving autoinhibition. Consistent with these previous findings, we found that the Nedd4L I37A+F38A mutant, which was not recruited to FCHO2-generated membrane tubules, enhanced autoubiquitination, irrespective of FCHO2 expression (Fig 5D and E). These results suggest that I37 and F38 are critical for interactions with the HECT domain as well as for membrane binding and curvature sensing. Interestingly, the Nedd4L I37A+F38A mutant, as'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='sensing. Interestingly, the Nedd4L I37A+F38A mutant, as well as the C922A mutant, showed little αENaC ubiquitination (Fig EV4), indicating that membrane binding of Nedd4L is critical for ENaC ubiquitination.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='In vitro activation of Nedd4L by FCHO2-generated membrane curvature'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='To examine the properties of Nedd4L catalytic activity, we carried out an in vitro ubiquitination assay using an αENaC peptide as a substrate. The αENaC peptide possesses a conserved PY motif for interaction with the Nedd4L WW domain, and it is fused with monomeric streptavidin (mSA) (Lim et al, 2013) to bind to liposomes containing biotinylated phospholipids (biotinylated liposomes) (Sakamoto et al, 2017) (Fig 9A and B). We found that Ca2+ and PS were required for Nedd4L catalytic activity (Fig 9C and D), and that 0.05 µm pore-size liposomes were most effective in stimulating Ca2+- and PS-dependent Nedd4L activity (Fig 9E). Thus, the properties of Nedd4L catalytic activity are similar to those of its liposome binding. Furthermore, we found that Nedd4L did'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='of its liposome binding. Furthermore, we found that Nedd4L did not ubiquitinate PY motif-mutated (Y644A in αENaC) mSA-αENaC (Appendix Fig S3). These findings are consistent with defects in ENaC ubiquitination observed in Liddle syndrome.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='We next examined whether membrane localization of mSA-αENaC affects the membrane binding and catalytic activity of Nedd4L. Nedd4L bound only slightly to mSA-αENaC on PS-free (0% PS) liposomes, but the liposome binding was synergistically increased by elevating the PS percentage in liposomes (Fig 9F). By contrast, when free biotin competitively inhibited the binding of mSA-αENaC to biotinylated liposomes, the PS-dependent liposome binding of Nedd4L was significantly reduced, resulting in remarkable inhibition of mSA-αENaC ubiquitination (Fig 9D and F). Thus, membrane localization of the PY motif is critical for membrane binding and activation of Nedd4L. It is, therefore, likely that the PY motifs on membranes, such as ENaC, play a role in enhancing Nedd4L'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='on membranes, such as ENaC, play a role in enhancing Nedd4L catalytic activity by potentiating membrane binding through interactions with the WW domain.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='We then reconstituted in vitro FCHO2-induced recruitment and activation of Nedd4L. When liposomes (20% PS) were used, the FCHO2 BAR domain enhanced the liposome binding of Nedd4L (Fig 10A) and mSA-αENaC ubiquitination (Fig 10B). This enhancement was lost when the strength of interaction of Nedd4L with liposomes was increased by elevating the PS percentage in liposomes (~50% PS) (Fig 10C and Fig EV5).\\u3000In contrast to FCHO2, neither amphiphysin1 nor FBP17 enhanced Nedd4L activity (Fig 10D and Fig EV6A). Similarly, tubulation-deficient FCHO2 mutant (L136E) did not enhance Nedd4L activity (Fig 10E and Fig EV6A). These results are in good agreement with those obtained from in vivo experiments on FCHO2-induced recruitment and activation of Nedd4L. Electron'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='on FCHO2-induced recruitment and activation of Nedd4L. Electron microscopic analysis revealed that mSA-αENaC-associated liposomes were deformed by membrane-sculpting proteins into tubules with diameters similar to those of mSA-αENaC-free liposomes (Fig EV6B and see also Fig 7D). The diameter of membrane tubules generated by FCHO2 is consistent with that of 0.05 µm pore-size liposomes which are most effective in stimulating Nedd4L activity.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='To confirm whether FCHO2-induced recruitment and activation of Nedd4L are mediated by liposomes (i.e., membrane curvature) rather than by protein–protein interactions, we conducted two sets of experiments. Initially, we employed synthetic liposomes where 20% PS was substituted with 5% PI(4,5)P2. The FCHO2 BAR domain bound to PI(4,5)P2 as well as PS with high affinity, whereas the Nedd4L C2 domain did not (Fig 11A and B). The FCHO2 BAR domain is known to induce the tubulation of liposomes [5% PI(4,5)P2] (Henne et al, 2010). If Nedd4L could interact through protein–protein interactions with the FCHO2 BAR domain binding to liposomes [5% PI(4,5)P2], the BAR domain would facilitate the co-precipitation of Nedd4L in a co-sedimentation assay, thereby stimulating'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='of Nedd4L in a co-sedimentation assay, thereby stimulating mSA-αENaC ubiquitination. However, the FCHO2 BAR domain did not demonstrate such interactions (Fig 11C and D). Therefore, it is unlikely that Nedd4L interacts with the FCHO2 BAR domain on liposomes through protein–protein interactions. Secondly, we employed 0.05 µm pore-size liposomes that are most effective in stimulating Nedd4L activity. The diameter of 0.05 µm pore-size liposomes is consistent with that of membrane tubules generated by the FCHO2 BAR domain. The BAR domain is, thus, expected not to induce the tubulation of 0.05 µm pore-size liposomes. The FCHO2 BAR domain showed similar binding activity toward 0.8 and 0.05 µm pore-size liposomes (Fig 12A). When 0.8 µm pore-size liposomes were'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='liposomes (Fig 12A). When 0.8 µm pore-size liposomes were used, the FCHO2 BAR domain enhanced Nedd4L activity (Fig 12B). However, in the case of 0.05 µm pore-size liposomes, curvature-stimulated Nedd4L activity was not potentiated by the addition of the FCHO2 BAR domain. If Nedd4L had interacted with the BAR domain, we would have expected potentiation of Nedd4L activity. These results strongly suggest that FCHO2-induced activation of Nedd4L is mediated by membrane curvature and not by protein–protein interactions.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Discussion'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Nedd4L represents the ancestral Ub ligase with strong similarity to yeast Rsp5 which plays a key role in the trafficking, sorting, and degradation of a large number of proteins (Yang & Kumar, 2010). Consistently, Nedd4L regulates a growing number of substrates, including membrane receptors, transporters, and ion channels. Although Nedd4L exists in a catalytically autoinhibited state due to an intramolecular interaction between the N-terminal C2 and C-terminal HECT domains, little is known about its activation mechanism. In this study, we clearly demonstrated that Nedd4L is activated by FCHO2 F-BAR domain-induced membrane curvature. It has been shown that the activity of synptojanin-1, a phosphoinositide phosphatase, is modulated by membrane curvature'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='phosphatase, is modulated by membrane curvature (Chang-Ileto et al, 2011). Taken together, it is conceivable that membrane curvature serves as a signal for the activation of various enzymes involved in membrane trafficking.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Unique Properties of the Nedd4L C2 Domain'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Although both sytI and Nedd4L C2 domains are Ca2+-dependent PS-binding domains that can generate and sense membrane curvature, their properties show a number of significant differences. An interesting difference exists in the degrees of membrane curvatures generated by sytI and Nedd4L: the diameters of their membrane tubules are about 15–20 nm and 40–80 nm, respectively (Hui et al, 2009; Martens et al, 2007). This difference is not simply due to that between tandem and single C2 domains, because it has been shown that the sytI C2B domain alone generates membrane tubules with a similar diameter (Hui et al, 2009). SytI and Nedd4L C2 domains must therefore have respective unique structures that determine the specific degrees of membrane curvatures that they'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='determine the specific degrees of membrane curvatures that they sense and generate.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Another important difference between sytI and Nedd4L C2 domains exists in their Ca2+ requirements. SytI C2 domains require Ca2+ concentrations in the submillimolar range to generate membrane curvature (Hui et al, 2009). In contrast, submicromolar Ca2+ concentrations, which are comparable to intracellular levels in unstimulated cells, are sufficient for the Nedd4L C2 domain to sense membrane curvature. This is consistent with the observation that Nedd4L is recruited and activated in serum-starved cells. The different Ca2+ requirements between sytI and Nedd4L C2 domains are reflected in their functions. SytI C2 domains induce membrane curvature in response to an increase in intracellular Ca2+ concentration, which is essential for Ca2+-triggered membrane'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='concentration, which is essential for Ca2+-triggered membrane fusion (Hui et al, 2009; Martens et al, 2007). Thus, sytI C2 domains serve as a Ca2+ sensor and a curvature generator. In contrast, the Nedd4L C2 domain recognizes a specific degree of curvature without an increase in intracellular Ca2+ concentration. Therefore, the Nedd4L C2 domain serves as a curvature sensor to regulate the localization and activity of the E3 Ub ligase.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='A striking feature that distinguishes the Nedd4L C2 domain from other C2 domain proteins (except for at least a subset of Nedd4 family members) is that the Nedd4L C2 domain interacts not only with membranes but also with the intramolecular catalytic domain for its autoinhibition. Mutational experiments have revealed that I37 and F38 residues in the C2 domain are critical for interactions with both membranes and the HECT domain, suggesting that these two interactions are competitive. This idea is supported by the observation that the Ca2+ EC50 value of full-length Nedd4L is much higher than that of the C2 domain. The HECT domain in full-length Nedd4L may competitively inhibit the interaction of the C2 domain with membranes. It is likely that membranes'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='of the C2 domain with membranes. It is likely that membranes displace the HECT domain from the C2 domain when the strength of the C2 domain interaction with membranes is increased by the preferred membrane curvature. This displacement relieves the autoinhibition, resulting in Nedd4L activation. Thus, the membrane binding of Nedd4L is coupled with its activation.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Stimulatory and inhibitory effects of the FCHO2 BAR domain on the liposome binding and catalytic activity of Nedd4L'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='The FCHO2 BAR domain enhances both the liposome binding and catalytic activity of Nedd4L when the interaction strength of Nedd4L with liposomes is weak (20% PS) (Fig 10A and B). Conversely, the BAR domain inhibits the binding and activity of Nedd4L when the interaction of Nedd4L with liposomes is increased by elevating the PS percentage in liposomes (~50% PS) (Fig 10C and Fig EV5). Thus, the FCHO2 BAR domain exerts both stimulatory and inhibitory effects on Nedd4L, depending on the PS percentage in liposomes. However, given that 20% PS is comparable to the concentration in the inner leaflet of the plasma membrane, it is plausible that FCHO2 exhibits only a stimulatory effect on Nedd4L in cells. The mechanism underlying the stimulatory and inhibitory effects'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='The mechanism underlying the stimulatory and inhibitory effects in vitro is as follows: when using liposomes (20% PS), Nedd4L hardly binds to liposomes through PS (Fig. 9F and 10A). The addition of the FCHO2 BAR domain enhances the strength of Nedd4L’s interaction with PS by inducing membrane curvature. Consequently, Nedd4L gains a new binding to liposomes through PS, which augments its catalytic activity. In contrast, when using liposomes (~50% PS), Nedd4L binds to liposome through PS. The addition of the FCHO2 BAR domain hinders the PS-mediated liposome binding of Nedd4L, because FCHO2 and Nedd4L are PS-binding proteins and compete for PS binding in liposomes. Consequently, the BAR domain inhibits the catalytic activity of Nedd4L. This is consistent with'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='the catalytic activity of Nedd4L. This is consistent with the results obtained with 0.05 µm pore-size liposomes (Fig 12B). These 0.05 µm pore-size liposomes are comparable to liposomes with a higher PS percentage, as the interaction of Nedd4L with liposomes through PS is increased by membrane curvature. Therefore, when using 0.05 µm pore-size liposomes, the FCHO2 BAR domain inhibits the catalytic activity of Nedd4L.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Based on a model in which a subset of BAR domain proteins, such as FCHO2 and FBP17, drive liposome tubulation in vitro (Frost et al, 2008; Shimada et al, 2007), the FCHO2 BAR domain is considered to polymerize on the liposome surface into spiral dense coats by lateral and tip-to-tip interactions. When using liposomes (~50% PS), it is conceivable that the FCHO2 BAR domain densely packs the liposomes, leaving little free space for Nedd4L binding. Conversely, when using liposomes (20% PS), it is likely that the liposomes are loosely packed, providing ample surface space for Nedd4L binding. \\n\\n\\n\\nModel for ENaC Endocytosis'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Based on the present study, we propose a model for ENaC endocytosis through ubiquitination (Fig 13). First, FCHO2 binds to the plasma membrane and generates a specific degree of membrane curvature. FCHO2 interacts with clathrin adaptor proteins, such as Eps15, and recruits the clathrin machinery for CCP formation (Uezu et al, 2011; Henne et al, 2010). Eps15 has Ub-interacting motifs (Di Fiore et al, 2003). Nedd4L is subsequently recruited to and activated by FCHO2-generated curved membrane at the rim of nascent CCPs that ENaC enters. ENaC is then ubiquitinated and captured by Eps15 which FCHO2 interacts with. Thus, ubiquitinated ENaC is spatially restricted to CCPs and separated from unmodified ENaC. Overall, our findings suggest that in the early stage of'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='ENaC. Overall, our findings suggest that in the early stage of CME, a membrane deformation induced by FCHO2 can program, through both the recruitment and activation of Nedd4L, the gathering of a subclass of cargo molecules in CCPs.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Methods\\n\\n\\t\\n\\nConstruction'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Human αENaC, βENaC and γENaC cDNAs were kindly supplied by Dr. H. Kai (Kumamoto University, Kumamoto, Japan) (Sugahara et al, 2009). FLAG-αENaC was constructed by replacing the sequence [amino acid (aa) 169–174, TLVAGS] in the extracellular region with the FLAG sequence (DYKDDDK) as described previously (Firsov et al, 1996). This introduction of the epitope tag was carried out by polymerase chain reaction (PCR). FLAG-αENaC, βENaC, and γENaC were subcloned into pTRE-Tight-Hygro, pCAGI-Puro (Miyahara et al, 2000), and pCAGI-Bst (Umeda et al, 2006), respectively. pTRE-Tight-Hygro was constructed by inserting the hygromycin-resistance gene into pTRE-Tight (Clontech). FLAG-αENaC was also subcloned into pCAGI-Hygro, which was constructed by replacing the'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='into pCAGI-Hygro, which was constructed by replacing the puromycin-resistance gene of pCAGI-Puro with the hygromycin-resistance gene. Human Nedd4L (NM_015277, isoform 3) cDNA was obtained by reverse transcription (RT)-PCR and cloned in pCMV-Myc (Miyahara et al, 2000). Nedd4L was also cloned in pCAGI-puro-EGFP and pGEX-6P-hexahistidine (His6). pGEX-6P-His6 was constructed from pGEX-6P (GE Healthcare) to express an N-terminal GST-tagged, C-terminal His6-tagged protein. pCAGI-puro-EGFP was constructed by subcloning EGFP cDNA into pCAGI-puro to express a C-terminal GFP-tagged protein. The Nedd4L C2 domain (C2, aa 1–160) and C2 domain-deleted mutant (ΔC2, aa 150–955) were subcloned into pCMV-Myc and/or pGEX-6P. Mouse FCHO2 (full length, aa 1–809; BAR domain, aa'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='pGEX-6P. Mouse FCHO2 (full length, aa 1–809; BAR domain, aa 1–302) and human FBP17 (BAR domain, aa 1–300) were subcloned into pEGFP-C1 and/or pGEX-6P (Uezu et al, 2011; Tsujita et al, 2006). FCHO2 (aa 1–327) were subcloned into pGEX-6P. Human amphiphysin1 cDNA was obtained by RT-PCR and cloned in pEGFP-N3. Amphiphysin1 (BAR domain, aa 1–377) was then subcloned into pGEX-6P. mTagRFP-T-Clathrin-15 was a gift from Michael Davidson (Addgene plasmid # 58005 ; http://n2t.net/addgene:58005 ; RRID:Addgene_58005) (Shaner et al, 2008). Human hemagglutinin (HA)-tagged Ub cDNA in pCGN was kindly donated by Dr. M. Nakao (Kumamoto University, Kumamoto, Japan). Ub cDNA was subcloned into pET3a. Human UbcH7 cDNA was obtained by RT-PCR and cloned in pGEX-6P. pET21d-Ube1 was'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='was obtained by RT-PCR and cloned in pGEX-6P. pET21d-Ube1 was a gift from Cynthia Wolberger (Addgene plasmid # 34965 ; http://n2t.net/addgene:34965 ; RRID:Addgene_34965) (Berndsen & Wolberger, 2011). pRSET-mSA was a gift from Sheldon Park (Addgene plasmid # 39860 ; http://n2t.net/addgene:39860 ; RRID:Addgene_39860) (Lim et al, 2013). mSA-αENaC constructed in pRSET-mSA to express a fusion protein of the N-terminal (aa 7–40) and C-terminal (aa 625–659) intracellular regions of αENaC (see Fig. 9A). Mutagenesis was performed by PCR using mutated primers and a site-directed mutagenesis kit (Stratagene).'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Protein expression and purification'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Expression and purification of GST-tagged proteins were performed using standard protocols. His6-tagged proteins were purified with Talon metal affinity resin (Clontech) using the manufacturer’s protocol. The GST tag was cleaved off by PreScission protease (GE Healthcare). The GST tag of BAR domains did not inhibit liposome binding or tubulation, whereas the GST tag of Nedd4L inhibited these functions. As FCHO2 (BAR domain, aa 1–302) was easily aggregated by the removal of the GST tag, FCHO2 (aa 1–327) was used instead as a GST-cleaved form of the FCHO2 BAR domain. Full-length Nedd4L (N-terminal GST-tagged, C-terminal His6-tagged) was purified by glutathione–Sepharose (GE Healthcare) column chromatography. The GST tag was cleaved off and the Nedd4L sample'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='The GST tag was cleaved off and the Nedd4L sample was then subjected to Talon metal affinity column chromatography. After the eluate was passed through glutathione–Sepharose beads, the sample was dialyzed against buffer A (20 mM Tris-HCl at pH 7.5, 100 mM KCl, 0.5 mM EDTA, 1 mM DTT, and 20 % sucrose). The Nedd4L sample was stored at −80°C until use. Ub was expressed in E. coli strain BL21 (DE3) pJY2 and purified as previously described (Pickart & Raasi, 2005). His6-tagged mSA-αENaC was expressed in E. coli strain BL21 (DE3) pLysS and purified as previously described (Lim et al, 2013) with slight modifications. Briefly, the inclusion body was isolated and solubilized in buffer B (50 mM Tris-HCl at pH 8.0 and 6 M guanidine hydrochloride). The sample was then'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='at pH 8.0 and 6 M guanidine hydrochloride). The sample was then subjected to Talon metal affinity column chromatography. The eluate was added drop by drop to ice-cold refolding buffer (50 mM Tris-HCl at pH 8.0, 150 mM NaCl, 0.15 mg/ml D-biotin, 0.2 mg/ml oxidized glutathione, and 1 mg/ml reduced glutathione) under rapid stirring. The sample was then concentrated using a Centriprep centrifugal filter unit (Millipore) and centrifuged to remove aggregates. The supernatant was dialyzed against buffer C (20 mM Tris-HCl at pH 8.0). The mSA-αENaC sample was further purified by Mono Q anion exchange column chromatography as follows. The sample was applied to a Mono Q 5/50 GL column (GE Healthcare) equilibrated with buffer C. Elution was performed with a 50 ml'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='equilibrated with buffer C. Elution was performed with a 50 ml linear gradient of NaCl (0–200 mM) in buffer C. Fractions of 1 ml each were collected. The active fractions (fraction 18–22) were collected and dialyzed against buffer D (10 mM Tris-HCl at pH 7.5, 100 mM KCl, and 0.5 mM EDTA). The mSA-αENaC sample was stored at −80°C until use.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Antibodies\\n\\nRabbit anti-Nedd4L antibody was raised against GST-Nedd4L C2 (aa 1–160). Rabbit anti-FCHO2 antibody was obtained as previously described (Uezu et al, 2011). The following antibodies were purchased from commercial sources: mouse anti-Myc (9E10) (American Type Culture Collection); mouse anti-FLAG (M2) and mouse anti-α-tubulin (clone DM1A) (Sigma-Aldrich); mouse anti-clathrin heavy chain (BD Biosciences); rabbit anti-GFP (MBL Co.); rabbit anti-αENaC (Calbiochem); rabbit anti-βENaC (Proteintech); rabbit anti-γENaC (Abcam); mouse anti-transferrin receptor H68.4 (Santa Cruz); mouse anti-Ub antibody P4D1 (Santa Cruz); and secondary antibodies conjugated with Alexa Fluor 488, 594, and 647 (Invitrogen).\\n\\n\\n\\nCell culture and transfection'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Cell culture and transfection\\n\\nHeLa, HEK293, and COS7 cells were cultured in Dulbecco’s modified Eagle medium (DMEM) supplemented with 10% fetal calf serum. HEK293 cells were grown on poly-L-lysine-coated dishes. Transfection was performed by using Lipofectamine 2000 (Invitrogen) according to the manufacturer’s protocol.\\n\\n\\n\\nGeneration of stable HeLa cells expressing all three ENaC subunits'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='HeLa cells were transfected with the pTet-On Advanced vector (Clontech) to develop stable Tet-On cell lines for a doxycycline (Dox)-inducible expression system according to the manufacturer’s protocol. Tet-On cells were selected in 0.5 mg/ml G418. Cloned Tet-On cells were then transfected with pCAGI-Bst-γENaC and selected in 4 µg/ml blasticidin (in the presence of G418). Blasticidin-resistant cells were transfected with pCAGI-Puro-βENaC and selected in 1 µg/ml puromycin (in the presence of G418 and blasticidin). Puromycin-resistant cells were finally transfected with pTRE-Tight-Hygro-FLAG-αENaC and cultured in 0.2 mg/ml hygromycin (in the presence of G418, blasticidin, and puromycin). After selection, cells were cloned and maintained in 0.5 mg/ml G418, 0.2'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='cells were cloned and maintained in 0.5 mg/ml G418, 0.2 mg/ml hygromycin, 1 µg/ml puromycin, and 2.5 µg/ml blasticidin. FLAG-tagged αENaC expression was induced by treating cells with 1 µg/ml Dox in the presence of 40 µM amiloride overnight. Amiloride was added to prevent cellular Na+ overload.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Immunofluorescence microscopy'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Cells were fixed with 3.7% formaldehyde in phosphate-buffered saline (PBS) at room temperature for 15 min, and then permeabilized with 0.2% Triton X-100 in PBS for 10 min. After blocking with 1% bovine serum albumin (BSA) in PBS, cells were sequentially incubated with primary antibodies and fluorescence-conjugated secondary antibodies at room temperature for 1 h, respectively. Cells were then analyzed with either a fluorescence microscope (BX51; Olympus) or a confocal microscopy system (BX50, Fluoview FV300, and Fluoview FV1200; Olympus). A 60 × oil immersion objective (NA = 1.40, Olympus) was used. All comparable images were acquired under identical parameters. To visualize cell-surface FLAG-αENaC in αβγENaC-HeLa cells, cells were incubated with 5 µg/ml'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content=\"in αβγENaC-HeLa cells, cells were incubated with 5 µg/ml anti-FLAG M2 antibody at 4°C for 1 h to prevent internalization. Cells were then fixed, permeabilized, and stained with a secondary antibody. Cells were also sequentially incubated with primary (anti-FCHO2) and secondary antibodies. To quantify co-localization, the Pearson's correlation coefficient was calculated from three cells using the Fiji (ImageJ) Coloc2 plug-in.\"), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='RNA interference'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Stealth™ double-stranded RNAs were purchased from Invitrogen. siRNA sequences of human FCHO2, Nedd4L, and FBP17 were 5′-CCA CAG AUC UUA GAG UGG AUU AUA A-3′ (corresponding to nucleotides 2048–2072 relative to the start codon), 5′-GAA GAG UUG CUG GUC UGG CCG UAU U-3′ (corresponding to nucleotides 1778–1802 relative to the start codon), and 5′-CCC ACU UCA UAU GUC GAA GUC UGU U-3′for human FBP17 (corresponding to nucleotides 1804–1828 relative to the start codon), respectively. A double-stranded RNA targeting luciferase (GL-2, 5′-CGU ACG CGG AAU ACU UCG AAA UGU C-3′) was used as a control. HeLa cells were transfected with 20 nM siRNA using Lipofectamine 2000 according to the manufacturer’s protocol. After 24 h, a second transfection was performed, and cells'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='After 24 h, a second transfection was performed, and cells were cultured for 3 days and subjected to various experiments. For rescue experiments, cells were transfected with the intended plasmid at 36 h after the second transfection. After 18 h, cells were fixed and analyzed.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Quantitative real-time PCR analysis\\n\\nTotal RNA was isolated using ISOGEN (Nippon Gene, 319-90211), and cDNA was synthesized using ReverTra Ace qPCR RT Master Mix with gDNA Remover (Toyobo, FSQ-301), following manufacturer’s instruction. Real time PCR was carried out using Thunderbird Next SYBR (Toyobo, QPX-201) on StepOnePlus Real-Time PCR Systems (Applied Biosystems). The sequences of qPCR primers are 5′-TCT CCC ATC GCT TCA ACG AG-3′ and 5′-GTT GCA CCC AGC TTG AGA GA-3′ for human FBP17; 5′-GTC TCC TCT GAC TTC AAC AGC G-3′ and\\xa05′-ACC ACC CTG TTG CTG TAG CCA A-3′ for\\xa0human GAPDH. The expression of target genes was normalized to GAPDH mRNA levels.\\n\\n\\n\\nEndocytosis assay'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='αβγENaC-HeLa cells at approximately 90% confluence in a 10-cm dish were starved with serum-free medium (DMEM containing 1% BSA, 1 µg/ml Dox, and 40 µM amiloride) at 37°C for 4 h. For ENaC endocytosis, cells were then incubated with 5 µg/ml anti-FLAG M2 in 2 ml of DMEM containing 1% BSA at 4°C for 1 h to prevent internalization. After three washes with ice-cold PBS containing 1 mM MgCl2 and 0.1 mM CaCl2 (PBS-CM), endocytosis was allowed by incubating cells in prewarmed DMEM containing 1% BSA at 37°C for the indicated periods of time. Endocytosis was stopped by placing cells on ice and washing them with ice-cold PBS-CM. Antibody bound to the cell surface but not internalized was removed by acid stripping with 0.2 M acetic acid and 0.5 M NaCl solution (pH'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='stripping with 0.2 M acetic acid and 0.5 M NaCl solution (pH 2.5). After washing with ice-cold PBS-CM, internalized ENaC was detected by immunoblotting (IB) or immunofluorescence microscopy. For IB, cells were scraped in 1 ml of lysis buffer A (50 mM Tris-HCl at pH 8.0, 150 mM NaCl, 5 mM EDTA, 1% Triton X-100, 1 mM PMSF, 20 µg/ml leupeptin, and 1 µg/ml pepstatin A). Cells were solubilized at 4°C for 1 h on a rotating wheel, and centrifuged at 20,000 × g at 4°C for 15 min. Protein G-Sepharose beads (GE Healthcare) were added to the supernatant and incubated at 4°C for 3 h. The beads were then thoroughly washed with lysis buffer A and boiled in sodium dodecyl sulfate (SDS) sample buffer. The sample was subjected to SDS-polyacrylamide gel electrophoresis'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='sample was subjected to SDS-polyacrylamide gel electrophoresis (PAGE), followed by IB with anti-ENaC subunit antibodies. Internalization was expressed as a percentage of the initial amount of cell-surface ENaC subunit determined by incubating cells with anti-FLAG antibody at 4°C for 1 h and lysing them without incubation at 37°C or acid stripping. For immunofluorescence microscopy, cells were transfected with control or FCHO2 siRNA and then with GFP-siRNA-resistant (sr) FCHO2 mutant. After cells were labeled with anti-FLAG antibody on 4°C, they were incubated at 37°C for 10 min to allow ENaC endocytosis. After acid stripping, cells were fixed with 3.7% formaldehyde and permeabilized with 0.2% Triton X-100. The antibody-labeled, internalized αENaC was'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='0.2% Triton X-100. The antibody-labeled, internalized αENaC was visualized with a fluorescence-conjugated secondary antibody. The percentage of cells with internalized αENaC among cells expressing GFP-sr-FCHO2 was determined using MetaMorph imaging System software. All images were acquired under identical parameters. For TfR endocytosis, a cell surface biotinylation assay was carried out as previously described (Roberts et al, 2001). Briefly, the cell surface was labeled with Sulfo-NHS-SS-Biotin (Thermo Fisher Scientific) at 4°C. After cells were incubated with ice-cold quenching buffer (50 mM NH4Cl in PBS-CM) at 4°C, they were incubated with serum free medium for 10 min at 37°C. Biotin was removed from the cell surface by incubation with ice-cold MesNa'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='removed from the cell surface by incubation with ice-cold MesNa buffer (50 mM MesNa, 100 mM NaCl, and 100 mM Tris-HCl, pH 8.8). Cells were then incubated with iodoacetamide at 4°C. Cell lysates were then incubated with NeutrAvidin beads (Thermo Fisher Scientific). The beads were boiled in SDS sample buffer. Proteins were analyzed by IB with TfR antibody.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='ENaC expression and ubiquitination at the cell surface'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='αβγENaC-HeLa cells at approximately 90% confluence in a 10-cm dish were starved with serum-free medium (DMEM containing 1% BSA, 1 µg/ml Dox, and 40 µM amiloride) at 37°C for 4 h, washed three times with ice-cold PBS, and incubated at 4°C for 45 min with 2 ml of 1 mg/ml EZ-link Sulfo-NHS-SS-Biotin in PBS with gentle shaking. Subsequently, cells were extensively washed with ice-cold Tris-buffered saline, and then scraped into 1 ml of lysis buffer B (50 mM Tris-HCl at pH 7.5, 150 mM NaCl, 5 mM EDTA, 1% Triton X-100, 10 mM N-ethylmaleimide, 1 mM PMSF, 10 µg/ml leupeptin, and 1 µg/ml pepstatin A) containing 40 µM MG132. Cells were solubilized at 4°C for 1 h on a rotating wheel, and centrifuged at 20,000 × g at 4°C for 15 min. The supernatant (4 mg of protein)'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='20,000 × g at 4°C for 15 min. The supernatant (4 mg of protein) was incubated at 4°C for 3 h with 10 µg of anti-FLAG M2 antibody. Protein G-Sepharose beads (50 µl of 50% slurry) were then added to the sample, which was incubated at 4°C for 3 h. After beads were thoroughly washed with lysis buffer B containing 40 µM MG132, FLAG-tagged proteins were eluted from the beads with 200 µg/ml FLAG peptide (Sigma-Aldrich) in 0.5 ml of lysis buffer B. After centrifugation, the supernatant was incubated with 25 µl of NeutrAvidin beads at 4°C overnight. After the beads were thoroughly washed with lysis buffer B, bound proteins were eluted by incubating the beads at 4°C for 1 h with 150 µl of lysis buffer B containing 200 mM DTT. After centrifugation, 5 × SDS sample'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='B containing 200 mM DTT. After centrifugation, 5 × SDS sample buffer was added to the supernatant, which was then analyzed by IB with anti-FLAG and anti-Ub antibodies. The chemiluminescence intensity was quantified by scanning using ImageJ software (NIH).'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='TIRF Microscopy\\n\\nFor live observation with TIRF microscopy, HeLa cells were grown on a glass-bottomed dish as described previously (Shimada et al, 2007). Cells were observed at 10–12 h after transfection as described previously (Henne et al, 2010). Images were acquired for 5 min at 1 s intervals with a TIRF microscopy system (Olympus), a 100 × oil immersion objective NA = 1.45 (Olympus) and MetaMorph Imaging System Software. All comparable images were acquired under identical parameters. \\n\\n\\n\\nLiposome preparation'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Phospholipids were purchased from commercial sources: phosphatidylcholine (PC, 1,2-dioleoyl-sn-glycero-3-phosphocholine), phosphatidylethanolamine (PE, 1,2-dioleoyl-sn-glycero-3-phosphoethanolamine), PS (1,2-dioleoyl-sn-glycero-3-phospho-l-serine), rhodamine-PE [18:1 L-α-PE-N-(lissamine rhodamine B sulfonyl)], PEG2000-biotin-PE (1,2-distearoyl-sn-glycero-3-phosphoethanolamine-N-[biotinyl (polyethyleneglycol)-2000]), porcine brain PS, brain PI(4,5)P2, and porcine brain total lipid extract were obtained from Avanti Polar Lipids; cholesterol and Folch lipids [bovine brain lipids (Folch Fraction 1)] were obtained from Sigma-Aldrich; X-biotin-PE [N-((6-(biotinoyl)amino)hexanoyl)-1,2-dihexadecanoylsn-glycero-3-phosphoethanolamine] was obtained from Thermo Fisher'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='was obtained from Thermo Fisher Scientific; and fluorescein-PE [N-(fluorescein-5-thiocarbamoyl)-1,2-dihexadecanoyl-sn-glycero-3-phosphoethanolamine] was obtained from AAT Bioquest Inc.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Liposomes were prepared using brain lipids (Folch lipids, ~50% PS of total lipids), brain lipids (20% PS of total lipids, a mixture of 89.5% porcine brain total lipid extract and 10.5% porcine brain PS), or a synthetic lipid mixture [70% PC, 20% PE, 10% cholesterol (w/w), and varying percentages of PS or PI(4,5)P2 (with a corresponding reduction in PC)]. Lipid concentrations were monitored by including 1% rhodamine-PE (w/w). To prepare mSA-αENaC–associated liposomes, 2% PEG2000-biotin-PE (w/w) was included. Lipids were dried under nitrogen gas, desiccated for 2 h, and resuspended at 1 mg/ml in lipid buffer (50 mM HEPES-NaOH at pH 7.2, 100 mM KCl, 2 mM MgCl2, and 5 mM EGTA), followed by hydration at 37°C for 1 h. To make liposomes of different sizes, they'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='at 37°C for 1 h. To make liposomes of different sizes, they were subjected to five freeze–thaw cycles and centrifuged at 100,000 × g for 15 min. Precipitated liposomes were then resuspended in the same buffer to make a final concentration of 1 mg/ml. They were subjected to extrusion through polycarbonate filter membranes (Avanti Polar Lipids): liposomes were sequentially passed 9 times through a 0.8-µm filter, 21 times through a 0.4-µm filter, 21 times through a 0.2-µm filter, 21 times through a 0.1-µm filter, 21 times through a 0.05-µm filter, and 21 times through a 0.03-µm filter. Instead of extrusion through a 0.03-µm filter, liposomes with the smallest size were also prepared by sonication three times for 5 s with a Branson Sonifier 25 at power level 1.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='times for 5 s with a Branson Sonifier 25 at power level 1. Liposome diameters were examined by transmission electron microscopy.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Co-sedimentation assay'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='A co-sedimentation assay was performed as described (Uezu et al, 2011) with slight modifications. Briefly, 5–10 µg of protein was incubated at 25°C for 10 min with 0.5 mg/ml liposomes in 100 µl of lipid buffer containing various concentrations of CaCl2 and centrifuged at 165,000 × g at 25°C for 15 min. Equal amounts of the supernatants and pellets were subjected to SDS-PAGE, followed by Coomassie brilliant blue (CBB) staining. Proteins were quantified by scanning using ImageJ (NIH). The assay with liposomes of different sizes was performed using NeutrAvidin (Thermo Fisher Scientific Inc) as described (Sakamoto et al, 2017). Briefly, brain-lipid (Folch-lipid) liposomes (~50% PS) were supplemented with 1% X-biotin-PE and 1% rhodamine-PE, subjected to'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='with 1% X-biotin-PE and 1% rhodamine-PE, subjected to extrusion through filter membranes, and then sonicated. Purified Nedd4L protein (10 µg) was incubated with 0.2 mg/ml liposomes in 100 µl of lipid buffer containing 2mM CaCl2 (0.1 µM Ca2+). The sample was then incubated with 2 µg of NeutrAvidin and ultracentrifuged. Both total and supernatant fluorescence were measured. The proportion of precipitated liposomes of each size was ~100%.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='The assay with mSA-αENaC–associated liposomes was performed as follows. Liposomes were prepared from a synthetic lipid mixture [various percentages of PS or 5% PI(4,5)P2] or brain lipids supplemented with 2% PEG2000-biotin-PE and 1% rhodamine-PE. The liposomes (10 µg) were incubated with purified mSA-αENaC protein (35 pmol) at 4°C for 10 min in 25 µl of lipid buffer. The sample was then incubated at 25°C for 10 min with 0.07 µM (8 µg/ml) Nedd4L and/or 1.4 µM BAR domains in 50 µl of ubiquitination buffer (50 mM HEPES-NaOH at pH 7.2, 100 mM KCl, 2 mM MgCl2, 5 mM EGTA, and 0.1 mM DTT) containing 4 mM CaCl2 (0.7 µM Ca2+). Subsequently, the sample was subjected to ultracentrifugation. Equal amounts of the total samples and pellets were subjected to SDS-PAGE,'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='of the total samples and pellets were subjected to SDS-PAGE, followed by CBB staining and IB. The chemiluminescence intensity was quantified by scanning using ImageJ (NIH).'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='The assay with mSA-αENaC-associated liposomes (0.05 µm pore-size) was done as follows. Liposomes were prepared from brain lipids (20% PS) supplemented with 2% PEG2000-biotin-PE, 1% rhodamine-PE, and 0.5% fluorescein-PE (w/w) and extruded through membrane filters. To precipitate liposomes, the sample (50 μl) was first incubated with 1 µg of anti-fluorescein goat antibody (Novus Biologicals) at room temperature for 1 h and then incubated with 0.5 µg of protein A/G/L (BioVision) at room temperature for 1 h. Subsequently, the sample was centrifuged at 165,000 × g at 25°C for 15 min. Equal amounts of the supernatants and pellets were subjected to SDS-PAGE, followed by CBB staining and IB. The percentage of precipitated liposomes was ~100%.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Calculation of free Ca2+ concentrations\\n\\nAt 1.0, 2.0, 3.0, 3.5, 4.0, 4.5, and 4.9 mM CaCl2 in lipid buffer, free Ca2+ concentrations were calculated to be 0.04, 0.1, 0.3, 0.4, 0.7, 1.6, and 8.1 µM, respectively, as determined by the Ca-Mg-ATP-EGTA Calculator v1.0 using constants from the NIST database #46 v8. At 1.0, 2.0, 3.0, 3.5, 4.0, 4.5, and 4.9 mM CaCl2 in ubiquitination buffer, free Ca2+ concentrations were calculated to be 0.04, 0.1, 0.2, 0.4, 0.7, 1.4, and 6.7 µM, respectively.\\n\\n\\n\\nIn vitro tubulation assay'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='In vitro tubulation assay\\n\\nAn in vitro tubulation assay was performed as described (Uezu et al, 2011; Tsujita et al, 2006) with slight modifications. Briefly, liposomes were prepared from brain lipids (Folch lipids) supplemented with 1 % rhodamine-PE. The liposomes (0.2 mg/ml liposomes) were incubated with the Nedd4L C2 domain (4.5 µM) or BAR domains (1.4 µM) at 25°C for 10 min in lipid buffer containing 3 mM CaCl2 (0.3 µM Ca2+). The sample was examined by fluorescence or electron microscopy. The tubulation assay with mSA-αENaC-associated liposomes was performed using the sample mixture utilized in the co-sedimentation assay in which mSA-αENaC-associated liposomes (brain-lipid liposomes) were used.\\n\\n\\n\\nNegative-staining electron microscopy'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Negative-staining electron microscopy\\n\\nNegative-staining transmission electron microscopy was performed as described (Shimada et al, 2007). Box and whisker plots were used to represent the distribution of liposome diameters. \\n\\n\\n\\nIn vivo ubiquitination assay'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='HEK293 cells were used, because they formed membrane tubules upon FCHO2 expression more efficiently than did COS7 cells. HEK293 cells at approximately 90% confluence on a 6-well plate were transfected with 0.8 µg of pCMV-Myc-Nedd4L, 1.6 µg of pEGFP-BAR domain, and 1.6 µg of pCGN-HA-Ub. In some experiments, cells were transfected with various doses (0–1.5 µg) of pCMV-Myc-Nedd4L, 1.5 µg of pEGFP-FCHO2 BAR domain, 1 µg of pCGN-HA-Ub, and 0.5 µg of pCAGI-Hygro-FLAG-αENaC. Total transfected cDNA amount was held constant using empty pCMV-Myc vector. Amiloride (40 µM) was added to the medium when αENaC cDNA was transfected. At 21 h after transfection, cells were washed with PBS and incubated with serum-free DMEM containing 1% BSA. After 3 h incubation, 10 µM MG132'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='DMEM containing 1% BSA. After 3 h incubation, 10 µM MG132 was added to the medium. After further 4 h incubation, cells were solubilized in 150 µl of lysis buffer C (50 mM Tris-HCl at pH 7.5, 150 mM NaCl, 5 mM EDTA, 10 mM N-ethylmaleimide, 1 mM PMSF, 10 µg/ml leupeptin, and 1µg/ml pepstatin A) containing 1% SDS. After boiling, the sample was sonicated and then centrifuged at 20,000 × g for 15 min. The supernatant was diluted 10-fold with lysis buffer C containing 1% Triton X-100 and then incubated at 4°C for 3 h with either anti-Myc or anti-FLAG M2 antibody. Protein G-Sepharose beads were added to the sample, which was then incubated at 4°C for 3 h. After the beads were thoroughly washed with lysis buffer C containing 1% Triton X-100, bound proteins were'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='lysis buffer C containing 1% Triton X-100, bound proteins were eluted by boiling in SDS sample buffer for 5 min. These proteins were then subjected to SDS-PAGE, followed by IB.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='In vitro ubiquitination assay'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='mSA-αENaC-associated liposomes were made by incubating liposomes (10 µg) with mSA-αENaC (35 pmol) in 25 µl of lipid buffer, as described for the co-sedimentation assay. They were then incubated at 25°C for 10 min with 0.07 µM Nedd4L, 0.04 µM E1 (Ube1), 0.7 µM E2 (UbcH7), 7 µM Ub, and various concentrations of BAR domains in 50 µl of ubiquitination buffer containing 4 mM CaCl2 (0.7 µM Ca2+) and 2 mM ATP. Where indicated, various concentrations of CaCl2 were used. The ubiquitination reaction was started by adding ATP and stopped by adding 10 µl of 6 × SDS sample. The sample was then boiled, subjected to SDS-PAGE, and immunoblotted with anti-Ub and anti-αENaC antibodies. The chemiluminescence intensity was quantified by scanning using ImageJ (NIH).'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Other procedures\\n\\nStripping immunoblots for re-probing was carried out with the WB Stripping Solution Strong (Nacalai Tesque) according to the manufacturer’s protocol. Protein concentrations were quantified with BSA as a reference protein by the Bradford method (Bio-Rad) or the BCA method (Thermo Fisher Scientific). SDS-PAGE was performed as described (Laemmli, 1970).\\n\\n\\n\\nStatistical analysis\\n\\nAll experiments were performed at least three times (biological replication) and representative results are shown. Data are shown as the mean ± S.E.M. Student’s tests, one-way analysis of variance test, and two-way analysis of variance test were used to evaluate statistical significances between different treatment groups.\\n\\n\\n\\nData availability'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Data availability\\n\\nThis study includes no data deposited in external repositories.\\n\\n\\n\\n\\tAcknowledgments'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Acknowledgments\\n\\n\\t\\n\\n\\tWe thank Drs. H. Kai and M. Nakao for kindly providing us with ENaC and Ub cDNAs, respectively. We are also grateful to Dr. T. Takenawa (Kobe University, Kobe, Japan) for helpful discussions. This work was partly carried out at the Institute of Molecular Embryology and Genetics, the Gene Technology Centre, and Research Facilities of the School of Medicine, Kumamoto University. This study was supported by JSPS KAKENHI Grant Numbers 19K06643 (to H.N.), 19K06544 (to Y.S.), and Setsuro Fujii Memorial, The Osaka Foundation for Promotion of Fundamental Medical Research (to Y.S.) and by grants from the Mochida Memorial Foundation, the Takeda Science Foundation, and the Uehara Memorial Foundation (to K.K.).'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Disclosure and Competing Interests Statement\\n\\nThe authors declare no competing interests.\\n\\n\\n\\n\\n\\n\\nReferences \\n\\n\\n\\nAguet F, Antonescu CN, Mettlen M, Schmid S & Danuser G (2013) Advances in analysis of low signal-to-noise images link dynamin and AP2 to the function of an endocytic checkpoint. Dev Cell 26: 279–291\\n\\nBerndsen CE & Wolberger C (2011) A spectrophotometric assay for conjugation of ubiquitin and ubiquitin-like proteins. Anal Biochem 418: 102–110\\n\\nButterworth MB (2010) Regulation of the epithelial sodium channel (ENaC) by membrane trafficking. Biochim Biophys Acta 1802: 1166–77'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Chang-Ileto B, Frere SG, Chan RB, Voronov SV, Roux A, & Di Paolo G (2011) Synaptojanin 1-mediated PI(4,5)P2 hydrolysis is modulated by membrane curvature and facilitates membrane fission. Dev. Cell 20: 206–218\\n\\nDoherty GJ & McMahon HT (2009) Mechanisms of endocytosis. Annu Rev Biochem 78: 857–902\\n\\nDi Fiore PP, Polo S & Hofmann K (2003) When ubiquitin meets ubiquitin receptors: a signalling connection. Nat Rev Mol Cell Biol 4: 491–497\\n\\nEscobedo A, Gomes T, Aragon E, Martin-Malpartida P, Ruiz L, & Macias MJ (2014) Structural basis of the activation and degradation mechaisms of the E3 ubiquitin lagase Nedd4L. Structure 22: 1446–1457'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Firsov D, Schild L, Gautschi I, Mérillat AM, Schneeberger E & Rossier BC (1996) Cell surface expression of the epithelial Na channel and a mutant causing Liddle syndrome: a quantitative approach. Proc Natl Acad Sci U S A 93: 15370–15375\\n\\nFrost A, Perera R, Roux A, Spasov K, Destaing O, Egelman EH, De Camilli P & Unger VM (2008) Structural basis of membrane invagination by F-BAR domains. Cell 132: 807–817\\n\\nHenne WM, Boucrot E, Meinecke M, Evergren E, Vallis Y, Mittal R & McMahon HT (2010) FCHo proteins are nucleators of clathrin-mediated endocytosis. Science 328: 1281–4'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Henne WM, Kent HM, Ford MGJ, Hegde BG, Daumke O, Butler PJG, Mittal R, Langen R, Evans PR & McMahon HT (2007) Structure and Analysis of FCHo2 F-BAR Domain: A Dimerizing and Membrane Recruitment Module that Effects Membrane Curvature. Structure 15: 839–852\\n\\nHershko A & Ciechanover A (1998) THE UBIQUITIN SYSTEM. Annu Rev Biochem 67: 425–479\\n\\nHughey RP, Mueller GM, Bruns JB, Kinlough CL, Poland PA, Harkleroad KL, Carattino MD & Kleyman TR (2003) Maturation of the epithelial Na+ channel involves proteolytic processing of the alpha- and gamma-subunits. J Biol Chem 278: 37073–37082\\n\\nHui E, Johnson CP, Yao J, Dunning FM & Chapman ER (2009) Synaptotagmin-mediated bending of the target membrane is a critical step in Ca(2+)-regulated fusion. Cell 138: 709–21'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Ingham RJ, Gish G & Pawson T (2004) The Nedd4 family of E3 ubiquitin ligases: functional diversity within a common modular architecture. Oncogene 23: 1972–84\\n\\nLaemmli UK (1970) Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature 227: 680–685\\n\\nLehmann M, Lukonin I, Noe F, Schmoranzer J, Clementi C, Loerke D & Haucke V (2019) Nanoscale coupling of endocytic pit growth and stability. Sci Adv 5: eaax5775\\n\\nLemmon M (2008) Membrane recognition by phospholipid-binding domains. Nat Rev Mol Cell Biol 9: 99–111\\n\\nLim KH, Huang H, Pralle A & Park S (2013) Stable, high-affinity streptavidin monomer for protein labeling and monovalent biotin detection. Biotechnol Bioeng 110: 57–67'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Martens S, Kozlov MM & McMahon HT (2007) How synaptotagmin promotes membrane fusion. Science 316: 1205–8\\n\\nMiyahara M, Nakanishi H, Takahashi K, Satoh-Horikawa K, Tachibana K & Takai Y (2000) Interaction of nectin with afadin is necessary for its clustering at cell-cell contact sites but not for its cis dimerization or trans interaction. J Biol Chem 275: 613–618\\n\\nNishizuka Y (1992) Intracellular signaling by hydrolysis of phospholipids and activation of protein kinase C. Science 258: 607–14\\n\\nPeter BJ, Kent HM, Mills IG, Vallis Y, Butler PJG, Evans PR & McMahon HT (2004) BAR domains as sensors of membrane curvature: the amphiphysin BAR structure. Science 303: 495–9'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Pickart CM & Raasi S (2005) Controlled synthesis of polyubiquitin chains. Methods Enzym 399: 21–36\\n\\nRoberts M, Barry S, Woods A, van der Sluijs P, & Norman J (2001) PDGF-regulated rab4-dependent recycling of avb3 integrin from early endosomes is necessary for cell adhesion and spreading. Curr Biol 11: 1392–1402\\n\\nRossier BC (2014) Epithelial sodium channel (ENaC) and the control of blood pressure. Curr Opin Pharmacol 15: 33–46\\n\\nRotin D & Kumar S (2009) Physiological functions of the HECT family of ubiquitin ligases. Nat Rev Mol Cell Biol 10: 398–409\\n\\nRotin D & Staub O (2011) Role of the ubiquitin system in regulating ion transport. Pflugers Arch 461: 1–21'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Saffarian S, Coccuci E, & Kirchausen T (2009) Distinct dynamics of endocytic clathrin-coated pits and coated palques. PLoS Biol 7: e1000191\\n\\nSakamoto Y, Kikuchi K, Umeda K & Nakanishi H (2017) Effects of various spacers between biotin and the phospholipid headgroup on immobilization and sedimentation of biotinylated phospholipid-containing liposomes facilitated by avidin-biotin interactions. J Biochem 162: 221–226\\n\\nShaner NC, Lin MZ, McKeown MR, Steinbach PA, Hazelwood KL, Davidson MW & Tsien RY (2008) Improving the photostability of bright monomeric orange and red fluorescent proteins. Nat Methods 5: 545–551'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Shimada A, Niwa H, Tsujita K, Suetsugu S, Nitta K, Hanawa-Suetsugu K, Akasaka R, Nishino Y, Toyama M, Chen L, et al (2007) Curved EFC/F-BAR-domain dimers are joined end to end into a filament for membrane invagination in endocytosis. Cell 129: 761–772\\n\\nSugahara T, Koga T, Ueno-Shuto K, Shuto T, Watanabe E, Maekawa A, Kitamura K, Tomita K, Mizuno A, Sato T, et al (2009) Calreticulin positively regulates the expression and function of epithelial sodium channel. Exp Cell Res 315: 3294–3300\\n\\nTakano K, Toyooka K & Suetsugu S (2008) EFC/F-BAR proteins and the N-WASP–WIP complex induce membrane curvature-dependent actin polymerization. EMBO J 27: 2817–2828'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Takei K, Slepnev VI, Haucke V & De Camilli P (1999) Functional partnership between amphiphysin and dynamin in clathrin-mediated endocytosis. Nat Cell Biol 1: 33–39\\n\\nTraub LM (2009) Tickets to ride: selecting cargo for clathrin-regulated internalization. Nat Rev Mol Cell Biol 10: 583–96\\n\\nTsujita K, Suetsugu S, Sasaki N, Furutani M, Oikawa T & Takenawa T (2006) Coordination between the actin cytoskeleton and membrane deformation by a novel membrane tubulation domain of PCH proteins is involved in endocytosis. J Cell Biol 172: 269–279\\n\\nUezu A, Umeda K, Tsujita K, Suetsugu S, Takenawa T & Nakanishi H (2011) Characterization of the EFC/F-BAR domain protein, FCHO2. Genes Cells 16: 868–878'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Umeda K, Ikenouchi J, Katahira-Tayama S, Furuse K, Sasaki H, Nakayama M, Matsui T, Tsukita S, Furuse M & Tsukita S (2006) ZO-1 and ZO-2 independently determine where claudins are polymerized in tight-junction strand formation. Cell 126: 741–754\\n\\nWang J, Peng Q, Lin Q, Childress C, Carey D & Yang W (2010) Calcium activates Nedd4 E3 ubiquitin ligases by releasing the C2 domain-mediated auto-inhibition. J Biol Chem 285: 12279–12288\\n\\nWiesner S, Ogunjimi AA, Wang H-R, Rotin D, Sicheri F, Wrana JL & Forman-Kay JD (2007) Autoinhibition of the HECT-type ubiquitin ligase Smurf2 through its C2 domain. Cell 130: 651–862'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content=\"Yang B & Kumar S (2010) Nedd4 and Nedd4-2: closely related ubiquitin-protein ligases with distinct physiological functions. Cell Death Differ 17: 68–77\\n\\nYin Y, Arkhipov A & Schulten K (2009) Simulations of Membrane Tubulation by Lattices of Amphiphysin N-BAR Domains. Structure 17: 882–892\\n\\nZaccai N, Kadlecova Z, Dickson VK, Korobchevskaya K & Traub L (2022) FCHO control AP2's initiating role in endocytosis through a Ptd-Ins(4,5)P2-dependent switch. Sci Adv 8: eabn2018\\n\\nZhou R, Patel S V. & Snyder PM (2007) Nedd4-2 catalyzes ubiquitination and degradation of cell surface ENaC. J Biol Chem 282: 20207–20212\"), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Zhu K, Shan Z, Chen X, Cai Y, Cui L, Yao W, Wang Z, Shi P, Tian C, Lou J, et al (2017) Allosteric auto‐inhibition and activation of the Nedd4 family E3 ligase Itch. EMBO Rep 18: 1618–1630\\n\\n\\n\\n\\n\\n\\n\\n\\nFigure legends\\n\\n\\n\\nFigure 1. FCHO2 is required for Nedd4L-mediated ubiquitination and endocytosis of ENaC.\\n\\nA. Knockdown of FCHO2 or Nedd4L by siRNA. αβγENaC-HeLa cells treated with each siRNA were analyzed by immunoblotting (IB).'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='B. Inhibition of ENaC endocytosis by knockdown of FCHO2 or Nedd4L. After cells were transfected with each siRNA, cell-surface ENaC was labeled by incubating at 4°C with anti-FLAG antibody. Endocytosis was started by incubating cells at 37°C for the indicated periods of time and stopped by chilling cells on ice. Antibody bound to the cell surface but not internalized was removed by acid stripping. After cells were solubilized, the antibody-labeled internalized ENaC subunits were precipitated and analyzed by IB (upper panel) and quantified (bottom and right panels). Internalization was expressed as a percentage of the initial amount of cell-surface αENaC determined by lysing cells without incubation at 37°C or acid stripping. Data are shown as the mean ±'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content=\"at 37°C or acid stripping. Data are shown as the mean ± S.E.M. of three independent experiments. **P < 0.01; ***P < 0.001 (Student's t-test).\"), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content=\"C. Inhibition of transferrin receptor (TfR) endocytosis by FCHO2 knockdown. After cells were transfected with each siRNA, cell-surface proteins were biotinylated at 4°C. Cells were then incubated at 37°C for 10 min. Biotin was removed from the cell surface, and internalized biotinylated proteins were precipitated with avidin beads. Samples were subjected to IB with anti-TfR antibody and quantified. Internalization was expressed as a proportion of the initial amount of TfR on the cell surface. Data are shown as the mean ± S.E.M. of three independent experiments. *P < 0.05 (Student's t-test).\"), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='D. Restoration of αENaC endocytosis by FCHO2 re-expression in knockdown cells. After cells were transfected with the indicated siRNA and GFP construct, cell-surface αENaC was labeled with anti-FLAG antibody at 4°C. Cells were then incubated at 37°C for 10 min. After acid stripping, cells were subjected to immunofluorescence microscopy (left panels) and quantified (right panel). The antibody-labeled, internalized αENaC was visualized with a fluorescence-conjugated secondary antibody. The border of each cell is delineated by a solid line. Arrows indicate cells expressing GFP-tagged proteins. Scale bars, 10 μm. The proportion of cells with internalized αENaC among cells expressing control GFP or GFP-siRNA-resistant (sr) -FCHO2 is shown. Data are shown as the'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='GFP-siRNA-resistant (sr) -FCHO2 is shown. Data are shown as the mean ± S.E.M. of five independent images (31-71 cells expressing GFP-tagged proteins per image). ***P < 0.001 (two-way analysis of variance with post hoc test).'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content=\"E, F. Effects of FCHO2, Nedd4L, and FBP17 knockdown on ENaC ubiquitination and expression at the cell surface. After cells were transfected with each siRNA, cell-surface αENaC was biotinylated and sequentially precipitated with anti-FLAG antibody and then avidin beads. Samples were subjected to IB with anti-FLAG and anti-Ub antibodies (E) and quantified (F). E, ENaC ubiquitination, Asterisks indicate the same band of ubiquitinated FLAG-αENaC. F, ENaC expression at the cell surface. Data are shown as the mean ± S.E.M. of three independent experiments. *P < 0.05 (Student's t-test).\\n\\n\\n\\nFigure 2. Localization of cell-surface ENaC, FCHO2, and Nedd4L at clathrin-coated structures.\"), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='A, B. Localization of cell-surface ENaC. mTagRFP-clathrin light chain (CLC) (A) or Nedd4L C922A (catalytic inactive mutant)-GFP (B) was expressed in αβγENaC-HeLa cells. Cell-surface αENaC was labeled with anti-FLAG antibody at 4°C. Cells were then fixed and stained with anti-FCHO2 antibody. Samples were subjected to immunofluorescence microscopy (left panel). Insets represent enlarged images of the dashed boxes. Scale bar, 10 μm. Co-localization was analyzed (right panel). The mean values of Pearson’s correlation coefficient from three cells and the S.E.M. are shown. A. Co-localization of cell-surface ENaC with FCHO2 at clathrin-coated structures. B. Co-localization of cell-surface ENaC with Nedd4L C922A mutant and FCHO2.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='C. Co-localization of Nedd4L with FCHO2 at clathrin-coated structures. Nedd4L-GFP and mTagRFP-CLC were co-expressed in wild-type HeLa cells. Cells were then fixed and stained with anti-FCHO2 antibody (left panel). Insets represent enlarged images of the dashed boxes. Scale bar, 10 μm. Co-localization was analyzed (right panel). The mean values of Pearson’s correlation coefficient from three cells and the S.E.M. are shown.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Figure 3. Recruitment of Nedd4L to CCPs. DsRed-CLC were co-expressed with Nedd4L-GFP in wild-type HeLa cells. Images were acquired at 1 s intervals through TIRF microscopy. Upper panel, TIRF images of HeLa cells. Insets represent enlarged images of the dashed boxes. Bottom panel, selected snapshots (every 10 sec) from a time series were obtained from a single CCP indicated by the arrowhead. The time point at which Nedd4L started to accumulate was set 0. Scale bar, 2 μm. \\n\\n\\t\\n\\nFigure 4. Recruitment of Nedd4L to membrane tubules generated by FCHO2 in cells.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='GFP-BAR domains were co-expressed with Myc-Nedd4L proteins in COS7 cells (A) and HEK293 cells (B). Cells were serum-starved and subjected to immunofluorescence microscopy. Insets represent enlarged images of the dashed boxes. Scale bar, 10 μm. \\n\\n\\n\\nFigure 5. Activation of Nedd4L by membrane tubules generated by FCHO2 in cells. \\n\\nA–E. Nedd4L autoubiquitination. An in vivo ubiquitination assay was performed with various BAR domains and FCHO2 mutants (A, B) and Nedd4L mutants (D). GFP-BAR domains were co-expressed with Myc-Nedd4L proteins and HA-tagged Ub in HEK293 cells. Cell lysates were subjected to immunoprecipitation (IP) and analyzed by IB (A, B, D). WT, wild type. Cells were also subjected to immunofluorescence microscopy (C, E). Scale bar, 10 μm.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='F. αENaC ubiquitination. An in vivo ubiquitination assay was performed with various concentrations of Myc-Nedd4L using FALAG-αENaC as a substrate in the presence or absence of GFP-FCHO2 BAR domain. Cell lysates were subjected to IP. Samples were analyzed by IB.\\n\\n\\n\\nFigure 6. The Nedd4L C2 domain is a Ca2+-dependent PS binding domain.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='A co-sedimentation assay was performed with the C2 domain or full-length Nedd4L using brain-lipid liposomes (~50% PS) at various Ca2+ concentrations (A) and with the C2 domain using synthetic liposomes containing various percentages of PS at 0.3 μM or 1.0 mM Ca2+ (B). The supernatants (S) and pellets (P) were subjected to sodium dodecyl sulfate (SDS)-polyacrylamide gel electrophoresis (PAGE), followed by Coomassie brilliant blue (CBB) staining (upper panels). Bottom panels, quantitative analysis. (−) for [Ca2+] indicates EGTA alone. Data are shown as the mean ± S.E.M. of three independent experiments.\\n\\nA. Ca2+ dependency.\\n\\nB. PS dependency.\\n\\n\\n\\nFigure 7. Preference of Nedd4L for FCHO2-generated membrane curvature.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='A. Distribution of liposome diameters. Brain-lipid liposomes (~50% PS) of different diameters were prepared by extrusion through the indicated pore sizes and sonication. Left panel, electron microscopic images. Scale bars, 200 nm. Right panel, distribution of liposome diameters shown by boxplots (number of observations per pore size = 25). The center line inside the box corresponds to the median, the bounds of the box encompass data points between the first and third quartiles, and the whiskers extend to the minimum and maximum values including outliers. So, sonication.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='B-D. Sensing and generation of membrane curvature by Nedd4L. Co-sedimentation (B) and in vitro tubulation (C, D) assays were performed using brain-lipid liposomes (~50% PS) at 0.1 μM (B) and 0.3 μM Ca2+ (C, D). B, Preference of Nedd4L for a specific degree of membrane curvature. Liposomes were supplemented with X-biotin-PE and rhodamine-PE. Full-length Nedd4L was incubated with liposomes of different sizes, which were prepared by extrusion through the indicated pore sizes. The sample was then incubated with avidin beads and ultracentrifuged. The proportion of precipitated liposomes of each size was calculated to be ~100% by measuring both total and supernatant fluorescence. The supernatants (S) and pellets (P) were subjected to SDS- PAGE, followed by CBB'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='and pellets (P) were subjected to SDS- PAGE, followed by CBB staining (upper panel). Bottom panel, quantitative analysis. So, sonication. Data are shown as the mean ± S.E.M. of five independent experiments. *P < 0.05; **P < 0.01 (one-way analysis of variance with Tukey’s post hoc test). C, Curvature generation by Nedd4L. The assay was performed with the Nedd4L C2 domain. Samples were examined by electron microscopy. Scale bar, 100 nm. D, The degree of membrane curvature generated by Nedd4L is consistent with that generated by FCHO2. The assay was performed with the indicated protein. Left panel, electron microscopic image. Scale bar, 100 nm. Right panel, distribution of tubule diameters shown by boxplots (number of observations per protein = 14–33). The'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='by boxplots (number of observations per protein = 14–33). The center line inside the box corresponds to the median, the bounds of the box encompass data points between the first and third quartiles, and the whiskers extend to the minimum and maximum values including outliers.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Figure 8. Conserved hydrophobic residues of Nedd4L are responsible for sensing and generation of membrane curvature.\\n\\nA. Sequence alignment of C2 domains. Conserved residues are highlighted with the following color code: yellow, aspartates for Ca2+ binding; blue, very hydrophobic residues probably for penetration into the lipid bilayer; and brown, others. Dots show the mutation sites. Hs, Homo sapiens; Sc, Saccharomyces cerevisiae; and Rt, Rattus norvegicus. \\n\\nB. Inability of Nedd4L C2 domain mutants to sense membrane curvature in cells. GFP-FCHO2 BAR domain and Myc-Nedd4L C2 domain mutants were co-expressed in COS7 cells. Insets are enlarged images of dashed boxes. Scale bar, 10 μm.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='C, D. Inability of Nedd4L C2 domain mutants to bind to membranes and to generate membrane curvature in vitro. Co-sedimentation (C) and in vitro tubulation (D) assays were performed using brain-lipid liposomes (~50% PS) at 0.3 μM Ca2+. C, Membrane binding. The assay was performed with Nedd4L C2 domain mutants using various concentrations of liposomes. The supernatants (S) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining (upper panel). Bottom panel, quantitative analysis. Data are shown as the mean ± S.E.M. of three independent experiments. D, Curvature generation. The assay was performed with Nedd4L C2 domain mutants. Samples were examined by fluorescence microscopy. Scale bar, 10 μm.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Figure 9. Properties of the catalytic activity and membrane binding of Nedd4L. \\n\\nLiposomes supplemented with biotinylated phospholipids were incubated with mSA-αENaC (A, B). Samples were mixed with Nedd4L. Mixtures were subjected to ubiquitination (C-E) and co-sedimentation (F) assays. These assays were performed with brain-lipid liposomes (~50% PS) at various Ca2+ concentrations (C), with synthetic liposomes (various percentages of PS) at 0.7 μM Ca2+ (D, F), and with brain-lipid liposomes (20% PS) at 0.7 μM Ca2+ (E). Ubiquitination and liposome binding were quantified by scanning and expressed as ratios to maximum levels. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='A, B. Schematic diagrams of mSA-αENaC. A, Structure. B. Binding to biotinylated liposomes.\\n\\nC. Ca2+-dependent Nedd4L activity. Ubiquitination was detected by IB (upper panel) and the intensity was quantified by scanning (bottom panel). Data are shown as the mean ± S.E.M. of three independent experiments.\\n\\nD, Membrane localization of mSA-αENaC enhances the PS-dependent catalytic activity of Nedd4L. Assays were performed in the presence or absence of 40 μM biotin. Ubiquitination was detected by IB (left panel). The intensities were quantified by scanning (right panel). Data are shown as the mean ± S.E.M. of three independent experiments.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='E. Nedd4L activity is predominantly stimulated by the specific degree of membrane curvature. The assay was performed using liposomes of different sizes which were prepared by extrusion through the indicated pore sizes. Ubiquitination was detected by IB (left panel) and quantified (right panel). Data are shown as the mean ± S.E.M. of three independent experiments. ***P < 0.001 (one-way analysis of variance with Tukey’s post hoc test).'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='F. Membrane localization of mSA-αENaC enhances the PS-dependent membrane binding of Nedd4L. Assays were performed in the presence or absence of 40 μM biotin. Liposome binding was detected by IB (left panel). The intensities were quantified by scanning (right panel). T, total sample. P, pellet. Data are shown as the mean ± S.E.M. of three independent experiments.\\n\\n\\n\\n\\n\\nFigure 10. Recruitment and activation of Nedd4L by FCHO2-generated membrane curvature in vitro.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='mSA-αENaC-associated brain-lipid liposomes (20% PS) were mixed with Nedd4L in the presence or absence of BAR domains. Mixtures were subjected to co-sedimentation (A) and ubiquitination (B-E) assays. These assays were performed at 0.7 μM Ca2+ with brain-lipid liposomes (20%PS) (A, B, D, and E) and with the indicated (20% PS or ~50% PS) brain-lipid liposomes (C). Ubiquitination and liposome binding were quantified by scanning and expressed as ratios to maximum levels. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='A. FCHO2-induced recruitment of Nedd4L. The assay was performed with the FCHO2 BAR domain (1.4 μM, 50 μg/ml) in the presence or absence of biotin. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining (FCHO2 BAR) and IB (Nedd4L and mSA-αENaC).\\n\\nB, C. FCHO2-induced activation of Nedd4L. B, Effects of various doses of the FCHO2 BAR domain. The assay was performed with various concentrations of the FCHO2 BAR domain. Ubiquitination was detected by IB (left panel) and the intensity was quantified by scanning (right panel). C, Effects of the PS percentage in liposomes. The assay was performed with the FCHO2 BAR domain (1.4 μM). Samples were analyzed by IB. Data are shown as the mean ± S.E.M. of three independent experiments.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='D, E. Specificity of BAR domains and effect of an FCHO2 mutant. The assay was performed with various BAR domains (D) and an FCHO2 mutant (E) (1.4 μM each). Samples were analyzed by IB.\\n\\n\\n\\nFigure 11. Effects of PI(4,5)P2 on FCHO2-induced recruitment and activation of Nedd4L.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='A, B. Binding to PI(4,5)P2. A co-sedimentation assay was performed at 0.7 μM Ca2+ with the FCHO2 BAR or Nedd4L C2 domain using mSA-αENaC-associated synthetic liposomes containing 20% PS, 5% PI(4,5)P2 (A), or various percentages of PI(4,5)P2 (B). The supernatants (S) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining (A and upper panel in B). Bottom panel in B, quantitative analysis. Control, synthetic liposomes [0% PS and 0% PI(4,5)P2]. Data are shown as the mean ± S.E.M. of three independent experiments.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='C, D. Effects on FCHO2-induced recruitment and activation of Nedd4L. Co-sedimentation (C) and ubiquitination (D) assays were performed at 0.7 μM Ca2+ with mSA-αENaC-associated synthetic liposomes containing 20% PS or 5% PI(4,5)P2 in the presence or absence of the FCHO2 BAR domain. C, Nedd4L recruitment. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by IB with anti-Nedd4L antibody (upper panel) and CBB staining (lower panel). D, Nedd4L activation. Samples were analyzed by IB. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Figure 12. Effects of liposome size on FCHO2-induced activation of Nedd4L. Co-sedimentation (A) and ubiquitination (B) assays were performed at 0.7 μM Ca2+ with the FCHO2 BAR domain using brain-lipid liposomes (20% PS, 0.8 µm or 0.05 µm pore-size) that were associated with mSA-αENaC.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='A. Binding of the FCHO2 BAR domain to 0.8 µm and 0.05 µm pore-size liposomes. The co-sedimentation assay was performed using synthetic (0% PS) or brain-lipid liposomes (20% PS) containing rhodamine-PE and fluorescein-PE. Liposomes were precipitated with anti-fluorescein antibody. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by CBB staining. The proportion of precipitated liposomes of each size was calculated to be ~100% by measuring both total and supernatant fluorescence.\\n\\nB. Effects of liposome size on FCHO2-induced activation of Nedd4L. Samples were analyzed by IB. Asterisks indicate the same band of mono-ubiquitinated mSA-αENaC.\\n\\n\\n\\nFigure 13. Model of ENaC endocytosis.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Figure 13. Model of ENaC endocytosis. \\n\\nFCHO2 binds to the plasma membrane and generates a specific degree of membrane curvature according to the intrinsic curvature of its BAR domain. Nedd4L exists in a catalytically autoinhibited state due to an intramolecular interaction between the N-terminal C2 and C-terminal HECT domains. Nedd4L is recruited to FCHO2-generated membrane curvature at the rim of nascent CCPs that ENaC enters. Membrane binding of the C2 domain causes catalytic activation by relieving autoinhibition. ENaC is ubiquitinated and captured by adaptor proteins with Ub-interacting motifs, such as Eps15. \\n\\n\\n\\nExtended View Figure Legends\\n\\n\\n\\nFigure EV1. Expression of ENaC subunits in αβγENaC-HeLa cells.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='A, B. Doxycycline (Dox)-induced expression of αENaC. αβγENaC-HeLa cells were cultured overnight in the presence or absence of Dox. The lysates of parental HeLa and αβγENaC-HeLa cells were subjected to IB. A, 7.5% gel. B, Gradient gel (5-20%). Anti-γENaC antibody cross-reacted with two bands of endogenous proteins in parental HeLa cells, the lower band of which overlapped with γENaC (arrow) in αβγENaC-HeLa cells. Upon Dox-induced αENaC expression, an additional 70-kDa γENaC band (asterisk) was detected. It has been shown that co-expression of all three subunits induces ENaC maturation, including proteolytic cleavage of α- and γENaC (Hughey et al, 2003). The 70-kDa γENaC band is likely a cleavage product comprising the C-terminal region. The 20-kDa αENaC band'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='product comprising the C-terminal region. The 20-kDa αENaC band (arrowhead) is likely a cleavage product comprising the N-terminal region.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content=\"C. Association of α-, β-, and γENaC. When αENaC was immunoprecipitated with anti-FLAG antibody from αβγENaC-HeLa cells treated with Dox, β- and γENaC were co-precipitated. Asterisk, 70-kDa γENaC band. \\n\\n\\n\\nFigure EV2. Knockdown of FBP17 by siRNA. \\n\\nαβγENaC-HeLa cells were treated with each siRNA, and FBP17 mRNA levels were quantified using real-time PCR. The expression of FBP17 was normalized to GAPDH mRNA levels. Data are shown as the mean ± S.E.M. of three independent experiments. ***P < 0.001 (Student's t-test). \\n\\n\\n\\nFigure EV3. Inability of FCHO2 mutants to generate membrane tubules. \\n\\nGFP-FCHO2 BAR domain [wild type (WT) or mutant] was expressed in COS7 cells. Cells were then subjected to immunofluorescence microscopy. Scale bar, 10 μm.\"), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Figure EV4. Inability of Nedd4L mutants to ubiquitinate αENaC. An in vivo ubiquitination assay was performed with various Nedd4L constructs (each 0.5 µg) using FLAG-αENaC as a substrate in the presence or absence of GFP-FCHO2 BAR domain. Cell lysates were subjected to IP. Samples were analyzed by IB.\\n\\n\\n\\nFigure EV5. Inhibition of the liposome binding of Nedd4L by the FCHO2 BAR domain.\\n\\nA co-sedimentation assay was performed at 0.7 μM Ca2+ with control liposomes (0% PS) or brain-lipid liposomes (~50% PS) in the presence or absence of the FCHO2 BAR domain. The total sample (T) and pellets (P) were subjected to SDS-PAGE, followed by IB (upper panel) and CBB staining (lower panel).'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='Figure EV6. Membrane binding and curvature generation of BAR domains and an FCHO2 mutant. Co-sedimentation (A) and in vitro tubulation (B) assays were performed at 0.7 μM Ca2+ with the indicated BAR domains using brain-lipid liposomes (20% PS) that were associated with mSA-αENaC. \\n\\nA. Membrane binding. The supernatants (S) and pellets (P) were subjected to SDS-PAGE followed by CBB staining.'), Document(metadata={'source': './data/docx/EMBOJ-2023-114687.docx'}, page_content='B. Curvature generation. Left panel, electron microscopic image. Scale bar, 100 nm. Right panel, distribution of tubule diameters shown by boxplots (number of observations per protein = 22–26). The center line inside the box corresponds to the median, the bounds of the box encompass data points between the first and third quartiles, and the whiskers extend to the minimum and maximum values including outliers.\\n\\n\\t\\t8')]\n",
      "Processing document: ./data/docx/EMBOJ-2023-115257.docx\n",
      "Split documents: [Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='LINE-1 RNA triggers matrix formation in bone cells via a PKR-mediated inflammatory response\\n\\nArianna Mangiavacchi,1@ Gabriele Morelli,1 Sjur Reppe,2,3,4 Alfonso Saera-Vila,5 Peng Liu,1 Benjamin Eggerschwiler,6,7 Huoming Zhang,8 Dalila Bensaddek,8 Elisa A. Casanova,5 Carolina Medina Gomez,9 Vid Prijatelj,9 Francesco Della Valle,1 Nazerke Atinbayeva,1 Juan Carlos Izpisua Belmonte,10 Fernando Rivadeneira,9 Paolo Cinelli,5,11 Kaare Morten Gautvik,3* Valerio Orlando.1*@\\n\\n\\n\\n1 King Abdullah University of Science and Technology (KAUST), Biological Environmental Science and Engineering Division, Thuwal 23500-6900, Kingdom of Saudi Arabia\\u200b\\n\\n2 Oslo University Hospital, Department of Medical Biochemistry, Oslo, Norway.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content=\"3Lovisenberg Diaconal Hospital, Unger-Vetlesen Institute, Oslo, Norway.\\n\\n4Oslo University Hospital, Department of Plastic and Reconstructive Surgery, Oslo, Norway.\\n\\n5 Sequentia Biotech, Carrer Comte D'Urgell 240, Barcelona, 08036, Spain.\\n\\n6 Department of Trauma, University Hospital Zurich, Sternwartstrasse 14, 8091, Zurich, Switzerland\\n\\n7 Life Science Zurich Graduate School, University of Zurich, Winterthurerstrasse 190, 8057, Zurich, Switzerland.\\n\\n8 Core Labs, King Abdullah University of Science and Technology (KAUST), Thuwal 23500-6900, Kingdom of Saudi Arabia\\u200b.\\n\\n9 Department of Internal Medicine, Erasmus Medical Centre, Rotterdam, the Netherlands.\\n\\n10 Altos Labs, San Diego, CA, USA\"), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='10 Altos Labs, San Diego, CA, USA\\n\\n11 Center for Applied Biotechnology and Molecular Medicine, University of Zurich, Winterthurerstrasse 190, 8057, Zurich, Switzerland.\\n\\n\\n\\n*senior author\\n\\n@corresponding author\\n\\n\\n\\nAbstract'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Transposable elements (TEs) are mobile genetic modules of viral derivation that have been co-opted to become modulators of mammalian gene expression. TEs are a major source of endogenous dsRNAs, signaling molecules able to coordinate inflammatory responses in various physiological processes. Here, we provide evidence for a positive involvement of TEs in inflammation-driven bone repair and mineralization. In newly fractured mice bone, we observed an early transient upregulation of repeats occurring concurrently with the initiation of the inflammatory stage. In human bone biopsies, analysis revealed a significant correlation between repeats expression, mechanical stress and bone mineral density. We investigated a potential link between LINE-1 (L1) expression'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='We investigated a potential link between LINE-1 (L1) expression and bone mineralization by delivering a synthetic L1 RNA to osteoporotic patient-derived mesenchymal stem cells and observed a dsRNA-triggered protein kinase (PKR)-mediated stress response that led to strongly increased mineralization. This response was associated with a strong and transient inflammation, accompanied by a global translation attenuation induced by eIF2a phosphorylation. We demonstrated that L1 transfection reshaped the secretory profile of osteoblasts, triggering a paracrine activity that stimulated the mineralization of recipient cells.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Keywords: dsRNA/inflammation/osteoblast /PKR/transposable elements\\n\\n\\n\\nIntroduction'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Repetitive elements are, in origin, parasitic transposable elements (TEs) of viral derivation and during evolution have become regulatory modules of gene expression, eventually integrated into developmental, resilience, and cell identity programs.(Cosby et al, 2019) (Mangiavacchi et al, 2021) (Jachowicz et al, 2017) Although cells have evolved several defense mechanisms to prevent deleterious uncontrolled retrotransposon reactivation,(Slotkin & Martienssen, 2007) increasing evidence demonstrates that retrotransposons are also involved in non-pathological contexts, particularly as non-coding RNAs.(Mangiavacchi et al, 2021) As part of resilience mechanisms, all retrotransposons are a major source of endogenous double-stranded RNAs (dsRNAs), that serve as a'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='of endogenous double-stranded RNAs (dsRNAs), that serve as a cellular signaling molecule to coordinate inflammatory and immune responses in various physiological processes.(Fukuda et al, 2021) (De Cecco et al, 2019a) (Simon et al, 2019) (Sadeq et al, 2021) (Chen & Hur, 2022) (Zhang et al, 2020) The transient and controlled breaching of the activation threshold of dsRNA sensors allows the integration of innate immune functions within the frame of physiological processes.(Chen & Hur, 2022) For example, repetitive elements transcribed during development drive RIG-I-like receptors (RLR)-mediated inflammatory signals that regulate hematopoietic stem and progenitor cells (HSPC) formation.(Lefkopoulos et al, 2020) During cell replication, dsRNA-mediated activation'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='et al, 2020) During cell replication, dsRNA-mediated activation of PKR ensures proper progression of mitosis.(Kim et al, 2014) Moreover, epigenetically mediated derepression of repetitive elements leads to dsRNA production and to the activation of an inflammatory response stimulating anti-tumor T cell immunity.(Sheng et al, 2018) Furthermore, after skin injury, dsRNAs induce inflammatory pathways contributing to wound healing and hair regeneration.(Nelson et al, 2015) Several studies have demonstrated a positive effect of proinflammatory mediators on bone anabolism in vitro and in vivo,(Croes et al, 2015) (Croes et al, 2016) (Li et al, 2016) (Croes et al, 2017) (Laroche et al, 2011) suggesting that an inflammatory reaction may be harnessed for bone'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='that an inflammatory reaction may be harnessed for bone regenerative purposes. Indeed, sterile inflammation is a fundamental component of the osteogenic microenvironment: the inflammatory reaction is the earliest response to injury and is crucial to initiate and orchestrate fracture repair and activate bone anabolic processes. (Mountziaris et al, 2011) (Marsell & Einhorn, 2011) (Bahney et al, 2019) Accordingly, a dysregulated inflammation negatively impacts optimal bone regeneration, (Wheatley et al, 2019) (Gerstenfeld et al, 2001) (Recknagel et al, 2013) (Hurtgen et al, 2016) (Jiao et al, 2015) and often precedes the excessive bone formation associated with heterotopic ossification and several vertebral column pathological conditions. (Lories & Schett,'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='vertebral column pathological conditions. (Lories & Schett, 2012) (Balboni et al, 2006) Therefore, identifying the key molecular factors triggering the sterile inflammation that precedes regeneration in bone would allow us to modulate the inflammatory reaction in a spatiotemporal and contextual manner as well as to develop new anabolic strategies for the treatment of bone loss conditions, such as osteoporosis, or impaired bone repair.(Roberts & Ke, 2018)'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Here, we show that 1) In mice, immediately after bone fracture, a transient upregulation of TEs concurs with the initiation of the inflammatory stage; 2) in humans, an increased expression of repeats is observed in bones with a higher mechanical stress-induced anabolic demand and expression of TEs correlates with local mineral density; 3) The delivery of L1 RNA, but not control RNA, to human bone marrow-derived mesenchymal stem cells committed to osteoblasts stimulates a unique mineralizing phenotype in a dose-dependent manner; 4) L1-treated osteoblasts show upregulation of inflammatory genes and transcriptional changes characteristic for the earliest stage of bone repair; 5) cytoplasmic L1 RNA is sensed by dsRNA-activated protein kinase (PKR), which'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='L1 RNA is sensed by dsRNA-activated protein kinase (PKR), which mediates eIF2α phosphorylation and a global attenuation of protein synthesis; 6) the inhibition of PKR prevents inflammation, inhibits translation and induces mineralization caused by elevated levels of cytoplasmic L1 RNA; 7) cells transfected with L1 RNA undergo significant changes in their secretome composition and initiate a paracrine effect stimulating the mineralization of osteogenic competent recipient cells.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Our results indicate that TEs are induced by bone damage and stimulate bone mineralization through an inflammatory response triggered by dsRNA sensing and mediated by a paracrine activity.\\n\\n\\n\\nResults\\n\\nRetrotransposon expression is triggered by fracture in vivo and correlates with bone mineral density in human weight-bearing bones'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Inflammation is the earliest response in fracture healing. To investigate a possible involvement of TEs in the sterile inflammatory response triggered by bone injury, we analyzed TEs expression in time-course RNA-Seq repository data obtained from a full-fracture bone healing mice model.(Coates et al, 2019) We focused on TEs expression dynamics immediately after fracture (acute inflammatory stage) and observed four clusters of differentially expressed TEs (logFC > 0.5) between intact and post-fracture bone after 4h, a time point corresponding to the initiation of acute inflammation (Figure 1A). Cluster 1 includes TEs subfamilies whose expression slightly increases after fracture and during the whole healing process. Clusters 2 and 4 include TEs subfamilies'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='whole healing process. Clusters 2 and 4 include TEs subfamilies that are downregulated after fracture but show increased expression at later time points. The major group, cluster 3, is represented by TEs subfamilies whose expression is highly and transiently upregulated after fracture (Figure 1A and 1B). Interestingly, the induction of these TEs, mostly Long Interspersed Nuclear Elements (LINEs) and Long Terminal Repeats (LTRs) (Figure 1B), is transient and limited to the earliest phase of the inflammatory stage necessary and sufficient to initiate and orchestrate fracture repair and activate bone anabolic processes.(Mountziaris et al, 2011),(Marsell & Einhorn, 2011) These results suggest that, in bone, retrotransposon reactivation is an early event in'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='in bone, retrotransposon reactivation is an early event in response to fracture, possibly linked to inflammation initiation which is fundamental for a healthy healing. Indeed, retrotransposon involvement in posttraumatic tissue regeneration was previously found in other organisms.(Mashanov et al, 2012),(Zhu et al, 2012)'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='To assess gene expression dynamics immediately after fracture in humans represents a technical and ethical hurdle. A comparison between bones experiencing different degrees of mechanical loading may represent an alternative model to study stress-induced bone anabolism in humans. Mechanically loaded bones like the femur, which are subject to recurring microfractures that need to be repaired, are indicated to be more metabolically active than less loaded bones, like the iliac crest, as pointed out by previous transcriptome studies.(Aerssens et al, 1997) (Varanasi et al, 2010)'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='We compared the expression of TEs in trabecular bone from ilium and femoral head, two different skeletal sites, the latter experiencing a higher degree of mechanical loading, anabolic demand and bone turnover.(Aerssens et al, 1997) Femur (n=48) and ilium biopsies (n=71) were isolated from a cohort of clinically well-characterized donors (refer to cohort description and Appendix Table S1 and S2). Each cohort was divided into normal (BMD T-score >-1), osteopenic (−2.5 < BMD T-score ≤ −1), and osteoporotic (BMD T-score ≤ −2.5, with at least one fragility fracture). TEs show generally higher upregulation in healthy femurs (n=27) compared to healthy ilia (n=34) (Figure 1C and 1E). Each TEs order showed a high percentage of upregulated subfamilies, from 60% of'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='a high percentage of upregulated subfamilies, from 60% of DNA transposon to 75% of LINEs, and a low percentage of downregulated subfamilies, from 9% of LTRs to 3% of Short interspersed nuclear elements (SINEs) and LINEs, in femur compared to ilium (Figure 1D). These results suggested a positive correlation between TEs global expression and bone anabolic activity.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='To further test our hypothesis, we compared TEs expression in femurs from healthy donors (n=27), osteopenic (n=12) and osteoporotic patients (n=9) whose bone anabolism is increasingly compromised. As shown in Figure 2A, the global expression of TEs is reduced in patients. To further assess whether TEs expression is involved in local anabolism and associated with mineral density, we divided the femur cohort according to the DXA T-score measured in the femoral neck (FN). We divided the cohort into “high BMD” (FN T-score >-1) and “low BMD” (FN T-score <-1). Globally, TEs are upregulated in femurs with high bone mineral density (Figure 2B). In particular, more than 90% of differentially expressed TEs subfamilies are upregulated (Figure 2C). Interestingly,'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='TEs subfamilies are upregulated (Figure 2C). Interestingly, almost all SINE elements are unchanged between the two groups (Figure 2C). Finally, we assessed whether a positive correlation exists between TEs expression levels and local mineral density (FN T-score). We found that approximately 30% of L1 and LTR subfamilies expressed in the femoral bone are positively correlated with local BMD (Figure 2D). Altogether, these results strongly suggest a positive involvement of TEs in response to stress-induced anabolic demand and bone mineralization.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Increased cytoplasmic L1 repeat RNA stimulates osteoblast mineralization activity'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='The in vivo evidence suggested a positive involvement of retrotransposons in post-traumatic bone repair in mice, and a strong correlation between retrotransposon expression and bone mineral density in humans, and higher expression in loaded compared to unloaded bone. We then corroborated the results by studying the effects of increasing repeat RNA levels on differentiating osteoblasts derived from mesenchymal stem cells (MSCs) isolated from femurs of healthy donors. At day 5 of differentiation, a Cy5-conjugated full-length L1 RNA consensus sequence was transfected. Capping, 2’-O-Methylation of 5’ end, polyadenylation (200 adenosines), full substitution with 5-methylcytidine (m5C) and 75% substitution with pseudouridine were used to stabilize the RNA and to'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='with pseudouridine were used to stabilize the RNA and to bypass the intracellular innate immune system.(Koski et al, 2004) (Pardi et al, 2013) (Ludwig et al, 2010) (Karikó et al, 2005) (Nallagatla & Bevilacqua, 2008) (Karikó et al, 2008) (Anderson et al, 2010) (Kormann et al, 2011) Red fluorescent protein (RFP) mRNA, with the same modifications, was included as negative control. The exogenous L1 RNA accumulated in the cytoplasm (Figure 3A) and was gradually cleared by secretion along with matrix components (Figure 3B and 3C). The effect of L1 RNA delivery was tested in differentiating MSCs from two different and unrelated healthy donors. As shown, L1 RNA transfection markedly stimulated the production of mineralized matrix (Figure 3D). Osteoblast'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='the production of mineralized matrix (Figure 3D). Osteoblast mineralization increased dose-dependently with the concentration of the delivered L1 RNA, in agreement with saturation kinetics (Figure 3E). Moreover, the strongly enhanced mineralization was highly specific for L1 RNA, as demonstrated by the lack of significant changes in mineral deposition in cells transfected with negative control RNA, even at concentrations 100-200-fold higher (figure 3E). We also isolated MSCs from the femur of osteoporotic patients and selected those showing a markedly delayed and reduced in vitro production of bone matrix in contrast to those derived from healthy donors (Figure 3F). Strikingly, also in these cells with low osteogenic capacity, the delivery of L1 RNA'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='cells with low osteogenic capacity, the delivery of L1 RNA triggered a strong mineralization response despite their clearly compromised anabolic activity (Figure 3G). These results demonstrate convincingly that the ectopic delivery of L1 RNA stimulates the mineralization of bone-forming cells regardless of their prior intrinsic differentiation/anabolic potential.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Osteoporotic patient-derived cells transfected with L1 RNA show a higher expression of early differentiation markers, with a peak 48h after L1 transfection (day 7) (Figure 3H). Bone sialoprotein (IBSP), a late osteogenic gene, shows a similar profile, being almost 6 times more expressed in L1- than in RFP-transfected cells on day 7 (Appendix Figure S1A). Strikingly, although alkaline phosphatase (ALPL) expression slightly increases at later time points (Appendix Figure S1A), its activity is not enhanced in L1-treated cells (Appendix Figure S2A). However, the activity of ectonucleotide pyrophosphatase/phosphodiesterase 1 (ENPP1) is gradually and significantly reduced by L1 RNA transfection (Appendix Figure S2B). As ENPP1 is a major source of inorganic'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='(Appendix Figure S2B). As ENPP1 is a major source of inorganic pyrophosphate (PPi), one of the main physiological inhibitors of mineralization,(FLEISCH et al, 1966) (Johnson et al, 2000) (Hessle et al, 2002) the reduction of its activity may contribute to the enhanced mineralization observed in L1-RNA transfected cells.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Altogether, these data suggest that L1 RNA induces a unique mineralizing phenotype and apparently distinct from the well-known canonical differentiation mechanisms. \\n\\n\\n\\nL1 RNA delivery induces inflammatory pathways significantly overlapping those involved in bone fracture repair'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='To investigate the molecular response of osteoblasts to L1 RNA transfection, we profiled their transcriptome by Illumina RNA-Seq 24h post-transfection. We found 482 differentially expressed genes (DEG) (FDR<0.05) between L1 and RFP RNA-treated OB (268 upregulated and 214 downregulated) (Appendix Figure S3). Gene ontology (GO) enrichment analysis revealed that the early transcriptional signature of L1-treated osteoblasts is typical of that of an inflammatory response (Figure 4A and 4B). This is in accordance with the recent evidence of a link between L1 and other retrotransposons accumulation in the cytoplasm and inflammatory response in vitro and in vivo.(Fukuda et al, 2021) (De Cecco et al, 2019b) (Simon et al, 2019) GO term enrichment analysis comparison'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='(Simon et al, 2019) GO term enrichment analysis comparison between the early stage of fracture-induced bone healing in vivo(Coates et al, 2019) and differentiating osteoblasts in vitro 24h post-L1 delivery (Figure 4A) revealed a matching inflammatory response and shared upregulated pathways (Figure 4C) essential for effective bone repair.(Maruyama et al, 2020) (Gerstenfeld et al, 2003) (Kon et al, 2001) Recruitment of neutrophils at the site of injury also indicates a healthy repair process, particularly in bone.(Kovtun et al, 2016) Extracellular signal-regulated kinase 1 (ERK1) and 2 (ERK2), play crucial roles in bone formation(Xiao et al, 2000) (Xiao et al, 2002b) (Xiao et al, 2002a) (Matsushita et al, 2009) (Ge et al, 2007) and seem to be the most'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='et al, 2009) (Ge et al, 2007) and seem to be the most upstream initiators of tissue regeneration in planaria.(Owlarn et al, 2017) Upon bone injury, an initial transient stage of acute inflammation is a crucial factor in ensuring an effective regeneration(Maruyama et al, 2020) whereas an excessive/prolonged (chronic) inflammation is deleterious for the healing environment.(Chan et al, 2012) (Hurtgen et al, 2016) (Clark et al, 2017) (Brem & Tomic-Canic, 2007) (Timmen et al, 2014) (Schmidt-Bleek et al, 2012) (Weckbach et al, 2013) We followed the time-course expression of “inflammatory response” (GO:0006954) and “immune response” (GO:0006955) genes upregulated by L1 RNA immediately after transfection. As shown in Figure 4D, their expression is already strongly'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='As shown in Figure 4D, their expression is already strongly reduced at day 10 and almost entirely silenced by day 14, suggesting the transient dynamics of the inflammatory response triggered in vitro by L1 RNA. This is coherent with our previous observation that exogenous L1 RNA is gradually secreted by the cell together with matrix components (Figure 3B and C), eventually clearing out the initial inflammatory stimulus.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Cellular response to increased cytoplasmic level of L1 RNA is mediated by PKR'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='It has been previously demonstrated that the cytoplasmic accumulation of L1-derived cDNA triggers an inflammatory response mediated by the cyclic GMP-AMP synthase (cGAS)-stimulator of interferon genes (STING) cytosolic DNA sensing pathway.(De Cecco et al, 2019a) Immunofluorescence (IF) analysis reveals that the exogenous L1 RNA forms DNA:RNA hybrids (Appendix Figure S4), suggesting that L1 RNA is reversed transcribed into cDNA after its transfection. Notably, ORF1p was undetectable by WB in differentiating osteoblasts and in L1 transfected osteoblasts (Appendix Figure S5), indicating that the exogenous L1 RNA is not translated after transfection. To assess whether L1-induced inflammation was triggered by the sensing of its cDNA, we used Lamivudine 3TC and'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='by the sensing of its cDNA, we used Lamivudine 3TC and G140 (Figure 5A) to inhibit the ORF2-mediated reverse transcription of L1 RNA and cGAS activity, respectively. Moreover, we also performed a siRNA-mediated knockdown (KD) of cGAS (Appendix Figure S6). (Jones et al, 2008) (Lama et al, 2019) Surprisingly, none of the treatments impaired the mineralization induced by L1 RNA delivery (Figure 5A, Appendix Figure S6C), suggesting that the observed cellular response depends entirely on RNA sensing. As previously mentioned, repeats are a major source of endogenous dsRNA,(Sadeq et al, 2021) (Chen & Hur, 2022) and L1 RNA, in particular, is able to form intramolecular double-stranded structures.(Hur, 2019) As expected, we found a tight colocalization between L1'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='2019) As expected, we found a tight colocalization between L1 RNA and dsRNA IF signal (Figure 5B). One of the main cellular sensors of dsRNA, including those derived from retrotransposons, is PKR.(Kim et al, 2018) PKR activation results in the phosphorylation of eukaryotic translation initiation factor 2a (eIF2α) and in the subsequent inhibition of global protein synthesis and cell growth.(Donnelly et al, 2013) Interestingly, PKR has been shown to interact with L1 RNA in vitro.(Kim et al, 2018) Moreover, eIF2α phosphorylation has been demonstrated to promote autophagy in osteoblasts and to counteract BMD reduction in osteoporotic ovariectomized mice.(Li et al, 2019)'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='To demonstrate that L1 RNA is able to trigger a PKR-mediated intracellular response, we transfected L1 RNA in the presence or absence of the PKR inhibitor C16 and evaluated the levels of eIF2α phosphorylation by western blot (Figure 5C). The delivery of L1, but not RFP, strongly induces eIF2α phosphorylation and this effect is prevented by PKR inhibition (Figure 5C). Notably, eIF2α phosphorylation occurs already 6 hours post-transfection. This observation suggests that PKR activation is one of the first events induced by L1 RNA, thus corroborating the hypothesis of PKR as an endogenous L1 RNA sensor.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='We then characterized the proteome of cells 24h post-L1 RNA transfection by diaPASEF mass spectrometry (MS).(Meier et al, 2020a) We detected almost 8000 proteins in each sample, and observed that more than 88% of significantly differentially expressed proteins (DEP) were downregulated in L1-treated cells (Figure 5D, left panel). Moreover, among the top 15 downregulated biological processes and cellular components were translation and ribosomal subunits (Figure 5D, right panels, yellow circles), indicating that L1 RNA induces a global attenuation of protein synthesis. Indeed, a cap-dependent translation shutdown is the main consequence of eIF2α phosphorylation.(Donnelly et al, 2013) Finally, we show that neither PKR KD (Appendix Figure S6A, B) nor its'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='we show that neither PKR KD (Appendix Figure S6A, B) nor its pharmacological inhibition by C16 treatment (Figure 5E and 5F) prevents L1 RNA-triggered induction of inflammatory genes (Figure 5E) and mineral matrix deposition (Figure 5F, Appendix Figure S6B). As previously mentioned, a global reduction of protein synthesis occurs 24h after L1 transfection and only a little percentage of DEP is upregulated in L1 compared to RFP control (Figure 5D). Interestingly, some of the most upregulated proteins are involved in autophagy and vesicle trafficking (Figure 5G). Autophagy is a crucial process for mineralization and bone homeostasis both in vitro and in vivo,(Nollet et al, 2014) as autophagic vacuoles are exploited to secrete apatite crystals.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Altogether these data demonstrate that the PKR-mediated sensing of L1-derived dsRNA globally attenuates the translation via eIF2α phosphorylation and indicate autophagy as a possible mechanism involved in L1-induced osteoblasts mineralization.\\n\\n\\n\\nL1 RNA delivery reshapes the secretory profile of differentiating osteoblasts'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='An inflammatory response translates into a strong secretory activity and production of paracrine signals, and we therefore tested the effect of “L1-primed” osteoblast-derived conditioned medium in recipient differentiating osteoblasts (Figure 6A). We found that the “secretome” of cells transfected with L1 RNA has a significant paracrine effect on recipient osteoblasts since they showed an earlier formation of mineralized nodules, already 24h post the addition of conditioned media (Figure 6A, right panel), and higher deposition of mineral matrix at day 10 (Figure 6A, left panel). We subsequently isolated bulk and exosome-derived proteomes from conditioned media of untreated, RFP- and L1-treated differentiating osteoblasts and characterized them by MS.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='differentiating osteoblasts and characterized them by MS. Differential expression analysis of MS data revealed a unique secretome profile (for both bulk and vesicular proteomes) of L1-primed osteoblasts compared to RFP-primed and untreated osteoblasts (Figure 6B). Gene Ontology (GO) analysis of differentially expressed proteins showed an enrichment of pro-inflammatory factors (i.e. interleukins and chemokines) involved in immune response and chemotactic migration of immune cells (Figure 6C and D), crucial processes for tissue repair mechanisms in vivo. Pro-inflammatory molecules are also a major constituent of the senescence-associated secretory phenotype (SASP),(Coppé et al, 2010) whose transient delivery supports cellular plasticity and tissue'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='transient delivery supports cellular plasticity and tissue regeneration.(Ritschka et al, 2017) Notably, the most enriched protein in L1-specific bulk secretome is interleukin 8 (IL-8), an inflammatory chemokine involved in several regenerative processes, such as skin wound healing,(JIANG et al, 2012) therapeutic angiogenesis after stroke and ischemia,(Choi et al, 2021) (Hou et al, 2014) and osteochondral bone repair.(Lin et al, 2019) (Yang et al, 2018a) Tumor necrosis factor-inducible gene 6 (TSG6) is among the top 10 proteins enriched in the bulk secretome of L1-primed osteoblasts compared to RFP. TSG6 is an inflammatory factor with suggestive therapeutic effects in corneal wounds, myocardial infarction, injured central nervous system, chronic liver'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='infarction, injured central nervous system, chronic liver damage, and intervertebral disc degeneration.(Zhang et al, 2013) (Lee et al, 2015) (Lee et al, 2009) (Wang et al, 2017) (Yang et al, 2018b) Moreover, TSG6 induces autophagy influx both in vivo and in vitro.(Wang et al, 2017) Another protein positively involved in autophagy and found specifically in the secretome of L1 primed osteoblasts was ATG7 (autophagy-related 7). Osteoblast-specific ATG7 conditional knockout in mice reduced bone mass during developmental and in adult age.(Li et al, 2018) Type 1 lysophosphatidic acid receptor (LPAR1) is also specifically secreted by L1-primed osteoblasts and has proven to be positively involved in bone mineralization in vitro and in vivo.(Alioli et al, 2020)'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='bone mineralization in vitro and in vivo.(Alioli et al, 2020) (Gennero et al, 2011)'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Altogether, our findings demonstrate that cytoplasmic L1 RNA is sensed by PKR, whose activation leads to eIF2α phosphorylation with consequent reprogramming of transcription and translation. The ultimate consequence of this L1-induced, PKR-mediated stress response in osteoblasts is a major reprogramming of their secretory activity, leading to a paracrine effect on recipient osteogenic competent cells to trigger their mineralization activity.\\n\\n\\n\\nDiscussion'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='The concerted co-option of TEs markedly changed whole regulatory networks and integrated new functions into the eukaryote genome.(Cosby et al, 2019) (Mangiavacchi et al, 2021) (Carelli et al, 2022) However, while the role of TEs as evolutionary drivers is today well-established, their contribution to cell physiology, particularly in somatic cells, remains to be elucidated. Indeed, aside from their role in the nucleus, (Della Valle et al, 2022) (Jachowicz et al, 2017) (Lu et al, 2021) (Percharde et al, 2018) TEs represent a major source of endogenous dsRNA whose role in triggering inflammatory and innate immune responses in various physiological processes has increasingly gained attention.(Sadeq et al, 2021) (Chen & Hur, 2022) For instance, an early acute'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='et al, 2021) (Chen & Hur, 2022) For instance, an early acute inflammatory response is of vital importance in initiating and orchestrating bone repair and induce anabolism leading to fracture healing. (Mountziaris et al, 2011) (Marsell & Einhorn, 2011) (Bahney et al, 2019) Moreover, several studies demonstrated a positive effect of proinflammatory mediators on bone mineralization in vitro and in vivo,(Croes et al, 2015) (Croes et al, 2016) (Li et al, 2016) (Croes et al, 2017) indicating that the modulation of inflammatory reaction is a tool to pursue regenerative strategies in bone.(Roberts & Ke, 2018) We found that, in the bone of newly fractured mice, a group of TEs, mostly LINE and LTR, was early, markedly and transiently upregulated. Interestingly, the'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='early, markedly and transiently upregulated. Interestingly, the expression of these TEs was limited to the earliest phase of the inflammatory stage. In line with the hypothesis of a positive correlation between stress-induced bone mineralization and TEs activation, the expression dynamics in human bone biopsies revealed a significant upregulation of TEs in the femur, a bone subjected to high mechanical stresses, high turnover and anabolism, compared to ilium, not weight bearing. Furthermore, we detected a lower TEs expression in donors with compromised bone mineralization and a significant correlation between TEs expression and local BMD. We hypothesized that the expression dynamics of TEs in fracture/mechanical stress observed in vivo could mimic the'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='in fracture/mechanical stress observed in vivo could mimic the immunological threshold model of the sterile activation of dsRNA sensors as a response to stressful conditions. Indeed, the transient and controlled breaching of the activation threshold of dsRNA sensors may lead to sterile inflammation being integrated into physiological processes.(Chen & Hur, 2022) To test further this hypothesis, we delivered a full-length L1 consensus sequence as source of dsRNA(Hur, 2019) to differentiating osteoblasts in vitro and showed that L1 RNA initially accumulated in the cytoplasm and is gradually cleared by secretion. The increased cytoplasmic levels of L1 RNA are immediately sensed by PKR, a cytoplasmic dsRNA sensor whose activation induces the phosphorylation of'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='dsRNA sensor whose activation induces the phosphorylation of eIF2a(Kim et al, 2018) (Donnelly et al, 2013), leading to the attenuation of global protein synthesis (Donnelly et al, 2013), induction of inflammatory response, and stimulation of mineral matrix deposition. In support of the evidence that PKR is the early sensor of increased L1 RNA cytoplasmic levels, we show that inhibition of PKR activation by C16 prevents L1-induced eIF2a phosphorylation, induction of inflammatory genes, and mineral matrix deposition. Moreover, the inhibition of L1 cDNA formation and sensing by Lamivudine 3TC, G140, or cGAS KD does not affect the L1-induced phenotype. Thus, the evidence does not support that the L1-triggered inflammation is causally connected to a'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='that the L1-triggered inflammation is causally connected to a cGAS-STING-dependent sensing of cytosolic L1 cDNA, a mechanism demonstrated operative in senescence.(De Cecco et al, 2019a)'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='L1 RNA response showed high specificity, as demonstrated by the dose-dependent increase of mineralization and the lack of significant changes in mineral deposition when negative control RNA was transfected, even at 100-200-fold higher concentration. Although the molecular mechanism behind L1-induced stress response which stimulates the mineralization activity of osteoblasts remains elusive, our results indicate that L1 stimulates the mineral matrix deposition by bone-forming cells regardless of their prior intrinsic anabolic potential. Moreover, MS results suggested that increased autophagy, a process crucial for mineralization and bone homeostasis in vitro and in vivo(Nollet et al, 2014) may contribute to L1-induced hydroxyapatite deposition. Intriguingly,'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='to L1-induced hydroxyapatite deposition. Intriguingly, the osteoinductive properties of L1-primed osteoblast-derived secretome and the observation that L1 RNA was gradually secreted by the cells may suggest an involvement of L1 RNA in cell-to-cell communication, in line with the recent finding of other retrotransposons (i.e. ERV) acting as paracrine molecules.(Liu et al, 2021) This, together with the observed L1-dependent induction of CCL and CXCL chemokines and cytokines, being chemoattractants of immune cells and endothelial cells,(Bischoff et al, 2011) (Eash et al, 2010) (Strieter et al, 2005) and the composition of L1-primed osteoblast secretome, makes the in vivo delivery of L1 RNA a necessary step to evaluate its actual contribution to the paracrine'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='step to evaluate its actual contribution to the paracrine processes that orchestrate bone repair mechanisms. (Chaparro & Linero, 2016) (Nimiritsky et al, 2019)'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='The identification of repeats RNA, particularly L1, as potential resilient molecular factors involved in stress-induced, inflammation-mediated bone production points out new avenues to develop anabolic strategies for the treatment of bone loss conditions, such as osteoporosis or impaired bone repair.(Roberts & Ke, 2018) On the other hand, repeats may be novel targets in the treatment of those conditions characterized by excessive or heterotopic ossification preceded by deregulated/chronic inflammation.(Lories & Schett, 2012) (Balboni et al, 2006)\\n\\n\\n\\nMaterials and Methods\\n\\nParticipants and ethics'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Femoral bone biopsies were obtained from the caput region of postmenopausal women or men with a wide BMD range, i.e., from healthy to osteoporotic, who were undergoing hip replacement surgery due to osteoarthritis or fracture at Lovisenberg Diaconal Hospital (Oslo, Norway) or Diakonhiemmet Hospital (Oslo, Norway), respectively. The donors of femoral bones are listed in Appendix Table S1. The postmenopausal iliac bone donors were recruited from the outpatient clinic of Lovisenberg Diaconal Hospital (Oslo, Norway). Candidates filled out a questionnaire that included medication and lifestyle factors, and selected donors were deemed representative of the Oslo-based Norwegian ethnic female population aged 50 to 86\\u2009years. The iliac bone donors are listed in'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content=\"aged 50 to 86\\u2009years. The iliac bone donors are listed in Appendix Table S2 and have been described previously in detail.(Reppe et al, 2010) All donors taking medication or having diseases, other than primary osteoporosis, known to affect bone metabolism were excluded. The presence of bone-impairing diseases/conditions was excluded by extensive biochemical serum and urine analyses supported by X-ray examinations. The site-specific BMD of all donors was evaluated with Lunar Prodigy DEXA (GE Lunar, Madison, WI, USA) following the manufacturer's instructions. The precision of the instrument for measuring the lumbar spine (L2–L4) and hip BMD was 1.7% and 1.1%, respectively. The study was approved by the Norwegian Regional Ethical Committee (REK no 2010/2539,\"), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='by the Norwegian Regional Ethical Committee (REK no 2010/2539, Norway), all volunteers gave their written informed consent, and sampling and procedures were according to the Act of Biobanking in Norway.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Cell culture'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='For L1 RNA transfection experiments, human MSCs were isolated from osteoporotic femoral heads with the consent of the patient according to Swiss (BASEC-Nr. 2016-02159) ethical guidelines. Femoral heads were crushed and incubated in DMEM (PAN Biotech, Germany, Cat. No. P04-03550) supplemented with 1% (v/v) 100× Penicillin-Streptomycin Solution (Biowest, France, Cat. No. L0022), 10% (v/v) FCS (Biowest, France, Cat. No. S181S), 1% (v/v) 200mM L-Glutamine solution (Sigma, USA, Cat. No. G7513), 5ng/ml FGF-2 (Sigma, USA, Cat. No. F0291) and 10ng/ml FGF-4 (Sigma, USA, Cat. No. F8424) in a humidified atmosphere containing 5% CO2. To induce osteogenic differentiation, MSCs were seeded on Nunc™ 24-well plates (Thermo Fisher, USA, Cat. No. 142475) or 48-well plates'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='plates (Thermo Fisher, USA, Cat. No. 142475) or 48-well plates (Thermo Fisher, USA, Cat. No. 150687) at a density of 1.5\\u2009×\\u2009104 cells/cm2. After 24\\u2009h, differentiation was induced using StemPro® Osteogenesis Kit (Gibco/Life Technologies, USA, Cat. No. A10072-01). The medium was exchanged every 3 days. Cells were treated with Lamivudine 3TC (Sigma), 1mM final concentration; C16 (Merck), 500nM final concentration; G140 (InvivoGen), 150mM final concentration.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='L1 RNA transfection'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='The vector human-L1_pBluescript II sk (+) carrying the full-length L1 sequence was custom-prepared by GenScript, USA. Large-scale human L1 mRNA was in vitro transcribed, modified, and purified by TriLink Biotechnologies, USA, (ARCA capped and 2’Omethymalted (CapI), fully substituted with 5-methyl-C, 25% substitution of Cyanine-5-U and 75% substitution of Pseudo-U, enzymatically polyadenylated, DNase and phosphatase treated, silica membrane purified). Synthetic L1 RNA was transfected in MSCs differentiating to osteoblasts at day 5 using LipofectamineTM MessengerMAXTM (Invitrogen, USA, Cat. No. LMRNA003) with a modified protocol for low RNA amount. RFP mRNA (System Bioscience, USA, Cat. No. MR800A-1) was used as negative control. Bone matrix was quantified'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='was used as negative control. Bone matrix was quantified with OsteoImage Mineralization Assay (Lonza, Basel, Switzerland, Cat. No. LOPA503) and Alizarin Red staining. In conditioned media experiments, the medium was exchanged 6h after transfection. Conditioned medium was collected after three days and delivered to the recipient, differentiating osteoblasts at day 5. In knockdown experiments, siRNAs were transfected with Lipofectamine RNAiMAX (Thermo Scientific) in differentiating osteoblasts at day 3, followed by L1 RNA transfection at day 7. siRNAs: EIF2AK2- Targeting SMARTpool (Dharmacon, Horizon, E-003527-00-0010), MB21D1-Targeting SMARTpool (Dharmacon, Horizon, L-015607-02-0020), Non-targeting Pool (D-001810-10).'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Quantitative mineralization assay\\n\\nCells were washed in PBS and fixed in 4% (v/v) formaldehyde (Sigma, USA, Cat. No. F8775) in 1× PBS for 15 minutes. Mineralization was assessed by using the OsteoImage Mineralization Assay (Lonza, Basel, Switzerland, Cat. No. LOPA503) according to the manufacturer’s indication. Mineralization was quantified with GloMax® discover microplate reader (Promega, USA), selecting appropriate excitation (492) / emission (520) wavelengths.\\n\\nAlizarin Red staining'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Osteoblasts were washed with 1× PBS (Kantonsapotheke Zürich, Switzerland, Cat. No. A171012) and fixed with 4% (v/v) formaldehyde (Sigma, USA, Cat. No. F8775) in 1× PBS for 30\\u2009min. After washing twice with ddH2O, Alizarin Red staining solution (0.7\\u2009g Alizarin Red S (Sigma, USA, Cat. No. A5533) diluted in 50\\u2009ml ddH2O at pH\\u2009=\\u20094.2) was added for 20\\u2009min. Afterwards, cells were washed four times with ddH2O, dried, and stored in the dark until image acquisition. For absorbance measurement, Alizarin Red S was eluted from stained osteoblasts with 300\\u2009μl 10% (w/v) cetylpyridinium chloride in an aqueous 0.01\\u2009M Na2HPO4/NaH2PO4 solution at pH\\u2009=\\u20097 for 1\\u2009h. One hundred fifty microliters were transferred on a 96-well plate, and absorbance was measured at 560\\u2009nm. Ten'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='on a 96-well plate, and absorbance was measured at 560\\u2009nm. Ten percent (w/v) cetylpyridinium chloride in an aqueous 0.01\\u2009M Na2HPO4/NaH2PO4 solution was used as blank. Images were acquired, processed and analysed as previously described(Eggerschwiler et al, 2019)'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='ALPL activity assay: \\n\\nALPL activity was measured as in Liu et al., 2016.(Liu et al, 2016) Briefly, cells were fixed in 3.7% formaldehyde at RT for 10’ and then stained with a solution of 25% naphthol AS-BI phosphate (Thermo) and 0.75% Fast Blue BB (Sigma) dissolved in 0.1\\u2009M Tris buffer (pH 9.3) at RT for 15’. After staining, cells were quickly washed with PBS five times.\\n\\nENPP1 activity assay'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='ENPP1 activity assay\\n\\nENPP1 activity was calculated colorimetrically using the chromogenic substrate p-nitrophenyl-thymidine-5′-monophosphate, as described in (Ferreira et al., 2013). After PBS washing, cells were lysed in a solution of 0.1% Triton X-100, 0.2 M Tris-base, 1.6 mM MgCl2, pH 8.1. 50 ml of 1mM thymidine monophosphate p-nitrophenyl ester (Sigma) were added to 50 ml of cell lysate and incubated for 1 hour at 37°C. Then, four volumes of 0.1M NaOH were added to stop the reaction and absorbance was read at 410 nm using a spectrophotometer (Tecan Nanoquant Infinite 200 Pro Multimode). Absorbance was normalized to DNA content.\\n\\nImmunohistochemistry:'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Immunohistochemistry: \\n\\nCells were fixed in 4% paraformaldehyde in PBS at room temperature (RT) for 10’. After fixation, cells were permeabilized in 3% Triton X-100 in PBS for 3’ at RT. Blocking was performed with 4% BSA in PBS for 30’ at RT. After blocking, cells were incubated with primary antibodies overnight at 4°C, then washed with PBS and incubated with secondary antibodies for 1 hour at RT. After washing with PBS, nuclei were stained with Hoechst (Thermo Scientific) and mounted using Fluoro Gel with DABCO™ Mounting Medium (Electron Microscopy Sciences). Primary antibodies: DNA-RNA Hybrid (mouse, Merck Millipore, MABE1095), dsRNA (mouse, Merck Millipore, MABE1134).\\n\\nRNA extraction and cDNA preparation'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='RNA extraction and cDNA preparation\\n\\nCells were harvested and resuspended in 1ml of QIAzol Lysis reagent (Qiagen, Cat. No.79306). Total RNA was then purified with RNeasy Plus Mini kit (Qiagen, cat. No. 74134) with minimal modifications to manufacturer’s instructions. DNase treatment (RNase-free DNase set, Qiagen, Cat. No. 79254) was performed to remove any residual DNA. RNA quality and concentration were checked using a NanodropTM 2000 spectrophotometer (ThermoFisher). cDNA was synthesized from 200ng of each RNA sample using a Superscript III first-strand cDNA synthesis system (ThermoFisher, cat. No.18080051) according to the manufacturer’s protocol. Processing of human bone biopsies and RNA isolation has been described previously.(Reppe et al, 2010)'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Gene expression analysis'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Real-time quantitative polymerase chain reaction (qPCR) was performed with a 7900HT Fast Real-Time PCR system (Applied Biosystems). Each sample was analyzed in triplicate and normalized with the endogenous control Ribosomal Protein L13A (RPL13A) for cDNA input concentration. No template and no RT were included as negative controls. For each 15\\uf06dl reaction, 10ng (1ng for L1) of cDNA was mixed with 1\\uf06dM specific primers mix and 7.5\\uf06dl of SybrTM Select Master mix (Applied Biosystems, USA, Cat. No. 4472908). The reaction was incubated at 95°C for 10 minutes, followed by 40 cycles of denaturation at 95°C for 15 seconds, annealing at 60°C for 30 seconds and elongation at 72°C for 30 seconds. Ct values were calculated by 7900HT Fast Real-Time PCR RQ manager software'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='calculated by 7900HT Fast Real-Time PCR RQ manager software (Applied Biosystems, USA) and then normalized as \\uf044Ct between the gene of interest and the endogenous calibrator. Primers used in this study for gene expression analysis were designed using Primer3 (http://www.ncbi.nlm.nih.gov/tools/primer-blast/). In all primer pairs, each primer matches a different exon. Amplicons length was 80-130 nucleotides. Primer sequences are reported in Appendix Table S3.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Cell cycle analysis\\n\\n2x105 MSCs were trypsinized for 5 min at 37°C, washed with PBS and 2% BSA, passed through a 70\\uf06dM strainer (Corning, USA, Cat. No. 352350) and then fixed at -20°C for 30 min in 70% ethanol. After washing with PBS and 4% BSA, cells were resuspended in PBS and incubated for 1 hour at 37°C with RNase. Cells were then washed and resuspended in 100\\uf06dl of Flow Cytometry Staining Buffer (R&D System, USA, Cat. No. FC001). 10\\uf06dl of 1mg/ml Propidium iodide (PI) staining solution (Invitrogen, USA, Cat. No. P3566) was added to the single-cell solution, gently mixed, and incubated for 5 min in the dark. Cell cycle analysis was performed on BD FACSCanto II Flow Cytometry System (BD-Biosciences), using BD FACSDiva Software (BD-Biosciences).'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='RNA-Seq and data analysis\\n\\nRNA from human bone biopsies was sequenced at the Human Genotyping facility (HuGe-F) of Erasmus MC.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Total RNA-Seq library, from L1 RNA delivery experiments, was prepared with CORALL Total RNA library prep with RiboCop rRNA for Human/Mouse/Rat depletion kit (Lexogen GmbH, Vienna, Austria) following manufacturer’s instructions (library type: fr-secondstrand) by IGA Technology service (Italy). The final libraries were checked using both Qubit 2.0 Fluorometer (Invitrogen, Carlsbad, CA) and Agilent Bioanalyzer DNA assay or Caliper (PerkinElmer, Waltham, MA). Libraries were then prepared for sequencing and sequenced on paired-end 150 bp mode on NovaSeq6000 (Illumina, San Diego, CA). RNA-Seq read quality control (QC) analyses and filtering of high-quality reads were executed using FastQC v0.11.9 (http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) and'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='(http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) and BBDuk v35.85 (https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbduk-guide/)\\xa0by setting a minimum read length of 35 bp and a minimum Phred-quality score of 25. After trimming quality control, high-quality reads were aligned to the human genome reference (GRCh38) with STAR 2.7.3a,(Dobin et al, 2013) while FeatureCounts 1.6.3 package(Liao et al, 2014) was used to assign reads to genes. Next, lowly expressed genes across one or more experimental conditions were filtered to eliminate the “uninformative” genes using HTSFilter v1.30.1.(Rau et al, 2013) Filtered gene data were further processed with the EdgeR package v3.32.1(Robinson et al, 2010) to normalize (Trimmed Mean'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='v3.32.1(Robinson et al, 2010) to normalize (Trimmed Mean of M-7 values, TMM, method) the raw counts and perform differential gene expression analysis. Multiple testing correction was performed with the FDR method(Benjamini & Hochberg, 1995) and significance level was set at FDR < 0.05. Gene Ontology (GO) term enrichment was analyzed by performing hypergeometric tests11 for each individual term, and FDR correction was applied (FDR < 0.05). Expression of Interspersed Repeat elements was quantified using SQuiRE 0.9.9.92 (https://github.com/wyang17/SquIRE). SQuiRE provides locus-specific expression quantification along with subfamily-level expression estimates counting unambiguously mapped reads, as well as ambiguously mapped reads using an'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='mapped reads, as well as ambiguously mapped reads using an expectation–maximization (EM) algorithm.(Yang et al, 2019) Briefly, reference genome and Repeatmasker annotation were downloaded from UCSC and prepared for the analysis with squire Fetch and squire Clean, respectively. High-quality reads were mapped against the reference genome with STAR using squire Map and expression was quantified using squire Call. Differential expression analysis was performed with squire Call.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='DIA-MS analysis using TimsTOF MS'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Total protein extracts were prepared with RIPA buffer (50mMTris-cl pH 8.0, 5mM EDTA, 150mM Nacl, 15mM MgCl2, 1% NP-40, 1mM PMSF and 1X Protease Inhibitor Cocktail (PIC)). Further sonication step was included: (30S ON, 30S OFF, 10 cycles with Bioruptor). Equal 50 mg protein extracts were concentrated to 30 ml volume, then diluted in 8M urea in 0.1M Tris-HCl, followed by protein digestion with trypsin, according to the FASP protocol.(Wiśniewski et al, 2009) After overnight digestion, the peptides were eluted from the filters with 25 mM ammonium bicarbonate buffer. The eluted peptides were processed in the desalting step using Sep-Pag C18 Column (waters) based on the manufacturer’s instructions. Approximately 200 ng of peptide mixture per sample was analyzed'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Approximately 200 ng of peptide mixture per sample was analyzed using a timsTOF Pro 2 QTOF mass spectrometer coupled with a nanoElute liquid chromatography system (Bruker Daltonik GmbH, Germany). The sample was injected directly into an RP-C18 Aurora emitter column (75 µm\\u202fi.d.×\\u202f250 mm, 1.6 μm, 120 Å pore size) (Ion Opticks, Australia) using a one-column separation method. An 80-min gradient was established using mobile phase A (0.1% FA in H2O) and mobile phase B (0.1% FA in Acetonitrile): 2–25% B for 60\\u2009min, 25–37% for 10\\u2009min, ramping 37% to 95% in 5\\u2009min, and maintaining 95% B for 5\\u2009min. The column temperature was set at 50oC and the flow rate at 250 nl/min. The sample eluting from the separation column was introduced into the mass spectrometer via a'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='column was introduced into the mass spectrometer via a CaptiveSpray nano-electrospray ion source (Bruker Daltonik GmbH) with an electrospray voltage of 1.6\\u2009kV. The ion source temperature was set to 180\\u2009°C and a dry gas of 3 l/min. The samples were analyzed using diaPASEF scheme(Meier et al, 2020b) consisting of 24 cycles including a total of 48 mass width windows (13 Da (m/z) from m/z 400 to 1,000 and TIMS scan range from\\u20090.63\\u2009to 1.35\\u2009Vs\\u2009cm−2 (1/K0). The collisional energy increased linearly from 20.01 eV at 0.6 (1/K0) to 52.00 eV at 1.35\\u2009Vs\\u2009cm−2 (1/K0). The scan range for MS and MS/MS spectra was set to 100-1700 m/z. TIMS ramping time and accumulation time were set to 100 milliseconds. The diaPASEF data were analyzed by directDIA approach using Spectronaut'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='data were analyzed by directDIA approach using Spectronaut software (version 14) following manufacture instructions. Up or down regulated proteins were determined using DEP (Differential Enrichment analysis of Proteomics data) R package. Significant results (adjusted p-value < 0.05) were subjected to Gene Ontology enrichment analysis with clusterProfiler R package.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Western Blot'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Total protein extracts were prepared by lysing cells in extraction buffer (HEPES KOH [pH 8.5], NaCl 400 mM, EDTA 0.1 mM, EGTA 0.1 mM, DTT 1 mM, 1× protease inhibitor, SDS 1%). Proteins were separated by electrophoresis on BOLT 4%–12% bis-tris polyacrylamide precast gels in MES buffer (Life Technologies) and transferred to a 0,2 mm nitrocellulose membrane. Non-specific signals were blocked with 5% Milk-PBS-Tween0.5% and the membrane hybridized overnight at 4°C with primary and secondary antibodies diluted in blocking buffer. Horseradish peroxidase-conjugated secondary antibodies were revealed with the ECL chemiluminescence kit (Amersham), and signals were detected using ChemiDoc (Bio-Rad). Primary antibodies: PKR (rabbit, Abcam, ab32052), cGAS (rabbit,'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Primary antibodies: PKR (rabbit, Abcam, ab32052), cGAS (rabbit, Abcam, ab224144), LINE-1 ORF1 (mouse, Merck Millipore, MABC1152), EIF2a (mouse, Abcam, ab5369), phospho-EIF2a (rabbit, Abcam, ab32157), Histone H3 (rabbit, Merk, 06-755).'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Exosome isolation\\n\\nExosomes were isolated using Total Exosome Isolation Reagent (from cell culture media) (Thermo), following manufacturer’s instructions. The exosome pellet was resuspended in 50 ml of RIPA buffer and kept on ice for 30’, mixing every 5’. Samples were then sonicated (30’’ ON/30” OFF, 10 cycles), centrifuged at 13,000 x g for 20’ at 4°C and collected as supernatants.\\n\\nStatistical analysis\\n\\nStatistic tests used for data analysis are indicated in the figure legends. \\n\\nData availability'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Data availability\\n\\nThe RNA-seq transcriptomics data on human bone biopsies have been deposited to BioProject with the dataset identifier PRJNA764663 (https://www.ncbi.nlm.nih.gov/bioproject/PRJNA764663). The RNA-seq transcriptomics data on transfected osteoblasts have been deposited to GEO with the dataset identifier GSE201774 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201774) (enter token: ififogaehzoxlgh).\\n\\nThe mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD051195. Reviewer account details are username: reviewer_pxd051195@ebi.ac.uk ; password: R6v0mIYe\\n\\n\\n\\nAcknowledgment'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Acknowledgment\\n\\nWe acknowledge kaust Smarth Health Initiative (KSHI); the South-Eastern Norway Regional Health Authority; the Norwegian Osteoporosis Society; Anders Jahre Foundation for the Promotion of Science; the Lovisenberg Diaconal Hospital research fund; and the Research Council of Norway.\\n\\n\\n\\nFunding: We are grateful for support from: KAUST BAS/1/1037-01-01, the European Union project OSTEOGENE (no. FP6-502491); ZonMw\\xa0VIDI\\xa0016.136.367\\xa0grant, for funding the creation of the RNA-Seq dataset of hip primary bone and Oslo University Hospital, Ullevål, project #29750104.\\n\\n\\n\\nDisclosure and  competing interests statement: All the authors declare no competing interests.\\n\\n\\n\\nReferences'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='References\\n\\nAerssens J, Boonen S, Joly J & Dequeker J (1997) Variations in trabecular bone composition with anatomical site and age: potential implications for bone quality assessment. J Endocrinol 155: 411–421\\n\\nAlioli CA, Demesmay L, Laurencin-Dalacieux S, Beton N, Farlay D, Follet H, Saber A, Duboeuf F, Chun J, Rivera R, et al (2020) Expression of the type 1 lysophosphatidic acid receptor in osteoblastic cell lineage controls both bone mineralization and osteocyte specification. Biochim Biophys Acta - Mol Cell Biol Lipids 1865: 158715\\n\\nAnderson BR, Muramatsu H, Nallagatla SR, Bevilacqua PC, Sansing LH, Weissman D & Karikó K (2010) Incorporation of pseudouridine into mRNA enhances translation by diminishing PKR activation. Nucleic Acids Res'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Bahney CS, Zondervan RL, Allison P, Theologis A, Ashley JW, Ahn J, Miclau T, Marcucio RS & Hankenson KD (2019) Cellular biology of fracture healing. J Orthop Res 37: 35–50\\n\\nBalboni TA, Gobezie R & Mamon HJ (2006) Heterotopic ossification: Pathophysiology, clinical features, and the role of radiotherapy for prophylaxis. Int J Radiat Oncol 65: 1289–1299\\n\\nBenjamini Y & Hochberg Y (1995) Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing. J R Stat Soc Ser B 57: 289–300\\n\\nBischoff DS, Sakamoto T, Ishida K, Makhijani NS, Gruber HE & Yamaguchi DT (2011) CXC receptor knockout mice: characterization of skeletal features and membranous bone healing in the adult mouse. Bone 48: 267–274'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Brem H & Tomic-Canic M (2007) Cellular and molecular basis of wound healing in diabetes. J Clin Invest 117: 1219–1222\\n\\nCarelli FN, Cerrato C, Dong Y, Appert A, Dernburg A & Ahringer J (2022) Widespread transposon co-option in the Caenorhabditis germline regulatory network. Sci Adv 8\\n\\nDe Cecco M, Ito T, Petrashen AP, Elias AE, Skvir NJ, Criscione SW, Caligiana A, Brocculi G, Adney EM, Boeke JD, et al (2019a) L1 drives IFN in senescent cells and promotes age-associated inflammation. Nature 566: 73–78\\n\\nDe Cecco M, Ito T, Petrashen AP, Elias AE, Skvir NJ, Criscione SW, Caligiana A, Brocculi G, Adney EM, Boeke JD, et al (2019b) L1 drives IFN in senescent cells and promotes age-associated inflammation. Nature 566: 73–78'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Chan JK, Roth J, Oppenheim JJ, Tracey KJ, Vogl T, Feldmann M, Horwood N & Nanchahal J (2012) Alarmins: awaiting a clinical response. J Clin Invest 122: 2711\\n\\nChaparro O & Linero I (2016) Regenerative Medicine: A New Paradigm in Bone Regeneration. In Advanced Techniques in Bone Regeneration InTech\\n\\nChen YG & Hur S (2022) Cellular origins of dsRNA, their recognition and consequences. Nat Rev Mol Cell Biol 23: 286–301\\n\\nChoi J, Choi W, Joo Y, Chung H, Kim D, Oh SJ & Kim S-H (2021) FGF2-primed 3D spheroids producing IL-8 promote therapeutic angiogenesis in murine hindlimb ischemia. NPJ Regen Med 6: 48\\n\\nClark D, Nakamura M, Miclau T & Marcucio R (2017) Effects of Aging on Fracture Healing. Curr Osteoporos Rep 15: 601'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Coates BA, McKenzie JA, Buettmann EG, Liu X, Gontarz PM, Zhang B & Silva MJ (2019) Transcriptional Profiling of Intramembranous and Endochondral Ossification after Fracture in Mice. Bone 127: 577\\n\\nCoppé J-P, Desprez P-Y, Krtolica A & Campisi J (2010) The Senescence-Associated Secretory Phenotype: The Dark Side of Tumor Suppression. Annu Rev Pathol Mech Dis 5: 99–118\\n\\nCosby RL, Chang NC & Feschotte C (2019) Host-transposon interactions: conflict, cooperation, and cooption. Genes Dev 33: 1098–1116\\n\\nCroes M, Boot W, Kruyt MC, Weinans H, Pouran B, van der Helm YJM, Gawlitta D, Vogely HC, Alblas J, Dhert WJA, et al (2017) Inflammation-Induced Osteogenesis in a Rabbit Tibia Model. Tissue Eng Part C Methods 23: 673–685'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Croes M, Oner FC, Kruyt MC, Blokhuis TJ, Bastian O, Dhert WJA & Alblas J (2015) Proinflammatory Mediators Enhance the Osteogenesis of Human Mesenchymal Stem Cells after Lineage Commitment. PLoS One 10: e0132781\\n\\nCroes M, Öner FC, van Neerven D, Sabir E, Kruyt MC, Blokhuis TJ, Dhert WJA & Alblas J (2016) Proinflammatory T cells and IL-17 stimulate osteoblast differentiation. Bone 84: 262–270\\n\\nDobin A, Davis CA, Schlesinger F, Drenkow J, Zaleski C, Jha S, Batut P, Chaisson M & Gingeras TR (2013) STAR: ultrafast universal RNA-seq aligner. Bioinformatics 29: 15–21\\n\\nDonnelly N, Gorman AM, Gupta S & Samali A (2013) The eIF2α kinases: their structures and functions. Cell Mol Life Sci 70: 3493–3511'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Eash KJ, Greenbaum AM, Gopalan PK & Link DC (2010) CXCR2 and CXCR4 antagonistically regulate neutrophil trafficking from murine bone marrow. J Clin Invest 120: 2423–2431\\n\\nEggerschwiler B, Canepa DD, Pape HC, Casanova EA & Cinelli P (2019) Automated digital image quantification of histological staining for the analysis of the trilineage differentiation potential of mesenchymal stem cells. Stem Cell Res Ther\\n\\nFLEISCH H, RUSSELL RGG & STRAUMANN F (1966) Effect of Pyrophosphate on Hydroxyapatite and Its Implications in Calcium Homeostasis. Nature 212: 901–903'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Fukuda S, Varshney A, Fowler BJ, Wang S Bin, Narendran S, Ambati K, Yasuma T, Magagnoli J, Leung H, Hirahara S, et al (2021) Cytoplasmic synthesis of endogenous Alu complementary DNA via reverse transcription and implications in age-related macular degeneration. Proc Natl Acad Sci U S A 118\\n\\nGe C, Xiao G, Jiang D & Franceschi RT (2007) Critical role of the extracellular signal–regulated kinase–MAPK pathway in osteoblast differentiation and skeletal development. J Cell Biol 176: 709\\n\\nGennero I, Laurencin-Dalicieux S, Conte-Auriol F, Briand-Mésange F, Laurencin D, Rue J, Beton N, Malet N, Mus M, Tokumura A, et al (2011) Absence of the lysophosphatidic acid receptor LPA1 results in abnormal bone development and decreased bone mass. Bone 49: 395–403'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Gerstenfeld LC, Cho T-J, Kon T, Aizawa T, Cruceta J, Graves BD & Einhorn TA (2001) Impaired Intramembranous Bone Formation during Bone Repair in the Absence of Tumor Necrosis Factor-Alpha Signaling. Cells Tissues Organs 169: 285–294\\n\\nGerstenfeld LC, Cullinane DM, Barnes GL, Graves DT & Einhorn TA (2003) Fracture healing as a post-natal developmental process: Molecular, spatial, and temporal aspects of its regulation. J Cell Biochem 88: 873–884\\n\\nHessle L, Johnson KA, Anderson HC, Narisawa S, Sali A, Goding JW, Terkeltaub R & Millan JL (2002) Tissue-nonspecific alkaline phosphatase and plasma cell membrane glycoprotein-1 are central antagonistic regulators of bone mineralization. Proc Natl Acad Sci U S A 99: 9445–9'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Hou Y, Ryu CH, Jun JA, Kim SM, Jeong CH & Jeun S (2014) IL-8 enhances the angiogenic potential of human bone marrow mesenchymal stem cells by increasing vascular endothelial growth factor. Cell Biol Int 38: 1050–9\\n\\nHur S (2019) Double-Stranded RNA Sensors and Modulators in Innate Immunity. Annu Rev Immunol 37: 349–375\\n\\nHurtgen BJ, Ward CL, Garg K, Pollot BE, Goldma SM, McKinley TO, Wenke JC & Corona BT (2016) Severe muscle trauma triggers heightened and prolonged local musculoskeletal inflammation and impairs adjacent tibia fracture healing. J Musculoskelet Neuronal Interact 16: 122'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Jachowicz JW, Bing X, Pontabry J, Bošković A, Rando OJ & Torres-Padilla ME (2017) LINE-1 activation after fertilization regulates global chromatin accessibility in the early mouse embryo. Nat Genet\\n\\nJIANG WG, SANDERS AJ, RUGE F & HARDING KG (2012) Influence of interleukin-8 (IL-8) and IL-8 receptors on the migration of human keratinocytes, the role of PLC-γ and potential clinical implications. Exp Ther Med 3: 231–236\\n\\nJiao H, Xiao E & Graves DT (2015) Diabetes and Its Effect on Bone and Fracture Healing. Curr Osteoporos Rep 13: 327–335'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Johnson KA, Hessle L, Vaingankar S, Wennberg C, Mauro S, Narisawa S, Goding JW, Sano K, Millan JL & Terkeltaub R (2000) Osteoblast tissue-nonspecific alkaline phosphatase antagonizes and regulates PC-1. Am J Physiol Regul Integr Comp Physiol 279: R1365-77\\n\\nJones RB, Garrison KE, Wong JC, Duan EH, Nixon DF & Ostrowski MA (2008) Nucleoside analogue reverse transcriptase inhibitors differentially inhibit human LINE-1 retrotransposition. PLoS One\\n\\nKarikó K, Buckstein M, Ni H & Weissman D (2005) Suppression of RNA recognition by Toll-like receptors: The impact of nucleoside modification and the evolutionary origin of RNA. Immunity'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Karikó K, Muramatsu H, Welsh FA, Ludwig J, Kato H, Akira S & Weissman D (2008) Incorporation of pseudouridine into mRNA yields superior nonimmunogenic vector with increased translational capacity and biological stability. Mol Ther\\n\\nKim Y, Lee JH, Park J-E, Cho J, Yi H & Kim VN (2014) PKR is activated by cellular dsRNAs during mitosis and acts as a mitotic regulator. Genes Dev 28: 1310–1322\\n\\nKim Y, Park J, Kim S, Kim MA, Kang MG, Kwak C, Kang M, Kim B, Rhee HW & Kim VN (2018) PKR Senses Nuclear and Mitochondrial Signals by Interacting with Endogenous Double-Stranded RNAs. Mol Cell 71: 1051-1063.e6'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Kon T, Cho TJ, Aizawa T, Yamazaki M, Nooh N, Graves D, Gerstenfeld LC & Einhorn TA (2001) Expression of osteoprotegerin, receptor activator of NF-kappaB ligand (osteoprotegerin ligand) and related proinflammatory cytokines during fracture healing. J Bone Miner Res 16: 1004–1014\\n\\nKormann MSD, Hasenpusch G, Aneja MK, Nica G, Flemmer AW, Herber-Jonat S, Huppmann M, Mays LE, Illenyi M, Schams A, et al (2011) Expression of therapeutic proteins after delivery of chemically modified mRNA in mice. Nat Biotechnol'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Koski GK, Karikó K, Xu S, Weissman D, Cohen PA & Czerniecki BJ (2004) Cutting Edge: Innate Immune System Discriminates between RNA Containing Bacterial versus Eukaryotic Structural Features That Prime for High-Level IL-12 Secretion by Dendritic Cells. J Immunol\\n\\nKovtun A, Bergdolt S, Wiegner R, Radermacher P, Huber-Lang M & Ignatius A (2016) The crucial role of neutrophil granulocytes in bone fracture healing. Eur Cells Mater 32: 152–162\\n\\nLama L, Adura C, Xie W, Tomita D, Kamei T, Kuryavyi V, Gogakos T, Steinberg JI, Miller M, Ramos-Espiritu L, et al (2019) Development of human cGAS-specific small-molecule inhibitors for repression of dsDNA-triggered interferon expression. Nat Commun 10: 2261'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Laroche M, Livideanu C, Paul C & Cantagrel A (2011) Interferon Alpha and Pamidronate in Osteoporosis with Fracture Secondary to Mastocytosis. Am J Med 124: 776–778\\n\\nLee MJ, Kim DH, Ryu JS, Ko AY, Ko JH, Kim MK, Wee WR, Khwarg SI & Oh JY (2015) Topical TSG-6 Administration Protects the Ocular Surface in Two Mouse Models of Inflammation-Related Dry Eye. Invest Ophthalmol Vis Sci 56: 5175–81\\n\\nLee RH, Pulin AA, Seo MJ, Kota DJ, Ylostalo J, Larson BL, Semprun-Prieto L, Delafontaine P & Prockop DJ (2009) Intravenous hMSCs improve myocardial infarction in mice because cells embolized in lung are activated to secrete the anti-inflammatory protein TSG-6. Cell Stem Cell 5: 54–63'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Lefkopoulos S, Polyzou A, Derecka M, Bergo V, Clapes T, Cauchy P, Jerez-Longres C, Onishi-Seebacher M, Yin N, Martagon-Calderón N-A, et al (2020) Repetitive Elements Trigger RIG-I-like Receptor Signaling that Regulates the Emergence of Hematopoietic Stem and Progenitor Cells. Immunity 53: 934-951.e9\\n\\nLi C, Li G, Liu M, Zhou T & Zhou H (2016) Paracrine effect of inflammatory cytokine-activated bone marrow mesenchymal stem cells and its role in osteoblast function. J Biosci Bioeng 121: 213–219\\n\\nLi H, Li D, Ma Z, Qian Z, Kang X, Jin X, Li F, Wang X, Chen Q, Sun H, et al (2018) Defective autophagy in osteoblasts induces endoplasmic reticulum stress and causes remarkable bone loss. Autophagy 14: 1726–1741'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Li J, Li X, Liu D, Hamamura K, Wan Q, Na S, Yokota H & Zhang P (2019) eIF2α signaling regulates autophagy of osteoblasts and the development of osteoclasts in OVX mice. Cell Death Dis 10: 921\\n\\nLiao Y, Smyth GK & Shi W (2014) featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. Bioinformatics 30: 923–930\\n\\nLin D, Chai Y, Ma Y, Duan B, Yuan Y & Liu C (2019) Rapid initiation of guided bone regeneration driven by spatiotemporal delivery of IL-8 and BMP-2 from hierarchical MBG-based scaffold. Biomaterials 196: 122–137'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Liu W, Zhou L, Zhou C, Zhang S, Jing J, Xie L, Sun N, Duan X, Jing W, Liang X, et al (2016) GDF11 decreases bone mass by stimulating osteoclastogenesis and inhibiting osteoblast differentiation. Nat Commun 7: 12794\\n\\nLiu X, Liu Z, Sun L, Ren J, Wu Z, Jiang X, Ji Q, Wang Q, Fan Y, Cai Y, et al (2021) Resurrection of human endogenous retroviruses during aging reinforces senescence. bioRxiv\\n\\nLories RJU & Schett G (2012) Pathophysiology of New Bone Formation and Ankylosis in Spondyloarthritis. Rheum Dis Clin North Am 38: 555–567\\n\\nLu JY, Chang L, Li T, Wang T, Yin Y, Zhan G, Han X, Zhang K, Tao Y, Percharde M, et al (2021) Homotypic clustering of L1 and B1/Alu repeats compartmentalizes the 3D genome. Cell Res 31: 613–630'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Ludwig J, Schuberth C, Goldeck M, Schlee M, Li H, Juranek S, Sheng G, Micura R, Tuschl T, Hartmann G, et al (2010) Structural and functional insights into 5′-ppp RNA pattern recognition by the innate immune receptor RIG-I. Nat Struct Mol Biol\\n\\nMangiavacchi A, Liu P, Della Valle F & Orlando V (2021) New insights into the functional role of retrotransposon dynamics in mammalian somatic cells. Cell Mol Life Sci 78: 5245–5256\\n\\nMarsell R & Einhorn TA (2011) The biology of fracture healing. Injury 42: 551–555\\n\\nMaruyama M, Rhee C, Utsunomiya T, Zhang N, Ueno M, Yao Z & Goodman SB (2020) Modulation of the Inflammatory Response and Bone Healing. Front Endocrinol (Lausanne) 11: 386'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Mashanov VS, Zueva OR & García-Arrarás JE (2012) Posttraumatic regeneration involves differential expression of long terminal repeat (LTR) retrotransposons. Dev Dyn 241: 1625–1636\\n\\nMatsushita T, Chan YY, Kawanami A, Balmes G, Landreth GE & Murakami S (2009) Extracellular signal-regulated kinase 1 (ERK1) and ERK2 play essential roles in osteoblast differentiation and in supporting osteoclastogenesis. Mol Cell Biol 29: 5843–5857\\n\\nMeier F, Brunner AD, Frank M, Ha A, Bludau I, Voytik E, Kaspar-Schoenefeld S, Lubeck M, Raether O, Bache N, et al (2020a) diaPASEF: parallel accumulation-serial fragmentation combined with data-independent acquisition. Nat Methods 17: 1229–1236'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Meier F, Brunner AD, Frank M, Ha A, Bludau I, Voytik E, Kaspar-Schoenefeld S, Lubeck M, Raether O, Bache N, et al (2020b) diaPASEF: parallel accumulation-serial fragmentation combined with data-independent acquisition. Nat Methods 17: 1229–1236\\n\\nMountziaris PM, Spicer PP, Kasper FK & Mikos AG (2011) Harnessing and Modulating Inflammation in Strategies for Bone Regeneration. Tissue Eng Part B Rev 17: 393\\n\\nNallagatla SR & Bevilacqua PC (2008) Nucleoside modifications modulate activation of the protein kinase PKR in an RNA structure-specific manner. RNA'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Nelson AM, Reddy SK, Ratliff TS, Hossain MZ, Katseff AS, Zhu AS, Chang E, Resnik SR, Page C, Kim D, et al (2015) dsRNA Released by Tissue Damage Activates TLR3 to Drive Skin Regeneration. Cell Stem Cell 17: 139–151\\n\\nNimiritsky P, Eremichev R, Alexandrushkina N, Efimenko A, Tkachuk V & Makarevich P (2019) Unveiling Mesenchymal Stromal Cells’ Organizing Function in Regeneration. Int J Mol Sci 20: 823\\n\\nNollet M, Santucci-Darmanin S, Breuil V, Al-Sahlanee R, Cros C, Topi M, Momier D, Samson M, Pagnotta S, Cailleteau L, et al (2014) Autophagy in osteoblasts is involved in mineralization and bone homeostasis. Autophagy 10: 1965–1977'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Owlarn S, Klenner F, Schmidt D, Rabert F, Tomasso A, Reuter H, Mulaw MA, Moritz S, Gentile L, Weidinger G, et al (2017) Generic wound signals initiate regeneration in missing-tissue contexts. Nat Commun 2017 81 8: 1–13\\n\\nPardi N, Muramatsu H, Weissman D & Karikó K (2013) In vitro transcription of long RNA containing modified nucleosides. Methods Mol Biol\\n\\nPercharde M, Lin CJ, Yin Y, Guan J, Peixoto GA, Bulut-Karslioglu A, Biechele S, Huang B, Shen X & Ramalho-Santos M (2018) A LINE1-Nucleolin Partnership Regulates Early Development and ESC Identity. Cell\\n\\nRau A, Gallopin M, Celeux G & Jaffrézic F (2013) Data-based filtering for replicated high-throughput transcriptome sequencing experiments. Bioinformatics 29: 2146–2152'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Recknagel S, Bindl R, Brochhausen C, Göckelmann M, Wehner T, Schoengraf P, Huber-Lang M, Claes L & Ignatius A (2013) Systemic inflammation induced by a thoracic trauma alters the cellular composition of the early fracture callus. J Trauma Acute Care Surg 74: 531–537\\n\\nReppe S, Refvem H, Gautvik VT, Olstad OK, Høvring PI, Reinholt FP, Holden M, Frigessi A, Jemtland R & Gautvik KM (2010) Eight genes are highly associated with BMD variation in postmenopausal Caucasian women. Bone\\n\\nRitschka B, Storer M, Mas A, Heinzmann F, Ortells MC, Morton JP, Sansom OJ, Zender L & Keyes WM (2017) The senescence-associated secretory phenotype induces cellular plasticity and tissue regeneration. Genes Dev 31: 172–183'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Roberts SJ & Ke HZ (2018) Anabolic Strategies to Augment Bone Fracture Healing. Curr Osteoporos Rep 16: 289–298\\n\\nRobinson MD, McCarthy DJ & Smyth GK (2010) edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26: 139–140\\n\\nSadeq S, Al-Hashimi S, Cusack CM & Werner A (2021) Endogenous Double-Stranded RNA. Non-Coding RNA 7: 15\\n\\nSchmidt-Bleek K, Schell H, Schulz N, Hoff P, Perka C, Buttgereit F, Volk HD, Lienau J & Duda GN (2012) Inflammatory phase of bone healing initiates the regenerative healing cascade. Cell Tissue Res 347: 567–573'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Sheng W, LaFleur MW, Nguyen TH, Chen S, Chakravarthy A, Conway JR, Li Y, Chen H, Yang H, Hsu P-H, et al (2018) LSD1 Ablation Stimulates Anti-tumor Immunity and Enables Checkpoint Blockade. Cell 174: 549-563.e19\\n\\nSimon M, Van Meter M, Ablaeva J, Ke Z, Gonzalez RS, Taguchi T, De Cecco M, Leonova KI, Kogan V, Helfand SL, et al (2019) LINE1 Derepression in Aged Wild-Type and SIRT6-Deficient Mice Drives Inflammation. Cell Metab 29: 871-885.e5\\n\\nSlotkin RK & Martienssen R (2007) Transposable elements and the epigenetic regulation of the genome. Nat Rev Genet 8: 272–285\\n\\nStrieter RM, Burdick MD, Gomperts BN, Belperio JA & Keane MP (2005) CXC chemokines in angiogenesis. Cytokine Growth Factor Rev 16: 593–609'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Timmen M, Hidding H, Wieskötter B, Baum W, Pap T, Raschke MJ, Schett G, Zwerina J & Stange R (2014) Influence of antiTNF-alpha antibody treatment on fracture healing under chronic inflammation. BMC Musculoskelet Disord 15: 184\\n\\nDella Valle F, Reddy P, Yamamoto M, Liu P, Saera-Vila A, Bensaddek D, Zhang H, Prieto Martinez J, Abassi L, Celii M, et al (2022) LINE-1 RNA causes heterochromatin erosion and is a target for amelioration of senescent phenotypes in progeroid syndromes. Sci Transl Med 14: eabl6057\\n\\nVaranasi SS, Olstad OK, Swan DC, Sanderson P, Gautvik VT, Reppe S, Francis RM, Gautvik KM & Datta HK (2010) Skeletal Site-Related Variation in Human Trabecular Bone Transcriptome and Signaling. PLoS One 5: e10692'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Wang S, Lee C, Kim J, Hyun J, Lim M, Cha H-J, Oh S-H, Choi YH & Jung Y (2017) Tumor necrosis factor-inducible gene 6 protein ameliorates chronic liver damage by promoting autophagy formation in mice. Exp Mol Med 49: e380\\n\\nWeckbach S, Hohmann C, Braumueller S, Denk S, Klohs B, Stahel PF, Gebhard F, Huber-Lang MS & Perl M (2013) Inflammatory and apoptotic alterations in serum and injured tissue after experimental polytrauma in mice: distinct early response compared with single trauma or ‘double-hit’ injury. J Trauma Acute Care Surg 74: 489–498\\n\\nWheatley BM, Nappo KE, Christensen DL, Holman AM, Brooks DI & Potter BK (2019) Effect of NSAIDs on Bone Healing Rates. J Am Acad Orthop Surg 27: e330–e336'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Wiśniewski JR, Zougman A, Nagaraj N & Mann M (2009) Universal sample preparation method for proteome analysis. Nat Methods 6: 359–362\\n\\nXiao G, Gopalakrishnan R, Jiang D, Reith E, Benson MD & Franceschi RT (2002a) Bone morphogenetic proteins, extracellular matrix, and mitogen-activated protein kinase signaling pathways are required for osteoblast-specific gene expression and differentiation in MC3T3-E1 cells. J Bone Miner Res 17: 101–110\\n\\nXiao G, Jiang D, Gopalakrishnan R & Franceschi RT (2002b) Fibroblast growth factor 2 induction of the osteocalcin gene requires MAPK activity and phosphorylation of the osteoblast transcription factor, Cbfa1/Runx2. J Biol Chem 277: 36181–36187'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Xiao G, Jiang D, Thomas P, Benson MD, Guan K, Karsenty G & Franceschi RT (2000) MAPK Pathways Activate and Phosphorylate the Osteoblast-specific Transcription Factor, Cbfa1 *. J Biol Chem 275: 4453–4459\\n\\nYang A, Lu Y, Xing J, Li Z, Yin X, Dou C, Dong S, Luo F, Xie Z, Hou T, et al (2018a) IL-8 Enhances Therapeutic Effects of BMSCs on Bone Regeneration via CXCR2-Mediated PI3k/Akt Signaling Pathway. Cell Physiol Biochem 48: 361–370\\n\\nYang H, Tian W, Wang S, Liu X, Wang Z, Hou L, Ge J, Zhang X, He Z & Wang X (2018b) TSG-6 secreted by bone marrow mesenchymal stem cells attenuates intervertebral disc degeneration by inhibiting the TLR2/NF-κB signaling pathway. Lab Investig 98: 755–772'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Yang WR, Ardeljan D, Pacyna CN, Payer LM & Burns KH (2019) SQuIRE reveals locus-specific regulation of interspersed repeat expression. Nucleic Acids Res 47: e27–e27\\n\\nZhang R, Liu Y, Yan K, Chen L, Chen X-R, Li P, Chen F-F & Jiang X-D (2013) Anti-inflammatory and immunomodulatory mechanisms of mesenchymal stem cell transplantation in experimental traumatic brain injury. J Neuroinflammation 10: 106\\n\\nZhang X, Zhang R & Yu J (2020) New Understanding of the Relevant Role of LINE-1 Retrotransposition in Human Disease and Immune Modulation. Front cell Dev Biol 8: 657'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Zhu W, Kuo D, Nathanson J, Satoh A, Pao GM, Yeo GW, Bryant S V., Voss SR, Gardiner DM & Hunter T (2012) Retrotransposon long interspersed nucleotide element-1 (LINE-1) is activated during salamander limb regeneration. Dev Growth Differ 54: 673–685\\n\\n\\n\\nFigure Legends\\n\\nFigure 1 - TEs expression is induced after fracture and in mechanically loaded bone.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='A) Heatmap representation of RNAseq differentially expressed TEs (LogFC>0.5) between 4h post-injury and intact femurs at different time points of the bone healing process. N=5 biological replicates for each time point. N=10 biological replicates for intact femur. B) Heatmap representation of RNAseq upregulated TEs (LogFC>0.5) between 4h and intact at different time points of the bone healing process. N=5 biological replicates for each time point. N=10 biological replicates for intact femur. C) Heatmap representation of RNAseq differentially expressed TEs analysis [Fragments Per Kilobase of transcript per Million (FPKM) fold change] in femoral (n=27) and iliac (n=34) bone biopsies from healthy donors. D) Pie charts showing the number and percentage of'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='donors. D) Pie charts showing the number and percentage of differentially expressed TEs subfamilies between femoral and iliac healthy bone. E) Heatmap representation of RNAseq differentially expressed TEs analysis [Fragments Per Kilobase of transcript per Million (FPKM) fold change] in femoral (n=27) and iliac (n=34) bone biopsies from healthy donors. One heatmap for each TEs order is shown.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Figure 2 - TEs expression correlates with bone mineral density in human weight-bearing bone.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='A) Heatmap representation of RNAseq differentially expressed TEs analysis [Fragments Per Kilobase of transcript per Million (FPKM) fold change] in femoral bone from healthy (n=27), osteopenic (n=12), and osteoporotic (n=9) donors. B) Heatmap representation of RNAseq differentially expressed TEs analysis [Fragments Per Kilobase of transcript per Million (FPKM) fold change] in femoral bone with high BMD (FN T-score >-1) (n=27) and low BMD (FN T-score <-1) (n=21). C) Upper panels: pie charts showing the number and percentage of differentially expressed TEs subfamilies between high BMD and low BMD femoral bone. Lower panels: heatmap representation of RNAseq differentially expressed TEs analysis [Fragments Per Kilobase of transcript per Million (FPKM) fold'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='[Fragments Per Kilobase of transcript per Million (FPKM) fold change] in femoral bone with high BMD (n=27) and low BMD (n=21). One heatmap for each TEs order is shown. D) Upper panels: Heatmap representation of correlation analysis (P value<0.05) between TEs expression (FPKM) and local BMD (FN T-score) in femoral bone biopsies (n=48). Correlation between TE expression (FPKM values) and FN t-score was evaluated using Pearson method. Significant correlations (p-value < 0.05) were selected for the heatmap representation and arranged according to the Pearson correlation coefficient (r).  r > 0, positive correlations; r < 0, negative correlations. Lower panels: Pie charts showing the percentage of TEs subfamilies positively correlated to local BMD.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Figure 3 - L1 RNA delivery stimulates the mineralization of differentiating osteoblasts'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='A) Experimental workflow and flow cytometer analysis showing the percentage of positive cells 6 hours after L1 RNA delivery at day 5 of ex vivo osteogenesis. Intracellular localization of cy5 conjugated synthetic L1 RNA (red spots) three days after transfection is also shown from a typical experiment (right). B) qPCR analysis of intracellular L1 RNA level 24 h (d6), 5 days (d10), and 9 days (d14) post-transfection. The graph is shown as mean\\u2009±\\u2009s.d. of n\\u2009=3 independent experiments. C) cy5 conjugated synthetic L1 RNA (red) and osteoimage stained mineral matrix (green) detection 9 days post L1 RNA transfection. The white arrowheads indicate the colocalization between L1 RNA and hydroxyapatite. D) Osteoimage stained mineral matrix quantification (upper panels),'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='stained mineral matrix quantification (upper panels), images (central panels), and Alizarin Red (lower panels) 9 days after L1 RNA or RFP RNA delivery in 2 healthy donors-derived MSCs (D188 and D170). RFU= Relative Fluorescence Units. The graph is shown as mean\\u2009±\\u2009s.d. of n\\u2009=\\u200911 technical replicates. ****P\\u2009<\\u20090.00005 in Wilcoxon test. E) Alizarin Red staining of osteoblasts transfected with increasing doses of RFP RNA (upper panels) and L1 RNA (lower panels). F) Alizarin Red images (upper panel) and quantification (lower panel) of MSCs mineralization after 14, 17 and 21 days of ex vivo differentiation. MSCs were obtained from the femur of 4 healthy (D188, D239, D247, D170) and 4 OP patients (HUK7, HUK9, HUK12, HUK16). N=9 technical replicates for each donor'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='HUK9, HUK12, HUK16). N=9 technical replicates for each donor and time point.  Boxes in the boxplot indicate the interquartile range (50% of data), while the lower end and the upper end represent the 1st and the 3rd quartile, respectively. The solid line inside the box represents the median. Whiskers represent the max and min values. Values that are not within 1.5 times the interquartile range are considered as outliers and lie outside the whiskers. G) Osteoimage stained mineral matrix quantification (upper panels), images (central panels), and Alizarin Red (lower panels) 9 days after L1 RNA or RFP RNA delivery in 3 OP patients derived MSCs (HUK9, HUK12, HUK16). RFU= Relative Fluorescence Units. The graph is shown as mean\\u2009±\\u2009s.d. of n\\u2009=\\u200910-12 technical'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Units. The graph is shown as mean\\u2009±\\u2009s.d. of n\\u2009=\\u200910-12 technical replicates. ****P\\u2009<\\u20090.00005 in Wilcoxon test. H) qRT-PCR of early osteogenic genes in RFP and L1 transfected osteoblasts at different time points of osteogenic differentiation. Expression level is normalized on day 5 (not transfected osteoblasts). N=3 biological replicates. The graph is shown as mean\\u2009±\\u2009s.d. of n\\u2009=\\u20093 independent experiments. *P\\u2009<\\u20090.05; **P\\u2009<\\u20090.005, ***P\\u2009<\\u20090.0005, ****P\\u2009<\\u20090.00005 in Student’s t test.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Figure 4 - L1 RNA delivery induces an inflammatory response characteristic of bone repair process'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='A) Left panel: heatmap representation of RNAseq differentially expressed gene analysis [Fragments Per Kilobase of transcript per Million (FPKM) fold change] in osteoblasts transfected with negative control RNA (RFP) or L1 RNA. Right panel: bubble plot showing gene ontology (GO) enrichment analysis of L1 upregulated biological processes. B) Tree plot showing gene ontology (GO) enrichment analysis of L1 upregulated biological processes (left) and cellular components (right). C) Top 25 upregulated biological processes 24h post-L1 RNA delivery in vitro (yellow) and 4h post fracture in vivo (blue). Shared GO terms are shown in green. D) qPCR analysis of “inflammatory response” (GO:0006954) and “immune response” (GO:0006955) genes at different time points post L1'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='response” (GO:0006955) genes at different time points post L1 RNA transfection at day 5. * **P\\u2009<\\u20090.005, ****P\\u2009<\\u20090.00005 in two way ANOVA.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Figure 5 - PKR mediates L1 RNA-induced stress response and mineralization\\n\\nA) Upper panel: schematic representation of the pathway inhibited by Lamivudine 3TC and G140. Lower panel: Alizarin Red staining of differentiating osteoblasts transfected with L1 RNA and treated with Lamivudine 3TC and G140. B) IF on RFP and L1 transfected osteoblasts, 6h post-transfection, shows the colocalization between cy5 signal (L1 RNA, red) and 488-Anti-dsRNA signal (dsRNA, green). Nuclei are stained with Hoechst (blue).'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='C)Western blot analysis showing the ratio between total eIF2a and phosphorylated eIF2a (P- eIF2a). H3: endogenous standard Histone 3. D) Volcano plot (left) showing the number of significantly upregulated and downregulated protein 24h post-L1 RNA transfection. Right: tree plots showing gene ontology (GO) enrichment analysis of biological processes (upper panel) and cellular components (lower panel) downregulated by L1 RNA delivery (MS data). E) qPCR analysis of “inflammatory response” (GO:0006954) and “immune response” (GO:0006955) genes 24h post L1 RNA transfection with and without PKR inhibitor C16. F) Alizarin Red staining of differentiating osteoblasts transfected with L1 RNA with and without PKR inhibitor C16. G) Tree plots showing gene ontology (GO)'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='PKR inhibitor C16. G) Tree plots showing gene ontology (GO) enrichment analysis of biological processes upregulated by L1 RNA 24h after transfection (MS data).'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='Figure 6 - L1-RNA-induced changes in osteoblast secretome'), Document(metadata={'source': './data/docx/EMBOJ-2023-115257.docx'}, page_content='A) Left: Alizarin red staining on recipient OB 9 days after the delivery of conditioned media. Right: Microscope images of recipient OB 24h from the delivery of conditioned media. Mineralized nodules in OB receiving conditioned media from L1 treated OB are shown (orange arrows). B) Heat map of differentially expressed proteins in bulk secretome and exosomes derived from untransfected (NT), RFP- and L1-transfected osteoblasts. N=3 biological replicates. C) GO enrichment analysis of differentially expressed protein (adjusted p-value <0.05) in the bulk secretome of L1 compared to RFP transfected osteoblasts. D) GO enrichment analysis of differentially expressed protein (adjusted p-value <0.05) in the exosomes of L1 compared to RFP transfected osteoblasts.')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Number of tokens (513) exceeded maximum context length (512).\n",
      "Number of tokens (514) exceeded maximum context length (512).\n",
      "Number of tokens (515) exceeded maximum context length (512).\n",
      "Number of tokens (516) exceeded maximum context length (512).\n",
      "Number of tokens (517) exceeded maximum context length (512).\n",
      "Number of tokens (518) exceeded maximum context length (512).\n",
      "Number of tokens (519) exceeded maximum context length (512).\n",
      "Number of tokens (520) exceeded maximum context length (512).\n",
      "Number of tokens (521) exceeded maximum context length (512).\n",
      "Number of tokens (522) exceeded maximum context length (512).\n",
      "Number of tokens (523) exceeded maximum context length (512).\n",
      "Number of tokens (524) exceeded maximum context length (512).\n",
      "Number of tokens (525) exceeded maximum context length (512).\n",
      "Number of tokens (526) exceeded maximum context length (512).\n",
      "Number of tokens (527) exceeded maximum context length (512).\n",
      "Number of tokens (528) exceeded maximum context length (512).\n",
      "Number of tokens (529) exceeded maximum context length (512).\n",
      "Number of tokens (530) exceeded maximum context length (512).\n",
      "Number of tokens (531) exceeded maximum context length (512).\n",
      "Number of tokens (532) exceeded maximum context length (512).\n",
      "Number of tokens (533) exceeded maximum context length (512).\n",
      "Number of tokens (534) exceeded maximum context length (512).\n",
      "Number of tokens (535) exceeded maximum context length (512).\n",
      "Number of tokens (536) exceeded maximum context length (512).\n",
      "Number of tokens (537) exceeded maximum context length (512).\n",
      "Number of tokens (538) exceeded maximum context length (512).\n",
      "Number of tokens (539) exceeded maximum context length (512).\n",
      "Number of tokens (540) exceeded maximum context length (512).\n",
      "Number of tokens (541) exceeded maximum context length (512).\n",
      "Number of tokens (542) exceeded maximum context length (512).\n",
      "Number of tokens (543) exceeded maximum context length (512).\n",
      "Number of tokens (544) exceeded maximum context length (512).\n",
      "Number of tokens (545) exceeded maximum context length (512).\n",
      "Number of tokens (546) exceeded maximum context length (512).\n",
      "Number of tokens (547) exceeded maximum context length (512).\n",
      "Number of tokens (548) exceeded maximum context length (512).\n",
      "Number of tokens (549) exceeded maximum context length (512).\n",
      "Number of tokens (550) exceeded maximum context length (512).\n",
      "Number of tokens (551) exceeded maximum context length (512).\n",
      "Number of tokens (552) exceeded maximum context length (512).\n",
      "Number of tokens (553) exceeded maximum context length (512).\n",
      "Number of tokens (554) exceeded maximum context length (512).\n",
      "Number of tokens (555) exceeded maximum context length (512).\n",
      "Number of tokens (556) exceeded maximum context length (512).\n",
      "Number of tokens (557) exceeded maximum context length (512).\n",
      "Number of tokens (558) exceeded maximum context length (512).\n",
      "Number of tokens (559) exceeded maximum context length (512).\n",
      "Number of tokens (560) exceeded maximum context length (512).\n",
      "Number of tokens (561) exceeded maximum context length (512).\n",
      "Number of tokens (562) exceeded maximum context length (512).\n",
      "Number of tokens (563) exceeded maximum context length (512).\n",
      "Number of tokens (564) exceeded maximum context length (512).\n",
      "Number of tokens (565) exceeded maximum context length (512).\n",
      "Number of tokens (566) exceeded maximum context length (512).\n",
      "Number of tokens (567) exceeded maximum context length (512).\n",
      "Number of tokens (568) exceeded maximum context length (512).\n",
      "Number of tokens (569) exceeded maximum context length (512).\n",
      "Number of tokens (570) exceeded maximum context length (512).\n",
      "Number of tokens (571) exceeded maximum context length (512).\n",
      "Number of tokens (572) exceeded maximum context length (512).\n",
      "Number of tokens (573) exceeded maximum context length (512).\n",
      "Number of tokens (574) exceeded maximum context length (512).\n",
      "Number of tokens (575) exceeded maximum context length (512).\n",
      "Number of tokens (576) exceeded maximum context length (512).\n",
      "Number of tokens (577) exceeded maximum context length (512).\n",
      "Number of tokens (578) exceeded maximum context length (512).\n",
      "Number of tokens (579) exceeded maximum context length (512).\n",
      "Number of tokens (580) exceeded maximum context length (512).\n",
      "Number of tokens (581) exceeded maximum context length (512).\n",
      "Number of tokens (582) exceeded maximum context length (512).\n",
      "Number of tokens (583) exceeded maximum context length (512).\n",
      "Number of tokens (584) exceeded maximum context length (512).\n",
      "Number of tokens (585) exceeded maximum context length (512).\n",
      "Number of tokens (586) exceeded maximum context length (512).\n",
      "Number of tokens (587) exceeded maximum context length (512).\n",
      "Number of tokens (588) exceeded maximum context length (512).\n",
      "Number of tokens (589) exceeded maximum context length (512).\n",
      "Number of tokens (590) exceeded maximum context length (512).\n",
      "Number of tokens (591) exceeded maximum context length (512).\n",
      "Number of tokens (592) exceeded maximum context length (512).\n",
      "Number of tokens (593) exceeded maximum context length (512).\n",
      "Number of tokens (594) exceeded maximum context length (512).\n",
      "Number of tokens (595) exceeded maximum context length (512).\n",
      "Number of tokens (596) exceeded maximum context length (512).\n",
      "Number of tokens (597) exceeded maximum context length (512).\n",
      "Number of tokens (598) exceeded maximum context length (512).\n",
      "Number of tokens (599) exceeded maximum context length (512).\n",
      "Number of tokens (600) exceeded maximum context length (512).\n",
      "Number of tokens (601) exceeded maximum context length (512).\n",
      "Number of tokens (602) exceeded maximum context length (512).\n",
      "Number of tokens (603) exceeded maximum context length (512).\n",
      "Number of tokens (604) exceeded maximum context length (512).\n",
      "Number of tokens (605) exceeded maximum context length (512).\n",
      "Number of tokens (606) exceeded maximum context length (512).\n",
      "Number of tokens (607) exceeded maximum context length (512).\n",
      "Number of tokens (608) exceeded maximum context length (512).\n",
      "Number of tokens (609) exceeded maximum context length (512).\n",
      "Number of tokens (610) exceeded maximum context length (512).\n",
      "Number of tokens (611) exceeded maximum context length (512).\n",
      "Number of tokens (612) exceeded maximum context length (512).\n",
      "Number of tokens (613) exceeded maximum context length (512).\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing document: ./data/docx/EMBOJ-2023-115537.docx\n",
      "Split documents: [Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Autophagy adaptors mediate Parkin-dependent mitophagy by forming sheet-like liquid condensates\\n\\n\\n\\nZi Yang1,a, Saori R. Yoshii1, Yuji Sakai1,2,3,b, Jing Zhang1, Haruka Chino1,c, Roland L. Knorr1,4,d, Noboru Mizushima1,5*\\n\\n\\n\\n1Department of Biochemistry and Molecular Biology, Graduate School of Medicine, The University of Tokyo, Tokyo, Japan.\\n\\n2Department of Biosystems Science, Institute for Life and Medical Sciences, Kyoto University, Sakyo-ku, Kyoto, Japan\\n\\n3Interdisciplinary Theoretical and Mathematical Sciences (iTHEMS) Program, RIKEN, Wako, Saitama, Japan\\n\\n4Institute of Biology, Humboldt-Universität zu Berlin, Berlin, Germany\\n\\n5International Research Center for Neurointelligence (WPI-IRCN), UTIAS, The University of Tokyo, Tokyo, Japan'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='aPresent address: Leibniz Forschungsinstitut für Molekulare Pharmakologie (FMP) , Berlin, Germany\\n\\nbPresent address: School of Science/Graduate School of Nanobioscience, Yokohama City University, Yokohama, Japan\\n\\ncPresent address: Department of Cell Biology, Harvard Medical school, Boston, MA, USA\\n\\ndPresent address: University of Cologne, Faculty of Medicine and University Hospital Cologne\\n\\n\\n\\n*Corresponding author. Department of Biochemistry and Molecular Biology, Graduate School of Medicine, The University of Tokyo, 7-3-1 Hongo, Bunkyo-ku, Tokyo 113-0033, Japan; Tel: +81-3-5841-3440; Fax: +81-3-3815-1490; E-mail: nmizu@m.u-tokyo.ac.jp\\n\\n\\nAbstract'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='During PINK1- and Parkin-mediated mitophagy, autophagy adaptors are recruited to damaged mitochondria to promote their selective degradation. Autophagy adaptors such as optineurin (OPTN) and NDP52 facilitate mitophagy by recruiting the autophagy-initiation machinery, and assisting engulfment of damaged mitochondria through binding to ubiquitinated mitochondrial proteins and autophagosomal ATG8 family proteins. Here, we demonstrate that OPTN and NDP52 form sheet-like phase-separated condensates with liquid-like properties on the surface of ubiquitinated mitochondria. The dynamic and liquid-like nature of OPTN condensates is important for mitophagy activity, because reducing the fluidity of OPTN-ubiquitin condensates suppresses the recruitment of ATG9'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='OPTN-ubiquitin condensates suppresses the recruitment of ATG9 vesicles and impairs mitophagy. Based on these results, we propose a dynamic liquid-like, rather than a stoichiometric, model of autophagy adaptors to explain the interactions between autophagic membranes (i.e., ATG9 vesicles and isolation membranes) and mitochondrial membranes during Parkin-mediated mitophagy. This model underscores the importance of liquid-liquid phase separation in facilitating membrane-membrane contacts, likely through the generation of capillary forces.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Keywords\\n\\nautophagy; mitophagy; liquid–liquid phase separation; optineurin; wetting\\n\\n\\n\\n\\nIntroduction'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Macroautophagy (hereafter referred to as autophagy) is an intracellular degradation process mediated by a double-membraned organelle termed the autophagosome (Nakatogawa, 2020; Mizushima & Levine, 2020). Cytosolic components such as proteins and organelles are segregated into autophagosomes, which subsequently fuse with lysosomes for degradation. Autophagy can be either non-selective or selective (Lamark & Johansen, 2021; Vargas et al, 2023). Mitophagy, which is a selective type of autophagy, specifically degrades mitochondria and can be induced by mitochondrial damage or cellular stresses (Onishi et al, 2021; Ganley & Simonsen, 2022). For example, upon mitochondrial damage, numerous mitochondrial outer-membrane proteins are ubiquitinated by the'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='mitochondrial outer-membrane proteins are ubiquitinated by the PINK1-Parkin system, initiating the recruitment of autophagy adaptors, including optineurin (OPTN), calcium binding and coiled-coil domain 2 (CALCOCO2/NDP52), sequestosome 1 (SQSTM1/p62), Next to BRCA1 gene 1 protein (NBR1), and Tax1 binding protein 1 (TAX1BP1) (Lamark & Johansen, 2021; Vargas et al, 2023). These autophagy adaptors bridge the mitochondrial and autophagosomal membranes by interacting with mitochondrial ubiquitinated proteins via a ubiquitin-binding domain (UBD) and with autophagosomal ATG8 family proteins (i.e., LC3 and GABARAP family proteins in mammals) via the LC3-interacting region (LIR). In addition to their role in supporting efficient engulfment by autophagosomes, OTPN and'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='in supporting efficient engulfment by autophagosomes, OTPN and NDP52 also play crucial roles in inducing mitophagy by recruiting upstream autophagy factors (Lazarou et al, 2015); OPTN recruits ATG9 vesicles and TBK1, which subsequently recruits the PI3K complex, whereas NDP52 recruits the ULK1 complex through interaction with FIP200 (Yamano et al, 2020, 2024; Nguyen et al, 2023; Vargas et al, 2023).'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Liquid–liquid phase separation (LLPS) has been described in various biological processes (Musacchio, 2022; Hirose et al, 2023). LLPS involves the formation of liquid-like condensates through dynamic, multivalent interactions between various molecules that often possess intrinsically disordered regions. Emerging evidence suggests connections between autophagy and LLPS at multiple steps, including autophagosome formation and degradation (Noda et al, 2020; Ma et al, 2023). Notably, autophagy adaptors, such as p62, undergo LLPS through multivalent interaction with poly-ubiquitins and form cytosolic condensates (Zaffagnini et al, 2018; Sun et al, 2018). These condensates can be degraded by autophagy, which is termed “fluidophagy” (Agudo-Canalejo et al, 2021).'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='which is termed “fluidophagy” (Agudo-Canalejo et al, 2021). Furthermore, recent studies showed that p62 undergoes LLPS on mitochondria and lysosomes that are subjected to selective autophagy (Peng et al, 2021; Gallagher & Holzbaur, 2023).'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Generally, phase-separated condensates can deform when they contact a rigid surface in order to minimize the overall energy of the system, a phenomenon known as “wetting” (Kusumaatmaja et al, 2021b; Gouveia et al, 2022). The wetting behavior of intracellular condensates is determined by the interfacial tensions between the condensate–membrane, condensate–cytosol, and cytosol-membrane interfaces. Depending on the relative strengths of these interfacial tensions, the condensates exhibit partial wetting, complete wetting, or de-wetting. In contrast, elastic surfaces, including membranes to which phase-separated condensates adhere, also deform (Kusumaatmaja et al, 2021a, 2021b; Gouveia et al, 2022; Mangiarotti et al, 2023). This indeed happens during'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='al, 2022; Mangiarotti et al, 2023). This indeed happens during fluidophagy; autophagosomal membranes bend along the surface of p62 condensates (Agudo-Canalejo et al, 2021). Deformation of a membrane by a phase-separated condensate is determined by the energy balance between membrane deformation and the surface tension of the condensate, which is known as elastocapillarity (Style et al, 2017; Kusumaatmaja et al, 2021b; Mangiarotti et al, 2023).'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='By forming capillary bridges, wetting droplets can also induce adhesion of surfaces (Wexler et al, 2014). Whether LLPS and capillary bridges also contribute to the adhesion of cellular membranes is not presently known. Therefore, we hypothesized that phase-separated condensates of autophagy adaptors can wet both mitochondria and autophagic membranes, including ATG9 vesicles and isolation membranes (also called phagophores), thereby promoting their contact during Parkin-mediated mitophagy. In this study, we employ live-cell imaging and a mathematical model to provide evidence that the autophagy adaptors form phase-separated condensates on the surface of the mitochondrial membrane, resulting in sheet-like condensates that cover the surface of ubiquitinated'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='sheet-like condensates that cover the surface of ubiquitinated mitochondria upon mitophagy induction. These condensates accumulate between mitochondria or between mitochondria and autophagic membranes, exhibiting a dynamic nature with a liquid-like property. Furthermore, we demonstrate the essential role of these dynamic condensates in the recruitment of ATG9 vesicles and the initiation of mitophagy. Based on these results, we propose a dynamic liquid-like model, rather than a stoichiometric model, to describe the roles of autophagy adaptors in Parkin-mediated mitophagy.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Results\\n\\n\\n\\nAutophagy adaptors show distinct distributions during Parkin-mediated mitophagy'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='To better understand the role of each autophagy adaptor, we analyzed their localization during Parkin-mediated mitophagy. When mitophagy was induced by treatment with the mitochondrial uncoupler carbonyl cyanide m-chlorophenylhydrazone (CCCP) in HeLa cells expressing exogenous Parkin, the autophagy adaptors p62, OPTN, NBR1, NDP52, and TAX1BP1 translocated to mitochondria as previously observed (Fig. 1A and B) (Geisler et al, 2010; Wong & Holzbaur, 2014; Heo et al, 2015; Moore & Holzbaur, 2016; Gallagher & Holzbaur, 2023). Ubiquitin and all of these autophagy adaptors were distributed evenly on the surface of separate mitochondria (Mt; Fig. 1A). In contrast, these adaptors exhibited distinct localization patterns during the formation of isolation membranes'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='patterns during the formation of isolation membranes on mitochondria (Mt–IM; Fig. 1B and C). The signals of OPTN and NDP52 were enriched in areas where LC3B signals colocalized, while they were mostly absent from the LC3B-negative side of the same mitochondria (Fig. 1B and C). Endogenous OPTN and NDP52 also showed clear enrichment on the LC3B-positive areas in comparison to the LC3B-negative side, confirming that this localization is not caused by the overexpression of adaptors (Fig. 1D). However, this inhomogeneous distribution pattern was not observed for ubiquitin, p62, NBR1, and TAX1BP1 (Fig. 1B and C). Unlike OPTN and NDP52, p62 showed enrichment between clustered mitochondria (Mt-Mt), which is consistent with previous reports (Fig. EV1A and B) (Wong &'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='is consistent with previous reports (Fig. EV1A and B) (Wong & Holzbaur, 2014), which appears to be consistent with the role of p62 in promoting mitochondrial clustering (Narendra et al, 2010; Okatsu et al, 2010). These accumulation patterns cannot be explained by the stoichiometric interaction of the autophagy adaptors with ubiquitinated proteins that are evenly distributed on mitochondria.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='The localization of autophagy adaptors may be affected by their interaction with other adaptors (Turco et al, 2021; Gubas & Dikic, 2022). To examine the localization of each adaptor on its own, we used HeLa cells lacking all five autophagy adaptors (penta KO cells) to exclude the effect of other adaptors (Lazarou et al, 2015). In penta KO cells, OPTN and NDP52 exhibited significant enrichment in LC3B-positive areas, similar to that observed in wild-type cells, suggesting that this accumulation occurred independently of heterologous interaction with other adaptors (Fig. 1E and F). Mitophagy was not restored by exogenous expression of p62 or NBR1 in penta KO cells, and thus colocalization with LC3 could not be tested for these two adaptors (Lazarou et al,'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='LC3 could not be tested for these two adaptors (Lazarou et al, 2015). The accumulation of p62 between clustered mitochondria was also observed in penta KO cells (Fig. EV1C and D). Given the uniform distribution of ubiquitinated proteins on mitochondrial surfaces, these data suggest that autophagy adaptors exhibit non-stoichiometric enrichment between membranes. Hereafter, we used penta KO cells to study each autophagy adapter individually.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='OPTN and NDP52 show a dynamic exchange between the mitochondrial surface and cytosol'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='We hypothesized that the non-stoichiometric distribution of the autophagy adaptors on ubiquitinated mitochondria could be explained by LLPS. Typical condensates generated by LLPS exhibit rapid exchange of their components, which is often demonstrated by fluorescence recovery after photobleaching (FRAP) analysis (Taylor et al, 2019; McSwiggen et al, 2019). We first conducted FRAP experiments for separate (i.e., unclustered) mitochondria in CCCP-treated cells. As expected, ubiquitin showed virtually no recovery because it was conjugated to mitochondrial membrane proteins (Fig. 2A and B). Additionally, p62 recovered only slightly, suggesting a minute exchange between the mitochondrial surface and cytosol (Fig. 2A and B). In contrast, OPTN and NDP52 showed a'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='cytosol (Fig. 2A and B). In contrast, OPTN and NDP52 showed a rapid recovery, indicating a dynamic exchange of these adaptors between the mitochondrial surface and the cytosol (Fig. 2A and B).'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Next, we examined the dynamics of adaptors between mitochondria and isolation membranes. OPTN and NDP52 showed partial recovery (Fig. 2C and D). Although this result indicates some exchange of these adaptors between membranes, the incomplete recovery suggests the existence of a gel-like or immobile fraction. Treatment with 1,6-hexanediol, which can be used to dissolve condensates, dispersed OPTN and NDP52, but not p62, from mitochondria without isolation membranes (Fig. 2E, EV2A). In contrast, 2,5-hexanediol, which is considered by some studies as a negative control for 1,6-hexanediol, did not disperse the OPTN or NDP52 condensates (Fig. EV2B). These data suggest that OPTN and NDP52 accumulation is supported by weak interactions. In contrast, OPTN and'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='is supported by weak interactions. In contrast, OPTN and NDP52 at the mitochondria-isolation membrane contact sites remained unaffected by 1,6-hexanediol (Fig. EV2C). This may be due to the additional interaction with ATG8 proteins, not only with ubiquitin. These results indicate that OPTN and NDP52 on ubiquitinated mitochondria are mobile and can quickly exchange between the mitochondrial surface and the cytosol, supporting the hypothesis that they form phase-separated condensates on damaged mitochondria.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Mathematical models of condensate formation and localization of autophagy adaptors'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='To simulate the behavior of autophagy adaptors in the context of phase-separated condensates, we developed mathematical models (see Materials and Methods for details). First, we considered the case of two mitochondria approaching each other (Fig. 3A). In Parkin-mediated mitophagy, various proteins in the mitochondrial outer membrane are ubiquitinated. We modeled the mitochondrial outer membrane as a circle with a diameter of  nm. On these mitochondria, we assumed a ring-shaped ubiquitin layer with a thickness  nm and area-fraction (hereafter referred to as concentration)  (Milo & Phillips, 2015; Berry et al, 2018) (Fig. 3A). The closest distance between the two mitochondrial outer membranes was set to  (Fig. 3A). Here, we considered the dynamics of'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='was set to  (Fig. 3A). Here, we considered the dynamics of concentration changes of the adapter proteins and solvent components. We assumed that mitochondria were very large compared with proteins and did not move on a similarly short time scale, and that ubiquitin was tightly bound to mitochondrial membranes and immobile, as suggested by the FRAP experiment (Fig. 2B). The adaptor proteins were assumed to diffuse freely in solution and to bind to ubiquitin and themselves (self-interactions) with strengths  and , respectively. The adaptors showed accumulation and distribution on mitochondria similarly to what was observed in cells (Figure EV1) when   and  (Figure. EV3A), with the Boltzmann constant  and temperature . Therefore, we used   and  for further'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='and temperature . Therefore, we used   and  for further analyses.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Using the mathematical model, we assessed how the condensates deformed when the distance between two mitochondria  was changed. When two mitochondria were far apart ( nm), the model predicted that adaptors cover the entire mitochondrial surface (Fig. 3B). When the mitochondria were close enough to come into contact ( nm), the adaptors moved to the area between the mitochondria and filled the cleft between them (arrowheads, Fig. 3B).'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Next, we considered the case in which a growing isolation membrane surrounded and enclosed a single mitochondrion (Fig. 3C and D). A mitochondrion with ubiquitin was modeled as mentioned above, and the isolation membrane was modeled as a region of thickness  nm bound by two semicircles with the same center as the mitochondrion. The closest distance between the isolation membrane and the mitochondrial outer membrane was  nm, and the isolation membrane bent at an angle  to surround the mitochondrion. The surface of the isolation membrane was assumed to be covered by an LC3 region with a thickness  nm and concentration . The adaptors interacted with both LC3 and ubiquitin with strengths  and , respectively. A clear accumulation of the autophagy adaptor in the'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='A clear accumulation of the autophagy adaptor in the region in contact with the isolation membrane, along with depletion from the non-contacting region, was observed at  (Figure EV3B), similar to the OPTN distribution observed in cells (Fig. 1B). Therefore, we used  for further analyses.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Using this model, we determined the distribution of the adaptors when the isolation membrane appeared and elongated (changing the angle  surrounding the mitochondrion). In the absence of the isolation membrane, the adaptors wet the entire mitochondrial surface uniformly (Fig. 3D). However, with the appearance of the isolation membrane, the adaptors remobilized to the area between the isolation membrane and mitochondrion, and this adaptor-enriched region elongated together with the isolation membrane. These modeling results suggest that the distribution of the adaptors on a mitochondrion can change upon contact with another mitochondrion or an isolation membrane.\\n\\n\\n\\nOPTN condensates redistribute upon membrane contact'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Our mathematical models predict that the OPTN distribution changes when mitochondria cluster or are engulfed by isolation membranes. We then validated these predictions by using live-cell imaging. When two small mitochondria with uniform OPTN distributions on their surface approached each other, OPTN redistributed and formed a smooth surface that covered the two adjacent mitochondria (Fig. 4A–D, Movie EV1). Notably, strong OPTN enrichment was observed in the cleft between the two mitochondria (arrowheads, Fig. 4A and C), showing a pattern distinct from that of the outer membrane protein Omp25 (OPTN signal peaks localized outside of Omp25 signal peaks) (Fig. 4B and D). This phenomenon was apparent when the size of both (Fig. 4A) or one (Fig. 4C, Movie EV1)'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='when the size of both (Fig. 4A) or one (Fig. 4C, Movie EV1) of the two mitochondria was less than 1 µm. Although OPTN formed sheet-like rather than spherical condensates, this phenomenon appeared to be similar to condensate coalescence, which is one of the hallmarks of liquid-like condensates (Hyman et al, 2014).'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Furthermore, when an isolation membrane started to engulf a mitochondrion, OPTN, which was initially distributed uniformly on the mitochondrial surface, underwent redistribution to the area contacting the isolation membrane (Fig. 4E, Movie EV2). OPTN signals on the isolation membrane-negative side of the mitochondrion diminished during this process (Fig. 4E and F). The OPTN-enriched region expanded together with the isolation membrane thereafter (Fig. 4E). These live-cell observations are consistent with the predictions of our mathematical models (Fig. 3). Taken together, our experimental data support the hypothesis that OPTN forms sheet-like phase-separated condensates on the surface of ubiquitinated mitochondria, exhibiting liquid-like properties.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Liquid-like property of OPTN condensates is required for mitophagy'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='To investigate the importance of the liquid-like nature of OPTN condensates on mitochondria, we sought to reduce their fluidity by strengthening the interaction between ubiquitin and GFP–OPTN constructs. To achieve this, we utilized the anti-green fluorescent protein (GFP) nanobody, which exhibits a high binding affinity to GFP (Rothbauer et al, 2008) (Fig. 5A). Consequently, the interaction between anti-GFP nanobody-fused ubiquitin and GFP–OPTN was stronger than that between ubiquitin and GFP–OPTN (Fig. 5B). Expression of anti-GFP nanobody-fused ubiquitin (nanobody–mRuby–Ub) did not affect the recruitment of GFP–OPTN to mitochondria (Fig. 5A, (i) and (ii)) but partially reduced its FRAP recovery (Fig. 5C), suggesting a reduction in mobility. We then'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='recovery (Fig. 5C), suggesting a reduction in mobility. We then evaluated the effect of reduced OPTN fluidity on mitophagy using the recently developed HaloTag cleavage assay (Yim et al, 2022). This assay utilizes the ligand-dependent conformational change of HaloTag (Halo); ligand-free Halo is efficiently degraded in lysosomes, whereas ligand-bound Halo becomes resistant to lysosomal degradation. When Halo is expressed in the mitochondrial matrix by fusing to the mitochondrial presequence of Fo-ATPase subunit 9 and SNAP-tag (mtHalo–SNAP), we can measure the extent of mitochondrial degradation by quantifying the amount of processed free Halo out of the total amount of Halo (mtHalo–SNAP + processed Halo). Mitophagy activity, which was impaired in penta KO'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Halo). Mitophagy activity, which was impaired in penta KO cells, was restored by the expression of GFP–OPTN (Fig. 5D and E). Mitophagy activity was partially reduced in cells expressing GFP-OPTN and nanobody–mRuby–Ub (Fig. 5D and E).'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='The partial reduction in both FRAP recovery and mitophagy activity may be attributed to the remaining interaction between OPTN and endogenous ubiquitin. Therefore, we deleted the ubiquitin-binding domain in ABINs and NEMO (UBAN domain) from OPTN (GFP–OPTNΔUBAN), abolishing its interaction with ubiquitinated proteins (Fig. 5B). GFP–OPTNΔUBAN failed to accumulate on mitochondria in the presence of mRuby–Ub (Fig. 5A, (iii)). In contrast, the expression of nanobody–mRuby–Ub rescued the recruitment of GFP–OPTNΔUBAN to mitochondria (Fig. 5A, (iv)), albeit to a reduced extent, likely reflecting the loss of condensate formation. FRAP experiments revealed that, compared with wild-type GFP–OPTN (Fig. 5A, (i)), GFP–OPTNΔUBAN (Fig. 5A, (iv)) showed a substantial'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='5A, (i)), GFP–OPTNΔUBAN (Fig. 5A, (iv)) showed a substantial decrease in recovery on mitochondria when co-expressed with nanobody–mRuby–Ub, indicating a reduction in its dynamic properties (Fig. 5C). GFP–OPTNΔUBAN failed to restore mitophagy, even when it was co-expressed with nanobody–mRuby–Ub to rescue ubiquitin binding and mitochondrial localization (Fig. 5A, B, D, and E). This impairment was indeed due to the loss of binding to ubiquitin because a point mutation that abolished ubiquitin binding (E478G) (Li et al, 2018) also showed normal mitochondrial localization, low FRAP recovery, and mitophagy defect (Fig. 5D, E, EV4A, and B). Furthermore, the impaired mitophagy was not simply due to the altered binding orientation of OPTN. Replacement of its UBAN'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='altered binding orientation of OPTN. Replacement of its UBAN domain with GFP (OPTNΔUBAN–GFP), mimicking its binding to ubiquitin at the C-terminus, did not restore FRAP recovery or mitophagy activity, although it also showed clear recruitment to mitochondria in the presence of nanobody–mRuby–Ub (Fig. EV4A, B, C and D). The restricted orientation of OPTN due to the stoichiometric interaction between GFP and nanobody, as opposed to the more flexible orientation in liquid-like condensates, may also contribute to the impairment of autophagy.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Consistent with the results of mitophagy activity, the accumulation of the ULK1 complex component FIP200, likely representing isolation membranes, was observed in the presence of mRuby–Ub and GFP–OPTN (Fig. EV5A). In contrast, no clear FIP200 accumulation was observed in cells expressing nanobody–mRuby–Ub and GFP–OPTNΔUBAN, suggesting that mitophagy failed at the initiation step in these cells (Fig. EV5A). Together, these data suggest that the localization of OPTN on mitochondria and its binding to ubiquitin alone is insufficient for mitophagy, and the fluidity of OPTN is important for effective mitophagy induction.\\n\\n\\n\\nLiquid-like property of OPTN condensates is required for initiation of mitophagy by recruiting ATG9 vesicles and activating TBK1'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='ATG9 vesicles are considered to be one of the sources of autophagosomal membranes (Yamamoto et al, 2012). Upon mitophagy induction, ATG9 vesicles accumulate at the sites of autophagosome formation (Itakura et al, 2012). OPTN plays a crucial role in this process by recruiting ATG9 vesicles to induce mitophagy (Yamano et al, 2020). In general, liquid-like condensates can be involved in clustering small vesicles such as synaptic vesicles, which exemplify complete wetting (Milovanovic & De Camilli, 2017; Sansevrino et al, 2023). Indeed, ATG9 vesicles are incorporated into condensates containing a glaucoma-associated OPTN mutant (O’Loughlin et al, 2020) and synapsin (Park et al, 2023). We, therefore, hypothesized that the liquid-like properties of OPTN'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='therefore, hypothesized that the liquid-like properties of OPTN condensates on mitochondria facilitate the recruitment of ATG9 vesicles. Consistent with previous reports (Itakura et al, 2012; Yamano et al, 2020), ATG9A was recruited to depolarized mitochondria in cells expressing GFP–OPTN (Fig. 6A and B). However, this ATG9A recruitment was almost completely abolished in cells expressing nanobody–mRuby–Ub and GFP–OPTNΔUBAN (Fig. 6A and B), even though GFP–OPTNΔUBAN still retains the ATG9A-interacting domain and the ability to bind with the ATG9 vesicles (Fig. 6C and D). These data suggest that the fluidity of sheet-like OPTN condensates on the mitochondrial surface is crucial for recruiting ATG9 vesicles and executing mitophagy.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Furthermore, TBK1 is recruited by OPTN to the mitophagy initiation site to be phosphorylated and activated, driving the feed-forward mechanism of OPTN accumulation and TBK1 activation, which in turn phosphorylates the PI3K complex to initiate mitophagy (Nguyen et al, 2023; Yamano et al, 2024). Consistently, a clear accumulation of phosphorylated TBK1 on mitochondria was observed in the presence of GFP–OPTN and mRuby–Ub (Fig. EV5B). In contrast, only very faint signals of phosphorylated TBK1 were observed in the presence of nanobody–mRuby–Ub and GFP–OPTNΔUBAN (Fig. EV5B). These data suggest that the fluidity of OPTN is also important for the feed-forward mechanism between OPTN and TBK1 for the initiation of mitophagy. Together, we propose that autophagy'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='initiation of mitophagy. Together, we propose that autophagy adaptors phase separate on ubiquitinated mitochondria to amplify the initiation signals and wet the ATG9 vesicles to facilitate efficient mitophagy.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Discussion\\n\\n\\n\\nLLPS by autophagy adaptors during Parkin-mediated mitophagy'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='In the present study, we discovered that autophagy adaptors accumulate on ubiquitinated mitochondria with non-stoichiometric distribution compared with ubiquitin signals. They displayed characteristics consistent with phase-separated structures, including dynamic exchange with cytosol and redistribution upon coalescence. Notably, we observed the enrichment of OPTN at the isolation membrane-positive region, which expanded along with the elongation of the isolation membrane. These findings suggest that phase-separated adaptors facilitate the effective engulfment of mitochondria by the isolation membrane through the wetting effect and capillary forces (Fig. 6E, right panel). Moreover, the fluidity of OPTN condensates is critical for the localization of ATG9'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='of OPTN condensates is critical for the localization of ATG9 vesicles onto ubiquitinated mitochondria, indicating that partial or complete wetting of phase-separated adaptors to ATG9 vesicles may be crucial for their recruitment to and/or retention on mitochondria (Fig. 6E, left panel). Therefore, we propose that LLPS of autophagy adaptors plays a critical role in two distinct steps of mitophagy by producing capillary forces, first, at the initiation step by recruiting and retaining the ATG9 vesicles in the condensates and, second, at the autophagosome elongation step by facilitating the attachment between ubiquitinated mitochondria and the isolation membrane. Additionally, our experimental data suggest that NDP52 also exhibits liquid-like properties.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='data suggest that NDP52 also exhibits liquid-like properties. Further studies are needed to determine whether condensate formation is necessary for mitophagy induction by NDP52.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='LLPS in bulk and selective autophagy'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Recent reports have shed light on the importance of LLPS in selective autophagy (Noda et al, 2020). In lysophagy, p62 undergoes LLPS on lysosomes to facilitate efficient lysophagy together with HSP27, which maintains the fluidity of p62 and prevents its gelation (Gallagher & Holzbaur, 2023). This may also be mediated by the wetting between damaged lysosomes and the isolation membrane by phase-separated p62. Indeed, p62 itself forms phase-separated condensates along with ubiquitinated proteins, producing capillary forces by the wetting effect to facilitate the engulfment by the isolation membrane (Agudo-Canalejo et al, 2021). In our study, p62 exhibited low mobility, as determined by FRAP and 1,6-hexanediol experiments (Fig. 2B and EV2A). While these data'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='1,6-hexanediol experiments (Fig. 2B and EV2A). While these data may suggest that p62 is immobile, they appear to contradict the observed redistribution of p62 to the area between mitochondria upon clustering (Fig. EV1A and B). Thus, we speculate that p62 may exhibit some degree of fluidity despite its low exchange between the condensate and cytosol. Similarly to p62, Ape1, the cargo of the cytosol-to-vacuole targeting (Cvt) pathway in yeast, forms gel-like condensates with Atg19 on the surface of Ape1 condensates to be sequestered by the isolation membrane (Yamasaki et al, 2020). This association could also be mediated by partial wetting. These examples together with our observations highlight the importance of LLPS and the wetting phenomena in various'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='the importance of LLPS and the wetting phenomena in various selective autophagy processes.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='LLPS may also be important for non-selective bulk autophagy. In yeast, the Atg1 complex (the ULK1 complex in mammals) forms pre-autophagosomal structures (PASs), which have been shown to be condensates driven by LLPS (Fujioka et al, 2020). Phase separation in this context provides a high local concentration of the Atg1 complex to support the autoactivation of Atg1 kinase (Yamamoto et al, 2016). Mammalian autophagy factors also accumulate at the autophagosome formation site; FIP200, a subunit of the autophagy-initiation complex, undergoes LLPS triggered by calcium transients (Zheng et al, 2022). The liquid-like nature of these autophagy-initiating structures may also be important for the recruitment of Atg9/ATG9 vesicles through the wetting effect.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='The importance of phase separation during mitophagy'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='In contrast to Parkin-mediated mitophagy, hypoxia-induced mitophagy does not involve autophagy adaptors, and instead, the outer mitochondrial membrane proteins NIX (also known as BNIP3L) and BNIP3 directly interact with ATG8 homologs on the isolation membrane (Novak et al, 2010; Onishi et al, 2021). In this context, ATG8 proteins interact with NIX or BNIP3, likely in a one-to-one manner. This raises the question of why phase separation of autophagy adaptors is essential for Parkin-mediated mitophagy. One possibility is that the quick induction and execution of Parkin-mediated mitophagy requires amplification of the reactions through increased local concentration of adaptors and autophagy-related proteins facilitated by LLPS. This is consistent with the fact'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='proteins facilitated by LLPS. This is consistent with the fact that mitophagy-inducing signals are also amplified by a mechanism involving phosphorylated ubiquitin (Kane et al, 2014; Kazlauskaite et al, 2014; Koyano et al, 2014) and OPTN-TBK1 axis (Nguyen et al, 2023; Yamano et al, 2024). Indeed, we observed that the amount of OPTN recruited to mitochondria was greater with wild-type OPTN than with OPTNΔUBAN (in the presence of nanobody-Ub), likely reflecting the enhanced recruitment by condensate formation (the reduced recruitment of OPTNΔUBAN may also be due to a lower expression level of nanobody–mRuby–Ub compared to abundant endogenous ubiquitin, which may further contribute to reduced mitophagy activity) (Fig. 5A and EV4A). Alternatively,'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='reduced mitophagy activity) (Fig. 5A and EV4A). Alternatively, Parkin-mediated mitophagy requires a high level of precision and selectivity; mitophagy should be induced only for damaged mitochondria, which is ensured by the recruitment and retention of ATG9 vesicles only on phase-separated adaptor-positive mitochondria. This strict specificity would enable proper quality control of the mitochondria.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Materials and Methods\\n\\n\\n\\nCell culture and generation of stable cell lines'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='HeLa cells, HeLa penta-knockout (penta KO) cells, and human embryonic kidney (HEK) 293T cells authenticated by RIKEN were cultured in Dulbecco’s modified Eagle’s medium (DMEM; D6546; Sigma-Aldrich) supplemented with 10% fetal bovine serum (FBS; 173012; Sigma-Aldrich) and 2 mM l-glutamine (25030-081; Gibco) in a 5% CO2 incubator at 37°C. For stable expression, retrovirus was produced with HEK293T cells transfected with pMRX-IP-based, pMRX-IB-based, pMRX-IU-based, or pMRX-IN-based retroviral plasmids, pCG-VSV-G, and pCG-gag-pol by using Lipofectamine 2000 (11668019; Thermo Fisher Scientific). After transfection, cells were further incubated at 37°C for 24h. The viral supernatant was collected by filtration through a 0.45-μm filter unit (Ultrafree-MC;'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='by filtration through a 0.45-μm filter unit (Ultrafree-MC; Millipore) and then used for infection. Cells were plated onto 6-cm dishes 18 h before infection, and the medium was replaced with viral supernatant diluted 1.5-fold with 8 μg/mL polybrene (H9268; Sigma-Aldrich). Two days later, cells were selected in a medium containing 2 μg/mL puromycin (P8833; Sigma-Aldrich), 4 μg/mL blasticidin S hydrochloride (022-18713; FUJIFILM Wako Pure Chemical Corporation), 1.5 mg/mL neomycin, or 250 μg/mL zeocin (R25005; Thermo Fisher Scientific).'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Plasmids'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='The pMRX-IPU, pMRX-IBU and pMRX-INU plasmids were generated by modifying the multi-cloning site of pMRX-IP (Saitoh et al, 2002; Kitamura et al, 2003), pMRX-IB, and pMRX-IN, respectively. DNA fragments encoding ubiquitin, p62 (NM_003900.5), OPTN (NM_001008211.1), NBR1 (NM_001291571.2), NDP52 (NM_001261391.2), TAX1BP1 (NM_001206901.1), ATG9A (NM_001077198.3), LC3A (NM_032514.4), and LC3B (NM_022818.5) were inserted into pMRX-IPU, pMRX-INU, or pMRX-IBU. DNAs encoding the HA epitope, monomeric enhanced GFP with the A206K mutation (mGFP), codon-optimized ultra-stable GFP (muGFP) (Scott et al, 2018), codon-optimized mRuby3 (Bajar et al, 2016), HaloTag7 (G1891; Promega), 3×FLAG, and SNAP-tag (New England BioLabs, N9181S) were used for tagging. The mitochondrial'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='BioLabs, N9181S) were used for tagging. The mitochondrial presequence of Neurospora crassa Fo-ATPase subunit 9 (residues 1–69) was added to HaloTag7-SNAP to deliver the reporter into the mitochondrial matrix (mtHalo-SNAP) (Eura, 2003). Truncated OPTN (OPTNΔUBAN, aa 445–502) was prepared by PCR-mediated site-directed mutagenesis. The resulting plasmids were sequenced.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Antibodies and reagents'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='The primary antibodies used in this study were rabbit polyclonal anti-OPTN (Proteintech, 10837-AP), rabbit polyclonal anti-ATG9A (MBL, PD042), mouse monoclonal anti-Halo (Promega, G9211), mouse monoclonal anti-HSP90 (BD Transduction Laboratories, 610419), rabbit polyclonal anti-GFP (Thermo Fisher Scientific, A6455), rabbit polyclonal anti-NDP52 (Proteintech, 12229-1-AP), rabbit polyclonal anti-RB1CC1 (Proteintech, 17250-1-AP), and rabbit monoclonal anti-phospho-TBK1 (Ser172) (Cell Signaling Technology, 5483) antibodies. The secondary antibodies used were HRP-conjugated goat polyclonal anti-rabbit IgG (Jackson ImmunoResearch Laboratories, 111-035-144) and HRP-conjugated goat polyclonal anti-mouse IgG (Jackson ImmunoResearch Laboratories, 115-035-003)'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='IgG (Jackson ImmunoResearch Laboratories, 115-035-003) antibodies. To induce mitophagy, HeLa cells were treated with 20 μM carbonyl cyanide m-chlorophenyl hydrazine (CCCP; Sigma-Aldrich) for 45 min, or 10 μM oligomycin (Cabiochem, 495455-10MGCN) and 4 μM antimycin A (Sigma-Aldrich, A8674) for 18 h. After cells had been treated with oligomycin and antimycin A for >6 h, 10 μM Q-VD-OPH (SM Biochemicals, SMPH001) was added to block apoptotic cell death. To inhibit the formation of isolation membranes in the 1,6-hexanediol (Sigma-Aldrich, 240117-50G) treatment experiment, wortmannin (Sigma-Aldrich, W1628-1MG) was added into the medium.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Immunoprecipitation and immunoblotting'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Cell lysates were prepared in a lysis buffer (50 mM Tris-HCl pH 7.4, 150 mM NaCl, 1 mM EDTA, 1% Triton X-100, EDTA-free protease inhibitor cocktail [19543200; Roche]). After centrifugation at 17,700 × g for 10 min, the supernatants were subjected to immunoprecipitation using anti-FLAG M2 magnetic beads (M8823-1ML; Sigma-Aldrich). Precipitated immunocomplexes were washed three times with lysis buffer and boiled in sample buffer (46.7 mM Tris-HCl, pH 6.8, 5% glycerol, 1.67% sodium dodecyl sulfate, 1.55% dithiothreitol, and 0.02% bromophenol blue). For immunoprecipitation of ATG9 vesicles, cells were disrupted by Dounce homogenization with hypotonic lysis buffer (10 mM HEPES, pH 7.9, 1.5 mM MgCl2 and 10 mM KCl, EDTA-free protease inhibitor cocktail [19543200;'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='and 10 mM KCl, EDTA-free protease inhibitor cocktail [19543200; Roche]). Disruption was carried out by applying 35 strokes while the cell suspension was cooled on ice. The supernatants were subjected to immunoprecipitation using anti-FLAG M2 magnetic beads (M8823-1ML; Sigma-Aldrich). For immunoblotting, the samples were separated by SDS-PAGE and transferred to Immobilon-P polyvinylidene difluoride membranes (Millipore, WBKLS0500) with the Trans-Blot Turbo Transfer System (Bio-Rad). After incubation with the relevant antibody in 5% skim milk in 20 mM Tris-HCl, 150 mM NaCl, and 0.1% Tween 20 (02194841-CF; MP Biomedicals), the signals from incubation with SuperSignal West Pico Chemiluminescent Substrate (Thermo Fisher Scientific, 34579) were detected with the'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='(Thermo Fisher Scientific, 34579) were detected with the FUSION SOLO.7S.EDGE imaging system (Vilber-Lourmat). Contrast and brightness adjustment and quantification were performed using the image processing software Fiji (Schindelin et al, 2012).'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Fluorescence recovery after photobleaching'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='In-cell fluorescence recovery after photobleaching (FRAP) analyses were performed with an Olympus Fluoview FV3000 confocal microscope equipped with a 60×\\u2009oil-immersion objective lens (1.40 NA, Olympus). The chamber was maintained at 37°C and continuously supplied with humidified 5% CO2. Bleaching was performed using 80% laser power (488 or 561\\u2009nm laser), and images were captured every 5\\u2009s for 30 frames. Recovery curves and fitting were analyzed using OriginPro 2022. Fluorescence intensity of the bleached spot, an unbleached control spot, and the background was measured using the software package Fiji. Background intensity was subtracted, and the intensity values of the region of interest are reported relative to the pre-bleached images during image'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='are reported relative to the pre-bleached images during image acquisition. Each data point represents the mean and standard error of fluorescence intensities in more than three unbleached (control) or bleached (experimental) spots.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Live-cell imaging'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Living imaging was conducted with the Olympus SpinSR10 spinning-disk confocal super-resolution microscope equipped with a Hamamatsu ORCA-Flash 4.0 camera, a UPLAPO OHR 100 × (NA 1.50) lens, and the SORA disk in place. The microscope was operated with Olympus cellSens Dimension 2.3 software. Cells were passaged onto a four-chamber glass-bottom dish (Greiner Bio-One) more than 24 h before imaging. To induce mitophagy, cells were incubated with 10 μM CCCP in the presence of 200 nM SF650-conjugated Halo ligand (GoryoChemical, A308-02). Images were processed using the image processing software package Fiji (Schindelin et al, 2012). Fluorescence intensity of indicated fluorophores was measured by Fiji, and spline connected graphs were created using OriginPro 2022'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='and spline connected graphs were created using OriginPro 2022 software.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Immunofluorescence imaging\\n\\nCells were fixed with 4% PFA in 0.1\\u2009M phosphate buffer (pH 7.3) for 15\\u2009min at room temperature (RT) and washed with PBS. Fixed cells were incubated with 10 µg/mL digitonin for 5 min at RT and washed with PBS. Then, cells were incubated with primary antibodies diluted (1:1000) in blocking buffer (3% BSA, in PBS) for 1\\u2009h at RT, washed with PBS, incubated with Alexa Fluor-conjugated secondary antibodies (1:1000) in blocking buffer for 60\\u2009min at RT, washed again, and mounted on coverslips. Colocalization was calculated by Fiji plugins BIOP JACoP, with the threshold of the ATG9A channel fixed to 500. ATG9A-positive areas out of GFP-positive areas were calculated to obtain the overlapping areas.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Mathematical model of autophagy adaptor condensate formation\\n\\nWe formulated a mathematical model of autophagy adaptor condensate formation. For the case of two mitochondria approaching each other (Fig. 3A), the free energy, , of the system can be written as\\n\\n\\n\\nwith the interaction energy, , and the entropic energy, , expressed as\\n\\n\\n\\n\\n\\nwhere , , and  are the adaptor, ubiquitin, and solvent concentrations, respectively,  and  represent the strength of the adaptor self-interaction and adaptor–ubiquitin interaction, respectively,  represents the surface tension of the condensates, and  represents the mitochondrial area exclusion effect, introduced to prevent proteins from entering the mitochondrial interior ().'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='For the case in which an isolation membrane surrounded a mitochondrion (Fig. 3C), the interaction energy, , is instead expressed as\\n\\n\\n\\nwhere  and  represent the LC3 concentrations and the strength of the adaptor–LC3 interaction, respectively, and  represents the mitochondrial area exclusion effect, introduced to prevent proteins from entering the lumen of the isolation membrane (). The solvent concentration that appears in the entropic energy (3) was also modified to . \\n\\n\\tFor the numerical simulation, the space was discretized by a lattice with 10 nm on each side, and a concentration field was assigned to each location. The time evolution of the system was assumed to follow the Cahn–Hilliard equation,'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='which is commonly used in analyses of phase separation dynamics (Berry et al, 2018). Here,  is the mobility of the protein with the diffusion constant  μm2/sec (Milo & Phillips, 2015) expressed in terms of the Boltzmann constant  and temperature . We considered a 3000 nm × 3000 nm square region with periodic boundary conditions containing two mitochondria and surrounding cytosolic components as a system and assumed that adaptor proteins are uniformly distributed in the cytosol around mitochondria at a concentration of  in the initial state. \\n\\n\\n\\nStatistical analysis'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Statistical analysis\\n\\nStatistical analysis was performed using OriginPro 2022 software. The statistical methods used for each analysis are specified in the figure legends. No sample size calculation was performed, the experiments were not randomized, and investigators were not blinded during experiments.\\n\\n\\n\\nData Availability\\n\\nThis study includes no data deposited in external repositories.\\n\\n\\n\\nAcknowledgments'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Acknowledgments\\n\\nWe thank Michael Lazarou (Walter and Eliza Hall Institute of Medical Research) for providing pentaKO HeLa cells, Toshio Kitamura for pMXs-IP (The University of Tokyo), Shoji Yamaoka (Tokyo Medical and Dental University, Tokyo, Japan) for pMRXIP, and Teruhito Yasui (National Institutes of Biomedical Innovation, Health and Nutrition (NIBIOHN), Osaka, Japan) for pCG-VSV-G and pCG-gag-pol. pKanCMV-mClover3-mRuby3 was a gift from Michael Lin (Addgene plasmid # 74252 ; http://n2t.net/addgene:74252 ; RRID:Addgene_74252).'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='This work was supported by the Exploratory Research for Advanced Technology (ERATO) research funding program of the Japan Science and Technology Agency (JST) (JPMJER1702 to NM), a Grant-in-Aid for Specially Promoted Research (22H04919 to NM) and a Grant-in-Aid for Scientific Research (C) (23K05715 to Y.S.) from the Japan Society for the Promotion of Science (JSPS), World Research Hub (WRH) Program of the International Research Frontiers Initiative, Tokyo Institute of Technology (RLK), and German Research Foundation (Project 460056461 to RLK). The numerical computations have been performed with the RIKEN supercomputer system (HOKUSAI).\\n\\n\\n\\nDisclosure and competing interests statement \\n\\nThe authors declare that they have no conflict of interest.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='The authors declare that they have no conflict of interest.\\n\\n\\n\\n\\nReferences\\n\\nAgudo-Canalejo J, Schultz SW, Chino H, Migliano SM, Saito C, Koyama-Honda I, Stenmark H, Brech A, May AI, Mizushima N, et al (2021) Wetting regulates autophagy of phase-separated compartments and the cytosol. Nature 591: 142–146\\n\\nBajar BT, Wang ES, Lam AJ, Kim BB, Jacobs CL, Howe ES, Davidson MW, Lin MZ & Chu J (2016) Improving brightness and photostability of green and red fluorescent proteins for live cell imaging and FRET reporting. Sci Rep 6: 20889\\n\\nBerry J, Brangwynne CP & Haataja M (2018) Physical principles of intracellular organization via active and passive phase transitions. Reports on Progress in Physics 81: 046601'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Eura Y (2003) Two Mitofusin Proteins, Mammalian Homologues of FZO, with Distinct Functions Are Both Required for Mitochondrial Fusion. J Biochem 134: 333–344\\n\\nFujioka Y, Alam JM, Noshiro D, Mouri K, Ando T, Okada Y, May AI, Knorr RL, Suzuki K, Ohsumi Y, et al (2020) Phase separation organizes the site of autophagosome formation. Nature 578: 301–305\\n\\nGallagher ER & Holzbaur ELF (2023) The selective autophagy adaptor p62/SQSTM1 forms phase condensates regulated by HSP27 that facilitate the clearance of damaged lysosomes via lysophagy. Cell Rep 42: 112037\\n\\nGanley IG & Simonsen A (2022) Diversity of mitophagy pathways at a glance. J Cell Sci 135: jcs259748'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Geisler S, Holmström KM, Skujat D, Fiesel FC, Rothfuss OC, Kahle PJ & Springer W (2010) PINK1/Parkin-mediated mitophagy is dependent on VDAC1 and p62/SQSTM1. Nat Cell Biol 12: 119–131\\n\\nGouveia B, Kim Y, Shaevitz JW, Petry S, Stone HA & Brangwynne CP (2022) Capillary forces generated by biomolecular condensates. Nature 609: 255–264\\n\\nGubas A & Dikic I (2022) A guide to the regulation of selective autophagy receptors. FEBS J 289: 75–89\\n\\nHeo J-M, Ordureau A, Paulo JA, Rinehart J & Harper JW (2015) The PINK1-PARKIN Mitochondrial Ubiquitylation Pathway Drives a Program of OPTN/NDP52 Recruitment and TBK1 Activation to Promote Mitophagy. Mol Cell 60: 7–20'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Hirose T, Ninomiya K, Nakagawa S & Yamazaki T (2023) A guide to membraneless organelles and their various roles in gene regulation. Nat Rev Mol Cell Biol 24: 288–304\\n\\nHyman AA, Weber CA & Jülicher F (2014) Liquid-Liquid Phase Separation in Biology. Annu Rev Cell Dev Biol 30: 39–58\\n\\nItakura E, Kishi-Itakura C, Koyama-Honda I & Mizushima N (2012) Structures containing Atg9A and the ULK1 complex independently target depolarized mitochondria at initial stages of Parkin-mediated mitophagy. J Cell Sci 125: 1488–1499\\n\\nKane LA, Lazarou M, Fogel AI, Li Y, Yamano K, Sarraf SA, Banerjee S & Youle RJ (2014) PINK1 phosphorylates ubiquitin to activate Parkin E3 ubiquitin ligase activity. Journal of Cell Biology 205: 143–153'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Kazlauskaite A, Kondapalli C, Gourlay R, Campbell DG, Ritorto MS, Hofmann K, Alessi DR, Knebel A, Trost M & Muqit MMK (2014) Parkin is activated by PINK1-dependent phosphorylation of ubiquitin at Ser65. Biochemical Journal 460: 127–141\\n\\nKitamura T, Koshino Y, Shibata F, Oki T, Nakajima H, Nosaka T & Kumagai H (2003) Retrovirus-mediated gene transfer and expression cloning: powerful tools in functional genomics. Exp Hematol 31: 1007–1014\\n\\nKoyano F, Okatsu K, Kosako H, Tamura Y, Go E, Kimura M, Kimura Y, Tsuchiya H, Yoshihara H, Hirokawa T, et al (2014) Ubiquitin is phosphorylated by PINK1 to activate parkin. Nature 510: 162–166'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Kusumaatmaja H, May AI, Feeney M, McKenna JF, Mizushima N, Frigerio L & Knorr RL (2021a) Wetting of phase-separated droplets on plant vacuole membranes leads to a competition between tonoplast budding and nanotube formation. Proceedings of the National Academy of Sciences 118: e2024109118\\n\\nKusumaatmaja H, May AI & Knorr RL (2021b) Intracellular wetting mediates contacts between liquid compartments and membrane-bound organelles. J Cell Biol 220: e202103175\\n\\nLamark T & Johansen T (2021) Mechanisms of Selective Autophagy. Annu Rev Cell Dev Biol 37: 143–169\\n\\nLazarou M, Sliter DA, Kane LA, Sarraf SA, Wang C, Burman JL, Sideris DP, Fogel AI & Youle RJ (2015) The ubiquitin kinase PINK1 recruits autophagy receptors to induce mitophagy. Nature 524: 309–314'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Li F, Xu D, Wang Y, Zhou Z, Liu J, Hu S, Gong Y, Yuan J & Pan L (2018) Structural insights into the ubiquitin recognition by OPTN (optineurin) and its regulation by TBK1-mediated phosphorylation. Autophagy 14: 66–79\\n\\nMa X, Li P & Ge L (2023) Targeting of biomolecular condensates to the autophagy pathway. Trends Cell Biol 33: 505–516\\n\\nMangiarotti A, Chen N, Zhao Z, Lipowsky R & Dimova R (2023) Wetting and complex remodeling of membranes by biomolecular condensates. Nat Commun 14: 2809\\n\\nMcSwiggen DT, Mir M, Darzacq X & Tjian R (2019) Evaluating phase separation in live cells: diagnosis, caveats, and functional consequences. Genes Dev 33: 1619–1634\\n\\nMilo R & Phillips R (2015) Cell Biology by the Numbers Garland Science'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Milovanovic D & De Camilli P (2017) Synaptic Vesicle Clusters at Synapses: A Distinct Liquid Phase? Neuron 93: 995–1002\\n\\nMizushima N & Levine B (2020) Autophagy in Human Diseases. New England Journal of Medicine 383: 1564–1576\\n\\nMoore AS & Holzbaur ELF (2016) Dynamic recruitment and activation of ALS-associated TBK1 with its target optineurin are required for efficient mitophagy. Proceedings of the National Academy of Sciences 113: E3349–E3358\\n\\nMusacchio A (2022) On the role of phase separation in the biogenesis of membraneless compartments. EMBO J 41: e109952\\n\\nNakatogawa H (2020) Mechanisms governing autophagosome biogenesis. Nat Rev Mol Cell Biol 21: 439–458'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Narendra DP, Kane LA, Hauser DN, Fearnley IM & Youle RJ (2010) p62/SQSTM1 is required for Parkin-induced mitochondrial clustering but not mitophagy; VDAC1 is dispensable for both. Autophagy 6: 1090–1106\\n\\nNguyen TN, Sawa-Makarska J, Khuu G, Lam WK, Adriaenssens E, Fracchiolla D, Shoebridge S, Bernklau D, Padman BS, Skulsuppaisarn M, et al (2023) Unconventional initiation of PINK1/Parkin mitophagy by Optineurin. Mol Cell 83: 1693-1709.e9\\n\\nNoda NN, Wang Z & Zhang H (2020) Liquid–liquid phase separation in autophagy. Journal of Cell Biology 219: e202004062\\n\\nNovak I, Kirkin V, McEwan DG, Zhang J, Wild P, Rozenknop A, Rogov V, Löhr F, Popovic D, Occhipinti A, et al (2010) Nix is a selective autophagy receptor for mitochondrial clearance. EMBO Rep 11: 45–51'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Okatsu K, Saisho K, Shimanuki M, Nakada K, Shitara H, Sou YS, Kimura M, Sato S, Hattori N, Komatsu M, et al (2010) P62/SQSTM1 cooperates with Parkin for perinuclear clustering of depolarized mitochondria. Genes to Cells 15: 887–900\\n\\nO’Loughlin T, Kruppa AJ, Ribeiro ALR, Edgar JR, Ghannam A, Smith AM & Buss F (2020) OPTN recruitment to a Golgi-proximal compartment regulates immune signalling and cytokine secretion. J Cell Sci 133: 1–15\\n\\nOnishi M, Yamano K, Sato M, Matsuda N & Okamoto K (2021) Molecular mechanisms and physiological functions of mitophagy. EMBO J 40: e104705'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Park D, Wu Y, Wang X, Gowrishankar S, Baublis A & De Camilli P (2023) Synaptic vesicle proteins and ATG9A self-organize in distinct vesicle phases within synapsin condensates. Nat Commun 14: 455\\n\\nPeng S, Chen X-H, Chen S, Zhang J, Wang C, Liu W, Zhang D, Su Y & Zhang X (2021) Phase separation of Nur77 mediates celastrol-induced mitophagy by promoting the fluidity of p62/SQSTM1 condensates. Nat Commun 12: 5989\\n\\nRothbauer U, Zolghadr K, Muyldermans S, Schepers A, Cardoso MC & Leonhardt H (2008) A Versatile Nanotrap for Biochemical and Functional Studies with Fluorescent Fusion Proteins. Molecular & Cellular Proteomics 7: 282–289'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Saitoh T, Nakano H, Yamamoto N & Yamaoka S (2002) Lymphotoxin-beta receptor mediates NEMO-independent NF-kappaB activation. FEBS Lett 532: 45–51\\n\\nSansevrino R, Hoffmann C & Milovanovic D (2023) Condensate biology of synaptic vesicle clusters. Trends Neurosci 46: 293–306\\n\\nSchindelin J, Arganda-Carreras I, Frise E, Kaynig V, Longair M, Pietzsch T, Preibisch S, Rueden C, Saalfeld S, Schmid B, et al (2012) Fiji: an open-source platform for biological-image analysis. Nat Methods 9: 676–682\\n\\nScott DJ, Gunn NJ, Yong KJ, Wimmer VC, Veldhuis NA, Challis LM, Haidar M, Petrou S, Bathgate RAD & Griffin MDW (2018) A Novel Ultra-Stable, Monomeric Green Fluorescent Protein For Direct Volumetric Imaging of Whole Organs Using CLARITY. Sci Rep 8: 667'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Style RW, Jagota A, Hui C-Y & Dufresne ER (2017) Elastocapillarity: Surface Tension and the Mechanics of Soft Solids. Annu Rev Condens Matter Phys 8: 99–118\\n\\nSun D, Wu R, Zheng J, Li P & Yu L (2018) Polyubiquitin chain-induced p62 phase separation drives autophagic cargo segregation. Cell Res 28: 405–415\\n\\nTaylor NO, Wei M-T, Stone HA & Brangwynne CP (2019) Quantifying Dynamics in Phase-Separated Condensates Using Fluorescence Recovery after Photobleaching. Biophys J 117: 1285–1300\\n\\nTurco E, Savova A, Gere F, Ferrari L, Romanov J, Schuschnig M & Martens S (2021) Reconstitution defines the roles of p62, NBR1 and TAX1BP1 in ubiquitin condensate formation and autophagy initiation. Nat Commun 12: 5212'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Vargas JNS, Hamasaki M, Kawabata T, Youle RJ & Yoshimori T (2023) The mechanisms and roles of selective autophagy in mammals. Nat Rev Mol Cell Biol 24: 167–185\\n\\nWexler JS, Heard TM & Stone HA (2014) Capillary Bridges between Soft Substrates. Phys Rev Lett 112: 066102\\n\\nWong YC & Holzbaur ELF (2014) Optineurin is an autophagy receptor for damaged mitochondria in parkin-mediated mitophagy that is disrupted by an ALS-linked mutation. Proceedings of the National Academy of Sciences 111: E4439–E4448\\n\\nYamamoto H, Fujioka Y, Suzuki SW, Noshiro D, Suzuki H, Kondo-Kakuta C, Kimura Y, Hirano H, Ando T, Noda NN, et al (2016) The Intrinsically Disordered Protein Atg13 Mediates Supramolecular Assembly of Autophagy Initiation Complexes. Dev Cell 38: 86–99'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Yamamoto H, Kakuta S, Watanabe TM, Kitamura A, Sekito T, Kondo-Kakuta C, Ichikawa R, Kinjo M & Ohsumi Y (2012) Atg9 vesicles are an important membrane source during early steps of autophagosome formation. Journal of Cell Biology 198: 219–233\\n\\nYamano K, Kikuchi R, Kojima W, Hayashida R, Koyano F, Kawawaki J, Shoda T, Demizu Y, Naito M, Tanaka K, et al (2020) Critical role of mitochondrial ubiquitination and the OPTN-ATG9A axis in mitophagy. J Cell Biol 219: e201912144\\n\\nYamano K, Sawada M, Kikuchi R, Nagataki K, Kojima W, Endo R, Kinefuchi H, Sugihara A, Fujino T, Watanabe A, et al (2024) Optineurin provides a mitophagy contact site for TBK1 activation. EMBO J 43: 754–779'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Yamasaki A, Alam JM, Noshiro D, Hirata E, Fujioka Y, Suzuki K, Ohsumi Y & Noda NN (2020) Fluidity Is a Critical Determinant for Selective Autophagy of Protein Condensates. Mol Cell 77: 1163-1175.e9\\n\\nYim WW-Y, Yamamoto H & Mizushima N (2022) A pulse-chasable reporter processing assay for mammalian autophagic flux with HaloTag. Elife 11: e78923\\n\\nZaffagnini G, Savova A, Danieli A, Romanov J, Tremel S, Ebner M, Peterbauer T, Sztacho M, Trapannone R, Tarafder AK, et al (2018) p62 filaments capture and present ubiquitinated cargos for autophagy. EMBO J 37: e98308'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Zheng Q, Chen Y, Chen D, Zhao H, Feng Y, Meng Q, Zhao Y & Zhang H (2022) Calcium transients on the ER surface trigger liquid-liquid phase separation of FIP200 to specify autophagosome initiation sites. Cell 185: 4082-4098.e22\\n\\n\\xa0\\n\\n\\n\\n\\nFigure legends\\n\\n\\n\\nFigure 1. Autophagy adaptors show distinct distributions during Parkin-mediated mitophagy.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='(A and B) Representative images (left) and spline graphs of the intensity profiles along the indicated arrows (from position 0) in the respective images (right) of wild-type HeLa cells expressing one of the GFP-tagged autophagy adaptors or ubiquitin, mRuby–Omp25, and HaloTag7–LC3B at 45 min after CCCP treatment. Separate mitochondria without contacting other mitochondria or isolation membranes (A; Mt) or mitochondria with an isolation membrane (B; Mt–IM) are shown. The y-axis in each of the graphs indicates the fluorescence intensity. Scale bars indicate 1 μm.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='(C) The relative intensity of GFP-tagged adaptors or ubiquitin (a and b) in LC3B-positive areas compared with LC3B-negative areas in panel B. GFP signals were normalized to the mRuby–Omp25 signals (a′ and b′) using the formula Relative Intensity = (a/b)/(a′/b′), where a and a′ represent the intensities in the LC3B-positive area, and b and b′ represent the intensities in the LC3B-negative area. Solid horizontal bars indicate the means, and dots indicate the data from five structures. Differences were statistically analyzed by one-way analysis of variance with Dunnett’s post-hoc test.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='(D) Immunostaining of endogenous OPTN or NDP52 in wild-type HeLa cells expressing mRuby–Omp25 and HaloTag7–LC3B at 45 min after CCCP treatment. Scale bars indicate 4 µm and 1 µm (magnified images).\\n\\n(E) Representative images (left) and spline graphs of the intensity profiles along the indicated arrows (from position 0) in the respective images (right) of HeLa cells lacking all five autophagy adaptors (penta KO cells) expressing indicated GFP-tagged autophagy adaptors, mRuby–Omp25, and HaloTag7–LC3B at 45 min after CCCP treatment. Mitochondria with an isolation membrane (Mt–IM) are shown. The y-axis in each of the graphs indicates the fluorescence intensity. Scale bars indicate 1 μm.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='(F) The relative intensity of GFP-tagged adaptors or ubiquitin (a and b) in LC3B-positive areas compared with LC3B-negative areas in panel E. Relative intensities were calculated as in (C). Solid horizontal bars indicate the means, and dots indicate the data from five structures. Differences were statistically analyzed by one-way analysis of variance with Dunnett’s post-hoc test.\\n\\n\\n\\nFigure 2. OPTN and NDP52 show a dynamic exchange between the mitochondrial surface and cytosol.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='(A–D) Representative images (A and C) and quantification (B and D) of GFP fluorescence recovery after photobleaching (FRAP) on separate mitochondria (Mt) (A and B) or on mitochondria with an isolation membrane (Mt–IM) (C and D) in HeLa cells lacking all five autophagy adaptors (penta KO cells) expressing GFP-tagged adaptors or ubiquitin at 45 min after CCCP treatment. The photobleached areas are indicated by dotted lines. The magnified panels display time-lapse images of the photobleached areas. Scale bars indicate 2 μm and 1 μm (magnified images). Data were collected from four structures and were fitted to the equation y = a*(1 − exp(−b*x)). The dark shaded areas represent the 95% confidence intervals, and the light shaded areas represent the 95%'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='intervals, and the light shaded areas represent the 95% prediction intervals.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='(E) Penta KO cells expressing GFP–OPTN and GFP–NDP52 at 45 min after treatment with CCCP and wortmannin. Wortmannin was added to inhibit autophagosome formation so that OPTN is not sequestered in a closed compartment. Images of cells before (upper panels) and 2 min after (lower panels) the addition of 10% 1,6-hexanediol are displayed. Scale bars, 5 μm and 2 μm (magnified images).\\n\\n\\n\\nFigure 3. Mathematical models of condensate formation and localization of autophagy adaptors.\\n\\n(A) Schematic diagram of two separated mitochondria.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='(A) Schematic diagram of two separated mitochondria.\\n\\n(B) Sheet-like condensates of autophagy adaptors on mitochondria deform when the inter-mitochondrial distance  was changed. The strength of adaptor self-interaction (), adaptor–ubiquitin interaction (), mitochondrial area exclusion () and surface tension () were set to , , , and , respectively. The color indicates the adaptor concentration (). Arrowheads indicate autophagy adaptors filling the cleft between the two mitochondria.\\n\\n(C) Schematic diagram of a single mitochondrion with an isolation membrane.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='(D) Sheet-like condensates of autophagy adaptors on mitochondria deformed when the isolation membrane (arrowheads) appeared and the angle  surrounding the mitochondria changed. The strength of adaptor self-interaction (), adaptor–ubiquitin interaction (), adaptor–LC3 interaction (), mitochondrial area exclusion (), isolation membrane area exclusion (), and surface tension () were set to , ,  , and , respectively. The color indicates the adaptor concentration ().\\n\\n\\n\\nFigure 4. OPTN condensates redistribute upon contact with membranes.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='(A–D) Time-lapse images (A and C) and spline graphs (B and D) of two separate mitochondria approaching and contacting each other. Events of the attachment of two mitochondria with comparable sizes (A and B) and with differing sizes (C and D) were observed. The line graphs represent the intensity profiles along the indicated arrows (from position 0) in the respective images shown (B and D). The arrowheads indicate redistributed OPTN signals filling the cleft between two adjacent mitochondria. Scale bars indicate 1 μm. See also Movie EV1 for the time-lapse video corresponding to (C).'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='(E and F) Time-lapse images (E) and a spline graph (F) of a mitochondrion with an isolation membrane elongating on its surface. The arrowhead indicates the elongating isolation membrane. The line graph represent the intensity profiles along the indicated arrows (from position 0) in the respective images shown. Scale bars indicate 1 μm. See also Movie EV2 for the time-lapse video corresponding to (E).\\n\\n\\n\\nFigure 5. Liquid-like property of OPTN condensates is required for mitophagy.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='(A) HeLa cells lacking all five autophagy adaptors (penta KO cells) expressing both GPF–OPTN and mRuby–Ub (i), both GPF–OPTN and anti-GFP nanobody–mRuby–Ub (ii), boh GFP–OPTNΔUBAN and mRuby–Ub (iii), or both GFP–OPTNΔUBAN and anti-GFP nanobody–mRuby–Ub (iv) at 45 min after CCCP treatment. Time-lapse images of GFP FRAP are shown. Photobleached areas are circled by dotted lines. Scale bars indicate 4 μm and 2 μm (magnified images). \\n\\n(B) Interaction between GFP–OPTN and OPTN mutants and FLAG–Ub or FLAG–nanobody–Ub was investigated by immunoprecipitation with an anti-FLAG antibody and immunoblotting with an anti-GFP antibody.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='(C) Quantification of GFP FRAP on separate mitochondria (Mt) in penta KO cells expressing both GFP–OPTN and mRuby–Ub (i; gray), both GFP–OPTN and anti-GFP nanobody–mRuby–Ub (ii; red), or both GFP–OPTNΔUBAN and anti-GFP nanobody–mRuby–Ub (iv; blue) at 45 min after CCCP treatment. Data were collected from four structures and were fitted to the equation y = a*(1 − exp(−b*x)). The dark shading represents the 95% confidence intervals, and the light shading represents the 95% prediction intervals.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='(D and E) Representative data (D) and quantification (E) of HaloTag (Halo) processing assay using cells expressing the indicated OPTN and Ub constructs. Cells expressing the mtHalo–SNAP mitophagy reporter were treated without (0 h) and with oligomycin and antimycin for 18 h. The amount of processed Halo (proc. Halo) indicates the relative amount of mitochondria degraded in lysosomes. Solid horizontal bars indicate the means, and dots indicate the data from at least three independent cultures. Differences were statistically analyzed by one-way analysis of variance with Dunnett’s post-hoc test.\\n\\n\\n\\nFigure 6. Liquid-like property of OPTN condensates is required for the recruitment of ATG9 vesicles.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content=\"(A and B) Localization of ATG9 vesicles under normal (CCCP−) or mitophagy-inducing conditions (CCCP+, 60 min) (A) and quantification (B) of ATG9A colocalization with mitochondria. Endogenous ATG9 was immunostained in penta KO cells expressing both GFP–OPTN and mRuby–Ub or both GFP–OPTNΔUBAN and anti-GFP nanobody–mRuby–Ub (nano–mRuby–Ub) together with mitochondrially targeted Halo–SNAP (mtHalo–SNAP). Scale bars indicate 4 μm and 2 μm (magnified images). Solid horizontal bars indicate the means, each dot indicates the mean value from one field of view with ≥10 cells (n=15). Differences were statistically analyzed by Welch's two-tailed t test.\"), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content=\"(C and D) Representative data (C) and quantification (D; p=0.99) of organelle immunoprecipitation in cells expressing GFP–OPTN or GFP–OPTNΔUBAN in the presence or absence of FLAG-ATG9A. ATG9 vesicles were immunoprecipitated by FLAG M2 beads and GFP was blotted to detect the interaction between GFP–OPTN (WT or ΔUBAN) and ATG9 vesicles at 60 min after CCCP treatment. Solid horizontal bars indicate the means, and dots indicate the data from five independent experiments. Differences were statistically analyzed by Welch's two-tailed t test.\"), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='(E) The models of ATG9 vesicle recruitment (left) and isolation membrane elongation (right) mediated by liquid-like OPTN condensates. Partial or complete wetting of phase-separated OPTN enables the recruitment of ATG9 vesicles to ubiquitinated mitochondria (left). Wetting of OPTN on mitochondria and isolation membranes also facilitates the engulfment of mitochondria by the isolation membranes (right).\\n\\n\\n\\nFigure EV1. Localization of autophagy adaptors between mitochondria during Parkin-mediated mitophagy.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='(A) Representative images (left) and spline graphs of the intensity profiles along the indicated arrows (from position 0; right) of wild-type HeLa cells expressing one of the GFP–tagged autophagy adaptors or ubiquitin and mRuby–Omp25 at 45 min after CCCP treatment. Mitochondrial clusters (Mt–Mt) are shown. The y-axis in each of the graphs indicates the fluorescence intensity.\\n\\n(B) The relative intensity of each adaptor was calculated as in Figure 1C. Solid horizontal bars indicate the means, and dots indicate the data from five structures. Differences were statistically analyzed by one-way ANOVA with Dunnett’s post-hoc test.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='(C) Representative images (left) and spline graphs of the intensity profiles along the indicated arrows (from position 0; right) of HeLa cells lacking all five autophagy adaptors (penta KO cells) expressing one of the GFP–tagged autophagy adaptors or ubiquitin and mRuby–Omp25 at 45 min after CCCP treatment. Mitochondrial clusters (Mt–Mt) are shown. The y-axis in each of the graphs indicates the fluorescence intensity.\\n\\n(D) The relative intensity of each adaptor was calculated as in Figure 1C. Solid horizontal bars indicate the means, and dots indicate the data from five structures. Differences were statistically analyzed by one-way ANOVA with Dunnett’s post-hoc test.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Figure EV2. p62 on mitochondrial clusters and OPTN and NDP52 at the mitochondria-isolation membrane contact sites are resistant to 1,6-hexanediol treatment.\\n\\n(A) Penta KO cells expressing GFP–p62 at 45 min after CCCP treatment. Images of cells before (left) and 2 min after (right) the addition of 10% 1,6-hexanediol are displayed. Scale bars, 5 μm and 2 μm (magnified images).\\n\\n(B) Penta KO cells expressing GFP–OPTN (left) or GFP–NDP52 (right) at 45 min after CCCP treatment. Images of cells before (upper panels) and 2 min after (lower panels) the addition of 10% 2,5-hexanediol are displayed. Scale bars, 5 μm and 2 μm (magnified images).'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='(C) Penta KO cells expressing GFP–OPTN (left) or GFP–NDP52 (right) at 45 min after CCCP treatment. Mitochondria associated with isolation membranes (LC3) were analyzed. Images of cells before (upper panels) and 2 min after (lower panels) the addition of 10% 1,6-hexanediol are displayed. Scale bars, 5 μm and 2 μm (magnified images).\\n\\n\\n\\nFigure EV3: Droplet distribution depends on the binding strength.\\n\\n(A) Dependence of droplet distribution on two mitochondria on Ub-binding strength () and self-binding strength (). Mitochondrial area exclusion () and surface tension () were set to  and , respectively.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='(B) Dependence of droplet distribution on isolation membrane and mitochondria on LC3-binding strength (). The other parameters were set to , ,  and . The bending angle of the isolation membrane was set to .\\n\\n\\n\\nFigure EV4. Loss of mobility and mitophagy activity with ubiquitin binding mutants of OPTN.\\n\\n(A) Penta KO cells expressing both GFP–OPTN and mRuby–Ub (i), GPF–OPTN (E478G) and mRuby–Ub (ii), both GPF–OPTN (E478G) and anti-GFP nanobody–mRuby–Ub (iii), both GFP–OPTNΔUBAN and anti-GFP nanobody–mRuby–Ub (iv), and both OPTNΔUBAN–GFP and anti-GFP nanobody–mRuby–Ub (v) at 45 min after CCCP treatment. Time-lapse images of GFP FRAP are shown. Photobleached areas are circled by dotted lines. Scale bars indicate 4 μm and 2 μm (magnified images).'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='(B) Quantification of GFP FRAP on separate mitochondria (Mt) in penta KO cells expressing both GFP–OPTN and mRuby–Ub (i), both GPF–OPTN (E478G) and anti-GFP nanobody–mRuby–Ub (iii), both GFP–OPTNΔUBAN and anti-GFP nanobody–mRuby–Ub (iv), and both OPTNΔUBAN–GFP and anti-GFP nanobody–mRuby–Ub (v) at 45 min after CCCP treatment. Data were collected from four structures and were fitted to the equation y = a*(1 − exp(−b*x)). The dark shading represents the 95% confidence intervals, and the light shading represents the 95% prediction intervals.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='(C and D) Representative data (C) and quantification (D) of HaloTag (Halo) processing assay using cells expressing the indicated OPTN and Ub constructs. Cells expressing the mtHalo–SNAP mitophagy reporter were treated without (0 h) and with oligomycin and antimycin for 18 h. The amount of processed Halo (proc. Halo) indicates the relative amount of mitochondria degraded in lysosomes. Solid horizontal bars indicate the means, and dots indicate the data from three independent cultures. Differences were statistically analyzed by one-way analysis of variance with Dunnett’s post-hoc test.\\n\\n\\n\\nFigure EV5. Recruitment of TBK1 and FIP200 during mitophagy.'), Document(metadata={'source': './data/docx/EMBOJ-2023-115537.docx'}, page_content='Figure EV5. Recruitment of TBK1 and FIP200 during mitophagy.\\n\\n(A, B) Localization of FIP200 and phosphorylated TBK1 under mitophagy-inducing conditions (CCCP, 60 min). Endogenous FIP200 (A) and phosphorylated TBK1 (B) were immunostained in penta KO cells expressing both GFP–OPTN and mRuby–Ub or both GFP–OPTNΔUBAN and anti-GFP nanobody–mRuby–Ub together with mitochondrially targeted Halo–SNAP (mtHalo–SNAP). Scale bars indicate 4 μm and 2 μm (magnified images).\\n\\n\\n\\n2')]\n",
      "Processing document: ./data/docx/EMBOR-2023-58706-T.docx\n",
      "Split documents: [Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Distinct mechanisms control the specific synaptic functions of Neuroligin 1 and Neuroligin 2\\n\\nJinzhao Wang1,2,4,5, Thomas C. Südhof1,2,3, * and Marius Wernig1,4,5, *\\n\\n1Institute for Stem Cell Biology and Regenerative Medicine\\n\\n2Department of Molecular and Cellular Physiology  \\n\\n3Howard Hughes Medical Institute\\n\\n4Department of Pathology \\n\\n5Department of Chemical and Systems Biology\\n\\nStanford University School of Medicine\\n\\nStanford, CA 94305, USA\\n\\n* co-corresponding authors\\n\\nEmails: tcs1@stanford.edu; wernig@stanford.edu\\n\\nKey Words: neuroligin, synapse specific, extracellular domain, intracellular domain, distinct mechanisms\\n\\n\\n\\n\\nAbstract'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Neuroligins are postsynaptic cell-adhesion molecules that regulate synaptic function with a remarkable isoform specificity. Although Nlgn1 and Nlgn2 are highly homologous and biochemically interact with the same extra- and intracellular proteins, Nlgn1 selectively functions in excitatory synapses whereas Nlgn2 functions in inhibitory synapses. How this excitatory/ inhibitory (E/I) specificity arises is unknown. Using a comprehensive structure-function analysis, we here expressed wild-type and mutant neuroligins in functional rescue experiments in cultured hippocampal neurons lacking all endogenous neuroligins. Electrophysiology confirmed that Nlgn1 and Nlgn2 selectively restored excitatory and inhibitory synaptic transmission, respectively, in'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='and inhibitory synaptic transmission, respectively, in neuroligin-deficient neurons, aligned with their synaptic localizations. Chimeric Nlgn1-Nlgn2 constructs reveal that the extracellular neuroligin domains confer synapse specificity, whereas their intracellular sequences are exchangeable. However, the cytoplasmic sequences of Nlgn2, including its Gephyrin-binding motif that is identically present in the Nlgn1, is essential for its synaptic function whereas they are dispensable for Nlgn1. These results demonstrate that although the excitatory vs. inhibitory synapse specificity of Nlgn1 and Nlgn2 are both determined by their extracellular sequences, these neuroligins enable normal synaptic connections via distinct intracellular mechanisms.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Introduction'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Neuroligins (Nlgns) are evolutionarily conserved postsynaptic cell adhesion molecules that bind to presynaptic neurexins (Nrxns) (Ichtchenko et al, 1995; Ichtchenko et al, 1996; Nguyen & Sudhof, 1997). In vertebrates, four genes encode Nlgns: Nlgn1, Nlgn2, Nlgn3, and Nlgn4. Among these, Nlgn1, Nlgn2, and Nlgn3 exhibit high conservation across species, while Nlgn4 displays variability between rodents and humans and is expressed at relatively low levels in mice (Bolliger et al, 2008; Ichtchenko et al., 1995; Ichtchenko et al., 1996; Jamain et al, 2003). Nlgns are type 1 transmembrane proteins that comprise a single, large extracellular domain consisting of a constitutively dimeric, enzymatically inactive esterase-homology domain, a transmembrane region (TMR),'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='esterase-homology domain, a transmembrane region (TMR), and a short cytoplasmic tail. Remarkably, the targeting of Nlgns to synapses is selective and specific. Nlgn1 is only localized to glutamatergic synapses (Song et al, 1999) while Nlgn2 is exclusively present at GABAergic, dopaminergic, and cholinergic synapses (Graf et al, 2004; Takacs et al, 2013; Uchigashima et al, 2016; Varoqueaux et al, 2004). Nlgn3 is found in both excitatory and inhibitory synapses (Budreck & Scheiffele, 2007), and Nlgn4 functions in glycinergic synapses in mice and in GABAergic synapses in human neurons (Hoon et al, 2011; Marro et al, 2019; Zhang et al, 2018). Consistent with their localization to synapses, Nlgns perform multiple synapse-specific functions. Compelling initial'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='perform multiple synapse-specific functions. Compelling initial evidence supporting the synaptic function of Nlgns was obtained through overexpression experiments, which demonstrated significant increases in synapse density and synaptic transmission in transfected neurons (Chubykin et al, 2007; Scheiffele et al, 2000). Since then, a large number of functional studies in invertebrates and vertebrates using both deletions of and mutations in Nlgn genes revealed a panoply of important roles for Nlgns in shaping synapses, confirming the central contribution of Nlgns to the functional architecture of synaptic connections (Banovic et al, 2010; Calahorro & Ruiz-Rubio, 2012; Chubykin et al., 2007; Hu et al, 2012; Hunter et al, 2010; Maglioni et al, 2022; Sudhof,'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='et al, 2012; Hunter et al, 2010; Maglioni et al, 2022; Sudhof, 2008; Sun et al, 2011; Sun et al, 2023; Tabuchi et al, 2007; Tu et al, 2015; Varoqueaux et al, 2006; Xing et al, 2014).'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Despite much investigation, however, several key questions about Nlgn function remain unaddressed. Possibly the most important of these questions regards the observation that different Nlgn isoforms, despite a high degree of sequence similarity, perform largely non-overlapping distinct functions in the same neurons. How are different Nlgns targeted to distinct synapses where they perform different roles? This question represents an intriguing cell biological challenge given that Nlgns are postsynaptic, and each neuron may receive inputs from thousands of neurons using the whole range of neurotransmitter types. Since Nlgns interact with synapse-specific molecules at both the post-synaptic (on the same cell) and pre-synaptic side (with molecules from the'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='the same cell) and pre-synaptic side (with molecules from the projecting neuron), E/I specificity could be conferred to various Nlgns by intra- and/or extra-cellular mechanisms. For example, Nlgn2 is primarily localized to inhibitory synapses and Nlgn2 loss-of-function results in impaired inhibitory but not excitatory synaptic transmission (Chanda et al, 2017; Chubykin et al., 2007; Gibson et al, 2009; Liang et al, 2015; Poulopoulos et al, 2009; Zhang et al, 2015). Intracellularly, Nlgn2 binds to the inhibitory synapse-specific molecules gephyrin, collybistin, GARLH3, and GARLH4, which are plausible candidates to recruit Nlgn2 to inhibitory synapses (Poulopoulos et al., 2009; Yamasaki et al, 2017). However, the intracellular domains of different Nlgns are'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='However, the intracellular domains of different Nlgns are highly conserved, which argues against a strong contribution of intracellular interaction partners as a mechanism of E/I specificity. On the extracellular side facing pre-synaptic terminals, several molecules are also known to bind to Nlgns. The best characterized Nlgn interacting proteins are the three Nrxns (Nrxn1-3), each of which is expressed in hundreds of alternatively spliced variants (Treutlein et al, 2014). With this impressive possibility of combinations, it is conceivable that specific \"Nrxn codes\" could decorate different classes of synapses (Aoto et al, 2015; Dai et al, 2019; Dai et al, 2021). More recently, the extracellular MDGA proteins have been described as Nlgn regulators by'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='MDGA proteins have been described as Nlgn regulators by binding to Nlgns in competition with Nrxns (Connor et al, 2019; Lee et al, 2013; Tanaka et al, 2012). Crystal structures of MDGA1/Nlgn1 and MDGA1/Nlgn2 complexes revealed that two N-terminal lg domains of MDGA1 straddle the Nlgn1 or Nlgn2 homodimer, such that each Nlgn homodimer binds to two MDGA1 molecules (Connor et al, 2016; Gangwar et al, 2017; Kim et al, 2017; Lee et al., 2013). The binding of Nlgns to MDGAs has been hypothesized to restrain Nlgn functions (Elegheert et al, 2017b; Pettem et al, 2013), but much about the mechanisms and relative importance of this interaction is unknown. In summary, how Nlgns target and exert their functions at specific types of synapses remains a mystery.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Here, we begin to address some of these critical questions using expression of specific Nlgns and their mutants in cultured hippocampal neurons in which all Nlgn isoforms were genetically deleted, with functional analyses performed by electrophysiology and imaging. This approach reduces the complexity of potential interactions among different Nlgn isoforms that can be present in the same postsynaptic compartment and elucidates the inherent functions of the single Nlgn molecules expressed in this system, thereby simplifying functional interpretations.\\n\\n\\nResults\\n\\nSpecific synaptic targeting is an inherent property of Nlgn1 and Nlgn2'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='The co-existence of various Nlgns in synapses complicates the conclusions about a particular Nlgn using conventional experimental approaches. We therefore sought to reduce the complexity to a neuronal cell system that contains only a single Nlgn of interest. To this end, we generated quadruple conditional knock-out (cKO) mice where all 4 endogenous Nlgn genes are conditionally deleted upon expression of Cre recombinase. Our experimental setup is illustrated in Fig. 1A and entails (i) the generation of primary hippocampal cultures of postnatal day 0 (P0) mice, (ii) the expression by lentiviral transduction of active Cre recombinase or inactive Cre recombinase as a control, and (iii) the co-expression of a specific Nlgn construct. We performed immunoblots on'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='of a specific Nlgn construct. We performed immunoblots on lysates from Nlgn1234 cKO hippocampal cultures infected with lentiviral Cre or Cre (Fig. EV1A and B). Cre expression caused a substantial loss of Nlgn1, Nlgn2, and Nlgn3 (~70%-75%). Nlgn4 is not significantly expressed in this culture system (Chanda et al., 2017; Hoon et al., 2011; Varoqueaux et al., 2006).'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='To investigate the localization of Nlgn1 and Nlgn2 (that are known to localize to specifically to excitatory and inhibitory synapses, respectively (Graf et al., 2004; Poulopoulos et al., 2009; Song et al., 1999; Varoqueaux et al., 2004)), in the absence of any other Nlgns, we expressed N-terminally hemagglutinin (HA)-tagged, full-length Nlgn1 or Nlgn2 under control of the Synapsin promoter in hippocampal neurons after Cre-mediated deletion of all endogenous Nlgns (Fig. 1B). Co-labeling with excitatory and inhibitory synapse markers allowed us to assess the specific targeting of Nlgn1 and Nlgn2 by fluorescence imaging. We detected strong expression of the infected Nlgn1 and Nlgn2 on MAP2-positive dendrites in a punctate pattern (Fig. 1C-F). We also'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='dendrites in a punctate pattern (Fig. 1C-F). We also co-stained HA with Homer1 or Gephyrin as markers of excitatory and inhibitory synapses, respectively (Fig. 1C and 1E). Consistent with reports about endogenous Nlgn1 and Nlgn2 (Song et al., 1999; Varoqueaux et al., 2004), we found that in the absence of endogenous Nlgns, the majority of HA-Nlgn1 puncta were co-localized with Homer1, representing ~60% of the observed puncta. In contrast, HA-Nlgn1 puncta showed little co-localization with Gephyrin, accounting for only ~27% of the observed puncta. HA-Nlgn2 puncta, conversely, predominantly co-localized with Gephyrin, with ~59% of the puncta showing this co-localization. Only ~28% of HA-Nlgn2 puncta co-localized with Homer1 (Fig. 1D and F). We also analyzed'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content=\"co-localized with Homer1 (Fig. 1D and F). We also analyzed the colocalization of Homer1 and Gephyrin to address synapse specificity at the confocal level. We found ~21% Homer1 co-localization with Gephyrin and ~30% Gephyrin co-localization with Homer1 (Fig. EV1C and D). However, the background of positive colocalization with the 'wrong' synapse is high due to the insufficient imaging resolution for separating closely spaced synapses and the significant impact of thresholding on apparent colocalizations. Although these findings are limited by the resolution of confocal microscopy, they are consistent with previous conclusions that Nlgn1 primarily targets excitatory synapses while Nlgn2 primarily targets inhibitory synapses, showing that Nlgn1 and Nlgn2\"), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='targets inhibitory synapses, showing that Nlgn1 and Nlgn2 exhibit this specificity even in the absence of other Nlgns.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='The Gephyrin-binding domain of Nlgn2 is necessary for its inhibitory synapse-specific function but does not confer inhibitory synapse specificity'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='To investigate the molecular mechanisms targeting Nlgn2 to inhibitory synapses, we generated Nlgn2 constructs with a truncation of the intracellular domain (Nlgn2-GPI) and a chimeric Nlgn2 construct whose intracellular domain was switched with the corresponding Nlgn1 sequence (Fig. 2A). We then expressed the constructs in our Nlgn1234 cKO hippocampal neuron system. Cell surface staining of Nlgn2-GPI and Nlgn2-Nlgn1 constructs in the Cre-infected neurons confirmed proper surface localization, indicating that the introduced mutations do not interfere with protein synthesis and processing in the secretory pathway (Fig. EV2A). Remarkably, co-labeling with antibodies against the inhibitory synapse protein Gephyrin revealed that both constructs localized to'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='protein Gephyrin revealed that both constructs localized to inhibitory synapses, but Nlgn2-Nlgn1 exhibited a higher degree of localization with Gephyrin compared to Nlgn2-GPI (Fig. EV2B), and expression levels no significant change (Fig. EV2C).'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Next, we assessed the synaptic function of the Nlgn2 constructs using electrophysiology. As expected, Cre-infected Nlgn1234 cKO hippocampal neurons exhibited a ~50% decrease of evoked GABAR-mediated inhibitory postsynaptic currents (IPSC) compared to Cre-EGFP infected cultures (Fig. 2B and C), consistent with previous findings (Chanda et al., 2017; Gan & Sudhof, 2020). Expression of full-length Nlgn2 resulted in a full rescue of the GABAR-mediated IPSCs, demonstrating that Nlgn2 is the main Nlgn isoform responsible for the inhibitory synapse phenotype observed with Ngln1234 deletions (Fig. 2B and C). Intriguingly, expression of Nlgn2-GPI, i.e. Nlgn2 without an intracellular domain, failed to rescue the phenotype (Fig. 2B and C), revealing that Nlgn2'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='to rescue the phenotype (Fig. 2B and C), revealing that Nlgn2 function for inhibitory synapses requires an intracellular domain presumably due to the absence of synaptic targeting motifs in the GPI-anchor domain and the impact of membrane anchoring on protein trafficking and synaptic recruitment. Given the sequence similarity between the intracellular domains of Nlgns despite their different synaptic functions, we assessed the function of a chimeric Nlgn2 with an intracellular domain of Nlgn1. Consistent with previous results (Nguyen et al, 2016), the Nlgn2-Nlgn1 construct could also rescue the GABAR-mediated IPSC phenotype in Nlgn1234 cKO neurons (Fig. 2B and C). Thus, the intracellular Nlgn2 domain is necessary for inhibitory synapse function but does not'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='is necessary for inhibitory synapse function but does not confer specificity.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='To further understand how Nlgns might function at synapses, we aligned Nlgn1 and Nlgn2 intracellular sequences and confirmed that both Nlgn1 and Nlgn2 contain a highly conserved Gephyrin-binding sequence (Fig. 3A), even though Nlgn1 is primarily localized to excitatory synapses and has no function in inhibitory synapses (Chanda et al., 2017; Song et al., 1999; Zhang et al., 2015). This raised the question whether the Gephyrin-binding sequence is responsible for the Nlgn1 and Nlgn2 intracellular domain rescue of Nlgn2 function. To address this question, we deleted the entire 15 amino acid-long Gephyrin-binding sequence from the Nlgn2-Nlgn1 construct or introduced the Y770A point mutation which was shown to prevent Nlgn2 binding to Gephyrin by blocking the'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='was shown to prevent Nlgn2 binding to Gephyrin by blocking the phosphorylation of this residue (Giannone et al, 2013; Poulopoulos et al., 2009) (Fig. 3B). Unexpectedly, expression of Nlgn2-Nlgn1Y770A construct fully rescued the IPSC phenotype, whereas the construct with the deletion of the entire Gephyrin-binding sequence (Nlgn2-Nlgn1 DelGeph) did not rescue (Fig. 3C and D). Again, we confirmed that the expressed constructs are properly transported to the cell surface by neuronal surface staining (Fig. EV2D). In line with our electrophysiology data, co-labeling with Gephyrin antibodies showed that the Nlgn2-Nlgn1 Y770A mutant displayed a greater degree of co-localization with Gephyrin than the Nlgn2-Nlgn1DelGeph mutant (Fig. EV2E), and expression levels no'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Nlgn2-Nlgn1DelGeph mutant (Fig. EV2E), and expression levels no significant change (Fig. EV2F).These data show that the Gephyrin-binding sequence is critical for proper Nlgn2 function in inhibitory synapses but that either the interaction of Nlgn2 with Gephyrin alone may not be functionally necessary or that the Gephyrin interaction mediated by Y770 phosphorylation may be less relevant in a cellular context than in vitro.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='An intracellular sequence adjacent to the Gephyrin-binding region is necessary for Nlgn2 inhibitory function'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Given the relative brevity of the intracellular Nlgn sequence, we set out to map the function of other intracellular Nlgn2 regions more comprehensively. To this end, we constructed a series of expanding Nlgn2 deletion constructs: Nlgn2-mt1 (Nlgn2 PDZ domain with the Nrxn PDZ domain) to test whether PSD95 and S-SCAM interaction with the Nlgn2 PDZ domain affects Nlgn2 inhibitory function; Nlgn2-mt2 (delete proline-rich domain) to test whether collybistin binding with the Nlgn2 proline-rich domain affects Nlgn2 inhibitory function; Nlgn2-mt3 (delete a 21-residue before the gephyrin binding domain); Nlgn-mt4 and Nlgn5 mt are deleted large part of Nlgn2 except gephyrin binding domain (Fig. 4A and B). All constructs were confirmed to be localized to the neuronal'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='All constructs were confirmed to be localized to the neuronal cell surface (Fig. EV3A). First, we assessed the specificity of the PDZ domain using the Nlgn2-mt1 construct and collybistin domain using the Nlgn2-mt2 construct and found that this construct fully rescued the IPSC phenotype (Fig. 4C and D). We then tested Nlgn2-mt5 and Nlgn2-mt4, which proved to abolish the Nlgn2 function (Fig. 4C and D). Subsequent smaller deletions demonstrated that a 21-residue was necessary for the Nlgn2 function in addition to the previously identified Gephyrin interacting domain (Fig. 4C and D). No interactor is known for this sequence, suggesting that other intracellular mechanisms in addition to gephyrin binding enable the inhibitory synapse function of Nlgn2.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='The extracellular domain of Nlgn2 is required for inhibitory synapse function'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content=\"So far, we could establish that the intracellular domain of Nlgn2 is necessary for its function in GABAergic synaptic transmission but that, at the same time, the Nlgn1 intracellular domain functionally substituted for the intracellular domain of Nlgn2. Thus, the Nlgn2 intracellular domain does not determine its inhibitory synapse specificity. Therefore, we asked whether Nlgn2's intracellular domain would be sufficient to recruit Nlgn1 to function in GABAergic neurotransmission and created a Nlgn1-Nlgn2 construct consisting of extracellular Nlgn1 and intracellular Nlgn2 domains (Fig. 5A). Electrophysiological recordings showed that the Nlgn1-Nlgn2 chimeric construct was unable to rescue the IPSC phenotype in Nlgn1-4 cKO neurons (Fig. 5B and C). Again, we\"), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='phenotype in Nlgn1-4 cKO neurons (Fig. 5B and C). Again, we confirmed that Nlgn1-Nlgn2 properly localizes to the cell surface (Fig. EV2A). This observation suggests that the extracellular domain of Nlgn2 plays a crucial role in determining the specificity of Nlgn2 function in GABAergic synaptic transmission.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Nlgn2 binding to MDGAs does not confer inhibitory synapse specificity to Nlgn2'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='To examine what extracellular sequences mediate the inhibitory synapse function of Nlgn2, we constructed a series of Nlgn2 variants with informative mutations in the extracellular domain. We introduced into Nlgn2 a series of 5 point mutations (Q370A/E372A/L374A/N375A/D377A) that are predicted to abolish Nrxn binding to Nlgn2 based on previous results for Nlgn1 (Ko et al, 2009), and a second set of 3 point mutations (F433A/M434A/W438A) that correspond to Nlgn1 mutations which block dimerization (Ko et al., 2009). Unfortunately, these two sets of Nlgn2 mutations prevented surface transport of Nlgn2, suggesting that they may hinder proper folding of Nlgn2, which limited their usefulness (Fig. EV3B and C).'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Recently the cell-adhesion molecules MDGA1 and MDGA2 were shown to compete with Nrxns for binding to Nlgns (Connor et al., 2016; Gangwar et al., 2017; Kim et al., 2017; Lee et al., 2013). Crystal structures of the MDGA1/Nlgn2 complex provided critical insights into this interaction, showing that the two N-terminal lg domains of MDGA1 bind to the Nlgn2 homodimer (Elegheert et al., 2017b; Gangwar et al., 2017; Kim et al., 2017). To interrogate whether the interaction of MDGAs with the extracellular domain of Nlgn2 confers synaptic specificity, we generated a Nlgn2 mutant (Nlgn2-MDGA1mt) containing three point mutations H278A, D362K, E372K that were previously shown to interfere with MDGA1 binding (Fig. 5D) (Gangwar et al., 2017). The Nlgn2-MDGA1mt construct'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='(Fig. 5D) (Gangwar et al., 2017). The Nlgn2-MDGA1mt construct properly localized to the neuronal surface and co-localized with Gephyrin puncta (Fig. EV2A and B). In line with these findings, functional rescue experiments revealed that the Nlgn2-MDGA1mt construct fully retained the ability of Nlgn2 to rescue the IPSC impairment of Nlgn1234 cKO neurons (Fig. 5E and F). These data suggest that MDGA1 does not directly suppress the inhibitory function of Nlgn2 in synapses. This finding is consistent with recent work showing that MDGA1 targets APP, but not Nlgn2, in hippocampal CA1 GABAergic neural circuits (Kim et al, 2022).'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='The extracellular domain of Nlgn1 is sufficient for the excitatory synapse specificity and function of Nlgn1'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Next, we asked whether the domain requirements we observed for Nlgn2 also apply to other Nlgns. Therefore, we turned to Nlgn1 that is known to function exclusively in excitatory synapses (Chanda et al., 2017; Chubykin et al., 2007; Gan & Sudhof, 2020; Ko et al., 2009; Song et al., 1999) using equivalent experiments with Nlgn1234 cKO hippocampal neurons expressing Cre-EGFP. As expected, we observed a significant decrease in both AMPAR- and NMDAR-mediated excitatory synaptic responses when we compared Cre-infected with Cre-infected neurons (Fig. 6B-D) (Chanda et al., 2017; Gan & Sudhof, 2020). The phenotype was more pronounced in NMDAR responses (~63% decrease) compared to AMPAR responses (~39% decrease) (Fig. 6B and C), which is consistent with previous'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='decrease) (Fig. 6B and C), which is consistent with previous findings (Chanda et al., 2017; Gan & Sudhof, 2020).'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='To investigate Nlgn1 function, we performed rescue expression experiment with various Nlgn1 variants (Fig. 6A). As expected, the excitatory synaptic phenotype could be fully restored by full-length Nlgn1 expression (Fig. 6B-D). However, in contrast to what we had observed for Nlgn2, the Nlgn1-GPI construct (Nlgn1 without an intracellular domain) also rescued the excitatory synapse phenotype. Consistent with a lack of importance of the intracellular Nlgn1 domain, a Nlgn2-Nlgn1 construct was unable to rescue the glutamatergic synaptic phenotype (Fig. 6B-D). Importantly, we confirmed the surface expression Nlgn1-GPI and Nlgn2-Nlgn1. Co-localization of the constructs with Homer1 revealed that a larger fraction of Nlgn1-GPI, and a smaller fraction of Nlgn2-Nlgn1'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='fraction of Nlgn1-GPI, and a smaller fraction of Nlgn2-Nlgn1 were co-localized with Homer1 (Fig. EV4A and B), and expression levels no significant change (Fig. EV4C). We also found Nlgn1-Nlgn2 is sufficient for the glutamatergic synaptic transmission function of Nlgn1, but Nlgn2-WT doesn’t (Fig. EV5). These findings demonstrate that - unlike for Nlgn2 - the extracellular domain of Nlgn1 is sufficient for Nlgn1 function and also confers specificity to excitatory synaptic transmission.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Nrxn- and MDGA1-binding and dimerization of Nlgn1 are not required for glutamatergic synaptic transmission'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='The observation that the extracellular domain of Nlgn1 is critical for its excitatory synapse function suggests that its physical interaction with presynaptic binding partners is important. Known Nlgn1-binding partners include Nrxns and MDGAs. To test the necessity of these interactions, we generated Nlgn1 variants that are unable to bind Nrxns and MDGA1, respectively (Arac et al, 2007; Ko et al., 2009). To more generally assess the importance of extracellular Nlgn1 interactions, we also generated a Nlgn1 variant that is unable to dimerize (Fig. 7A). All constructs were appropriately expressed on the neuronal surface (Fig. EV4D).  Also, and remarkably, all constructs exhibited a high degree of co-localization with Homer1 (Fig. EV4E), and expression levels'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='co-localization with Homer1 (Fig. EV4E), and expression levels no significant change (Fig. EV4F). In concordance with these results, electrophysiological measurements demonstrated that all three constructs fully rescued the excitatory phenotype of Nlgn1234 cKO neurons (Fig. 7B-D). These results suggest the surprising notion that neither Nrxn- nor MDGA1-binding and not even dimerization of Nlgn1 is necessary for Nlgn1 function despite the functional necessity of its extracellular, but not its intracellular domain.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Discussion'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Nlgn1 and Nlgn2 function selectively in glutamatergic excitatory and GABAergic inhibitory synapses, respectively, but how their synapse specificity is determined remains unclear. In this study, we sought to investigate the mechanisms that confer this intriguing synapse specificity onto Nlgn1 and Nlgn2 and to understand how two closely related proteins can possess such contrasting functions. To eliminate the potentially confounding effects of other Nlgns that may heterodimerize with exogenously expressed Nlgns or otherwise influence a specific Nlgn under investigation, we here resorted to a reductionistic approach that employs a primary neuronal culture system lacking endogenous Nlgns. Specifically, we used as a general substrate of functional analyses'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='we used as a general substrate of functional analyses neurons that lack all four Nlgns, in which we then expressed a specific Nlgn mutant. Surprisingly, we found that different molecular mechanisms guide Nlgn1 and Nlgn2 function in this system. Our results allow us to draw five main conclusions.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='First, deletion of all four Nlgns results in a major dysfunction of both GABAergic and glutamatergic synapses, demonstrating that neuronal Nlgns are crucial for proper synaptic transmission. These results are consistent with previous findings using Nlgn123 cKO and Nlgn1234 cKO mice (Chanda et al., 2017; Gan & Sudhof, 2020). Our findings further corroborate the notion that Nlgn1 is important for glutamatergic and Nlgn2 for GABAergic neurotransmission since Nlgn1 could selectively rescue the glutamatergic defect and Nlgn2 the GABAergic defect of Nlgn1234 cKO neurons. Moreover, Nlgn1 molecules were primarily targeted to Homer1-positive excitatory synapses and Nlgn2 to Gephyrin-positive inhibitory synapses.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Second, Nlgn2’s function at inhibitory synapses requires its intracellular domain. Given this finding, interactions mediated by the intracellular domain would represent an attractive mechanism to target Nlgn2 to GABAergic synapses given the many inhibitory synapse-specific scaffolding proteins that are known to bind to the Nlgn2 cytoplasmatic domain (Poulopoulos et al., 2009; Tyagarajan et al, 2011). Unexpectedly, however, the Nlgn2 intracellular domain was unable to confer onto Nlgn2 its GABAergic synapse specificity since the Nlgn1 intracellular domain was fully capable of substituting for the Nlgn2 intracellular domain. A conspicuous similarity between Nlgn1 and Nlgn2 is the Gephyrin-binding sequence that is equally present in the Nlgn1 and Nlgn2'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='sequence that is equally present in the Nlgn1 and Nlgn2 cytoplasmic domain despite the fact that Nlgn1 exclusively functions in excitatory synapses lacking Gephyrin. Gephyrin is a key postsynaptic scaffolding protein at GABAergic synapses that enables the recruitment of GABAA receptors to synapses (Choii & Ko, 2015; Tyagarajan & Fritschy, 2014). Importantly, we found that the Gephyrin-interacting domain is necessary for Nlgn2 function, suggesting that recruitment by Gephyrin is indeed important. However, Gephyrin may not be the only important Nlgn2 binding partner. A Nlgn2 variant that carries the Y770A mutation, previously shown to disrupt Gephyrin binding (Giannone et al., 2013; Poulopoulos et al., 2009), was still fully functional. This finding suggests'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='al., 2009), was still fully functional. This finding suggests that potentially other co-factors may stabilize the Gephyrin-Nlgn2 interaction overcoming the Y770A mutation. A previous study also showed an only partial functional impairment of Nlgn2 by the Y770A mutation (Nguyen et al., 2016). However, in this paper the deletion of the entire Gephyrin binding sequence also only partly impaired Nlgn2 function whereas in our experiments such a deletion was functional deleterious. Overall, these results prompted us to look for additional cytosolic Nlgn2 sequences that are important for Nlgn2 function. Through targeted mutagenesis, we identified a new sequence of 21-residue that is just as critical as the better characterized Gephyrin-binding sequence and may'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='as the better characterized Gephyrin-binding sequence and may represent a binding surface for additional postsynaptic proteins at GABAergic synapses. Of note, the two amino acid residues K749 and R750 were shown to be ubiquitinated and methylated, respectively (Guo et al, 2014; Wagner et al, 2012).'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Third, we found that different Nlgns use distinct functional mechanisms at their target synapses. Whereas Nlgn2 requires its intracellular domain for enabling inhibitory synaptic connections, Nlgn1 -consistent with previous results (Jiang et al, 2017; Wu et al, 2019)- does not. This may be our conceptually most intriguing result: Despite the fact that Nlgn1 and Nlgn2 are both enabling synapse function, albeit at different types of synapses, their mechanisms of action differ critically.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Fourth, the extracellular but not the intracellular domain of Nlgn1 and Nlgn2 determines their specificity for glutamatergic and GABAergic synapses, respectively. The Intracellular domain of Nlgn2 (but intriguingly not of Nlgn1) is necessary for synaptic localization and its \"specific\" recruitment to synapses. Chimeric Nlgn2-Nlgn1 molecules enabled GABAergic synaptic transmission, but not glutamatergic, while chimeric Nlgn1-Nlgn2 molecules mediated glutamatergic, but not GABAergic synaptic transmission. The specificity for glutamatergic vs. GABAergic synapses is dictated by the Nlgn1, and Nlgn2 extracellular domains, respectively. Nrxns are well-studied pre-synaptic binding partners of Nlgns and thus could provide synapse specificity. However, we found that'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='thus could provide synapse specificity. However, we found that Nlgn1 variants unable to bind Nrxns still rescued the excitatory phenotypes as reported previously (Ichtchenko et al., 1995; Ichtchenko et al., 1996; Jiang et al., 2017; Ko et al., 2009; Sudhof, 2017; Wu et al., 2019). These results suggest that Nlgns function via interactions with other binding partners in addition to Nrxns. Indeed, recently MDGA proteins have been identified as Nlgn interaction partners, binding through their lg1-lg2 domains to the lobes of the Nlgn extracellular dimer at sites overlapping with the Nrxn binding interface (Elegheert et al, 2017a; Gangwar et al., 2017; Kim et al., 2017). They thus emerge as candidates that may provide synapse specificity for Nlgns. Remarkably,'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='that may provide synapse specificity for Nlgns. Remarkably, however, we found that mutant Nlgn1 predicted to lack MDGA binding was fully able to rescue Nlgn1 function, indicating that this class of interaction is also not providing specificity. Additionally, in vivo experiments show that MDGA1 targets APP, but not Nlgn2, in hippocampal CA1 GABAergic neural circuits to regulate GABAergic synaptic transmission (Kim et al., 2022). In addition to MDGAs, at least Nlgn3 has been shown to bind to PTPRD (Yoshida et al, 2021), which in turn binds to neurexins (Han et al, 2018), suggesting a possible synapse maintenance mechanism involving multiple trans-synaptic interactions.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content=\"Finally, we conclude that Nlgn1 may be acting as a monomer in regulating synaptic transmission and that no binding event that involves signal transduction mediated by dimerization plays a role in Nlgn1's specificity. Mutations disrupting Nlgn1 dimerization did not impair Nlgn1 function in our hippocampal cell assay. Intriguingly, a previous study showed that Nlgn1 overexpression enhances both NMDAR- and AMPAR- EPSCs in neurons, even when dimerization is disrupted (Ko et al., 2009).\"), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Our studies also raise new questions. With the conclusion that extracellular mechanisms must be responsible for functional specificity of Nlgns, but neither Nrxn nor MDGA interactions are themselves sufficient to account for the specificity of Nlgns, there must be other thus-far unidentified proteins present at excitatory and inhibitory pre-synaptic compartments that interact with Nlgns. In the case of Nlgn2, the intracellular domain is critical for synaptic function, but its exact role continues to be unclear. For example, Nlgn2 activation by an extracellular signal could stimulate a defined signal-transduction cascade, akin to a conventional receptor/ligand-induced signaling pathway. Unraveling these signaling mechanisms will contribute to a comprehensive'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='these signaling mechanisms will contribute to a comprehensive understanding of the molecular pathways governing Nlgn-mediated synaptic processes. These questions are crucial for a general understanding of the diversity of Nlgns functions. Detailed molecular, genetic, and biochemical approaches may lead to a better insight into these questions.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Methods\\n\\nMouse breeding and husbandry\\n\\nAll animal experiments were performed with male and female newborn mice according to institutional guidelines and approved by the Administrative Panel on Laboratory Animal Care of Stanford University School of Medicine (Protocol ID: 18846). A detailed description of generating Nlgn1234 cKO mice is describe in the previous study (Wu et al., 2019). All experiments except for Figure EV1, were performed in a “blinded” fashion (i.e., the experimenter was unaware of whether a sample represented a test or control sample).\\n\\nPlasmids and Lentivirus preparation'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='All plasmids were in a pFSW67 backbone and used the human synapsin promoter, for all neuroligin added an N-terminal HA peptide fused to the mature coding sequence. Lentivirus were prepared as described previously (Chanda et al., 2017). Briefly, the lentiviral expressing vectors (12 mg) were co-transfected with three helper plasmids (pRSV-REV (4mg), pMDLg/pRRE (8mg), and VSV-G (6mg)) into HEK293T cells cultured in T-75 flasks using calcium phosphate transfection. After a complete medium exchange at 7h, supernatants were subsequently collected at 48h, filtered via 0.45 mm filter and ultracentrifuged at 64000 g for 2h. Pellets were resuspended in MEM and stored at -80 C before use. All viruses used in this paper are as below:\\n\\nLenti-Syn-Cre-EGFP'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Lenti-Syn-Cre-EGFP\\n\\nLenti-Syn-Cre-EGFP\\n\\nLenti-Syn-HA-Nlgn1-WT (Nlgn1-wild-type)\\n\\nLenti-Syn-HA-Nlgn1-GPI (We deleted all transmembrane regions and cytoplasmic sequences of Nlgn1 and attaches its extracellular domains to the membrane using a GPI-anchor)\\n\\nLenti-Syn-HA-Nlgn1-Nlgn2 (We made the extracellular domain of Nlgn1 transplanted onto Nlgn2 intercellular.)\\n\\nLenti-Syn-HA-Nlgn1-Nrxnmt (Nlgn1 with Nrxn binding site mutant was generated by making previously described mutations Q395A/E397A/L399A/N400A/D402A (Arac et al., 2007).)\\n\\nLenti-Syn-HA-Nlgn1-dimmt (Nlgn1 dimerization mutant was generated by making previously described mutations F458A/M459A/W463A (Arac et al., 2007; Ko et al., 2009).)'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Lenti-Syn-HA-Nlgn1-MDGA1mt (We aligned the Nlgn1 and Nlgn2 sequences, made the Nlgn1 with MDGA1 binding site mutations H303A/D387K/E397K.)\\n\\nLenti-Syn-HA-Nlgn2-WT (Nlgn2-wild-type)\\n\\nLenti-Syn-HA-Nlgn2-GPI (We deleted all transmembrane regions and cytoplasmic sequences of Nlgn2 and attaches its extracellular domains to the membrane using a GPI-anchor.)\\n\\nLenti-Syn-HA-Nlgn2-Nlgn1 (We made extracellular domain of Nlgn2 transplanted onto Nlgn1 intercellular.)\\n\\nLenti-Syn-HA-Nlgn2-Nrxnmt (We aligned the Nlgn1 and Nlgn2 sequences, made the Nlgn2 with Nrxn binding site mutations Q370A/E372A/L374A/N375A/D377A.)\\n\\nLenti-Syn-HA-Nlgn2-dimmt (We aligned the Nlgn1 and Nlgn2 sequences, made the Nlgn2 dimerization mutations F433A/M434A/W438A.)'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Lenti-Syn-HA-Nlgn2-MDGA1mt (Nlgn2 with MDGA1 binding site mutant was generated by making previously described mutations H278A/D362K/E372K (Gangwar et al., 2017).)\\n\\nLenti-Syn-HA-Nlgn2mt1 (We replaced the Nlgn2-PDZ domain HSTTRV with Nrxn-PDZ domain KKNKDKEYYV.)\\n\\nLenti-Syn-HA-Nlgn2mt2 (We deleted Nlgn2 intracellular sequences PPPPPPPPSLHPFPPPPPTATSHNNTLPHP.)\\n\\nLenti-Syn-HA-Nlgn2mt3 (We deleted Nlgn2 intracellular sequences EEELVSLQLKRGGGVGADPAE.)\\n\\nLenti-Syn-HA-Nlgn2mt4 (We deleted Nlgn2 intracellular sequences GSGSGVPGGGPLLPTAGRELPEEEELVSLQLKRGGGVGADPAE.)\\n\\nLenti-Syn-HA-Nlgn2mt5 (We deleted Nlgn2 intracellular sequences GSGSGVPGGGPLLPTAGRELPEEEELVSLQLKRGGGVGADPAE and PPPPPPPPSLHPFPPPPPTATSHNNTLPHP.)'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Lenti-Syn-HA-Nlgn2-Nlgn1 Y-A (We made extracellular domain of Nlgn2 transplanted onto Nlgn1 intercellular and made gephyrin binding point mutation as previously described Y770A (Poulopoulos et al., 2009).)\\n\\nLenti-Syn-HA-Nlgn2-Nlgn1 dele Geph (We deleted the whole Nlgn1 with gephyrin binding domain sequences PDYTLALRRAPDDVP)\\n\\nNeuronal culture and lentivirus infection'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Cell culture of hippocampal primary neurons from both male and female Nlgn1234 cKO newborn (P0) mice was performed as described previously (Wang et al, 2022). Briefly, dissected hippocampus was digested at 37 C for 25 min with 10U/ml papain in HBSS buffer, washed three times with plating medium (MEM supplemented with 0.5% glucose, 0.02% NaHCO3, 0.1 mg/ml transferrin, 10% FBS, 2 mM L-glutamine, and 0.025% mg/ml insulin), and then gently dissociated in plating medium, and filter with 70 mm cell strainer, and seeded on Matrigel pre-coated coverslip placed inside 24-well plate. The day of plating was considered as 0 days in vitro (DIV 0). After 24 h (DIV 1), 90% of the plating medium was replaced with neuronal growth medium (Neurobasal supplemented with 0.5%'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='with neuronal growth medium (Neurobasal supplemented with 0.5% glucose, 0.02% NaHCO3, 0.1 mg/ml transferrin, 5% FBS, 2% B27 supplement, and 0.5 mM L-glutamine), At DIV3, 50% of the medium was replaced with fresh growth medium additionally supplemented with finally concertation 4 mM Ara-C. Cultured hippocampal neurons were infected at DIV 4 with lentiviruses expressing Cre-EGFP or Cre-EGFP and subsequently were infected at DIV 6 with lentiviruses expressing different neuroligin constructs.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Immunohistochemistry'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='All solutions were made fresh and filtered via a 0.22 mm filter prior to staring experiments. For HA surface-labeling experiments were performed as described previously (Trotter et al, 2019). Briefly, primary neurons were first washed at room temperature once with a normal Tyrode’s bath solution, and then incubated for 20 min with purified Mouse anti-HA monoclonal antibody (1:500; BioLegend, Cat. #: 901501) or Rabbit anti-HA monoclonal antibody (1:500; Cell Signaling Technologies, Cat. #: 3724S) diluted in normal Tyrode’s bath solution. Cultures were then gently washed three times with normal Tyrode’s bath solution, followed by fixation for 20 min at 4 C with 4% PFA. Following fixation, cultures were washed three times with Dulbecco’s PBS (DPBS). For'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='were washed three times with Dulbecco’s PBS (DPBS). For surface labeling experiments (All the Nlgn constructs were all incubated live with an anti-HA monoclonal antibody and then fixed and further processed) to be used for conventional imaging, cultures were blocked for 1 h at room temperature with antibody dilution buffer (ADB) without Triton -100, which contains 5% normal goat serum diluted in DPBS. Cultures were then labeled with Alexa Fluor-conjugated secondary antibody (1:1000; Invitrogen) diluted in ADB for 1h at room temperature. Next for non-surface labeling, cells were washed and permeabilized and blocked for 1 h with ADB with Triton -100, which contains 0.2% Triton -100 and 5% normal goat serum diluted in DPBS. Non-surface primary antibodies'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='goat serum diluted in DPBS. Non-surface primary antibodies Guinea pig anti-Gephyrin monoclonal antibody (1:200, Synaptic Systems, Cat. #: 147318) or Mouse anti-Gephyrin monoclonal antibody (1:1000, Synaptic Systems, Cat. #: 147011)/ Rabbit anti-Homer1 monoclonal antibody (1:500, Millipore, Cat. #: ABN37) or Guinea pig anti-Homer1 monoclonal antibody (1:500, Synaptic Systems, Cat. #: 160005) /Chicken anti-MAP2 monoclonal antibody (1:1000, Encor, Cat. #: CPCA-MAP2) were diluted in ADB, and cells were incubated overbought at 4 C or 2h at room temperature. Cultures were washed three times and then incubated with either Alexa Fluor-conjugated secondary antibodies (1:1000; Invitrogen) in ABD for 1h. Following three washed, coverslip for conventional imaging were'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Following three washed, coverslip for conventional imaging were inverted onto glass microscope slides with Fluoromount-G Mounting medium (Southern Biotech). In certain experiments, the number of cells analyzed was relatively low, as indicated in Fig1D and 1F. Despite this, the observed effect size and the consistency of the results suggest that the sample size was sufficient.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Immunoblotting'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Hippocampal neurons in 24-well plates were lysed in RIPA buffer containing 150 mM NaCl, 1% Triton X-100, 0.1% SDS, 25 mM Tris-HCl, pH 7.4, and Complete EDTA-Free Protease Inhibitor Cocktail (Sigma Millipore). Lysates were incubated on ice for 30 min and clarified by centrifugation at 14,000 g for 30 min at 4°C. Lysates were boiled in sample buffer containing 1% b -mercaptoethanol at 42 C for 30 min. Proteins were analyzed by SDS-PAGE using 4%-20% Mini-Protean TGX precast gels (Bio-Rad). Proteins were transferred onto nitrocellulose membranes for 7 min at 25 V using the Trans-Blot Turbo transfer system (Bio-Rad). Membranes were blocked in 5% milk diluted in TBST for 1 h at room temperature. Membranes were then incubated overnight at 4°C with the following'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='were then incubated overnight at 4°C with the following primary antibodies diluted in 5% BSA diluted in TBST solution: Mouse anti-Nlgn1 (1:1000, Synaptic Systems, Cat. #: ab308451), Mouse anti-Nlgn2 (1:1000, Synaptic Systems, Cat. #: ab317510), Rabbit anti-Nlgn3 (639B, 1:500, T.C.S. laboratory), and Rabbit anti-Tuj1 (1:1000, BioLegend, Cat. #: 802001) Membranes were subsequently incubated for 1 h at room temperature with the following compatible secondary antibodies (LI-COR), diluted 1:10,000 in blocking solution: IRDye 800LT donkey anti-mouse or anti-Rabbit; IRDye 800CW donkey anti-mouse or anti-Rabbit. Quantitative analysis was performed by a dual-channel infrared imaging system, an Odyssey Infrared Imager CLX, and Image Studio 5.2.5 software (LI-COR).'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Confocal image acquisition and analysis\\n\\nSerial confocal z-stack images were acquired using a Nikon confocal microscope (A1RSi) with a 40 objective. Images were analyzed with by NIS-Elements AR analysis software. \\n\\nElectrophysiology'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Electrophysiology \\n\\nElectrophysiological recordings were performed from primary hippocampal culture neurons plated on coverslip, which were placed in a recording chamber mounted on a fixed stage inverted phase-contrast microscope (Olympus). Patch electrodes (3-5 M) were pulled from borosilicate glass capillary tubes (Warner Instruments) using a PC-10 pipette puller (Narishige). Whole-cell capacitance and series resistances were recorded and compensated to >80%, and in addition, series resistances were less than two times the tip resistance. The Tyrode’s bath solution contained (in mM): 129 NaCl, 5 KCl, 2 CaCl2, 1 MgCl2, 0.01 glycine, 30 D-glucose and 25 HEPES, pH 7.2–7.4.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='All recording under voltage-clamp model with a pipette solution containing (in mM): 135 CsCl, 10 HEPES, 10 EGTA, 2 Mg-ATP, 2 Na2GTP, and 5 QX-314, pH 7.35 (adjusted with CsOH). Presynaptic Action-potential for evoked synaptic responses were triggered by 0.5-ms current (40-90 \\uf06dA) injections through a local extracellular electrode (FHC concentric bipolar electrode, Catalogue number CBAEC75) placed ~100 \\uf06dm from the soma of neurons recorded. Inhibitory synaptic currents were made in presence of 20 mM CNQX and 50 mM APV to block AMPAR and NMDAR-mediated currents respectively. GABAR-IPSCs were recorded at -60 mV and measured at the peak of the current.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Excitatory synaptic currents were made in presence of 100 mM picrotoxin to block inhibitory currents and a small (10 nM) amount of TTX to reduced epileptiform activity. AMPAR-EPSCs were recorded as the peak current (~2 ms window) at -70 mV and NMDAR-EPSCs at +40 mV. \\n\\nQuantifications and statistical analyses'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Quantifications and statistical analyses\\n\\nElectrophysiological data were analyzed in Clampfit 10.7 (Molecular Devices). For clarity, all stimulus were blanked and not shown in the figures. All data were shown as means  SEM, numbers of cells and batches analyzed are shown in the bars. Statistical significance (p<0.05; p<0.01; p<0.001; p<0.0001, non-significant comparisons are indicted as NS) was analyzed with Prism 9, GraphPad. Unpaired t test was used for comparison between two groups. One-way ANOVA was used for comparison among more than two groups. \\n\\nData availability\\n\\nNo data have been deposited to public repositories.\\n\\nAcknowledgements'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Acknowledgements\\n\\nThis study was supported by grant from the National Institute of Mental Health (NIMH) (2R01MH092931 to M.W and T.C.S.)\\n\\n\\n\\nDisclosure and competing interests statement \\n\\nThe authors declare no competing interest.\\n\\n\\n\\n\\n\\n\\n\\n\\nREFERENCES\\n\\nAoto J, Foldy C, Ilcus SM, Tabuchi K, Sudhof TC (2015) Distinct circuit-dependent functions of presynaptic neurexin-3 at GABAergic and glutamatergic synapses. Nat Neurosci 18: 997-1007\\n\\nArac D, Boucard AA, Ozkan E, Strop P, Newell E, Sudhof TC, Brunger AT (2007) Structures of neuroligin-1 and the neuroligin-1/neurexin-1 beta complex reveal specific protein-protein and protein-Ca2+ interactions. Neuron 56: 992-1003'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Banovic D, Khorramshahi O, Owald D, Wichmann C, Riedt T, Fouquet W, Tian R, Sigrist SJ, Aberle H (2010) Drosophila neuroligin 1 promotes growth and postsynaptic differentiation at glutamatergic neuromuscular junctions. Neuron 66: 724-738\\n\\nBolliger MF, Pei J, Maxeiner S, Boucard AA, Grishin NV, Sudhof TC (2008) Unusually rapid evolution of Neuroligin-4 in mice. Proc Natl Acad Sci U S A 105: 6421-6426\\n\\nBudreck EC, Scheiffele P (2007) Neuroligin-3 is a neuronal adhesion protein at GABAergic and glutamatergic synapses. Eur J Neurosci 26: 1738-1748\\n\\nCalahorro F, Ruiz-Rubio M (2012) Functional phenotypic rescue of Caenorhabditis elegans neuroligin-deficient mutants by the human and rat NLGN1 genes. PLoS One 7: e39277'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Chanda S, Hale WD, Zhang B, Wernig M, Sudhof TC (2017) Unique versus Redundant Functions of Neuroligin Genes in Shaping Excitatory and Inhibitory Synapse Properties. J Neurosci 37: 6816-6836\\n\\nChoii G, Ko J (2015) Gephyrin: a central GABAergic synapse organizer. Exp Mol Med 47: e158\\n\\nChubykin AA, Atasoy D, Etherton MR, Brose N, Kavalali ET, Gibson JR, Sudhof TC (2007) Activity-dependent validation of excitatory versus inhibitory synapses by neuroligin-1 versus neuroligin-2. Neuron 54: 919-931'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Connor SA, Ammendrup-Johnsen I, Chan AW, Kishimoto Y, Murayama C, Kurihara N, Tada A, Ge Y, Lu H, Yan R et al (2016) Altered Cortical Dynamics and Cognitive Function upon Haploinsufficiency of the Autism-Linked Excitatory Synaptic Suppressor MDGA2. Neuron 91: 1052-1068\\n\\nConnor SA, Elegheert J, Xie Y, Craig AM (2019) Pumping the brakes: suppression of synapse development by MDGA-neuroligin interactions. Curr Opin Neurobiol 57: 71-80\\n\\nDai J, Aoto J, Sudhof TC (2019) Alternative Splicing of Presynaptic Neurexins Differentially Controls Postsynaptic NMDA and AMPA Receptor Responses. Neuron 102: 993-1008 e1005'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Dai J, Patzke C, Liakath-Ali K, Seigneur E, Sudhof TC (2021) GluD1 is a signal transduction device disguised as an ionotropic receptor. Nature 595: 261-265\\n\\nElegheert J, Cvetkovska V, Clayton AJ, Heroven C, Vennekens KM, Smukowski SN, Regan MC, Jia W, Smith AC, Furukawa H et al (2017a) Structural Mechanism for Modulation of Synaptic Neuroligin-Neurexin Signaling by MDGA Proteins. Neuron 96: 242-244\\n\\nElegheert J, Cvetkovska V, Clayton AJ, Heroven C, Vennekens KM, Smukowski SN, Regan MC, Jia W, Smith AC, Furukawa H et al (2017b) Structural Mechanism for Modulation of Synaptic Neuroligin-Neurexin Signaling by MDGA Proteins. Neuron 95: 896-913 e810'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Gan KJ, Sudhof TC (2020) SPARCL1 Promotes Excitatory But Not Inhibitory Synapse Formation and Function Independent of Neurexins and Neuroligins. J Neurosci 40: 8088-8102\\n\\nGangwar SP, Zhong X, Seshadrinathan S, Chen H, Machius M, Rudenko G (2017) Molecular Mechanism of MDGA1: Regulation of Neuroligin 2:Neurexin Trans-synaptic Bridges. Neuron 94: 1132-1141 e1134\\n\\nGiannone G, Mondin M, Grillo-Bosch D, Tessier B, Saint-Michel E, Czondor K, Sainlos M, Choquet D, Thoumine O (2013) Neurexin-1beta binding to neuroligin-1 triggers the preferential recruitment of PSD-95 versus gephyrin through tyrosine phosphorylation of neuroligin-1. Cell Rep 3: 1996-2007'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Gibson JR, Huber KM, Sudhof TC (2009) Neuroligin-2 deletion selectively decreases inhibitory synaptic transmission originating from fast-spiking but not from somatostatin-positive interneurons. J Neurosci 29: 13883-13897\\n\\nGraf ER, Zhang X, Jin SX, Linhoff MW, Craig AM (2004) Neurexins induce differentiation of GABA and glutamate postsynaptic specializations via neuroligins. Cell 119: 1013-1026\\n\\nGuo A, Gu H, Zhou J, Mulhern D, Wang Y, Lee KA, Yang V, Aguiar M, Kornhauser J, Jia X et al (2014) Immunoaffinity enrichment and mass spectrometry analysis of protein methylation. Mol Cell Proteomics 13: 372-387'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Han KA, Ko JS, Pramanik G, Kim JY, Tabuchi K, Um JW, Ko J (2018) PTPsigma Drives Excitatory Presynaptic Assembly via Various Extracellular and Intracellular Mechanisms. J Neurosci 38: 6700-6721\\n\\nHoon M, Soykan T, Falkenburger B, Hammer M, Patrizi A, Schmidt KF, Sassoe-Pognetto M, Lowel S, Moser T, Taschenberger H et al (2011) Neuroligin-4 is localized to glycinergic postsynapses and regulates inhibition in the retina. Proc Natl Acad Sci U S A 108: 3053-3058\\n\\nHu Z, Hom S, Kudze T, Tong XJ, Choi S, Aramuni G, Zhang W, Kaplan JM (2012) Neurexin and neuroligin mediate retrograde synaptic inhibition in C. elegans. Science 337: 980-984'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Hunter JW, Mullen GP, McManus JR, Heatherly JM, Duke A, Rand JB (2010) Neuroligin-deficient mutants of C. elegans have sensory processing deficits and are hypersensitive to oxidative stress and mercury toxicity. Dis Model Mech 3: 366-376\\n\\nIchtchenko K, Hata Y, Nguyen T, Ullrich B, Missler M, Moomaw C, Sudhof TC (1995) Neuroligin 1: a splice site-specific ligand for beta-neurexins. Cell 81: 435-443\\n\\nIchtchenko K, Nguyen T, Sudhof TC (1996) Structures, alternative splicing, and neurexin binding of multiple neuroligins. J Biol Chem 271: 2676-2682'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Jamain S, Quach H, Betancur C, Rastam M, Colineaux C, Gillberg IC, Soderstrom H, Giros B, Leboyer M, Gillberg C et al (2003) Mutations of the X-linked genes encoding neuroligins NLGN3 and NLGN4 are associated with autism. Nat Genet 34: 27-29\\n\\nJiang M, Polepalli J, Chen LY, Zhang B, Sudhof TC, Malenka RC (2017) Conditional ablation of neuroligin-1 in CA1 pyramidal neurons blocks LTP by a cell-autonomous NMDA receptor-independent mechanism. Mol Psychiatry 22: 375-383\\n\\nKim J, Kim S, Kim H, Hwang IW, Bae S, Karki S, Kim D, Ogelman R, Bang G, Kim JY et al (2022) MDGA1 negatively regulates amyloid precursor protein-mediated synapse inhibition in the hippocampus. Proc Natl Acad Sci U S A 119'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Kim JA, Kim D, Won SY, Han KA, Park D, Cho E, Yun N, An HJ, Um JW, Kim E et al (2017) Structural Insights into Modulation of Neurexin-Neuroligin Trans-synaptic Adhesion by MDGA1/Neuroligin-2 Complex. Neuron 94: 1121-1131 e1126\\n\\nKo J, Zhang C, Arac D, Boucard AA, Brunger AT, Sudhof TC (2009) Neuroligin-1 performs neurexin-dependent and neurexin-independent functions in synapse validation. EMBO J 28: 3244-3255\\n\\nLee K, Kim Y, Lee SJ, Qiang Y, Lee D, Lee HW, Kim H, Je HS, Sudhof TC, Ko J (2013) MDGAs interact selectively with neuroligin-2 but not other neuroligins to regulate inhibitory synapse development. Proc Natl Acad Sci U S A 110: 336-341'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Liang J, Xu W, Hsu YT, Yee AX, Chen L, Sudhof TC (2015) Conditional neuroligin-2 knockout in adult medial prefrontal cortex links chronic changes in synaptic inhibition to cognitive impairments. Mol Psychiatry 20: 850-859\\n\\nMaglioni S, Schiavi A, Melcher M, Brinkmann V, Luo Z, Laromaine A, Raimundo N, Meyer JN, Distelmaier F, Ventura N (2022) Neuroligin-mediated neurodevelopmental defects are induced by mitochondrial dysfunction and prevented by lutein in C. elegans. Nat Commun 13: 2620\\n\\nMarro SG, Chanda S, Yang N, Janas JA, Valperga G, Trotter J, Zhou B, Merrill S, Yousif I, Shelby H et al (2019) Neuroligin-4 Regulates Excitatory Synaptic Transmission in Human Neurons. Neuron 103: 617-626 e616'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Nguyen QA, Horn ME, Nicoll RA (2016) Distinct roles for extracellular and intracellular domains in neuroligin function at inhibitory synapses. Elife 5\\n\\nNguyen T, Sudhof TC (1997) Binding properties of neuroligin 1 and neurexin 1beta reveal function as heterophilic cell adhesion molecules. J Biol Chem 272: 26032-26039\\n\\nPettem KL, Yokomaku D, Takahashi H, Ge Y, Craig AM (2013) Interaction between autism-linked MDGAs and neuroligins suppresses inhibitory synapse development. J Cell Biol 200: 321-336\\n\\nPoulopoulos A, Aramuni G, Meyer G, Soykan T, Hoon M, Papadopoulos T, Zhang M, Paarmann I, Fuchs C, Harvey K et al (2009) Neuroligin 2 drives postsynaptic assembly at perisomatic inhibitory synapses through gephyrin and collybistin. Neuron 63: 628-642'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Scheiffele P, Fan J, Choih J, Fetter R, Serafini T (2000) Neuroligin expressed in nonneuronal cells triggers presynaptic development in contacting axons. Cell 101: 657-669\\n\\nSong JY, Ichtchenko K, Sudhof TC, Brose N (1999) Neuroligin 1 is a postsynaptic cell-adhesion molecule of excitatory synapses. Proc Natl Acad Sci U S A 96: 1100-1105\\n\\nSudhof TC (2008) Neuroligins and neurexins link synaptic function to cognitive disease. Nature 455: 903-911\\n\\nSudhof TC (2017) Synaptic Neurexin Complexes: A Molecular Code for the Logic of Neural Circuits. Cell 171: 745-769'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Sun M, Xing G, Yuan L, Gan G, Knight D, With SI, He C, Han J, Zeng X, Fang M et al (2011) Neuroligin 2 is required for synapse development and function at the Drosophila neuromuscular junction. J Neurosci 31: 687-699\\n\\nSun Y, Li M, Geng J, Meng S, Tu R, Zhuang Y, Sun M, Rui M, Ou M, Xing G et al (2023) Neuroligin 2 governs synaptic morphology and function through RACK1-cofilin signaling in Drosophila. Commun Biol 6: 1056\\n\\nTabuchi K, Blundell J, Etherton MR, Hammer RE, Liu X, Powell CM, Sudhof TC (2007) A neuroligin-3 mutation implicated in autism increases inhibitory synaptic transmission in mice. Science 318: 71-76'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Takacs VT, Freund TF, Nyiri G (2013) Neuroligin 2 is expressed in synapses established by cholinergic cells in the mouse brain. PLoS One 8: e72450\\n\\nTanaka H, Miyazaki N, Matoba K, Nogi T, Iwasaki K, Takagi J (2012) Higher-order architecture of cell adhesion mediated by polymorphic synaptic adhesion molecules neurexin and neuroligin. Cell Rep 2: 101-110\\n\\nTreutlein B, Gokce O, Quake SR, Sudhof TC (2014) Cartography of neurexin alternative splicing mapped by single-molecule long-read mRNA sequencing. Proc Natl Acad Sci U S A 111: E1291-1299\\n\\nTrotter JH, Hao J, Maxeiner S, Tsetsenis T, Liu Z, Zhuang X, Sudhof TC (2019) Synaptic neurexin-1 assembles into dynamically regulated active zone nanoclusters. J Cell Biol 218: 2677-2698'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Tu H, Pinan-Lucarre B, Ji T, Jospin M, Bessereau JL (2015) C. elegans Punctin Clusters GABA(A) Receptors via Neuroligin Binding and UNC-40/DCC Recruitment. Neuron 86: 1407-1419\\n\\nTyagarajan SK, Fritschy JM (2014) Gephyrin: a master regulator of neuronal function? Nat Rev Neurosci 15: 141-156\\n\\nTyagarajan SK, Ghosh H, Yevenes GE, Nikonenko I, Ebeling C, Schwerdel C, Sidler C, Zeilhofer HU, Gerrits B, Muller D et al (2011) Regulation of GABAergic synapse formation and plasticity by GSK3beta-dependent phosphorylation of gephyrin. Proc Natl Acad Sci U S A 108: 379-384'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Uchigashima M, Ohtsuka T, Kobayashi K, Watanabe M (2016) Dopamine synapse is a neuroligin-2-mediated contact between dopaminergic presynaptic and GABAergic postsynaptic structures. Proc Natl Acad Sci U S A 113: 4206-4211\\n\\nVaroqueaux F, Aramuni G, Rawson RL, Mohrmann R, Missler M, Gottmann K, Zhang W, Sudhof TC, Brose N (2006) Neuroligins determine synapse maturation and function. Neuron 51: 741-754\\n\\nVaroqueaux F, Jamain S, Brose N (2004) Neuroligin 2 is exclusively localized to inhibitory synapses. Eur J Cell Biol 83: 449-456'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Wagner SA, Beli P, Weinert BT, Scholz C, Kelstrup CD, Young C, Nielsen ML, Olsen JV, Brakebusch C, Choudhary C (2012) Proteomic analyses reveal divergent ubiquitylation site patterns in murine tissues. Mol Cell Proteomics 11: 1578-1585\\n\\nWang J, Miao Y, Wicklein R, Sun Z, Wang J, Jude KM, Fernandes RA, Merrill SA, Wernig M, Garcia KC et al (2022) RTN4/NoGo-receptor binding to BAI adhesion-GPCRs regulates neuronal development. Cell 185: 218\\n\\nWu X, Morishita WK, Riley AM, Hale WD, Sudhof TC, Malenka RC (2019) Neuroligin-1 Signaling Controls LTP and NMDA Receptors by Distinct Molecular Pathways. Neuron 102: 621-635 e623'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Xing G, Gan G, Chen D, Sun M, Yi J, Lv H, Han J, Xie W (2014) Drosophila neuroligin3 regulates neuromuscular junction development and synaptic differentiation. J Biol Chem 289: 31867-31877\\n\\nYamasaki T, Hoyos-Ramirez E, Martenson JS, Morimoto-Tomita M, Tomita S (2017) GARLH Family Proteins Stabilize GABA(A) Receptors at Synapses. Neuron 93: 1138-1152 e1136\\n\\nYoshida T, Yamagata A, Imai A, Kim J, Izumi H, Nakashima S, Shiroshima T, Maeda A, Iwasawa-Okamoto S, Azechi K et al (2021) Canonical versus non-canonical transsynaptic signaling of neuroligin 3 tunes development of sociality in mice. Nat Commun 12: 1848'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Zhang B, Chen LY, Liu X, Maxeiner S, Lee SJ, Gokce O, Sudhof TC (2015) Neuroligins Sculpt Cerebellar Purkinje-Cell Circuits by Differential Control of Distinct Classes of Synapses. Neuron 87: 781-796\\n\\nZhang B, Gokce O, Hale WD, Brose N, Sudhof TC (2018) Autism-associated neuroligin-4 mutation selectively impairs glycinergic synaptic transmission in mouse brainstem synapses. J Exp Med 215: 1543-1553\\n\\n\\n\\n\\nFIGURE LEGENDS\\n\\nFigure 1. Nlgn1 and Nlgn2 are specifically localized on excitatory and inhibitory synapses.\\n\\nSchematic overview of experimental timeline and set-up.\\n\\nSchematic of Nlgn1-WT and Nlgn2-WT constructs.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Schematic of Nlgn1-WT and Nlgn2-WT constructs. \\n\\nRepresentative image from DIV14-16 cultured Nlgn1234 conditional knockout mice neurons, infected with Cre (blue) and either Nlgn1-WT (top) or Nlgn2-WT (bottom). The neurons were labeled with antibodies to Homer1 (red), HA (green), and MAP2 (blue). Scale bar: 20 mm. The right panels show an enlarged box area (arrowheads indicate HA puncta overlapped with Homer1 puncta). Scale bar: 5 mm.\\n\\nSummary graph of the HA-Homer1 overlap percentage in Nlgn1-WT and Nlgn2-WT conditions. (Bar and line graphs indicate mean  SEM; numbers of cells/experiments = 13/3 and 11/3 for each column, left to right. Statistical significance was assessed by unpaired t test, p<0.0001).'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Representative image of hippocampal neurons from DIV14-16 cultured Nlgn1234 conditional knockout mice, infected with Cre (blue) and either Nlgn1-WT (top) or Nlgn2-WT (bottom). The neurons were labeled with antibodies to Gephyrin (red), HA (green), and MAP2 (blue). Scale bar: 20 mm. The right panels show an enlarged box area (arrowheads indicate HA puncta overlapped with Gephyrin puncta). Scale bar: 5 mm.\\n\\nSummary graph of the HA-Gephyrin overlap percentage in Nlgn1-WT and Nlgn2-WT conditions. (Bar and line graphs indicate mean  SEM; numbers of cells/experiments = 15/3 and 14/3 for each column, left to right. Statistical significance was assessed by unpaired t test, p<0.0001).'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Figure 2. The intracellular sequence of Nlgn2 is necessary for inhibitory synapse function but can be fully replaced with the intracellular sequence of Nlgn1.\\n\\nSchematic of Nlgn2-WT, Nlgn2-GPI (Nlgn2 chimeric construct only has extracellular domain), Nlgn2-Nlgn1 (Nlgn2 extracellular domain with Nlgn1 intercellular domain) constructs. \\n\\nRepresentative traces of evoked GABAR-IPSC recorded from DIV14-16 cultured Nlgn1234 conditional knockout mice neurons in the five conditions (Cre, Cre, Cre+Nlgn2-WT, Cre+Nlgn2-GPI and Cre+Nlgn2-Nlgn1).'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Summary graph of evoked GABAR-IPSC amplitude in all conditions (Bar and line graphs indicate mean  SEM; numbers of cells/experiments = 16/3, 16/3, 16/3, 14/3 and 14/3 for each column, left to right).  Nonsignificant p>0.05; p<0.01; p<0.001; p<0.0001, one-way ANOVA with post-hoc Dunnett’s Multiple comparisons. \\n\\n\\n\\n\\nFigure 3. The cytoplasmic gephyrin binding motif is required for Nlgn2 function whereas tyrosine phosphorylation is not.\\n\\nAlignment of Nlgn1 and Nlgn2 gephyrin binding sequence.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Alignment of Nlgn1 and Nlgn2 gephyrin binding sequence.  \\n\\nSchematic of Nlgn2-Nlgn1Y770A (Nlgn2 extracellular domain with Nlgn1 intercellular domain with a gephyrin binding point mutation), Nlgn2-Nlgn1DelGeph (Nlgn2 extracellular domain with Nlgn1 intercellular domain truncation of the entire gephyrin binding sequence) constructs.\\n\\nRepresentative traces of evoked GABAR-IPSC recorded from DIV14-16 cultured Nlgn1234 conditional knockout mice neurons in the four conditions (Cre, Cre, Cre+Nlgn2-Nlgn1Y770A, Cre+Nlgn2-Nlgn1DelGeph).'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Summary graph of evoked GABAR-IPSC amplitude in all conditions (Bar and line graphs indicate mean  SEM; numbers of cells/experiments = 15/3,16/3,15/3, and 15/3 for each column, left to right).  Nonsignificant p>0.05; p<0.001, one-way ANOVA with post-hoc Dunnett’s Multiple comparisons. Nonsignificant relations are indicated as ns.\\n\\n\\n\\n\\n\\n\\nFigure 4. A conserved cytoplasmic 21 residue sequence is necessary for Nlgn2 function, whereas the PDZ-domain binding sequence is dispensable. \\n\\nAlignment of Nlgn1 and Nlgn2 intracellular domain. Colored boxes show the different Nlgn2 intracellular domain truncations.\\n\\nSchematic of Nlgn2-mt1, Nlgn2-mt2, Nlgn2-mt3, Nlgn2-mt3, Nlgn2-mt4 and Nlgn2-mt5 constructs.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Representative traces of evoked GABAR-IPSC recorded from DIV14-16 cultured Nlgn1234 conditional knockout mice neurons in the seven conditions (Cre, Cre, Cre+Nlgn2-mt1, Cre+Nlgn2-mt2, Cre+Nlgn2-mt3, Cre+Nlgn2-mt4, Cre+Nlgn2-mt5).\\n\\nSummary graph of evoked GABAR-IPSC amplitude in all conditions (Bar and line graphs indicate mean  SEM; numbers of cells/experiments = 19/4,19/4,19/4,19/4,17/4,13/3, and 13/3 for each column, left to right).  Nonsignificant p>0.05; p<0.01; p<0.001, one-way ANOVA with post-hoc Dunnett’s Multiple comparisons. Nonsignificant relations are indicated as ns.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Figure 5. Different from the functional promiscuity of the Nlgn1 and Nlgn2 cytoplasmic sequences, their extracellular sequences encode synapse type-specificity that is independent of MDGA binding.\\n\\nSchematic of Nlgn1-Nlgn2 (Nlgn1 extracellular domain with Nlgn2 intercellular domain) constructs. \\n\\nRepresentative traces of evoked GABAR-IPSC recorded from DIV14-16 cultured Nlgn1234 conditional knockout mice neurons in the three conditions (Cre, Cre, Cre+Nlgn1-Nlgn2). Note that we rescued Nlgn1-Nlgn2 here from the identical three batches of Figure 2, so the Cre and Cre traces here are the same as in Figure 2.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Summary graph of evoked GABAR-IPSC amplitude in all conditions (Bar and line graphs indicate mean  SEM; numbers of cells/experiments = 16/3,16/3, and 12/3 for each column, left to right). p<0.0001, one-way ANOVA with post-hoc Dunnett’s Multiple comparisons. Nonsignificant relations are indicated as ns. Note that we rescued Nlgn1-Nlgn2 here from the identical three batches of Figure 2, so the Cre and Cre data here are the same as in Figure 2.\\n\\nSchematic of Nlgn2-MDGA1mt (Nlgn2 with MDGA1-binding point mutation) constructs. \\n\\nRepresentative traces of evoked GABAR-IPSC recorded from DIV14-16 cultured Nlgn1234 conditional knockout mice neurons in the three conditions (Cre, Cre, Cre+Nlgn2-MDGA1mt).'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Summary graph of evoked GABAR-IPSC amplitude in all conditions (Bar and line graphs indicate mean  SEM; numbers of cells/experiments = 16/3,15/3, and 18/3 for each column, left to right).  Nonsignificant p>0.05; p<0.01, one-way ANOVA with post-hoc Dunnett’s Multiple comparisons. Nonsignificant relations are indicated as ns.\\n\\n\\n\\n\\nFigure 6. Nlgn1 extracellular domain provides the specificity of Nlgn1 excitatory function.\\n\\nSchematic of Nlgn1-WT, Nlgn1-GPI (Nlgn1 chimeric construct only has extracellular domain), Nlgn2-Nlgn1 constructs.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Representative traces of evoked AMPAR- and NMDAR-EPSCs recorded from DIV14-16 cultured Nlgn1234 conditional knockout mice neurons. AMPAR-EPSCs and NMDAR-EPSC recorded at -70 mV and +40 mV, respectively, in the five conditions (Cre, Cre, Cre+Nlgn1-WT, Cre+Nlgn1-GPI, Cre+Nlgn2-Nlgn1).  \\n\\nSummary graph of evoked AMPAR-EPSC amplitude in all conditions (Bar and line graphs indicate mean  SEM; numbers of cells/experiments = 22/4, 17/4, 22/4, 23/4, and 16/3 for each column, left to right). Nonsignificant p>0.05;p<0.05, one-way ANOVA with post-hoc Dunnett’s Multiple comparisons. Nonsignificant relations are indicated as ns.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Summary graph of evoked NMDAR-EPSC amplitude in all conditions (Bar and line graphs indicate mean  SEM; numbers of cells/experiments = 22/4, 17/4, 22/4, 23/4, and 16/3 for each column, left to right). Nonsignificant p>0.05; p<0.01; p<0.0001, one-way ANOVA with post-hoc Dunnett’s Multiple comparisons. Nonsignificant relations are indicated as ns.\\n\\n\\n\\n\\nFigure 7. Nlgn1 function in excitatory synapses does not require Neurexin-binding, MDGA-binding, or dimerization.\\n\\nSchematic of Nlgn1-NRXNmt (Nlgn1 with neurexin-binding point mutation), Nlgn1-dimmt (Nlgn1 dimerization point mutation), Nlgn1-MDGA1mt (Nlgn1 with MDGA1-binding point mutation) constructs.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Representative traces of evoked AMPAR- and NMDAR-EPSCs recorded from DIV14-16 cultured Nlgn1234 conditional knockout mice neurons. AMPAR-EPSCs and NMDAR-EPSC recorded at -70 mV and +40 mV, respectively, in the five conditions (Cre, Cre, Cre+Nlgn1-NRXNmt, Cre+Nlgn1-dimmt, Cre+Nlgn1-MDGA1mt).  \\n\\nSummary graph of evoked AMPAR-EPSC amplitude in all conditions (Bar and line graphs indicate mean  SEM; numbers of cells/experiments = 17/4, 14/4, 15/4, 18/4, and 15/4 for each column, left to right). Nonsignificant p>0.05; p<0.05, one-way ANOVA with post-hoc Dunnett’s Multiple comparisons. Nonsignificant relations are indicated as ns.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Summary graph of evoked NMDAR-EPSC amplitude in all conditions (Bar and line graphs indicate mean  SEM; numbers of cells/experiments = 17/4, 14/4, 15/4, 18/4, and 15/4 for each column, left to right). Nonsignificant p>0.05; p<0.01, one-way ANOVA with post-hoc Dunnett’s Multiple comparisons. Nonsignificant relations are indicated as ns.\\n\\n\\n\\n\\nExpanded View Figure LEGENDS\\n\\nFigure EV1. Neuroligin protein measurement and co-staining Homer1 with Gephyrin in Nlgn1234 cKO cultured hippocampal neurons (related to Fig 1).'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Representative images of western blot for Nlgn1, Nlgn2, and Nlgn3 protein expressions from DIV14-16 cultured Nlgn1234 conditional knockout mice neurons, infected with either Cre or Cre. Note that the Nlgn3 antibody detects Nlgn3 (blue arrow) and nonspecific band (red arrow).\\n\\nSummary graphs of western blot analysis for Nlgn1, Nlgn2, and Nlgn3 protein expressions. (Bar and line graphs indicate mean  SEM; samples/experiments=3/3. 3 technical replicates. Statistical significance was assessed by unpaired t test, p<0.001; p<0.0001).'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Representative image from DIV14-16 cultured Nlgn1234 conditional knockout mice neurons. The neuron was labeled with antibodies to Homer1 (Green), Gephyrin (red), and MAP2 (blue). Scale bar: 20 mm. The right panels show an enlarged box area (arrowheads indicate Homer1 puncta overlapped with Gephyrin puncta) Scale bar: 5 \\uf06dm. \\n\\nSummary graph of Homer1 overlap percentage with Gephyrin and Gephyrin overlap percentage with Homer1 (Bar and line graphs indicate mean  SEM; numbers of cells/experiment=14/1).\\n\\n\\n\\n\\nFigure EV2. Nlgn2 is specifically localized on inhibitory synapses is determined by the extracellular sequence of Nlgn2 and cytoplasmic gephyrin binding motif is required whereas tyrosine phosphorylation is not (related to Fig 2, Fig3 and Fig 5).'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Representative image from DIV14-16 cultured Nlgn1234 conditional knockout mice neurons, infected with Cre (blue) and Nlgn2-GPI, Nlgn1-Nlgn2, Nlgn2-Nlgn1, and Nlgn2-MDGA1mt (from top to bottom). The neurons were labeled with antibodies to Gephyrin (red), HA (green), and MAP2 (blue). Scale bar: 20 mm. The right panels show an enlarged box area (arrowheads indicate HA puncta overlapped with Gephyrin puncta). Scale bar: 5 mm.\\n\\nSummary graph of the HA-Gephyrin overlap percentage in Nlgn2-GPI, Nlgn1-Nlgn2, Nlgn2-Nlgn1, and Nlgn2-MDGA1mt conditions.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Summary graph of the surface levels of HA-tagged Nlgn2 forms relative to MAP2 signal. B and C (Bar and line graphs indicate mean  SEM; numbers of cells/experiments = 10/3, 11/3, 9/3 and 9/3 for each column, left to right. Statistical significance was assessed by one-way ANOVA with post-hoc Dunnett’s Multiple comparisons, Nonsignificant p>0.05;  p<0.01; p<0.0001).\\n\\nRepresentative image from DIV14-16 cultured Nlgn1234 conditional knockout mice neurons, infected with Cre (blue) and Nlgn2-Nlgn1Y770A and Nlgn2-Nlgn1DelGeph. The neurons were labeled with antibodies to Gephyrin (red), HA (green), and MAP2 (blue). Scale bar: 20 mm. The right panels show an enlarged box area (arrowheads indicate HA puncta overlapped with Gephyrin puncta). Scale bar: 5 mm.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Summary graph of the HA-Gephyrin overlap percentage in Nlgn2-Nlgn1Y770A and Nlgn2-Nlgn1DelGeph conditions. \\n\\nSummary graph of the surface levels of HA-tagged Nlgn2-Nlgn1Y770A and Nlgn2-Nlgn1DelGeph relative to MAP2 signal. E and F (Bar and line graphs indicate mean  SEM; numbers of cells/experiments = 19/3 and 12/3 for each column, left to right.  Statistical significance was assessed by one-way ANOVA with post-hoc Dunnett’s Multiple comparisons, Nonsignificant p>0.05;  p<0.0001).'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Figure EV3. All Nlgn2 intracellular domain truncation constructs can properly traffic to the neuron membrane surface, but Nlgn2-NRXNmt and Nlgn2-dimmt constructs traffic to the neuron membrane surface signification decrease compared to the Nlgn2-WT construct (related to Fig 4).\\n\\nRepresentative image from DIV14-16 cultured Nlgn1234 conditional knockout mice neurons, infected with Cre (blue) and Nlgn2-mt1, Nlgn2-mt2, Nlgn2-mt3, Nlgn2-mt4 and Nlgn2-mt5 (from left to right). The neurons were labeled with antibodies to HA (green) and MAP2 (blue). Scale bar: 20 mm.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Representative image from DIV14-16 cultured Nlgn1234 conditional knockout mice neurons, infected with Nlgn2-WT, Nlgn2-NRXNmt, and Nlgn2-dimmt. The neurons were labeled with antibodies to total HA (red) and surface HA (green). Scale bar: 20 mm. \\n\\nSummary graph of the surface HA/total HA in Nlgn2-WT, Nlgn2-NRXNmt, and Nlgn2-dimmt conditions (Bar and line graphs indicate mean  SEM. numbers of cells/experiments = 10/3, 9/3 and 7/3 for each column, left to right.  Statistical significance was assessed by one-way ANOVA with post-hoc Dunnett’s Multiple comparisons, p<0.0001).'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Figure EV4. Nlgn1 is specifically localized on excitatory synapses is determined by the extracellular sequence of Nlgn1, and does not require Neurexin-binding, MDGA-binding, or dimerization (related to Fig 6 and Fig 7).\\n\\nRepresentative image from DIV14-16 cultured Nlgn1234 conditional knockout mice neurons, infected with Cre (blue) and Nlgn1-GPI, Nlgn1-Nlgn2, and Nlgn2-Nlgn1. The neurons were labeled with antibodies to Homer1 (red), HA (green), and MAP2 (blue). Scale bar: 20 mm. The right panels show an enlarged box area (arrowheads indicate HA puncta overlapped with Homer1 puncta). Scale bar: 5 mm.\\n\\nSummary graph of the HA-Homer1 overlap percentage in Nlgn1-GPI, Nlgn1-Nlgn2, and Nlgn2-Nlgn1 conditions.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Summary graph of the surface levels of HA-tagged Nlgn1 forms relative to MAP2 signal. B and C (Bar and line graphs indicate mean  SEM; numbers of cells/experiments = 10/3, 9/3, and 15/3 for each column, left to right. Statistical significance was assessed by one-way ANOVA with post-hoc Dunnett’s Multiple comparisons, Nonsignificant p>0.05;  p<0.01; p<0.0001).\\n\\nRepresentative image from DIV14-16 cultured Nlgn1234 conditional knockout mice neurons, infected with Cre (blue) and Nlgn1-NRNXmt, Nlgn1-dimmt, and Nlgn1-MDGA1mt. The neurons were labeled with antibodies to Homer1 (red), HA (green), and MAP2 (blue). Scale bar: 20 mm. The right panels show an enlarged box area (arrowheads indicate HA puncta overlapped with Homer1 puncta). Scale bar: 5 mm.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Summary graph of the HA-Homer1 overlap percentage in Nlgn1-NRNXmt, Nlgn1-dimmt, and Nlgn1-MDGA1mt conditions.\\n\\nSummary graph of the surface levels of HA-tagged Nlgn1 forms relative to MAP2 signal. E and F (Bar and line graphs indicate mean  SEM; numbers of cells/experiments = 14/3, 9/3, and 9/3 for each column, left to right. Statistical significance was assessed by one-way ANOVA with post-hoc Dunnett’s Multiple comparisons, Nonsignificant p>0.05).\\n\\n\\n\\n\\n\\n\\nFigure EV5. Nlgn1-Nlgn2 is sufficient for the glutamatergic synaptic transmission function of Nlgn1, but Nlgn2-WT doesn’t (related to Fig 6).'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Summary graph of evoked AMPAR-EPSC amplitude in all conditions (Bar and line graphs indicate mean  SEM; numbers of cells/experiments = 22/4, 17/4, 22/4, 7/2, and 11/3 for each column, left to right). Nonsignificant p>0.05; p<0.05, one-way ANOVA with post-hoc Dunnett’s Multiple comparisons. Nonsignificant relations are indicated as ns. Note that we rescued Nlgn1-Nlgn2 and Nlgn2-WT here from the same batches of Figure 6, so the Cre and Cre data here are the same as in Figure 6.'), Document(metadata={'source': './data/docx/EMBOR-2023-58706-T.docx'}, page_content='Summary graph of evoked NMDAR-EPSC amplitude in all conditions (Bar and line graphs indicate mean  SEM; numbers of cells/experiments = 22/4, 17/4, 22/4, 7/2, and 11/3 for each column, left to right). Nonsignificant p>0.05; p<0.001; p<0.0001, one-way ANOVA with post-hoc Dunnett’s Multiple comparisons. Nonsignificant relations are indicated as ns. Note that we rescued Nlgn1-Nlgn2 and Nlgn2-WT here from the same batches of Figure 6, so the Cre and Cre data here are the same as in Figure 6.\\n\\n\\n\\n\\n\\n\\n\\n1')]\n",
      "Processing document: ./data/docx/EMBOR-2024-58727V1.docx\n",
      "Split documents: [Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Title: Targeting the transcription factor YY1 is synthetic lethal with loss of the histone demethylase KDM5C\\n\\n\\n\\nQian Zheng1#, Pengfei Li2#, Yulong Qiang1#, Jiachen Fan1, Yuzhu Xing1, Ying Zhang1, Fan Yang1*, Feng Li1,3* and Jie Xiong4* \\n\\n1Department\\xa0of\\xa0Medical Genetics, TaiKang Medical School (School of Basic Medical Sciences), Wuhan University, Wuhan 430071, China\\n\\n2Inner Mongolia Key Laboratory of Molecular Pathology, Inner Mongolia Medical University, Huhhot, Inner Mongolia, 010059, China\\n\\n3Hubei Provincial Key Laboratory of Allergy and Immunology, Wuhan University, Wuhan 430071, China\\n\\n4Department\\xa0of\\xa0Immunology, TaiKang Medical School (School of Basic Medical Sciences), Wuhan University, Wuhan 430071, China'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='#These authors contribute equally to this paper.   \\n\\nCorrespondence to:\\n\\nJie Xiong\\n\\nE-mail: jiexiong@whu.edu.cn，Tel: +86 27 68759222\\n\\nDepartment\\xa0of\\xa0Immunology, TaiKang Medical School (School of Basic Medical Sciences), Wuhan University, Wuhan 430071, China\\n\\nFeng Li\\n\\nE-mail: fli222@whu.edu.cn, Tel: +86 27 68759222\\n\\nDepartment\\xa0of\\xa0Medical Genetics, TaiKang Medical School (School of Basic Medical Sciences), Wuhan University, Wuhan 430071, China.\\n\\nHubei Provincial Key Laboratory of Allergy and Immunology, Wuhan University, Wuhan 430071, China \\n\\nFan Yang\\n\\nE-mail: yangfan2022@whu.edu.cn, Tel: +86 27 68759222\\n\\nDepartment\\xa0of\\xa0Medical Genetics, TaiKang Medical School (School of Basic Medical Sciences), Wuhan University, Wuhan 430071, China.\\n\\n\\n\\n\\n\\n\\nAbstract'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='An understanding of the enzymatic and scaffolding functions of epigenetic modifiers is important for the development of epigenetic therapies for cancer. The H3K4me2/3 histone demethylase KDM5C has been shown to regulate transcription. The diverse roles of KDM5C are likely determined by its interacting partners, which are still largely unknown. In this study, we screen for KDM5C-binding proteins and show that YY1 interacts with KDM5C. A synergistic antitumor effect is exerted when both KDM5C and YY1 are depleted, and targeting YY1 appears to be a vulnerability in KDM5C-deficient cancer cells. Mechanistically, KDM5C promotes global YY1 chromatin recruitment, especially at promoters. Moreover, an intact KDM5C JmjC domain but not KDM5C histone demethylase'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='an intact KDM5C JmjC domain but not KDM5C histone demethylase activity is required for KDM5C-mediated YY1 chromatin binding. Transcriptional profiling reveals that dual inhibition of KDM5C and YY1 increases transcriptional repression of cell cycle- and apoptosis-related genes. In summary, our work demonstrates a synthetic lethal interaction between YY1 and KDM5C and suggests combination therapies for cancer treatments.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Key word: KDM5C; YY1; chromatin recruitment; promoter; cancer therapy\\n\\n\\nIntroduction\\n\\nEpigenetic regulators change phenotype acquisition or gene expression without altering the DNA sequence. By conferring a fitness advantage, mutational or non-mutational epigenetic reprogramming has been implicated in the evolution of tumors during malignant progression and metastasis. In addition, numerous studies have shown that aberrant epigenetic alterations contribute to cancer therapeutic resistance(Hogg et al, 2020; Miranda Furtado et al, 2019; Zafar et al, 2021). Therefore, targeting epigenetic regulators is an attractive strategy for cancer therapy.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='The histone demethylase KDM5C is a member of the SMC homolog family. By specifically removing one or two methyl groups from Lys4 in histone H3, the dynamic balance in histone H3K4 methylation is maintained. KDM5C carries a composite catalytic region in which JmjN and JmjC domains are located. JmjC is critical for catalyzing the demethylation of H3K4me2/3, while the JmjN domain interacts with the JmjC domain for the complete catalytic function of KDM5C. H514 is a highly conserved amino acid that is located in the catalytic core of the JmjC domain, and its mutation eliminates KDM5C demethylase activity. KDM5C can suppress transcription by reducing the level of H3K4me2/3 via its demethylase activity; however, whether modulation of H3K4 methylation status is'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='however, whether modulation of H3K4 methylation status is the only mechanism or the predominant mechanism by which KDM5C regulates gene expression is unclear.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='KDM5C is frequently mutated or aberrantly expressed in various cancer types. Several studies have indicated that KDM5C is involved in regulating different biological processes in tumor cells through its histone demethylase activity and functions as a tumor suppressor. For example, the high-risk HPV E6 protein degrades KDM5C in cervical cancer cells, which results in super enhancers activation of the proto-oncogenes c-MET and EGFR by changing the H3K4me3/H3K4me1 ratio(Chen et al, 2018). In clear cell renal cell carcinoma cells, KDM5C inhibits the expression of genes related to glycogen metabolism by reducing H3K4me3 levels at the promoters of these genes, while inactivating KDM5C mutation confers cell resistance to ferroptosis(Zheng et al, 2021). However,'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='cell resistance to ferroptosis(Zheng et al, 2021). However, the therapeutic strategy for KDM5C-deficient tumors is unknown. Intriguingly, KDM5C has also been shown to promote tumorigenesis, although the exact mechanism is unclear; in general, these observations suggest that KDM5C potentially regulates gene expression via a distinct mechanism.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='To better understand the mechanisms underlying how KDM5C regulates gene expression, we screened and thus identified Yin Yang 1 (YY1) as a KDM5C-interacting partner. Targeting YY1 in KDM5C-deficient tumor cells effectively suppressed cell proliferation and tumorigenesis. Chromatin Immunoprecipitation followed by next generation sequencing (ChIP-seq) analysis with multiple engineered cell lines revealed that KDM5C is essential for YY1 chromatin recruitment, which is partially dependent on an intact JmjC domain in KDM5C. Simultaneous loss of KDM5C and YY1 led to significantly enhanced transcriptional inhibition of a number of essential cell fate genes, elevated the apoptosis rate and led to cell cycle arrest. Our study suggests a novel scaffolding function of'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='arrest. Our study suggests a novel scaffolding function of epigenetic regulators and provides a rationale for the development of new combination treatments.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Results\\n\\nYY1 is identified as a partner of KDM5C'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Previous studies have shown that KDM5C plays diverse roles in cancer progression. Indeed, a systemic The Cancer Genome Atlas (TCGA) clinical data-based analysis clearly revealed that high KDM5C expression was positively or negatively correlated with cancer patient survival, depending on the cancer type (Figure 1A), reinforcing the idea that KDM5C may exhibit a distinct regulatory effect on gene expression, prompting us to screen the interacting partners of KDM5C. Tandem affinity purification (TAP) was subsequently performed. We generated a HEK293T cell line that stably expressed SBP- and Flag-tagged KDM5C, and the lysates of these cells were incubated with streptavidin-coated beads. The bead-bound proteins were subsequently eluted with buffer containing'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='proteins were subsequently eluted with buffer containing biotin. Next, the captured proteins were incubated with anti-Flag beads, and the proteins bound to these beads were eluted and analyzed by mass spectrometry (MS). The results of the MS analysis indicated that a number of proteins potentially interact with KDM5C, including YY1 (Figure 1B), the overexpression of which has been observed in the vast majority of cancers(Gao et al, 2019; van Leenders et al, 2007). Given the pivotal role played by YY1 in the transcriptional regulation of numerous genes(Meliala et al, 2020), we focused on the interaction between KDM5C and YY1. To confirm their interaction, mutual coimmunoprecipitation (co-IP) assays were performed in HEK293T cells, and the results clearly'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='assays were performed in HEK293T cells, and the results clearly showed that KDM5C bound to YY1 (Figure 1C). Co-IP with endogenous proteins was also performed and the results confirmed that KDM5C interacts with YY1 in mammalian cells (Figure 1D). In addition, we also performed the semi-endogenous co-IP with YY1 and KDM5C-Flag in 769-P renal cancer cell lines. As shown in Figure 1E, KDM5C clearly bound to YY1. To delineate the interaction, a series of KDM5C truncation mutants were constructed and used for further analysis (Figure 1F). We found that multiple regions, including the JmjN and JmjC domains, which are critical for H3K4me2/3 removal, were also involved in the interaction between KDM5C and YY1 (Figure 1G).'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='YY1 depletion significantly inhibits the proliferation and tumorigenicity of KDM5C-deficient tumor'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Many studies have shown that YY1 functions as an oncogene because its overexpression is observed in multiple cancers(Li et al, 2022; Qiao et al, 2019; Wang et al, 2022). YY1 tethers cofactors and promotes or represses transcription, depending on its interacting partner, which includes c-myc and p53(Shrivastava et al, 1993; Sui et al, 2004), prompting us to investigate the role of the YY1 KDM5C interaction in tumorigenesis. To achieve this goal, we first selected human kidney proximal tubular epithelial cell line (HK-2), and several renal cancer cell lines (ACHN, 769-P and RCC4) with different KDM5C and YY1 protein expression levels, as described in our previous studies(Chen et al., 2018; Zheng et al., 2021) (Figure 2A). Interestingly, knocking down YY1 in'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='et al., 2021) (Figure 2A). Interestingly, knocking down YY1 in KDM5C-deficient RCC4 tumor cells significantly reduced the colony formation ability of the cells, which was not observed in the KDM5C-proficient tumor cell lines ACHN and 769-P (Figure 2B and EV1B-D). Similar results were also obtained in a CCK8-based tumor cell proliferation assay (Figure 2C and EV1E-F). Notably, tumor growth was markedly inhibited when both of KDM5C and YY1 were depleted, whereas the loss of either KDM5C or YY1 only negligibly influenced ACHN cell growth (Figure 2D). These results strongly indicate that KDM5C might be important for YY1-related tumorigenesis, suggesting that YY1 may be a vulnerable target in KDM5C-deficient tumor cells.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='KDM5C colocalizes with YY1 and contributes to YY1 genome-wide chromatin recruitment'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='The stimulatory effect of YY1 on tumor cell growth was at least partially dependent on the presence of KDM5C. Given that YY1 always promotes tumorigenesis by increasing oncogene transcription, it is unlikely that YY1 tethers KDM5C because KDM5C has H3K4me2/3 demethylase activity, which is associated with repressed gene expression. A recent study revealed that KDM5B, which is a member of the KDM5 demethylase family, recruited a methyltransferase to suppress antitumor immunity, prompting us to explore whether KDM5C has a scaffolding function similar to that of KDM5B(Zhang et al, 2021). We generated a stable KDM5C-depleted ACHN cell lines (Figure EV1B), and ChIP-seq assays were subsequently performed. As shown in Figure 3A, the intensity of the YY1 binding'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='As shown in Figure 3A, the intensity of the YY1 binding peak was reduced upon loss of KDM5C, implying that KDM5C might be a general regulator of YY1 activity by mediating its chromatin recruitment. Next, we asked to what extent KDM5C co-localizes with YY1. To evaluate the genomic co-occupancy of KDM5C and YY1, we performed the cleavage under targets and tagmentation (CUT&Tag) assay in the ACHN renal cell line, and the annotation of the YY1 and KDM5C peaks revealed an obvious overlap (n=3645) between YY1 and KDM5C binding events (Figure 3B). Further analyzed revealed a significant overlap, with 7,957 genes being concurrently bound by both YY1 and KDM5C (Hypergeometric test, P=4.87e-17). This overlap accounts for 56.32% of the total genes bound by KDM5C and'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='accounts for 56.32% of the total genes bound by KDM5C and 66.71% of the total genes bound by YY1. Moreover, a heatmap and a metaplot analysis revealed that the enrichment of YY1 was reduced after KDM5C was depleted in the peaks (n=3645) where YY1 and KDM5C were directly related (Figure 3C). Reduced enrichment of YY1 was also observed in HK-2 cells with KDM5C depletion (Figure EV1A and EV2A). These results revealed that the intensity of the YY1 binding peaks was partially dependent on the KDM5C protein level in both nontumoral cell and tumor cells, suggesting that KDM5C mediated YY1 chromatin recruitment may represent a universal mechanism of action. The localization of the KDM5C-dependent YY1 binding peaks was subsequently analyzed, and the results revealed'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='peaks was subsequently analyzed, and the results revealed that most of the peaks were enriched in promoters (Figure 3D), implying that KDM5C may regulate gene expression via a distinct mechanism.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='An intact catalytic domain but not the demethylation activity of KDM5C is required for YY1 recruitment'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Since our results suggested that KDM5C functions as a scaffold and that KDM5C is essential for YY1 binding to a promoter, we asked, whether the demethylase activity of KDM5C is involved in YY1 recruitment. We generated ACHN cell lines that stably expressed KDM5C-WT or its enzymatically inactive mutant H514A and performed ChIP-seq to evaluate the effects of KDM5C or its mutants on YY1 recruitment (Figure EV1B). As shown in Figure 4A, ectopic KDM5C WT but not the KDM5C-H514A mutant enhanced YY1 chromatin binding. A similar trend was also observed in HK-2 cells (Figure EV2A). Because the primary function of KDM5C is to catalyze the removal of methyl groups from H3K4me2/3, we evaluated whether H3K4me3 abundance plays a role in YY1 recruitment.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='We first performed ChIP-seq of H3K4me3 in ACHN shKDM5C cells and analyzed the change in the signal intensity of this mark. Increased H3K4me3 enrichment was clearly observed in the KDM5C/YY1 overlapping region (Figure 4B), but the opposite trend of YY1 recruitment was observed in KDM5C-depleted cells (Figure 3C). To further understand the relationship between YY1 recruitment and H3K4me3, we focused on the overlapping regions of YY1 and H3K4me3 (n=11087), which were classified into KDM5C demethylase-dependent peaks (n=9943) and KDM5C demethylase-independent peaks (n=1144), as shown in Figure 4C. Reduced YY1 ChIP-seq signals were observed in almost all of the overlapping YY1 and H3K4me3 regions (Figure 4D-E), regardless of the effect of KDM5C on H3K4me3'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='(Figure 4D-E), regardless of the effect of KDM5C on H3K4me3 intensity. We therefore hypothesized that an intact KDM5C JmjC domain, rather than KDM5C histone demethylase activity, is essential for YY1 recruitment.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='To further explore the mechanism underlying the effects of KDM5C, we evaluated the interaction between KDM5C-H514A and YY1 and found that the mutation did not influence their interaction (Figure EV3), indicating that the effect of KDM5C-H514A on YY1 chromatin recruitment was unlikely to be caused by disrupted protein binding activity. Because the catalytic composite JmjN/C domain binds the substrate H3 tail harboring methylated H3K4 and since a previous study revealed that none of the KDM5C PHD domains exhibited strong binding affinity for unmethylated or methylated histones(Iwase et al, 2007), we speculated that the replacement of the histidine residue at position 514 with an alanine residue destroyed the integrity of the JmjN/C domain and influenced the'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='destroyed the integrity of the JmjN/C domain and influenced the chromatin-binding activity of KDM5C. To test this hypothesis, we expressed KDM5C-WT or KDM5C-H514A in HEK293T cells and isolated chromatin-bound proteins. Interestingly, although the protein levels in whole-cell lysates were indistinguishable, H154A exhibited obviously reduced chromatin-binding capacity (Figure 4F), which might help explain why KDM5C-H514A failed to recruit YY1 to chromatin. In addition, tumor cell proliferation was markedly inhibited only when both of KDM5C and YY1 were depleted in the KDM5C-proficient tumor cells ACHN and 769-P (Figure 4G-H and EV1B-C). The inhibitory effect was blunted when KDM5C-WT was re-expressed; in contrast, KDM5C-H514A only moderately restored the'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='in contrast, KDM5C-H514A only moderately restored the growth of ACHN and 769-P double knockdown cells. A similar trend was also observed in the colony formation assay (Figure 4J). In KDM5C-deficient RCC4, cells overexpression of KDM5C-WT largely restored cell sensitivity to YY1 depletion (Figure 4I and EV1D). These result collectively suggest that an intact catalytic domain of KDM5C is required for YY1 recruitment.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='KDM5C cooperates with YY1 to regulate essential cell fate gene expression'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='To systematically evaluate the effect of KDM5C-mediated YY1 recruitment on gene expression, we performed RNA-seq with shKDM5C, shYY1 or double-knockdown ACHN cells. As shown in Figure 5A-B, the loss of KDM5C or YY1 led to the upregulated or downregulated expression of several genes. In contrast, the simultaneous loss of KDM5C and YY1 resulted in altered transcription of thousands of genes (1511 genes whose expression was downregulated versus 797 genes whose expression was upregulated, |log2FC|≥1.5, p value<0.05, Figure 5C), indicating that YY1 recruitment by KDM5C likely contributed to gene expression activation. A similar pattern was also observed via RNA-seq with HK-2 nontumoral cells (Figure EV4). Compared with the loss of KDM5C or YY1 alone, double'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='EV4). Compared with the loss of KDM5C or YY1 alone, double knockdown caused transcriptional repression of a sharply increased number of genes (1099 newly identified downregulated genes in total), as shown in Figure 5D. Given that depletion of both YY1 and KDM5C led to significant suppression of tumor cell proliferation (Figure 4), we next investigated the gene expression patterns by analyzing the RNA-seq results and found that the expression of several cell cycle- and cell apoptosis-related genes, including JAM3, KNTC1, SMC2, SMC4, LMNB1 and FOXM1, was markedly downregulated in the double-knockdown cell group (Figure 5E). These results were verified by qPCR in ACHN cells (Figure 5F).'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='We compared the RNA-seq results with our CUT&Tag data, and found that almost half of the downregulated genes (49.96%) in the KDM5C/YY1 double-knockdown group were annotated in the genomic regions where YY1 and KDM5C overlapped, as shown in Figure EV5B. We then analyzed the YY1 peaks at the promoters of individual genes via the IGV browser. As shown in Figure EV5C-D, KDM5C depletion reduced theYY1 peak intensity at the promoters of the indicated genes, and the presence of KDM5C increased the amount of YY1 recruited to these promoters. KDM5C-H514A marginally increased the YY1 intensity in the indicated regions. Moreover, ChIP-qPCR was performed in HK-2 cells and the result revealed that YY1 enrichment at the promoter region of the indicated genes was'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='enrichment at the promoter region of the indicated genes was increased after KDM5C-WT was expressed, whereas KDM5C-H514A failed to increase YY1 binding (Figure EV2C). These results strongly suggest that KDM5C increases the expression of essential cell-fate genes by recruiting YY1 to their promoters.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Simultaneous loss of KDM5C and YY1 significantly promotes cell cycle arrest and cell apoptosis'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Previous studies have shown that SMC2/4, LMNB1, and KNTC1 are involved in cell cycle arrest and apoptosis regulation. To confirm the synergistic effect of KDM5C and YY1 on tumorigenicity and cancer cell proliferation, we analyzed the cell cycle progression and apoptosis rate of tumor cells in which KDM5C, YY1 or both had been depleted. We first measured the ratio of renal cancer cells in each phase of the cell cycle, and the results indicated that the loss of both KDM5C and YY1 caused an increased ratio of 769-P tumor cells arrested in the G2/M phase. Similar results were also observed in KDM5C-deficient RCC4 cells after YY1 depletion (Figure 6A). Moreover, double knockdown of KDM5C and YY1 but not loss of either gene individually greatly increased the'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='but not loss of either gene individually greatly increased the percentage of apoptotic cells in 769-P tumors. In addition, the absence of YY1 in RCC4 cells induced a significant increase in the percentage of apoptosis cells (Figure 6B). These results confirmed the inhibitory effect of YY1 depletion on KDM5C-deficient cells (Figure 4). Taken together, our study revealed a synergistic regulatory effect of KDM5C and YY1 on cancer cell proliferation and cell cycle progression.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Discussion'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='As a member of the GLI-Kruppel family of zinc finger DNA-binding proteins, YY1 has been proven to regulate genes either by directly binding to a promoter or indirectly by associating with chromatin-remodeling proteins or histone modifiers (Gordon et al, 2006; Khachigian, 2018). YY1 is frequently overexpressed in many cancer types and reshapes the oncogenic gene expression network(Bonavida & Kaufhold, 2015; Sarvagalla et al, 2019). Although YY1 can directly bind to a promoter DNA sequence, the mechanism underlying its recruitment to chromatin is still not fully understood. In addition, although targeting YY1 might be an attractive therapeutic strategy, a robust biomarker to analyze its effect is lacking. In this study, we found that YY1 is a KDM5C-associated'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='lacking. In this study, we found that YY1 is a KDM5C-associated protein, and the JmjC domain of KDM5C was essential for the KDM5C–YY1 interaction. Moreover, we demonstrated that KDM5C mediates the chromatin recruitment of YY1, which is important for YY1 binding to promoter regions and the transcription of cell fate-related genes. Importantly, targeting YY1 effectively inhibited the proliferation and tumorigenicity of tumor cells with low KDM5C expression. Considering these findings, we propose the following model: YY1 chromatin recruitment is at least partially dependent on the presence of KDM5C, and in tumor cells with high KDM5C expression, a small amount of YY1 can be recruited to chromatin even after KDM5C is knocked down. In contrast, depleting YY1 in'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='even after KDM5C is knocked down. In contrast, depleting YY1 in low KDM5C-expressing tumor cells completely eliminated chromatin-bound YY1 and promoted tumor regression (Figure 7).'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='As a specific H3K4 histone demethylase, KDM5C plays dual roles as both a proto-oncogene and tumor suppressor(Chen et al., 2018; Ji et al, 2015; Shen et al, 2016; Shen et al, 2021). KDM5C interacts with specific transcription factors, such as MYC(Outchkourov et al, 2013; Secombe et al, 2007), and downregulates gene expression by reducing H3K4me2/3 levels at enhancer or promoter regions. However, studies, including our previous reports, have shown that KDM5C also upregulates the expression of certain genes, but the relevant molecular mechanism of KDM5C effect on gene expression is unclear(Outchkourov et al., 2013; Scandaglia et al, 2017; Zheng et al., 2021). A recent study showed that the histone demethylase KDM5B recruited the H3K9 methyltransferase SETDB1'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='demethylase KDM5B recruited the H3K9 methyltransferase SETDB1 to suppress antitumor immunity in a KDM5B demethylase-independent manner, implying that the scaffolding function of this epigenetic regulator was important in certain contexts, a finding that has been appreciated to a lesser degree in prior studies(Zhang et al., 2021). Our finding that KDM5C recruited YY1 to chromatin in a histone demethylase-independent manner reinforces the evidence suggesting that the scaffolding function of epigenetic regulators might be critical for tumorigenesis. Indeed, epigenetic inhibitors targeting the catalytic domain often show limited efficacy and intolerable toxicity, and our findings strongly suggest that targeting the scaffolding function should be considered in'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='that targeting the scaffolding function should be considered in the development of new cancer prevention strategies and therapies.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Abnormalities in enzymes that regulate epigenetic modifications are important drivers of tumor occurrence and cancer progression(Dawson, 2017). Therefore, epigenome-based therapies might be developed to block the dysregulated expression of proto-oncogenes in tumor cells and increase antitumor responses(Miranda Furtado et al., 2019; Nebbioso et al, 2018). KDM5C has been shown to be frequently mutated or aberrantly expressed in various cancers. The degree of YY1 chromatin recruitment was decreased in KDM5C-deficient tumor cells, and further inhibition of YY1 expression might completely suppress YY1-directed transcription and significantly impair the proliferation of tumor cells. Furthermore, simultaneously targeting KDM5C and YY1 effectively inhibited tumor'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='targeting KDM5C and YY1 effectively inhibited tumor growth in KDM5C-proficient tumor cells. In recent years, the induction of synthetic lethality has been applied to the identification of novel drug targets, opening new avenues to tumor-targeted therapy(Nijman & Friend, 2013). Considering the definition of synthetic lethality and the results presented here, we propose that KDM5C and YY1 form a synthetically lethal gene pair, providing a rationale for the clinical treatment of patients with KDM5C-deficient cancer by targeting YY1.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='In summary, our work provides mechanistic insight into YY1 chromatin recruitment and suggests an important role for the scaffolding function of the histone demethylase KDM5C. Because proteolysis-targeting chimeras (PROTACs) are used to target and degrade proteins with important scaffolding functions and serve as a potential therapeutic strategy in cancer, future work focusing on the design and synthesis of KDM5C PROTAC drugs is important, and further studies in this research direction are warranted.\\n\\n\\n\\n\\nMethods\\n\\nVectors and plasmids'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Methods\\n\\nVectors and plasmids\\n\\nThe pHAGE vector carries a puromycin resistance cassette, endowing the transfected cells with resistance to puromycin, hence enabling us to select and generate stable cells. The overexpression vectors used in our context (KDM5C or YY1 with HA or Flag tags and their truncated constructs) were constructed using pHAGE for mammalian cell expression. The KDM5C enzymatically inactive mutant H514A was generated via site-directed mutagenesis PCR from the pHAGE-KDM5C plasmid. shRNAs targeting human KDM5C and YY1 were constructed using pLKO.1. The construction of pLKO.1 is detailed on the addgene website. All of the constructs were validated by DNA sequencing. The targeted sequences of KDM5C and YY1 genes are as follows:'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='shCtrl: 5’-CAACAAGATGAAGAGCACCAA-3’; \\n\\nshKDM5C-1: 5’-AGTACCTGCGGTATCGGTATA-3’; \\n\\nshKDM5C-2: 5’-GCCACACTTGAG GCCATAATC-3’; \\n\\nshYY1-1: 5’-GACGACGACTACATTGAACAA-3’; \\n\\nshYY1-2: 5’-GCCTCTCCTTTGTATATTATT-3’. \\n\\n\\n\\nCell culture'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='The human kidney proximal tubular epithelial cell line HK-2 and human renal cancer cell lines (ACHN, 769-P and RCC4) were obtained from the American Type Culture Collection (ATCC). HEK293T cells were obtained from the Institute of Cellular Resources, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences. HK-2, ACHN, RCC4, and HEK293T cells were cultured in high-glucose Dulbecco’s modified Eagle’s medium (DMEM, HyClone) while 769-P cells were cultured in Roswell Park Memorial Institute-RPMI medium(1640, HyClone) and both media were supplemented with 10% fetal bovine serum (FBS, Cell-box, AUS-01S-02) in a humidified atmosphere containing 5% CO2 at 37 ℃. The cell lines were authenticated by STR profiling, and mycoplasma testing was performed'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='by STR profiling, and mycoplasma testing was performed monthly with a MycoBlue Mycoplasma Detector (Vazyme, D101-01).'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='For transient protein expression, the cells were transfected with the indicated constructs via NEOFECT DNA transfection reagent (NeoFect, TF20121201) according to the manufacturer’s instructions. After 48 h of transfection, the cells were harvested for the following assays. To generate stable overexpression or knockout cell lines, appropriate constructs and two packaging plasmids psPAX2 and pMD2.G were cotransfected into HEK293T cells. After 60 h, the lentivirus particles from the culture medium were collected and filtered with 0.45 µm filter, and then added to the host cells along with 8 μg/mL polybrene (Solarbio, H8761). Infected cells were then selected in puromycin (Solarbio, P8230)-containing medium. The protein levels of the target genes were'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='medium. The protein levels of the target genes were determined via western blotting with the corresponding antibodies. To generate the double-knockdown cells, we selected a GFP-labeled KDM5C knockdown plasmid (pLKO.1-GFP) to infect cells depleted of YY1, and then screened the cells for puromycin resistance and green fluorescence. To restore KDM5C or H514A in knockdown cells, we inserted a hygromycin resistance gene into pHAGE-KDM5C/H514A-puro, which could be used to screen cells expressing KDM5C/H514A. The cells involved in the cell cycle and apoptosis assays did not contain fluorescence, which might have affected the results. We confirmed protein expression via Western blotting.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='RNA extraction and qRT-PCR\\n\\nTotal RNA was extracted using the RNA isolater Total RNA Extraction Reagent (Vazyme, R401-01) according to the manufacturer’s instructions. Reverse transcription reactions were performed by using ReverTra Ace qPCR RT Master Mix with gDNA Remover (TOYOBO, FSQ-301). RT-PCR was performed with ChamQ SYBR qPCR Master Mix (Vazyme, Q311-02) on a CFX ConnectTM Real-Time System (Bio-Rad). The housekeeping gene expression levels of β-actin or GAPDH were used for normalization. Each reaction was performed in triplicates. For quantification of gene expression, the 2-△△Ct method was used. The primer sequences of indicated genes were as follows:\\n\\nh-JAM3-RT \\n\\nForward : 5’-GAGACTCAGCCCTTTATCGC-3’;\\n\\nReverse : 5’- TGTTGCCATCTTGCCTACTG-3’;'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Reverse : 5’- TGTTGCCATCTTGCCTACTG-3’; \\n\\nh-KNTC1-RT\\n\\nForward : 5’-CTCACGAAGTTGCACAGG-3’;\\n\\nReverse : 5’-GCCTCCAGCTCTTGTTTG-3’;\\n\\nh-SMC2-RT\\n\\nForward : 5’-GTTTTGGATGGTCTGGAGTT-3’;\\n\\nReverse : 5’-ATTGGAGCAGGTTTGAAGAG-3’;\\n\\nh-SMC4-RT\\n\\nForward : 5’-TCGACCACCTAAGAAAAGTTG-3’;\\n\\nReverse : 5’-CCATGAAGTAAAGGGGAGTG-3’;\\n\\nh-LMNB1-RT\\n\\nForward : 5’-CTGTCTCCAAGCCCTTCTT-3’;\\n\\nReverse : 5’-ACTCGCCTCTGATTCTTCC-3’;\\n\\nh-FOXM1-RT\\n\\nForward : 5’-GCTGCCCCTTCCTGTTC-3’;\\n\\nReverse : 5’-TGGACTCTGCCACTTCCTT-3’;\\n\\nh-GAPDH-RT \\n\\nForward : 5’-ACATCAAGAAGGTGGTGAAG-3’;  \\n\\nReverse : 5’-CTGTTGCTGTAGCCAAATTC-3’;\\n\\n\\n\\nWestern blotting'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='The cell pellets were lysed with RIPA buffer (Beyotime, P0013B) containing a protease inhibitor mixture cocktail (MedChemExpress, HY-K0010). Then, the cells were centrifuged at 4 ℃ at 13,000 rpm for 10 min to remove insoluble cellular components. Total protein concentrations were measured with a BCA protein assay kit (Beyotime, P0012). Proteins (20–100 μg) were separated via 8-15% sodium dodecyl sulphate-polyacrylamide gel electrophoresis (SDS-PAGE) and transferred to nitrocellulose membranes (NC, PALL, 66485) in Tris-glycine buffer. The blots were blocked with 5% nonfat milk then incubated with diluted primary antibodies at 4 ℃ overnight. Then, appropriate horseradish peroxidase (HRP)-conjugated secondary antibodies were incubated with the blots for 1 h at'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='secondary antibodies were incubated with the blots for 1 h at room temperature. The signals were visualized with an enhanced chemiluminescence detection kit (ECL, Vazyme, E411-05) and exposured to X-ray film (Sigma, America). The primary antibodies used were purchased from commercial sources: anti-KDM5C (Cell Signaling Technology, D29B9; BETHYL, A301-034A; Proteintech, 14426-1-AP), anti-YY1 (Cell Signaling Technology, D5D9Z), anti-Flag (Proteintech, 20543-1-AP), anti-HA (Proteintech, 66006-2-Ig), anti-GAPDH (Proteintech, 60004-1-Ig), anti-β-actin (Proteintech, 66009-1-Ig), anti-H3 (Cell Signaling Technology, 2650) and anti-H3K4me3 (Cell Signaling Technology, 9751).'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Coimmunoprecipitation (co-IP)\\n\\nFor the immunoprecipitation assay, appropriately constructed plasmids were transfected into the HEK293T and 769-P cell lines. After 48 h of transfection, the cells were harvested and then lysed in IP lysis buffer (Beyotime, P0013) with EDTA-free protease inhibitor cocktail (MedChemExpress, HY-K0010) for 30 min on ice, sonicated, and then insoluble proteins were removed by centrifugation at 4 ℃ at 12000 rpm for 10 min. An aliquot of the cleared cell lysate was collected as an input control. The remaining lysates were subsequently incubated with anti-HA or anti-Flag magnetic beads (MedChemExpress, HY-K0201, HY-K0207) overnight at 4 ℃ with gentle shaking.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='For endogenous protein interactions, the cell lysates were processed as described above. One tenth of each lysates was used as the input, half of the remainder was incubated with KDM5C antibody, and the other half was incubated with homologous IgG for 6 hours at 4 ℃. The lysates were then mixed with ProteinA/G agarose and incubated overnight at 4 ℃.\\n\\nMagnetic beads or agarose beads with a complex of antigen and antibody were washed four times in TBS-T buffer (100 nM Tris-HCl, 0.9% NaCl, 0.1% Tween-20). The proteins pulled down by the specified antibodies were eluted with 2×SDS loading buffer through boiling the beads, and then verified by Western blotting.\\n\\n\\n\\nTandem affinity purofication and mass spectrometry (TAP-MS)'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Tandem affinity purofication and mass spectrometry (TAP-MS)\\n\\nHEK293T cells overexpressing SBP-Flag-KDM5C were harvested for Tap-MS. Cell pellets were lysed with IP lysis buffer (Beyotime, P0013), and the supernatant was subsequently incubated with strep beads overnight at 4 ℃. After three washes, SBP-linked protein were elutied with biotin solution, and then incubated with anti-Flag magnetic beads (MedChemExpress, HY-K0207). After enrichment of the target proteins, we washed the beads to remove nonspecific proteins and then performed SDS-PAGE on the products of the two groups. All of the lanes of the gel were collected for mass spectrometry analysis.\\n\\n\\n\\nProliferation ability assay（CCK8）'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Proliferation ability assay（CCK8）\\n\\nThe cells were seeded in a 96-well plate at 30-50% confluence according to the cell growth rate. When the cells just attached at the initial point, the OD450 nm was read every 24 h via a cell counting kit 8 (CCK8, MedChemExpress, HY-K0301) according to the manufacturer’s instruction. The CCK8 solution was diluted with complete medium, incubated with the cells for 1-2 h, and then read with a microplate reader (BioTek Epoch, America) at 450 nm. The cell viability assay was performed in triplicate and repeated three times. Cell growth curves were drawn and analyzed via GraphPad Prism software V9.4.1.\\n\\n\\n\\nColony formation assay'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Colony formation assay\\n\\nThe cells were seeded into a 6-well plate at the same cell density. Each well contained 1000 cells, which were cultivated in a cell incubator. Fresh cell culture medium was added every three days. After four weeks (ACHN and 769-P) or eight weeks (RCC4), the cell culture medium was removed. The cells were gently washed with PBS, fixed with 4% paraformaldehyde for 20 min at 37 ℃, and stained with 0.1% crystal violet in PBS. The assay was performed in triplicate and repeated two or three times. \\n\\n\\n\\nCell cycle assay'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Cell cycle assay\\n\\nThe cells were gently collected at the exponential growth phase, washed with cold PBS, and then resuspended in 70% ethanol for more than 30 min at 4 ℃. Then, the cells were washed with cold PBS, incubated with propidium iodide(PI) solution (Byotime, C1052) and protected from light at 37 ℃ for 30 min. Red fluorescence was detected via flow cytometry (LSRFortessaX20, BD) at Ex=488 nm, and then analyzed with Flowjo. The assay was repeated two or three times.\\n\\n\\n\\nCell apoptosis assay'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Cell apoptosis assay\\n\\nIn accordance with the instructions of the Annexin V-FITC/PI apoptosis kit (Multi Sciences, AP101), we collected 1-10×10^5 cells and resuspended them in 500μl of 1×Binding buffer, and add 5 μl of Annexin V-FITC and 10 μl of PI to each sample. The samples were mixed gently and protected from light for 5 min, then Annexin V-FITC (Ex = 488 nm; Em = 530 nm) and PI (Ex = 535 nm; Em = 615 nm) were detected via flow cytometry (LSRFortessaX20, BD) with. The results were analyzed via Flowjo.\\n\\n\\n\\nIsolation of chromatin-bound proteins from subcellular fractions'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='After the transfection of HEK293T cells with empty vector, Flag-KDM5C or Flag-H514A, we collected the cells and lysed them with E1 buffer (50 nM HEPES-KOH, 140 nM NaCl, 1mM EDTA, 10% glycerol, 0.5% NP-40, 0.25% Triton X-100, 1mM DTT, Protease inhibitor cocktail). The cells were subsequently centrifuged 1,100× g at 4 ℃ for 2 min after which the supernatant was obtained as the cytoplasmic fraction. Next, we resuspended the pellet in E2 buffer (10nM Tris-HCl, 200mM NaCl, 1mM EDTA, 0.5mM EGTA, Protease inhibitor cocktail). After centrifugation, the supernatant was obtained as the nuclear fraction. We resuspended the pellet in E3 buffer (500 mM Tris-HCl, 500mM NaCl, protease inhibitor cocktail) and sonicated it on ice for 5 min, 2 s ON/45 s OFF. The supernatant'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='sonicated it on ice for 5 min, 2 s ON/45 s OFF. The supernatant was centrifuged at 4 ℃ for 10 min at 16,000× g, and transferred to fresh microcentrifuge tubes. The protein concentrations were analyzed via Western blotting.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Chromatin immunoprecipitation followed by next-generation sequencing (ChIP-seq)\\n\\nThe cells were plated in 10 cm dishes until they reached confluence and were then fixed with 37% fresh formaldehyde to a final concentration of 1% for 13 min at room temperature with rotation, after which the reaction was quenched with 0.125 mM glycine for 10 min. The final number of Chip-seq cells in each group was 4×107.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='The cells were washed three times with ice-cold PBS and then harvested bwith a cell scraper and lysed in lysis buffer 1 (50mM HEPES-KOH, 140mM NaCl, 1mM EDTA, 10% glycerin, 0.5% NP-40, 0.25% TritonX-100), lysis buffer 2 (pH 8.0 10mM Tris-HCl, 200 mM NaCl 1mM EDTA, 0.5mM EGTA), and lysis buffer 3 (pH 8.0 10mM Tris-HCl, 100mM NaCl, 1mM EDTA, 0.5mM EGTA, 0.2% N-lauroyl sarcosine sodium salt). The chromatin was sonicated in lysis buffer 3 to ensure that the size of the chromatin fraction was between 200-500bp. 80 μL of each sample was transferred to a clean tube as the input DNA. The chromatin sample was diluted with ChIP dilution buffer (20mM Tris-HCl, 150mM NaCl, 1mM EDTA, 1% TritonX-100). 6 μL of antibody was added to each tube and the mixturewas incubated'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='of antibody was added to each tube and the mixturewas incubated at 4 ℃ overnight. To each tube, 80μL of protein G Sepharose 4 Fast Flow (GE Healthcare, 17061805) was added, and the immune complexes were collected by incubation at 4 ℃ for at least 6 h with gentle rotation. The bound complexes were washed once with each of the following buffers: low-salt wash buffer (0.1% SDS, 1% TritonX-100, 2 mM EDTA, 20 mM Tris-HCl pH 8.0, 150 mM NaCl), high-salt wash buffer (0.1% SDS, 1% Triton X-100, 2 mM EDTA, 20 mM Tris-HCl pH 8.0, 500 mM NaCl), LiCl wash buffer (0.25 M LiCl, 1% NP-40, 1mM EDTA, 10 mM Tris-HCl pH 8.0) and TE buffer (10 mM Tris-HCl pH 8.0, 1mM EDTA). The beads were incubated at 65 ℃ with TE buffer containing proteinase K, RNase and SDS overnight, and'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='TE buffer containing proteinase K, RNase and SDS overnight, and the DNA was extracted using phenol/chloroform. ChIP-seq was performed by Beijing Novogene Biotech Co., Ltd.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='ChIP-qPCR\\n\\nThe qPCR templates were obtained from ChIP. For method details, please refer to “qRT-PCR”. We calculated the fold enrichment of those genes via YY1-ChIP, and IgG was used as a negative control. The calculation formula is as followed:\\n\\nΔCt [normalized ChIP] = (Ct [ChIP] - (Ct [Input] -Log2 (Input Dilution Factor))\\n\\nΔΔCt [ChIP/NIS] = ΔCt [normalized ChIP] - ΔCt [ IgG ]\\n\\nFold Enrichment = 2^ (-ΔΔCt [ChIP/NIS])\\n\\nPrimer sequences used in ChIP-qPCR are as followed:\\n\\nJAM3\\n\\nForward : 5’-TGGTAACTGGGGCGGGTCGCA-3’;\\n\\nReverse : 5’-AGCTTCGGCCCGCTCTGCTTC-3’;\\n\\nKNTC1\\n\\nForward : 5’-CAACACCGCCTTACTTCC-3’;\\n\\nReverse : 5’-CCGGGACTCTGAACTATGGC-3’;\\n\\nSMC2\\n\\nForward : 5’-GGAGCTACCATTATCGAGAC-3’;\\n\\nReverse : 5’-CTTGACGCACCCCAAAAG-3’;\\n\\nSMC4'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Reverse : 5’-CTTGACGCACCCCAAAAG-3’;\\n\\nSMC4\\n\\nForward : 5’-AAAAGAATCCCTCGCTCTTC-3’;\\n\\nReverse : 5’-TGACAGCAAGAGAAATCTCC-3’;\\n\\nLMNB1\\n\\nForward : 5’-TTGCCCAAGGGCCAGATTTTA-3’;\\n\\nReverse : 5’-AATGCAAGACGAGGGTGACT-3’;\\n\\nFOXM1\\n\\nForward : 5’-ATTTTCCCACAGTGAACGAT\\n\\nReverse : 5’-AACCTTGTCTGCCATTGTATC\\n\\n\\n\\nCleavage Under Targets and Tagmentation assay'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Cleavage Under Targets and Tagmentation assay\\n\\nThe Cleavage Under Targets and Tagmentation (CUT&Tag) assay was conducted according to the manual instructions of Hyperactive Universal CUT&Tag Assay Kit for Illumina Pro (Vazyme, TD904). A ChIP-grade anti-KDM5C antibody (Bethyl, A301-034A) and anti-YY1 (CST, D5D9Z) were used for immunoprecipitation. Rabbit IgG (ABclonal, AC042) was used as a negative control. The library was constructed with the TruePrep Index Kit V2 for Illumina (Vazyme, TD202). \\n\\n\\n\\nChIP-seq and CUT&Tag data analysis'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='The raw data (raw reads) in fastq format were first processed via fastp software to obtain clean data, from which reads containing adapters were removed. The reference genome hg38 and gene model annotation files were downloaded directly from the genome website. The index of the reference genome was built using Bowtie2, and the clean reads were aligned to the reference genome with Bowtie2. After the reads were mapped to the reference genome, MACS2 peak-calling software was used to identify regions enriched with IP over the background. A q-value threshold of 0.05 was used for all of the datasets. Only peaks with counts greater than 200 were retained in the CUT&Tag data. Bedtools reldist and bedtools intersect were used to compare two Chip-seq or CUT&Tag'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='bedtools intersect were used to compare two Chip-seq or CUT&Tag datasets in the global or promoter region. Motif analysis was performed via Homer. Chipseeker was used to analyze genomic distribution.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='RNA-seq and data analysis\\n\\nTotal RNA was extracted from the corresponding cells. The concentration was measured by Qubit 3.0 fluorometer (Life Technologies) and RNA purity was checked via the kaiaoK5500®Spectrophotometer (Kaiao, Beijing, China). Sample integrity was assessed using an Agilent 2100 RNA Nano 6000 assay kit (Agilent Technologies). RNA-seq was performed by Annoroad Gene Technology Corporation (Beijing).'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Briefly, sequencing libraries were constructed using the NEBNext Ultra RNA Library Prep Kit for Illumina (NEB, USA). Clustering of the index-coded samples was performed on a cBot cluster generation system using the HiSeq PE Cluster Kit v4-cBot-HS (Illumina) according to the manufacturer’s instructions. After cluster generation, the libraries were sequenced on an Illumina platform and 150 bp paired-end reads were generated.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='For the data analysis, an index of the reference genome was built using HISAT2 v2.0.5 and paired-end clean reads were aligned to the reference genome via HISAT2 v2.0.5. FeatureCounts v1.5.0-p3 was used to count the numbers of reads mapped to each gene. Differential expression analysis of the two groups was performed using the DESeq2 R package. Heatmaps were generated via pheatmap (v1.0.12) package in R.\\n\\n\\n\\nXenograft assay'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Xenograft assay\\n\\nAll animal experiments were approved by the Animal Research Ethics Board of Wuhan University (WAEF-2022-0113) and confirmed by the animal experimental guidelines. Four-week-old female BALB/c nude mice were purchased from Beijing Vital River Laboratory Animal Technology (China) and were housed under specific pathogen-free conditions in Animal Experiment Center-Animal Biosafety Level-III Laboratory of Wuhan University. The animals were randomly divided into four groups of 6 mice each. The mice were injected with 100 μL of cells suspended in Matrigel Basement Membrane Matrix (Corning, 356234) at a population of 3 × 106 ACHN indicated cells into the left or right dorsal flank subcutaneously.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='The tumor size was estimated once a week via a digital Vernier caliper. The tumor volume was calculated with the following formula: tumor volume (mm3) =length (mm)×width2 (mm)× 0.5. Tumors were havested 8 weeks after injection.\\n\\n\\n\\nStatistical analysis\\n\\nThe quantitative experiment was repeated three times or more independently. The graphs were generated with GraphPad Prism software. Two-tailed Student’s t-test, one-way ANOVA and the Wilcoxon rank-sum test were used to assess the significance of the experiments. P values < 0.05 were considered to indicate the significant differences. \\n\\n\\n\\n\\n\\n\\nData Availability'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Data Availability\\n\\nThe sequencing data are available in GEO Data Sets with the accession number GSE270718 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE270718), GSE270719 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE270719) and GSE275838 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE275838). All other raw data support the findings of this study are available from the corresponding author upon reasonable request.\\n\\n\\n\\n\\nAcknowledgments'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Acknowledgments\\n\\nThis work was supported by Natural Science Foundation of China (grant numbers 82203491 and 82173137), Natural Science Foundation of Hubei Province of China (grant number 2022CFA008), China Postdoctoral Science Foundation (grant number 2022TQ0237), Natural Science Foundation of Inner Mongolia (grant number 2024QN08005) and Translational Medicine and Interdisciplinary Research Joint Fund of Zhongnan Hospital of Wuhan University (grant number ZNJC202212). \\n\\n\\n\\n\\n\\n\\n\\n\\nDisclosure Statement & Competing Interests \\n\\nThese authors declare that they have no conflict of interest.\\n\\n\\n\\n\\nReferences'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='References\\n\\nBonavida B, Kaufhold S (2015) Prognostic significance of YY1 protein expression and mRNA levels by bioinformatics analysis in human cancers: a therapeutic target. Pharmacol Ther 150: 149-168\\n\\nChen X, Loo JX, Shi X, Xiong W, Guo Y, Ke H, Yang M, Jiang Y, Xia S, Zhao M et al (2018) E6 Protein Expressed by High-Risk HPV Activates Super-Enhancers of the EGFR and c-MET Oncogenes by Destabilizing the Histone Demethylase KDM5C. Cancer Res 78: 1418-1430\\n\\nDawson MA (2017) The cancer epigenome: Concepts, challenges, and therapeutic opportunities. Science 355: 1147-1152'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Gao P, Li L, Yang L, Gui D, Zhang J, Han J, Wang J, Wang N, Lu J, Chen S et al (2019) Yin Yang 1 protein ameliorates diabetic nephropathy pathology through transcriptional repression of TGFbeta1. Sci Transl Med 11\\n\\nGordon S, Akopyan G, Garban H, Bonavida B (2006) Transcription factor YY1: structure, function, and therapeutic implications in cancer biology. Oncogene 25: 1125-1142\\n\\nHanahan D (2022) Hallmarks of Cancer: New Dimensions. Cancer Discov 12: 31-46\\n\\nHogg SJ, Beavis PA, Dawson MA, Johnstone RW (2020) Targeting the epigenetic regulation of antitumour immunity. Nat Rev Drug Discov 19: 776-800'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Iwase S, Lan F, Bayliss P, de la Torre-Ubieta L, Huarte M, Qi HH, Whetstine JR, Bonni A, Roberts TM, Shi Y (2007) The X-linked mental retardation gene SMCX/JARID1C defines a family of histone H3 lysine 4 demethylases. Cell 128: 1077-1088\\n\\nJi X, Jin S, Qu X, Li K, Wang H, He H, Guo F, Dong L (2015) Lysine-specific demethylase 5C promotes hepatocellular carcinoma cell invasion through inhibition BMP7 expression. BMC Cancer 15: 801\\n\\nKhachigian LM (2018) The Yin and Yang of YY1 in tumor growth and suppression. Int J Cancer 143: 460-465\\n\\nLi Y, Li J, Li Z, Wei M, Zhao H, Miyagishi M, Wu S, Kasim V (2022) Homeostasis Imbalance of YY2 and YY1 Promotes Tumor Growth by Manipulating Ferroptosis. Adv Sci (Weinh) 9: e2104836'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content=\"Meliala ITS, Hosea R, Kasim V, Wu S (2020) The biological implications of Yin Yang 1 in the hallmarks of cancer. Theranostics 10: 4183-4200\\n\\nMiranda Furtado CL, Dos Santos Luciano MC, Silva Santos RD, Furtado GP, Moraes MO, Pessoa C (2019) Epidrugs: targeting epigenetic marks in cancer treatment. Epigenetics 14: 1164-1176\\n\\nNebbioso A, Tambaro FP, Dell'Aversana C, Altucci L (2018) Cancer epigenetics: Moving forward. PLoS Genet 14: e1007362\\n\\nNijman SM, Friend SH (2013) Cancer. Potential of the synthetic lethality principle. Science 342: 809-811\"), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Outchkourov N, MuinO J, Kaufmann K, Van Ijcken WJ, Koerkamp MJ, Van LD, P. GD, Holstege FP, Grosveld F, Timmers HT (2013) Balancing of Histone H3K4 Methylation States by the Kdm5c/SMCX Histone Demethylase Modulates Promoter and Enhancer Function. Cell Reports 3: 1071-1079\\n\\nQiao K, Ning S, Wan L, Wu H, Wang Q, Zhang X, Xu S, Pang D (2019) LINC00673 is activated by YY1 and promotes the proliferation of breast cancer cells via the miR-515-5p/MARK4/Hippo signaling pathway. J Exp Clin Cancer Res 38: 418\\n\\nSarvagalla S, Kolapalli SP, Vallabhapurapu S (2019) The Two Sides of YY1 in Cancer: A Friend and a Foe. Front Oncol 9: 1230'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Scandaglia M, Lopez-Atalaya JP, Medrano-Fernandez A, Lopez-Cascales MT, Del Blanco B, Lipinski M, Benito E, Olivares R, Iwase S, Shi Y et al (2017) Loss of Kdm5c Causes Spurious Transcription and Prevents the Fine-Tuning of Activity-Regulated Enhancers in Neurons. Cell Rep 21: 47-59\\n\\nSecombe J, Li L, Carlos L, Eisenman RN (2007) The Trithorax group protein Lid is a trimethyl histone H3K4 demethylase required for dMyc-induced cell growth. Genes Dev 21: 537-551\\n\\nShen H, Xu W, Guo R, Rong B, Gu L, Wang Z, He C, Zheng L, Hu X, Hu Z et al (2016) Suppression of Enhancer Overactivation by a RACK7-Histone Demethylase Complex. Cell 165: 331-342'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Shen HF, Zhang WJ, Huang Y, He YH, Hu GS, Wang L, Peng BL, Yi J, Li TT, Rong R et al (2021) The Dual Function of KDM5C in Both Gene Transcriptional Activation and Repression Promotes Breast Cancer Cell Growth and Tumorigenesis. Adv Sci (Weinh) 8: 2004635\\n\\nShrivastava A, Saleque S, Kalpana GV, Artandi S, Goff SP, Calame K (1993) Inhibition of transcriptional regulator Yin-Yang-1 by association with c-Myc. Science 262: 1889-1892\\n\\nSui G, Affar el B, Shi Y, Brignone C, Wall NR, Yin P, Donohoe M, Luke MP, Calvo D, Grossman SR et al (2004) Yin Yang 1 is a negative regulator of p53. Cell 117: 859-872'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='van Leenders GJ, Dukers D, Hessels D, van den Kieboom SW, Hulsbergen CA, Witjes JA, Otte AP, Meijer CJ, Raaphorst FM (2007) Polycomb-group oncogenes EZH2, BMI1, and RING1 are overexpressed in prostate cancer with adverse pathologic and clinical features. Eur Urol 52: 455-463\\n\\nWang W, Qiao S, Li G, Cheng J, Yang C, Zhong C, Stovall DB, Shi J, Teng C, Li D et al (2022) A histidine cluster determines YY1-compartmentalized coactivators and chromatin elements in phase-separated enhancer clusters. Nucleic Acids Res 50: 4917-4937\\n\\nZafar A, Wang W, Liu G, Wang X, Xian W, McKeon F, Foster J, Zhou J, Zhang R (2021) Molecular targeting therapies for neuroblastoma: Progress and challenges. Med Res Rev 41: 961-1021'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Zhang SM, Cai WL, Liu X, Thakral D, Luo J, Chan LH, McGeary MK, Song E, Blenman KRM, Micevic G et al (2021) KDM5B promotes immune evasion by recruiting SETDB1 to silence retroelements. Nature 598: 682-687\\n\\nZheng Q, Li P, Zhou X, Qiang Y, Fan J, Lin Y, Chen Y, Guo J, Wang F, Xue H et al (2021) Deficiency of the X-inactivation escaping gene KDM5C in clear cell renal cell carcinoma promotes tumorigenicity by reprogramming glycogen metabolism and inhibiting ferroptosis. Theranostics 11: 8674-8691\\n\\n\\n\\n\\n\\n\\nFigure Legends\\n\\nFigure 1 YY1 was identified as a KDM5C binding protein'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Figure 1 YY1 was identified as a KDM5C binding protein\\n\\nA. Forest plot of the hazard ratio (HR) showing the association between high expression levels of KDM5C and overall survival in multiple cancers. For each type of cancer, the HR and 95% confidence interval (CI) are plotted as red dots and horizontal lines on the basis of a Cox proportional hazards model; when the HR=1, the study object had no effect on overall survival; when the HR>1, the study object was determined to be a risk factor; and when the HR<1, the study object was identified as a protective factor.\\n\\nB. Strategy for identifying KDM5C-binding partners in HEK293T cells via TAP-MS.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='C-D. Co-IP followed by Western blotting was used to detect the interaction between KDM5C and YY1 exogenously (C) and endogenously (D) in HEK 293 cells.\\n\\nE. Semi-endogenous co-IP with KDM5C-Flag and YY1 in the renal cancer cell line 769-P.\\n\\nF. Schematic representation of KDM5C domains and corresponding constructs.\\n\\nG. Semi-endogenous co-IP with KDM5C truncation mutants and endogenous YY1 in HEK293T cells.\\n\\n\\n\\nFigure 2 YY1 is a vulnerable target in KDM5C-deficient tumor cells\\n\\nA. Western blotting showing KDM5C expression in HK-2 human kidney proximal tubular epithelial cells and renal cancer cell lines (ACHN, 769-P and RCC4).'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='B. Effect of YY1 knockdown on the survival of the indicated tumor cell lines as measured by a colony formation assay (n=3 biological replicates).\\n\\nC. Proliferation of the indicated cell lines after YY1 was depleted, as measured by a CCK8 assay. The data are shown as the mean ± SD of three independent experiments after analysis by one-way ANOVA. In ACHN cells, P=0.116 (shYY1-1) and P=0.1222 (shYY1-2); in 769-P cells, P=0.7353 (shYY1-1) and P=0.9113 (shYY1-2); and in RCC4 cells, P=0.0007 (shYY1-1) and P=2E-05 (shYY1-2).\\n\\nD. Xenograft tumors excised from nude mice (n = 6/group) that had been subcutaneously injected with the indicated tumor cells. Statistical significance was determined by one-way ANOVA. P=1E-05'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Figure 3 KDM5C is essential for YY1 genome-wide chromatin recruitment\\n\\nA. Violin plot showing the alteration in YY1 binding peaks after KDM5C depletion. YY1 binding peaks in both ACHN parental and shKDM5C cells were counted (n=17,030). The data are presented as mean ± SD. Statistical significance was determined by the Mann-Whitney U test, P= 2.22E-16.\\n\\nB. Venn diagram showing the overlapping peaks of YY1 and KDM5C in the co-occupied regions. Representative data from two independent CUT-Tag experiments.\\n\\nC. Heatmap and metaplot showing the chromatin enrichment of YY1 at overlapping YY1 and KDM5C peak regions (n=3645) in ACHN parental and shKDM5C cells.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='D. Pie chart showing the annotated genomic distribution of altered YY1 peaks on different DNA elements in shKDM5C or KDM5C overexpressing cells.\\n\\n\\n\\nFigure 4 An intact JmjC domain but not KDM5C H3K4 demethylation activity is required for KDM5C-mediated YY1 recruitment\\n\\nA. Heatmap and metaplot showing the distribution of YY1 peaks in the overlapping regions of YY1 and KDM5C (n=3645) in ACHN cells expressing the KDM5C-WT or KDM5C-H514A mutant.\\n\\nB. Loss of KDMC5 resulted in increased H3K4me3 signal intensity in the YY1 and KDM5C overlapping region (n=3645) in ACHN parental and shKDM5C cells.\\n\\nC. Venn diagram showing the number of YY1 peaks in the H3K4me3/YY1 overlapping region (n=11087) with or without modulation by KDM5C.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='D-E. Heatmaps and metaplots revealed reduced YY1 enrichment at KDM5C demethylase dependent regions (n=9943) or KDM5C demethylase independent regions (n=1144) in ACHN parental and shKDM5C cells.\\n\\nF. KDM5C protein levels in whole-cell lysates or chromatin fractions from HEK293T cells transfected with KDM5C-WT or KDM5C-H514A mutant construct.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='G-I. Effect of the re-expression of the KDM5C-WT or KDM5C-H514A mutant on the proliferation of ACHN (G), 769-P (H) or RCC4 (I) cells after the depletion KDM5C, YY1 or both, as determined by the CCK8 assay. Each experiment was repeated three times with similar results. “Dsh” indicates cells in which KDM5C and YY1 were both knocked down. The data are shown as the mean ± SD of three independent experiments after analysis via Student’s t test. In ACHN cells, P=5.37E-08 (Dsh) and P=3.48E-07 (Dsh+KDM5C); in 769-P cells, P=4.45E-10 (Dsh) and P=7.49E-10 (Dsh+KDM5C); in ACHN cells, P=5.86E-09 (Dsh) and P=2.72E-07 (Dsh+KDM5C).\\n\\nJ. Colony formation assay with the indicated cell lines(n=3). “Dsh” indicates cells with KDM5C and YY1 simultaneously knocked down.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Figure 5 KDM5C and YY1 coregulate the expression of a number of essential cell-fate genes\\n\\nA-C. Volcano plots showing the change in the number of transcripts in shKDM5C, shYY1 or double shKDM5C ACHN cells, as determined by RNA-seq analysis (|log2 FC|>1.5, p value <0.05). “Double sh” indicates cells with KDM5C and YY1 simultaneously knocked down.\\n\\nD. Venn diagrams showing the number of downregulated genes in shKDM5C, shYY1 or shKDM5C/YY1 double-knockdown cells.\\n\\nE. Heatmap showing the normalized gene expression profile in shKDM5C, shYY1 or double sh ACHN cells. “Double sh” indicates cells with KDM5C and YY1 simultaneously knocked down.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='F. qPCR was used to assess the transcription of select genes in the indicated ACHN cells. Each experiment was repeated three times with similar results. “Double sh” indicates cells with KDM5C and YY1 double knocked down. The data are shown as the mean ± SD of three independent experiments after analysis by two tailed Student’s T-test. P=2.86E-05 (JAM3), P=4.06E-06 (KNTC1), P=1.31E-04(SMC2), P=9.18E-06(SMC4), P=1.91E-05(LMNB1) and P=1.47E-04 (FOXM1).\\n\\n\\n\\nFigure 6 Simultaneous targeting of KDM5C and YY1 effectively induces cell cycle arrest in the G2/M phase and apoptosis.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='A. Cell cycle analysis with the indicated cell lines via flow cytometry. The percentage of cells in each phase of the cell cycle is shown on the Y-axis. Each experiment was repeated three times with similar results. “Double sh” indicates cells with simultaneous knockdown of KDM5C and YY1.\\n\\nB. Flow cytometry–based Annexin-V/PI staining showing cells in the indicated lineages undergoing apoptosis. The quantitative results indicate the ratio of apoptotic cells. Each experiment was repeated three times with similar results. “Double sh” indicates cells with simultaneous knockdown of KDM5C and YY1. The data are shown as the mean ± SD of three independent experiments after analysis by two tailed Student’s T-test. P=5.3E-05 (RCC4) and P=1.1E-06 (769-P).'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Figure 7 A mechanistic hypothesis model for KDM5C-mediated YY1 recruitment. \\n\\nYY1 and KDM5C constitute a pair of synthetic lethal genes. KDM5C recruits YY1 to the promoter regions of cell cycle- or apoptosis-related genes. Targeting YY1 increases the vulnerability of KDM5C-deficient cancer cells.\\n\\n\\n\\n\\nExpanded View Figure Legends\\n\\nFigure EV1 Validation of the constructed cell lines used in this study\\n\\nA-D. KDM5C and YY1 protein levels in the indicated cells were measured via Western blotting. “Dsh” indicates cells in which KDM5C and YY1 were both knocked down.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='E-F. Proliferation rate of the indicated cell lines upon KDM5C depletion, as determined via a CCK8 assay. The data are shown as the mean ± SD of three independent experiments after analysis via one-way ANOVA. In 769-P cells, P=0.2701 (shKDM5C -1) and P=0.4753 (shKDM5C-2); in ACHN cells, P=0.4308 (shKDM5C-1) and P=0.3869 (shYY1-2).\\n\\n\\n\\nFigure EV2 ChIP-seq experiments targeting YY1 in HK-2 cells \\n\\nA. Heatmap and metaplot showing the chromatin enrichment of YY1 in HK-2 cells in the presence or absence of KDM5C.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='B-C. ChIP-PCR assay showing the enrichment of YY1 at the promoters of the indicated genes in HK-2 cells after the depletion of YY1, KDM5C or both (B), or in cells expressing KDM5C-WT or KDM5C-H514A mutant (C). Each experiment was repeated three times with similar results. Statistical significate was determined by one-way ANOVA. In B, P=1.24E-06 (JAM3), P=2.27E-06 (KNTC1), P=2.82E-08 (SMC2), P=2.41E-05 (SMC4) and P=3.96E-05 (FOXM1), respectively. In C, P=4.53E-06 (JAM3), P=4.46E-06 (KNTC1), P=5.53E-06(SMC2), P=6.82E-06 (SMC4), P=1.05E-06 (FOXM1) and P=0.0002 (LMNB1), respectively.\\n\\n\\n\\nFigure EV3 Evaluation of the interaction between YY1 and KDM5C-WT or KDM5C-H514A mutant.'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='Co-IP followed by Western blotting showing the interaction between YY1 and KDM5C-WT or KDM5C-H514A mutant in HEK293T cells.\\n\\n\\n\\nFigure EV4 RNA-seq in HK-2 derived cells.\\n\\nA-C. Volcano plots showing the change in the number of transcripts in shKDM5C, shYY1 or double-knockdown HK-2 cells, as determined by RNA-seq analysis (|log2 FC|>1.5, p value <0.05). \\n\\nD. Heatmap showing the normalized gene expression profile in shKDM5C, shYY1 or double sh ACHN cells. “Double sh” indicates cells in which KDM5C and YY1 were both knocked down.\\n\\n\\n\\nFigure EV5 Effects of KDM5C on YY1 enrichment at promoter regions of specific genes'), Document(metadata={'source': './data/docx/EMBOR-2024-58727V1.docx'}, page_content='A-B. The connections between the ChIP-seq and RNA-seq data were analyzed, and the genomic regions of the downregulated genes in the KDM5C/YY1 double-knockdown group were annotated.\\n\\nC-D. ChIP-seq snapshots reflecting changes in YY1 enrichment at the promoter regions of the JAM3, LNMB1, FOXM1, KNTC1, SMC2 and SMC4 genes in the indicated cell lines. The promoters are indicated by H3K4me3 peaks.\\n\\n\\n\\n\\n\\n9')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Number of tokens (513) exceeded maximum context length (512).\n",
      "Number of tokens (514) exceeded maximum context length (512).\n",
      "Number of tokens (515) exceeded maximum context length (512).\n",
      "Number of tokens (516) exceeded maximum context length (512).\n",
      "Number of tokens (517) exceeded maximum context length (512).\n",
      "Number of tokens (518) exceeded maximum context length (512).\n",
      "Number of tokens (519) exceeded maximum context length (512).\n",
      "Number of tokens (520) exceeded maximum context length (512).\n",
      "Number of tokens (521) exceeded maximum context length (512).\n",
      "Number of tokens (522) exceeded maximum context length (512).\n",
      "Number of tokens (523) exceeded maximum context length (512).\n",
      "Number of tokens (524) exceeded maximum context length (512).\n",
      "Number of tokens (525) exceeded maximum context length (512).\n",
      "Number of tokens (526) exceeded maximum context length (512).\n",
      "Number of tokens (527) exceeded maximum context length (512).\n",
      "Number of tokens (528) exceeded maximum context length (512).\n",
      "Number of tokens (529) exceeded maximum context length (512).\n",
      "Number of tokens (530) exceeded maximum context length (512).\n",
      "Number of tokens (531) exceeded maximum context length (512).\n",
      "Number of tokens (532) exceeded maximum context length (512).\n",
      "Number of tokens (533) exceeded maximum context length (512).\n",
      "Number of tokens (534) exceeded maximum context length (512).\n",
      "Number of tokens (535) exceeded maximum context length (512).\n",
      "Number of tokens (536) exceeded maximum context length (512).\n",
      "Number of tokens (537) exceeded maximum context length (512).\n",
      "Number of tokens (538) exceeded maximum context length (512).\n",
      "Number of tokens (539) exceeded maximum context length (512).\n",
      "Number of tokens (540) exceeded maximum context length (512).\n",
      "Number of tokens (541) exceeded maximum context length (512).\n",
      "Number of tokens (542) exceeded maximum context length (512).\n",
      "Number of tokens (543) exceeded maximum context length (512).\n",
      "Number of tokens (544) exceeded maximum context length (512).\n",
      "Number of tokens (545) exceeded maximum context length (512).\n",
      "Number of tokens (546) exceeded maximum context length (512).\n",
      "Number of tokens (547) exceeded maximum context length (512).\n",
      "Number of tokens (548) exceeded maximum context length (512).\n",
      "Number of tokens (549) exceeded maximum context length (512).\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing document: ./data/docx/EMBOR-2024-59101V1-T.docx\n",
      "Split documents: [Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='ER-to-lysosome-associated degradation\\n\\nacts as failsafe mechanism upon ERAD dysfunction\\n\\n\\n\\n\\n\\nElisa Fasana1+, Ilaria Fregno1+, Carmela Galli1+, Tatiana Soldà1 and Maurizio Molinari1,2*\\n\\n\\n\\n1Faculty of Biomedical Sciences, Institute for Research in Biomedicine, Università della Svizzera italiana (USI), 6500 Bellinzona, Switzerland\\n\\n2School of Life Sciences, École Polytechnique Fédérale de Lausanne, 1015 Lausanne, Switzerland\\n\\n\\n\\n+Equal contribution\\n\\n*Correspondence to: maurizio.molinari@irb.usi.ch'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='The endoplasmic reticulum (ER) produces proteins destined to organelles of the endocytic and secretory pathways, the plasma membrane, and the extracellular space. While native proteins are transported to their intra- or extra-cellular site of activity, folding-defective polypeptides are retro-translocated across the ER membrane into the cytoplasm, poly-ubiquitylated and degraded by 26S proteasomes in a process called ER-associated degradation (ERAD). Large misfolded polypeptides, such as polymers of alpha1 antitrypsin Z (ATZ) or mutant procollagens, fail to be dislocated across the ER membrane and instead enter ER-to-lysosome-associated degradation (ERLAD) pathways. Here, we show that pharmacological or genetic inhibition of ERAD components, such as the'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='or genetic inhibition of ERAD components, such as the a1,2-mannosidase EDEM1 or the OS9 ERAD lectins triggers the delivery of the canonical ERAD clients Null Hong Kong (NHK) and BACE457D to degradative endolysosomes under control of the ER-phagy receptor FAM134B and the LC3 lipidation machinery. Our results reveal that ERAD dysfunction is compensated by activation of FAM134B-driven ERLAD pathways that ensure efficient lysosomal clearance of orphan ERAD clients.'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Keywords\\n\\nEndoplasmic reticulum (ER); ER-associated degradation (ERAD); ER-to-lysosome-associated degradation (ERLAD); ER-phagy; Protein quality control.\\n\\n\\nIntroduction'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Introduction\\n\\nMaintenance of cellular homeostasis relies on efficient clearance of folding-defective gene products. ERAD, for ER-associated protein degradation, is an acronym coined to describe the proteasomal clearance from the ER of misfolded pro-alpha factor in a reconstituted yeast system (McCracken & Brodsky, 1996). ERAD now defines all the pathways in eukaryotic cells that ensure recognition of folding-defective polypeptides in the ER lumen or membrane and control their transport into the cytosol for poly-ubiquitylation and degradation by 26S proteasomes (Fig. 1A) (Christianson et al, 2023; Hebert et al, 2010; Sun & Brodsky, 2019).'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Recently, it has been observed that large misfolded proteins fail to be transported across the ER membrane into the cytosol and fail therefore to enter ERAD pathways. These polypeptides are segregated in specialized ER subdomains and are eventually delivered to degradative endolysosomes or autolysosomes (as defined in (Bright et al, 2016; Huotari & Helenius, 2011) and (Klionsky et al, 2014), respectively, Fig. 1B) in Metazoan cells, or to vacuoles in yeast and plant cells (Rudinskiy & Molinari, 2023). The term ERLAD for ER-to-lysosome-associated degradation was coined to describe the delivery from the ER to degradative LAMP1-positive endolysosomes of disease-causing ATZ polymers (Fregno et al, 2018; Fregno et al, 2021) or of misfolded procollagen (Forrester'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Fregno et al, 2021) or of misfolded procollagen (Forrester et al, 2019) (Fig. 1B, arrows 1 and 2, respectively). Nowadays, ERLAD is an umbrella term that embraces all the autophagic and non-autophagic pathways that deliver ERAD-resistant misfolded polypeptides from the ER to degradative endolysosomes/autolysosomes/vacuoles (Fig. 1B) (Fregno & Molinari, 2019; Gubas & Dikic, 2022; Klionsky et al, 2021; Rudinskiy & Molinari, 2023). Mechanistically, the clearance from the ER of ATZ polymers and of procollagen molecules relies on cycles of de-/re-glucosylation of N-linked oligosaccharides that prolong calnexin (CNX) binding to favor engagement of the ER membrane protein FAM134B (Fregno et al., 2018; Fregno et al., 2021). FAM134B has been first described as a'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Fregno et al., 2021). FAM134B has been first described as a LC3-binding protein, whose role in autophagic clearance of ER subdomains is activated upon nutrient restriction (Khaminets et al, 2015). More recently, our group established that its role as an ER-phagy receptor (i.e., a protein that controls lysosomal clearance of ER portions) is also activated by the luminal accumulation of ERAD resistant misfolded polypeptides (Forrester et al., 2019; Fregno et al., 2018; Fregno et al., 2021). Disposal of ATZ monomers proceeds via ERAD. In contrast, clearance from cells of ATZ polymers relies on their delivery into lysosomal degradative compartments and involves ATG5 and ATG7 autophagy gene products that control LC3 lipidation  (Chu et al, 2014; Fregno et al.,'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='that control LC3 lipidation  (Chu et al, 2014; Fregno et al., 2018; Hidvegi et al, 2010; Kroeger et al, 2009; Pastore et al, 2013; Sun et al, 2023; Teckman & Perlmutter, 2000). Despite the intervention of the LC3 lipidation machinery, in all mammalian cell lines tested in our lab, the lysosomal clearance of ATZ polymers does not require intervention of autophagosomes (Fig. 1B, arrow 1) (Fregno et al., 2018; Fregno et al., 2021). In contrast, autophagosomes are involved in disposal of procollagen molecules, which proceeds via macro-ER-phagy (Fig. 1B, arrow 2) (Forrester et al., 2019). A variety of ERLAD clients is found in the literature, but for few of them the degradative mechanisms have been characterized in molecular detail (Rudinskiy & Molinari, 2023).'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='characterized in molecular detail (Rudinskiy & Molinari, 2023). Certainly, a variety of ERLAD pathways does operate in eukaryotic cells. These catabolic programs are regulated by client- and tissue-specific ER-phagy receptors, whose activity in promoting clearance of ER subdomains is activated by the intraluminal or membrane accumulation of misfolded polypeptides, including FAM134B (Forrester et al., 2019; Fregno et al., 2018; Fregno et al., 2021), CCPG1 (Ishii et al, 2023; Smith et al, 2018) and FAM134B-2 (Kohno et al, 2019). Their activation ensures delivery of ER content to be removed from cells to degradative organelles via LC3-dependent delivery (Fig. 1B, arrow 1), macro-ER-phagy (arrow 2) or micro-ER-phagy (arrow 3).'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Notably, ERAD inhibition delays, rather than blocks, degradation of ERAD clients, hinting at alternative pathways intervening to ensure efficient clearance of misfolded proteins generated in the ER under conditions of ERAD impairment or overload (Molinari, 2007). To uncover a possible interplay of proteasomal (ERAD) and lysosomal (ERLAD) quality control of defective gene products synthesized in the ER, we assessed the capacity of the ERLAD machinery to support clearance of Null Hong Kong (NHK) and BACE457D, two classical ERAD clients, upon pharmacologic or genetic ERAD inactivation. The NHK variant of alpha1 antitrypsin is a disease-causing folding-defective glycoprotein (Sifers et al, 1988). BACE457D is a folding-defective splice variant of beta-secretase'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='is a folding-defective splice variant of beta-secretase (Molinari et al, 2002). Characterization of the machineries ensuring proteasomal clearance of NHK and BACE457D from the mammalian ER revealed general principles of ERAD: i) the role of mannose processing by EDEM proteins to extract terminally misfolded polypeptides from the CNX folding cycle (Chiritoiu et al, 2020; Hirao et al, 2006; Liu et al, 1997, 1999; Molinari et al, 2003; Oda et al, 2003; Olivari et al, 2006); ii) the engagement of OS9 ERAD lectins that deliver misfolded polypeptides to various client-specific dislocons embedded in the ER membrane that control ERAD clients retro-translocation into the cytosol for poly-ubiquitylation and proteasomal degradation (Fig. 1A) (Bernasconi et al, 2010a;'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='and proteasomal degradation (Fig. 1A) (Bernasconi et al, 2010a; Bernasconi et al, 2008; Christianson et al, 2008; Ninagawa et al, 2011; Sugimoto et al, 2017)); iii) the involvement of members of the protein disulfide isomerase (Guerra et al, 2018; Molinari et al., 2002; Ushioda et al, 2008) and of the peptidyl-prolyl cis/trans isomerase superfamilies (Bernasconi et al, 2010b). As such, NHK and BACE457D are among the best characterized clients of the ERAD machinery. Here, we monitor their fate in cells with pharmacologically- or genetically-induced dysfunction of ERAD. Our data reveal the intervention of FAM134B-driven ERLAD pathways to warrant efficient clearance of misfolded ERAD clients in cells characterized by dysfunctional ERAD.'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Results\\n\\nBafA1 stabilizes NHK upon prolonged pharmacologic inhibition of 26S-proteasomes or a1,2- mannosidases'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='To assess a possible intervention of ERLAD to compensate dysfunctional ERAD, the degradation of 35S-methionine/cysteine radiolabeled NHK was determined in mock-treated HEK293 cells (Figs. 2A, lanes 1-3 (for one representative experiment), 2B-2C, mock (for quantification of 3 biological replicates)), in HEK293 cells incubated with the dipeptide boronic acid Bortezomib/PS341, a selective inhibitor of 26S proteasomes (Adams et al, 1999) (Figs. 2A, lanes 4-5, 2B, PS), or with the alkaloid kifunensine (KIF), a selective inhibitor of a1,2 mannosidases of the glycosyl hydrolase 47 family members, which includes the three EDEM proteins (Elbein et al, 1990; Liu et al., 1999; Moremen & Molinari, 2006; Olivari & Molinari, 2007; Vallee et al, 2000) (Figs. 2A, lanes'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Olivari & Molinari, 2007; Vallee et al, 2000) (Figs. 2A, lanes 8-9, 2C, KIF). The immunoisolation from cell lysates of radiolabeled NHK after 75 or 120 min of chase (Fig. 2A) reveals that both PS341 (Fig. 2B, PS) and KIF (Fig. 2C, KIF) delay, rather than block ERAD of NHK. The slower decay of NHK in cells with dysfunctional ERAD (i.e., in cells exposed to PS341 or to KIF) is further reduced upon lysosomal inactivation with BafA1 (Fig. 2B, lanes 6, 7 and 10, 11 and the grey zone in Figs. 2B, 2C). Notably, lysosomal inhibitors per se, have no evident inhibitory activity on NHK disposal (Liu et al., 1999) (Fig. 2D). Thus, lysosomal inactivation does not impact on clearance of NHK (Fig. 2D), unless the ERAD pathway is dysfunctional (Fig. 2A, lanes 6, 7, 10, 11'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='the ERAD pathway is dysfunctional (Fig. 2A, lanes 6, 7, 10, 11 and grey zones in Figs. 2B, 2C).'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='The ERAD client NHK is normally not delivered to endolysosomes for clearance'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='The canonical ERLAD client ATZ is delivered to LAMP1-positive endolysosomes for clearance (Fregno et al., 2018; Fregno et al., 2021). Consistently, when mouse 3T3 cells expressing HA-tagged ATZ were incubated with BafA1 to inhibit lysosomal hydrolases and preserve delivered material in the lumen of degradative organelles (Klionsky et al, 2008), ATZ accumulates in endolysosomes that display LAMP1 at the limiting membrane as seen in Confocal laser scanning microscopy (CLSM) (Fig. 2E). ATZ delivery to the degradative compartment is quantified with LysoQuant (Morone et al, 2020), a deep-learning-based analysis software for segmentation and classification of fluorescence images (Fig. 2I, ATZ+BafA1). The ERAD client NHK, whose clearance from cells is not delayed'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='The ERAD client NHK, whose clearance from cells is not delayed upon inhibition of lysosomal enzymes with BafA1 (Fig. 2D) (Liu et al., 1999) is poorly delivered within LAMP1-positive degradative compartments (Figs. 2F and 2J and quantification in 2I and 2O, NHK+BafA1).'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Pharmacologic inhibition of ERAD triggers lysosomal delivery of NHK'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Motivated by the results of the biochemical analyses showing that inactivation of lysosomal hydrolases stabilizes NHK when ERAD is inactive (Figs. 2A-2C), we examined the delivery to endolysosomes of NHK upon pharmacologic inhibition of the ERAD machinery in mouse 3T3 cells. CLSM reveals the enhanced accumulation of NHK in endolysosomes that display LAMP1 at the limiting membrane in cells where ERAD has been inhibited with KIF (Figs. 2G, 2I), or with PS341 (Figs. 2H, 2I). Thus, inhibition of ERAD at early (KIF) and late steps (PS341) of client’s selection, activates channeling of misfolded polypeptides into ERLAD pathways that deliver them to degradative LAMP1-positive compartments. The degradative nature of the LAMP1-positive endolysosomes where NHK'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='nature of the LAMP1-positive endolysosomes where NHK accumulates in cells treated with BafA1 (Figs. 2F-2I, 2K, 2M, 2O) is confirmed by the disappearance of the NHK immunoreactivity from the endosome’s lumen upon BafA1 washout (Figs. 2L, 2N-2O) and (Fregno et al., 2018).'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Genetic inhibition of ERAD upon silencing of EDEM1 triggers delivery of NHK to degradative endolysosomes'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='EDEM1 is an active a1,2-mannosidase that ensures extraction of terminally misfolded glycoproteins from the CNX chaperone system to direct them for proteasomal degradation (Molinari et al., 2003; Oda et al., 2003; Olivari et al., 2006; Olivari & Molinari, 2007). Silencing of EDEM1 expression delays ERAD of glycoproteins (Molinari et al., 2003). To verify whether silencing of EDEM1 expression diverts NHK into the ERLAD pathway to compensate for ERAD dysfunction, we compared delivery of HA-tagged NHK in the LAMP1-positive endolysosomes in HEK293 cells expressing a scrambled short hairpin RNA (shCTRL), or a short hairpin RNA targeting the EDEM1 sequence (shEDEM, Fig. 3A). Cells were exposed to BafA1 to preserve the NHK fraction possibly delivered in the lumen'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='to preserve the NHK fraction possibly delivered in the lumen of endolysosomes. CLSM reveals the enhanced accumulation of NHK in endolysosomes that display LAMP1 at the limiting membrane in cells with reduced EDEM1 levels (Fig. 3B, lower panels), which has been quantified by LysoQuant (Fig. 3C).'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Genetic inhibition of ERAD upon silencing of OS9 proteins triggers delivery of NHK to degradative endolysosomes'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='De-mannosylation of N-glycans on misfolded glycoproteins generates binding sites for the OS9 ERAD lectins, which are expressed in two ER stress-induced splice variants in the ER lumen (Bernasconi et al., 2008). OS-9 association facilitates proteasomal clearance of NHK by promoting the delivery of the misfolded polypeptide to the HRD1/SEL1L retro-translocation machinery (Fig. 1A) (Bernasconi et al., 2010a; Bernasconi et al., 2008; Christianson et al., 2008). We and others previously showed that silencing of OS9 expression delays ERAD of NHK (Bernasconi et al., 2008; Christianson et al., 2008). To verify whether under these conditions NHK was diverted into the ERLAD pathways to compensate ERAD dysfunction, we monitored NHK delivery in the LAMP1-positive'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='dysfunction, we monitored NHK delivery in the LAMP1-positive endolysosomes in HEK293 cells expressing a control short hairpin scrambled RNA (shCTRL), or two short hairpin RNAs targeting the OS9.1 and OS9.2 sequences (shOS9.1/OS9.2 A and shOS9.1/OS9.2 B, Fig. 3D). shOS9.1/OS9.2 A reduced by about 30% the cellular level of OS9.1, and by about 70% the level of OS9.2 (Fig. 3D). shOS9.1/OS9.2 B significantly reduced only the level of OS9.2 (Fig. 3D). CLSM reveals that in cells expressing shOS9.1/OS9.2 A NHK delivery to the degradative LAMP1-positive organelles is enhanced (Figs. 3E, panels in the middle, 3F) to higher levels compared to cells expressing shOS9.1/OS9.2 B (Figs. 3E, lower panels, 3F).'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='FAM134B drives lysosomal delivery of NHK upon ERAD inhibition'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Misfolded polypeptides that fail to engage the ERAD machinery (ATZ polymers are shown as examples in Figs. 2E, 2I) are segregated in ER subdomains displaying FAM134B at the limiting membrane and are eventually delivered to LAMP1-positive endolysosomal compartments for clearance (Fregno et al., 2018; Fregno et al., 2021; Rudinskiy & Molinari, 2023). If ERAD clients would co-opt the same pathway for lysosomal clearance upon ERAD inactivation, it is expected that NHK delivery to the LAMP1-positive compartment is substantially inhibited in cells lacking FAM134B. To test this hypothesis, the experiments described in the previous sections were reproduced in mouse embryonic fibroblasts (MEF) subjected to CRISPR/Cas9 genome editing to knockout FAM134B (Fig. 4A,'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='to CRISPR/Cas9 genome editing to knockout FAM134B (Fig. 4A, lane 2). As shown above for other cell lines, also in wild type (WT) MEF (Figs. 4B, upper panels, 4C) and in MEF lacking the FAM134B (Figs. 4B, lower panels, 4C) the ERAD client NHK is not delivered to the LAMP1-positive endolysosomes. Inactivation of the ERAD pathway with KIF promotes NHK delivery to the LAMP1-positive degradative compartments WT MEF (Figs. 4D, 4K), which is virtually abolished upon deletion of FAM134B (Figs. 4E, 4K, pCDNA3). The back-transfection of V5-tagged FAM134B in FAM134B-KO MEF (Fig. 4A, lane 3) restores NHK delivery to the LAMP1-positive endolysosomes (Figs. 4F, 4K). The back-transfection of FAM134BLIR (Fig. 4A, lane 4), an inactive variant of the ER-phagy receptor that'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='4A, lane 4), an inactive variant of the ER-phagy receptor that carries mutations in the LIR domain preventing engagement of cytosolic LC3 molecules (Fregno et al., 2018), is not restoring NHK delivery to endolysosomes (Figs. 4G, 4K). The FAM134B-driven deviation of NHK into the ERLAD pathways is also induced when ERAD of NHK is abolished upon MEF exposure to the proteasomal inhibitor PS341 (Figs. 4H-4K).'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Involvement of autophagy genes in compensatory ERLAD pathways'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='ER-phagy receptors control clearance of ER subdomains upon engagement of various autophagy gene products that will determine if ERLAD will proceed via macro-ER-phagy involving double membrane autophagosomes (as for misfolded procollagen) (Forrester et al., 2019), or via other types of ER-phagy that will not involve autophagosomes and their biogenesis (as for ATZ) (Chino & Mizushima, 2023; Fregno et al., 2018; Fregno et al., 2021; Reggiori & Molinari, 2022). Previous work in our lab showed that FAM134B-controlled delivery of ATZ polymers to LAMP1-positive endolysosomes involves the LC3 lipidation machinery but does not involve the autophagosome biogenesis machinery. Consistently, deletion of the autophagy gene Atg7 that abolishes LC3 lipidation (Komatsu et'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='autophagy gene Atg7 that abolishes LC3 lipidation (Komatsu et al, 2005) prevents ATZ delivery to endolysosomal compartments. In contrast, deletion of ATG13, a crucial component of the autophagosome biogenesis machinery (Hosokawa et al, 2009; Kaizuka & Mizushima, 2016; Suzuki et al, 2014) does not impact on ATZ delivery to the degradative district (Fregno et al., 2018). The repetition of these experiments to monitor the case of the ERAD client NHK confirms that the misfolded protein is not normally delivered to the LAMP1-positive endolysosomal compartment (Figs. 5A, 5H and Figs. 2-4), unless the ERAD pathway has been inactivated upon cell exposure to the a1,2-mannosidases inhibitor KIF or the proteasomal inhibitor PS341 (Figs. 5B, 5C, 5H and Figs. 2, 4). In'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='inhibitor PS341 (Figs. 5B, 5C, 5H and Figs. 2, 4). In cells lacking ATG7, which show impaired generation of the lipidated form of LC3 (LC3-II, Fig. 5D), the delivery of NHK to the LAMP1-positive degradative compartment is abolished (Figs. 5E, 5H). In cells lacking ATG13 (Fig. 5F), the delivery of NHK to the LAMP1-positive degradative compartment proceeds unperturbed (Figs. 5G, 5H). These results recapitulate the phenotype previously observed for clearance of ATZ polymers, which is hampered in cells with defective LC3 lipidation  (Chu et al., 2014; Fregno et al., 2018; Hidvegi et al., 2010; Kroeger et al., 2009; Pastore et al., 2013; Sun et al., 2023; Teckman & Perlmutter, 2000), but remains unaffected in cells with defective autophagosome biogenesis (Fregno'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='in cells with defective autophagosome biogenesis (Fregno et al., 2018).'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Pharmacologic and genetic inhibition of ERAD triggers lysosomal delivery of BACE457D'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Monitoring the fate of BACE457D, another folding defective polypeptide that has been used extensively to characterized mechanistically the ERAD pathways (Bernasconi et al., 2010a; Bernasconi et al., 2010b; Cali et al, 2008; Eriksson et al, 2004; Horimoto et al, 2013; Molinari et al., 2003; Molinari et al., 2002; Ninagawa et al., 2011; Olivari et al., 2006; Olivari et al, 2005; Sokolowska et al, 2015), we confirmed that ERLAD acts as surrogate catabolic pathway to remove misfolded polypeptides from cells with dysfunctional ERAD (Fig. 6). As shown above for NHK, biochemical analyses of BACE457D decay confirm that the inhibition of lysosomal enzymes does not affect clearance from the ER (Molinari et al., 2002), unless cells have dysfunctional ERAD upon'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='et al., 2002), unless cells have dysfunctional ERAD upon inhibition of cytosolic proteasomes with PS341 (Fig. 6A, lanes 6, 7 and grey zones in Fig. 6B) or of luminal mannosidases with KIF (Fig. 6A, lanes 10, 11 and grey zones in Fig. 6C). CLSM analyses confirm delivery of BACE457D within LAMP1-positive endolysosomes in cells with dysfunctional ERAD (Figs. 6D-6G), and the involvement of FAM134B in the surrogate catabolic pathways activated under these circumstances (Figs. 6H-6O).'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Discussion'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='A stringent protein quality control operates in the ER of eukaryotic cells. Proteins that have completed their folding program are released from ER-resident molecular chaperones and exit the ER to be delivered to their final intra- or extra-cellular destination. Folding is error prone, and the rate of misfolding is substantially enhanced by mutations in the polypeptide sequence. The incapacity to fold correctly, results in selection of the aberrant gene product for degradation, or in the formation of aggregates that are retained in the biosynthetic compartment in soluble or insoluble forms. Dedicated machineries are available in the ER lumen and membranes to distinguish misfolded or incompletely folded polypeptides to be retained in the ER lumen, from'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='folded polypeptides to be retained in the ER lumen, from native and functional proteins to be released (Ellgaard et al, 1999). Incompletely folded polypeptides retained in the ER are exposed to the folding environment and can eventually reach the native, transport-permissive architecture. When folding is impossible, the polypeptides are actively deviated into ERAD pathways that ensure their transport into the cytosol for proteasomal degradation (Fig. 1A). An increasing number of misfolded proteins is emerging in the literature that cannot enter ERAD pathways. In many cases, these are large polypeptides or polypeptides that are prone to form aggregates or polymers. A variety of ERLAD pathways are available in nucleated cells to segregate ERAD-resistant'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='are available in nucleated cells to segregate ERAD-resistant polypeptides in dedicated ER subdomains and to deliver them to endolysosomal/vacuolar degradative compartments (Fig. 1B) (Rudinskiy & Molinari, 2023).'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='For ERAD clients, numerous studies show that clearance from the ER is delayed, and not abolished upon inactivation of the ubiquitin proteasome system. This also applies for cytosolic misfolded proteins, whose clearance from cells hosting inactive proteasomes has been proposed to rely on intervention of the giant protease tripeptidyl peptidase II, which shows enhanced activity in proteasome-inhibitor adapted cell (Geier et al, 1999; Glas et al, 1998; Tomkinson, 2019). Our study demonstrates that the intervention of ERLAD pathways is not limited to clearance of large proteins that fail to be dislocated across the ER membrane for ERAD. Rather, ERLAD may be engaged by ERAD clients when their preferred road-to-destruction is dysfunctional or saturated by an'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='road-to-destruction is dysfunctional or saturated by an excess of aberrant gene products (Fig. 7).'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Methods'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Expression plasmids and antibodies-HA-tagged ATZ and NHK and V5-tagged FAM134B and FAM134BLIR are subcloned in pcDNA3.1 plasmids. BACE457D is subcloned in pCDNA3 plasmid. pDest-EGFP-Rab7 is a kind gift from T. Johansen. The polyclonal anti-HA (Sigma cat. H6908), the anti-V5 (Invitrogen cat. 46-0705) and the anti-LAMP1 (Hybridoma Bank, 1D4B deposited by J.T. August) are described in (Fregno et al., 2021). The anti-FAM134B is a kind gift of M. Miyazaki (University of Colorado Denver); the anti-EDEM1 is from Sigma (cat. E8406); the anti OS-9 is from Novus Biologicals (cat. BC100-520); the anti-ATG7 and anti-ATG13 are from Sigma (cat. SAB4200304 and SAB4200100, respectively); the anti LC3 from Novus (BC100-520); the anti GAPDH from Merck (MAB374).'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='LC3 from Novus (BC100-520); the anti GAPDH from Merck (MAB374). Alexa-conjugated secondary antibodies are from Thermo Fisher; HRP-conjugated Protein A is from Invitrogen; HRP-conjugated anti-mouse is from SouthernBiotech).'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Cell Lines, transient transfections, pharmacologic inhibition-Flp-InTM-3T3 cells (Thermo Fisher) stably expressing ATZ-HA or NHK-HA were generated following manufacturer instructions and cultured in DMEM supplemented with 10% FCS and 150g/ml Hygromycin. MEF and HEK293 cells were grown in DMEM/10% FBS. HEK293FT cells expressing reduced levels of OS9.1 and OS9.2 are described in (Bernasconi et al., 2008). FAM134B-deficient MEF cells (CRISPR FAM134B) were generated using CRISPR/Cas9 genome editing protocol as described in (Fumagalli et al, 2016).'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Transient transfections were performed using JetPrime transfection reagent (PolyPlus) following manufacturer’s protocol. BafA1 (Calbiochem) was used at 50 nM for 12 h; KIF (Toronto Research Chemicals) was used at 200μM for 12h; PS341 (LubioScience) was used 12h at 100nM for Flp-InTM-3T3 cells or at 5nM for MEF cells. In wash/out experiments Flp-InTM-3T3 cells were incubated for 8h with 100nM BafA1 and indicated drugs.\\n\\nTo induce autophagy in MEF WT and ATG7KO, cells were washed three times with Earle’s balanced salt solution (EBSS, Thermo Fisher) and then incubated for 4h with 100nM BafA1.\\n\\n\\n\\nCell lysis, immunoprecipitation, and Western blot'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='After treatments, cells were washed with ice-cold PBS containing 20 mM NEM and lysed with RIPA Buffer in HBS pH 7.4 supplemented with protease inhibitors. Post-nuclear supernatants (PNS) were collected after centrifugation at 10,600 g for 10 min. For immunoprecipitations, PNSs were diluted with lysis buffer and incubated with Protein A (1:10 w/v, swollen in PBS) and select antibodies at 4°C. After three washes of the immunoprecipitates with 0.5% Triton X-100 in HBS pH 7.4, beads were denatured for 5 min at 95°C and subjected to SDS–PAGE. Proteins were transferred to PVDF membranes using the Trans-Blot Turbo Transfer System (Bio-Rad). Membranes were blocked with 10% (w/v) non-fat dry milk (Bio-Rad) in TBS-T and stained with primary antibodies diluted in'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='in TBS-T and stained with primary antibodies diluted in TBS-T followed by HRP-conjugated secondary antibodies or Protein A diluted in TBS-T. Membranes were developed using Western Bright ECL or Quantum (Advansta) and signals captured on Fusion FX (Vilber). Images were quantified with the Evolution Capture Edge (Vilber).'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Metabolic labeling\\n\\nSeventeen hours after transient transfections, cells were pulsed with 0.05 mCi [35S]methionine/cysteine mix and chased for the indicated time points with DMEM supplemented with 5mM cold methionine and cysteine. Cells were detergent-solubilized and radiolabeled proteins were revealed with Typhoon FLA 9500, version 1.0 (GE Healthcare). Radioactive signals were quantified using the ImageQuant software (Molecular Dynamics, GE Healthcare).'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Confocal laser scanning microscopy-Cells plated on Alcian Blue-treated glass coverslips were washed with PBS and fixed at room temperature for 20 min in 3.7% formaldehyde diluted in PBS. Cells were permeabilized for 15 min with 0.05% saponin, 10% goat serum, 10mM HEPES, 15mM glycine (PS) and exposed for 90 min to primary antibodies diluted 1:100 in PS. After washes with PS, cells were incubated for 45 min with Alexa Fluor-conjugated secondary antibodies diluted 1:300 in PS. Cells were rinsed with PS and water and mounted with Vectashield (Vector Laboratories). Confocal images were acquired on a Leica TCS SP5 microscope with a Leica HCX PL APO lambda blue 63.0X1.40 OIL UV objective. IF images were acquired by two different people, double-blinded, and raw'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='were acquired by two different people, double-blinded, and raw data were quantified using LysoQuant (Morone et al., 2020).'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Statistical analyses-Plots and statistical analyses were performed using GraphPad Prism10 (GraphPad Software Inc.). In this study, one-way ANOVA with Dunnett’s multiple comparisons test and unpaired t-test were used to assess statistical significance. An adjusted P-value < 0.05 was considered as statistically significant.\\n\\n\\n\\nData availability\\n\\nThis study includes no data deposited in external repositories.\\n\\n\\n\\nAcknowledgments\\n\\nWe thank members of Molinari’s group for discussions and the IRB Microscopy core facility for assistance with microscopy.\\n\\nMM is supported by Swiss National Science Foundation grants 310030_214903 and 320030_227541, Eurostar E!2228, Innosuisse 35449.1 IP-LS. \\n\\n\\n\\n\\n\\n\\n\\nDisclosure and competing interests statement'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Disclosure and competing interests statement\\n\\nAuthors declare no competing interests.\\n\\n\\n\\n\\n\\n\\nAdams J, Palombella VJ, Sausville EA, Johnson J, Destree A, Lazarus DD, Maas J, Pien CS, Prakash S, Elliott PJ (1999) Proteasome inhibitors: a novel class of potent and effective antitumor agents. Cancer Res 59: 2615-2622\\n\\nBernasconi R, Galli C, Calanca V, Nakajima T, Molinari M (2010a) Stringent requirement for HRD1, SEL1L, and OS-9/XTP3-B for disposal of ERAD-LS substrates. The Journal of cell biology 188: 223-235'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Bernasconi R, Pertel T, Luban J, Molinari M (2008) A Dual Task for the Xbp1-responsive OS-9 Variants in the Mammalian Endoplasmic Reticulum: Inhibiting Secretion of Misfolded Protein Conformers and Enhancing their Disposal. J Biol Chem 283: 16446-16454\\n\\nBernasconi R, Solda T, Galli C, Pertel T, Luban J, Molinari M (2010b) Cyclosporine A-sensitive, cyclophilin B-dependent endoplasmic reticulum-associated degradation. PLoS One 5: e13008\\n\\nBright NA, Davis LJ, Luzio JP (2016) Endolysosomes Are the Principal Intracellular Sites of Acid Hydrolase Activity. Curr Biol 26: 2233-2245'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Cali T, Galli C, Olivari S, Molinari M (2008) Segregation and rapid turnover of EDEM1 by an autophagy-like mechanism modulates standard ERAD and folding activities. Biochem Biophys Res Commun 371: 405-410\\n\\nChino H, Mizushima N (2023) ER-Phagy: Quality and Quantity Control of the Endoplasmic Reticulum by Autophagy. Cold Spring Harb Perspect Biol 15: 1-16\\n\\nChiritoiu M, Chiritoiu GN, Munteanu CVA, Pastrama F, Ivessa NE, Petrescu SM (2020) EDEM1 Drives Misfolded Protein Degradation via ERAD and Exploits ER-Phagy as Back-Up Mechanism When ERAD Is Impaired. Int J Mol Sci 21: 3468-3494\\n\\nChristianson JC, Jarosch E, Sommer T (2023) Mechanisms of substrate processing during ER-associated protein degradation. Nat Rev Mol Cell Biol 24: 777-796'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Christianson JC, Shaler TA, Tyler RE, Kopito RR (2008) OS-9 and GRP94 deliver mutant alpha1-antitrypsin to the Hrd1/SEL1L ubiquitin ligase complex for ERAD. Nat Cell Biol 10: 272-282\\n\\nChu AS, Perlmutter DH, Wang Y (2014) Capitalizing on the Autophagic Response for Treatment of Liver Disease Caused by Alpha-1-Antitrypsin Deficiency and Other Genetic Diseases. Biomed Research International: 1-8\\n\\nElbein AD, Tropea JE, Mitchell M, Kaushal GP (1990) Kifunensine, a potent inhibitor of the glycoprotein processing mannosidase I. J Biol Chem 265: 15599-15605\\n\\nEllgaard L, Molinari M, Helenius A (1999) Setting the standards: quality control in the secretory pathway. Science 286: 1882-1888'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content=\"Eriksson KK, Vago R, Calanca V, Galli C, Paganetti P, Molinari M (2004) EDEM contributes to maintenance of protein folding efficiency and secretory capacity. J Biol Chem 279: 44600-44605\\n\\nForrester A, De Leonibus C, Grumati P, Fasana E, Piemontese M, Staiano L, Fregno I, Raimondi A, Marazza A, Bruno G et al (2019) A selective ER-phagy exerts procollagen quality control via a Calnexin-FAM134B complex. EMBO J 38: e99847\\n\\nFregno I, Fasana E, Bergmann TJ, Raimondi A, Loi M, Solda T, Galli C, D'Antuono R, Morone D, Danieli A et al (2018) ER-to-lysosome-associated degradation of proteasome-resistant ATZ polymers occurs via receptor-mediated vesicular transport. EMBO J 37: e99259\"), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Fregno I, Fasana E, Solda T, Galli C, Molinari M (2021) N-glycan processing selects ERAD-resistant misfolded proteins for ER-to-lysosome-associated degradation. EMBO J 40: e107240\\n\\nFregno I, Molinari M (2019) Proteasomal and lysosomal clearance of faulty secretory proteins: ER-associated degradation (ERAD) and ER-to-lysosome-associated degradation (ERLAD) pathways. Crit Rev Biochem Mol Biol 54: 153-163\\n\\nFumagalli F, Noack J, Bergmann TJ, Presmanes EC, Pisoni GB, Fasana E, Fregno I, Galli C, Loi M, Soldà T et al (2016) Translocon component Sec62 acts in endoplasmic reticulum turnover during stress recovery. Nat Cell Biol 18: 1173-1184'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Geier E, Pfeifer G, Wilm M, Lucchiari-Hartz M, Baumeister W, Eichmann K, Niedermann G (1999) A giant protease with potential to substitute for some functions of the proteasome. Science 283: 978-981\\n\\nGlas R, Bogyo M, McMaster JS, Gaczynska M, Ploegh HL (1998) A proteolytic system that compensates for loss of proteasome function. Nature 392: 618-622\\n\\nGubas A, Dikic I (2022) ER remodeling via ER-phagy. Molecular Cell 82: 1492-1500\\n\\nGuerra C, Brambilla Pisoni G, Solda T, Molinari M (2018) The reductase TMX1 contributes to ERAD by preferentially acting on membrane-associated folding-defective polypeptides. Biochem Biophys Res Commun 503: 938-943\\n\\nHebert DN, Bernasconi R, Molinari M (2010) ERAD substrates: which way out? Semin Cell Dev Biol 21: 526-532'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Hidvegi T, Ewing M, Hale P, Dippold C, Beckett C, Kemp C, Maurice N, Mukherjee A, Goldbach C, Watkins S et al (2010) An Autophagy-Enhancing Drug Promotes Degradation of Mutant alpha 1-Antitrypsin Z and Reduces Hepatic Fibrosis. Science 329: 229-232\\n\\nHirao K, Natsuka Y, Tamura T, Wada I, Morito D, Natsuka S, Romero P, Sleno B, Tremblay LO, Herscovics A et al (2006) EDEM3, a soluble EDEM homolog, enhances glycoprotein ERAD and mannose trimming. J Biol Chem 281: 9650-9658'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Horimoto S, Ninagawa S, Okada T, Koba H, Sugimoto T, Kamiya Y, Kato K, Takeda S, Mori K (2013) The Unfolded Protein Response Transducer ATF6 Represents a Novel Transmembrane-type Endoplasmic Reticulum-associated Degradation Substrate Requiring Both Mannose Trimming and SEL1L Protein. J Biol Chem 288: 31517-31527\\n\\nHosokawa N, Hara T, Kaizuka T, Kishi C, Takamura A, Miura Y, Iemura S, Natsume T, Takehana K, Yamada N et al (2009) Nutrient-dependent mTORC1 Association with the ULK1-Atg13-FIP200 Complex Required for Autophagy. Molecular Biology of the Cell 20: 1981-1991\\n\\nHuotari J, Helenius A (2011) Endosome maturation. EMBO J 30: 3481-3500'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Ishii S, Chino H, Ode KL, Kurikawa Y, Ueda HR, Matsuura A, Mizushima N, Itakura E (2023) CCPG1 recognizes endoplasmic reticulum luminal proteins for selective ER-phagy. Molecular Biology of the Cell 34: 1-18\\n\\nKaizuka T, Mizushima N (2016) Atg13 Is Essential for Autophagy and Cardiac Development in Mice. Mol Cell Biol 36: 585-595\\n\\nKhaminets A, Heinrich T, Mari M, Grumati P, Huebner AK, Akutsu M, Liebmann L, Stolz A, Nietzsche S, Koch N et al (2015) Regulation of endoplasmic reticulum turnover by selective autophagy. Nature 522: 354-358'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content=\"Klionsky DJ, Abdel-Aziz AK, Abdelfatah S, Abdellatif M, Abdoli A, Abel S, Abeliovich H, Abildgaard MH, Abudu YP, Acevedo-Arozena A et al (2021) Guidelines for the use and interpretation of assays for monitoring autophagy (4th edition). Autophagy 17: 1-382\\n\\nKlionsky DJ, Elazar Z, Seglen PO, Rubinsztein DC (2008) Does bafilomycin A(1) block the fusion of autophagosomes with lysosomes? Autophagy 4: 849-850\\n\\nKlionsky DJ, Eskelinen EL, Deretic V (2014) Autophagosomes, phagosomes, autolysosomes, phagolysosomes, autophagolysosomes... wait, I'm confused. Autophagy 10: 549-551\"), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Kohno S, Shiozaki Y, Keenan AL, Miyazaki-Anzai S, Miyazaki M (2019) An N-terminal-truncated isoform of FAM134B (FAM134B-2) regulates starvation-induced hepatic selective ER-phagy. Life Sci Alliance 2: e201900340\\n\\nKomatsu M, Waguri S, Ueno T, Iwata J, Murata S, Tanida I, Ezaki J, Mizushima N, Ohsumi Y, Uchiyama Y et al (2005) Impairment of starvation-induced and constitutive autophagy in Atg7-deficient mice. The Journal of cell biology 169: 425-434\\n\\nKroeger H, Miranda E, MacLeod I, Perez J, Crowther DC, Marciniak SJ, Lomas DA (2009) Endoplasmic Reticulum-associated Degradation (ERAD) and Autophagy Cooperate to Degrade Polymerogenic Mutant Serpins. J Biol Chem 284: 22793-22802'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Liu Y, Choudhury P, Cabral CM, Sifers RN (1997) Intracellular disposal of incompletely folded human alpha1-antitrypsin involves release from calnexin and post-translational trimming of asparagine-linked oligosaccharides. J Biol Chem 272: 7946-7951\\n\\nLiu Y, Choudhury P, Cabral CM, Sifers RN (1999) Oligosaccharide modification in the early secretory pathway directs the selection of a misfolded glycoprotein for degradation by the proteasome. J Biol Chem 274: 5861-5867\\n\\nMcCracken AA, Brodsky JL (1996) Assembly of ER-associated protein degradation in vitro: dependence on cytosol, calnexin, and ATP. The Journal of cell biology 132: 291-298\\n\\nMolinari M (2007) N-glycan structure dictates extension of protein folding or onset of disposal. Nat Chem Biol 3: 313-320'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Molinari M, Calanca V, Galli C, Lucca P, Paganetti P (2003) Role of EDEM in the release of misfolded glycoproteins from the calnexin cycle. Science 299: 1397-1400\\n\\nMolinari M, Galli C, Piccaluga V, Pieren M, Paganetti P (2002) Sequential assistance of molecular chaperones and transient formation of covalent complexes during protein degradation from the ER. The Journal of cell biology 158: 247-257\\n\\nMoremen KW, Molinari M (2006) N-linked glycan recognition and processing: the molecular basis of endoplasmic reticulum quality control. Curr Opin Struct Biol: 592-599'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Morone D, Marazza A, Bergmann TJ, Molinari M (2020) Deep learning approach for quantification of organelles and misfolded polypeptide delivery within degradative compartments. Mol Biol Cell 31: 1512-1524\\n\\nNinagawa S, Okada T, Takeda S, Mori K (2011) SEL1L Is Required for Endoplasmic Reticulum-associated Degradation of Misfolded Luminal Proteins but not Transmembrane Proteins in Chicken DT40 Cell Line. Cell structure and function 36: 187-195\\n\\nOda Y, Hosokawa N, Wada I, Nagata K (2003) EDEM as an acceptor of terminally misfolded glycoproteins released from calnexin. Science 299: 1394-1397'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Olivari S, Cali T, Salo KE, Paganetti P, Ruddock LW, Molinari M (2006) EDEM1 regulates ER-associated degradation by accelerating de-mannosylation of folding-defective polypeptides and by inhibiting their covalent aggregation. Biochem Biophys Res Commun 349: 1278-1284\\n\\nOlivari S, Galli C, Alanen H, Ruddock L, Molinari M (2005) A Novel Stress-induced EDEM Variant Regulating ER-associated Glycoprotein Degradation. J Biol Chem 280: 2424-2428\\n\\nOlivari S, Molinari M (2007) Glycoprotein folding and the role of EDEM1, EDEM2 and EDEM3 in degradation of folding-defective glycoproteins. FEBS Lett 581: 3658-3664'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Pastore N, Blomenkamp K, Annunziata F, Piccolo P, Mithbaokar P, Maria Sepe R, Vetrini F, Palmer D, Ng P, Polishchuk E et al (2013) Gene transfer of master autophagy regulator TFEB results in clearance of toxic protein and correction of hepatic disease in alpha-1-anti-trypsin deficiency. EMBO Mol Med 5: 397-412\\n\\nReggiori F, Molinari M (2022) ER-phagy: mechanisms, regulation and diseases connected to the lysosomal clearance of the endoplasmic reticulum. Physiol Rev 102: 1393-1448\\n\\nRudinskiy M, Molinari M (2023) ER-to-lysosome-associated degradation in a nutshell: mammalian, yeast, and plant ER-phagy as induced by misfolded proteins. Febs Letters: 1928-1945'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Sifers RN, Brashears-Macatee S, Kidd VJ, Muensch H, Woo SL (1988) A frameshift mutation results in a truncated alpha 1-antitrypsin that is retained within the rough endoplasmic reticulum. J Biol Chem 263: 7330-7335\\n\\nSmith MD, Harley ME, Kemp AJ, Wills J, Lee M, Arends M, von Kriegsheim A, Behrends C, Wilkinson S (2018) CCPG1 Is a Non-canonical Autophagy Cargo Receptor Essential for ER-Phagy and Pancreatic ER Proteostasis. Developmental Cell 44: 217-232\\n\\nSokolowska I, Pilka ES, Sandvig K, Wegrzyn G, Slominska-Wojewodzka M (2015) Hydrophobicity of protein determinants influences the recognition of substrates by EDEM1 and EDEM2 in human cells. BMC Cell Biol 16: 1-18'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Sugimoto T, Ninagawa S, Yamano S, Ishikawa T, Okada T, Takeda S, Mori K (2017) SEL1L-dependent Substrates Require Derlin2/3 and Herp1/2 for Endoplasmic Reticulum-associated Degradation. Cell Struct Funct 42: 81-94\\n\\nSun Y, Wang X, Yang X, Wang L, Ding J, Wang CC, Zhang H, Wang X (2023) V-ATPase recruitment to ER exit sites switches COPII-mediated transport to lysosomal degradation. Dev Cell 58: 2761-2775\\n\\nSun Z, Brodsky JL (2019) Protein quality control in the secretory pathway. The Journal of cell biology 218: 3171-3187\\n\\nSuzuki H, Tabata K, Morita E, Kawasaki M, Kato R, Dobson RCJ, Yoshimori T, Wakatsuki S (2014) Structural Basis of the Autophagy-Related LC3/Atg13 LIR Complex: Recognition and Interaction Mechanism. Structure 22: 47-58'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Teckman JH, Perlmutter DH (2000) Retention of mutant alpha(1)-antitrypsin Z in endoplasmic reticulum is associated with an autophagic response. American Journal of Physiology-Gastrointestinal and Liver Physiology 279: G961-G974\\n\\nTomkinson B (2019) Tripeptidyl-peptidase II: Update on an oldie that still counts. Biochimie 166: 27-37\\n\\nUshioda R, Hoseki J, Araki K, Jansen G, Thomas DY, Nagata K (2008) ERdj5 is required as a disulfide reductase for degradation of misfolded proteins in the ER. Science 321: 569-572\\n\\nVallee F, Karaveg K, Herscovics A, Moremen KW, Howell PL (2000) Structural basis for catalysis and inhibition of N-glycan processing class I alpha 1,2-mannosidases. J Biol Chem 275: 41287-41298'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Figure 1-Proteasomal and lysosomal pathways for clearance of misfolded proteins from the ER. A ER-associated degradation (ERAD) pathways rely on mannose processing by EDEM proteins that interrupts futile folding cycles in the CNX chaperone system and engages mannose-binding lectins (OS9). Retro-translocation machineries (Dislocons) built around membrane-embedded E3 ubiquitin ligases ensure client-specific dislocation across the ER membrane and poly-ubiquitylation that precedes proteasomal degradation. B ER-to-lysosome-associated degradation (ERLAD) pathways. For ATZ and pro-collagen, selection for ERLAD relies on cycles of de-/re-glucosylation that prolong CNX binding and lead to association with the FAM134B (Forrester et al., 2019; Fregno et al., 2018;'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='with the FAM134B (Forrester et al., 2019; Fregno et al., 2018; Fregno et al., 2021). ERLAD clients are eventually cleared in endolysosomes or in autolysosomes, where they are delivered via mechanistically distinct pathways shown with arrows 1-3.'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Figure 2-The ERAD client NHK is re-directed to LAMP1-positive endolysosomes for degradation upon pharmacologic inactivation of ERAD. (A) HEK239 cells transiently expressing the 35S-NHK are chased for the indicated time without (lanes 1-3) or with PS341 (lanes 4, 5), PS341/BafA1 (lanes 6, 7), KIF (lanes 8, 9), or KIF/BafA1 (10-11). Radiolabeled NHK is immunoisolated at the end of the indicated chase time and separated in SDS-PAGE. Densitometric quantification of the 35S-NHK band in this representative experiment is shown. (B) Quantification of 35S-NHK after a 10- or 120-min chase in cells mock treated or exposed to PS341 or PS341/BafA1. Individual data points at 120-min (%, normalized to 10’ mock): mock= 49, 37 and 49%; PS341= 73, 55 and 52%; PS341/BafA1='), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='mock= 49, 37 and 49%; PS341= 73, 55 and 52%; PS341/BafA1= 91, 63 and 68%. (C) Same as (B) for Kif, KIF/BafA1. Individual data points at 120-min (%, normalized to 10-min mock): mock= 49, 37 and 49%; KIF= 80, 53 and 54%; KIF/BafA1= 99, 64 and 83%. (D) BafA1 does not inhibit clearance of the ERAD client NHK. n=3 for 10- and 120-min; n=2 for 120-min BafA1. (E) ATZ, a canonical ERLAD client accumulates within LAMP1-positive endolysosomes in 3T3 cells exposed to BafA1. (F) The ERAD client NHK is not delivered within LAMP1-positive endolysosomes. (G-H) NHK is delivered and accumulates within endolysosomes inactivated with BafA1 upon ERAD inhibition with KIF (G) or with PS341 (H). (I) LysoQuant quantification of ATZ delivery within LAMP1-positive endolysosomes in'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='of ATZ delivery within LAMP1-positive endolysosomes in panel E, and of NHK in panels F-H (n=18, 19, 23 and 13 cells, respectively. (J-N) Analysis of NHK delivery to LAMP1-positive endolysosomes as in (F-H) but with drug incubation of 8h and with additional BafA1 washout conditions (L, N).O LysoQuant quantification of NHK accumulation within LAMP1 positive endolysosomes (panels J-N, n=31, 24, 31, 31 and 33 cells respectively.'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Data Information: (B, C) N=3 independent experiments, mean +/- SEM  is shown. (D) mean +/- SEM  is shown.(I, O) mean is shown. N=3 independent experiments. One-way analysis of variance (ANOVA) and Dunnett’s multiple comparison test, ****P<0.0001. (E-H, J-N) Scale bars: 10M, Insets are shown with white squares.'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Figure 3-The ERAD client NHK is re-directed to endolysosomes upon genetic inactivation of ERAD. A Western blot showing the efficiency of EDEM1 silencing in HEK293 cells. B Delivery of NHK in control cells (upper panels) and in cells with reduced ERAD upon EDEM1 silencing (lower panels). C LysoQuant quantification of B (n=39 and 25 cells, respectively). D Western blot showing the efficiency of OS9.1/OS9.2 silencing. Two different oligos are used to reduce OS9.1/OS9.2 level. Quantification of OS9.1 and OS9.2 levels, normalized on GAPDH, is shown. E Delivery of NHK in control cells (upper panels) and in cells with reduced ERAD upon OS9.1/OS9.2 silencing (lower panels). F LysoQuant quantification of E (n=35, 36 and 18 cells, respectively).'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Data Information: (C, F) Mean is shown. N=3 independent experiments. (C) Unpaired t-test, *** P<0.001. (F) Mean is shown. N=3 independent experiments One-way analysis of variance (ANOVA) and Dunnett’s multiple comparison test, ****P<0.0001, ***P<0.001 (B, E) Scale bars: 10M. Insets are shown with white squares.'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Figure 4-FAM134B drives delivery of the ERAD client NHK into the ERLAD pathway. (A Western blot showing deletion of endogenous FAM134B in MEF and levels of recombinant FAM134B-V5 and FAM134BLIR-V5 upon back-transfection. B NHK does not accumulate in LAMP1-positive endolysosomes in wild type (upper panels) and FAM134B-KO cells (lower panels). C LysoQuant quantification of B (n=17 and 27 cells, respectively). D NHK is delivered in LAMP1-positive endolysosomes upon ERAD inhibition with KIF in wild type MEF. E NHK is not delivered in LAMP1-positive endolysosomes upon ERAD inhibition with KIF in MEF lacking FAM134B. F Back-transfection of FAM134B in FAM134B-KO MEF restores NHK delivery within endolysosomes. G Back-transfection of FAM134BLIR that cannot engage'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='G Back-transfection of FAM134BLIR that cannot engage LC3 fails to restore NHK delivery within endolysosomes. (H-J) Analysis of NHK delivery to LAMP1-positive lysosomes in FAM134B-KO cells as in (E-G), when ERAD is inhibited with PS341. K LysoQuant quantification of D-J (n=39, 35, 37, 39, 31, 30 and 35 cells, respectively).'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Data Information (C) Mean is shown. N=3 independent experiments, Unpaired t-test, nsP> 0.05. (K) Mean is shown. N=2 independent experiments. (B, D-J) Scale bars: 10M. Insets are shown with white squares.\\n\\n\\n\\n\\n\\n\\nFigure 5-The LC3 lipidation machinery is involved, autophagosome biogenesis is dispensable, for delivery of NHK to LAMP1-positive endolysosomes upon ERAD inactivation. (A-C) Same as Figs. 2F-2H, in MEF. (A) The ERAD client NHK is not delivered to LAMP1-positive endolysosomes. NHK is delivered and accumulates within endolysosomes inactivated with BafA1 upon ERAD inhibition with KIF (B) or with PS341 (C).'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='D Western blots showing deletion of ATG7 (left panels) and defective LC3 lipidation in ATG7-KO upon starvation (right panels). (E) NHK does not accumulate in LAMP1-positive endolysosomes in ATG7-KO cells. (F) Western blot showing deletion of ATG13 in MEF. (G) NHK delivery within LAMP1-positive endolysosomes is not perturbed in ATG13-KO MEF. (H ) LysoQuant quantification of A-C, E, G (n=17, 57, 29, 46 and 50 cells, respectively). \\n\\nData Information: (H) Mean is shown. N=3 independent experiments. One-way analysis of variance (ANOVA) and Dunnett’s multiple comparison test, nsP> 0.05, ****P<0.0001. (A-C, E, G) Scale bars: 10M. Insets are shown with white squares.'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Figure 6-The ERAD client BACE457D is re-directed to endolysosomes for degradation upon pharmacologic inactivation of ERAD. (A-C) Same as Figs. 2A-2C for BACE457D. (A) HEK239 cells transiently expressing the 35S- BACE457D are chased for the indicated time without (lanes 1-3) or with PS341 (lanes 4, 5), PS341/BafA1 (lanes 6, 7), KIF (lanes 8, 9), or KIF/BafA1 (10-11). Radiolabelled BACE457D is immunoisolated at the end of the indicated chase time and separated in SDS-PAGE. Densitometric quantification of the 35S-BACE457D band in the representative experiment is shown. (B-C) Quantification of 35S- BACE457D after a 10- or 120-min chase in cells mock treated, exposed to PS341 or PS341/BafA1 (B) or to KIF, KIF/BafA1 (C). Individual data points at 120-min (%,'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='or to KIF, KIF/BafA1 (C). Individual data points at 120-min (%, normalized to 10-min mock): for (B) mock= 36 and 54%; PS341= 70 and 84%; PS341/BafA1= 97 and 99%. For (C): mock= 36 and 54%; KIF= 72 and 75%; KIF/BafA1= 91 and 85%. (D) The ERAD client BACE457D does not accumulate within LAMP1-positive endolysosomes upon BafA1 exposure in MEF cells. (E) BACE457D is delivered and accumulates within endolysosomes upon ERAD inhibition with KIF. (F) Same as E, when ERAD is inhibited with PS341. (G) LysoQuant quantification of D-F (n=33, 37 and 17 cells, respectively). (H) BACE457D is not delivered in LAMP1-positive endolysosomes in MEF lacking FAM134B.'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='(I-N) Same as Fig. 4E-J for BACE457D. Analysis of BACE457D delivery to LAMP1-positive endolysosomes in FAM134B KO cells treated with BafA1 and KIF (I-K) or BafA1 and PS341 (L-N). BACE457D is not delivered to endolysosomes in cells with deficient ERAD. Back-transfection of FAM134B (J, M) but not FAM134BLIR (K, N) restores delivery. (O) LysoQuant quantification of H-N (n=24, 40, 36, 47, 14, 20 and 16 cells, respectively). \\n\\nData Information: (B, C) N=2 independent experiments, mean +/- SEM is shown. (G, O) Mean is shown. N=2 independent experiments. (D-F, H-N) Scale bars: 10M. Insets are shown with white squares.'), Document(metadata={'source': './data/docx/EMBOR-2024-59101V1-T.docx'}, page_content='Figure 7-Schematic representation of how pharmacologic (blue) and genetic (red) ERAD perturbation triggers compensatory ERLAD programs. Defective proteasomal clearance of NHK and BACE457D activates FAM134B-driven ERLAD. Other clients or maintenance of ER homeostasis in specific tissues may activate other ER-phagy receptors (e.g., CCPG1 in the pancreas and FAM134B-2 in the liver).\\n\\n\\n\\n4\\n\\n\\n\\n1')]\n",
      "Processing document: ./data/docx/EMM-2023-18636.docx\n",
      "Split documents: [Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='DUSP6 inhibition overcomes Neuregulin/HER3-driven therapy tolerance in HER2+ breast cancer\\n\\n\\n\\nMajid Momeny1,2§, Mari Tienhaara3,4, Mukund Sharma1,4,  Deepankar Chakroborty3,4, Roosa Varjus1, Iina Takala3,4, Joni Merisaari1, Artur Padzik1, Andreas Vogt5, Ilkka Paatero1, Klaus Elenius3,4, Teemu D. Laajala6, Kari J. Kurppa3,4, Jukka Westermarck1,4,§\\n\\n\\n\\n1Turku Bioscience Centre, University of Turku and Åbo Akademi University, Turku, Finland\\n\\n2The Brown Foundation Institute of Molecular Medicine, McGovern Medical School, The University of Texas Health Science Center at Houston, TX, USA.\\n\\n3Medicity Research Laboratories, Faculty of Medicine, University of Turku, Turku, Finland\\n\\n4Institute of Biomedicine, University of Turku, Turku, Finland'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='4Institute of Biomedicine, University of Turku, Turku, Finland\\n\\n5University of Pittsburgh Drug Discovery Institute, Department of Computational and Systems Biology, Pittsburgh Technology Center, Pittsburgh, PA, USA\\n\\n6Department of Mathematics and Statistics, University of Turku, Turku, Finland\\n\\n\\n\\n\\n\\n§ Co-corresponding authors: jukwes@utu.fi, majid.momeny@uth.tmc.edu\\n\\n\\n\\n\\n\\n\\n\\nAbstract'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Despite clinical benefits of tyrosine kinase inhibitors (TKIs) in cancer, most tumors can reactivate proliferation under TKI therapy. Here we present transcriptional profiling of HER2+ breast cancer cells transitioning from dormant drug tolerant cells to re-proliferating cells under continuous HER2 inhibitor (HER2i) therapy. Focusing on phosphatases, expression of dual-specificity phosphatase DUSP6 was found inhibited in dormant cells, but strongly induced upon re-growth. DUSP6 expression also selectively associated with poor patient survival in HER2+ breast cancers. DUSP6 overexpression conferred apoptosis resistance, whereas its pharmacological blockade prevented therapy tolerance development under HER2i therapy. DUSP6 targeting also synergized with'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='under HER2i therapy. DUSP6 targeting also synergized with clinically used HER2i combination therapies. Mechanistically DUSP6 is a positive  regulator of HER3 expression, and its impact on HER2i tolerance was mediated by neuregulin-HER3 axis. In vivo, genetic targeting of DUSP6 reduced tumor growth in brain metastasis model, whereas its pharmacological targeting induced synthetic lethal therapeutic effect in combination with HER2i. Collectively this work demonstrates that DUSP6 drives escape from HER2i-induced dormancy, and that DUSP6 is a druggable target to overcome HER3-driven TKI resistance.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Keywords: Non-genetic drug tolerance, Lapatinib, Neratinib, BCI, FOXM1\\n\\n\\n\\n\\n\\n\\nThe paper explained:\\n\\n\\n\\nProblem\\n\\nThe molecular mechanisms for the acquisition of resistance to HER2-targeted therapies in breast cancer are still elusive, especially when the drug-tolerant persister cells start to regrow under the treatment. \\n\\nResults'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Results\\n\\nThe oncogenic phosphatase DUSP6 is increased in the drug-tolerant emerging persister cells and plays central roles in the acquisition of HER2i resistance. Moreover, DUSP6 blockade potentiates therapeutic sensitivity in the primary resistant models via inhibition of the HER3 signaling pathway. Inhibition of DUSP6 offers advantages over AKT blockade, including induction of apoptotic cell death, preempting ligand-induced rescue, and inhibition of HER3. Ultimately, DUSP6 inhibition reverses HER2i resistance in vivo and reduces the outgrowth of HER2+ cell in the brain metastasis models.\\n\\nImpact'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Impact\\n\\nThese findings have implications in the clinical management of HER2+ breast cancer and warrant development and clinical investigation of DUSP6 inhibitors in combination with HER2-directed therapies in the patients.\\n\\n\\n\\n\\n\\n\\n\\n\\nIntroduction'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='To develop therapeutic resistance, tumor cells undergo distinct evolutionary stages, starting with induction of dormancy and non-genetic drug tolerance, followed by epigenetic changes and finally resistance-conferring genetic mutations  (De Conti et al, 2021; Hata et al, 2016; Marine et al, 2020). These different phases were originally demonstrated for EGFR-targeted therapies in non-small cell lung cancer (NSCLC) cells (Sharma et al, 2010), but the concept has been expanded more recently to other malignancies including HER2+ breast cancer (Chang et al, 2022; De Conti et al., 2021; Dhimolea et al, 2021; Hata et al., 2016; Kurppa et al, 2020; Sharma et al., 2010). Based on these studies, there is ample of omics data from dormant cells (also called as drug'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='is ample of omics data from dormant cells (also called as drug tolerant persisters; DTPs) from different cancer types treated with variety of therapies. However, our understanding of the molecular mechanisms behind the regrowth of DTPs under continuous therapy is still rudimentary. Especially, to our knowledge there are no published studies describing transcriptional landscapes of transition from DTP to drug tolerant expanding cells (DTEP), or transition of DTEP cells to long term resistant (LR) cells upon TKI therapies.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='The human epidermal growth factor receptor 2 (HER2; encoded by ERBB2) is overexpressed in 15-25% of human breast cancers and associates with a poor patient survival (Arteaga & Engelman, 2014; Haikala & Janne, 2021). HER2 belongs to the ERBB family of receptor tyrosine kinase (RTK) with four members: HER1 (EGFR), HER2, HER3 and HER4. Upon ligand binding, the ERBB receptors homo-and heterodimerize and activate downstream signaling pathways including PI3K/AKT and RAS/MAPK/ERK, which regulate cell proliferation, survival and the metastatic dissemination (Arteaga & Engelman, 2014; Haikala & Janne, 2021). Multiple HER2-targeted therapies, including the monoclonal antibody trastuzumab and small molecule tyrosine kinase inhibitors (TKIs) have been approved for the'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='tyrosine kinase inhibitors (TKIs) have been approved for the treatment of HER2-overexpressing (HER2+) breast cancer (Goutsouliak et al, 2020). Application of anti-HER2 agents in combination with chemotherapy has significantly improved the patients’ outcome. However, patients initially responsive to the HER2 inhibitors (HER2is) almost inevitably succumb to disease relapse (Goutsouliak et al., 2020). Moreover, HER2+ breast tumors have an inherent tendency to develop brain metastasis, a significant clinical challenge for the treatment of these patients (Fecci et al, 2019). Therefore, there is a pressing need for novel and more efficacious therapeutic strategies to overcome resistance to HER2is.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='HER3 is an obligate heterodimerization partner for HER2 and plays essential roles in HER2-driven tumorigenesis, and resistance to HER2is (Haikala & Janne, 2021; Wilson et al, 2012). Consistent with the effects of HER3 overexpression, the HER3 ligand neuregulin (NRG, a.k.a Heregulin; HRG) promotes trastuzumab resistance in HER2+ breast cancer cells (Haikala & Janne, 2021). Importantly, the NRG-HER3 axis also promotes resistance to a wide range of TKIs and chemotherapies (Erjala et al, 2006; Haikala & Janne, 2021; Knuefermann et al, 2003; Recondo et al, 2020; Wilson et al., 2012; Yonesaka et al, 2011). Despite the importance of HER3 in cancer progression and therapy resistance, development of HER3 small molecule inhibitors has been challenging due to its'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='HER3 small molecule inhibitors has been challenging due to its impaired kinase activity (Haikala & Janne, 2021; Xie et al, 2014). Moreover, the clinical activity of HER3 monoclonal antibodies either as monotherapies or in combination with chemo- and targeted therapies have been marginal (Cleary et al, 2017; Haikala & Janne, 2021; Schneeweiss et al, 2018). To this end, there is a pressing need to identify novel strategies to inhibit HER3 activity and/or expression for the treatment of HER3-dependent human malignancies (Gaborit et al, 2015; Haikala & Janne, 2021; Xie et al., 2014).'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='There is emerging evidence that phosphatases are novel and “druggable” targets in oncology (Lazo et al, 2018; Vainonen et al, 2021). Inhibition of oncogenic phosphatases, or re-activation of tumor suppressor phosphatases, by small molecule therapies halt tumor growth, retard malignant progression, and enhance therapeutic sensitivity in various neoplasms (Lazo et al., 2018; Vainonen et al., 2021). Despite this, the contribution of phosphatases to resistance to HER2is is still poorly understood. Dual-specificity phosphatases (DUSPs) belong to the superfamily of protein tyrosine phosphatases and dephoshorylate both tyrosines and serines or threonines. A subgroup of the DUSPs are mitogen-activated protein kinase (MAPK) phosphatases that selectively interact'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='protein kinase (MAPK) phosphatases that selectively interact with and dephosphorylate the MAPKs (Patterson et al, 2009; Zandi et al, 2022). For instance, DUSP6 displays a high degree of substrate selectivity for the extracellular signal-regulated kinase (ERK), but not P38 or c-Jun N-terminal kinase (JNK) (Zandi et al., 2022). However, recent studies indicate that at least some cancer relevant DUSP6 functions may by ERK-independent (Kong et al, 2023). DUSP6 is indicated in clinical cancer progression, and its genetic inhibition prevents tumor cell growth  (Shojaee et al, 2015; Wu et al, 2018). Notably, DUSP6 is a druggable phosphatase (Korotchenko et al, 2014; Molina et al, 2009; Vainonen et al., 2021; Zandi et al., 2022). The best characterized DUSP6'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='et al., 2021; Zandi et al., 2022). The best characterized DUSP6 inhibitor molecule BCI ((E)-2-Benzylidene-3-(cyclohexylamino)-2,3-dihydro-1H-inden-1-one), is a semi-allosteric inhibitor of both DUSP1 and 6 that phenocopies genetic DUSP6 inhibition in several cancer models  (Kong et al., 2023; Korotchenko et al., 2014; Molina et al., 2009; Vainonen et al., 2021). However, the role and potential of therapeutic targeting of DUSP6 in overcoming resistance to HER2is is currently unknown.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Here we present transcriptional analysis of HER2i treated HER2+ cancer cells upon 9 months of continuous HER2i treatment. In addition to revealing first global gene expression programs associated with the DTP-DTEP and DTEP-LR therapy tolerance transitions, we use complementary genetic and pharmacological approaches to demonstrate that DUSP6 has critical role in regrowth of DTEP cells under HER2i therapies. Mechanistically, DUSP6 drives non-genetic HER2i tolerance via regulation of HER3 expression, and by abrogating neuregulin-elicited apoptosis resistance. Collectively our findings provide a strong pre-clinical rationale to further advance in DUSP6 blockade for HER3 targeting in general, and especially for the clinical management of HER2+ breast cancer'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='especially for the clinical management of HER2+ breast cancer patients with resistance to HER2i.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Results\\n\\n\\n\\nDevelopment of acquired HER2i resistance by long-term treatment of drug sensitive HER2+ breast cancer cells with lapatinib'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='To model the full range of development of HER2i resistance starting from primary sensitive phase, via drug-tolerant persister (DTP) development, and re-emergence of proliferative drug-tolerant expanded persister (DTEP) cells, to long-term resistant cells (LR), the HER2i sensitive cell line BT474, and its brain seeking variant BT474Br (Zhang et al, 2013), were exposed to therapeutically relevant 1 M of lapatinib every 3 days for up to 9 months  (Fig. 1A). Both cell lines followed a similar pattern of lapatinib tolerance development, where at the 9-day timepoint only a few DTP cells could be microscopically observed, whereas the emergence DTEP population took about 6 months (Fig. 1A). Following this, the plates were fully populated by the LR cells after 9'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='this, the plates were fully populated by the LR cells after 9 months of continuous lapatinib treatment (Fig. 1A). Importantly, in addition to lapatinib, the LR clones of both BT474 and BT474Br displayed strong cross resistance to tucatinib (a HER2i), afatinib (a HER2/EGFR inhibitor), and neratinib (a HER2/HER4/EGFR inhibitor) (Figure EV1).  This indicates that the acquired resistance is not specific to lapatinib but is driven by a mechanism that is generally relevant to the ERBB family of RTKs.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Transcriptomic landscape of acquired lapatinib resistance in BT474 cells'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Recent studies have focused on the molecular characterization of DTP cells in response to kinase inhibitor therapies (Chang et al., 2022; Kurppa et al., 2020; Marsolier et al, 2022) but there is no published information about transcriptional profiles of TKI treated cells at the DTP-DTEP or DTEP-LR transitions. To this end, the transcriptional profiles from each functional state of lapatinib drug tolerance and resistance development in BT474 cells were surveyed by bulk RNA-sequencing. From three technical replicates per condition, and by using statistical criteria of |logFC|>2 and Benjamini-Hochberg adjusted p <0.05, upregulation of 144, 1169 and 16 genes was found upon the control-DTP, DTP-DTEP, and DTEP-LR transitions of BT474 cells, respectively (Fig. 1B,'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='and DTEP-LR transitions of BT474 cells, respectively (Fig. 1B, Dataset EV1). On the other hand, the number of downregulated genes were 517, 930 and 28, respectively (Fig. 1B, Dataset EV1). The highest number of differentially regulated genes upon the DTP-DTEP transition indicates that this is the transition phase where the BT474 cell fate is most robustly impacted during the resistance development. When assessing the patterns of gene expression changes by unsupervised soft clustering analysis (Futschik & Carlisle, 2005), we identified six approximately similar size gene clusters with distinct regulation patterns (Appendix Figure S1A). The genes included in these clusters are listed in the Dataset EV2. Such regulation patterns indicate that neither gene'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='EV2. Such regulation patterns indicate that neither gene activation nor gene repression characterize lapatinib resistance development in BT474 cells, but unique gene expression programs are involved in each of these steps.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Gene regulatory mechanisms and hallmarks associated with HER2i resistance transitions'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='To understand gene regulatory mechanisms controlling transitions between the different phases on lapatinib tolerance and resistance development, we predicted the transcription factor binding sites enriched on promoter regions of the differentially regulated genes (Fig. 1B, Dataset EV3). Using FDR < 0.05 as a cut-off, the most highly enriched transcription factor elements in genes downregulated upon the control-DTP transition were E2F4, FOXM1, FLI1, E2F1 and NFYA (Fig. 1B, Dataset EV3). Strikingly, most of these transcription factor binding sites were enriched also in the genes significantly upregulated in DTEP cells (Fig. 1B in bold, Dataset EV3). This indicates that these transcription factors are inactivated co-ordinately upon the development of the DTP'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='are inactivated co-ordinately upon the development of the DTP state and re-activated upon re-growth of the DTEPs. E2F4, FOXM1 and E2F1 are all indicated in development and progression of breast cancer.  We validated inhibition of selected BT474 cell DTP FOXM1 and E2F target genes across different HER2+ cells treated with lapatinib to reach the DTP state (Figure EV2).  On the other hand, binding sites for transcription factors CBX2, ESR1, TCF12 and ZNF217 were found significantly enriched in the genes downregulated upon DTP to DTEP transition, whereas STAT1 and STAT2 were enriched among the genes upregulated in LR cells as compared to DTEPs (Fig. 1B, Dataset EV3).'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='To identify cancer hallmark processes involved in each transition phase, the entire transcriptomics data was re-analysed by Gene Set Variation Analysis (GSVA). Consistent with recent evidence from other DTP models (Chang et al., 2022; Dhimolea et al., 2021; Kurppa et al., 2020), MYC signaling was inhibited in the lapatinib-treated BT474 DTPs, but reactivated in DTEPs (Fig. 1C, gene sets 3 and 6).  Similar gene regulation pattern was observed for E2F1 targets, G2/M checkpoint, mTOR signaling, and androgen response (Fig. 1C, gene set 3), ROS signaling, oxidative phosphorylation, and DNA repair (Fig. 1C, gene set 6). Interestingly, the DTP-DTEP transition was associated also with downregulation of several cancer hallmark gene sets, most apparently seen in set'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='several cancer hallmark gene sets, most apparently seen in set 4 where KRAS signaling, EMT, WNT/ß-catenin, and inflammatory response genes were all suppressed upon proliferation reactivation (Fig. 1C). Additional hallmark gene sets regulated between the lapatinib resistance development transitions are displayed in appendix figure S1B.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Collectively these results demonstrate that unique gene clusters and biological processes are involved in each step of lapatinib resistance development. This bulk RNA-sequencing data provides a rich resource for future studies of gene regulatory mechanisms in HER2i tolerance and resistance development. However,  it is clear that future single cell RNA sequencing studies are needed to understand clonality of gene expression changes induced during these transitions. \\n\\n\\n\\n\\n\\nPhosphatase gene expression landscape in DTPs and DTEPs'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Recent data indicates that development of cancer therapy resistance is initiated by non-genetic signaling rewiring mediated by post-translational regulation of intracellular signaling pathways (De Conti et al., 2021; Hata et al., 2016; Marine et al., 2020). Protein phosphorylation is the most prevalent post-translational modification in cancer cells, and cancer cell phosphoproteomes are regulated by kinases and phosphatases. Whereas the role of kinases in non-genetic therapy tolerance development has been extensively studied (Marine et al., 2020), the importance of phosphatases to development of non-genetic kinase inhibitor therapy tolerance have been thus far poorly characterized. Therefore, we focused on the dynamics of phosphatase gene regulation during'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='focused on the dynamics of phosphatase gene regulation during lapatinib tolerance development. Importantly, there were only four phosphatase genes (CDC25A, CDC25C, DUSP6 and SYNJ1) that were synchronously downregulated in DTP cells but upregulated in DTEPs versus DTPs (Fig. 2A-C, Dataset EV1). We rationalized that these four phosphatases might be particularly relevant for allowing the regrowth of the DTEP population under continuous lapatinib treatment. Even though a large group of other genes were found differentially regulated between the DTEP and LR populations (Fig. 1B, Dataset EV1 and EV2), none of the phosphatase genes were significantly regulated in the DTEP-LR transition (Dataset EV1). This indicates that regulation of phosphatase gene expression'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='This indicates that regulation of phosphatase gene expression could be primarily relevant to the early non-genetic phases of acquired lapatinib resistance.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Clinical association of DUSP6 with poor prognosis HER2+ breast cancer'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Based on the above analysis, CDC25A, CDC25C, DUSP6 and SYNJ1 were the only phosphatases that were significantly regulated during both therapy tolerance transitions (Fig. 2A-C). To evaluate potential clinical relevance of the selected four phosphatases in human breast cancer, we first examined their expression levels across breast cancer subtypes in the METABRIC dataset (Dataref:(Pereira et al, 2016). Interestingly, DUSP6 was the only one of these four phosphatases that was selectively overexpressed in the target HER2+ subtype of breast cancers (Fig. 2D). The closest functional orthologue for DUSP6, DUSP1 did not show HER2+ selective overexpression (Appendix Figure S2A). DUSP6 overexpression in HER2+ breast cancer was confirmed in the TCGA breast invasive'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='HER2+ breast cancer was confirmed in the TCGA breast invasive carcinoma dataset  (Dataref: (Cerami et al, 2012; de Bruijn et al, 2023; Gao et al, 2013)(Appendix Figure S2B). Further, when the 1082 breast invasive carcinoma samples from the TCGA BRCA-dataset (Dataref: (Cerami et al., 2012; de Bruijn et al., 2023; Gao et al., 2013) were divided to DUSP6high and DUSP6low groups based on their DUSP6 mRNA expression levels, HER2+ and luminal A subtypes were clearly enriched among the DUSP6high samples (Fig. 2E). HER2 positivity was also found enriched among DUSP6high tumors based on HER2 immunohistochemistry of samples available for staining in the Breast Invasive Carcinoma (TCGA, Firehose legacy) dataset (Dataref: (Cerami et al., 2012; de Bruijn et al., 2023;'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='dataset (Dataref: (Cerami et al., 2012; de Bruijn et al., 2023; Gao et al., 2013) (Appendix Figure S2C).'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='To study the prognostic value of DUSP6 expression in HER2+ breast cancers from the TCGA BRCA-dataset, the patients with high ERBB2 (gene coding for HER2) expression were further divided into DUSP6high and low groups. Comparison of the survival outcomes between the two groups indicate that high DUSP6 expression predicts poor overall and disease-free survival among HER2+ patients (Fig. 2F,G). In multivariable analysis of high ERBB2 expressing tumors, increase in DUSP6 expression and large tumor size (T4) remained significant independent prognostic factors (Appendix Figure S2D). However, DUSP6 mRNA expression was neither a prognostic factor in Luminal B or Basal subtypes, nor across unselected breast cancer patient population (Appendix Figure S2E), further'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='breast cancer patient population (Appendix Figure S2E), further highlighting the selective connection between DUSP6 and HER2 in breast cancer progression.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Interestingly, luminal B cancers were clearly enriched in DUSP6low samples as compared to DUSP6high (Fig. 2E). Luminal B are estrogen receptor (ER) positive cancers that often also express HER2. Clinically luminal B cancers are less aggressive than HER2+ cancers. This data may suggest that DUSP6 has a role in suppressing ER positivity, and thereby increasing the relative numbers of HER2+ cancers over luminal B cancers. Although the details of this regulation remain unclear, this hypothesis is supported by strongly decreased expression of ER protein and ESR1 mRNA, coding for ER, in HER2+ DUSP6high samples as compared to HER2+ DUSP6low samples (Appendix Figure S2F,G).'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='These data demonstrate that DUSP6 is clinically associated with the aggressive HER2+ breast cancer but may also have a broader role in defining breast cancer subtype development.  \\n\\n\\n\\nDUSP6 promotes the HER2i tolerance and DTP-DTEP transition under continuous lapatinib treatment'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Clinical relevance of DUSP6 in HER2+ breast cancers among the four candidate phosphatases differentially regulated at DTP-DTEP transition, together with the feasibility of DUSP6 targeting by small molecules (Kong et al., 2023; Korotchenko et al., 2014; Molina et al., 2009; Vainonen et al., 2021; Zandi et al., 2022), motivated us to select DUSP6 as the phosphatase to be focused in this study. We confirmed differential expression of DUSP6 across different resistance acquisition transitions in BT474 cells. Consistent with the RNA sequencing results (Fig. 2C), DUSP6 protein expression was strongly induced upon the DTP-DTEP transition (Fig. 3A). While the DUSP6 mRNA levels were diminished in the fully resistant LR clones compared to DTEPs (Fig. 2C), its protein'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='resistant LR clones compared to DTEPs (Fig. 2C), its protein levels remained robustly elevated presumably via post-translational stabilization mechanisms (Fig. 3A). Validating that these effects were not cell line specific, DUSP6 protein was also increased in BT474BrLR cells as compared to the parental cells (Appendix Figure 3A). We further validated inhibition of DUSP6 upon DTP phase across independent set of lapatinib treated HER2+ cells including BT474Br, and two cell lines (EFM192A, HCC1419) from (Data ref:(Chang et al., 2022)( GSE155342), or NSCLC, melanoma, and colorectal cancer cells de novo treated with various TKIs (Fig. 3B). Therefore, inhibition of DUSP6 expression appears as a general mechanism associated with establishment of TKI-induced'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='general mechanism associated with establishment of TKI-induced therapy tolerance. Selected E2F1 and FOXM1 target genes were also confirmed to be downregulated in TKI-treated NSCLC, MM, and CRC DTP cells (Appendix Figure S3B). Notably, among the transcription factors differentially implicated upon HER2i tolerance development (Fig. 1B), forkhead box transcription factor M1\\xa0 (FOXM1) and NFYA bind to DUSP6 promoter (Dataset EV4), and coinciding with DUSP6 expression, FOXM1 gene expression is downregulated in DTPs and upregulated in DTEPs (Appendix Figure S3C). Functionally, a small molecule inhibitor of FOXM1 (FDI-6) \\xa0(Wang et al, 2021) inhibited DUSP6 expression in a time-dependent fashion in BT474 cells (Appendix Figure S3D,E). Together with its role as'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='BT474 cells (Appendix Figure S3D,E). Together with its role as breast cancer oncogene involved in therapy resistance (Zhang et al, 2021), these data imply FOXM1 as a viable candidate inducing DUSP6 expression during the DTP-DTEP transition.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='To evaluate whether the hallmark gene sets co-regulated with DUSP6 during therapy tolerance transitions (Fig. 1C) could be functionally downstream of DUSP6, we performed a RNAseq analysis from DUSP6 depleted MDA-MB-453 cells and compared these two gene sets. Notably, there was a marked overlap between hallmark gene sets from these two conditions in which DUSP6 expression was suppressed (Fig. 1C versus Fig. 3C; overlapping gene sets highlighted in red). Especially interesting finding was that the DUSP6 knockdown cells displayed a gene expression pattern linked to dormant cancer cells such as inhibition of MYC, E2F1 targets, and the PI3K/AKT/mTOR signaling, as well as activation of the interferon response (Chang et al., 2022; Dhimolea et al., 2021; Kurppa et'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='response (Chang et al., 2022; Dhimolea et al., 2021; Kurppa et al., 2020; Marine et al., 2020). The finding that siRNA-mediated DUSP6 depletion (Fig. 3C) recapitulates the gene expression profile in the lapatinib-induced DTP cells (Fig. 1C), clearly indicates that inhibition of DUSP6-driven gene expression programs functionally contribute to the HER2i-elicited growth inhibition.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='To functionally validate that increased DUSP6 expression in DTEP cells contributes to their survival, we ectopically overexpressed DUSP6 in BT474 cells, and subjected the cells to treatment with lapatinib, neratinib, afatinib, or tucatinib. Importantly, DUSP6 overexpression was able to dampen both cell viability inhibition, and apoptosis induction, by all four tested HER2is (Fig. 3D,E and Figure EV3A-D). Mechanistically, the anti-apoptotic activity of DUSP6 in lapatinib treated BT474 cells appeared to be independent of ERK MAPK regulation, as overexpression of the KIM mutant of DUSP6 (R64A,R65A), incapable in ERK binding (Nichols et al, 2000), had equally strong impact as the wild-type DUSP6 (Fig. 3F and Appendix Figure S3F). ERK-independent antiapoptotic'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='3F and Appendix Figure S3F). ERK-independent antiapoptotic activity for DUSP6 in cancer cells was suggested also recently (Kong et al., 2023). As a complementary pharmacological  approach, BT474 cells were treated with lapatinib or neratinib alone, or in combination with small molecule DUSP6 inhibitor BCI for 6 months. BCI is a semi-allosteric inhibitor of both DUSP1 and DUSP6 and several studies have demonstrated that BCI phenocopies genetic DUSP6 inhibition in cancer (Kong et al., 2023; Ramkissoon et al, 2019; Shojaee et al., 2015). Notably, as compared to the monotherapies, combination with BCI preempted the DTEP development in both lapatinib and neratinib treated cells (Fig. 3G). Strongly indicative of selective drug interaction rather than overall'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='indicative of selective drug interaction rather than overall toxicity by BCI, the BCI used at the given concentration for 6 months did not kill all BT474 cells but potently synergized with the HER2is (Fig. 3G). The fact that DUSP6 overexpression inhibited HER2i-elicited cell killing (Fig. 3E), whereas BCI abrogated development of HER2i tolerant cells (Fig. 3G), strongly support selective effects for BCI on DUSP6. Indeed, the role of DUSP1 as a primary BCI target was overruled by the results that  DUSP1 was neither expressed at the protein level in BT474 cells (Fig. 3A), nor its mRNA was found differentially regulated between any of the acquired resistance phases (Fig. 2A, B, Dataset EV1). In harmony, DUSP1 overexpression had clearly weaker activity than'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='harmony, DUSP1 overexpression had clearly weaker activity than DUSP6 overexpression in the rescue experiments (Figure EV3D). To further validate the selectivity of BCI as a DUSP6 inhibitor, we generated DUSP6 knockout (DUSP6KO) MDA-MB-453 cells by CRISPR/CAS9 (Appendix Figure S3G). Indeed, three independent single cell clones of DUSP6KO cells were significantly less sensitive to BCI-elicited inhibition of cell viability as compared to the control cells (Appendix Figure S3H).'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Collectively, these results identify DUSP6 downregulation to be functionally relevant for establishment of the DTP phase, whereas its transcriptional induction contributes to DTP-DTEP transition in lapatinib treated HER2+ cells. \\n\\n\\n\\nDUSP6 targeting kills HER2i resistant breast cancer cells and synergizes with HER2i combination therapies'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='After discovering the role for DUSP6 in the development of HER2i tolerance, we wanted to address its role in HER2+ breast cancer cells with stable HER2i resistance. To this end, we compared pharmacological DUSP6 targeting against a library of available small molecule modulators of other phosphatases (Lazo et al., 2018; Vainonen et al., 2021). Across either HER2i resistant cells lines, inhibition of cell viability by phosphatase targeting was observed only with DUSP6 inhibitor BCI, its derivative BCI-215, and with FTY-720 that reactivates protein phosphatase 2A (PP2A) by SET inhibition (Saddoughi et al, 2013)(Figure EV4). The phenocopying results between BCI and BCI-215 is an important additional evidence for the selectivity of BCI type of drugs towards'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='evidence for the selectivity of BCI type of drugs towards DUSPs, as BCI-215 was demonstrated to  activate only the DUSP1 and 6 target MAPKs among the 43 tested kinases (Chan et al, 2020; Kaltenmeier et al, 2017). Based on the response to kinase inhibitors and anti-apoptotic antagonists, the cells that are HER2i resistant, but sensitive to DUSP6 inhibition, are co-dependent on PI3K/AKT, PLK1 and AXL kinase activities, and on the anti-apoptotic proteins IkB, survivin, cIAP and/or XIAP (Figure EV4). Further corroborating the role for DUSP6 as the anti-apoptotic target in stably HER2i resistance cells, siRNA-mediated depletion of DUSP6 induced apoptosis in MDA-MB-453 cells (Fig. 4A). This was not seen with DUSP1 inhibition (Fig. 4A). DUSP6 depletion also'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='not seen with DUSP1 inhibition (Fig. 4A). DUSP6 depletion also induced apoptosis in another HER2i resistant HER2+ cell line MDA-MB-361 (Appendix Figure S4A). Furthermore, independent clones of CRISPR/CAS9 targeted MDA-MB-453 DUSP6 KO cells showed impaired long-term colony growth potential (Fig. 4B). This was due to loss of DUSP6 expression, as lentiviral re-expression of DUSP6 rescued the phenotype (Appendix Figure S4B-E).'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Notably, the genetic DUSP6 targeting also sensitized MDA-MB-453 cells to several HER2 targeting approaches. Indeed, whereas the parental MDA-MB-453 cells were resistant to the clinically relevant concentrations of lapatinib, neratinib and trastuzumab, this resistance was abrogated  in the DUSP6-siRNA targeted cells (Fig. 4C, Appendix Figure S4F). These results also validate that DUSP6 inhibition sensitizes to HER2 inhibition regardless of whether small molecule inhibitors or therapeutic antibody (trastuzumab) is used. The impact of genetic DUSP6 targeting in HER2i sensitization was recapitulated by BCI treatment (Fig. 4D). Additionally, DUSP6 depletion increased sensitivity to the combination of neratinib and capecitabine (Fig. 4E), which is a combination'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='of neratinib and capecitabine (Fig. 4E), which is a combination therapy in clinical use (Saura et al, 2020). Furthermore, DUSP6 depletion enhanced sensitivity to the combination of tucatinib+trastuzumab+capecitabine (Appendix Figure S4G), which improves progression free survival and overall survival in patients with HER2+ metastatic breast cancer (Murthy et al, 2020).'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Together with DUSP6 overexpression experiments in HER2i sensitive cells (Fig. 3G-I), these data provide strong evidence that DUSP6 contributes to the HER2i resistance both in monotherapy and combination therapy settings. \\n\\n\\n\\nDUSP6 inhibition overcomes HER2 inhibitor resistance in vivo'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='To validate the in vivo relevance of the results, we used both genetic and pharmacological targeting of DUSP6 in HER2i resistant xenograft models. To start with, we evaluated the impact of CRISPR/CAS9-mediated DUSP6 knockout on the    xenograft growth of MDA-MB-453 cells in immunocompromised BALB/cOlaHsd-Foxn1nu mice. Importantly, the two DUSP6 KO clones showed significant and indistinguishable antitumor effects as compared to the control cells (Fig. 5A). Next, we asked whether pharmacological DUSP6 blockade overcomes HER2i resistance in vivo. For this purpose, we used two different HER2+ cell models, MDA-MB-453 or HCC1954. When xenografts with these two cell lines had reached the approximate size of 100 mm3, the mice were randomized into four treatment'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='size of 100 mm3, the mice were randomized into four treatment groups; vehicle, lapatinib/neratinib (50 mg/kg), BCI (50 mg/kg), and lapatinib/neratinib+BCI. Importantly, validating the in vivo HER2i resistance of both chosen HER2+ cell models, tumors from both cell lines were fully resistant to clinically relevant doses of either lapatinib or neratinib (Fig. 5B-E). Notably, both MDA-MB-453 and HCC1954 tumors also displayed strong resistance to BCI monotherapy (Fig. 5B-E), indicating for tumor microenvironment-mediated impact as compared to the in vitro cultures. However, combination of BCI very efficiently preempted the lapatinib or neratinib resistance phenotype (Fig. 5B-E). Indicating for potential clinical utility, DUSP6 and HER2-targeted therapies'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='potential clinical utility, DUSP6 and HER2-targeted therapies displayed a clear synthetic lethal drug interaction when assessed by a waterfall blot in both cell models (Fig. 5C, E). The dramatic combinatorial activity of DUSP6 and HER2 targeting on cellular viability was also evidenced by lack of cytoplasmic eosin staining of the MDA-MB-453 xenograft tumor after 24 d of treatment (Fig. 5F). Consistent with previous studies with BCI (Kesarwani et al, 2017; Ramkissoon et al., 2019; Shojaee et al., 2015; Wu et al., 2018), we did not observe any apparent signs of toxicity or weight loss (Appendix Figure S4D) in any treatment groups. In addition, normal fibroblasts displayed several folds decreased sensitivity to BCI in an in vitro cell viability assay (Appendix'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='to BCI in an in vitro cell viability assay (Appendix Figure S4E), further indicating for a favorable dose-window between impact of DUSP6 inhibition in HER2i resistance, and its effects to normal cells.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='DUSP6 targeting does not lead to the compensatory HER3 induction characteristic to AKT inhibition'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Constitutive activity of PI3K/AKT signaling pathway is strongly associated with HER2i therapy resistance (Berns et al, 2007; Majewski et al, 2015; Nagata et al, 2004). Further, clinical studies of HER2i plus PI3K/AKT inhibitors demonstrated some clinical activity, but did not lead to approval of these combinations (Hudis et al, 2013; Saura et al, 2014). On the other hand, PI3K/AKT activation was one of the DUSP6-driven hallmark gene sets associated with the DTP-DTEP transition (Fig. 1C and 3F). Therefore, it was relevant to compare the quantitative and qualitative differences between DUSP6 blockade versus AKT inhibition in combination with the HER2i in the resistant models. Having demonstrated phenocopying growth effects between genetic DUSP6 inhibition and'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='growth effects between genetic DUSP6 inhibition and BCI, as well as resistance of DUSP KO cells to BCI, these experiments were mostly performed by comparing the compounds MK2206 (AKTi) and BCI (DUSP6i) as the alternative pharmacological HER2i combination approaches.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='In cell viability assay, both BCI and MK2206 synergized with already low micromolar concentrations of lapatinib and neratinib in MDA-MB-453 and HCC1954 cells, respectively (Fig. 6A,B, and Appendix Figures S4C and S5A). However, as compared to AKT inhibition, DUSP6 targeting had qualitatively superior pro-apoptotic activity. Regardless of efficient inhibition of AKT phosphorylation, MK2206 did not induce apoptosis alone, or in combination with lapatinib (Fig. 6C,D). In contrast, treatment with BCI plus lapatinib triggered apoptosis across all the tested cell models (Fig. 6C,D, and Appendix Figure S5B).'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='HER3 is a key pro-survival receptor in HER2+ breast cancer cells (Haikala & Janne, 2021). Further, compensatory induction of HER3 is thought to be one of the primary mechanisms behind the resistance to the combination of HER2 inhibition and AKT targeting (Chandarlapaty et al, 2011; Wilson et al., 2012). Accordingly, AKT inhibition resulted in HER3 induction in MDA-MBA-453 cells treated with either MK2206 alone, and with MK2206-lapatinib combination (Fig. 6C,E). Pharmacological inhibition of the AKT upstream kinase PI3K also induced HER3 expression (Appendix Figure S5C). In contrast, BCI decreased HER2 and HER3 expressions as a monotherapy, and in combination with lapatinib (Fig. 6C, F). Notably, both HER2 and HER3 were downregulated 3 h already after BCI'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='both HER2 and HER3 were downregulated 3 h already after BCI treatment, and this coincided with the induction of ERK phosphorylation, and destabilization of DUSP6, both serving as signs for BCI target engagement (Fig. 6G). Accordingly, siRNA-mediated knockdown of DUSP6 reduced HER2 and HER3 protein levels in MDA-MB-453, HCC1954, and the HER3+ triple negative breast cancer cell line MDA-MB-468 (Fig. 6H and Appendix Figures S5D,E). Indicative of transcriptional regulation, DUSP6 inhibition decreased HER2 and HER3 mRNA expression, but did not impact HER2 and HER3 protein stability (Appendix Figure S5F,G). Further, indicative of selective effects on HER2 and HER3 among the ERBB receptors, EGFR expression was unaffected by DUSP6 targeting in MDA-MB-468 (Appendix'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='was unaffected by DUSP6 targeting in MDA-MB-468 (Appendix Figure S5E). The tumor material from the BCI-treated MDA-MB-453 xenograft model was further used for in vivo validation of HER3 as DUSP6 downstream target. Indeed, BCI-treated tumors displayed vastly decreased tumor cell immunopositivity for HER3, and to lesser extent HER2 (Fig. 6I). Finally, clinically high DUSP6 tumors had significantly higher expression of tyrosine 1298 phosphorylated HER3 (Fig. 6J).'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='These findings indicate that when HER2i is combined AKTi, the HER2+ cells survive due to compensatory induction of HER3, whereas DUSP6 blockade rather inhibits HER3, and therefore BCI+lapatinib treated cells succumb to apoptosis. \\n\\n\\n\\nDUSP6 targeting overcomes NRG-mediated HER2i therapy tolerance'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='HER3 is activated in tumors by tumor microenvironment derived ligand NRG, and NRG-activated HER3 drives HER2i tolerance (Kodack et al, 2017; Leung et al, 2015). On the other hand, our results indicate that DUSP6 may drive HER2i resistance by promoting the expression of HER3. With this background, we compared the capacity of DUSP6 and AKT inhibition to reverse HER2i tolerance by the NRG/HER3 axis. As expected, (Kodack et al., 2017; Leung et al., 2015), NRG treatment induced tolerance to both neratinib and lapatinib across all four tested primarily sensitive cell lines (Fig. 7A and Appendix Figure S6A). The NRG-mediated rescue from both HER2is was abrogated in a concentration dependent manner after treatment with BCI, but not with MK2206 (Fig. 7A and Appendix'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='treatment with BCI, but not with MK2206 (Fig. 7A and Appendix Figure S6A). The only exception was HCC2218 cells, in which also MK2206 reversed NRG-elicited HER2i tolerance (Fig. 7A and Appendix Figure S6A). These results were validated at the level of differential apoptosis induction between DUSP6 and AKT targeting. Whereas NRG completely prevented HER2i-elicited induction of apoptosis in BT474 cells, and AKT inhibition could not reverse this, DUSP6 inhibition restored the apoptotic activity (Fig. 7B). On the other hand, RNAi-mediated HER3 knockdown triggered apoptosis in MDA-MB-453 cells (Appendix Figure S6B). To functionally prove that inhibition of HER3 expression mediates BCI effects, we performed a rescue experiment in MDA-MB-453 cells with ectopic'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='performed a rescue experiment in MDA-MB-453 cells with ectopic stable overexpression of HER3 under heterologous CMV promoter (Appendix Figure S6C). As compared to control cells, the HER3 overexpressing cells displayed significant resistance to BCI (Fig. 7C).'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='To link these results back to the development of HER2i tolerance, we found that targeting of DUSP6, but not AKT, reversed the NRG-elicited effects on lapatinib tolerance in the DTP assay (Fig. 7D). This BCI effect was likely due to DUSP6 inhibition, as siRNA-mediated DUSP6 depletion also abrogated the NRG-induced cell survival in BT474 cells (Appendix Figure S6D). Importantly, deregulation of NRG-ERBB axis was evidenced also during the DTP-DTEP transition of lapatinib treated cells as, concomitantly with DUSP6 regulation (Fig. 2C), NRG3 and ERBB4 were found upregulated in DTEP versus DTP cells (Appendix Figure S6E). Further, providing clinical validation for the link between NRG and DUSP6, NRG mRNA was significantly overexpressed in breast tumors with high'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='mRNA was significantly overexpressed in breast tumors with high DUSP6 mRNA expression (Fig. 7E).'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Due to brain microenvironment derived NRG, HER3 has a key role in brain metastatic growth of HER2+ breast cancers (Da Silva et al, 2010; Haikala & Janne, 2021).  Based on the newly discovered role for DUSP6 in promoting HER3 expression, we investigated contribution of DUSP6 in the brain metastatic outgrowth by using GFP-positive MDA-MB-361 cells as yet additional HER2+ in vivo model. To this end, we quantitated the zebrafish intracranial tumor area derived from GFP+ MDA-MB-361 cells transfected prior tumor implantation either with control or DUSP6 targeting siRNA. As shown in Fig. 7F, the zebrafish larva with DUSP6 targeted cells had significantly smaller tumors 3d after the intracranial injection. Further, mice bearing the intracranial DUSP6 KO MDA-MB-453'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Further, mice bearing the intracranial DUSP6 KO MDA-MB-453 tumors had a significantly longer survival time as compared with the CAS9 control group (Fig. 7G).'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Collectively these results demonstrate that DUSP6 inhibition is superior to AKT blockade as a HER2i combination therapy strategy due to its newly discovered capacity to inhibit the NRG-HER3 axis and has potential in inhibiting brain metastatic growth of HER2+ cells.  \\n\\n\\n\\nA DUSP6-HER3 feed forward loop drives the HER2i tolerance'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='After identifying previously unrecognized regulation of HER3 by DUSP6 and demonstrating its functional relevance, we asked whether the HER3 reciprocally regulates DUSP6 expression. Indeed, DUSP6 mRNA expression was potently inhibited already with the smallest tested concentration of neratinib or lapatinib in the HER2i sensitive cells (BT474, BT474Br) (Fig. 8A and Appendix Figure S7A; in green). However, among the HER2i resistant cell lines, DUSP6 was not inhibited by HER2is in MDA-MB-361, MDA-MB-453, BT474Br-LR, and BT474-LR cells, whereas HCC1954 showed an intermediate phenotype (Fig. 8A and Appendix Figure S7A; in red). Additionally, this regulation was specific to DUSP6, as DUSP1 mRNA expression was not inhibited by HER2 targeting in any of the tested'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='was not inhibited by HER2 targeting in any of the tested cell modes (Fig. 8A and Appendix Figure S7A). Clinically DUSP6 and HER3 mRNA expression also correlated in HER2+ cancer samples in the TCGA-BRCA dataset (Cerami et al., 2012)(Appendix Figure S7B). Differential regulation of DUSP6 expression by HER2i in sensitive (BT474Br, BT474) versus resistant (MDA-MB-361, MDA-MB-453, BT474BrLR) cells was validated at the protein level by western blotting (Fig. 8B,C and Figure EV5). Importantly, RNAi-mediated knockdown of HER3 decreased DUSP6 expression (Fig. 8D). HER3 dependent DUSP6 regulation was further supported by induction of DUSP6 mRNA and protein expression by treatment of serum-starved BT474 or BT474Br cells with the HER3 ligand NRG (Fig. 8E, F and'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='BT474 or BT474Br cells with the HER3 ligand NRG (Fig. 8E, F and Appendix Figures S7C,D).  Induction of DUSP6 was selective for NRG, since hepatocyte growth factor (HGF), which activates MET RTK, failed to increase DUSP6 in BT474 cells (Appendix Figure S7C,D). To gain insights into the mechanism by which HER3 regulates DUSP6, the NRG treated cells were co-treated either with AKTi MK2206, or the MEK1/2 inhibitor trametinib. Consistent with a previous report that DUSP6 is a transcriptional ERK1/2 target (Zandi et al., 2022), treatment with trametinib inhibited both ERK phosphorylation and DUSP6 expression in all tested cell lines (Fig. 8E-G and Appendix Figure S7E), whereas AKT inhibition by MK2206 further increased DUSP6 expression (Fig. 8E-G). Notably,'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='MK2206 further increased DUSP6 expression (Fig. 8E-G). Notably, HER2i-elicited inhibition of DUSP6 expression correlated with the capability of HER2i to impact ERK phosphorylation (Fig. 8F, G and Figure EV5A,B).'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Collectively these results reveal that inhibition of the newly discovered DUSP6-HER3 feed-forward loop has a major contribution to HER2i therapy response, and that lack of DUSP6 inhibition is a novel  HER2i resistance mechanism in acquired resistant cells in which HER2i fails to target MEK activity (Fig. 8H).\\n\\n\\n\\n\\n\\n\\nDiscussion'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Cancer therapy resistance is initiated by non-genetic signaling rewiring resulting in major changes in the epigenetic and transcriptional landscapes of DTPs (Chang et al., 2022; De Conti et al., 2021; Marine et al., 2020; Sharma et al., 2010). It is commonly believed that transition from dormant DTPs to proliferating DTEPs is a pivotal step towards development of the disseminated disease, and eventually late-stage genetic changes which determine the ultimate therapy resistance (Aguirre-Ghiso, 2021; Marine et al., 2020). Here, we report the first transcriptomic analysis of the DTP-DTEP transition in TKI treated cancer cells. In addition to importance of the discovery of the role of DUSP6 in DTP-DTEP transition, the transcriptome data will provide a rich'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='DTP-DTEP transition, the transcriptome data will provide a rich resource for future studies to understand different stages of the HER2i resistance development. Based on the fuzzy clustering data (Appendix figure S1 and Dataset EV2), the gene clusters 1, 2 and 6, contain genes that are strongly regulated upon the DTP-DTEP transition and are therefore contain many additional genes involved in the evasion of the DTP cells from lapatinib-induced growth inhibition. The longitudinal transcriptomics profiles also allowed us to identify potential master transcription factors involved in the lapatinib resistance development. The functional relevance of the differentially expressed genes associated with each therapy tolerance transition obviously need to be validated'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='therapy tolerance transition obviously need to be validated in future studies stemming from this novel resource. Further, to understand therapy tolerance development projectories at the single cell level, similar studies are needed by employing single cell RNA sequencing technologies.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Phosphatases have recently emerged as novel druggable targets for cancer therapy (Lazo et al., 2018; Vainonen et al., 2021; Zandi et al., 2022). However, their contribution to development of non-genetic TKI therapy tolerance is still poorly understood. In the present study, we asked which phosphatases might contribute to the development of HER2i tolerance and resistance and explored their potential role as therapy targets in combination with the HER2i therapies in HER2+ breast cancer. Via transcriptomics analysis of lapatinib tolerance development over six months, we discovered an important role for DUSP6 in the re-growth of HER2i tolerant cells and showed that its genetic and pharmacological blockade potentiates HER2i sensitivity in vitro and in vivo in'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='blockade potentiates HER2i sensitivity in vitro and in vivo in multiple different HER2+ cell models. Mechanistically, we discovered that transcriptional DUSP6 inhibition is an important contributor to effective HER2i response whereas the lack of HER2i-elicited DUSP6 inhibition in the resistant cells provides a novel explanation for drug resistance in these cells (Fig. 8H). We further discovered DUSP6 as a novel activator of HER3 expression and demonstrated that HER3 inhibition is a prerequisite for the therapeutic impact of DUSP6 inhibition. Inhibition of HER3 by DUSP6 targeting appears also an important mechanistic explanation for the superior apoptotic activity as compared to PI3K/AKT targeting in combination with HER2i. As we demonstrate, the therapeutic'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='in combination with HER2i. As we demonstrate, the therapeutic impact of DUSP6 targeting is not compromised by the NRG-mediated feedback to HER3 as is the case with AKT targeting and thereby our data identify DUSP6 targeting as a novel approach to target NRG-HER3 axis to overcome HER2i resistance. These are important advantages favoring DUSP6 blockade versus AKT inhibition as a future strategy for treatment of HER2+ breast cancer. A particularly important translational finding in our study is that DUSP6 defines therapy sensitivity of the lapatinib resistant cells in response to several ERBB TKIs with different specificities, and to their chemotherapy combinations. This is consistent with the proposed HER3-DUSP6 positive feedback mechanism, as HER3 or NRG'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='proposed HER3-DUSP6 positive feedback mechanism, as HER3 or NRG overexpression confer resistance to several cancer drugs  (Erjala et al., 2006; Haikala & Janne, 2021; Knuefermann et al., 2003; Recondo et al., 2020; Wilson et al., 2012; Yonesaka et al., 2011). Thereby identification of DUSP6 targeting as a novel approach for HER3 inhibition may have broad ramifications in different combination therapy settings across different cancer types.  HER2 and 3 play also essential roles in the development of brain metastasis which causes death of more than 30% of stage IV HER2+ breast cancer patients (Berghoff et al, 2014; Da Silva et al., 2010; Fecci et al., 2019). To this end, DUSP6-mediated regulation of HER2 and 3 suggested that its inhibition may retard'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='of HER2 and 3 suggested that its inhibition may retard development of brain metastasis. Indeed, our findings directly support these conclusions as DUSP6 knockout in tumor cells improves survival of mice with intracranial HER2+ tumors and the outgrowth of HER2+ breast tumor cells in a zebrafish intracranial model. Linking our results further to future therapy of HER2+ brain metastasis, we showed that DUSP6 knockdown increases sensitivity to tucatinib+trastuzumab+capecitabine combination regimen, which show significant clinical activity in HER2+ breast cancer patients with brain metastasis (Murthy et al., 2020).'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='While demonstrated here for the first time in the context of HER2i therapy resistance, genetic DUSP6 inhibition has recently been shown to inhibit malignant phenotypes in other cancer types (Shojaee et al., 2015; Wu et al., 2018; Zandi et al., 2022). Consistent with these reports, genetic inhibition of DUSP6 in our study resulted in significant inhibition of HER2+ breast cancer cell viability, HER2i resistance, colony forming potential, and in vivo tumor growth. Mechanistically, DUSP6 inhibition converted cytostatic response to HER2i to an apoptotic response, which is considered as a paramount for cancer therapy strategies aiming for cancer cure. On the other hand, overexpression of DUSP6 converted HER2i sensitive cells to resistant in both cell viability'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='HER2i sensitive cells to resistant in both cell viability and apoptosis assays.  Importantly, all our main conclusions remained valid regardless of whether DUSP6 was inhibited either by siRNA, CRISPR/CAS9, or by pharmacological inhibitors BCI and BCI-215. We further demonstrated nearly immediate target engagement, as well as resistance of three independent DUSP6 knock-out clones to BCI-elicited cell viability inhibition. Therefore, together with recent results from the others (Kong et al., 2023; Shojaee et al., 2015), we are confident that the results achieved by BCI and BCI-215 are dependent on their inhibitory effects on DUSP6 despite a recent report questioned their selectivity using other cellular system (Thompson et al, 2022).'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='The results identified DUSP6 as a novel combination therapy target with existing clinical HER2 targeting strategies including both small molecule and antibody mediated HER2 inhibition and chemotherapy combinations. In that regard, our xenograft results demonstrate significant potential for pharmacological DUSP6 inhibition in overcoming HER2i resistance in vivo by using two different HER2i resistant cell lines, and two different HER2is. Whereas neither HER2 nor DUSP6 inhibition alone did not have therapeutic effect on tumors, the combination showed very potent synthetic lethal phenotype further validating our in vitro results. The lack of monotherapy effect of BCI, combined with clear evidence for downstream target mechanism engagement in vivo also'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='for downstream target mechanism engagement in vivo also alleviates concerns about overall cellular toxicity behind the BCI-elicited HER2i synergy. This is consistent with previous findings demonstrating antitumor effects with BCI and BCI-215 without obvious systemic toxicity in in vivo models of gastric cancer, leukemia, and malignant peripheral nerve sheath tumor (Kesarwani et al., 2017; Ramkissoon et al., 2019; Shojaee et al., 2015; Wu et al., 2018).'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='In summary, we provide first transcriptional map of DTP-DTEP transition under TKI tolerance development in cancer. The results specifically identify DUSP6 targeting as a novel approach to target HER3-mediated ERBB TKI resistance. Ultimately, the work provides proof-of-principle evidence to encourage development of next generation DUSP6 inhibitors (with brain penetrance) to test the clinical relevance of the presented therapy scenarios.\\n\\n\\nMaterials and Methods\\n\\n\\n\\nReagents'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Materials and Methods\\n\\n\\n\\nReagents\\n\\nTissue culture reagents including regular RPMI, DMEM, RPMI and FBS were purchased from Sigma. The recombinant NRG, HGF and β-estradiol (E2) were from Peprotech. BCI was purchased from Axon Medchem. BCI-215 was provided by Dr. Andreas Vogt, University of Pittsburgh Drug Discovery Institute, Pittsburgh, PA, USA. The HER2 inhibitors and the other compounds used in the drug screening were purchased from Adooq Bioscience. All the agents were dissolved in DMSO and the final concentration of DMSO did not exceed 0.1% [v/v] in all the treatments.\\n\\nCloning and plasmids'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Cloning and plasmids\\n\\npBABE-puro-gateway-ERBB2 was a gift from Matthew Meyerson (Addgene plasmid No. 40978; http://n2t.net/addgene:40978; RRID:Addgene_40978). ERBB3 wild-type was cloned from pBABE-puro-gateway-ERBB3 (Koivu et al, 2021) into pLenti CMV Puro DEST (w118-1), a gift from Eric Campeau and Paul Kaufman (Addgene plasmid # 17452;http://n2t.net/addgene:17452 ; RRID:Addgene_17452) through Gateway cloning (Chakroborty et al, 2019) to create pLenti-CMV-Puro-ERBB3. The DUSP6 plasmid was purchased from Addgene (#27975) and the Kinase interaction motif (KIM) mutants were prepared by site directed mutagenesis (GenScript Inc) generating R64A, R65A double mutant defective in ERK binding as described (Nichols et al., 2000).\\n\\nGeneration of stable lines'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='To overexpress wild-type ERBB2, pBABE-puro-gateway-ERBB2 was transfected (using Fugene6 transfection reagent; Promega Catalog # E2692) into amphotropic Phoenix HEK293T cells (a gift from Dr. Garry Nolan) to generate retroviruses, which were used to transduce MDA-MB-453, as described previously (Chakroborty et al., 2019). pLenti-CMV-Puro-ERBB3 was co-transfected with virus-packaging plasmids pMLDg/pRRE (addgene #12251), pMD2.G (addgene #12259), and pRSV-Rev (addgene #12253) into HEK293T cells using Fugene6 transfection reagent to produce lentiviruses. The lentivirus-containing supernatant was used to transduce MDA-MB-453 to over-express wild-type ERBB3. After viral transduction cells were treated with 1 µg/mL puromycin (Gibco) for 48 h to select the cells'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='with 1 µg/mL puromycin (Gibco) for 48 h to select the cells with stable expression of the respective introduced transgenes.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='For DUSP1 and 6 overexpression, lentiviral particles containing full length of either DUSP1 (Genecopoeia),  DUSP6 (Addgene #27975), or control empty (Genecopoeia) vector were generated in HEK293FT packaging cell line (complete medium: high glucose DMEM, 10% FBS, 0,1mM NEAA, 1mM MEM Sodium Pyruvate, 6mM L-Glutamine, 1% Pen/Strep and 0,5mg/ml Geneticin) by transient transfection of transfer vector 2nd generation packaging plasmid-psPAX2 (Addgene #12259) and envelope vector-pMD2 (Addgene #12260) with the ratio (7:2:1) using calcium-phosphate precipitation method. Seventy-two hours post-transfection medium containing viral vectors was collected, concentrated for 2 h by ultracentrifugation in swing-out rotor SW-32Ti (Beckman Coulter), 26,000g, resuspended in'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='rotor SW-32Ti (Beckman Coulter), 26,000g, resuspended in residual medium and flash-frozen in liquid nitrogen. Functional titer   ̴1x108 was measured in HEK293FT cells and FACS (BD LSRFortessa, Becton Dickinson). To obtain stable overexpression of DUSP1, DUSP6 or double DUSP1+6 population on day zero, 8x104 cells were seeded in a 24-well plate. 24 h later, the cells were transduced with MOI 60 of lentiviral stocks in a low volume of full media. Medium containing viral particles was removed 16 h later. Cells expressing DUSP1 and GFP indicative of lentiviral integration were collected by fluorescence assisted cell sorting (BD FACSaria II cell sorter, Becton Dickinson). FACS gating was set at 10% top high fluorescence signal. DUSP6 transduced BT474 cells were'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='top high fluorescence signal. DUSP6 transduced BT474 cells were selected with 3 µg/mL of puromycin. DUSP1 and 6 expressing cells were obtained by sequential transduction (MOI 2, 6, 10), puromycin selection and later GFP fluorescence assisted cell sorting (BD FACSaria II cell sorter, Becton Dickinson). The levels of protein were confirmed by Western blot analysis.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='For stable overexpression of DUSP6 and DUSP6 KIM mutant, DUSP6 (Addgene #27975) and DUSP6R64-65A lentiviral vectors were co-transfected with virus-packaging plasmids pMLDg/pRRE, (Addgene #12251), pMD2.G (Addgene #12259), and pRSV-Rev (Addgene #12253) into HEK293T cells using Fugene6 transfection reagent to produce lentiviruses. 48 h after transfection viral supernatants were collected, filtered, and added to pre-plated BT474 cells along with 10 μg/mL polybrene. Transduced cells were selected with 2 μg/mL puromycin, and the protein levels were confirmed by Western blot analysis.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='To rescue the DUSP6 KO MDA-MB-453 clones, the pLEX307-hygro-DUSP6 lentiviral vector was co-transfected with virus-packaging plasmids pMLDg/pRRE, (Addgene #12251), pMD2.G (Addgene #12259), and pRSV-Rev (Addgene #12253) into HEK293T cells using FuGENE6 transfection reagent. 48 hours after transfection viral supernatants were collected, filtered, and added to MDA-MB-453 DUSP6 KO cells (g1, clone 22) along with 10 μg/mL polybrene. Transduced cells were selected with hygromycin (250µg/ml).The MDA-MB-453 CAS9 control, DUSP6 KO, and DUSP6 transduced KO cells were seeded at low density and maintained for 10 d. The colonies were stained with 0.5% crystal violet in methanol and imaged using an Epson scanner. Colony quantification was done by using the ImageJ'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='scanner. Colony quantification was done by using the ImageJ ColonyArea plugin.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='FACS\\n\\nMDA-MB-453 cells transduced with lentiviruses encoding wild-type ERBB3 were washed with azide-free PBS, trypsinized and suspended in ice-cold sorting buffer (PBS +1% Goat serum, Life technologies catalog # PCN5000). The cells were incubated with anti-ERBB3 (MAB3481, R&D systems) for 1 h on ice, and with anti-mouse Alexa Fluor 405 (A-31553, Invitrogen) for 30 min on ice in a dark environment. Single cell suspension was analyzed and sorted on Sony SH800 Cell Sorter to select cell-pools with high surface ERBB3 expression.\\n\\nCRISPR/CAS9 knockout system'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='A two-component CRISPR system was used to generate DUSP6 KO cells (Adli, 2018). DUSP6 sgRNAs (seq#1- CATCGAGTCGGCCATCAACG, seq#2-GACTGGAACGAGAATACGGG, seq#3- CCATGATAGATACGCTCAGA) were selected using DeskGEN platform and cloned according to F. Zhang lab protocol. Separate lentivectors containing spCas9 (lentiCas9-Blast a gift from Feng Zhang (Addgene plasmid # 52962) and sgRNA (lentiGuide-Puro a gift from Feng Zhang (Addgene plasmid # 52963) were produced in HEK293FT packaging cell line by transient cotransfection. Shortly, 40-70% confluent HEK293FT cells were used for transfections with 14 μg of transfer vector, 4 μg of packaging vector psPAX2 (gift from Didier Trono (Addgene plasmid # 12260), 2 μg envelope vector pMD2.G (gift from Didier Trono (Addgene'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='2 μg envelope vector pMD2.G (gift from Didier Trono (Addgene plasmid # 12259) mixed in 0.45 mL water, 2.5M CaCl2, and 2x HeBS (274 mM NaCl, 10 mM KCl, 1.4 mM Na2HPO4, 15 mM D-glucose, 42 mM Hepes, pH 7.06) per 10 cm dish. Before adding to the cells, the DNA-HeBS mix was incubated for 30 min at room temperature. After overnight incubation medium with DNA precipitate was gently removed from the cells and replaced with a full fresh medium. Media containing viral particles was collected after 72 h, spun at 300 rpm for 5 min at room temperature to remove cell debris, filtered through 0.45 µm PES filter, and concentrated by ultracentrifugation for 2 hours at 25,000rpm, 4°C (Beckman Coulter). The pellet containing lentiviral particles was suspended in the residual'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='containing lentiviral particles was suspended in the residual medium, incubated for ~2 hours in +4˚C with occasional mild vortex, aliquoted, snap-frozen and stored in -70˚C. P24 ELISA measured physical lentiviral titer with a serial dilution of virus stock according to manufacturer protocol.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='To generate DUSP6KO clones in MDA-MB-453, 1e+05 cells were seeded on a 24-well plate. The next day the cells were transduced with Lenti-Cas9 (MOI 1, 5, 10), and 72 h later, 8 μg/mL of Blasticidin was applied to select only Cas9 expressing cells. Cells transduced with the smallest number of Lenti-Cas9 particles that survived after the parallel control well was cleared proceeded to the next step. In the second stage, the mixed pool of stably expressing Cas9 cells was transduced with Lenti sgRNA vectors (MOI 5,10,15,20), and 72 h later, 1 μg/uL Puromycin was applied on the cells to select double-positive Cas9+/Lenti sgRNA+ cells. Based on Western blot results, cell populations showing the highest reduction in DUSP6 protein levels were single sorted (Sony SH800'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='in DUSP6 protein levels were single sorted (Sony SH800 cell sorter, Sony Biotechnology Inc) and re-grown into a clonal cell population. On average, about 20 clones per sgRNA population were screened using Western blotting and qPCR. Sanger sequencing was used to confirm full knockout status.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content=\"Cell Culture and Transfections\\n\\nAll cell lines were purchased from the American Type Culture Collection (ATCC) and were maintained at 37 °C and 5% CO2 in a humidified incubator and cultured according to the ATCC recommendations. Cell line authentication was performed by STR profiling and using Cell IDTM system (Promega). The cultures were routinely tested for mycoplasma contamination. The information about siRNAs for DUSP1, DUSP6, HER2, HER3, AKT1 and negative control siRNAs are in Table EV1. Transient transfections were performed with lipofectamine RNAiMAX reagent (ThermoFisher) according to the manufacturer's instructions. \\n\\nCell growth assays\"), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Cell growth assays\\n\\nThe cells were seeded at a density of 2 x 103 into 96-well plates and treated with increasing concentrations of the drugs for 48 h. Cell viability was determined using WST-1 assay (Sigma). Vehicle-treated cells were used as the control group. For the clonogenic survival assay, cells were seeded in 6-well plates at a density of 1000 cells/well. After 24 h, the media was changed, and the cells were maintained for another 10 d. The resulting colonies were stained/fixed with 0.5% crystal violet imaged using an inverted microscope.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='To yield DTPs in NSCLC, melanoma and colorectal cancer cell lines, the following treatments were carried out: NSCLC cell line HCC827, melanoma cell line A375, and colorectal cancer HT-29 cell line were cultured under standard conditions. The following treatments were applied for 10 d: HCC827, 1 µM osimertinib; A375, 1 µM dabrafenib+100 nM trametinib; HT-29, 1 µM dabrafenib+10 µg/ml cetuximab. \\n\\nDrug combination analysis'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Drug combination analysis\\n\\nTo explore the efficacy of drug combinations, growth inhibition was determined by WST-1 assay and the results were evaluated by Bliss SynergyFinder (Ianevski et al, 2017). Visualization of synergy scores is depicted as a synergy map. An average synergy score of 0 is considered additive, less than 0 as antagonistic and over 0 as a synergistic outcome. \\n\\nCaspase 3/7 activity assay\\n\\nThe Caspase-Glo® 3/7 Assay System (Promega), a luminescence-based assay for detection of active caspase-3 and 7, was used to quantitatively determine apoptotic cell death. Following caspase cleavage of the proluciferin DEVD substrate, a substrate for luciferase is released and results in luciferase reaction and production of light.\\n\\nWestern blot analysis'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Western blot analysis\\n\\nThe cells were lysed for 30 min in ice-cold RIPA buffer (50 mM Tris-HCl, pH 8.0, 150 mM NaCl, 1.0% NP-40, 0.5% sodium deoxycholate and 0.1% SDS) containing protease and phosphatase inhibitors (ThermoFisher). The assay was performed with primary and Licor secondary antibodies (Licor Biosciences). β-actin was used as the loading control. The list of antibodies is in Table EV2.\\n\\nAnalysis of gene expression by quantitative reverse transcription-PCR'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Quantitative reverse transcription-PCR (qRT-PCR) analysis was done on a QuantStudio Real-Time PCR instrument (ThermoFisher) using PowerUp™ SYBR® Green Master Mix (ThermoFisher). The primer sequences are listed in Table EV3. The target gene expression levels were normalized to beta-2-microglobulin (B2M) levels. For calculations, 2 –ΔΔCT formula was used, with ΔΔCT = (CTTarget – CTB2M) experimental sample – (CTTarget – CTB2M) control samples, where CT is the cycle threshold.\\n\\nAnimal studies'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='The animal experiments were performed according to the Animal Experiment Board in Finland (ELLA) for the care and use of animals under the licenses 4161/04.10.07/2015 and 9241/2018. The animals were kept under pathogen-free conditions in individually ventilated cages in an animal care facility. Mice were kept on a 12 h light/dark cycle with access to autoclaved water and irradiated chow ad libitum and were allowed to adapt to the facility for 1 week before starting the experiments. For the subcutaneous experiments, MDA-MB-453 (5 x 106) and HCC1954 (3 x 106) cells were injected into the right flank of six- to eight-week-old BALB/cOlaHsd-Foxn1nu mice (Envigo, France). Mice with tumor size 100 mm3 were randomized into experimental and control groups. Tumor'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='mm3 were randomized into experimental and control groups. Tumor dimensions were measured with Vernier calipers and tumor volume were calculated as 1/2 larger diameter x (smaller diameter)2.  For the intracranial model, 1 x 105 cells in 5 µL of PBS were inoculated into the brain of anaesthetized mice. The mice were then imaged with bioluminescence and based on the bioluminescence signal were randomized to experimental and control groups, as descried earlier (Merisaari et al, 2020).  Mice were euthanized when they became moribund when they reached defined study end points.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Zebrafish studies'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Zebrafish embryo xenograft studies were performed under the license ESAVI/9339/04.10.07/2016 (National Animal Experimentation Board, Regional State Administrative Agency for Southern Finland). Briefly, the DUSP6 knockdown GFP-MDA-MB-361 cells were injected into the brain of zebrafish embryos from the dorsal side. One day after injection (1 dpi), successfully transplanted embryos were placed in CellView glass bottom 96-well plate (1 embryo/well) and embryos were incubated in E3 + PTU at 33 °C. The xenografted embryos were imaged using a Nikon Eclipse Ti2 fluorescence microscope and a 2x Nikon Plan-Apochromat (NA 0.06) objective. Each embryo was imaged at 1dpi and 4dpi using brightfield illumination and a GFP fluorescence filter set (excitation with 470nm'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='and a GFP fluorescence filter set (excitation with 470nm LED). Each image was inspected manually to filter out severely malformed, dead or out of focus embryos. The tumor area was measured using ImageJ (NIH). The fold change in tumor size was calculated as follows: GFP intensity (4dpi)/GFP intensity (1dpi)'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='RNA-sequencing'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='RNA-sequencing was conducted at the Finnish Functional Genomic Center, The University of Turku, Finland. RNA was harvested using the NucleoSpin RNA purification kit (Macherey‐Nagel), followed by treatment with DNase to remove genomic DNA. RNA (300 ng) was reverse transcribed using the Illumina TruSeq Stranded Total mRNA kit. The quality of the samples was ensured using Agilent Bioanalyzer 2100 or Advanced Analytical Fragment Analyzer. Sample concentration was measured with Qubit® Fluorometric Quantitation (Life Technologies) and/or KAPA Library Quantification kit for Illumina platform, KAPA Biosystems. Sequencing run was performed using the Illumina NovaSeq 6000 instrument. Genomic alignment was performed using Rsubread v. 2.0.0 and the reads were mapped to'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='performed using Rsubread v. 2.0.0 and the reads were mapped to the human reference genome hg38. Aligned reads were assigned to RefSeq gene models using the same R package with its default settings.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Differentially expressed genes and pathways were identified using the R package limma. Latest hallmark gene sets were downloaded from the Molecular Signatures Database version 7.4 (http://www.gsea-msigdb.org/gsea/msigdb/index.jsp) and used within Gene Set Variation Analysis (GSVA) allowing pathway enrichment estimates for each sample (Hanzelmann et al, 2013). Data was transformed using log(x+1) after normalization and the RNA-sequencing pipeline run. Heatmaps were plotted using the ComplexHeatmap package (Gu et al, 2016). R statistical software version 4.0.3 was used for the statistical analyses and visualizations.\\n\\ncBioPortal database analyses'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='The correlation analysis between gene expression and breast cancer subtypes was examined using the METABRIC breast cancer cohort with PAM50 classification (Pereira et al., 2016), TCGA breast invasive carcinoma dataset and the TCGA Firehose legacy dataset (Gao et al., 2013). The RNA-seq data from breast invasive carcinoma samples and the relevant clinical information are available at the TCGA data portal, cBioPortal for Cancer Genomics (http://www.cbioportal.org/). According to the PAM50 classification, METABRIC breast cancer patients were divided into 5 subtypes, including the basal (n=199), HER2+(n=220), Lum A (n=679), Lum B (n=461) and Normal-like (n=140). ER and HER2 status were assessed using the patient’s IHC information. The survival outcomes were'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='using the patient’s IHC information. The survival outcomes were extracted from the TCGA breast invasive carcinoma dataset.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='For multivariable Cox regression analyses, N=121 HER2+ patients from the TCGA-BRCA cohort were extracted, with reported pathological T-stages and lymph node statuses included as clinical factors and relapse free survival modelled as the end-point. Further, transcriptomics was included as log2-transformed values for RSEM normalized expression of DUSP6.\\n\\nTranscription factor binding analyses'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Transcription factor binding analyses\\n\\nTo predict transcription factors enriched in the significantly differentially regulated genes during the therapy tolerance transitions we used ChEA3: transcription factor enrichment analysis by orthogonal omics integration tool (Keenan et al, 2019). Differentially regulated genes were provided as input to the tool and the significantly associated transcription factors were obtained using ENCODE (Encyclopedia of DNA Elements) ChIP-seq library. The output file consists of a list of transcription factors arranged based on a scaled rank where a lower value means a higher significance in the gene list. An FDR value < 0.05 was considered significant.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='To predict the transcription factors binding the DUSP6 promotor region, Cistrome DB Toolkit was used (Zheng et al, 2019). As an input 4000 base pairs upstream of the DUSP6 gene were entered in the Cistrome DB Toolkit. This provided a list of transcription factors based on the regulatory potential (RP score) derived by comparing the CHIP-seq data sets.\\n\\nStatistical analysis'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Differentially expressed genes and pathways were identified using the R package limma (Ritchie et al, 2015).  Latest hallmark gene sets were downloaded from the Molecular Signatures Database version 7.4 (http://www.gsea-msigdb.org/gsea/msigdb/index.jsp) and used within Gene Set Variation Analysis (GSVA) allowing pathway enrichment estimates for each sample (Hanzelmann et al., 2013), with FDR-cutoff <0.25 used for the pathway enrichment analyses. Data was transformed using log(x+1) after normalization and the RNA-sequencing pipeline run. Heatmaps were plotted using the ComplexHeatmap package (Gu et al., 2016). R statistical software (R Core Team (2023). R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna,'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='computing. R Foundation for Statistical Computing, Vienna, Austria. URL: https://www.R-project.org) version 4.0.3 was used for the statistical analyses and visualizations.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='All data were evaluated in triplicate against the vehicle-treated control cells and collected from three independent experiments. In addition to R analyses, data were visualized and analyzed using GraphPad Prism 8.3.0 using one-way ANOVA and the unpaired two-tailed student’s t test. All such data are presented as mean ± standard deviation (SD).\\n\\nData Availability\\n\\nThe gene expression data from this publication have been deposited to the GEO database (https://www.ncbi.nlm.nih.gov/geo/) and assigned the identifier GSE231526 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE231526). \\n\\n\\n\\nDisclosure and competing interests statement\\n\\n\\n\\nThe authors declare no competing interests.\\n\\n\\n\\n\\n\\nAuthor contributions'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Author contributions\\n\\nMajid Momeny: Conceptualization; Validation; Supervision; Visualization; Writing; Investigation; Formal analysis; Methodology. Mari Tienhaara: Validation; Investigation; Methodology. Mukund Sharma: Validation; Investigation; Methodology. Deepankar Chakroborty: Methodology. Roosa Varjus: Formal analysis. Iina Takala: Methodology. Joni Merisaari: Methodology. Artur Padzik: Methodology; Validation. Andreas Vogt: Methodology. Ilkka Paatero: Methodology; Validation. Klaus Elenius: Supervision; Methodology. Teemu D. Laajala: Methodology; Formal analysis. Kari J. Kurppa: Supervision; Investigation; Methodology. Jukka Westermarck: Conceptualization; Supervision; Validation; Project administration; Writing-review and editing.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Acknowledgement'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content=\"Taina Kalevo-Mattila is acknowledged for superior technical support and the entire Turku Bioscience Centre personnel is thanked for excellent working environment. We acknowledge Zhong-Yin Zhang for PRL1i compound, Norma O'Donovan, National Institute for Cellular Biotechnology, Dublin City University, Ireland, for some HER2+ breast cancer cell lines used in this study, and Dihua Yu, The University of Texas MD Anderson Cancer Center, USA, for the BT474Br cells. We further acknowledge important contributions of the following core facilities of Turku Bioscience Centre (University of Turku and Åbo Akademi University) supported by Biocentre Finland: Finnish Functional Genomics Center, Screening unit, and Zebrafish unit. This study was supported by funding from\"), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='and Zebrafish unit. This study was supported by funding from Finnish Cancer Associations (JW), Foundation of Finnish Cancer Institute (MM), Maud Kuistila Foundation (MM), Turku University Foundation (MM), Finnish Cancer Institute (TDL), Finnish Cultural Foundation (TDL), and Finnish Cultural Foundation (KJK).'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='References:\\n\\nAdli M (2018) The CRISPR tool kit for genome editing and beyond. Nat Commun 9: 1911\\n\\nAguirre-Ghiso JA (2021) Translating the Science of Cancer Dormancy to the Clinic. Cancer Res 81: 4673-4675\\n\\nArteaga CL, Engelman JA (2014) ERBB receptors: from oncogene discovery to basic science to mechanism-based cancer therapeutics. Cancer Cell 25: 282-303\\n\\nBerghoff AS, Bartsch R, Preusser M, Ricken G, Steger GG, Bago-Horvath Z, Rudas M, Streubel B, Dubsky P, Gnant M et al (2014) Co-overexpression of HER2/HER3 is a predictor of impaired survival in breast cancer patients. Breast 23: 637-643'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Berns K, Horlings HM, Hennessy BT, Madiredjo M, Hijmans EM, Beelen K, Linn SC, Gonzalez-Angulo AM, Stemke-Hale K, Hauptmann M et al (2007) A functional genetic approach identifies the PI3K pathway as a major determinant of trastuzumab resistance in breast cancer. Cancer Cell 12: 395-402\\n\\n[DATASET]  Cerami E, Gao J, Dogrusoz U, Gross BE, Sumer SO, Aksoy BA, Jacobsen A, Byrne CJ, Heuer ML, Larsson E et al (2012) The cBio cancer genomics portal: an open platform for exploring multidimensional cancer genomics data. Cancer Discov 2: 401-404'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Chakroborty D, Kurppa KJ, Paatero I, Ojala VK, Koivu M, Tamirat MZ, Koivunen JP, Janne PA, Johnson MS, Elo LL et al (2019) An unbiased in vitro screen for activating epidermal growth factor receptor mutations. J Biol Chem 294: 9377-9389\\n\\nChan LN, Murakami MA, Robinson ME, Caeser R, Sadras T, Lee J, Cosgun KN, Kume K, Khairnar V, Xiao G et al (2020) Signalling input from divergent pathways subverts B cell transformation. Nature 583: 845-851\\n\\nChandarlapaty S, Sawai A, Scaltriti M, Rodrik-Outmezguine V, Grbovic-Huezo O, Serra V, Majumder PK, Baselga J, Rosen N (2011) AKT inhibition relieves feedback suppression of receptor tyrosine kinase expression and activity. Cancer Cell 19: 58-71'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content=\"[DATASET] Chang CA, Jen J, Jiang S, Sayad A, Mer AS, Brown KR, Nixon AML, Dhabaria A, Tang KH, Venet D et al (2022) Ontogeny and Vulnerabilities of Drug-Tolerant Persisters in HER2+ Breast Cancer. Cancer discovery 12: 1022-1045. https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE155342\\n\\nCleary JM, McRee AJ, Shapiro GI, Tolaney SM, O'Neil BH, Kearns JD, Mathews S, Nering R, MacBeath G, Czibere A et al (2017) A phase 1 study combining the HER3 antibody seribantumab (MM-121) and cetuximab with and without irinotecan. Invest New Drugs 35: 68-78\"), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Da Silva L, Simpson PT, Smart CE, Cocciardi S, Waddell N, Lane A, Morrison BJ, Vargas AC, Healey S, Beesley J et al (2010) HER3 and downstream pathways are involved in colonization of brain metastases from breast cancer. Breast Cancer Res 12: R46\\n\\n[DATASET]  de Bruijn I, Kundra R, Mastrogiacomo B, Tran TN, Sikina L, Mazor T, Li X, Ochoa A, Zhao G, Lai B et al (2023) Analysis and Visualization of Longitudinal Genomic and Clinical Data from the AACR Project GENIE Biopharma Collaborative in cBioPortal. Cancer Res 83: 3861-3867\\n\\nDe Conti G, Dias MH, Bernards R (2021) Fighting Drug Resistance through the Targeting of Drug-Tolerant Persister Cells. Cancers (Basel) 13'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content=\"Dhimolea E, de Matos Simoes R, Kansara D, Al'Khafaji A, Bouyssou J, Weng X, Sharma S, Raja J, Awate P, Shirasaki R et al (2021) An Embryonic Diapause-like Adaptation with Suppressed Myc Activity Enables Tumor Treatment Persistence. Cancer Cell 39: 240-256 e211\\n\\nErjala K, Sundvall M, Junttila TT, Zhang N, Savisalo M, Mali P, Kulmala J, Pulkkinen J, Grenman R, Elenius K (2006) Signaling via ErbB2 and ErbB3 associates with resistance and epidermal growth factor receptor (EGFR) amplification with sensitivity to EGFR inhibitor gefitinib in head and neck squamous cell carcinoma cells. Clin Cancer Res 12: 4103-4111\"), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Fecci PE, Champion CD, Hoj J, McKernan CM, Goodwin CR, Kirkpatrick JP, Anders CK, Pendergast AM, Sampson JH (2019) The Evolving Modern Management of Brain Metastasis. Clin Cancer Res 25: 6570-6580\\n\\nFutschik ME, Carlisle B (2005) Noise-robust soft clustering of gene expression time-course data. J Bioinform Comput Biol 3: 965-988\\n\\nGaborit N, Abdul-Hai A, Mancini M, Lindzen M, Lavi S, Leitner O, Mounier L, Chentouf M, Dunoyer S, Ghosh M et al (2015) Examination of HER3 targeting in cancer using monoclonal antibodies. Proc Natl Acad Sci U S A 112: 839-844'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='[DATASET] Gao J, Aksoy BA, Dogrusoz U, Dresdner G, Gross B, Sumer SO, Sun Y, Jacobsen A, Sinha R, Larsson E et al (2013) Integrative analysis of complex cancer genomics and clinical profiles using the cBioPortal. Sci Signal 6: pl1\\n\\nGoutsouliak K, Veeraraghavan J, Sethunath V, De Angelis C, Osborne CK, Rimawi MF, Schiff R (2020) Towards personalized treatment for early stage HER2-positive breast cancer. Nat Rev Clin Oncol 17: 233-250\\n\\nGu Z, Eils R, Schlesner M (2016) Complex heatmaps reveal patterns and correlations in multidimensional genomic data. Bioinformatics 32: 2847-2849\\n\\nHaikala HM, Janne PA (2021) Thirty Years of HER3: From Basic Biology to Therapeutic Interventions. Clin Cancer Res'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Hanzelmann S, Castelo R, Guinney J (2013) GSVA: gene set variation analysis for microarray and RNA-seq data. BMC Bioinformatics 14: 7\\n\\nHata AN, Niederst MJ, Archibald HL, Gomez-Caraballo M, Siddiqui FM, Mulvey HE, Maruvka YE, Ji F, Bhang HE, Krishnamurthy Radhakrishna V et al (2016) Tumor cells can follow distinct evolutionary paths to become resistant to epidermal growth factor receptor inhibition. Nat Med 22: 262-269\\n\\nHudis C, Swanton C, Janjigian YY, Lee R, Sutherland S, Lehman R, Chandarlapaty S, Hamilton N, Gajria D, Knowles J et al (2013) A phase 1 study evaluating the combination of an allosteric AKT inhibitor (MK-2206) and trastuzumab in patients with HER2-positive solid tumors. Breast Cancer Res 15: R110'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content=\"Ianevski A, He L, Aittokallio T, Tang J (2017) SynergyFinder: a web application for analyzing drug combination dose–response matrix data. Bioinformatics 33: 2413-2415\\n\\nKaltenmeier CT, Vollmer LL, Vernetti LA, Caprio L, Davis K, Korotchenko VN, Day BW, Tsang M, Hulkower KI, Lotze MT et al (2017) A Tumor Cell-Selective Inhibitor of Mitogen-Activated Protein Kinase Phosphatases Sensitizes Breast Cancer Cells to Lymphokine-Activated Killer Cell Activity. J Pharmacol Exp Ther 361: 39-50\\n\\nKeenan AB, Torre D, Lachmann A, Leong AK, Wojciechowicz ML, Utti V, Jagodnik KM, Kropiwnicki E, Wang Z, Ma'ayan A (2019) ChEA3: transcription factor enrichment analysis by orthogonal omics integration. Nucleic Acids Res 47: W212-W224\"), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Kesarwani M, Kincaid Z, Gomaa A, Huber E, Rohrabaugh S, Siddiqui Z, Bouso MF, Latif T, Xu M, Komurov K et al (2017) Targeting c-FOS and DUSP1 abrogates intrinsic resistance to tyrosine-kinase inhibitor therapy in BCR-ABL-induced leukemia. Nat Med 23: 472-482\\n\\nKnuefermann C, Lu Y, Liu B, Jin W, Liang K, Wu L, Schmidt M, Mills GB, Mendelsohn J, Fan Z (2003) HER2/PI-3K/Akt activation leads to a multidrug resistance in human breast adenocarcinoma cells. Oncogene 22: 3205-3212\\n\\nKodack DP, Askoxylakis V, Ferraro GB, Sheng Q, Badeaux M, Goel S, Qi X, Shankaraiah R, Cao ZA, Ramjiawan RR et al (2017) The brain microenvironment mediates resistance in luminal breast cancer to PI3K inhibition through HER3 activation. Sci Transl Med 9'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Koivu MKA, Chakroborty D, Tamirat MZ, Johnson MS, Kurppa KJ, Elenius K (2021) Identification of Predictive ERBB Mutations by Leveraging Publicly Available Cell Line Databases. Mol Cancer Ther 20: 564-576\\n\\nKong T, Laranjeira ABA, Yang K, Fisher DAC, Yu L, Poittevin De La Fregonniere L, Wang AZ, Ruzinova MB, Fowles JS, Fulbright MC et al (2023) DUSP6 mediates resistance to JAK2 inhibition and drives leukemic progression. Nat Cancer 4: 108-127\\n\\nKorotchenko VN, Saydmohammed M, Vollmer LL, Bakan A, Sheetz K, Debiec KT, Greene KA, Agliori CS, Bahar I, Day BW et al (2014) In vivo structure-activity relationship studies support allosteric targeting of a dual specificity phosphatase. Chembiochem 15: 1436-1445'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Kurppa KJ, Liu Y, To C, Zhang T, Fan M, Vajdi A, Knelson EH, Xie Y, Lim K, Cejas P et al (2020) Treatment-Induced Tumor Dormancy through YAP-Mediated Transcriptional Reprogramming of the Apoptotic Pathway. Cancer Cell 37: 104-122 e112\\n\\nLazo JS, McQueeney KE, Burnett JC, Wipf P, Sharlow ER (2018) Small molecule targeting of PTPs in cancer. Int J Biochem Cell Biol 96: 171-181\\n\\nLeung WY, Roxanis I, Sheldon H, Buffa FM, Li JL, Harris AL, Kong A (2015) Combining lapatinib and pertuzumab to overcome lapatinib resistance due to NRG1-mediated signalling in HER2-amplified breast cancer. Oncotarget 6: 5678-5694'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Majewski IJ, Nuciforo P, Mittempergher L, Bosma AJ, Eidtmann H, Holmes E, Sotiriou C, Fumagalli D, Jimenez J, Aura C et al (2015) PIK3CA mutations are associated with decreased benefit to neoadjuvant human epidermal growth factor receptor 2-targeted therapies in breast cancer. J Clin Oncol 33: 1334-1339\\n\\nMarine JC, Dawson SJ, Dawson MA (2020) Non-genetic mechanisms of therapeutic resistance in cancer. Nat Rev Cancer 20: 743-756\\n\\nMarsolier J, Prompsy P, Durand A, Lyne AM, Landragin C, Trouchet A, Bento ST, Eisele A, Foulon S, Baudre L et al (2022) H3K27me3 conditions chemotolerance in triple-negative breast cancer. Nat Genet 54: 459-468'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Merisaari J, Denisova OV, Doroszko M, Le Joncour V, Johansson P, Leenders WPJ, Kastrinsky DB, Zaware N, Narla G, Laakkonen P et al (2020) Monotherapy efficacy of blood-brain barrier permeable small molecule reactivators of protein phosphatase 2A in glioblastoma. Brain Commun 2: fcaa002\\n\\nMolina G, Vogt A, Bakan A, Dai W, Queiroz de Oliveira P, Znosko W, Smithgall TE, Bahar I, Lazo JS, Day BW et al (2009) Zebrafish chemical screening reveals an inhibitor of Dusp6 that expands cardiac cell lineages. Nat Chem Biol 5: 680-687\\n\\nMurthy RK, Loi S, Okines A, Paplomata E, Hamilton E, Hurvitz SA, Lin NU, Borges V, Abramson V, Anders C et al (2020) Tucatinib, Trastuzumab, and Capecitabine for HER2-Positive Metastatic Breast Cancer. N Engl J Med 382: 597-609'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content=\"Nagata Y, Lan KH, Zhou X, Tan M, Esteva FJ, Sahin AA, Klos KS, Li P, Monia BP, Nguyen NT et al (2004) PTEN activation contributes to tumor inhibition by trastuzumab, and loss of PTEN predicts trastuzumab resistance in patients. Cancer Cell 6: 117-127\\n\\nNichols A, Camps M, Gillieron C, Chabert C, Brunet A, Wilsbacher J, Cobb M, Pouyssegur J, Shaw JP, Arkinstall S (2000) Substrate recognition domains within extracellular signal-regulated kinase mediate binding and catalytic activation of mitogen-activated protein kinase phosphatase-3. J Biol Chem 275: 24613-24621\\n\\nPatterson KI, Brummer T, O'Brien PM, Daly RJ (2009) Dual-specificity phosphatases: critical regulators with diverse cellular targets. Biochem J 418: 475-489\"), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Pereira B, Chin SF, Rueda OM, Vollan HK, Provenzano E, Bardwell HA, Pugh M, Jones L, Russell R, Sammut SJ et al (2016) The somatic mutation profiles of 2,433 breast cancers refines their genomic and transcriptomic landscapes. Nature communications 7: 11479\\n\\nRamkissoon A, Chaney KE, Milewski D, Williams KB, Williams RL, Choi K, Miller A, Kalin TV, Pressey JG, Szabo S et al (2019) Targeted Inhibition of the Dual Specificity Phosphatases DUSP1 and DUSP6 Suppress MPNST Growth via JNK. Clin Cancer Res 25: 4117-4127'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Recondo G, Bahcall M, Spurr LF, Che J, Ricciuti B, Leonardi GC, Lo YC, Li YY, Lamberti G, Nguyen T et al (2020) Molecular Mechanisms of Acquired Resistance to MET Tyrosine Kinase Inhibitors in Patients with MET Exon 14-Mutant NSCLC. Clin Cancer Res 26: 2615-2625\\n\\nRitchie ME, Phipson B, Wu D, Hu Y, Law CW, Shi W, Smyth GK (2015) limma powers differential expression analyses for RNA-sequencing and microarray studies. Nucleic Acids Res 43: e47\\n\\nSaddoughi SA, Gencer S, Peterson YK, Ward KE, Mukhopadhyay A, Oaks J, Bielawski J, Szulc ZM, Thomas RJ, Selvam SP et al (2013) Sphingosine analogue drug FTY720 targets I2PP2A/SET and mediates lung tumour suppression via activation of PP2A-RIPK1-dependent necroptosis. EMBO Mol Med 5: 105-121'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Saura C, Bendell J, Jerusalem G, Su S, Ru Q, De Buck S, Mills D, Ruquet S, Bosch A, Urruticoechea A et al (2014) Phase Ib study of Buparlisib plus Trastuzumab in patients with HER2-positive advanced or metastatic breast cancer that has progressed on Trastuzumab-based therapy. Clin Cancer Res 20: 1935-1945\\n\\nSaura C, Oliveira M, Feng YH, Dai MS, Chen SW, Hurvitz SA, Kim SB, Moy B, Delaloge S, Gradishar W et al (2020) Neratinib Plus Capecitabine Versus Lapatinib Plus Capecitabine in HER2-Positive Metastatic Breast Cancer Previously Treated With 2 HER2-Directed Regimens: Phase III NALA Trial. J Clin Oncol 38: 3138-3149'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Schneeweiss A, Park-Simon TW, Albanell J, Lassen U, Cortes J, Dieras V, May M, Schindler C, Marme F, Cejalvo JM et al (2018) Phase Ib study evaluating safety and clinical activity of the anti-HER3 antibody lumretuzumab combined with the anti-HER2 antibody pertuzumab and paclitaxel in HER3-positive, HER2-low metastatic breast cancer. Invest New Drugs 36: 848-859\\n\\nSharma SV, Lee DY, Li B, Quinlan MP, Takahashi F, Maheswaran S, McDermott U, Azizian N, Zou L, Fischbach MA et al (2010) A chromatin-mediated reversible drug-tolerant state in cancer cell subpopulations. Cell 141: 69-80'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Shojaee S, Caeser R, Buchner M, Park E, Swaminathan S, Hurtz C, Geng H, Chan LN, Klemm L, Hofmann WK et al (2015) Erk Negative Feedback Control Enables Pre-B Cell Transformation and Represents a Therapeutic Target in Acute Lymphoblastic Leukemia. Cancer Cell 28: 114-128\\n\\nThompson EM, Patel V, Rajeeve V, Cutillas PR, Stoker AW (2022) The cytotoxic action of BCI is not dependent on its stated DUSP1 or DUSP6 targets in neuroblastoma cells. FEBS Open Bio 12: 1388-1405\\n\\nVainonen JP, Momeny M, Westermarck J (2021) Druggable cancer phosphatases. Sci Transl Med 13'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Wang SP, Wu SQ, Huang SH, Tang YX, Meng LQ, Liu F, Zhu QH, Xu YG (2021) FDI-6 inhibits the expression and function of FOXM1 to sensitize BRCA-proficient triple-negative breast cancer cells to Olaparib by regulating cell cycle progression and DNA damage repair. Cell death & disease 12: 1138\\n\\nWilson TR, Fridlyand J, Yan Y, Penuel E, Burton L, Chan E, Peng J, Lin E, Wang Y, Sosman J et al (2012) Widespread potential for growth-factor-driven resistance to anticancer kinase inhibitors. Nature 487: 505-509\\n\\nWu QN, Liao YF, Lu YX, Wang Y, Lu JH, Zeng ZL, Huang QT, Sheng H, Yun JP, Xie D et al (2018) Pharmacological inhibition of DUSP6 suppresses gastric cancer growth and metastasis and overcomes cisplatin resistance. Cancer Lett 412: 243-255'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Xie T, Lim SM, Westover KD, Dodge ME, Ercan D, Ficarro SB, Udayakumar D, Gurbani D, Tae HS, Riddle SM et al (2014) Pharmacological targeting of the pseudokinase Her3. Nat Chem Biol 10: 1006-1012\\n\\nYonesaka K, Zejnullahu K, Okamoto I, Satoh T, Cappuzzo F, Souglakos J, Ercan D, Rogers A, Roncalli M, Takeda M et al (2011) Activation of ERBB2 signaling causes resistance to the EGFR-directed therapeutic antibody cetuximab. Sci Transl Med 3: 99ra86\\n\\nZandi Z, Kashani B, Alishahi Z, Pourbagheri-Sigaroodi A, Esmaeili F, Ghaffari SH, Bashash D, Momeny M (2022) Dual-specificity phosphatases: therapeutic targets in cancer therapy resistance. J Cancer Res Clin Oncol 148: 57-70'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Zhang S, Huang WC, Zhang L, Zhang C, Lowery FJ, Ding Z, Guo H, Wang H, Huang S, Sahin AA et al (2013) SRC family kinases as novel therapeutic targets to treat breast cancer brain metastases. Cancer Res 73: 5764-5774\\n\\nZhang YL, Ma Y, Zeng YQ, Liu Y, He EP, Liu YT, Qiao FL, Yu R, Wang YS, Wu XY et al (2021) A narrative review of research progress on FoxM1 in breast cancer carcinogenesis and therapeutics. Ann Transl Med 9: 1704\\n\\nZheng R, Wan C, Mei S, Qin Q, Wu Q, Sun H, Chen CH, Brown M, Zhang X, Meyer CA et al (2019) Cistrome Data Browser: expanded datasets and new tools for gene regulatory analysis. Nucleic Acids Res 47: D729-D735\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nFigure legends'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Figure 1: Transcriptional landscape of lapatinib tolerance and resistance development in HER2+ cells. (A) Development of lapatinib resistance in HER2+ breast cancer cells. BT474 and BT474Br cells were treated with 1 M of lapatinib for 9 days (d), 6 months (mos) and 9 mos to yield Drug-tolerant persister (DTP), Drug-tolerant expanding perister (DTEP) and Long term resistant (LR) clones, respectively. The cells were stained with crystal violet (0.5% w/v) and the images were acquired with an inverted microscope. The transcriptional profiles from each functional state of lapatinib drug tolerance and resistance development from three parallel BT474 cell plates were surveyed by RNA-sequencing. (B) The number of the genes with a significant change in their'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='(B) The number of the genes with a significant change in their expression during the resistance acquisition (see Dataset EV1 for individual gene names). Transcription factor (TF) binding motifs significantly enriched in significantly regulated genes in each transition are indicated. Bolding indicates shared TF binding sites between DTP downregulated and DTEP upregulated genes. (C) Differentially expressed pathways were identified using the R package limma and hallmark gene sets were used for GSVA analysis to reveal hallmarks and signal transduction pathways involved in each step of the resistance acquisition. Red indicates those hallmarks and pathways that are overlapping with processes regulated by DUSP6 depletion in figure 3F.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Figure 2: Clinical association of DUSP6 overexpression with poor prognosis HER2+ breast cancers. (A, B) Volcano plots visualizing differentially expressed genes in (A) Control-DTP and (B) DTP-DTEP transitions. The volcano blots indicate all genes that were significantly regulated during these transitions (|logFC |<2 and FDR <0.05), whereas only the phosphatase genes among these are indicated by names. The four phosphatase genes significantly regulated in both transitions (DUSP6, CDC25A, CDC25C, and SYNJ1) are indicated in bold. Differentially expressed genes were identified using the R package limma (n=3). (C) Changes in the DUSP6, CDC25A, CDC25C, and SYNJ1 mRNA levels during the acquisition of lapatinib resistance in BT474 cells. Data is based on RNA'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content=\"of lapatinib resistance in BT474 cells. Data is based on RNA sequencing analysis (Dataset EV1) and was analyzed by one-way ANOVA followed by Tukey's multiple comparisons test. Statistically significant values of *p < 0.05, **p < 0.01 and ***p < 0.001 were determined (n=3). (D) Differential expression of DUSP6, CDC25A, CDC25C, and SYNJ1 in different breast cancer subtypes. Data were extracted from the METABRIC dataset and categorized into five molecular subtypes according to the PAM50 gene expression subtype classification (basal, claudin-low, HER2+, Luminal A, and Luminal B). Data were analyzed by one-way ANOVA followed by Tukey's multiple comparisons test and shown as mean  standard deviation (SD). Statistically significant values of *p < 0.05, **p < 0.01\"), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='(SD). Statistically significant values of *p < 0.05, **p < 0.01 and ****p < 0.0001 were determined (basal=209, claudin-low=218, HER2+=224, LumA=700 and LumB=475). (E) Breast cancer patients from the TCGA-BRCA dataset were divided into DUSP6 high (LogFC>1, FDR<0.05) and low expression (LogFC<-1, FDR<0.05) groups and the clinical breast cancer subtypes were compared between the two groups. NA; not available. (F,G) Subgroup of 113 patient cases with high tumor ERBB2 mRNA expression (LogFC>1, FDR<0.05) were divided into DUSP6high and DUSP6low groups and their overall survival (OS) (G) (Log-rank Test p value=0.0220) and disease-specific progression free survival (PFS)(H)(Log-rank Test p value=0.0259) was tested according to DUSP6 status.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Figure 3: Functional involvement of DUSP6 in HER2i tolerance development.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='(A) Expression of DUSP6 protein in different stages of lapatinib resistance development by Western blot analysis. (B) Relative expression of DUSP6 mRNA in indicated cell lines either at the untreated control situation, or in the DTP state after following treatments: BT474Br, Lapatinib 1M for 9 days; EFM192A and HCC1419, Lapatinib 2.5 M for 14 days; HCC827(EGFRmut NSCLC), 1 µM Osimertinib for 10 days; A375(BRAFV600E mutant malignant melanoma (MM)), 1 µM dabrafenib+100 nM trametinib for 10 days; and HT-29(BRAFV600E mutant colorectal cancer (CRC)), 1 µM dabrafenib+10 µg/ml cetuximab for 10 days. Data for EFM192 and HCC1419 cells was obtained from (DataRef: (Chang et al., 2022)(GSE155342), and for other cells by qRT-PCR analysis of the de novo treated samples.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content=\"other cells by qRT-PCR analysis of the de novo treated samples. Shown is data from two-three repeat samples. (C) Transcriptional profile of MDA-MB-453 cells after DUSP6 knockdown by 3 different siRNA and compared with 3 different scramble controls, followed by the GSVA analysis of the Hallmark gene sets. The Hallmark gene sets overlapping with the gene sets regulated to same direction in DUSP6 low expressing DTEP cells (Fig. 1C) are indicated with red.  (D, E) Ectopic overexpression of DUSP6 in BT474 cells inhibits lapatinib effects on cell viability (G) and apoptosis (H), as measured by WST1 cell viability assay and caspase 3/7 activity, respectively. Data was analyzed by two-way ANOVA followed by Tukey' post hoc test.  Statistically significant values of\"), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content=\"by Tukey' post hoc test.  Statistically significant values of **p < 0.01 and ****p < 0.0001 were determined. n=3.  (F) Ectopic overexpression of either wild-type DUSP6 (DUSPWT) or ERK binding deficient KIM mutant of DUSP6 (DUSP6MUT) in BT474 cells inhibits lapatinib effects on apoptosis. Shown is a result from a representative experiment from three repeats with similar results. Data were analyzed by two-way ANOVA followed by Tukey' post hoc test.  Statistically significant values of **p < 0.01 and ****p < 0.0001 were determined. (G) DUSP6 inhibitor BCI preempts DTEP development in BT474 cells treated with either lapatinib or neratinib for 6 months. The cells were stained and fixed with crystal violet in methanol (0.5% w/v) and the images were acquired with\"), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='violet in methanol (0.5% w/v) and the images were acquired with an inverted microscope.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Figure 4: Both genetic and pharmacological DUSP6 targeting overcomes HER2i resistance. (A) RNAi-mediated DUSP6 knockdown, but not of DUSP1, induces apoptotic cell death in MDA-MB-453 cells, as shown by Western blotting for PARP-1 cleavage. (B) CRISPR/CAS9-mediated DUSP6 knockout hinders the clonogenic growth of MDA-MB-453 cells, as compared to the CAS9 expressing controls. Shown are three independent single cell clones created with two independent gRNAs (g1 and g2). The cells were seeded at low density and maintained for 10 d. The colonies were stained/fixed with 0.5% crystal violet in methanol and imaged using an inverted microscope. (C) DUSP6 siRNA knockdown increases sensitivity of HER2i resistant MDA-MB-453 cells to HER2-targeted therapies. Cell'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content=\"resistant MDA-MB-453 cells to HER2-targeted therapies. Cell viability was measured by WST-1 assay after 48 h of drug treatment. Data were collected from three independent experiments each performed in triplicate and analyzed by one-way ANOVA followed by Tukey's multiple comparisons test. Statistically significant values of **p < 0.01, ***p < 0.001 and ****p < 0.0001 were determined. (D) A 2D synergy map of neratinib-BCI combination in MDA-MB-453 cells calculated by Bliss SynergyFinder (Ianevski et al., 2017). Higher score (in red) indicates for higher degree of drug synergy. The cultures were treated with increasing concentrations of the compounds for 48 h and cell viability was measured by WST-1 assay. (E) DUSP6 siRNA knockdown increases sensitivity of\"), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content=\"WST-1 assay. (E) DUSP6 siRNA knockdown increases sensitivity of HER2i resistant MDA-MB-453 cells to combination with capecitabine and neratinib. Cell viability was measured by WST-1 assay after 48 h of drug treatment. Data were collected from three independent experiments each performed in triplicate and analyzed by one-way ANOVA followed by Tukey's multiple comparisons test. Statistically significant values of ****p < 0.0001 were determined.\"), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content=\"Figure 5: DUSP6 inhibition overcomes HER2 inhibitor resistance in vivo. (A) Subcutaneous xenograft growth of two independent DUSP6 single cell knockout clones of MDA-MB-453 cells targeted with two different gRNA guides. One-way ANOVA followed by Tukey's multiple comparisons test **p < 0.01 comparing each clone to CAS9 expressing control cells . Data are shown as mean  SD (n=5) (B-E) The effect of BCI in combination with lapatinib or neratinib in two HER2 inhibitor resistant xenograft models: HCC1954 (B, C) or MDA-MB-453 (D, E). Data are shown as mean  SD (n=10 in each treatment group). Mice with tumor size ∼100 mm3 were randomized into the experimental and the control groups and tumor volumes were measured every 3 d. Data were analyzed by one-way ANOVA\"), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content=\"were measured every 3 d. Data were analyzed by one-way ANOVA followed by Tukey's multiple comparisons test. Statistically significant values of ****p < 0.0001 were determined. C and E panels display percentual change in the tumor volume from the start of the therapy as water-fall blots in HCC1954 and MDA-MB453 models, respectively. (F) H&E staining of the representative MDA-MB-453 xenograft tumors from the control, lapatinib, BCI and lapatinib+BCI groups at day 24. Scale bar 200 µm.\"), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Figure 6: DUSP6 targeting inhibits HER3 expression  (A) A 2D synergy map of lapatinib+MK2206(AKTi) or (B) lapatinib+BCI(DUSP6i) combination in MDA-MB-453 cells calculated by the Bliss SynergyFinder. Higher score (in red) indicates for higher degree of drug synergy. The cultures were treated with increasing concentrations of the compounds for 48 h and cell viability was measured by WST-1 assay. (C) Comparison of the effects of lapatinib+BCI and lapatinib+MK2206 on apoptosis induction (PARP-1 cleavage), and HER2 or HER3 protein levels by Western blot analysis. The cells were treated with lapatinib (1 M), MK2206 (2.5 µM) and BCI (2.5 µM) and their combinations for 48 h. (D) Quantification of PARP1 cleavage from three repeats of (C). Data were analyzed by'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content=\"PARP1 cleavage from three repeats of (C). Data were analyzed by one-way ANOVA followed by Tukey's multiple comparisons test. Statistically significant values of **p < 0.01 and ****p < 0.0001 were determined. (E,F) The dose-dependent effects of MK2206 (E) and BCI (F) on the expression of HER2 and HER3 protein levels in MDA-MB-453 cells by Western blot analysis after 48h of treatment. (G) The time-dependent effects of BCI (2.5 µM) on the expression of HER2 and HER3 protein levels and apoptosis induction (PARP cleavage) in MDA-MB-453 cells by Western blot analysis. Increase in phosphorylated ERK (p-ERK1/2) and inhibition of DUSP6 both indicate for early target engagement by BCI.  (H) DUSP6 knockdown by siRNA inhibits HER2 and HER3 protein expression in\"), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='knockdown by siRNA inhibits HER2 and HER3 protein expression in MDA-MB-453 cells. (I) Effects of BCI (50 mg/kg) therapy on HER2 and HER3 protein levels in the MDA-MB-453 xenograft tissue on day 24. Shown is immunohistochemical analysis of HER2 and HER3 from the adjacent paraffin embedded tissue slices from Fig. 5F. Scale bar 200 µm. (J) Breast cancer patients from the TCGA-BRCA dataset were divided into DUSP6 high (LogFC>1, FDR<0.05) and low expression (LogFC<-1, FDR<0.05) profiles and expression of phosphorylated HER3 (p-HER3Y1298) was compared between the two groups. Data were analyzed by two-tailed t test; **p < 0.01. (DUSP6high=142, DUSP6low=149).'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Fig. 7. DUSP6 targeting overcomes Neuregulin/HER3-mediated HER2i therapy tolerance'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='(A) Comparison of the potential of BCI or MK2206 treatment to overcome Neuregulin (NRG)-mediated rescue from the anti-proliferative activity of neratinib. The cells were treated with NRG (10 ng/mL), lapatinib (1\\xa0µM), MK2206 (1, 2.5, and 5 µM), and BCI (1, 2.5, and 5 µM) for 48 h and cell viability was measured by WST-1 assay.  Data were collected from three independent experiments each performed with three technical repeat samples. (B) Comparison of the effects of BCI and MK2206 on NRG-mediated evasion from neratinib-induced apoptotic cell death, as measured by Western blot analysis for cleaved PARP-1. The cells were treated with NRG (10 ng/mL), lapatinib (1 M), MK2206 (2.5 µM) and BCI (2.5 µM) for 48 h. (C) HER3 overexpression rescues MDA-MB-453 cells from'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='for 48 h. (C) HER3 overexpression rescues MDA-MB-453 cells from BCI-elicited inhibition of cell viability. The control or HER3 overexpressing MDA-MB-453 were treated with BCI (3 µM) for 48 h and cell viability was measured by WST-1 assay. Shown is data from four technical replicate samples from a representative of three experiments with similar results. The data was analysed by Two-way ANOVA + Tukey’s post hoc test, ****p < 0.0001 (D) Comparison of the effects of lapatinib+BCI and lapatinib+MK2206 on NRG-mediated rescue from the anti-growth activity of lapatinib, as shown by crystal violet staining. The cells were treated with NRG (10 ng/mL), lapatinib (1 M), MK2206 (2.5 µM) and BCI (2.5 µM) for 48 h, stained/fixed with 0.5% crystal violet in methanol and'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='48 h, stained/fixed with 0.5% crystal violet in methanol and imaged by an inverted microscope (images acquired at 10x magnification). (E) Breast cancer patients from the TCGA-BRCA dataset were divided into DUSP6 high (LogFC>1, FDR<0.05) and low expression (LogFC<-1, FDR<0.05) profiles and the neuregulin (NRG1) mRNA levels were compared between the two groups. Data were analyzed by two-tailed t test; ****p < 0.0001. (DUSP6high=167, DUSP6low=181).  (F) The effect of DUSP6 knockdown on the brain metastatic outgrowth of MDA-MB-361 cells in a zebrafish model. GFP positive MDA-MB-361 cells transfected either with control scrambled siRNA or DUSP6 siRNA were injected into zebrafish embryo brain and the GFP intensity was measured 3 days after by microscopy. Data'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='the GFP intensity was measured 3 days after by microscopy. Data were analyzed by two-tailed t test; ****p < 0.0001. Scale bar 100 µm (siSCR=37, siDUSP6=43). (G) Improved overall survival of mice with intracranially injected DUSP6 KO MDA-MB-453 cells as compared to the CAS9 positive control cell injected mice. Survival data were analyzed by log-rank Mantel–Cox test, **p < 0.01.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Figure 8. Feed-forward loop between NRG/HER3 and DUSP6 determines HER2i sensitivity'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='(A) DUSP1 and DUSP6 mRNA levels were determined by qRT-PCR analysis after treatment with increasing concentrations of neratinib for 48 h in indicated cell lines. Red denotes for HER2i resistant cell lines and green HER2i sensitive cells. Data were collected from three independent experiments each performed in triplicate. (B, C) Comparison of the effect of neratinib treatment (48 h) on DUSP6, p-AKT and p-ERK1/2 between HER2i sensitive BT474Br (B) and HER2i resistant MDA-MB-361 (C) cells, respectively. (D) The effect of siRNA-mediated HER3 knockdown on DUSP6 expression in MDA-MB-453 cells by Western blot analysis. (E) NRG-mediated induction of DUSP6 mRNA via MEK activation as measured by qRT-PCR analysis after treatment with NRG (10 ng/mL), MK2206(AKTi)'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content=\"analysis after treatment with NRG (10 ng/mL), MK2206(AKTi) (2.5\\xa0µM), and trametinib(MEKi) (100 nM)\\xa0for 48 h. Data were analyzed by one-way ANOVA followed by Tukey's multiple comparisons test. Statistically significant values of *p < 0.05 and ****p < 0.0001 were determined (n=3). (F) The effect of NRG on DUSP6 protein expression BT474 cells via MEK activation. The cells were serum-starved for 24 h, followed by treatment with NRG (10 ng/mL), MK2206 (2.5 µM) and trametinib (100 nM) for 48 h. (G) Inhibition of DUSP6 expression in HER2i resistant MDA-MB-453 cells by MEKi Trametinib. The cells were treated with MK2206 (2.5 µM) or trametinib (100 nM) for 48 h. (H) A schematic illustration of the discovered HER3/DUSP6 feed forward loop in HER2+ breast cancer cells.\"), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='HER3/DUSP6 feed forward loop in HER2+ breast cancer cells. NRG binding to HER3 induces MEK/ERK-mediated DUSP6 expression which feeds back to increased HER2 and HER3 expression (left panel). In HER2i sensitive cells (middle panel) inhibition of HER3 results in DUSP6 inhibition and loss of DUSP6 driven cancer hallmarks. In HER2i resistant cells (right panel), MEK is not inhibited by HER2i but its constitutive activity (MEKca.) drives DUSP6-HER2/3 positive feed-back loop resulting in HER3-mediated multitherapy resistance and cancer progression.'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Expanded view figure legends\\n\\n\\n\\nFigure EV1: Profiling of a panel of HER2+ breast cancer cell lines for their sensitivity to HER2 targeting small molecule tyrosine kinase inhibitors \\n\\nThe primary HER2i sensitive cells are marked on green, and the acquired HER2i resistant cells in red. The long-term resistant (LR) BT-474 and BT-474Br generated by 9-month treatment with lapatinib in this study are denoted in bold. The cells were treated with the increasing concentrations of the indicated HER2i compounds (in µM) for 48 h and cell viability was measured using WST-1 assay.\\n\\n\\n\\nFigure EV2: E2F1 and FOXM1 target genes are inhibited across HER2+ DTP cells'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='A) mRNA levels of selected predicted FOXM1 and E2F1 target genes regulated during different steps of acquisition of lapatinib resistance in BT474 cells (Fig. 1). Data is blotted based on RNA sequencing analysis (Dataset EV1). (B, C) Expression of the predicted FOXM1 and E2F1 target genes in HER2+ HCC1419 and EFM192A cells between the DTP and the control cells. The RNA-seq data was obtained from (Dataref: (Chang et al., 2022)(GSE155342). The cells were treated with lapatinib (2.5 µM) for 14 days to reach the DTP state. Data are shown as mean  SD (n=2) (D) Changes in the expression of FOXM1, PLK1, CCNB1, E2F1, CCNE2 and CDC6 in BT474Br cells between the control and the DTP cells treated for 9 days with lapatinib (1 µM). Data is based on Q-PCR analysis from'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='with lapatinib (1 µM). Data is based on Q-PCR analysis from three technical replicates. The analysis was limited to only these genes due to lack of sufficient mRNA material from the strongly growth suppressed DTP cells. Data are shown as mean  SD (n=3).'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Figure EV3: DUSP6 overexpression protects HER2+ cells from HER2i-induced  cell death'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content=\"(A, B) Ectopic overexpression of DUSP6 in BT474 cells inhibits neratinib-elicited effects on cell viability and apoptosis, as measured by WST1 cell viability assay and caspase 3/7 activity, respectively. Data were collected from three independent experiments each performed in triplicate and analyzed by two-way ANOVA followed by Tukey' post hoc test.  Statistically significant values of ****p < 0.0001 were determined. (C) Ectopic overexpression of DUSP6 in BT474 cells inhibits the afatinib-elicited effects on cell viability, as measured by WST1 assay. Data were collected from three independent experiments each performed in triplicate and analyzed by two-way ANOVA followed by Tukey' post hoc test. Statistically significant values of ****p < 0.0001 were\"), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content=\"test. Statistically significant values of ****p < 0.0001 were determined. (D) Ectopic overexpression of DUSP6, but not of DUSP1, inhibits the Tucatinib-elicited effects on cell viability, as measured by WST1 assay. Data were collected from three independent experiments each performed in triplicate and analyzed by two-way ANOVA followed by Tukey' post hoc test. Statistically significant values of ****p < 0.0001 were determined.\"), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='Figure EV4: The anti-proliferative activities of a library of small molecule modulators of phosphatases, kinases, and anti-apoptotic proteins in HER2i resistant cells. The indicated cells were treated with the increasing concentrations (in M) of the compounds for 48 h and cell viability was measured using WST-1 assay. The long-term resistant (LR) BT474 and BT474Br generated de novo by 9-month treatment with lapatinib in this study had comparable drug sensitivity profile to acquired resistant cell line MDA-MB-453. The primary target of the used compound is indicated in parenthesis. \\n\\n\\n\\nFigure EV5: Lack of DUSP6 inhibition is associated with HER2i resistance'), Document(metadata={'source': './data/docx/EMM-2023-18636.docx'}, page_content='(A,B) The effects of tucatinib on DUSP6 expression and the signaling pathway activities in  (A) HER2i sensitive (green) BT474 cells or (B) HER2i resistant (red) MDA-MB-453 and BT474BrLR cells. The cells were treated with increasing concentrations of tucatinib for 48 h, followed by Western blot analysis. (C) DUSP6 expression in MDA-MB-453 cells is resistant to multiple HER2is including antibody therapy with trastuzumab. The cells were treated with indicated drugs for 48 h, followed by Western blot analysis. \\n\\n\\n\\n\\n\\n1')]\n",
      "Processing document: ./data/docx/EMM-2023-19044.docx\n",
      "Split documents: [Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Research Article\\n\\nTitle: Ependymal cell lineage reprogramming as a potential therapeutic intervention for hydrocephalus\\n\\n\\n\\nAuthors:  Konstantina Kaplani1, Maria - Eleni Lalioti1, Stella Vassalou1, Georgia Lokka1, Evangelia Parlapani1, Georgios Kritikos1, Zoi Lygerou2, Stavros Taraviras1,\\n\\n\\n\\nAffiliations: \\n\\n1Department of Physiology, School of Medicine, University of Patras, Patras, Greece\\n\\n2Department of General Biology, School of Medicine, University of Patras, Patras, Greece\\n\\n\\n\\nCorresponding author: Stavros Taraviras\\n\\nCorresponding author’s address: Department of Physiology, School of Medicine, University of Patras, Asklepiou Street 1, Rio 26504, Patras, Greece\\n\\nCorresponding author’s phone and fax: +30 2610 997676'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Corresponding author’s phone and fax: +30 2610 997676\\n\\nCorresponding author’s e-mail address: taraviras@med.upatras.gr\\n\\n\\n\\nRunning title: GemC1 and McIdas promote ependymal reprogramming. \\n\\n\\n\\nKeywords: ependymal cells, reprogramming, hydrocephalus, McIdas, GemC1\\n\\n\\n\\n\\nABSTRACT'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Hydrocephalus is a common neurological condition, characterized by the excessive accumulation of cerebrospinal fluid in the cerebral ventricles. Primary treatments for hydrocephalus mainly involve neurosurgical cerebrospinal fluid diversion, which hold high morbidity and failure rates, highlighting the necessity for the discovery of novel therapeutic approaches. Although the pathophysiology of hydrocephalus is highly multifactorial, impaired function of the brain ependymal cells plays a fundamental role in hydrocephalus. Here we show that GemC1 and McIdas, key regulators of multiciliated ependymal cell fate determination, induce direct cellular reprogramming towards ependyma. Our study reveals that ectopic expression of GemC1 and McIdas reprograms cortical'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='that ectopic expression of GemC1 and McIdas reprograms cortical astrocytes and programs mouse embryonic stem cells into ependyma. McIdas is sufficient to establish functional activity in the reprogrammed astrocytes. Furthermore, we show that McIdas’ expression promotes ependymal cells regeneration in two different postnatal hydrocephalus mouse models: an intracranial hemorrhage and a genetic form of hydrocephalus and ameliorates the cytoarchitecture of the neurogenic niche. Our study provides evidence on the restoration of ependyma in animal models mimicking hydrocephalus that could be exploited towards future therapeutic interventions.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='INTRODUCTION'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Hydrocephalus is a neurological disorder defined by the abnormal accumulation of cerebrospinal fluid (CSF) within the brain ventricles, leading to increased head circumference (Schrander-Stumpel & Fryns, 1998). The condition can arise as a primary clinical feature triggered by a genetic cause (congenital hydrocephalus) or secondary (acquired hydrocephalus) to other insults such as central nervous system (CNS) infections, tumors, trauma, or hemorrhage (Rekate, 2009; Kahle et al, 2016; Kousi & Katsanis, 2016). Malfunction of the ependymal cells is a leading cause of the pathophysiological mechanism of hydrocephalus as these cells have a key role in CSF circulation and composition (Ji et al, 2022). In addition, hydrocephalus is linked with disorganization of'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='In addition, hydrocephalus is linked with disorganization of the subventricular neurogenic niche’s  cytoarchitecture and neurogenesis defects, as ependymal cells were shown to provide structural and functional support to the adult neural stem cells (Paez-Gonzalez et al, 2011; McAllister, James P. et al, 2017; Rodríguez & Guerra, 2017).'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Under homeostatic conditions, ependymal cells form one of the largest epithelia of the brain, lying adjacent to the ventricular lumen and bear unique structural characteristics. The cell surface exposed to the ventricular lumen contains a patch of approximately 50 modified centrioles, called basal bodies, which are tiny microtubule-based organelles responsible for nucleating an equal number of motile cilia. Cilia  are anchored οn their apical surface, beating in the same direction and in a coordinated fashion, thus contributing to efficient CSF circulation throughout the brain (Mirzadeh et al, 2008, 2010). Constant ciliary beating facilitates secreted molecules dispersion to other regions of the CNS. Such circulating molecules in the CSF influence'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='of the CNS. Such circulating molecules in the CSF influence neurogenesis and neuroblasts’ migration to the olfactory bulbs (Lim et al, 2000; Sawamoto et al, 2006).'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Ependymal cell differentiation is a multi-step process orchestrated by a tightly regulated transcriptional program which is responsible for activating numerous genes that promote amplification and docking of centrioles and the formation of cilia (Kyrousi et al, 2017; Arbi et al, 2018). We have previously provided evidence that the Geminin family proteins, GEMC1 and MCIDAS, are the earliest regulators for the cell fate commitment to the ependymal lineage (Kyrousi et al, 2015). GemC1 lies upstream of McIdas inducing its transcriptional activation, while both upregulate genes that are essential for centriole amplification and for promoting the downstream transcriptional machinery of multiciliogenesis (Kyrousi et al, 2015; Arbi et al, 2016; Lalioti et al,'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(Kyrousi et al, 2015; Arbi et al, 2016; Lalioti et al, 2019a). GEMC1 and MCIDAS cooperate with E2F4/5 transcription factors, to upregulate cMyb and Ccno, which are implicated in centriole amplification (Tan et al, 2013; Wallmeier et al, 2014; Funk et al, 2015; Arbi et al, 2018). In addition, we and others have provided evidence that GemC1 and McIdas drive the transcriptional activation of P73 and FOXJ1 transcription factors, which are implicated in basal bodies docking and cilia motility (Kyrousi et al, 2015; Arbi et al, 2016; Nemajerova et al, 2016; Marshall et al, 2016; Lalioti et al, 2019a).'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Given the multifaceted roles that ependymal cells have in hydrocephalus pathophysiology, restoration of ependymal cells could significantly contribute to hydrocephalus therapy. With the advancements in direct reprogramming-based strategies for conditions affecting the central nervous system (Bocchi et al, 2022), it is plausible to anticipate that cellular reprogramming directed towards ependymal cells could represent a significant breakthrough in the treatment of hydrocephalus. In the present study we showed that ectopic expression of GemC1 or McIdas promotes programming of mouse pluripotent embryonic stem cells and reprogramming of cortical astrocytes into ependymal cells, with McIdas being more efficient on establishing functional motile cilia in'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='being more efficient on establishing functional motile cilia in reprogrammed astrocytes. In addition, forced expression of McIdas in both a genetic and an acquired hydrocephalus mouse model promoted in vivo reprogramming of cells residing in the ventricular walls of the diseased brain into functional ependymal cells. Importantly, reprogrammed ependymal cells were able to beat their cilia in a coordinated fashion and formed pinwheel structures together with neural stem cells in hydrocephalic mice, revealing their potential to regenerate the architecture of the neurogenic niche, which is disrupted in hydrocephalus. Collectively, our data suggest that GEMC1 and MCIDAS can orchestrate the transcriptional program of multiciliogenesis establishing ependymal cell'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='program of multiciliogenesis establishing ependymal cell fate and differentiation, providing proof of principle evidence that restoration of ependymal cells could potentially contribute to hydrocephalus management.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='RESULTS\\n\\nGemC1 and McIdas promote programming of mouse embryonic stem cells and reprogramming of cortical astrocytes into ependymal cells'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='GemC1 and McIdas are the most upstream regulators driving multiciliogenesis in vertebrate organisms (Stubbs et al, 2012; Ma et al, 2014; Boon et al, 2014; Kyrousi et al, 2015; Terré et al, 2016; Arbi et al, 2016). We therefore postulated that GemC1 and McIdas would be capable of inducing stem cell programming towards multiciliated ependymal cells. To examine this hypothesis, mouse embryonic stem cells (mESCs) were transfected with plasmids encoding McIdas or GemC1 in conjunction with GFP (referred to as McIdas and GemC1 respectively), or GFP alone as a control. Transfected mESCs were cultured for four days in the absence of Leukemia Inhibitory Factor (LIF) and subjected to immunostaining for FoxJ1, a key transcription factor for the differentiation of'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='FoxJ1, a key transcription factor for the differentiation of ependymal cells, and Pericentrin which marks nascent basal bodies (Fig EV1A). Our analysis showed that 87% of McIdas- and 39% of GemC1-overexpressing mESCs were expressing FoxJ1, while only 1% FoxJ1 positive cells were detected in the control (Fig EV1B). In addition, we identified that 90% of McIdas- and 59% of GemC1-overexpressing mESCs showed accumulation of Pericentrin staining (Fig EV1C), thus presented multiple nascent basal bodies, corresponding to differentiating ependymal cells. Furthermore, we examined whether McIdas and GemC1-overexpressing mESCs acquire multiple cilia, a unique structural characteristic of the ependymal cells. Double immunofluorescence experiments were performed six'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='cells. Double immunofluorescence experiments were performed six days following mESCs transfection using anti-Pericentrin and anti-acetylated-a-tubulin antibodies to mark the basal bodies and cilia, respectively (Fig EV1D). Our analysis revealed that 66% of McIdas- and 21% of GemC1- overexpressing mESCs develop multiple cilia arising from numerous basal bodies.  On the contrary, control mESCs develop neither multiple basal bodies nor cilia (Fig EV1D, E). Our data show that ectopic expression of McIdas and GemC1 programs mESCs into multiciliated ependymal cells.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Next, we sought out to explore the reprogramming potential of McIdas and GemC1 in astrocytes that under physiological conditions do not differentiate into ependymal cells. Astrocytes were isolated from the cortex of neonatal mice and were subsequently infected with lentiviruses expressing GFP-McIdas or GFP-GemC1 (referred to as McIdas and GemC1, respectively), while lentiviruses expressing GFP alone were used as control. Cells were subsequently cultured under differentiating conditions and immunofluorescence experiments were conducted to assess the expression of known ependymal and astrocytic markers (Fig. 1A). Ectopic expression of both McIdas and GemC1 resulted in P73 expression, a marker of the early steps of ependymogenesis, in 37% and 36% of McIdas-'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='the early steps of ependymogenesis, in 37% and 36% of McIdas- and GemC1- infected cells, respectively, while in the control condition only 4% of the transduced cells was expressing P73 (Fig. 1B, C, and EV 2A, B). Notably, concurrent changes in the expression pattern of the astrocytic marker S100β were observed in approximately 20% of McIdas- and 18% of GemC1- infected astrocytes, as S100β expression was detected around their cell body, which is characteristic for ependymal cells (Mirzadeh et al, 2008) (Fig. 1B-D and EV. 2A-C). On the contrary, control astrocytes retained a cytoplasmic expression of the S100β protein, a typical feature of astrocytes.  Overall, our data suggest that ectopic expression of McIdas and GemC1 can lead to the acquisition of the'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='of McIdas and GemC1 can lead to the acquisition of the ependymal cell fate.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Subsequently, we examined whether McIdas- and GemC1-transduced astrocytes successfully differentiate into ependymal cells. We performed immunofluorescence experiments using specific antibodies recognizing FoxJ1 and Pericentrin, to identify astrocytes that acquired the ependymal fate and successfully managed to multiply their basal bodies, respectively. Our analysis showed that fourteen days post infection 49% of McIdas- and 19% of GemC1- infected astrocytes were expressing FoxJ1 (Fig. 1E, F and EV2D, E). In addition, 28% of the McIdas- and 12% of GemC1- infected astrocytes possessed multiple basal bodies, indicated by the accumulation of Pericentrin signal (Fig. 1G and EV2F). Moreover, expression of a GFP lentivirus did not result into FoxJ1 expression nor'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='of a GFP lentivirus did not result into FoxJ1 expression nor accumulation of Pericentrin.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='In addition, we examined the ability of McIdas and GemC1 to repress the astrocytic identity of the transduced astrocytes. Towards this direction, the combined expression of the astrocytic markers GFAP and S100β was assessed through immunofluorescence experiments at fourteen days post infection. Our analysis revealed that McIdas ectopic expression resulted to 23% of GFAP+/S100β+ double positive cells compared to 31% in the control condition (Appendix Fig. S1A, B). We did not observe statistically significant change in the percentage of GFAP+/S100β+ double positive cells following GemC1 ectopic expression (Appendix Fig. S1C). In addition, we examined the percentage of infected cells that were GFAP- and reveal S100β expression around the cell body, which'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='GFAP- and reveal S100β expression around the cell body, which correspond to ependymal cells. We showed that 28% McIdas- and 22% GemC1- infected astrocytes present this expression pattern (Appendix Fig. S1A, B), which is in accordance with our analysis performed in earlier stages of reprogramming (Fig. 1D). Our results indicate that a significant percentage of the infected astrocytes are reprogrammed to the ependymal lineage and possess unique ependymal morphological characteristics. Moreover, we showed that McIdas is more potent in the downregulation of the astrocytic identity of the transduced cells.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='GemC1 is positioned upstream of McIdas in the molecular pathway governing ependymogenesis, as it regulates the transcriptional activation of McIdas and controls the initial fate commitment towards the ependymal lineage (Kyrousi et al, 2015, 2017). Our results suggest that GemC1 was less effective than McIdas in reprogramming of astrocytes. We have therefore examined whether ectopic expression of GemC1 in astrocytes would not induce McIdas expression at sufficient levels. To test this hypothesis, we conducted immunofluorescence experiments using an McIdas specific antibody and quantified McIdas fluorescence in McIdas- and GemC1-infected astrocytes seven days post infection (Appendix Fig. S2A). Our results revealed that McIdas’ ectopic expression led to'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Our results revealed that McIdas’ ectopic expression led to higher expression levels compared to GemC1 (Appendix Fig. S2B), providing a potential mechanism for their differential ability to induce reprogramming.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Given the greater capacity of McIdas to induce the ependymal differentiation in astrocytes compared to GemC1, we hypothesized that McIdas would reprogram cortical astrocytes into fully mature ependymal cells carrying multiple basal bodies nucleating motile cilia. To address this point, we initially performed immunofluorescence experiments in transduced astrocytes twenty-two days post infection using antibodies against Centriolin (also known as Cep110), a known centrosomal marker, and Meig1 which was recently characterized as ependymal cell marker (MacDonald et al, 2021). We detected accumulation of both Centriolin (Appendix Fig.S3A) and Meig1 (Appendix Fig.S3B) signal in McIdas-infected astrocytes, verifying the generation of multiple basal bodies upon'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='verifying the generation of multiple basal bodies upon McIdas’ ectopic expression. Next, we performed immunofluorescence experiments making use of antibodies against acetylated a-tubulin, which stains ciliary axonemes, and the astrocytic marker glial fibrillary acidic protein (GFAP). The ectopic expression of McIdas resulted in the downregulation of GFAP expression and the generation of multiple cilia on the surface of the reprogrammed astrocytes, revealing the loss of the astrocytic identity and the establishment of the ependymal differentiation (Fig. 2A). Our analysis showed that 18% of McIdas- infected astrocytes were reprogrammed into multiciliated ependymal cells based on acetylated-a-tubulin staining (Fig. 2C), which was consistent with the percentage'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='staining (Fig. 2C), which was consistent with the percentage of McIdas- infected cells that downregulate the expression of the astrocytic marker GFAP (Fig. 2B). Additionally, live imaging microscopy experiments were performed in transduced astrocytes twenty-two days post infection to address the ability of the reprogrammed astrocytes multicilia to beat (Movies EV1-4). Using high-speed video microscopy, we acquired fast video recordings from GFP (Movies EV1-2) and McIdas- infected astrocytes (Movies EV3-4), which revealed the presence of ciliary motility in McIdas- infected cells as opposed to the absence of motility in GFP- infected cells. Importantly, our analysis showed that McIdas expression established cilia movement that was sufficient to propel'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='established cilia movement that was sufficient to propel fluorescent particles added to the imaging medium, highlighting the functionality of the ependymal cells and their capability to create a fluid flow. On the contrary, astrocytes infected with a control virus were unable to propel the fluorescent particles, as were only observed to have a rather stable forward and backward motion (Fig. 2D).'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Our findings show that GemC1 and McIdas promote the early steps of the multiciliogenesis program in both embryonic stem cells and astrocytes. Importantly, McIdas has a greater capacity for eliciting direct reprogramming into functional ependymal cells.\\n\\n\\n\\nMcIdas induces ependymal cells regeneration in a mouse model of intracranial hemorrhage hydrocephalus'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='In both human and murine hydrocephalus, the disrupted ependymal cell population is replaced by astrocytes. However, this astrocytic scarring along the denuded ventricular wall fails to restore the ependymal functions (Sival et al, 2011; Roales-Buján et al, 2012; Guerra et al, 2015; McAllister, James P. et al, 2017). Based on our finding that McIdas can successfully reprogram cortical astrocytes into functional ependymal cells, we examined whether McIdas could induce direct reprogramming of periventricular cells in hydrocephalic models. We used an established mouse model of intracranial hemorrhage hydrocephalus induced by the elevated concentration of Lysophosphatidic Acid (LPA) in the brain (Yung et al, 2015; Lummis et al, 2019), which closely mirrors the'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='et al, 2015; Lummis et al, 2019), which closely mirrors the most common type of human hydrocephalus.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Based on previously established protocols (Yung et al, 2011; Lummis et al, 2019), postnatal day five (P5) mice received intracranial injections of LPA and mouse brains were examined two days later to evaluate hydrocephalus occurrence. Although no obvious differences were observed in the size of LPA-treated brains macroscopically (Fig. EV3A), coronal brain sections revealed ventricular dilation in LPA-injected animals compared to non-injected ones (Fig. EV3B), confirming the development of hydrocephalus. In addition, acetylated-a-tubulin immunofluorescence revealed the ciliary disruption in ependymal cells alongside the ventricular walls upon LPA administration (Fig. EV3C), in line with previous studies (Lummis et al, 2019). On the contrary, ependymal cells'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='studies (Lummis et al, 2019). On the contrary, ependymal cells with multiple cilia covered the entire ventricular walls in control animals (Fig. EV3C). Given that the ventricular walls of the LPA-injected hydrocephalic mice were denuded from ependymal cells we wished to determine their cellular composition. We performed immunofluorescence with an antibody against the astrocytic marker glial fibrillary acidic protein (GFAP). Our analysis showed that GFAP positive cells with long processes, which likely corresponded to reactive astrocytes, covered the walls of the lateral ventricles in regions where ependymal cells localize under physiological conditions (Fig. EV3C), in line with previous findings (Roales-Buján et al, 2012).'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='To assess whether McIdas can induce direct reprogramming into ependyma in LPA-induced hydrocephalus, GFP-McIdas (McIdas) or GFP expressing plasmids were electroporated onto P7 brains isolated from LPA-treated mice by targeting the dorsolateral wall of one lateral ventricle to introduce plasmids in cells located at periventricular regions. Subsequently, thick coronal brain sections at the level of the lateral ventricles were cultured and immunofluorescence was performed using established ependymal markers (Fig. 3A). Five days post electroporation, 77% of the McIdas-electroporated cells expressed P73, as opposed to 13% of the GFP-electroporated cells, suggesting that astrocytes that replaced damaged ependymal cells in LPA treated hydrocephalic mouse models'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='ependymal cells in LPA treated hydrocephalic mouse models acquired the ependymal fate (Fig. 3B).'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='As ependymal cells normally require approximately two weeks until they acquire their mature characteristics in vivo, we followed differentiation of electroporated cells at later time points and examined whether they differentiate into fully mature multiciliated cells. Towards this direction, we performed immunolabeling experiments making use of antibodies recognizing Pericentrin to mark basal bodies and acetylated-a-tubulin to detect cilia (Fig. 3C). Nine days post electroporation 67% of McIdas electroporated cells carried multiple basal bodies (Fig. 3D), while 16% of McIdas-electroporated cells carried multiple cilia as revealed by acetylated-a-tubulin immunostaining (Fig. 3E). Basal bodies amplification or multiple cilia formation was not detected in any'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='or multiple cilia formation was not detected in any of the control cells, electroporated with a GFP expressing plasmid.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Collectively, our data show that McIdas forced expression in brain periventricular cells has the potential to regenerate the ependymal cells in a mouse model of intracranial hemorrhage hydrocephalus. \\n\\n\\n\\nMcIdas promotes in vivo reprogramming towards functional ependymal cells in a model of congenital hydrocephalus'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='We have recently described that genetic mutations in the human and mouse GemC1 gene lead to hydrocephalus (Lalioti et al, 2019b). Moreover, we have shown that radial glial cells lacking functional GemC1 fail to commit to the ependymal lineage and therefore do not develop ependymal cells during embryogenesis. We took advantage of the GemC1-knockout genetic mouse model of hydrocephalus we have generated to investigate whether reprogramming towards the ependymal cells could be achieved in a model of congenital hydrocephalus.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='We initially examined whether McIdas could program radial glial cells derived from GemC1-knockout hydrocephalic mice into ependymal cells ex vivo. Postnatal radial glial cells isolated from GemC1-knockout mice were infected with lentiviruses expressing either a GFP-McIdas (McIdas) fusion protein or GFP as a control. Infected cells were analyzed at different time points for the expression of known ependymal markers by immunofluorescence experiments, using specific antibodies recognizing FOXJ1, Pericentrin and acetylated-a-tubulin proteins (Fig. EV4A). At five days post infection, 86% of the McIdas-infected cells expressed FoxJ1 and showed accumulation of multiple basal bodies, based on Pericentrin immunostaining (Fig. EV4B, C). Moreover, our analysis'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='immunostaining (Fig. EV4B, C). Moreover, our analysis revealed that 97% of the McIdas transduced cells were multiciliated 15 days following infection, based on acetylated-a-tubulin and Pericentrin staining (Fig. EV3D, E). By contrast, in GFP transduced cells neither FoxJ1 expression nor multiple basal bodies and cilia were observed. In addition, similar experiments were performed by ectopically expressing P73 in postnatal radial glial cells isolated from GemC1-knockout mice. Ectopic expression of P73 was not sufficient to induce FoxJ1 expression nor multiple basal bodies or multiple cilia formation in GemC1-knockout radial glial cells (Fig. EV4A-E). These results show that cells derived from a genetic model of hydrocephalus can be programmed ex vivo into'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='a genetic model of hydrocephalus can be programmed ex vivo into multiciliated ependymal cells.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='In order to examine whether McIdas promotes cellular reprogramming towards the ependymal cell lineage in vivo, we have used our genetic mouse model of hydrocephalus which was previously established (Lalioti et al, 2019b). Plasmid vectors encoding McIdas in conjunction with GFP or GFP alone were electroporated into one lateral ventricle of newborn GemC1-knockout mouse brains targeting periventricular cells which could be either neural progenitor cells or reactive astrocytes that have replaced the lost ependymal cell population (Roales-Buján et al, 2012; Gonzalez-Cano et al, 2016; Abdi et al, 2018; Lalioti et al, 2019b). Reprogramming towards the ependymal cells was evaluated through immunofluorescence, using molecular markers that determine different stages'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='using molecular markers that determine different stages of ependymogenesis. Interestingly, four days post electroporation 61% and 58% of McIdas-electroporated cells were immunopositive for the ependymal markers P73 (Fig. 4A, B) and FoxJ1, respectively (Fig. EV5A, B). The expression of P73 and FoxJ1 was not detected in control animals, in agreement to what was previously described (Lalioti et al, 2019b), as GemC1-knockout mice lack ependymal cells. We also showed that McIdas further promoted the differentiation of the electroporated ependymal cells as basal bodies amplification was observed, based on Pericentrin immunofluorescence (Fig. 4C). More specifically, upon McIdas overexpression 44% of the electroporated cells acquired multiple basal bodies (Fig.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='the electroporated cells acquired multiple basal bodies (Fig. 4D). In addition, immunofluorescence against acetylated-a-tubulin revealed that McIdas overexpression resulted in the formation of cilia in a significant number of cells, corresponding to 7% of McIdas electroporated cells (Fig. 4E). Noticeably, ciliated cells were located close to the ventricles, resembling normal mature multiciliated ependymal cells, which form a single-layered ciliated epithelium adjacent to the brain ventricle.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='In order to examine whether the cilia that were generated upon McIdas ectopic expression were motile, thick coronal brain sections from GemC1-knockout mice covering the lateral brain ventricles were obtained and used in live imaging microscopy experiments (Fig. 5A). Electroporated cells were identified under a FITC filter and their cilia beating capacity was examined in fast video records. A proportion of cells with multiple beating cilia that were GFP-negative in sections derived from McIdas- electroporated brains were observed. We have previously shown that GemC1-knockout mice are unable to generate ependymal cells (Lalioti et al, 2019b), therefore cells carrying multiple beating cilia that do not appear GFP-positive were possibly observed due to the loss'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='not appear GFP-positive were possibly observed due to the loss of the transgene because of its transient expression and were thus undetectable under the FITC filter. In addition to this, we cannot exclude the possibility of dilution of the GFP signal due to continued proliferation. In agreement with what was previously described cilia were not detected in brain sections following GFP overexpression. However, McIdas electroporated cells exhibited functional beating cilia (Fig. 5B and Movies EV5, 6). Kymographs were generated (Fig. 5C) following previously described methods by Francis et al. (Francis & Lo, 2013) and ciliary beating frequency of McIdas induced ependymal cells was estimated to have a mean value of 10 Hz (Fig. 5D).'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Our findings show that McIdas induces ependymal cells restoration in the diseased hydrocephalic brain.\\n\\n\\n\\nMcIdas expression contributes to the restoration of the subventricular zone niche cytoarchitecture in hydrocephalic mice.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='The adult subventricular zone niche has a well-defined architecture formed by pinwheel-like structures. These structures are assembled by the large apical domain of ependymal cells which surround the smaller apical domain of adult neural stem cells (aNSCs) and constitute a unique micro-environment for coordinating adult neurogenesis (Mirzadeh et al, 2008; Paez-Gonzalez et al, 2011; Kokovay et al, 2012). To assess whether McIdas is able to restore the subventricular zone niche’s cytoarchitecture in hydrocephalic mice, whole-mount immunofluorescence was performed on ventricular walls of the lateral ventricles of GemC1-knockout hydrocephalic mice at P7-P9, previously electroporated with an McIdas-expressing plasmid or GFP alone. Specific antibodies against'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='plasmid or GFP alone. Specific antibodies against VCAM1 and GFAP, both markers of aNSCs, and β-catenin to delineate cell boundaries were used. Our analysis showed that McIdas-electroporated cells had a larger apical surface compared to control GFP-electroporated cells, which is characteristic for multiciliated ependymal cells. In addition, McIdas-electroporated cells were surrounding VCAM1- (Fig. 6A) and GFAP-immunopositive cells (Fig. 6B) reminiscent of pinwheel structures. On the contrary, control GFP-electroporated cells had a small apical surface and pinwheel structures were not detectable as it was previously described (Lalioti et al, 2019b). We also quantified the number of electroporated cells that were detected in pinwheel structures, revealing that'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='cells that were detected in pinwheel structures, revealing that 18% of McIdas-expressing cells were able to form pinwheel structures as opposed to GFP-electroporated cells that were not detected in pinwheels (Fig. 6C). Our data suggest that McIdas expression is sufficient to instruct the regeneration of ependymal cells in hydrocephalic mice which have the potential to re-build the lost architecture of the subventricular zone niche.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='DISCUSSION\\n\\nDuring mammalian brain development neural progenitor cells transition from an uncommitted to a restricted state, through a process that is influenced by lineage-specific gene expression programs, resulting to terminally differentiated cells. Reprogramming allows the conversion of differentiated cells from one lineage to another type of differentiated cell. Efficient reprogramming is highly dependent on the selection of pioneer factors related to the desired cell type. Here, we demonstrate that GEMC1 and MCIDAS, members of the geminin family, constitute potent reprogramming factors towards the ependymal fate both ex vivo and in vivo.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='GemC1 and McIdas drive the cell fate commitment of a subpopulation of brain radial glial cells towards multiciliated ependymal cells by coordinating gene expression of essential factors for ependymal cell differentiation (Kyrousi et al, 2015). In addition, we have recently provided evidence that GemC1 expression is necessary and sufficient to establish the appropriate chromatin organization at multiple loci for ependymal cells differentiation (Lalioti et al, 2019b). In light of this evidence, we hypothesized that GemC1 and McIdas could promote direct reprogramming towards the ependymal lineage in different cellular populations under physiology and disease. We initially demonstrated that both GemC1 and McIdas overexpression were able to trigger the'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='both GemC1 and McIdas overexpression were able to trigger the differentiation of mESCs into ependymal cells, as evidenced by the upregulation of key factors of ependymogenesis and the generation of multiple cilia in the GemC1- and McIdas overexpressing cells. We also showed that GemC1 and McIdas overexpression induce the expression of P73 and FoxJ1, two well-established transcription factors of the molecular pathway of multiciliogenesis, in cultured cortical astrocytes. Importantly, we demonstrated that GemC1 and McIdas activate the ependymal fate and induce the formation of multiple basal bodies and cilia in the reprogrammed cells. The ectopic expression of GemC1 resulted in a lower percentage of cells with multiple basal bodies, a unique structural'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='of cells with multiple basal bodies, a unique structural feature of ependymal cells, revealing a differential reprogramming potential of the two factors. We showed that GemC1 does not induce sufficient levels of McIdas expression, that could contribute to the reduced ability of GemC1 to execute as efficiently the ependymal differentiation program. Moreover, using video microscopy experiments we showed that McIdas ectopic expression was sufficient to reprogram cortical astrocytes into fully functional multiciliated ependymal cells, as reprogrammed cells possessed multiple beating cilia that were able to beat in a co-ordinated way and generate fluid flow. In addition, McIdas overexpression was sufficient to induce the ependymal fate in radial glial cells'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='sufficient to induce the ependymal fate in radial glial cells derived from the brain of hydrocephalic mice in great contrast to P73, further highlighting McIdas’ ability to induce the ependymal cell fate. The stem cell properties of the mESCs and radial glial cells used in our study are likely to facilitate the adoption of the ependymal fate. Since this aligns with their differentiation spectrum their conversion to ependyma could be considered as differentiation into the ependymal cell lineage.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='It has been shown that in human and experimental hydrocephalus, disrupted ependyma is replaced by astroglial cells (Sival et al, 2011; Roales-Buján et al, 2012; Guerra et al, 2015; McAllister, James P. et al, 2017). This astrocytic scarring along the denuded ventricular wall contributes to the pathophysiology of the disease as astrocytes do not restore ependymal functions; as they do not display cilia, produce appropriate trophic signals, or facilitate the production of neurons (Sival et al, 2011; Roales-Buján et al, 2012). Therefore, our strategy revealing the successful conversion of isolated cortical astrocytes into ependymal cells is of particular importance for treatment of hydrocephalus.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='To investigate whether McIdas would also reprogram cells in vivo, we used a mouse model of congenital hydrocephalus in which GemC1 has been inactivated, and a model of acquired hydrocephalus, triggered by intracranial hemorrhage. It was previously shown that GemC1 deficiency in mice results in the loss of the ependymal cells due to failure on activating the molecular program for the cell fate commitment towards the ependymal lineage (Terré et al, 2016; Lalioti et al, 2019b). Moreover, GemC1 deficiency has been recently associated with congenital hydrocephalus in both humans and mice (Lalioti et al, 2019b). In this study, we showed that McIdas forced expression induces the upregulation of P73 and FoxJ1 in brain periventricular cells, which are likely to be'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='FoxJ1 in brain periventricular cells, which are likely to be  neural progenitor cells or/and reactive astrocytes that have replaced the ependymal cell population in the hydrocephalic brain. The activation of the P73 and FoxJ1 genes uncovered the ability of McIdas to promote the initial steps of the ependymal cell’s differentiation program in the hydrocephalic brain. We also showed that McIdas-overexpressing cells generated multiple basal bodies and cilia, both of which are unique features of mature multiciliated ependymal cells, thus revealing the successful reprogramming towards ependyma. Importantly, the multiple cilia of the reprogrammed cells presented functional activity, as we were able to detect beating cilia upon in vivo overexpression of McIdas in'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='detect beating cilia upon in vivo overexpression of McIdas in the hydrocephalic mice. Using the LPA-induced intracranial hydrocephalus mouse model, we demonstrated McIdas ability to convert periventricular cells into ependymal cells. Previous study has shown that LPA causes substantial depletion of the ependymal monolayer, loss of ependymal cilia and subsequent ependymal apoptotic cell death (Lummis et al, 2019). In line with these findings, we did not observe multiciliated ependymal cells in periventricular regions after LPA treatment. Instead, we observed GFAP-positive cells, likely reactive astrocytes, covering the lateral ventricle walls where ependymal cells typically reside under normal conditions, as it has been previously shown in hydrocephalic mice'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='as it has been previously shown in hydrocephalic mice (Páez et al, 2007; Roales-Buján et al, 2012). Therefore, it is highly likely that McIdas reprograms astrocytes into ependymal cells in LPA-treated mice. However, we cannot rule out the possibility that McIdas might also enhance the survival of a subset of ependymal cells resistant to LPA-induced apoptosis.\\u200b'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Collectively, in our experiments we employed diverse cellular systems and technical approaches, including plasmid and lentiviral vectors, to induce the overexpression of McIdas and GemC1 suggesting that both GemC1 and McIdas can induce the programming or reprogramming of different cellular populations irrespectively of the cell system and vector used. While our approach in the genetic mouse model of hydrocephalus and the LPA-induced intracranial hydrocephalus likely targets periventricular cells, which are neural progenitor cells or/and reactive astrocytes, we cannot rule out the possibility that other cellular populations might also be targeted. Recent evidence from reprogramming experiments in the mouse retina and the brain indicates that the insert'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content=\"in the mouse retina and the brain indicates that the insert sequences of overexpression constructs can alter the specificity of the construct's promoter, introducing a bias toward the initial cell population being targeted (Wang et al, 2021; Le et al, 2022). Additionally, future genetic lineage analysis using transgenic mice to label neural progenitors and/or astrocytes would be needed to clarify which cell population is differentiated into ependymal cells in our approach. One of the major clinical features in human patients with hydrocephalus is the ventricular zone disruption phenomenon, characterized by the loss of neural stem cells and ependyma (McAllister, James P. et al, 2017). Of note, sustained neuron generation depends on the maintenance of the\"), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='sustained neuron generation depends on the maintenance of the proper cytoarchitecture of the Ventricular-Subventricular Zone (V-SVZ) niche of the brain, which requires preserving the structure of the ependymal cell monolayer intact (Paez-Gonzalez et al, 2011). In the present study, we targeted V-SVZ-resident cells in the hydrocephalic brain in which McIdas overexpression was induced and observed that McIdas-overexpressing cells populated areas of the ventricular zone niche next to the lateral ventricles, where ependymal cells normally reside under homeostasis. We have used SVZ electroporation to target the periventricular cells of one lateral ventricle of the hydrocephalic brain of GemC1-knockout mice, which results to a limited number of electroporated'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='mice, which results to a limited number of electroporated cells.  Therefore, following this experimental approach, the small number of ependymal cells which is generated is not possible to contribute to a significant reduction in the volume of the lateral ventricles as a readout of resolved or reduced hydrocephalus. It has been described that disruption of the ventricular/subventricular zone (V/SVZ) is a critical and consistent factor in the pathophysiology of hydrocephalus (McAllister, James P. et al, 2017; Jiménez et al, 2014). In our study we showed that McIdas ectopic expression contributes to the restoration of the lost cytoarchitecture of the niche in hydrocephalic mice, as we identified reprogrammed ependymal cells forming pinwheel-like structures'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='reprogrammed ependymal cells forming pinwheel-like structures with resident neural stem cells in the targeted ventricular wall. Our findings suggest that McIdas forced expression in human patients with hydrocephalus would contribute to the re-establishment of the SVZ niche leading to brain functions’ improvement by enhancing neuronal production in human patients with hydrocephalus and ameliorating brain function. The in vivo reprogramming approach indicates that the in-situ conversion of cells into ependyma in its native microenvironment is feasible. Our in vivo approach allowed the exploration of the influence of the diseased environment of the hydrocephalic brain on the reprogramming process. Our data support the idea that functional maturation and'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Our data support the idea that functional maturation and integration of the reprogrammed cells in the native tissue can be achieved through our in vivo reprogramming method. Since the molecular program of multiciliogenesis is conserved across different organisms and tissues (Arbi et al, 2018), it is tempting to speculate that such a reprogramming approach could be applied to other diseases that impact ciliated tissues, such as the lung and the reproductive system. For instance, a rare genetic disorder affecting the mucociliary clearance mechanism, known as Reduced Generation of Multiple Motile Cilia (RGMC), has been linked to mutations in MCIDAS. Therefore, it would be highly interesting to investigate whether these conditions could be addressed in the'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='investigate whether these conditions could be addressed in the future through a genetic treatment implementing McIdas’ overexpression in the affected tissue.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Together, our analysis proposes MCIDAS and GEMC1 as potent reprogramming factors towards the ependymal cell lineage and demonstrate a proof of concept that restoring ependymal cells could contribute in managing hydrocephalus. Although the present study yielded intriguing findings, our implementation holds certain limitations. In particular, our in vivo studies have been conducted to mouse models of hydrocephalus. Despite sharing a similar pathophysiology with the human disease, the anatomical and physiological characteristics of the mouse and human brain possess certain differences. Therefore, to validate the effectiveness of our approach, additional experiments in disease models closer to human pathophysiology will be required. The pig model demonstrates'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='pathophysiology will be required. The pig model demonstrates anatomical and physiological similarities to humans and replicates the pathophysiology of human hydrocephalus, exhibiting features such as ventriculomegaly, V/SVZ disruption, and neuroinflammation and could thus be a valuable model for examining the translational success of our approach to the human conditions (Garcia-Bonilla et al, 2023). Additionally, since hydrocephalus in our study appeared during early developmental stages, our findings are particularly relevant to postnatal hydrocephalus. Therefore, their applicability to adult hydrocephalus will require further investigation using appropriate adult animal models. Although we achieved McIdas overexpression in the periventricular cells of the'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='McIdas overexpression in the periventricular cells of the diseased brain through electroporation, such an approach cannot be used in humans. Therefore, alternative gene delivery methods compatible with therapeutic protocols, such as AAV gene delivery strategies in the human brain (Chamakioti et al, 2022; Au et al, 2022) should be established. Our study shows that regeneration of multiciliated ependymal cells can be succeeded in hydrocephalic mouse models while further experiments are needed to show the applicability of our findings in treating hydrocephalus in humans.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='METHODS\\n\\nMouse knockout strain\\n\\nC57/Bl6 mice carrying a knockout (KO) allele of the GemC1 gene were initially generated by the trans-NIH Knock-Out Mouse Project (KOMP) and obtained from the KOMP repository.  The GemC1-KO allele was maintained on a mixed strain background. Inactivation of the GemC1 gene was carried out using the “knockout-first” strategy, as previously described (Arbi et al, 2016; Lalioti et al, 2019b). This approach involves disrupting gene function by adding RNA processing signals without removing any of the targeted genes (Testa et al, 2004). The GemC1-KO allele was created by placing a lacZ-neo cassette between the second and third exon of the gene.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Mice were housed in the animal house of the University of Patras, where a standard light cycle of 12 hours of light and 12 hours of darkness was maintained. Continuous access to food and water was ensured at all times. To ensure animal welfare handling and environmental stressors like noise, vibrations, and strong odors were minimized to necessary levels. All animal-related procedures were conducted in strict accordance with EU directives and approved by the Veterinary Administrations of the Prefecture of Achaia, Greece, and the Research Ethics Committee of the University of Patras, Greece.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='In all experiments, mice were used without regard to their gender. Experiments involving animal usage were conducted during the neonatal stage when the gender of the animals was not yet distinguishable.LPA preparation and injections\\n\\nTo generate an inducible mouse model of intracranial-hemorrhage hydrocephalus, we performed intraventricular delivery of lysophosphatidic acid (LPA) in wild-type (WT) mice at postnatal day five (P5), following previously published works (Yung et al, 2011; Lummis et al, 2019).'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Powdered 18:1 LPA (857130P, Avanti Polar Lipids) was initially dissolved in methanol and subsequently vacuum-dried. LPA was then reconstituted in Hanks’ balanced salt solution (HBSS 14065-049, Gibco) by sonicating in a water bath for 5 min to generate a 10 mM stock, as previously described (Yung et al, 2011; Lummis et al, 2019). Stock solution was stored at −20°C until use. Using a glass microcapillary, 3 μl of the 10 mM LPA stock solution together with Fast Green (0.1%, Sigma) were injected into the lateral ventricle of P5 mice upon hypothermia-anesthetization. Mice were sacrificed two days post LPA injections at P7 and were examined or further processed as described in the respective experiments.\\n\\nPlasmid constructs'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Plasmid constructs\\n\\nFor neonatal electroporation and mESCs transfection experiments mouse cDNAs for McIdas and GemC1 were cloned in pCAGGS-IRES-GFP vectors between SacI/EcoRV and NheI/SmaI restriction sites respectively, as described previously (Kyrousi et al, 2015). \\n\\nNeonatal electroporation'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Neonatal electroporation\\n\\nNeonatal electroporation was performed at P0-P1. Anesthesia was induced via hypothermia. Using a pulled glass microcapillary, 2 μg plasmid DNA together with Fast Green (0.1%, Sigma) were injected in the lateral ventricle of the mouse brains. Mice were subsequently electroporated with 5 pulses of 100V for 50ms each followed by intervals of 950ms. An ECM830 Electroporator (Harvard Apparatus) and 7 mm platinum electrodes (BTX) were used. Following electroporation mice were placed back to cages and left to grow for 4 or 6-14 days before sacrificed as described in the respective experiments. \\n\\nEx vivo electroporation and organotypic brain slice cultures'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Ex vivo electroporation and organotypic brain slice cultures\\n\\nFor ex vivo electroporation, mice were sacrificed 2 days post LPA-injections at P7. Mouse brains were dissected and placed in cold DMEM/F-12 medium (Gibco). Subsequently, 2 μg plasmid DNA of a pCAGGS-IRES-GFP (GFP) or pCAGGS-McIdas-IRES-GFP (McIdas) vector together with Fast Green (0.1%, Sigma) were injected in the lateral ventricle using a pulled glass microcapillary. Brains were then electroporated with 5 pulses applied at 100V for 50ms each at intervals of 950ms. An ECM830 Electroporator (Harvard Apparatus) and 7 mm platinum electrodes (BTX) were used.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='For organotypic brain slice cultures 250 μm thick coronal slices were obtained from the electroporated brains using a vibrating microtome (Leica VT1000 S). Consecutive slices containing the lateral ventricles were collected and placed in culturing medium [2% FBS (Gibco), 1% GlutaMAX Supplement (Gibco) and 1% Penicillin-Streptomycin (Gibco) in DMEM/F-12]. Subsequently, brain slices were placed onto 0.4 μm membrane inserts (Millipore) in 6-well plates and cultured for 5 or 9 days in culturing medium, performing medium change every three days. \\n\\nLentiviral production'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Lentiviral production\\n\\nA second generation packaging system was used to generate lentiviral expression particles as previously described (Arbi et al, 2016). The pLV-Dest-CAG lentiviral expression vector was used and was kindly provided by Dr. M. Gotz, Helmholtz Center, Munich. The cloning strategy for generating the lentiviral expression vectors for GFP and GemC1 with an N-terminal GFP tag (GFP-GemC1) was described elsewhere (Arbi et al, 2016).'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='McIdas with an N-terminal GFP tag (GFP-McIdas) was initially cloned into the KpnI/XhoI restriction sites of the pENTR1AminusCmR vector. p73 with a C-terminal GFP tag (p73-GFP) was initially cloned into the SalI/NotI restriction sites of the pENTR1AminusCmR vector. An LR recombination reaction, using the Gateway LR Clonase II enzyme mix (Invitrogen), was performed between the attL-containing entry clone and the attR-containing destination pLVDest-CAG vector.\\n\\nPrimary cultures of postnatal cortical astroglia and lentiviral infection'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='To establish primary cultures of postnatal cortical astroglia, the cerebral cortex of postnatal P5-P7 wild-type mice was dissected from the brain. Cells were subsequently mechanically dissociated and cultured under proliferating conditions for 7 days in astro medium containing DMEM/F-12 (Gibco), 10% FBS (Gibco), 2% B27 Supplement (Gibco), 1% GlutaMAX Supplement (Gibco), 1% Penicillin-Streptomycin (Gibco), epidermal growth factor (10ng/ml) and basic fibroblast growth factor (10ng/ml), as previously described (Heinrich et al, 2011).'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='For lentiviral infection, astroglial cells were plated onto poly-D-lysine pre-coated coverslips 7 days upon proliferation initiation and were left to attach and grow for 2-4 hours. Cells were then transduced with concentrated lentiviral vectors. The infection mix was removed 24 hours later and was replaced by astro medium which did not contain FBS, epidermal growth factor and basic fibroblast growth factor to favor differentiation. Cells were analyzed 7 and 14 days later. \\n\\nPrimary cultures of postnatal radial glial cells and lentiviral infection'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='To establish primary cultures of postnatal radial glial cells (pRGCs), the lateral walls of the lateral ventricles of P0-P1 mice were dissected, mechanically dissociated, and then plated onto poly-D-lysine pre-coated coverslips. pRGCs were cultured in proliferation medium [DMEM-high glucose (Gibco), 10% FBS (Gibco), and 1% penicillin/streptomycin (Gibco)] for three days (Paez-Gonzalez et al, 2011; Kyrousi et al, 2015; Lalioti et al, 2019b). Subsequently, pRGCs were infected with unconcentrated viral particles performing plate-spin infection. Proliferation medium was changed 24 hours post-infection and later replaced with differentiation medium [DMEM-high glucose (Gibco), 2% FBS (Gibco), and 1% penicillin/streptomycin (Gibco)] 48 hours post-infection. Cells'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='penicillin/streptomycin (Gibco)] 48 hours post-infection. Cells were processed for analysis 5 or 15 days upon differentiation initiation.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Cultures of mouse embryonic stem cells (mESCs) and transfection\\n\\nMouse embryonic stem cells were grown onto gelatin-coated (0.1% gelatin, Sigma) flasks in GMEM BHK-21 medium (Gibco), supplemented with 10% FBS (Biosera, Embryonic Stem Cells tested), 0.1 mM β-mercaptoethanol (Gibco), 2 mM L-glutamine (Sigma), 1 mM sodium pyruvate (Pan Biotech), 1% penicillin/streptomycin (Gibco), 1% MEM Non-Essential Amino Acids (Biosera) and 2.000 U/ml LIF (Millipore). Cells were passaged every other day and were mycoplasma tested.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Reverse mESCs transfection was performed using the jet-OPTIMUS reagent according to manufacturer’s instructions. Briefly, the transfection mixes containing DNA, jet-OPTIMUS buffer and transfection reagent were prepared. Afterwards, transfection mixes were placed on gelatin pre-coated glass coverslips and then mESCs were immediately seeded. Cells were cultured in the presence of LIF for 24 hours. The following day the medium was changed (LIF withdrawal), and the cells were cultured in differentiating conditions for up to 6 days and were then processed for analysis.\\n\\nVideo microscopy of ependymal cilia'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Video microscopy of ependymal cilia\\n\\nFor the study of ependymal cilia in primary cultures of reprogrammed postnatal cortical astroglia, cells were grown under proliferating conditions in astro medium for 7 days. Cells were then plated onto poly-D-lysine coated glass-bottom dishes (ibidi) and transduced with unconcentrated lentiviral vectors. 24 hours post infection, medium was replaced by astro medium which did not contain FBS, epidermal growth factor and basic fibroblast growth factor to favor differentiation. Cells were used in video microscopy experiments 22 days later.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='To perform video microscopy experiments in reprogrammed postnatal cortical astroglia, the culturing medium was replaced by live imaging medium containing MEM (Sigma), 20% FBS (Gibco), 4 mM L-glutamine (Sigma) and 1 mM sodium pyruvate (Pan Biotech). In addition, to track cilia generated fluid flow 0.5 μm red fluorescent microbeads (1μl/ml medium; Sigma) were added to the imaging medium. Video microscopy experiments were performed using an Olympus IX83 microscope equipped with a Hamamatsu Orca-Flash 4.0 sCMOS camera. During live imaging microscopy experiments cells were maintained at 37°C and 5% CO2. Videos were obtained using a 60x water immersion objective at 80 frames/sec, while 500 frames were acquired for each field.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='For the study of brain ependymal cilia, mice were sacrificed at P7-P15, brains were dissected in HBSS at room temperature and 200 μm thick coronal slices were obtained using a vibrating microtome. Slices containing the lateral ventricles were placed in DMEM-GlutaMAX (Gibco) media pre-warmed at 37°C. \\n\\nTo perform video microscopy experiments, brain slices were placed onto glass-bottom dishes containing live imaging medium with 0.5 μm red fluorescent microbeads (1μl/ml medium). Video recordings were obtained as described in the preceding paragraph.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='For cilia beat frequency analysis, the ImageJ image processing program was used to firstly generate a graphical representation of cilia beating, in the form of a kymograph, as previously described by Drummond I. (Drummond, 2009). Cilia beat frequency for each multiciliated cell was then calculated with the correlation of the time scale of the video recording with the kymograph, following the method described by Francis R. and Lo C. (Francis & Lo, 2013).'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='To track the trajectory of the moving microbeads the MTrackJ Plugin of the ImageJ program was used. In short, the point selection tool was used to manually mark the position of each red fluorescent microbead in each frame. Consecutive marks on the corresponding video recordings created the trajectory of the microbeads.\\n\\nWhole-mount dissection'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Whole-mount dissection\\n\\nSVZ whole mounts were obtained as previously described (Lalioti et al, 2019b). Briefly, the brain was isolated and the electroporated lateral ventricle was carefully separated from the posterior region of the telencephalon, followed by the extraction of the hippocampus and septum. Fixation of the dissected lateral wall was performed in 4% PFA/0.1% Triton X-100 at 4oC overnight. Staining was then performed (see below, immunofluorescence) and the lateral wall was further dissected from the surrounding brain parenchyma, resulting in tissue slivers with a thickness of 200-300 micrometers. These slivers were then affixed to a glass slide using Mowiol 4-88 (Calbiochem) mounting media and covered with a coverslip.\\n\\n\\n\\nImmunofluorescence'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Immunofluorescence\\n\\nFor immunofluorescence, cultured cells were fixed for 10 min with 4% PFA (or 10 min PFA 4% followed by 10 min fixation in Methanol where stated) and incubated in blocking solution, for 1hr at room temperature. Postnatal cortical astroglial cells were incubated in blocking solution containing 5% FBS, 3% BSA, 0.1% Triton X-100 in 1× PBS. pRGCs and mESCs were incubated in blocking solution containing 5% Normal Goat Serum (Jackson Immunoresearch) and 0.3% Triton-X100 in 1x PBS. Cells were incubated with primary antibodies in blocking solution at 4°C, overnight (Kalogeropoulou et al, 2022).'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Brains dissected from newborn mice (up to P7) were fixed overnight with 4% PFA.  Subsequently, brains were rinsed with 1x PBS and cryopreserved with incubation in 30% sucrose in 1x PBS for ~48 hours. Brains were then frozen in 7.5% gelatin supplemented with 15% sucrose and sectioned at 12 μm using a CM1850 Leica cryostat. Brain coronal cryosections were postfixed with 4% PFA for 10 min, treated with 0.3% Triton X-100 in 1× PBS for 5 min, and incubated in blocking solution containing 5% FBS, 3% BSA, 0.1% Triton X-100 in 1× PBS, for 1-3 hrs. Samples were incubated with primary antibodies in blocking solution at 4°C, overnight.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='For immunofluorescence of cultured brain slices, membrane inserts were removed, rinsed in 1x PBS and fixed with 4% PFA at 4°C, overnight. Membrane inserts were then subjected to consecutive washes in 1x PBS (5 min), 0.1% Triton X-100 in 1× PBS (5 min) and 0.5% Triton X-100 in 1× PBS (20 min once) and were incubated in blocking solution containing 1% FBS, 0.1% Triton X-100 in 1× PBS for 2-3 hrs at room temperature. Samples were incubated with primary antibodies in blocking solution at 4°C, overnight.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Immunofluorescence of SVZ whole-mounts was performed as previously described (Mirzadeh et al, 2008, 2010). Briefly, whole-mounts were incubated for 1-2 hrs at room temperature in blocking solution, containing 10% normal goat serum in 0.1 M PBS with 0.5-2% Triton-X100. Samples were incubated with primary antibodies in blocking solution at 4oC for 24-48 hrs.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='The following primary antibodies were used: chicken anti-GFP (1020, Aves Laboratories, 1:500 for brain sections; 1:1000 for cells, 1:250 for whole mounts), mouse anti-S100β (S2532, Sigma, 1:250), rabbit anti-P73 (ab40658, Abcam, 1:300), mouse anti-FoxJ1 (14-9965, eBioscience, 1:500), rabbit anti-pericentrin (PRB432C, Covance, 1:750 for brain sections; 1:1500 for cells), rabbit anti-GFAP (Z0334,  DakoCytomation, 1:1000 for cells, 1:500 for whole-mounts), mouse anti-acetylated α-tubulin (T6793, Sigma, 1:1000 for brain sections; 1:1500 for cells), rabbit anti-McIdas ((Pefani et al, 2011), 1:250), mouse anti-centriolin (sc-365521, Santa Cruz, 1:100, cells’  fixation in PFA-Methanol), rabbit anti-Meig1 (orb185566, Biorbyt, 1:100), rat anti-VCAM1 (550547, BD'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(orb185566, Biorbyt, 1:100), rat anti-VCAM1 (550547, BD Biosciences, 1:100), mouse anti β-catenin (610153, BD Biosciences, 1:500), guinea pig anti-double-cortin (AB2253, Millipore, 1:500).'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='The following Alexa Fluor-labeled secondary antibodies (Invitrogen) were used in the respective blocking solution (1:1000) for 1 hr (cryosections and cells) at room temperature or overnight (membrane inserts) at 4°C: Alexa Fluor 488 goat anti-chicken, Alexa Fluor 568 goat anti-mouse IgG1, Alexa Fluor 647 donkey anti-rabbit, Alexa Fluor 568 goat anti-rabbit, Alexa Fluor 647 goat anti-mouse IgG1, Alexa Fluor 568 goat anti-mouse, Alexa Fluor 568 goat anti-rabbit, Alexa Fluor 647 donkey anti-mouse, Alexa Fluor 568 goat anti-guinea pig and Alexa Fluor 568 goat anti-rat. DNA was stained either with Draq-5 (1:1000, Biostatus) or Dapi (1:1500, Sigma). Samples were mounted in Mowiol 4-88 (Calbiochem).\\n\\nMicroscopy and data analysis'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Microscopy and data analysis\\n\\nImages of fluorescent samples were acquired with a confocal Leica TCS SP5 and a Leica TCS SP8 system equipped with a fluorescent Leica DMI600B or Leica DMi8 microscope, respectively. 20x, 40x and 63x lenses were used. Digital images were processed with Adobe Photoshop and the ImageJ image processing program. For still images related to video microscopy experiments, individual frames were cropped and adjusted for brightness and contrast in ImageJ.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Quantification of cells on overexpressed brains from in vivo experiments was performed on cryosections, vibratome sections or whole-mounts. In all instances, each brain region containing electroporated cells was imaged and quantified per frame. All quantifications are expressed as percentages of the total cell count. Quantification of radial glial cells, cortical astrocytes, or mESCs from ex-vivo experiments was performed in at least two independent experiments, with a minimum of 15 different images analyzed for each condition in each experiment. Fluorescent intensity per nucleus was measured using the open-source platform ImageJ.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='The number of independent experiments performed for each analysis is mentioned in the corresponding figure legend. Normality was tested for each experiment using the Shapiro-Wilk test to determine if the sample data came from a normally distributed population. Since our data did not follow a normal distribution, statistical analysis was performed with the non-parametric two-tailed Mann–Whitney test. Significant differences in medians were determined by p value: P<0.05 (*P<0.05, **P<0.01, ***P<0.001). Each graph displayed the median ± interquartile range (IQR). All statistical analyses and graph preparation were performed in GraphPad Prism 6.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='The study did not incorporate blinding or randomization, and all samples were included without applying specific inclusion or exclusion criteria. These choices were guided by the exploratory nature of the research.DATA AVAILABILITY\\n\\nThis study includes no data deposited in external repositories.\\n\\n\\n\\nACKNOWLEDGMENTS'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='We thank the Advanced Light Microscopy Facility of the Medical School at the University of Patras and the Experimental Animal Facility of the University of Patras for support with experiments. We are also grateful to Andriana Charalampopoulou and members from Professor’s Benedikt Berninger laboratory for their assistance in astrocytic reprogramming experiments. We also thank Dr. Christina Kyrousi and members of our laboratories for helpful discussions and Nikolaos Giakoumakis for his assistance with whole-mount images processing. This study was supported by the Hellenic Foundation for Research and Innovation (H.F.R.I.) under the “2nd Call for H.F.R.I. Research Projects to support Faculty Members & Researchers” (Project Number: 2735), the Hydrocephalus'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='& Researchers” (Project Number: 2735), the Hydrocephalus Association USA and Fondation Santé to S.T., and the State Scholarships Foundation (IKY) to K.K.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='DISCLOSURE AND COMPETING INTERESTS STATEMENT \\n\\nZoi Lygerou is an EMBO Council member. This has no bearing on the editorial consideration of thisarticle for publication.\\n\\nTHE PAPER EXPLAINED:\\n\\nMedical issue\\n\\nHydrocephalus is a condition where fluid builds up in the brain cavities, leading to serious health issues. Standard treatment involves surgery, which is risky and often fails over time. Ependymal cells have multiple tiny hair-like structures (cilia) on their surface to circulate the fluid, and their impairment may lead to hydrocephalus. \\n\\nResults'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Results\\n\\nWe showed that GemC1 and McIdas, which are known regulators of multiciliated ependymal cell fate determination, can instruct different cell types to differentiate into ependymal cells. We further managed to convert brain cells of hydrocephalic animal models into ependymal cells. \\n\\nClinical impact\\n\\nOur findings suggest new therapeutic intervention aiming at the regeneration of damaged ependymal cells in human hydrocephalus. .\\n\\n\\n\\n\\n\\nREFERENCES \\n\\nAbdi K, Lai C-H, Paez-Gonzalez P, Lay M, Pyun J & Kuo CT (2018) Uncovering inherent cellular plasticity of multiciliated ependyma leading to ventricular wall transformation and hydrocephalus. Nat Commun 9: 1655'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Arbi M, Pefani D, Kyrousi C, Lalioti M, Kalogeropoulou A, Papanastasiou AD, Taraviras S & Lygerou Z (2016) GemC1 controls multiciliogenesis in the airway\\xa0epithelium. EMBO Rep 17: 400–413\\n\\nArbi M, Pefani D-E, Taraviras S & Lygerou Z (2018) Controlling centriole numbers: Geminin family members as master regulators of centriole amplification and multiciliogenesis. Chromosoma 127: 151–174\\n\\nAu HKE, Isalan M & Mielcarek M (2022) Gene Therapy Advances: A Meta-Analysis of AAV Usage in Clinical Settings. Frontiers in Medicine 8\\n\\nBocchi R, Masserdotti G & Götz M (2022) Direct neuronal reprogramming: Fast forward from new concepts toward therapeutic approaches. Neuron 110: 366–393'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Boon M, Wallmeier J, Ma L, Loges NT, Jaspers M, Olbrich H, Dougherty GW, Raidt J, Werner C, Amirav I, et al (2014) MCIDAS mutations result in a mucociliary clearance disorder with reduced generation of multiple motile cilia. Nat Commun 5: 4418\\n\\nChamakioti M, Karantzelis N & Taraviras S (2022) Advanced Gene-Targeting Therapies for Motor Neuron Diseases and Muscular Dystrophies. International Journal of Molecular Sciences 23: 4824\\n\\nDrummond I (2009) Studying cilia in zebrafish. Methods Cell Biol 93: 197–217\\n\\nFrancis R & Lo C (2013) Ex vivo method for high resolution imaging of cilia motility in rodent airway epithelia. J Vis Exp'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Funk MC, Bera AN, Menchen T, Kuales G, Thriene K, Lienkamp SS, Dengjel J, Omran H, Frank M & Arnold SJ (2015) Cyclin O (Ccno) functions during deuterosome-mediated centriole amplification of multiciliated cells. EMBO J 34: 1078–1089\\n\\nGarcia-Bonilla M, Nair A, Moore J, Castaneyra-Ruiz L, Zwick SH, Dilger RN, Fleming SA, Golden RK, Talcott MR, Isaacs AM, et al (2023) Impaired neurogenesis with reactive astrocytosis in the hippocampus in a porcine model of acquired hydrocephalus. Experimental Neurology 363: 114354'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Gonzalez-Cano L, Fuertes-Alvarez S, Robledinos-Anton N, Bizy A, Villena-Cortes A, Fariñas I, Marques MM & Marin MC (2016) p73 is required for ependymal cell maturation and neurogenic SVZ cytoarchitecture: p73 Regulates SVZ Cytoarchitecture. Developmental Neurobiology 76: 730–747\\n\\nGuerra MM, Henzi R, Ortloff A, Lichtin N, Vío K, Jiménez AJ, Dominguez-Pinos MD, González C, Jara MC, Hinostroza F, et al (2015) Cell Junction Pathology of Neural Stem Cells Is Associated With Ventricular Zone Disruption, Hydrocephalus, and Abnormal Neurogenesis. J Neuropathol Exp Neurol 74: 653–671'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Heinrich C, Gascón S, Masserdotti G, Lepier A, Sanchez R, Simon-Ebert T, Schroeder T, Götz M & Berninger B (2011) Generation of subtype-specific neurons from postnatal astroglia of the mouse cerebral cortex. Nat Protoc 6: 214–228\\n\\nJi W, Tang Z, Chen Y, Wang C, Tan C, Liao J, Tong L & Xiao G (2022) Ependymal Cilia: Physiology and Role in Hydrocephalus. Frontiers in Molecular Neuroscience 15\\n\\nJiménez AJ, Domínguez-Pinos M-D, Guerra MM, Fernández-Llebrez P & Pérez-Fígares J-M (2014) Structure and function of the ependymal barrier and diseases associated with ependyma disruption. Tissue Barriers 2: e28426\\n\\nKahle KT, Kulkarni AV, Limbrick DD & Warf BC (2016) Hydrocephalus in children. The Lancet 387: 788–799'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Kalogeropoulou A, Mougkogianni M, Iliadou M, Nikolopoulou E, Flordelis S, Kanellou A, Arbi M, Nikou S, Nieminuszczy J, Niedzwiedz W, et al (2022) Intrinsic neural stem cell properties define brain hypersensitivity to genotoxic stress. Stem Cell Reports 17: 1395–1410\\n\\nKokovay E, Wang Y, Kusek G, Wurster R, Lederman P, Lowry N, Shen Q & Temple S (2012) VCAM1 is essential to maintain the structure of the SVZ niche and acts as an environmental sensor to regulate SVZ lineage progression. Cell Stem Cell 11: 220–230\\n\\nKousi M & Katsanis N (2016) The Genetic Basis of Hydrocephalus. Annu Rev Neurosci 39: 409–435'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Kyrousi C, Arbi M, Pilz G-A, Pefani D-E, Lalioti M-E, Ninkovic J, Götz M, Lygerou Z & Taraviras S (2015) Mcidas and GemC1 are key regulators for the generation of multiciliated ependymal cells in the adult neurogenic niche. Development 142: 3661–3674\\n\\nKyrousi C, Lygerou Z & Taraviras S (2017) How a radial glial cell decides to become a multiciliated ependymal cell. Glia 65: 1032–1042\\n\\nLalioti M-E, Arbi M, Loukas I, Kaplani K, Kalogeropoulou A, Lokka G, Kyrousi C, Mizi A, Georgomanolis T, Josipovic N, et al (2019a) GemC1 governs multiciliogenesis through direct interaction with and transcriptional regulation of p73. J Cell Sci 132'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Lalioti M-E, Kaplani K, Lokka G, Georgomanolis T, Kyrousi C, Dong W, Dunbar A, Parlapani E, Damianidou E, Spassky N, et al (2019b) GemC1 is a critical switch for neural stem cell generation in the postnatal brain. Glia\\n\\nLe N, Appel H, Pannullo N, Hoang T & Blackshaw S (2022) Ectopic insert-dependent neuronal expression of GFAP promoter-driven AAV constructs in adult mouse retina. Front Cell Dev Biol 10\\n\\nLim DA, Tramontin AD, Trevejo JM, Herrera DG, García-Verdugo JM & Alvarez-Buylla A (2000) Noggin Antagonizes BMP Signaling to Create a Niche for Adult Neurogenesis. Neuron 28: 713–726'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Lummis NC, Sánchez-Pavón P, Kennedy G, Frantz AJ, Kihara Y, Blaho VA & Chun J (2019) LPA1/3 overactivation induces neonatal posthemorrhagic hydrocephalus through ependymal loss and ciliary dysfunction. Sci Adv 5: eaax2011\\n\\nMa L, Quigley I, Omran H & Kintner C (2014) Multicilin drives centriole biogenesis via E2f proteins. Genes Dev 28: 1461–1471\\n\\nMacDonald A, Lu B, Caron M, Caporicci-Dinucci N, Hatrock D, Petrecca K, Bourque G & Stratton JA (2021) Single Cell Transcriptomics of Ependymal Cells Across Age, Region and Species Reveals Cilia-Related and Metal Ion Regulatory Roles as Major Conserved Ependymal Cell Functions. Front Cell Neurosci 15'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Marshall CB, Mays DJ, Beeler JS, Rosenbluth JM, Boyd KL, Santos\\xa0Guasch GL, Shaver TM, Tang LJ, Liu Q, Shyr Y, et al (2016) p73 Is Required for Multiciliogenesis and Regulates the Foxj1-Associated Gene Network. Cell Reports 14: 2289–2300\\n\\nMcAllister, James P., Guerra, Maria Montserrat, Ruiz, Leandro Castaneyra, Jimenez, Antonio J., Dominguez-Pinos, Dolores, Sival, Deborah, den Dunnen, Wilfred, Morales, Diego M., Schmidt, Robert E., Rodriguez, Esteban M., et al (2017) Ventricular Zone Disruption in Human Neonates With Intraventricular Hemorrhage. Journal of neuropathology and experimental neurology 76: 358–375'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Mirzadeh Z, Doetsch F, Sawamoto K, Wichterle H & Alvarez-Buylla A (2010) The subventricular zone en-face: wholemount staining and ependymal flow. J Vis Exp: 1938\\n\\nMirzadeh Z, Merkle FT, Soriano-Navarro M, Garcia-Verdugo JM & Alvarez-Buylla A (2008) Neural Stem Cells Confer Unique Pinwheel Architecture to the Ventricular Surface in Neurogenic Regions of the Adult Brain. Cell Stem Cell 3: 265–278\\n\\nNemajerova A, Kramer D, Siller SS, Herr C, Shomroni O, Pena T, Suazo CG, Glaser K, Wildung M, Steffen H, et al (2016) TAp73 is a central transcriptional regulator of airway multiciliogenesis. Genes Dev'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Páez P, Bátiz L-F, Roales-Buján R, Rodríguez-Pérez L-M, Rodríguez S, Jiménez AJ, Rodríguez EM & Pérez-Fígares JM (2007) Patterned neuropathologic events occurring in hyh congenital hydrocephalic mutant mice. J Neuropathol Exp Neurol 66: 1082–1092\\n\\nPaez-Gonzalez P, Abdi K, Luciano D, Liu Y, Soriano-Navarro M, Rawlins E, Bennett V, Garcia-Verdugo JM & Kuo CT (2011) Ank3-Dependent SVZ Niche Assembly Is Required for the Continued Production of New Neurons. Neuron 71: 61–75\\n\\nRekate HL (2009) A Contemporary Definition and Classification of Hydrocephalus. Seminars in Pediatric Neurology 16: 9–15'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Roales-Buján R, Páez P, Guerra M, Rodríguez S, Vío K, Ho-Plagaro A, García-Bonilla M, Rodríguez-Pérez L-M, Domínguez-Pinos M-D, Rodríguez E-M, et al (2012) Astrocytes acquire morphological and functional characteristics of ependymal cells following disruption of ependyma in hydrocephalus. Acta Neuropathol 124: 531–546\\n\\nRodríguez EM & Guerra MM (2017) Neural Stem Cells and Fetal-Onset Hydrocephalus. Pediatr Neurosurg 52: 446–461\\n\\nSawamoto K, Wichterle H, Gonzalez-Perez O, Cholfin JA, Yamada M, Spassky N, Murcia NS, Garcia-Verdugo JM, Marin O, Rubenstein JLR, et al (2006) New neurons follow the flow of cerebrospinal fluid in the adult brain. Science 311: 629–632'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Schrander-Stumpel C & Fryns JP (1998) Congenital hydrocephalus: nosology and guidelines for clinical approach and genetic counselling. Eur J Pediatr 157: 355–362\\n\\nSival DA, Guerra M, den Dunnen WFA, Bátiz LF, Alvial G, Castañeyra-Perdomo A & Rodríguez EM (2011) Neuroependymal denudation is in progress in full-term human foetal spina bifida aperta. Brain Pathol 21: 163–179\\n\\nStubbs JL, Vladar EK, Axelrod JD & Kintner C (2012) Multicilin promotes centriole assembly and ciliogenesis during multiciliate cell differentiation. Nat Cell Biol 14: 140–147'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Tan FE, Vladar EK, Ma L, Fuentealba LC, Hoh R, Espinoza FH, Axelrod JD, Alvarez-Buylla A, Stearns T, Kintner C, et al (2013) Myb promotes centriole amplification and later steps of the multiciliogenesis program. Development 140: 4277–4286\\n\\nTerré B, Piergiovanni G, Segura-Bayona S, Gil-Gómez G, Youssef SA, Attolini CS-O, Wilsch-Bräuninger M, Jung C, Rojas AM, Marjanović M, et al (2016) GEMC1 is a critical regulator of multiciliated cell differentiation. EMBO J 35: 942–960\\n\\nTesta G, Schaft J, van der Hoeven F, Glaser S, Anastassiadis K, Zhang Y, Hermann T, Stremmel W & Stewart AF (2004) A reliable lacZ expression reporter cassette for multipurpose, knockout-first alleles. Genesis 38: 151–158'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Wallmeier J, Al-Mutairi DA, Chen C-T, Loges NT, Pennekamp P, Menchen T, Ma L, Shamseldin HE, Olbrich H, Dougherty GW, et al (2014) Mutations in CCNO result in congenital mucociliary clearance disorder with reduced generation of multiple motile cilia. Nat Genet 46: 646–651\\n\\nWang L-L, Serrano C, Zhong X, Ma S, Zou Y & Zhang C-L (2021) Revisiting astrocyte to neuron conversion with lineage tracing in vivo. Cell 184: 5465-5481.e16\\n\\nYung YC, Mutoh T, Lin M, Noguchi K, Rivera RR, Choi JW, Kingsbury MA & Chun J (2011) Lysophosphatidic Acid Signaling May Initiate Fetal Hydrocephalus. Sci Transl Med 3: 99ra87\\n\\nYung YC, Stoddard NC, Mirendil H & Chun J (2015) Lysophosphatidic acid (LPA) signaling in the nervous system. Neuron 85: 669–682\\n\\n\\n\\n\\n\\nFIGURE LEGENDS'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='FIGURE LEGENDS\\n\\nFigure 1.\\n\\nMcIdas forced expression promotes ex vivo reprogramming of murine cortical astrocytes into ependymal cells. \\n\\n(A) Schematic representation of experimental procedures for (B-G). Astrocytes were isolated from the cortex of neonatal mice and were cultured under proliferating conditions. Subsequently, cells were transduced with lentiviral vectors encoding GFP or GFP-McIdas (McIdas) and cultured under differentiating conditions for indicated time points.\\n\\n(B) Cortical astrocytes were immunostained against GFP (green) to mark infected cells, P73 (grey) and S100β (red) at day 7 of differentiation. Arrows point to McIdas-infected cells which are P73+ and represent cells committed to the ependymal lineage.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(C-D) The graph depicts the percentage of the infected cells that co-express the ependymal marker P73 over the total number of the infected cells (C) The percentage of the infected cells which display S100β staining around their cell body, which corresponds to ependymal cells, is depicted in the graph (D) Data are presented as the median ±interquartile range (IQR) of three independent experiments. Statistical significance was determined using the non-parametric two-tailed Mann–Whitney test (***p < 0.0001).'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(E) Transduced astrocytes were immunostained at differentiation day 14 against GFP (green), FoxJ1 (grey) and pericentrin (PCNT, red). Note the presence of multiple basal bodies, based on PCNT signal accumulation, in McIdas-infected cells which co-express the ependymal marker FoxJ1 (arrows). \\n\\n(F-G) The graphs present the percentage of the infected cells that express FoxJ1 over the total number of the infected cells (F) and the percentage of FoxJ1-positive infected cells which also display accumulation of PCNT signal (G). Data are presented as the median ±IQR of three independent experiments. Statistical significance was determined using the non-parametric two-tailed Mann–Whitney test (***p < 0.0001). \\n\\nDNA was stained with Dapi (blue). Scale bars, 10 μm.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='DNA was stained with Dapi (blue). Scale bars, 10 μm. \\n\\nAbbreviations, dd: differentiation day\\n\\n\\n\\nFigure 2. \\n\\nMcIdas reprograms cortical astrocytes into functional ependymal cells. \\n\\n(A) Immunofluorescence of transduced cortical astrocytes at differentiation day 22 with antibodies against GFP (green), Glial Fibrillary Acidic Protein (GFAP, grey) which labels astrocytes and acetylated a-tubulin (ACT, red), a marker of ciliary axonemes. McIdas expression triggers the formation of multiple cilia, tiny hair-like structures which appear on the surface of the reprogrammed astrocytes (arrow). Note the downregulation of the GFAP signal in the reprogrammed multiciliated cells. DNA was stained with Dapi (blue). Scale bars, 10 μm'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(B-C) The graph shows the percentage of infected cells that express GFAP and correspond to an astrocytic population (B) The percentage of the infected cells which possess multiple cilia based on ACT staining is depicted in the graph (C) Data are presented as the median ±IQR of two independent experiments. Statistical significance was determined using the non-parametric two-tailed Mann–Whitney test (***p < 0.0001).'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(D) Live imaging microscopy experiments were performed on cortical astrocytes transduced either with GFP or McIdas lentiviruses. The positions of fluorescent beads included in the live imaging medium, were marked with colored dots on selected video frames. Τrack of one representative bead is shown in the right panel. Two independent experiments were performed. Scale bar, 10 μm\\n\\nAbbreviations, dd: differentiation day\\n\\n\\n\\nFigure 3. \\n\\nMcIdas induces ependymal cells regeneration in a mouse model of intracranial hemorrhage hydrocephalus.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(A) Mice received unilateral intracranial injection of Lysophosphatidic acid (LPA) to induce hydrocephalus. Coronal brain slices derived from brains electroporated either with a GFP or GFP-McIdas (McIdas) plasmid were cultured for 5 days. Immunofluorescence experiments were performed with antibodies against GFP (green) and the ependymal marker P73 (red). Inserts show higher magnification images. Arrows indicate GFP+P73+ cells.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(B) The graph depicts the percentage of the electroporated cells that co-express the ependymal marker P73, over the total number of the electroporated cells, at day 5 of culture. Data are presented as the median ± IQR of three independent experiments. Statistical significance was determined using the non-parametric two-tailed Mann–Whitney test (***p = 0.0002). \\n\\n(C) Coronal brain slices were cultured for 9 days and immunostained against GFP (green), pericentrin (PCNT, red) and acetylated a-tubulin (ACT, grey) to detect mature multiciliated cells. Note the presence of multiple basal bodies and cilia in McIdas electroporated cells.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(D-E) The percentage of the electroporated cells that carry multiple basal bodies, based on PCNT accumulation signal, upon 9 days of culture, is presented in the graph (D), (***p < 0.0001). The graph shows the percentage of multiciliated cells based on PCNT and ACT staining (E) upon 9 days of culture, (p=0.0013). Data are presented as the median ±IQR of three independent experiments. Statistical significance was determined using the non-parametric two-tailed Mann–Whitney test. \\n\\nDNA was stained with Dapi (blue). Scale bars, 10 μm. .\\n\\n\\n\\nFigure 4.\\n\\nMcIdas ectopic expression promotes in vivo reprogramming into ependymal cells in a genetic mouse model of hydrocephalus.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(A) Neonatal subventricular zone electroporation was conducted at GemC1-knockout mice at postnatal day 1 (P1) with plasmids encoding GFP, or GFP-McIdas (McIdas). Mice were sacrificed 4 days post electroporation and coronal brain sections were immunostained with antibodies against GFP (green) and P73 (red). Arrows point to McIdas-electroporated cells co-expressing P73, which represent cells committed to the ependymal lineage. Higher magnification of the boxed region is shown in the right panel.\\n\\n(B) The graph shows the percentage of the electroporated cells expressing P73. Data are presented as the median ±IQR of three independent experiments. Statistical significance was determined using the non-parametric two-tailed Mann–Whitney test (***p < 0.0001).'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(C) Coronal brain sections from GemC1-knockout electroporated mice were stained with specific antibodies against GFP (green), Pericentrin (PCNT, red) and acetylated a-tubulin (ACT grey). Multiple basal bodies and cilia on the surface of McIdas electroporated cells are indicated by an arrow. Higher magnification of the boxed region is shown in the right panel.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(D-E) Quantification of the percentage of the electroporated cells with multiple basal bodies, based on PCNT accumulation (D), (***p < 0.0001), and the percentage of multiciliated cells based on PCNT and ACT staining over the total number of the electroporated cells (E), (P=0.0041). Data are presented as the median ±IQR of two independent experiments. Statistical significance was determined using the non-parametric two-tailed Mann–Whitney test. \\n\\nDNA was stained with Dapi (blue). Scale bars, 10 μm. .\\n\\nAbbreviations, LV: lateral ventricle\\n\\n\\n\\nFigure 5.\\n\\nMcIdas establishes cilia motility in reprogrammed cells in a genetic mouse model of hydrocephalus.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(A) Schematic representation of experimental procedures for (B-D). GFP or GFP-McIdas (McIdas) expressing plasmids were electroporated at the lateral ventricle of postnatal day 1 (P1) GemC1-knockout mouse brains. Coronal brain sections, that contained the lateral ventricles, were obtained from the electroporated mice at ages P7-P15 and were used in live imaging experiments.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(B) Cilia still images from GFP or McIdas electroporated brains at indicated time points in milliseconds. FITC fluorescent cells are presented together with brightfield images. Arrows indicate the positions of cilia at indicated time points. Cilia motility was observed upon McIdas ectopic expression as evidenced by cilia changing positions at indicated time points. Higher magnification of the boxed regions is shown in the right panels.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(C) Representative kymograms depicting cilia motility upon GFP or McIdas ectopic expression.  Left panel images show the lines used to obtain the kymograms. These images are extracted from the image shown in B, which was utilized to generate the kymograph.(D) The graph shows cilia beat frequency of electroporated cells quantified from high-speed brightfield cilia motility movies. Data are presented as the median ±IQR (n=2 for GFP (control) and n=4 for McIdas). Statistical significance was determined using the non-parametric two-tailed Mann–Whitney test (***p < 0.0001).\\n\\n\\n\\nFigure 6.\\n\\nEctopic expression of McIdas restores the subventricular zone niche cytoarchitecture.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(A-B) GemC1-knockout mouse brains were electroporated with plasmids encoding GFP or McIdas-GFP (McIdas) at postnatal day 1 (P1). Mice were sacrificed at ages P7-P9 and lateral ventricular wall whole-mount immunofluorescence was conducted with antibodies against GFP (green), β-catenin (red), and Vascular Cell adhesion Molecule 1 (VCAM1, grey) (A) or Glial Fibrillary Acidic Protein (GFAP, grey) (B). \\n\\nFormation of pinwheel structures by McIdas-reprogrammed ependymal cells surrounding VCAM1+ neural stem cells (arrow, A) or GFAP-expressing neural stem cells (B) was observed. Note the enlarged apical domain of McIdas-reprogrammed ependymal cells. \\n\\nHigher magnification of the boxed region is shown in the right panel.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(C) The graph shows the percentage of electroporated cells, shown in A, identified in pinwheel structures. Data are presented as the median ±IQR (n=3 for GFP (control) and n=4 for McIdas). Statistical significance was determined using the non-parametric two-tailed Mann–Whitney test (***p < 0.0001.). \\n\\nScale bars, 10 μm. \\n\\n\\n\\n\\n\\nEV Figure Legends\\n\\n\\n\\nFigure EV1.\\n\\nMouse embryonic stem cells programming into ependymal cells.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Mouse embryonic stem cells programming into ependymal cells.\\n\\n(A) Mouse embryonic stem cells (mESCs) were transfected with GFP, GFP-McIdas (McIdas) or GFP-GemC1 (GemC1) expressing vectors and subsequently cultured under differentiating conditions. Immunofluorescence was performed with antibodies against GFP (green), FoxJ1 (red) and Pericentrin (PCNT, magenta) at differentiation day 4. The arrow points to multiple basal bodies in McIdas and GemC1 transfected cells which co-express the ependymal marker FoxJ1. Higher magnification of the boxed regions is shown in the right panels.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(B-C) The graphs present the percentage of transfected mESCs that express FoxJ1 over the total number of the transfected cells (B) and the percentage of transfected cells which display multiple basal bodies based on the accumulation of PCNT signal (C) Data are presented as the median ±IQR of two independent experiments. Statistical significance was determined using the non-parametric two-tailed Mann–Whitney test (***p < 0.0001). \\n\\n(D) GFP (green), PCNT (magenta) and acetylated-a-tubulin (ACT red) immunostaining in transfected mESCs at differentiation day 6. McIdas and GemC1 transfected mESCs present multiple basal bodies and cilia.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(E) The graph shows the percentage of transfected cells with multiple basal bodies (PCNT accumulation) and multiple cilia (ACT labeled cilia) over the total number of the transfected cells. Data are presented as the median ±IQR of two independent experiments. Statistical significance was determined using the non-parametric two-tailed Mann–Whitney test (***p < 0.0001). \\n\\nDNA was stained with Dapi (blue). Scale bars, 10 μm, \\n\\nAbbreviations, dd: differentiation day\\n\\n\\n\\nFigure EV2.\\n\\nGemC1 drives the commitment of murine cortical astrocytes towards the ependymal cell lineage.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(A) Cortical astrocytes were transduced with lentiviruses encoding GFP or GFP-GemC1 (GemC1). Immunofluorescence experiments were performed at day 7 of differentiation with specific antibodies against GFP (green) to mark the infected cells, P73 (grey) and S100β (red). Arrow point to a P73+ infected cell, committed to the ependymal lineage.\\n\\n(B-C) Graph depicting the percentage of infected cells which express P73 over the total number of infected cells (B). The graph shows the percentage of the infected cells which display S100β staining around their cell body (C). Three independent experiments were analyzed. Data are presented as the median ±IQR. Statistical significance was determined using the non-parametric two-tailed Mann–Whitney test (***p < 0.0001).'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(D) Astrocytes were immunostained against GFP (green), FoxJ1 (grey) and pericentrin (PCNT red) at differentiation day 14. The arrow points to a GemC1-infected cell which expresses FoxJ1 and possesses multiple basal bodies.\\n\\n(E-F) Graph presenting the percentage of the infected cells that express FoxJ1 over the total number of the infected cells (E) The graph shows the percentage of infected cells which express the ependymal marker FoxJ1 and display accumulation of PCNT signal (F). Data are presented as the median ±IQR from three independent experiments. Statistical significance was determined using the non-parametric two-tailed Mann–Whitney test (***p < 0.0001). \\n\\nDNA was stained with Dapi (blue). Scale bars, 10 μm\\n\\nAbbreviations, dd: differentiation day'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='Abbreviations, dd: differentiation day\\n\\n\\n\\nFigure EV3.\\n\\nLysophosphatidic Acid (LPA) administration causes hydrocephalus in early postnatal stages in mice.\\n\\n(A) Top views of Postnatal day 7 (P7) wild type mouse brains that either received or not injection of LPA at the lateral ventricle at P5. No differences on the size of the brain were observed macroscopically. \\n\\n(B) Dapi staining on coronal brain sections from P7 mouse brains reveals the dilation of the lateral ventricles after LPA injections. Scale bar, 100 μm.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(C) Immunofluorescence for Glial Fibrillary Acidic Protein (GFAP, green) which labels astrocytes and acetylated a-tubulin (ACT, red), a marker of ciliary axonemes on coronal sections from P7 mouse brains at the dorsal-lateral and lateral regions of the lateral walls.  Arrowheads point to multiple cilia in control ependymal cells (ep). Arrows show GFAP positive astrocytes (as) in the lateral wall of LPA-injected brains, where ciliary disruption is observed. Higher magnification of the boxed regions is shown in the right panels. Scale bars, 10 μm\\n\\nDNA was stained with Dapi or Draq-5 (blue). \\n\\n\\n\\nFigure EV4.\\n\\nEx vivo reprogramming of radial glial cells to ependyma can be achieved through McIdas ectopic expression.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(A) Schematic representation of experimental procedures for (B-E). Postnatal radial glial cells (pRGCs) were isolated from GemC1-knockout newborn mice, cultured and infected with lentiviruses expressing a GFP-McIdas (McIdas) or a GFP-P73 (P73) fusion protein, while GFP alone was used as a control. Cells were then cultured under differentiating conditions and analyzed at indicated timepoints.\\n\\n(B) Transduced radial glial cells were co-stained with antibodies against GFP (green) to mark infected cells, FoxJ1 (red) and pericentrin (PCNT, grey) 5 days after the initiation of differentiation. The arrow indicates the accumulation of PCNT signal in FoxJ1 expressing cells upon McIdas ectopic expression.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(C) The graph presents the percentage of infected cells which express FoxJ1 and display multiple basal bodies (accumulation of PCNT signal). Three independent experiments were analyzed. Data are presented as the median ±IQR. Statistical significance was determined using the non-parametric two-tailed Mann–Whitney test (***p < 0.0001). \\n\\n(D) Radial glial cells infected with GFP, McIdas, or P73 lentiviruses were labeled with antibodies against GFP (green), pericentrin (PCNT, grey) and acetylated a-tubulin (ACT, red) to detect mature multiciliated cells at differentiation day 15 (arrow).'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(E) The percentage of the infected cells which displayed multiple basal bodies, based on PCNT staining and simultaneously multiple cilia, based on ACT staining, was analyzed. Data are presented as the median ±IQR of three independent experiments. Statistical significance was determined using the non-parametric two-tailed Mann–Whitney test (***p < 0.0001). \\n\\nDNA was stained with Hoechst (blue). Scale bars, 10 μm,.\\n\\n\\n\\nFigure EV5.\\n\\nMcIdas drives the commitment to the ependymal cell lineage in a genetic mouse model for hydrocephalus.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(A) Subventricular zone electroporation was conducted at postnatal day 1 (P1) GemC1-knockout mice with plasmids encoding GFP, or GFP-McIdas (McIdas). Coronal brain sections were stained with antibodies against GFP (green) to mark the electroporated cells and FoxJ1 (red), a marker of committed ependymal cells, 4 days post electroporation. Arrows point to McIdas-electroporated cells expressing FoxJ1. Higher magnification of the boxed region is shown in the right panel.'), Document(metadata={'source': './data/docx/EMM-2023-19044.docx'}, page_content='(B) Graph depicting the percentage of electroporated cells that express FoxJ1 over the total number of the electroporated cells. Two independent experiments were analyzed. Data are presented as the median ±IQR. Statistical significance was determined using the non-parametric two-tailed Mann–Whitney test (***p < 0.0001). \\n\\nDNA was stained with Dapi (blue). Scale bars, 10 μm \\n\\n\\n\\n\\n\\n2')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/jabreu/PycharmProjects/python-playground/.venv/lib/python3.11/site-packages/nltk/translate/bleu_score.py:577: UserWarning: \n",
      "The hypothesis contains 0 counts of 2-gram overlaps.\n",
      "Therefore the BLEU score evaluates to 0, independently of\n",
      "how many N-gram overlaps of lower order it contains.\n",
      "Consider using lower n-gram order or use SmoothingFunction()\n",
      "  warnings.warn(_msg)\n",
      "/Users/jabreu/PycharmProjects/python-playground/.venv/lib/python3.11/site-packages/nltk/translate/bleu_score.py:577: UserWarning: \n",
      "The hypothesis contains 0 counts of 3-gram overlaps.\n",
      "Therefore the BLEU score evaluates to 0, independently of\n",
      "how many N-gram overlaps of lower order it contains.\n",
      "Consider using lower n-gram order or use SmoothingFunction()\n",
      "  warnings.warn(_msg)\n",
      "/Users/jabreu/PycharmProjects/python-playground/.venv/lib/python3.11/site-packages/nltk/translate/bleu_score.py:577: UserWarning: \n",
      "The hypothesis contains 0 counts of 4-gram overlaps.\n",
      "Therefore the BLEU score evaluates to 0, independently of\n",
      "how many N-gram overlaps of lower order it contains.\n",
      "Consider using lower n-gram order or use SmoothingFunction()\n",
      "  warnings.warn(_msg)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing document: ./data/docx/MSB-2023-12087.docx\n",
      "Split documents: [Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='What we can learn from deep space communication for reproducible bioimaging and data analysis \\n\\nTatiana Woller1,2, Christopher J. Cawthorne3, Romain Raymond Agnes Slootmaekers4, Ingrid Barcena Roig5, Alexander Botzki6, Sebastian Munck7,2,#. \\n\\nVIB Technology Training, Data Core, and VIB BioImaging Core, Ghent & Leuven, Belgium \\n\\nKU Leuven, Department of Neuroscience, Leuven, Belgium \\n\\nNuclear Medicine and Molecular Imaging, Department of Imaging and Pathology, KU Leuven, Leuven, Belgium \\n\\nNSANGA, 36 Pakenstraat, Leuven, 3001, Belgium \\n\\nSupport for Research Data Management (RDM), KU Leuven, Leuven, Belgium \\n\\nVIB Technology Training, Ghent, Belgium \\n\\nVIB BioImaging Core, Leuven, Belgium'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='VIB BioImaging Core, Leuven, Belgium \\n\\n# to whom correspondence should be addressed: Sebastian.munck@kuleuven.be'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='Multiple initiatives have attempted to define and recommend the annotation of images with metadata. However, proper documentation of complex and evolving projects is a difficult task, and the variety of storage methods – electronic labnotebooks, metadata servers, repositories and manuscripts – along with data from different time points of a given project leads to either redundancy in annotation or omissions. In this Commentary, we discuss how to tackle this problem, taking inspiration from space communication which uses error-correction protocols based on redundancy for data transmission. We provide a proof of concept using an Artificial Intelligence (AI) language model to digest redundant metadata entries of this manuscript and visualize the differences to'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='entries of this manuscript and visualize the differences to complete metadata entries, highlight inconsistencies and correct human error to improve the documentation for more reproducibility and reusability.'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='The reproducibility issue and related initiatives \\n\\nThe reproducibility crisis – many scientific studies are difficult or impossible to reproduce –   threatens science’s very fabric and public credibility. A survey conducted by Nature in 2016 showed that more than 70% of researchers did not succeed in reproducing someone else’s experiments; more than half could not reproduce their own experiment (Baker, 2016). This failure to reproduce experiments is often attributed to multiple factors, most commonly a lack of access to raw data, insufficient documentation and the inability to manage complex datasets (https://www.nature.com/articles/d42473-019-00004-y).'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='Reproducibility can be easily confounded with replicability, and its definition depends on the research domain. In biomedical research and computational biology, including bioimaging, ‘reproducibility’ indicates the ability to obtain the same results by using the same data and methods, while ‘replicability’ stands for researchers arriving at the same conclusion using their own data and methods (interestingly, the meanings of reproducibility and repeatability are swapped in computer science and microbiology (Plesser, 2018).'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='Gunderson et al. define four types of reproducibility based on the quality of the documentation (Gundersen & Kjensmo, 2018). The lowest degree of reproducibility is ‘R1 description’ that encompasses a textual description of the experiment. ‘R2 code’ contains the code/workflow and its associated metadata but lacks the original data. ‘R3 data’ refers to the available dataset and the associated metadata without the workflow for creating the metadata. ‘R4 experiments’ is the highest degree of reproducibility with dataset, code and associated documentation.'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='Addressing reproducibility via documentation inspired multiple initiatives within the field and beyond. These initiatives attempt to standardize the documentation that accompanies the generation of a bioimaging dataset – and by extension other data analysis disciplines. Organizations such as EOSC assert that the quality of data and associated metadata will improve /if it is Findable, Accessible Interoperable, and Reusable (FAIR; Wilkinson et al, 2016), and will enhance the reproducibility of research (https://zenodo.org/records/7515816). The FAIRification of data requires work at many levels from ontology to reproducible analysis pipelines. Recent ontologies such as REMBI (Sarkans et al, 2021), MITI (Schapiro et al, 2022), and EDAM Bioimaging'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='et al, 2021), MITI (Schapiro et al, 2022), and EDAM Bioimaging (https://bioportal.bioontology.org/ontologies/EDAM-BIOIMAGING) provide a starting point to report metadata associated with analysis.'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='The typical and most common tool for documenting experiments is the electronic labnotebook (Myers et al, 2001), which does not necessarily accommodate the aforementioned standardization of metadata like REMBI, MITI or EDAM. These metadata standards are increasingly incorporated with rich file formats (like OME-TIFF and OME -Zarr) or data tools such as OMERO to find and browse images alongside metadata; https://www.openmicroscopy.org/omero/) and ManGO for associating any file with predetermined metadata on modern data management systems like iRODS (https://irods.org/; https://github.com/kuleuven/mango-metadata-schemas).'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='One of the consequences of using additional data annotations like REMBI, is that they can lead to a fragmentation of metadata. Information about sample preparation and experimental conditions is typically stored in the labnotebook, whilst information about image acquisition and analysis is typically found in the associated metadata file, so there is a variable degree of redundancy when these sources are combined into a framework such as REMBI. However, as anyone who has had to “recover” data about an image from a labnotebook knows, redundancy can be a feature, not a bug. Seemingly cryptic information (time information, cell naming, filter idiosyncrasies, etc.) may allow confirmation of the identity of an image file via probabilistic inference. This suggests'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='of an image file via probabilistic inference. This suggests that it can be valuable to keep electronic labnotebook entries, imaging metadata, and the final write-up as separate data entries, as they can all be used to reconstruct complete information in the presence of human error or ‘noise’.'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='Using redundant sources to reconstruct complete information: lessons from space communication'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='The use of redundant sources to deal with noise is a well-established strategy in space communication, which is gaining increasing interest with current international efforts for unmanned space exploration. A central challenge for controlling robotic probes in outer space or on an alien planet is accurate and reliable communication. The finding that information can be accurately transmitted over noisy channels is based on Claude Shannon’s landmark publication (Shannon, 1948). That it can be corrected is based on Richard Hamming’s pioneering invention of the first error-correcting code in 1950 (Hamming, 1950). Today’s telecommunication routinely employs different strategies for redundancy, error detection and correction, including applications for unreliable'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='detection and correction, including applications for unreliable storage mediums.'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='The challenge of creating FAIR bioimage data is similar to the problems in deep-space communication. It can be viewed as a transmission to scientists in the future who would find the data useful or to another entity with which direct communication is not possible, such as another lab, or a successor. As with space communication, the chance that information is lost or parts were not passed on properly and hence are unrecoverable, needs to be accounted for.'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='In space communication, the error correction is intended to address noise due to weak signal strength and distance; for bioimage data, we can consider the ‘noise’ as imperfect user documentation. In addition to information included in publications – theoretically ‘stand-alone descriptions’ containing all the information needed for replication – the lack of a universal/integrated system that leverages electronic labnotebooks and standardized metadata leads to both omission and (unused) redundancy.'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='Here, we propose to use the redundancy inherent in different sources of data documentation – namely the electronic labnotebooks, metadata fileservers, manuscripts, GitHub resources, images, and so on that comprise the typical ‘data package’ for a bioimaging experiment – to create the most possible complete annotation and enable cross correction if necessary, using a similar conceptual model. This should consolidate the different metadata sources, help to complete missing information, and future-proof data and support documentation for better reproducibility.  \\n\\n\\n\\nApplication of AI'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='Whilst the call to proofread and consolidate various metadata entries might be noble, it is unlikely that many researchers will adopt it due to time constraints and its tedious nature. Artificial intelligence (AI) based language models are powerful tools for creating structured outputs that can readily take over tedious proofreading tasks for complementing the human part of the documentation. Commonly-available language models such as GPT-4 have already been used for the post-hoc transformation of free-text radiology reports into structured reporting (Adams et al, 2023). Language models can consequently be used to query if a specific diagnosis is present and to ‘digest’ various sources into a structured report for example in the form of a JSON file.'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='Such a report can be based on the latest recommendations for metadata such as, for example, REMBI or checklists as proposed by Schmied et al, (2023). Using an AI language model proofreading together with a metadata catalogue, can highlight gaps and contradictions and integrate an error correction for the various metadata entries to improve the overall annotation of the data. Redundancy of different (meta)data sources and representation of their consistency can be considered similar to error correction in space communication (Fig 1A and B). In addition, our analysis offers a feedback on the entries and their completeness.'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='Such a report could be published alongside a manuscript and could even be a prerequisite for submission. It could be seen as analogous to a preregistered report, where the study proposal is peer-reviewed, only that here the documentation is retrospectively reviewed and reproducibility and long-term validity are enhanced.  \\n\\n\\t \\n\\nA practical example'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='A practical example \\n\\n\\n\\nAs a proof of concept for the proposed approach we have created a workflow where the multimodal Large Language Model GPT-4 reads a labnotebook entry, a corresponding MANGO metadata file and a publication. Using the labnotebook entry that started this project, the Metadata entry created for the image file of Figure 1B when uploaded to the KU Leuven iRODS storage system, and this manuscript we tested the consistency between entries using a list of 5 keywords. We effectively tested for the completeness of the title, the authors, the topic, the methodology and the repository used; this list is a placeholder for a checklist, a metadata standard or a published template like REMBI as mentioned above.'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='In the future, it would be desirable to align resources like REMBI or a recommendation list targeted at image analysis (Schmied et al, 2023) with the GPT-4 queries. We also see that this is a community effort, where metadata schemas are evolving and hopefully over time converging on a community-agreed standard. Based on our five keyword query we created a Jupyter notebook (https://github.com/vib-bic-training/Reproducibility_RDM.git), which was used to generate a report summarizing the findings of the proofreading and comparing the different entries (the exemplary labnotebook file used here as well as the metadata file can be found on the GitHub repository). We interacted with GPT4 using an API key. We also used a larger pipeline package (en_core_web_lg,'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='key. We also used a larger pipeline package (en_core_web_lg, 685k unique vectors) for tokenization, which could be customized towards a specific domain, such as BioImaging. The ‘digestion’, the text that GPT-4 found in the manuscript, the labnotebook entry, and the metadata file based on the keywords are given in Table 1. It is impressive to see that even with these five simple keywords detailed descriptions can be extracted from the sources and compared. The consistency between the entries can be visualized in the form of a heatmap per keyword and source (Figure 1C). In a scenario where one of three data entries is different – for instance, a concentration – that value can now be corrected based on the majority of entries: the heatmap readily shows how'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='based on the majority of entries: the heatmap readily shows how similarity varies across the files with a ‘1’ describing perfect similarity. Beyond the proof of concept stage and regarding the use of large language models in general, consistency over time, correctness, hallucinations, and confidence in the answer as well as the availability of the language model need to be monitored carefully for future implementations.'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='Overall, we believe that the procedure outlined here can reduce errors in reporting and improve the overall reproducibility and FAIRness of bioimage data. Generating interpretable readouts in the form of heatmaps that highlight where metadata differs or is missing, could help to consolidate records and complete information more easily. This should improve the overall quality of reporting and future-proof the reproducibility and reusability of data for follow-up studies. Given the simplicity of the approach, it can be easily adopted, allowing image data to boldly go FAIR (where too little data has gone before). \\n\\n \\n\\n\\n\\nAcknowledgements'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='Acknowledgements \\n\\nThe authors like to thank Christof De Bo for help with the Figure design. Part of the resources and services used in this work were provided by the VSC (Flemish Supercomputer Center), funded by the Research Foundation - Flanders (FWO) and the Flemish Government. SM and CC are supported by \\n\\nFWO I000123N Flanders BioImaging: Leading Imaging Application Integrated Service and Enablement (FBI-LIAISE). SM is supported by FWO I001322N - 3D SUper-Resolution to cryo-Electron MIcroscopy to study nanoscale subcellular dynamics and structure that alter in Neurodegenerative Diseases - 3SURE MIND.  \\n\\n\\n\\nDisclosure and competing interests statement\\n\\nThe authors declare that they have no conflict of interest. \\n\\nReferences'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='References \\n\\nAdams LC, Truhn D, Busch F, Kader A, Niehues SM, Makowski MR & Bressem KK (2023) Leveraging GPT-4 for \\n\\nPost Hoc Transformation of Free-text Radiology Reports into Structured Reporting: A Multilingual Feasibility Study. Radiology 307: e230725 \\n\\nBaker M (2016) 1,500 scientists lift the lid on reproducibility. Nature 533: 452–454 \\n\\nGundersen OE & Kjensmo S (2018) State of the Art: Reproducibility in Artificial Intelligence. Proceedings of the AAAI Conference on Artificial Intelligence 32 \\n\\nHamming RW (1950) Error detecting and error correcting codes. The Bell System Technical Journal 29: 147–160 \\n\\nMyers J, Mendoza E & Hoopes B (2001) A Collaborative Electronic Laboratory Notebook. (https://papers.ssrn.com/abstract=2969589) [PREPRINT]'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='Plesser HE (2018) Reproducibility vs. Replicability: A Brief History of a Confused Terminology. Frontiers in Neuroinformatics 11 \\n\\nSarkans U, Chiu W, Collinson L, Darrow MC, Ellenberg J, Grunwald D, Hériché J-K, Iudin A, Martins GG, Meehan T, et al (2021) REMBI: Recommended Metadata for Biological Images—enabling reuse of microscopy data in biology. Nat Methods 18: 1418–1422 \\n\\nSchapiro D, Yapp C, Sokolov A, Reynolds SM, Chen Y-A, Sudar D, Xie Y, Muhlich J, Arias-Camison R, Arena S, et al (2022) MITI minimum information guidelines for highly multiplexed tissue images. Nat Methods 19: 262–267'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='Schmied C, Nelson MS, Avilov S, Bakker G-J, Bertocchi C, Bischof J, Boehm U, Brocher J, Carvalho MT, Chiritescu C, et al (2023) Community-developed checklists for publishing images and image analyses. Nat Methods: 1–12 \\n\\nShannon CE (1948) A mathematical theory of communication. The Bell System Technical Journal 27: 379–423 \\n\\nWilkinson MD, Dumontier M, Aalbersberg IjJ, Appleton G, Axton M, Baak A, Blomberg N, Boiten J-W, da Silva Santos LB, Bourne PE, et al (2016) The FAIR Guiding Principles for scientific data management and stewardship. Sci Data 3: 160018 \\n\\n \\n\\nFigure legends'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='Figure 1. Linking metadata sources and digesting them with language models to generate structured outputs and representations of similarity. A) Illustration of Hamming code for error correction in data transmission. Transmission of data (d) and parity (p) bits enables error correction via redundancy (https://en.wikipedia.org/wiki/Hamming_code). B) Diagram showing the different sources of metadata information and how to bundle them. Three independent resources – the electronic labnotebook, the data-associated metadata, and the publication – are shown as redundant entries. An AI language model can be used to extract required and standardized data elements for verification, using codewords as a means of error correction analogous to error correction in'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='as a means of error correction analogous to error correction in communication. C) Heatmap display of similarities between sources by keyword. A Jupyter notebook using GPT-4 has been used to create a structured output in the form of a CSV file, (see Table 1). The digestion of a labnotebook entry, a metadata file server file, and this manuscript are used to check for keywords. The consistency of the keywords across the sources is displayed in a heatmap using the cosine distance for semantic similarity estimation (https://spacy.io/api/doc).'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='Table 1. Proof of concept report. A Jupyter notebook using GPT-4 has been used to create a structured output in the form of a Table (CSV file). The digestion of a labnotebook entry, a ManGO metadata file and this manuscript are used to check for keywords and their consistent use.  \\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\xa0\\n\\nKeywords\\n\\nSource\\n\\nTitle\\n\\nAuthors\\n\\nTopic\\n\\nMethodology\\n\\nRepository\\n\\nManuscript\\n\\nWhat we can learn from deep space communication for reproducible bioimaging and data analysis\\n\\nTatiana Woller, Christopher J. Cawthorne, Romain Raymond Agnes Slootmaekers, Ingrid Barcena Roig, Alexander Botzki, Sebastian Munck'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='The paper discusses the challenge of maintaining reproducibility in BioImage data analysis, much like the challenges in deep space communications. It recommends the use of AI language model proofreading to improve error correction and consequently, the fidelity of the data.\\n\\nThe researchers created a workflow where the Large Language Model GPT-4 reads a lab notebook entry, a corresponding ManGO metadata file, and a publication to look for a list of keywords and check them for consistency across these metadata sources. Error correction is then performed by using AI to highlight gaps and contradictions.\\n\\nhttps://github.com/vib-bic-training/Reproducibility_RDM.git\\n\\nELN'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='ELN\\n\\nWhat we can learn from deep space communication for reproducible BioImaging and data analysis\\n\\nNot specified\\n\\nThe project aims to understand the learnings from deep space communication, which can be applied for better and reproducible BioImaging and data analysis.\\n\\nThe project relies on a Python-based main script which interacts with GPT-4 through an API (application programming interfaces). The user must obtain an API key from OpenAI to run the analysis. It accepts input in pdf, txt, json files format.\\n\\nhttps://github.com/vib-bic-training/Reproducibility_RDM.git\\n\\nManGO Metadata\\n\\nWhat we can learn from deep space communication for reproducible BioImaging and data analysis'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='Tatiana Woller, Christopher Cawthorne, Romain Raymond Agnes Slootmaekers, Ingrid Barcena Roig, Alex Botzki, Sebastian Munck\\n\\nThe study discusses what can be learned from deep space communication for making BioImaging and data analysis more reproducible.\\n\\nThe research utilized a large language model (GPT-4) for its methodology.\\n\\nhttps://github.com/vib-bic-training/Reproducibility_RDM.git\\n\\nAll sources\\n\\n\"What we can learn from deep space communication for reproducible BioImaging and data analysis\"\\n\\nTatiana Woller, Christopher J. Cawthorne, Romain Raymond Agnes Slootmaekers, Ingrid Barcena Roig, Alexander Botzki, Sebastian Munck.'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='The main focus is on improving the reproducibility of BioImage and data analysis, by drawing inspiration from deep space communication techniques. It suggests using AI language model proofreading to consolidate various metadata entries, and improve error correction, consequently enhancing the fidelity of data and its documentation, thereby paving the way for higher reproducibility and reusability.'), Document(metadata={'source': './data/docx/MSB-2023-12087.docx'}, page_content='The main methodology involves the use of a large language model AI, specifically GPT-4, to process various metadata sources such as lab notebook entries, metadata files, and publications. This AI assists in checking for consistency across these metadata sources and performs error correction by highlighting gaps and contradictions. A Jupyter notebook was created for proofreading and comparing the different entries. The AI model\\'s response in the form of a structured report was then used to visualize the similarities between the sources and to correct inconsistencies.\\n\\nThe code and workflows for the study have been made available in the following GitHub repository: \"https://github.com/vib-bic-training/Reproducibility_RDM.git\".')]\n"
     ]
    }
   ],
   "source": [
    "import json\n",
    "import uuid\n",
    "\n",
    "scores_local, scores_openai = [], []\n",
    "\n",
    "results = []\n",
    "\n",
    "query = \"\"\"\n",
    "    Please provide the Data Availability statement from this document.\n",
    "    Extract the estatemen exactly as it is written in the document. Make sure not to repeat the estatement twice.\n",
    "    If you can not find any estatement, please provide the following answer:\n",
    "    \n",
    "    `This study includes no data deposited in external repositories.`\n",
    "    \"\"\"\n",
    "\n",
    "openai_llm = OpenAI(\n",
    "    model=\"gpt-3.5-turbo-instruct\",\n",
    "    temperature=0,\n",
    "    max_retries=2,\n",
    "    max_tokens=512,\n",
    "    frequency_penalty=2\n",
    ")\n",
    "\n",
    "for docx, truth in zip(DOCX_FILES, TRUTH_FILES):\n",
    "    print(f\"Processing document: {docx}\")\n",
    "    \n",
    "    with open(truth) as f:\n",
    "        truth_data = json.load(f)\n",
    "\n",
    "    truth_answer = truth_data[\"data_availability\"][\"section_text\"]\n",
    "    truth_answer = remove_html_tags(truth_answer)\n",
    "    raw_documents = extract_document_text(docx, engine='docx2txt')\n",
    "    text_splitted = text_splitter.split_documents(raw_documents)\n",
    "    \n",
    "    print(\"Split documents:\", text_splitted)\n",
    "\n",
    "    # Build vectorstore\n",
    "    vectorstore = Chroma.from_documents(\n",
    "        documents=text_splitted,\n",
    "        embedding=embedding_model,\n",
    "        persist_directory=None,  # No persistence\n",
    "        collection_name=str(uuid.uuid4())  # ensure uniqueness\n",
    "    )\n",
    "    retriever = vectorstore.as_retriever(search_kwargs={\"k\": 3})\n",
    "    \n",
    "    # Debug retrieved documents\n",
    "    retrieved_docs = retriever.get_relevant_documents(query)\n",
    "\n",
    "    # QA chain\n",
    "    qa_chain_openai = RetrievalQA.from_chain_type(\n",
    "        llm=openai_llm,\n",
    "        chain_type=\"refine\",\n",
    "        retriever=retriever,\n",
    "    )\n",
    "    qa_chain_local= RetrievalQA.from_chain_type(\n",
    "        llm=ctrans_llm,         # Our ctransformers-based LLM\n",
    "        chain_type=\"refine\",    # \"stuff\", \"refine\", or \"map_reduce\"\n",
    "        retriever=retriever,\n",
    "    )\n",
    "    response_openai = qa_chain_openai(query)\n",
    "    response_local= qa_chain_local(query)\n",
    "        \n",
    "    # Evaluate\n",
    "    score_openai = sentence_bleu([truth_answer], response_openai[\"result\"], weights=(1, 0, 0, 0))\n",
    "    scores_openai.append(score_openai)\n",
    "    score_local = sentence_bleu([truth_answer], response_local[\"result\"], weights=(1, 0, 0, 0))\n",
    "    scores_local.append(score_local)\n",
    "    \n",
    "    results.append(\n",
    "        {\n",
    "            \"truth\": truth_answer,\n",
    "            \"openai\": response_openai[\"result\"],\n",
    "            \"local\": response_local[\"result\"],\n",
    "        }\n",
    "    )\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/var/folders/ps/jnmyqjq57sncdxvsr5297tk00000gq/T/ipykernel_37481/2574596975.py:6: MatplotlibDeprecationWarning: The 'labels' parameter of boxplot() has been renamed 'tick_labels' since Matplotlib 3.9; support for the old name will be dropped in 3.11.\n",
      "  plt.boxplot(data, labels=['Local Scores', 'OpenAI Scores'])\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAjcAAAHHCAYAAABDUnkqAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjkuMCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy80BEi2AAAACXBIWXMAAA9hAAAPYQGoP6dpAABMy0lEQVR4nO3deVhUZf8/8DeMMAyrC8iiCIgYqKgBQkh8XcLQTCW1KDOX1DKXNKwnMTfURHM3XHKv1PJx7UlLTdPcSAtcEwQV1ExwKQGVQJnP7w9/TE4ssowOHN+v65pL5z73uc/nDDPMmzP3mWMiIgIiIiIihTA1dgFEREREhsRwQ0RERIrCcENERESKwnBDREREisJwQ0RERIrCcENERESKwnBDREREisJwQ0RERIrCcENERESKwnBDZCAmJiaYOHGiscuotC+//BLe3t4wMzNDzZo1jV1OuaWnp8PExASrVq0ydik6bdu2Rdu2bY1dBtETg+GGDObcuXN4++230bBhQ1hYWMDW1hYhISGYN28ecnNzjV0elUFycjL69esHT09PLF26FEuWLCmx78SJE2FiYoLr168/xgqV7+7du5g/fz5atWoFGxsbWFtbo1WrVpg/fz7u3r1r7PJK9Morr8DExAQffvhhscv37t0LExMTbNiw4aFjXbt2DSNGjIC3tzc0Gg3q1q2LwMBAfPjhh7h165ahSycFqmHsAkgZtm3bhpdffhlqtRp9+vRBs2bNkJ+fjwMHDuCDDz7Ab7/9VuobpRLk5uaiRo3q/ZLau3cvtFot5s2bh0aNGhm7nCfO7du30blzZ/z000948cUX0a9fP5iammL79u0YMWIENm3ahG3btsHKysrYperJzs7Gt99+C3d3d3z11VeYNm0aTExMKjTWn3/+iYCAAGRnZ+PNN9+Et7c3bty4gRMnTmDRokV45513YG1tbeA9IKWp3r+JqUpIS0vDq6++Cjc3N/z4449wdnbWLRs6dCjOnj2Lbdu2GbHCR0er1SI/Px8WFhawsLAwdjmVdvXqVQColh9HKUFUVBR++uknfPrppxg2bJiu/Z133sGCBQswbNgwvP/++1i0aJERqyxq48aNKCgowIoVK9C+fXvs27cPbdq0qdBYy5cvx8WLF3Hw4EG0bt1ab1l2djbMzc0NUXKZ3L59u8oFSSobfixFlfbJJ5/g1q1bWL58uV6wKdSoUSOMGDFCd//evXuYPHkyPD09oVar4e7ujjFjxiAvL09vPXd3d7z44ovYu3cvAgICoNFo4Ovri7179wIANm3aBF9fX1hYWMDf3x9Hjx7VW79fv36wtrbG+fPnER4eDisrK7i4uGDSpEkQEb2+M2fOROvWrVGnTh1oNBr4+/sXe/jcxMQEw4YNw5o1a9C0aVOo1Wps375dt+zBOTc5OTkYOXIk3N3doVarUbduXXTo0AGJiYl6Y65fvx7+/v7QaDSwt7dH7969cfny5WL35fLly4iIiIC1tTUcHBzw/vvvo6CgoISfjL6FCxfqanZxccHQoUNx8+ZNvcd7woQJAAAHBweDzSH68ccfERoaCisrK9SsWRPdunVDUlJSkX6XL1/GgAED4OLiArVaDQ8PD7zzzjvIz88HcP8v+vfffx++vr6wtraGra0tOnXqhOPHj1eorrKOV/hxyn//+198/PHHqF+/PiwsLPDcc8/h7NmzRcZdsmQJPD09odFoEBgYiP3795epnt9//x3Lly9H+/bt9YJNoaFDh6Jdu3ZYtmwZfv/9d137g8/Jp556Svd62LdvX5ExLl++jDfffBOOjo5Qq9Vo2rQpVqxYUan9BYA1a9agQ4cOaNeuHXx8fLBmzZoy7XNxzp07B5VKhWeeeabIMltb2yJ/RBw+fBgvvPACatWqBSsrKzRv3hzz5s3T61OW52Dhx6ynT59Gr169UKtWLTz77LO65atXr9a9TmvXro1XX30Vly5d0hsjNTUVPXr0gJOTEywsLFC/fn28+uqryMrKqvDjQRUkRJVUr149adiwYZn79+3bVwBIz549ZcGCBdKnTx8BIBEREXr93Nzc5KmnnhJnZ2eZOHGizJkzR+rVqyfW1tayevVqadCggUybNk2mTZsmdnZ20qhRIykoKNDbjoWFhXh5eckbb7whcXFx8uKLLwoAGTdunN626tevL0OGDJG4uDiZPXu2BAYGCgDZunWrXj8A4uPjIw4ODhITEyMLFiyQo0eP6pZNmDBB17dXr15ibm4uUVFRsmzZMpk+fbp06dJFVq9ereuzcuVKASCtWrWSOXPmyOjRo0Wj0Yi7u7v89ddfRfaladOm8uabb8qiRYukR48eAkAWLlz40Md8woQJAkDCwsLk008/lWHDholKpZJWrVpJfn6+iIhs3rxZXnrpJQEgixYtki+//FKOHz/+0DGvXbtWYp8ffvhBatSoIY0bN5ZPPvlEYmJixN7eXmrVqiVpaWm6fpcvXxYXFxextLSUkSNHyuLFi2XcuHHi4+Ojexx++eUX8fT0lNGjR8tnn30mkyZNknr16omdnZ1cvnxZN1ZaWpoAkJUrV5b6mJR1vD179ggAefrpp8Xf31/mzJkjEydOFEtLSwkMDNQbc9myZQJAWrduLfPnz5eRI0dKzZo1pWHDhtKmTZtS61myZIkAkFWrVpXYp/D5snTpUl0bAGnWrJnY29vLpEmTZPr06eLm5iYajUZOnjyp65eRkSH169cXV1dXmTRpkixatEi6du0qAGTOnDkV2l+R+z87U1NT+fLLL0VEZNKkSVKrVi3Jy8vT61c47vr160t9HKZOnfrQx6HQzp07xdzcXNzc3GTChAmyaNEieffddyUsLEzXp6zPwcLnc5MmTaRbt26ycOFCWbBggYiITJkyRUxMTCQyMlIWLlyoG+PB12leXp54eHiIi4uLTJkyRZYtWyYxMTHSqlUrSU9Pf+i+kGEx3FClZGVlCQDp1q1bmfofO3ZMAMjAgQP12t9//30BID/++KOuzc3NTQDIoUOHdG07duwQAKLRaOTChQu69s8++0wAyJ49e3RthSFq+PDhujatViudO3cWc3NzvTflO3fu6NWTn58vzZo1k/bt2+u1AxBTU1P57bffiuzbv8ONnZ2dDB06tMTHIj8/X+rWrSvNmjWT3NxcXfvWrVsFgIwfP77IvkyaNElvjMI3oNJcvXpVzM3N5fnnn9cLf3FxcQJAVqxYoWsrS2ApT9+WLVtK3bp15caNG7q248ePi6mpqfTp00fX1qdPHzE1NZVffvmlyBharVZERP7++2+9+kXuBxm1Wq33uJQ13JR1vMI3ZR8fH7037Hnz5gkAXYAo/Hm2bNlSr19haHlYuBk5cqQA0IXl4iQmJgoAiYqK0rUBEADy66+/6touXLggFhYW8tJLL+naBgwYIM7OznL9+nW9MV999VWxs7PTvQbKur+FZs6cKRqNRrKzs0VEJCUlRQDI5s2b9fqVNdxkZGSIg4ODABBvb28ZPHiwrF27Vm7evKnX7969e+Lh4SFubm56fwiI/POcESn7c7Dw+fzaa6/pjZWeni4qlUo+/vhjvfaTJ09KjRo1dO1Hjx4t0/7R48GPpahSsrOzAQA2NjZl6v/dd98BuD+34EGjRo0CgCJzc5o0aYLg4GDd/aCgIABA+/bt0aBBgyLt58+fL7LNBw/xFx7Cz8/Px65du3TtGo1G9/+//voLWVlZCA0NLfIREgC0adMGTZo0ecie3p+3cvjwYfzxxx/FLv/1119x9epVDBkyRO9Qe+fOneHt7V3sPKXBgwfr3Q8NDS12nx+0a9cu5OfnY+TIkTA1/eclP2jQINja2j6y+VBXrlzBsWPH0K9fP9SuXVvX3rx5c3To0EH3XNBqtdiyZQu6dOmCgICAIuMUTkxVq9W6+gsKCnDjxg1YW1vjqaeeKvbn9DDlHa9///568z1CQ0MB/POcK/x5Dh48WK9fv379YGdn99B6cnJyAJT+WipcVvi6KxQcHAx/f3/d/QYNGqBbt27YsWMHCgoKICLYuHEjunTpAhHB9evXdbfw8HBkZWUV2eeH7W+hNWvWoHPnzrravLy84O/vX+GPphwdHXH8+HEMHjwYf/31FxYvXoxevXqhbt26mDx5su4j5aNHjyItLQ0jR44sMkes8DlT1ufgg/79Gtu0aRO0Wi1eeeUVvcfNyckJXl5e2LNnDwDofsY7duzAnTt3KrTvZDgMN1Qptra2AP75xfwwFy5cgKmpaZEzcZycnFCzZk1cuHBBr/3BAAP88wvE1dW12Pa//vpLr93U1BQNGzbUa2vcuDGA+9+HUmjr1q145plnYGFhgdq1a8PBwQGLFi0q9rNyDw+Ph+0mgPtzkU6dOgVXV1cEBgZi4sSJem8Mhfv61FNPFVnX29u7yGNhYWEBBwcHvbZatWoV2ed/K2k75ubmaNiwYZHtGEpp++fj44Pr16/j9u3buHbtGrKzs9GsWbNSx9NqtZgzZw68vLygVqthb28PBwcHnDhxokJzGso73r+fi7Vq1QLwz3OucH+9vLz0+pmZmRV5DhanMByU9loqKQD9e5vA/ef5nTt3cO3aNVy7dg03b97EkiVL4ODgoHfr378/gH8mkxd62P4CQFJSEo4ePYqQkBCcPXtWd2vbti22bt1aJISVlbOzMxYtWoQrV67gzJkzmD9/PhwcHDB+/HgsX74cwP25OQBKfd6U9Tn4oH+/vlNTUyEi8PLyKvLYJSUl6R43Dw8PREVFYdmyZbC3t0d4eDgWLFjA+TZGwrOlqFJsbW3h4uKCU6dOlWu9sp4mqlKpytVe+Fddeezfvx9du3bF//3f/2HhwoVwdnaGmZkZVq5cibVr1xbp/+BRntK88sorCA0NxebNm7Fz507MmDED06dPx6ZNm9CpU6dy11nSPj8ppk6dinHjxuHNN9/E5MmTUbt2bZiammLkyJHQarWPfDxDPueK4+PjAwA4ceIEWrZsWWyfEydOAECZjhw+qHB/evfujb59+xbbp3nz5nr3y7K/q1evBgC89957eO+994r03bhxoy48VYSJiQkaN26Mxo0bo3PnzvDy8sKaNWswcODACo/5MP9+fWu1WpiYmOD7778v9jF58LT0WbNmoV+/fvjmm2+wc+dOvPvuu4iNjcXPP/+M+vXrP7KaqSiGG6q0F198EUuWLEF8fLzeR0jFcXNzg1arRWpqqu6XOQBkZmbi5s2bcHNzM2htWq0W58+f1x2tAYCUlBQA988OAu7/ArawsMCOHTugVqt1/VauXFnp7Ts7O2PIkCEYMmQIrl69Cj8/P3z88cfo1KmTbl/PnDmD9u3b66135swZgz0WD27nwSMI+fn5SEtLQ1hYmEG2U9p2/y05ORn29vawsrKCRqOBra3tQwPyhg0b0K5dO91f7oVu3rwJe3v7ctdn6PEK9zc1NVXv53n37l2kpaWhRYsWpa7fqVMnqFQqfPnll+jTp0+xfb744gvUqFEDHTt21GtPTU0t0jclJQWWlpa6o302NjYoKCgw2M9bRLB27Vq0a9cOQ4YMKbJ88uTJWLNmTaXCzYMaNmyIWrVq4cqVKwAAT09PAMCpU6dK3KeyPgdL4+npCRGBh4eH3u+Rkvj6+sLX1xdjx47FoUOHEBISgsWLF2PKlCkPXZcMhx9LUaX95z//gZWVFQYOHIjMzMwiy8+dO6c7NfOFF14AAMydO1evz+zZswHcn29iaHFxcbr/iwji4uJgZmaG5557DsD9v1BNTEz0TqlOT0/Hli1bKrzNgoKCIoej69atCxcXF90p7wEBAahbty4WL16sdxr8999/j6SkJIM9FmFhYTA3N8f8+fP1/upevnw5srKyHsljDtwPdi1btsTnn3+ud8r5qVOnsHPnTt1zwdTUFBEREfj222/x66+/FhmnsGaVSlXkKMn69euLnDZfVoYeLyAgAA4ODli8eLHu9HUAWLVqld7+l8TV1RX9+/fHrl27iv0em8WLF+PHH3/EgAEDihwFiI+P15szc+nSJXzzzTd4/vnnoVKpoFKp0KNHD2zcuLHYEHnt2rVy7Ol9Bw8eRHp6Ovr374+ePXsWuUVGRmLPnj0lzjkryeHDh4t8VAQAR44cwY0bN3QfMfn5+cHDwwNz584t8vgW/lzL+hwsTffu3aFSqRATE1Pk+SIiuHHjBoD786Du3bunt9zX1xempqZFvuaCHj0euaFK8/T0xNq1axEZGQkfHx+9byg+dOgQ1q9fj379+gEAWrRogb59+2LJkiW4efMm2rRpgyNHjuDzzz9HREQE2rVrZ9DaLCwssH37dvTt2xdBQUH4/vvvsW3bNowZM0b3F23nzp0xe/ZsdOzYEb169cLVq1exYMECNGrUSPcxQHnl5OSgfv366NmzJ1q0aAFra2vs2rULv/zyC2bNmgXg/lyM6dOno3///mjTpg1ee+01ZGZmYt68eXB3dy/2MH9FODg4IDo6GjExMejYsSO6du2KM2fOYOHChWjVqhV69+5dqfFnz54NS0tLvTZTU1OMGTMGM2bMQKdOnRAcHIwBAwYgNzcXn376Kezs7PS+Q2fq1KnYuXMn2rRpg7feegs+Pj64cuUK1q9fjwMHDqBmzZp48cUXMWnSJPTv3x+tW7fGyZMnsWbNmjLNZymOocczMzPDlClT8Pbbb6N9+/aIjIxEWloaVq5cWeYx58yZg+TkZAwZMgTbt2/XHaHZsWMHvvnmG7Rp00b3/HlQs2bNEB4ejnfffRdqtRoLFy4EAMTExOj6TJs2DXv27EFQUBAGDRqEJk2a4M8//0RiYiJ27dqFP//8s1z7u2bNGqhUqhLDcdeuXfHRRx/h66+/LnICQWm+/PJLrFmzBi+99BL8/f1hbm6OpKQkrFixAhYWFhgzZgyA+8+xRYsWoUuXLmjZsiX69+8PZ2dnJCcn47fffsOOHTsAoMzPwZJ4enpiypQpiI6ORnp6OiIiImBjY4O0tDRs3rwZb731Ft5//338+OOPGDZsGF5++WU0btwY9+7dw5dffqkLlvSYPfbzs0ixUlJSZNCgQeLu7i7m5uZiY2MjISEh8umnn8rff/+t63f37l2JiYkRDw8PMTMzE1dXV4mOjtbrI3L/VPDOnTsX2Q6AIqdYF57+O2PGDF1b3759xcrKSs6dOyfPP/+8WFpaiqOjo0yYMKHIKcDLly8XLy8vUavV4u3tLStXrtSdGvqwbT+4rPBU8Ly8PPnggw+kRYsWYmNjI1ZWVtKiRYtiv5Nm3bp18vTTT4tarZbatWvL66+/Lr///rten8J9+bfiaixJXFyceHt7i5mZmTg6Oso777xT5BTaipwKXtxNpVLp+u3atUtCQkJEo9GIra2tdOnSRU6fPl1kvAsXLkifPn3EwcFB1Gq1NGzYUIYOHao7Hfnvv/+WUaNGibOzs2g0GgkJCZH4+Hhp06aN3mnW5TkVvCzjlXQKc0nbWbhwoXh4eIharZaAgADZt29fkTFLk5eXJ3PmzBF/f3+xsrISS0tL8fPzk7lz5+q+k+hBhc/J1atX657DTz/9tN7XIhTKzMyUoUOHiqurq5iZmYmTk5M899xzsmTJknLtb35+vtSpU0dCQ0NL3RcPDw95+umnSx33306cOCEffPCB+Pn5Se3ataVGjRri7OwsL7/8siQmJhbpf+DAAenQoYPudda8eXP59NNP9fqU5Tn4sOf+xo0b5dlnnxUrKyuxsrISb29vGTp0qJw5c0ZERM6fPy9vvvmmeHp6ioWFhdSuXVvatWsnu3btKnV/6dEwETHQbDiiKqZfv37YsGEDL7RHimZiYoKhQ4fqffxK9KTjnBsiIiJSFIYbIiIiUhSGGyIiIlIUzrkhIiIiReGRGyIiIlIUhhsiIiJSlCfuS/y0Wi3++OMP2NjYlPn6RkRERGRcIoKcnBy4uLjA1LT0YzNPXLj5448/ilxRmoiIiKqHS5cuPfRCpE9cuLGxsQFw/8GxtbU1cjVERERUFtnZ2XB1ddW9j5fmiQs3hR9F2draMtwQERFVM2WZUsIJxURERKQoDDdERESkKAw3REREpCgMN0RERKQoDDdERESkKAw3REREpCgMN0RERKQoDDdERESkKAw3REREpChGDTf79u1Dly5d4OLiAhMTE2zZsuWh6+zduxd+fn5Qq9Vo1KgRVq1a9cjrJCIiourDqOHm9u3baNGiBRYsWFCm/mlpaejcuTPatWuHY8eOYeTIkRg4cCB27NjxiCslIiKi6sKo15bq1KkTOnXqVOb+ixcvhoeHB2bNmgUA8PHxwYEDBzBnzhyEh4c/qjKJiIioGqlWc27i4+MRFham1xYeHo74+PgS18nLy0N2drbejYiIiJSrWl0VPCMjA46Ojnptjo6OyM7ORm5uLjQaTZF1YmNjERMT87hKJCIiA7hz5w6Sk5NL7ZObm4v09HS4u7sX+/v/37y9vWFpaWmoEqkKq1bhpiKio6MRFRWlu5+dnQ1XV1cjVkRERA+TnJwMf39/g46ZkJAAPz8/g45JVVO1CjdOTk7IzMzUa8vMzIStrW2JqV2tVkOtVj+O8oiIyEC8vb2RkJBQap+kpCT07t0bq1evho+PT5nGpCdDtQo3wcHB+O677/TafvjhBwQHBxupIiIiehQsLS3LfJTFx8eHR2RIj1HDza1bt3D27Fnd/bS0NBw7dgy1a9dGgwYNEB0djcuXL+OLL74AAAwePBhxcXH4z3/+gzfffBM//vgj/vvf/2Lbtm3G2gUiIqqA1NRU5OTkVGqMpKQkvX8ry8bGBl5eXgYZi4zLRETEWBvfu3cv2rVrV6S9b9++WLVqFfr164f09HTs3btXb5333nsPp0+fRv369TFu3Dj069evzNvMzs6GnZ0dsrKyYGtra4C9ICKi8khNTUXjxo2NXUaxUlJSGHCqqPK8fxv1yE3btm1RWrYq7tuH27Zti6NHjz7CqoiI6FEqPGJT1rkyJSnv2VKlKZy/U9mjSVQ1VKs5N0REpAxO1ibwc1bBx6kyX7dmhRCPpgapR3NTBSdrE4OMRcbHcENERI/d2/7m8Nn3NrDP2JXc54P7NZEyMNwQEdFj91lCPiLHr4JPFTk9Oyk5GZ/N6oWuxi6EDILhhoiIHruMW4Lcmo0Bl5bGLgUAkJuhRcYto51fQwZWra4tRURERPQwDDdERESkKAw3REREpCgMN0RERKQoDDdERESkKAw3REREpCgMN0RERKQoDDdERESkKAw3REREpCgMN0RERKQoDDdERESkKAw3REREpCgMN0RERKQoDDdERESkKAw3REREpCgMN0RERKQoDDdERESkKAw3REREpCgMN0RERKQoDDdERESkKAw3REREpCgMN0RERKQoDDdERESkKAw3REREpCgMN0RERKQoDDdERESkKAw3REREpCgMN0RERKQoDDdERESkKAw3REREpCgMN0RERKQoDDdERESkKAw3REREpCgMN0RERKQoDDdERESkKAw3REREpCgMN0RERKQoDDdERESkKAw3REREpCgMN0RERKQoDDdERESkKAw3REREpCgMN0RERKQoDDdERESkKAw3REREpCgMN0RERKQoDDdERESkKAw3REREpCgMN0RERKQoDDdERESkKAw3REREpCgMN0RERKQoNYxdwIIFCzBjxgxkZGSgRYsW+PTTTxEYGFhi/7lz52LRokW4ePEi7O3t0bNnT8TGxsLCwuIxVk1ERBV1584dAEBiYmKlxsnNzUV6ejrc3d2h0WgqNVZSUlKl1qeqxajhZt26dYiKisLixYsRFBSEuXPnIjw8HGfOnEHdunWL9F+7di1Gjx6NFStWoHXr1khJSUG/fv1gYmKC2bNnG2EPiIiovJKTkwEAgwYNMnIlRdnY2Bi7BDIAExERY208KCgIrVq1QlxcHABAq9XC1dUVw4cPx+jRo4v0HzZsGJKSkrB7925d26hRo3D48GEcOHCgTNvMzs6GnZ0dsrKyYGtra5gdISKiMrt+/Tq2bNkCb29vWFpaVnicpKQk9O7dG6tXr4aPj0+l67KxsYGXl1elx6FHozzv30Y7cpOfn4+EhARER0fr2kxNTREWFob4+Phi12ndujVWr16NI0eOIDAwEOfPn8d3332HN954o8Tt5OXlIS8vT3c/OzvbcDtBRETlZm9vj4EDBxpsPB8fH/j5+RlsPKr+jBZurl+/joKCAjg6Ouq1Ozo66g5Z/luvXr1w/fp1PPvssxAR3Lt3D4MHD8aYMWNK3E5sbCxiYmIMWjsRERFVXdXqbKm9e/di6tSpWLhwIRITE7Fp0yZs27YNkydPLnGd6OhoZGVl6W6XLl16jBUTERHR42a0Izf29vZQqVTIzMzUa8/MzISTk1Ox64wbNw5vvPGG7nCmr68vbt++jbfeegsfffQRTE2LZjW1Wg21Wm34HSAiIqIqyWhHbszNzeHv7683OVir1WL37t0IDg4udp07d+4UCTAqlQoAYMR50URERFSFGPVU8KioKPTt2xcBAQEIDAzE3Llzcfv2bfTv3x8A0KdPH9SrVw+xsbEAgC5dumD27Nl4+umnERQUhLNnz2LcuHHo0qWLLuQQERHRk82o4SYyMhLXrl3D+PHjkZGRgZYtW2L79u26ScYXL17UO1IzduxYmJiYYOzYsbh8+TIcHBzQpUsXfPzxx8baBSIiIqpijPo9N8bA77khIlKGxMRE+Pv7IyEhgaeCPwHK8/5drc6WIiIiInoYhhsiIiJSFIYbIiIiUhSGGyIiIlIUhhsiIiJSFIYbIiIiUhSGGyIiIlIUhhsiIiJSFIYbIiIiUhSGGyIiIlIUhhsiIiJSFIYbIiIiUhSGGyIiIlIUhhsiIiJSFIYbIiIiUhSGGyIiIlIUhhsiIiJSFIYbIiIiUhSGGyIiIlIUhhsiIiJSFIYbIiIiUhSGGyIiIlIUhhsiIiJSFIYbIiIiUhSGGyIiIlKUGsYugIiI6N/u3LmD5OTkUvskJSXp/fsw3t7esLS0rHRtVPUx3BARUZWTnJwMf3//MvXt3bt3mfolJCTAz8+vMmVRNcFwQ0REVY63tzcSEhJK7ZObm4v09HS4u7tDo9GUaUx6MpiIiBi7iMcpOzsbdnZ2yMrKgq2trbHLISIiojIoz/s3JxQTERGRojDcEBERkaIw3BAREZGiMNwQERGRojDcEBERkaIw3BAREZGiMNwQERGRojDcEBERkaIw3BAREZGi8PILpFgFBQXYv38/rly5AmdnZ4SGhkKlUhm7LCIiesR45IYUadOmTWjUqBHatWuHXr16oV27dmjUqBE2bdpk7NKIiOgRY7ghxdm0aRN69uwJX19fxMfHIycnB/Hx8fD19UXPnj0ZcIiIFI4XziRFKSgoQKNGjeDr64stW7bA1PSf/K7VahEREYFTp04hNTWVH1EREVUjvHAmPbH279+P9PR0jBkzRi/YAICpqSmio6ORlpaG/fv3G6lCIiJ61BhuSFGuXLkCAGjWrFmxywvbC/sREZHyMNyQojg7OwMATp06VezywvbCfkREpDwMN6QooaGhcHd3x9SpU6HVavWWabVaxMbGwsPDA6GhoUaqkIiIHjWGG1IUlUqFWbNmYevWrYiIiNA7WyoiIgJbt27FzJkzOZmYiEjB+CV+pDjdu3fHhg0bMGrUKLRu3VrX7uHhgQ0bNqB79+5GrI6IiB41ngpOisVvKCYiUo7yvH/zyA0plkqlQtu2bY1dBhERPWacc0NERESKwnBDREREisJwQ0RERIrCcENERESKwnBDREREisJwQ0RERIrCcENERESKYvRws2DBAri7u8PCwgJBQUE4cuRIqf1v3ryJoUOHwtnZGWq1Go0bN8Z33333mKolIiKiqs6oX+K3bt06REVFYfHixQgKCsLcuXMRHh6OM2fOoG7dukX65+fno0OHDqhbty42bNiAevXq4cKFC6hZs+bjL56IiIiqJKNefiEoKAitWrVCXFwcgPtXbXZ1dcXw4cMxevToIv0XL16MGTNmIDk5GWZmZhXaJi+/QEREVP2U5/3baB9L5efnIyEhAWFhYf8UY2qKsLAwxMfHF7vO//73PwQHB2Po0KFwdHREs2bNMHXqVBQUFJS4nby8PGRnZ+vdiIiISLmMFm6uX7+OgoICODo66rU7OjoiIyOj2HXOnz+PDRs2oKCgAN999x3GjRuHWbNmYcqUKSVuJzY2FnZ2drqbq6urQfeDiIiIqhajTyguD61Wi7p162LJkiXw9/dHZGQkPvroIyxevLjEdaKjo5GVlaW7Xbp06TFWTERERI+b0SYU29vbQ6VSITMzU689MzMTTk5Oxa7j7OwMMzMzqFQqXZuPjw8yMjKQn58Pc3PzIuuo1Wqo1WrDFk9ERERVltGO3Jibm8Pf3x+7d+/WtWm1WuzevRvBwcHFrhMSEoKzZ89Cq9Xq2lJSUuDs7FxssCEiIqInj1E/loqKisLSpUvx+eefIykpCe+88w5u376N/v37AwD69OmD6OhoXf933nkHf/75J0aMGIGUlBRs27YNU6dOxdChQ421C0RERFTFGPV7biIjI3Ht2jWMHz8eGRkZaNmyJbZv366bZHzx4kWYmv6Tv1xdXbFjxw689957aN68OerVq4cRI0bgww8/NNYuEBERURVj1O+5MQZ+zw0REVH1Uy2+54aIiIjoUWC4ISIiIkVhuCEiIiJFYbghIiIiRWG4ISIiIkVhuCEiIiJFMUi4yc7OxpYtW5CUlGSI4YiIiIgqrELh5pVXXkFcXBwAIDc3FwEBAXjllVfQvHlzbNy40aAFEhEREZVHhcLNvn37EBoaCgDYvHkzRAQ3b97E/PnzMWXKFIMWSERERFQeFQo3WVlZqF27NgBg+/bt6NGjBywtLdG5c2ekpqYatEAiIiKi8qhQuHF1dUV8fDxu376N7du34/nnnwcA/PXXX7CwsDBogURERETlUaELZ44cORKvv/46rK2t0aBBA7Rt2xbA/Y+rfH19DVkfERERUblUKNwMGTIEgYGBuHTpEjp06KC7cnfDhg0554aIiIiMqlJXBc/Pz0daWho8PT1Ro0aFctJjx6uCExERVT+P/Krgd+7cwYABA2BpaYmmTZvi4sWLAIDhw4dj2rRpFRmSiIiIyCAqFG6io6Nx/Phx7N27V28CcVhYGNatW2ew4oiIiIjKq0KfJW3ZsgXr1q3DM888AxMTE11706ZNce7cOYMVR0RERFReFTpyc+3aNdStW7dI++3bt/XCDhEREdHjVqFwExAQgG3btunuFwaaZcuWITg42DCVEREREVVAhT6Wmjp1Kjp16oTTp0/j3r17mDdvHk6fPo1Dhw7hp59+MnSNRERERGVWoSM3zz77LI4fP4579+7B19cXO3fuRN26dREfHw9/f39D10hERERUZuU+cnP37l28/fbbGDduHJYuXfooaiIiIiKqsHIfuTEzM8PGjRsfRS1ERERElVahj6UiIiKwZcsWA5dCREREVHkVmlDs5eWFSZMm4eDBg/D394eVlZXe8nfffdcgxRERERGVV4WuLeXh4VHygCYmOH/+fKWKepR4bSkiIqLqpzzv3xU6cpOWllahwoiIiIgetQrNuXmQiKASFxYnIiIiMqgKh5svvvgCvr6+0Gg00Gg0aN68Ob788ktD1kZERERUbhX6WGr27NkYN24chg0bhpCQEADAgQMHMHjwYFy/fh3vvfeeQYskIiIiKqsKTyiOiYlBnz599No///xzTJw4sUrPyeGEYiIiouqnPO/fFfpY6sqVK2jdunWR9tatW+PKlSsVGZKIiIjIICoUbho1aoT//ve/RdrXrVsHLy+vShdFREREVFEVmnMTExODyMhI7Nu3Tzfn5uDBg9i9e3exoYeIiIjocanQkZsePXrg8OHDsLe3x5YtW7BlyxbY29vjyJEjeOmllwxdIxEREVGZVWhCcXXGCcVERETVzyOfUPzdd99hx44dRdp37NiB77//viJDEhERERlEhcLN6NGjUVBQUKRdRDB69OhKF0VERERUURUKN6mpqWjSpEmRdm9vb5w9e7bSRRERERFVVIXCjZ2dXbFX/j579iysrKwqXRQRERFRRVUo3HTr1g0jR47EuXPndG1nz57FqFGj0LVrV4MVR0RERFReFQo3n3zyCaysrODt7Q0PDw94eHjA29sbderUwcyZMw1dIxEREVGZVehL/Ozs7HDo0CH88MMPOH78ODQaDVq0aIHQ0FBD10dERERULuU6chMfH4+tW7cCAExMTPD888+jbt26mDlzJnr06IG33noLeXl5j6RQIiIiorIoV7iZNGkSfvvtN939kydPYtCgQejQoQNGjx6Nb7/9FrGxsQYvkoiIiKisyhVujh07hueee053/+uvv0ZgYCCWLl2KqKgozJ8/n9eWIiIiIqMqV7j566+/4OjoqLv/008/oVOnTrr7rVq1wqVLlwxXHREREVE5lSvcODo6Ii0tDQCQn5+PxMREPPPMM7rlOTk5MDMzM2yFREREROVQrnDzwgsvYPTo0di/fz+io6NhaWmpd4bUiRMn4OnpafAiiYiIiMqqXKeCT548Gd27d0ebNm1gbW2Nzz//HObm5rrlK1aswPPPP2/wIomIiIjKykREpLwrZWVlwdraGiqVSq/9zz//hLW1tV7gqWrKc8l0IiIiqhrK8/5d4S/xK07t2rUrMhwRERGRwVTo8gtEREREVRXDDRERESkKww0REREpCsMNERERKUqVCDcLFiyAu7s7LCwsEBQUhCNHjpRpva+//homJiaIiIh4tAUSERFRtWH0cLNu3TpERUVhwoQJSExMRIsWLRAeHo6rV6+Wul56ejref/99vS8RJCIiIjJ6uJk9ezYGDRqE/v37o0mTJli8eDEsLS2xYsWKEtcpKCjA66+/jpiYGDRs2PAxVktERERVnVHDTX5+PhISEhAWFqZrMzU1RVhYGOLj40tcb9KkSahbty4GDBjw0G3k5eUhOztb70ZERETKZdRwc/36dRQUFOhdaRy4f4HOjIyMYtc5cOAAli9fjqVLl5ZpG7GxsbCzs9PdXF1dK103ERERVV1G/1iqPHJycvDGG29g6dKlsLe3L9M60dHRyMrK0t0uXbr0iKskIiIiY6rQ5RcMxd7eHiqVCpmZmXrtmZmZcHJyKtL/3LlzSE9PR5cuXXRtWq0WAFCjRg2cOXOmyFXJ1Wo11Gr1I6iejO3OnTtITk4utU9ubi7S09Ph7u4OjUbz0DG9vb1haWlpqBKJiMgIjBpuzM3N4e/vj927d+tO59Zqtdi9ezeGDRtWpL+3tzdOnjyp1zZ27Fjk5ORg3rx5/MjpCZOcnAx/f3+DjpmQkAA/Pz+DjklERI+XUcMNAERFRaFv374ICAhAYGAg5s6di9u3b6N///4AgD59+qBevXqIjY2FhYUFmjVrprd+zZo1AaBIOymft7c3EhISSu2TlJSE3r17Y/Xq1fDx8SnTmEREVL0ZPdxERkbi2rVrGD9+PDIyMtCyZUts375dN8n44sWLMDWtVlOD6DGxtLQs81EWHx8fHpEhInpCmIiIGLuIxyk7Oxt2dnbIysqCra2tscuhRywxMRH+/v78uImIqJorz/s3D4kQERGRojDcEBERkaIw3BAREZGiMNwQERGRojDcEBERkaIw3BAREZGiMNwQERGRojDcEBERkaIw3BAREZGiMNwQERGRojDcEBERkaIw3BAREZGiMNwQERGRojDcEBERkaIw3BAREZGiMNwQERGRojDcEBERkaIw3BAREZGiMNwQERGRotQwdgFERETlVVBQgP379+PKlStwdnZGaGgoVCqVscuiKoJHboiIqFrZtGkTGjVqhHbt2qFXr15o164dGjVqhE2bNhm7NKoiGG6IiKja2LRpE3r27AlfX1/Ex8cjJycH8fHx8PX1Rc+ePRlwCABgIiJi7CIep+zsbNjZ2SErKwu2trbGLocescTERPj7+yMhIQF+fn7GLoeIKqGgoACNGjWCr68vtmzZAlPTf/4+12q1iIiIwKlTp5CamsqPqBSoPO/fPHJDRETVwv79+5Geno4xY8boBRsAMDU1RXR0NNLS0rB//34jVUhVBcMNERFVC1euXAEANGvWrNjlhe2F/ejJxXBDRETVgrOzMwDg1KlTxS4vbC/sR08uhhsiIqoWQkND4e7ujqlTp0Kr1eot02q1iI2NhYeHB0JDQ41UIVUVDDdERFQtqFQqzJo1C1u3bkVERITe2VIRERHYunUrZs6cycnExC/xIyKi6qN79+7YsGEDRo0ahdatW+vaPTw8sGHDBnTv3t2I1VFVwXBDRETVSvfu3dGtWzd+QzGViOGGiIiqHZVKhbZt2xq7DKqiOOeGiIiIFIXhhoiIiBSF4YaIiIgUheGGiIiIFIXhhoiIiBSF4YaIiIgUheGGiIiIFIXhhoiIiBSF4YaIiIgUheGGiIiIFIXhhoiIiBSF4YaIiIgUheGGiIiIFIXhhoiIiBSF4YaIiIgUheGGiIiIFIXhhoiIiBSF4YaIiIgUheGGiIiIFIXhhoiIiBSF4YaIiIgUheGGiIiIFIXhhoiIiBSF4YaIiIgUheGGiIiIFKVKhJsFCxbA3d0dFhYWCAoKwpEjR0rsu3TpUoSGhqJWrVqoVasWwsLCSu1PRERETxajh5t169YhKioKEyZMQGJiIlq0aIHw8HBcvXq12P579+7Fa6+9hj179iA+Ph6urq54/vnncfny5cdcOREREVVFJiIixiwgKCgIrVq1QlxcHABAq9XC1dUVw4cPx+jRox+6fkFBAWrVqoW4uDj06dPnof2zs7NhZ2eHrKws2NraVrp+qtoSExPh7++PhIQE+Pn5GbscIiKqoPK8fxv1yE1+fj4SEhIQFhamazM1NUVYWBji4+PLNMadO3dw9+5d1K5d+1GVSURERNVIDWNu/Pr16ygoKICjo6Neu6OjI5KTk8s0xocffggXFxe9gPSgvLw85OXl6e5nZ2dXvGAiIiKq8ow+56Yypk2bhq+//hqbN2+GhYVFsX1iY2NhZ2enu7m6uj7mKomIiOhxMmq4sbe3h0qlQmZmpl57ZmYmnJycSl135syZmDZtGnbu3InmzZuX2C86OhpZWVm626VLlwxSOxEREVVNRg035ubm8Pf3x+7du3VtWq0Wu3fvRnBwcInrffLJJ5g8eTK2b9+OgICAUrehVqtha2urdyMiIiLlMuqcGwCIiopC3759ERAQgMDAQMydOxe3b99G//79AQB9+vRBvXr1EBsbCwCYPn06xo8fj7Vr18Ld3R0ZGRkAAGtra1hbWxttP4iIiKhqMHq4iYyMxLVr1zB+/HhkZGSgZcuW2L59u26S8cWLF2Fq+s8BpkWLFiE/Px89e/bUG2fChAmYOHHi4yydHrHU1FTk5ORUaoykpCS9fyvLxsYGXl5eBhmLiIgeDaN/z83jxu+5qR5SU1PRuHFjY5dRrJSUFAYcIqLHrDzv30Y/ckNUnMIjNqtXr4aPj0+Fx8nNzUV6ejrc3d2h0WgqVVNSUhJ69+5d6aNJRET0aDHcUJXm4+NT6W8WDgkJMVA1RERUHVTr77khIiIi+jeGGyIiIlIUhhsiIiJSFIYbIiIiUhSGGyIiIlIUhhsiIiJSFIYbIiIiUhSGGyIiIlIUhhsiIiJSFIYbIiIiUhSGGyIiIlIUhhsiIiJSFIYbIiIiUhSGGyIiIlIUhhsiIiJSFIYbIiIiUhSGGyIiIlIUhhsiIiJSFIYbIiIiUhSGGyIiIlIUhhsiIiJSFIYbIiIiUhSGGyIiIlIUhhsiIiJSFIYbIiIiUhSGGyIiIlIUhhsiIiJSFIYbIiIiUhSGGyIiIlIUhhsiIiJSFIYbIiIiUhSGGyIiIlIUhhsiIiJSFIYbIiIiUhSGGyIiIlKUGsYugKgkTtYm0NxMAf6oGhlcczMFTtYmxi6DiIgeguGGqqy3/c3hs+9tYJ+xK7nPB/drIiKiqo3hhqqszxLyETl+FXy8vY1dCgAgKTkZn83qha7GLoSIiErFcENVVsYtQW7NxoBLS2OXAgDIzdAi45YYuwwiInqIqjGZgYiIiMhAGG6IiIhIURhuiIiISFEYboiIiEhRGG6IiIhIURhuiIiISFEYboiIiEhRGG6IiIhIURhuiIiISFEYboiIiEhRGG6IiIhIURhuiIiISFF44Uyqku7cuQMASExMrNQ4ubm5SE9Ph7u7OzQaTaXGSkpKqtT6RET0eDDcUJWUnJwMABg0aJCRKynKxsbG2CUQEVEpGG6oSoqIiAAAeHt7w9LSssLjJCUloXfv3li9ejV8fHwqXZeNjQ28vLwqPQ4RET06DDdUJdnb22PgwIEGG8/Hxwd+fn4GG4+IiKquKjGheMGCBXB3d4eFhQWCgoJw5MiRUvuvX78e3t7esLCwgK+vL7777rvHVCkRERFVdUYPN+vWrUNUVBQmTJiAxMREtGjRAuHh4bh69Wqx/Q8dOoTXXnsNAwYMwNGjRxEREYGIiAicOnXqMVdOREREVZGJiIgxCwgKCkKrVq0QFxcHANBqtXB1dcXw4cMxevToIv0jIyNx+/ZtbN26Vdf2zDPPoGXLlli8ePFDt5ednQ07OztkZWXB1tbWcDtCVVJiYiL8/f2RkJDAj6WIiKqx8rx/G3XOTX5+PhISEhAdHa1rMzU1RVhYGOLj44tdJz4+HlFRUXpt4eHh2LJlS7H98/LykJeXp7ufnZ1d+cKpSrhz547urKqSFJ6+XdbTuCs7gZmIiIzPqOHm+vXrKCgogKOjo167o6NjiW9aGRkZxfbPyMgotn9sbCxiYmIMUzBVKcnJyfD39y9T3969e5epH4/wEBFVf4o/Wyo6OlrvSE92djZcXV2NWBEZire3NxISEkrtU94v8fP29jZUeUREZCRGDTf29vZQqVTIzMzUa8/MzISTk1Ox6zg5OZWrv1qthlqtNkzBVKVYWlqW6ShLSEjIY6iGiIiqCqOeLWVubg5/f3/s3r1b16bVarF7924EBwcXu05wcLBefwD44YcfSuxPRERETxajfywVFRWFvn37IiAgAIGBgZg7dy5u376N/v37AwD69OmDevXqITY2FgAwYsQItGnTBrNmzULnzp3x9ddf49dff8WSJUuMuRtERERURRg93ERGRuLatWsYP348MjIy0LJlS2zfvl03afjixYswNf3nAFPr1q2xdu1ajB07FmPGjIGXlxe2bNmCZs2aGWsXiIiIqAox+vfcPG78nhsiIqLqpzzv30b/hmIiIiIiQ2K4ISIiIkVhuCEiIiJFYbghIiIiRWG4ISIiIkVhuCEiIiJFYbghIiIiRWG4ISIiIkVhuCEiIiJFMfrlFx63wi9kzs7ONnIlREREVFaF79tlubDCExducnJyAACurq5GroSIiIjKKycnB3Z2dqX2eeKuLaXVavHHH3/AxsYGJiYmxi6HHrHs7Gy4urri0qVLvJYYkcLw9f1kERHk5OTAxcVF74LaxXnijtyYmpqifv36xi6DHjNbW1v+8iNSKL6+nxwPO2JTiBOKiYiISFEYboiIiEhRGG5I0dRqNSZMmAC1Wm3sUojIwPj6ppI8cROKiYiISNl45IaIiIgUheGGiIiIFIXhhoiIiBSF4YaqrYkTJ6Jly5bGLoOIiKoYhhuqsH79+iEiIsLYZZRq8+bNeOaZZ2BnZwcbGxs0bdoUI0eONHZZRFXGpUuX8Oabb8LFxQXm5uZwc3PDiBEjcOPGDaPWFR8fD5VKhc6dOxdZlp6eDhMTExw7dqzE9dPS0tCrVy+4uLjAwsIC9evXR7du3ZCcnPwIq6aqguGGFGv37t2IjIxEjx49cOTIESQkJODjjz/G3bt3H9k2CwoKoNVqH9n4RIZ0/vx5BAQEIDU1FV999RXOnj2LxYsXY/fu3QgODsaff/5ptNqWL1+O4cOHY9++ffjjjz/Kte7du3fRoUMHZGVlYdOmTThz5gzWrVsHX19f3Lx589EU/P+3S1WEEFVQ3759pVu3biUu37t3r7Rq1UrMzc3FyclJPvzwQ7l7965ueUFBgUyfPl08PT3F3NxcXF1dZcqUKbrl//nPf8TLy0s0Go14eHjI2LFjJT8/X7d8woQJ0qJFixK3P2LECGnbtu1D9+N///ufBAQEiFqtljp16khERIRu2Z9//ilvvPGG1KxZUzQajXTs2FFSUlJ0y1euXCl2dnbyzTffiI+Pj6hUKklLS5O///5bRo0aJS4uLmJpaSmBgYGyZ88e3Xrp6eny4osvSs2aNcXS0lKaNGki27Zte2itRIbUsWNHqV+/vty5c0ev/cqVK2JpaSmDBw/Wtbm5ucmkSZPk1VdfFUtLS3FxcZG4uDi99f766y8ZMGCA2Nvbi42NjbRr106OHTumW174mv3iiy/Ezc1NbG1tJTIyUrKzs/XGycnJEWtra0lOTpbIyEj5+OOP9ZanpaUJADl69Gix+3X06FEBIOnp6aXu/6VLl+TVV1+VWrVqiaWlpfj7+8vPP/+sW75w4UJp2LChmJmZSePGjeWLL77QWx+ALFy4ULp06SKWlpYyYcIEERHZsmWLPP3006JWq8XDw0MmTpyo+92n1WplwoQJ4urqKubm5uLs7CzDhw8vtU4qP4YbqrDSws3vv/8ulpaWMmTIEElKSpLNmzeLvb297sUvcj+81KpVS1atWiVnz56V/fv3y9KlS3XLJ0+eLAcPHpS0tDT53//+J46OjjJ9+nTd8oeFm9jYWHFwcJCTJ0+W2Gfr1q2iUqlk/Pjxcvr0aTl27JhMnTpVt7xr167i4+Mj+/btk2PHjkl4eLg0atRIF7JWrlwpZmZm0rp1azl48KAkJyfL7du3ZeDAgdK6dWvZt2+fnD17VmbMmCFqtVoXjDp37iwdOnSQEydOyLlz5+Tbb7+Vn376qbSHm8igbty4ISYmJnrP9wcNGjRIatWqJVqtVkTuhxsbGxuJjY2VM2fOyPz580WlUsnOnTt164SFhUmXLl3kl19+kZSUFBk1apTUqVNHbty4ISL3X7PW1tbSvXt3OXnypOzbt0+cnJxkzJgxettevny5BAQEiIjIt99+K56enro6RB4ebn7//XcxNTWVmTNnyr1794rtk5OTIw0bNpTQ0FDZv3+/pKamyrp16+TQoUMiIrJp0yYxMzOTBQsWyJkzZ2TWrFmiUqnkxx9/1I0BQOrWrSsrVqyQc+fOyYULF2Tfvn1ia2srq1atknPnzsnOnTvF3d1dJk6cKCIi69evF1tbW/nuu+/kwoULcvjwYVmyZEmJPyeqGIYbqrDSws2YMWPkqaee0vuFtGDBArG2tpaCggLJzs4WtVqtF2YeZsaMGeLv76+7/7Bwc+vWLXnhhRcEgLi5uUlkZKQsX75c/v77b12f4OBgef3114tdPyUlRQDIwYMHdW3Xr18XjUYj//3vf0XkfrgBoPfX6YULF0SlUsnly5f1xnvuueckOjpaRER8fX11v+yIjOHnn38WALJ58+Zil8+ePVsASGZmpojcDzcdO3bU6xMZGSmdOnUSEZH9+/eLra2t3utLRMTT01M+++wzEbn/mrW0tNQ7UvPBBx9IUFCQ3jqtW7eWuXPniojI3bt3xd7eXu/I58PCjYhIXFycWFpa6o4gTZo0Sc6dO6db/tlnn4mNjY0ueP1b69atZdCgQXptL7/8srzwwgu6+wBk5MiRen2ee+65IoHxyy+/FGdnZxERmTVrljRu3FjvKDQZHufc0CORlJSE4OBgmJiY6NpCQkJw69Yt/P7770hKSkJeXh6ee+65EsdYt24dQkJC4OTkBGtra4wdOxYXL14scw1WVlbYtm0bzp49i7Fjx8La2hqjRo1CYGAg7ty5AwA4duxYiTUkJSWhRo0aCAoK0rXVqVMHTz31FJKSknRt5ubmaN68ue7+yZMnUVBQgMaNG8Pa2lp3++mnn3Du3DkAwLvvvospU6YgJCQEEyZMwIkTJ8q8X0SGJOX4kvrg4OAi9wtfC8ePH8etW7dQp04dved9Wlqa7nkPAO7u7rCxsdHdd3Z2xtWrV3X3z5w5gyNHjuC1114DANSoUQORkZFYvnx5ufZr6NChyMjIwJo1axAcHIz169ejadOm+OGHHwDcf+0//fTTqF27drHrJyUlISQkRK8tJCRE77UPAAEBAXr3jx8/jkmTJuk9BoMGDcKVK1dw584dvPzyy8jNzUXDhg0xaNAgbN68Gffu3SvXvtHD1TB2AfRk0mg0pS6Pj4/H66+/jpiYGISHh8POzg5ff/01Zs2aVe5teXp6wtPTEwMHDsRHH32Exo0bY926dejfv/9D6ygLjUajF+Ju3boFlUqFhIQEqFQqvb7W1tYAgIEDByI8PBzbtm3Dzp07ERsbi1mzZmH48OGVroeoLBo1agQTExMkJSXhpZdeKrI8KSkJtWrVgoODQ5nGu3XrFpydnbF3794iy2rWrKn7v5mZmd4yExMTvUn4y5cvx7179+Di4qJrExGo1WrExcXBzs6uTPUAgI2NDbp06YIuXbpgypQpCA8Px5QpU9ChQweDvPaB+39EPejWrVuIiYlB9+7di/S1sLCAq6srzpw5g127duGHH37AkCFDMGPGDPz0009FHhuqOB65oUfCx8cH8fHxen8VHjx4EDY2Nqhfvz68vLyg0Wiwe/fuYtc/dOgQ3Nzc8NFHHyEgIABeXl64cOFCpetyd3eHpaUlbt++DQBo3rx5iTX4+Pjg3r17OHz4sK7txo0bOHPmDJo0aVLiNp5++mkUFBTg6tWraNSokd7NyclJ18/V1RWDBw/Gpk2bMGrUKCxdurTS+0dUVnXq1EGHDh2wcOFC5Obm6i0rPOIRGRmpF9x//vlnvX4///wzfHx8AAB+fn7IyMhAjRo1ijzv7e3ty1TTvXv38MUXX2DWrFk4duyY7nb8+HG4uLjgq6++qvD+mpiYwNvbW++1f+zYsRLPCPPx8cHBgwf12g4ePFjqax+4/zicOXOmyGPQqFEjmJref8vVaDTo0qUL5s+fj7179yI+Ph4nT56s8L5RMYz8sRhVY3379pW2bdvK0aNH9W4XL17UTSgeOnSoJCUlyZYtW4pMKJ44caLUqlVLPv/8czl79qzEx8fLsmXLRETkm2++kRo1ashXX30lZ8+elXnz5knt2rXFzs5Ot/7D5txMmDBBPvjgA9mzZ4+cP39eEhMTpV+/fqLRaCQ5OVlERPbs2SOmpqa6CcUnTpyQadOm6cbo1q2bNGnSRPbv3y/Hjh2Tjh07FplQ/GBNhV5//XVxd3eXjRs3yvnz5+Xw4cMydepU2bp1q4jcP5Nr+/btcv78eUlISJCgoCB55ZVXKviTIKqYlJQUsbe3l9DQUPnpp5/k4sWL8v3330uzZs3Ey8tLbz5K4dlN06dPlzNnzkhcXJyoVCrZvn27iNw/C+jZZ5+VFi1ayI4dOyQtLU0OHjwoY8aMkV9++UVEin/NzpkzR9zc3EREZPPmzWJubi43b94sUut//vMf3STjspwt1bVrV1m/fr389ttvkpqaKsuWLRMrKyuZNGmSiIjk5eVJ48aNJTQ0VA4cOCDnzp2TDRs26CYUb968WczMzGThwoWSkpKim1D84NwfFDNnafv27VKjRg2ZOHGinDp1Sk6fPi1fffWVfPTRRyJy/3fGsmXL5OTJk3Lu3DkZO3asaDQauX79+sN/YFRmDDdUYX379hUARW4DBgwQkbKdCj5lyhRxc3MTMzMzadCggd5EvA8++EDq1Kkj1tbWEhkZKXPmzClXuPnxxx+lR48eulMuHR0dpWPHjrJ//369fhs3bpSWLVuKubm52NvbS/fu3XXLCk8Ft7OzE41GI+Hh4cWeCv5v+fn5Mn78eHF3dxczMzNxdnaWl156SU6cOCEiIsOGDRNPT09Rq9Xi4OAgb7zxBn+5kVGkp6dL3759xdHRUczMzMTV1VWGDx9e5Pno5uYmMTEx8vLLL4ulpaU4OTnJvHnz9PpkZ2fL8OHDxcXFRTfW66+/LhcvXhSRh4ebF198UW/C7oMOHz4sAOT48eMPDTfXrl2Td999V5o1aybW1tZiY2Mjvr6+MnPmTCkoKNDb9x49eoitra1YWlpKQECAHD58WLe8LKeCFzche/v27dK6dWvRaDRia2srgYGBujOiNm/eLEFBQWJraytWVlbyzDPPyK5du4rdD6o4E5FyzCYjIqInkru7O0aOHMlv+KZqgXNuiIiISFEYboiIiEhR+LEUERERKQqP3BAREZGiMNwQERGRojDcEBERkaIw3BAREZGiMNwQERGRojDcEFGlXLt2De+88w4aNGgAtVoNJycnhIeHF7kuj7G1bdsWJiYmJd7atm1r7BKJyEB4VXAiqpQePXogPz8fn3/+ORo2bIjMzEzs3r0bN27ceGTbzM/Ph7m5ebnW2bRpE/Lz8wEAly5dQmBgIHbt2oWmTZsCQLnHI6Kqi0duiKjCbt68if3792P69Olo164d3NzcEBgYiOjoaHTt2lWv39tvvw1HR0dYWFigWbNm2Lp1q275xo0b0bRpU6jVari7u2PWrFl623F3d8fkyZPRp08f2Nra4q233gIAHDhwAKGhodBoNHB1dcW7776ru+rzv9WuXRtOTk5wcnKCg4MDgPtXxnZyckKvXr0wfvx4vf7Xrl2Dubm57qrxhTW89tprsLKyQr169bBgwYIij8fAgQPh4OAAW1tbtG/fHsePH6/go0tEFcVwQ0QVZm1tDWtra2zZsgV5eXnF9tFqtejUqRMOHjyI1atX4/Tp05g2bRpUKhUAICEhAa+88gpeffVVnDx5EhMnTsS4ceOwatUqvXFmzpyJFi1a4OjRoxg3bhzOnTuHjh07okePHjhx4gTWrVuHAwcOYNiwYeXej4EDB2Lt2rV6+7B69WrUq1cP7du317XNmDFDV8Po0aMxYsQI/PDDD7rlL7/8Mq5evYrvv/8eCQkJ8PPzw3PPPYc///yz3DURUSUY97qdRFTdbdiwQWrVqiUWFhbSunVriY6OluPHj+uW79ixQ0xNTeXMmTPFrt+rVy/p0KGDXtsHH3wgTZo00d13c3OTiIgIvT4DBgyQt956S69t//79YmpqKrm5uaXW/O+rSufm5kqtWrVk3bp1uj7NmzeXiRMn6tXQsWNHvXEiIyOlU6dOum3b2trK33//rdfH09NTPvvss1LrISLD4pEbIqqUHj164I8//sD//vc/dOzYEXv37oWfn5/uyMuxY8dQv359NG7cuNj1k5KSEBISotcWEhKC1NRUFBQU6NoCAgL0+hw/fhyrVq3SHT2ytrZGeHg4tFot0tLSyrUPFhYWeOONN7BixQoAQGJiIk6dOoV+/frp9QsODi5yPykpSVfPrVu3UKdOHb2a0tLScO7cuXLVQ0SVwwnFRFRpFhYW6NChAzp06IBx48Zh4MCBmDBhAvr16weNRmOQbVhZWendv3XrFt5++228++67Rfo2aNCg3OMPHDgQLVu2xO+//46VK1eiffv2cHNzK/P6t27dgrOzM/bu3VtkWc2aNctdDxFVHMMNERlckyZNsGXLFgBA8+bN8fvvvyMlJaXYozc+Pj5FThs/ePAgGjdurJuXUxw/Pz+cPn0ajRo1MkjNvr6+CAgIwNKlS7F27VrExcUV6fPzzz8Xue/j46OrJyMjAzVq1IC7u7tBaiKiiuHHUkRUYTdu3ED79u2xevVqnDhxAmlpaVi/fj0++eQTdOvWDQDQpk0b/N///R969OiBH374AWlpafj++++xfft2AMCoUaOwe/duTJ48GSkpKfj8888RFxeH999/v9Rtf/jhhzh06BCGDRuGY8eOITU1Fd98802FJhQXGjhwIKZNmwYRwUsvvVRk+cGDB/HJJ58gJSUFCxYswPr16zFixAgAQFhYGIKDgxEREYGdO3ciPT0dhw4dwkcffYRff/21wjURUQUYe9IPEVVff//9t4wePVr8/PzEzs5OLC0t5amnnpKxY8fKnTt3dP1u3Lgh/fv3lzp16oiFhYU0a9ZMtm7dqlu+YcMGadKkiZiZmUmDBg1kxowZettxc3OTOXPmFNn+kSNHpEOHDmJtbS1WVlbSvHlz+fjjjx9a978nFBfKyckRS0tLGTJkSJF13NzcJCYmRl5++WWxtLQUJycnmTdvnl6f7OxsGT58uLi4uIiZmZm4urrK66+/LhcvXnxoTURkOCYiIsYOWEREVUF6ejo8PT3xyy+/wM/PT2+Zu7s7Ro4ciZEjRxqnOCIqM865IaIn3t27d3Hjxg2MHTsWzzzzTJFgQ0TVC+fcENET7+DBg3B2dsYvv/yCxYsXG7scIqokfixFREREisIjN0RERKQoDDdERESkKAw3REREpCgMN0RERKQoDDdERESkKAw3REREpCgMN0RERKQoDDdERESkKAw3REREpCj/D3TEbknSjKFlAAAAAElFTkSuQmCC",
      "text/plain": [
       "<Figure size 640x480 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import matplotlib.pyplot as plt\n",
    "# Combine the data into a list\n",
    "data = [scores_local, scores_openai]\n",
    "\n",
    "# Create a box plot\n",
    "plt.boxplot(data, labels=['Local Scores', 'OpenAI Scores'])\n",
    "\n",
    "# Add a title and labels\n",
    "plt.title('Comparison of Local and OpenAI Scores')\n",
    "plt.xlabel('Score Type')\n",
    "plt.ylabel('Scores')\n",
    "\n",
    "# Display the plot\n",
    "plt.show()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "import html\n",
    "import difflib\n",
    "\n",
    "    \n",
    "def text_diff(expected, actual):\n",
    "    # First unescape both texts\n",
    "    expected = html.unescape(expected)\n",
    "    actual = html.unescape(actual)\n",
    "    \n",
    "    matcher = difflib.SequenceMatcher(None, expected, actual)\n",
    "    output = []\n",
    "    \n",
    "    for tag, i1, i2, j1, j2 in matcher.get_opcodes():\n",
    "        if tag == 'equal':\n",
    "            output.append(expected[i1:i2])\n",
    "        elif tag == 'delete':\n",
    "            output.append('\\033[91m' + expected[i1:i2] + '\\033[0m')\n",
    "        elif tag == 'insert':\n",
    "            output.append('\\033[92m' + actual[j1:j2] + '\\033[0m')\n",
    "        elif tag == 'replace':\n",
    "            output.append('\\033[91m' + expected[i1:i2] + '\\033[0m')\n",
    "            output.append('\\033[92m' + actual[j1:j2] + '\\033[0m')\n",
    "            \n",
    "    return ''.join(output)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "********EMBOJ-2023-114195.docx:********\n",
      "gpt-3.5-turbo: \n",
      "The datasets produced in this study are available in the following databases:\n",
      "\n",
      "All data: BioStudies S-BIAD928: https://www.ebi.ac.uk/biostudies/studies?query=S-BIAD928 \n",
      "\n",
      "\n",
      "Comparison with ground truth:  \u001b[92m\n",
      "\u001b[0mThe datasets produced in this study are available in the following databases:\u001b[91m \u001b[0m\u001b[92m\n",
      "\n",
      "\u001b[0mAll data: BioStudies S-BIAD928: https://www.ebi.ac.uk/biostudies/studies?query=S-BIAD928\n",
      "********EMBOJ-2023-114687.docx:********\n",
      "gpt-3.5-turbo: \n",
      "This study includes no data deposited in external repositories. \n",
      "\n",
      "\n",
      "Comparison with ground truth:  \u001b[92m\n",
      "\u001b[0mThis study includes no data deposited in external repositories.\n",
      "********EMBOJ-2023-115257.docx:********\n",
      "gpt-3.5-turbo: \n",
      "The RNA-seq transcriptomics data on human bone biopsies have been deposited to BioProject with the dataset identifier PRJNA764663 (https://www.ncbi.nlm.nih.gov/bioproject/PRJNA764663). The RNA-seq transcriptomics data on transfected osteoblasts have been deposited to GEO with the dataset identifier GSE201774 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201774) (enter token: ififogaehzoxlgh).\n",
      "\n",
      "The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD051195. Reviewer account details are username: reviewer_pxd051195@ebi.ac.uk ; password: R6v0mIYe \n",
      "\n",
      "\n",
      "Comparison with ground truth:  \u001b[92m\n",
      "\u001b[0mThe RNA-seq transcriptomics data on human bone biopsies have been deposited to BioProject with the dataset identifier PRJNA764663 (https://www.ncbi.nlm.nih.gov/bioproject/PRJNA764663). The RNA-seq transcriptomics data on transfected osteoblasts have been deposited to GEO with the dataset identifier GSE201774 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201774) (enter token: ififogaehzoxlgh).\u001b[91m \u001b[0m\u001b[92m\n",
      "\n",
      "\u001b[0mThe mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD051195. Reviewer account details are username: reviewer_pxd051195@ebi.ac.uk ; password: R6v0mIYe\n",
      "********EMBOJ-2023-115537.docx:********\n",
      "gpt-3.5-turbo: `This study includes no data deposited in external repositories.` \n",
      "\n",
      "\n",
      "Comparison with ground truth:  \u001b[92m`\u001b[0mThis study includes no data deposited in external repositories.\u001b[92m`\u001b[0m\n",
      "********EMBOR-2023-58706-T.docx:********\n",
      "gpt-3.5-turbo: \n",
      "This study includes no data deposited in external repositories. \n",
      "\n",
      "\n",
      "Comparison with ground truth:  \u001b[91mN\u001b[0m\u001b[92m\n",
      "This study includes n\u001b[0mo data\u001b[91m have been\u001b[0m deposited \u001b[92min ex\u001b[0mt\u001b[91mo pub\u001b[0m\u001b[92merna\u001b[0ml\u001b[91mic\u001b[0m repositories.\n",
      "********EMBOR-2024-58727V1.docx:********\n",
      "gpt-3.5-turbo: \n",
      "The sequencing data are available in GEO Data Sets with the accession number GSE270718 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE270718), GSE270719 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE270719) and GSE275838 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE275838). All other raw data support the findings of this study are available from the corresponding author upon reasonable request. \n",
      "\n",
      "\n",
      "Comparison with ground truth:  \u001b[92m\n",
      "\u001b[0mThe sequencing data are available in GEO Data Sets with the accession number GSE270718 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE270718), GSE270719 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE270719) and GSE275838 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE275838). All other raw data support the findings of this study are available from the corresponding author upon reasonable request.\n",
      "********EMBOR-2024-59101V1-T.docx:********\n",
      "gpt-3.5-turbo: \n",
      "Data Availability: This study includes no data deposited in external repositories. \n",
      "\n",
      "\n",
      "Comparison with ground truth:  \u001b[92m\n",
      "Data Availability: \u001b[0mThis study includes no data deposited in external repositories.\n",
      "********EMM-2023-18636.docx:********\n",
      "gpt-3.5-turbo: Data Availability: Dataref: (Cerami et al., 2012; de Bruijn et al., 2023; Gao et al., 2013) (Appendix Figure S2C). \n",
      "\n",
      "\n",
      "Comparison with ground truth:  \u001b[91mThe gene expression d\u001b[0m\u001b[92mD\u001b[0mata \u001b[91mfrom th\u001b[0m\u001b[92mAva\u001b[0mi\u001b[91ms pu\u001b[0m\u001b[92mla\u001b[0mb\u001b[92mi\u001b[0mli\u001b[91mc\u001b[0m\u001b[92mty: D\u001b[0mat\u001b[92maref: (Ceram\u001b[0mi\u001b[91mon\u001b[0m \u001b[91mh\u001b[0m\u001b[92met \u001b[0ma\u001b[91mve\u001b[0m\u001b[92ml.,\u001b[0m \u001b[91mbeen\u001b[0m\u001b[92m2012;\u001b[0m de\u001b[91mpos\u001b[0m\u001b[92m Bru\u001b[0mi\u001b[92mjn e\u001b[0mt\u001b[91med\u001b[0m \u001b[91mt\u001b[0m\u001b[92mal., 2023; Ga\u001b[0mo \u001b[92me\u001b[0mt\u001b[91mh\u001b[0m\u001b[92m al., 2013) (Appendix Figur\u001b[0me \u001b[91mGEO database (https://www.ncbi.nlm.nih.gov/geo/) and assigned the identifier G\u001b[0mS\u001b[91mE\u001b[0m2\u001b[91m31526 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE231526\u001b[0m\u001b[92mC\u001b[0m).\u001b[91m \u001b[0m\n",
      "********EMM-2023-19044.docx:********\n",
      "gpt-3.5-turbo: `This study includes no data deposited in external repositories.` \n",
      "\n",
      "\n",
      "Comparison with ground truth:  \u001b[92m`\u001b[0mThis study includes no data deposited in external repositories.\u001b[92m`\u001b[0m\n",
      "********MSB-2023-12087.docx:********\n",
      "gpt-3.5-turbo: \n",
      "This study includes no data deposited in external repositories. \n",
      "\n",
      "\n",
      "Comparison with ground truth:  \u001b[92m\n",
      "\u001b[0mThis study includes no data deposited in external repositories.\n"
     ]
    }
   ],
   "source": [
    "for result, paper in zip(results, DOCX_FILES):\n",
    "    print(f\"********{paper.split('/')[-1]}:********\")\n",
    "    print(\"gpt-3.5-turbo:\", result[\"openai\"], \"\\n\\n\")\n",
    "    print(\"Comparison with ground truth: \", text_diff(result[\"truth\"], result[\"openai\"]))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".venv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
